<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/5/607" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/5/607.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/5/607.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/5/607.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/5/607.atom" />
<meta name="HW.pisa" content="ggg;2/5/607" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Comparative Genetic Mapping and Discovery of Linkage Disequilibrium Across Linkage Groups in White Clover (Trifolium repens L.)" />
<meta name="DC.Identifier" content="10.1534/g3.112.002600" />
<meta name="DC.Date" content="2012-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Isobe et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="White clover ( Trifolium repens L.) is an allotetraploid species (2n = 4X = 32) that is widely distributed in temperate regions and cultivated as a forage legume. In this study, we developed expressed sequence tag (EST)–derived simple sequence repeat (SSR) markers, constructed linkage maps, and performed comparative mapping with other legume species. A total of 7982 ESTs that could be assembled into 5400 contigs and 2582 singletons were generated. Using the EST sequences that were obtained, 1973 primer pairs to amplify EST-derived SSR markers were designed and used for linkage analysis of 188 F1 progenies, which were generated by a cross between two Japanese plants, ‘273-7’ and ‘T17-349,’ with previously published SSR markers. An integrated linkage map was constructed by combining parental-specific maps, which consisted of 1743 SSR loci on 16 homeologous linkage groups with a total length of 2511 cM. The primer sequences of the developed EST-SSR markers and their map positions are available on &lt;http://clovergarden.jp/&gt;. Linkage disequilibrium (LD) was observed on 9 of 16 linkage groups of a parental-specific map. The genome structures were compared among white clover, red clover ( T. pratense L.), Medicago truncatula , and Lotus japonicus . Macrosynteny was observed across the four legume species. Surprisingly, the comparative genome structure between white clover and M. truncatula had a higher degree of conservation than that of the two clover species." />
<meta name="DC.Contributor" content="Sachiko N. Isobe" />
<meta name="DC.Contributor" content="Hiroshi Hisano" />
<meta name="DC.Contributor" content="Shusei Sato" />
<meta name="DC.Contributor" content="Hideki Hirakawa" />
<meta name="DC.Contributor" content="Kenji Okumura" />
<meta name="DC.Contributor" content="Kenta Shirasawa" />
<meta name="DC.Contributor" content="Shigemi Sasamoto" />
<meta name="DC.Contributor" content="Akiko Watanabe" />
<meta name="DC.Contributor" content="Tsuyuko Wada" />
<meta name="DC.Contributor" content="Yoshie Kishida" />
<meta name="DC.Contributor" content="Hisano Tsuruoka" />
<meta name="DC.Contributor" content="Tsunakazu Fujishiro" />
<meta name="DC.Contributor" content="Manabu Yamada" />
<meta name="DC.Contributor" content="Mistuyo Kohara" />
<meta name="DC.Contributor" content="Satoshi Tabata" />
<meta name="article:published_time" content="2012-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Comparative Genetic Mapping and Discovery of Linkage Disequilibrium Across Linkage Groups in White Clover (Trifolium repens L.)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;White clover (&lt;i&gt;Trifolium repens&lt;/i&gt; L.) is an allotetraploid species (2n = 4X = 32) that is widely distributed in temperate regions and cultivated as a forage legume. In this study, we developed expressed sequence tag (EST)–derived simple sequence repeat (SSR) markers, constructed linkage maps, and performed comparative mapping with other legume species. A total of 7982 ESTs that could be assembled into 5400 contigs and 2582 singletons were generated. Using the EST sequences that were obtained, 1973 primer pairs to amplify EST-derived SSR markers were designed and used for linkage analysis of 188 F&lt;sub&gt;1&lt;/sub&gt; progenies, which were generated by a cross between two Japanese plants, ‘273-7’ and ‘T17-349,’ with previously published SSR markers. An integrated linkage map was constructed by combining parental-specific maps, which consisted of 1743 SSR loci on 16 homeologous linkage groups with a total length of 2511 cM. The primer sequences of the developed EST-SSR markers and their map positions are available on http://clovergarden.jp/. Linkage disequilibrium (LD) was observed on 9 of 16 linkage groups of a parental-specific map. The genome structures were compared among white clover, red clover (&lt;i&gt;T. pratense&lt;/i&gt; L.), &lt;i&gt;Medicago truncatula&lt;/i&gt;, and &lt;i&gt;Lotus japonicus&lt;/i&gt;. Macrosynteny was observed across the four legume species. Surprisingly, the comparative genome structure between white clover and &lt;i&gt;M. truncatula&lt;/i&gt; had a higher degree of conservation than that of the two clover species.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/05/01" />
<meta name="citation_mjid" content="ggg;2/5/607" />
<meta name="citation_id" content="2/5/607" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/5/607" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/5/607.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/5/607.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/5/607.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.002600" />
<meta name="citation_pmid" content="22670230" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="5" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="607" />
<meta name="citation_lastpage" content="617" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Sachiko N. Isobe" />
<meta name="citation_author_institution" content="Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" />
<meta name="citation_author_email" content="sisobe@kazusa.or.jp" />
<meta name="citation_author" content="Hiroshi Hisano" />
<meta name="citation_author_institution" content="Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" />
<meta name="citation_author" content="Shusei Sato" />
<meta name="citation_author_institution" content="Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" />
<meta name="citation_author" content="Hideki Hirakawa" />
<meta name="citation_author_institution" content="Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" />
<meta name="citation_author" content="Kenji Okumura" />
<meta name="citation_author_institution" content="Forage Crop Breeding Research Team, National Agricultural Research Center for Hokkaido Region, Sapporo, Hokkaido 062-8555, Japan" />
<meta name="citation_author" content="Kenta Shirasawa" />
<meta name="citation_author_institution" content="Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" />
<meta name="citation_author" content="Shigemi Sasamoto" />
<meta name="citation_author_institution" content="Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" />
<meta name="citation_author" content="Akiko Watanabe" />
<meta name="citation_author_institution" content="Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" />
<meta name="citation_author" content="Tsuyuko Wada" />
<meta name="citation_author_institution" content="Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" />
<meta name="citation_author" content="Yoshie Kishida" />
<meta name="citation_author_institution" content="Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" />
<meta name="citation_author" content="Hisano Tsuruoka" />
<meta name="citation_author_institution" content="Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" />
<meta name="citation_author" content="Tsunakazu Fujishiro" />
<meta name="citation_author_institution" content="Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" />
<meta name="citation_author" content="Manabu Yamada" />
<meta name="citation_author_institution" content="Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" />
<meta name="citation_author" content="Mistuyo Kohara" />
<meta name="citation_author_institution" content="Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" />
<meta name="citation_author" content="Satoshi Tabata" />
<meta name="citation_author_institution" content="Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Arabidopsis Genome Initiative;citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana.;citation_pages=796-815;citation_volume=408;citation_year=2000;citation_issue=6814;citation_pmid=11130711;citation_doi=10.1038/35048692" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=E. Asamizu;citation_author=Y. Nakamura;citation_author=S. Sato;citation_author=H. Fukuzawa;citation_author=S. Tabata;citation_title=A large scale structural analysis of cDNA in a unicellular green alga Chlamydomonas reinhardtii. I. Generation of 3433 non-redundant expressed sequence tags;citation_pages=369-373;citation_volume=31;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=S. S. Atwood;citation_author=H. D. Hill;citation_title=The regularity of meiosis in microsporocytes of Trifolium repens;citation_pages=730-735;citation_volume=27;citation_year=1940;citation_doi=10.2307/2436899" />
<meta name="citation_reference" content="citation_author=M. J. Baker;citation_author=W. M. Williams;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=B. Barrett;citation_author=A. Griffiths;citation_author=M. Schreiber;citation_author=N. Ellison;citation_author=C. Mercer;citation_title=A microsatellite map of white clover.;citation_pages=596-608;citation_volume=109;citation_year=2004;citation_issue=3;citation_pmid=15103407" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=B. A. Barrett;citation_author=I. J. Baird;citation_author=D. R. Woodfield;citation_title=A QTL Analysis of White Clover Seed Production;citation_pages=1844-1850;citation_volume=45;citation_year=2005;citation_issue=5;citation_doi=10.2135/cropsci2004.0679" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=R. Berloo;citation_title=GGT2.0: Versatile Software for visualization and analysis of genetic data;citation_volume=99;citation_year=2008;citation_pmid=18222930;citation_doi=10.1093/jhered/esm109" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=N. M. Casey;citation_author=D. Milbourne;citation_author=S. Barth;citation_author=M. Febrer;citation_author=G. Jenkins;citation_title=The genetic location of the self-incompatibility locus in white clover (Trifolium repens L.).;citation_pages=567-576;citation_volume=121;citation_year=2010;citation_issue=3;citation_pmid=20383486;citation_doi=10.1007/s00122-010-1330-9" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=N. O. Cogan;citation_author=M. T. Abberton;citation_author=K. F. Smith;citation_author=G. Kearney;citation_author=A. H. Marshall;citation_title=Individual and multi-environment combined analyses identify QTLs for morphogenetic and reproductive development traits in white clover (Trifolium repens L.).;citation_pages=1401-1415;citation_volume=112;citation_year=2006;citation_issue=8;citation_pmid=16699790;citation_doi=10.1007/s00122-006-0241-2" />
<meta name="citation_reference" content="citation_journal_title=Molecular phylogenetics and evolution;citation_journal_abbrev=Mol Phylogenet Evol;citation_author=N. W. Ellison;citation_author=A. Liston;citation_author=J. J. Steiner;citation_author=W. M. Williams;citation_author=N. L. Taylor;citation_title=Molecular phylogenetics of the clover genus (Trifolium--Leguminosae).;citation_pages=688-705;citation_volume=39;citation_year=2006;citation_issue=3;citation_pmid=16483799;citation_doi=10.1016/j.ympev.2006.01.004" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. Ewing;citation_author=P. Green;citation_title=Base-calling of automated sequencer traces using phred. II. Error probabilities;citation_volume=8;citation_year=1998;citation_pmid=9521922;citation_doi=10.1101/gr.8.3.186" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=B. Ewing;citation_author=L. Hillier;citation_author=M. C. Wendl;citation_author=P. Green;citation_title=Base-Calling of Automated Sequencer Traces Using Phred.  I. Accuracy Assessment;citation_pages=175-185;citation_volume=8;citation_year=1998;citation_issue=3;citation_pmid=9521921;citation_doi=10.1101/gr.8.3.175" />
<meta name="citation_reference" content="citation_journal_title=Annals of Botany;citation_journal_abbrev=Ann Bot;citation_author=C. Feuillet;citation_author=B. Keller;citation_title=Comparative Genomics in the Grass Family: Molecular Characterization of Grass Genome Structure and Evolution;citation_pages=3-10;citation_volume=89;citation_year=2002;citation_issue=1;citation_pmid=12096816;citation_doi=10.1093/aob/mcf008" />
<meta name="citation_reference" content="citation_title=Forage Legumes for Temperate Grasslands;citation_author=J. Frame;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=J. George;citation_author=T. I. Sawbridge;citation_author=N. O. Cogan;citation_author=A. R. Gendall;citation_author=K. F. Smith;citation_title=Comparison of genome structure between white clover and Medicago truncatula supports homoeologous group nomenclature based on conserved synteny.;citation_pages=905-911;citation_volume=51;citation_year=2008;citation_issue=11;citation_pmid=18956023" />
<meta name="citation_reference" content="citation_author=J. M. Gillett;citation_year=1985" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=D. L. Gustine;citation_author=A. M. Sanderson;citation_title=Molecular Analysis of White Clover Population Structure in Grazed Swards during Two Growing Seasons;citation_pages=1143-1149;citation_volume=41;citation_year=2001;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=D. L. Gustine;citation_author=P. W. Voigt;citation_author=E. C. Brummer;citation_author=Y. A. Papadopoulos;citation_title=Genetic Variation of RAPD Markers for North American White Clover Collections and Cultivars;citation_pages=343-347;citation_volume=42;citation_year=2002;citation_issue=2;citation_doi=10.2135/cropsci2002.0343" />
<meta name="citation_reference" content="citation_journal_title=Mol. Genet. Genomics;citation_author=M. L. Hand;citation_author=R. C. Ponting;citation_author=M. C. Drayton;citation_author=K. A. Lawless;citation_author=N. O. I. Cogan;citation_title=Identification of homologous, because and paralogous sequence variants in an outbreeding allopolyploid species based on comparison with progenitor taxa;citation_pages=293-304;citation_volume=4;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=M. A. Hughes;citation_author=A. L. Sharif;citation_author=M. A. Dunn;citation_author=E. Oxtoby;citation_author=A. Pancoro;citation_title=Restriction fragment length polymorphism segregation analysis of the Li locus in Trifolium repens L.;citation_pages=407-414;citation_volume=14;citation_year=1990;citation_issue=3;citation_pmid=1983287;citation_doi=10.1007/BF00028776" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=S. Isobe;citation_author=R. Kölliker;citation_author=H. Hisano;citation_author=S. Sasamoto;citation_author=T. Wada;citation_title=Construction of a consensus linkage map for red clover (Trifolium pratense L.).;citation_pages=57-57;citation_volume=9;citation_year=2009;citation_pmid=19442273;citation_doi=10.1186/1471-2229-9-57" />
<meta name="citation_reference" content="citation_journal_title=Plant Sci.;citation_author=E. Jones;citation_author=L. Hughes;citation_author=M. Drayton;citation_author=M. Abberton;citation_author=T. Michaelson-Yeates;citation_title=An SSR and AFLP molecular marker-based genetic map of white clover (Trifolium repens L.);citation_pages=531-539;citation_volume=165;citation_year=2003;citation_doi=10.1016/S0168-9452(03)00212-7" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=B. Julier;citation_author=S. Flajoulot;citation_author=P. Barre;citation_author=G. Cardinet;citation_author=S. Santoni;citation_title=Construction of two genetic linkage maps in cultivated tetraploid alfalfa (Medicago sativa) using microsatellite and AFLP markers.;citation_pages=9-9;citation_volume=3;citation_year=2003;citation_pmid=14683527;citation_doi=10.1186/1471-2229-3-9" />
<meta name="citation_reference" content="citation_journal_title=Acta Biol. Hung.;citation_author=G. B. Kiss;citation_author=A. Kereszt;citation_author=P. Kiss;citation_author=G. Endre;citation_title=Colormapping: a non-mathematical procedure for genetic mapping;citation_pages=125-142;citation_volume=19;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=R. Kölliker;citation_author=E. S. Jones;citation_author=M. Z. Z. Jahufer;citation_author=J. W. Forster;citation_title=Bulked AFLP analysis for the assessment of genetic diversity in white clover (Trifolium repens L.);citation_pages=305-315;citation_volume=121;citation_year=2001a;citation_doi=10.1023/A:1012048103585" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=R. Kölliker;citation_author=E. S. Jones;citation_author=M. C. Drayton;citation_author=M. P. Dupa;citation_author=J. W. Forster;citation_title=Development and characterisation of simple sequence repeat (SSR) markers for white clover (Trifolium repens L.);citation_pages=416-424;citation_volume=102;citation_year=2001b;citation_doi=10.1007/s001220051662" />
<meta name="citation_reference" content="citation_author=M. S. O’Bleness;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=P. Rice;citation_author=I. Longden;citation_author=A. Bleasby;citation_title=EMBOSS: the European Molecular Biology Open Software Suite.;citation_pages=276-277;citation_volume=16;citation_year=2000;citation_issue=6;citation_pmid=10827456;citation_doi=10.1016/S0168-9525(00)02024-2" />
<meta name="citation_reference" content="citation_author=S. Rozen;citation_author=H. Skaletsky;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=DNA Research;citation_journal_abbrev=DNA Res;citation_author=S. Sato;citation_author=S. Isobe;citation_author=E. Asamizu;citation_author=N. Ohmido;citation_author=R. Kataoka;citation_title=Comprehensive Structural Analysis of the Genome of Red Clover (Trifolium pratense L.).;citation_pages=301-364;citation_volume=12;citation_year=2005;citation_issue=5;citation_pmid=16769692;citation_doi=10.1093/dnares/dsi018" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=S. Sato;citation_author=Y. Nakamura;citation_author=T. Kaneko;citation_author=E. Asamizu;citation_author=T. Kato;citation_title=Genome structure of the legume, Lotus japonicus;citation_volume=15;citation_year=2008;citation_pmid=18511435;citation_doi=10.1093/dnares/dsn008" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. Schmutz;citation_author=S. B. Cannon;citation_author=J. Schlueter;citation_author=J. Ma;citation_author=T. Mitros;citation_title=Genome sequence of the palaeopolyploid soybean;citation_pages=178-183;citation_volume=14;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. K. Sledge;citation_author=I. M. Ray;citation_author=G. Jiang;citation_title=An expressed sequence tag SSR map of tetraploid alfalfa (Medicago sativa L.).;citation_pages=980-992;citation_volume=111;citation_year=2005;citation_issue=5;citation_pmid=16075206;citation_doi=10.1007/s00122-005-0038-8" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. M. Tashiro;citation_author=Y. Han;citation_author=M. J. Monteros;citation_author=J. H. Bouton;citation_author=W. A. Parrott;citation_title=Leaf Trait Coloration in White Clover and Molecular Mapping of the Red Midrib and Leaflet Number Traits;citation_pages=1260-1268;citation_volume=50;citation_year=2010;citation_issue=4;citation_doi=10.2135/cropsci2009.08.0457" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=R. L. Tatusov;citation_author=N. D. Fedorova;citation_author=J. D. Jackson;citation_author=A. R. Jacobs;citation_author=B. Kiryutin;citation_title=The COG database: an updated version includes eukaryotes.;citation_pages=41-41;citation_volume=4;citation_year=2003;citation_pmid=12969510;citation_doi=10.1186/1471-2105-4-41" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=S. Temnykh;citation_author=G. DeClerck;citation_author=A. Lukashova;citation_author=L. Lipovich;citation_author=S. Cartinhour;citation_title=Computational and Experimental Analysis of Microsatellites in Rice (Oryza sativa L.): Frequency, Length Variation, Transposon Associations, and Genetic Marker Potential;citation_pages=1441-1452;citation_volume=11;citation_year=2001;citation_issue=8;citation_pmid=11483586;citation_doi=10.1101/gr.184001" />
<meta name="citation_reference" content="citation_title=JoinMAP 4: Software for the Calculation of Genetic Linkage Maps in Experimental Populations;citation_author=J. W. Van Ooijen;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. Wang;citation_author=M. C. Drayton;citation_author=J. George;citation_author=N. O. Cogan;citation_author=R. C. Baillie;citation_title=Identification of genetic factors influencing salt stress tolerance in white clover (Trifolium repens L.) by QTL analysis.;citation_pages=607-619;citation_volume=120;citation_year=2010;citation_issue=3;citation_pmid=19865805;citation_doi=10.1007/s00122-009-1179-y" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Y. Zhang;citation_author=M. L. Sledge;citation_author=J. H. Bouton;citation_title=Genome mapping of white clover (Trifolium repens L.) and comparative analysis within the Trifolieae using cross-species SSR markers.;citation_pages=1367-1378;citation_volume=114;citation_year=2007;citation_issue=8;citation_pmid=17356868;citation_doi=10.1007/s00122-007-0523-3" />
<meta name="citation_reference" content="citation_title=The Genus Trifolium;citation_author=M. Zohary;citation_author=D. Heller;citation_year=1984" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Comparative Genetic Mapping and Discovery of Linkage Disequilibrium Across Linkage Groups in White Clover (Trifolium repens L.)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/5.cover-source.jpg" />
<meta name="twitter:description" content="White clover ( Trifolium repens L.) is an allotetraploid species (2n = 4X = 32) that is widely distributed in temperate regions and cultivated as a forage legume. In this study, we developed expressed sequence tag (EST)–derived simple sequence repeat (SSR) markers, constructed linkage maps, and performed comparative mapping with other legume species. A total of 7982 ESTs that could be assembled into 5400 contigs and 2582 singletons were generated. Using the EST sequences that were obtained, 1973 primer pairs to amplify EST-derived SSR markers were designed and used for linkage analysis of 188 F1 progenies, which were generated by a cross between two Japanese plants, ‘273-7’ and ‘T17-349,’ with previously published SSR markers. An integrated linkage map was constructed by combining parental-specific maps, which consisted of 1743 SSR loci on 16 homeologous linkage groups with a total length of 2511 cM. The primer sequences of the developed EST-SSR markers and their map positions are available on &lt;http://clovergarden.jp/&gt;. Linkage disequilibrium (LD) was observed on 9 of 16 linkage groups of a parental-specific map. The genome structures were compared among white clover, red clover ( T. pratense L.), Medicago truncatula , and Lotus japonicus . Macrosynteny was observed across the four legume species. Surprisingly, the comparative genome structure between white clover and M. truncatula had a higher degree of conservation than that of the two clover species." />
<meta name="og-title" property="og:title" content="Comparative Genetic Mapping and Discovery of Linkage Disequilibrium Across Linkage Groups in White Clover (Trifolium repens L.)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/5/607" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="White clover ( Trifolium repens L.) is an allotetraploid species (2n = 4X = 32) that is widely distributed in temperate regions and cultivated as a forage legume. In this study, we developed expressed sequence tag (EST)–derived simple sequence repeat (SSR) markers, constructed linkage maps, and performed comparative mapping with other legume species. A total of 7982 ESTs that could be assembled into 5400 contigs and 2582 singletons were generated. Using the EST sequences that were obtained, 1973 primer pairs to amplify EST-derived SSR markers were designed and used for linkage analysis of 188 F1 progenies, which were generated by a cross between two Japanese plants, ‘273-7’ and ‘T17-349,’ with previously published SSR markers. An integrated linkage map was constructed by combining parental-specific maps, which consisted of 1743 SSR loci on 16 homeologous linkage groups with a total length of 2511 cM. The primer sequences of the developed EST-SSR markers and their map positions are available on &lt;http://clovergarden.jp/&gt;. Linkage disequilibrium (LD) was observed on 9 of 16 linkage groups of a parental-specific map. The genome structures were compared among white clover, red clover ( T. pratense L.), Medicago truncatula , and Lotus japonicus . Macrosynteny was observed across the four legume species. Surprisingly, the comparative genome structure between white clover and M. truncatula had a higher degree of conservation than that of the two clover species." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/5.cover-source.jpg" />
    <title>Comparative Genetic Mapping and Discovery of Linkage Disequilibrium Across Linkage Groups in White Clover (Trifolium repens L.) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q0wst1" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__KPgiwEdjo5O9HdRryHqQs1TpsV-PeBb9B7sGBvwM9FQ__opzZWgPNgQzoEgfATRS5qCKwBw_6XcNjDHoDVOzQv8M__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__YUqwBlv6bNoCToiBWgAUSPtppdS8sbGfzmAH3qEDcI0__Mvk93h3ScFH5Xi9WhibVKlf368fHw-V7HWqWX2QMvtw__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XNWYaQubFGWYWZNytUHkWs-b34LV21DsB4s_6gchI4k__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__7cpBA0GVdrMqkZGGZEW-_SPF2HrbTCmVg9-lrP5l_AQ__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__TV2D4R18tPiSIOmeQ3vyMqHObN_FDmQ6EEIrfE5sAoA__c9BBZFy1BrgGgF_Auu9Fmysb3Z7J56eJGYvMcLQrLks__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q0wst1"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__OaHnYyci68QTwctdRYR30kEfu0eYWBrLyMqy8THv5Ik__IyfJQYM1q1QEPwYsK_Eslh2RQkxpy6yo-nR0p4DWPQQ__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__fhq_kmKS3zjIxlsAa3GE1ubLajyXUgXXTXULzAtiqfg__Lm1_Ls-bdpbWRyl0q4o0mmHyxB-139AJ68dE4zOskLQ__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"cCy0fWzWfelbCUsHhvlycGtdsiHy4ys53V1BCPeLobk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459276","apath":"\/ggg\/2\/5\/607.atom","pisa":"ggg;2\/5\/607","ac":{"\/ggg\/2\/5\/607.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/5\/607.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/5\/607"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/5\/607"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/5\/607":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Y64DnVlPycbC+M07vkhgENig6NPtufkt+CiuZszJYot1PprJq+o89WpGb5aEbkM7fh+gq+wgRJjR30CbBhMDW27I0p1NuaWFxBvoZdHUQoHjC\\\\\\\/XICuRQa\\\\\\\/n166pGx4CXbtdfM0JQLcnm8waKSKET4OhncJEZkDcSXRWC7SZWwOo0Vmhm1QAmLe0g7wIB6VzqYU\\\\\\\/6f+saDorgAQoYrEr6YuGK2305WHQMmj29VqDu5uklhIX+6OA7n3fPdoNZLc5oELBaa3VosXMiyojmmnkJo4f8AiHepXKrbHLWOCqS\\\\\\\/gAOAm6K6Q4395ZTigflcwkFuHenkRBLSvgZLToyga7Aur3WFQ3aZy\\\\\\\/E92qNAVr60gbq7rwVE1rBn65g\\\\\\\/ChNEcwD\\\\\\\/t495tM+BxfrVUgo3Ymzjx\\\\\\\/B8P1Xk45Bue66Ob6uoIrxdfOBuUgi\\\\\\\/qiWIpxtrba97DYBQaw8cfdr99pPOjgoJnxctSB0yfKN7Pmf+C7Tfg3mmeVU0b\\\\\\\/22lUta9pDvU7Fkmc5\\\\\\\/Cf7\\\\\\\/KovTfwsY49WXiLje26PeEdT39uYDMs06wUzzmK5JAI62Dys4Hc9rzcNjc+o7LRGQ7pfGtrzN43oPDTtK7+cJse1K2OwA1RmymuK4pP\\\\\\\/I6BbwNDj8oJttf75NmywFZAU\\\\\\\/D+\\\\\\\/wTBiKQ7AYQZoyJKIE\\\\\\\/gXeW4UOmhPqUlYuYxVUXIAaYGGRrDElMPbQPly62+dNnGGS30SJTRzC3ONeBj1fxFfAvVR6zxRAy5PxGGOSWSFuHiWmJOa4zy+FhJSXyN4OrzNN6wv0Dzbe9Nhz7GdbU7KB6oaaeOvBeYjfePfL4rwqi+jpeEZH9Xdlz1zgxcxTjjmweh0kCNwVkiZp92qXjEychgtuFRj1oeybjlPoCB2acmK6BIlQdekWMJdp+g0PyNCbDr9+1uumENdesh35WjcPruq0c1uxiT05NbQnQ40NKMrjHe7AQmcnUw29cI08Syik09DNClESDUt\\\\\\\/KdgzuyfWgxcmkkdrz9ZvskjvF4T2P+O\\\\\\\/cjcjaZ3FGJNIt5GFTEK7ZsMkQSQ7czEWz1UERLQwzI05WvXz17w0uoaA6cpry5V9m+H+pWwvFxFrbwIR9QgftNx1pn125NrmPYo6HgbYgXt2lKV+vLDMa+mKwqyvLXe6023f6s07a6T4V48ye7NbOyBfR6TllbPRAaLFS\\\\\\\/MHBxiqbnD+0nIuNWgDOLSoZwiJw7Iz1Z3lKt44OR1287ZzJxEt44B8k7RNKdnQf3TlqHXpiSwiDbbAQ1xYBIACQrTKW5BPiaO\\\u0022,\\\u0022iv\\\u0022:\\\u0022vhJY8oXdVITA06Qd2Cx+yA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228fd630b23a32d53f60bdc42840e4f78a01f46710c026b01e43bc0510609c87b3\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CglHa+0ZkrljBoWw+WTHUdcyM5PnSx\\\\\\\/+WzInboJ4clp7T0pvbpl8jmCrCBLuOredPiF7JfmAtKyj21uz00ji0DOYU6LCZ42ir5+X3lRpi6zzNOpthBe\\\\\\\/YH96Atm+KXvNd1FK81vBkkKsn2l+ZbOORspklJ8aYhwcyMhQXfDAqGrkEB09r5OOpGz6l+1QdQNNTs34UrW8rAMjUPR+L1W7CJdacSjX0qA8Rm1rVzuSGWteMe1t0rPznWmdQ1QkXbh0a\\\\\\\/j7GSI7E8\\\\\\\/HzfQcPRyKNHc6kUqXwPMj\\\\\\\/2gdm7h5BUwAjkud5CX0FHIgZ8O3+6duY4oCAtvgsSkcdEY9GIUdipXH7gh3Q1hEyoRLcUmMYNjWGepphIXw+8ef76aYjXujJ2DN36NsEx4Yi5LfRcaqM0b5MGApnOPkceHihNXjR31+JS6N50HVw88ST4CuiTKarFnNDr1jFw2CjKuUQIGrDeXMkHkqc0Jp1bl2ONE7iy2tXLIO3IemeWF6YnanHKyC3gBaTHOBc1MHFlIlsRvvNlV\\\\\\\/zzR5YsjgOewcsWYVPAPz\\\\\\\/bRSBp\\\\\\\/hwpki05h9ia9Q9o8HXfmnJtJs5IF4LWH0TQfEBCVcVkW67j1LZ+3maLmo0vr3ZgqXnBQ48tgT9+m0MGgxAYEZQfumOADkcTSv6pOjRCAEmvb95wFuge2So2RsftXE\\\\\\\/xuOGQ72Qt4t0bGcycVeuzUVy1Q2F9kSmOT34XKcZqwpFTmsSn5qhiqKdFEKJZuern\\\\\\\/LrC8b9vQoj19yeQPHazNeg8PFgPtOcZKbu426vCewYTJB9GEc2v0LQkFeqEadxh0rpz8N9n5GhQGK7DfjqY1\\\\\\\/DoHCE0m\\\\\\\/qDCIkdyR2zN565wzN9zd3UsGOzQBx3iGaM7sfQjuSpnpvEhKAKqKoN3sqzo3Kuj+QunbreyVIr5bam+yuhdKJmsOJXBD4vWCi8Qz7kNsqf7aUOadKl7lJf\\\\\\\/bPNMqfT0ZYvA+Tero2grPJCOsb74YkD64KzKtMMQSlOScFZQH4LQrSIEaYAB4F3Ow6GxaCct88EQlNHa6HjxAjwptp8iJECBnR9qE\\\\\\\/FYe4zLgn8nbxNv3e6rYJ6amh6n8wqYg0D70xj+nFgH\\\\\\\/7D6+FXRmGk+jY5h03pFxkpMqbsVmpB\\\\\\\/VhyRAsVUX84cEusd34KnHg7knU5\\\\\\\/KJL4HG0kl\\\\\\\/HTrVc9Ke072Ii1TZ2jWLDzroL\\\\\\\/NFUcdc+knncsF+XCoXv3M8Trpm+c+TstAX0qS6cgjGEz8tlFaP+SSljUKkRc3624P1vrvxHiJxxG0\\\\\\\/qRE6Fs4P4qzmmnewnwTlR9\\\\\\\/8fKgrFPQD9UMsXcQj1jbVxkSWWfcGHOESzOgY79iY4TTV\\\\\\\/IRpLhIdg5SATEW0n3AbAJuN8peCStkV4xP9B+H4xCAIyRvUY57U0zYTedhMdc7R9vZTErKWsTjiVhFdoimhjabCFVgigHZ0kzX6NNgwNPDXDxhY\\\\\\\/mWoMNx4q69yciG6mrS3C\\\\\\\/SVLVtdBe5M8e3GGYm5w2nMxU9dmFx\\\\\\\/C2xYKDOw7cdYK0EFthgwX5K8L12ti6hxJaJuPPwD4le3iUE3O6xNZSIJlqf7lPK\\\\\\\/9mPaufwBI9rY2WCWWc5gkmU\\\\\\\/F59FH51hYpv9INVMlf8zYVYu7yF6n4jXuFFQV5VR4y2b1TkPLbUTpZb174JAHP5IJ\\\\\\\/PBdQJUJ2kqMjq+RK3F6AGfCZ5VydM+KcHiy0zRy5wpY+lOZtVyxN3nGXXlnM4nM\\\\\\\/1WDWrYNHxj+MTcHvEFuW9\\\\\\\/FXSWd1SC26v7Yn5tguxKjQ7rFgYKBC\\\\\\\/6BobI4LYWqcZMreDlI4H79DF5npaj4gWwju560s4cRzX1oXeY29pzaZH91HwwT9fwgtbu\\\\\\\/dhwkY0muzTEWP+aLTlhudhj+IAHa3p19+b\\\\\\\/fHy44SSQvzSBPwt8neV0E92kI\\\\\\\/ilO4Dma2POvCk0yHU5yE01jZcmoAjfpY9XbF8Q1UZHnsjpp6Sgx0lNvO+klTrRQ6a4exv3sOFp\\\\\\\/yDbclQrBTHO4E3JT+taEmO6RFlcCHjDYT30yGg+3pXUlyeH1bv\\\\\\\/EIYLL7ZSndeLXNxXYEtBtUVPDrsOKEEE7nJkUWSG0SV3VBVlQOa3qwGzXuKdn9Euoxbs5+xW1cw88cHiEEO0+whP0s\\\\\\\/cJxeEVQG9g7dyCTITvUXy5CtkGlXry6ucnjYQABN\\\\\\\/zW1kJPl2mLhracY7JjRebbEMbi8ZOv9l4FGsbHd90UyPWmBW+EQt4PUnbivB8gxoYM8aUoXswvUOtmNtYr3q0gLN31u3IW5Ii1YRCMiAP6n0H+Y1nkgxyWis7Sp2wF8OHoUpV5Ra6gB\\\\\\\/4Cw7qAqxBQltQwUqntuA7oLROFz3Y7sC1zr7U2fXhaL6YazJomDwxXP3V0HVzy\\\\\\\/z1ch8GLNYrXAnWjW\\\\\\\/m44euCdNTor+GWvmbc\\\\\\\/IU\\\\\\\/0TycDuvzfqY0M0ZYyQPORfEDdE0BqcnjcY97+ArrIcH1XGo94hwYX0Cou5UNVGOHE6rIKj1dYjukZ33vtnjnccZRfB+rqCK9EO\\\\\\\/6r\\\\\\\/bNu\\\\\\\/1pi6FMMztdnAS0TrbNeIIN+oMiE43+BhIn1BNP92U1wn0\\\\\\\/gaEJNh9VHVbGrO8qsmPb16m9YXixg46U9XOYu7Vzl00TbDXntqrdRIWXlB\\\\\\\/r7fj8xcqrJwTexWkCkeL2TLRxdm7ptTZrK32EUjJNJW\\\\\\\/TBYHkSSDRqA4Ap3+VpqN04Zy75ySXLPfV5lKFH3M3aQa44JIQ+mS6ET\\\\\\\/EJaUn1LY6\\\\\\\/sjqI5yLQcCQqXdPFrKCUm+T3lY46bolKSO6HYIwLFPRYmFKit7BBY7M29eN8zsodbgD0W8RTjpDAPN0wgWGDbmzyd8rrRmOlw6ZbFGAfeCUAufc6rmgfsIi0dzF1EhHppKvkP3SR3MTik4pgwMgEYWa03CmHp29ncp7VMH0CGNbSH1GpETQhPbAsW9Wi+7amy5oC9aMplUVGgo0k9aTsCex0+6qf7Ww8qViWFi\\\\\\\/H5NFgMGkHT\\\\\\\/w2cRUCmVhuwwuliT4TFKzPZ1w4f8cE+hdtbcRJc2xYUj22fcK1kbo5X5uTcFutsz\\\\\\\/UmB6d36bZrvD2iZ\\\\\\\/h4OX5JqxDrtzAdjQgCo8Db88DyzAdcll\\\\\\\/C08yr1xCzXSkQp2y24a4K36KisOPfmLmJiG28aDLkw3EiUH7w454dAZKqTH8tQcjLGmxEhaqReZl4b\\\\\\\/EWQYbomhw4saZLtMDsLi9E3PINrP\\\\\\\/1fia7PlUucFWVJmA9DZJF\\\\\\\/c5BL0kXsPSdZU7fdRrnUE+ZO1cywMcc\\\\\\\/WxErODws9vVxbIhrfm\\\\\\\/nLxOhcUzsKOeqhic6kNNVapE0Dt\\\\\\\/etE8O5M+WnZlLXMZg0mGbpw2AWID+ePyNzr+\\\\\\\/ck0Ecw0gsd8\\\\\\\/ibaahFF9qZPAzyTK0M\\\\\\\/5AT6hsCaI2t9ocQpsKVB1wV5qSpI4AKVtKWD4S0urZDN4OtwheaLaLiXLxNGZ1w0DW91xf2Cu7UgFK6sPNEK6KYMdR7iyvyqZDzgYoDevnC1qZThaJItly+uaHd\\\\\\\/UAkpRhW9QjWtVQrUrpoKHg7lU8EWsTA6a1qTABDjfz5AX3nMTyrdSw5+bx4x9ZgDRJxamD1ujouK7S64q3io6AcQiNXuds1zporfjmx7DikYWORiOVBsTdaWBBswsS10AiQTFagpPH17pNck5KCzQW9S+vdDZJQvq+DDQ0r7HxkEM3oAt4NOM+57Cx77u+4q2fltFI1dTOenM4xC3QqK1j8lIaLWGYrlAqfa0OInvizTAdLNgd2QieTDW7J7knxq28Ng4xq\\\\\\\/pJYhOOLMp0dDIJIUVdGl\\\\\\\/HaI8Wck8ETM1MHn4xz1CF72UkqvXjsyxhk64xaeN9UuCHHkmF3E8molEnY6MnDSDhdOMaSsTmyNCJB51SeOaO0BPW4+kW0Ote3UDK0RZD+K89QixFVn7TVvKdBaly15k4yLId+7yjTLPMkorTlVepZt1wsEtbbk8d0YU76\\\\\\\/no9GbVYIDY2ZfnxRUEDAOOgu0gDkcJKoiII+SRrciXdPo0eOaqGQq\\\\\\\/tXvHEWbYcT0MR4rfhUquUhwOBx2s8hpWutXE5XWwTA3dmefnFDzlAfNldoy4yHPPh4QZABOUMXLISB2jXcRxImizRaad3V893xwZY04CpftGDIP3G\\\\\\\/M4p4nq2x\\\\\\\/I3\\\\\\\/cVLL\\\\\\\/Po6LHVE1Vb\\\\\\\/D3G0e7Nr9SzloRgr3CbOV8teWVrcFv2AT5\\\\\\\/5WjDfg8oLEQp7FP30t5uVvSEwpg9D9HqoctnNYrmt2M9IrViupO0zkUy5lk1n+bgkSNhnRKvgbpE9rtzFvKxl6VZcs54hu+w3NZsyDTdwdRd0Gyj7jFTXzPEEC61HfipNaVORYkdbiArUJqBt2TIo8LQXlTPhOVm07vTQRmygY49ly5NBANJ3AjrFIYSaGtvNf2wbkjbtL6tw\\\\\\\/KcIU5USx4FOf+vlvD6cUdo3\\\\\\\/UwSlyU8mhZYCSk4diCMacY0TdumdsZHW+PcZkQXYo5i0BrK3afdynNRjtkG4dagavIfoLOSblFBzn235H\\\\\\\/dZKYj9gKPmAyVe0Z2dhTNrJ13mV7vs4sK0SGJKUjo1Vsdg0m5jfvowZz1wqCKgJBanNLxHkUn3jqjewinAVQfUlMRH+HW0qpscy\\\\\\\/+w+IcG8W7Jw4113SqdQ5acqbpxuyVV3rfc1V6BAoJQi2BurOkdqOih\\\\\\\/Z6gGoSXVppRPrPoDrvlB6JmRqnOWOMhqgSn0zKWSg9ShGeptuNpR1jR4bs7wTKcJtN9Tcplj2TBaezlPYk6E6Rr7X4TOdXf8y5zyvkJLc2h2vubcH7Xl8mdp12EoSmraoiPKgv0INZyAaY99\\\\\\\/SxVrpIQz2O7haB6Z1QXb71aS5aMW6L2Z11hYpKIg7BaZFtZBkBz\\\\\\\/FdavBRyeSiwS74nvsmIjZNyV6z4I9DQpMf+n7pULkBTfqAZV6Oh\\\\\\\/KpbkzwtSvHhLGzm5fSkIzFlXcWl77QDKYN1rzwajKgMGlohc94rm3HF9LeKx3iduYsI5CcHQlj49BeIb5xTqxmyMNxNHlXLYhqTujIyuJkvuaiRXLslFZimAVd9tv5FKTruODddPDCcwwAo4GdIxxwNaKmciCFGa676cpHuT6IezkCYkfzoUydmBWvhbW1Shn8Tn3NJytaC963xsaoqbaCRAo0Q1iQAEYTnIx9ljVjB9c0+MLTIC35PtvnAfEy94WyZhPhbRcMbmZtb2JaRd81P0hP3PWMlPgWUeX32K4V0BVSMiCJkZNFlxy2sOukRhYsdcUlJ0t5UBewNvZ7dzOgcLvY2wWW5oFU6lj9abqX5FtrP13T5szMx4F3OU9sV23KDuVcXlRmmMUefdhcpKpwEMidC8GSoOvkP9ylDxC5jWiiUmpTcBCFyUU9QxglZeTPgkITQwtMRLOmoi5kUWTOzb3w9EPLUw61p2AIBp974P2pG6jajsjQVpWfQmaD6+WBHgfC4J6+UHqvEqdoLVGNWj0ZyjlePmJLmwubbyMrpVacrSpiuYybTSzHqewy6+ttUsD7czwLnpvfqyuKI4shTlWTYYDNnFDHwMnZDHkF4iYjiDccBTX4eXWTOUrS3RvJkSSneJs8YX6tphSu2nzA4gpzE3twoNpO61YbskeedZ31z9Azq3eV4RXeP9Rieth5hvucnxp31lUihUhQrOO8lUt+fZk36QN+SqVbwL0cMbJFcDhiez71sLs8LqO37T5LlO82p577YMKnJgxZ+dk\\\\\\\/xDCJCaLuitVfm\\\\\\\/me3pflMNGVYNcIaoi2RErAkgX+Pt0TIeLB88Bdy7IP7WX6R9Ak2kr1pMzdVPfY45w33B60VIsqZBkYsxLFsKvVc0NoeLTPO330sBDvfAq9dRIjMZHotlEuzLo0hj0+krzJi2ozH\\\\\\\/RNlhzHpQAjlPtJCuone8th4kIMyili8O082jvG\\\\\\\/FmG3IhDxI3pwHnpRUBunKods0fdZC\\\\\\\/H6C4Dr+UllA5xNkKbQqcz5CM+1ezDMAFRSMK+CkhZZQxKVq7N47fYgoYKkJVd9yFbnHcSH88W8OMw23WZQpi1IoBx6SwTn\\\\\\\/506OI8xBYx5kjesJnzuOdt5C1rKkpod0lqxNaGgC0I5G7JvsDufEBQAUtHViQBcWSgWSE4niz9cyAnyOLWLpixattwYKNUVpmRTVK5tpqfks4lrqd6ORbYu63SbV76Fee0xuFTTIobFI9bjbIUV\\\\\\\/imoVEYAAjuH4cdFTHQSTpni3fcwQy0vGJ0iTo47SXYNfvsXR18RXDSAtWEVFKIkumxDLbpllQsOuoord9Q8KTCgY7e+tWQ5cI3I+hdBsYnilLr+c31\\\\\\\/Xe0ZNlzig3\\\\\\\/ECnb3ZRBjOFKHht3WhKUaUVDBix31xefFn9n6vf6mR6qh128qS1eVi6oMTd3BmfBvE790oqDsD8p40j+mIX55xNVkq6YxASSuc2ds+hC10XsAC3ndAMwPfUCrTWAclIXugEpEQX3Doisaw84VOx3md4T8pxiNPnn7xOEo\\\\\\\/21yIDIIJGxobfGPPaJEAXDTmFSqARf3ySYHAfKgiD5kiVBLymYjs39MAxfmT1t8\\\\\\\/ickH6+60PDaWF8Q56uIF3MW9cJCsYYOUI4sF8A9pQqZDee\\\\\\\/MEKrO3GjRegMsWnL3U6IzcXqoeR3cf1pTX4\\\\\\\/LxYHekhsYgjYi2AsYu8hctMfvbCJWxatwhQfbcPBnu5DywtvQcxJGKgdGnVDhsdS6B5hcnBpNgOnCY195AEsYbhYow2WY4uIvlYIsOEP\\\\\\\/qBwhD7Xtzqep0vWSyIAkf9vE\\\\\\\/wmiR5RDgVPr7dzJq75tzFJBbTppJOZLVW3MQFvso\\\\\\\/nCVesQBYSaBhMQ9PyupJQHfY0yzuoCAIW3UdX1uFRwh7YpevHBqO20YqjLV3i0m+ALsh97Wx4jY+3REjwng9sPm9P6bTTRxC+QDqiNLHL8U6+2vU2bsL7UzeUC0hcjoeYcQr62gXuoj60O7QQe\\\\\\\/e0lLvsbFys3ch03bVQBKeRIaMUoJVMRuKsq5coAfubVZkiECsVtoDnTDyGj5FHWgz9WLqWS1tKqa8hmjLK7O\\\\\\\/L0r7w4YKto5+R9\\\\\\\/vIQJ9hQ26s6Loa\\\\\\\/+QlodgbfzWBIHGAHaGD9+6YcwNsD2aliKD9W1Qb3a51oPskdPoD2kIigh929BZJs8eR8pG303KB6hKO6rjMj9AjFHYC0OoNUlsua7WlmIpqHvX59Je354ZX9Jpn0Kuo7CDe1wn+BXOjLHeKlLnZyyH7bqpO7Tl2TW6QLxCPbVchuT\\\\\\\/QfeS8UNf3YR5\\\\\\\/z+1f4d0NIkQ2PSc8xeP6A9UbyuNfahKarjdQE9D7z7iXHtohPASrOTzb7nMSW3bMqLDHp4k\\\\\\\/kYf0a\\\\\\\/6IhrU5I9qpE4XOzDdSonMGApO5ma0pEY0ded37m+2E8LrOtgRJGuCCtVb1UTWmoBs0fe2Dp5eYd\\\\\\\/LJMxbBprUfY0jZufkPguv8BjJNtDbh3teM19VL+B4\\\\\\\/lc0r1iT2EVAApshhuJkmXF\\\\\\\/ZRGHmZF212tHmvRoATE2QJ\\\\\\\/fHDJu0axP5mFi\\\\\\\/lrcvlBugdmzjA7i9kk5GDevM9qq2m65MtV31dK\\\\\\\/4dFsxqFy3szxwdDlFcPrzPZvlGdglVRVhYriDwqfvKIOULiClx69b0+Y1OER2Y+Ua\\\\\\\/bjTMN4mlQk\\\\\\\/Lzp7OK075v+wXW8xDVaFhSN7NNqvR\\\\\\\/MAFVnbMhypBt3b\\\\\\\/QPmswk+2YJ1lMWTgoqTNUMfofIht65Ngcg67eNHThTSFKhKPVr51LUhieHrKR\\\\\\\/OKQmO6cGLynzRd8gGqjWGELFeC0pniPvHgN96RmISaxkmH46xAT5B6S33GwSaNcal9D5TbopYBKf3e3ZYSn0N5mc\\\\\\\/cLqA\\\\\\\/ITBd4z2bCbgbm8995zqX2kq+gOKZf1bqaLUhr9eMRnAncgMEHsj1b\\\\\\\/O3egR3GKHqAJZtITluPfq30kDzuFHyJE\\\\\\\/3uA0b0MKNV\\\\\\\/6zs70+bD+WlNZ5hoj7OHsTfEjdZgOMS5RQtBv08sMuAhCs\\\\\\\/xfB+SXcl5\\\\\\\/JV3JN9BUppRXJxZeKSDDM4w0V1UMwl+lYAKdsxwf7nKm7f50F4Lj\\\\\\\/uVwErY8KwFcoDZop+pzCY38lkpbp\\\\\\\/nmUdJB6WrCQrxjiIYFLW\\\\\\\/7MzpQ5JX1dHRI\\\\\\\/UvYDCRkyL6g\\\\\\\/1zYjTIP+VlxuegPcnxyYjGJLaHQzQv7GEqvd6RP9PKkFc45xGE5YoFA5K57grmXT8vMerKfis\\\\\\\/8+FRo7aGORKvvZ3YA\\\\\\\/WDElHXbyPccTYi790YGDiI0wo14nea8ydiO\\\\\\\/D7o+kL8ohiepQtiZ1nkUInGv4oYrvmjf8ocJF19T9PTwnw9HchDL5vOSYdXlWwV5QIBMQgP+xuSB7cl1sHIVox6Amf1XNZU6Jl3OHA\\\\\\\/LmiRM9JDXk2wJFrS2JuxGvcOfYFzLpBPyRHV\\\\\\\/jgnMGN4Wg\\\\\\\/CXEG0k+aCMgqL\\\\\\\/LAfbpbWmT1twqZQ\\\\\\\/CmUa\\\\\\\/\\\\\\\/gSDRBj6PO3wyebt3b\\\\\\\/Ug+rGQWhVCDWcFViSmn13DRu1YIUU6WBnCY11LRTWGczzf6vChT8H0spWaxeI+fPjo6b7zlA2A2qedNVKYS\\\\\\\/ksdHzUvCumATayhpd4uRbNSr8i1Bdx16wTQ2bNIGvA8YJjTzn2VRfVQXLoGgEPdfj5NQo4D4hxQRHc1r2NAWYuQcG9YpHz20\\\\\\\/3HewthH1VpcipzHp+FeYX2H8YeBYX\\\\\\\/59jz+NNYVXhbmg2I9SuJhhMOt57zN074gggGi0YEByow2keZyclhfK+J\\\\\\\/Qi4h466eoMU8fB+Lu\\\\\\\/51wM77p6MgUE+nar+y0fHR\\\\\\\/kEGsbiwJ9PWNoBIPr4RVCL8qhR6zt9LwuwMVtdVuToAXI0\\\\\\\/2xGtjFwnHNJOUf5pgk8IvmyBVthskv+g86bMwP0vMGpGoLCyn7rWCOSEiypUyYdK9kBAYIn3X3RNPT\\\\\\\/8MlBsizs+znexZveBCcF17AtiF1VrlGs2IYNljpn57kxopxotjK\\\\\\\/YvPgjmZ\\\\\\\/0CZWnY5brfQGMyzYwgCCG6EWnxolX+U72CSfBrzw5wpyk144Dc3mH9NrwIklGN1SgjMlcRr47pPU3\\\\\\\/axB1n3qFJ46oQq0a\\\\\\\/otZXOUbc6TB90a\\\\\\\/b26rmzJ6y36bgpo3kMgNSBvhA50\\\\\\\/95WODTWkl7vADNJpyI853UICP6XhUIPisxyqLr7WIKW16jLKby7Xl2PRUXEiq5F1ZMi0BiWCNOmuBG4GsQmDxZZkl33PB8+kdnZWqKl59t9W+8MSVeDCaXlFsPEJ6azybPfmnqnA1\\\\\\\/LTHXVYeN\\\\\\\/w+P6gEP9URgOsutR7v+i+p4wbLEAITwwxs64nO5Z\\\\\\\/D1mVUaWM6h69juGQUy5o1btwOyCIJupnA1qfycBz3Ta34CDpIdAzWOOrvDMOz9Hl6HfCVSwIa\\\\\\\/0aMfX4+7JdBnfmhuSWLaMBVgLGImbVDbz\\\\\\\/9xPKNNJpDnV5JW1c4Ca0FiDwbH1SRZY9NqjepYx4j6WMWDU24kcWnfKSaJYvURNFPzXEz5Rp9v5Epp5aEz0vtNN3CFWvq+ek2Q2t6BaH+Sl+qJ9\\\\\\\/eLJbVnyT5+2Dftt9nswYkuZ1EAzRfozJgKJ\\\\\\\/vu8VI09ixb0tHvtzyov\\\\\\\/ZRCZdJfrYlr0TNRK573PtKo5ZaJq\\\\\\\/sjkLSBtBChubKrMbjfrLvR4bvlvCy+eRxte1sPnP7YTYW9gX5VFNSMxZHYHTPxCZE6zdJ0jWHGGKTjCRJjHanyYi4yBAiwPbiwj8y7xMtefp6QWxegL4kFRzcHMe2bJtHqw7o01F+K82NR2FFTk18BebS+z20fn\\\\\\\/zYO4Xt5NL7GbnB3Mi68Sc98jHuYHOzdXyfrN96HZmjoQYU4k5tXMVqKRwRsHoChRZZcUEq3KO+YcqjQHlJx45Ft\\\\\\\/yhV4yFn10+\\\\\\\/+WrO\\\\\\\/8WbacwdcxZV+5gZJ\\\\\\\/Hm5kyxl\\\\\\\/bzpcgy2hqqalmuZcdsENuUSLuX9hGa+JIPrW1CD+FC9TNZoyxnOOqjrldrQRfCH2GyBvUALe7vMMsr7qBjv4iJtgc6DJlntP9fDVPuLnVqzmJPhVmwLrb4NV6vNYe5\\\\\\\/5Ei8xFL+dlrFZyRLk1FimSf69zSsdMV57rJroRNrYdynxHbUIay+KxvvWN4WT6Dx0JquApripXfMoimrT6SXHFF0o2P1jW9mme\\\\\\\/0XGk7z5B0bRn\\\\\\\/oRxzv3iqYXcuYPBg\\\\\\\/j99aJH+xbVlXjA8V4Dux\\\\\\\/fVB7vXFZNJhqtc\\\\\\\/HyXE8Jx3vLq+czyNXMYRjh7z4qonw5y77sbgimToZ20acW\\\\\\\/zh0v7ynsClalondBbCZl\\\\\\\/kkOf3bGpjbCdlQB+Lmk5j4qBVr9IA3OH2CJ+dAh+TkAycjg5Yb7qVYsMw98JMT30thnNUAZ2SHAAUNN1T3ZwLiQUb2H12zaVC3EymQiFPkLpzq9YhmDl2Y5THiukOtcNSJOuWf4\\\\\\\/cYElVWHc1AMTRZ7KEgy64FoO0pfa5vb3in0pKJYT\\\\\\\/db2PyRgAL0MKfFeQk0HGXxeNegVfiPJOVqG+AE6vhvpqR5RsOh4MeTlm5YnmYVa59vM\\\\\\\/6pUHBH92RkEPz1On\\\\\\\/YgQzqVNlYoHtjYXNKw0wWkVqPeXaxNBGq6ONhu9yxjcxUnBMpy3yogtJ0e1sdo49HgEkmv1Rn5pCZgdnlgOfPa7F1M+Tdd\\\\\\\/ZniUBe409T0Dzm\\\\\\\/8uMS7rt7PW+iUdq7xonI92faujy\\\\\\\/FuSc5r3kL2DGSJndcfT1AJ+pAO0AKeOUeSG8VD1TVJXsqoPJPmV3K8o8vBlPVeDKxz\\\\\\\/vHHfE6jVpFhkS51V1YZZ55\\\\\\\/CvB3hOMaCwTXL9hKK1763A+YhRBBVPnYCTQm2Fvmk1CLRkBjrdMKo2zVfSlDAf5HofsT60LZLBiAovcsV+4ad1vblrAKBB4n1+4zXy2I0JBLj3Al\\\\\\\/LTipC7URvIZv+n2rD+26xTlsJY35d5yo5uGklavk0sLARpDlp024Xy0UGs47VRiDact64jVxe+i0e\\\\\\\/IKBa9Ed1y5K36xVmzbwGZ6jXJVvXDFtrwKMn+Skd7xt4pqF5ToMPFqEZz\\\\\\\/RViDDT6vNIYSuEiu5uO\\\\\\\/Bk2qX+BxdQ0WXlru8aEO3VhD5qp8inUgmZX+1CXZglaCU7xcgedPkF2\\\\\\\/kq4mGVZ9nXdKm6AzHIhoZcqkQWaVxZrahOp\\\\\\\/8tVZW\\\\\\\/kKYLp\\\\\\\/TIhJG1cU+xYMb5c9qcJ6gGKl8kSFV1828uxOt5KJCf7gQBjJ1lNkvMrf4QcqxumHpHZmlxxldPVdA7OIw3URuF9Gto208YGAbghyIxnVXUQ14N+bm\\\\\\\/VT0EfPmviO0gjE\\\\\\\/NuUtziDiNdPmMYdzQkqCOZvG0dTxDv+AknpKCYjxPG9ZWcgEX45ct4HlZBQWabk2dVJvzysoBSF3ZOzix23WsMs+La5tLeukcl3gNIwr\\\\\\\/Phvh3Jo\\\\\\\/+ITw0cwo8m\\\\\\\/GRZwDknVOQps4AymKoEc2rOba2ol2+eAKeiw7Py8GW\\\\\\\/o5Jo9dzqRa+Au2zjVUOw7nr9CvM7kCSL3fauwUvt3pDLy6JrG4KRBG8LW0IGrd9kOoR2g60WNeGU5lSMDV7DltXMtgERwzZBvD\\\\\\\/\\\\\\\/GgRct1sF797vMZ5cFO8Bjod3rC8PXB2sRZie8u4d1iMeBQX6+T6HKASSrcxvs2c2kDti0Ra5Ish7ma50SqqffZT4wQ51Dy+2r0HoL9WNDb3L5W2vdqC2+QWpBPrLusX94e6uUHfmsv+I7J8w3ykebj\\\\\\\/3bvOVDkhUVO6Guu11z0V\\\\\\\/JvTf6dP8xsFMqBraZT\\\\\\\/Ja6Wt6mxjWhl+uJ\\\\\\\/3Rti5jSXsT47rGI7tkBjNvkR6Ou0aAzKq+F5MbZsLRgyZRs6pE8YOhH7yYmDwMubIVGOxjf5yGKYM\\\\\\\/9wRv3hCHdQxABJRKOhyi439tzBfDRJF+OcMOERXzgsUlPXGYeGrqPGHS0vVuN\\\\\\\/yqgtZ4i+CZSBQsCFqAZqlmxOu9CnNH0yggOUHSDK08oZxuiK2khq7mnvJ1wzZ5yO8u9kDOQCPJuDk7IDkUYX8u6eSjrUC36YzjIHVp9A1i6vwpyx\\\\\\\/vRczThU+x34kp0HFF6F7dqkymvH7J4m4C\\\\\\\/LmLW5UjmWNwLgRS5HxkWDGK9L7E5ZBLHZdrfaUASvqYP43MnL2jSWRp+20jkcS89jTnDT2F2pgYcQLm54nlewzo3OCAymNB3ttehr7bv2k+qdJ85TsyINjI1+osSfJsAXh\\\\\\\/xMTbnEky3bqE2gP7uUWVPON9zIKWN5\\\\\\\/8sE\\\\\\\/pOu9w4V4xBu0W5nYNoc2yU8AQqTVoNl7+gfGByuMpHLUgTy+DICD9H4zp\\\\\\\/B1FIPXs1Gu9fKD1fer4gnZV7v5QqJwaIjm6nAGJLlbm+fy5LvnUIqcmrA875rZ4PmMl5je1rqxa9saoI\\\\\\\/IfWvuA8OGtt3p6VnABcc6o91oDJ7tMzJAmdPMK\\\\\\\/9Yk6drirCKOJeUUCQwkyIz7rY8+WZBFGyMPFe913s6RYO9sWOcZyDchYwfGf55fKA+BC\\\\\\\/cMvbFooJrX5Z2JOYqAHP2+TZLYJS9T5xuExobgc3wfdovzkctnXzFtEoJuCm8RjED+3f6DtR\\\\\\\/8soL5QWHnTwoKNorD2SkomLXcNyPfRIvuLXaRO3w7i\\\\\\\/+v4OQHqmVdi923EZdez0eioTA\\\\\\\/DAGI85wxvkY3M8cfUXBCtYDwfsTKqAf1seQXcxxbunXt7CLWE\\\\\\\/gLvoHmDq+aE4YBCBQigaQuTBccY\\\\\\\/1vXESQobmZGRWnp8EP9WJNPAnp3eamm04VeUKH7+9Ip3NTyJnMgxG6jdh0lo\\\\\\\/8rMgO9dcMLIF1oG\\\\\\\/o4ttEglN6jUOrmV5dm5lZtO3fHMFdSTrUfOwlm\\\\\\\/RGVZrjEDDX1+Db8U+QEB1RGu\\\\\\\/Ha9WK40dEYb2tJkhZ9KMizJGXjtiH\\\\\\\/guUg9iK5cT58LemxygBo1l9S+UlS71BhCJjgv7I9rHvlXxAwnPYKnsBURxn9gHGoXniduh7edW9qeS7nHZjY0DNewLUNBgfri6Zspngqpg\\\\\\\/R2gFHCntkxCRcMcKrJ\\\\\\\/BW+cKpPUkdCQaXdpU+HT\\\\\\\/zUiqHQZXKvb02dZio2t6EtApJLdALqK67zyXyJsaqRkglgKjyCMcRwFAnM9vy5dxT3CMUfsuWzXe7fujshGDGE7cpLSkL2eeTy9GQJ9zkpZqwjBPGNJkC6+nLx6h8op61KBhLRHaldjaiYMv5k6UlTdHPRaKs\\\\\\\/npEKJFstS1RXrN7VaiXYVKoS77vhMiLa1JSYtbsHcvsXSzvSiNuVoJ\\\\\\\/Tvd96cLywXau59ffEoL+uJVj0HKyLaTyT4GCohGeCHFneAjJ\\\\\\\/XcR1fHBVAGlruh7u7JNhX2c0IFko1OLWVNkJ15R5TmDYNpLYDoX53xpqb+xs23krqt\\\\\\\/6FK9Ezdv7GM\\\\\\\/BT9kkACF9GAz9HQHb4lJaALMO\\\\\\\/GNkVPwUEfOnPbb+0xpkg2UDNDdQc89kicVy7QW5Lx2HckSDc6VSTrQi05gfcohLjTtytN7AhGHJqcUmXRTcSfKYo+ibkvlecAeZ+KtUKPVsIDUGTSkWCuzvyoxIspJ6NYNnvrCw5OxSukE66N1kWa3VvmmZN5uagZz6k6s\\\\\\\/kzgXwm5les2Y2nU3MaXaz6fxnW5282UIBJgJVJDWR17gqH5idzATcdtg7YXonUCTAgOBlguZUn07pWSurQ9O5qxc1eEaAwnZi33HHos39wD5jObbYEIxVxp5A8pxnvCizC5uNtAaQXez7zDEZp2wCFRrirMzNh4tpSWQIOZTf62qDDrMgChT8KJ4iyCpFvI0zqIt2cow1LS7dANPuEp6qa1LQ8fWzDJds1x8bmTZ\\\\\\\/dGoJTCSyEI2uPf+FA\\\\\\\/+bQK0GkeDug8UWw\\\\\\\/5OEQnBApwQeT0C17FvXtEAWmF4Qk93fQPj\\\\\\\/UNVas+R3MiNlhKPTAn2bYqYRc\\\\\\\/q8ih4+09ATIaPEB1XINSIAvGHO861BjsM5atlq+qgy+azmbea4wwlBSqhPj\\\\\\\/EzLfWBgP4MAsXRIdb8aTY4hMdeNfeIQ58j75RjKYnRI3t3Q0vu5LlbWOjrWxQBDvcZeM1Dh8GMhQaTKDBRIg9IderXuqWoE75Xwzn2pdyPAKQZreuDNAepdH7afMqw9+TVVKQUvxLB8LcHn5uz4gPhgu86AuJej6aoG3wPMHMEQiMnkSQa9m0eu1fPaxB+6GwYzmfbizfjlBgNVVFu5ORugd14zCfEdiIlZjhqwkRRD3ENW5S37yFqi3pJbLvSYlNw3h+NP3tgehlEGG1SEEUiJh38mGhN49N67yeZnoFejjgMEG6DfnwkBRvaelcyCGR2ULacMuxVGP\\\\\\\/p01WlnNPNE4KuGtSQsBk09b5PgnM8qhNaOV4+4LqhMm3XPRDM9Ivz+auYxU37lQA3gtAP9xk40n3SYNuBLyxmFGyksjnFAw5YsnP+CJ93dcxCJgL15qi65O02o9n2m3VG5scDbQh1tU2vhZPXwuAMZlQyesZax1rhKcPuVA1PljoOoDlYm4PAcTr1V8d5P9GP5TFgbGtAePLV+gTHMqfUuQNSW50ZiBAMF4V4Cptgq\\\\\\\/bQ6u4LioQZ9+mRhx0sQCq+axnBRlbjKxR+3HZJrhYUIW9P3anE363qeu4frUwjECrgyyDwGNNWwKBPUX8h9\\\\\\\/HZLGcK5pB5SJLf0NTmSXuYyr1y2lY6qW5JeCAw+IWU16sJ5XReTem3RNKBCVtA05wIilDQD9qMa36JgvNk5YVXYJw4BlcbBUgNZCCxv0OX\\\\\\\/MZFXy\\\\\\\/J5pe8mGLglUHaV0QkTVKL3GJCZ9a3waiGPncvsVhAJLZoZYKYhuH5lpLGnRs1f8qyz1NhvpZZj2NZ0GOkydJQdGzWvj5S4\\\\\\\/uGuXbBKnPprzaCd9\\\\\\\/9Mjqcj60Be5mqvAHI189R+A5LcRk6lWTjyuGB9sVPwEyeYrCpSPiB0VGkAwuxYf7htjH4HNbBqIujFyjM2iSZ\\\\\\\/QTvOjrr3tvAoABF1kKeQp+BPtajWbMeg0NAhs10y3zt9onWk+HsUuYHLZCNLQAq77MoL7AT3lfj7vwM+WbtL55Rc6xVOCjIezh5p3oW+zBeHS4idQwJ3MgRF4rWgSAvr3lRIUhS+FkRN9ibl9ttJY\\\\\\\/mnOQguvntxPqjjmhJFrzM9mRZ80igZYb35wOQEKJUSkAjTTG8SXh8HhXw86UMU65lfCcsj9Dg3f7\\\\\\\/kl6w1DYnLsLiwJ921QM+OQnw\\\\\\\/ne8Z4QsLmCPMltGqh+uvkHDINeMvbkZFXHLIOpTa2CZoea+TKdL024c9Cl2SdGyeA1G1UJLBwCELgWkTte2J8p2lOfNXqdQmKHjD+RbFnoMggW+fREEDijWCeNYGpkSBIou7Lu7fzE1HNuBpowD1QPcNMcWyxoshB7P\\\\\\\/XhWw83FMC8WQwHns04D\\\\\\\/+fcZn63tIF72+IAOEh\\\\\\\/YQt47Z7g9ChDMQ9AK36D9ehVmW0Nu6NKRHwa\\\\\\\/bNAd+hhQzOmKvWzTr25wfrRR7UuBvWg0tA4Z8FRzCGhLKvf9FzUUpCmRQsNMZhUlCqjgmLq1QZw9+zxAx3z9w+KjEpImDhe\\\\\\\/6o6EfN2RGqXONGNgsPAVpNfmiI46YRxXZqimA11v+agYAUaTTD7EmDcbdCRtTI5kkOK2nWQBJvtZXCRwA6WGgqzNVyt4Zyd7tMDpYm0UTefWqsmH5PDdcVJmEQKNIR7CPhv+gCE3g6Y3skyBbGaNWUm7avNX8UvE7A6OUBxJcB3W4N04vMQh6dCOtvEs8z5o\\\\\\\/EhFxi2EZf52j2c+hd5uEVqzitehdW8tOLvg\\\\\\\/xidpki2O+o7AFj6z6kZv31CWAjHaCW0O\\\\\\\/NqDPhz5IUM0+UAgaA9ljk0RG6nD046+0ZtR\\\\\\\/l7ffYIIW71BmQzLyBJp+Dg3Dp62C8PyNolgiTGY1E3V9Ay6NRV7dJgf32OPOfyj2cBu0HgDfcAgxU6RonkksjM+IHjH\\\\\\\/pNChTwDfYWrhDFPQsrwcyJED6BsnUrvzcjiwA1nBIm1H9A9CbuVcXydA7JwIFc8UgKW5Lj+uHVVrd17fIXngQKN6Ez67OyfOYf9XIuKZgsjiIyphye\\\\\\\/3AdJZhY4o95RNZdDdjj1A3Gs5AN6xBOBsDhaS4ONazBEyS01n+cpxT0EcgPLEVv9l9TbnGSN4MkOhYa63pYwn1vChKqrE6GCWeXCd0ovSjqevlCcK88Ox5Z08CbqWpObpHLMxRskq7crq+vy\\\\\\\/qzmuhZnRXuiZVl1VAHasjxw+HI3NWeoRTtWCoqooRoVrWHqJV7vizXM7vgzTqd1FDmEklZMeuikz6NgXbBO4qwsCILla\\\\\\\/s25gQ+pArUvrZtajj5ds3QulyMAyPpPICCpmcth\\\\\\\/F3FwLnywrTBCm4HbVaGARXaAsFxqNnm84XQOY2GEQBQXIKGc+2A5\\\\\\\/TNzAGWFu4t40wPMwpVf0+5y3sd+bCdxpQEb+DzsNGKgjYBd\\\\\\\/lJmjRJMwT2RYUGJbBQTxmLD3lToyqDe3P3UfKbb8xR6Yuy++vocA9AnQItgNLWOr0mTuWahLLplJZyg65y6swpbPo1yvJOWupaCzV9z+2\\\\\\\/sAcMk2PBLvJSthcFUEx6XppoX0SbCr2DDWIJkhyDxoPCnnvHPU11WGykTbuAnOcnNLpRpomWDMqjKyaqmbeCaWt9\\\\\\\/ycS751Oyo6BabpB1o7bQ+tFjLo0DlHi3YL7eNfDDHLtfk\\\\\\\/zpTwTKJkzNaDa05iYWbNsO0ueccB2yPwlOlwXldEjfeZr6Q64mqmmAlo6Oofd1nl1lrLdVvTwJDUKNYxpGaO7wGOKG1tUAqmJAR7NHz9hif6jvjrLZmGRN9N20cO4HfHNP0cQW0DU6E0X16BVi7kLfqXQXNZvwj4eJj7P8U2uNgCtzQ4Cz1rt2baiVumjuj7wUDfPiF0dz9azfbNYimHFbIlyrtSBbv2eVUJAfNtUd4P7xx4fMGuEaum7h296TdL97eGXL7aynaKiY84GBMON9X3D\\\\\\\/uXh9h9uHFMeoAkUQUg\\\\\\\/j1AQbO9wh9LEo5pe6nYn\\\\\\\/1kDl+FeSxOMzLvaGJHgA9leBBOZ7Cm7759bnvDBjdrrUugBa8EBlhRXyLtD18VtYVKG93vO\\\\\\\/fiISRfPOJ0ULyu0A7HFTC9OQMZCpr8tUid\\\\\\\/26\\\\\\\/SOkqqPwAaLtPhCTpeD7XphPjzXnlnWxNO0ermgv9voG485p33tAH2JlsD0foIsQDDtNVV+S7kYcfTC+P0BO7eRdpWlAxozvky4fpbue4QlCOIBvhV3FuLiiAjOIMZbA5kAAQoec+hk\\\\\\\/zFyeEtf41vKeH+\\\\\\\/Vif9n51Fd67wM6DYn1ppNI3C8xJjxiI44M0dgFlwaw7HoZrbgaFjBcg2pJXJvaHj6RJiSajge+cmUUeCci50L0LYOm5vtEvtp54OK95DjJDI0+H\\\\\\\/9NXTm4hTdN3OPN2gaUkKA88taOE47XHWj29G7623DO41Dvt5khcukHSXvGdRg9rzUNc6+OGwKi7D+MBkLf\\\\\\\/ZlQq34mzrxm30+okN0prZEvYrJfyrvfeHp0e9kN2SeMrp\\\\\\\/3tEf+7hZArD90pqgIPOig6qOkfBTkKvKni7CtoD9tnDiXBn70HmFtKP9aidQPQkpGv+XKg5mvcIBEJjpN8paZRo5xv6E9+ycvoyBMiGKjdUA6HAGjJYdXRRDeREKNz0FiNskYl4UOGT7\\\\\\\/hx0TJzOn6wa5A5q9vND465VrR76Psq\\\\\\\/QqPKtUnsfSlOJ\\\\\\\/0pjuk\\\\\\\/yTItfkBUIHpNCJ4ydGQZ+UhJQqMxzw3Q3T9OMsrSQZ0In92yZPufxIVwLr\\\\\\\/19BBVi5JpI8beoxCJKdBrQObhkCjX\\\\\\\/04PqmUB+amxg7uAXv1PYjMNu2G0GsnOLuiDdWSLFv5Xxn4y1HGClrZ27bssyoTQMeJx\\\\\\\/U4NDuT65u2nhU9IC\\\\\\\/veLgbtmmVujTv2ymz3njU1tQFnl77fOpLwCUyZHfBMxFajOAyQKIQ5I8JO\\\\\\\/E9r2vp0i2ZwtC8NDwTwETjOIUJwXF5wbGLET4v5VHjkKRM0QhdgpCvsx452MHdl3Xq9Um6ssPnUroQYJGK8zywApQ6jeten\\\\\\\/827sgP7ALfdGVp\\\\\\\/n7AmYey1RspXAaDNmX5zZVu9XKXU91n8BNbx1KVv2+\\\\\\\/yhakIBCha5d7BhbP6iplsDxv8o9fi4tJ\\\\\\\/lxWFhpQPlZFvCseLd1LYc0IIEVKyzD8H6oed+AEPa0cvZyPW8V8UnkH66tLXfRt2JvtfygYdPKkz+ycsaDkJPxHBHwjPqXzXbDTrd7kB\\\\\\\/aQZtUZu7zOzAMrbzht59bHDQpliFIOLbGWBI5HR\\\\\\\/36Q2Q2NKRyEEpJaJfanvWNbV6ghS12NJcLGM3fz2rm7y9dbCsnu8koIgb13BDGs\\\\\\\/k+kXz4M4TIvSz\\\\\\\/vgV2W6L\\\\\\\/3LvrAa6M9WsguE0Jpn1KJXmDSztn6gp0tSQhbHIbupuI6VWv1iaSjOmfbrAtvmK9r05wfMRpf8kSDR6G6n2nTl11PNI883HUohlbJPxm2CUTqyiXNeYWHVtq0HHQO3fxk2UN1acULA9nHNgMON2bj91LEmbsKMPrOPrjFrOAhAzYI7YZD01yXLhymHgVL1JFviZMHNNTv3SG4gQ0nSwZbYrr2Nz7g9Vdbk8R8mgjqce4o3NYJV6XCUHnH0Fnelw7BjritYjRMMGL7VNRL1Bf743bRF309dn4n8eN59jfRPvUz6kOmkVvsF0hy7xDYY9r5yqsdTm8hBxm1ybYvck2pAsU5QXV3WSECoLaJJ7yrnoiXbreeLRj7YHBxNH0MWcr+mfZ4zr7bCxNeOL59BIkMRTSUG63Nhpx6O5BoA89U3bSyMLxUSt\\\\\\\/ugKdroRkcVxvYftL67LoPnIECqQaaX49rWDfQSQuZKJ2ArPeuWR\\\\\\\/3mBJkm5Dx0f5+3V34cqNdPfkI6QMsylFBZk3hwJJtRpVluiiNKhw7deIZfmHhMQVia24J4HXIvL4W9jAUTxMrj4bErNMs1tA9vvAc9v+OKbV1y70rNXxcFf5m39R4ySrQvCf4RBaE\\\\\\\/0kau3zqvRtSmxVXRFnUX2\\\\\\\/ChoO\\\\\\\/pgFbEfchuQkxjRBJRSSI0QiOh6MsjBcxqDLDXGRa8BWffdz2ZO4kuKRmcgKGD98QDD5aSVIT1MaBxpSi0\\\\\\\/ur5S6yXxUMOPGhywwfuqPbHo\\\\\\\/8X7IOBOelA8MbXPBt9J3OHgp6CaOvPi\\\\\\\/lBLhQY6Flp3skefED9Qz7xSYBdB1mq4bTzpEtIviXi+UkV5T9I0udwTCheD6SR6MmGgg8h7E9+e7H0VYRRGmmpAtxPYOu0XJUGAdo7vny7\\\\\\\/hFdO9X3kdK7yhkDDDKTfEjJnNtwLVVjR8o+VQsk866iXgoeloxYU55TKTW\\\\\\\/QVPfhnUMDRwZR5XKgyRrZZbZkvG1VuKvb\\\\\\\/AHa8AnKLwUw8PyNLjIbKh1oKZwJgsWkO4kKeSdXamV+rJ9hX1CtdUfaXpPUP7LXKQ2hfL5Js9auf79q6PnmQ4qUT4D7vCZhjGYMEfvvQICDNorXOXzuEfHjOdxBUur7q0kBsedkqQP4LNZwA42zjkRWKA44I5e0oMjjxDbZGLxKSOe1NXBHa6A2RaiPI4IDrpvCrSEDD6QIaICmpeaB1Tj8nCgZX2x8Z0nDRTuqputd3gdd4qqDEpdW+pWpT1aG5QpBRuZ8Em2Lxp6hK6LUalJVQThgrsgWoC1NQidvyTN4IbN397Fj0kIA3n8ENVVnQrK6YH\\\\\\\/LDWFm8BNOiBVNrKioikLR9Rgv6ESXqjncGuEhq\\\\\\\/H7PM483xk84L0Vc50YkeUCGA\\\\\\\/MjHb4FYrA+pfX6A6vBONupPn3EDBP47yy6xT\\\\\\\/wkAq94cxBHMGtzgkIr8SOG3wcVleuXT+jZZyHcsW5qmaQWNOY6Mrq+UEYwVmh3g5YaCxPGvCabDc\\\\\\\/fF8ZWk4kzuM8VhMwkh1do6iVE2347zwkmnqJsNVQNeRGSEUG6suf4Z7lDnoOiffDYgQgEh\\\\\\\/9LPJu8jSuhoBJUmL+aaHhp6qfCgxQxA0NW47YhZCdL1f1NAF4QxUK6J\\\\\\\/VqIrIiH6LiKXk3fiZjDwgz8Wfaz0aNNZHk6WdZElKHcJSanaklBPi5y86QfKkxNy0N0Q+pU\\\\\\\/S2xD9cbcWUG+VK5ApRoz9fre+tNYu+e8YB24umTYk3Cajj4NsQtkg9xAoEdMrtVJ\\\\\\\/o3C0TykM7XF8DuS5r3z8WohTDVxfGwoWVR64GPSbwxnAT6\\\\\\\/W38niYBsSZmDseuGDQpMi0TA7JvSWfX5rYMjyfW1EU6IlFpouq1sIA9MHgVqVaJf+VTFyWhqY0MFPoiRsQ67zCtUm+9g38y9eXmKSzEXoL0ddFJCvB5bKOebtMO7gRpmZw9o11YpVJTfEZlSSTfksvgd8GlD+xxIn+NyiMQugfJ31A582k\\\\\\\/ilYiAHj9hKo59prCHrwwDXHOojlQCzMk70Pgg20XInHVYjejUZ0yZDXNoop38ChkKuQgqt3vYa73yyn6gE+pJWfz1y1toO5mAPw5NtLDBWagvD5QRnbeHnsOSzd5OhFypzE7snD6+XbxUyWvW348MdwMKBufDCAknllu2CX2DtDBk+H+MEbKbGMAOe4J3QmBcm9YLmmp3WkIjw3VUZoLefH+Fszw2HhQNmCNBlfNlu0VlqOn\\\\\\\/\\\\\\\/lb90Lv9ONYI7diEHfvSzb5NfhQ+gqBWurK2SjCzh\\\\\\\/D2rzAMpOhuNKqILxDLz6hX\\\\\\\/mzOQ2Y96BIU\\\\\\\/DnW+jun47L4uZV1GPGjsbMH23OQblRySBFS8ljSXLqiLZe0hltFmNJ9aXYRljY4\\\\\\\/1fXrsUeqgCUjCf+P4g+zbHFy\\\\\\\/oQLFSM0aL9SicZgZkbDK5o7Ay7k8V9oncyI9YytwEJD5Xhr57SJqL+sZ0Z3vN6DYvGRbidJiKIP2QRIK8Cx\\\\\\\/a9sbo6FgmQMD8jUQ3deOW9DhHxwU9NJPk2kqMggdJHbg4N9uPr61qoNtEMfy8dswy4Dw5CSzaTBT6L76OVTCAs24oueJIrr4EEGYD75kAITGk9QasrS4uEc\\\\\\\/WVljwZO9grzyNx2vNI92YUpTk0KWNTMPI1dk3Lnt8K05PC61k5DNcPyH9wlpsPwRDhIP3GdXa\\\\\\\/0x0gELgVJdd1r1iLSvW\\\\\\\/Tqg1tka2KT30o\\\\\\\/m4XGhbNlfSCJgA4qCQu\\\\\\\/vns5WHEs8GQQmxt70i8+\\\\\\\/k4TeLPl\\\\\\\/E61B8DRSWAQ06dhKnrotYSXd3D0ULtUDEj\\\\\\\/3Z1iuh7XznZEN6LZ60XnbwXMwqGuLD7oBBltzozqB3DKN2XA\\\\\\\/HBsmhG1ia0pbiACeW2wp91q\\\\\\\/9RKS0p9kUlSCbf5IVcZIqL3PGfP7ob16eYh5VguGm46WPy7nSS77hUaYWQW2sBV8zdSLd+kUNCU8aYTplKqXqwo40cJU1QKRSdqn9XVMSRUjyf1Zdu0luf\\\\\\\/Mxz7le8UGqVKhO2WuwbviA90qQBfJxXzJMRmMtoS5QLOh\\\\\\\/wCoOl+nzHALBM\\\\\\\/6cRoorgOJfKvDfQz5zrw42YdiPlRQesXZ7MvkLoO5b7iZss9Cb+j9VfXVFmc1vXMP7qy\\\\\\\/QGLd\\\\\\\/vchvmHObXVFOccToGCxABr0Ukcsxq2\\\\\\\/uyB0\\\\\\\/0iVSjL3CpyrjJUooJ6sk4+Dj64R9Z3nx7wU571uX5Jn9nMdHl6TCXE7R8hnMzbg\\\\\\\/iDb7igHpj504VSJJ+CZNTcYmLZz8tUvR\\\\\\\/rsysihKErpM9bEo5SXfqSQnqHbmWYChofEYAw5IzLiwxa6DYNoj1KUksaXCXAqNIODjjO+3FZbeIrWexhrhjfVxJyb6np6UDAKlEbXhm7ZxhCj8z2\\\\\\\/zMx\\\\\\\/VbFO924q8uhDzZWRFuLSQzJcV9KfY1QtpX5wtqSpkjiZ08oIkWnUnEl6rcu0ISDf4s+DKOvTbCRQbv7\\\\\\\/XnCTBUY2Gc7i2RZ7xJwRV7SLPVuTMzEGuB4mIH25XJEslqXyrmLwfynEwd254aGhkOZLB4tb92EY3+qSEZkm0izsrNcgSA0fwjtYLdD356GIBqM7g9n9uMACoFyLLJshglo4vZMRK+Sh6YUwOe9YyBWokVyLTZxxJbhXbBTeFVorHl1nYifQnDj5fnB9f9oEM\\\\\\\/zUA1RX2toOp6u+CUErvHBHNBSfG84OqE41FZjlBBz3Teb7U53OxD4bWyb+KQu3CTluC\\\\\\\/Iqv4MUbuTy5KY1Uv2cdtSIGNU4B1pdfoQpmqCjp7aj9xJWJaq3QsqBC6CaC0VXEb0aQoOytHo0d0QLpPpeJVEeiMlVnGCQXPN4JODH7SNcikTMhKGOHRQoxobr4QZIr2ewCeicQsMIRHtUxILANG0rxTrSewdttHXZ0dxUfeRwWHbjR18xgcEI9Aaqci1AQysHEOLSgQ1vjBJm9pBwJzjrpptEFB7EYY\\\\\\\/8l0BLnooirunA16lzp1EO\\\\\\\/53mDfw0YNCv5u0t24bg9Q\\\\\\\/I2leIBD0YDqYq8+6XUr878IemUBYIEbZRU3OU7TyK1bPvVgZ9DZWUOg3vgmmr8mRBIyV1tvtf1gBZa3J\\\\\\\/vosKChKc7NNev38GelDeJhkkFIdLcrIyuW6qFYmVuAjyjWhh4uXny0NDfwBQYpozshFz909DaNIqJqe\\\\\\\/fYnHwchrIUIGamo6blh7OmxD7Q6KmMxi8KVt0XsC7JnFg+dBnafoDaMq47P6TIZGmxy4hcFHLcCoXKKKwkZNsXSr3Peoqf02+mHCsZuZH5AAmftYTZMNqzVSLkJEbIRBux7G5wFeZLkP9ETYO\\\\\\\/RUplQl9g414a+FTXcBKmyYDKjqnBf4kwIArzFjiWVoB\\\\\\\/pUoIZkVtpWgDMcRhYtasVlgzMm3R6kvLORNtuR8lYOJThOFbIqMGjVEdU0AcW5\\\\\\\/Pofx06y5gwjDfqf97t6sXui\\\\\\\/x3pkQmy+QgU84oyZxNtwiQnvWAwanFWrveaw0ULfEZiEagOMXnwS1llnh\\\\\\\/Run95XehYp3xmwDMHgcMYtCNzWFtwOoF0r4b0Ct5rX5XzyBveD26WNhOLkI7XXcuJpCtjuDgOQtHi1aNeNOZLYLAkjmHoE+H\\\\\\\/ZDueZrklyF9r\\\\\\\/\\\\\\\/aC04FoQXsPycdszc298I6NdVyaBQbc9OEYbdtqF2P1sA3pyARUcMiJQdCxj4shgGrL3nElXQlnQ8UVgX9jzwvgGAZshIT+aoc6zm7WLCIoDcC5gR1BWt6sGSfAjLGFsmxn0syXa9HuaU\\\\\\\/RkI+gY24oY\\\\\\\/y\\\\\\\/a8rcLaEdRzv1BfiWXVHeEF5Y8z2TQjbX6kNtP54DDmRjTC6Wtrx+\\\\\\\/0etnD3WWfyIbtxomnwnxhWFv1gN8pJCjSikBr1yN1WWh30Ob8WN0KZq71yQq6YqY6fYYHx68AJD0+iPF98aVmTAyXLBNNmkmEztq3xxtJCG1h0tikn9zeSB+zSZ08jzXnxVJ5CNiMmiM4fpd16DgD2ldk\\\\\\\/AlAhqI\\\\\\\/WLMW4FCzqReldRwH7LKcs8hPb0HZP4XFRMwRaULK0Y7oJ8Hy0L\\\\\\\/+35c+Tg\\\\\\\/no3+ijGlPgd+hJ0qfDdH\\\\\\\/7\\\\\\\/JWm0e4SoQj+yL2uLMJXCq88e\\\\\\\/pWBFE8OFmHAGENpZYilg744votMEb2+gKgPmuugsMrv9fYLP0Io0jip7RVRow93l0oGA7NaIxxEMevT5pQ9tnuUHOLbSyVv89nzO\\\\\\\/KtJOSQkYbq4Mv\\\\\\\/SdBieGLiFwn4Eibx4wXcZKAAQHYrTC+qEWNPRNF6o3krB9HLgHTj5rm7E7gxDh1okfHLtvcdsnwG3LAiuklTAC+ryuxA3Pry3oQ\\\\\\\/glODLS4rzCZPxXaF0rXdKYBDZrLFdRzCM0qnAM37FatLj3+E1DooCVAmm+nOawF3ZeWfGriRLYd++SN5YTTXkMNSsEVZbbPymaueVA\\\\\\\/cPkZPRqbpG21fqPftwwu0qerJL5PuiU3KSNv5yRHxTSt9F5CjMJBnQ7T+83mzU6K12jxx8Q4cb3aJvznEJT\\\\\\\/dYsbYZDIN1kcKR0hqFM90wegeu\\\\\\\/TSX1Nu66YypP7nW0qrQjD7sQzdTtsEXf9zmcz+5Clh3a0golbArVpQZJ0mXjGNI8O4Apjuyj2QjuQhg8Tx2pofxivQqdWZWbUvFOfqEvgTAX18TJaPzia4z4HBiasw1UP9ysB9stQn8Y2SRLBVwGn5NSBMx85S6i6946Oq8e5G3k5qkze2sxBdP2+dX77J7GJOImtQJzU1IOSRChjpnCgt74ZsLJegMfy6PmAKqIy4Al7GBhk16D00uTuPSQGySmqf8UHtYsZZWsLdCwim6gqg8ISiYdZIPBv8ZsaOmoCu\\\\\\\/Q8La4\\\\\\\/YMq4766oPH4hChe7l7UM9IEfu5oTbyu8VO3XzdUZHShdn5od0VPa8kWOTyAOAIiLkmWQ2vEBaweDfZrajsJzSFSo7Q5SMQFpgd6w5Xf1YcBlKpvcQ1WzPwlE1iklWpkhQipD+wJLjgKqITFymKkqjbuxzJjRYMJoAgBLWxVPwB84IWk0LIIvy6Cq6iTEcKaT0KUTgdl+yR2GbDAc1DKf7mNX\\\\\\\/w\\\\\\\/fzZe2Q1TJ0NvAx02krXgPqE49ay59FcWL7MLNykwMKawNx8mR\\\\\\\/O+6LKFPcl9ZYN6qNk33esp9L8TJTeyNG2KVo0azCOt\\\\\\\/d4QGCT2jMV++raWOaBSYlgc9y8fFtckYGx3QpB4RO1h7lilwBzgLYCr18ewCByoq4XZCqUVIo2g1SB\\\\\\\/k2fofmrLFjoIYCKWwTRcLvx20Tdt\\\\\\\/TY1wfus6iUcF+L9Z1KiYL44EDloy92mx5TNbuCJXNqUXc\\\\\\\/FeDFQg3F0ooYDt+oftMsylOHZUTFtWEG45kc9abBc+z3U8Dvp1XqqNzs5142yQW16L4VNebZEZ+ZPkr0ECXdtGCI++D8YSnanvztF8tD65cpVL9oQLYROyDaCg9QksswOtzfgHG9kjlVCf84JOl3BxDajYK404aJOhdXkB1DCfqj91HRWe+wGNdHFZCnZUeB5o8YfJSCViBQmuFjezkb6PJ1OVf6BjmnGEEaijj6p7KTgJbr4uj52U\\\\\\\/NaB\\\\\\\/TBgS20F70AjlRWgV13+oNtkyEKzGDOliGGP2eWLOW+eJXGH5vwtr1w5wy03UkKCNcNtVx+NSi0Y9ONRmaa1NDkJkjFUxwaNAl1Gh66J6iF04exQDVzLXPHTkfRMW2537C7\\\\\\\/U2DzlddRd0KAGhHrJNOmcy5xcaSqW7GdG\\\\\\\/KsgeuyVia\\\\\\\/lM87n86jej4aYI10AWvzo62nYaUnVXQ1ONOsPn8BVhxg8jg0DmiFn065pJApitQX+6aKwTL7vkBK8cF5KS3xJGXqntcjkAZL1kPO+xienWaakKTIomUCRyX9WYqY2tDS1UDjp8rlyJOSkaVKcIrD3+UGZadt97DFJm71Y7pqUbOYZfPZ9HDRRC+3e14XC01AL20Y96Fl90aYUSwzBy4F54+uiDaWoIO6vwoII1AYAohiw4+5qdu4sREi2A71fQ2RfDQpyqs9OA\\\\\\\/51ST3bBEZ6iPfI1ocE6tUr1MUfPkBHHPmpzoVSGj3XzZw3vyT5d9IgWNLYemNRDi59C9UdH\\\\\\\/X\\\\\\\/BLs2RoL814T9ZuawNSOafbbEa6JNRu3foiys+ll+3Koc\\\\\\\/maThj0JRGzEHQ4bfEUMdtQAkT3JORSMwPTnD7KuptXr011Lt5U1kIFz986m55K2UnhoIms4uHuWnQQa0Cwarpo0ICM7XAlK6zv5ylxfDpFYn96PCLzyU0ixgKgMxsVAhUORYTtoaiw7MZrLkU3Iqfk7XuzdNFWtcMe+bfWgHZThQ538DKKOlQTo2da81OPTLi9C4p77J7xlmhJPZSbFWKhf9sbMnTqbZIbA+OzOyxefSawz3OWGe3Ac1I0sLDyd5MReO3SY66tsaiDg0nyDcPKwxb4jTF3NMY7swsJMWZEy675wBNsHze4aXHR4ccY38xXVmhFui9kWGsj7gVT+uXxJeVnQLTEia5kiNcsuQ8UpTa3mVDcoRuv0pLhVnzFIB+bhFEaaEtUEA2VpMag\\\\\\\/\\\\\\\/O+8fVLIl+ZVM7hGau5PD0Gn3yzvmgM+\\\\\\\/xz6keiLW\\\\\\\/jikC7fbCGpxGjvOLWY2wViP5F+bEMrgR1+MfoQcUMqaeb\\\\\\\/c2BrM5CHPEfQQE\\\\\\\/dtL5G+TIdPL+uerXsOo0up18lJ6oS81U+vOzKWFBbz6gUcty5\\\\\\\/SSG7P+DapwgtUmyNSZ6IeDW8RWP4ss8kFTz\\\\\\\/Qp4onoZOAnqG0UKTxeFiPBS8faQkz2tRxv0J1480QsyKsM5IEMZwEXp0+QDqgEt9cyc8KAT9RhyVnaAzovrFRGmTqcIs66noj7S74C9Y+O6zh1PtIe1OGzaA5OecGnsEQwJfT71+fz+2fDGbgGyZTTaXmao+npghOdjGxUd58O9KG3SHlTXJxBiSb4ztpXCLxQYpyyOmFLWFJYSTRpZkPY5\\\\\\\/ES5V5kGo8gvSHXqoPTZ79PbPy\\\\\\\/6DgvCmKe5z7q\\\\\\\/BoCtC2zyaSL4S3CepJIUdMWoXkHoEd+3bHm2CmwO4u0dQWUV+4hnMtvRb+q4L5Yy7+08HeBIk0xdDbS\\\\\\\/R6Fem0xCN9+TDVG+ju5ZVhHxfnQw4CUZsraLbOvaMqRqxA1cM\\\\\\\/POEH+7CUM0wuWrZHZVig7zdpiiEOX\\\\\\\/TB2yKMWTzg4EVIhNAAPuj1dAUVKWbE\\\\\\\/N6yhddoTPc7OnQX5U6k2wUaWXkwY4u\\\\\\\/fxZl+6sGxoEvd6Md66PBssReYzRPP7GfqLrqCCJEwACtgAorSLDNlhnC29AsOHUZ0J63WNEjZO+eZFbeCKDwEUhOFaprdAWMBC8vlRjAS4BaGXnSbXF2zRFc+kcKBQig3OJTygsRllLroLuGchNpLljomiY\\\\\\\/WSV6SlRZobEhPnc8188\\\\\\\/tMtSVR2azLIygsrJ10ChqAMIpanqroV5YwrMSMfWC8I9sIWOzO0Fhmd5n8ZjeiUb3+uXhwntzmu3qaTGMINtyvaf7+GJGFgKG0HYKEsKbG\\\\\\\/QoQ\\\\\\\/DlhtagZuYc0laE4hnFokvKkdBzfFvPHEB8Q46DqZh3Ds9Ic\\\\\\\/yiS+npv94O64z8qNcNCHjdJYE3FZ8FcwSkIyyQIxWAb7GUB8cd9Dco6afqMRQZ5SYDdfr7dNFsJH2tW6oAeZwvm0cmKlCaVTXmPri+Fkrqto8k\\\\\\\/qhaBw\\\\\\\/xsJBigOjyvdFlyw0VZ7puhkxTHuS1N2RKbaWFd7jCsRno\\\\\\\/8XKD8H5GKc14uRQLRrHHEuAnQitLWnW6EZEKxtL1waER11143BIrw9P0f3ITGSXv0Lja1blQ4zBnAq4OTyTeBdXkk5nuLwBysTMe2VOQtNNFyGm1JXVHkwoyBdVMMKKEIeAhadoi6pv8d40dM4QPDYWr\\\\\\\/7MC6MO6qan9sX6G2X+\\\\\\\/87mc\\\\\\\/Y++s06imGwFMTVs6KhcjFZhaha5UvtQoJsJhi0cCXNKXywO7Uul4ZKTnSwsseWgvKC\\\\\\\/CW\\\\\\\/CLVeJNKr7y2sQmx2HW2Dg0TyBJuFO\\\\\\\/EkHsB\\\\\\\/GPD1mcLoJEAxorL7FD+PkNZk2J0uqnLQ2iIGONtnC8ESvyovoq87XRQPbmwpMLE\\\\\\\/vKK1SF9U6lvUJYlRTE5XakYZsm9tgc1FbCVwXtuoE2x1LbfvQn+rohPuVuQXKmTAs38zMA+2wGFnh1Q7D0WTb5jUSTig9KqqvtajmBhihiifJq33h6c0Q51qjNc7ZLblLBmwf2MLPvaTm4GAG3HDjaAFJC\\\\\\\/u1noIMUZZDdEL4EdESeRF9mD2YIsMl6+1B0Xvqo80kPAgv4BP9SfjP7NOCBnsuBVW0xqU86JUu+yGJKltDsKfshrONLvee8kDQKOOZIbCg5j1aDMSv3bgpybbWyXu0W6lPhG54esnHCCs3qW9Z\\\\\\\/ZhJT9baOwAzes9ZHFze+voOfiW6BBvmDKzt9\\\\\\\/C+YsnBCEJFD2zyD3Ovorb6+IYJNuJfI+ITxyAIvDW0aUVfYTYtRgzZxRu193Myh\\\\\\\/d1JkXKM3BTgZYqxWWsaPYaofLd3YM9AW3Xnb9XYrWGSvBbiCFtDeCYOecmZ136VEAIdftJAMteSt57mQJtMfjz8hySqKBUAoTtX9CG1dxjJplAqFpH1bEB\\\\\\\/G8ZyNLiJiUZdyeN7gpsEwC\\\\\\\/2gKyKP3IPeNmGHpkHX8hE1Oe1zmjDksXXj188Qf+k\\\\\\\/i5oga083MIY2TCJJw5d7lU8IYylD9yPkP0jA7dMfS5JyZiqK8TmOauOs2yTBswaPong1y4uILrKnLSNTxPHpaqaEEjgLfCQvPAV08owvAZT18jWQIzn8KKgKMjGXCj0agyUTRifxWlLVqHccR8Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022SuxxDlx4o8x8v6Wn2rbKTQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220150d139771f3350f83bdf0c08d435f58c72f46fbb511626605392332413bf74\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224FYO9sTEYLRLZbIo5uWrxsXFfIcVk0ug+vAsMFRpQn9l+MQ56e6fwYr5LQYBP6OzKcCLOIO0N+4sgcdFiUpsTVHILV3kQVDbZkjBlGyrgtOryzJFqMtVXP0FBqUeN0HSQtrVqI8er8eKub8dFTY4Cfz50nZmPwTT+AnZKIq5F9ZJrTqOAjdLOzFIDXNMsZ4pPhtITz1PoD41iv4w7Vi0tHoLHAOJjCO4FGlKpBTsZzTObFiCJi80\\\\\\\/DN+rrw2+y+gDOm2ZXE7rWx6BLZxfyY+eh9kUAftgrlZ4OoC45FLcF7FSe5y0M8GtCpmusH61C3FRWF3YLoSs+5YSaQGiwjmzzzT+8WgEoeLnaJkQavqlLK6pcuQnjjW0sw0hmtOVxPak5kURgtbQZW6+4YgCkNgpOtyVfMdSpt65bUt+xoJGZ\\\\\\\/yTow0pO0hkmyhoYjCpWl7zJzy3fpb\\\\\\\/v+aLq1tOaQXNszmDDgwb8C4ZJy3CPL72eNghgvii5zIleSbUcquX4XqDlH2VUdTjDH17zUCzAYBBZWC1BZK6URIV8t8BIU7LBpsyJ4GR\\\\\\\/wRSmo2N63Gvqoiyvbr91vIajWzKze9ZbPLi3Nkbllrm8NlbFVg09HubrvBvUQd119Iho\\\\\\\/l1\\\\\\\/0uk+ojFOXePMaExLkFXNe\\\\\\\/2wXGVVBi7sjAI8nsK6c61nUXjaVQiWEgGZU\\\\\\\/c1m\\\\\\\/ZgWZS16nwiKmQQkgEU+jXZHVLnPe8y1EZ0s6fh9nh5IgleOBYkZTgqpxQYzQSMjqeEjK7tBnWUteM5cYgW7xVAFwQkdN9vqia5e6mY3La9dZcvSsOK9XhFvXMi21x8yZQj\\\\\\\/\\\\\\\/XHlpQYLN9Wr9V6kYIVxkJ4Si+NZnPdDOPVsZ+AGECj11kSHwz5+d2UwDTazO19IBOGFDlRfLZ+VLELycU9c0QYzbDEXB6B6\\\\\\\/f+T6twsMcqViIHNt3mmbGuR2khMkxaf8Tb\\\\\\\/bXdfyzAsl+CjXcZEMQmjhj3xQfklzNc7WZaEvPaDQxG914R+tR3BBo1F5ghDRJVaZHDTJ+4fgWTknrMTPnYKxhbt6vqOzBF6n6Z7VdnXH0iGNQddVb5m7Is9Hz220jstPPO8y6w8RaYRFYubvylW09ECdEcy0yeuDESPbw9Iyc8JVe2uj7Wq81u3rnRJ4Yv8JHKoc4f\\\\\\\/V\\\\\\\/VniQK85KfmngOuX7F3BZFrcQG3KbWkh1cM\\\\\\\/ERDHKxQM1wdHHBM6srDetNsJWWk\\\\\\\/LmkPNAVFqDyakwZQmZtX+YloejmZRkVWnjulSMQqRG0ooqvAOnnkvxOpC8hetmIUMFvoxK11sml7yUTXhhs3\\\\\\\/dafoxT1XqDSYTOnr4Vf0267v54DdwJcx0b5j4rw53vzoWlmDBr2+VCwPCCKssFZoUFnodij4MXxXdHf0qvX0ikOh9krWseHZzvn\\\\\\\/wsnv+SuE14VtuGnZC4S1sdKjUrORZNThF3LRygouajBT29n6mwujzrYg2DyJfWWoiqnYR\\\\\\\/Whi\\\\\\\/qkyW7Xtg\\\\\\\/zlRINWKk+Lu+99XiYdPf4d0\\\\\\\/zf2cMQUKDPMc+iDZ3PkkU47fw5FB4KUMeRYe9KQGFZIfNAbhEYRBYVHtdckx4cIsNnFqM1ZcsZ90p5aZqAU2lDufSMSvcAjlqsV77r3W7U2VoRJPBIy9j0E6VWRU0H3PgTlLTn6YOmRZZE+xuEuMZxHXuiG5ZKVlDf6NsEbdhB7xq4jcGOsWaZa\\\\\\\/ElPxrAnt8qlvkEExyPptNK97SE9OpsG0vWrXfycnZOGHU4XtwzPIB0MzUj2F0joddAb2y6NX\\\\\\\/GWS1o8cTeipX8uxS2Hyo58j1uVlizJpFM16M\\\\\\\/ZaotgOCsGaSCPE5R1kpXXaDqTwvMyeS8j5mKYUbxf6n3e+xqGvC9K7LNkTMFO3T0NE\\\\\\\/vdYie3SfUUT27\\\\\\\/OHhMCBKRWV+OAn8wfOB\\\\\\\/YRyTMqV\\\\\\\/yaJ8z8iMAsGs2BLVU4AMHU0384mmNTVbO8UZCMYJs34FiMmAus65KkQT50dV0WaJP0MJhiI+RXFdqY4TuqAFvDQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00225E0wabP1B7LawB\\\\\\\/7Qc5P4g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022020e1c646701817760cb017206fdd7f36823eb8a6e2ef35b17984670c6fe9498\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459276 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/5/607" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-gFjAWozkNQhNs9maXm6X5jE3YPteML_7JwsQyVQDCzg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/5/607" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ImXdtCSBGaH6MCrpQsmYS8aiBAzU7tfZvhjbbfc4Ygo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/5/597" title="Aging Uncouples Heritability and Expression-QTL in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/5/619" title="The Yeast Ubr1 Ubiquitin Ligase Participates in a Prominent Pathway That Targets Cytosolic Thermosensitive Mutants for Degradation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459276" id="node459276" data-pisa="ggg;2/5/607" data-pisa-master="ggg;2/5/607" data-apath="/ggg/2/5/607.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparative Genetic Mapping and Discovery of Linkage Disequilibrium Across Linkage Groups in White Clover (<em>Trifolium repens</em> L.)</h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sachiko N. Isobe</span>, <span class="highwire-citation-author" data-delta="1">Hiroshi Hisano</span>, <span class="highwire-citation-author" data-delta="2">Shusei Sato</span>, <span class="highwire-citation-author" data-delta="3">Hideki Hirakawa</span>, <span class="highwire-citation-author" data-delta="4">Kenji Okumura</span>, <span class="highwire-citation-author" data-delta="5">Kenta Shirasawa</span>, <span class="highwire-citation-author" data-delta="6">Shigemi Sasamoto</span>, <span class="highwire-citation-author" data-delta="7">Akiko Watanabe</span>, <span class="highwire-citation-author" data-delta="8">Tsuyuko Wada</span>, <span class="highwire-citation-author" data-delta="9">Yoshie Kishida</span>, <span class="highwire-citation-author" data-delta="10">Hisano Tsuruoka</span>, <span class="highwire-citation-author" data-delta="11">Tsunakazu Fujishiro</span>, <span class="highwire-citation-author" data-delta="12">Manabu Yamada</span>, <span class="highwire-citation-author" data-delta="13">Mistuyo Kohara</span> and <span class="highwire-citation-author" data-delta="14">Satoshi Tabata</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">607-617;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002600 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459276" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sachiko N. Isobe</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sachiko%2BN.%2BIsobe" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Isobe%20SN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASachiko%2BN.%2BIsobe" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sisobe@kazusa.or.jp" class="" data-icon-position="" data-hide-link-title="0">sisobe@kazusa.or.jp</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Hiroshi Hisano</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hiroshi%2BHisano" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hisano%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHiroshi%2BHisano" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shusei Sato</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shusei%2BSato" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sato%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShusei%2BSato" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hideki Hirakawa</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hideki%2BHirakawa" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hirakawa%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHideki%2BHirakawa" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kenji Okumura</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Forage Crop Breeding Research Team, National Agricultural Research Center for Hokkaido Region, Sapporo, Hokkaido 062-8555, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenji%2BOkumura" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Okumura%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKenji%2BOkumura" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Kenta Shirasawa</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenta%2BShirasawa" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shirasawa%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKenta%2BShirasawa" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Shigemi Sasamoto</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shigemi%2BSasamoto" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sasamoto%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShigemi%2BSasamoto" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Akiko Watanabe</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Akiko%2BWatanabe" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Watanabe%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAkiko%2BWatanabe" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Tsuyuko Wada</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tsuyuko%2BWada" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wada%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATsuyuko%2BWada" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Yoshie Kishida</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yoshie%2BKishida" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kishida%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYoshie%2BKishida" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Hisano Tsuruoka</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hisano%2BTsuruoka" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsuruoka%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHisano%2BTsuruoka" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Tsunakazu Fujishiro</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tsunakazu%2BFujishiro" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fujishiro%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATsunakazu%2BFujishiro" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Manabu Yamada</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manabu%2BYamada" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yamada%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AManabu%2BYamada" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Mistuyo Kohara</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mistuyo%2BKohara" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kohara%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMistuyo%2BKohara" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Satoshi Tabata</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Satoshi%2BTabata" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tabata%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASatoshi%2BTabata" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/5/607" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459276" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459276/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/5/607.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459276" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459276/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/5/607.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459276" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459276/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/5/607.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459276" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459276/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/5/607.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459276" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459276/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">White clover (<em>Trifolium repens</em> L.) is an allotetraploid species (2n = 4X = 32) that is widely distributed in temperate regions and cultivated as a forage legume. In this study, we developed expressed sequence tag (EST)–derived simple sequence repeat (SSR) markers, constructed linkage maps, and performed comparative mapping with other legume species. A total of 7982 ESTs that could be assembled into 5400 contigs and 2582 singletons were generated. Using the EST sequences that were obtained, 1973 primer pairs to amplify EST-derived SSR markers were designed and used for linkage analysis of 188 F<sub>1</sub> progenies, which were generated by a cross between two Japanese plants, ‘273-7’ and ‘T17-349,’ with previously published SSR markers. An integrated linkage map was constructed by combining parental-specific maps, which consisted of 1743 SSR loci on 16 homeologous linkage groups with a total length of 2511 cM. The primer sequences of the developed EST-SSR markers and their map positions are available on <a href="http://clovergarden.jp/">http://clovergarden.jp/</a>. Linkage disequilibrium (LD) was observed on 9 of 16 linkage groups of a parental-specific map. The genome structures were compared among white clover, red clover (<em>T. pratense</em> L.), <em>Medicago truncatula</em>, and <em>Lotus japonicus</em>. Macrosynteny was observed across the four legume species. Surprisingly, the comparative genome structure between white clover and <em>M. truncatula</em> had a higher degree of conservation than that of the two clover species.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/comparative-map" class="hw-term hw-article-keyword hw-article-keyword-comparative-map" rel="nofollow">comparative map</a></li><li class="kwd"><a href="/keyword/white-clover" class="hw-term hw-article-keyword hw-article-keyword-white-clover" rel="nofollow">white clover</a></li><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">linkage disequilibrium</a></li><li class="kwd"><a href="/keyword/expressed-sequence-tag%E2%80%93simple-sequence-repeat" class="hw-term hw-article-keyword hw-article-keyword-expressed-sequence-tag&#x2013;simple-sequence-repeat" rel="nofollow">expressed sequence tag–simple sequence repeat</a></li></ul><p id="p-6">White clover (<em>Trifolium repens</em> L.) is widely distributed in temperate regions of the world and is cultivated as a forage legume mainly in grazing pasture. It propagates both by seeds and stolons, and it grows well in a wide range of soil and environmental conditions (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Frame 2005</a>). White clover is an allotetraploid species (2n = 4X = 32) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Atwood and Hill 1940</a>). The putative diploid progenitors are <em>T. occidentale</em> and <em>T. pallescens</em> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Ellison <em>et al.</em> 2006</a>). White clover has an outcrossing reproduction system and exhibits strong gametophytic self-incompatibility, which causes high levels of genetic heterozygosity in natural and synthetic populations (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Baker and Williams 1987</a>). The polyploidy and wide genetic diversity in white clover have made it difficult to achieve rapid advances in genetic and genomic studies.</p><p id="p-7">Molecular genetics studies of white clover have been conducted during the past quarter century. In the initial stage, segregation and diversity analyses were performed with restriction fragment length polymorphism (RFLP), random amplified polymorphic DNA (RAPD), and amplified fragment length polymorphism (AFLP) markers (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hughes <em>et al.</em> 1990</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gustine and Sanderson 2001</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kölliker <em>et al.</em> 2001a</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gustine <em>et al.</em> 2002</a>). In later work, a total of 117 and 677 simple sequence repeat (SSR) markers were developed by <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kölliker <em>et al.</em> (2001b)</a> and <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Barrett <em>et al.</em> (2004)</a>, respectively. The first linkage map was constructed with 78 SSR and 57 AFLP markers using an F<sub>2</sub> mapping population cross between parental inbred lines (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jones <em>et al.</em> 2003</a>). Subsequently, a linkage map suggesting homeologous pairing of linkage groups was reported by <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Barrett <em>et al.</em> (2004)</a>, which consisted of 493 SSR loci on 16 homeologous linkage groups of 1144 cM. Although several linkage maps were reported after this map (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Zhang <em>et al.</em> 2007</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Wang <em>et al.</em> 2010</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Casey <em>et al.</em> 2010</a>), it continues to be the densest linkage map published to date. Single nucleotide polymorphism (SNP) discovery has been considered a challenging task in white clover because of its allotetraploidy. Despite this difficulty, primers for 58 SNP markers and SNP sites on 20 candidate gene sequences relating to stress tolerance are reported by <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cogan <em>et al.</em> (2006)</a> and <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hand <em>et al.</em> (2008)</a>.</p><p id="p-8">Along with the advance of linkage maps, quantitative trait loci (QTL) mapping has been progressing mainly for marker-assisted selection (MAS) in white clover. Significant QTLs were identified for seed production (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Barrett <em>et al.</em> 2005</a>), for 11 morphological and reproductive traits (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Cogan <em>et al.</em> 2006</a>), and for salt stress tolerance (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Wang <em>et al.</em> 2010</a>). In addition to QTL, qualitative trait loci were mapped onto linkage groups of white clover, including the red leaf locus (<em>R</em> locus) and the self-incompatibility locus (<em>S</em> locus) (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Barrett <em>et al.</em> 2004</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Casey <em>et al.</em> 2010</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Tashiro <em>et al.</em> 2010</a>). Although these results suggest the availability of MAS or the possibility of map-based cloning of genes, obvious progress toward these goals is not reported. The lack of a sufficient number of polymorphic DNA markers and a saturated linkage map could prevent the progress of molecular genetics in white clover.</p><p id="p-9">Comparative genetic mapping is an effective strategy for sharing genetic and genomic information between model species and those with more complex genome structures (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Feuillet and Keller 2002</a>). White clover is a member of Trifolieae, which includes the genera <em>Trifolium</em>, <em>Medicago</em>, and <em>Melilotus</em> (Gilett 1985). Of the tribe Trifolieae, the genus <em>Trifolium</em> is the largest and contains approximately 255 species (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Zohary and Heller 1984</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Ellison <em>et al.</em> 2006</a>). The comparative genetic mapping of white clover was performed first by using red clover (<em>T. pretense</em> L.) and alfalfa (<em>Medicago sativa</em>) with 167 and 37 commonly mapped markers, respectively (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Zhang <em>et al.</em> 2007</a>). The result suggested the existence of putative macro-colinearity between the genomes of the two species. A further analysis was performed between white clover and the model legume <em>Medicago truncatula</em>, which has a basic chromosome number that is the same as white clover (2n = 2X = 16) (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">George <em>et al.</em> 2008</a>). A total of 159 ESTs mapped on a white clover linkage map showed significant synteny with the genome of <em>M. truncatula</em>. This result indicated the predominant colinearity between most of the homeologous groups (HG) of white clover and chromosomes of <em>M. truncatula</em>, except for F and H in white clover and chromosomes 2 and 6 in <em>M. truncatula</em>. Before this report, linkage groups of the white clover map were named using letters (A–H). <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">George <em>et al.</em> (2008)</a> renamed the linkage groups according to the syntenic chromosome number of <em>M. truncatula</em>.</p><p id="p-10">To accelerate the advance of molecular genetics in white clover, we performed EST-SSR marker development and constructed an integrated high-density linkage map. For broadening the knowledge across legume species, the genome structure of white clover was compared with that of red clover and two model legumes, <em>M. truncatula</em> and <em>Lotus japonicus</em>. The resulting EST-SSR markers, integrated linkage map, and observed macrosynteny produced by this work will be valuable resources for genetic mapping, QTL analysis, and molecular breeding of white clover in the future.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials</h3><p id="p-11">An integrated genetic linkage map was constructed using a full-sib mapping population of 188 individuals derived from a cross between ‘273-7’ as a female parent and ‘T17-349’ as a male parent. ‘273-7’ is a wild accession in the Hokkaido region in Japan, with characteristics of early flowering, large leaves, a red leaf mark, and a sparse stolon network. ‘T17-349’ was derived from ‘Hokkai 1,’ a breeding line of the National Agricultural Research Center for Hokkaido Region (Japan). ‘Hokkai 1’ was bred by a maternal line selection method, which consisted of 10 maternal lines generated in eight countries (United States, Turkey, Afghanistan, Iraq, Switzerland, France, Spain, and New Zealand). ‘T17-349’ was selected for its special characteristics of micro-leaves, late flowering, a white leaf mark, and a dense stolon network.</p></div><div id="sec-3" class="subsection"><h3>Development of EST-SSR markers</h3><p id="p-12">Total RNAs were extracted from 68 g of seedlings of a Swedish white clover variety ‘Sonja’ using the Plant RNA Purification Reagent (Invitrogen). The seeds were sown on petri dishes containing moistened filter papers, and whole seedlings were used for RNA extraction when primary leaves were fully developed. Purification of polyadenylated RNA and conversion to cDNA were performed as described previously (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Asamizu <em>et al.</em> 1999</a>). The synthesized cDNA was resolved by 1% agarose gel electrophoresis, and a fraction ranging from 1 to 3 kb was recovered. The recovered fragments were cloned into the <em>Eco</em> RI-<em>Xho</em> I site of a pBluescript II SK(‑) plasmid vector (Stratagene) and introduced into an <em>E. coli</em> ElectroTen-Blue strain (Stratagene) by electroporation. For generation of ESTs, plasmid DNA was amplified from the colonies using TempliPhi (GE Healthcare) and was subjected to sequencing using the BigDye Terminator Cycle Sequencing Kit (Applied Biosystems). The reaction mixtures were run on the automated DNA sequencer ABI PRISM 3730 (Applied Biosystems).</p><p id="p-13">Sequencing chromatograms were evaluated with PHRED (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Ewing <em>et al.</em> 1998</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Ewing and Green 1998</a>) and vector-derived sequences were trimmed with CROSSMATCH (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Ewing and Green 1998</a>). The EST reads were quality-trimmed by the PHRED quality score at a position where five ambiguous bases (PHRED score under 16) were found within 15 contiguous bases. Reads that comprised &gt;50 bp of contiguous quality were submitted to the DDBJ/EMBL/GenBank databases with the accession numbers <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=FY454661&amp;atom=%2Fggg%2F2%2F5%2F607.atom">FY454661</a> to <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=FY469874.&amp;atom=%2Fggg%2F2%2F5%2F607.atom">FY469874.</a> The PHRAP program was used for clustering of ESTs to identify non-redundant white clover ESTs (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Ewing and Green 1998</a>).</p><p id="p-14">A similarity search was performed for non-redundant white clover ESTs using the BLASTX program against protein-encoding genes deduced in the genomes of <em>Arabidopsis thaliana</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Arabidopsis Genome Initiative 2000</a>), <em>L. japonicus</em> (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Sato <em>et al.</em> 2008</a>), soybean (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Schmutz <em>et al.</em> 2010</a>), and <em>M. truncatula</em> (release 3.0; <a href="http://www.medicago.org/genome/">http://www.medicago.org/genome/</a>). The EST contigs were classified into KOG categories according to the results of BLASTX searches against amino acid sequences in the KOG set (<a href="http://www.ncbi.nlm.nih.gov/COG/">http://www.ncbi.nlm.nih.gov/COG/</a>) (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Tatusov <em>et al.</em> 2003</a>). These sequence similarities were judged to be significant when the E-value was less than 1e−10.</p><p id="p-15">SSRs ≥15 nucleotides in length, which contained all possible combinations of di-nucleotide (NN), tri-nucleotide (NNN), and tetra-nucleotide (NNNN) repeats, were identified from the non-redundant white clover ESTs using the SSRIT (Simple Sequence Repeat Identification Tool) program (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Temnykh <em>et al.</em> 2001</a>) for perfect SSRs, and the fuzznuc tool from EMBOSS version 6.1.0 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Rice <em>et al.</em> 2000</a>) for SSRs with mismatches. Primer pairs for amplification of SSR-containing regions were designed based on the flanking sequences of each SSR with the assistance of the PRIMER3 program (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Rozen and Skaletsky 2000</a>), so that amplified fragment sizes were between 90 and 300 bp in length.</p></div><div id="sec-4" class="subsection"><h3>Amplification of SSR markers and polymorphic analysis</h3><p id="p-16">DNA was extracted from young leaves of white clover using DNeasy Plant Mini Kit (Qiagen). A total of 4619 primer pairs of SSR markers, including 1973 SSR markers developed in this study (hereafter WCS markers), 2518 red clover SSR markers (RCS markers) developed by <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Sato <em>et al.</em> (2005)</a>, and 128 SSR markers on a published white clover linkage map (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Zhang <em>et al.</em> 2007</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Kölliker <em>et al.</em> 2001b</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Julier <em>et al.</em> 2003</a>; <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Barrett <em>et al.</em> 2004</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Sledge <em>et al.</em> 2005</a>), were used for comparative polymorphic analysis among the two parents and four randomly selected F<sub>1</sub> progeny. The names of markers and their sources are listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. The primer sequences of WCS markers are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002600/-/DC1/TableS1.xls">Table S1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461499/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461499/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Sources of SSR markers, numbers of markers screened, polymorphic markers, polymorphic ratio, and numbers of mapped markers and loci used for segregation analysis of the ’273-7’ × ’T17-349’ mapping population</span><div class="sb-div caption-clear"></div></div></div><p id="p-19">PCR amplification was performed in 5-µl reaction volumes using 0.6 ng of genomic DNA in 1X PCR buffer (Bioline), 3 mM MgCl<sub>2</sub>, 0.08 U of BIOTAQ DNA polymerase (Bioline), 0.8 mM dNTPs, and 0.4 µM of each primer. A modified touchdown PCR protocol was followed as described by <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Sato <em>et al.</em> (2005)</a>. The PCR products were separated by electrophoresis using 10% polyacrylamide gels. The primer pairs giving polymorphisms among the mapping parents and/or F<sub>1</sub> progenies were selected and used for segregation analysis of a mapping population of 188 progenies.</p></div><div id="sec-5" class="subsection"><h3>Linkage analysis</h3><p id="p-20">Segregation data obtained from a mapping population of 188 progeny were analyzed by a combination of a color map method (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kiss <em>et al.</em> 1998</a>), which employed a comparison of graphical genotypes for mapping, and the JoinMap program v.4 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Van Ooijen 2006</a>). Generally, the procedure for linkage map construction is two steps: grouping and ordering. In this study, the Grouping module of the JoinMap program did not give a reliable result. Therefore, the grouping step was performed carefully as described below.</p><p id="p-21">First, segregated marker loci were categorized into two parental-specific data sets by comparison of the sizes of polymorphic bands of parents and progenies. The segregation data were re-scored using the ‘HAP1’ population type codes employed in the JoinMap analysis. Next, the segregated marker loci in each parental-specific data set were roughly classified into 16 linkage groups using the color map method. Then, the robustness of the data sets for each linkage group was confirmed by the Grouping module of JoinMap using a logarithm of odds (LOD) threshold of 10. Finally, homeologous linkage groups within each parental-specific data set and corresponding linkage groups between the two parental-specific data sets were assumed by comparison of the names of bi-parental and multiple polymorphic marker loci. The locus orders in each parental-specific map were calculated by a Regression Mapping module of JoinMap. Each parental-specific data set was handled as a ‘HAP1’ population type, and the following parameters were used for the calculation: Kosambi’s mapping function, LOD ≥ 1.0, REC frequency ≤ 0.4, goodness-of-fit jump threshold for removal of loci = 5.0, number of added loci after which to perform a ripple = 1, and third round = yes.</p><p id="p-22">After this procedure, the linkage map derived from the ‘273-7’-specific data were considered reasonable, whereas an error persisted in the ‘T17-349’-specific data. This error was because one of the 16 linkage groups of the ‘T17-349’-specific data consisted of an extremely larger number of marker loci (32.4% of the total number of marker loci) than the other 15 groups, and corresponding marker loci classified to the largest linkage group were found on most of the linkage groups of the ‘273-7’-specific data. In addition, graphical genotypes of the largest linkage group showed mosaic patterns (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002600/-/DC1/FigureS1.pdf">Figure S1</a>). Therefore, we concluded that the loci belonging to multiple linkage groups were not correctly integrated into the largest linkage group. Meanwhile, most of the 16 linkage groups of the ‘273-7’-specific data showed one-on-one colinearity to the eight chromosomes in <em>M. truncatula</em> by comparative analysis. Therefore, the marker loci consisting of the largest linkage group of the ‘T17-349’-specific data were carefully disassembled into multiple linkage groups according to the classified groups of the corresponding markers of the ‘273-7’-specific data and chromosomes of <em>M. truncatula</em>, as well as their segregation pattern by color mapping. Finally, the reclassified linkage groups were confirmed again based on their robustness under the Grouping module of the JoinMap program.</p><p id="p-23">For construction of an integrated linkage map, parental-specific data sets were integrated into one data set by the Combine Groups for Mapping Integration module, and then ordered by the Regression Mapping module of JoinMap. The parameters used for the mapping module of an integrated map were the same as for the parental-specific maps.</p><p id="p-24">The GGT 2.0 program was employed to determine LD between loci mapped on the parental-specific maps and to draw graphical genotypes (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Berloo 2008</a>). To decrease the calculation volume, marker loci were selected on the linkage maps approximately every 5 cM to determine the R<sup>2</sup>, which is one of the indexes of LD. The total numbers of marker loci used for ‘273-7’- and ‘T17-349’-specific maps were 428 and 401, respectively. The names of marker loci used for the calculation are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002600/-/DC1/TableS2.xls">Table S2</a>.</p></div><div id="sec-6" class="subsection"><h3>Comparative mapping</h3><p id="p-25">To compare the genome structures of white clover and red clover, SSR markers developed in this study and those on a published white clover linkage map (<a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">Zhang <em>et al.</em> 2007</a>) were mapped on an integrated linkage map of red clover. A full-sib mapping population of 188 individuals derived from a cross between ‘HR’ and ‘R130’ was used for map construction (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Sato <em>et al.</em> 2005</a>). Accession-specific maps of ‘HR’ and ‘R130’ were previously reported with 997 and 810 marker loci, respectively, including SSR, AFLP, and RFLP (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Isobe <em>et al.</em> 2009</a>). Polymorphic analyses were performed between ‘HR’ and ‘R130’ with 1973 WCS and 128 SSR markers (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The methods for amplification of SSR markers and linkage analysis were the same as described above. An integrated linkage map of red clover was developed by combining the two parental-specific segregation data sets obtained in this and a previous study (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Isobe <em>et al.</em> 2009</a>). In addition to reconstruction of a red clover linkage map, integration of homeologous linkage groups of the white clover linkage map was performed by the Combine Groups for Mapping Integration module of JoinMap v.4. The cDNA sequences adjacent to the mapped EST-SSR markers on white and red clover maps were compared by BLASTN, and an E-value of less than 1e−20 was considered significant.</p><p id="p-26">Syntenic regions between the genomes of white clover, red clover, and two model legumes, <em>M. truncatula</em> and <em>L. japonicas</em>, were detected by identifying the conservation of relative location of genes and genomic regions. The sources of genome sequences of the two model legumes were described previously in the section <em>Development of EST-SSR Markers</em>. The cDNA sequences adjacent to the mapped EST-SSR markers on the white clover map were compared with the gene sequences in the reference genomes using BLASTX program with a cutoff E-value ≤ 1e−10. A synteny block was defined as the region where three or more conserved homologs were located within a 10-cM region in the white and red clover linkage map, and a 500-kb DNA stretch in the reference genomes. The syntenic regions were plotted using Cicros (<a href="http://circos.ca/">http://circos.ca/</a>).</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Features of white clover ESTs</h3><p id="p-27">A total of 15,214 cDNA clones were sequenced from their 5′ ends, and a total of 10,290,123 qualified bases, of which the average GC content was 42.8%, were obtained. To identify the number of independent EST species, clustering of the EST sequences was performed using the PHRAP program. As a result, 7982 potential non-redundant EST sequences were generated, including 5400 contigs and 2582 singletons. When these non-redundant EST sequences were searched for similarity against proteome databases of three legume genomes (soybean, <em>L. japonicas</em>, and <em>M. truncatula</em>) and the <em>A. thaliana</em> genome, 7082 non-redundant ESTs had significant similarity (E-value &lt; 1e−10) to the registered sequences, whereas the remaining 900 ESTs were not previously identified as sequences that show significant similarity on the published genomes for the four reference species (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002600/-/DC1/TableS3.xls">Table S3</a>).</p><p id="p-28">To investigate the functional classification of white clover ESTs, non-redundant EST sequences were compared with the eukaryotic clusters of orthologous groups (KOGs) by BLASTX and classified into KOG categories of assigned orthologous <span class="underline">g</span>roups (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Tatusov <em>et al.</em> 2003</a>). Of 7982 non-redundant white clover EST sequences, 5193 showed similarity to KOG sequences with functional classifications. The distribution of non-redundant white clover EST sequences assigned to KOG functional categories is shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002600/-/DC1/FigureS2.pdf">Figure S2</a>.</p></div><div id="sec-9" class="subsection"><h3>SSR features and marker development</h3><p id="p-29">A total of 1266 di-, tri-, and tetra-nucleotide SSRs that were ≥15 bp were identified in the non-redundant EST sequences. Provided that the total size of the non-redundant EST sequences is 6.0 Mbp, the frequency of occurrence of the SSRs in transcribed regions of the white clover genome was estimated to be one SSR in every 4.7 kb. Di-, tri-, and tetra-nucleotide SSRs accounted for 19.0%, 69.8%, and 11.1% of the identified SSRs, respectively (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The SSR motifs on the previously reported 32 EST-SSR markers [prs markers, <a id="xref-ref-5-7" class="xref-bibr" href="#ref-5">Barrett <em>et al.</em> (2004)</a>] consisted of 1 di-, 24 tri-, 4 tetra-, and 3 penta-nucleotide repeats. The main difference between the study of <a id="xref-ref-5-8" class="xref-bibr" href="#ref-5">Barrett <em>et al.</em> (2004)</a> and our study was the inclusion of penta-nucleotide repeats by <a id="xref-ref-5-9" class="xref-bibr" href="#ref-5">Barrett <em>et al.</em> (2004)</a>. Among the assigned 1266 SSR regions, qualified primer pairs could be designed on 241 SSR regions. These primer pairs were employed for validation of EST-SSR markers. In addition to the perfect SSR, “mismatch SSRs” were identified, which are SSRs showing 1- or 2-bp mismatch sequences to the repeat motif on the identified SSRs. The numbers of identified 1- and 2-bp mismatch SSRs were 1440 and 4316, respectively. To increase the numbers for the candidate EST-SSR markers, an additional 456 and 1276 primer pairs were designed on the SSR regions that allowed the presence of 1- and 2-bp mismatch motifs in the SSRs, respectively (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). As a result, a total of 1973 EST-SSR markers were designed and named WCS markers.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461444/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461444/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">SSRs in non-redundant white clover ESTs and designed EST-SSR primers</span><div class="sb-div caption-clear"></div></div></div><p id="p-30">In terms of the repeat motifs among the 1973 generated WCS markers, 1589 (81%) were tri-nucleotide repeats, whereas 139 (7%) and 245 (12%) were di- and tetra-nucleotide repeats, respectively. Three types of di-nucleotide repeats were observed, with poly (AG)n the most frequently observed, constituting 67% of the di-nucleotide repeats (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Among the tri-nucleotide repeats, the poly (AAG)n motif was most abundant (371, 23.3%), followed by poly (AAC)n (252, 15.9%), poly (GGT)n (247, 15.5%), and poly (ATC)n (230, 14.5%). The tetra-nucleotide repeats, specifically, poly (AAAG)n, poly (AAAT)n, poly (AAAC)n, and poly (AATG)n, were more commonly observed compared with other motifs, and collectively they added up to 72% of the tetra-nucleotide repeats. The details of the designed white clover EST-SSR primers, along with the corresponding SSR motif, product size, primer sequence, and images of 10% polyacrylamide gel electrophoresis, are provided on the web database at <a href="http://clovergarden.jp/">http://clovergarden.jp/</a> and in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002600/-/DC1/TableS1.xls">Table S1</a>.</p></div><div id="sec-10" class="subsection"><h3>Construction of parental-specific linkage maps</h3><p id="p-31">Polymorphisms of the white clover mapping population were examined for a total of 4619 SSR markers, including 1973 white clover SSR markers (WCS), 2518 red clover SSR markers (RCS), and 128 other SSR markers on a previously published white clover linkage map (<a id="xref-ref-39-6" class="xref-bibr" href="#ref-39">Zhang <em>et al.</em> 2007</a>) (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). As a result, 874, 305, and 99 polymorphic markers were screened in WCS, RCS, and other markers. The polymorphic ratio of WCS markers was 44.3%, whereas that of RCS markers was 12.1%. A total of 1797 polymorphic loci were identified from 1278 markers (data not shown). Multiple loci generated from single markers were indicated by attaching lowercase letters after the marker names (<em>e.g.</em> WCS0403a and WCS0403b). Of the 1797 loci, 424, 789, and 584 showed bi-parental, ‘273-7’-specific, and ‘T17-349’-specific polymorphisms, respectively.</p><p id="p-32">Of the 1173 loci showing polymorphisms on ‘273-7’, 1059 were mapped onto 16 linkage groups, representing a total map length of 2322.9 cM (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002600/-/DC1/TableS2.xls">Table S2</a>). The length of each linkage group ranged from 97.6 cM (LG6a) to 186.5 cM (LG1b). The mean locus density and segregation distortion were 2.19 cM and 35.7%, ranging from 1.43 cM·locus<sup>-1</sup> (LG8a) to 4.07 cM·locus<sup>-1</sup> (LG6a) and from 19.5% (LG3a) to 66.0% (LG2b), respectively. Most of the linkage groups, except LG6a and LG6b, showed colinearity to eight chromosomes of <em>M. truncatula</em> (data not shown). Therefore, the linkage group numbers were designated based on the corresponding <em>M. truncatula</em> chromosome number. The numbers of loci showing significant similarity with genome sequences on the corresponding <em>M. truncatula</em> chromosomes were 22 (LG1a, 41% of the total number of mapped loci), 32 (LG1b, 39%), 17 (LG2a, 31%), 20 (LG2b, 43%), 37 (LG3a, 45%), 54 (LG3b, 53%), 52 (LG4a, 52%), 32 (LG4b, 46%), 37 (LG5a, 58%), 33 (LG5b, 70%), 1 (LG6a, 4%), 6 (LG6b, 11%), 18 (LG7a, 47%), 36 (LG7b, 50%), 43 (LG8a, 48%), and 35 (LG8b, 45%). Homeologous linkage groups were distinguished by randomly attached lower case letters (<em>e.g.</em> ‘a’ and ‘b’).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461500/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461500/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Description of parental-specific and integrated maps</span><div class="sb-div caption-clear"></div></div></div><p id="p-36">As briefly described in <em>Materials and Methods</em>, the grouping procedure of the ‘T17-349’-specific map was more complicated than that of the ‘273-7’-specific map. When we classified a total of 1008 loci showing polymorphisms on ‘T17-349’ onto 16 linkage groups, we found that one of the linkage groups was consistent with a much larger number of loci (327/1008, 32.4%) than the other 15 linkage groups (data not shown). All of the loci of the largest linkage groups were successfully ordered by the JoinMap software, but the result appeared to be incorrect because graphical genotypes of mapping populations showed mosaic patterns (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002600/-/DC1/FigureS1.pdf">Figure S1</a>). Moreover, the corresponding loci of the linkage map were mapped on most of linkage groups of the ‘273-7’-specific map, and no colinearity was found between the linkage group and chromosomes of <em>M. truncatula</em>. Therefore, we concluded that the largest linkage group was consistent with loci that were originally generated from multiple chromosomes and showing similar segregation patterns. For this reason, the loci consisting of the largest linkage group of the ‘T17-349’-specific data were disassembled to multiple linkage groups according to the homeologous linkage groups of the corresponding markers of the ‘273-7’-specific map and chromosomes of <em>M. truncatula</em>, along with their segregation pattern by color mapping. As a result, the ‘T17-349’-specific map was constructed with 863 loci of 16 linkage groups, with a total length of 2450.3 cM (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002600/-/DC1/TableS2.xls">Table S2</a>). The graphical genotypes of the individuals showed less inconsistency than the previous map (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002600/-/DC1/FigureS3.pdf">Figure S3</a>). By comparison of the marker positions between the two parental-specific maps, the linkage groups of the ‘T17-349’-specific map were numbered according to the names of the ‘273-7’-specific map. The numbers of loci showing significant similarity with genome sequences on the corresponding <em>M. truncatula</em> chromosomes were 25 (LG1a, 40% of the total number of mapped loci), 14 (LG1b, 38%), 14 (LG2a, 23%), 29 (LG2b, 50%), 27 (LG3a, 53%), 37 (LG3b, 59%), 33 (LG4a, 53%), 44 (LG4b, 58%), 36 (LG5a, 56%), 35 (LG5b, 55%), 3 (LG6a, 14%), 2 (LG6b, 13%), 32 (LG7a, 57%), 24 (LG7b, 55%), 36 (LG8a, 42%), and 11 (LG8b, 28%). The length and locus density of each linkage group of the ‘T17-349’-specific map ranged from 89.6 (LG8b) to 252.3 cM (LG7a) and from 1.71 cM·locus<sup>-1</sup> (LG3b) to 7.77 cM·locus<sup>-1</sup> (LG6b), respectively. Segregation distortion (<em>P</em> &lt; 0.05) of each linkage group tended to be higher than that of ‘273-7’ and ranged from 35.9% (LG5a) to 75.8% (LG4a), with 53.3% as a mean.</p></div><div id="sec-11" class="subsection"><h3>Construction of an integrated linkage map</h3><p id="p-37">An integrated linkage map was constructed by combining the segregation data of the ‘273-7’- and ‘T17-349’-specific maps. Most of the linkage groups were successfully integrated; however, LG6a was not integrated because only two bi-parental loci were commonly mapped on the two parental-specific maps. Therefore, the number of mapped loci and the length of LG6a were calculated as follows. Number of loci = (total number of the parental-specific maps) – (number of commonly mapped markers on the parental-specific maps); length = average length of the parental-specific maps. As a result, 1743 loci generated from 1189 SSR markers were mapped onto 16 linkage groups, which totaled 2511.3 cM in length (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002600/-/DC1/TableS2.xls">Table S2</a>). The length of each linkage group varied from 122.9 (LG2a) to 207.8 cM (LG2b). The locus density and segregation distortion were 1.44 cM·locus<sup>-1</sup> and 45.6%, ranging from 0.98 cM·locus<sup>-1</sup> (LG3b) to 2.12 cM·locus<sup>-1</sup> (LG2b) and from 28.2% (LG3a) to 62.0% (LG1b), respectively.</p><p id="p-38">The numbers of bi-parental, ‘273-7’-specific, and ‘T17-349’-specific loci mapped onto the integrated linkage map were 407, 764, and 532, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002600/-/DC1/TableS4.xls">Table S4</a>). Both parental-specific and bi-parental markers mapped randomly onto most regions of the integrated linkage groups, except for several distal regions (<em>i.e.</em> top of LG1a, LG2b, LG4b, LG 5a, LG7a, and LG 7b, and bottom of LG1b, LG2b, LG4b, LG5b, and LG6b) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The largest gap between two loci on the integrated linkage groups was 19.1 cM, between RCS3956 and WCS0963a on LG7a.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/5/607/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="An integrated linkage map of white clover generated from a mapping population of &#x2018;273-7&#x2019; &#xD7; &#x2018;T17-349&#x2019;. Positions of bi-parental, &#x2018;273-7&#x2019;-specific, and &#x2018;T17-349&#x2019;-specific loci are indicated by colors of linkage bars. Those showing normal segregation of bi-parental, &#x2018;273-7&#x2019;-specific, and &#x2018;T17-349&#x2019;-specific loci are green, red, and blue, respectively. Those showing distorted segregation of bi-parental, &#x2018;273-7&#x2019;-specific, and &#x2018;T17-349&#x2019;-specific loci are yellow-green, pink, and aqua, respectively. Distorted loci are preferentially represented when multiple loci, including the distorted loci, are closely located. Bars attached to linkage groups show numbers of mapped multiple loci per each 5 cM. The orange and yellow colors indicate when all multiple loci generated from a single marker are mapped onto the same or homeologous linkage groups, respectively, whereas purple represents other cases." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-528848873" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;An integrated linkage map of white clover generated from a mapping population of &#x2018;273-7&#x2019; &#xD7; &#x2018;T17-349&#x2019;. Positions of bi-parental, &#x2018;273-7&#x2019;-specific, and &#x2018;T17-349&#x2019;-specific loci are indicated by colors of linkage bars. Those showing normal segregation of bi-parental, &#x2018;273-7&#x2019;-specific, and &#x2018;T17-349&#x2019;-specific loci are green, red, and blue, respectively. Those showing distorted segregation of bi-parental, &#x2018;273-7&#x2019;-specific, and &#x2018;T17-349&#x2019;-specific loci are yellow-green, pink, and aqua, respectively. Distorted loci are preferentially represented when multiple loci, including the distorted loci, are closely located. Bars attached to linkage groups show numbers of mapped multiple loci per each 5 cM. The orange and yellow colors indicate when all multiple loci generated from a single marker are mapped onto the same or homeologous linkage groups, respectively, whereas purple represents other cases.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/5/607/F1.medium.gif" width="297" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/2/5/607/F1.medium.gif" width="297" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/5/607/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/5/607/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461463" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-39" class="first-child">An integrated linkage map of white clover generated from a mapping population of ‘273-7’ × ‘T17-349’. Positions of bi-parental, ‘273-7’-specific, and ‘T17-349’-specific loci are indicated by colors of linkage bars. Those showing normal segregation of bi-parental, ‘273-7’-specific, and ‘T17-349’-specific loci are green, red, and blue, respectively. Those showing distorted segregation of bi-parental, ‘273-7’-specific, and ‘T17-349’-specific loci are yellow-green, pink, and aqua, respectively. Distorted loci are preferentially represented when multiple loci, including the distorted loci, are closely located. Bars attached to linkage groups show numbers of mapped multiple loci per each 5 cM. The orange and yellow colors indicate when all multiple loci generated from a single marker are mapped onto the same or homeologous linkage groups, respectively, whereas purple represents other cases.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">The number of mapped multiple loci generated from a single marker was 1.5 on average and ranged from 1 to 5 (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002600/-/DC1/TableS2.xls">Table S2</a>). Multiple loci were classified as Type I and Type II (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002600/-/DC1/TableS2.xls">Table S2</a>). Type I was defined as multiple loci identified by observation of multiple bands on 10% polyacrylamide gels. Type II was defined as those generated from single bi-parental segregation data and located on different linkage groups of the parental-specific maps. The total number of multiple loci was 962 (55.3% of all mapped loci), of which the numbers of Type I and Type II were 186 and 836, respectively (60 loci overlapped). Of the 962 multiple loci, 324 and 409 were mapped onto the same or homeologous linkage groups, respectively, whereas 229 multiple loci were mapped onto other linkage groups (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002600/-/DC1/TableS4.xls">Table S4</a>). The multiple loci were identified across all linkage groups (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), but they did not always map randomly. Homeologous multiple loci were frequently observed on LG3a and LG3b and the upper half of LG4a and LG4b, and less frequently mapped on LG2a and LG2b.</p></div><div id="sec-12" class="subsection"><h3>Detection of LD across linkage groups</h3><p id="p-41">The genome-wide LD was estimated for the marker loci mapped onto parental-specific linkage maps (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). High r<sup>2</sup> values were observed between most of the adjacent marker loci mapped onto each linkage group. When considering marker loci mapped onto different linkage groups, LD showing r<sup>2</sup> &gt; 0.5 was observed between 87 locus pairs located on LG1a, LG1b, LG2a, LG3b, LG5b, LG6a, LG6b, LG7b, and LG8a of the ‘T17-349’-specific map, whereas r<sup>2</sup> &gt; 0.5 was identified only between single marker pairs of WCS1722a (LG3b) and WCS0080b(LG7b) of the ‘273-7’-specific map. Of the regions showing significant LD across linkage groups of the ‘T17-349’-specific map, four regions had extremely high LD compared with other regions, including 75–80 cM of LG1a, 55–59 cM of LG3b, 143–156 cM of LG5b, and 54–63 cM of LG7b.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/5/607/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Patterns of LD blocks of the parental-specific maps. Upper right and lower left triangles show LD blocks of &#x2018;273-7&#x2019;-specific and &#x2018;T17-349&#x2019;-specific maps, respectively. Colors in triangles indicate magnitudes of LD (r2) as listed on the right side." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-528848873" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Patterns of LD blocks of the parental-specific maps. Upper right and lower left triangles show LD blocks of &#x2018;273-7&#x2019;-specific and &#x2018;T17-349&#x2019;-specific maps, respectively. Colors in triangles indicate magnitudes of LD (r&lt;sup&gt;2&lt;/sup&gt;) as listed on the right side.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/5/607/F2.medium.gif" width="440" height="330"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/2/5/607/F2.medium.gif" width="440" height="330"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/5/607/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/5/607/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461434" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-42" class="first-child">Patterns of LD blocks of the parental-specific maps. Upper right and lower left triangles show LD blocks of ‘273-7’-specific and ‘T17-349’-specific maps, respectively. Colors in triangles indicate magnitudes of LD (r<sup>2</sup>) as listed on the right side.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Comparison with the integrated linkage map of red clover</h3><p id="p-43">To investigate genome synteny between white clover and red clover, 2071 SSR markers developed from white clover and <em>M. truncatula</em> sequences (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>) were examined for mapping onto the red clover linkage map. As a result, 240 loci generated from 238 markers were newly mapped onto an integrated linkage map of red clover developed from a cross between ‘HR’ and ‘R130’ (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002600/-/DC1/TableS5.xls">Table S5</a>). The total number of mapped loci and the genetic length of the map were 1714 and 833.9 cM, respectively. Each of the two homeologous linkage groups of the white clover integrated map were integrated to a single linkage group to simplify the comparison of white and red clover genomes.</p><p id="p-44">Of the 1743 loci mapped onto the white clover integrated linkage map, 951 showed sequence similarities to the ESTs or genome sequences of markers mapped onto the integrated linkage map of red clover. As shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, alignment of homologs along each linkage group revealed an obvious syntenic relationship. The proposed syntenic regions were white clover (wc) HG1–red clover (rc) LG1; wc HG2–rc LG2, LG5, and LG7; wc HG3–rc LG2, LG3, and LG7; wc HG4–rc LG2, LG3, and LG4; wc HG5–rc LG2 and LG4; wc HG6–rc LG2 and LG7; wc HG7–rc LG6; and wc HG8–rc LG2, LG4, and LG5.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/5/607/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Graphical view of syntenic blocks between white and red clover. Loci generated common or orthologous markers identified by BLASTX searches with a cutoff E-value &#x2264; 1e&#x2212;10. (A) Colors of lines represent white clover homeologous linkage groups of orthologous loci. (B) Colors of lines represent red clover linkage groups of orthologous loci." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-528848873" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Graphical view of syntenic blocks between white and red clover. Loci generated common or orthologous markers identified by BLASTX searches with a cutoff E-value &#x2264; 1e&#x2212;10. (A) Colors of lines represent white clover homeologous linkage groups of orthologous loci. (B) Colors of lines represent red clover linkage groups of orthologous loci.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/5/607/F3.medium.gif" width="271" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/2/5/607/F3.medium.gif" width="271" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/5/607/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/5/607/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461452" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-45" class="first-child">Graphical view of syntenic blocks between white and red clover. Loci generated common or orthologous markers identified by BLASTX searches with a cutoff E-value ≤ 1e−10. (A) Colors of lines represent white clover homeologous linkage groups of orthologous loci. (B) Colors of lines represent red clover linkage groups of orthologous loci.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Comparison with the genomes of two model legumes, <em>M. truncatula</em> and <em>L. japonicus</em></h3><p id="p-46">Of the 1096 sequences that corresponded to mapped SSR markers, 784 and 807 showed sequence similarities to the genes of <em>M. truncatula</em>, and <em>L. japonicus</em>, respectively, and 725 were common to two genomes. By considering the genes with highest similarity score as putative orthologs, the map locations of the white clover markers and the corresponding genes of the other legumes were compared. As shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002600/-/DC1/FigureS4.pdf">Figure S4</a>, the alignment of homologous sequence pairs along each linkage group revealed an obvious syntenic relationship. Syntenic relationships seemed to be highest against <em>M. truncatula</em> (Mt), in which the syntenic relationships spanned whole chromosome between wc HG1–Mt chr1, wc HG2–Mt chr2, wc HG3–Mt chr3, wc HG5–Mt chr5, and wc HG7–Mt chr7. A segmental syntenic blocks were observed between wc HG1–Mt chr7; wc HG4–Mt chr4 and 8; and wc HG8–Mt chr4 and 8. No synteny blocks were observed between white clover HGs and Mt chr6. A segmental level of syntenic relationships were detected against <em>L. japonicus</em> (Lj) as follows: wc HG1–Lj chr5; wc HG2–Lj chr6; wc HG3–Lj chr1; wc HG4–Lj chr3 and 4; wc HG5–Lj chr2; wc HG7–Lj chr1; and wc HG8–Lj chr3 and 4.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/5/607/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Graphical view of syntenic relationship between white clover, red clover, M. truncatula, and L. japonicus. Homologous regions were identified by BLASTX searches with a cutoff E-value &#x2264; 1e&#x2212;10. Synteny blocks were defined as the region where three or more conserved homologs were located within a 10-cM region in the white and red clover linkage maps, and a 500-kb DNA stretch in the reference genomes. Syntenic regions between the two clover species and model legumes are connected by colored lines. Line colors represent white clover homeologous linkage groups and red clover linkage groups of orthologous loci. (A) Syntenic relationships between two clover species and M. truncatula. (B) Syntenic relationships between two clover species and L. japonicus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-528848873" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Graphical view of syntenic relationship between white clover, red clover, &lt;em&gt;M. truncatula&lt;/em&gt;, and &lt;em&gt;L. japonicus&lt;/em&gt;. Homologous regions were identified by BLASTX searches with a cutoff E-value &#x2264; 1e&#x2212;10. Synteny blocks were defined as the region where three or more conserved homologs were located within a 10-cM region in the white and red clover linkage maps, and a 500-kb DNA stretch in the reference genomes. Syntenic regions between the two clover species and model legumes are connected by colored lines. Line colors represent white clover homeologous linkage groups and red clover linkage groups of orthologous loci. (A) Syntenic relationships between two clover species and &lt;em&gt;M. truncatula&lt;/em&gt;. (B) Syntenic relationships between two clover species and &lt;em&gt;L. japonicus.&lt;/em&gt;&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/5/607/F4.medium.gif" width="280" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/2/5/607/F4.medium.gif" width="280" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/5/607/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/5/607/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461439" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-47" class="first-child">Graphical view of syntenic relationship between white clover, red clover, <em>M. truncatula</em>, and <em>L. japonicus</em>. Homologous regions were identified by BLASTX searches with a cutoff E-value ≤ 1e−10. Synteny blocks were defined as the region where three or more conserved homologs were located within a 10-cM region in the white and red clover linkage maps, and a 500-kb DNA stretch in the reference genomes. Syntenic regions between the two clover species and model legumes are connected by colored lines. Line colors represent white clover homeologous linkage groups and red clover linkage groups of orthologous loci. (A) Syntenic relationships between two clover species and <em>M. truncatula</em>. (B) Syntenic relationships between two clover species and <em>L. japonicus.</em></p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-48">The original aim of this study was to develop EST-SSR markers and linkage maps for molecular genetics analysis in white clover. We achieved this by generating 1973 WCS markers and an integrated map with 1743 loci. WGS markers were designed based on the accumulated 7982 non-redundant white clover EST sequences. Although the number of accumulated EST sequences was limited, the proportion of KOG functional categories represented by the accumulated EST sequences was comparable to the proportion predicted in the genomes of <em>Arabidopsis</em>, <em>L. japonicas</em>, and rice. In addition, the distribution of the identified SSR motifs in the accumulated white clover EST sequences was basically similar to the distribution found in the genomes of other legume species, such as <em>L. japonicas</em> (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Sato <em>et al.</em> 2008</a>). Therefore, no specific bias should be observed in the accumulated white clover EST sequences.</p><p id="p-49">In parallel, we discovered the existence of LD across linkage groups on the ‘T17-349’-specific map. Although the segregation pattern of polymorphic bands in an F<sub>1</sub> population of an outcrossing species is complex, most of the polymorphic bands of loci showing high LDs mapped onto different linkage groups and were clearly resolved (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002600/-/DC1/FigureS5.pdf">Figure S5</a>). In addition, the patterns of graphical genotypes of those loci were not inconsistent with other markers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002600/-/DC1/FigureS3.pdf">Figure S3</a>). Therefore, these data indicate that both the graphical genotypes and the mapping position of loci were adequate. We considered this the most unique result in this study, because linkage analysis has been generally performed under a prerequisite assumption of the absence of linkage between chromosomes. Therefore, it was impossible to identify LD across linkage groups without the assistance of reference information. In this study, we used two reference sources for identification of LD across white clover linkage groups, including another parental-specific map (‘273-7’) and a predominant macrosynteny between <em>M. truncatula</em>. This work is an example of the application of comparative genomics to reveal unique molecular genetics phenomena of species that have a complex genome structure. The LD across linkage groups was observed only in the ‘T17-349’-specific map, which suggested that it was a specific behavior of haplotype combination. Based on the experimental procedures and results of this study, it is not possible to specify the biological factors that affect the phenomenon. However, the result suggested that systematic selection was occurring to promote (or avoid) specific combinations of haplotype blocks across chromosomes. Further analysis, such as an investigation of the segregation of haplotypes of BC<sub>1</sub>F<sub>1</sub> populations derived from a cross between multiple F<sub>1</sub> and ‘T17-349,’ would be required to determine the biological factors that affect the phenomenon.</p><p id="p-50">Of the 1973 designed primer pairs of WCS markers, 1776 were amplified fragments with DNAs of the ‘273-7’×’T17-349’ F<sub>1</sub> mapping population (data not shown). To our knowledge, a total of 168 primer sequences, including 90 SSR and 78 SNP markers, are published for DNA markers generated from the white clover genome or EST sequences (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Kölliker <em>et al.</em> 2001b</a>; <a id="xref-ref-5-10" class="xref-bibr" href="#ref-5">Barrett <em>et al.</em> 2004</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Cogan <em>et al.</em> 2006</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Hand <em>et al.</em> 2008</a>). We expect that the large number of primer sequences published in this study will help advance the study of molecular genetics in white clover.</p><p id="p-51">Forty-four percent of the 1973 WCS markers showed polymorphisms in the F<sub>1</sub> mapping population. The polymorphic ratio was 2.4 times higher than that observed in polymorphic analyses of RCS markers with a red clover F<sub>1</sub> mapping population (18.2%) (<a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Sato <em>et al.</em> 2005</a>). Meanwhile, the average number of mapped loci generating a single marker in white clover was 1.47, which was 1.4 times higher than that observed in red clover (1.03) (<a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">Sato <em>et al.</em> 2005</a>). Although both white and red clover have self-incompatibility systems and are considered to maintain high degree of heterozygosity within populations, white clover showed significantly higher polymorphisms than red clover. Considering that red clover is a diploid species, the possible reason is a higher number of multiple loci caused by allotetraploidy of white clover.</p><p id="p-52">The integrated linkage map constructed in this study consisted of 1743 loci on 2511 cM. The length of the linkage map was longer than those previously published: 1144 cM in <a id="xref-ref-5-11" class="xref-bibr" href="#ref-5">Barrett <em>et al.</em> (2004)</a> and 1877 cM in <a id="xref-ref-39-7" class="xref-bibr" href="#ref-39">Zhang <em>et al.</em> (2007)</a>. The mapped loci were randomly located across linkage groups, and the average interval between loci was 1.4 cM. However, we were unsuccessful in obtaining an integrated linkage group of LG6a because of the low number of bridging loci. Therefore, we concluded that the integrated linkage map was almost saturated except for LG6a. The segregation distortion ratio varied between parental-specific maps as well as among linkage groups. Most of the linkage groups in the ‘T17-349’-specific map showed a higher segregation distortion ratio than those observed in the ‘273-7’-specific map. By contrast, no common pattern was observed for variation of segregation distortion among linkage groups between the parental-specific maps. Therefore, it was concluded that the behavior of segregation distortion in white clover was specific for haplotype combination.</p><p id="p-53"><em>T. occidentale</em> and <em>T. pallescens</em> are putative diploid progenitors of white clover. In this study, we randomly suffixed ‘a’ and ‘b’ to each homeologous linkage group without consideration of their origin. <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Casey <em>et al.</em> (2010)</a> pointed out a risk of random suffixing, because it would mislead identification of two subgenome groups. We temporarily added suffixes for identification of the 16 linkage groups, but the suffixes were replaced immediately with other suffixes that reflect the progenitor of the subgenomes. <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Hand <em>et al.</em> (2008)</a> demonstrated the identification of progenitors of subgenomes by an <em>in vitro</em> gene-associated SNP discovery approach and found close similarity between the genome of <em>T. occidentale</em> and the linkage group LG3(A)O. In their study, progenitor-specific SNPs were identified on stress tolerance–related genes. We consider that SNPs on amplicons of mapped EST-SSR markers also can be used for identification of the progenitor of genomes. By application of the approach reported by <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Hand <em>et al.</em> (2008)</a>, it would be possible to identify progenitors of all homeologous linkage groups in white clover.</p><p id="p-54">Significant synteny blocks were identified between white clover and red clover. Although the identified synteny blocks in this study showed more complex patterns, the results are consistent with those identified by <a id="xref-ref-39-8" class="xref-bibr" href="#ref-39">Zhang <em>et al.</em> (2007)</a>. <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">George <em>et al.</em> (2008)</a> compared genome structures between white clover and <em>M. truncatula</em>, and they identified predominant synteny between the two species, except HG2(F) and HG6(H) in white clover and chr2 and chr6 in <em>M. trancatula</em>. Although only 14 markers were commonly mapped between the two studies, most of our results agree with <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">George <em>et al.</em> (2008)</a>. Prominent synteny was observed between wc HG2 and Mt chr2. Translocation was observed between wc HG1 and Mt chr7, wc HG4 and Mt chr8, and wc HG8 and Mt chr4. It is interesting that clearer macrosynteny was observed between white clover and <em>M. truncatula</em> than between white and red clover. Because the ancestor chromosome number in the genus <em>Trifolium</em> is supposed to be 2n = 16, our results suggest that the genome structures of white clover and <em>M. truncatula</em> have not drastically diverged after genus <em>Trifolium</em> and <em>Medicago</em> diverged (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Ellison <em>et al.</em> 2006</a>). White clover belongs to the section <em>Trifoliastum</em>, which consists of species with a basic chromosome number of 8, whereas red clover belongs to the section <em>Trifolium</em>, which consists of species with basic chromosome numbers of 5,6,7,8, and 24 (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Ellison <em>et al.</em> 2006</a>). Therefore, genome rearrangement of red clover might have happened after the divergence of white clover and red clover into different sections. In our results, HG6 consisted of fewer loci than other HGs, and chromosome 6 of <em>M. truncatula</em> did not show significant macrosynteny between two clover species. <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">O’Bleness (2008)</a> described that chromosome 6 of <em>M. truncatula</em> was different from other chromosomes: it is the smallest and has multiple heterochromatic sites and retroviral elements scattered throughout its arm. The unique structure of the chromosome might be conserved in white clover HG6 and cause the difficulty of mapping EST-SSR loci.</p><p id="p-55">Along with recent remarkable advances of technologies in genome analysis, knowledge of the genetics and genomics of plant species has rapidly progressed. However, there are still many hurdles in genetic and genomic analysis in polyploid and outcrossing species because of the difficulty in distinguishing between heterozygous, paralogous, and homeologous sequences. There are still many unresolved issues in the genetic analysis of polyploidy and outcrossing species, such as QTL identification, gene expression, and inbreeding depression. We anticipate that the EST-SSR markers and linkage maps developed in this study will accelerate the progress of genetics in white clover and other polyploid and outcrossing species.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-56">We thank Brent Barrett and Andrew Griffiths from AgResearch Limited, Grassland Research Center, for providing many suggestions for linkage map construction in white clover. This work was supported by the Kazusa DNA Research Institute Foundation and the National Agricultural Research Center for Hokkaido Region.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002600/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002600/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">Sequence data from this article have been deposited with the EMBL/GenBank Data Libraries under accession nos. FY454661–FY469874.</p></li><li class="fn" id="fn-9"><p id="p-57"><em>Communicating editor: S. A. Jackson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-11-09"><span class="received-label">Received </span>November 9, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-03-16"><span class="accepted-label">Accepted </span>March 16, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Isobe <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.1" data-doi="10.1038/35048692"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Arabidopsis Genome Initiative</span>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Analysis of the genome sequence of the flowering plant <em>Arabidopsis thaliana</em></span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">408</span>: <span class="cit-fpage">796</span>–<span class="cit-lpage">815</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D408%26rft.issue%253D6814%26rft.spage%253D796%26rft.epage%253D815%26rft.atitle%253DAnalysis%2Bof%2Bthe%2Bgenome%2Bsequence%2Bof%2Bthe%2Bflowering%2Bplant%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35048692%26rft_id%253Dinfo%253Apmid%252F11130711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35048692&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11130711&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F607.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165831300037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Asamizu</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukuzawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tabata</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">A large scale structural analysis of cDNA in a unicellular green alga <em>Chlamydomonas reinhardtii</em>. I. Generation of 3433 non-redundant expressed sequence tags</span>. <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">369</span>–<span class="cit-lpage">373</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft.volume%253D31%26rft.spage%253D369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.3" data-doi="10.2307/2436899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1940</span> <span class="cit-article-title">The regularity of meiosis in microsporocytes of <em>Trifolium repens</em></span>. <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">730</span>–<span class="cit-lpage">735</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft.volume%253D27%26rft.spage%253D730%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2436899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2436899&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-2.5.607.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">W. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-comment"><em>White Clover</em>. CABI Publishing, Wallingford, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schreiber</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellison</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mercer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A microsatellite map of white clover</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">596</span>–<span class="cit-lpage">608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBarrett%26rft.auinit1%253DB.%26rft.volume%253D109%26rft.issue%253D3%26rft.spage%253D596%26rft.epage%253D608%26rft.atitle%253DA%2Bmicrosatellite%2Bmap%2Bof%2Bwhite%2Bclover.%26rft_id%253Dinfo%253Apmid%252F15103407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15103407&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F607.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223097700017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.6" data-doi="10.2135/cropsci2004.0679"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woodfield</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A QTL analysis of white clover seed production</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1844</span>–<span class="cit-lpage">1850</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBarrett%26rft.auinit1%253DB.%2BA.%26rft.volume%253D45%26rft.issue%253D5%26rft.spage%253D1844%26rft.epage%253D1850%26rft.atitle%253DA%2BQTL%2BAnalysis%2Bof%2BWhite%2BClover%2BSeed%2BProduction%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2004.0679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2004.0679&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000231764400023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.7" data-doi="10.1093/jhered/esm109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berloo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">GGT2.0: Versatile Software for visualization and analysis of genetic data</span>. <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">232</span>–<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesm109%26rft_id%253Dinfo%253Apmid%252F18222930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6Ijk5LzIvMjMyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi81LzYwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.8" data-doi="10.1007/s00122-010-1330-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casey</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milbourne</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barth</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Febrer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genetic location of the self-incompatibility locus in white clover (<em>Trifolium repens</em> L.)</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">567</span>–<span class="cit-lpage">576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DCasey%26rft.auinit1%253DN.%2BM.%26rft.volume%253D121%26rft.issue%253D3%26rft.spage%253D567%26rft.epage%253D576%26rft.atitle%253DThe%2Bgenetic%2Blocation%2Bof%2Bthe%2Bself-incompatibility%2Blocus%2Bin%2Bwhite%2Bclover%2B%2528Trifolium%2Brepens%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1330-9%26rft_id%253Dinfo%253Apmid%252F20383486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1330-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20383486&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F607.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.9" data-doi="10.1007/s00122-006-0241-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cogan</span>  <span class="cit-name-given-names">N. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abberton</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kearney</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Individual and multi-environment combined analyses identify QTLs for morphogenetic and reproductive development traits in white clover (<em>Trifolium repens</em> L.)</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet</abbr>. <span class="cit-vol">112</span>: <span class="cit-fpage">1401</span>–<span class="cit-lpage">1415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DCogan%26rft.auinit1%253DN.%2BO.%26rft.volume%253D112%26rft.issue%253D8%26rft.spage%253D1401%26rft.epage%253D1415%26rft.atitle%253DIndividual%2Band%2Bmulti-environment%2Bcombined%2Banalyses%2Bidentify%2BQTLs%2Bfor%2Bmorphogenetic%2Band%2Breproductive%2Bdevelopment%2Btraits%2Bin%2Bwhite%2Bclover%2B%2528Trifolium%2Brepens%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0241-2%26rft_id%253Dinfo%253Apmid%252F16699790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0241-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16699790&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F607.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237509700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.10" data-doi="10.1016/j.ympev.2006.01.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellison</span>  <span class="cit-name-given-names">N. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liston</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steiner</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">N. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Molecular phylogenetics of the clover genus (<em>Trifolium</em>-Leguminosae)</span>. <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">688</span>–<span class="cit-lpage">705</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.stitle%253DMol%2BPhylogenet%2BEvol%26rft.aulast%253DEllison%26rft.auinit1%253DN.%2BW.%26rft.volume%253D39%26rft.issue%253D3%26rft.spage%253D688%26rft.epage%253D705%26rft.atitle%253DMolecular%2Bphylogenetics%2Bof%2Bthe%2Bclover%2Bgenus%2B%2528Trifolium--Leguminosae%2529.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2006.01.004%26rft_id%253Dinfo%253Apmid%252F16483799%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2006.01.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16483799&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F607.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238155300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.11" data-doi="10.1101/gr.8.3.186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Base-calling of automated sequencer traces using <em>phred</em>. II. Error probabilities</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">186</span>–<span class="cit-lpage">194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.8.3.186%26rft_id%253Dinfo%253Apmid%252F9521922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1101/gr.8.3.186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9521922&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F607.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.12" data-doi="10.1101/gr.8.3.175"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendl</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Base-calling of automated sequencer traces using <em>phred</em>. I. Accuracy assessment</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DEwing%26rft.auinit1%253DB.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D175%26rft.epage%253D185%26rft.atitle%253DBase-Calling%2Bof%2BAutomated%2BSequencer%2BTraces%2BUsing%2BPhred.%2B%2BI.%2BAccuracy%2BAssessment%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.8.3.175%26rft_id%253Dinfo%253Apmid%252F9521921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjgvMy8xNzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzUvNjA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.13" data-doi="10.1093/aob/mcf008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feuillet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Comparative genomics in the grass family: molecular characterization of grass genome structure and evolution</span>. <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2BBotany%26rft.stitle%253DAnn%2BBot%26rft.aulast%253DFEUILLET%26rft.auinit1%253DC.%26rft.volume%253D89%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D10%26rft.atitle%253DComparative%2BGenomics%2Bin%2Bthe%2BGrass%2BFamily%253A%2BMolecular%2BCharacterization%2Bof%2BGrass%2BGenome%2BStructure%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcf008%26rft_id%253Dinfo%253Apmid%252F12096816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjY6Ijg5LzEvMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNS82MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-2.5.607.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frame</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source">Forage Legumes for Temperate Grasslands</span>. <span class="cit-publ-name">Science Publishers, Inc.</span>, <span class="cit-publ-loc">New Hampshire</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sawbridge</span>  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cogan</span>  <span class="cit-name-given-names">N. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gendall</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Comparison of genome structure between white clover and <em>Medicago truncatula</em> supports because group nomenclature based on conserved synteny</span>. <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">905</span>–<span class="cit-lpage">911</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DGeorge%26rft.auinit1%253DJ.%26rft.volume%253D51%26rft.issue%253D11%26rft.spage%253D905%26rft.epage%253D911%26rft.atitle%253DComparison%2Bof%2Bgenome%2Bstructure%2Bbetween%2Bwhite%2Bclover%2Band%2BMedicago%2Btruncatula%2Bsupports%2Bhomoeologous%2Bgroup%2Bnomenclature%2Bbased%2Bon%2Bconserved%2Bsynteny.%26rft_id%253Dinfo%253Apmid%252F18956023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18956023&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F607.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-book no-rev-xref" id="cit-2.5.607.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gillett</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-comment">Taxonomy and morphology, pp. 7–69 in <em>Clover Science and Technology</em>, edited by N. L. Taylor. American Society of Agronomy, Wisconsin</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gustine</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanderson</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Molecular analysis of white clover population structure in grazed swards during two growing seasons</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">1143</span>–<span class="cit-lpage">1149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DGustine%26rft.auinit1%253DD.%2BL.%26rft.volume%253D41%26rft.issue%253D4%26rft.spage%253D1143%26rft.epage%253D1149%26rft.atitle%253DMolecular%2BAnalysis%2Bof%2BWhite%2BClover%2BPopulation%2BStructure%2Bin%2BGrazed%2BSwards%2Bduring%2BTwo%2BGrowing%2BSeasons%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000170881200024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.18" data-doi="10.2135/cropsci2002.0343"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gustine</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voigt</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brummer</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papadopoulos</span>  <span class="cit-name-given-names">Y. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genetic variation of RAPD markers for North American white clover collections and cultivars</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DGustine%26rft.auinit1%253DD.%2BL.%26rft.volume%253D42%26rft.issue%253D2%26rft.spage%253D343%26rft.epage%253D347%26rft.atitle%253DGenetic%2BVariation%2Bof%2BRAPD%2BMarkers%2Bfor%2BNorth%2BAmerican%2BWhite%2BClover%2BCollections%2Band%2BCultivars%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2002.0343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2002.0343&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000174093500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hand</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponting</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drayton</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawless</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cogan</span>  <span class="cit-name-given-names">N. O. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Identification of homologous, because and paralogous sequence variants in an outbreeding allopolyploid species based on comparison with progenitor taxa</span>. <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">293</span>–<span class="cit-lpage">304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BGenet.%2BGenomics%26rft.volume%253D4%26rft.spage%253D293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.20" data-doi="10.1007/BF00028776"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharif</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunn</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oxtoby</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pancoro</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Restriction fragment length polymorphism segregation analysis of the Li locus in <em>Trifolium repens</em> L</span>. <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">407</span>–<span class="cit-lpage">414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DHughes%26rft.auinit1%253DM.%2BA.%26rft.volume%253D14%26rft.issue%253D3%26rft.spage%253D407%26rft.epage%253D414%26rft.atitle%253DRestriction%2Bfragment%2Blength%2Bpolymorphism%2Bsegregation%2Banalysis%2Bof%2Bthe%2BLi%2Blocus%2Bin%2BTrifolium%2Brepens%2BL.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00028776%26rft_id%253Dinfo%253Apmid%252F1983287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00028776&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1983287&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F607.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.21" data-doi="10.1186/1471-2229-9-57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Isobe</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kölliker</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hisano</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasamoto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wada</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Construction of a consensus linkage map for red clover (<em>Trifolium pratense</em> L.)</span>. <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DIsobe%26rft.auinit1%253DS.%26rft.volume%253D9%26rft.spage%253D57%26rft.epage%253D57%26rft.atitle%253DConstruction%2Bof%2Ba%2Bconsensus%2Blinkage%2Bmap%2Bfor%2Bred%2Bclover%2B%2528Trifolium%2Bpratense%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-9-57%26rft_id%253Dinfo%253Apmid%252F19442273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-9-57&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19442273&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F607.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.22" data-doi="10.1016/S0168-9452(03)00212-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drayton</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abberton</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michaelson-Yeates</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">An SSR and AFLP molecular marker-based genetic map of white clover (<em>Trifolium repens</em> L.)</span>. <abbr class="cit-jnl-abbrev">Plant Sci.</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">531</span>–<span class="cit-lpage">539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSci.%26rft.volume%253D165%26rft.spage%253D531%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9452%252803%252900212-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9452(03)00212-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000184742100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.23" data-doi="10.1186/1471-2229-3-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Julier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flajoulot</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barre</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardinet</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santoni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Construction of two genetic linkage maps in cultivated tetraploid alfalfa (<em>Medicago sativa</em>) using microsatellite and AFLP markers</span>. <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DJulier%26rft.auinit1%253DB.%26rft.volume%253D3%26rft.spage%253D9%26rft.epage%253D9%26rft.atitle%253DConstruction%2Bof%2Btwo%2Bgenetic%2Blinkage%2Bmaps%2Bin%2Bcultivated%2Btetraploid%2Balfalfa%2B%2528Medicago%2Bsativa%2529%2Busing%2Bmicrosatellite%2Band%2BAFLP%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-3-9%26rft_id%253Dinfo%253Apmid%252F14683527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-3-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14683527&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F607.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kiss</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kereszt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiss</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endre</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Colormapping: a non-mathematical procedure for genetic mapping</span>. <abbr class="cit-jnl-abbrev">Acta Biol. Hung.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BBiol.%2BHung.%26rft.volume%253D19%26rft.spage%253D125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.25" data-doi="10.1023/A:1012048103585"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kölliker</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jahufer</span>  <span class="cit-name-given-names">M. Z. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forster</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2001a</span> <span class="cit-article-title">Bulked AFLP analysis for the assessment of genetic diversity in white clover (<em>Trifolium repens</em> L.)</span>. <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">305</span>–<span class="cit-lpage">315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D121%26rft.spage%253D305%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1012048103585%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1012048103585&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000171802500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.26" data-doi="10.1007/s001220051662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kölliker</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drayton</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupa</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forster</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2001b</span> <span class="cit-article-title">Development and characterisation of simple sequence repeat (SSR) markers for white clover (<em>Trifolium repens</em> L.)</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">416</span>–<span class="cit-lpage">424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D102%26rft.spage%253D416%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220051662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220051662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000167232800030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-other" id="cit-2.5.607.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Bleness</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-comment">Sequencing and analysis of <em>Medicago truncatula</em> chromosome 6 and evolution of the RPG1 resistance cluster in soybean. Ph.D. Thesis, University of Oklahoma, Norman. Publication no. 3315900</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.28" data-doi="10.1016/S0168-9525(00)02024-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longden</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bleasby</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">EMBOSS: the European Molecular Biology Open Software Suite</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">276</span>–<span class="cit-lpage">277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DRice%26rft.auinit1%253DP.%26rft.volume%253D16%26rft.issue%253D6%26rft.spage%253D276%26rft.epage%253D277%26rft.atitle%253DEMBOSS%253A%2Bthe%2BEuropean%2BMolecular%2BBiology%2BOpen%2BSoftware%2BSuite.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252800%252902024-2%26rft_id%253Dinfo%253Apmid%252F10827456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(00)02024-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10827456&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F607.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087457200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-2.5.607.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rozen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skaletsky</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-comment">Primer3 on the WWW for general users and for biologist programmers, pp. 365–386 in <em>Bioinformatics Methods and Protocols</em>, edited by S. Misener and S. A. Krawetz. Humana Press, Totowa, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.30" data-doi="10.1093/dnares/dsi018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isobe</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asamizu</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohmido</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kataoka</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Comprehensive structural analysis of the genome of red clover (<em>Trifolium pretense</em> L.)</span>. <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BResearch%26rft.stitle%253DDNA%2BRes%26rft.volume%253D12%26rft.issue%253D5%26rft.spage%253D301%26rft.epage%253D364%26rft.atitle%253DComprehensive%2BStructural%2BAnalysis%2Bof%2Bthe%2BGenome%2Bof%2BRed%2BClover%2B%2528Trifolium%2Bpratense%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsi018%26rft_id%253Dinfo%253Apmid%252F16769692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjEyLzUvMzAxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi81LzYwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.31" data-doi="10.1093/dnares/dsn008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaneko</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asamizu</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome structure of the legume, <em>Lotus japonicus</em></span>. <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">227</span>–<span class="cit-lpage">239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsn008%26rft_id%253Dinfo%253Apmid%252F18511435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjE1LzQvMjI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi81LzYwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmutz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlueter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitros</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome sequence of the palaeopolyploid soybean</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D14%26rft.spage%253D178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.33" data-doi="10.1007/s00122-005-0038-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sledge</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ray</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">An expressed sequence tag SSR map of tetraploid alfalfa (<em>Medicago sativa</em> L.)</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">980</span>–<span class="cit-lpage">992</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSledge%26rft.auinit1%253DM.%2BK.%26rft.volume%253D111%26rft.issue%253D5%26rft.spage%253D980%26rft.epage%253D992%26rft.atitle%253DAn%2Bexpressed%2Bsequence%2Btag%2BSSR%2Bmap%2Bof%2Btetraploid%2Balfalfa%2B%2528Medicago%2Bsativa%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-0038-8%26rft_id%253Dinfo%253Apmid%252F16075206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-0038-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16075206&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F607.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232598600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.34" data-doi="10.2135/cropsci2009.08.0457"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tashiro</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monteros</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouton</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parrott</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Leaf trait coloration in white clover and molecular mapping of the red midrib and leaflet number traits</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1260</span>–<span class="cit-lpage">1268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DTashiro%26rft.auinit1%253DR.%2BM.%26rft.volume%253D50%26rft.issue%253D4%26rft.spage%253D1260%26rft.epage%253D1268%26rft.atitle%253DLeaf%2BTrait%2BColoration%2Bin%2BWhite%2BClover%2Band%2BMolecular%2BMapping%2Bof%2Bthe%2BRed%2BMidrib%2Band%2BLeaflet%2BNumber%2BTraits%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.08.0457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.08.0457&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.35" data-doi="10.1186/1471-2105-4-41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tatusov</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fedorova</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobs</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiryutin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The COG database: an updated version includes eukaryotes</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DTatusov%26rft.auinit1%253DR.%2BL.%26rft.volume%253D4%26rft.spage%253D41%26rft.epage%253D41%26rft.atitle%253DThe%2BCOG%2Bdatabase%253A%2Ban%2Bupdated%2Bversion%2Bincludes%2Beukaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-4-41%26rft_id%253Dinfo%253Apmid%252F12969510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-4-41&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12969510&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F607.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.36" data-doi="10.1101/gr.184001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Temnykh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeClerck</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lukashova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipovich</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cartinhour</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Computational and experimental analysis of microsatellites in rice (<em>Oryza sativa</em> L.): frequency, length variation, transposon associations, and genetic marker potential</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1441</span>–<span class="cit-lpage">1452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DTemnykh%26rft.auinit1%253DS.%26rft.volume%253D11%26rft.issue%253D8%26rft.spage%253D1441%26rft.epage%253D1452%26rft.atitle%253DComputational%2Band%2BExperimental%2BAnalysis%2Bof%2BMicrosatellites%2Bin%2BRice%2B%2528Oryza%2Bsativa%2BL.%2529%253A%2BFrequency%252C%2BLength%2BVariation%252C%2BTransposon%2BAssociations%252C%2Band%2BGenetic%2BMarker%2BPotential%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.184001%26rft_id%253Dinfo%253Apmid%252F11483586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjExLzgvMTQ0MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNS82MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-2.5.607.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-source">JoinMAP 4: Software for the Calculation of Genetic Linkage Maps in Experimental Populations</span>. <span class="cit-publ-name">Kyazma B.V.</span>, <span class="cit-publ-loc">Wageningen, The Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.38" data-doi="10.1007/s00122-009-1179-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drayton</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cogan</span>  <span class="cit-name-given-names">N. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baillie</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Identification of genetic factors influencing salt stress tolerance in white clover (<em>Trifolium repens</em> L.) by QTL analysis</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">607</span>–<span class="cit-lpage">619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D120%26rft.issue%253D3%26rft.spage%253D607%26rft.epage%253D619%26rft.atitle%253DIdentification%2Bof%2Bgenetic%2Bfactors%2Binfluencing%2Bsalt%2Bstress%2Btolerance%2Bin%2Bwhite%2Bclover%2B%2528Trifolium%2Brepens%2BL.%2529%2Bby%2BQTL%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1179-y%26rft_id%253Dinfo%253Apmid%252F19865805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1179-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19865805&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F607.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.607.39" data-doi="10.1007/s00122-007-0523-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sledge</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouton</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome mapping of white clover (<em>Trifolium repens</em> L.) and comparative analysis within the Trifolieae using cross-species SSR markers</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">1367</span>–<span class="cit-lpage">1378</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DY.%26rft.volume%253D114%26rft.issue%253D8%26rft.spage%253D1367%26rft.epage%253D1378%26rft.atitle%253DGenome%2Bmapping%2Bof%2Bwhite%2Bclover%2B%2528Trifolium%2Brepens%2BL.%2529%2Band%2Bcomparative%2Banalysis%2Bwithin%2Bthe%2BTrifolieae%2Busing%2Bcross-species%2BSSR%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0523-3%26rft_id%253Dinfo%253Apmid%252F17356868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0523-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17356868&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F607.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246360300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-book" id="cit-2.5.607.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zohary</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heller</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-source">The Genus Trifolium</span>. <span class="cit-publ-name">Israel Academy of Sciences and Humanities</span>, <span class="cit-publ-loc">Jerusalem</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/5/607.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/5/597" title="Aging Uncouples Heritability and Expression-QTL in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/5/619" title="The Yeast Ubr1 Ubiquitin Ligase Participates in a Prominent Pathway That Targets Cytosolic Thermosensitive Mutants for Degradation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/5"><span>Volume 2 Issue 5, May 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/5.cover-source.jpg?itok=3sppLJvG" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/5/607" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NPSxYNNig6H-v6d0xhlu1CS1jFgU_RVMUBj8akai4xQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/5/607" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/5/607" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/5/607" class="active" data-icon-position="" data-hide-link-title="0">Comparative Genetic Mapping and Discovery of Linkage Disequilibrium Across Linkage Groups in White Clover (Trifolium repens L.)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459276" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-rEHjpP_ufTY423SGsi-IRtexuG2P5BXzVphQsgQMjuA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/5/607.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/5/607" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-k7ggyCAwgZt99aYVnr4Fm8HeOGfEyizsThlopeVBn6E" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/5/607" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/5/607.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459276" id="node459276" data-pisa="ggg;2/5/607" data-pisa-master="ggg;2/5/607" data-apath="/ggg/2/5/607.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparative Genetic Mapping and Discovery of Linkage Disequilibrium Across Linkage Groups in White Clover (<em>Trifolium repens</em> L.)</h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sachiko N. Isobe</span>, <span class="highwire-citation-author" data-delta="1">Hiroshi Hisano</span>, <span class="highwire-citation-author" data-delta="2">Shusei Sato</span>, <span class="highwire-citation-author" data-delta="3">Hideki Hirakawa</span>, <span class="highwire-citation-author" data-delta="4">Kenji Okumura</span>, <span class="highwire-citation-author" data-delta="5">Kenta Shirasawa</span>, <span class="highwire-citation-author" data-delta="6">Shigemi Sasamoto</span>, <span class="highwire-citation-author" data-delta="7">Akiko Watanabe</span>, <span class="highwire-citation-author" data-delta="8">Tsuyuko Wada</span>, <span class="highwire-citation-author" data-delta="9">Yoshie Kishida</span>, <span class="highwire-citation-author" data-delta="10">Hisano Tsuruoka</span>, <span class="highwire-citation-author" data-delta="11">Tsunakazu Fujishiro</span>, <span class="highwire-citation-author" data-delta="12">Manabu Yamada</span>, <span class="highwire-citation-author" data-delta="13">Mistuyo Kohara</span> and <span class="highwire-citation-author" data-delta="14">Satoshi Tabata</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">607-617;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002600 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459276" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sachiko N. Isobe</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sachiko%2BN.%2BIsobe" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Isobe%20SN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASachiko%2BN.%2BIsobe" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sisobe@kazusa.or.jp" class="" data-icon-position="" data-hide-link-title="0">sisobe@kazusa.or.jp</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Hiroshi Hisano</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hiroshi%2BHisano" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hisano%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHiroshi%2BHisano" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shusei Sato</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shusei%2BSato" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sato%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShusei%2BSato" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hideki Hirakawa</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hideki%2BHirakawa" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hirakawa%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHideki%2BHirakawa" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kenji Okumura</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Forage Crop Breeding Research Team, National Agricultural Research Center for Hokkaido Region, Sapporo, Hokkaido 062-8555, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenji%2BOkumura" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Okumura%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKenji%2BOkumura" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Kenta Shirasawa</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenta%2BShirasawa" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shirasawa%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKenta%2BShirasawa" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Shigemi Sasamoto</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shigemi%2BSasamoto" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sasamoto%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShigemi%2BSasamoto" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Akiko Watanabe</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Akiko%2BWatanabe" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Watanabe%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAkiko%2BWatanabe" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Tsuyuko Wada</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tsuyuko%2BWada" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wada%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATsuyuko%2BWada" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Yoshie Kishida</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yoshie%2BKishida" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kishida%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYoshie%2BKishida" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Hisano Tsuruoka</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hisano%2BTsuruoka" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsuruoka%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHisano%2BTsuruoka" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Tsunakazu Fujishiro</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tsunakazu%2BFujishiro" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fujishiro%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATsunakazu%2BFujishiro" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Manabu Yamada</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manabu%2BYamada" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yamada%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AManabu%2BYamada" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Mistuyo Kohara</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mistuyo%2BKohara" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kohara%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMistuyo%2BKohara" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Satoshi Tabata</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Satoshi%2BTabata" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tabata%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASatoshi%2BTabata" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/5/607" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/5/607&amp;title=Comparative%20Genetic%20Mapping%20and%20Discovery%20of%20Linkage%20Disequilibrium%20Across%20Linkage%20Groups%20in%20White%20Clover%20%28Trifolium%20repens%20L.%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/5/607&amp;title=Comparative%20Genetic%20Mapping%20and%20Discovery%20of%20Linkage%20Disequilibrium%20Across%20Linkage%20Groups%20in%20White%20Clover%20%28Trifolium%20repens%20L.%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/5/607&amp;title=Comparative%20Genetic%20Mapping%20and%20Discovery%20of%20Linkage%20Disequilibrium%20Across%20Linkage%20Groups%20in%20White%20Clover%20%28Trifolium%20repens%20L.%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/5/607&amp;text=Comparative%20Genetic%20Mapping%20and%20Discovery%20of%20Linkage%20Disequilibrium%20Across%20Linkage%20Groups%20in%20White%20Clover%20%28Trifolium%20repens%20L.%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/5/607&amp;title=Comparative%20Genetic%20Mapping%20and%20Discovery%20of%20Linkage%20Disequilibrium%20Across%20Linkage%20Groups%20in%20White%20Clover%20%28Trifolium%20repens%20L.%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/5/607&amp;t=Comparative%20Genetic%20Mapping%20and%20Discovery%20of%20Linkage%20Disequilibrium%20Across%20Linkage%20Groups%20in%20White%20Clover%20%28Trifolium%20repens%20L.%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/5/607&amp;title=Comparative%20Genetic%20Mapping%20and%20Discovery%20of%20Linkage%20Disequilibrium%20Across%20Linkage%20Groups%20in%20White%20Clover%20%28Trifolium%20repens%20L.%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/5/607&amp;title=Comparative%20Genetic%20Mapping%20and%20Discovery%20of%20Linkage%20Disequilibrium%20Across%20Linkage%20Groups%20in%20White%20Clover%20%28Trifolium%20repens%20L.%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459276/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459276" id="node459276" data-pisa="ggg;2/5/607" data-pisa-master="ggg;2/5/607" data-apath="/ggg/2/5/607.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparative Genetic Mapping and Discovery of Linkage Disequilibrium Across Linkage Groups in White Clover (<em>Trifolium repens</em> L.)</h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sachiko N. Isobe</span>, <span class="highwire-citation-author" data-delta="1">Hiroshi Hisano</span>, <span class="highwire-citation-author" data-delta="2">Shusei Sato</span>, <span class="highwire-citation-author" data-delta="3">Hideki Hirakawa</span>, <span class="highwire-citation-author" data-delta="4">Kenji Okumura</span>, <span class="highwire-citation-author" data-delta="5">Kenta Shirasawa</span>, <span class="highwire-citation-author" data-delta="6">Shigemi Sasamoto</span>, <span class="highwire-citation-author" data-delta="7">Akiko Watanabe</span>, <span class="highwire-citation-author" data-delta="8">Tsuyuko Wada</span>, <span class="highwire-citation-author" data-delta="9">Yoshie Kishida</span>, <span class="highwire-citation-author" data-delta="10">Hisano Tsuruoka</span>, <span class="highwire-citation-author" data-delta="11">Tsunakazu Fujishiro</span>, <span class="highwire-citation-author" data-delta="12">Manabu Yamada</span>, <span class="highwire-citation-author" data-delta="13">Mistuyo Kohara</span> and <span class="highwire-citation-author" data-delta="14">Satoshi Tabata</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">607-617;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002600 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459276" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sachiko N. Isobe</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sachiko%2BN.%2BIsobe" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Isobe%20SN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASachiko%2BN.%2BIsobe" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sisobe@kazusa.or.jp" class="" data-icon-position="" data-hide-link-title="0">sisobe@kazusa.or.jp</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Hiroshi Hisano</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hiroshi%2BHisano" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hisano%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHiroshi%2BHisano" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shusei Sato</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shusei%2BSato" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sato%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShusei%2BSato" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hideki Hirakawa</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hideki%2BHirakawa" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hirakawa%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHideki%2BHirakawa" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kenji Okumura</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Forage Crop Breeding Research Team, National Agricultural Research Center for Hokkaido Region, Sapporo, Hokkaido 062-8555, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenji%2BOkumura" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Okumura%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKenji%2BOkumura" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Kenta Shirasawa</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenta%2BShirasawa" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shirasawa%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKenta%2BShirasawa" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Shigemi Sasamoto</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shigemi%2BSasamoto" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sasamoto%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShigemi%2BSasamoto" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Akiko Watanabe</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Akiko%2BWatanabe" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Watanabe%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAkiko%2BWatanabe" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Tsuyuko Wada</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tsuyuko%2BWada" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wada%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATsuyuko%2BWada" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Yoshie Kishida</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yoshie%2BKishida" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kishida%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYoshie%2BKishida" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Hisano Tsuruoka</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hisano%2BTsuruoka" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsuruoka%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHisano%2BTsuruoka" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Tsunakazu Fujishiro</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tsunakazu%2BFujishiro" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fujishiro%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATsunakazu%2BFujishiro" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Manabu Yamada</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manabu%2BYamada" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yamada%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AManabu%2BYamada" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Mistuyo Kohara</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mistuyo%2BKohara" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kohara%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMistuyo%2BKohara" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Satoshi Tabata</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Plant Genome Research, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Satoshi%2BTabata" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tabata%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASatoshi%2BTabata" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=607&amp;rft.epage=617&amp;rft.atitle=Comparative%20Genetic%20Mapping%20and%20Discovery%20of%20Linkage%20Disequilibrium%20Across%20Linkage%20Groups%20in%20White%20Clover%20%28Trifolium%20repens%20L.%29&amp;rft.volume=2&amp;rft.issue=5&amp;rft.date=2012-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Isobe%2C+Sachiko+N.&amp;rft.au=Hisano%2C+Hiroshi&amp;rft.au=Sato%2C+Shusei&amp;rft.au=Hirakawa%2C+Hideki&amp;rft.au=Okumura%2C+Kenji&amp;rft.au=Shirasawa%2C+Kenta&amp;rft.au=Sasamoto%2C+Shigemi&amp;rft.au=Watanabe%2C+Akiko&amp;rft.au=Wada%2C+Tsuyuko&amp;rft.au=Kishida%2C+Yoshie&amp;rft.au=Tsuruoka%2C+Hisano&amp;rft.au=Fujishiro%2C+Tsunakazu&amp;rft.au=Yamada%2C+Manabu&amp;rft.au=Kohara%2C+Mistuyo&amp;rft.au=Tabata%2C+Satoshi"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459276/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459276/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459276/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459276/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459276/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459276/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459276/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459276/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459276/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459276/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459276/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459276/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494625" id="node494625" data-pisa="ggg;9/11/3891" data-pisa-master="ggg;g3.119.400723" data-apath="/ggg/9/11/3891.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3891" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Mouse Mutation That Dysregulates Neighboring <em>Galnt17</em> and <em>Auts2</em> Genes Is Associated with Phenotypes Related to the Human AUTS2 Syndrome</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494624" id="node494624" data-pisa="ggg;9/11/3781" data-pisa-master="ggg;g3.119.400647" data-apath="/ggg/9/11/3781.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3781" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Transgenic Analyses in Drosophila Reveal That mCORL1 Is Functionally Distinct from mCORL2 and dCORL</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494622" id="node494622" data-pisa="ggg;9/11/3843" data-pisa-master="ggg;g3.119.400363" data-apath="/ggg/9/11/3843.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3843" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Analysis of Cell Cycle-Regulating Genes in the Symbiotic Dinoflagellate <em>Breviolum minutum</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/5/607" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/5/607.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/5/607.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/5/607.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/5/607.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2019 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
  </body>
</html>
