<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/5/555" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/5/555.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/5/555.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/5/555.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/5/555.atom" />
<meta name="HW.pisa" content="ggg;2/5/555" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Identification of Mutant Versions of the Spt16 Histone Chaperone That Are Defective for Transcription-Coupled Nucleosome Occupancy in Saccharomyces cerevisiae" />
<meta name="DC.Identifier" content="10.1534/g3.112.002451" />
<meta name="DC.Date" content="2012-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Hainer et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The highly conserved FACT (Facilitates Chromatin Transactions) complex performs essential functions in eukaryotic cells through the reorganization of nucleosomes. During transcription, FACT reorganizes nucleosomes to allow passage of RNA Polymerase II and then assists in restoring these nucleosomes after RNA Polymerase II has passed. We have previously shown, consistent with this function, that [Spt16][1] facilitates repression of the Saccharomyces cerevisiae [SER3][2] gene by maintaining nucleosome occupancy over the promoter of this gene as a consequence of intergenic transcription of [SRG1][3] noncoding DNA. In this study, we report the results of a genetic screen to identify mutations in [SPT16][1] that derepress [SER3][2] . Twenty-five [spt16][1] mutant alleles were found to derepress [SER3][2] without causing significant reductions in either [SRG1][3] RNA levels or [Spt16][1] protein levels. Additional phenotypic assays indicate that these mutants have general transcription defects related to altered chromatin structure. Our analyses of a subset of these [spt16][1] mutants reveal defects in [SRG1][3] transcription-coupled nucleosome occupancy over the [SER3][2] promoter. We provide evidence that these mutants broadly impair transcription-coupled nucleosome occupancy at highly transcribed genes but not at lowly transcribed genes. Finally, we show that one consequence shared by these mutations is the reduced binding of mutant [Spt16][1] proteins across [SRG1][3] and other highly transcribed genes. Taken together, our results highlight an important role for [Spt16][1] in orchestrating transcription-coupled nucleosome assembly at highly transcribed regions of the genome, possibly by facilitating the association of [Spt16][1] during this process.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883
 [3]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010" />
<meta name="DC.Contributor" content="Sarah J. Hainer" />
<meta name="DC.Contributor" content="Brittany A. Charsar" />
<meta name="DC.Contributor" content="Shayna B. Cohen" />
<meta name="DC.Contributor" content="Joseph A. Martens" />
<meta name="article:published_time" content="2012-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Identification of Mutant Versions of the Spt16 Histone Chaperone That Are Defective for Transcription-Coupled Nucleosome Occupancy in Saccharomyces cerevisiae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The highly conserved FACT (Facilitates Chromatin Transactions) complex performs essential functions in eukaryotic cells through the reorganization of nucleosomes. During transcription, FACT reorganizes nucleosomes to allow passage of RNA Polymerase II and then assists in restoring these nucleosomes after RNA Polymerase II has passed. We have previously shown, consistent with this function, that Spt16 facilitates repression of the &lt;i&gt;Saccharomyces cerevisiae SER3&lt;/i&gt; gene by maintaining nucleosome occupancy over the promoter of this gene as a consequence of intergenic transcription of &lt;i&gt;SRG1&lt;/i&gt; noncoding DNA. In this study, we report the results of a genetic screen to identify mutations in &lt;i&gt;SPT16&lt;/i&gt; that derepress &lt;i&gt;SER3&lt;/i&gt;. Twenty-five &lt;i&gt;spt16&lt;/i&gt; mutant alleles were found to derepress &lt;i&gt;SER3&lt;/i&gt; without causing significant reductions in either &lt;i&gt;SRG1&lt;/i&gt; RNA levels or Spt16 protein levels. Additional phenotypic assays indicate that these mutants have general transcription defects related to altered chromatin structure. Our analyses of a subset of these &lt;i&gt;spt16&lt;/i&gt; mutants reveal defects in &lt;i&gt;SRG1&lt;/i&gt; transcription-coupled nucleosome occupancy over the &lt;i&gt;SER3&lt;/i&gt; promoter. We provide evidence that these mutants broadly impair transcription-coupled nucleosome occupancy at highly transcribed genes but not at lowly transcribed genes. Finally, we show that one consequence shared by these mutations is the reduced binding of mutant Spt16 proteins across &lt;i&gt;SRG1&lt;/i&gt; and other highly transcribed genes. Taken together, our results highlight an important role for Spt16 in orchestrating transcription-coupled nucleosome assembly at highly transcribed regions of the genome, possibly by facilitating the association of Spt16 during this process.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/05/01" />
<meta name="citation_mjid" content="ggg;2/5/555" />
<meta name="citation_id" content="2/5/555" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/5/555" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/5/555.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/5/555.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/5/555.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.002451" />
<meta name="citation_pmid" content="22670226" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="5" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="555" />
<meta name="citation_lastpage" content="567" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Sarah J. Hainer" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University of Pittsburgh, Pittsburgh, Pennsylvania 15260" />
<meta name="citation_author" content="Brittany A. Charsar" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University of Pittsburgh, Pittsburgh, Pennsylvania 15260" />
<meta name="citation_author" content="Shayna B. Cohen" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University of Pittsburgh, Pittsburgh, Pennsylvania 15260" />
<meta name="citation_author" content="Joseph A. Martens" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University of Pittsburgh, Pittsburgh, Pennsylvania 15260" />
<meta name="citation_author_email" content="martens@pitt.edu" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=K. Adelman;citation_author=W. Wei;citation_author=M. B. Ardehali;citation_author=J. Werner;citation_author=B. Zhu;citation_title=Drosophila Paf1 Modulates Chromatin Structure at Actively Transcribed Genes;citation_pages=250-260;citation_volume=26;citation_year=2006;citation_issue=1;citation_pmid=16354696;citation_doi=10.1128/MCB.26.1.250-260.2006" />
<meta name="citation_reference" content="citation_author=F. M. Ausubel;citation_author=R. Brent;citation_author=R. E. Kingston;citation_author=D. D. Moore;citation_author=J. G. Seidman;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=L. Bai;citation_author=A. V. Morozov;citation_title=Gene regulation by nucleosome positioning.;citation_pages=476-483;citation_volume=26;citation_year=2010;citation_issue=11;citation_pmid=20832136;citation_doi=10.1016/j.tig.2010.08.003" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. Belotserkovskaya;citation_author=S. Oh;citation_author=V. A. Bondarenko;citation_author=G. Orphanides;citation_author=V. M. Studitsky;citation_title=FACT Facilitates Transcription-Dependent Nucleosome Alteration;citation_pages=1090-1093;citation_volume=301;citation_year=2003;citation_issue=5636;citation_pmid=12934006;citation_doi=10.1126/science.1085703" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=D. Biswas;citation_author=R. Dutta-Biswas;citation_author=D. J. Stillman;citation_title=Chd1 and yFACT act in opposition in regulating transcription;citation_volume=27;citation_year=2007;citation_pmid=17620414;citation_doi=10.1128/MCB.00978-07" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=D. G. Brickner;citation_author=I. Cajigas;citation_author=Y. Fondufe-Mittendorf;citation_author=S. Ahmed;citation_author=P. C. Lee;citation_title=H2A.Z-mediated localization of genes at the nuclear periphery confers epigenetic memory of previous transcriptional state.;citation_pages=e81-e81;citation_volume=5;citation_year=2007;citation_issue=4;citation_pmid=17373856;citation_doi=10.1371/journal.pbio.0050081" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. R. Cairns;citation_title=The logic of chromatin architecture and remodelling at promoters.;citation_pages=193-198;citation_volume=461;citation_year=2009;citation_issue=7261;citation_pmid=19741699;citation_doi=10.1038/nature08450" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. J. Carrozza;citation_author=B. Li;citation_author=L. Florens;citation_author=T. Suganuma;citation_author=S. K. Swanson;citation_title=Histone H3 methylation by Set2 directs deacetylation of coding regions by Rpd3S to suppress spurious intragenic transcription.;citation_pages=581-592;citation_volume=123;citation_year=2005;citation_issue=4;citation_pmid=16286007;citation_doi=10.1016/j.cell.2005.10.023" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=V. Cheung;citation_author=G. Chua;citation_author=N. N. Batada;citation_author=C. R. Landry;citation_author=S. W. Michnick;citation_title=Chromatin- and transcription-related factors repress transcription from within coding regions throughout the Saccharomyces cerevisiae genome.;citation_pages=e277-e277;citation_volume=6;citation_year=2008;citation_issue=11;citation_pmid=18998772;citation_doi=10.1371/journal.pbio.0060277" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=C. D. Clark-Adams;citation_author=F. Winston;citation_title=The SPT6 gene is essential for growth and is required for delta-mediated transcription in Saccharomyces cerevisiae.;citation_pages=679-686;citation_volume=7;citation_year=1987;citation_issue=2;citation_pmid=3029564;citation_doi=10.1128/MCB.7.2.679" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=C. D. Clark-Adams;citation_author=D. Norris;citation_author=M. A. Osley;citation_author=J. S. Fassler;citation_author=F. Winston;citation_title=Changes in histone gene dosage alter transcription in yeast.;citation_pages=150-159;citation_volume=2;citation_year=1988;citation_issue=2;citation_pmid=2834270;citation_doi=10.1101/gad.2.2.150" />
<meta name="citation_reference" content="citation_author=M. A. Collart;citation_author=S. Oliviero;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=J. S. Cox;citation_author=R. E. Chapman;citation_author=P. Walter;citation_title=The unfolded protein response coordinates the production of endoplasmic reticulum protein and endoplasmic reticulum membrane.;citation_pages=1805-1814;citation_volume=8;citation_year=1997;citation_issue=9;citation_pmid=9307975;citation_doi=10.1091/mbc.8.9.1805" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res. Int.;citation_title=Histone chaperones Spt16 and FACT: similarities and differences in modes of action at transcribed genes;citation_author=A. A. Duina;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. A. Duina;citation_author=A. Rufiange;citation_author=J. Bracey;citation_author=J. Hall;citation_author=A. Nourani;citation_title=Evidence that the localization of the elongation factor Spt16 across transcribed genes is dependent upon histone H3 integrity in Saccharomyces cerevisiae;citation_volume=177;citation_year=2007;citation_pmid=17603125;citation_doi=10.1534/genetics.106.067140" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. R. Evans;citation_author=N. K. Brewster;citation_author=Q. Xu;citation_author=A. Rowley;citation_author=B. A. Altheim;citation_title=The yeast protein complex containing cdc68 and pob3 mediates core-promoter repression through the cdc68 N-terminal domain;citation_pages=1393-1405;citation_volume=150;citation_year=1998;citation_pmid=9832518" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. Floer;citation_author=X. Wang;citation_author=V. Prabhu;citation_author=G. Berrozpe;citation_author=S. Narayan;citation_title=A RSC/nucleosome complex determines chromatin architecture and facilitates activator binding.;citation_pages=407-418;citation_volume=141;citation_year=2010;citation_issue=3;citation_pmid=20434983;citation_doi=10.1016/j.cell.2010.03.048" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Biosyst;citation_author=T. Formosa;citation_title=FACT and the reorganized nucleosome.;citation_pages=1085-1093;citation_volume=4;citation_year=2008;citation_issue=11;citation_pmid=18931784;citation_doi=10.1039/b812136b" />
<meta name="citation_reference" content="citation_journal_title=Biochim. Biophys. Acta;citation_title=The role of FACT in making and breaking nucleosomes;citation_author=T. Formosa;citation_pages=247-255;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=T. Formosa;citation_author=P. Eriksson;citation_author=J. Wittmeyer;citation_author=J. Ginn;citation_author=Y. Yu;citation_title=Spt16-Pob3 and the HMG protein Nhp6 combine to form the nucleosome-binding factor SPN.;citation_pages=3506-3517;citation_volume=20;citation_year=2001;citation_issue=13;citation_pmid=11432837;citation_doi=10.1093/emboj/20.13.3506" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Formosa;citation_author=S. Ruone;citation_author=M. D. Adams;citation_author=A. E. Olsen;citation_author=P. Eriksson;citation_title=Defects in SPT16 or POB3 (yFACT) in Saccharomyces cerevisiae cause dependence on the Hir/Hpc pathway: polymerase passage may degrade chromatin structure;citation_pages=1557-1571;citation_volume=162;citation_year=2002;citation_pmid=12524332" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=S. J. Hainer;citation_author=J. A. Martens;citation_title=Identification of histone mutants that are defective for transcription-coupled nucleosome occupancy;citation_volume=31;citation_year=2011;citation_pmid=21730290;citation_doi=10.1128/MCB.05195-11" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=S. J. Hainer;citation_author=J. A. Pruneski;citation_author=R. D. Mitchell;citation_author=R. M. Monteverde;citation_author=J. A. Martens;citation_title=Intergenic transcription causes repression by directing nucleosome assembly;citation_volume=25;citation_year=2011;citation_pmid=21156811;citation_doi=10.1101/gad.1975011" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=M. Hampsey;citation_title=A review of phenotypes in Saccharomyces cerevisiae.;citation_pages=1099-1133;citation_volume=13;citation_year=1997;citation_issue=12;citation_pmid=9301019;citation_doi=10.1002/(SICI)1097-0061(19970930)13:12&lt;1099::AID-YEA177&gt;3.0.CO;2-7" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=F. C. Holstege;citation_author=E. G. Jennings;citation_author=J. J. Wyrick;citation_author=T. I. Lee;citation_author=C. J. Hengartner;citation_title=Dissecting the regulatory circuitry of a eukaryotic genome.;citation_pages=717-728;citation_volume=95;citation_year=1998;citation_issue=5;citation_pmid=9845373;citation_doi=10.1016/S0092-8674(00)81641-4" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=A. Jamai;citation_author=A. Puglisi;citation_author=M. Strubin;citation_title=Histone chaperone spt16 promotes redeposition of the original h3-h4 histones evicted by elongating RNA polymerase.;citation_pages=377-383;citation_volume=35;citation_year=2009;citation_issue=3;citation_pmid=19683500;citation_doi=10.1016/j.molcel.2009.07.001" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=C. D. Kaplan;citation_author=L. Laprade;citation_author=F. Winston;citation_title=Transcription Elongation Factors Repress Transcription Initiation from Cryptic Sites;citation_pages=1096-1099;citation_volume=301;citation_year=2003;citation_issue=5636;citation_pmid=12934008;citation_doi=10.1126/science.1087374" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=M. Kim;citation_author=S. H. Ahn;citation_author=N. J. Krogan;citation_author=J. F. Greenblatt;citation_author=S. Buratowski;citation_title=Transitions in RNA polymerase II elongation complexes at the 3′ ends of genes;citation_volume=23;citation_year=2004;citation_pmid=14739930;citation_doi=10.1038/sj.emboj.7600053" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=P. Komarnitsky;citation_author=E. J. Cho;citation_author=S. Buratowski;citation_title=Different phosphorylated forms of RNA polymerase II and associated mRNA processing factors during transcription;citation_pages=2452-2460;citation_volume=14;citation_year=2000;citation_issue=19;citation_pmid=11018013;citation_doi=10.1101/gad.824700" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=N. J. Krogan;citation_author=M. Kim;citation_author=S. H. Ahn;citation_author=G. Zhong;citation_author=M. S. Kobor;citation_title=RNA Polymerase II Elongation Factors of Saccharomyces cerevisiae: a Targeted Proteomics Approach;citation_pages=6979-6992;citation_volume=22;citation_year=2002;citation_issue=20;citation_pmid=12242279;citation_doi=10.1128/MCB.22.20.6979-6992.2002" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=B. Li;citation_author=M. Carey;citation_author=J. L. Workman;citation_title=The role of chromatin during transcription.;citation_pages=707-719;citation_volume=128;citation_year=2007;citation_issue=4;citation_pmid=17320508;citation_doi=10.1016/j.cell.2007.01.015" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=Y. Liu;citation_author=H. Huang;citation_author=B. O. Zhou;citation_author=S. S. Wang;citation_author=Y. Hu;citation_title=Structural analysis of Rtt106p reveals a DNA binding role required for heterochromatin silencing;citation_volume=285;citation_year=2010;citation_pmid=20007951;citation_doi=10.1074/jbc.M109.055996" />
<meta name="citation_reference" content="citation_journal_title=Chromosome research :  an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology;citation_journal_abbrev=Chromosome Res;citation_author=K. Luger;citation_title=Dynamic nucleosomes.;citation_pages=5-16;citation_volume=14;citation_year=2006;citation_issue=1;citation_pmid=16506092;citation_doi=10.1007/s10577-005-1026-1" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. Luger;citation_author=A. W. Mader;citation_author=R. K. Richmond;citation_author=D. F. Sargent;citation_author=T. J. Richmond;citation_title=Crystal structure of the nucleosome core particle at 2.8 A resolution.;citation_pages=251-260;citation_volume=389;citation_year=1997;citation_issue=6648;citation_pmid=9305837;citation_doi=10.1038/38444" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=E. A. Malone;citation_author=C. D. Clark;citation_author=A. Chiang;citation_author=F. Winston;citation_title=Mutations in SPT16/CDC68 suppress cis- and trans-acting mutations that affect promoter function in Saccharomyces cerevisiae.;citation_pages=5710-5717;citation_volume=11;citation_year=1991;citation_issue=11;citation_pmid=1922073;citation_doi=10.1128/MCB.11.11.5710" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. A. Martens;citation_author=L. Laprade;citation_author=F. Winston;citation_title=Intergenic transcription is required to repress the Saccharomyces cerevisiae SER3 gene.;citation_pages=571-574;citation_volume=429;citation_year=2004;citation_issue=6991;citation_pmid=15175754;citation_doi=10.1038/nature02538" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=J. A. Martens;citation_author=P. Y. Wu;citation_author=F. Winston;citation_title=Regulation of an intergenic transcript controls adjacent gene transcription in Saccharomyces cerevisiae;citation_pages=2695-2704;citation_volume=19;citation_year=2005;citation_issue=22;citation_pmid=16291644;citation_doi=10.1101/gad.1367605" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=P. B. Mason;citation_author=K. Struhl;citation_title=The FACT Complex Travels with Elongating RNA Polymerase II and Is Important for the Fidelity of Transcriptional Initiation In Vivo;citation_pages=8323-8333;citation_volume=23;citation_year=2003;citation_issue=22;citation_pmid=14585989;citation_doi=10.1128/MCB.23.22.8323-8333.2003" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=A. Mayer;citation_author=M. Lidschreiber;citation_author=M. Siebert;citation_author=K. Leike;citation_author=J. Soding;citation_title=Uniform transitions of the general RNA polymerase II transcription complex.;citation_pages=1272-1278;citation_volume=17;citation_year=2010;citation_issue=10;citation_pmid=20818391;citation_doi=10.1038/nsmb.1903" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. McCullough;citation_author=R. Rawlins;citation_author=A. Olsen;citation_author=H. Xin;citation_author=D. J. Stillman;citation_title=Insight into the mechanism of nucleosome reorganization from histone mutants that suppress defects in the FACT histone chaperone;citation_pages=835-846;citation_volume=188;citation_year=2011;citation_pmid=21625001;citation_doi=10.1534/genetics.111.128769" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. H. McCusker;citation_author=J. E. Haber;citation_title=Cycloheximide-resistant temperature-sensitive lethal mutations of Saccharomyces cerevisiae;citation_pages=303-315;citation_volume=119;citation_year=1988;citation_pmid=3294103" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=C. N. Myers;citation_author=G. B. Berner;citation_author=J. H. Holthoff;citation_author=K. Martinez-Fonts;citation_author=J. A. Harper;citation_title=Mutant versions of the S. cerevisiae transcription elongation factor Spt16 define regions of Spt16 that functionally interact with histone H3.;citation_pages=e20847-e20847;citation_volume=6;citation_year=2011;citation_issue=6;citation_pmid=21673966;citation_doi=10.1371/journal.pone.0020847" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=G. J. Narlikar;citation_author=H. Y. Fan;citation_author=R. E. Kingston;citation_title=Cooperation between complexes that regulate chromatin structure and transcription.;citation_pages=475-487;citation_volume=108;citation_year=2002;citation_issue=4;citation_pmid=11909519;citation_doi=10.1016/S0092-8674(02)00654-2" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. F. O’Donnell;citation_author=N. K. Brewster;citation_author=J. Kurniawan;citation_author=L. V. Minard;citation_author=G. C. Johnston;citation_title=Domain organization of the yeast histone chaperone FACT: the conserved N-terminal domain of FACT subunit Spt16 mediates recovery from replication stress;citation_pages=5894-5906;citation_volume=32;citation_year=2004;citation_issue=19;citation_pmid=15520471;citation_doi=10.1093/nar/gkh922" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=A. F. O’Donnell;citation_author=J. R. Stevens;citation_author=R. Kepkay;citation_author=C. A. Barnes;citation_author=G. C. Johnston;citation_title=New mutant versions of yeast FACT subunit Spt16 affect cell integrity.;citation_pages=487-502;citation_volume=282;citation_year=2009;citation_issue=5;citation_pmid=19727824;citation_doi=10.1007/s00438-009-0480-4" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=G. Orphanides;citation_author=G. LeRoy;citation_author=C. H. Chang;citation_author=D. S. Luse;citation_author=D. Reinberg;citation_title=FACT, a factor that facilitates transcript elongation through nucleosomes.;citation_pages=105-116;citation_volume=92;citation_year=1998;citation_issue=1;citation_pmid=9489704;citation_doi=10.1016/S0092-8674(00)80903-4" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Orphanides;citation_author=W. H. Wu;citation_author=W. S. Lane;citation_author=M. Hampsey;citation_author=D. Reinberg;citation_title=The chromatin-specific transcription elongation factor FACT comprises human SPT16 and SSRP1 proteins.;citation_pages=284-288;citation_volume=400;citation_year=1999;citation_issue=6741;citation_pmid=10421373;citation_doi=10.1038/22350" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. W. Pfaffl;citation_title=A new mathematical model for relative quantification in real-time RT-PCR;citation_pages=e45-e45;citation_volume=29;citation_year=2001;citation_issue=9;citation_pmid=11328886;citation_doi=10.1093/nar/29.9.e45" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=J. A. Pruneski;citation_author=S. J. Hainer;citation_author=K. O. Petrov;citation_author=J. A. Martens;citation_title=The Paf1 complex represses SER3 transcription in Saccharomyces cerevisiae by facilitating intergenic transcription-dependent nucleosome occupancy of the SER3 promoter;citation_volume=10;citation_year=2011;citation_pmid=21873510;citation_doi=10.1128/EC.05141-11" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=D. Reinberg;citation_author=R. J. Sims;citation_title=de FACTo nucleosome dynamics;citation_volume=281;citation_year=2006;citation_pmid=16766522;citation_doi=10.1074/jbc.R600007200" />
<meta name="citation_reference" content="citation_author=M. D. Rose;citation_author=F. Winston;citation_author=P. Heiter;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=M. A. Schwabish;citation_author=K. Struhl;citation_title=Evidence for Eviction and Rapid Deposition of Histones upon Transcriptional Elongation by RNA Polymerase II;citation_pages=10111-10117;citation_volume=24;citation_year=2004;citation_issue=23;citation_pmid=15542822;citation_doi=10.1128/MCB.24.23.10111-10117.2004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. K. Shirra;citation_author=S. E. Rogers;citation_author=D. E. Alexander;citation_author=K. M. Arndt;citation_title=The Snf1 protein kinase and Sit4 protein phosphatase have opposing functions in regulating TATA-binding protein association with the Saccharomyces cerevisiae INO1 promoter;citation_pages=1957-1972;citation_volume=169;citation_year=2005;citation_pmid=15716495;citation_doi=10.1534/genetics.104.038075" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. S. Sikorski;citation_author=P. Hieter;citation_title=A system of shuttle vectors and yeast host strains designed for efficient manipulation of DNA in Saccharomyces cerevisiae;citation_pages=19-27;citation_volume=122;citation_year=1989;citation_pmid=2659436" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=R. Simic;citation_author=D. L. Lindstrom;citation_author=H. G. Tran;citation_author=K. L. Roinick;citation_author=P. J. Costa;citation_title=Chromatin remodeling protein Chd1 interacts with transcription elongation factors and localizes to transcribed genes;citation_pages=1846-1856;citation_volume=22;citation_year=2003;citation_issue=8;citation_pmid=12682017;citation_doi=10.1093/emboj/cdg179" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=E. Smith;citation_author=A. Shilatifard;citation_title=The chromatin signaling pathway: diverse mechanisms of recruitment of histone-modifying enzymes and varied biological outcomes.;citation_pages=689-701;citation_volume=40;citation_year=2010;citation_issue=5;citation_pmid=21145479;citation_doi=10.1016/j.molcel.2010.11.031" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=S. L. Squazzo;citation_author=P. J. Costa;citation_author=D. L. Lindstrom;citation_author=K. E. Kumer;citation_author=R. Simic;citation_title=The Paf1 complex physically and functionally associates with transcription elongation factors in vivo.;citation_pages=1764-1774;citation_volume=21;citation_year=2002;citation_issue=7;citation_pmid=11927560;citation_doi=10.1093/emboj/21.7.1764" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. R. Stevens;citation_author=A. F. O’Donnell;citation_author=T. E. Perry;citation_author=J. J. Benjamin;citation_author=C. A. Barnes;citation_title=FACT, the Bur kinase pathway, and the histone co-repressor HirC have overlapping nucleosome-related roles in yeast transcription elongation.;citation_pages=e25644-e25644;citation_volume=6;citation_year=2011;citation_issue=10;citation_pmid=22022426;citation_doi=10.1371/journal.pone.0025644" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Stuwe;citation_author=M. Hothorn;citation_author=E. Lejeune;citation_author=V. Rybin;citation_author=M. Bortfeld;citation_title=The FACT Spt16 “peptidase” domain is a histone H3–H4 binding module;citation_volume=105;citation_year=2008;citation_pmid=18579787;citation_doi=10.1073/pnas.0712293105" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. N. Tomson;citation_author=C. P. Davis;citation_author=M. H. Warner;citation_author=K. M. Arndt;citation_title=Identification of a role for histone H2B ubiquitylation in noncoding RNA 3′-end formation through mutational analysis of Rtf1 in Saccharomyces cerevisiae;citation_volume=188;citation_year=2011;citation_pmid=21441211;citation_doi=10.1534/genetics.111.128645" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=A. P. VanDemark;citation_author=M. Blanksma;citation_author=E. Ferris;citation_author=A. Heroux;citation_author=C. P. Hill;citation_title=The structure of the yFACT Pob3-M domain, its interaction with the DNA replication factor RPA, and a potential role in nucleosome deposition.;citation_pages=363-374;citation_volume=22;citation_year=2006;citation_issue=3;citation_pmid=16678108;citation_doi=10.1016/j.molcel.2006.03.025" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=A. P. VanDemark;citation_author=H. Xin;citation_author=L. McCullough;citation_author=R. Rawlins;citation_author=S. Bentley;citation_title=Structural and functional analysis of the Spt16p N-terminal domain reveals overlapping roles of yFACT subunits;citation_volume=283;citation_year=2008;citation_pmid=18089575;citation_doi=10.1074/jbc.M708682200" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=D. D. Winkler;citation_author=K. Luger;citation_title=The histone chaperone FACT: structural insights and mechanisms for nucleosome reorganization;citation_volume=286;citation_year=2011;citation_pmid=21454601;citation_doi=10.1074/jbc.R110.180778" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=D. D. Winkler;citation_author=U. M. Muthurajan;citation_author=A. R. Hieb;citation_author=K. Luger;citation_title=Histone chaperone FACT coordinates nucleosome interaction through multiple synergistic binding events;citation_volume=286;citation_year=2011;citation_pmid=21969370;citation_doi=10.1074/jbc.M111.301465" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. Winston;citation_author=D. T. Chaleff;citation_author=B. Valent;citation_author=G. R. Fink;citation_title=Mutations affecting Ty-mediated expression of the HIS4 gene of Saccharomyces cerevisiae;citation_pages=179-197;citation_volume=107;citation_year=1984;citation_pmid=6329902" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=F. Winston;citation_author=C. Dollard;citation_author=S. L. Ricupero-Hovasse;citation_title=Construction of a set of convenient Saccharomyces cerevisiae strains that are isogenic to S288C.;citation_pages=53-55;citation_volume=11;citation_year=1995;citation_issue=1;citation_pmid=7762301;citation_doi=10.1002/yea.320110107" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. Wittmeyer;citation_author=T. Formosa;citation_title=The Saccharomyces cerevisiae DNA polymerase alpha catalytic subunit interacts with Cdc68/Spt16 and with Pob3, a protein similar to an HMG1- like protein;citation_pages=4178-4190;citation_volume=17;citation_year=1997;citation_issue=7;citation_pmid=9199353;citation_doi=10.1128/MCB.17.7.4178" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=S. Zheng;citation_author=J. J. Wyrick;citation_author=J. C. Reese;citation_title=Novel trans-tail regulation of H2B ubiquitylation and H3K4 methylation by the N terminus of histone H2A;citation_volume=30;citation_year=2010;citation_pmid=20498280;citation_doi=10.1128/MCB.00324-10" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Identification of Mutant Versions of the Spt16 Histone Chaperone That Are Defective for Transcription-Coupled Nucleosome Occupancy in Saccharomyces cerevisiae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/5.cover-source.jpg" />
<meta name="twitter:description" content="The highly conserved FACT (Facilitates Chromatin Transactions) complex performs essential functions in eukaryotic cells through the reorganization of nucleosomes. During transcription, FACT reorganizes nucleosomes to allow passage of RNA Polymerase II and then assists in restoring these nucleosomes after RNA Polymerase II has passed. We have previously shown, consistent with this function, that [Spt16][1] facilitates repression of the Saccharomyces cerevisiae [SER3][2] gene by maintaining nucleosome occupancy over the promoter of this gene as a consequence of intergenic transcription of [SRG1][3] noncoding DNA. In this study, we report the results of a genetic screen to identify mutations in [SPT16][1] that derepress [SER3][2] . Twenty-five [spt16][1] mutant alleles were found to derepress [SER3][2] without causing significant reductions in either [SRG1][3] RNA levels or [Spt16][1] protein levels. Additional phenotypic assays indicate that these mutants have general transcription defects related to altered chromatin structure. Our analyses of a subset of these [spt16][1] mutants reveal defects in [SRG1][3] transcription-coupled nucleosome occupancy over the [SER3][2] promoter. We provide evidence that these mutants broadly impair transcription-coupled nucleosome occupancy at highly transcribed genes but not at lowly transcribed genes. Finally, we show that one consequence shared by these mutations is the reduced binding of mutant [Spt16][1] proteins across [SRG1][3] and other highly transcribed genes. Taken together, our results highlight an important role for [Spt16][1] in orchestrating transcription-coupled nucleosome assembly at highly transcribed regions of the genome, possibly by facilitating the association of [Spt16][1] during this process.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883
 [3]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010" />
<meta name="og-title" property="og:title" content="Identification of Mutant Versions of the Spt16 Histone Chaperone That Are Defective for Transcription-Coupled Nucleosome Occupancy in Saccharomyces cerevisiae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/5/555" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The highly conserved FACT (Facilitates Chromatin Transactions) complex performs essential functions in eukaryotic cells through the reorganization of nucleosomes. During transcription, FACT reorganizes nucleosomes to allow passage of RNA Polymerase II and then assists in restoring these nucleosomes after RNA Polymerase II has passed. We have previously shown, consistent with this function, that [Spt16][1] facilitates repression of the Saccharomyces cerevisiae [SER3][2] gene by maintaining nucleosome occupancy over the promoter of this gene as a consequence of intergenic transcription of [SRG1][3] noncoding DNA. In this study, we report the results of a genetic screen to identify mutations in [SPT16][1] that derepress [SER3][2] . Twenty-five [spt16][1] mutant alleles were found to derepress [SER3][2] without causing significant reductions in either [SRG1][3] RNA levels or [Spt16][1] protein levels. Additional phenotypic assays indicate that these mutants have general transcription defects related to altered chromatin structure. Our analyses of a subset of these [spt16][1] mutants reveal defects in [SRG1][3] transcription-coupled nucleosome occupancy over the [SER3][2] promoter. We provide evidence that these mutants broadly impair transcription-coupled nucleosome occupancy at highly transcribed genes but not at lowly transcribed genes. Finally, we show that one consequence shared by these mutations is the reduced binding of mutant [Spt16][1] proteins across [SRG1][3] and other highly transcribed genes. Taken together, our results highlight an important role for [Spt16][1] in orchestrating transcription-coupled nucleosome assembly at highly transcribed regions of the genome, possibly by facilitating the association of [Spt16][1] during this process.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883
 [3]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/5.cover-source.jpg" />
    <title>Identification of Mutant Versions of the Spt16 Histone Chaperone That Are Defective for Transcription-Coupled Nucleosome Occupancy in Saccharomyces cerevisiae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q0wst1" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__KPgiwEdjo5O9HdRryHqQs1TpsV-PeBb9B7sGBvwM9FQ__opzZWgPNgQzoEgfATRS5qCKwBw_6XcNjDHoDVOzQv8M__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__YUqwBlv6bNoCToiBWgAUSPtppdS8sbGfzmAH3qEDcI0__Mvk93h3ScFH5Xi9WhibVKlf368fHw-V7HWqWX2QMvtw__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XNWYaQubFGWYWZNytUHkWs-b34LV21DsB4s_6gchI4k__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__7cpBA0GVdrMqkZGGZEW-_SPF2HrbTCmVg9-lrP5l_AQ__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__TV2D4R18tPiSIOmeQ3vyMqHObN_FDmQ6EEIrfE5sAoA__c9BBZFy1BrgGgF_Auu9Fmysb3Z7J56eJGYvMcLQrLks__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q0wst1"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__OaHnYyci68QTwctdRYR30kEfu0eYWBrLyMqy8THv5Ik__IyfJQYM1q1QEPwYsK_Eslh2RQkxpy6yo-nR0p4DWPQQ__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__fhq_kmKS3zjIxlsAa3GE1ubLajyXUgXXTXULzAtiqfg__Lm1_Ls-bdpbWRyl0q4o0mmHyxB-139AJ68dE4zOskLQ__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"TwkQ3km-0gEgpuancp5JUU04qPY1mQLvV-fBpNP4S94","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459281","apath":"\/ggg\/2\/5\/555.atom","pisa":"ggg;2\/5\/555","ac":{"\/ggg\/2\/5\/555.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/5\/555.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/5\/555"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/5\/555"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/5\/555":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mUPeceK3T1cyiJQQ4gy0jhIgFpVQ4Y7VNPoURgteit6yzlZtqDazwOUBvDC1tTjsEuyhyQrycnvxmFoOMrfr00SUmZrqUSNWIbMEQT3FflYXKGYnD\\\\\\\/O5Lv0F+hwqtEygkKibhE0F\\\\\\\/lhjkCNXe9xPaHwUQJz8W8+8eWw6ICMpbBMZXchdCJlMKqCHrmwW4QwACDrd2GUcO2c6DxkKvuTzwLsJXH1zuFYJauBhQ0Lwzx4WnqBPT2GJ3uBpfXk3RWyXF2NaeFF2lgc3zfs7WMVfDRnLVeoHEJslznsG\\\\\\\/YRHUdkhroLTKr4pk33V7\\\\\\\/cbGSj4hrYcdsvcygUCIq6gayc2vhJpY9EY+ER5+6u4DfEXjkxGTADgpvj0Hbvn7La+TBVjfhT2\\\\\\\/t7u7QHlvLHjx\\\\\\\/5DwYTgmuOLbeNBHXj4oREdFyw2bavDPFCnMspdD3mmtaWr00xSemGi3YoZTAmRHZph\\\\\\\/Z1oOQBF\\\\\\\/uwxXTiKnassv10liKlBU5YCmPJPzBXcCos38DWZJCJ4dfN4GXCn9XCRcSgiU7VZJHeiKLB94nlNU7eAqqA\\\\\\\/mAUKnIHp8jzkUHhtMU3FSQx8cukv\\\\\\\/b9JxsthWIlWtroVDDoLh4zr4Td9nauQ\\\\\\\/SMU17aVRt0zxZZiyKgh2wvyfyr3aunUlBAuv7lc+43HBcIIKoSlMc7PXYFRW4bP3CVyK2g7NU9dJGDHkV3W0r36X\\\\\\\/QzBScIKyNmnBhHsWSh8xbCPVepx9RxHtUbUMKRBWQ7CoAva84nMdSZfRsNz9xpxXTJO7cfX+m2D+gCrsHFdw7aYtv9iohmG1XPu43GUgYjkB3dDrAZvV9i4aArkQRvW+M8IBoQ0IAIcdNMFlxplbmp33RCuNtYlscYCIBsvVoucyPm8mHFuK+JJeljQ2lHrGNG1tFM3l8+BjK1a0jvMN9EN7\\\\\\\/8nCqlWLxKDo\\\\\\\/gJBaKgGFamEkGFBg1fjT5DlcX+RsGmIT2FxftytIjAE7rsFHx+3MHxyZ8MliJlbqgg2iwuLD\\\\\\\/YOoylrt8p8RTfuab4mykiOqSmUzAycuLhQ3OWU5lw8v+6MHoAWwjhAF8pf3SmlG34gBcCG9+2UaNWFCP34oc295KnuAT1moAEXQdItXKOPWefeugFHwU83LWeLIoicNXxN76c9tmWPJF\\\\\\\/ojnfKXXwyAkiiPkX09Y8z2tF2K0zTO2MA81DwFiI0ZlZIUJ19CP\\\\\\\/JPhsf\\\\\\\/LyztzFpSiCtra9BCXWuZKgqeq562N2pvwk7lBnXjTC1\\\\\\\/K\\\\\\\/aRGOW3z3pIMmZ+BvONnX\\\\\\\/sQ\\\u0022,\\\u0022iv\\\u0022:\\\u00224TnbeLvAP9ymQv7EyWKBkg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229a0780d7a06fd8e64578329a9abeacdfc7ee350a75f623a720b7ebe6e4ccee0a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Uy+B0h2BpU1z8G2wCesEdG8XgqsTtiHUDHD03LxDryesVVj4Bw95V1b95mxdGc3ngPBJ1IwLtS6yAOXwEQ70VPLNdzCRF+vyFZqvZEOQ\\\\\\\/soVLgkn06bO4AsnFxzDAPmL5coCRR5hdLtXG42KS\\\\\\\/ko3Wg8Zw49L2Swpc7UgEYrK\\\\\\\/bEb8mzfkjUoDgJ0r+d8loVaoZXQgtot4IRpDTZPri\\\\\\\/6O0KrFv3fw2kw\\\\\\\/FC1S4VmEsZu5EwOcozy8cgdf12GEUw3PIpG6n\\\\\\\/8sOWv0BX8vXXRddlBCLapVXogQd9Bwr0zUJGuvpdm10lXDvcMGwe7l4TrFuLWUJglS+YA+r6mi5A\\\\\\\/xPay1HBr2uJEOksLDtJHKifLQuaxpE16CIisAWvYVmwGNr30L\\\\\\\/XnocZWWLM9eRYWWTEyx3i\\\\\\\/8XkhysCuWWDkBAL60fNI251MG+obkcqcelE0N8AEl3b0VD7g8+e\\\\\\\/dE5lj2F32SEBuDwBCtmKDehcizkY6hBkvtpKfqZfQ3cFkBrmHdqc5wIEOTknhdRac6FXHGssXBB60hacbpwlfJ4yxeCEkbXZcxQn3UL59qBMvrljHo41Ued52JQydKWJezBeiMGaReRNAMNW5DPr8dRnfbU5YN+yfUvQFv6KPZDNyGefnwGs37sG2YTcpuzvOWyGx9J8HDN75eNnZWIZ8y0Fx0d5TqqNNVa7h1SBQV9W8MxhJjPQc7Do\\\\\\\/3fpzvgsFeJc2hyNwr7XWDp8MiDK0OCZe9OB4zybdkBVvjizOAZ0y24mZ6yvuiiXDwD\\\\\\\/xc0TvrgWTwcxVv6ukQVkxK9B17dZIbVan4z40q2RtMxeUOI9Uc4q0RPRbCKnysNPJn87oNXPhRcWWfIrh4lYDvpHs5B6xWaRhqOWHAlZYfY3mZXhfCObnsRQ0zb1422TCRoWeTa0Iope4RUa11yFWsonA5n8K8qPxAjeOVTwJBJjqRBt2sFYE4poGpkpQ3MmXr+xg2Al0A1HeijHPyjucDuu9VDcQ5Sm\\\\\\\/cV6zGD+lD+eaSIJeRznGy+wdHSJ0Yroa1Ijp+wdyXKgJfullive3IMapvpQlKTvZrWvZZ9lFdruGwnBevidy9Ys3ZvEz201p5svSgURyVHl8LpI8EDYoRgBNITukfDkTp36lbKjXW23UuUvrCPAroIZ\\\\\\\/e3vf4q8zHzSP+3AgTWOodDMWknRMH9A1SZGu5\\\\\\\/C6KYw5ieIdU5AKSVHhIizah7CQrwDNG8qP9m+guExJA5ZI3Hi3WkQPb\\\\\\\/4N\\\\\\\/IKABH3KfFnwgzZsujCKTzwpOSDdnfLaNo\\\\\\\/T1yvjDQC1O34TI\\\\\\\/So8SJ7wo6RxqRGVkslE+aUh3H5qmgAzQK+M9gJjBKcXh3D+okrSMi6ft9Up47n2jbWTxu1nrs65sBmECcfDY6IMnT0tO0NjZCxGfB\\\\\\\/oKKXtiqerUbt6sTouuryvrgv7ULTlQiZMeNc2D2hw8ep\\\\\\\/FLYZ3Ia0Rq9bDZKoQ6Fxgz1Bi0luHt4SwvNb7\\\\\\\/Z0NrMZVAb4JoJn+Yuxr8hjFmXYxvwgKt9yr7eTByZnH0otLHxiJjJ32bwNdFdw5i5IgSWLg44q0LpL7XFImkAOf3OkVEvF\\\\\\\/iFWrzCbjZrRD8Di+PktLE+xmUIPuz7RNkno7HWZx8XbLs4RSioEskf5T818GD5eCIy4FSLjhFIIVLnkFuaL\\\\\\\/ncsQCWWlGALmigQcI7W4SrGIyQjolKIil\\\\\\\/Zn4M0K+N\\\\\\\/FIOLP11mbCeymcwI19IptYGzUUtOVVHPUBdY+FF\\\\\\\/CMYeabEAqOKt8tI2YTLxlLr70roeJHCjMYhdFXuv\\\\\\\/y\\\\\\\/8hCmjZwjAYdMpueTl+mj3qBENEmoC7Kb1C1fXsW4YsamwNCgFQx9LEzm\\\\\\\/4nE0AGcNb8hGaEy2zvafPgdTow4hYECIBJH\\\\\\\/eDlmZAl2Ee695\\\\\\\/wPK+c3cjTbywlyZOf0lQn4kF9WZ+AnnzDAUBbFFa86cRRDYQ78nr+gyaSm7hC1lOQbIXt2niTaTpdE+imfX4ni+URkH3Scbm1n18IGtUIyk+tBb4n4MXSpCSfkk5qE2AidPy21aWm052FFaVKA\\\\\\\/PvFZaSuTGSDVTazsirAzmjfQpMXK6jeg7ZKKywwsaX1CKTUGgthV2+4OlJ3ofApvdD3EKwQXle7YabDD6s5TUFaSgTOa0FiB8TGx2dQiRscGy9Cfsp4KIde4BCX3L9DIkiJVByNxJkHvoj\\\\\\\/uE2r+jZ4xU6KRABqPLIBQ7GKh5Ev6s2nmynxvkGmDBeouidmnlRdlNHa17HhuNYVlwCVVw3QYklOxtyRTnvH1B11sRC8JLyV9WsWZ14IoQh4kdKe3fpj1V7IvGGlxZ0tMqJL1lNKzQpLorisQl6R9nuu+zkKM21\\\\\\\/OPgp1zDaJc4rhjaFELJu3Thfw8H56V6QqXFFFALF5VfuQiIJ391t1FU1PMQHLJ8LyKkls\\\\\\\/BAb6vyCJwGm3BobUTBgogdCKnR9oGOKTW1hNYXrPFOM8y4aPvML\\\\\\\/lJzbIUTBePdCi4b6xc6O8MzydFlvG9wZWVCzpQbTgJGsZKhIJH8H58SeMjG7vhUYCK5LT7fZ\\\\\\\/kr0tMturfqw3Gzo7+fQy3xaKRMzOEGih8RFYvxl7ZjhxEHGnkYuhMB6JoBnEXz6Gf0oX+bYrRzFkohyFENNmu4WPqXRffpqVv4l1KLTl2pT1P\\\\\\\/kaDjoCEzbwP1nw8mFWYr13+nwGxt80JeCOyDzEJGNUXf9ChDf12iDRFYsOG465Eomq34AUnIKjfkb0lM\\\\\\\/Pg9Lht9TPy0C81bzzgKE7htm6jT2ZCo0nRXV+pVSA3DY31\\\\\\\/8Q3kDG+rZomIUIASFWb1NcoyRdqIqwU46Yygk3CyXv19pRdr7QDsJUWp6FmOxRg4EBTgSywVjvmXmkgXJy++viO5MJgEgNtBYHWqaweiUSyG8jBS\\\\\\\/EGsEMa5bGEyH4fLOdOHWPgoPDQCb7ZG3nra882gmAQyEvvQB2Pe3l1RrFgAhNsKjHwfu7NLqrTMaHBiwdYtU0Yrbnu76P4pYprrSHJBqNP2\\\\\\\/bhxvenPElc9eZbw00hz6k1vp5DMQxhXOptc3TQ1Xnldr8zEk\\\\\\\/dpw1y6EKAzea44kIkq0GHIpxwRXYV5qI6YcmIousGa1MizXT\\\\\\\/Lni99rVfOwosvOEc\\\\\\\/s4Vc379Vmf+lu\\\\\\\/eFyGFVonEx2yjNP0RaOGdj5ZkyPfuUVbTPKEYDRqJGJMpCqQ2IoW8B2+0TYV0hBNe5ILpjAbkOpaJFG2xbMIZ8WE3LwfUdp19mxN\\\\\\\/Yldzb5m+bMREvvd26Iqve9BNncnMytlQbe1XIDAudw8POOCn60GbbVhb5lKysrwBcHCl\\\\\\\/YdiFp7LfpnPgVP6JvYWxVtSoj0K5f+0UavihgvGsEHojR4tGZwGyJhFx57jSwlcUrLoxEc5Cpg\\\\\\\/ai0du49asrFYA6MJtm2JgHjF4OnhmHW8ufMvUn+P7k1y0viP14i6gLFB3cf9HMXI7bQjp\\\\\\\/LSox9Hd\\\\\\\/eBL9B6rBLWOPj4YHQNhJd83AlcTInQkOXSPmdR9y6vE6yNghPGeA9ZE1JW3ZWQS1+ZEPwaP7vEF8V9x8DBf1gjH2KsZgtDWZiavVUhiv9IVRhMpiitcdScdiqcJdJMtR2gblqSDXziNyLDVoYnJ1isnx8lesuKho6\\\\\\\/JQWXB64nAm4biUOjR1b\\\\\\\/RMOG\\\\\\\/Xtr8kEZXG2PqYhg2QYErqLQo\\\\\\\/3dpWRoa+lz1C00oUP7lVVr07V4KGhhLCbhoMUZ0tQnetxb8WDjTdhD0sJXWGpVRSYShkI88FjtCLccWXwx0zto0zRMdgFymZ1xRHy9PJ\\\\\\\/qXjyJLvvAYDyQfm229eMFkZrfGKv2B3\\\\\\\/hiOiUJm\\\\\\\/7ZDtfrtmfNPCsfzYK7u90IyDs5PA\\\\\\\/tFm6zuBLpC9Qw4o49d8k8QOlTnpVL2tcMGzRkNF8xyF2CRccL4+tneQT8U8d26XvomSLK92YsJX3AMkRybSV6Jnq2dorj5DEq1Q1bwNzmpICBqh\\\\\\\/i1H7ihEf5WOgBPK4FobiI\\\\\\\/hKDB0sXk2igRqL7eSy3Iz4Eh+esK46dTcGv3dwioQZhDazVZu4o01pK1ABsrn+rU0m7Rz8QAdCH4i4H4w5wcqYkZ16HalJInYe+n5pW1OhNl0Mj8uESDvHNfBltRC+bysLLCdLSBLHSyIiJZ4OtBxThEA2AXbSOZjZrTQFV64+Px6Tab8\\\\\\\/PrQVQV+bvnA0IFrnXxm7XeMRC4a83yO8mU7VUQZAJs7KH1f9t6n4HMZdblxAZ+VOf0DGI5RExqroG6rLWIY37ssF3IKVC0EpMQSqZy41R4\\\\\\\/fF35wgj6GwhZe0XLQlWkTUxsEQcBnHVJ\\\\\\\/DLlNSbB7zlEwr5DGF7uhDtYWQR1YxJvPG6cpThYeCaYZKLcYeixMf7Gz5prEpKphGBRGlZggBiOxDT6KJyjDzkE1Ntaw5fZKxZMFMKZhuKMERQPBuQ4oGnvTFZTtrvNro7Pj4eSpYsLgbPputXIJ2cYz2uIskadvGmo4gsz60kUSA8ZqURAhVaKSTao2anJi69kHLvPkXyjJPiacCxSCMiEh9bam0kNYuOETbPK8FYDD3hIJJNJ9B2PSoyDTc\\\\\\\/r3cPese+qGSxgZguLjbIVVSuhGSo0g02QqLaH5pvlBWzK3pMvoA46xTr\\\\\\\/LUB19\\\\\\\/kcHPSiO1hfJhCoTbQVvJO5tLPPhEfB4GymxSBNTLvx36cs7uVooY5gLnDzP+4yMpxH8RCJmF180FFCB\\\\\\\/unT6IsI63gxo+mEqtSHw97TlhgjZFfmP3xMN0PQ1I2WdSfRz5kJqOzdYVNgbW4sTy\\\\\\\/sf+pnACWCi74PLb94uwka8nq8B8a+vdWqbiGzUsbQkqqQgGS2u7T2PLGMBd4ll4SY8El\\\\\\\/t6yDmG9MV54lqt6tmhhw+zKZ2A7xfukwy4CsEcwKhidwhSBGUxCT6\\\\\\\/nYezScPH9cDIjPryPJ3BE0G7PiGFllX35jiwbEnBYeRtEwr9PXvoXO\\\\\\\/1vlkW\\\\\\\/Ecpos6urVzU5ulOcqQKbV\\\\\\\/GoWqXJWBt3wz4wsBJ\\\\\\\/G+WN9BKssSPuX0Dlpgq+1GGCeuFTiVRmqda28wGXELIn64nNttzUFlQMnSJ7ASPE0NbTPGktkbAXbnA7+l1WgHMSch\\\\\\\/b274baMnIVkLD21ZVZ\\\\\\\/k8t4MrEMUviCro05IfkcY+CQeFkkITnrOz7ZSzFKXTPs2VMRzkNS85ehFff22FgMPiu+wHQKAl047E54n3364B008xnj1XXI6qKPTtaK3XJVUc488rsvB6nRigXOb2SIRQMHjE6lu+kJvmxvlHEhlx9eGdJoMKJqvpJhil5LDJdEBpMtZEbBvu9zn9MuTKcIXbPHgwEoSGt1z07xpWDL7PasHzFH4WebCoelWsHYmNGTBrjVfD2lirKKvIIx6+mPaQzZ5ThdXwMH9q63YJVuFpwV\\\\\\\/QwbJK\\\\\\\/6JgMDnESjEULRzepFT9Sx2\\\\\\\/1J9m9bNxjj7cU5F5WpqdZv\\\\\\\/bnMOmJIbomW8FJfsYKE1whN1Vb6bk+\\\\\\\/UzYBgnEPuQHP+TWvFPhiCwqGpejXJh8k+nJfdfUugj1E5BZBC3IDdqU6O\\\\\\\/LdDwBMlc1zowt0YMWmII2smkQYT2MqQYImpDnz57SOa5g3sze7JeuqHE4Cu07sQVSz+h5r2QPvFlz6dGwIC9pHBcQKhPofqTALloWjcEqI9dzBr9+WhRHi8F0oMrOyI72b2KgWmQsEHdJPueBwCscRj6OpIXMvU3kyG0Zu1ZDebBDBFGP77dwnxu9n+ucxMma22ttYwUgNJTW+5Vuut1SBYnJTV0wtCizGWkqwNpYRdYs0DWnMiRowvU6wuGfvVJHeUKZmAo9CPyV0Y7rcywuPD0DKFFVVfhA+EJz5HIfZxgiFE0S3j0L+XPwn\\\\\\\/Vy285qwJakipQJxeBvarDlylTmIjHzyr5NJREh3jhe\\\\\\\/+4QJ6FBvgfwRFLPyiYIbihA57umkeqheUFLKghzotGMzUcHV2ii6S6l5FITJbhtEYWxARjPb+NnVSyVhgB8n5Vryz\\\\\\\/yvWmPuyv2XFiQ04MQwCOOxnX4jQwSpIVSb2KD9CYmVziCDid0NxrwC\\\\\\\/sSeazYx1kfHzhwbNVllHgV\\\\\\\/fRNjIpK7F1FQdUg97odYsFefEF7HJNK84gKA5JXJ4wqkE7zFcHw6FbiY9RjnVT1Zm6ttiHdvL\\\\\\\/V9pWzNe74HUuhKKNWisfkfJnddS1qZeMfVIKCPs0oGAerpTtjrTeKf8ALko9K\\\\\\\/cnvMqDs6d8eo0oYOlwmnPk88UMGnv5tMUZ1AXcGJTcZTI28vjF7T\\\\\\\/8TQmrvN0uHwbITg3DtTTyGXMyFNihBSFMj+ER6VnfPLiGTDXjuXcsDQFv\\\\\\\/mMs\\\\\\\/3D+DUSqrS5AGxKHw523YWriWgCUjrVcsn2qtKgTAzVd+Y\\\\\\\/Q1eZTwXNeJDUwiQL+dFs1wckdeNtX+uWxKF0FGIv9YABf9gnafIK\\\\\\\/b2AdVZKzhoHRd9fYuwasDzWcQ8LpKbPD9C3os0wflXqIspF07ifObHERv90ivkYNavQQcFV4rLj5tSNeZ5kC9fV+gjGVMsReZiijGk5\\\\\\\/6\\\\\\\/tNSrWg3bqpOucnPutKUnHagaxK7qEd\\\\\\\/SF2R48Sc9lG9cw17X1tX6MFd5jxzNNysjNIdxJf94O\\\\\\\/gY2MEVui9sYPG7V0WOkx76yIPId8G851jM44mgahMEIWsGUWjJOuVcQt82vfBvcYKVhJs9yfrMLk1iN\\\\\\\/8R61lBHsjHidh3Tx5K7ljrdHP\\\\\\\/Go8iY1WHAGE539r9C8k\\\\\\\/5DInVIM0O\\\\\\\/l7iZY23S7HBHzVqcgmZIdFPW4n+ZVW1bZ7DzXoSl+OLhjBcU5i3\\\\\\\/Oqgsn0ThYB9rPO1PEBmDl8cDEDsPkCiRFMg9kMmLeHRvzw+iYvWTrj0WeS\\\\\\\/gGUeXEOmFuJeLj8WEsjpvPN5Z0GMqGiwEM1X5zxskfLJ\\\\\\\/ODb5OK5VFuMtPfZLvbiNQ5whCtb8RXovK5EtDBJOC1yRnPnCQagvDM8ZTNeVfwRP8iNrcXbyyCN00e0wM2SsW6JvbBr6\\\\\\\/UdYLmV+LBS21UG60fNJLoz7hiVw7Q4s0F6XQeSt+uJKs4YanLj+5Wb\\\\\\\/abG8QPOLB1c59Vp3Wsr0ou94mGD4gHaP7aI1LC7\\\\\\\/DJdVEDPndDBkbQE7rkpBnEmwkKtBpBbGeHCNmyv\\\\\\\/IgnEE8YJBYJ4NP7h0mjebBLAWHlQbXfJEDDiIGVS9onsjfHlpJINJvq5EoacKHcCBlv20JRwz65YEMeCJf89RXIOZHm28Qa1FkYDHrNT4JYVrfgONfrvB\\\\\\\/0CDGD\\\\\\\/CQDN6XvosIbdVDroMCzdaSAB8ly3r28BO88\\\\\\\/UU5Lct0v\\\\\\\/K26Rw2PzBFIgn41uN6o3wz39BAEPMaO0GTMgVRzLyB4g7iouyxn92B6+jE0fyETBY8oee\\\\\\\/2TauygL4c1IceF\\\\\\\/oyxuyZXVi3bdjXLaaxh6oW+6YmpR8RyBaUX7+DeYjJd9RJwS+U9iNpUPBqHZXODQ35DzVtFbkbHeizJHk4CUajJR72t8I+E75jG2MDKbGzODO+FsScKWo33uDPHkJRMthJWVmjOXnCe32NfHEbJaN8RWJfkI6ew4uXz7tqXQ8CcVmrzTw\\\\\\\/MxnTIxlainOnvJjcnd0NNg1p3Deui9puuuXI9h52zjNeEgzzd24slU0M\\\\\\\/W1YIACOsSuT0pd0zZZ7tRl9AeA8dNh2lYqap1Tk0PmPpeVnTEtyPx8RiJvtqH9NabX3Doyr3deaYTUnAZcQ\\\\\\\/4Wd2q\\\\\\\/mVGrUn4pXscKzCSy6NoCiLjCIRNTvMVjehvFQ3IhsGfhMs9FfgmabwA45933ofRuvPNfyk52Bn3h+RulFPWiEIon0mc+9OfgKY9qEjmnob3YYxlo3gMCsJR\\\\\\\/4P6ZGLx4xX8a\\\\\\\/9oVQpsTB4Bsl1ds6RpGgP0x3SK9+I83pSFNUV8g2yAhc+EcJkPj5jC1JlPqkPZnICskrH4fwWh08xMRarsXaRrQUHK796nACKauJPQX+RSzsBDZgePseB9W03k4Jwp77EAT8g+q31z7Zt3QCiyEA\\\\\\\/fYIpYLdL23J+e4cUAQpJdcLL3ok9eUQFaWjTsZM3FPzXDDVsY5qIY0ha5Ig8Olg+\\\\\\\/YJ1oj+F7cklvOdPZogCLNGczfDtiaCMqsFakznKHQ6ZSOGZZpnvXSj8\\\\\\\/Q0O73F21gHvrwSYZ0hd3W\\\\\\\/unvYmmuNR4F9ZtoGINPJmrdn4P9LSfG+s8Xq+Q705sHjdome3rnsnmLzbjfRh8ATbc3YBY8794teas9wF8lND4nh2hzKdR8v\\\\\\\/z7TKSkxJgMjEbONvqZmWvWuY3haP5LYWl4uLI62kcRC3yegNQJ1FlPBv5DFBrLwsSYeIGj39pb3Vm4GuS3PanBfcgPuCt0Kyd6zhjVRhaqnLPYJGvumxAcaJHVAid\\\\\\\/qkc7Sf68QyAFmlOLqB9MW3SAiGX341uPAzNjIhDt04dpv3f8Od6nYOYucEkqUP5dCXURmd4\\\\\\\/Q53pQY48Q3i7Hhq24QR5CJWPvS+t+SVjAfRLbrmiyh1t4T4okQo6cMnUEnPuFSErWJ6y8n7o7JwAXztieICSjZLK+HRe9j3TJAPZHCcmPDeYnBeMvz2BGBMg8dkiT2zUDKhaMqzR59U2A4Pf9w+ABmvKv2JIwGWCWl664\\\\\\\/bXMXQs2zNb9EUqyoJGa9iuueYTL5Ok3rSxTb8Hu0OdPsSCUux+HSj9MrqmQACooif4DVixk0WKOmOdJh0ziLWeYPQ2G8+HkITItCLpCw5rkGtRXTPTbSM2d1YAcP61qtzatznNWBY0FhGMiROSVWIxI85fr6yECRwxRWHhIUiWeL9gND6QU1DbT9iK9VxUSRDQmLVH7S6Qw+0hpJJB45UqaXX82VWjkHwisNPxWVRHQ32HoL3sobXVtalgTl65+SCCtetVFYURppcwmC7JFv4suFqD4Q8AEf0omSuxaGma+OSO3uLBqvxH2t0wYZkBa2KdKnWwo0lRCknflK2WgyTfaOoOatrI8zr3RgxOCWkIoelRlILrs4UsqWguEBbnHTHSgINA\\\\\\\/JnwjDHD2PlAAzPHqfpdUk\\\\\\\/+h0Irv4N\\\\\\\/vSN3Ny2qlkW3T\\\\\\\/glvIUtUUSf5+NO+hkPHxBVb6qwRDo97pWZUjrSKq7H9faBOpXVoaUGwd5slNXEdwOj9Y01xpekdBac2lRvz5ipBAROoQUU+VrkKrlQ+HKzT0l+8iTKZXWyxcXhQZJZGXUsmiF2Y35HI2YKMudFonah9bUxSdb0iJGwgxAny5pq\\\\\\\/Gt4NQDraSTRmvXwKB1LqYwxPeDZLs1dvNPCq7N67iSgL7dwTrsmJEcWnx493RS3Pm6FrlqrfBbypykbgnU0fLoXd7gesiDIlyudwxwUCgAl8e5ss749ECORZiyqV1zCOTfPi4COS9WY3a4pA+b0\\\\\\\/jtjNvj5qpP8dIKFlBm3SEgilk8h+qkPTZlvobPfrQRXG0EtfXZuQD\\\\\\\/HVM\\\\\\\/Gt5SFJFi\\\\\\\/cbcka64xBED7ApbHp+lXViUbTbsS4O3izpWjqYh1XdyseD4QksKyEVSQ+zjwABWPy298MrFKNWUE8cuYQPE3rdFIIbzbw9TwkSRCQhugN40kaXcaYL9sXug8uBrzaNKKbfdh+ZRzPd9pyDy4wj0PS7b90sc8aMW7lJDD8KTzINuDlg38t\\\\\\\/3O+ofHPA9pZbKNI6pcpsLDc+XFmNnjbGP2GSVpITYcnRAjIiMPrZj26RRFihJkHlWMD8\\\\\\\/ykxICnmyP6h79D4vMJ+uZpDv3jHgQMOSmZ\\\\\\\/g8EArU9IgiQb1SapXHHW37R5RYNEgzY7QkL+OhOAAM3AgZV5QmxNnXkkY9st2NBE3Ci7Vw\\\\\\\/YKFJZZg1toFJ5FUxACz1E5EuCgxr6iSx4E53KSAIhN+p5e7S85YjK6UeOakFBdEAvWO9GTSUsh1WzInxmL+r6KzKLpXd7mhOz4SRjhrrOMnqcH+IJLfNmfYfCN0A2z2zmDWCWtvLLXFkTuNFdimpqk7RUaYknnt98+G4nmgr8lSppNI5t7l50sfcAMjpOkmON2ZWuVAds1E028w77NWrKUXEnTBQZqruaiVsbuihWiP9bivcESl2D0bZsqGBcNtparaE8SaimvlLsVh0DOuOSaoDlDkC8fgiE9MXCKcrnH9Trdz6hOS31VMXDGatX6EUNn9dqacjrCLs+C5jCc4AsaPWURheDOoKMxrw9AtlpXc4ZasKVbcDtkIxJK4uF+0afbLWXJDrNIuGfgas7zovyO5LKm6a6xBijx9K96F3cJa2BbfalKHzps48p0nuCwP9omwINuMWILuMf6U6LbTgJQZUE0jgGWKCzbLP3Cr3p4ZSxLf8hezYP9Ao6uZnYE\\\\\\\/yl\\\\\\\/0F16SJ6KoIUAaadZzcT\\\\\\\/DYO1e8biVhDrTonie38xLZh+KKUNZML+DvotjNPvxZ+2jZKLZQc+S18\\\\\\\/WklksEuD3UwdLXNvNxIyYQrmVLemAd8Z2MEJ7jTXiI6W50Kq1z4Uii6AtKLTtRSZsuDKpilbH3J4hfOQITvbYSpb8RxIDHlzCoQVUbkkm+3m5sFnovx7Ropvtrqz07OjFJ3FEwfdPQyS7GGWkp8GYorjwiNGu985C8zW\\\\\\\/b4LouLZ0cnHTCOHiP9+mqRC0Xf0Heevc7Qh+HsOg0gjuFV8HcJGAvoJOiQspLe8W+BYd2QeRSoF1LLySyF7UCrhytBQUZKeEtkQl\\\\\\\/ynEmsj5rIwvrGSKZhZNr3u5X+Tk0JwaSmMNSSoJAtVFzl8a29dPBYOsudFh16VYU8iiPqbw9e+fMeAfk+dczs+naSQ+G57JE1iLrlhAlCseenSfVD4yhjedkITBGgxaXYVVgZjIJFCmPq4\\\\\\\/rYnUuyAqcG3lCA+LtXM1UVrEWPuntRjliwCV5w+2nQ3uXNeaI8TOWiGOQDEKG27OvtTSarPYYgfubhpWBInBIW7h5mc4mMDnunFqjIY1yk3jpIJfDjUk08nHCaFp077F\\\\\\\/7abw7XB5ndO1BRVofwJKUy\\\\\\\/UgGyiNY9Pql7M\\\\\\\/9+AJyw6LcXrJPVdQAwYnbTfUZ+MiPr0MZS+FdlSBvBSNsM2f5kwbOEuHDiQ4mxiCjmNC9Pg7DFal9Y86J2gpzTqMf0TbEeXg9ScZbTwiYp2VBtEvmYd8fvXyUHEIC8Yj8o1OhKUT0SahPTu0Ie\\\\\\\/a2FgeZYaiSG5PW7vLQYFZKlo0ludum900+pOtaI3mvlcIT36Al6RnpkE76rZHxuM+Pld4BmRpbExP6CLfL49hDbfTpmaSlIbhE9rS2SfaE1zN6u5hW74XY3gDhyHoDE9S4SVPFS8KCRp29QO0366jcn9L8c2PWdHwfAot0O2IWBUCe4J7h1PywiBObsMEcgs\\\\\\\/WirbfTwE2E7HJrLgeo+wEvpVjIslBr6d2pMub3HXW12G8ePVX+otwwxaTr7qWKRpxCW7IrLKgfEQxD4hbxnxRfQUNPt5bVJrTAouR+DDZy13gS+RCZo+66S4hNqshQydCHnGR7KX7Wk57Zsp8y8nQSJOIn6UYvgEwSHTd0I4Q28tK4i8SWmgalBont+2i9QezUYeJ4TRyTid+JWOjfuH7CiUQi5hy+vLKWBGDmSBIvoB5Nz\\\\\\\/l7nTbmglFOE5qdey37Y+jqtBC8OD4y\\\\\\\/2Dw8QoSU+Q6JQtCUC5nNxcduBCY7Yef1T5YiDjbJyzchKek7BXDYS6y+YC4Km5GYehytJs9Io1KsQqI2mfMoGZDtRJtgIDaySyHmJgfFxgUB8T0NbbGlXZ55PJIlA41L4v35DLJqlKnpe3vw8xtFfDamvdB2huegvH12i46EQpQfQ5teoNofXLWqRRgbIX+KOQ4MRFfoC27nSam0\\\\\\\/CZVotWJQyGd+GDveL2IA751cklNXbQq9kCApD+1+U1JFa2lVP2pqOxZUUwlJ+V6kl3RZSIMuTzlA\\\\\\\/SRHEmQAPYsR1KB4r6LbRh6cBzWGAtwUl7IIg+d5fLjEz0VTKjUK7cmOklgDw207exfD8nVGlaWeUEOPSH9cid3z4fQBd604pRIv6uoWq0jJdO6dVcY70+QzI\\\\\\\/6e5tXP424QTLsyyMd+JaCE1TvuS\\\\\\\/jwEM8bGMZmjZWLcFV\\\\\\\/8ZETJDgZrJB2ECz5y5qj45iDB+G\\\\\\\/iFAie14zV4PlK9eZki00D+P61z7eNfHCbanuz9EVO8nD8fT9\\\\\\\/heS2e4twrlpLFY7xgo2JKiXOb7ALoSiWxBEIHFUxMJagY1xyNBz49Yb77YBS6IjlB9OW\\\\\\\/aQ17VEt\\\\\\\/8jDP\\\\\\\/H8OhcC5dbmmc9ZzQlWaRtj4SdX9b\\\\\\\/64vDSg7mh1\\\\\\\/ozy8aq3SRw7uOwQZqJ1MY\\\\\\\/lnJRLJJSQD51HsbJBizbLla\\\\\\\/GCKRGikPSDtnB0ZcK\\\\\\\/Vmxdasbm8A0VgXour0tZwXdXUJJPAZW0hih\\\\\\\/jA7DZzs5Bw4SsMJSyDpxac+F5a6nxdGAq0RmXTdcKpKgPBXIoU2jiLrgN6MdSTCYJMqnlDi7gFVE9nLaAdzaDsO1l77cldJjzhZvLesIpuFbE\\\\\\\/M9Ng3US9juKk18mILCYZKy0QjLyBSCwzilmYmK3UuiTqtdIvso8WqMZiOgp4kikrK3048CH9\\\\\\\/QmzSasJkXHNNKjP8KbWKufUkgFLjE2uSC9YfJbywAgYznotllrYEFVbJ6KH44ZSoih3NquEefL8hpeMp8Hh+gO6RC+EjB\\\\\\\/rqkjxWvIENpSJLIvQHGkqt7lLNFZFHs0NnTkfrUz+Sab+tS1VVqfNHjaK3gg0Kd1oxOlb7diVbmjX2PBsLQ6X5jzjrIa3s3bhAxOSZPvnKMHUFIGlCUPeAc8ROoDqaV9JRTnN20QSf92XGRgLcOztou3CaXw8WXPWzzp5ICU1DEmq7BvbXN+W\\\\\\\/lbVP52f1wIjEqxj9bhdi6gUrJkmqUdx3EYMEUnmrK8gQ23VthBOOPbkcNQKe9A4E94cAKpRNVcRRjIDkaQiY3nPqrOL5e7JL6T9iL7HWLsV1JXL67dQpAzfNfGq\\\\\\\/zWS\\\\\\\/MHYOCf5PgAm0MQHDZ5UxL2mvgXa\\\\\\\/kkQLb26tB9bx5mEzh1aFg49nhmnykxTBgREUej5zrDlVo1Zo2tfk54aLD+Yliw2jXIVxHjOXKDlvEFmgzQiIYxZMTHl+YQ1KXNpKlvGrVTHjVqupDAOkHsPdkvky7yeV1xs51c3elD1P4Jxmk5LJNJqVfzRLRSzjWt0ZlmUAZO7QqLbxVIStHUikfcFe+bUkZW8S7VBirZEQ2ynIyQbt01xD2LWGib0DMZa1RLpky3IxulLbNmkVg7RlSMsv56ciynL5HKR9Bf1WqSznTp\\\\\\\/6+QcBmJdM7GkyRArnidqn0z9qiYt\\\\\\\/G\\\\\\\/zuRXENzhfKKcJrTH05TCLYy5HjL+CS1i58JWTkijCNCGc99nMWGEWeDeYFpdxczylyczDWQ4E\\\\\\\/yQTkCCtcwUJUUUA5h34A6uNvf4tsYLPg8d7uRhViWtmbHQwr0WFzAv0y7Ly7EN2Hno\\\\\\\/KhHGXOqCfatYd96jCtx\\\\\\\/knkwOr9RbA4DLQMd6tLYjQnstFnErQinKispTLCOkfaHSeApU\\\\\\\/7PPgzxdxvDey0u7kK\\\\\\\/5j4mHqRquD+d2l3hnNUZ98jGsxgzAy2g18bdFEFmmqjBVEdHW0ogpAjChnZrRKT5cXmvmWbQb1uftBBF0fNA7iSLPzBoX1chp18rWC8AxdFBfb\\\\\\\/PetmxC3oWyDx0sDrVWhroWkcfAEiMPLKZCdpOVY3Nk0hCXNs4q7w4fVjXe3McRVWJ0SUej4ZGyBXolcDc02navWz9CmbK1gUHVDtdurxZbzhvyc4+b2m14gNZU0E5LMl8hkw8SDDyPmf0Ga4PlzYMpNe3I0T\\\\\\\/bAjjNf7n9zwnavAjQAFI6BDbAI1UIfPn0K5XKyGjHh6l4upj5k\\\\\\\/NzAkQjgDuyL5TSmMUjDtD0DN8udQqs3DT2CmiMZ\\\\\\\/02N702lqmFo4PZAxSiQZqPrBpama4j6T9kMzn3oHpryAIgYdYEA3dlP4v2WkyD1Pw83lWBLSxd33k92n+A68esaVcuF4YMZ91HcN5PNR0kMNtjuyjsfdJa\\\\\\\/MBwvt1Z4r63ILP\\\\\\\/dsORtEd3q7zMsqgNMP9d64dkNY3mrsPjl9b0Eunqo2Mv0L5Hc0fez97hBUnyMmhqaCBODinc3E\\\\\\\/g+jGM\\\\\\\/BNcqOIKaUqCP0JaCIK1HVffbamClx9jNIc4ViHxQOIBdZe+KsixtOqfowwM+\\\\\\\/3mlbgE3iHd2tED62BLrj\\\\\\\/gys8kvgKOrJemriYfn1VzYrFZzozQI6CiW6gmZpdDxDT1S2XOg6CygKSNBj9Ww9xg5tkXJIdFTbYD7\\\\\\\/Viy7XIgBDv6cwDTlhkd4OS1rxfRZVBVCNYMNt37tHzVmoGPpv7VgRAr3XTgwWVOBi5KWHTgidUxxW4qpncb5h3s6HONrfDZ17AA\\\\\\\/s9VT5dKACTCkazlw9ERHfxMd9FotEYaPvpNs02LesT5yVHPsdSS4vJ752DkCg4f3viWzP2ABQ1NMGibFAH\\\\\\\/ST5C7RvFZJLIlB3QwVu\\\\\\\/TZIe4fOxjtj93Nsq89RFiDA55aTkNDuGdwtjNDmzB332WnTs9LZ9lA8nQKehW3JRQk4n7oDiVrbinjxEJaTORylTpI5Ggt2YaaCwXL4wGLssU28TqFe7xzn0QVo3oUqrCZC7GGX71zuf3EhKKO4B6Iwqm5Tol+8SgClMBsPxcOIoig5X0I4uzY5suaqe4lYo92EdEXxsl3kbxGIxwKwqQzN4l60QE\\\\\\\/FBTwk5zU\\\\\\\/eM\\\\\\\/phjoVYU\\\\\\\/r9o1LaU2rqBbF0aqGgHQzBic\\\\\\\/egWD5flEeOhH+wkNQLHTfEcSg\\\\\\\/hLh77CnzA\\\\\\\/6BG9e\\\\\\\/hGVQYZlyDL2xUIGT3cXDr+n\\\\\\\/pN3Nk2trtu8t9LLfwoRdcHNISNOcTCKfVeHlhCXzeWhhgQrpn2GbRsS2KSNK2cp0lXUh+I7+8BS+122dWnrY5ts3xRtggxkBqk6VjCthz198cHLiP1YU1WoDNPFN0wGr8o6B+8kW6K7p56vsN2Lmw3eIsDY6UT6CEfZC4MxonPja+YR6Rt0qNQVrtafhtwCFzs9b9f5\\\\\\\/JkUwY6jIVk5d803yPPVZrswjH26N\\\\\\\/CyQlzhkr6ZlxKFBnP+s+uMqFSSuTYdHPTbNaaiQoJ6+y5GLssxcgzyOYimBw6xpMveMwO9toyngwVMJT1OnKYb2hzAo9XDO8qEBjD6Wxctcy6eMWS\\\\\\\/F8gQMxzYubfZGrzmYR7Rdy9L2v1ZRPrTaNun+ndx9ymIpQ41D5h23Yx+iQkvHoGekeI99on1UPTFhCioXAHosU0ayFsBOEObMXOY9vvmgil9ynmANKuigqhNdaZZkRCMe82ONvCG181\\\\\\\/H7Wv\\\\\\\/SVZjtWI0y4uLs9lvJrQWlrXghlhtAp9kGsB0NeltTrOL3a1Ny1mcAnSoO2U+RA0WcaAWoU2xQxXDnB5GIiPGsQe81M8tmW6tThNW7prEcfIgj+8yqkeum1O+A2Wds06iMlgxk8mpeGHAesqaOCnq+TVShGg1G8e4OCKi1gVmJX2jcxYYVfBDXvJm4YDABmQ\\\\\\\/z7LDl73zC2aCA8+Za1k\\\\\\\/fNm7v6GTD\\\\\\\/YmlSfkJCuPR+nvPT6DN01gsYrWI\\\\\\\/\\\\\\\/TvBPF\\\\\\\/Sl+ZXMNb2lY6KUhl7rmauNN0x0KgezfNQPV5J\\\\\\\/cqTlaPV4m4w9FyxZt+LWoVxRil+GFjs5EMpB3bpo79ZpYPRn3MlLyyLayEqhZdtZBmHgSTw0NIrRqhXKRqhQjgA1ThAOvoQNDLkDDx5+NQI+tDFVVUzC3oRrOCCX2dcOIFCeMOf1Lc19HsQDXMunw2Tn9U1dH3PQmHmKChVNXhE1n1CJOeGxQIlZ3PxBy8XZ42yiXQ4wOVvbHzS83zP4ztOB4zndSvlx1KmPoE9V7V2dCUypR5Qvh6XmQXZCnYaS0CJKNo49wpC3rOgjKg7OuhJD26OeimhvPvzmw7Sm4EGlTb9W3LENWr81sereeDTK1cx2mZ4k+E3iKNGQXz7gOQJj5F51VwtuNSabuRRsvTPrnS4KIcPVcD+\\\\\\\/FKao5CLKInvj+jnzQ2LIyMVJo5TZsO+ain\\\\\\\/NZMC\\\\\\\/s5lOvsvF74amJCWrGPMPGNGxVxktg0qeHKT+fKylN\\\\\\\/mzY7IvsnIQrtmveUPW1j6crb2RO3qzepUUsaSxfHAoUbgBjdbAJ8FicDTiBT3EMJLQCDSl0JYmkABi5LaJsiogXqR+\\\\\\\/GuJRRX2hlNuFgb5G0SFHDSrQNAnuTk1yQfFeTXBTxklX4gWWUVr7WgR+6Z5SzLT3liF230yHehYbEEeWaemwDATPtbHtz4HSuObY5VNozHjRoh52oOLfzHCNztWn\\\\\\\/PZvzc6eI\\\\\\\/uXRFQd1WGCDM75RdKzRLtKqsSp+Lh2xXgMNoqyakLwhjgDNBENgs9xeYmMLXyDKj+H0XYIA7e7xzWab71dCGaxP4V7iUKsLoLpdLkpc2KHDPgzkJwBXr4RQ0myys4aPCbPHIqHJIaSNjE6vYVSa1jbdm1hHE4jbZELl9jpi1cN4TjoN\\\\\\\/+7ZVazKP9b8HNr20LAP4xTktvVRCrzGKx5vsIvnGJAKvNmzAswpU8qdTw1xNkxz4VjfrREky\\\\\\\/A8HvAS1sJ5ZTXXRMOZge6JoKwldEeDnQs5TlBvzbrjVpYoGz1Re7Ho7WzO3shRG68JiAAW7yuh+e\\\\\\\/u5SlAE5UB3LkaCyKqUioZXp0ZPGS7NT\\\\\\\/0IYUXVVjqJNgzYpPceWxaK6qH518g3XhyNeclJKLC\\\\\\\/z1vVTXme1kvHhCQOzCLPLAA6YDnT7X1IAzaVrOMIu7umGrHna87dyMj8P2YepzagdtlaXzbDPq4+heqdFdwAhQ5GsrYBa6OoXsYYcSKy+bvJVeYRIppnHLvvQV2pzBFsPd6EHFaor\\\\\\\/s+7nmeBbFzxoOkywAkRINW3E7zT8+9Hz8QYNPZ6wjamMt12n6kJrO7FFzxKyvkD4m0oGAGBG3pG+PIpDzd0auy4D4t5R9sUs3nCh0oBZTvEF35NwZFJyiiTCDbtNy3RtnMGgtqoX0lyLlQJTWoS8v8fhylHK5lcRy+2LHnsXSEF6d13X6LTuSNQovNKhFITBa0ll0Bq8+SzSdsi9HdxF7yYh1rKpRIu0SW+f1dzzBPeLn9dSBsiTJ7R92Lt56u0dW0wMHEWcYcfhAIBHzNl7x7L9fL6+LUlab+S1dVMp8YZWkD4OifOlkjHNVWhYNh\\\\\\\/\\\\\\\/0uVu\\\\\\\/30klnFi22VOhz+BZ4v+bfUqXjIs96ssu8KyiTAyyCgg+yzLA6dukhbqvKUlo7+F4RCfDh7K58jmvTNnCP7IIodtzHv5rAC3s0f6M\\\\\\\/ffU1UsRDLVV0CDW9TJC9AR1lK7F79V2K9uqxHpUbIeH7elOFpEIS57ubukQTNvBSa0KB9NN6tT+x18yaJcwT6qCmkl8hSyNdEOMK6F3D3n7zsClbvNZBBvGR4ZxgCyt3ApnithuWP1KFg1bhbyXeUuaqXUyBwhU+g5Kv\\\\\\\/vPfoJjRWklXOFMrqrVr8CzO1nP3DInPeWSxUP6sQ3tGDqqIvE39N8sGnOXLZzCCzLYes1OmfSj8ognJLN\\\\\\\/L0B\\\\\\\/NDodLrisuCNdVxSTpxMizoJmmIIE5QKF6Bq+5LI1dCWge2CrjgjxJg1AgVxvS9E10EXZc1uDZQCwCL6EqVJRGNMS+B0CNuXdrFo8OW6NscAgLB2G3FXPw4UZ2syxNCHCxsL0tXoMTPe0uu2qOscsPD9JzUQ1+hzzuzwXHVLJ5x5aUuaJnh6IkcvurXksfBAE1m9dUlKoHdGef\\\\\\\/cea57jdIXQBQYM4\\\\\\\/kSMD8QrFljAnZA5O3tDqOfY2XKwMSB\\\\\\\/Z39khDLpeNbzq8vgh1jzgZBwbyFBCtgo6eoF4qo\\\\\\\/CE5drbpL8DaVHeMX+59GtAVqJsZ2527vs1\\\\\\\/8+Qk5TkrNljKzVv0AQGEAWrFCHTrGPI5ZxnGNAolSEjSb2pS85EmXLqqVZFkC4v1LJkDO1uXNrg7fD+Oljl7YsvRFje4JLJ24mRmBE2p7ytlcRK9QWZHNZb2DVLsonn+26qRDvQ+JtXX5CuCqy2erV9Zm1EivRF2NmGYssmafoKnQFavKThu1CX9k1S0F3wE+Ds7I7Z9N6mVXqT1G7xMrWydBmrF82vU6A5LOfvl\\\\\\\/tebko4O+NPmjFofVP0xW\\\\\\\/oXGENpNo4nothqRaC9LydqWgSo9\\\\\\\/z\\\\\\\/NtpTocFhpvinVBkYDjGEZ7Fs60MmX6cTmhyV7H7PwLV74EvGUy7x1vmJYtQhoLetgrlVcyujh96xfD9SgKXSw6s2SIOYXcgKZEfnl\\\\\\\/I9fJtc4CdvQHo9rvYwqMXKUHnYnLwd0+gJeGD\\\\\\\/+gmzZaMxCoqRovvRQQ6R6sVCqptfccrz+ntckZd0kxtjmUxQhcB4vnNRcfOYtUq71vM7agHOfrXA7lmlbiZ9ZqpsWcJIodd7IuA6DiQthM9ciSswVo2k\\\\\\\/FimAbDjEuVP2jMSh\\\\\\\/aULCpUClTSI7LvLirN8+xLpnzheHzFkMUMUdvkCuXL5YTETmCSofjhMkArMTi3uAryopYqHvluUZQ4updwlxYJ+pHeyUrwp9emq4V0UZfa1dr1VCJaQyHR4MypmtZi90c6ijarxCYXw2OB+kV4qoummS8zlSxK+R6\\\\\\\/ODVbwCnESqqKHcoWfDxY3xsUrNTS5B6msUUooZ4eDWRscQ9IqHzTU46fV0vzLUiRm71U1otuIUqcd5fp9nCBvnHm8CTlnnMulsX1ErdMB4dA3eR7TQO94AK+oFzgKQb3xLOXsN1FB6HX2N8E93sePEjFHwHTQPJ75MpsckJbIKaRrs5cQQj1\\\\\\\/3StgkrpuftuLEE0Gqj6Ev\\\\\\\/YictPzJfCLP0kmjhhzRhB5cLODPKmNutqD9JQ8UQsunAJznVjyUNtObmNZyHCmJLfxPMaMJG6wysdTxOP9Ud+\\\\\\\/gvP5mLq61IEtzLUK0p93TxXPu5fBm33I6JaHzbamEPcr6rloLjB+BoPF7s2meA+q7i2HbvSZy05RO+cuaH5tKEwshRczz5kjWA1df6ktgA3\\\\\\\/zpfFjWHQOD+cpCIt4dUZAqCrZur69msDjRIMSkpCIPifbt9vm2T6Pl02nVpbYppf9T\\\\\\\/bv76tZUuXePijCrG6Bzf\\\\\\\/At0TiO1kjDvw7sICdQltJFdKmrvyPbKmcyOkqyICw6IeJoUL3ItQ74Hsv0GrAFZVRD1MiH51bNFlJTZ6XXlbc8Xec24Uxf9smyWYJZOwoFvQuOhbqG11yoMxmxbrBHJLeuERReUitiTfedW\\\\\\\/6jI0IFpa+7TaKFGaOMtkCUeUpGfOxAzxVNOkdhjarPqwbdX+r+1nYU+1EgqIhAzUI36extTEyjBvLd7OhIWORW9+e2eQt1WtrVCTsElbPzGE5\\\\\\\/H1Vz+JXKWmOSClh7evla\\\\\\\/2ghLkqzXmXLdrLuVKNNfNUEyKrQKDjaUNMHE+jxWYuUZg1E7gZFYqKS1LzTpKzA5nHpFk4lq8dmIeD+mv6Z3duY1ORQVcw8j\\\\\\\/Io1Vit2FaeECR5aFhXW9itzTLiERU++lZl9U3S7oANAdmYLXFkYiX3OnBOK87smx4KntDAEucGN4X8TsY1lX1uaKgWz5NOcDReMczBk5H+zedCdQ06zqGeCFYg8giP7U1FYqzpDOUw91YB8cnG\\\\\\\/gg0mQeFnTZ0m1C5L2\\\\\\\/ArBHbUMnYSclnQRj6hosBftDM1yzuQERxLN0MDC1oKGqAsqmVQaiRJJ8G3N3IrzKGD2cXf3bk6Kz7p90m8ZO+66C9SENBFDD4V1CtIPheI09U4RF9io4UFazcQDnrObZgQCRdyAKiEAAtdaX+Nyk67LgmfteTVrOKVkL6n429T\\\\\\\/Fx5f2heZ9r2\\\\\\\/r6+FhU4HZfVyCRe23UpP5KI7PTbFPuDAqzezoFPa6jyTArKp6cXFxOJQpMztS5W7DcID10\\\\\\\/4lKDkFujdGIDlZ6lxgeEYqK7JqTQkqRxH89PbQ4WkEmn\\\\\\\/0DcPQMn3MsW1QIp82yLSW44iJxzZM+g0UOPKQHkPBcx++ZX948tzqc703Io4yIVBORVGbKoVxu\\\\\\\/uJw+RBQYI6H2rg9qlWATONNkXY+VOj\\\\\\\/Tprc1Cl6hXTG7Wj9Xw6r+rQ6tzXA94uxggdaVXzke2eX6cDRhdSS3cX44qSJcDJO\\\\\\\/HuB9p1nMtPrmj2NSiR+mF5QOTaUn959ZDvDKsl4B4p68kCbWjb6iIvGyqcgfGZ5w290BOgPwiZwX+SOopJww4aqE2ft7928fu+Gtq55tZJJhHmu4wOK9LrUrfPhPLP0HY4CYiTBTMGL+R4EOXmgwbzoxriJOOq8eReXvJ86kevOLlNwKLhaoCOwtcgdcegDCSyq8fZxMapGhCW6KEHUlvSGBtDIWDJfUDwjFJSsDRnIqUkLHjiopfsf\\\\\\\/eL4iGUOGyx5UGJ+JYAdv4J1AqeS\\\\\\\/bZ9dAZTSKULXAbHrgnu\\\\\\\/BoWjS32jrKyrzb6dcXEQ+0LbbRe37zFNMw+qHSsZbFWMGdFHCD509VhYpqKAtuUbben9pSmD19fK6osptkX2A48r698N0RkmVHAw7\\\\\\\/6uIKj+7dWB7JBYz1cWLBCGc0Wsxlb4Pza7ixC3b9RPb82pjOAS9HmzK2asX91NtE0xCESmthfGmJKGsUNpiWPu2uCusJ1yLAD5tPYhUifUSzG0+iCqtsoPXKRZeTOm82lyF8HpyQEP0n0FupTEpQFC\\\\\\\/yUwUf7VpElpSTSMEMxNMryL2jXj7Jg\\\\\\\/P8xRORmslMrVdZHtqUYR0Vv3UuXMezxQHySftvG7sOywBCY2qLgboVHDEHoWU1oXIdpqW0qrjYZVjt1IUPWsIPIXH\\\\\\\/SUXqA97sdOzH\\\\\\\/JkwyDzmQ95YbAcyiwuL5EKlGmSTwoO6X6ZT6a9WUJqr9UPpqJ8UwYvdC6AamuoAsoEF69QBoJ15oJrBpjS8EaBxSrSGtk43m6M0RMcYAzhsHMUPqgmR7WmBzMSZcCfL5I5Z5eKzR4a\\\\\\\/u6pdY3gNZuFs7kTW5wskcqG4hOFU0uqUY0dZAxLoFVLun56cjkLovhTPlKiiSI\\\\\\\/Kxz6WCsh0zXuGvFUG5aUxujb4hDUFRnd+cxKSsjBAihxVUQ0Thi\\\\\\\/BaSwPloetbYxnEct52ucgEp9QZI\\\\\\\/hxtkoZBPyxvkHMEwCmTIQfYJaSnKpoPKpCI76GNlL7UOSEVKrdlj+77GAZMdloq8SjXjfbjUKjcg6fXGrz7MuSwrDcEZ4re2yYPr2v4\\\\\\\/PbFDYQ4i0fmLZFAGeNmP\\\\\\\/wHK3xFbfju3\\\\\\\/9CNrlHyshSqX+fjcJavQPkzAW3cpEdYQtP61GW3pjCKD\\\\\\\/INqbQiAlyEgP3Gs\\\\\\\/XZbNUVd6t1CZSNz3Bfy3UHl9hPwsUWpUMkgLLu24Xpri8yeML6H2Ki0yqDw\\\\\\\/s2YeG3N\\\\\\\/4TWzwJOv1BIsajQhxeVYwTepELmbU57aS0m8yUjjFXFokEAQ5fH2GPsZCWtwpeoDX\\\\\\\/JBvJL+zAr2ycOcEmh+d\\\\\\\/LmOcI1AInC3+vBTnF3HRbFTCze4aCfNT8EWK0TskTSyUpxPKGlNpbDHLAmHZVIB806CeP0nRgtqzwDwCGgbrcfKymDNl7ibSyr9hzOVGrnxM\\\\\\\/3uF0gcygRg5PPP9wD3GdmdauYw7DbQ5jzG7A2rzLxg2nWW5VSxwT4tEKN4nL+rFHk9gSj79fUISG5fKLvi8FZ67gEVEWyFAYOWNpdHfFJuf1\\\\\\\/om0x9PlhEnLWGEUM7VRGK0N+m1SckD2NZjQjCFaYSMIiLdWx9g9GD3bXTPmp3LPcGjYcqgiL2QNuDSwmXJwssBW3kTlDU\\\\\\\/XDHMXW4ITSUkf0NFz96McrRP7OhAgUKnEMEDbv4pQX2+lk566yzVx0nTBV93Hst9pqdjcNXofzo3RjxurBbJK0eBUACXQaEkN+G1FShMNujPrts2yTmmQzqrouj9V9elIWFzhvvuBNVW8Qi3WV+Gbic7RhnvKHuB5wvncSfEckrPLuXBmYhLuBKlcsMDjntbT4Z3IQP5ei087B2d85uTvcuW63L2TQtGBf+jIW6zfq0i3qmvrPAvUNFuZIRjUdAQQCgS1gGydBdCw0+3TwXS\\\\\\\/Cy7oKKQ53BvZNddtYG5Fd8Wd\\\\\\\/xJpjtTe4ydczHjvn9rBqIAkdOFrOJXKg\\\\\\\/spC5ORfYYCGOjU3MWmB8tVZ\\\\\\\/wt122+OjAcCPgHjoz0r6BNHhiVlp7MyQw9uVHyQDOCbXOmVTjudhYryxzYL3jdU6whmb0E2Ve2LFxYBjVs+9JxQR2rM7l81iXNdKLWMMO5sXlPaQvM1594CsfD0eSHpLItfQVQW0gHmx6tE9x0MLyp03wyyz9+aoxUR\\\\\\\/GGSiG1f3QyJmPP54QcGA\\\\\\\/wSfMXT\\\\\\\/nf5zvq\\\\\\\/7TIOrxIuH49ftIc8f5fG+6hTV0YF7KrzcQ\\\\\\\/QzmhvAFGzazZKWCmEWzT\\\\\\\/RJLjvIUqtl6NBBasAnhclx3prKRhNWJ73qB7qxBfzUlwR3F1Hc+H5EjtvpUMLdN1z2ZV4Lsfp1x3\\\\\\\/Hl2N9U0SbbBRkQmjzE6YRaDyHw+Rw1jzIi71zcALbkLTAC92TgrcYxzi8fP9vSN6MLfxFjwlJ3jKO\\\\\\\/NbPW4GjkpZczXSPfJzWpIKMTLs0j1HCosTzSBtFYTYBRBwkquE6o9Jh2l0LDhxuq6zYcXIRHjnhsntKtAt0nxJJKeg8pu10VNDBwCdhIYshTIUDkl8a+Ib4dEUrQkmVB\\\\\\\/rCXzR0OcF\\\\\\\/Db3FRoPKkRCcejaVxidZam4YJgIE4dST77RXtvyauBuRXuA0jS7v7F9Pk3N9NhS9CdPbtoKLrNeeEOSFv1BRHlEc+S1Ko9jCIO2CwbRAHEZpXNJ6VJPV26tSBrWuDTMV1KdN\\\\\\\/\\\\\\\/JIcihRxz2saPR5gTo9T8AsGq19rV9+zr8s4anby+9uyi6MBIGHMaecDrkd0wsrnra4FuhCFcw2btU1h2pt+UzqOHrgsnj8PKyMSexLoBf0Dk4BNUkbImiIPFJ38\\\\\\\/OYEFKdvecVa3+CSMo1p98KifctqS0q8fBgxFIvQP6ILJa4MKQgFHB6JdcBXnqVrX2ZZc\\\\\\\/YBcycyZ8Zvq4ldlYr+YYJx5qWCclQq1pKUcIK+m26sQzAiJfDgZrug6wRmXxHFJixCzY23iK7EKdOgpR4c9cCMqGUC1gu18utksrJeEbNw\\\\\\\/ItYD2VsiJC0W9Ai5Cgmh5Mf2aC17lvFzBoKFi6G23na093AybVLYmsrmLbsJ4phLVKcpTYHO89\\\\\\\/FNMusgeb+I4tDmxOVRw\\\\\\\/QAGzAUR0EphLoXn+wXrgOfxFLlmDovk71QQId67M6xesXQz2wAS73qYrR7YFfPUFebs5oCfRRQ+bWKxAufpe8XK5BfSNSwLALSUG87+scKUzAW1yP+X2FADmlHOyEK01Va78Q2NuzxopYckCl7NlJAKujXIfKCgI\\\\\\\/9ZwwVbjQmWuzpYxe9Yb75XNGLMrwB4BPIqw6kC61BHcyiJNSgHhFVAtOHD55s0aqAYhcI2usY90kB17dEoA3XZDWAqbIAUDdnz83Ncriv+TidIPE4zZ+tU1tBf6M6Mo\\\\\\\/2ozThPFt3Y5RR507\\\\\\\/sojIgBBdJgMEsSVjD8kXlo\\\\\\\/6Y5+QEelQSnvHcLqHrsR9zcVQOhR3rjzpFdwuxR5lSNeKlebrYP\\\\\\\/7bALv9uO0rgEMkpSmYvGw+wS18gNBTQ5J6NNdtx9RLWRpqt\\\\\\\/QantseebNERNHMAzrWHGtIe6mSUrEv12BW+7C67bbvqq\\\\\\\/AH3XZE1rpaEFpHaoEto6cyzK\\\\\\\/ckcEXuJPM3Btu+Q3dCM9MSTCIPJStewsjNqBUn\\\\\\\/Jz2sGhTmR3p\\\\\\\/jllJ5fTK4tWuP2Gt3iVunsxer6GiKqr4s5NXI7Yy2p5puBghMMsXKNkE\\\\\\\/y7IxUBTLy8aHNNHtHiLCR0oHHSTFJ73IhWoJJ\\\\\\\/d0OCOBsUFvBARcfT7OtdsVJbl45t9yIpEOvrQzIpGYhyM7ihwe4YtBaqemYx1QXy5c6+qwgEFIkACGr\\\\\\\/6imucrgKzvtdpB91+0JntCiMXVA34sBGj2YOuxQ1GhPt7jkkb50EUILAeaBtmv\\\\\\\/pyQVxF3JYa6VZ3Qpu0zqs3COxdiwnY8P1ZdhgxhlCSMzKS5k4OqSLuw8nr6hPiWEUNvWA\\\\\\\/H9PyeYb3JAXmaBWeNm0RbDrm2snXf+3qFLtTcj4pmmk9gSoxLw+fJnlVEajT+lH6THnlVyvrqhnOAy7D1cLw0DOhvk3sbgEDCX3XfGgEKaT7qCQcciQkt9CE6zV\\\\\\\/Z9s3hK\\\\\\\/dYci8crtCg+bnjtlDOYQlELnaZu8FXYFr3EH2nAuzaZudD1XvhFVhCDBBCuIGSyzomq8UOXs\\\\\\\/0V9U8A8UazyPNZAzeSLYs5x7tF5GIhgmKAFKFGTtWAx3Q8ShoyWoJzvBFrgtlwzxRdyWE8J4kelXGzdaBJWpzKNRvJ4C+OLxS+9HZo0FYhp94dNILHc\\\\\\\/gR4sjJaS2uu15Gew8kNScnKTjVFriNgSNk4SHOxoOcBI7Tie9+61BcqpZlcNjL+Ao30KQsb5TFqEWA97De3v3YmpGtyrMoN71u\\\\\\\/8ZJOo\\\\\\\/VkG451XENwyXnABjrQUMCRyNGpza8W2l737vT3XHfe2Rk9VLvhO7\\\\\\\/81+WIDjNM6LJ9TxzDDFH87cM1lyhDB+jXsJ3q+uy8itH0WOvsddMBM0q3iRbvHStrz+a9ApQrUpBH4RrgOEgS9C4VUlFeU9Cui5qxl3+YlsrSx2OCt5zt80CaEXvDpGeadHv76reFOD8YhmfyV6Pf3jU5qMvkIapjM9dyN3nNDdaWB9Ajtlnxn4SwR61z1TL4kZNYhBZ7TMT4UG\\\\\\\/RYawJ28v48h\\\\\\\/3AYl18pGxHItFvpwfOtjrK8DT4WaTDWMb9Kace0oxPLF8fU6LT32pODKBp09mtgjmSHYPFzBOX7oH0wSoqr0dbg6uq5NGpuYRbN194v+ixg5XmIGHZ6J2Py3jm3T+1nxl2rad3r3hyTwDMotxL2YebjsASmnvQEijTAt0vtDjnpUMYoWea8A\\\\\\\/plO3u54i5f8MhHiW3rTFT9xCDrZpIh9C4JGsxRu2igMzl0C356Jx3h98tj++18FAGoehqKDYGX72fN21dlJ6gHawOuwfZGM+u+kYsfFv\\\\\\\/jBxmDmXNzTFAufDGVwzyLu\\\\\\\/cQyuZf3Jt4RX9rZwcEFbJTT+vhx7Xri1sUwrD\\\\\\\/PttVJ\\\\\\\/lWPlR6W8YRhlOAEswU40ZZIziKN9+G6zXaD2vFqJ9QIZ7HXJKK78C0Z8rX1Bidg0JRmF66gLQKkFj81PAZJSBlOyO4w3SkGWOVe5nSTifal2uSBH7F5UTcJMS8GM2Ey96alt5UmUNFQAoCH1OKKbz0nh6PLfhxPX4JKlmoimEeaixrJG2bKx\\\\\\\/m1ZmhqsieS8c+HdupSANaLd40o2ufpwLY\\\\\\\/\\\\\\\/YYGY24NxlmeSrtqDKvnqZZQRXzYFITU2IDoooOcUivtxwt9+JBfz98iAEFrbx0VOMMbsuUtrZkJBqR0RQu4wDKJUkUp9PiQXyVJyp06qoYyWdU\\\\\\\/KcFQOkAV09Hs0+blhL86vhkUCmQ6BGONuStMyAMdL5xEH8pzU8mZizwYSzogbEJWo7NOIbt1zPaIa7G6Qe4i6hvdJb\\\\\\\/zUwpVW2IV\\\\\\\/vQsJltKbR17WHNF\\\\\\\/xqy5eipT8XgIGCZVtm+B+j\\\\\\\/fV91CwaJbdsl3jNIfirC55FbW\\\\\\\/S1giSuerX\\\\\\\/s2dr5pekp5MRY0zbv6dKQahTR2XwS\\\\\\\/CeHda76ZbfFYichiQmelJOYe9wN4UKigxV5RVPzs+DGMVv1nu\\\\\\\/r3oowi6VAsp3x9LEeTzxoeW\\\\\\\/a7KunBfJxAb99zOjgMCcPlBZuQJnLamukWsp2vVTHP5b8eFWlU4tJDt2gOf7qdR6WIkJU+e6bJKW6Sy9Q4K7Mv+VyqMx4uYPU\\\\\\\/2Wwd2UuoeALPxcYXp\\\\\\\/WtfQz4PvxR7AJ3JT1uRxhN0zYbK4rB9bhZJvBizsCgz8nyhKlfddyxMESCalk+RCsnDt4V1Fm\\\\\\\/373QR1T+1bc8\\\\\\\/uwCUOHYq2tvs3SFYgreZ9MFt9heQ7UUeXomMZRYyMvagDA2oUrHcYR0VGKZOEB0ByriVQzy6SCOLmbiCkY1RjXcvtEuKAysOmmrriEm+2qgwYUh2OGLkNBTzV1l3A24lfeJqctv18Rguyq\\\\\\\/qS+86vEPtA+9KzxsWNAsznNAzVKQyQW8XzlFwlQH+rmvy7US4L3conImmqeyaF4bactjp1tMHW5WyweZ8SECVFY2bLmx6By7eEl9ksJEnLHP4A6v+26KpKWKdo3y96WX40SIClJwl4LkJUEdntjXoDioX4TukA0AOURmYQ4wWqIciSNxcwJrKGHwfWaiuiFpRcsDwC07iQAz\\\\\\\/6TjgIzMA36sQ3w6BMxRsKvNv8QUZ79lfgDqloUoQ\\\\\\\/i3rRjqyTHkwY4dfvOsNFCpgrEZfF6vbYOhd6UWp3EsHyzhXMJfirK48XrkWhDrc94meQaYy3HrfXaalK8XA+TCfcy5pVdeyDD8nOarAOYQfuVsnuhAS8JWUS8Xk\\\\\\\/QmLC5jpdNEi6ZfbuUnh7edBz+W13Kd01dR2h2q\\\\\\\/yKtmcKWQtN\\\\\\\/KiPqLbX9cJJT5ibi4VxRO48oxJ16y\\\\\\\/dtYD0\\\\\\\/1M8d27X1T3XkYUhQzsua8AIIKO7DjEFPCfgQAu\\\\\\\/RNwhUAeZGSA+pxoKD+lcCIYVi0cEuUF8pED2fw5OSYEbQIuxxUwV5orBBHCurVjlp7ex\\\\\\\/pLxONkhvwNKujg3DElG09ruZBQCe7+czlzJOMPrEprDwvix4r7At\\\\\\\/RaTPrT0balaROHnrZsVAgtwnBSlyLPS+1v3elZ\\\\\\\/A1c0w+0Fv6Aewg2JaUWSRPdcxSk+M5MsvORptyrrUnsQ5X5Z06a6aicNZhYGNGEXyGGe8XgifCbsU5VUyYy7\\\\\\\/9X4eheG1gUg8IODTMaRVGIJWI0KeHqRQ7NRD\\\\\\\/jWsAqRF22Z0DkVgD9QF8EVOczdtQWf42sB1vNQCR48jj\\\\\\\/tn\\\\\\\/6Na4amWmzECE7e244zUvBBPM9bUOXI+hTHl8ZvBPQDOv5pJR+VuOuLge+EldAosoU5AToKW6HdjNX7tEIyfiGMLlpLW\\\\\\\/DwYhPrpHGle2LnXEbgoh52QYY8pfqcgEp1Kys5iJqFYXfMB8TYH2qHjLUA23pVi+xoX3LRQDQJTz1ftubav2HrKXIRfWdCVkWCt\\\\\\\/UcFjyz8\\\\\\\/E193L96QbyaH\\\\\\\/zT6mzYznjbubZD43DDoUFRufaJOpff9qsinrtJSxa+tRbMdfTdTuHfQ3CkCozviWO+DB9MlKu5rhx9kjqmIgZ3RcxwtLos5Rzm1UHlemEFQYJTgd2jOgPBeaRkkXt3kL\\\\\\\/\\\\\\\/H2GW4JrjvM+Mlcq2h3T9vRUwRPQtHaRFcndJ4PW6Yn4VOZ4mY7s09uXHbWqOu35Nu2HOUfc7qeSgERwGAsGX3z0RlyP1pfSNYooLgpkhrenarbct8sZd5ZJIsZjmq6t+WIUPGZBfskLmNjIp8OZrboBKk17qhZboqKMnxedRhFie\\\\\\\/xR92fCWRibuJrqF7FTDYmXXeqy3HY4\\\\\\\/0VIshdA2312vxyhhackbVXPuBEFBUpbeWRAmC4S3Xb\\\\\\\/U8WfgRsk6WPEz6zilSB68vriJPiClvn4cAD7rLD6SHQul2l5YDjSYoey\\\\\\\/jvRSg7IOgjLtwqo+a3zb5j7z+zoODHkFe2lg\\\\\\\/9aJCi7ahQwT2LNLcBaRiiHlVgtqCaOlhoE5suUIOnBEsQuI46trSKcATyrf9zU+RnegxhndnpHuq2bORr3Fhf1CoHdJhwROjlFGIpG\\\\\\\/eEU\\\\\\\/fDPPBFV7iY0ZCVvpA91OP9ENa\\\\\\\/+r43EluFQuYW2rJRhfnUVd1EuQvNfTpz9r7gi7n+BazrJJkfD08omwjV5HTkBGg1PhSNQfN476cc+f8dk\\\\\\\/rZ0+zgR6JSfOE2uKigxCExZtGXi2imLszWiRv3ECcLO600OA9OHujNZfSstnGoJdEAnCH0LXs\\\\\\\/yByDm05HvJNu9mnh6jjikJX0Pqd6CX4tMXZBNkPLF0DQo2KOvnn4snR\\\\\\\/\\\\\\\/hh6oPseLT1e4VGqbNPFY43vBmkyMkr8cVILhfwxOz3pAxffu8NizVczwnqpGGGQENEglAhKDuUY3BRN0ctwpkfblcdUb7t+\\\\\\\/xTKbJ4I7BHdwrp+daPFbr3xGy9sT3fcXd9BgnAAhO81FVnRNLn2Wv7vIrTDAOZtmPcQvsi0Mvp5mrcMU8g==\\\u0022,\\\u0022iv\\\u0022:\\\u00222Lt8SKnkg1fntnlI1Y1qgQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228481261fd1f54f194d0cb2db4c8fe250353582357e5ed50ee07a08918ea5a8da\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bchotJtt2An7\\\\\\\/JQhPXzhkz\\\\\\\/oYVkQhvEAeGkoggP+UrFxvc2Btvfe71f6NeLIuI2166IN86tgtaEveSNaXwHQdtYLniWcat4q4dQHxZoDHLmIcRpKXBO6XSyQ8LqvqdmMvjfBr1qtLhZ+fPB\\\\\\\/LDZsVKusxbFDLjXV1+cbr6e98Sspa6rwc7rbKq+9y8BEpgnEux0skBn1ULIwibqvztGBOzTLy\\\\\\\/2JpZQ8as\\\\\\\/9zv2g7JD1mxLhZS+2wBLg6DIfXFkgqdwYhMa3Zl4NzCn0s194s4IijsIkpYQGngezaOWPMoPE96tTRAXPpU2lVsjhjObpOW5VRXACn6HMl+kM+0Nojpj4Reu7UJjj+VxEcGfvc50\\\\\\\/WHAuiEJj3ybUYs2NUqU70OoxgPdz6vPtYJZJXnJlb7qvf0WQ0DMvgsnsYXYm+2+XVt5dlatcoGihEvPAj7CXObLSBvpbc5\\\\\\\/bU+uJCkCVrXYFkNTAd0GqpQuMMXb2+OZoAYQBKQ1EjRZVgovk8JaEtwf4L+WO6SwRfN495mhAOHL\\\\\\\/o47IRy4DVmiwOdC0zUJsS6o03YSejG12Uoz0\\\\\\\/AYgA06muO\\\\\\\/Q4B97AoUi8FtHM3pZ5Qs56GSnyL0FWDdr3ILULQ+BPwoYnxqJoxCtXmHyUygfXy71F4FPreFFySvdAn85e9UQiJr6qZ6w\\\\\\\/PtrJ+bLDVzzBl8GNwRFxyUz6fL1R0bgwkzG25O3e8K02m9sTmagLYop\\\\\\\/puJoCVXMAw1GQknISjBptULiAtLnhUe\\\\\\\/baVf4d963t4lbodcbqPtiEx0TOLMO3F5jVyVC8qRZa99WWvywY4ThG0rsQXip0HAeLZBFTweTWVUooIDok0ViB+rIWDArgM+EYnCNlEhw\\\\\\\/s+pzeLks24F+Q3jNSbcsBB8J80ekt4OR4KsGBIvv\\\\\\\/S+ByqUhxosADX0MO\\\\\\\/hs70UCY1oWCsUg4LUTWoN+QP\\\\\\\/PyslOTMEuRj6+1Q7JNa6nc7sFu4tTiXF7p3kxJlJ801CpNEY\\\\\\\/AN\\\\\\\/Hcl5WWhmdCdak23o0K+yhHpniT7K\\\\\\\/tMHMcwBQf3JLSnX5EWa+3ehGptcRTN+mVcuoJhgz7VHW9r3BkXAamqlMvJpLDHMUFABZag8PjqJnLDhTLXBwgBzqZe2S7+jaxL8FQ4gQVGLpiO92qILNagTkDM3ZxiPw1Wnz3hVAU2anvypvvAMBEWw9JVKRy7ZkfuZ6BjgUTBVTqCaTz9nMbqWrh7Ei3J+ZJ69TOkDWhxk0+AUGzJAfnJNOxBQ940vdIscxr\\\\\\\/mIYfk36ImwNQX7Bi8acRujvrjo4iVPpw5TZSt\\\\\\\/cGpNsaDsILfgKhe9JzU+2MHuLp54CEzHGUcnDMgKeteMCCS1ogt+HGwcPlLwouyJSaxP9XflbkZjd6+PEvypBJMpxaGzfwQEXiuNh6K3es5xopOCus9fpOUP0MfBkknxOFRABhOxA8RVCf\\\\\\\/yY9tWDfdbow+\\\\\\\/0GHOp1uux4qZMLfiMzzhjlQNN726hSh4xBRKNtgHdF9D2mAFDo0QvnIiTY1I5x1b3lGY93vKu2XuYym\\\\\\\/BFnIcl6W4w1t79GLgobFACamU2mxTEI36q+3QDJzgehpeeEHt80JDZz5tjzNfPFhgtXUpCTmGheTb8oWq7L3lJX5a3P+LksDMP21N+12jLjr+RM35MZUl3wscVbp\\\\\\\/VQ+tf94V\\\\\\\/7vB0zrxrWmS8dj9aERjZO4Gy\\\\\\\/uPhUH5L6wENU4TsN36\\\\\\\/MAjO1qrG3Z29Jaz5kHS\\\\\\\/REeQU0VHzjSYMMFd1EQh8tosKT9XoJc0RInirA1c8oyiZ10yZY0xEVyThAjwVD\\\\\\\/2536aBOExdoBRnjp0SN+262xUxhL+HRaOf3ew2ydEJKwZJLcTHpgtUTeAogMFFvm9Os2IGysfq3nuQwhUIpJSCl7IaqxX5Fd3SzjrOdLQA6GIdjY\\\\\\\/S3SCCBtRUSDxZB1HvSBrjVgOKeeoHyjXcV03y3BgAvqBpbVyTPrkydPkbIH9eygV\\\\\\\/lN23Wbk2PTsKF\\\\\\\/kDDzZHk\\\\\\\/VU1w7Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Z2eu+s\\\\\\\/mnRUZiLruQEz\\\\\\\/Lw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fc44da15183e3a579f0f7493d2fc3247f5a11e3173d5e5bcdb4d6aaaceb9839b\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459281 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/5/555" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5w5hkOx1aSLANcV6iggfP8XUpaLcGU5TCZpfC1Kich0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/5/555" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YU-BLsHQ-NSGYFzxq3nUBb5j7h5sKDwgYM4lNDQWt9E" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/5/551" title="Simplified Insertion of Transgenes Onto Balancer Chromosomes via Recombinase-Mediated Cassette Exchange" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/5/569" title="Regulated Expression of Chromobox Homolog 5 Revealed in Tumors of ApcMin/+ ROSA11 Gene Trap Mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459281" id="node459281" data-pisa="ggg;2/5/555" data-pisa-master="ggg;2/5/555" data-apath="/ggg/2/5/555.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification of Mutant Versions of the Spt16 Histone Chaperone That Are Defective for Transcription-Coupled Nucleosome Occupancy in <em>Saccharomyces cerevisiae</em></h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sarah J. Hainer</span>, <span class="highwire-citation-author" data-delta="1">Brittany A. Charsar</span>, <span class="highwire-citation-author" data-delta="2">Shayna B. Cohen</span> and <span class="highwire-citation-author" data-delta="3">Joseph A. Martens</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">555-567;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002451 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459281" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sarah J. Hainer</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Biological Sciences, University of Pittsburgh, Pittsburgh, Pennsylvania 15260</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BJ.%2BHainer" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hainer%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASarah%2BJ.%2BHainer" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Brittany A. Charsar</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Biological Sciences, University of Pittsburgh, Pittsburgh, Pennsylvania 15260</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brittany%2BA.%2BCharsar" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Charsar%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrittany%2BA.%2BCharsar" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shayna B. Cohen</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Biological Sciences, University of Pittsburgh, Pittsburgh, Pennsylvania 15260</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shayna%2BB.%2BCohen" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cohen%20SB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShayna%2BB.%2BCohen" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Joseph A. Martens</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Biological Sciences, University of Pittsburgh, Pittsburgh, Pennsylvania 15260</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BA.%2BMartens" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martens%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BA.%2BMartens" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:martens@pitt.edu" class="" data-icon-position="" data-hide-link-title="0">martens@pitt.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/5/555" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459281" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459281/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/5/555.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459281" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459281/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/5/555.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459281" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459281/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/5/555.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459281" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459281/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/5/555.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459281" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459281/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The highly conserved FACT (<span class="underline">Fa</span>cilitates <span class="underline">C</span>hromatin <span class="underline">T</span>ransactions) complex performs essential functions in eukaryotic cells through the reorganization of nucleosomes. During transcription, FACT reorganizes nucleosomes to allow passage of RNA Polymerase II and then assists in restoring these nucleosomes after RNA Polymerase II has passed. We have previously shown, consistent with this function, that <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> facilitates repression of the <em>Saccharomyces cerevisiae <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> gene by maintaining nucleosome occupancy over the promoter of this gene as a consequence of intergenic transcription of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> noncoding DNA. In this study, we report the results of a genetic screen to identify mutations in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a></em> that derepress <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>. Twenty-five <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutant alleles were found to derepress <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> without causing significant reductions in either <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> RNA levels or <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> protein levels. Additional phenotypic assays indicate that these mutants have general transcription defects related to altered chromatin structure. Our analyses of a subset of these <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants reveal defects in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> transcription-coupled nucleosome occupancy over the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> promoter. We provide evidence that these mutants broadly impair transcription-coupled nucleosome occupancy at highly transcribed genes but not at lowly transcribed genes. Finally, we show that one consequence shared by these mutations is the reduced binding of mutant <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> proteins across <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> and other highly transcribed genes. Taken together, our results highlight an important role for <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> in orchestrating transcription-coupled nucleosome assembly at highly transcribed regions of the genome, possibly by facilitating the association of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> during this process.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/fact" class="hw-term hw-article-keyword hw-article-keyword-fact" rel="nofollow">FACT</a></li><li class="kwd"><a href="/keyword/spt16" class="hw-term hw-article-keyword hw-article-keyword-spt16" rel="nofollow">Spt16</a></li><li class="kwd"><a href="/keyword/transcription" class="hw-term hw-article-keyword hw-article-keyword-transcription" rel="nofollow">transcription</a></li><li class="kwd"><a href="/keyword/chromatin" class="hw-term hw-article-keyword hw-article-keyword-chromatin" rel="nofollow">chromatin</a></li><li class="kwd"><a href="/keyword/intergenic-dna" class="hw-term hw-article-keyword hw-article-keyword-intergenic-dna" rel="nofollow">intergenic DNA</a></li></ul><p id="p-4">In eukaryotes, genomic DNA is packaged with proteins to form chromatin, a repeating array of nucleosomes that contain 147 bp of DNA wrapped around an octamer of histone proteins (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Luger <em>et al.</em> 1997</a>). In general, this stable association of DNA and histone proteins poses a significant obstacle to many cellular processes, including transcription, DNA replication, and DNA repair, that rely on proteins being able to interact with DNA [reviewed in <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bai and Morozov (2010)</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Duina (2011)</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Li <em>et al.</em> (2007)</a>, and <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Luger (2006)</a>]. Not surprisingly, eukaryotes express a large group of factors with a range of activities that contribute to the reorganization of chromatin to facilitate these processes. These include chromatin remodelers that use the energy of ATP hydrolysis to reposition or remove histones, posttranslational histone modifiers that covalently add chemical moieties (acetyl, methyl, phosphate, ubiquitin groups) to histone residues that alter their function, and histone chaperones that interact with histones to contribute to the disassembly and reassembly of nucleosomes [reviewed in <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cairns (2009)</a>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Narlikar <em>et al.</em> (2002)</a>, and <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Smith and Shilatifard (2010)</a>].</p><p id="p-5">The highly conserved heterodimer FACT (<span class="underline">Fa</span>cilitates <span class="underline">C</span>hromatin <span class="underline">T</span>ransactions) is a prominent member of the histone chaperone family with reported functions in multiple nuclear processes including DNA replication, DNA repair, transcription initiation, and transcription elongation [reviewed in <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Duina (2011)</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Formosa (2008</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">2011</a>), and <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Winkler and Luger (2011)</a>]. Its role in transcription elongation has been particularly well supported by both genetic and biochemical experiments involving yeast and mammalian systems (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Formosa 2011</a>). These include the sensitivity of yeast FACT mutants to the transcription elongation inhibitor 6-azauracil, the genetic interaction of these mutants with other known elongation factors, the colocalization of FACT with RNA Polymerase II (Pol II) across transcribed regions of eukaryotic genomes, the physical association of FACT with other transcription elongation factors, and the requirement of human FACT to allow RNA Pol II to transcribe a nucleosomal DNA template <em>in vitro</em> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Belotserkovskaya <em>et al.</em> 2003</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Formosa <em>et al.</em> 2001</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">2002</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Krogan <em>et al.</em> 2002</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Orphanides <em>et al.</em> 1998</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Simic <em>et al.</em> 2003</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Squazzo <em>et al.</em> 2002</a>). Although the precise molecular functions of FACT in transcription elongation remain under investigation, several studies have strongly implicated FACT in facilitating the nucleosome dynamics that occur during transcription elongation. These studies suggest that FACT associates with a nucleosome in front of RNA Pol II resulting in the reorganization of histones that eventually lead to the displacement of H2A-H2B dimers and the passage of RNA Pol II (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Belotserkovskaya <em>et al.</em> 2003</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">McCullough <em>et al.</em> 2011</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Orphanides <em>et al.</em> 1998</a>). Once RNA Pol II has passed, FACT is also required to assist in the reassembly of nucleosomes to protect recently transcribed DNA from spurious transcription from cryptic intragenic promoters (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Belotserkovskaya <em>et al.</em> 2003</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Formosa <em>et al.</em> 2002</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jamai <em>et al.</em> 2009</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Orphanides <em>et al.</em> 1999</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Schwabish and Struhl 2004</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Stuwe <em>et al.</em> 2008</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">VanDemark <em>et al.</em> 2008</a>).</p><p id="p-6">Although a role for FACT in facilitating transcription-dependent nucleosome dynamics has been well documented, less is known concerning the precise contribution of the individual FACT subunits. Yeast FACT is composed of two proteins, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004534">Pob3</a>, that are essential for viability and can bind nucleosomes <em>in vitro</em> when aided by a third protein, the HMG box-containing protein Nhp6 (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Formosa <em>et al.</em> 2001</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Wittmeyer and Formosa 1997</a>). <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004534">Pob3</a> consists of three separate domains defined by limited proteolysis: an N-terminal (NT/D) domain that is thought to be involved in dimerization with <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>, a middle (M) domain that contains a double pleckstrin homology motif, and an acidic C-terminal (C) domain (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Liu <em>et al.</em> 2010</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">VanDemark <em>et al.</em> 2006</a>). The pleckstrin homology motif has been implicated in assisting the interactions between FACT and RPA, an essential protein involved in DNA replication and repair (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">VanDemark <em>et al.</em> 2006</a>). <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> has been characterized as having four distinct domains, referred to as the N-terminal (NTD), dimerization (D), middle (M), and C-terminal (C) domains (<a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">VanDemark <em>et al.</em> 2006</a>, <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">2008</a>). Structures of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-NTD, the one domain that is dispensable for viability, from both <em>S. cerevisiae</em> and <em>S. pombe</em> have recently been solved by X-ray crystallography, revealing a motif that is structurally similar to bacterial aminopeptidases (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Stuwe <em>et al.</em> 2008</a>; <a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">VanDemark <em>et al.</em> 2008</a>). Although interactions between the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-NTD and histones H2A, H3, and H4 have been reported, the fact that this domain is expendable for <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> functions <em>in vivo</em> suggests that there are likely to be other regions of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> that functionally and physically interact with histones (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">O’Donnell <em>et al.</em> 2004</a>; <a id="xref-ref-62-4" class="xref-bibr" href="#ref-62">VanDemark <em>et al.</em> 2008</a>). The <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-D domain is thought to interface with the NT/D domain of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004534">Pob3</a> to form the FACT dimer (<a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">VanDemark <em>et al.</em> 2006</a>). Although molecular functions of the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-M domain are not known, mutations altering residues within this domain have resulted in phenotypes indicative of transcription initiation and elongation defects, defects in replication, and defects in cell wall integrity, indicating the functional significance of this domain (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Myers <em>et al.</em> 2011</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">O’Donnell <em>et al.</em> 2009</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Stevens <em>et al.</em> 2011</a>). <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-C is an acidic domain that is essential for viability whose most 3′ end has recently been shown to functionally interact with histone H3 (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Belotserkovskaya <em>et al.</em> 2003</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Evans <em>et al.</em> 1998</a>). Recent <em>in vitro</em> analysis of the human <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-C domain has implicated this domain in the active displacement of nucleosomal DNA during nucleosome reorganization (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Winkler <em>et al.</em> 2011</a>).</p><p id="p-7">We have recently provided evidence that FACT contributes to a new mechanism of gene regulation operating at the <em>S. cerevisiae <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> gene based on its ability to promote transcription-coupled nucleosome dynamics (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hainer <em>et al.</em> 2011</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Martens <em>et al.</em> 2004</a>). In the presence of serine, transcription of intergenic <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> DNA initiates 5′ of the adjacent <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> gene, which encodes an enzyme for serine biosynthesis (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Martens <em>et al.</em> 2004</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">2005</a>). As a consequence of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> transcription across the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> promoter, FACT assists in the assembly and maintenance of nucleosomes over this region that is otherwise depleted of nucleosomes (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Hainer <em>et al.</em> 2011</a>). The presence of these nucleosomes at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> promoter inhibits the binding of transcription factors required to induce <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> transcription. In this report, we present the results of an unbiased genetic screen to identify mutations of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a></em> that derepress <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> transcription. Our analyses of these mutants indicate that the integrity of both the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-D and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-M domains not only are required for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> transcription-dependent nucleosome assembly and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> repression but also are more broadly required for transcription-coupled nucleosome occupancy at highly transcribed genes. We provide evidence suggesting a possible role for the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-D and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-M domains in promoting the association of FACT to genes being actively transcribed.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains and media</h3><p id="p-8">All <em>S. cerevisiae</em> strains used in this study (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002451/-/DC1/002451SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002451/-/DC1/TableS1.pdf">Table S1</a>) are isogenic to a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004071">GAL2</a><sup>+</sup></em> derivative of S288c (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Winston <em>et al.</em> 1995</a>). All strains were constructed by transformation or by genetic crosses (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ausubel <em>et al.</em> 1991</a>). YJ920 and YADP50 have been previously described (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Hainer <em>et al.</em> 2011</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Myers <em>et al.</em> 2011</a>). Strains YJ1089-YJ1092 were derived from YJ920. The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em>Δ::<em>KanMX</em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">lys2</a>-128</em>δ alleles have been previously described (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Clark-Adams <em>et al.</em> 1988</a>; <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Myers <em>et al.</em> 2011</a>). The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005212">lyp1</a></em>Δ::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>pr<em>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> allele was generated by replacing the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> open reading frame (ORF) in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005212">lyp1</a></em>Δ::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>pr<em>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hainer and Martens 2011</a>) with a PCR product containing the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> open reading frame that was amplified from pRS403 (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Sikorski and Hieter 1989</a>). pAO01 and p<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> are centromeric plasmids marked with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em>, respectively, that contain wild-type <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a></em> (kindly provided by A. Duina) (<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Myers <em>et al.</em> 2011</a>). Derivatives of pAO01 containing <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-G132D</em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-T828I/P859S</em> alleles were generated by standard cloning methods and verified by sequencing. All other <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants characterized in this study are expressed from plasmids derived from pAO01. Yeast extract-peptone-dextrose (YPD), synthetic complete (SC), omission (SC−), 5-fluoroorotic acid (5-FOA), and galactose media have been previously described (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Rose 1991</a>). YPD was supplemented with 5 μg/mL cyclohexamide (CHX) or 200 mM hydroxyurea (HU) as indicated. 3-amino-1,2,4-triazole (3-AT; Sigma) was added to SC medium lacking leucine and histidine at the indicated concentrations.</p></div><div id="sec-3" class="subsection"><h3>Screen for <em>spt16</em> mutants that derepress <em>SER3</em></h3><p id="p-9">Using a previously described strategy (<a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Myers <em>et al.</em> 2011</a>), two regions of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a></em>, from +764 to +2044 (region B) and from +1430 to +3521 (region C), were amplified from pAO01 plasmid (gift from A. Duina) using GoTaq polymerase (Invitrogen) and standard PCR conditions. Amplified DNA was cotransformed into YJ1089 with pAO01 plasmid that had been digested with either <em>Eag</em>1 and <em>Sna</em>B1 (region B) or <em>Sna</em>B1 and <em>Xba</em>I (region C). Transformants containing gap-repaired plasmids were selected on SC medium lacking leucine and then replica-plated onto medium containing 5-FOA to select for cells that lost the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em>-marked plasmid carrying a wild-type copy of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a></em> (p<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em>). The resulting colonies were then replica-plated to SC medium lacking histidine and leucine that was supplemented with 5 mM 3-AT. Candidate plasmids were recovered from strains resistant to 5 mM 3-AT, retransformed into YJ1089, and retested for their ability to confer 3-AT resistance. The region of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a></em> that was subjected to PCR mutagenesis was first subcloned into a new copy of pAO01 before retransformation. For each plasmid that retested for 3-AT resistance, both strands of the entire <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a></em> gene were sequenced and compared with the wild-type gene.</p></div><div id="sec-4" class="subsection"><h3>Northern analysis</h3><p id="p-10">Cells were grown to approximately 2 × 10<sup>7</sup> cells/ml in YPD at 30°. Total RNA isolation and Northern analysis was performed as previously described (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Collart and Oliviero 2001</a>). Radiolabeled DNA probes to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> (−454 to −123 relative to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> ATG), <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> (+111 to +1342), and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000013">SCR1</a></em> (−163 to +284) were generated by random-primed labeling of PCR fragments amplified from genomic DNA. RNA levels were quantified using a PhosphorImager (FLA-5000) and ImageJ software.</p></div><div id="sec-5" class="subsection"><h3>Western analysis</h3><p id="p-11">Whole cell extracts (WCE) were prepared from cells grown in YPD at 30° to approximately 3 × 10<sup>7</sup> cells/ml using trichloroacetic acid as previously described (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cox <em>et al.</em> 1997</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Zheng <em>et al.</em> 2010</a>). Equal amounts of WCE were separated by 10% acrylamide SDS-PAGE, transferred to Protean nitrocellulose (Whatman), and assayed by immunoblotting. The antibodies used to detect <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004534">Pob3</a>, and G6PDH were as follows: anti-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> (1:500; gift from T. Formosa), anti-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004534">Pob3</a> (1:2000; gift from T. Formosa), anti-G6PDH (1:50,000; Sigma). After incubation with HRP-conjugated IgG secondary antibody (1:5000; GE Healthcare), the immunoreactive proteins were visualized by enhanced chemiluminescence detection (PerkinElmer) using a Kodak image station 440<em>CF</em>. <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004534">Pob3</a> protein levels were calculated by measuring their signal intensities in these Western blots using Kodak ID 3.6 software and normalizing these values to those obtained for the G6PDH control.</p></div><div id="sec-6" class="subsection"><h3>Dilution growth assays</h3><p id="p-12">Cells were grown at 30° overnight to saturation then washed twice with water. Starting at 1 × 10<sup>8</sup> cells/ml, cultures were serially diluted 10-fold. Three microliters of each dilution was spotted onto indicated media and incubated at 30° for the indicated number of days.</p></div><div id="sec-7" class="subsection"><h3>Nucleosome scanning assay</h3><p id="p-13">Cells were grown at 30° to approximately 2 × 10<sup>7</sup> cells/ml in YPD and subjected to a nucleosome-scanning assay, as previously described (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Hainer <em>et al.</em> 2011</a>). For each of the 38 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> primer pairs, the amount of template protected from digestion by micrococcal nuclease (MNase) was calculated as a ratio between MNase-digested and MNase-undigested samples and then normalized to the amount of MNase-protected control template (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000224">GAL1</a></em> NB) that is located within a well-positioned nucleosome in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000224">GAL1</a></em> promoter (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brickner <em>et al.</em> 2007</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Floer <em>et al.</em> 2010</a>).</p></div><div id="sec-8" class="subsection"><h3>Chromatin immunoprecipitation (ChIP)</h3><p id="p-14">Cells were grown in YPD at 30° to approximately 2 × 10<sup>7</sup> cells/ml. Chromatin was prepared as previously described (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Shirra <em>et al.</em> 2005</a>). Histone H3, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>, or <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001283">Rpb3</a> were immunoprecipitated by incubating sonicated chromatin overnight at 4° with 1 μl anti-histone H3 antisera [previously described in <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Tomson <em>et al.</em> (2011)</a>], 1 μl anti-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> antisera (kindly provided by T. Formosa), or 2.5 μl anti-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001283">Rpb3</a> antisera (W0012; Neoclone) followed by the addition of IgG-Sepharose beads (GE Healthcare) for 2 hr at 4°. Dilutions of input DNA and immunoprecipitated DNA were analyzed by qPCR reactions. Primer sets that amplify the following regions were used for qPCR: <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>-41 (−921 to −828, relative to +1 ATG of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>); <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>-25 (−338 to −289, relative to +1 ATG of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>); <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>-22 (−300 to −200, relative to +1 ATG of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>); <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>-7 (+195 to +295); <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000036">PYK1</a></em> (5′: +62 to +164, 3′: +1173 to +1279); <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em> (5′: +691 to +794, 3′: +1689 to +1791); <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005446">ADH1</a></em> (+845 to +943); <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003809">CYC1</a></em> (+122 to +217); <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001857">TUB2</a></em> (5′: +105 to +202, 3′: +1083 to +1189); and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000224">GAL1</a></em> (5′: +79 to +175, 3′: +1366 to + 1487). Histone H3, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>, and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001283">Rpb3</a> ChIP signals for each gene were normalized to a No ORF control template, which is located within a region of chromosome V that lacks open reading frames (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Komarnitsky <em>et al.</em> 2000</a>).</p></div><div id="sec-9" class="subsection"><h3>Quantitative PCR (qPCR)</h3><p id="p-15">All qPCR data for the nucleosome scanning and ChIP assays were obtained by using an ABI StepOne Plus Real-time system using SYBR green reagents (Fermentas) and the indicated primers (<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Hainer <em>et al.</em> 2011</a>). Calculations were performed using Pfaffl methodology (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Pfaffl 2001</a>).</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Identification of <em>spt16</em> mutants that derepress <em>SER3</em></h3><p id="p-16">We recently described a new mechanism of gene regulation in <em>Saccharomyces cerevisiae</em> whereby transcription of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> ncDNA assembles nucleosomes over the promoter of the adjacent <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> gene to maintain <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> repression (<a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">Hainer <em>et al.</em> 2011</a>). Furthermore, we provided evidence that the histone chaperones, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003348">Spt6</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>, are required to maintain this nucleosome occupancy, and repress <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>, likely through their ability to disassemble and reassemble nucleosomes during active transcription (<a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Belotserkovskaya <em>et al.</em> 2003</a>; <a id="xref-ref-23-7" class="xref-bibr" href="#ref-23">Hainer <em>et al.</em> 2011</a>). To investigate the role of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> in this mechanism, we performed an unbiased genetic screen to identify novel mutations in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a></em> that derepress <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> during <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> transcription. A PCR-based strategy that has been previously described [see <em>Materials and Methods</em>; <a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Myers <em>et al.</em> (2011)</a>] was used to target mutagenesis of the 3′ half of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a></em> that excludes most of the N-terminal domain (NTD), which is dispensable for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> repression (S. J. Hainer, unpublished data). These PCR fragments were cotransformed with a gapped <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a></em>-marked plasmid that contained homology to the PCR fragments into an <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>Δ <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a>Δ</em> strain containing an integrated <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>pr<em>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> reporter (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>) and expressing a wild-type copy of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a></em> from a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em>-marked plasmid (YJ1089). Following gap-repair and loss of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em>-marked plasmid expressing <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a></em>, we screened for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants that derepress the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>pr<em>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> reporter by their ability to confer growth in the presence of 3-AT, a competitive inhibitor of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> gene product (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/5/555/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Identification of spt16 mutants that derepress an ectopically expressed SER3pr-HIS3 reporter gene. (A) Diagram of SER3pr-HIS3 reporter. The LYP1 ORF was replaced by SER3 intergenic sequence from &#x2212;713 to &#x2212;1, including SRG1 and its promoter, fused to the HIS3 ORF. Block arrows beneath the diagram indicate the expected SRG1 and SER3-HIS3 transcripts in wild-type and mutant strains grown in serine-rich media (YPD). The expected growth of these strains on SC-His-Leu plates containing 3-AT is indicated on the right. (B) Growth assays indicating that newly isolated spt16 mutants derepress SER3pr-HIS3 reporter. spt16&#x394; cells (YJ1089) containing LEU2-marked plasmids expressing either wild-type or mutant Spt16 protein, as indicated, were grown to saturation in YPD, diluted to 108 and then spotted in a 10-fold serial dilution series on SC-His-Leu (control) and SC-His-Leu + 10 mM 3-AT plates. Plates were incubated at 30&#xB0; for three days. Results were obtained for two independent growth assays in which each plate contained control strains and five to six mutants. Shown are representative dilutions for the control strains and each spt16 mutant strain. Each Spt16 mutant protein is named to describe the location and nature of the amino acid substitution. The locations of the amino acid substitutions in each of these mutants are also indicated (marked by stars) in diagrams of Spt16; gray ovals indicate the N-terminal (NTD), dimerization (D), middle (M) domains, and the C-terminal acidic tail region (C). Note that the five mutants marked by an asterisk each have an additional silent mutation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-202268667" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Identification of &lt;em&gt;spt16&lt;/em&gt; mutants that derepress an ectopically expressed &lt;em&gt;SER3&lt;/em&gt;pr&lt;em&gt;-HIS3&lt;/em&gt; reporter gene. (A) Diagram of &lt;em&gt;SER3&lt;/em&gt;pr&lt;em&gt;-HIS3&lt;/em&gt; reporter. The &lt;em&gt;LYP1&lt;/em&gt; ORF was replaced by &lt;em&gt;SER3&lt;/em&gt; intergenic sequence from &#x2212;713 to &#x2212;1, including &lt;em&gt;SRG1&lt;/em&gt; and its promoter, fused to the &lt;em&gt;HIS3&lt;/em&gt; ORF. Block arrows beneath the diagram indicate the expected &lt;em&gt;SRG1&lt;/em&gt; and &lt;em&gt;SER3-HIS3&lt;/em&gt; transcripts in wild-type and mutant strains grown in serine-rich media (YPD). The expected growth of these strains on SC-His-Leu plates containing 3-AT is indicated on the right. (B) Growth assays indicating that newly isolated &lt;em&gt;spt16&lt;/em&gt; mutants derepress &lt;em&gt;SER3&lt;/em&gt;pr&lt;em&gt;-HIS3&lt;/em&gt; reporter. &lt;em&gt;spt16&lt;/em&gt;&#x394; cells (YJ1089) containing &lt;em&gt;LEU2&lt;/em&gt;-marked plasmids expressing either wild-type or mutant Spt16 protein, as indicated, were grown to saturation in YPD, diluted to 10&lt;sup&gt;8&lt;/sup&gt; and then spotted in a 10-fold serial dilution series on SC-His-Leu (control) and SC-His-Leu + 10 mM 3-AT plates. Plates were incubated at 30&#xB0; for three days. Results were obtained for two independent growth assays in which each plate contained control strains and five to six mutants. Shown are representative dilutions for the control strains and each &lt;em&gt;spt16&lt;/em&gt; mutant strain. Each Spt16 mutant protein is named to describe the location and nature of the amino acid substitution. The locations of the amino acid substitutions in each of these mutants are also indicated (marked by stars) in diagrams of Spt16; gray ovals indicate the N-terminal (NTD), dimerization (D), middle (M) domains, and the C-terminal acidic tail region (C). Note that the five mutants marked by an asterisk each have an additional silent mutation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/5/555/F1.medium.gif" width="440" height="407"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/2/5/555/F1.medium.gif" width="440" height="407"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/5/555/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/5/555/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461508" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-17" class="first-child">Identification of <em>spt16</em> mutants that derepress an ectopically expressed <em>SER3</em>pr<em>-HIS3</em> reporter gene. (A) Diagram of <em>SER3</em>pr<em>-HIS3</em> reporter. The <em>LYP1</em> ORF was replaced by <em>SER3</em> intergenic sequence from −713 to −1, including <em>SRG1</em> and its promoter, fused to the <em>HIS3</em> ORF. Block arrows beneath the diagram indicate the expected <em>SRG1</em> and <em>SER3-HIS3</em> transcripts in wild-type and mutant strains grown in serine-rich media (YPD). The expected growth of these strains on SC-His-Leu plates containing 3-AT is indicated on the right. (B) Growth assays indicating that newly isolated <em>spt16</em> mutants derepress <em>SER3</em>pr<em>-HIS3</em> reporter. <em>spt16</em>Δ cells (YJ1089) containing <em>LEU2</em>-marked plasmids expressing either wild-type or mutant Spt16 protein, as indicated, were grown to saturation in YPD, diluted to 10<sup>8</sup> and then spotted in a 10-fold serial dilution series on SC-His-Leu (control) and SC-His-Leu + 10 mM 3-AT plates. Plates were incubated at 30° for three days. Results were obtained for two independent growth assays in which each plate contained control strains and five to six mutants. Shown are representative dilutions for the control strains and each <em>spt16</em> mutant strain. Each Spt16 mutant protein is named to describe the location and nature of the amino acid substitution. The locations of the amino acid substitutions in each of these mutants are also indicated (marked by stars) in diagrams of Spt16; gray ovals indicate the N-terminal (NTD), dimerization (D), middle (M) domains, and the C-terminal acidic tail region (C). Note that the five mutants marked by an asterisk each have an additional silent mutation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">With this screen, we initially identified 522 mutants that permit growth on medium containing 5 mM 3-AT. Forty <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a></em>-containing plasmids were then recovered from strains that conferred resistance up to 40 mM 3-AT to enrich for mutations that most strongly derepress <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>. The remaining 482 plasmids have not yet been examined. After retesting for their ability to derepress the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>pr<em>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> reporter, the entire <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a></em> gene contained on each of these plasmids was sequenced. Sequencing of the 38 plasmids that successfully retested identified 25 unique <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants harboring nucleotide changes that result in either single (12), double (11), or triple (2) amino acid substitutions. For the 12 single amino acid substitution mutants, the location of the altered amino acids varies: 3 are located at the very 3′ end of the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-NTD, 3 are found in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-D, and the remaining 6 are found in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-M, including 3 residues that are within 13 amino acids of each other (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). Interestingly, 9 of the 13 double or triple mutants contain 1 of the single amino acid substitutions, indicating that the effect on the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> reporter from these mutation combinations is likely through the isolated single substitution. Interestingly, only one of these mutations, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-E857K</em>, has been previously reported (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">O’Donnell <em>et al.</em> 2009</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Stevens <em>et al.</em> 2011</a>). In these studies, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-E857K</em> was isolated as a dominant suppressor of a transcription defect caused by the insertion of a δ element 5′ of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">LYS2</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000535">HIS4</a></em> genes (Spt<sup>−</sup> phenotype) and was found to genetically interact with mutations in other transcription elongation factors.</p></div><div id="sec-12" class="subsection"><h3>Phenotypic analysis of the <em>spt16</em> mutants</h3><p id="p-19">To further characterize these mutants, we tested the strains for temperature sensitivities and growth defects on YPD medium supplemented with cyclohexamide (CHX), hydroxyurea (HU), mycophenolic acid (MPA), and caffeine. Surprisingly, we found that not one of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants that we isolated confers a growth defect at elevated temperatures (39°) or in the presence of HU (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>), phenotypes that have been previously described for other <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> alleles, including <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-G132D</em> (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>, row 2) and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-T828I/P859S</em> (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>, row 3) (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Formosa <em>et al.</em> 2001</a>). Interestingly, one mutant, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-S715G/D718G</em>, confers cold sensitivity at 15°, and a number of the mutants cause varying sensitivities to CHX (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2A</a>). No detectable growth defects were observed when strains expressing any one of the isolated <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants were exposed to MPA or caffeine (S. B. Cohen, unpublished data).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/5/555/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phenotypic characterization of newly isolated spt16 mutants. (A) Temperature, cyclohexamide (CHX), and hydroxyurea (HU) sensitivity of spt16 mutants. spt16&#x394; cells (YJ1091) expressing either wild-type or the indicated mutant alleles of SPT16 from a LEU2-marked plasmid were grown to saturation in YPD at 30&#xB0;, diluted to 108, spotted in 10-fold serial dilutions, and incubated for the indicated number of days on solid media. Cells spotted on YPD were incubated at 30&#xB0;, 39&#xB0;, or 15&#xB0;, and cells spotted on YPD+CHX or YPD+HU were grown at 30&#xB0;. Results are a representative of two biological replicates. (B) Dominance test. Cells expressing wild-type SPT16 from its genomic location and the SER3pr-HIS3 reporter (YJ1090) were transformed with plasmids containing either wild-type or the indicated mutant SPT16 alleles. Transformants were grown to saturation in YPD at 30&#xB0; and spotted on SC-His-Leu (control) and SC-His-Leu + 10 mM solid medium, which were then incubated at 30&#xB0; for three days. Results are representative of two biological replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-202268667" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phenotypic characterization of newly isolated &lt;em&gt;spt16&lt;/em&gt; mutants. (A) Temperature, cyclohexamide (CHX), and hydroxyurea (HU) sensitivity of &lt;em&gt;spt16&lt;/em&gt; mutants. &lt;em&gt;spt16&lt;/em&gt;&#x394; cells (YJ1091) expressing either wild-type or the indicated mutant alleles of &lt;em&gt;SPT16&lt;/em&gt; from a &lt;em&gt;LEU2&lt;/em&gt;-marked plasmid were grown to saturation in YPD at 30&#xB0;, diluted to 10&lt;sup&gt;8&lt;/sup&gt;, spotted in 10-fold serial dilutions, and incubated for the indicated number of days on solid media. Cells spotted on YPD were incubated at 30&#xB0;, 39&#xB0;, or 15&#xB0;, and cells spotted on YPD+CHX or YPD+HU were grown at 30&#xB0;. Results are a representative of two biological replicates. (B) Dominance test. Cells expressing wild-type &lt;em&gt;SPT16&lt;/em&gt; from its genomic location and the &lt;em&gt;SER3&lt;/em&gt;pr-&lt;em&gt;HIS3&lt;/em&gt; reporter (YJ1090) were transformed with plasmids containing either wild-type or the indicated mutant &lt;em&gt;SPT16&lt;/em&gt; alleles. Transformants were grown to saturation in YPD at 30&#xB0; and spotted on SC-His-Leu (control) and SC-His-Leu + 10 mM solid medium, which were then incubated at 30&#xB0; for three days. Results are representative of two biological replicates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/5/555/F2.medium.gif" width="440" height="219"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/2/5/555/F2.medium.gif" width="440" height="219"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/5/555/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/5/555/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461487" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">Phenotypic characterization of newly isolated <em>spt16</em> mutants. (A) Temperature, cyclohexamide (CHX), and hydroxyurea (HU) sensitivity of <em>spt16</em> mutants. <em>spt16</em>Δ cells (YJ1091) expressing either wild-type or the indicated mutant alleles of <em>SPT16</em> from a <em>LEU2</em>-marked plasmid were grown to saturation in YPD at 30°, diluted to 10<sup>8</sup>, spotted in 10-fold serial dilutions, and incubated for the indicated number of days on solid media. Cells spotted on YPD were incubated at 30°, 39°, or 15°, and cells spotted on YPD+CHX or YPD+HU were grown at 30°. Results are a representative of two biological replicates. (B) Dominance test. Cells expressing wild-type <em>SPT16</em> from its genomic location and the <em>SER3</em>pr-<em>HIS3</em> reporter (YJ1090) were transformed with plasmids containing either wild-type or the indicated mutant <em>SPT16</em> alleles. Transformants were grown to saturation in YPD at 30° and spotted on SC-His-Leu (control) and SC-His-Leu + 10 mM solid medium, which were then incubated at 30° for three days. Results are representative of two biological replicates.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">We also tested whether the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants that we isolated are dominant for repression of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>pr<em>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> reporter. YJ1090 cells containing wild-type <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a></em> at its genomic location, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>pr<em>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em>, and a plasmid expressing either wild-type or mutant version of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> were spotted onto media containing 3-AT to test for expression of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>pr<em>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> reporter. One mutant allele, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-E857K</em>, is dominant for derepression of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> reporter gene, suggesting that it may be a gain-of-function mutation (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>). Interestingly, our analysis of the more complex mutants identified amino acid substitutions at <em>I626T</em> and one or both of <em>T651A</em> and <em>H471Y</em> as intragenic suppressors of the dominant effect of this <em>E857K</em> substitution. Additionally, our analysis revealed that whereas the mutant <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-S765P</em> allele alone does not confer dominance, it is synthetically dominant for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> derepression with either a <em>K800E</em> or <em>L865P</em> substitution mutation.</p></div><div id="sec-13" class="subsection"><h3><em>spt16</em> mutants derepress endogenous <em>SER3</em></h3><p id="p-22">We next determined the effect of these <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants on endogenous <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> RNA levels. We transformed plasmids containing either wild-type <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a></em>, a previously characterized <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-G132D</em> mutant (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Malone <em>et al.</em> 1991</a>), or one of our newly isolated <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants into YJ1091 and YJ1092 strains and performed Northern assays on these strains (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3, A and B</a>). For these and subsequent experiments, we limited our analysis to the 12 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants having single amino acid substitutions. All of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants tested derepress <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> with effects ranging from very strong (30-fold increase for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-E857K</em>) to milder (2-fold increase for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-Y297H</em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-N580D</em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-E671G</em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-S765P</em>) that are similar to what we had previously observed for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-G132D</em> (<a id="xref-ref-23-8" class="xref-bibr" href="#ref-23">Hainer <em>et al.</em> 2011</a>). Although we did observe some variability in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> RNA levels between experiments, average results from four independent experiments indicate that these <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants do not significantly alter <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> RNA levels. Consistent with these Northern data, we found equivalent levels of RNA Pol II localized across the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> transcription unit in strains expressing either wild-type or mutant versions of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6B</a>). Moreover, Western analyses showed that these newly isolated <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants do not alter the levels of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> or its interacting partner, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004534">Pob3</a> (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, C and D</a>). Taken together, these data identify amino acids in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> that are critical for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> repression.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/5/555/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Single amino acid substitutions in Spt16 strongly derepress endogenous SER3. (A) Northern blot analysis examining the effect of spt16 mutants on SER3, SRG1, and SCR1 (loading control). Total RNA was isolated from spt16&#x394; cells (YJ1091) carrying plasmid-borne wild-type or mutant SPT16 alleles that were grown to a density of &#x223C;2 &#xD7; 107 cells/ml in YPD at 30&#xB0;. Note that there are two independent strains with the spt16-L669S mutation. (B) Quantitation of Northern data. SRG1 (gray bars) and SER3 (white bars) RNA levels for the spt16 mutants are normalized to the level of the SCR1 loading control and are relative to strains expressing wild-type SPT16 (arbitrarily set to 1). Each bar indicates the mean RNA level &#xB1; SEM from four independent experiments using two transformations each of YJ1091 and YJ1092. (C) Western analysis examining the effect of spt16 mutant alleles on mutant Spt16 and Pob3 protein levels. Whole cell extracts were prepared from the same set of strains described in panel A grown to &#x223C;3 &#xD7; 107 cells/ml in YPD at 30&#xB0; and subjected to Western analysis using anti-Spt6 and anti-Pob3 antibodies (kindly provided by T. Formosa). Blots were reprobed with anti-G6PDH antibody as a loading control. Note that there are two independent spt16-L669S mutants. (D) Quantitation of Western data. Spt16 (gray bars) and Pob3 (white bars) protein levels are normalized to the G6PDH loading control and are relative to strains expressing wild-type SPT16 (arbitrarily set to 1). Each bar indicates the mean protein level &#xB1; SEM from three independent experiments using the same set of strains as in panel B." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-202268667" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Single amino acid substitutions in Spt16 strongly derepress endogenous &lt;em&gt;SER3&lt;/em&gt;. (A) Northern blot analysis examining the effect of &lt;em&gt;spt16&lt;/em&gt; mutants on &lt;em&gt;SER3&lt;/em&gt;, &lt;em&gt;SRG1&lt;/em&gt;, and &lt;em&gt;SCR1&lt;/em&gt; (loading control). Total RNA was isolated from &lt;em&gt;spt16&lt;/em&gt;&#x394; cells (YJ1091) carrying plasmid-borne wild-type or mutant &lt;em&gt;SPT16&lt;/em&gt; alleles that were grown to a density of &#x223C;2 &#xD7; 10&lt;sup&gt;7&lt;/sup&gt; cells/ml in YPD at 30&#xB0;. Note that there are two independent strains with the &lt;em&gt;spt16-L669S&lt;/em&gt; mutation. (B) Quantitation of Northern data. &lt;em&gt;SRG1&lt;/em&gt; (gray bars) and &lt;em&gt;SER3&lt;/em&gt; (white bars) RNA levels for the &lt;em&gt;spt16&lt;/em&gt; mutants are normalized to the level of the &lt;em&gt;SCR1&lt;/em&gt; loading control and are relative to strains expressing wild-type &lt;em&gt;SPT16&lt;/em&gt; (arbitrarily set to 1). Each bar indicates the mean RNA level &#xB1; SEM from four independent experiments using two transformations each of YJ1091 and YJ1092. (C) Western analysis examining the effect of &lt;em&gt;spt16&lt;/em&gt; mutant alleles on mutant Spt16 and Pob3 protein levels. Whole cell extracts were prepared from the same set of strains described in panel A grown to &#x223C;3 &#xD7; 10&lt;sup&gt;7&lt;/sup&gt; cells/ml in YPD at 30&#xB0; and subjected to Western analysis using anti-Spt6 and anti-Pob3 antibodies (kindly provided by T. Formosa). Blots were reprobed with anti-G6PDH antibody as a loading control. Note that there are two independent &lt;em&gt;spt16-L669S&lt;/em&gt; mutants. (D) Quantitation of Western data. Spt16 (gray bars) and Pob3 (white bars) protein levels are normalized to the G6PDH loading control and are relative to strains expressing wild-type &lt;em&gt;SPT16&lt;/em&gt; (arbitrarily set to 1). Each bar indicates the mean protein level &#xB1; SEM from three independent experiments using the same set of strains as in panel B.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/5/555/F3.medium.gif" width="440" height="347"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/2/5/555/F3.medium.gif" width="440" height="347"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/5/555/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/5/555/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461470" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-23" class="first-child">Single amino acid substitutions in Spt16 strongly derepress endogenous <em>SER3</em>. (A) Northern blot analysis examining the effect of <em>spt16</em> mutants on <em>SER3</em>, <em>SRG1</em>, and <em>SCR1</em> (loading control). Total RNA was isolated from <em>spt16</em>Δ cells (YJ1091) carrying plasmid-borne wild-type or mutant <em>SPT16</em> alleles that were grown to a density of ∼2 × 10<sup>7</sup> cells/ml in YPD at 30°. Note that there are two independent strains with the <em>spt16-L669S</em> mutation. (B) Quantitation of Northern data. <em>SRG1</em> (gray bars) and <em>SER3</em> (white bars) RNA levels for the <em>spt16</em> mutants are normalized to the level of the <em>SCR1</em> loading control and are relative to strains expressing wild-type <em>SPT16</em> (arbitrarily set to 1). Each bar indicates the mean RNA level ± SEM from four independent experiments using two transformations each of YJ1091 and YJ1092. (C) Western analysis examining the effect of <em>spt16</em> mutant alleles on mutant Spt16 and Pob3 protein levels. Whole cell extracts were prepared from the same set of strains described in panel A grown to ∼3 × 10<sup>7</sup> cells/ml in YPD at 30° and subjected to Western analysis using anti-Spt6 and anti-Pob3 antibodies (kindly provided by T. Formosa). Blots were reprobed with anti-G6PDH antibody as a loading control. Note that there are two independent <em>spt16-L669S</em> mutants. (D) Quantitation of Western data. Spt16 (gray bars) and Pob3 (white bars) protein levels are normalized to the G6PDH loading control and are relative to strains expressing wild-type <em>SPT16</em> (arbitrarily set to 1). Each bar indicates the mean protein level ± SEM from three independent experiments using the same set of strains as in panel B.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Effect of <em>spt16</em> mutants on nucleosome occupancy over the <em>SER3</em> promoter</h3><p id="p-24">To examine the effect of a subset of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants on nucleosome occupancy at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>, we performed nucleosome-scanning assays on six of the single amino acid substitutions that most strongly derepress <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). As previously described (<a id="xref-ref-23-9" class="xref-bibr" href="#ref-23">Hainer <em>et al.</em> 2011</a>), MNase protection across <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> was normalized to the protection of a well-studied, nucleosome-bound region of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000224">GAL1</a></em> promoter whose digestion by MNase is unaffected by these <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants (S. J. Hainer, unpublished data; see <em>Materials and Methods</em> for details). Compared with strains containing wild-type control plasmids, protection from MNase digestion was reduced across the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em>-transcribed region in all the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants examined to degrees approximately equal to or exceeding that of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-G132D</em> (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>), which we had previously shown to decrease nucleosome occupancy across the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> locus (<a id="xref-ref-23-10" class="xref-bibr" href="#ref-23">Hainer <em>et al.</em> 2011</a>). MNase protection across the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> promoter region was most dramatically reduced in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-E857K</em> mutant (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4H</a>), which is consistent with the strong derepression of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> that is observed in this mutant. The other six mutants that displayed more modest defects in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> repression also had more modest reductions in the MNase protection across the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> promoter. However, we did observe subtle differences in the MNase protection patterns between these mutants. Two of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants resulted in greater sensitivity to MNase toward the 5′ of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> relative to the 3′ of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-K579E</em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-L669S</em>), compared with the other mutants that had increases in MNase sensitivity that were more evenly distributed across the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> transcription unit (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>, compare −400 and −200 regions in panels D and F with panels E and G).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/5/555/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of spt16 mutants on chromatin structure at SER3. (A) Diagram of the SER3 locus. The gray ovals mark the position of nucleosomes when wild-type cells are grown in SER3-repressing conditions (YPD). The block arrow indicates SRG1 transcription. (B&#x2013;H) Nucleosome scanning assays were performed on spt16&#x394; cells (YJ1091 and YJ1092) carrying plasmids expressing either wild-type SPT16 or mutant spt16 alleles as indicated. Mononucleosome-sized DNA fragments were generated by micrococcal nuclease (MNase) digestion of formaldeyde-treated chromatin that was isolated from cells grown to &#x223C;2 &#xD7; 107 cells/mL in YPD media at 30&#xB0;. MNase protection across the SER3 locus relative to a positioned nucleosome within the GAL1 promoter was determined by qPCR. For each PCR amplicon, the mean MNase protection &#xB1; SEM from three independent experiments is plotted at its midpoint. Shown below each graph is a diagram of the SER3 locus indicating the positions of nucleosomes (gray ovals) extrapolated from the MNase protection data for each spt16 mutant. The light-gray ovals are indicative of less dramatic reductions in MNase protections compared with the wild-type control shown in panel A. (I) Histone H3 ChIP was performed on chromatin isolated from the same strains used in panels B&#x2013;H. The amount of immunoprecipitated DNA was determined by qPCR as a fraction of the input that was then normalized to a control region in chromosome V and made relative to strains expressing wild-type SPT16 (arbitrarily set to 1). Each bar represents the mean &#xB1; SEM of three independent experiments. Below the graph is a schematic of SER3 with black bars corresponding to the regions amplified by qPCR." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-202268667" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Effect of &lt;em&gt;spt16&lt;/em&gt; mutants on chromatin structure at &lt;em&gt;SER3&lt;/em&gt;. (A) Diagram of the &lt;em&gt;SER3&lt;/em&gt; locus. The gray ovals mark the position of nucleosomes when wild-type cells are grown in &lt;em&gt;SER3&lt;/em&gt;-repressing conditions (YPD). The block arrow indicates &lt;em&gt;SRG1&lt;/em&gt; transcription. (B&#x2013;H) Nucleosome scanning assays were performed on &lt;em&gt;spt16&lt;/em&gt;&#x394; cells (YJ1091 and YJ1092) carrying plasmids expressing either wild-type &lt;em&gt;SPT16&lt;/em&gt; or mutant &lt;em&gt;spt16&lt;/em&gt; alleles as indicated. Mononucleosome-sized DNA fragments were generated by micrococcal nuclease (MNase) digestion of formaldeyde-treated chromatin that was isolated from cells grown to &#x223C;2 &#xD7; 10&lt;sup&gt;7&lt;/sup&gt; cells/mL in YPD media at 30&#xB0;. MNase protection across the &lt;em&gt;SER3&lt;/em&gt; locus relative to a positioned nucleosome within the &lt;em&gt;GAL1&lt;/em&gt; promoter was determined by qPCR. For each PCR amplicon, the mean MNase protection &#xB1; SEM from three independent experiments is plotted at its midpoint. Shown below each graph is a diagram of the &lt;em&gt;SER3&lt;/em&gt; locus indicating the positions of nucleosomes (gray ovals) extrapolated from the MNase protection data for each &lt;em&gt;spt16&lt;/em&gt; mutant. The light-gray ovals are indicative of less dramatic reductions in MNase protections compared with the wild-type control shown in panel A. (I) Histone H3 ChIP was performed on chromatin isolated from the same strains used in panels B&#x2013;H. The amount of immunoprecipitated DNA was determined by qPCR as a fraction of the input that was then normalized to a control region in chromosome V and made relative to strains expressing wild-type &lt;em&gt;SPT16&lt;/em&gt; (arbitrarily set to 1). Each bar represents the mean &#xB1; SEM of three independent experiments. Below the graph is a schematic of &lt;em&gt;SER3&lt;/em&gt; with black bars corresponding to the regions amplified by qPCR.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/5/555/F4.medium.gif" width="284" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/2/5/555/F4.medium.gif" width="284" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/5/555/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/5/555/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461433" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-25" class="first-child">Effect of <em>spt16</em> mutants on chromatin structure at <em>SER3</em>. (A) Diagram of the <em>SER3</em> locus. The gray ovals mark the position of nucleosomes when wild-type cells are grown in <em>SER3</em>-repressing conditions (YPD). The block arrow indicates <em>SRG1</em> transcription. (B–H) Nucleosome scanning assays were performed on <em>spt16</em>Δ cells (YJ1091 and YJ1092) carrying plasmids expressing either wild-type <em>SPT16</em> or mutant <em>spt16</em> alleles as indicated. Mononucleosome-sized DNA fragments were generated by micrococcal nuclease (MNase) digestion of formaldeyde-treated chromatin that was isolated from cells grown to ∼2 × 10<sup>7</sup> cells/mL in YPD media at 30°. MNase protection across the <em>SER3</em> locus relative to a positioned nucleosome within the <em>GAL1</em> promoter was determined by qPCR. For each PCR amplicon, the mean MNase protection ± SEM from three independent experiments is plotted at its midpoint. Shown below each graph is a diagram of the <em>SER3</em> locus indicating the positions of nucleosomes (gray ovals) extrapolated from the MNase protection data for each <em>spt16</em> mutant. The light-gray ovals are indicative of less dramatic reductions in MNase protections compared with the wild-type control shown in panel A. (I) Histone H3 ChIP was performed on chromatin isolated from the same strains used in panels B–H. The amount of immunoprecipitated DNA was determined by qPCR as a fraction of the input that was then normalized to a control region in chromosome V and made relative to strains expressing wild-type <em>SPT16</em> (arbitrarily set to 1). Each bar represents the mean ± SEM of three independent experiments. Below the graph is a schematic of <em>SER3</em> with black bars corresponding to the regions amplified by qPCR.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">To confirm that the changes in MNase protection across the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> transcription unit caused by these <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants reflect changes in nucleosome occupancy, we measured histone occupancy across this region by ChIP. For the most part, histone H3 occupancy across the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> transcription unit was reduced in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants to degrees that correlate with the results of our MNase experiments (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4I</a>). Taken together, these data identify <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> residues whose integrity is required to maintain <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> repression by facilitating <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> transcription–dependent nucleosome occupancy across the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> promoter.</p></div><div id="sec-15" class="subsection"><h3>Effect of <em>spt16</em> mutations on phenotypes associated with defects in transcription and chromatin structure</h3><p id="p-27">Having shown a role for at least six of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> single mutants in regulating chromatin structure at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>, we tested whether all 12 single mutants confer other phenotypes indicative of chromatin-related transcriptional defects. We first determined whether these <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants can confer an Spt<sup>−</sup> phenotype (suppressor of Ty δ element insertion), which is caused by defects in chromatin and aberrant transcription initiation (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Clark-Adams <em>et al.</em> 1988</a>). <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em>Δ strains containing the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">lys2</a>-128</em>δ allele were transformed with plasmids containing either wild-type <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a></em> or mutant <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> alleles and assayed for their ability to grow on medium lacking lysine (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). As a control, we also introduced a plasmid expressing the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-G132D</em> allele, which has been previously shown to have an Spt<sup>−</sup> phenotype (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Evans <em>et al.</em> 1998</a>). Compared with the cells expressing wild-type <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a></em>, most of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants grew robustly in the absence of lysine, similar to what was observed for the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-G132D</em> control, indicating that these mutants confer a strong Spt<sup>−</sup> phenotype (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>). In contrast, the two <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants that most weakly derepress <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-E671G</em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-E679G</em>, had no detectable Spt<sup>−</sup> phenotype.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/5/555/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of spt16 mutants for phenotypes associated with defects in transcription and chromatin. (A) Assay for Spt&#x2212; phenotype. spt16&#x394; cells containing the lys2-128&#x3B4; allele and plasmids that express either wild-type or mutant Spt16, as indicated, were grown to saturation in YPD at 30&#xB0;, diluted to 108, and spotted in 10-fold serial dilutions onto solid synthetic complete medium (SC) and synthetic complete medium lacking lysine (SC-Lys). Plates were incubated at 30&#xB0; for 3 days. Results are representative of two independent assays using transformations of YJ1091. (B) Assay for cryptic transcription initiation. spt16&#x394; cells containing the FLO8-HIS3 reporter and plasmids that express either wild-type or mutant Spt16, as indicated, were grown to saturation in YPD at 30&#xB0;. Serial diluted cells were spotted onto solid synthetic complete medium with or without histidine containing either glucose (SC and SC-His) or galactose (SC/Gal and SC/Gal-His) as a carbon source. Plates were incubated at 30&#xB0; for either 3 days (SC, SC/Gal, and SC/Gal-His) or 6 days (SC-His). Results are representative of two independent assays using transformants of YJ1092. (C) Assay for suppression of histone H3 L61W mutant. spt16&#x394; cells expressing the H3-L61W mutant as its sole source of histone H3 and plasmids expressing either wild-type or mutant Spt16, as indicated, were grown to saturation in YPD at 30&#xB0;. Serial diluted cells were spotted onto solid YPD medium as described in panel A and grown at 30&#xB0; (3 days) or 15&#xB0; (17 days). Results were generated with strains derived from YADP50 (kindly provided by A. Duina) and are representative of two independent assays." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-202268667" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Analysis of &lt;em&gt;spt16&lt;/em&gt; mutants for phenotypes associated with defects in transcription and chromatin. (A) Assay for Spt&lt;sup&gt;&#x2212;&lt;/sup&gt; phenotype. &lt;em&gt;spt16&#x394;&lt;/em&gt; cells containing the &lt;em&gt;lys2-128&lt;/em&gt;&#x3B4; allele and plasmids that express either wild-type or mutant Spt16, as indicated, were grown to saturation in YPD at 30&#xB0;, diluted to 10&lt;sup&gt;8&lt;/sup&gt;, and spotted in 10-fold serial dilutions onto solid synthetic complete medium (SC) and synthetic complete medium lacking lysine (SC-Lys). Plates were incubated at 30&#xB0; for 3 days. Results are representative of two independent assays using transformations of YJ1091. (B) Assay for cryptic transcription initiation. &lt;em&gt;spt16&#x394;&lt;/em&gt; cells containing the &lt;em&gt;FLO8-HIS3&lt;/em&gt; reporter and plasmids that express either wild-type or mutant Spt16, as indicated, were grown to saturation in YPD at 30&#xB0;. Serial diluted cells were spotted onto solid synthetic complete medium with or without histidine containing either glucose (SC and SC-His) or galactose (SC/Gal and SC/Gal-His) as a carbon source. Plates were incubated at 30&#xB0; for either 3 days (SC, SC/Gal, and SC/Gal-His) or 6 days (SC-His). Results are representative of two independent assays using transformants of YJ1092. (C) Assay for suppression of histone H3 L61W mutant. &lt;em&gt;spt16&#x394;&lt;/em&gt; cells expressing the H3-L61W mutant as its sole source of histone H3 and plasmids expressing either wild-type or mutant Spt16, as indicated, were grown to saturation in YPD at 30&#xB0;. Serial diluted cells were spotted onto solid YPD medium as described in panel A and grown at 30&#xB0; (3 days) or 15&#xB0; (17 days). Results were generated with strains derived from YADP50 (kindly provided by A. Duina) and are representative of two independent assays.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/5/555/F5.medium.gif" width="440" height="120"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/2/5/555/F5.medium.gif" width="440" height="120"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/5/555/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/5/555/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461506" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-28" class="first-child">Analysis of <em>spt16</em> mutants for phenotypes associated with defects in transcription and chromatin. (A) Assay for Spt<sup>−</sup> phenotype. <em>spt16Δ</em> cells containing the <em>lys2-128</em>δ allele and plasmids that express either wild-type or mutant Spt16, as indicated, were grown to saturation in YPD at 30°, diluted to 10<sup>8</sup>, and spotted in 10-fold serial dilutions onto solid synthetic complete medium (SC) and synthetic complete medium lacking lysine (SC-Lys). Plates were incubated at 30° for 3 days. Results are representative of two independent assays using transformations of YJ1091. (B) Assay for cryptic transcription initiation. <em>spt16Δ</em> cells containing the <em>FLO8-HIS3</em> reporter and plasmids that express either wild-type or mutant Spt16, as indicated, were grown to saturation in YPD at 30°. Serial diluted cells were spotted onto solid synthetic complete medium with or without histidine containing either glucose (SC and SC-His) or galactose (SC/Gal and SC/Gal-His) as a carbon source. Plates were incubated at 30° for either 3 days (SC, SC/Gal, and SC/Gal-His) or 6 days (SC-His). Results are representative of two independent assays using transformants of YJ1092. (C) Assay for suppression of histone H3 L61W mutant. <em>spt16Δ</em> cells expressing the H3-L61W mutant as its sole source of histone H3 and plasmids expressing either wild-type or mutant Spt16, as indicated, were grown to saturation in YPD at 30°. Serial diluted cells were spotted onto solid YPD medium as described in panel A and grown at 30° (3 days) or 15° (17 days). Results were generated with strains derived from YADP50 (kindly provided by A. Duina) and are representative of two independent assays.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Next, we tested whether these <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants permit the production of aberrant intragenic transcripts, a phenotype that has been associated with defects in transcription-coupled nucleosome reassembly (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Carrozza <em>et al.</em> 2005</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kaplan <em>et al.</em> 2003</a>). For these experiments, we employed a previously described <em>GAL</em>pr-<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">FLO8</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> reporter gene in which <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> gene expression is dependent on transcription initiation from a cryptic promoter within the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">FLO8</a></em> coding sequence (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cheung <em>et al.</em> 2008</a>). Therefore, cryptic intragenic transcription can be measured by the growth of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a></em>Δ cells containing this reporter construct on medium lacking histidine. For this assay, strains expressing plasmid-borne <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a></em> or the indicated <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutant alleles were monitored for growth on medium lacking histidine. When grown in galactose-containing medium, all but two of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants allowed cells to grow in the absence of histidine, indicative of robust transcription initiation from the cryptic promoter within the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">FLO8</a></em> coding sequence (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>). For the most part, these data correlate well with the Spt<sup>−</sup> phenotypic data, suggesting that the molecular defects resulting in these two phenotypes are likely related. Interestingly, only those mutations within the N-terminal domain of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> allowed cells to grow in glucose-containing medium lacking histidine, suggesting that these mutants permit cryptic transcription initiation even in the absence of significant levels of transcription across this region.</p><p id="p-30">Finally, we tested the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants for their ability to suppress a cold-sensitive (cs) phenotype of a histone mutant, H3 L61W, a phenotype that has been previously described for a distinct class of mutations located within the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> M-domain (<a id="xref-ref-42-7" class="xref-bibr" href="#ref-42">Myers <em>et al.</em> 2011</a>). For this assay, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em>Δ cells containing the H3 L61W mutant as the sole source of histone H3 (YADP50) and plasmid-borne copies of wild-type or mutant versions of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a></em> were monitored for growth on YPD at 15° (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C</a>). When compared with the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-E735G</em> control (kindly provided by A. Duina), none of our identified <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants suppressed the cold sensitivity of H3 L61W. Therefore, the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants we isolated as being defective for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> repression represent a distinct class of mutants from those that suppress the cs phenotype of the H3 L61W mutant and may define functionally distinct regions of the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-D and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-M domains.</p></div><div id="sec-16" class="subsection"><h3>Occupancy of mutant versions of Spt16 is reduced across <em>SRG1</em> and the <em>SER3</em> promoter region</h3><p id="p-31">We next considered the possibility that these mutant versions of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> fail to be recruited normally to transcribed regions, which may account for their multiple phenotypes related to defects in transcription-coupled nucleosome occupancy. Therefore, we performed ChIP experiments to assess the binding of selected <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> mutant proteins across the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> transcription unit (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a>). In general, we detected reduced binding of most of the mutant versions of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> that parallels the loss of histone H3 occupancy across this region that we observed in these mutant versions (compare <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6A</a> with <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4I</a>). The lone exception is the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-K579E</em> mutant where we detected a stronger decrease in the occupancy of the mutant protein expressed from this allele than expected based on a relatively modest decrease in histone H3 occupancy. Because <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> strongly colocalizes with RNA Pol II across transcribed genes, we tested whether the decrease in the occupancy of the mutant versions of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> might be indirect due to a decrease in RNA Pol II occupancy at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>. To this end, we performed ChIP analysis of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001283">Rpb3</a>, a subunit of RNA Pol II, over <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6B</a>). Consistent with our Northern analysis (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>), we found that all but one of these <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants did not cause a decrease in RNA Pol II occupancy compared with cells expressing wild-type <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a></em>. Interestingly, the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-L669S</em> mutant did cause a slight but significant decrease (<em>P</em> &lt; 0.05) in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001283">Rpb3</a> binding across <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em>. However, by normalizing the binding of this mutant version of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> to <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001283">Rpb3</a> binding, it is clear that this minor decrease in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001283">Rpb3</a> binding alone cannot account for the reduced binding of this mutant version of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> across <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6C</a>). Taken together, these data indicate that the amino acids defined by these mutants are required to maintain <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> colocalization with RNA Pol II across <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em>.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/5/555/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relative occupancy of Spt16 and RNA Pol II across SER3 in spt16 mutants. Spt16 (A) and Rpb3 (B) ChIP experiments were performed on chromatin prepared from spt16&#x394; strains expressing either wild-type or mutant Spt16, as indicated, that were grown in YPD at 30&#xB0;. The amount of immunoprecipitated DNA at four locations across SER3 (indicated by black bars in the diagram of SER3 below the graphs) was determined by qPCR as a fraction of the input material and normalized to a control region in chromosome V. Each bar represents the mean &#xB1; SEM of three independent experiments using strains derived from YJ1091 and YJ1092. Occupancy of these factors in the strains expressing wild-type Spt16 was arbitrarily set to 1 at each SER3 location. (C) Occupancy of Spt16 across SER3 was recalculated relative to Rpb3 occupancy." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-202268667" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relative occupancy of Spt16 and RNA Pol II across &lt;em&gt;SER3&lt;/em&gt; in &lt;em&gt;spt16&lt;/em&gt; mutants. Spt16 (A) and Rpb3 (B) ChIP experiments were performed on chromatin prepared from &lt;em&gt;spt16&lt;/em&gt;&#x394; strains expressing either wild-type or mutant Spt16, as indicated, that were grown in YPD at 30&#xB0;. The amount of immunoprecipitated DNA at four locations across &lt;em&gt;SER3&lt;/em&gt; (indicated by black bars in the diagram of &lt;em&gt;SER3&lt;/em&gt; below the graphs) was determined by qPCR as a fraction of the input material and normalized to a control region in chromosome V. Each bar represents the mean &#xB1; SEM of three independent experiments using strains derived from YJ1091 and YJ1092. Occupancy of these factors in the strains expressing wild-type Spt16 was arbitrarily set to 1 at each &lt;em&gt;SER3&lt;/em&gt; location. (C) Occupancy of Spt16 across &lt;em&gt;SER3&lt;/em&gt; was recalculated relative to Rpb3 occupancy.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/5/555/F6.medium.gif" width="221" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/2/5/555/F6.medium.gif" width="221" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/5/555/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/5/555/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461485" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-32" class="first-child">Relative occupancy of Spt16 and RNA Pol II across <em>SER3</em> in <em>spt16</em> mutants. Spt16 (A) and Rpb3 (B) ChIP experiments were performed on chromatin prepared from <em>spt16</em>Δ strains expressing either wild-type or mutant Spt16, as indicated, that were grown in YPD at 30°. The amount of immunoprecipitated DNA at four locations across <em>SER3</em> (indicated by black bars in the diagram of <em>SER3</em> below the graphs) was determined by qPCR as a fraction of the input material and normalized to a control region in chromosome V. Each bar represents the mean ± SEM of three independent experiments using strains derived from YJ1091 and YJ1092. Occupancy of these factors in the strains expressing wild-type Spt16 was arbitrarily set to 1 at each <em>SER3</em> location. (C) Occupancy of Spt16 across <em>SER3</em> was recalculated relative to Rpb3 occupancy.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Effect of <em>spt16</em> mutants on histone H3, Spt16, and RNA Pol II occupancy at other genes</h3><p id="p-33">To investigate whether the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants that reduce nucleosome occupancy across <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> have a general defect in transcription-coupled nucleosome occupancy, we measured histone H3 occupancy across the coding sequences of a subset of yeast genes by ChIP (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>). At three highly transcribed genes, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em> (100 mRNA/hr), <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000036">PYK1</a></em> (95 mRNA/hr), and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005446">ADH1</a></em> (125 mRNA/hr) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Holstege <em>et al.</em> 1998</a>), histone H3 levels were reduced in all of the mutants to a similar extent as we observed across <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em>. Conversely, histone H3 occupancy at three lowly transcribed genes, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000224">GAL1</a></em> (repressed), <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001857">TUB2</a></em> (12 mRNA/hr), and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003809">CYC1</a></em> (10 mRNA/hr) (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Holstege <em>et al.</em> 1998</a>), was unaffected in the mutants.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/5/555/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relative occupancy of histone H3, Spt16, and Rpb3 across the coding regions of a subset of yeast genes. Histone H3 (A), Spt16 (B), and Rpb3 (C) was measured by ChIP within the coding region of three highly transcribed genes: PMA1, PYK1, and ADH1 (top panels in A, B, and C) and over three lowly transcribed genes: GAL1, TUB2, and CYC1 (bottom panels in A, B, and C) as described in Figure 6. The regions assayed by qPCR are marked with the black bars in the diagram provided for each gene. All values represent the mean &#xB1; SEM of three independent experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-202268667" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relative occupancy of histone H3, Spt16, and Rpb3 across the coding regions of a subset of yeast genes. Histone H3 (A), Spt16 (B), and Rpb3 (C) was measured by ChIP within the coding region of three highly transcribed genes: &lt;em&gt;PMA1&lt;/em&gt;, &lt;em&gt;PYK1&lt;/em&gt;, and &lt;em&gt;ADH1&lt;/em&gt; (top panels in A, B, and C) and over three lowly transcribed genes: &lt;em&gt;GAL1&lt;/em&gt;, &lt;em&gt;TUB2&lt;/em&gt;, and &lt;em&gt;CYC1&lt;/em&gt; (bottom panels in A, B, and C) as described in Figure 6. The regions assayed by qPCR are marked with the black bars in the diagram provided for each gene. All values represent the mean &#xB1; SEM of three independent experiments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/5/555/F7.medium.gif" width="253" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/2/5/555/F7.medium.gif" width="253" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/5/555/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/5/555/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461491" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-34" class="first-child">Relative occupancy of histone H3, Spt16, and Rpb3 across the coding regions of a subset of yeast genes. Histone H3 (A), Spt16 (B), and Rpb3 (C) was measured by ChIP within the coding region of three highly transcribed genes: <em>PMA1</em>, <em>PYK1</em>, and <em>ADH1</em> (top panels in A, B, and C) and over three lowly transcribed genes: <em>GAL1</em>, <em>TUB2</em>, and <em>CYC1</em> (bottom panels in A, B, and C) as described in <a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6</a>. The regions assayed by qPCR are marked with the black bars in the diagram provided for each gene. All values represent the mean ± SEM of three independent experiments.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">We next examined the occupancy of these mutant derivatives of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> across the coding sequence of this subset of yeast genes (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a>). Consistent with our results at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em>, we found that at the highly transcribed genes, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000036">PYK1</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005446">ADH1</a></em>, the binding of the mutant <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> proteins were generally reduced in accordance with the decrease in histone H3 levels across these regions. Interestingly, the decrease in the occupancy of these mutant <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> proteins was greater at the 5′ end of these genes compared with regions toward the 3′ end. For the most part, these changes in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> binding occurred in the absence of any change in RNA Pol II binding to these regions (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7C</a>). Interestingly, a small but significant decrease in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001283">Rpb3</a> levels in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-L669S</em> mutant (<em>P</em> &lt; 0.05) was detected at these highly transcribed genes, comparable to what we observed at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em>. In contrast to what we observed at highly transcribed genes, occupancy of the mutant <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> proteins and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001283">Rpb3</a> at three lowly transcribed genes, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000224">GAL1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001857">TUB2</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003809">CYC1</a></em>, were largely unaffected. Importantly, we show that when these mutants are grown in galactose-containing medium to induce high levels of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000224">GAL1</a></em> expression, we detected reduced occupancy of both H3 and the mutant <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> proteins to the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000224">GAL1</a></em> coding sequence similar to what we observed for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> and other highly transcribed genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002451/-/DC1/FigureS1.pdf">Figure S1</a>). Thus, we have identified mutant <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> alleles that cause reduced occupancy of both the mutant version of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> encoded by these alleles and histones specifically over highly transcribed regions of the genome. Taken together, our studies suggest that the integrity of the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-D and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-M domains is generally required to maintain nucleosome occupancy at highly transcribed genes, possibly by facilitating <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> recruitment to those genes.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-36"><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> is an essential, highly conserved component of the FACT elongation complex with a dual role in transcription elongation—the disassembly of nucleosomes to allow the passage of RNA Pol II and their reassembly in the wake of RNA Pol II [reviewed in <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Duina (2011)</a>, <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Formosa (2011)</a>, <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Reinberg and Sims (2006)</a>, and <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Winkler and Luger (2011)</a>]. In this work, we provide evidence indicating that the integrity of both the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-D and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-M domains are required to support the histone chaperone activities of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> during transcription elongation. We utilized a previously characterized system in which this activity of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> is required for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> transcription–dependent repression of the <em>S. cerevisiae <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> gene (<a id="xref-ref-23-11" class="xref-bibr" href="#ref-23">Hainer <em>et al.</em> 2011</a>) to identify a largely novel class of mutations in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a></em> that derepress <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em>. Six mutations that most strongly derepress <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> contain single amino acid substitutions in either the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-D or <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-M domain. For this subset of mutants, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> transcription–coupled nucleosome occupancy over the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> promoter is reduced to degrees that generally correlate with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> derepression. Moreover, we provide evidence that these mutations broadly disrupt transcription-coupled nucleosome occupancy at highly transcribed regions of the yeast genome. Finally, we show that while these mutant versions of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> are expressed at wild-type levels, their association with highly transcribed genes is significantly reduced. These data suggest that the integrity of the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-D and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-M domains are required for transcription-coupled nucleosome occupancy, possibly by promoting or maintaining FACT association with transcribed regions of the genome.</p><p id="p-37">With one exception (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-E857K</em>), the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants that we identified in this work are distinct from those that have been previously identified by other genetic approaches (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Formosa <em>et al.</em> 2002</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Malone <em>et al.</em> 1991</a>; <a id="xref-ref-42-8" class="xref-bibr" href="#ref-42">Myers <em>et al.</em> 2011</a>; <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">O’Donnell <em>et al.</em> 2009</a>; <a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Stevens <em>et al.</em> 2011</a>). Although most of this new class of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants confers an Spt<sup>−</sup> phenotype similar to many previously characterized <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants, additional phenotypic studies indicate that there are important functional differences between these mutants. First, these mutants do not confer lethality at elevated temperature as is common for many previously characterized <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutant alleles (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Formosa <em>et al.</em> 2002</a>; <a id="xref-ref-42-9" class="xref-bibr" href="#ref-42">Myers <em>et al.</em> 2011</a>; <a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">O’Donnell <em>et al.</em> 2009</a>). This result suggests that the amino acid substitutions caused by these mutations are not likely to affect the general stability of the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> protein. Furthermore, these results indicate that the ability of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> to promote nucleosome assembly during transcription is not essential for viability. Second, these mutants do not confer a growth defect in the presence of hydroxyurea, a phenotype conferred by other <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Formosa <em>et al.</em> 2002</a>; <a id="xref-ref-42-10" class="xref-bibr" href="#ref-42">Myers <em>et al.</em> 2011</a>; <a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">O’Donnell <em>et al.</em> 2009</a>) that is indicative of a defect in DNA replication and/or DNA repair (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hampsey 1997</a>). Therefore, this new group of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants may define an activity for <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> that is specific to its role in transcription elongation rather than a histone chaperone activity that may be generally required for all of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> functions. Third, these <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants do not suppress a cold-sensitive growth defect conferred by a histone H3 L61W as has been recently described for a distinct set of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutant alleles (<a id="xref-ref-42-11" class="xref-bibr" href="#ref-42">Myers <em>et al.</em> 2011</a>). This is somewhat surprising given that both groups of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants have amino acid substitutions within the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-M domain. Moreover, one of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants isolated as a suppressor of the cold sensitivity of the histone H3 L61W mutant contains a glutamine substitution of glutamic acid residue at position 847, the same residue that, when substituted for a lysine, confers strong <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> derepression and transcription-coupled nucleosome assembly defects. However, the lysine substitution did not suppress the cold sensitivity of the H3 L61W mutation. Taken together, these data show that we have identified a new class of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants that interferes with <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> activity specific to its role in transcription-coupled nucleosome assembly rather than its generally required functions in transcription, cell viability, and/or DNA replication/DNA repair.</p><p id="p-38">During our phenotypic analyses, we found that most of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants that were isolated based on their ability to derepress <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> also confer sensitivity to cyclohexamide, a phenotype that has not been previously described for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutant alleles. Cyclohexamide is a potent inhibitor of eukaryotic protein synthesis that is normally toxic to yeast cells (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Mccusker and Haber 1988</a>). However, at low doses, a sensitivity to this drug has been shown to reveal mutations that reduce protein synthesis or impair cell-cycle progression (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Hampsey 1997</a>). Therefore, although the identification of this phenotype may be interesting, the interpretation of the data are unclear. We hypothesize that the subset of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants causing cyclohexamide sensitivity do so as a result of the misregulation of one or more genes encoding proteins that are essential for viability, regulate protein synthesis, or regulate intracellular levels of cyclohexamide.</p><p id="p-39">Interestingly, the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-E857K</em> allele, which we found to confer a dominant negative effect on <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> repression, was previously isolated as a dominant suppressor of the transcription defects of δ element insertions just 5′ of both the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">LYS2</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000535">HIS4</a></em> genes (<a id="xref-ref-45-6" class="xref-bibr" href="#ref-45">O’Donnell <em>et al.</em> 2009</a>; <a id="xref-ref-58-4" class="xref-bibr" href="#ref-58">Stevens <em>et al.</em> 2011</a>). This is not surprising given the striking similarities between <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> repression by <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> transcription and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">LYS2</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000535">HIS4</a></em> repression by the δ element insertions (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Clark-Adams and Winston 1987</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Martens <em>et al.</em> 2004</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Winston <em>et al.</em> 1984</a>). Both <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> and the δ element insertion promote transcription across the promoters of their adjacent genes, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> and either <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">LYS2</a></em> or <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000535">HIS4</a></em>, respectively. Our finding that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> derepression in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-E857K</em> mutant is the result of reduced <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029010">SRG1</a></em> transcription–dependent nucleosome assembly at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> promoter suggests that a similar transcription defect in nucleosome occupancy may play a role in alleviating repression of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">LYS2</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000535">HIS4</a></em> caused by these δ element insertions. Interestingly, we found that whereas three of the five double mutants containing the <em>E857K</em> substitution also act in a dominant manner, two of these combinations, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-I626T/E857K</em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a>-T651A/H741Y/E857K</em>, do not. Moreover, we found that the level of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000883">SER3</a></em> derepression in these two mutant alleles to be significantly lower to that caused by the <em>E857K</em> substitution alone (B. A. Charsar, unpublished data). Therefore, <em>I626T</em> and one or both of <em>T651A</em> and <em>H741Y</em> substitutions appear to suppress the negative effects of the <em>E857K</em> substitution.</p><p id="p-40">Our analysis of the single amino acid substitutions in the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-D and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-M domains revealed a strong correlation between defective transcription-dependent nucleosome assembly and reduced association of these mutant versions of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> at highly transcribed regions of the yeast genome. Several possible models could account for these observations. First, these mutant versions of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> may interfere with the normal recruitment of FACT to transcribed DNA. In this model, the reduced recruitment of FACT would be the cause of the defect in transcription-coupled nucleosome assembly. Although several studies have determined that FACT physically associates with DNA that is being transcribed (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Duina <em>et al.</em> 2007</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kim <em>et al.</em> 2004</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Mason and Struhl 2003</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Mayer <em>et al.</em> 2010</a>), the molecular mechanism of this association is not known. Previous studies have implicated a number of factors that may facilitate <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> association with transcribed DNA, including the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000966">Chd1</a> chromatin-remodeling factor, the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000483">Paf1</a> elongation complex, RNA Pol II, and histone proteins (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adelman <em>et al.</em> 2006</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Biswas <em>et al.</em> 2007</a>; <a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Formosa <em>et al.</em> 2001</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Mason and Struhl 2003</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Pruneski <em>et al.</em> 2011</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Simic <em>et al.</em> 2003</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Winkler <em>et al.</em> 2011</a>). It is conceivable that the amino acid substitutions within the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-D and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-M domains that interfere with transcription-coupled nucleosome assembly do so by altering FACT interactions with one or more of these factors. Second, the reduction in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> association with transcribed regions may be a consequence of the reduced nucleosome occupancy due to a defect in transcription-coupled nucleosome assembly. In this model, the amino acid substitutions in the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-D and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-M domains would not alter initial <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> recruitment to transcribed DNA or its ability to associate with nucleosomal DNA but, rather, interfere with its nucleosome remodeling activity that leads to disassembly and/or reassembly of nucleosomes during transcription. Additional molecular and biochemical experiments to investigate the affect of these mutants on FACT interactions with other proteins and the nucleosome remodeling activity of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> will be necessary to distinguish between these models.</p><p id="p-41">Although the possibility that the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-D and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-M domains may directly mediate protein-protein interactions or FACT nucleosome remodeling activity is intriguing, we cannot rule out a more indirect role for these domains. For example, it is possible that the three mutations in the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-D domain may simply disrupt the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004534">Pob3</a> interface (<a id="xref-ref-61-5" class="xref-bibr" href="#ref-61">VanDemark <em>et al.</em> 2006</a>, <a id="xref-ref-62-5" class="xref-bibr" href="#ref-62">2008</a>). However, if this were the case, we would expect any changes in the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004534">Pob3</a> dimer interface to be subtle, specifically affecting the activity of FACT in transcription-dependent nucleosome assembly rather than in a more general histone chaperone role. Large perturbations in the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003348">Spt6</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004534">Pob3</a> interaction would most likely lead to more broad defects in cell growth and DNA replication/repair, which were not detected in these mutants by our phenotypic assays.</p><p id="p-42">In summary, we have identified a novel class of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants that specifically impairs transcription-coupled nucleosome occupancy across highly transcribed regions of the <em>S. cerevisiae</em> genome and results in reduced association of the mutant <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> proteins to these regions. These mutants are likely to be useful molecular tools to further elucidate the dynamic function of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> in maintaining chromatin architecture during transcription.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-43">We thank Andrea Duina and Tim Formosa for providing plasmids, strains, and antibodies used in this work. We are grateful to Erin Walker for her assistance in piloting the genetic screen described in this report and to the 2009 University of Pittsburgh Summer Gene Team (supported by the National Center for Research Resources and the Division of Program Coordination, Planning, and Strategic Initiatives of the National Institutes of Health through Grant Number R25-RR-020463) led by Justin Pruneski, Alison Slinskey Legg, and Lewis Jacobson for performing the large-scale screen for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">spt16</a></em> mutants. S.J.H. was supported by an Andrew Mellon predoctoral fellowship. B.A.C. was supported by undergraduate research awards provided by the Department of Biological Sciences from an Undergraduate Science Education Grant from the Howard Hughes Medical Institute and from a generous donation by Samuel D. Colella. This work was supported by NIH Grant GM-080470 to J.A.M.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002451/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002451/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-44"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-02-01"><span class="received-label">Received </span>February 1, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-03-04"><span class="accepted-label">Accepted </span>March 4, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Hainer <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.1" data-doi="10.1128/MCB.26.1.250-260.2006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adelman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ardehali</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werner</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Drosophila Paf1 modulates chromatin structure at actively transcribed genes</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DAdelman%26rft.auinit1%253DK.%26rft.volume%253D26%26rft.issue%253D1%26rft.spage%253D250%26rft.epage%253D260%26rft.atitle%253DDrosophila%2BPaf1%2BModulates%2BChromatin%2BStructure%2Bat%2BActively%2BTranscribed%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.26.1.250-260.2006%26rft_id%253Dinfo%253Apmid%252F16354696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjI2LzEvMjUwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi81LzU1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-2.5.555.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ausubel</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brent</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kingston</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seidman</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-comment">(Editors)</span>, <span class="cit-pub-date">1991</span> <span class="cit-comment"><em>Current Protocols in Molecular Biology</em>. John Wiley and Sons, New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.3" data-doi="10.1016/j.tig.2010.08.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morozov</span>  <span class="cit-name-given-names">A. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Gene regulation by nucleosome positioning</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">476</span>–<span class="cit-lpage">483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DBai%26rft.auinit1%253DL.%26rft.volume%253D26%26rft.issue%253D11%26rft.spage%253D476%26rft.epage%253D483%26rft.atitle%253DGene%2Bregulation%2Bby%2Bnucleosome%2Bpositioning.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2010.08.003%26rft_id%253Dinfo%253Apmid%252F20832136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2010.08.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20832136&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283678500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.4" data-doi="10.1126/science.1085703"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Belotserkovskaya</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bondarenko</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orphanides</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Studitsky</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">FACT facilitates transcription-dependent nucleosome alteration</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">301</span>: <span class="cit-fpage">1090</span>–<span class="cit-lpage">1093</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBelotserkovskaya%26rft.auinit1%253DR.%26rft.volume%253D301%26rft.issue%253D5636%26rft.spage%253D1090%26rft.epage%253D1093%26rft.atitle%253DFACT%2BFacilitates%2BTranscription-Dependent%2BNucleosome%2BAlteration%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1085703%26rft_id%253Dinfo%253Apmid%252F12934006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDEvNTYzNi8xMDkwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi81LzU1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.5" data-doi="10.1128/MCB.00978-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Biswas</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dutta-Biswas</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Chd1 and yFACT act in opposition in regulating transcription</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">6279</span>–<span class="cit-lpage">6287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00978-07%26rft_id%253Dinfo%253Apmid%252F17620414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8xOC82Mjc5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi81LzU1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.6" data-doi="10.1371/journal.pbio.0050081"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brickner</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cajigas</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fondufe-Mittendorf</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahmed</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">H2A.Z-mediated localization of genes at the nuclear periphery confers epigenetic memory of previous transcriptional state</span>. <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DBrickner%26rft.auinit1%253DD.%2BG.%26rft.volume%253D5%26rft.issue%253D4%26rft.spage%253De81%26rft.epage%253De81%26rft.atitle%253DH2A.Z-mediated%2Blocalization%2Bof%2Bgenes%2Bat%2Bthe%2Bnuclear%2Bperiphery%2Bconfers%2Bepigenetic%2Bmemory%2Bof%2Bprevious%2Btranscriptional%2Bstate.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0050081%26rft_id%253Dinfo%253Apmid%252F17373856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0050081&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17373856&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.7" data-doi="10.1038/nature08450"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cairns</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The logic of chromatin architecture and remodelling at promoters</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCairns%26rft.auinit1%253DB.%2BR.%26rft.volume%253D461%26rft.issue%253D7261%26rft.spage%253D193%26rft.epage%253D198%26rft.atitle%253DThe%2Blogic%2Bof%2Bchromatin%2Barchitecture%2Band%2Bremodelling%2Bat%2Bpromoters.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08450%26rft_id%253Dinfo%253Apmid%252F19741699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08450&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19741699&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269654600031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.8" data-doi="10.1016/j.cell.2005.10.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carrozza</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Florens</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suganuma</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swanson</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Histone H3 methylation by Set2 directs deacetylation of coding regions by Rpd3S to suppress spurious intragenic transcription</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">581</span>–<span class="cit-lpage">592</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DCarrozza%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D123%26rft.issue%253D4%26rft.spage%253D581%26rft.epage%253D592%26rft.atitle%253DHistone%2BH3%2Bmethylation%2Bby%2BSet2%2Bdirects%2Bdeacetylation%2Bof%2Bcoding%2Bregions%2Bby%2BRpd3S%2Bto%2Bsuppress%2Bspurious%2Bintragenic%2Btranscription.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2005.10.023%26rft_id%253Dinfo%253Apmid%252F16286007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2005.10.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16286007&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233497400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.9" data-doi="10.1371/journal.pbio.0060277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batada</span>  <span class="cit-name-given-names">N. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michnick</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Chromatin- and transcription-related factors repress transcription from within coding regions throughout the Saccharomyces cerevisiae genome</span>. <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DCheung%26rft.auinit1%253DV.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De277%26rft.epage%253De277%26rft.atitle%253DChromatin-%2Band%2Btranscription-related%2Bfactors%2Brepress%2Btranscription%2Bfrom%2Bwithin%2Bcoding%2Bregions%2Bthroughout%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0060277%26rft_id%253Dinfo%253Apmid%252F18998772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0060277&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18998772&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.10" data-doi="10.1128/MCB.7.2.679"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark-Adams</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The SPT6 gene is essential for growth and is required for delta-mediated transcription in Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">679</span>–<span class="cit-lpage">686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DClark-Adams%26rft.auinit1%253DC%2BD%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D679%26rft.epage%253D686%26rft.atitle%253DThe%2BSPT6%2Bgene%2Bis%2Bessential%2Bfor%2Bgrowth%2Band%2Bis%2Brequired%2Bfor%2Bdelta-mediated%2Btranscription%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.7.2.679%26rft_id%253Dinfo%253Apmid%252F3029564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjc6IjcvMi82NzkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzUvNTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.11" data-doi="10.1101/gad.2.2.150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark-Adams</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Norris</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osley</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fassler</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Changes in histone gene dosage alter transcription in yeast</span>. <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">150</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DClark-Adams%26rft.auinit1%253DC%2BD%26rft.volume%253D2%26rft.issue%253D2%26rft.spage%253D150%26rft.epage%253D159%26rft.atitle%253DChanges%2Bin%2Bhistone%2Bgene%2Bdosage%2Balter%2Btranscription%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.2.2.150%26rft_id%253Dinfo%253Apmid%252F2834270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6NzoiMi8yLzE1MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNS81NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-2.5.555.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collart</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliviero</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-comment">Preparation of yeast RNA. Curr. Protoc. Mol. Biol. Chapter 13: Unit 13.12</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.13" data-doi="10.1091/mbc.8.9.1805"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The unfolded protein response coordinates the production of endoplasmic reticulum protein and endoplasmic reticulum membrane</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1805</span>–<span class="cit-lpage">1814</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DCox%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D8%26rft.issue%253D9%26rft.spage%253D1805%26rft.epage%253D1814%26rft.atitle%253DThe%2Bunfolded%2Bprotein%2Bresponse%2Bcoordinates%2Bthe%2Bproduction%2Bof%2Bendoplasmic%2Breticulum%2Bprotein%2Band%2Bendoplasmic%2Breticulum%2Bmembrane.%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.8.9.1805%26rft_id%253Dinfo%253Apmid%252F9307975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjgvOS8xODA1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi81LzU1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duina</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Histone chaperones Spt16 and FACT: similarities and differences in modes of action at transcribed genes</span>. <abbr class="cit-jnl-abbrev">Genet. Res. Int.</abbr> <span class="cit-comment">2011: Article ID 625201</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.15" data-doi="10.1534/genetics.106.067140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duina</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rufiange</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bracey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nourani</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evidence that the localization of the elongation factor Spt16 across transcribed genes is dependent upon histone H3 integrity in Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.067140%26rft_id%253Dinfo%253Apmid%252F17603125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc3LzEvMTAxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi81LzU1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brewster</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rowley</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altheim</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The yeast protein complex containing cdc68 and pob3 mediates core-promoter repression through the cdc68 N-terminal domain</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">1393</span>–<span class="cit-lpage">1405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D150%26rft.spage%253D1393%26rft_id%253Dinfo%253Apmid%252F9832518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9832518&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077401300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.17" data-doi="10.1016/j.cell.2010.03.048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Floer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prabhu</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berrozpe</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Narayan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A RSC/nucleosome complex determines chromatin architecture and facilitates activator binding</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">407</span>–<span class="cit-lpage">418</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DFloer%26rft.auinit1%253DM.%26rft.volume%253D141%26rft.issue%253D3%26rft.spage%253D407%26rft.epage%253D418%26rft.atitle%253DA%2BRSC%252Fnucleosome%2Bcomplex%2Bdetermines%2Bchromatin%2Barchitecture%2Band%2Bfacilitates%2Bactivator%2Bbinding.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.03.048%26rft_id%253Dinfo%253Apmid%252F20434983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.03.048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20434983&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277180800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.18" data-doi="10.1039/b812136b"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Formosa</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">FACT and the reorganized nucleosome</span>. <abbr class="cit-jnl-abbrev">Mol. Biosyst.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1085</span>–<span class="cit-lpage">1093</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BBiosyst%26rft.aulast%253DFormosa%26rft.auinit1%253DT.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253D1085%26rft.epage%253D1093%26rft.atitle%253DFACT%2Band%2Bthe%2Breorganized%2Bnucleosome.%26rft_id%253Dinfo%253Adoi%252F10.1039%252Fb812136b%26rft_id%253Dinfo%253Apmid%252F18931784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1039/b812136b&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18931784&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Formosa</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The role of FACT in making and breaking nucleosomes</span>. <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr>. <span class="cit-pub-date">1819</span>: <span class="cit-fpage">247</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.20" data-doi="10.1093/emboj/20.13.3506"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Formosa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eriksson</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittmeyer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ginn</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Spt16-Pob3 and the HMG protein Nhp6 combine to form the nucleosome-binding factor SPN</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3506</span>–<span class="cit-lpage">3517</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DFormosa%26rft.auinit1%253DT.%26rft.volume%253D20%26rft.issue%253D13%26rft.spage%253D3506%26rft.epage%253D3517%26rft.atitle%253DSpt16-Pob3%2Band%2Bthe%2BHMG%2Bprotein%2BNhp6%2Bcombine%2Bto%2Bform%2Bthe%2Bnucleosome-binding%2Bfactor%2BSPN.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F20.13.3506%26rft_id%253Dinfo%253Apmid%252F11432837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjAvMTMvMzUwNiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNS81NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Formosa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruone</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eriksson</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Defects in SPT16 or POB3 (yFACT) in Saccharomyces cerevisiae cause dependence on the Hir/Hpc pathway: polymerase passage may degrade chromatin structure</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">1557</span>–<span class="cit-lpage">1571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D162%26rft.spage%253D1557%26rft_id%253Dinfo%253Apmid%252F12524332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12524332&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180502300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.22" data-doi="10.1128/MCB.05195-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hainer</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martens</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of histone mutants that are defective for transcription-coupled nucleosome occupancy</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3557</span>–<span class="cit-lpage">3568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.05195-11%26rft_id%253Dinfo%253Apmid%252F21730290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIzMS8xNy8zNTU3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi81LzU1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.23" data-doi="10.1101/gad.1975011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hainer</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pruneski</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monteverde</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martens</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Intergenic transcription causes repression by directing nucleosome assembly</span>. <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1975011%26rft_id%253Dinfo%253Apmid%252F21156811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6NzoiMjUvMS8yOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNS81NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.24" data-doi="10.1002/(SICI)1097-0061(19970930)13:12&lt;1099::AID-YEA177&gt;3.0.CO;2-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hampsey</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">A review of phenotypes in Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1099</span>–<span class="cit-lpage">1133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DHampsey%26rft.auinit1%253DM.%26rft.volume%253D13%26rft.issue%253D12%26rft.spage%253D1099%26rft.epage%253D1133%26rft.atitle%253DA%2Breview%2Bof%2Bphenotypes%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%252819970930%252913%253A12%253C1099%253A%253AAID-YEA177%253E3.0.CO%253B2-7%26rft_id%253Dinfo%253Apmid%252F9301019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(19970930)13:12&lt;1099::AID-YEA177&gt;3.0.CO;2-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9301019&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XU74400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.25" data-doi="10.1016/S0092-8674(00)81641-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holstege</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jennings</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wyrick</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hengartner</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Dissecting the regulatory circuitry of a eukaryotic genome</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">717</span>–<span class="cit-lpage">728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHolstege%26rft.auinit1%253DF.%2BC.%26rft.volume%253D95%26rft.issue%253D5%26rft.spage%253D717%26rft.epage%253D728%26rft.atitle%253DDissecting%2Bthe%2Bregulatory%2Bcircuitry%2Bof%2Ba%2Beukaryotic%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981641-4%26rft_id%253Dinfo%253Apmid%252F9845373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81641-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9845373&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077253700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.26" data-doi="10.1016/j.molcel.2009.07.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jamai</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puglisi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strubin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Histone chaperone spt16 promotes redeposition of the original h3-h4 histones evicted by elongating RNA polymerase</span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">377</span>–<span class="cit-lpage">383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DJamai%26rft.auinit1%253DA.%26rft.volume%253D35%26rft.issue%253D3%26rft.spage%253D377%26rft.epage%253D383%26rft.atitle%253DHistone%2Bchaperone%2Bspt16%2Bpromotes%2Bredeposition%2Bof%2Bthe%2Boriginal%2Bh3-h4%2Bhistones%2Bevicted%2Bby%2Belongating%2BRNA%2Bpolymerase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2009.07.001%26rft_id%253Dinfo%253Apmid%252F19683500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2009.07.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19683500&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269278900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.27" data-doi="10.1126/science.1087374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laprade</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Transcription elongation factors repress transcription initiation from cryptic sites</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">301</span>: <span class="cit-fpage">1096</span>–<span class="cit-lpage">1099</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKaplan%26rft.auinit1%253DC.%2BD.%26rft.volume%253D301%26rft.issue%253D5636%26rft.spage%253D1096%26rft.epage%253D1099%26rft.atitle%253DTranscription%2BElongation%2BFactors%2BRepress%2BTranscription%2BInitiation%2Bfrom%2BCryptic%2BSites%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1087374%26rft_id%253Dinfo%253Apmid%252F12934008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDEvNTYzNi8xMDk2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi81LzU1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.28" data-doi="10.1038/sj.emboj.7600053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahn</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krogan</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenblatt</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buratowski</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Transitions in RNA polymerase II elongation complexes at the 3′ ends of genes</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">354</span>–<span class="cit-lpage">364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7600053%26rft_id%253Dinfo%253Apmid%252F14739930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo4OiIyMy8yLzM1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNS81NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.29" data-doi="10.1101/gad.824700"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Komarnitsky</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buratowski</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Different phosphorylated forms of RNA polymerase II and associated mRNA processing factors during transcription</span>. <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2452</span>–<span class="cit-lpage">2460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DKomarnitsky%26rft.auinit1%253DP.%26rft.volume%253D14%26rft.issue%253D19%26rft.spage%253D2452%26rft.epage%253D2460%26rft.atitle%253DDifferent%2Bphosphorylated%2Bforms%2Bof%2BRNA%2Bpolymerase%2BII%2Band%2Bassociated%2BmRNA%2Bprocessing%2Bfactors%2Bduring%2Btranscription%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.824700%26rft_id%253Dinfo%253Apmid%252F11018013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE0LzE5LzI0NTIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzUvNTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.30" data-doi="10.1128/MCB.22.20.6979-6992.2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krogan</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahn</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobor</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">RNA polymerase II elongation factors of Saccharomyces cerevisiae: a targeted proteomics approach</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">6979</span>–<span class="cit-lpage">6992</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DKrogan%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D22%26rft.issue%253D20%26rft.spage%253D6979%26rft.epage%253D6992%26rft.atitle%253DRNA%2BPolymerase%2BII%2BElongation%2BFactors%2Bof%2BSaccharomyces%2Bcerevisiae%253A%2Ba%2BTargeted%2BProteomics%2BApproach%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.22.20.6979-6992.2002%26rft_id%253Dinfo%253Apmid%252F12242279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8yMC82OTc5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi81LzU1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.31" data-doi="10.1016/j.cell.2007.01.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Workman</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The role of chromatin during transcription</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLi%26rft.auinit1%253DB.%26rft.volume%253D128%26rft.issue%253D4%26rft.spage%253D707%26rft.epage%253D719%26rft.atitle%253DThe%2Brole%2Bof%2Bchromatin%2Bduring%2Btranscription.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.01.015%26rft_id%253Dinfo%253Apmid%252F17320508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.01.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17320508&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245098600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.32" data-doi="10.1074/jbc.M109.055996"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">B. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Structural analysis of Rtt106p reveals a DNA binding role required for heterochromatin silencing</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">285</span>: <span class="cit-fpage">4251</span>–<span class="cit-lpage">4262</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M109.055996%26rft_id%253Dinfo%253Apmid%252F20007951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyODUvNi80MjUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi81LzU1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.33" data-doi="10.1007/s10577-005-1026-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luger</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Dynamic nucleosomes</span>. <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosome%2Bresearch%2B%253A%2B%2Ban%2Binternational%2Bjournal%2Bon%2Bthe%2Bmolecular%252C%2Bsupramolecular%2Band%2Bevolutionary%2Baspects%2Bof%2Bchromosome%2Bbiology%26rft.stitle%253DChromosome%2BRes%26rft.aulast%253DLuger%26rft.auinit1%253DK.%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253D5%26rft.epage%253D16%26rft.atitle%253DDynamic%2Bnucleosomes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10577-005-1026-1%26rft_id%253Dinfo%253Apmid%252F16506092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10577-005-1026-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16506092&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235703700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.34" data-doi="10.1038/38444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mader</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richmond</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sargent</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richmond</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Crystal structure of the nucleosome core particle at 2.8 A resolution</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">389</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLuger%26rft.auinit1%253DK.%26rft.volume%253D389%26rft.issue%253D6648%26rft.spage%253D251%26rft.epage%253D260%26rft.atitle%253DCrystal%2Bstructure%2Bof%2Bthe%2Bnucleosome%2Bcore%2Bparticle%2Bat%2B2.8%2BA%2Bresolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F38444%26rft_id%253Dinfo%253Apmid%252F9305837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/38444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9305837&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XW77200038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.35" data-doi="10.1128/MCB.11.11.5710"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Mutations in SPT16/CDC68 suppress cis- and trans-acting mutations that affect promoter function in Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">5710</span>–<span class="cit-lpage">5717</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMalone%26rft.auinit1%253DE%2BA%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D5710%26rft.epage%253D5717%26rft.atitle%253DMutations%2Bin%2BSPT16%252FCDC68%2Bsuppress%2Bcis-%2Band%2Btrans-acting%2Bmutations%2Bthat%2Baffect%2Bpromoter%2Bfunction%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.11.11.5710%26rft_id%253Dinfo%253Apmid%252F1922073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxMS8xMS81NzEwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi81LzU1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.36" data-doi="10.1038/nature02538"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martens</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laprade</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Intergenic transcription is required to repress the Saccharomyces cerevisiae SER3 gene</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">429</span>: <span class="cit-fpage">571</span>–<span class="cit-lpage">574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMartens%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D429%26rft.issue%253D6991%26rft.spage%253D571%26rft.epage%253D574%26rft.atitle%253DIntergenic%2Btranscription%2Bis%2Brequired%2Bto%2Brepress%2Bthe%2BSaccharomyces%2Bcerevisiae%2BSER3%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02538%26rft_id%253Dinfo%253Apmid%252F15175754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02538&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15175754&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221767700042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.37" data-doi="10.1101/gad.1367605"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martens</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">P. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Regulation of an intergenic transcript controls adjacent gene transcription in Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2695</span>–<span class="cit-lpage">2704</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DMartens%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D19%26rft.issue%253D22%26rft.spage%253D2695%26rft.epage%253D2704%26rft.atitle%253DRegulation%2Bof%2Ban%2Bintergenic%2Btranscript%2Bcontrols%2Badjacent%2Bgene%2Btranscription%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1367605%26rft_id%253Dinfo%253Apmid%252F16291644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE5LzIyLzI2OTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzUvNTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.38" data-doi="10.1128/MCB.23.22.8323-8333.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mason</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Struhl</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The FACT complex travels with elongating RNA polymerase II and is important for the fidelity of transcriptional initiation in vivo</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">8323</span>–<span class="cit-lpage">8333</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMason%26rft.auinit1%253DP.%2BB.%26rft.volume%253D23%26rft.issue%253D22%26rft.spage%253D8323%26rft.epage%253D8333%26rft.atitle%253DThe%2BFACT%2BComplex%2BTravels%2Bwith%2BElongating%2BRNA%2BPolymerase%2BII%2Band%2BIs%2BImportant%2Bfor%2Bthe%2BFidelity%2Bof%2BTranscriptional%2BInitiation%2BIn%2BVivo%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.23.22.8323-8333.2003%26rft_id%253Dinfo%253Apmid%252F14585989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8yMi84MzIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi81LzU1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.39" data-doi="10.1038/nsmb.1903"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mayer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lidschreiber</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siebert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leike</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soding</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Uniform transitions of the general RNA polymerase II transcription complex</span>. <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1272</span>–<span class="cit-lpage">1278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DMayer%26rft.auinit1%253DA.%26rft.volume%253D17%26rft.issue%253D10%26rft.spage%253D1272%26rft.epage%253D1278%26rft.atitle%253DUniform%2Btransitions%2Bof%2Bthe%2Bgeneral%2BRNA%2Bpolymerase%2BII%2Btranscription%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.1903%26rft_id%253Dinfo%253Apmid%252F20818391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.1903&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20818391&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282563600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.40" data-doi="10.1534/genetics.111.128769"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCullough</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rawlins</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Insight into the mechanism of nucleosome reorganization from histone mutants that suppress defects in the FACT histone chaperone</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">835</span>–<span class="cit-lpage">846</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D188%26rft.spage%253D835%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.128769%26rft_id%253Dinfo%253Apmid%252F21625001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.111.128769&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21625001&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haber</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Cycloheximide-resistant temperature-sensitive lethal mutations of Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">303</span>–<span class="cit-lpage">315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D119%26rft.spage%253D303%26rft_id%253Dinfo%253Apmid%252F3294103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=3294103&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.42" data-doi="10.1371/journal.pone.0020847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berner</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holthoff</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez-Fonts</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harper</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mutant versions of the S. cerevisiae transcription elongation factor Spt16 define regions of Spt16 that functionally interact with histone H3</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e20847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMyers%26rft.auinit1%253DC.%2BN.%26rft.volume%253D6%26rft.issue%253D6%26rft.spage%253De20847%26rft.epage%253De20847%26rft.atitle%253DMutant%2Bversions%2Bof%2Bthe%2BS.%2Bcerevisiae%2Btranscription%2Belongation%2Bfactor%2BSpt16%2Bdefine%2Bregions%2Bof%2BSpt16%2Bthat%2Bfunctionally%2Binteract%2Bwith%2Bhistone%2BH3.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0020847%26rft_id%253Dinfo%253Apmid%252F21673966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0020847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21673966&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=21673966&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.43" data-doi="10.1016/S0092-8674(02)00654-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Narlikar</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kingston</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Cooperation between complexes that regulate chromatin structure and transcription</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">475</span>–<span class="cit-lpage">487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DNarlikar%26rft.auinit1%253DG.%2BJ.%26rft.volume%253D108%26rft.issue%253D4%26rft.spage%253D475%26rft.epage%253D487%26rft.atitle%253DCooperation%2Bbetween%2Bcomplexes%2Bthat%2Bregulate%2Bchromatin%2Bstructure%2Band%2Btranscription.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252802%252900654-2%26rft_id%253Dinfo%253Apmid%252F11909519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(02)00654-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11909519&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174039000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.44" data-doi="10.1093/nar/gkh922"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Donnell</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brewster</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurniawan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minard</span>  <span class="cit-name-given-names">L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Domain organization of the yeast histone chaperone FACT: the conserved N-terminal domain of FACT subunit Spt16 mediates recovery from replication stress</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">5894</span>–<span class="cit-lpage">5906</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DO%2527Donnell%26rft.auinit1%253DA.%2BF.%26rft.volume%253D32%26rft.issue%253D19%26rft.spage%253D5894%26rft.epage%253D5906%26rft.atitle%253DDomain%2Borganization%2Bof%2Bthe%2Byeast%2Bhistone%2Bchaperone%2BFACT%253A%2Bthe%2Bconserved%2BN-terminal%2Bdomain%2Bof%2BFACT%2Bsubunit%2BSpt16%2Bmediates%2Brecovery%2Bfrom%2Breplication%2Bstress%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh922%26rft_id%253Dinfo%253Apmid%252F15520471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMi8xOS81ODk0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi81LzU1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.45" data-doi="10.1007/s00438-009-0480-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Donnell</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kepkay</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barnes</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">New mutant versions of yeast FACT subunit Spt16 affect cell integrity</span>. <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">487</span>–<span class="cit-lpage">502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DO%2527Donnell%26rft.auinit1%253DA.%2BF.%26rft.volume%253D282%26rft.issue%253D5%26rft.spage%253D487%26rft.epage%253D502%26rft.atitle%253DNew%2Bmutant%2Bversions%2Bof%2Byeast%2BFACT%2Bsubunit%2BSpt16%2Baffect%2Bcell%2Bintegrity.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-009-0480-4%26rft_id%253Dinfo%253Apmid%252F19727824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-009-0480-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19727824&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.46" data-doi="10.1016/S0092-8674(00)80903-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orphanides</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">LeRoy</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luse</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinberg</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">FACT, a factor that facilitates transcript elongation through nucleosomes</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DOrphanides%26rft.auinit1%253DG.%26rft.volume%253D92%26rft.issue%253D1%26rft.spage%253D105%26rft.epage%253D116%26rft.atitle%253DFACT%252C%2Ba%2Bfactor%2Bthat%2Bfacilitates%2Btranscript%2Belongation%2Bthrough%2Bnucleosomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252980903-4%26rft_id%253Dinfo%253Apmid%252F9489704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)80903-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9489704&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071473700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.47" data-doi="10.1038/22350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orphanides</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lane</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hampsey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinberg</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The chromatin-specific transcription elongation factor FACT comprises human SPT16 and SSRP1 proteins</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">400</span>: <span class="cit-fpage">284</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DOrphanides%26rft.auinit1%253DG.%26rft.volume%253D400%26rft.issue%253D6741%26rft.spage%253D284%26rft.epage%253D288%26rft.atitle%253DThe%2Bchromatin-specific%2Btranscription%2Belongation%2Bfactor%2BFACT%2Bcomprises%2Bhuman%2BSPT16%2Band%2BSSRP1%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F22350%26rft_id%253Dinfo%253Apmid%252F10421373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/22350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10421373&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081503800049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.48" data-doi="10.1093/nar/29.9.e45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pfaffl</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A new mathematical model for relative quantification in real-time RT-PCR</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">e45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DPfaffl%26rft.auinit1%253DM.%2BW.%26rft.volume%253D29%26rft.issue%253D9%26rft.spage%253De45%26rft.epage%253De45%26rft.atitle%253DA%2Bnew%2Bmathematical%2Bmodel%2Bfor%2Brelative%2Bquantification%2Bin%2Breal-time%2BRT-PCR%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F29.9.e45%26rft_id%253Dinfo%253Apmid%252F11328886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjI5LzkvZTQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi81LzU1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.49" data-doi="10.1128/EC.05141-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pruneski</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hainer</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">K. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martens</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The Paf1 complex represses SER3 transcription in Saccharomyces cerevisiae by facilitating intergenic transcription-dependent nucleosome occupancy of the SER3 promoter</span>. <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1283</span>–<span class="cit-lpage">1294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.05141-11%26rft_id%253Dinfo%253Apmid%252F21873510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czoxMDoiMTAvMTAvMTI4MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNS81NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.50" data-doi="10.1074/jbc.R600007200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reinberg</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sims</span>  <span class="cit-name-given-names">R. J.</span>  <span class="cit-name-suffix">3rd</span>.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">de FACTo nucleosome dynamics</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">23297</span>–<span class="cit-lpage">23301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.R600007200%26rft_id%253Dinfo%253Apmid%252F16766522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvMzMvMjMyOTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzUvNTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-book" id="cit-2.5.555.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heiter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-comment"><em>Methods in Yeast Genetics: A Laboratory Course Manual</em>. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.52" data-doi="10.1128/MCB.24.23.10111-10117.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwabish</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Struhl</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Evidence for eviction and rapid deposition of histones upon transcriptional elongation by RNA polymerase II</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">10111</span>–<span class="cit-lpage">10117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DSchwabish%26rft.auinit1%253DM.%2BA.%26rft.volume%253D24%26rft.issue%253D23%26rft.spage%253D10111%26rft.epage%253D10117%26rft.atitle%253DEvidence%2Bfor%2BEviction%2Band%2BRapid%2BDeposition%2Bof%2BHistones%2Bupon%2BTranscriptional%2BElongation%2Bby%2BRNA%2BPolymerase%2BII%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.24.23.10111-10117.2004%26rft_id%253Dinfo%253Apmid%252F15542822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjExOiIyNC8yMy8xMDExMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNS81NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.53" data-doi="10.1534/genetics.104.038075"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shirra</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arndt</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The Snf1 protein kinase and Sit4 protein phosphatase have opposing functions in regulating TATA-binding protein association with the Saccharomyces cerevisiae INO1 promoter</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1957</span>–<span class="cit-lpage">1972</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D169%26rft.spage%253D1957%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.038075%26rft_id%253Dinfo%253Apmid%252F15716495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.104.038075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15716495&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sikorski</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A system of shuttle vectors and yeast host strains designed for efficient manipulation of DNA in Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D122%26rft.spage%253D19%26rft_id%253Dinfo%253Apmid%252F2659436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2659436&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989U362600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.55" data-doi="10.1093/emboj/cdg179"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simic</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindstrom</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tran</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roinick</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costa</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Chromatin remodeling protein Chd1 interacts with transcription elongation factors and localizes to transcribed genes</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1846</span>–<span class="cit-lpage">1856</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DSimic%26rft.auinit1%253DR.%26rft.volume%253D22%26rft.issue%253D8%26rft.spage%253D1846%26rft.epage%253D1856%26rft.atitle%253DChromatin%2Bremodeling%2Bprotein%2BChd1%2Binteracts%2Bwith%2Btranscription%2Belongation%2Bfactors%2Band%2Blocalizes%2Bto%2Btranscribed%2Bgenes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252Fcdg179%26rft_id%253Dinfo%253Apmid%252F12682017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo5OiIyMi84LzE4NDYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzUvNTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.56" data-doi="10.1016/j.molcel.2010.11.031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shilatifard</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The chromatin signaling pathway: diverse mechanisms of recruitment of histone-modifying enzymes and varied biological outcomes</span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">689</span>–<span class="cit-lpage">701</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DSmith%26rft.auinit1%253DE.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D689%26rft.epage%253D701%26rft.atitle%253DThe%2Bchromatin%2Bsignaling%2Bpathway%253A%2Bdiverse%2Bmechanisms%2Bof%2Brecruitment%2Bof%2Bhistone-modifying%2Benzymes%2Band%2Bvaried%2Bbiological%2Boutcomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.11.031%26rft_id%253Dinfo%253Apmid%252F21145479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.11.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21145479&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285405800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.57" data-doi="10.1093/emboj/21.7.1764"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Squazzo</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costa</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindstrom</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumer</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simic</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The Paf1 complex physically and functionally associates with transcription elongation factors in vivo</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1764</span>–<span class="cit-lpage">1774</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DSquazzo%26rft.auinit1%253DS.%2BL.%26rft.volume%253D21%26rft.issue%253D7%26rft.spage%253D1764%26rft.epage%253D1774%26rft.atitle%253DThe%2BPaf1%2Bcomplex%2Bphysically%2Band%2Bfunctionally%2Bassociates%2Bwith%2Btranscription%2Belongation%2Bfactors%2Bin%2Bvivo.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F21.7.1764%26rft_id%253Dinfo%253Apmid%252F11927560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo5OiIyMS83LzE3NjQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzUvNTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.58" data-doi="10.1371/journal.pone.0025644"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Donnell</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benjamin</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barnes</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">FACT, the Bur kinase pathway, and the histone co-repressor HirC Have overlapping nucleosome-related roles in yeast transcription elongation</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e25644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DStevens%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D6%26rft.issue%253D10%26rft.spage%253De25644%26rft.epage%253De25644%26rft.atitle%253DFACT%252C%2Bthe%2BBur%2Bkinase%2Bpathway%252C%2Band%2Bthe%2Bhistone%2Bco-repressor%2BHirC%2Bhave%2Boverlapping%2Bnucleosome-related%2Broles%2Bin%2Byeast%2Btranscription%2Belongation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0025644%26rft_id%253Dinfo%253Apmid%252F22022426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0025644&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22022426&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.59" data-doi="10.1073/pnas.0712293105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stuwe</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hothorn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lejeune</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rybin</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bortfeld</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The FACT Spt16 “peptidase” domain is a histone H3–H4 binding module</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">8884</span>–<span class="cit-lpage">8889</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0712293105%26rft_id%253Dinfo%253Apmid%252F18579787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzI2Lzg4ODQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzUvNTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.60" data-doi="10.1534/genetics.111.128645"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tomson</span>  <span class="cit-name-given-names">B. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warner</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arndt</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of a role for histone H2B ubiquitylation in noncoding RNA 3′-end formation through mutational analysis of Rtf1 in Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.128645%26rft_id%253Dinfo%253Apmid%252F21441211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzIvMjczIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi81LzU1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.61" data-doi="10.1016/j.molcel.2006.03.025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanDemark</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanksma</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heroux</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The structure of the yFACT Pob3-M domain, its interaction with the DNA replication factor RPA, and a potential role in nucleosome deposition</span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">363</span>–<span class="cit-lpage">374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DVandemark%26rft.auinit1%253DA.%2BP.%26rft.volume%253D22%26rft.issue%253D3%26rft.spage%253D363%26rft.epage%253D374%26rft.atitle%253DThe%2Bstructure%2Bof%2Bthe%2ByFACT%2BPob3-M%2Bdomain%252C%2Bits%2Binteraction%2Bwith%2Bthe%2BDNA%2Breplication%2Bfactor%2BRPA%252C%2Band%2Ba%2Bpotential%2Brole%2Bin%2Bnucleosome%2Bdeposition.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2006.03.025%26rft_id%253Dinfo%253Apmid%252F16678108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2006.03.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16678108&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237618200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.62" data-doi="10.1074/jbc.M708682200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanDemark</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCullough</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rawlins</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Structural and functional analysis of the Spt16p N-terminal domain reveals overlapping roles of yFACT subunits</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">283</span>: <span class="cit-fpage">5058</span>–<span class="cit-lpage">5068</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M708682200%26rft_id%253Dinfo%253Apmid%252F18089575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyODMvOC81MDU4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi81LzU1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.63" data-doi="10.1074/jbc.R110.180778"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winkler</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luger</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The histone chaperone FACT: structural insights and mechanisms for nucleosome reorganization</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">18369</span>–<span class="cit-lpage">18374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.R110.180778%26rft_id%253Dinfo%253Apmid%252F21454601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODYvMjEvMTgzNjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzUvNTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.64" data-doi="10.1074/jbc.M111.301465"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winkler</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muthurajan</span>  <span class="cit-name-given-names">U. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieb</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luger</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Histone chaperone FACT coordinates nucleosome interaction through multiple synergistic binding events</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">41883</span>–<span class="cit-lpage">41892</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M111.301465%26rft_id%253Dinfo%253Apmid%252F21969370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODYvNDgvNDE4ODMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzUvNTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaleff</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valent</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Mutations affecting Ty-mediated expression of the HIS4 gene of Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D107%26rft.spage%253D179%26rft_id%253Dinfo%253Apmid%252F6329902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=6329902&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984ST06000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.66" data-doi="10.1002/yea.320110107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dollard</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ricupero-Hovasse</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Construction of a set of convenient Saccharomyces cerevisiae strains that are isogenic to S288C</span>. <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DWinston%26rft.auinit1%253DF.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D53%26rft.epage%253D55%26rft.atitle%253DConstruction%2Bof%2Ba%2Bset%2Bof%2Bconvenient%2BSaccharomyces%2Bcerevisiae%2Bstrains%2Bthat%2Bare%2Bisogenic%2Bto%2BS288C.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.320110107%26rft_id%253Dinfo%253Apmid%252F7762301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.320110107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7762301&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F555.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QF61400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.67" data-doi="10.1128/MCB.17.7.4178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wittmeyer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Formosa</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The Saccharomyces cerevisiae DNA polymerase alpha catalytic subunit interacts with Cdc68/Spt16 and with Pob3, a protein similar to an HMG1-like protein</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">4178</span>–<span class="cit-lpage">4190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DWittmeyer%26rft.auinit1%253DJ.%26rft.volume%253D17%26rft.issue%253D7%26rft.spage%253D4178%26rft.epage%253D4190%26rft.atitle%253DThe%2BSaccharomyces%2Bcerevisiae%2BDNA%2Bpolymerase%2Balpha%2Bcatalytic%2Bsubunit%2Binteracts%2Bwith%2BCdc68%252FSpt16%2Band%2Bwith%2BPob3%252C%2Ba%2Bprotein%2Bsimilar%2Bto%2Ban%2BHMG1-%2Blike%2Bprotein%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.17.7.4178%26rft_id%253Dinfo%253Apmid%252F9199353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE3LzcvNDE3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNS81NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.555.68" data-doi="10.1128/MCB.00324-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wyrick</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reese</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Novel trans-tail regulation of H2B ubiquitylation and H3K4 methylation by the N terminus of histone H2A</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">3635</span>–<span class="cit-lpage">3645</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00324-10%26rft_id%253Dinfo%253Apmid%252F20498280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIzMC8xNC8zNjM1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi81LzU1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/5/555.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/5/551" title="Simplified Insertion of Transgenes Onto Balancer Chromosomes via Recombinase-Mediated Cassette Exchange" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/5/569" title="Regulated Expression of Chromobox Homolog 5 Revealed in Tumors of ApcMin/+ ROSA11 Gene Trap Mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/5"><span>Volume 2 Issue 5, May 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/5.cover-source.jpg?itok=3sppLJvG" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/5/555" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-A0wVj9V03Sq7c0wBQyKZs6dF9_iphPAn82EJceQgCkU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/5/555" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/5/555" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/5/555" class="active" data-icon-position="" data-hide-link-title="0">Identification of Mutant Versions of the Spt16 Histone Chaperone That Are Defective for Transcription-Coupled Nucleosome Occupancy in Saccharomyces cerevisiae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459281" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ReWuBWCWzhnRgupodlRRqr2OquoiagAfU00hZOkkezo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/5/555.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/5/555" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-lQnE6X9ukkQvMRDU_x6FIXpGVZaKQAfW-drEjFe7Vp8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/5/555" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/5/555.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459281" id="node459281--2" data-pisa="ggg;2/5/555" data-pisa-master="ggg;2/5/555" data-apath="/ggg/2/5/555.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification of Mutant Versions of the Spt16 Histone Chaperone That Are Defective for Transcription-Coupled Nucleosome Occupancy in <em>Saccharomyces cerevisiae</em></h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sarah J. Hainer</span>, <span class="highwire-citation-author" data-delta="1">Brittany A. Charsar</span>, <span class="highwire-citation-author" data-delta="2">Shayna B. Cohen</span> and <span class="highwire-citation-author" data-delta="3">Joseph A. Martens</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">555-567;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002451 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459281--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sarah J. Hainer</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Biological Sciences, University of Pittsburgh, Pittsburgh, Pennsylvania 15260</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BJ.%2BHainer" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hainer%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASarah%2BJ.%2BHainer" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Brittany A. Charsar</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Biological Sciences, University of Pittsburgh, Pittsburgh, Pennsylvania 15260</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brittany%2BA.%2BCharsar" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Charsar%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrittany%2BA.%2BCharsar" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shayna B. Cohen</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Biological Sciences, University of Pittsburgh, Pittsburgh, Pennsylvania 15260</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shayna%2BB.%2BCohen" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cohen%20SB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShayna%2BB.%2BCohen" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Joseph A. Martens</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Biological Sciences, University of Pittsburgh, Pittsburgh, Pennsylvania 15260</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BA.%2BMartens" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martens%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BA.%2BMartens" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:martens@pitt.edu" class="" data-icon-position="" data-hide-link-title="0">martens@pitt.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/5/555" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/5/555&amp;title=Identification%20of%20Mutant%20Versions%20of%20the%20Spt16%20Histone%20Chaperone%20That%20Are%20Defective%20for%20Transcription-Coupled%20Nucleosome%20Occupancy%20in%20Saccharomyces%20cerevisiae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/5/555&amp;title=Identification%20of%20Mutant%20Versions%20of%20the%20Spt16%20Histone%20Chaperone%20That%20Are%20Defective%20for%20Transcription-Coupled%20Nucleosome%20Occupancy%20in%20Saccharomyces%20cerevisiae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/5/555&amp;title=Identification%20of%20Mutant%20Versions%20of%20the%20Spt16%20Histone%20Chaperone%20That%20Are%20Defective%20for%20Transcription-Coupled%20Nucleosome%20Occupancy%20in%20Saccharomyces%20cerevisiae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/5/555&amp;text=Identification%20of%20Mutant%20Versions%20of%20the%20Spt16%20Histone%20Chaperone%20That%20Are%20Defective%20for%20Transcription-Coupled%20Nucleosome%20Occupancy%20in%20Saccharomyces%20cerevisiae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/5/555&amp;title=Identification%20of%20Mutant%20Versions%20of%20the%20Spt16%20Histone%20Chaperone%20That%20Are%20Defective%20for%20Transcription-Coupled%20Nucleosome%20Occupancy%20in%20Saccharomyces%20cerevisiae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/5/555&amp;t=Identification%20of%20Mutant%20Versions%20of%20the%20Spt16%20Histone%20Chaperone%20That%20Are%20Defective%20for%20Transcription-Coupled%20Nucleosome%20Occupancy%20in%20Saccharomyces%20cerevisiae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/5/555&amp;title=Identification%20of%20Mutant%20Versions%20of%20the%20Spt16%20Histone%20Chaperone%20That%20Are%20Defective%20for%20Transcription-Coupled%20Nucleosome%20Occupancy%20in%20Saccharomyces%20cerevisiae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/5/555&amp;title=Identification%20of%20Mutant%20Versions%20of%20the%20Spt16%20Histone%20Chaperone%20That%20Are%20Defective%20for%20Transcription-Coupled%20Nucleosome%20Occupancy%20in%20Saccharomyces%20cerevisiae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459281/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459281" id="node459281--2" data-pisa="ggg;2/5/555" data-pisa-master="ggg;2/5/555" data-apath="/ggg/2/5/555.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification of Mutant Versions of the Spt16 Histone Chaperone That Are Defective for Transcription-Coupled Nucleosome Occupancy in <em>Saccharomyces cerevisiae</em></h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sarah J. Hainer</span>, <span class="highwire-citation-author" data-delta="1">Brittany A. Charsar</span>, <span class="highwire-citation-author" data-delta="2">Shayna B. Cohen</span> and <span class="highwire-citation-author" data-delta="3">Joseph A. Martens</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">555-567;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002451 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459281--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sarah J. Hainer</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Biological Sciences, University of Pittsburgh, Pittsburgh, Pennsylvania 15260</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BJ.%2BHainer" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hainer%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASarah%2BJ.%2BHainer" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Brittany A. Charsar</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Biological Sciences, University of Pittsburgh, Pittsburgh, Pennsylvania 15260</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brittany%2BA.%2BCharsar" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Charsar%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrittany%2BA.%2BCharsar" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shayna B. Cohen</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Biological Sciences, University of Pittsburgh, Pittsburgh, Pennsylvania 15260</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shayna%2BB.%2BCohen" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cohen%20SB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShayna%2BB.%2BCohen" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Joseph A. Martens</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Biological Sciences, University of Pittsburgh, Pittsburgh, Pennsylvania 15260</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BA.%2BMartens" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martens%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BA.%2BMartens" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:martens@pitt.edu" class="" data-icon-position="" data-hide-link-title="0">martens@pitt.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=555&amp;rft.epage=567&amp;rft.atitle=Identification%20of%20Mutant%20Versions%20of%20the%20Spt16%20Histone%20Chaperone%20That%20Are%20Defective%20for%20Transcription-Coupled%20Nucleosome%20Occupancy%20in%20Saccharomyces%20cerevisiae&amp;rft.volume=2&amp;rft.issue=5&amp;rft.date=2012-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hainer%2C+Sarah+J.&amp;rft.au=Charsar%2C+Brittany+A.&amp;rft.au=Cohen%2C+Shayna+B.&amp;rft.au=Martens%2C+Joseph+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459281/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459281/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459281/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459281/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459281/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459281/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459281/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459281/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459281/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459281/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459281/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459281/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494625" id="node494625" data-pisa="ggg;9/11/3891" data-pisa-master="ggg;g3.119.400723" data-apath="/ggg/9/11/3891.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3891" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Mouse Mutation That Dysregulates Neighboring <em>Galnt17</em> and <em>Auts2</em> Genes Is Associated with Phenotypes Related to the Human AUTS2 Syndrome</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494624" id="node494624" data-pisa="ggg;9/11/3781" data-pisa-master="ggg;g3.119.400647" data-apath="/ggg/9/11/3781.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3781" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Transgenic Analyses in Drosophila Reveal That mCORL1 Is Functionally Distinct from mCORL2 and dCORL</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494622" id="node494622" data-pisa="ggg;9/11/3843" data-pisa-master="ggg;g3.119.400363" data-apath="/ggg/9/11/3843.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3843" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Analysis of Cell Cycle-Regulating Genes in the Symbiotic Dinoflagellate <em>Breviolum minutum</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/5/555" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/5/555.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/5/555.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/5/555.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/5/555.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2019 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
  </body>
</html>
