<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/2/223" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/2/223.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/2/223.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/2/223.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2.cover.gif" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/2/2/223/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/2/223.atom" />
<meta name="HW.pisa" content="ggg;2/2/223" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Resource of Quantitative Functional Annotation for Homo sapiens Genes" />
<meta name="DC.Identifier" content="10.1534/g3.111.000828" />
<meta name="DC.Date" content="2012-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Tasan et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The body of human genomic and proteomic evidence continues to grow at ever-increasing rates, while annotation efforts struggle to keep pace. A surprisingly small fraction of human genes have clear, documented associations with specific functions, and new functions continue to be found for characterized genes. Here we assembled an integrated collection of diverse genomic and proteomic data for 21,341 human genes and make quantitative associations of each to 4333 Gene Ontology terms. We combined guilt-by-profiling and guilt-by-association approaches to exploit features unique to the data types. Performance was evaluated by cross-validation, prospective validation, and by manual evaluation with the biological literature. Functional-linkage networks were also constructed, and their utility was demonstrated by identifying candidate genes related to a glioma FLN using a seed network from genome-wide association studies. Our annotations are presented—alongside existing validated annotations—in a publicly accessible and searchable web interface." />
<meta name="DC.Contributor" content="Murat Taşan" />
<meta name="DC.Contributor" content="Harold J. Drabkin" />
<meta name="DC.Contributor" content="John E. Beaver" />
<meta name="DC.Contributor" content="Hon Nian Chua" />
<meta name="DC.Contributor" content="Julie Dunham" />
<meta name="DC.Contributor" content="Weidong Tian" />
<meta name="DC.Contributor" content="Judith A. Blake" />
<meta name="DC.Contributor" content="Frederick P. Roth" />
<meta name="article:published_time" content="2012-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Resource of Quantitative Functional Annotation for Homo sapiens Genes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The body of human genomic and proteomic evidence continues to grow at ever-increasing rates, while annotation efforts struggle to keep pace. A surprisingly small fraction of human genes have clear, documented associations with specific functions, and new functions continue to be found for characterized genes. Here we assembled an integrated collection of diverse genomic and proteomic data for 21,341 human genes and make quantitative associations of each to 4333 Gene Ontology terms. We combined guilt-by-profiling and guilt-by-association approaches to exploit features unique to the data types. Performance was evaluated by cross-validation, prospective validation, and by manual evaluation with the biological literature. Functional-linkage networks were also constructed, and their utility was demonstrated by identifying candidate genes related to a glioma FLN using a seed network from genome-wide association studies. Our annotations are presented—alongside existing validated annotations—in a publicly accessible and searchable web interface.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/02/01" />
<meta name="citation_mjid" content="ggg;2/2/223" />
<meta name="citation_id" content="2/2/223" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/2/223" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/2/223.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/2/223.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/2/223.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.000828" />
<meta name="citation_pmid" content="22384401" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="2" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="223" />
<meta name="citation_lastpage" content="233" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Murat Taşan" />
<meta name="citation_author_institution" content="Donnelly Centre for Cellular &amp; Biomolecular Research, University of Toronto, Toronto, Ontario M5S-3E1, Canada" />
<meta name="citation_author_institution" content="Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115" />
<meta name="citation_author" content="Harold J. Drabkin" />
<meta name="citation_author_institution" content="Mouse Genome Informatics, The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author" content="John E. Beaver" />
<meta name="citation_author_institution" content="Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115" />
<meta name="citation_author" content="Hon Nian Chua" />
<meta name="citation_author_institution" content="Donnelly Centre for Cellular &amp; Biomolecular Research, University of Toronto, Toronto, Ontario M5S-3E1, Canada" />
<meta name="citation_author_institution" content="Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115" />
<meta name="citation_author" content="Julie Dunham" />
<meta name="citation_author_institution" content="Donnelly Centre for Cellular &amp; Biomolecular Research, University of Toronto, Toronto, Ontario M5S-3E1, Canada" />
<meta name="citation_author" content="Weidong Tian" />
<meta name="citation_author_institution" content="Institute of Biostatistics, School of Life Sciences, Fudan University, Shanghai 200433, P. R. China" />
<meta name="citation_author" content="Judith A. Blake" />
<meta name="citation_author_institution" content="Mouse Genome Informatics, The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author" content="Frederick P. Roth" />
<meta name="citation_author_institution" content="Donnelly Centre for Cellular &amp; Biomolecular Research, University of Toronto, Toronto, Ontario M5S-3E1, Canada" />
<meta name="citation_author_institution" content="Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115" />
<meta name="citation_author_institution" content="Center for Cancer Systems Biology, Dana Farber Cancer Institute, Boston, Massachusetts 02115" />
<meta name="citation_author_institution" content="Samuel Lunenfeld Research Institute, Mt. Sinai Hospital, Toronto, Ontario M5G-1X5, Canada" />
<meta name="citation_author_email" content="fritz.roth@utoronto.ca" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. Barrell;citation_author=E. Dimmer;citation_author=R. P. Huntley;citation_author=D. Binns;citation_author=C. O’Donovan;citation_title=The GOA database in 2009–an integrated Gene Ontology Annotation resource;citation_pages=396-403;citation_volume=37;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. Barrett;citation_author=D. B. Troup;citation_author=S. E. Wilhite;citation_author=P. Ledoux;citation_author=D. Rudnev;citation_title=NCBI GEO: archive for high-throughput functional genomic data;citation_volume=37;citation_year=2009;citation_pmid=18940857;citation_doi=10.1093/nar/gkn764" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. E. Beaver;citation_author=M. Tasan;citation_author=F. D. Gibbons;citation_author=W. Tian;citation_author=T. R. Hughes;citation_title=FuncBase: a resource for quantitative gene function annotation;citation_volume=26;citation_year=2010;citation_pmid=20495000;citation_doi=10.1093/bioinformatics/btq265" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc., B;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing;citation_pages=289-300;citation_volume=57;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. F. Berriz;citation_author=F. P. Roth;citation_title=The Synergizer service for translating gene, protein and other biological identifiers;citation_volume=24;citation_year=2008;citation_pmid=18697767;citation_doi=10.1093/bioinformatics/btn424" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. Bieri;citation_author=D. Blasiar;citation_author=P. Ozersky;citation_author=I. Antoshechkin;citation_author=C. Bastiani;citation_title=WormBase: new content and better access;citation_volume=35;citation_year=2007;citation_issue=Database issue;citation_pmid=17169986;citation_doi=10.1093/nar/gkl1076" />
<meta name="citation_reference" content="citation_journal_title=Cancer Research;citation_journal_abbrev=Cancer Res.;citation_author=M. Bredel;citation_author=C. Bredel;citation_author=D. Juric;citation_author=G. R. Harsh;citation_author=H. Vogel;citation_title=Functional Network Analysis Reveals Extended Gliomagenesis Pathway Maps and Three Novel MYC-Interacting Genes in Human Gliomas;citation_pages=8679-8689;citation_volume=65;citation_year=2005;citation_issue=19;citation_pmid=16204036;citation_doi=10.1158/0008-5472.CAN-05-1204" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=L. Breiman;citation_title=Random forests;citation_pages=5-32;citation_volume=45;citation_year=2001;citation_issue=1;citation_doi=10.1023/A:1010933404324" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. A. Crosby;citation_author=J. L. Goodman;citation_author=V. B. Strelets;citation_author=P. Zhang;citation_author=W. M. Gelbart;citation_title=FlyBase: genomes by the dozen;citation_volume=35;citation_year=2007;citation_issue=Database issue;citation_pmid=17169988;citation_doi=10.1093/nar/gkl1073" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Dai;citation_author=P. Wang;citation_author=A. D. Boyd;citation_author=G. Kostov;citation_author=B. Athey;citation_title=Evolving gene/transcript definitions significantly alter the interpretation of GeneChip data;citation_pages=e175-e175;citation_volume=33;citation_year=2005;citation_issue=20;citation_pmid=16284200;citation_doi=10.1093/nar/gni179" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=M. Deng;citation_author=T. Chen;citation_author=F. Sun;citation_title=An integrated probabilistic model for functional prediction of proteins.;citation_pages=463-475;citation_volume=11;citation_year=2004;citation_issue=2–3;citation_pmid=15285902;citation_doi=10.1089/1066527041410346" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Mouse Genome Informatics Group;citation_title=The mouse genome database (mgd): new features facilitating a model system;citation_volume=35;citation_year=2007;citation_issue=Database issue;citation_pmid=17135206;citation_doi=10.1093/nar/gkl940" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=K. C. Gunsalus;citation_author=W. C. Yueh;citation_author=P. MacMenamin;citation_author=F. Piano;citation_title=RNAiDB and PhenoBlast: web tools for genome-wide phenotypic mapping projects;citation_pages=D406-410;citation_volume=32;citation_year=2004;citation_issue=90001;citation_pmid=14681444;citation_doi=10.1093/nar/gkh110" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. C. Gunsalus;citation_author=H. Ge;citation_author=A. J. Schetter;citation_author=D. S. Goldberg;citation_author=J. D. Han;citation_title=Predictive models of molecular machines involved in Caenorhabditis elegans early embryogenesis.;citation_pages=861-865;citation_volume=436;citation_year=2005;citation_issue=7052;citation_pmid=16094371;citation_doi=10.1038/nature03876" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Hamosh;citation_author=A. F. Scott;citation_author=J. S. Amberger;citation_author=C. A. Bocchini;citation_author=V. A. McKusick;citation_title=Online Mendelian Inheritance in Man (OMIM), a knowledgebase of human genes and genetic disorders;citation_pages=514-517;citation_volume=33;citation_year=2005;citation_issue=Database issue;citation_doi=10.1093/nar/gki033" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. Huttenhower;citation_author=E. M. Haley;citation_author=M. A. Hibbs;citation_author=V. Dumeaux;citation_author=D. R. Barrett;citation_title=Exploring the human genome with functional maps;citation_volume=19;citation_year=2009;citation_pmid=19246570;citation_doi=10.1101/gr.082214.108" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=C. Huttenhower;citation_author=O. Hofmann;citation_title=A quick guide to large-scale genomic data mining.;citation_pages=e1000779-e1000779;citation_volume=6;citation_year=2010;citation_issue=5;citation_pmid=20523745;citation_doi=10.1371/journal.pcbi.1000779" />
<meta name="citation_reference" content="citation_journal_title=Omics :  a journal of integrative biology;citation_journal_abbrev=OMICS;citation_author=T. Joshi;citation_author=Y. Chen;citation_author=J. M. Becker;citation_author=N. Alexandrov;citation_author=D. Xu;citation_title=Genome-scale gene function prediction using multiple sources of high-throughput data in yeast Saccharomyces cerevisiae.;citation_pages=322-333;citation_volume=8;citation_year=2004;citation_issue=4;citation_pmid=15703479;citation_doi=10.1089/omi.2004.8.322" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=U. Karaoz;citation_author=T. M. Murali;citation_author=S. Letovsky;citation_author=Y. Zheng;citation_author=C. Ding;citation_title=Whole-genome annotation by using evidence integration in functional-linkage networks;citation_volume=101;citation_year=2004;citation_pmid=14981259;citation_doi=10.1073/pnas.0307326101" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. S. Keshava Prasad;citation_author=R. Goel;citation_author=K. Kandasamy;citation_author=S. Keerthikumar;citation_author=S. Kumar;citation_title=Human Protein Reference Database–2009 update;citation_volume=37;citation_year=2009;citation_pmid=18988627;citation_doi=10.1093/nar/gkn892" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=O. D. King;citation_author=J. C. Lee;citation_author=A. M. Dudley;citation_author=D. M. Janse;citation_author=G. M. Church;citation_title=Predicting phenotype from patterns of annotation;citation_pages=183-189;citation_volume=19;citation_year=2003;citation_issue=Suppl 1;citation_doi=10.1093/bioinformatics/btg1024" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=S. Ko;citation_author=H. Lee;citation_title=Integrative approaches to the prediction of protein functions based on the feature selection.;citation_pages=455-455;citation_volume=10;citation_year=2009;citation_pmid=20043848;citation_doi=10.1186/1471-2105-10-455" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. R. G. Lanckriet;citation_author=T. De Bie;citation_author=N. Cristianini;citation_author=M. I. Jordan;citation_author=W. S. Noble;citation_title=A statistical framework for genomic data fusion;citation_volume=20;citation_year=2004;citation_pmid=15130933;citation_doi=10.1093/bioinformatics/bth294" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=I. Lee;citation_author=S. V. Date;citation_author=A. T. Adai;citation_author=E. M. Marcotte;citation_title=A Probabilistic Functional Network of Yeast Genes;citation_pages=1555-1558;citation_volume=306;citation_year=2004;citation_issue=5701;citation_pmid=15567862;citation_doi=10.1126/science.1099511" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=I. Lee;citation_author= B. Lehner;citation_author=T. Vavouri;citation_author= J. Shin;citation_author= A. G. Fraser;citation_author=E. M. Marcotte;citation_title=Predicting genetic modifier loci using functional gene networks;citation_volume=20;citation_year=2010;citation_pmid=20538624;citation_doi=10.1101/gr.102749.109" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=I. Lee;citation_author=U. M. Blom;citation_author=P. I. Wang;citation_author=J. E. Shim;citation_author=E. M. Marcotte;citation_title=Prioritizing candidate disease genes by network-based boosting of genome-wide association data;citation_volume=21;citation_year=2011;citation_pmid=21536720;citation_doi=10.1101/gr.118992.110" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Letovsky;citation_author=S. Kasif;citation_title=Predicting protein function from protein/protein interaction data: a probabilistic approach;citation_pages=197-204;citation_volume=19;citation_year=2003;citation_issue=Suppl 1;citation_doi=10.1093/bioinformatics/btg1026" />
<meta name="citation_reference" content="citation_journal_title=Future Oncol.;citation_author=Y. Li;citation_author=G. Bu;citation_title=LRP5/6 in Wnt signaling and tumorigenesis;citation_pages=673-681;citation_volume=1;citation_year=2005;citation_pmid=16556044;citation_doi=10.2217/14796694.1.5.673" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Linghu;citation_author=E. S. Snitkin;citation_author=Z. Hu;citation_author=Y. Xia;citation_author=C. DeLisi;citation_title=Genome-wide prioritization of disease genes and identification of disease-disease associations from an integrated human functional linkage network.;citation_pages=R91-R91;citation_volume=10;citation_year=2009;citation_issue=9;citation_pmid=19728866;citation_doi=10.1186/gb-2009-10-9-r91" />
<meta name="citation_reference" content="citation_journal_title=Current cancer drug targets;citation_journal_abbrev=Curr Cancer Drug Targets;citation_author=H. H. Luu;citation_author=R. Zhang;citation_author=R. C. Haydon;citation_author=E. Rayburn;citation_author=Q. Kang;citation_title=Wnt/beta-catenin signaling pathway as a novel cancer drug target.;citation_pages=653-671;citation_volume=4;citation_year=2004;citation_issue=8;citation_pmid=15578921;citation_doi=10.2174/1568009043332709" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=N. J. Mulder;citation_author=R. Apweiler;citation_author=T. K. Attwood;citation_author=A. Bairoch;citation_author=A. Bateman;citation_title=A large-scale analysis of mRNA polyadenylation of human and mouse genes;citation_pages=201-212;citation_volume=33;citation_year=2005;citation_issue=Database issue;citation_pmid=15647503;citation_doi=10.1093/nar/gki158" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=T. M. Murali;citation_author=C. J. Wu;citation_author=S. Kasif;citation_title=The art of gene function prediction.;citation_pages=1474-1475;citation_volume=24;citation_year=2006;citation_issue=12;citation_pmid=17160037;citation_doi=10.1038/nbt1206-1474" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. Nash;citation_author=S. Weng;citation_author=B. Hitz;citation_author=R. Balakrishnan;citation_author=K. R. Christie;citation_title=Expanded protein information at SGD: new pages and proteome browser;citation_pages=468-471;citation_volume=35;citation_year=2007;citation_issue=Database issue;citation_doi=10.1093/nar/gkl931" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. P. O’Brien;citation_author=M. Remm;citation_author=E. L. L. Sonnhammer;citation_title=Inparanoid: a comprehensive database of eukaryotic orthologs;citation_pages=476-480;citation_volume=33;citation_year=2005;citation_issue=Database issue;citation_doi=10.1093/nar/gki107" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=L. Pena-Castillo;citation_author=M. Tasan;citation_author=C. L. Myers;citation_author=H. Lee;citation_author=T. Joshi;citation_title=A critical assessment of Mus musculus gene function prediction using integrated genomic evidence;citation_pages=S2;citation_volume=9;citation_year=2008;citation_issue=Suppl 1" />
<meta name="citation_reference" content="citation_journal_title=J. Neuro-oncol.;citation_author=D. Piekutowska-Abramczuk;citation_author= E. Ciara;citation_author= E. Popowska;citation_author=W. Grajkowska;citation_author= B. Dembowska-Bagiska;citation_title=The frequency of NBN molecular variants in pediatric astrocytic tumors;citation_pages=161-168;citation_volume=96;citation_year=2009;citation_pmid=19629396" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. Pujana;citation_author=J. D. Han;citation_author=L. M. Starita;citation_author=K. N. Stevens;citation_author=M. Tewari;citation_title=Network modeling links breast cancer susceptibility and centrosome dysfunction.;citation_pages=1338-1349;citation_volume=39;citation_year=2007;citation_issue=11;citation_pmid=17922014;citation_doi=10.1038/ng.2007.2" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=B. Schwikowski;citation_author=P. Uetz;citation_author=S. Fields;citation_title=A network of protein-protein interactions in yeast.;citation_pages=1257-1261;citation_volume=18;citation_year=2000;citation_issue=12;citation_pmid=11101803;citation_doi=10.1038/82360" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. Shete;citation_author=F. J. Hosking;citation_author=L. B. Robertson;citation_author=S. E. Dobbins;citation_author=M. Sanson;citation_title=Genome-wide association study identifies five susceptibility loci for glioma.;citation_pages=899-904;citation_volume=41;citation_year=2009;citation_issue=8;citation_pmid=19578367;citation_doi=10.1038/ng.407" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=C. L. Smith;citation_author=C.-A. W. Goldsmith;citation_author=J. T. Eppig;citation_title=The Mammalian Phenotype Ontology as a tool for annotating, analyzing and comparing phenotypic information.;citation_pages=R7-R7;citation_volume=6;citation_year=2005;citation_issue=1;citation_pmid=15642099;citation_doi=10.1186/gb-2005-6-5-p7" />
<meta name="citation_reference" content="citation_journal_title=Journal of bioinformatics and computational biology;citation_journal_abbrev=J Bioinform Comput Biol;citation_author=A. Sokolov;citation_author=A. Ben-Hur;citation_title=Hierarchical classification of gene ontology terms using the GOstruct method.;citation_pages=357-376;citation_volume=8;citation_year=2010;citation_issue=2;citation_pmid=20401950;citation_doi=10.1142/S0219720010004744" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. Szklarczyk;citation_author=A. Franceschini;citation_author=M. Kuhn;citation_author=M. Simonovic;citation_author=A. Roth;citation_title=The STRING database in 2011: functional interaction networks of proteins, globally integrated and scored;citation_volume=39;citation_year=2011;citation_pmid=21045058;citation_doi=10.1093/nar/gkq973" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. Tasan;citation_author=W. Tian;citation_author=D. P. Hill;citation_author=F. D. Gibbons;citation_author=J. A. Blake;citation_title=An en masse phenotype and function prediction system for Mus musculus;citation_pages=S8;citation_volume=9;citation_year=2008;citation_issue=Suppl 1" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=W. Tian;citation_author=L. V. Zhang;citation_author=M. Taan;citation_author=F. D. Gibbons;citation_author=O. D. King;citation_title=Combining guilt-by-association and guilt-by-profiling to predict Saccharomyces cerevisiae gene function;citation_pages=S7;citation_volume=9;citation_year=2008;citation_issue=Suppl 1" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=O. G. Troyanskaya;citation_author=K. Dolinski;citation_author=A. B. Owen;citation_author=R. B. Altman;citation_author=D. Botstein;citation_title=A Bayesian framework for combining heterogeneous data sources for gene function prediction (in Saccharomyces cerevisiae);citation_volume=100;citation_year=2003;citation_pmid=12826619;citation_doi=10.1073/pnas.0832373100" />
<meta name="citation_reference" content="citation_journal_abbrev=J Proteomics;citation_author=P. I. Wang;citation_author=E. M. Marcotte;citation_title=It's the machine that matters: Predicting gene function and phenotype from protein networks.;citation_pages=2277-2289;citation_volume=73;citation_year=2010;citation_issue=11;citation_pmid=20637909;citation_doi=10.1016/j.jprot.2010.07.005" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. L. Wong;citation_author=L. V. Zhang;citation_author=A. H. Y. Tong;citation_author=Z. Li;citation_author=D. S. Goldberg;citation_title=Combining biological networks to predict genetic interactions;citation_volume=101;citation_year=2004;citation_pmid=15496468;citation_doi=10.1073/pnas.0406614101" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=Z. Wu;citation_author=R. A. Irizarry;citation_author=R. Gentleman;citation_author=F. Martinez-Murillo;citation_author=F. Spencer;citation_title=A model-based background adjustment for oligonucleotide expression arrays;citation_pages=909-917;citation_volume=99;citation_year=2004;citation_doi=10.1198/016214504000000683" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Resource of Quantitative Functional Annotation for Homo sapiens Genes" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2.cover.gif" />
<meta name="twitter:description" content="The body of human genomic and proteomic evidence continues to grow at ever-increasing rates, while annotation efforts struggle to keep pace. A surprisingly small fraction of human genes have clear, documented associations with specific functions, and new functions continue to be found for characterized genes. Here we assembled an integrated collection of diverse genomic and proteomic data for 21,341 human genes and make quantitative associations of each to 4333 Gene Ontology terms. We combined guilt-by-profiling and guilt-by-association approaches to exploit features unique to the data types. Performance was evaluated by cross-validation, prospective validation, and by manual evaluation with the biological literature. Functional-linkage networks were also constructed, and their utility was demonstrated by identifying candidate genes related to a glioma FLN using a seed network from genome-wide association studies. Our annotations are presented—alongside existing validated annotations—in a publicly accessible and searchable web interface." />
<meta name="og-title" property="og:title" content="A Resource of Quantitative Functional Annotation for Homo sapiens Genes" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/2/223" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The body of human genomic and proteomic evidence continues to grow at ever-increasing rates, while annotation efforts struggle to keep pace. A surprisingly small fraction of human genes have clear, documented associations with specific functions, and new functions continue to be found for characterized genes. Here we assembled an integrated collection of diverse genomic and proteomic data for 21,341 human genes and make quantitative associations of each to 4333 Gene Ontology terms. We combined guilt-by-profiling and guilt-by-association approaches to exploit features unique to the data types. Performance was evaluated by cross-validation, prospective validation, and by manual evaluation with the biological literature. Functional-linkage networks were also constructed, and their utility was demonstrated by identifying candidate genes related to a glioma FLN using a seed network from genome-wide association studies. Our annotations are presented—alongside existing validated annotations—in a publicly accessible and searchable web interface." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2.cover.gif" />
    <title>A Resource of Quantitative Functional Annotation for Homo sapiens Genes | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q0wst1" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__KPgiwEdjo5O9HdRryHqQs1TpsV-PeBb9B7sGBvwM9FQ__opzZWgPNgQzoEgfATRS5qCKwBw_6XcNjDHoDVOzQv8M__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__YUqwBlv6bNoCToiBWgAUSPtppdS8sbGfzmAH3qEDcI0__Mvk93h3ScFH5Xi9WhibVKlf368fHw-V7HWqWX2QMvtw__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XNWYaQubFGWYWZNytUHkWs-b34LV21DsB4s_6gchI4k__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__7cpBA0GVdrMqkZGGZEW-_SPF2HrbTCmVg9-lrP5l_AQ__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__TV2D4R18tPiSIOmeQ3vyMqHObN_FDmQ6EEIrfE5sAoA__c9BBZFy1BrgGgF_Auu9Fmysb3Z7J56eJGYvMcLQrLks__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q0wst1"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__OaHnYyci68QTwctdRYR30kEfu0eYWBrLyMqy8THv5Ik__IyfJQYM1q1QEPwYsK_Eslh2RQkxpy6yo-nR0p4DWPQQ__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__fhq_kmKS3zjIxlsAa3GE1ubLajyXUgXXTXULzAtiqfg__Lm1_Ls-bdpbWRyl0q4o0mmHyxB-139AJ68dE4zOskLQ__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"S4Sxg9geF3TfTAMZiaTV0yU-x2tmoq2sIpvEpJ1HHao","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459234","apath":"\/ggg\/2\/2\/223.atom","pisa":"ggg;2\/2\/223","ac":{"\/ggg\/2\/2\/223.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/2\/223.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/2\/223"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/2\/223"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/2\/223":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Ae6Cxt\\\\\\\/B5lZhVAwJPPSEUBC4o6hlkGijoK4NkHoP162IW2jcmo311gNR0CXgW+Q0stjtcuVholQVpY5FRnhsQBrB7dYstp\\\\\\\/4rxhiWqBbv67A\\\\\\\/5XmWHnsjZ0XwSJdCdFEbM0+eugFyqws4Ta7+3qCcm\\\\\\\/+8AUJ6+cZmeOIq92beJzbunx1S\\\\\\\/G3FYu4+4YNhT91WiH8nF4g+Wp3kTmpT6zV9LYJR9F8CzSKBWZiFegDQQYDFY9Iro+x6ojHoIeezg5EzwYiJbYTFykOPNgeOOsKcTb4FTdYvAA5pXHB32JYhPEN2RBkA3sfGE+ehD+7+xD38SMVTL7tc\\\\\\\/q63Hg6wUWY\\\\\\\/r9EZM85jhyRBX07s\\\\\\\/vP9B1p1ub4EFdAqAWY+Jgfs1iyoRMYWHyOQ8xu9j0bmPbvMmCvQPCwSxLkfumQiQuYOj1HmxQCSCgyaPKjOHPYYGSQegabMBoWIWcLRVHBs0jR8aNOV0ObgUkb8cEZ0YloDSj9keSFXeRN\\\\\\\/\\\\\\\/GdQz1nLZT\\\\\\\/9Oo0yCnV61bZQ5HF9nF3TKcXCV0UiUEdWZAy1pzlVAruMRBtjzq3y092qRFSzvtii85nzuWZAcZlE62rrVSCKWeqN0OXEJKfymxexxqvfO0yMTQl6GQ702ys2bdosjArSqrF0TPsprqS1Nyl1xyU+kSZCXVzVTo0A+ZYurEQ2eupMpUPwE5MFjOt7LB7ZdvZ\\\\\\\/UFWlNfrZ6ovJYIEHbVUX513npPv4bOU8cw5PE154KE2DwdM2Ttvz56CZ6MZaDW\\\\\\\/kYxcg7yYJkbSzKtNL9Y9wcme2ZaB0KCz3j76pwsUuzUQhTAJfx79ntiWghP2YQkchyV+J2fQL31pJP1GF2ltA3MyPm3Ut8bvGUnl6WI7Vkx3UQbPNeYDbSwljP\\\\\\\/5CO3HRtYWj2tqn3\\\\\\\/OkAHPj9pZuSL5qdY5vsZUEkRLQB6CoOPkLjn4BgNsf3e6GJB+Mjq07FUBjDu73JjAAZwz1rRXJFWRxf2qt246kxHK4SsyFSrNMipDTf1mnIUPGG5Mh0rzSOK5cwdgZGk5GAaBy+6HdILn7BT6v36OoxIo7IQRn397XCCVYLPR3nqYHPyZ1eM\\\\\\\/1ryPs9gwxjR1HCAch5uDeOAlE78Krz25giF93vCcvKHn3Gr0dfCQq6yXHTQSKqS0nmjKtVk8bzXWj\\\\\\\/gCcLwMA07trkssJvDcYF00OnIW4H6tclGIIDh3pydvm4hTSjN+qXJGNMyMD2QPc+IVaFNv9SQ31w\\\\\\\/dYrgO2kDl81KF0\\\\\\\/SJMT7TuUxVFaS9JF0mzn\\\\\\\/B\\\u0022,\\\u0022iv\\\u0022:\\\u0022DXMVa1RjNYlhTiykLpgp9w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002220ae5a54af2219902d91484e127de1b81f1412f312c4f05c2921013d8afd1b7f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022gcCsJW1MKZogBFuUex7dVrnpNgY4cH1vuc6eeuFhVPGV5OCIWjP8fAkyxqjtZDF0GcgiLifyavNd90YisVbJALfzZsr74Qu6MEfnQSFclu3LDR1TnM14hpS6lKP6w3MW+owG8xwd1OwUaCl0VlbDUlXFZpX9Ydx5vMhlXMqIKlil5WXNCp97MkkzYOoC8HFn1CEZ6WUxND6PKqQ5X6UrDdIb0dCAJMyiPAcr6h1IsavKpFjma\\\\\\\/YYpfbw0taBQlHjI2cV2D9SxrVqgr0KHUvfSkmNYsK6uiPmHE4Ju9h72xVxl0JzayerpAd6qLKbw1XhdgKE8MHwcV+LlUQiZxYqHy\\\\\\\/rp1mniSwHjU0Y98Ho2UqbbL5ElMYHcG2X90S98GSKJcvCj\\\\\\\/om6mLX10kBawKXkCr3\\\\\\\/yOGkmHSHY1hNnQXN2yn7W1nYJUt1p9GgzzzX23i\\\\\\\/DsW2BnwpqOlPt67ZbnSxS6yJBD5g3hRGoKSs8DHoA+Q2h3ImkWBzDqLsh+udjFgz2mPQ+fObUvSDmHro94wdZNOZMpX+ERlkhDBY+Mc8d0Gm\\\\\\\/eJg5eSkq5NGieqztEDHQpJSwcK27UEn6bjrMjKNOhtVhpkr9ZUZXe6xR0oRvdAiGC99\\\\\\\/ivXwgqnQa0I7gbfknzRRtuGc\\\\\\\/3jSME0ZangnvPsNUXBxtutDK7joD\\\\\\\/z30JhfWuw6dpXSJCA9rok3Jv9f0MU4dtLypU3FnorOxqX9BWl6TMG7H2qTY04NOqwS+uINQWJO6ctGWZcnpGyLI7OngE8b+ts+7eI278AXJeh8JU8htPmPYUvHDEqJT75SqeSirQouWjPpwXE+9PPO3i8gdi+k5O9hsgkmgircGmcaVDtUUouo\\\\\\\/vszvdlNGRF0hWzazTJKa\\\\\\\/6\\\\\\\/uHaMki8e4v4FWbPYI14AIh+xboe6Szmnnj6rTfjZL3m\\\\\\\/fKpsauSx1Hq08tVEexEgDgrBOwqT7y\\\\\\\/bXJCs492KY1GvUJNq38KhqhwCidPO5MKjL0EM4Q91u7+3Po\\\\\\\/CHcxtdmxjhSNjWyAAOa09av3vy7ZgjQ4EQcf0EWmEcfQs\\\\\\\/KP\\\\\\\/QyceJGuBA5yM+SRI48nqeSWG8Hec73U5xJyPpp9y93c8yTE7Ob+V1klYrirsnpB13d2NpGmbg2UgEqfNGncrS8GBvFp\\\\\\\/zgWehoNvfsVgpQeFm3waPLMBQAmTpOmiekpZ5h3y33f6cAWLGs3GatvrFCd44Yi29NeDPC\\\\\\\/dm\\\\\\\/BvtCX2IgVh9zZejA8ddV4ckirpS9UP7i9dDyxn725vHR3jidTfbexT47OrYtp+PtggT5PqNgj8xBJf2wrvg+Q7CjvbcEkDG6dcR27zygzS0YWgApE9DooRgoV2QbmRjetUqPPld7P3k1uawLcPNfLsBPr909aC2LeZodiW2\\\\\\\/nvASbw8\\\\\\\/cSFgNcrxfeVXQPkiuGCK0Xqkufan3cg9n7T7sMgb3yrx+QxM9iVvcSYU+2VMb2yXe4vHy5eB3G+Zs90Q+YGhfvo+B9nHWnh\\\\\\\/MWN4eiDCW89Q04PtPm\\\\\\\/YXSn41byrf2VH7CtYlschfABhs\\\\\\\/pAPr4oh1RSayxlx9aR4mz7PTawmTnI4aOzVjRjYgi0XIHE0gc1Oa5djQhPhxcrTeVoYeGAKz8dGTruXDvs9CBKN9E9v9vgmXbNh2hkO0CIPh9k6evM\\\\\\\/XaQvRqD2aEM0sCQfrtrbm0EAlAt6CS5YxrfCDdvhpnfzuA16YR91uBwi63M84ez8i\\\\\\\/DbOoBtT4KX1n0V3yI4MNVnZMqTdhPAwFKqJq8jtN3hTcU8XbpIo8cptid8SXOpKiv4ah3omlTHChEnehT3fYGEvZKdg5uYLmnaKkjT6\\\\\\\/jAH8iFyBIW7eM9292Sts3TbdxRYXvHcmuBmJLBXexgo7\\\\\\\/NSsbc1j1NcTDtgUnwdZL37u5cRrEnrgQOBjprjrHwkGnxxY440S7pUP5fnUgwDs8h5V1ImbSz7q\\\\\\\/OahjFcj572ATyBFxMAY22uKJR8yaqP3f\\\\\\\/9Gh1A0viApZCQifUj4QWcXpSWMdNkL\\\\\\\/ukSnTA2zvHdV24o3TylrLkFYFbuB7ACP7nR9Iy2k6rBM3RYxnqYS+yK5F0ff62m12GY+BXwqTrLzOTCWb4mG8IisXW4iHS0koSkVPA9OYelbE1m0\\\\\\\/Mo1qh+coclhJPZRpYyKaE\\\\\\\/31aZbrTExKVNH8iyVRHcUsonQh7FwTq0XvTRNYg3D0EFbGfeVtgpTbbEYFAjjs6wkO86gxLKYAbcd+nrcKehDZfVc83DtJfek2jU1WFGWpj7jCcBC8f+oyOxWRRc77bM54LXo9U8eKaSpoI+AbtHRVVeI\\\\\\\/GqFVJYIVl0O2xSG9vwOcxHpD7ywKPxrglgrFhgtasvZpY85ixnSB9E0dfPXUEOKQWFdhBZkzWTkku+9dHvht1VSj27ZYA4JjD1CmRtSC\\\\\\\/kl+RpbjPmqjFA4BOn29MSy10TuEXU9K\\\\\\\/qmLzl4CPueoT3V6i2WsvmM7msdXtELt1JqyyxdEZIuaDuAym67EmtYcS2DTNecRYASn5pz5Ay\\\\\\\/LPDpukGNbFuxFlxzNJLO0QEHBH\\\\\\\/hqmO6IsOE5Aim6CXc9YkwgYDmbrpZhi14eUfIjYrPASDLcNctuUEt+ELL2IRwZTY9XlhRA4oQTS0iOnwNlvaCmCrxX8U28dPAkONd32qswPXY63qL\\\\\\\/ScGrV6WMzt1fD+HYoeC6Qt2l+DARzycATbCgdYyCKlY\\\\\\\/8YPW27U0dPhPPeOJGk14dFP6R4zXsKJutzd81WTvGWryMLGA10ZSTh5shC6pyL\\\\\\\/zM8XCfVkHWFfdtwDaAk2WzecuGLEieAuSM6wNMaWXW7L959NfP9BxjLmDZUbD6VPbqkxn5SOfQt82VmyF7kGfeZOs2hKP98p+5vFtgSiktYHpKeNX4KRpb8ER7hL9wR3BffIsrGNK6stzt1eY8FNer8I53ZOVZexrBhNH14Hd4s6jU\\\\\\\/qmKHO\\\\\\\/8kfBPukxxUxZMFdBp72IiFvd3zrUa9k4pVkUsO+jMIsMNregjcW7ZPQfSukqH5GmsW7G\\\\\\\/o3WWV9\\\\\\\/l9cgjtEXgdMqL4Cd7TDc+Ch2rRp\\\\\\\/7VtKj9EH1kwiNhHIEJXj9GAI+STRdt4oDPE9\\\\\\\/hK8tq6t6xUYFsZ\\\\\\\/I5fH3wFqgn\\\\\\\/1oGq3lNdB2rhmg932EdiZ0g2ahsJoVrdMgCPfg7CkAt83Fvbk+thLqy095E9\\\\\\\/cCDn+H53gKxgkEHez9C4zZhB5aRC2eqnpPjcTRUHB7G7xbXsb1SLBIoXoa+rORdNzD+N2nUpanLqWJHA3t9NrRd1QIHrjnw+NYBiKBj4Dw1g1+UQsQQWz7SNTlALFZSr3g9Y5U2mkffA+4\\\\\\\/Haiu6sik0kV5LaY27ucnysYCfqMqR7reCetADiVFmOnU+CHenkhxWd44XrMUQdvQFm+ajYvDwe7qINKdYBSsZTafdz2OFXsNtdQjSuks4k2KPXXDhcjKCF6n2K2z8\\\\\\\/03mLLNYiH3yPSBe7QkZQRWFcVp9fNrIitugepEdmkNWDbsj6+zwx7gK5Kot8vVpw2OtAwgxca72hzp16KdHgGt3AG6kXIBXrNNGrOvdwJOpFrtKcOCsvxhoJv\\\\\\\/10OxF791h+ZQud5GJsIvMnT91l2iqHGYhHr2gYN6\\\\\\\/AB01VUv9uP0YngmLcoig94rZnnWZC2c9Ccn6X1dWzd8EOtftUQS\\\\\\\/r0brGEQrwYKEXhGj48epbZm8\\\\\\\/jlDh\\\\\\\/9cYityHccbKXT4xH32rgmaxcP69JkUuF0JKstaMMj9c+KijfZEbdqwPsxv0b4hEyKQ+QMUOZwW3mWazxNRtBbCxioNlxhqq+TaTR0jhRtnSljvENbHy2IsWsKWHUscStiszY6eXAaKOs5LfxxpjOQrRYPniP106Vh4jSjhFJLhTwvW65Y3v15gcJNpqc1R0Up8LJPv\\\\\\\/zpWGZrfYJdaTH\\\\\\\/JhrxcRV8eB1cnDVJsgTIOckAA4bDskfT1+zxWKyQOaWbjWzTDcMWES\\\\\\\/ZGXiq44czuwB3S48AUt6Wj8I\\\\\\\/bjjzbRuWtJ2n5e\\\\\\\/mWFOQUy40ZEVXb98VXfipJeR60JNzBc9nRHHe\\\\\\\/Whkz6oSCtYcP53eeMa4QUqnjjRjpIvSy\\\\\\\/5QTgkJ00GqzVUoxi5U2SFQY4fFe\\\\\\\/pGVUXCPbnAbc1u9A2g74kgOvt+WcAKKFktmY7LjTujq7P+L+uR6m4X0Am60nPAipBbScnp\\\\\\\/ifsPJhUncGOaZT0XP5ebT3Tzv58vGlAHm0vEP9EBAfc8farmjAOfeJ9AZyw3U3jzVrCuPFOkngj55ZmCWGe6oEr5sT8S\\\\\\\/gKs1oR54bzW\\\\\\\/ZAD46ONY7uC1RK3hUEU48fWRzsj4pBcNU0clUhgy5AqIqxIBVW5I1uw3\\\\\\\/5tvYfbGWp+bFFJAWuWO27cDWbjcyHeOFzdR26aekI+s9wNXlS4GarIFTcVxHuQtVSYnybjmRwgIxeEoYhZm89P+n3ipn2BNXgM\\\\\\\/wE0umxxDztKbuhffXWQuCi9TZqAzS8SW2fXrGqqEOJKTY5fGMtaL231QYNIBGCLJ4tElKXTu7yp1cpvQbFl29dWxYjJYqZ+CwFCWDmHGoWpov+kmgt4QeUG1owCeo2TGVErtnjdRP1yv\\\\\\\/P0YpEQYlOsAfPL9H077c1kxFpy9vMNv04Tbj8agmNKEXusRZ0CCJcbAwM+Ts14F2RnjCtRjbGnETcRO4F4vpfPx0vwKOSZqmkPipNfzsP3mAYUicbIfywb+lJbJUpt96BLE7nic6Et+2oxpzh9dXQs\\\\\\\/QDkp2NH6cpCySgB3PAq2zxFYu4mrvfywrCXJMDHAXddGZApHc+kkkKZYShJTcvmzNDNDSN6OkI5zV7bwgAmkmMZmrdpSDWTaCdG+XAF89IlXOjlPZcFhnwupDusmQE35gbPM0kNUbIuGiVVaH7XRo++KExpSPn48SeWTid8KtU85MHxb834S6LBBA9ok3xblUy27TStUGcDFKc94fpwLkWXvtlNLX5Scv9ReayfgDP9azAbqiG7HHSXaB1psG5wQc77xuK5BXDjBvaEuwQEJW44Xlo0oUBrjp3wwRR3MS\\\\\\\/0yMEd\\\\\\\/JjFhqKPtMAgkzQnm2S1aFT1j64TKohDGGl2bVksBZRh6Zy7fUIwA0a+7W1WR2hSFUNpny8sUyhRW\\\\\\\/j3PdjVTOFE3rL3tK+BDsyzBlSVxE8SbvqN5TqtMUbytoa34kStscO\\\\\\\/EDgMwEeYmK4UIl1nulSSHWsJSujduzcqWdjSrcFH4Iu+8mjYA8qZovwSpEpLoMSBKXLIAQfUUHNmsXBbpjPy9yDjPwNdVRiDP0tgQj9hvGtL2EqyDkwFFqcr40u1931xUpTWk5qToigNhuPOlx953WXBd8p\\\\\\\/5JmdXQaDtj3l+zpEqSX9xLMgB7R+gkCClta0Q+EN27eaA+yR4Y1R+Qt+mbC5ArjraGCn6ZjBGk1av7aT5lGdRU8hSWMGw6cciNB8Oi2M\\\\\\\/xAhWoVjDRFORMkAySNc4wGeMhe9tLTH\\\\\\\/ll5lgyBsHFJxqbV9OL83Se6xO3qKD3wOaPQe74rnV+8thVGbO\\\\\\\/nWO9fFvUMnAqEif\\\\\\\/Eb\\\\\\\/X8NMRCvijGsLcJ6etgdmt0VsWk7BA4mcfEDXq3TFPF9pvzgp9rwZWlWD7Z+tJnKAntmLBcS9fJeHwcCNrBBa0R7GjO0xMhDsRwxy1u6jUp0gftkO0ZTdu5frwPx1KCP3QAvHQQSe99QSgmg9vrYnqzfjdJzMjuSD9kpyJwKOP2VMTLyT4t6c\\\\\\\/XVL7fJwwiJJwbOXSX2xE+GG2x1ux56NDYD0+RgzhIU0x6EXrkO5weQb9QURvGjvQZbylBpYE83qHzpRTXhNfi4nHnhrMSFhpeabV8qTPNf8IPw\\\\\\\/kj7tpGSnZh\\\\\\\/aZKY8s\\\\\\\/8Vwtgf8RBlD8oK86TD1uLnipZuxgImRZy19BLGpfnh21j3JT3zsZJqLuL+7L2NkkN\\\\\\\/EbGGx2nWj3sbgQD92\\\\\\\/XLT77rDmY6aZ0hVA4HLLVc1flf48aZpYbFwXp9\\\\\\\/ZwFaMr8BnuzGoInBrbDjuUhGf6m7RSWadVzjPndmRwzze0bSN0j182m4Aggs9lWhKZ1bJb0RoUowg8nWGeKrsJ2W0LHVBNZHRhZEeOTTY81+dpz7A1zHXbYzgv39srb4DLFTE\\\\\\\/ia6ZR8ooh4sxN2s6fR91M0QsWdMWSuMefXCEv4dBwbkEizrQQiEjQ8BNIj6NCQ02GinT94nH8mSTA1c61Dht9X0nAx5dFtMqqcta3gdILuL36R8Q\\\\\\\/PHDttOUkMMYeQr6Sxu6RybDohBY9YR8bS91XGjBXDtKdG0lUG3NG7s4njlGyxKbIEbueg3PLKvwAlG8g7jW8aFqCxUzs2CeCk+vk3pzVGwf1Ve8LR3HW5prKTFta7etRY53gUK1CtVINVIVAZT6KwMJ+H3ctUHfL1M\\\\\\\/wQAYnW09kXAi5AnvE2ts4c3ZOxi61NwsYsGqQEhxnj5Z4OZTnYnNkewcH5aVz7T+Ee8NtujdGwwc7t8Cy\\\\\\\/yLNlnLEVQ6aYL1uE27FtXvqOXdDoBniNI9Bgu1IucyAHAtjcegCQhCXa9YutURmmdRGh7fJ0Ui7JAHyvOVZ52EZjP\\\\\\\/g8tcE0R9p8YqFfi5Ilsxm+4Jr6GoGtLunivf4KtwYFz5CNtojrNiTu7Ti7gSGuRx2gxlkD10yctH+ivVx6nm3UNIxIpfgB81PuZnv9B877+PVl0o0T01molGPvfH8h18UWAu\\\\\\\/hv8wkalG\\\\\\\/bvGEpZjhgIZWo4WmzSG3PRUP9JTrAcyfmJ9LRjEEmYnyYduEU2IQnhrOsDv3FJ8Q+LrDoj8614qJBPzWXGWMEFHt4zopdW9M6lptUd4qkgz3t34tb9o6pzE6tpID2n+3vgS9XvIiDdt43En78VOI+lM2kdN+2BsUBft2zFkJBBDNlf13tb0Y04zv\\\\\\\/gbEp4b1itDqzKnMwZpgnLXHqpupTyJAT9LY4COk1VV2TJetu8LfJNCoFKAcmvRMnuevU5Z7zylrkE6mqCJ8rhlx6MEqjAmVVPgtvrmVoK8rv8EPqvOMDbaH1hRNhGTh0gTt5HeHkfD7ktDjjQLj6oZOULw1\\\\\\\/t5pZJFNPUKB0Np0mk7nT7XYtorfXUfHkP1Q7mJeLdZXXEsQfg2zdZZ48JuHL9jk5IZqEUM69E8YdDCZv7q2QwJL8ciyUehwMnDi5ShOXEJ9vJOiUhAidUpnNT\\\\\\\/FdvUdeKnZ0q2BGtTZHWF\\\\\\\/ysW20OF1eMIm7k6r935ikcqHCUqOx3q8DpFI+\\\\\\\/at3nWNa2Y\\\\\\\/7AJegLpItt+g2scvCcBXfBVFZq0pHrWvE16xBFCPl6KElATQ1SdacEY8HbTkgt8RoN+U93ccEmaz+C7I4ko9bbSnTOSlHoqA+w7qj7PFjOF3R\\\\\\\/Sc5nklg2OOtYKw4I7b643eASfMLyseNoWZ2zkhbaPQyQtnJRexd27N\\\\\\\/NHcLsTjp0SqJpnQZc3qhpMQrT3PHmC3k6foQk1M4nBxpAaBNVTrcUFfBR7tV+WxGAfg6rgdndcrRKpLEY\\\\\\\/L46KM2fTIsTTMeUirUsFMi7qlgoMdF3wxPVtpm8ykFWRSYdd4Vi6WpjnPg9iMnN0AhN2pc2lb4Gcn0\\\\\\\/NoNG\\\\\\\/VQ10TyZzZl5\\\\\\\/\\\\\\\/XGvFHdNdWuneV3LFo7DrzCPKQgt5wH2BB2vzbjWFioXJtgZYN80P6ItsOg6u\\\\\\\/6VAsQpES4mgLf97yNvhVJKgMsPqynPISEYd0Vsg\\\\\\\/g+W7AacNAQlUcwZ6nclAlFHN9jSUtsRsocmwbviMXidqUBOB5HPzY9l75jhv7QD199xhUU8BQtY7uF\\\\\\\/NFxCdr1ziX2jj8dHwRMZndpILbrUdm3FR9tXFa8PMuatydFlyVdf9Q7cY8Jh60Jg9ydH4DpX0gW2utydw3FWht5Ade0wioIOmsKgCpScOFqY4uFNz\\\\\\\/9kSVLHEDqddQW3MgGnz0i1DpNGvuLSJmX21vh3w\\\\\\\/9kbiKL\\\\\\\/ZnHffew6wJyw\\\\\\\/SbU+8tHLSxiZYqiasiImCg01ZT5NRMvWSncoCOqtqQE4g\\\\\\\/A5HX3LSTp\\\\\\\/w\\\\\\\/Ol5Lgt9pHj5PMMO8oGagWCtRuuPc7fTzvIwx+bsfqkVS6TcAbpj62py4vA7BaWLYo8e3E2vO4AUCBAUhBakLNBIO6pJxBrnuaFhso2hYJQIjdyEve7T3reM0OqALayYjXVN\\\\\\\/taLbFoo0lZWXYq5IgGl+yQbq4AiyUZ8pS6xUeBreKfdktq5A39uGKOE5PZGQYqWZWR6XcM32xkB7bOn0FqQ2\\\\\\\/BIzX6LXA2ZE8N9jntpNhh+HVORLI5IsBLkHbzEPL78cx0PKpi6sD3bbcKSxr0TcBrCmA1Qul8\\\\\\\/XqB9TuXsHcJZzDNybNo3mTebHRZ8YiVncQXaVnp76IaI+Q\\\\\\\/lsCYS4BFifklyxwiuDCtKZzLXajHXvpzL4kE0KTVmRzIf4fZy6QoHUPql4FOx8fHK0qIsVCcTzVSf2qNI6LnTeWtdBYb0anWEFpLL1e6yTGB8Us7D81zwgpyz85i10W3HfvsdaD8WKMTncYYw769G8K67259HtSKNBUej5DcOXpiZFsHu\\\\\\\/6zumVmTHM4\\\\\\\/zN2wRpn\\\\\\\/UKZwpSD1odq6bqO1p5xJ4wmkgMBysInp5w4SNKZKLWhbDjezzV6JnRWG3JKXdljSVhTSeK4GwTpi6skgkNw3Xk\\\\\\\/3y13mRJmid8jS4Kjqyp+BTLlOO6Yvh4I4mx3dWlFFT56bWEjLZjIH7lXYGSKtpa0vGHvKTOcY+pqtt5luj54ngOCY3KYQISKCE9w7Y+R1SfV+UcWnYnKXSh2HbYuLGhC2FOokdlJMMqFDIgCkXbLaPEPfUw+PRCN7hUxxsHzlpzEZNYJVIKmFpk\\\\\\\/OCtzpftSuV69+DIGb9K7YgFLpq054A6zptkugkYF\\\\\\\/7dh1Bu7cdZT7ta7VyFuGko0l1jqEgwJYnH1L0\\\\\\\/p\\\\\\\/1+bomWEHgt5jM6J7hNbCr8k7JL5BCKP\\\\\\\/KtAUsOlQMMAn5ZmGidJL+esJFXH8ShcIwM3rG9UEudsaU5rCnp710TEEVU5tK9mQ9QkS\\\\\\\/BzcxyEHYIcheCLwbI2AcJ6hgqVXXtZYkHWBIouvAQuofySSdOeflwFSEYYjyniJY0jT8KDmf1K1ledAimQo3Pt+buDmH1l4xVB76T32fVY4AHEg4RA3Owx5jy53pZ1m\\\\\\\/\\\\\\\/4RlxCEK7ndvdvA0GzByeUcVOa7TN\\\\\\\/lz76zVtDwBrPAOcpZxKh4xu8VSVnvL0\\\\\\\/DHHEIVVaWSQxLvMk6VBejBzJpslkS5NfhZtUmzu3jnFdEUT7jfJdRZOhw4XG7xv02efDRRDD8BzCACMYfdQ0jsJ7zGCGr5pzobOX\\\\\\\/l7a0Ekbhbiryl3E+gDl1+1Av932xgMGgF06U542DxPmkTSjC8AWYAbfXtXj7L5y5I5jX2zMjwCBENuyElv9mGaqIj\\\\\\\/4ri+lvABCQz\\\\\\\/FevDJ0fGP+JwLviKKhe\\\\\\\/7rIhPoVe3NE1mEJXadMKZjiPV1sVMUlo30JhMWvyG\\\\\\\/GolT5Zcq7vYjaYYdNYj0Dd3NcFxXj35MJw5YxojT7dzEM9R+UHvDq3hA5hyu9m9masXwrbbxRbP+d6\\\\\\\/MEy4SGlLyUhosnKm1QA2tw+fljKIFWg4SFB+4KIomhJs\\\\\\\/5DEkXF1cQ6RB9F8qvCD\\\\\\\/jtc9\\\\\\\/VyJJIYutP5\\\\\\\/d2seIEg\\\\\\\/9QtlmCgjc7zuYvlG1hMapumGVpZI\\\\\\\/0+qVYmD8mSdfZMbbPHYvJ4AJY8BJf6Ejk475PF1zJldRe3\\\\\\\/tLhfI7RGkPOlfasXHF8evE8aqwk8tz9n\\\\\\\/J0w7GOpk9h66pBDzdV+sng\\\\\\\/rgGWOCQVLc+t5ECbFkqQQksQ+NAsIag3mNuJHfwTG9ywrax5o6qYg6hx9\\\\\\\/waYZXQEAfh7\\\\\\\/+53ug8l65dgQ8KC91HzLR8OTQ8Du7+k4ZsoxQATDzArl4agsu9Lur7twtzZWyBq21Zry397Le52THPXyZIneBkP403RuChkSB3u7ngxte8wEM0\\\\\\\/AuVzcNlB+2ZznSSHC54w5i8RenI5\\\\\\\/tr1WLrLvb+OF3QNZVURUEMETsPm+3QVURCvc6EX5QvKniTU4SNcL8jAPaOFu9UWWxdYotN3mAEaePQ2A+DW6xFVYxTaRt\\\\\\\/bZ3s7HEh3r7tgSYpV+i9vnumUhey3eXK\\\\\\\/5J2zwFeq+PYo695uCRGDjV\\\\\\\/J\\\\\\\/8TpORd8oYj3CuYxNkrXoDmbN++3ujt6BQInc9hBmN30AxMXgpOAtLjZTWs7o0Z7E159eTYlAJRFStM6TBKVd0Q4PiVsYBG7UYjjTagi22KzqbUTXIVMCxz6ixpEtshBopgkRdY6BrGYl88qCioPJWCYxQWQTbjhxXysRj\\\\\\\/EuXtL6EEa8EM7VFblQF0MBQCyUfT5CdX4jZBrYA2SLFHbX\\\\\\\/Ew3BsxLToiOTq60jA6hvc0x4pmv6xaOiLkfv3qHDDRLB8Jn2Wa9tQAcgQDkJFnB8fgRw\\\\\\\/qrDdvscrWrDgRFKDcBPNcX5abBZ1YtrLU9iNNXlqV6QsvpXfTXvyT43PgliZwmvhz9mXjD4f+XBteXKF\\\\\\\/geMw8ElN9bVskXZfdZGeAtJhMvrZawPqmF7ChVo+JW9bDJWqrKtO3YB19\\\\\\\/ZAEsNcDlD7UkO+NxjvY+HX0S2M3fWTBBDhkQvVotvgVu2mxdi21JhyKTdcm+4VljE3tz2wZ7UGSexEzjatMWBxgyuSG9DMZrAvqoZVH8Zb7urvKqW+uJ4iFPoi4KStD1R6SsJgvGjAjF\\\\\\\/8ydlH8gY5dxdoxmifXqjt+mbvoifl43u84q5k6OCJJO9JzjWwb4\\\\\\\/uyOScLPUHGR0LcruamoHJrhuvPYfNQvuek+4jA7zQq9FZtCfKYl0edRMmO26q1deF2isI3rbE6D+x5flg7Oe5Gyzce4fJgeYk6jAXCMBrjGMGTBrt66mwT19YeMwSSn+ah7BA83aEnUgjMeMaggbtUxcFOJP3z6\\\\\\\/7QiSgpEG4ojgpxpD8gZN3bABrLymZ6xjeQ4QNXxafe6am2Y5Uox63gxOHRZwJf83YP4MdzwjaQITzeBCEfB4u4vdah3AxkzQ1JlSzZqH0EGDBE0oaUxebgKPlZuZZRktLiiI+cbx1lac8+31f+sG\\\\\\\/0lxlybKYEjnuuqlc\\\\\\\/ruGQ6frTK4tA\\\\\\\/7YIExxqUp9bEvg7iZ9fBqeADL4aWzcWeyY2lUpNrytz97pevCVAz3ZHiUj1fZEfCbClxiiQBCHZiSnsOct7oNflRqHyAoEjs2pNXycmE33WEVjCZb2fJepQWVEjJmeOQfUq3ZTVCSBLJwNe1uV+RFEvEtodN5XuuhJ2g2pTePQWrg4+qJd9\\\\\\\/ulEuUegYAL4f+pESIgkAFZm\\\\\\\/r+FPmbGya0bO+iVuEmraBTnQ1lpick1d\\\\\\\/N\\\\\\\/U0IsDSWbWB8jQH7a4iDq0OuLL4\\\\\\\/5j9opRB4na6hsKpstb\\\\\\\/y7yquBBNH3erhGg2\\\\\\\/0KtDW0NE79Tf5fT6peWgeS0PwAjr1p3wB4Bk\\\\\\\/CO93pYyeRLDWIshABExxqlkDMQxas\\\\\\\/7ZqhUS92GNlFmvoDCmqDwxRgC6FTowW\\\\\\\/NCVyugjzZm\\\\\\\/PKZAXTrdMCuz\\\\\\\/iftzxjnubkhB5QCEptIapz9ZSWmiTHM\\\\\\\/Gqm+qjqRuCfeq51+iIUIxdF3dEg5xuRgLZp7IQfXT20OPfT62eUQ9XDm3g5H9IZK\\\\\\\/SXYtNWiG0c9rQN8lxtUj+0lDSwJIuy54AZwX\\\\\\\/M8lviyQa4oFwSUcLwg9L5xpYI\\\\\\\/d2EPzG1iFt1BuiMnTeG\\\\\\\/f8hJP1pu+qHe5a1GmwH72ZCxBBGt7Q2UaZGG+tuYFsPrhrRjgfuznLMnrFxtSxnNj9Ncbck6s0fzmsO9kc6S6y4\\\\\\\/5y7qKa8ZSrAHbLOGhQH1\\\\\\\/Tu2kKgcofLLgB0x3ds4Q4b195YYM1Zri3BeeEtqASQ9dEnOpQPB9m0WffZCqHIqIrHjUxFEAZa7EsbGDlW4\\\\\\\/9ijqcEXcdKy48BpzX6fGrARTUFHKA56DJ44T5zSU8BCjrY02E8jZGCLhUHI0SrBWvoInn9+1aw0qIcnDXAB7isc3hDiWcB7LG5cB6eu\\\\\\\/wnDUs3DctPWOUumNNGQIAbeF17MXcZlfUMyDtEJvAdS41aJTxU7OieD0r8niim9HR+BI7\\\\\\\/G3\\\\\\\/1U1aWyMpkXUdsXXJ7vQuu8YdlNmUZE8BgipwbBEPSKd4ch1puS44eAzcNP6v6LWtbwIVvBjrVPU9fT+EZlJusC\\\\\\\/RHsaNA+HhqTUnKvkuLIyBZP315\\\\\\\/+feIPlQWCCWT5iQdbVMqWbVv\\\\\\\/9q78DOLtOYvNQ4MoZadlmLuR62AuJA8MZgjdKjSkz2eQj4xaxUlWzeXTLjOrTLtVfPXVUnOGx6kOLlIn1pZGcj0HlN0ahzsHFFXtjUtziV6BldssCG2yS4SW6qwcrqrVb84EDJj6VCrlZvZ\\\\\\\/\\\\\\\/vnR5lyGHe9OXLdBKiRY7UWM9y5oj632ID0KZIt0Niamq\\\\\\\/LtntZYsCsxH05AZDo8MBPnmyRcC0O9otE9TnRP0PWvhiHfopjebtSVH\\\\\\\/oFi44hZ5AngLCbe4U9JzlyEH6IU2vWITbhZrUeRkyvwFAm2I1vZABX53TgBeXP\\\\\\\/VLwXbuzovIZqdeQ4ROCZpTCzpCng13s0sOJPGVlcbaqKocPGuHnJcebEfix\\\\\\\/kGr7GJ5ycTxCX7F4ZSIkwaqox7UGiRuP57xZWBFXyHxaYk8slVtJTCV8FasKcsKEhnAwipZcI3djTuEoa2o72VvcG9mYMQJqLWeP0xLC1Ksh0fp24KYRbdQ+EUcyTNupEmwgxm4DwTOcscLNf2MA0YgH\\\\\\\/bjrH5tXiARR9zhlnZDvVQr6tD6470VR6sB4Bb7b3npocfZ1szJKDt5LfOyUjpjtFqFXZzJncf\\\\\\\/0ZPXFJ2CfMTnDnYy6k4nUWqC9GUbbfqJfKIB0toreS6EOQf4AiSWJfoZqQaHAqnIekZFZ0hU7yChFQ+NhczfX9SNo6xqRG5ZBJYXnoFPPr8nZeno+txI8A1Sxbxembr8D1tstvWvowh++36YP\\\\\\\/lVuUzs2QThqxf5F8cTtDkh\\\\\\\/quKjMSkKLTFAR9MQT4\\\\\\\/nyMIb5NJLGdycB3DIUchlPtCwS+WiNql+kSo2COnFGw3VpN\\\\\\\/c5mn3aV5vdwFejZG3wFp4fvtgHdUrsNkRFreB9tdtv6l61y6x331oFGUnPnT7YijVNVneLPdnqj6PQOKY9\\\\\\\/07mPdak6lwduJJl0qnCsZGBf910eULMWDW37GS6m87alRMXM+w74o1EQqaKs0oRe\\\\\\\/M9RSC8C0C+YaOdOaSEkRAdXPfm7pjWNjQ5LWs\\\\\\\/XPghkcW0ToHEI3v\\\\\\\/nxfyXCfkUaM1L7XPsuDae5dGUEFl5szXNxLOivMFK\\\\\\\/KXscqEl2iiH4zyqLQlQc28on1NALz2f47KHsF7RNqCoKtTnC9a6uTH6jcXzzfeJ\\\\\\\/l0\\\\\\\/Ka9qTyxAEjaZSfo3a8itay1cv8LD4g9ieEKs2DYFmhTc+dZlu5fTnjD8B\\\\\\\/g2SSQlTjf2rT3sJAMMyFPjRidrnI9\\\\\\\/z58n4KGXA2OIijs3RL5PpLf9axizDFXrnmMlbXrWxpZ4BcxpDBbOuIrUqxhuZGXLiKOGBN86frjJfC0AwFAMzc2ot8xm7jciDMChOlpcSMZBjmHDo1iCGSGX19FdxolnyGruIIspy65bPdcIGbxkQB9xYfRfe93hQ4+xjJwxAcBrPknbfkPZtTLPv+VXy+maVo8HfBeHIOl7s6mOlueYRTONelmpI7VqDTM36DVrTjJQj6KgTlWxSoVtybEthRbcdbTRFv655Hh2k9dv1q7uXD1iPXmCJrynik6ub3IOAmu3dcbFgY6Di2ii4+p5gdfeEEv2lQQvbjet3+eNPnBteBachKDD\\\\\\\/XGAqYP0pLVWj+ZDrJ6QuSEbmlJrAykW\\\\\\\/QUBZIbQUBOX4f3Lad6ru+0RQbWtrUPvHb86Bq9SqnadQt8uH56M5GsCQ7K6M6VKx+OvX9yGIxIf5+im2p1cRDTBUC5NHUDJJF7d6yybg9R\\\\\\\/1g80mauYrNz5n4ajU2TzSoJw+QHm35wOOt4Sl1SOliN7uCfvDA+k3HDyfYmSvvhVT+oxbmD1Ul0PRKhAbDnDT7qOrSgrU5OzMLi5+QTN73BZTRPhfXg\\\\\\\/Uo9A5RohSTtVa+d5OLDZyPvKSJZcHR+A7B0Ig0ajnxgbPas7BkJQxaE03PmjfRiS2TD1LMPwRILUIf\\\\\\\/bwJ9zWxch\\\\\\\/g6E3zXenF3tlpi+9aAEWMxbL9KdfI++tGgznr62lmFARY9v64ymzqOoIz4\\\\\\\/NWcoJSH+RI1kdDz3\\\\\\\/b6jCZPL6A+Tbx4X1RJLvseBI4eb+X3rn\\\\\\\/4V2rJt37ADmNMLkM+dy\\\\\\\/2s1z4BeLqYDWjyw29OrY3zJswJfGjYbjSZhV8WMJpvl6C8pcxBGKO8xHL8RQzM+ajX0FIzJ7C3SPjzfw7hj8PkepWdAlNtVEugpojfP7oc9HLXEQKPEnqokvt+h3fX8O6NBtzGK+ZigAGlUpkHigAeb4d+KEWMf5t3BXHa2\\\\\\\/cxOiBbXHygzWZJEXTVfH1ztAN9xoOMI69yk+b14nCijoTwNYkvEz7XNmfJi8VRYG44yq0CB9lf5JryvpbaC8VZJL8qdAo2DofYwbk3234Lir1dkjGFfLUDIUu2wHaBxvbYX1vocrHUPdjPJxRHpIYlZCV95UTW1HepVYo3t1OHGRbkEcL+hMQOFZ5FnXVq2Oyg0FDqwojl63igbk681F0NhYEYf9pzYYZUe9ZssiILSwid3mUT0HbO4dU+UrL4wjVkOrvLebsRsjZCaIh\\\\\\\/qli+0KUQDcmO7jeMnSJpsNnyQEyePlvzi4Sy0CKVqngEUIyRnA2MHl0fxkD1KSaS7rkNwiTw8f9Iz+nls3w2VhFCQO2jhLjA3DnKhY378dIP51bcODXkG5kOLmJMgDvIUv0qB6HV99BHeGEv\\\\\\\/1+gvkSOGqG\\\\\\\/TbnkGMbajvnNhjAx4Tb1ab5UTNG0LAjnebZ2\\\\\\\/FnaPLqadDk8nueQ6iakCBlu++\\\\\\\/xVmE7lxjPZEPPi+db5f9FHwWjQ81\\\\\\\/9UGnqWSR+mz0Qr7AUDw8QfwX2wozjQ9O0OkwWgOWWa\\\\\\\/6Fm0nUKZ5Iqaog4tCoPWGeiPsYFYHTo9cNZ3PcsVzvRAGNJsZEAH+9mndxsEueXOqRb9zMCIbzzcum0wtX5Px8J2ohpEpQC5ZB37zS0xy4CNqCMVnPy55IWIkscWmgVFncyoEE50hlLZsDF3pEM3Azm\\\\\\\/hcclGVVoxaOwODDqglhum9gh+4rCdWcAmSJUCaFKZ7KiwiA8uyhVmwklp5bdUaZgszp+vC\\\\\\\/rWkbh6k3EHUK1Ppb6GeUVXMkcod6BSdQ4x0MWQqh\\\\\\\/tIOB9hwtHuX3AIHX2TQqSjxrzpcV83I+vixuHgxXOgnhpmLwTxFrX5MGJmhLsbfIf1UMAqrMoAMhdC4ZeN+A2oFtASUR65Fvsh\\\\\\\/XAG67h4jc7UD1koOeQH0lbU5lmjzpAPVc68jXpzt92BAFs7gMhQYzKUvXdlkhs9jrLXv8fPvS9pmnSYYbv4awLXnVxnaxyOoAN4hC3ZFhXOVItpL53XOwOizgFuAYOmyUeYRYKygat+1qeq63BHA2oMTuD1tIDLiySq4x0sz1xqodGDl1\\\\\\\/A3IOa3o2DJdsAzltc8oDkCDcQnbAtE5WOwVOCTlYuDyzMRTwQQcQda398yzKN++yLtC43iX+Jc0QLlX+71LAjXITL3L+S0A8L9pPbo\\\\\\\/0yOeImeBko\\\\\\\/s+36Rp3351WSiDBUaY9h7KLZdzxUXoiM+ni5+8up524yivvtiV2jsZFaClgP2zrdaFG5jiEBK9CZNuwyT9WMGh8aiZVPQ3SjTyoOxENiUn4WcjkNtrYNOw56jxecTRDbnVuL6EkrJHWP0agc\\\\\\\/SpdPlm6kJkIgTFrsQJw8gM7\\\\\\\/hvRAtP18ASuSxOgwjNIFg6s6WxmNQtXjt8AF3Npq7X5yb3unkQ006XCVscCvzNedFWobbGG8AXao0DYt+gBF084i4qaBPhktVhE0Hrx1muiIcVu9+LWmyiYbb3cqW0KL+mSaZWf1ijWr5rShTd90ppfCClBDdzWRa6I\\\\\\\/15ADm85v2VtlrElspOsZwD2Myz8eFYhvTpKVrZJEFAkoHXWkoqTqQL3NHMR3\\\\\\\/O6gVNOfS1aqc6x37cMCQls\\\\\\\/MEiW2OleQYEUaGEGiyAy4LPR2gFzmbXLDNGh2f+pzlVpCdrt\\\\\\\/+cQz+JEcw7tBw3Xi7FVB5oJktnZy3ZWF9XudNpwYS1VVRfCTMKsJQho0woYH+XzsKF5KrL+ebckevPRKjgTVECZzMENh86bTHdnWjWIUI8WEJf4XDc2n9+WxYPOScjoQmi\\\\\\\/\\\\\\\/4NjO+uktllKCsS+AeiFBeCbAUFU628iYUio0RoWBXQ\\\\\\\/aYCfzDcWY5svtvhxlQ+7wbeHHnRiFfwSyJIewwQ5AsbUqWdK7IvDpq91o7l2rskMFPMJipxfMKdY2WfkaF\\\\\\\/IRyHSEfNyAOloeHR2eJASSBxxMIFeR6bn0IkXlwE5iAk\\\\\\\/tVzoSi3EG5Uc5WFC1sOyzSWgho6apQFYciSJx1b4bShKWLAYAN\\\\\\\/HwCZtE7P7i9b+TiOh3YRN0LRMGVdpG+lnKHsEiZwTtzA9\\\\\\\/nLQj6lnhzKBI9Ss5YMPK9IytdsejRtplsoZIX+8Tc8Vyxr5bjwo5FgA02RbHUIstIaGEcmiE1T5JKrF++l+YVRlw3iwxWNze7xfhldZxQZvXRPOh3SP7odF3WbOJFE7nR7BkQvUr2alWcUcyyIm54aiQuJOtF+AhbmbnNkjEcPjaBamuI1ogX6pEVn1Ghp5xQqiwkfJyaXys7bJ7qUVyBKX73xrUn4xzuEzWHMlH5xKp0QAbUTN7giIAvLxAxNEas5y0NHzW9qTJ3tajJaCEZhMbT3OT14roezVPXxOdtbMInDsr74XyN4hHXNnOOjIB6i\\\\\\\/STM\\\\\\\/TVK23T60OAJmURNxXMbfEIEhOzivEjiSXl9jMrpZ5WDzifcUstwyXGlbILMRUp4u83RnbIiihNw26f4dL8OAEr9WKgrOUArAZz6Jcp5vPAqRLwstcyvRISR4tpSn1aj3XG0SNDViaGR0d7D\\\\\\\/OWnKmu1WdREneur9oo1Gmmw9N5\\\\\\\/0yQLiaOYynVbQT8hKey2oddMzQOStNeDmEQa+WmhqyuW0cjmAsxDyb2wmcReynez6oKR6LzMees3XllNOpxxD5xbAAQTK8vxF\\\\\\\/COJvpRjfLBq3vTfbpXYaIxF2CHb6TSa2KqUUfNcekiR+GCH1o7OdiON2hvDolqZG4UuQt\\\\\\\/zuUusHreBgCyHUN4B0t5UEHIC6w+JWMK5YeOQMdMQsAIdK4sJPW8eHH7ywbsYvUNzufav8QxzzZNOg5ZcjeuruzdAwbxEFmtddnC3gzKV6\\\\\\\/+sP6N1QtBiZylLmftp8\\\\\\\/9j8T5ce+q2wwMek3VXWMnQdy6AgbzSq7FyOu8HzLk0C40OWPu4xsB+ZAaazJI5Hp11VImkp6IDfuxMtIRa83TQOBmk8aBh+fzKbzvoF6fOU8KlIVDJKSSoQZs5ZvYtI4Ke\\\\\\\/bRFlcFCztmSboNRkXmO6WJkWe2SkijFSZ0M5kikAPRr\\\\\\\/kT6451mQFN7E52VNWa8Vvka+hPY7kmyrFlgubC+62sT2Q8TxCZloSTLGrd7FWUCIgmvTr590iVwADuNN7mr2u2o0uyylNfPbVfSGJA4Slvt9pfeC+bdgIOtcAg1nec3nP1AeQBgdctGEJj0oNmIVjnWgU+gV6uyXgYuqYTP+lIYMrop474ogxMQAiNHgtOY5X4nc9iyDCuh0Fo6oTqr9L7uZBtB+fU3lbUdmrpmG+cKasZEJVY4pPrtlcYSOGM4xwP2wsOWunVA36Pehhi6Pq5tffjxr8diB29RfQJqe777ux\\\\\\\/dGKBaDPlV4DymzZT620X+8IqsmiPqxVqjdcplGMfIgl6SjOkQqz15DBhewl1MuE9UE+UCIzoUkIbEB73tZUESlpjZIU+mBI1BbbBXH9n\\\\\\\/UTwMbN381604kFSPdM2V+JD1blQc+32KVeDXTEEWkjZuPNLfRTZIwVDRSRYk\\\\\\\/95vlfz0GT3VOOLG9X461VDI9JmgcMtcu6UZ6epHU5FT85DtgKQl3tL8QXK9q0A0Zu5bygEIXmMFK71RuzE4ZeCMhCZpWZyqW+Lk0qeeaZTOwmlsSfFlOZbqVS\\\\\\\/fKSTA48jGuUEqWUMGnI6\\\\\\\/guP+Ega9cCGwLwWmLBGx8GCbcsUeAoZpwsGvUoOhz4Y9pgjhkpelaNjtrC3M4QWlIdog9fOjY2nfPWbb9qYufj3GvzrRc3xSoMUxHmcjMQH8V\\\\\\\/kuHmKbiU8HUUwhmJDlYcFqYLW\\\\\\\/736\\\\\\\/s6Bh80qdyizD2EnURmcPFd8Vyr5CyOyUPytBcKw8nuS9qcfZycavRsKWYCusxA0sieEPv7XYQBY1Yah\\\\\\\/1mY910Vab9Cm11u8hRh7FG4aep8pw8UTp3IMrP63J8IYmQA5r2NfLJOkbh2ym5UzqVT8R1BQQdRg\\\\\\\/DfDwkuzDym7wxCjf\\\\\\\/6sA8bZPI0JwmzhoaY2u0cov4v7TLO8WG7YbG60RonadLdDImR01b7lrtJNP4kf1zvV54gcWjh04FTSb7MeQLCq4ws5ERJRxCQDmDAdmMa8g\\\\\\\/1nm5iVGG24WpILOM0qfvm0arSnDotCgg9Y3KmvAj7tDj4Ws\\\\\\\/tv9yWKYYrnFZSRKe\\\\\\\/LQ+aKmILiwkxbP6civp02fHRnu5J1LIwc4WUd3x4wKp\\\\\\\/I3JmmCyyOonbAnWw8XbJ\\\\\\\/99Gvi0rizajxFm53bs9ryh\\\\\\\/3+fAOApAylUZ2zi3A6lR3wFAZphBoZfpnDEUbQBd+5na2Nx9DUDougny2uLevzWaxVHNjPGkEShaZvO7DyLtkeAeKc0YAS7I+VdBuBBD7\\\\\\\/wIGdZ2o0yQM8Jym4HRhZfkXsLaSiVNw\\\\\\\/PtZ8+2WLKvJEwtwwho3choTrGibwexTAtgk88xxrKaWLOPHNIFt2bI2aYyklrRaA85t96KreKR7QNgCuzw1AEM2FHQbYGSf8xf482gjZ8ci3tW5rFNlhRIkQhqj74AuZmOQB9kjdIoP4UaA+N1HlVEB6Avps3kjc2MbnV+8IfBQXJEG+ycjEj7V5gV9sEXSzY6AVEoiwwxPGsD98\\\\\\\/RHfR2N4k65dT8kCEM32GWtugAoAB9xO8H5hTnN1Svy9T9bBUvl13dBuKnQ6v046F96XLSejtQLWyfFbOl2rTgdpnQ8DAedIN04e3AVsMGGowsGQS\\\\\\\/XskrmKSQVHc0vfimMyGNxEIPlei4R6d5XJ6JUoHkbwZTDwHlGsXY8AvGBf4AI1obN8xuwbjdAdHVZA2vX6PD\\\\\\\/oJdgkRuVUvnrfsOFT3xaHMtejBj7KL+O274t2v9G4+FDtdMWbqRZlXCPH42nDN3l9R+hxo1FxaNaqWOMbPV\\\\\\\/3daBhPI0zjF+S+UyVoxE8uNdrv308KdtKGfnPcpUUoNkOdQuTBa\\\\\\\/xD964VBEitsig1DuvGy+o8FeEhdgsvQHMMgxoEGVi7H10mhHyOPqTeZvhF3Hb24agwjPG84SUr9yV29pi41c9xlLtdVaz2+n4Oxsjxna+pp3+8A2l9bCaOXwVZCXyCOeRMmm9eYteCOkUdka12gHu9H4fn1pcI8D9QPIsoigEOVoCDdoDILnDZ4Kn1g4LZTSRF7vgmrNvtllhe1d0aHfuvR7YBjty9JSkamY\\\\\\\/eeVi4pJIWjTIaWhGSnUpjhMIel8YmzAjHxvYECfxxHRCSDofoxEZQiAhk8Tf9Muptf4EKhGJwaHkwCjxdgHJaFmYgPTJjHtOptvcagrO78vK62vTmpCCcpvYzf1coHjFEEIaUvSYfV7i0lAKHFqQ\\\\\\\/dF3NFY9laV6RCqTgqvsuBTLxEugEqh6VTwdIMm5p0dlbYCXqb+PNIkKEZqa9mODTi2yq\\\\\\\/7m5hmoNcfoTaglqGmF09EkjrSSmD28N3zw3GDl5bhXBSumzfbVCNXsdrmbpjauDr08Dm4be3FUlxoLMwRUnvBeFGJkhjX+wk6tBGA2HXw\\\\\\\/h2slWoGxkYSUCGt4iWTEGCSGluExrP7NN+NGv7Evrs+ULB1NfP3k7JeAFbZIjGRnjw8PAUahB5GO4LeSvRBzrD66iYt2Lj+BkOIUPXFSv9wboErsDjkp6CVmM6gyWV29VVm4l+4sC8SDVBgPE2XFvYXEaD8Y74ZQsMOMDdfnrqflWFwGkpoFUKNcp980X+mu\\\\\\\/gRRSPW\\\\\\\/b0ZRGNqQ0q2EQ6fEGF7jYpM4BzwFsa4MyiraIzTzUgUKxVgczr\\\\\\\/WFhAgMhfS3yfm+lyTYrQ96fw71gp+fAbSLGI2KoR9kY4AcFVApbIQDy5Vwcx3dsskl6A0DkQKj9XSuNEbMOs3QwAyOo8ApJ5b0f11NKYUoU83wSf9fhoDCwDpmEkqLeo9oSNrYl36lgSZB7TNhW\\\\\\\/0xP6olccsZFh5FWlz5Djk3LTn9TZ+L1\\\\\\\/PNQtIwV5JGu1EWvBLzKxu8Bydnl6JVQdLuHH4P8YGblFQQvXG7xhUDkYLeRm54bne0b\\\\\\\/E5qBl+Wk2dGaTMAPJ+iG60NSi9WGwvRVV91tjLRxSyDozE67evq3dToG636TMukPfGvCo1FELGSzgf1mp1yh45zwGpwxTLdOq3mXa\\\\\\\/7x5pbieyHyVg\\\\\\\/jEM+auf483Qa2oE1DabH6rbNfrLziP3mQDZKrn8Cm4Sjp1ybvXdAV\\\\\\\/vjyRIXKIcODwD28qKxAV6gmZqtHcvypX5tLKea+EPAoJ6c\\\\\\\/1ZY6jAdxyN\\\\\\\/\\\\\\\/duSs5Wyhiif0+wZ82kTZ9hrzm71NOWSCk4WZPv7fA1Zgb7qp1MRj4xzJ1KLHmVgxRy3ugbY8nUUY7FKEwtLzi8uuhkz2N7T9O3ihoRg1YVqAWfpI2i+jKBVe+ahy4KQYY29nW1e9MT3Zn+yC\\\\\\\/5453KjdshppSuzq\\\\\\\/bWFkzZ+7G08My5eFqs7U9w6lRRsQt271T7jVMak40RhJz4eZUDFsnYzWqg5FZF4J2RbI4w\\\\\\\/nLLaREhFTxuHLC0xc1AdKLGeGoJ0ijnljm7AaIwvz3ztOeda0bRruiv14JX1qCmKWmu\\\\\\\/oT157loeYUvTW7kPMXZJodRf40X3nPqzKWNnfrUgFFheOdgYbvpU36ndjHRhx5zXMv9M+v5lABYQ3v2HIARzIuefpCGv9Ffies\\\\\\\/7p\\\\\\\/4EdAhV2csW8Mon0G7u2p6bMCCY2LzCBf5Roq7pAYL95UqFRCDSVbXBA2nDCQ0Qz8SAjSyaJ8DzKLSD6wFyHvXEyMULuhKq9bjDEJoNpCvGijncMSOEgCuqG\\\\\\\/zubCeh\\\\\\\/8PZe8+oZYEbqUnlWVc+QZinP+01ub9\\\\\\\/yXpQwDMmG13uLVyPMy9Sz3MuwLzhf\\\\\\\/kDKEFSi\\\\\\\/9FP9xwTO0pVqLNQptsiIWR6UkdMOACMHKA8jEwB\\\\\\\/5I7HnU7xIte7NU6\\\\\\\/CzOrpnox3vVJBbx1iMTz8XGth7YaaqQMofNq8gvZCLTsP7YmoahILgZ5UYdkosGB+JxU4u7KB7TNN882S+RQu6GczG\\\\\\\/CaeAqMBykuNwKt\\\\\\\/+PvA3EvvmMGffNHtKQmDH6FVqaAZ1D2KSv\\\\\\\/lwSjBiEj4o5+Kl0BDGXNXHmlhDBQ4aHeMoHpBv766TIh2AcQjT4U2UPRCfZa7LJ6T0qmQ+0qk+UdTY+vnUpYtt5DH6txvdvblUAwyPlqXv7e9OPME9CpKUrjZ9zhKhmffOCxGsIVdzrTeyWvtspvDwZxeyHhApzvVrpkby\\\\\\\/FvK1B6hDe9T6bdZY\\\\\\\/\\\\\\\/kuCIoIQIxn1h92sAzCopjMtkMCh8jSde7247F6Tzgd7De9idca2SNh25HobxqI7+Mdf7KmQTRbDWilo9\\\\\\\/ZLvwHkxUPvrmwMG1waqx4aSCFSOvEgD9L5prtu+TEXoNZ8Or56eK8qYR0aeQQuw6p+gHqH4MUhJsjvANF+\\\\\\\/R4RaAOo3jYrZG7cAOK7CFrgWCHwznPzr3m1l3JxuYoJYXKu4s4+qnc7RGv9R+db7ATY4cTb8mzo15g8uVwoG46zuFmKlwtCvDpL3MO\\\\\\\/7uKpNFGO74v\\\\\\\/STx9xJi4ViEUR7j3xNycNbOqmU8wcPRAtsHzKm+UnHPirvkqc3BXxv4mTWbpwSgsaf3Ko3+XKJDJvvri06zsEX\\\\\\\/b6BN6vBIRfPt9mWkeX6zeJpSztHIWC2XawTigLqn6bi5lmpg\\\\\\\/EBEP7MfLVfrQwiDADYTae1tyV34wkvIL8Hqw07\\\\\\\/YsOUfg0iZi3M3sqeqrhju5sEk2PTq6+MeZzJMg7GyePWaPPCI5WJMayrvJmlcuWFVjNWAs5XfvRLPvEyIuiClLY8q58hWNPAHAoM0rW4QGDAyjim4CyXY+Ecs1lJcik59KLoZvrJXWXfJyJ1EwBOR+aJ4jiMPSRUAw1WWKZRGE3\\\\\\\/50vN3dr5WV8BgUTJpAqrGWINOHFLUpffhceU5IOiVhODVXwcaHtFkM4VNUL2AbCr3op2fQiJrSc2oOn\\\\\\\/BZTPZZQeA1WYynnQXCy0wllxG4sWWSCVKJbWT+5LK2ByZB7ZBPnVhv6gPoN1nvdwrJ95P23t0WAfhN0CYXiBdgcvLc6XZiGGEW1i67s1mHUHdvVfeMj9BGr31Y0npAbt887qWsOVWmAKzTm5jWybBoztNsmC8Lfq+fnWrPoKSRIMnhV2B2xt4LOQoPAYsetxGwUL1zeZNzsSKUrqyxaJ6muRxIOCMuEUrDGiiFsSay1DC0DjumKi3AJTkjjw9weiBcU8QiyQD\\\\\\\/cd\\\\\\\/0REgGSfU\\\\\\\/K8NbQQ4G5ICSzxwr6FdyGRZXIw1q9T53YZCZORL4ia5rWayFxy\\\\\\\/Iupf8gIk6EomMLyfmFICQSV8stxYlnNgoum4eSpsqfz0Rnl5cmR8D6RgH5DD\\\\\\\/EZMQn1WinXiBiRvaEXBak5JGmyu6VREVQGI0BF4HsU+eIo+vzHGtt7++rDDIGoOLP8DraKX8NaEX2XXKueaSx25wXWOeulk7GciZs6Q5ww77Um+v7QP8WNKEHNno9etpJmHSy1oKixK7zKMOvRmijzW0CL6dwjB99HbFjcKSkKXnRr+vhpxpHmgklFpvZo35F0cxRGuj1BjA8CuUpSGLMXgSmjrCgPPTFMqirAnvJbzSqYdZNqWKONdFG\\\\\\\/Vk+AN32aV9O8W+HA7ClimvFnWA+g7DSkwTfGQCOHahAI0FvazC8z+audigpOk+bkEEHs2qr4qDf7lXGyEesNbt1crzJjpnl1hbzOddGM6v+V6r34WEieoWNydH4mKFtlc3+cYOXdkL5EFYLJJ9cFMKksUq5jjmthsA89iYdHwBT7ilfnYYGnPaI8Q3yxmZUToR9EspKi9MBBSdPVGr5VVpAHKQE0PxLkAj0GBcf8BOIbIoW203fQgyGQ3\\\\\\\/wz74zAr3eV+o1EHxnvzZ7hJCls\\\\\\\/OBR6DC\\\\\\\/OJacMnQeO8\\\\\\\/BgoP50qO1nddlY5BTksfJYiZ3nvzG9Z9Ierm7Jda\\\\\\\/kT9gf0Pb5+gJf2XPEM+4hEsyTZvuROygkGiMldQnPzmq\\\\\\\/2CDvT3Dpdyk77iPJKUXdf4O24y1HvKXn0MeN9yeTyTCHDhY\\\\\\\/ttXvFBKieTdNciOCAW9JNepPdSbHJDs7zAGaOUznN2xBDxrZ4rgVvTZO91cJaUb5HXhKtSqfOUmVr00lkEQVfm69X9L1bQ0JklBSoq1SzqeEtRb3AONRXGTyrVUAUbgRT0f75ROR+OaejMlZg55E+TP6RpqOsMzjVjzJHrOoNhJBkNx+TLsP6cCoplhG3UR50aG7+4Yr7W1S8yOVnJ7PeSEHj1MLT9hqMxRfTDP31BoksF3\\\\\\\/ICaPrcjXcYmrBUVCMZ+3gR0DxL0CDZrZREswQfO5Y8uYgjkO7H2MGhW4wqbto9vZlPdXrfLQrp6nbK+1oR9ugNsF7oCY89op+w4+0ZgWSAZTlXwS8iSV6mVW0cmhjQgbejkE5h6Wfz1jRTbyUunkINGqcpKaJz2O1kLg6o82noG5P1rLFt1g6JAsVdAZUz51eeK3STsT+3JsJ1y7ihZH9HR6QpOon7zCKEA8GRcbFqiI0SmsSSNO9cLRN1Ab\\\\\\\/yD0gb3Qy7X+WnOTbYRW1po9XuOrH8wa\\\\\\\/BQ5aw8fveJvWWXzbZs4pueQLUHptqsWcL+YEL2Q0iItnBCFVXIjy4IGalc4+ljB2INZR9FVsHP4aPtN6AbW3L1w9xS5n+OF\\\\\\\/TYfrvgBSgW9QxqitkkUZBRApyb7l4ufASwrnwPZ43RJs47f5LJ25CGmqaCaCn+lNszTEePs86gUP6SDevmNRnyGqKp8805x3g1w8nizDSxOXa6i399XRtQ4yUsRiWPBkxZv49bzgsM0TjTdiSK0jHaYDJM4hmUIu8eJo9vwvFbWyz0E5D3acojcLeRXFysivWxY60xG1jbbr5jBf42JFr32MRKuNRu5BNYV4UChUU4OtbR4oHe0BIRWSXcYYfHhFNgpptYuY+IOvTzWZwIWjr0q3GlqXw86DelM9LHQLd1gRKu74npHLKfeqfFpyBTDRb2+tdQ5mhPGU9sSiOLD6MI6Nv\\\\\\\/sEGL4kPA4j7fhaS0NKu7eJilXvDAjX84xVQJ1jwyyKQ57SyOXlrRfGWuzRqheLtIOmNiNc8FHzbkcoMslYTYTYuLNCk0C1aAaUPy4UZt5zlepeOnZaExmsEe5mDi8FGcYfDL56lfD0s79y5DlV8IC5JMaGDQ8VJ68IxlxX8I9IyH+sOmwRELUHRR8tkz00V8VZc9UVl6l\\\\\\\/RsQWHmmoQisa+JQxHVd+aUMgcjOq0C4Yeb6bALDXNlE87P9jFmepx\\\\\\\/f\\\\\\\/7xU4+8oLd+PdA0rg1llVrNA395GNfftP2yzHiQ0ulkeHlelhYPKMSCxTZ3YxmnMz72tdKC43mo0q3xFfvOeTnwBfDYHCrbAns43szoNbceLsK4vI6+\\\\\\\/+XBuQ6VdvUwyc2HurMncD6sS0OT75HJrCo0IHG\\\\\\\/UaMrKYDumDkuGYRX0fNHUWjX2T8jOOOYn6JvsiAfJqTJAVk9BHPClV+Y4LTy3uqyylGNzufGW7VmqbQFCSQ+0EMuQc59Bed6+e3h2l09ArdbgggiV35lE5iBQCDNBgxMjl8IChJ5M1lWR7vDZLzcaHOoRM6Y6xPaYXm8rlTloEucW5JDX1BDvIgcytmdFUvEVQvR58eORKorA3bc0HbZLS\\\\\\\/lbHXebHSUMR+zMff++2IsGMfMbEmscdr\\\\\\\/jDfMSfRZ5nMLRttEB+AjaljEGNmAk1JGIbc5x7\\\\\\\/EDkuVZOueG1rbRW3Gnu8qFB5iiFwqkIzhAd6\\\\\\\/rfdGwC6DUYJsq0voQHj7U2ALb9g2WtPGxkRM+CT9Kw6I6IFERMisxn97RED\\\\\\\/nnzvGSiN1zzz7w0JqQZn\\\\\\\/L77o+gjvKTChVeJbSBZTQxI9qVoyUVIqKD2A7\\\\\\\/w2KSdWKD7ICQf7AOhRE3aTVSht4Z5DU4AyYM4szM2SH4JbBpLbbZ3BgH7zgiJ7noLlC9ZQsooFZSbZ2EjjGYY1ZkePFmfBrDFDqhnUYDYTLe5bnMb2kw9BTDBFh5KzFGeHjSsS3fCz8S5DUKT344XG79f0hYb0XQhhLqcvHzJ37io133klYNFNODkCJlmk7NMRGzACvJCWEwMtIiIzz05qVC5KOUfba9T98j+C5i+1GcnwKuTEe4Z+7Uq6a2IE2Vt05yN0Ak0vAknU6u0BkNAiwET4\\\\\\\/Cg88TzLIoGD0hcI\\\\\\\/1llXYZlTzxuH0PySQP6BXq+RNFMiGtT5GwPHpcnwIH6knYrRzjuuQHEr5RXKXcwtVblinR4PKI2TMbdh4lwqc5fJ77QLqw9Zso+ar3kgT2dnueQZ4fDxTirTeTj1vQfWPkJDCvwGZWAXvfeXxaUfbjBP31KDD4TnzcEELuHbb\\\\\\\/Fw3IrfVy3L9VBqYcUWPjreatA4u7BdqVQsii7JQcjLTi5wIdQV4H\\\\\\\/Vlarq6DbHtUsvyqINDVqm6\\\\\\\/\\\\\\\/TdLMUlBAhBbPHsw\\\\\\\/flsGdJgwFQ8g7OzgcejivRZJL6YsOM+ZkPuxj5y6eUhUCQ+f6T+LQbLRphxJpwNW7CpS9KRZUI0YST71REZrI0a5w5hdPyyOnVJADsJ6SqChdQhB9CmG0maQjpE26\\\\\\\/V7D\\\\\\\/ULIDYwvC2dZ2HCntpO08sdaQxcqpoTb3ry2R+DfOz1C8kEaPj9uim0etbG35VC9dQ42eLm+TgaU\\\\\\\/Z1u6RHvDCERYIhKDazMQaAIZ0JPLGHkn7OnSiRc0ij7SV3UOVrPatcWZhNZFhWqzCzd3PCveTNFaZH4t1GTui69C6DcmHUDy5IzTjtjUk4Fs\\\\\\\/0mgDMWcnKeLeFJhpKWh\\\\\\\/f9M5mfNTyo2+kThlY4rWN87lUhsehOZjRPvv8wkNXd877xwSXngav0StaCgt+l0AxRjN6R8KfmqezDZWYESS3RSUyVAelfVCT3nXAOBSqppgWPRVI5mX\\\\\\\/8Pb4WSKAJOGPtQZbW1eonTyYFUBeMFXyhoRXnfJk57AEWiEhM9jgTFz8kr3ydFoyX41R++Lcg0OHq+RS\\\\\\\/WukQL9oHmgT8aXWiSPq3IxB+zq5bCPATDKsIMQZ1xODQE77kT4G5fD0z3+HAYMZk8DS9vCIWSUrRTZFFSYZJr4soHv1SnPg5qBBXi6wCnht99qLCkBArpN+9iExKUpAcGA0QeY+IAA15mEBva5UIqM\\\\\\\/g717+I2rd+dcwAYhfJMP0cz\\\\\\\/AfybDw4szBWYNw+4O3lS8C1YYgKN4iNfqyW8Su8uOwK6m0Jm8ugp9hxoulAzklsOJcD8mvMZBV\\\\\\\/VhK3\\\\\\\/btlrHcDxV4nGAjWOrS1VeaHPfPOJJ6mWR156Qjmu\\\\\\\/yoALXL3wNbWo1UPqJ2Nx8v+36rWw+eST7EIQE9CTdY2SDiuGqlgKAszE\\\\\\\/Ou5F+h+nS2+2O94JcsbQu7a4DL\\\\\\\/v\\\\\\\/WHedg4xcoFzpQgkV5auJipWof+pUig2CQLQx5loPtjhJiQBGHvYXTiiz0vCEY2OUdXfspKl9M9Z2fA92KmShLB2Vu9RzEEz+WoOSwxuNKcYsEZpP6xGUneiANQjmDMxg\\\\\\\/nLPh2rsMnZ4RqQcMHX98DrYDXIsxVEpWRl3rJs+A8qViTUB1daXMmkLsO9fye29W0zJ+Kjgnab4onpZBKlB30u+AqgQVhDkppHi+VjOP1YupJBVwfpSiogZVlKDliPprKTQkm7\\\\\\\/hS664cZwFIHe\\\\\\\/rlQKuqg2ljf2qUYUzwKRU9JKkaypJU+yDZnnJN1bk4Ps5dvzMziN2fzD+D3tsaZqhJL58nVzEZIRGMH14a8XBZuhFPJFXzxbDDyuHyjIytMXI8p5wxv7v04THsxIjVtzaiPPfwwGmxxdIOvbRUeJHQp8tMYwEwx9s+wliUxStURQ1lST5SfPRH9C9zA2FUdGG21ByM2Y6jIDVA52pMTlufQ7UKwpS0sZqGUJXBtMSjWI336iVESVP8ZHlyRw8MPYse3AIXqRTXoINPwKHpifOZ+YSpOS4qmznpzMaY\\\\\\\/io4i7Nv7tC0sIvChwEAaID+gPG8gVif6gLs2QdQgNaKb9FKrppcTHKa6\\\\\\\/7in15ANDeVL6fmXCcgjBteZXerUJokRRwfiZKoYVOs8NIRank5\\\\\\\/LbImTaDzWKpVCrnh5mf0zTOFJAVkjU7N1wfgiYHl7zsd68JRdG9UvOsDfpJIy5DKr0mQzVf2KTeWh2HUfPBR6zhbA9dFKqFRuJ3bIagBYajHvqaz2\\\\\\\/H2NkNJK8+jKfq1ArMV5Rexmdv1HiIoX4EPT7Y5Lg3x6UPn2pk8zRqcgYdFzMvznbW75en\\\\\\\/gYZWfww5yWRspk4UhDys1rhkEe4q0Hsuf1PsjNOnsXMyV1AFtRvxk5wbN3SxgZf4wVQ9xpZM3WDF5tLMIY3hpqcDX08g\\\\\\\/4AwB4pggnxneaVdNvsOUtmhYgOD\\\\\\\/vaKNu9y8wkOhA3\\\\\\\/6E3EB3UujBmqE91O5BaDmehG4z26m8KrSd5u60Ik3q7lkGgPvdYKUeEH6Q5INLi3hgSxiYhwIJJJkQBCDf\\\\\\\/mgpuadCsqcj8kS3f45uL\\\\\\\/UE6oJMqTYsByaBVznnl8ETfsKgu+Etd5+zrVTcMkqPeLe\\\\\\\/sL8KsHxjdfqjvemjiPw\\\\\\\/PYOj8AQGtSo6YWOYhQwMmmL\\\\\\\/562ZSJtC7Hpog51OaRfuL7qC\\\\\\\/kMWOxjACPAGlqOsIVK39\\\\\\\/AbAOWVPLly+TZ6\\\\\\\/0qA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Z06YRMnAyz7mGTSZ21O6dg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221054919b2e6891c51fbf12f6a5b93f418ab05c60fc414e28dbc8aed656a2d8d4\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RK3CIVkwW2oMYz4qcecjmqNZ3mja9vMlO0IpzjO\\\\\\\/VwiSHbSf6cGeXC3X\\\\\\\/\\\\\\\/MJsXU8wM+59ZReN0sC2fDUB9Ev9u4FPy2cmaP+Ig87tqRdRQtaFXtaGrNkny43ygYYvAUG5wjs+BYcCH2l6Zx\\\\\\\/NLskbOkL1eFp8+QTDVWIz5fdnxQTPbOI5PuaVwT+6tEbwti9oSVGO8UpXaR8CyQrK0+gQTIoMHff5KBsFMjxsiu4ylgiARV9qYr4ceKSaSt0b23eUhwem9DW1EJCi\\\\\\\/sS2LI0UAz0r3petxITEjvXgt+YIrV6Mwj\\\\\\\/ttmaxDchOUExz4nZ\\\\\\\/2iB+iL6KI3v2Una7tOg8hTTVHYneFxFWSdV5YjOtXj2sD+glB6lvhYNdFH9G99NT4OFdwQKl8VbLoZwfTH7aKUpBWyfY6z7hZN0T261DeZR4\\\\\\\/ieD12r526ZkZe5XPbI62WoyxR1ThM04Q79OGA68BOwaN2ebl\\\\\\\/ZdraQZzz0hTNiiTp6pORFWzFQs9Lh92pIXwwdrpbyOWcNMcdQahf5VG2oSUksc+boHCtwRGny8CUP5R\\\\\\\/eNibY6rS7n9RynGIJIhEUp9lUxW4D9CxycVS+v1UXY+iR\\\\\\\/LqUP74oO7vpr3sPMUFnpOQohj5SZQ+pDQX3+SJqKRQQRywreWx\\\\\\\/VSI+w8Q7eHtTAJWZpHIWmDLubaqZkglT5A3A8US1HfOnL8eMdWv1TS98ObIsC8BROzQv3sui1JMElW8o27NHkY04JklYMgHkdM3\\\\\\\/IjfXaGVTZ2SUTtNYt0u56ujVeXV14t7MczuASsgAKDw4zc5zaAAnQLvDu5MwpPA+QMV8qOqYr8maIXxA4uAPsmXfIqxPJwVL8eylpiu+ThowmaSb+hkoABj\\\\\\\/41iN7yw2uMll5bRzqJjYH38\\\\\\\/fdP+yEsvk9OTDVnA\\\\\\\/bfC4O\\\\\\\/idaqb4jtIIW+DhY64YGlS6ubFtWZQoCsWxSkjhE8XW0CpQxJ5mcwxolYVFGW28xMZU2GPNDlmbwATm2l+Vqer\\\\\\\/h7Uv+NV4n+hDGRp3Ym1FTC\\\\\\\/W3tchYthUfh8s277VaKKkHtGnlYQr5aDaw\\\\\\\/c77rrAwwkmap8lINEWWbr+eKdIf\\\\\\\/w+3igmoj7qhxZcG\\\\\\\/\\\\\\\/fBYb21I5w1sWggow8UVNMKGgTrmL061L6hSd5gTUCgNykfxtFYNwUM3A2DcUCjBQEp0\\\\\\\/ocSVAtuix7s4DooVdjXaAM4JHQNh+Ssv3qnknlyjntlzs+33V8scUWzxRb+lKS5gValkzBE7T4BMhjlfKBkcQZqBSwhd8OwjmlvkPCEX3Br\\\\\\\/maVUtF1v10VJKR5YGwk\\\\\\\/I+\\\\\\\/ptbREU+eCOKHz3Orvm3waGGofFG1f60bxgxuSx29tfeu6ZgJX31n3J3Ucl8pZgPLdbKfVTP8umld2Pg1ysSlHNmGCuYrhx7cVWqIuSFBye+QYnsDhNngHrB1B7ZWf53FvbufAwgrfH75izUluYo+xAOZTIkoefF70xAHuwFKygrCwMef1QGkT2URe2lWDBDdqAZbhLBhtoenhUXCfb+YdBBmdP6+AhHwxvBzaSMXzctO\\\\\\\/dhf0KU\\\\\\\/wof2KpE0nmtOTpJvION1jCe2YMdedFJBreiiUcMGAZ7tguvGxSOVj\\\\\\\/yGfHrb\\\\\\\/sjNG7+owrk43KWl4uM6FfXtR1eV60LXMxuMBHyJDU8SyYQa6gPPsn7ZMSwhzeXw9Nem5FAiLhRl0A3LUiWGanZPfOM1jj+Bsd01KBHP15F\\\\\\\/6bMaUs5cmrt9tRSjuIZ4Rwv5atE0UIkTPyw4jjZZXS\\\\\\\/U3LPiZjGWNJHq859YDyraINjRrNg0GAzJHEqnOT4oEiOgFG7eMdOBZACg\\\\\\\/XXXGBHxCB2XidqnVSHuVmQ8BfCZ+0YRnm0HZZIOBn9l6+5dnT0f5OBhfZdnKK+C3bIjZPakf\\\\\\\/6hz8s+6HE4p\\\\\\\/sg4mSRe3B7VylNxQWNFoKmA1lhYn52hOhpMxwhOOyJBqL3Q66oldtVACv8LaN8P4AWpRmqZ25Z1M9DGvsbduTpXF8sDv4Y9fA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022O7yE1ArG2vx5jsNrKbQzMQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229ba29d7152fe0d86f00a985536bfffc5ffdb90c2325254aa49be834682a5cb00\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459234 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/2/223" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1mnG_46EOCxDIg8SXiJGiPa_BCf4QKY52RFPdMsGJ7s" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/2/223" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fSGsYChnGS_jco0Xaz5MfbKOPT92P5pW3_oqHf8PFPw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/2/213" title="Expression Quantitative Trait Loci for Extreme Host Response to Influenza A in Pre-Collaborative Cross Mice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/2/235" title="Extent With Modification: Leg Patterning in the Beetle Tribolium castaneum and the Evolution of Serial Homologs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459234" id="node459234" data-pisa="ggg;2/2/223" data-pisa-master="ggg;2/2/223" data-apath="/ggg/2/2/223.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Resource of Quantitative Functional Annotation for <em>Homo sapiens</em> Genes</h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Murat Taşan</span>, <span class="highwire-citation-author" data-delta="1">Harold J. Drabkin</span>, <span class="highwire-citation-author" data-delta="2">John E. Beaver</span>, <span class="highwire-citation-author" data-delta="3">Hon Nian Chua</span>, <span class="highwire-citation-author" data-delta="4">Julie Dunham</span>, <span class="highwire-citation-author" data-delta="5">Weidong Tian</span>, <span class="highwire-citation-author" data-delta="6">Judith A. Blake</span> and <span class="highwire-citation-author" data-delta="7">Frederick P. Roth</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">223-233;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000828 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459234" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Murat Taşan</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Donnelly Centre for Cellular &amp; Biomolecular Research, University of Toronto, Toronto, Ontario M5S-3E1, Canada</span><span class="nlm-aff">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Murat%2BTa%C5%9Fan" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ta%C5%9Fan%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMurat%2BTa%25C5%259Fan" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Harold J. Drabkin</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Mouse Genome Informatics, The Jackson Laboratory, Bar Harbor, Maine 04609</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harold%2BJ.%2BDrabkin" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Drabkin%20HJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHarold%2BJ.%2BDrabkin" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John E. Beaver</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BE.%2BBeaver" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beaver%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BE.%2BBeaver" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hon Nian Chua</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Donnelly Centre for Cellular &amp; Biomolecular Research, University of Toronto, Toronto, Ontario M5S-3E1, Canada</span><span class="nlm-aff">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hon%2BNian%2BChua" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chua%20HN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHon%2BNian%2BChua" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Julie Dunham</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Donnelly Centre for Cellular &amp; Biomolecular Research, University of Toronto, Toronto, Ontario M5S-3E1, Canada</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julie%2BDunham" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulie%2BDunham" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Weidong Tian</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institute of Biostatistics, School of Life Sciences, Fudan University, Shanghai 200433, P. R. China</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Weidong%2BTian" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tian%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWeidong%2BTian" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Judith A. Blake</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Mouse Genome Informatics, The Jackson Laboratory, Bar Harbor, Maine 04609</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Judith%2BA.%2BBlake" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blake%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJudith%2BA.%2BBlake" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Frederick P. Roth</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Donnelly Centre for Cellular &amp; Biomolecular Research, University of Toronto, Toronto, Ontario M5S-3E1, Canada</span><span class="nlm-aff">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115</span><span class="nlm-aff">Center for Cancer Systems Biology, Dana Farber Cancer Institute, Boston, Massachusetts 02115</span><span class="nlm-aff">Samuel Lunenfeld Research Institute, Mt. Sinai Hospital, Toronto, Ontario M5G-1X5, Canada</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frederick%2BP.%2BRoth" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Roth%20FP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrederick%2BP.%2BRoth" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fritz.roth@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">fritz.roth@utoronto.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/2/223" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459234" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459234/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/2/223.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459234" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459234/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/2/223.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459234" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459234/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/2/223.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459234" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459234/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/2/223.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459234" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459234/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">The body of human genomic and proteomic evidence continues to grow at ever-increasing rates, while annotation efforts struggle to keep pace. A surprisingly small fraction of human genes have clear, documented associations with specific functions, and new functions continue to be found for characterized genes. Here we assembled an integrated collection of diverse genomic and proteomic data for 21,341 human genes and make quantitative associations of each to 4333 Gene Ontology terms. We combined guilt-by-profiling and guilt-by-association approaches to exploit features unique to the data types. Performance was evaluated by cross-validation, prospective validation, and by manual evaluation with the biological literature. Functional-linkage networks were also constructed, and their utility was demonstrated by identifying candidate genes related to a glioma FLN using a seed network from genome-wide association studies. Our annotations are presented—alongside existing validated annotations—in a publicly accessible and searchable web interface.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/function-prediction" class="hw-term hw-article-keyword hw-article-keyword-function-prediction" rel="nofollow">function prediction</a></li><li class="kwd"><a href="/keyword/gene-function-human" class="hw-term hw-article-keyword hw-article-keyword-gene-function-human" rel="nofollow">gene function human</a></li><li class="kwd"><a href="/keyword/gene-ontology" class="hw-term hw-article-keyword hw-article-keyword-gene-ontology" rel="nofollow">Gene Ontology</a></li><li class="kwd"><a href="/keyword/machine-learning" class="hw-term hw-article-keyword hw-article-keyword-machine-learning" rel="nofollow">machine learning</a></li></ul><p id="p-5">An abundance of high-throughput laboratory techniques and computational methods has led to a deluge of human genomic and proteomic data (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bieri <em>et al.</em> 2007</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Crosby <em>et al.</em> 2007</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Eppig <em>et al.</em> 2007</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Nash <em>et al.</em> 2007</a>). Even when an individual researcher can assemble all available evidence, they are left with the task of weighing this evidence to infer likely functions for genes of interest.</p><p id="p-6">Fewer than one-third of human genes have a Gene Ontology (GO) annotation based on evidence derived from specific study of that gene (as opposed to prediction; see <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>), providing little guidance to researchers wanting to investigate sparsely annotated genes. Computational integration of diverse evidence can help in assigning function, and ideally these inferences can reflect the shades of gray in our current knowledge, as opposed to the “black or white” annotation that is most appropriate for archival annotation.</p><p id="p-7">Automated annotation has traditionally relied on sequence similarity to transfer function from one gene to another, but homology-based approaches fail in the common scenario in which a gene/function relationship is unknown or poorly characterized in all detectable homologs. Toward complementary approaches, integration of diverse data types have been adopted for use in making predictions of gene function (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Troyanskaya <em>et al.</em> 2003</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Deng <em>et al.</em> 2004</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Joshi <em>et al.</em> 2004</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Karaoz <em>et al.</em> 2004</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lanckriet <em>et al.</em> 2004</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Wong <em>et al.</em> 2004</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Huttenhower <em>et al.</em> 2009</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Linghu <em>et al.</em> 2009</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bologna <em>et al.</em> 2011</a>) and modeling term-to-term relationships [(<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">King <em>et al.</em> 2003</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Sokolov and Ben-Hur 2010</a>); see refs. (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Murali <em>et al.</em> 2006</a>) and (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Huttenhower and Hofmann 2010</a>) for general reviews on the concepts]. For example, one large effort (‘MouseFunc’) was undertaken by a consortium of research groups to compare function prediction methods using standardized training and testing datasets for <em>Mus musculus</em> (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Pena-Castillo <em>et al.</em> 2008</a>).</p><p id="p-8">Here we integrated an extensive set of <em>Homo sapiens</em> data and inferred quantitative associations between 21,341 human genes (12,925 of which had no existing associations on the basis of direct–ie, nonpredicted–evidence) and each of 4333 GO terms. Our models exploit both gene features and gene−gene relationships by using both guilt-by-profiling (GBP) and guilt-by-association (GBA) approaches to function prediction (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Taşan <em>et al.</em> 2008</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Tian <em>et al.</em> 2008</a>). We provide estimates of our models’ accuracy, including a prospective evaluation in which we consider annotations that were made after our training data set was assembled. Literature-based follow-up investigations are performed for a sample of high-confidence novel predictions. In the course of making these predictions, we constructed multiple functional linkage networks [FLNs–where an edge between two genes indicates some level of shared function (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lee <em>et al.</em> 2004</a>)], capturing different categories of biological relationships. We find that FLNs are independently useful, which we illustrate here by identifying candidate glioma-related genes given only “seed” glioma-related genes identified from systematic unbiased genome-wide association (GWA) studies. All gene/term prediction scores from this project—as well as our FLNs—are made freely available to the public via a web-accessible resource (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beaver <em>et al.</em> 2010</a>), which has been adapted here to host quantitative function annotations for <em>H. sapiens</em>.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Genomic data integration</h3><div id="sec-3" class="subsection"><h4>Genes:</h4><p id="p-9">We used 36,396 Ensembl Gene IDs as the initial set of identifiers for <em>H. sapiens</em> genes, which was further reduced to 21,341 genes as described in the section <em>Predictive features</em>. All mappings between alternative identifiers and Ensembl Gene IDs were performed using the Synergizer translation resource (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Berriz and Roth 2008</a>).</p></div><div id="sec-4" class="subsection"><h4>Predictive features:</h4><p id="p-10">We downloaded 6121 protein domain signatures from Interpro (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Mulder <em>et al.</em> 2005</a>) that could be associated with any one of the downloaded Ensembl Gene IDs. We also identified 2542 disease descriptions from Online Mendelian Inheritance in Man [OMIM (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hamosh <em>et al.</em> 2005</a>)] that were assigned to Ensembl Gene IDs. Phenotype terms from the Mammalian Phenotype Ontology (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Smith <em>et al.</em> 2005</a>) were downloaded and transferred from <em>M. musculus</em> to human via a homology mapping as provided by Mouse Genome Informatics (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Eppig <em>et al.</em> 2007</a>); because many of the phenotype terms are quite specific and have a nearly one-to-one mapping to GO BP terms, we limited our use of phenotype terms to only those at the top level of the phenotype ontology: 34 such terms mapped to our Ensembl Gene IDs. Phylogenetic profiles were built for each gene on the basis of ortholog presence in the genome of any of 38 possible organisms, as determined by the default threshold used by InParanoid (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">O’Brien <em>et al.</em> 2005</a>). The 21,341 genes used in this study had at least one “gene-centric” feature listed previously.</p><p id="p-11">Correlation coefficients were computed for pairs of genes using all Affymetrix HG-U133A (PL96) GEO (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barrett <em>et al.</em> 2009</a>) datasets for which CEL files were available (97 datasets in total). Each dataset was preprocessed separately using GCRMA background correction, normalization, and probeset summarization (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Wu <em>et al.</em> 2004</a>), with probeset-to-gene mappings provided by BrainArray Custom CDFs (Version 11) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dai <em>et al.</em> 2005</a>). Minimum-jackknife-1 Pearson correlation coefficients were computed for each gene-pair in each of the 97 datasets. The 35,948 protein interactions downloaded from HPRD (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Keshava Prasad <em>et al.</em> 2009</a>) were mapped to Ensembl Gene IDs; filtering of this dataset to remove homo-dimers left 33,811 protein-protein interactions within our gene set.</p></div><div id="sec-5" class="subsection"><h4>GO terms and associations:</h4><p id="p-12">Our models were trained using the human GO associations [from the GOA project (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Barrell <em>et al.</em> 2009</a>)] from August 2008, with associations from May 2009 used for the prospective evaluation. All IEA associations were removed, and the remaining associations were up-propagated through the GO graph. Only those 4333 GO terms with at least 3 and at most 300 distinct (non-IEA) gene associations were included for this study. We used the 8416 genes containing some non-IEA association to any one of these terms as the training set of genes, with the remaining 12925 genes being the undetermined set. Terms were divided into 12 categories, as described previously.</p></div></div><div id="sec-6" class="subsection"><h3>Model construction and evaluation</h3><div id="sec-7" class="subsection"><h4>GBP models:</h4><p id="p-13">In total, we used 8735 gene-centric predictive features: 6121 distinct InterPro IDs, 2542 OMIM Morbid Map IDs, 34 Mammalian Phenotype Ontology IDs, and 38 species used to build phylogenetic profiles. Collectively, we call these the GBP feature set.</p><p id="p-14">For each GO term, we trained a separate random forest classifier as an ensemble of 100 recursive partitioning or decision tree classifiers, using the GBP feature set and the training set of genes as gold-standard positive examples. The classifiers produced scores for each gene/term combination: predictive scores for undetermined genes, and out-of-bag (OOB) scores<sup>2</sup> for training genes. Precision-vs-recall performance using OOB scores was calculated for each GO term, then combined by term category to obtain aggregate measures of performance over all GO terms (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000828/-/DC1/0008282SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000828/-/DC1/FigureS1.pdf">Figure S1</a>).</p></div><div id="sec-8" class="subsection"><h4>GBA models:</h4><p id="p-15">Twelve GBA classifiers were built (one for each GO term category), with each producing a separate FLN that provides scores to all pairs of genes (<em>i.e.</em> <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/223/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/223/embed/mml-math-1.gif"/></noscript></span></span> gene-pair scores). Here again we used random forests, but in this case the set of objects to be scored consisted of gene pairs (rather than genes).</p><p id="p-16">Among the 8416 training-set genes, gene pairs that shared any GO term within the same category were used as positive training examples of functionally-linked genes. To illustrate by example, our BP [31,100] FLN was constructed using gene pairs that shared any one (or more) associations to any BP term with fewer than 101 (non-IEA) total associations. Such pairs were deemed functionally linked and given a score of “1” in the training network. All remaining pairs among the 8416 training genes were considered <em>not</em> to be functionally linked (with respect to the current GO term category) and were scored as “0” in the training network. Predictions of edge weights for all other pairs of genes (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/223/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/223/embed/mml-math-2.gif"/></noscript></span></span> pairs) are then made using random forest models trained on these reference networks, and the <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/223/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/223/embed/mml-math-3.gif"/></noscript></span></span> OOB edge scores for the training edges are used for assessing model performance (described below).</p><p id="p-17">For each of our Interpro, OMIM, Mammalian Phenotype Ontology, and phylogenetic tree datasets, shared patterns of annotation between pairs of genes were scored using a method that is functionally similar to the PhenoBlast score used previously for measuring similarity between pairs of <em>Caenorhabditis elegans</em> strains (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gunsalus <em>et al.</em> 2004</a>). The score between two genes is computed as <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/223/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/223/embed/mml-math-4.gif"/></noscript></span></span>, where <em>f<sub>i</sub></em> is the frequency of the <em>i</em>-th feature shared by both genes. Each such score was then discretized by placing gene-pairs into bins corresponding to the membership in the top 10%, 20%, 30%, 40%, and 50% quantiles. Thus, each of these datasets led to five binary features used in training our functional-linkage models.</p><p id="p-18">Already in binary form, protein−protein interactions from Human Protein Reference Database (HPRD) are included as a predictive feature. Each expression dataset was also discretized by using the top 1%, 5%, and 10% quantiles, leading to three binary features per expression dataset.</p><p id="p-19">To make individual gene/term predictions from our FLNs, we use a probabilistic approach. For GO terms in evaluation categories with association counts in the ranges [11, 30], [31, 100], and [101, 300], we compute gene/term scores one term at a time, starting by identifying the “core set” of genes that have positive associations with the current GO term. A probability density function is constructed from the edge weights between all pairs of genes in the core set (forming a clique), using a Gaussian kernel density estimation (the “core-to-core” distribution). A “pseudo-count-like” edge weight uniformly distributed over [0,1] is added to this collection to prevent the density estimators from dropping to zero during numerical computations with a reasonable kernel bandwidth. Similarly, the weights between all pairs of noncore training genes and the core set of genes are used to estimate a “noncore-to-core” distribution. Finally, a gene not in the training set is scored (for the current GO term) by identifying all edges between that gene and core set genes; the log-likelihood ratio (LLR) of these edges having arisen from the “core-to-core” relative to the “noncore-to-core” distributions is then computed using the estimated probability density functions.</p><p id="p-20">For GO terms with 10 or fewer gene associations, the process is similar to that described, except that “core-to-core” and “noncore-to-core” distributions are estimated from core-to-core and noncore-to-core gene-pair sets pooled across all GO terms in the evaluation category. We selected this method after empirical observation that the limited number of core genes for many such GO terms led to poor density function estimation (results not shown).</p><p id="p-21">Performance of annotation transfer by LLRs was estimated using a leave-one-out (LOO) approach: each gene in the training set was scored using the OOB edge weights computed during random forest construction, and distributions were re-estimated after excluding all edges incident to the current held-out gene. The transferred LOO scores for each GO term were then evaluated similarly to the scores produced by the GBP models (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000828/-/DC1/FigureS2.pdf">Figure S2</a>).</p></div><div id="sec-9" class="subsection"><h4>Combined models:</h4><p id="p-22">We combined the GBA and GBP scores for each gene/term via logistic regression, referring to the combined scores as “HF” scores. The general form of the combination is:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/223/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/223/embed/mml-math-5.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/223/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/223/embed/mml-math-6.gif"/></noscript></span></span> is the posterior HF log-odds and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/223/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/223/embed/mml-math-7.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/223/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/223/embed/mml-math-8.gif"/></noscript></span></span> are the log-likelihood ratios of the GBP and GBA models, all for each gene <em>i</em>. <em>LO<sup>prior</sup></em> is the prior log-odds of a gene being associated with a particular term (invariant across genes for each term).</p><p id="p-23">The α-term in the aforementioned equation is systematically optimized to maximize the LOO estimates of the area under the PR curve (average precision) for each term. For GO terms in evaluation categories with association counts in the ranges [11,30], [31,100], and [101,300], the α coefficients are determined separately for each GO term. To avoid overfitting the coefficients for the GO terms in evaluation categories in the range [3,10], a single coefficient is found for all GO terms within the category (<em>i.e.</em> α<sub>BP[3,10],</sub> α<sub>CC[3,10],</sub> α<sub>MF[3,10],</sub>).</p></div></div><div id="sec-10" class="subsection"><h3>Prospective evaluation</h3><p id="p-24">For prospective evaluation, each term was evaluated individually (as mentioned previously), but with changes to the set of genes used for the evaluation. First, all existing positive associations made prior to the start date were removed from each term’s list of gene scores. Second, only those genes that received <em>any</em> association (<em>i.e.</em>, to any other term in the GO) in the time since the start date were retained; there were 1170 such genes. The idea here is that any new association for a gene made in the prospective time window indicates that this gene had some curation done and thus had the opportunity to be newly associated with at least one term (and possibly more). Genes not receiving any new association in that time window were not considered for prospective validation. Aggregate prospective performance across all GO terms is shown in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>.</p></div><div id="sec-11" class="subsection"><h3>Selection of 36 predictions for literature follow-up</h3><p id="p-25">Three gene/term pairs from each of the term categories were chosen for a literature-based follow-up. We considered only gene/term pairs for which the gene had no known or predicted (<em>i.e.</em>, IEA-based) function within the same GO branch—we call these “novel” predictions. The top three HF scores within each term category after this filtering were then chosen. A list of these 36 predictions, along with their assessments and literature references, is available as Supplementary Information.</p></div><div id="sec-12" class="subsection"><h3>Microarray differential expression analysis</h3><p id="p-26">When evaluating differential expression for our glioma network follow-up, we used Welch’s two sample <em>t</em>-tests, followed by a multiple-testing correction (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Benjamini and Hochberg 1995</a>). We note that the <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bredel <em>et al.</em> (2005)</a> dataset was not included as part of our microarray co-expression data set used for training our FLNs (to address circularity in the discovery process).</p></div><div id="sec-13" class="subsection"><h3>Computational tools</h3><p id="p-27">Random forest code was written in Java for this study, and all statistical analyses (<em>e.g.</em>, model performance estimates) were performed using R. All software used in generating this resource is freely available upon request.</p></div></div><div class="section results" id="sec-14"><h2 class="">Results</h2><div id="sec-15" class="subsection"><h3>Data integration</h3><p id="p-28">All data collected were from public sources and fell into the two broad classes of gene property and gene-pair relationship (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), and 4333 GO terms were chosen for classification (see <em>Materials and Methods</em> for a description of the selection criteria). Terms were separated into 12 distinct term categories on the basis of the aspect of function it describes, that is, biological process (BP), cellular component, or molecular function (MF), and its breadth, that is, the number of unique genes to which it has been assigned ([3,10], [11,30], [31,100], or [101,300] non-IEA associations, following the paradigm introduced in <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Pena-Castillo <em>et al.</em> 2008</a>, <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). These distinctions allow us to separately study inferences to GO terms of similar “type” and permit an understanding of how performance differs among these types. Gene features and GO term associations were gathered in 2008, with the prospective evaluation reported here based on term associations made approximately 1 year later (2009). All associations with IEA evidence codes were removed from this study (forcing our models to rely on ground truth rather than on previous predictions), with the remaining association types used for model training being distributed as shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/223/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="General strategy for the combined GBP and GBA approach. (GBP: guilt-by-profiling, GBA: guilt-by-association, HF: human function, FLN: functional-linkage network.)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-860623627" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;General strategy for the combined GBP and GBA approach. (GBP: guilt-by-profiling, GBA: guilt-by-association, HF: human function, FLN: functional-linkage network.)&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/223/F1.medium.gif" width="379" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/2/223/F1.medium.gif" width="379" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/223/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/223/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461334" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-29" class="first-child">General strategy for the combined GBP and GBA approach. (GBP: guilt-by-profiling, GBA: guilt-by-association, HF: human function, FLN: functional-linkage network.)</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/223/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distinct Ensembl Gene IDs grouped into broad classes depending on status of association with at least one GO term." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-860623627" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distinct Ensembl Gene IDs grouped into broad classes depending on status of association with at least one GO term.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/223/F2.medium.gif" width="440" height="246"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/2/223/F2.medium.gif" width="440" height="246"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/223/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/223/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461312" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-30" class="first-child">Distinct Ensembl Gene IDs grouped into broad classes depending on status of association with at least one GO term.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461235/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461235/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 </span> <span class="caption-title">Number of distinct genes (21,341 total genes) with at least one “gene-centric” feature attributed to it for each predictive data type</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461226/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461226/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2 </span> <span class="caption-title">Number of distinct genes (of 21,341 total genes) appearing in at least one “gene-pair-centric” data type</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461229/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461229/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3 </span> <span class="caption-title">Number of terms in each of the 12 evaluation categories</span><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/223/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Counts of association types between Ensembl Gene IDs and GO terms (by GO evidence code). The barplot represents the evidence code distribution within the smaller (non-IEA) slice of the pie." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-860623627" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Counts of association types between Ensembl Gene IDs and GO terms (by GO evidence code). The barplot represents the evidence code distribution within the smaller (non-IEA) slice of the pie.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/223/F3.medium.gif" width="440" height="433"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/2/2/223/F3.medium.gif" width="440" height="433"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/223/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/223/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461299" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-34" class="first-child">Counts of association types between Ensembl Gene IDs and GO terms (by GO evidence code). The barplot represents the evidence code distribution within the smaller (non-IEA) slice of the pie.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Quantitative association of 21,341 <em>H. sapiens</em> genes to 4333 GO terms</h3><p id="p-35">Predictions of gene/term associations were made by combining GBP and GBA approaches. Quantitative scores describing the strength of association between each GO term and human gene were computed (for a total of 4333⋅21,341 ~ 92 × 10<sup>6</sup> gene/term scores).</p><p id="p-36">In the GBP approach (relying on “gene-centric” data), a separate random forest classifier (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Breiman 2001</a>) was constructed for each GO term, providing direct gene/term predictions that we call the “GBP set.” The GBA approach is complementary to GBP in that it focuses on “transferring” annotation from one gene to another via evidence of a biological relationship between the genes. To this end, data best represented as “edge-centric” (<em>e.g.</em>, protein−protein interactions) were used to construct FLNs, graphs describing relationships (edges) between genes (vertices), where edge weights reflect confidence levels of two genes sharing a function (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Lee <em>et al.</em> 2004</a>). We constructed 12 probabilistic FLN networks (one for each of the aforementioned 12 categories), also using random forests (one per FLN). Associations between each gene <em>g</em> and each term <em>t</em> were then transferred probabilistically by examining the strength of connection between <em>g</em> and a ‘core set’ of genes already annotated with term <em>t</em> (details in <em>Materials and Methods</em>). The collection of gene/term scores derived from this process is called the “GBA set.”</p><p id="p-37">Finally, the GBP and GBA scores were combined for each GO term via a logistic regression model optimized to maximize precision <em>vs.</em> recall (PR) performance (see <em>Materials and Methods</em>), resulting in the human function (HF) set of quantitative confidence scores.</p></div><div id="sec-17" class="subsection"><h3>Predictive performance</h3><p id="p-38">Our predictions were evaluated (using a cross-validation scheme) for each term, and for all three sets of scores (GBP, GBA, and HF). To obtain a global view of performance, evaluation measures were aggregated into the 12 term categories and illustrated using contour analysis that shows the characteristic trade-off of accuracy <em>vs.</em> recovery for predictive models (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000828/-/DC1/FigureS1.pdf">Figure S1</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000828/-/DC1/FigureS2.pdf">Figure S2</a>). For a given score threshold τ, <em>precision</em> is the proportion of (known) “true” gene/term pairs among all gene/term pairs scoring above τ; <em>recall</em> is the proportion of “true” gene/term pairs scoring above τ among all possible “true” gene/term pairs. The combination of GBP and GBA scoring methods provide an increase in average precision for many different term types (insets of <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). We also find that our performance (in terms of precision) compares favorably with the best performance achieved from the MouseFunc study (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>), with mean precision at 20% recall (P20R) ranging between approximately 20% and 70%.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/223/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Aggregated HF performance for each of the 12 GO term categories. Contour lines indicate the fraction of GO terms that met or exceeded each cumulative precision vs. recall performance point. Contours are shown for median GO term (50% contour; heavy blue dashed line) and every other decile of GO terms. The mean performance (heavy blue solid line is also shown. Gray area exceeds performance of all classifiers. Red and green lines indicate median and mean performances (dashed and solid, respectively) for GBP and GBA classifiers. The median of average cumulative precision across all classifiers in each category is shown in inset (error bars show standard error of the median)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-860623627" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Aggregated HF performance for each of the 12 GO term categories. Contour lines indicate the fraction of GO terms that met or exceeded each cumulative precision &lt;em&gt;vs.&lt;/em&gt; recall performance point. Contours are shown for median GO term (50% contour; heavy blue dashed line) and every other decile of GO terms. The mean performance (heavy blue solid line is also shown. Gray area exceeds performance of all classifiers. Red and green lines indicate median and mean performances (dashed and solid, respectively) for GBP and GBA classifiers. The median of average cumulative precision across all classifiers in each category is shown in inset (error bars show standard error of the median).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/223/F4.medium.gif" width="332" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/2/2/223/F4.medium.gif" width="332" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/223/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/223/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461267" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-39" class="first-child">Aggregated HF performance for each of the 12 GO term categories. Contour lines indicate the fraction of GO terms that met or exceeded each cumulative precision <em>vs.</em> recall performance point. Contours are shown for median GO term (50% contour; heavy blue dashed line) and every other decile of GO terms. The mean performance (heavy blue solid line is also shown. Gray area exceeds performance of all classifiers. Red and green lines indicate median and mean performances (dashed and solid, respectively) for GBP and GBA classifiers. The median of average cumulative precision across all classifiers in each category is shown in inset (error bars show standard error of the median).</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/223/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparing current performance with previous mouse gene function predictions. Precision at 20% recall for best-performing groups of the MouseFunc project (Pena-Castillo et al. 2008) was compared with HF scores in this study. Differences reflect a combination of both algorithms and datasets used but indicate that overall prediction performance is at least comparable and for some term categories represents a substantial improvement. Error bars indicate standard error of the mean; variance across terms for MouseFunc not available, preventing error bar computation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-860623627" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparing current performance with previous mouse gene function predictions. Precision at 20% recall for best-performing groups of the MouseFunc project (Pena-Castillo &lt;em&gt;et al.&lt;/em&gt; 2008) was compared with HF scores in this study. Differences reflect a combination of both algorithms and datasets used but indicate that overall prediction performance is at least comparable and for some term categories represents a substantial improvement. Error bars indicate standard error of the mean; variance across terms for MouseFunc not available, preventing error bar computation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/223/F5.medium.gif" width="440" height="438"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/2/2/223/F5.medium.gif" width="440" height="438"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/223/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/223/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461341" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-40" class="first-child">Comparing current performance with previous mouse gene function predictions. Precision at 20% recall for best-performing groups of the MouseFunc project (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Pena-Castillo <em>et al.</em> 2008</a>) was compared with HF scores in this study. Differences reflect a combination of both algorithms and datasets used but indicate that overall prediction performance is at least comparable and for some term categories represents a substantial improvement. Error bars indicate standard error of the mean; variance across terms for MouseFunc not available, preventing error bar computation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">Our HF predictions were also evaluated <em>prospectively</em> using gene/term associations made after the start date. Because all genes will not have been studied in this time window (and thus not given the opportunity to gain new annotations), we evaluated performance for the subset of 1170 genes that gained at least one new GO annotation during this period (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). Recall for each term was calculated using only gene/term pairs recently (<em>i.e.</em>, in the time window) assigned to that term. For lower recall levels (<em>e.g.</em>, &lt;20%), our precision generally ranged between 20% and 40%.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/223/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prospective evaluation of precision vs. recall. Predictions made on July 19, 2008, evaluated on new associations made between July 19, 2008, and May 28, 2009. Dashed lines indicate each 10% contour, heavy dashed line is median (50% contour), and the heavy blue solid line is mean performance. Contours indicate what fraction of classifiers in the evaluation category exceeded the shown performance. Gray area exceeds performance of all classifiers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-860623627" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Prospective evaluation of precision &lt;em&gt;vs.&lt;/em&gt; recall. Predictions made on July 19, 2008, evaluated on new associations made between July 19, 2008, and May 28, 2009. Dashed lines indicate each 10% contour, heavy dashed line is median (50% contour), and the heavy blue solid line is mean performance. Contours indicate what fraction of classifiers in the evaluation category exceeded the shown performance. Gray area exceeds performance of all classifiers.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/223/F6.medium.gif" width="330" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/2/2/223/F6.medium.gif" width="330" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/223/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/223/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461239" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 </span> <p id="p-42" class="first-child">Prospective evaluation of precision <em>vs.</em> recall. Predictions made on July 19, 2008, evaluated on new associations made between July 19, 2008, and May 28, 2009. Dashed lines indicate each 10% contour, heavy dashed line is median (50% contour), and the heavy blue solid line is mean performance. Contours indicate what fraction of classifiers in the evaluation category exceeded the shown performance. Gray area exceeds performance of all classifiers.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Literature-based validation of top novel predictions</h3><p id="p-43">Having evaluated the performance of quantitative annotations by cross-validation and prospective analysis, we wished to more carefully examine the most interesting HF predictions, that is, those that are both novel and most likely to be correct. To restrict ourselves to the most novel predictions, we excluded any predictions in which the gene had previously been annotated to any GO term in the same branch as the predicted GO term. We further excluded gene/term pairs that had previously been predicted (on the basis of annotation with an IEA evidence code). For each of the 12 GO categories, we examined literature related to the three top-scoring novel predictions, leading to 36 high-scoring gene/term predictions from the HF model for in-depth literature evaluation.</p><p id="p-44">Of the predictions examined, we deemed 14 to be “true” or “likely true” (9 and 5, respectively) given evidence available in the current body of published literature. The success rate corresponds to 14 of 36 (39%) and thus appears similar to our prospective estimates of precision (described previously). Of the remaining predictions, 14 were unclear, in that we found no existing literature that either supported or refuted the prediction. These predictions can be viewed as hypotheses for further study. The final 8 predictions were either “likely false” (6) or “false” (2). (A full list of the 36 literature-evaluated predictions and chosen categories, with PubMed IDs, where available, is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000828/-/DC1">File S1</a> for this article.)</p></div><div id="sec-19" class="subsection"><h3>FLNs for functional gene set expansion</h3><p id="p-45">In addition to their contribution toward predicted gene/term associations, FLNs have previously been shown to be valuable tools for elucidating relationships between genes [<em>e.g.</em>, predicting genetic interactions via patterns of functional linkage (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lee <em>et al.</em> 2010</a>)]. We further find that FLNs are useful for identifying disease-associated candidates by extracting subsets of genes that were functionally linked with particular seed genes of interest.</p><p id="p-46">As an example of how “induced subgraphs” from FLNs can aid in annotation of genes even in the absence of any particular seed set of genes, we selected the single gene <em>POLR2J2</em>, which was found to be “likely true” in the aforementioned literature evaluation for association with the GO MF term “DNA-directed RNA polymerase complex” (GO:0000428). Considering only the edges scoring in the top 0.1% of the FLN for the term category BP [3,30], the 3 linked neighbors of <em>POLR2J2</em> are <em>POLR2J3</em>, <em>POLR2J</em>, and <em>POLR1C</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000828/-/DC1/FigureS3.pdf">Figure S3</a>). The nonseed genes with prefix <em>POLR2J</em> are each subunits of the RNA Pol II complex, and <em>POLR1C</em> is a subunit of the RNA Pol I complex. We note that <em>POLR2J2</em> has no cataloged interaction partners in the HPRD (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Keshava Prasad <em>et al.</em> 2009</a>), limiting the prospects of GBA via physical or genetic interactions alone. Although this is a case of an already named and characterized gene, it serves to illustrate the (1) automated identification of strong functional hypotheses and (2) identification of groups of additional genes enriched for functional relationships with a particular incompletely annotated gene of interest.</p><p id="p-47">To illustrate how one may use FLNs to identify novel candidate disease genes [an idea supported by recent similar work in ref. (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lee <em>et al.</em> 2011</a>)], we examined a seed set of candidate genes identified from a meta-analysis of GWA studies examining glioma susceptibility (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Shete <em>et al.</em> 2009</a>). Five of the six genes identified by the meta-analysis were among the 21,341 genes examined in our work, which we used as seeds in the BP [3,30] FLN with an edge-weight threshold retaining only the top 0.01% of edges. The resulting subgraph induced by this seed set of five glioma-associated genes recruited 21 additional genes (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). Genes common to many forms of cancer were among those found (<em>e.g.</em>, <em>BRCA2</em>, <em>TP53</em>, <em>CDK4</em>, <em>XRCC4</em>, none of which were seeds originating from the GWAS meta-analysis). Two seed-centered components (one from the seed <em>TERT</em>, the other from the seeds <em>CDKN2A</em> and <em>CDKN2B</em>) are connected via a functional link between the two nonseed genes <em>BRCA2</em> and <em>NBN</em>. This approach generalizes the strategy of inducing disease networks from known disease-associated seed genes using physical interaction data (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Pujana <em>et al.</em> 2007</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/223/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Illustrated use of the human FLN to identify additional candidate genes from a set of seed genes. This network was identified using five glioma genes identified in unbiased genome-wide association studies (in red). The FLN used was the top 0.01% of edges (in red) from the BP [3,30] FLN. To illustrate that the FLN extends beyond known protein interactions, all cataloged HPRD interactions between these 26 genes are shown as blue edges." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-860623627" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Illustrated use of the human FLN to identify additional candidate genes from a set of seed genes. This network was identified using five glioma genes identified in unbiased genome-wide association studies (in red). The FLN used was the top 0.01% of edges (in red) from the BP [3,30] FLN. To illustrate that the FLN extends beyond known protein interactions, all cataloged HPRD interactions between these 26 genes are shown as blue edges.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/223/F7.medium.gif" width="440" height="395"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7&#x2002;" src="https://www.g3journal.org/content/ggg/2/2/223/F7.medium.gif" width="440" height="395"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/223/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/223/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461337" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7 </span> <p id="p-48" class="first-child">Illustrated use of the human FLN to identify additional candidate genes from a set of seed genes. This network was identified using five glioma genes identified in unbiased genome-wide association studies (in red). The FLN used was the top 0.01% of edges (in red) from the BP [3,30] FLN. To illustrate that the FLN extends beyond known protein interactions, all cataloged HPRD interactions between these 26 genes are shown as blue edges.</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">To illustrate the greater potential for detection of related genes when FLNs are used, as opposed to using only protein interactions alone, we also display (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>) all interactions cataloged in HPRD between the 26 genes in our glioma network. Among interesting candidate genes retrieved using the FLN, the <em>NBN</em> gene had previously been postulated to have a role in nonglioma forms of cancer, and recent evidence has (independently from the GWA studies) implicated <em>NBN</em> variants in pediatric glioblastoma (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Piekutowska-Abramczuk <em>et al.</em> 2009</a>). Also of interest is the recovery of the <em>LRP5</em> gene, a low-density lipoprotein receptor-related protein that has been identified as potentially oncogenic (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Li and Bu 2005</a>) and part of the cancer-drug-targeted Wnt/beta-catenin signaling pathway (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Luu <em>et al.</em> 2004</a>). We note that both the <em>NBN</em> and <em>LRP5</em> genes would not have been recovered using the HPRD interaction network alone (with a “1-nearest-neighbor” expansion). To further analyze the value of our glioma network, we examined a microarray expression dataset from an experiment comparing (among other tumor types) glioblastoma samples and normal brain tissue [(<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Bredel <em>et al.</em> 2005</a>) NCBI GEO (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Barrett <em>et al.</em> 2009</a>) record GDS1813]. Of the 21 genes in our glioma network for which we could compute test statistics, we found that 10 show differential expression (FDR &lt; 0.1), including the <em>NBN</em> and <em>LRP5</em> genes, shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000828/-/DC1/FigureS4.pdf">Figure S4</a>.</p></div><div id="sec-20" class="subsection"><h3>Resource availability</h3><p id="p-50">Our predictions are made available as a resource to the community via an online web-accessible and searchable browser (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Beaver <em>et al.</em> 2010</a>), available via <a href="http://func.mshri.on.ca/human">http://func.mshri.on.ca/human</a>, with periodic data updates (trained on current gene annotations). Our FLNs, not included in the <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000828/-/DC1">File S1</a> because of file size constraints, are available for download at <a href="http://llama.mshri.on.ca/~mtasan/HumanFunc_publication_support">http://llama.mshri.on.ca/~mtasan/HumanFunc_publication_support</a>.</p></div></div><div class="section discussion" id="sec-21"><h2 class="">Discussion</h2><div id="sec-22" class="subsection"><h3>Combining GBP and GBA methods allows full data exploitation</h3><p id="p-51">Our quantitative gene function annotations combine both gene-centric and gene-pair relationship evidence. The contribution of relationship-based (GBA) scoring tends to increase as the terms become more specific. Given that gene-centric (GBP) models are trained separately for each GO term, it stands to reason that performance must degrade for GO terms with very few annotated genes to use as positive training examples. By contrast, FLNs are trained by aggregating information from many related GO terms. This lends support to the notion of using the FLNs for inferring specific functions. Also we find that terms in the MF (molecular function) branch of the GO vocabulary often rely more heavily on the GBP scores; this is likely to be due, at least in part, to the strong association between protein domain pattern signatures and specific biochemical functions.</p></div><div id="sec-23" class="subsection"><h3>Emphasis on high precision at low recall</h3><p id="p-52">We expect that a common use of this resource will be experimental biologists examining only the top few quantitative annotations for any gene or term to generate hypotheses for testing. Presentation of a ranked list of annotations, each in the context of underlying evidence, permits filtering based on (often extensive) user knowledge that was not modeled by our approach. Thus, our GBP and GBA predictions are combined to maximize average precision, which is largely driven by the accuracy of the top predictions.</p><div id="sec-24" class="subsection"><h4>Cross-validation vs. prospective performance evaluation:</h4><p id="p-53">Differences between the training performance estimates and the prospective estimates reflect both the bias in the state of the associations at the start date (<em>i.e.</em>, some genes and terms are better-studied than others) and the types of genes and terms recently curated by GOA. Even limiting the evaluated genes to those that have gained some new annotation in the time period, there are several reasons why one might expect to see lower precision-recall performance for the prospective evaluation as compared to that seen with cross-validation. First, each freshly-annotated gene gained an average of only 3.1 new GO term associations, which is lower than the total number of terms that we might ultimately expect to be added for each gene. Second, recently-annotated functions may be intrinsically harder to predict (<em>i.e.</em>, where the function is obvious, annotations will tend to occur at an earlier time). Third, recent advances may be on functions that were less well-studied previously, so that the training data are sparsely available or is not representative.</p></div></div><div id="sec-25" class="subsection"><h3>Variable importance</h3><p id="p-54">As part of our GBP process, we obtain a variable importance measure for each predictive feature, measuring that feature’s contribution to the performance of the classifier. These can assist in understanding <em>why</em> a prediction has been made, possibly suggesting a validation experiment or simply providing more general clues to help guide the researcher. Strong evidence of particular features leading to accurate gene/term associations may lead to the establishment of empirically derived annotation-transfer rules. As an example, the gene <em>RAET1L</em> was predicted to have MHC class I receptor activity, and based solely on the available literature, this prediction was deemed “unclear.” That two of the top three Interpro protein domain pattern signatures with high importance for predicting MHC class I receptor activity had been assigned to this gene (a MHC class I alpha chain pattern (IPR001039) and a MHC classes I and II-like antigen recognition pattern (IPR011162)) yields insight into why this prediction was made, and can guide curators in their decision on whether to make a GO term association for this gene. Such feature identification can also be used to help streamline future prediction efforts [by focusing efforts on those predictive features that are most informative for the task at hand (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Ko and Lee 2009</a>)].</p></div><div id="sec-26" class="subsection"><h3>A novel method for annotation transfer through FLNs</h3><p id="p-55">To derive GBA scores for each term, we use the edge weights of the appropriate FLN to transfer associations from genes currently associated with that term (the “core set”) to those genes with “unknown” association status. This process calculates, for each candidate gene in turn, a likelihood ratio that measures whether the distribution of weights connecting the candidate to the core set is more consistent with ‘core-to-core’ pairs (that are known to share the function) than ‘core-to-non-core’ pairs (that are not known to share the function; see <em>Material and Methods</em> for details). We have previously experimented with other forms (see <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Wang and Marcotte 2010</a> for a review) of association transfer from FLNs, specifically, using average weight of (the top-<em>n</em>) edges linking “unknown” genes to “core set” genes (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Tian <em>et al.</em> 2008</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Taşan <em>et al.</em> 2008</a>) and have found the likelihood-ratio approach applied here to be more robust and empirically better-performing than the ‘top-<em>n</em>’ approach in cross-validation evaluation (results not shown).</p></div><div id="sec-27" class="subsection"><h3>Utility of FLNs</h3><p id="p-56">Physical interaction networks have been shown to help transfer gene function (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Schwikowski <em>et al.</em> 2000</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Letovsky and Kasif 2003</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gunsalus <em>et al.</em> 2005</a>), and extending these approaches to include predicted <em>functional</em> interactions can drastically increase the power of discovery. The STRING database (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Szklarczyk <em>et al.</em> 2011</a>) is an example of a publicly accessible repository of predictions of functional interactions made through various evidence sources (<em>e.g.</em>, literature co-citation or existing protein−protein interaction reports). Our functional linkage networks differ from those in STRING in that we make predictions for all pairs of genes at different levels of resolution and with respect to different types of functional links (<em>e.g.</em>, BP—a shared role in some biological process <em>vs.</em> MF—sharing a particular molecular activity). When comparing the STRING (v9.0) interaction scores with those of each of our 12 FLNs, we see a significant correlation in each case (<em>P</em> &lt; 3 × 10<sup>−12</sup> for all 12 correlations measured). However, these correlations were relatively weak, with Pearson correlations ranging from <em>r</em> = 0.07 (category cellular component [3,10]) to <em>r</em> = 0.20 (category MF [3,300]). Although STRING provides highly reliable functional associations between genes, it is limited to only those pairs with strong scores (~3.3 million in STRING v9.0, as opposed to the ~200 million in each of the 12 FLNs described previously). A larger range of scores can provide additional information, for instance, establishing densities of scores for use in label propagation (as described in the section <em>A novel method for annotation transfer through FLNs</em>), thus we see a continued role for our FLNs in ongoing research.</p><p id="p-57">Biological networks can reveal previously unknown players in the etiology of a phenotype, solely based on of their proximity in a FLN to known genes associated with the phenotype. For example, a seed set of genes associated with breast cancer has been successfully used to identify additional disease genes via protein interaction relationships (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Pujana <em>et al.</em> 2007</a>). Here we extend this notion in two ways: First, we use edges that aggregate multiple sources of evidence about pairwise functional relationships. Second, our predictive models include <em>essentially all</em> genes, including those that have neither existing physical interaction evidence nor existing GO annotations.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-58">We thank Research Computing groups at Harvard Medical School and Mt. Sinai Hospital for computational resources and support. This work was supported by U.S. National Institutes of Health (NIH) grants HG003224, HG004233, HG001715, HL107440, and MH087394 (supporting F.P.R.), by NIH DK071507 (supporting W.T.), by NIH HG002273 (supporting J.A.B. and H.J.D.), by NIH F32 HG004098 (supporting M.T.), by the A*STAR Foundation (supporting H.N.C.), and by the Canada Excellence Research Chairs Program. F.P.R. also acknowledges support by a Fellowship from the Canadian Institute for Advanced Research.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000828/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000828/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2"><span class="fn-label">2</span> An unbiased scoring method such that the score for each gene is obtained only from the subset of decision trees that were not trained using that gene, that is, similar in spirit to cross-validation.</p></li><li class="fn" id="fn-6"><p id="p-59"><em>Communicating editor: J. M. Cherry</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-08-04"><span class="received-label">Received </span>August 4, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-11-23"><span class="accepted-label">Accepted </span>November 23, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Tasan <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrell</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dimmer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huntley</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Binns</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Donovan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The GOA database in 2009–an integrated Gene Ontology Annotation resource</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">396</span>–<span class="cit-lpage">403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D37%26rft.spage%253D396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.2" data-doi="10.1093/nar/gkn764"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troup</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilhite</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ledoux</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rudnev</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">NCBI GEO: archive for high-throughput functional genomic data</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">D885</span>–<span class="cit-lpage">D890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn764%26rft_id%253Dinfo%253Apmid%252F18940857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNy9zdXBwbF8xL0Q4ODUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.3" data-doi="10.1093/bioinformatics/btq265"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beaver</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tasan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibbons</span>  <span class="cit-name-given-names">F. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">FuncBase: a resource for quantitative gene function annotation</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1806</span>–<span class="cit-lpage">1807</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq265%26rft_id%253Dinfo%253Apmid%252F20495000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMTQvMTgwNiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8yMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing</span>. <abbr class="cit-jnl-abbrev">J. R. Stat. Soc., B</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%252C%2BB%26rft.volume%253D57%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.5" data-doi="10.1093/bioinformatics/btn424"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berriz</span>  <span class="cit-name-given-names">G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">F. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The Synergizer service for translating gene, protein and other biological identifiers</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2272</span>–<span class="cit-lpage">2273</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn424%26rft_id%253Dinfo%253Apmid%252F18697767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjQvMTkvMjI3MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8yMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.6" data-doi="10.1093/nar/gkl1076"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bieri</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blasiar</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozersky</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antoshechkin</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bastiani</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">WormBase: new content and better access</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>(<span class="cit-issue">Database issue</span>): <span class="cit-fpage">506</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl1076%26rft_id%253Dinfo%253Apmid%252F17169986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjM1LzIvNTA2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8yLzIyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-web" id="cit-2.2.223.7"><div class="cit-metadata"><cite><span class="cit-comment">Bologna, G., A.-L. Veuthey, M. Pagni, L. Lane, and A. Bairoch 2011  A preliminary study on the prediction of human protein functions, <em>in</em> ‘Proceedings of the 4th international conference on Interplay between natural and artificial computation - Volume Part I’, IWINAC’11, Springer-Verlag, Berlin, Heidelberg, pp. 334–343. Available at<strong>:</strong> <em><a href="http://dl.acm.org/citation.cfm?id=2009405.2009440">http://dl.acm.org/citation.cfm?id=2009405.2009440</a></em></span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.8" data-doi="10.1158/0008-5472.CAN-05-1204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bredel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bredel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juric</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harsh</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Functional network analysis reveals extended gliomagenesis pathway maps and three novel MYC-interacting genes in human gliomas</span>. <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">8679</span>–<span class="cit-lpage">8689</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BResearch%26rft.stitle%253DCancer%2BRes.%26rft.aulast%253DBredel%26rft.auinit1%253DM.%26rft.volume%253D65%26rft.issue%253D19%26rft.spage%253D8679%26rft.epage%253D8689%26rft.atitle%253DFunctional%2BNetwork%2BAnalysis%2BReveals%2BExtended%2BGliomagenesis%2BPathway%2BMaps%2Band%2BThree%2BNovel%2BMYC-Interacting%2BGenes%2Bin%2BHuman%2BGliomas%26rft_id%253Dinfo%253Adoi%252F10.1158%252F0008-5472.CAN-05-1204%26rft_id%253Dinfo%253Apmid%252F16204036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjEwOiI2NS8xOS84Njc5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8yLzIyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.9" data-doi="10.1023/A:1010933404324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breiman</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Random forests</span>. <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">45</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">5</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D45%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1010933404324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1010933404324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000170489900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.10" data-doi="10.1093/nar/gkl1073"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crosby</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodman</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strelets</span>  <span class="cit-name-given-names">V. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gelbart</span>  <span class="cit-name-given-names">W. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">FlyBase: genomes by the dozen</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>(<span class="cit-issue">Database issue</span>): <span class="cit-fpage">486</span>–<span class="cit-lpage">491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl1073%26rft_id%253Dinfo%253Apmid%252F17169988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjM1LzIvNDg2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8yLzIyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.11" data-doi="10.1093/nar/gni179"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyd</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kostov</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Athey</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Evolving gene/transcript definitions significantly alter the interpretation of GeneChip data</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">e175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DDai%26rft.auinit1%253DM.%26rft.volume%253D33%26rft.issue%253D20%26rft.spage%253De175%26rft.epage%253De175%26rft.atitle%253DEvolving%2Bgene%252Ftranscript%2Bdefinitions%2Bsignificantly%2Balter%2Bthe%2Binterpretation%2Bof%2BGeneChip%2Bdata%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgni179%26rft_id%253Dinfo%253Apmid%252F16284200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMy8yMC9lMTc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8yLzIyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.12" data-doi="10.1089/1066527041410346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">An integrated probabilistic model for functional prediction of proteins</span>. <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">2–3</span>): <span class="cit-fpage">463</span>–<span class="cit-lpage">475</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DDeng%26rft.auinit1%253DM.%26rft.volume%253D11%26rft.issue%253D2-3%26rft.spage%253D463%26rft.epage%253D475%26rft.atitle%253DAn%2Bintegrated%2Bprobabilistic%2Bmodel%2Bfor%2Bfunctional%2Bprediction%2Bof%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1089%252F1066527041410346%26rft_id%253Dinfo%253Apmid%252F15285902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/1066527041410346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15285902&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222588300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.13" data-doi="10.1093/nar/gkl940"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eppig</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bult</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richardson</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite><span class="cit-auth cit-collab">Mouse Genome Informatics Group</span>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The mouse genome database (mgd): new features facilitating a model system</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>(<span class="cit-issue">Database issue</span>): <span class="cit-fpage">D630</span>–<span class="cit-lpage">D637</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl940%26rft_id%253Dinfo%253Apmid%252F17135206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNS9zdXBwbF8xL0Q2MzAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.14" data-doi="10.1093/nar/gkh110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gunsalus</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yueh</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacMenamin</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piano</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">RNAiDB and PhenoBlast: web tools for genome-wide phenotypic mapping projects</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">D406</span>–<span class="cit-lpage">D410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGunsalus%26rft.auinit1%253DK.%2BC.%26rft.volume%253D32%26rft.issue%253D90001%26rft.spage%253DD406%26rft.epage%253D410%26rft.atitle%253DRNAiDB%2Band%2BPhenoBlast%253A%2Bweb%2Btools%2Bfor%2Bgenome-wide%2Bphenotypic%2Bmapping%2Bprojects%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh110%26rft_id%253Dinfo%253Apmid%252F14681444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzMi9zdXBwbF8xL0Q0MDYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.15" data-doi="10.1038/nature03876"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gunsalus</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schetter</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldberg</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Predictive models of molecular machines involved in Caenorhabditis elegans early embryogenesis</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">436</span>: <span class="cit-fpage">861</span>–<span class="cit-lpage">865</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGunsalus%26rft.auinit1%253DK.%2BC.%26rft.volume%253D436%26rft.issue%253D7052%26rft.spage%253D861%26rft.epage%253D865%26rft.atitle%253DPredictive%2Bmodels%2Bof%2Bmolecular%2Bmachines%2Binvolved%2Bin%2BCaenorhabditis%2Belegans%2Bearly%2Bembryogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03876%26rft_id%253Dinfo%253Apmid%252F16094371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03876&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16094371&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231116500049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.16" data-doi="10.1093/nar/gki033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamosh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amberger</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bocchini</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKusick</span>  <span class="cit-name-given-names">V. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Online Mendelian Inheritance in Man (OMIM), a knowledgebase of human genes and genetic disorders</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>(<span class="cit-issue">Database issue</span>): <span class="cit-fpage">514</span>–<span class="cit-lpage">517</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D33%26rft.spage%253D514%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.17" data-doi="10.1101/gr.082214.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huttenhower</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hibbs</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dumeaux</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Exploring the human genome with functional maps</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1093</span>–<span class="cit-lpage">1106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.082214.108%26rft_id%253Dinfo%253Apmid%252F19246570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTA5MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8yMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.18" data-doi="10.1371/journal.pcbi.1000779"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huttenhower</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hofmann</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A quick guide to large-scale genomic data mining</span>. <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000779</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DHuttenhower%26rft.auinit1%253DC.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De1000779%26rft.epage%253De1000779%26rft.atitle%253DA%2Bquick%2Bguide%2Bto%2Blarge-scale%2Bgenomic%2Bdata%2Bmining.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000779%26rft_id%253Dinfo%253Apmid%252F20523745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000779&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20523745&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.19" data-doi="10.1089/omi.2004.8.322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Joshi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexandrov</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genome-scale gene function prediction using multiple sources of high-throughput data in yeast <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">OMICS</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">322</span>–<span class="cit-lpage">333</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOmics%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bintegrative%2Bbiology%26rft.stitle%253DOMICS%26rft.aulast%253DJoshi%26rft.auinit1%253DT.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253D322%26rft.epage%253D333%26rft.atitle%253DGenome-scale%2Bgene%2Bfunction%2Bprediction%2Busing%2Bmultiple%2Bsources%2Bof%2Bhigh-throughput%2Bdata%2Bin%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fomi.2004.8.322%26rft_id%253Dinfo%253Apmid%252F15703479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/omi.2004.8.322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15703479&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227144400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.20" data-doi="10.1073/pnas.0307326101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karaoz</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murali</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Letovsky</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Whole-genome annotation by using evidence integration in functional-linkage networks</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">2888</span>–<span class="cit-lpage">2893</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0307326101%26rft_id%253Dinfo%253Apmid%252F14981259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAxLzkvMjg4OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8yMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.21" data-doi="10.1093/nar/gkn892"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keshava Prasad</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kandasamy</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keerthikumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Human Protein Reference Database–2009 update</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">D767</span>–<span class="cit-lpage">D772</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn892%26rft_id%253Dinfo%253Apmid%252F18988627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNy9zdXBwbF8xL0Q3NjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.22" data-doi="10.1093/bioinformatics/btg1024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">O. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dudley</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janse</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Predicting phenotype from patterns of annotation</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">Suppl 1</span>): <span class="cit-fpage">183</span>–<span class="cit-lpage">189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D19%26rft.spage%253D183%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg1024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg1024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.23" data-doi="10.1186/1471-2105-10-455"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ko</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Integrative approaches to the prediction of protein functions based on the feature selection</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DKo%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.spage%253D455%26rft.epage%253D455%26rft.atitle%253DIntegrative%2Bapproaches%2Bto%2Bthe%2Bprediction%2Bof%2Bprotein%2Bfunctions%2Bbased%2Bon%2Bthe%2Bfeature%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-455%26rft_id%253Dinfo%253Apmid%252F20043848%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-455&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20043848&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.24" data-doi="10.1093/bioinformatics/bth294"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lanckriet</span>  <span class="cit-name-given-names">G. R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Bie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cristianini</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">W. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A statistical framework for genomic data fusion</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">2626</span>–<span class="cit-lpage">2635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth294%26rft_id%253Dinfo%253Apmid%252F15130933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjAvMTYvMjYyNiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8yMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.25" data-doi="10.1126/science.1099511"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Date</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adai</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcotte</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A probabilistic functional network of yeast genes</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">306</span>: <span class="cit-fpage">1555</span>–<span class="cit-lpage">1558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLee%26rft.auinit1%253DI.%26rft.volume%253D306%26rft.issue%253D5701%26rft.spage%253D1555%26rft.epage%253D1558%26rft.atitle%253DA%2BProbabilistic%2BFunctional%2BNetwork%2Bof%2BYeast%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1099511%26rft_id%253Dinfo%253Apmid%252F15567862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDYvNTcwMS8xNTU1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8yLzIyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.26" data-doi="10.1101/gr.102749.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehner</span>  <span class="cit-name-given-names"> B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vavouri</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names"> J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names"> A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcotte</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Predicting genetic modifier loci using functional gene networks</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1143</span>–<span class="cit-lpage">1153</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.102749.109%26rft_id%253Dinfo%253Apmid%252F20538624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzgvMTE0MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8yMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.27" data-doi="10.1101/gr.118992.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blom</span>  <span class="cit-name-given-names">U. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">P. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shim</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcotte</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Prioritizing candidate disease genes by network-based boosting of genome-wide association data</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1109</span>–<span class="cit-lpage">1121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.118992.110%26rft_id%253Dinfo%253Apmid%252F21536720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzcvMTEwOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8yMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.28" data-doi="10.1093/bioinformatics/btg1026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Letovsky</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasif</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Predicting protein function from protein/protein interaction data: a probabilistic approach</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">Suppl 1</span>): <span class="cit-fpage">197</span>–<span class="cit-lpage">204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D19%26rft.spage%253D197%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg1026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg1026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.29" data-doi="10.2217/14796694.1.5.673"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bu</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">LRP5/6 in Wnt signaling and tumorigenesis</span>. <abbr class="cit-jnl-abbrev">Future Oncol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">673</span>–<span class="cit-lpage">681</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFuture%2BOncol.%26rft.volume%253D1%26rft.spage%253D673%26rft_id%253Dinfo%253Adoi%252F10.2217%252F14796694.1.5.673%26rft_id%253Dinfo%253Apmid%252F16556044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2217/14796694.1.5.673&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16556044&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.30" data-doi="10.1186/gb-2009-10-9-r91"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Linghu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snitkin</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeLisi</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide prioritization of disease genes and identification of disease-disease associations from an integrated human functional linkage network</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLinghu%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D9%26rft.spage%253DR91%26rft.epage%253DR91%26rft.atitle%253DGenome-wide%2Bprioritization%2Bof%2Bdisease%2Bgenes%2Band%2Bidentification%2Bof%2Bdisease-disease%2Bassociations%2Bfrom%2Ban%2Bintegrated%2Bhuman%2Bfunctional%2Blinkage%2Bnetwork.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-9-r91%26rft_id%253Dinfo%253Apmid%252F19728866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-9-r91&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19728866&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.31" data-doi="10.2174/1568009043332709"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luu</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haydon</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rayburn</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Wnt/beta-catenin signaling pathway as a novel cancer drug target</span>. <abbr class="cit-jnl-abbrev">Curr. Cancer Drug Targets</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">653</span>–<span class="cit-lpage">671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bcancer%2Bdrug%2Btargets%26rft.stitle%253DCurr%2BCancer%2BDrug%2BTargets%26rft.aulast%253DLuu%26rft.auinit1%253DH.%2BH.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253D653%26rft.epage%253D671%26rft.atitle%253DWnt%252Fbeta-catenin%2Bsignaling%2Bpathway%2Bas%2Ba%2Bnovel%2Bcancer%2Bdrug%2Btarget.%26rft_id%253Dinfo%253Adoi%252F10.2174%252F1568009043332709%26rft_id%253Dinfo%253Apmid%252F15578921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2174/1568009043332709&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15578921&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225958500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.32" data-doi="10.1093/nar/gki158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mulder</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Apweiler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Attwood</span>  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bairoch</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bateman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">InterPro, progress and status in 2005</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>(<span class="cit-issue">Database issue</span>): <span class="cit-fpage">201</span>–<span class="cit-lpage">205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DTian%26rft.auinit1%253DB.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D201%26rft.epage%253D212%26rft.atitle%253DA%2Blarge-scale%2Banalysis%2Bof%2BmRNA%2Bpolyadenylation%2Bof%2Bhuman%2Band%2Bmouse%2Bgenes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki158%26rft_id%253Dinfo%253Apmid%252F15647503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjMzLzEvMjAxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8yLzIyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.33" data-doi="10.1038/nbt1206-1474"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murali</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasif</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The art of gene function prediction</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1474</span>–<span class="cit-lpage">1475</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DMurali%26rft.auinit1%253DT.%2BM.%26rft.volume%253D24%26rft.issue%253D12%26rft.spage%253D1474%26rft.epage%253D1475%26rft.atitle%253DThe%2Bart%2Bof%2Bgene%2Bfunction%2Bprediction.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1206-1474%26rft_id%253Dinfo%253Apmid%252F17160037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1206-1474&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17160037&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242795800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.34" data-doi="10.1093/nar/gkl931"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nash</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hitz</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balakrishnan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christie</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Expanded protein information at SGD: new pages and proteome browser</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>(<span class="cit-issue">Database issue</span>): <span class="cit-fpage">468</span>–<span class="cit-lpage">471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D35%26rft.spage%253D468%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl931&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.35" data-doi="10.1093/nar/gki107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Brien</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Remm</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonnhammer</span>  <span class="cit-name-given-names">E. L. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Inparanoid: a comprehensive database of eukaryotic orthologs</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>(<span class="cit-issue">Database issue</span>): <span class="cit-fpage">476</span>–<span class="cit-lpage">480</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D33%26rft.spage%253D476%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pena-Castillo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tasan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joshi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A critical assessment of Mus musculus gene function prediction using integrated genomic evidence</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">Suppl 1</span>): <span class="cit-fpage">S2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D9%26rft.spage%253DS2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piekutowska-Abramczuk</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ciara</span>  <span class="cit-name-given-names"> E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Popowska</span>  <span class="cit-name-given-names"> E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grajkowska</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dembowska-Bagiska</span>  <span class="cit-name-given-names"> B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The frequency of NBN molecular variants in pediatric astrocytic tumors</span>. <abbr class="cit-jnl-abbrev">J. Neuro-oncol.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">161</span>–<span class="cit-lpage">168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BNeuro-oncol.%26rft.volume%253D96%26rft.spage%253D161%26rft_id%253Dinfo%253Apmid%252F19629396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19629396&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.38" data-doi="10.1038/ng.2007.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pujana</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Starita</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tewari</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Network modeling links breast cancer susceptibility and centrosome dysfunction</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1338</span>–<span class="cit-lpage">1349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DPujana%26rft.auinit1%253DM.%2BA.%26rft.volume%253D39%26rft.issue%253D11%26rft.spage%253D1338%26rft.epage%253D1349%26rft.atitle%253DNetwork%2Bmodeling%2Blinks%2Bbreast%2Bcancer%2Bsusceptibility%2Band%2Bcentrosome%2Bdysfunction.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2007.2%26rft_id%253Dinfo%253Apmid%252F17922014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2007.2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17922014&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250575900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.39" data-doi="10.1038/82360"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwikowski</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uetz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fields</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A network of protein-protein interactions in yeast</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1257</span>–<span class="cit-lpage">1261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DSchwikowski%26rft.auinit1%253DB.%26rft.volume%253D18%26rft.issue%253D12%26rft.spage%253D1257%26rft.epage%253D1261%26rft.atitle%253DA%2Bnetwork%2Bof%2Bprotein-protein%2Binteractions%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F82360%26rft_id%253Dinfo%253Apmid%252F11101803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/82360&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11101803&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165659000021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.40" data-doi="10.1038/ng.407"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shete</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hosking</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dobbins</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide association study identifies five susceptibility loci for glioma</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">899</span>–<span class="cit-lpage">904</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DShete%26rft.auinit1%253DS.%26rft.volume%253D41%26rft.issue%253D8%26rft.spage%253D899%26rft.epage%253D904%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bidentifies%2Bfive%2Bsusceptibility%2Bloci%2Bfor%2Bglioma.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.407%26rft_id%253Dinfo%253Apmid%252F19578367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.407&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19578367&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268432900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.41" data-doi="10.1186/gb-2005-6-5-p7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldsmith</span>  <span class="cit-name-given-names">C.-A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eppig</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The Mammalian Phenotype Ontology as a tool for annotating, analyzing and comparing phenotypic information</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">R7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DSmith%26rft.auinit1%253DC.%2BL.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253DR7%26rft.epage%253DR7%26rft.atitle%253DThe%2BMammalian%2BPhenotype%2BOntology%2Bas%2Ba%2Btool%2Bfor%2Bannotating%252C%2Banalyzing%2Band%2Bcomparing%2Bphenotypic%2Binformation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2005-6-5-p7%26rft_id%253Dinfo%253Apmid%252F15642099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2005-6-5-p7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15642099&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.42" data-doi="10.1142/S0219720010004744"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sokolov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ben-Hur</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Hierarchical classification of gene ontology terms using the GOstruct method</span>. <abbr class="cit-jnl-abbrev">J. Bioinform. Comput. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbioinformatics%2Band%2Bcomputational%2Bbiology%26rft.stitle%253DJ%2BBioinform%2BComput%2BBiol%26rft.aulast%253DSokolov%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D357%26rft.epage%253D376%26rft.atitle%253DHierarchical%2Bclassification%2Bof%2Bgene%2Bontology%2Bterms%2Busing%2Bthe%2BGOstruct%2Bmethod.%26rft_id%253Dinfo%253Adoi%252F10.1142%252FS0219720010004744%26rft_id%253Dinfo%253Apmid%252F20401950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1142/S0219720010004744&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20401950&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.43" data-doi="10.1093/nar/gkq973"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szklarczyk</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franceschini</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simonovic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The STRING database in 2011: functional interaction networks of proteins, globally integrated and scored</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">D561</span>–<span class="cit-lpage">D568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq973%26rft_id%253Dinfo%253Apmid%252F21045058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzOS9zdXBwbF8xL0Q1NjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tasan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibbons</span>  <span class="cit-name-given-names">F. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">An en masse phenotype and function prediction system for Mus musculus</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">Suppl 1</span>): <span class="cit-fpage">S8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D9%26rft.spage%253DS8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibbons</span>  <span class="cit-name-given-names">F. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">O. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Combining guilt-by-association and guilt-by-profiling to predict <em>Saccharomyces cerevisiae</em> gene function</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">Suppl 1</span>): <span class="cit-fpage">S7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D9%26rft.spage%253DS7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.46" data-doi="10.1073/pnas.0832373100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Troyanskaya</span>  <span class="cit-name-given-names">O. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dolinski</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Owen</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altman</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A Bayesian framework for combining heterogeneous data sources for gene function prediction (in Saccharomyces cerevisiae)</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">8348</span>–<span class="cit-lpage">8353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0832373100%26rft_id%253Dinfo%253Apmid%252F12826619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAwLzE0LzgzNDgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.47" data-doi="10.1016/j.jprot.2010.07.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">P. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcotte</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">‘It’s the machine that matters: predicting gene function and phenotype from protein networks’</span>. <abbr class="cit-jnl-abbrev">J. Proteomics</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">2277</span>–<span class="cit-lpage">2289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BProteomics%26rft.aulast%253DWang%26rft.auinit1%253DP.%2BI.%26rft.volume%253D73%26rft.issue%253D11%26rft.spage%253D2277%26rft.epage%253D2289%26rft.atitle%253DIt%2527s%2Bthe%2Bmachine%2Bthat%2Bmatters%253A%2BPredicting%2Bgene%2Bfunction%2Band%2Bphenotype%2Bfrom%2Bprotein%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jprot.2010.07.005%26rft_id%253Dinfo%253Apmid%252F20637909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jprot.2010.07.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20637909&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283903000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.48" data-doi="10.1073/pnas.0406614101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldberg</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Combining biological networks to predict genetic interactions</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">15682</span>–<span class="cit-lpage">15687</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0406614101%26rft_id%253Dinfo%253Apmid%252F15496468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzQ0LzE1NjgyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8yLzIyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.223.49" data-doi="10.1198/016214504000000683"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irizarry</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gentleman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez-Murillo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spencer</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A model-based background adjustment for oligonucleotide expression arrays</span>. <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">909</span>–<span class="cit-lpage">917</span> .</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D99%26rft.spage%253D909%26rft_id%253Dinfo%253Adoi%252F10.1198%252F016214504000000683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/016214504000000683&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000225337400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/2/223.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/2/213" title="Expression Quantitative Trait Loci for Extreme Host Response to Influenza A in Pre-Collaborative Cross Mice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/2/235" title="Extent With Modification: Leg Patterning in the Beetle Tribolium castaneum and the Evolution of Serial Homologs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/2"><span>Volume 2 Issue 2, February 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/2.cover.gif?itok=i7gjzosG" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/2/223" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1oAQS3yH21BmQeSTyylouvk49fW2UGyYR3s5SM7G7jo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/2/223" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/2/223" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/2/223" class="active" data-icon-position="" data-hide-link-title="0">A Resource of Quantitative Functional Annotation for Homo sapiens Genes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459234" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form--bIoQIM3e1LyAKa11TeMprkhGBctBMB1MxGorW2arxA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/2/223.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/2/223" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-WSDtavCUf5-HeKFPlfMRl2sb_vPsnHYmOADmh5jxsyE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/2/223" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/2/223.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459234" id="node459234" data-pisa="ggg;2/2/223" data-pisa-master="ggg;2/2/223" data-apath="/ggg/2/2/223.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Resource of Quantitative Functional Annotation for <em>Homo sapiens</em> Genes</h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Murat Taşan</span>, <span class="highwire-citation-author" data-delta="1">Harold J. Drabkin</span>, <span class="highwire-citation-author" data-delta="2">John E. Beaver</span>, <span class="highwire-citation-author" data-delta="3">Hon Nian Chua</span>, <span class="highwire-citation-author" data-delta="4">Julie Dunham</span>, <span class="highwire-citation-author" data-delta="5">Weidong Tian</span>, <span class="highwire-citation-author" data-delta="6">Judith A. Blake</span> and <span class="highwire-citation-author" data-delta="7">Frederick P. Roth</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">223-233;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000828 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459234" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Murat Taşan</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Donnelly Centre for Cellular &amp; Biomolecular Research, University of Toronto, Toronto, Ontario M5S-3E1, Canada</span><span class="nlm-aff">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Murat%2BTa%C5%9Fan" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ta%C5%9Fan%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMurat%2BTa%25C5%259Fan" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Harold J. Drabkin</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Mouse Genome Informatics, The Jackson Laboratory, Bar Harbor, Maine 04609</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harold%2BJ.%2BDrabkin" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Drabkin%20HJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHarold%2BJ.%2BDrabkin" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John E. Beaver</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BE.%2BBeaver" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beaver%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BE.%2BBeaver" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hon Nian Chua</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Donnelly Centre for Cellular &amp; Biomolecular Research, University of Toronto, Toronto, Ontario M5S-3E1, Canada</span><span class="nlm-aff">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hon%2BNian%2BChua" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chua%20HN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHon%2BNian%2BChua" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Julie Dunham</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Donnelly Centre for Cellular &amp; Biomolecular Research, University of Toronto, Toronto, Ontario M5S-3E1, Canada</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julie%2BDunham" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulie%2BDunham" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Weidong Tian</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institute of Biostatistics, School of Life Sciences, Fudan University, Shanghai 200433, P. R. China</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Weidong%2BTian" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tian%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWeidong%2BTian" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Judith A. Blake</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Mouse Genome Informatics, The Jackson Laboratory, Bar Harbor, Maine 04609</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Judith%2BA.%2BBlake" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blake%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJudith%2BA.%2BBlake" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Frederick P. Roth</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Donnelly Centre for Cellular &amp; Biomolecular Research, University of Toronto, Toronto, Ontario M5S-3E1, Canada</span><span class="nlm-aff">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115</span><span class="nlm-aff">Center for Cancer Systems Biology, Dana Farber Cancer Institute, Boston, Massachusetts 02115</span><span class="nlm-aff">Samuel Lunenfeld Research Institute, Mt. Sinai Hospital, Toronto, Ontario M5G-1X5, Canada</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frederick%2BP.%2BRoth" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Roth%20FP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrederick%2BP.%2BRoth" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fritz.roth@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">fritz.roth@utoronto.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/2/223" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/2/223&amp;title=A%20Resource%20of%20Quantitative%20Functional%20Annotation%20for%20Homo%20sapiens%20Genes" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/2/223&amp;title=A%20Resource%20of%20Quantitative%20Functional%20Annotation%20for%20Homo%20sapiens%20Genes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/2/223&amp;title=A%20Resource%20of%20Quantitative%20Functional%20Annotation%20for%20Homo%20sapiens%20Genes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/2/223&amp;text=A%20Resource%20of%20Quantitative%20Functional%20Annotation%20for%20Homo%20sapiens%20Genes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/2/223&amp;title=A%20Resource%20of%20Quantitative%20Functional%20Annotation%20for%20Homo%20sapiens%20Genes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/2/223&amp;t=A%20Resource%20of%20Quantitative%20Functional%20Annotation%20for%20Homo%20sapiens%20Genes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/2/223&amp;title=A%20Resource%20of%20Quantitative%20Functional%20Annotation%20for%20Homo%20sapiens%20Genes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/2/223&amp;title=A%20Resource%20of%20Quantitative%20Functional%20Annotation%20for%20Homo%20sapiens%20Genes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459234/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459234" id="node459234" data-pisa="ggg;2/2/223" data-pisa-master="ggg;2/2/223" data-apath="/ggg/2/2/223.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Resource of Quantitative Functional Annotation for <em>Homo sapiens</em> Genes</h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Murat Taşan</span>, <span class="highwire-citation-author" data-delta="1">Harold J. Drabkin</span>, <span class="highwire-citation-author" data-delta="2">John E. Beaver</span>, <span class="highwire-citation-author" data-delta="3">Hon Nian Chua</span>, <span class="highwire-citation-author" data-delta="4">Julie Dunham</span>, <span class="highwire-citation-author" data-delta="5">Weidong Tian</span>, <span class="highwire-citation-author" data-delta="6">Judith A. Blake</span> and <span class="highwire-citation-author" data-delta="7">Frederick P. Roth</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">223-233;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000828 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459234" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Murat Taşan</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Donnelly Centre for Cellular &amp; Biomolecular Research, University of Toronto, Toronto, Ontario M5S-3E1, Canada</span><span class="nlm-aff">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Murat%2BTa%C5%9Fan" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ta%C5%9Fan%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMurat%2BTa%25C5%259Fan" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Harold J. Drabkin</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Mouse Genome Informatics, The Jackson Laboratory, Bar Harbor, Maine 04609</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harold%2BJ.%2BDrabkin" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Drabkin%20HJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHarold%2BJ.%2BDrabkin" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John E. Beaver</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BE.%2BBeaver" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beaver%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BE.%2BBeaver" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hon Nian Chua</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Donnelly Centre for Cellular &amp; Biomolecular Research, University of Toronto, Toronto, Ontario M5S-3E1, Canada</span><span class="nlm-aff">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hon%2BNian%2BChua" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chua%20HN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHon%2BNian%2BChua" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Julie Dunham</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Donnelly Centre for Cellular &amp; Biomolecular Research, University of Toronto, Toronto, Ontario M5S-3E1, Canada</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julie%2BDunham" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulie%2BDunham" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Weidong Tian</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institute of Biostatistics, School of Life Sciences, Fudan University, Shanghai 200433, P. R. China</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Weidong%2BTian" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tian%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWeidong%2BTian" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Judith A. Blake</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Mouse Genome Informatics, The Jackson Laboratory, Bar Harbor, Maine 04609</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Judith%2BA.%2BBlake" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blake%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJudith%2BA.%2BBlake" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Frederick P. Roth</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Donnelly Centre for Cellular &amp; Biomolecular Research, University of Toronto, Toronto, Ontario M5S-3E1, Canada</span><span class="nlm-aff">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115</span><span class="nlm-aff">Center for Cancer Systems Biology, Dana Farber Cancer Institute, Boston, Massachusetts 02115</span><span class="nlm-aff">Samuel Lunenfeld Research Institute, Mt. Sinai Hospital, Toronto, Ontario M5G-1X5, Canada</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frederick%2BP.%2BRoth" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Roth%20FP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrederick%2BP.%2BRoth" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fritz.roth@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">fritz.roth@utoronto.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=223&amp;rft.epage=233&amp;rft.atitle=A%20Resource%20of%20Quantitative%20Functional%20Annotation%20for%20Homo%20sapiens%20Genes&amp;rft.volume=2&amp;rft.issue=2&amp;rft.date=2012-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ta%C5%9Fan%2C+Murat&amp;rft.au=Drabkin%2C+Harold+J.&amp;rft.au=Beaver%2C+John+E.&amp;rft.au=Chua%2C+Hon+Nian&amp;rft.au=Dunham%2C+Julie&amp;rft.au=Tian%2C+Weidong&amp;rft.au=Blake%2C+Judith+A.&amp;rft.au=Roth%2C+Frederick+P."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459234/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459234/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459234/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459234/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459234/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459234/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459234/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459234/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459234/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459234/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459234/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459234/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494625" id="node494625" data-pisa="ggg;9/11/3891" data-pisa-master="ggg;g3.119.400723" data-apath="/ggg/9/11/3891.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3891" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Mouse Mutation That Dysregulates Neighboring <em>Galnt17</em> and <em>Auts2</em> Genes Is Associated with Phenotypes Related to the Human AUTS2 Syndrome</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494624" id="node494624" data-pisa="ggg;9/11/3781" data-pisa-master="ggg;g3.119.400647" data-apath="/ggg/9/11/3781.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3781" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Transgenic Analyses in Drosophila Reveal That mCORL1 Is Functionally Distinct from mCORL2 and dCORL</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494622" id="node494622" data-pisa="ggg;9/11/3843" data-pisa-master="ggg;g3.119.400363" data-apath="/ggg/9/11/3843.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3843" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Analysis of Cell Cycle-Regulating Genes in the Symbiotic Dinoflagellate <em>Breviolum minutum</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/2/223" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/2/223.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/2/223.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/2/223.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/2/223.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2019 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
  </body>
</html>
