<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/2/175" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/2/175.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/2/175.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/2/175.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2.cover.gif" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/2/2/175/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/2/175.atom" />
<meta name="HW.pisa" content="ggg;2/2/175" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="HTreeQA: Using Semi-Perfect Phylogeny Trees in Quantitative Trait Loci Study on Genotype Data" />
<meta name="DC.Identifier" content="10.1534/g3.111.001768" />
<meta name="DC.Date" content="2012-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Zhang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="With the advances in high-throughput genotyping technology, the study of quantitative trait loci (QTL) has emerged as a promising tool to understand the genetic basis of complex traits. Methodology development for the study of QTL recently has attracted significant research attention. Local phylogeny-based methods have been demonstrated to be powerful tools for uncovering significant associations between phenotypes and single-nucleotide polymorphism markers. However, most existing methods are designed for homozygous genotypes, and a separate haplotype reconstruction step is often needed to resolve heterozygous genotypes. This approach has limited power to detect nonadditive genetic effects and imposes an extensive computational burden. In this article, we propose a new method, HTreeQA, that uses a tristate semi-perfect phylogeny tree to approximate the perfect phylogeny used in existing methods. The semi-perfect phylogeny trees are used as high-level markers for association study. HTreeQA uses the genotype data as direct input without phasing. HTreeQA can handle complex local population structures. It is suitable for QTL mapping on any mouse populations, including the incipient Collaborative Cross lines. Applied HTreeQA, significant QTLs are found for two phenotypes of the PreCC lines, white head spot and running distance at day 5/6. These findings are consistent with known genes and QTL discovered in independent studies. Simulation studies under three different genetic models show that HTreeQA can detect a wider range of genetic effects and is more efficient than existing phylogeny-based approaches. We also provide rigorous theoretical analysis to show that HTreeQA has a lower error rate than alternative methods." />
<meta name="DC.Contributor" content="Zhaojun Zhang" />
<meta name="DC.Contributor" content="Xiang Zhang" />
<meta name="DC.Contributor" content="Wei Wang" />
<meta name="article:published_time" content="2012-02-01" />
<meta name="article:section" content="Mouse Genetic Resources" />
<meta name="citation_title" content="HTreeQA: Using Semi-Perfect Phylogeny Trees in Quantitative Trait Loci Study on Genotype Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;With the advances in high-throughput genotyping technology, the study of quantitative trait loci (QTL) has emerged as a promising tool to understand the genetic basis of complex traits. Methodology development for the study of QTL recently has attracted significant research attention. Local phylogeny-based methods have been demonstrated to be powerful tools for uncovering significant associations between phenotypes and single-nucleotide polymorphism markers. However, most existing methods are designed for homozygous genotypes, and a separate haplotype reconstruction step is often needed to resolve heterozygous genotypes. This approach has limited power to detect nonadditive genetic effects and imposes an extensive computational burden. In this article, we propose a new method, HTreeQA, that uses a tristate semi-perfect phylogeny tree to approximate the perfect phylogeny used in existing methods. The semi-perfect phylogeny trees are used as high-level markers for association study. HTreeQA uses the genotype data as direct input without phasing. HTreeQA can handle complex local population structures. It is suitable for QTL mapping on any mouse populations, including the incipient Collaborative Cross lines. Applied HTreeQA, significant QTLs are found for two phenotypes of the PreCC lines, white head spot and running distance at day 5/6. These findings are consistent with known genes and QTL discovered in independent studies. Simulation studies under three different genetic models show that HTreeQA can detect a wider range of genetic effects and is more efficient than existing phylogeny-based approaches. We also provide rigorous theoretical analysis to show that HTreeQA has a lower error rate than alternative methods.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/02/01" />
<meta name="citation_mjid" content="ggg;2/2/175" />
<meta name="citation_id" content="2/2/175" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/2/175" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/2/175.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/2/175.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/2/175.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.001768" />
<meta name="citation_pmid" content="22384396" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="2" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Mouse Genetic Resources" />
<meta name="citation_firstpage" content="175" />
<meta name="citation_lastpage" content="189" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Zhaojun Zhang" />
<meta name="citation_author_institution" content="Department of Computer Science, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Xiang Zhang" />
<meta name="citation_author_institution" content="Department of Electrical Engineering and Computer Science, Case Western Reserve University, Cleveland, Ohio 44106" />
<meta name="citation_author" content="Wei Wang" />
<meta name="citation_author_institution" content="Department of Computer Science, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599" />
<meta name="citation_author_email" content="weiwang@cs.unc.edu" />
<meta name="citation_reference" content="citation_journal_title=European journal of human genetics :  EJHG;citation_journal_abbrev=Eur J Hum Genet;citation_author=J. Akey;citation_author=L. Jin;citation_author=M. Xiong;citation_title=Haplotypes vs single marker linkage disequilibrium tests: what do we gain?;citation_pages=291-300;citation_volume=9;citation_year=2001;citation_issue=4;citation_pmid=11313774;citation_doi=10.1038/sj.ejhg.5200619" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. L. Aylor;citation_author=W. Valdar;citation_author=W. Foulds-Mathes;citation_author=R. J. Buus;citation_author=R. A. Verdugo;citation_title=Genetic analysis of complex traits in the emerging Collaborative Cross;citation_volume=21;citation_year=2011;citation_pmid=21406540;citation_doi=10.1101/gr.111310.110" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Besenbacher;citation_author=T. Mailund;citation_author=M. H. Schierup;citation_title=Local phylogeny mapping of quantitative traits: higher accuracy and better ranking than single-marker association in genomewide scans;citation_pages=747-753;citation_volume=181;citation_year=2009;citation_pmid=19064712" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. A. Churchill;citation_author=R. W. Doerge;citation_title=Empirical threshold values for quantitative trait mapping.;citation_pages=963-971;citation_volume=138;citation_year=1994;citation_issue=3;citation_pmid=7851788" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Collaborative Cross Consortium;citation_title=The genome architecture of the Collaborative Cross Mouse Genetic Reference Population;citation_pages=389-401;citation_volume=190;citation_year=2012;citation_pmid=22345608;citation_doi=10.1534/genetics.111.132639" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=B. Devlin;citation_author=K. Roeder;citation_title=Genomic control for association studies;citation_pages=997-1004;citation_volume=55;citation_year=1999;citation_pmid=11315092;citation_doi=10.1111/j.0006-341X.1999.00997.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Z. Ding;citation_author=T. Mailund;citation_author=Y. S. Song;citation_title=Efficient whole-genome association mapping using local phylogenies for unphased genotype data;citation_volume=24;citation_year=2008;citation_pmid=18667442;citation_doi=10.1093/bioinformatics/btn406" />
<meta name="citation_reference" content="citation_journal_title=Appl. Math. Lett.;citation_author=A. Dress;citation_author=M. Steel;citation_title=Convex tree realizations of partitions;citation_pages=3-6;citation_volume=5;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Networks;citation_author=D. Gusfield;citation_title=An efficient algorithms for inferring evolutionary trees;citation_pages=19-28;citation_volume=21;citation_year=1991;citation_doi=10.1002/net.3230210104" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=D. Gusfield;citation_title=The Multi-State Perfect Phylogeny Problem with missing and removable data: solutions via integer-programming and chordal graph theory.;citation_pages=383-399;citation_volume=17;citation_year=2010;citation_issue=3;citation_pmid=20377452;citation_doi=10.1089/cmb.2009.0200" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Rhythms;citation_journal_abbrev=J Biol Rhythms;citation_author=J. R. Hofstetter;citation_author=J. A. Trofatter;citation_author=K. L. Kernek;citation_author=J. I. Nurnberger;citation_author=R. Mayeda;citation_title=New Quantitative Trait Loci for the Genetic Variance in Circadian Period of Locomotor Activity between Inbred Strains of Mice;citation_pages=450-462;citation_volume=18;citation_year=2003;citation_issue=6;citation_pmid=14667146;citation_doi=10.1177/0748730403259468" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. R. Hudson;citation_author=N. L. Kaplan;citation_title=Statistical properties of the number of recombination events in the history of a sample of DNA sequences;citation_pages=147-164;citation_volume=111;citation_year=1985;citation_pmid=4029609" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. M. Kang;citation_author=N. A. Zaitlen;citation_author=C. M. Wade;citation_author=A. Kirby;citation_author=D. Heckerman;citation_title=Efficient control of population structure in model organism association mapping;citation_pages=1709-1723;citation_volume=178;citation_year=2008;citation_pmid=18385116;citation_doi=10.1534/genetics.107.080101" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=F. Larribe;citation_author=S. Lessard;citation_author=N. J. Schork;citation_title=Gene mapping via the ancestral recombination graph.;citation_pages=215-229;citation_volume=62;citation_year=2002;citation_issue=2;citation_pmid=12167358;citation_doi=10.1006/tpbi.2002.1601" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=G. Lettre;citation_author=C. Lange;citation_author=J. N. Hirschhorn;citation_title=Genetic model testing and statistical power in population-based association studies of quantitative traits;citation_pages=358-362;citation_volume=362;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. Li;citation_author=T. Jiang;citation_title=Haplotype-based linkage disequilibrium mapping via direct data mining;citation_volume=21;citation_year=2005;citation_pmid=16249262;citation_doi=10.1093/bioinformatics/bti732" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=A. D. Long;citation_author=C. H. Langley;citation_title=The Power of Association Studies to Detect the Contribution of Candidate Genetic Loci to Variation in Complex Traits;citation_pages=720-731;citation_volume=9;citation_year=1999;citation_issue=8;citation_pmid=10447507" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=T. Mailund;citation_author=S. Besenbacher;citation_author=M. H. Schierup;citation_title=Whole genome association mapping by incompatibilities and local perfect phylogenies.;citation_pages=454-454;citation_volume=7;citation_year=2006;citation_pmid=17042942;citation_doi=10.1186/1471-2105-7-454" />
<meta name="citation_reference" content="citation_journal_title=Behavior genetics;citation_journal_abbrev=Behav Genet;citation_author=R. Mayeda;citation_author=J. R. Hofstetter;citation_title=A QTL for the genetic variance in free-running period and level of locomotor activity between inbred strains of mice.;citation_pages=171-176;citation_volume=29;citation_year=1999;citation_issue=3;citation_pmid=10547923;citation_doi=10.1023/A:1021639901679" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=P. McClurg;citation_author=M. T. Pletcher;citation_author=T. Wiltshire;citation_author=A. I. Su;citation_title=Comparative analysis of haplotype association mapping algorithms.;citation_pages=61-61;citation_volume=7;citation_year=2006;citation_pmid=16466585;citation_doi=10.1186/1471-2105-7-61" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. Q. Minichiello;citation_author=R. Durbin;citation_title=Mapping trait loci by use of inferred ancestral recombination graphs.;citation_pages=910-922;citation_volume=79;citation_year=2006;citation_issue=5;citation_pmid=17033967;citation_doi=10.1086/508901" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=A. Morris;citation_author=J. Whittaker;citation_author=D. Balding;citation_title=Fine-scale mapping of disease loci via shattered coalescent modeling of genealogies.;citation_pages=686-707;citation_volume=70;citation_year=2002;citation_issue=3;citation_pmid=11836651;citation_doi=10.1086/339271" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=P. Onkamo;citation_author=V. Ollikainen;citation_author=P. Sevon;citation_author=H. Toivonen;citation_author=H. Mannila;citation_author=J. Kere;citation_title=Association analysis for quantitative traits by data mining: QHPM.;citation_pages=419-429;citation_volume=66;citation_year=2002;citation_issue=Pt 5-6;citation_pmid=12485474;citation_doi=10.1046/j.1469-1809.2002.00131.x" />
<meta name="citation_reference" content="citation_journal_title=Pac. Symp. Biocomput.;citation_author=F. Pan;citation_author=L. McMillan;citation_author=F. Pardo-Manuel De Villena;citation_author=D. Threadgill;citation_author=W. Wang;citation_title=TREEQA: QUANTITATIVE GENOME-WIDE ASSOCIATION MAPPING USING LOCAL PERFECT PHYLOGENY TREES;citation_pages=415-426;citation_volume=426:;citation_year=2009" />
<meta name="citation_reference" content="citation_author=F. Pan;citation_author=L. Yang;citation_author=L. McMillan;citation_author=F. Pardo-Manuel De Villena;citation_author=D. Threadgill" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=I. Pe'er;citation_author=P. I. W. De Bakker;citation_author=J. Maller;citation_author=R. Yelensky;citation_author=D. Altshuler;citation_title=Evaluating and improving power in whole-genome association studies using fixed marker sets.;citation_pages=663-667;citation_volume=38;citation_year=2006;citation_issue=6;citation_pmid=16715096;citation_doi=10.1038/ng1816" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. L. Price;citation_author=N. J. Patterson;citation_author=R. M. Plenge;citation_author=M. E. Weinblatt;citation_author=N. A. Shadick;citation_title=Principal components analysis corrects for stratification in genome-wide association studies.;citation_pages=904-909;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16862161;citation_doi=10.1038/ng1847" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=A. Roberts;citation_author=F. Pardo-Manuel de Villena;citation_author=W. Wang;citation_author=L. McMillan;citation_author=D. W. Threadgill;citation_title=The polymorphism architecture of mouse genetic resources elucidated using genome-wide resequencing data: implications for QTL discovery and systems genetics.;citation_pages=473-481;citation_volume=18;citation_year=2007;citation_issue=6-7;citation_pmid=17674098;citation_doi=10.1007/s00335-007-9045-1" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. Scheet;citation_author=M. Stephens;citation_title=A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.;citation_pages=629-644;citation_volume=78;citation_year=2006;citation_issue=4;citation_pmid=16532393;citation_doi=10.1086/502802" />
<meta name="citation_reference" content="citation_journal_title=IEEE/ACM Trans. Comput. Biol. Bioinf.;citation_author=P. Sevon;citation_author=H. Toivonen;citation_author=V. Ollikainen;citation_title=TreeDT: tree pattern mining for gene mapping;citation_pages=174-1085;citation_volume=3;citation_year=2006;citation_doi=10.1109/TCBB.2006.28" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. L. Svenson;citation_author=D. M. Gatti;citation_author=W. Valdar;citation_author=C. E. Welsh;citation_author=R. Cheng;citation_title=High-Resolution Genetic Mapping Using the Mouse Diversity Outbred Population;citation_pages=437-447;citation_volume=190;citation_year=2012;citation_issue=2;citation_pmid=22345611;citation_doi=10.1534/genetics.111.132597" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=The Complex Trait Consortium;citation_title=The Collaborative Cross, a community resource for the genetic analysis of complex traits;citation_pages=1133-1137;citation_volume=36;citation_year=2012" />
<meta name="citation_reference" content="citation_title=Statistical Methods in Genetic Epidemiology;citation_author=D. C. Thomas;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=W. Valdar;citation_author=L. C. Solberg;citation_author=D. Gauguier;citation_author=S. Burnett;citation_author=P. Klenerman;citation_title=Genome-wide genetic association of complex traits in heterogeneous stock mice.;citation_pages=879-887;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16832355;citation_doi=10.1038/ng1840" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=K. Wang;citation_author=V. Sheffield;citation_title=A constrained-likelihood approach to marker-trait association studies.;citation_pages=768-780;citation_volume=77;citation_year=2005;citation_issue=5;citation_pmid=16252237;citation_doi=10.1086/497434" />
<meta name="citation_reference" content="citation_title=Resampling-Based Multiple Testing: Examples and Methods for P-Value Adjustment;citation_author=P. H. Westfall;citation_author=S. S. Young;citation_year=1993" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=H. Yang;citation_author=Y. Ding;citation_author=L. Hutchins;citation_author=J. Szatkiewicz;citation_author=T. Bell;citation_title=A customized and versatile high-density genotyping array for the mouse.;citation_pages=663-666;citation_volume=6;citation_year=2009;citation_issue=9;citation_pmid=19668205;citation_doi=10.1038/nmeth.1359" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. Yang;citation_author=J. R. Wang;citation_author=J. P. Didion;citation_author=R. J. Buus;citation_author=T. A. Bell;citation_title=Subspecific origin and haplotype diversity in the laboratory mouse.;citation_pages=648-655;citation_volume=43;citation_year=2011;citation_issue=7;citation_pmid=21623374;citation_doi=10.1038/ng.847" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Zöllner;citation_author=J. K. Pritchard;citation_title=Coalescent-based association mapping and fine mapping of complex trait loci;citation_volume=169;citation_year=2005;citation_pmid=15489534;citation_doi=10.1534/genetics.104.031799" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="HTreeQA: Using Semi-Perfect Phylogeny Trees in Quantitative Trait Loci Study on Genotype Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2.cover.gif" />
<meta name="twitter:description" content="With the advances in high-throughput genotyping technology, the study of quantitative trait loci (QTL) has emerged as a promising tool to understand the genetic basis of complex traits. Methodology development for the study of QTL recently has attracted significant research attention. Local phylogeny-based methods have been demonstrated to be powerful tools for uncovering significant associations between phenotypes and single-nucleotide polymorphism markers. However, most existing methods are designed for homozygous genotypes, and a separate haplotype reconstruction step is often needed to resolve heterozygous genotypes. This approach has limited power to detect nonadditive genetic effects and imposes an extensive computational burden. In this article, we propose a new method, HTreeQA, that uses a tristate semi-perfect phylogeny tree to approximate the perfect phylogeny used in existing methods. The semi-perfect phylogeny trees are used as high-level markers for association study. HTreeQA uses the genotype data as direct input without phasing. HTreeQA can handle complex local population structures. It is suitable for QTL mapping on any mouse populations, including the incipient Collaborative Cross lines. Applied HTreeQA, significant QTLs are found for two phenotypes of the PreCC lines, white head spot and running distance at day 5/6. These findings are consistent with known genes and QTL discovered in independent studies. Simulation studies under three different genetic models show that HTreeQA can detect a wider range of genetic effects and is more efficient than existing phylogeny-based approaches. We also provide rigorous theoretical analysis to show that HTreeQA has a lower error rate than alternative methods." />
<meta name="og-title" property="og:title" content="HTreeQA: Using Semi-Perfect Phylogeny Trees in Quantitative Trait Loci Study on Genotype Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/2/175" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="With the advances in high-throughput genotyping technology, the study of quantitative trait loci (QTL) has emerged as a promising tool to understand the genetic basis of complex traits. Methodology development for the study of QTL recently has attracted significant research attention. Local phylogeny-based methods have been demonstrated to be powerful tools for uncovering significant associations between phenotypes and single-nucleotide polymorphism markers. However, most existing methods are designed for homozygous genotypes, and a separate haplotype reconstruction step is often needed to resolve heterozygous genotypes. This approach has limited power to detect nonadditive genetic effects and imposes an extensive computational burden. In this article, we propose a new method, HTreeQA, that uses a tristate semi-perfect phylogeny tree to approximate the perfect phylogeny used in existing methods. The semi-perfect phylogeny trees are used as high-level markers for association study. HTreeQA uses the genotype data as direct input without phasing. HTreeQA can handle complex local population structures. It is suitable for QTL mapping on any mouse populations, including the incipient Collaborative Cross lines. Applied HTreeQA, significant QTLs are found for two phenotypes of the PreCC lines, white head spot and running distance at day 5/6. These findings are consistent with known genes and QTL discovered in independent studies. Simulation studies under three different genetic models show that HTreeQA can detect a wider range of genetic effects and is more efficient than existing phylogeny-based approaches. We also provide rigorous theoretical analysis to show that HTreeQA has a lower error rate than alternative methods." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2.cover.gif" />
    <title>HTreeQA: Using Semi-Perfect Phylogeny Trees in Quantitative Trait Loci Study on Genotype Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q0wst1" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__KPgiwEdjo5O9HdRryHqQs1TpsV-PeBb9B7sGBvwM9FQ__opzZWgPNgQzoEgfATRS5qCKwBw_6XcNjDHoDVOzQv8M__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__YUqwBlv6bNoCToiBWgAUSPtppdS8sbGfzmAH3qEDcI0__Mvk93h3ScFH5Xi9WhibVKlf368fHw-V7HWqWX2QMvtw__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XNWYaQubFGWYWZNytUHkWs-b34LV21DsB4s_6gchI4k__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__7cpBA0GVdrMqkZGGZEW-_SPF2HrbTCmVg9-lrP5l_AQ__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__TV2D4R18tPiSIOmeQ3vyMqHObN_FDmQ6EEIrfE5sAoA__c9BBZFy1BrgGgF_Auu9Fmysb3Z7J56eJGYvMcLQrLks__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q0wst1"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__OaHnYyci68QTwctdRYR30kEfu0eYWBrLyMqy8THv5Ik__IyfJQYM1q1QEPwYsK_Eslh2RQkxpy6yo-nR0p4DWPQQ__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__fhq_kmKS3zjIxlsAa3GE1ubLajyXUgXXTXULzAtiqfg__Lm1_Ls-bdpbWRyl0q4o0mmHyxB-139AJ68dE4zOskLQ__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"mcopXwFoeAdfIZwJKjJ-DTgQbgAhTnK6_ZcTCyIwJnA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459239","apath":"\/ggg\/2\/2\/175.atom","pisa":"ggg;2\/2\/175","ac":{"\/ggg\/2\/2\/175.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/2\/175.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/2\/175"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/2\/175"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/2\/175":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022JstnSh2rX5YKKTIJsr6h8bZWTCQQTkxaJSFZyMKXEfZcPVTLHEgv7SE23LKh4kiQfa+FZSFGamGNqkZL3GOImyQR0O2byMmWwc2K+uOZxThdLAAU5wSxRdM\\\\\\\/B+0I1Wl\\\\\\\/5Ms9jxwbPCgbZ0+\\\\\\\/IBjXVk3YGArYfTXGAnXVktEoc85xYj0guOC\\\\\\\/LfwC4FR1Ppmt+ZqhQUffVZ\\\\\\\/hKpNsQu3i51lisHyxc+wsMWbDuw50nsjET3MY+aOMWjVyTk6MAlWbGOqX2XAgNw0Fj1Fvk7emCVnpqptDjUuAxrgM+6O5mFYCdGtfBVgVwLlx5+GLLOa8xmXgypBOq+Ra8Qf+jXGZcSbOzlhqoTyvHFq0FKkdQAsjQ2WbqvQiN68J8tRdvrk3z3YBtU\\\\\\\/JSuOAMtRKA3SD0T7ICAMJxX4MM8yYo8q4x\\\\\\\/9w1HTanwwdYSB+8Lg1Jm+oHhhyeLC5LWmnErhPEh38T\\\\\\\/cN97Yk2MJK7P1k6wrLiCHKLGbfVNgzsTYX6Gcwq7+cDlKg9LMvV1255OLBMKkpeD3iVD2qeSZaryBiTnoQVty5Fswreh6pOtyM5Zss\\\\\\\/rDZjUAJBPnNpLEczzMUqJH9tQgOzH0sBT1gXJZu07tGhFw2US44\\\\\\\/o2nM0A\\\\\\\/9KekbV+4HGBRjWKogSqLW3\\\\\\\/xnAT4iZBgo+1ide1RjZiUfz8RG\\\\\\\/u4CRZ3c4vIGwmzSYWqy6oEOWZzWOhi+w55G4bYVKN0b+qfWBJ3IfbXO9R5sqoQ1RUT1S3rZvhkLOP\\\\\\\/16GwvXXnutz3tzjsXT0WxsgvNsQELYX3CUArdhVHEkcR4dQjWnSlhlFOsIO1ijpEWW0g3NXMXqvoTfiogU1bz7FvYfoa3Wok78ffkxrqpe\\\\\\\/6FK3tDYOlxa6+c2aMNSNRr2a+J+TUNh1ZRzV+0wlWaVTyRxPIb8u1aWv5aro4hGZpw1FV77tWsNLovURcMP9GMxST9aHLWUZtzAhbbFnZRvkRpn8ZowkN312OeSNfLyUi\\\\\\\/mFt9ju9E94C4h6z\\\\\\\/sBiEMaFqpyERwauEVDNRaYzDe\\\\\\\/rVRZSxtqCKMFZ5VXqu2SKoF8gDilBp\\\\\\\/6ioYHapij4TnidpVub5Pd\\\\\\\/4yfHqNtGltqM\\\\\\\/xUQL8HuceEvzG+uMv9ELSOSOdQw9BHWUvPdARPQuJt4hq+XvW\\\\\\\/4WiAJOrOY4Fwig29Q0FuluprDbjZv\\\\\\\/qh4nb1hml5rx2CbeWXnAOE+8w8znlOb+piyueFHSFMQo\\\\\\\/qyMSMusw0wYRzDtnNHJHXr8fC4m2nzcRrr5fiyh5MYELNdntbt\\\u0022,\\\u0022iv\\\u0022:\\\u0022PG6A1jJ84SEtp95URSzOgA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c3e690db7b06a605d8e96467d0e72aa8bade13acf444f4124f1fd12de83d8da7\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022YbJzWyg1VE5zn8Xn6aPfMRyBHNGwWlcQwBJ1qKCIpzmELFpBRrA7fLMxoAIBtWMa7tGNjEwcHChr0DmkmamuYLpA+YYEuLtQEWRblNeEOB11FA7UhUFsh93HrEt9cocafO+9a5opchHTtGqzKUFcsqUDUCSR5IboJ18o\\\\\\\/YvnxlS1s9cvykyOjTsaCGJwB6NR2GytUKeURTdsy79BQl0JI9ncNExmwIOBL7QOpmCcY2cv1LYc7wONQdRcXBcjdQjTur1I8kFnD4TkIA9Owton5x9p64f59938mX\\\\\\\/u\\\\\\\/RR+KFxqTp1Nd4IdtWmXZAaikpLbl01aO1adIpJnA3ulC2c6zIk5RQzJfGqE2iAlBoLzUazTqwVqN9Q6WrYmSfgaPb9HVhPVj517OwY5bhOjoVO5uKwgZHXiEBZBzVODl5X+JUr7j7\\\\\\\/+2jvnV7Ht5OewmUZyk9xb3BEPyRXCoj3HzMGlx8IsOOsB1jJu2oAyo6yZfdbVieEntP0KEgbRg99IOXYS6pVTWIFuXlidhjX6uGi0sOUMWvt377vpdW8WJfdpQjAjC4HMW8DHW2d\\\\\\\/pnbf0WhsSoFoQb0JU2SGsH9iuxm2kuoLc7iv7jS75O0ap7DFchDAtEm1BA2+AvFdZxGhN2n4NiOoNohm+nN77ZUn1RQq0HJZ7RYuLAfA111tAcxjIufrgp8apPFjXMEgc5YBljITaCamEXAWKrycLXZbVrNaJr1ZEn6SgG7xYrPVnBS5mBdZzH6UTyp\\\\\\\/NSZVoFqgWNMbkcJi3gi177ETRUhR6Pxd+x9VcSNgmSFX8uQKi+\\\\\\\/3WaHmqvUvWaRPqA1y7ZWUMI\\\\\\\/Bo+wuz4TrJC92RVhfzbIIN6ebJv7Ohvg9dLs28wTy02v\\\\\\\/Cm1+PAx8NbLvN9P2gGGUcvgbl1pNoW1Q3W1Zu0uZLsV7UTrMniqA\\\\\\\/ZBQ0VW9W2yu0Mz0LoPBLQz0yGYNMmavrdGqEOp7lumlv168oziVpKWV0mcH0WyY9xhEeZYWjGVXpIHpHAWWq9BzOApha2XCSpRRroqbNEQnHMFFWvs0CeCw5Y9rewlhCgCY+vgttRp7i0T5W0w2C5tGscBjxBCMNUcjTyV52Wk4q8eQ5sutM7Ly6iEt8qtDy95YKON40\\\\\\\/iKAwSRnt+Kjrn7D9+s\\\\\\\/\\\\\\\/xKs9OIsn2pf5H2+6dwoaGG9+dCV3vrK\\\\\\\/jlE4APPc1ACMq+E5A6lj+vMSjHrMX29ZGGoJ7MmvZwd\\\\\\\/1iZh\\\\\\\/i0SXjxur7r62cfIcoF4RN9yuPPLyVpJwOy2a6HosnhISV6OaKenm\\\\\\\/lx8I\\\\\\\/C+iny8z\\\\\\\/yUkTTIw2n6gahfS8M8S+ruTFYGeREa+nS5L09Lx5JTLVEwS6yPuUwiztzU\\\\\\\/vnc\\\\\\\/wy3h4wLCzaD1CD\\\\\\\/lNqcDcQjpyODePR36\\\\\\\/fsmQ58nS02SH62agnaZ\\\\\\\/Gvww5JGEhff5H50C1Cfv24Uxk7DCpiuDL5pfZakn75b0IH0Qot8\\\\\\\/BXBAa2iset0lq3EoN04TWeeXeR8ECDRz\\\\\\\/RTF\\\\\\\/kb37\\\\\\\/RZCyJuPOn2zv0UnJ1jrGqYP0BPk4xaIKWFBbscOBt8Vz+TSDyfeLI1\\\\\\\/Pv2moamtFhkfujWDAT1uOb4idUxi9HrkCsG5Ly\\\\\\\/X3IB09GZBL7qKLvthV6NWjnDBksOQhsoeLSnaQmlKgSkd4JEAkg13POUylk4c7\\\\\\\/BApCpgHGBO5eliL9GJ8cZc7Dg\\\\\\\/xSQ9gnJ\\\\\\\/xHaKCF\\\\\\\/IhOlQeLPGYL4DvJHzQJnuvFpayDUDgeTwkmAlm2oUHw+4t6j\\\\\\\/9N9p5w89iWAA88q4vke2e0\\\\\\\/RhiSOUWW2gEDo6pil\\\\\\\/AAm\\\\\\\/6M4CKkdSdhaOJ9cXPRi\\\\\\\/b6fGqzLsLMHdWc2UPhlPO7oyPt+k9FR2pFE5Cvv05sOuPykWWIk+o+j9ewJH3K5hCLEANYHLLe4pFS8eyNAaiwXXcJ66cqLACC0xwYIQUh8fR+7FYoL1N6yjCPJ+d4YG7uPDvWPspuXU1+cdPm76PwVkXC\\\\\\\/FOHcpZ77qpiD1N\\\\\\\/bDNycmw2clmnysc4MXh86NeGASYdV48Vm4us+wVLb9nKtFS785pYX14jF28lPhn89JhQKMd9FZgQsaPrfr+7riylblgYnCuORnTEDNClO03vJszLxRsp7fLfQI5blX6cqgCeReYIFJ\\\\\\\/FoFWWwHvOnMu4yvaSo0DC5Kvm2FivZ7\\\\\\\/sKvPm0k+kRZG72YADvLGJyAFsN7YsDA8GULW7NhiVW0rB66s1CRt6AQU+iORApz4CDQK47Xjuis8vt8XOUOh\\\\\\\/ynp7H2jf\\\\\\\/9cETVGf6T3Ejqk0j3YT5LG\\\\\\\/Y75TZs5TiuwVloNi+YIcuMKaTv3DUjBrQSz7yv0nTGboBGBwKiRtnYoOGD7Tgh3hAmnQVG7lCt8yoRm5CgZ9CXNK1X5nXGdbehlYIZJqfudGlJmWPXbZNBrCXs3\\\\\\\/Y5sCyB0pIrJLGkA7yJrasu1LYqBSXiYKgER2WQmDitX7ipan2pXPIur8rZT7sPxrw7wzUGCDtkNy8SosH5BH3ChP5+vnuTcclT1bNl\\\\\\\/8y7pTYfBO5+\\\\\\\/5iy5z\\\\\\\/JK2QXA+DDZ9jswe15HE8iCsyQlixtf1HDKKE+imSoCTfjyTqJPfOyTd4zYzl6ueBfSoP6KXPGoxsd0VPQ2vgRc2IUDF7erUIOE5xnaiswZnjKP5Qw8f9o3Rhi2mE1jW704ukXDwLpbVEQzkq18P5jZpYbuleDTfYlXyQYDlF+Pizm6pNvcdo2m6HO7FKNhBkEynVDTjwwybwPhJOL5ZvdjR5ZANom40psG7ua\\\\\\\/HXQkW7\\\\\\\/lV61WkHtrld+w8UsDlQzmVo\\\\\\\/xnloevMAMc8jFaSzHS+lGCQHnQ3CO4uebrTCZFFYgzWs5LEo2nYBNKFIVK+r5F3m4zu3PzgjC\\\\\\\/ZCpa3UnpH0Qpea7Uq9FRCA7+3UYpooSV\\\\\\\/uas43uNbQG7w7tN19Vl7C6RqZXjXpqU8oEtJil6TGwKnyoborapA\\\\\\\/GnrfhGlFXw5jQREYJZA8Rsc7uj8dsmoKy3F9MhvrPYplKQwCpHzWUJ6LBXMSzjkZ2qVUTkXpLN7vMpC4dSzVmPN6+LXLvoeHXXGpBRfa9Ib53IbsQYAmDzana0uz7bpNx7ojVjkJv3Ze6XAEAPsYAVLVFxggnNssIX6TW23xFzpn4geELf4zfMDmBCf\\\\\\\/uZQ2LDigdQpRY6xGl\\\\\\\/jDWwULw40UD\\\\\\\/MrYPmocCOj7UdSp8OKDzs6c1Fj7bVU1LwAXy+MF9voSUu7StSBP9AeyPQG0eRMFK3tlqN1ku9VGjQsvVFm0bCwIsPRbJTfVzH\\\\\\\/h+ERTTGpwghGQSl9jEfBIGLJCNA6aguLraJ84RwOtv8eRGMk92dcQVQF1bsSzT+EQT1Sz+oNMiN7sbYj9gFKsTy3zMAgdFXbzslwbWAXTJcDaR4IdwzllOg5ht\\\\\\\/HcTPo2onId+Z5tXEI1UBC+oGIytcQWPzvGP5nmahcu1u+SLlFnmqCJLcd\\\\\\\/x\\\\\\\/cl5hys8aXTUpPeTlhzk1ANZTrIPbqHg1FFEWPKxrAeyTCGjzg9hqELEVUIFVy\\\\\\\/qN4mSRlkI1SOBvFqHhhWSjUwl8ae+dkF8uW5i3nv3lF8HJwkbkr8EQnTFbMvyqHJ0x0sRSwF+yVs4YRTc3\\\\\\\/ORPUrPWBZEsaRyh8oZIeyBiCBWtsOUQqOX4ITo4tM5ciyB1kt5quQEgcdJRPSyGR8MSzG45gdgg6eKUixMoEEG32wl1djf4Nq+6yD9hTK\\\\\\\/cw46UllnBE4mcZKNU7nwzkABlJY6j1uS4DqwMabxcm3isdzTYAuX3u8DIlEJGO3MOtDxSWAk6Ju2HPT++6GFikBzaFJB+FzcF5o20oeoxToCYiV7SN0Y9PJCshbcSyRHOXtS68RZBQ3T0oDb4iii5Xblt0afbQPYSgxMKZj1Ky4jlBXvdaZDYH1vpyfmQAg\\\\\\\/HhlB6YDlzhgBk04im5hiG700qe5IbQnViB6xAb3Nt3f0lluM5kth1VUnUTQ8XsXs\\\\\\\/CzUw8dz3bS7wv3XPfjqv5qFfaVwza5zrJ9xbXAXw1lmjdSno5Q7X4eCF8r8Hc7QUlGIUB5Yd91eFMdnMQd+YDZVfDivEcxyYts+rH0IJpBr4+G4fH+wajYLjeXlljn3Yq2BmJNROYW+VFPd8GS0TPBcoD+rzFg3HxRiDq8t4lkMs\\\\\\\/wTs4NraVrsH5\\\\\\\/OM4eZV8s4mi6oYyw1Nhz3W4DcauV2HzbXmxSk2tYCbj+Z8wb4x16D6cKbjI72SD3qI6tlaWmsDRKsLPXhRiZUF8mCaEOykmC0E642CvoUdVcIPdibz3lXGADoPnIcy\\\\\\\/SWIR0lYQ872ivn1QT9NrXHFex2eo8RtslVVuBbJnf4NA2dMZrpn1t4lYKZb57SAP3Ipr\\\\\\\/lG3J1NE9CBS1GXMXDlTA1ghIGZrjpRKhnmW\\\\\\\/Ks2cFliVkWd\\\\\\\/l+6QlXtBml5Q0S6ahqUj2YhwOPeIBPZRaDGQuFMBKDeOP2L8Mui9krqT8JrCeZNq1\\\\\\\/+6++uD3XMbjdacG9U2DU3xIIP25xagVQYAYFmprP1iuItsIKpXqypDaUQ4PriHDK\\\\\\\/8bE7BwyiWC4CMh\\\\\\\/7lypvYyFEp+Cy7sUoV7r\\\\\\\/D6xJpPiFt1XoFEmWMFvpxSeDxz10JPh+uls7etKB2szJqd96NHlTB7ow6Vw7xcQhGYcnFvS5A\\\\\\\/QPTU9vIuMSrDb9wU+k1f9kY6A+0jEpk\\\\\\\/DP0ecsuwfBVjQsiJPHpTg2S8bdd0fUaLQIvkWJs\\\\\\\/ZckdB09mIpKGP4sBLR5rgAqfAmO1aoW9NlGiFwO8bI0e9SMZZLAAKmTKXW8vTqVZlLa5NnltpbmzmEQZksaK6l\\\\\\\/\\\\\\\/Cd1wdpf7f2SqxOzhU9A47vCFbGNuZs8WVPlhfw+G9o1oiD1iUvNiKao5KS6z6ZlA7NVYYU2fnsC2AKXKyrmpFym3PPgaFunMh3glmXwEwtIrft4zVJd6YUbSUo1EICOngaP\\\\\\\/\\\\\\\/VAeEiGzl\\\\\\\/PCXGoO7GG4P6JMeC7p\\\\\\\/COw6sFwd7\\\\\\\/rTx1WB1xMkmO6YptgKwtkRWZ8ihJIwaO6iwSmkl9Bt3Bt+CtPoF4W6TbTyQXn1KBiYcaKEQV2I0DUXy5U7X33KojuJA4THBRpPPp3drOJQGCWMLPaJObk8VxA6Fi1UsxII7D03EEhkIyTTI5P0Z+D7E7EWGBRe2R3N1skrdTlQblz3YfdIvbhcUvfZafAnrt5LJHE+EEKcq\\\\\\\/h0337aI1bqziE2KEn16IEJRLtQNx8l9Ot14+NszvwXARhuv8ZGnaC18QqvDfMgdkpn3IAe+IBMfTQru7ZgxzACE2VkH9LiiCSVCBHns8atEs266bvHFVYBQINyKBGN5DSHvAu5qEnyM5bNetQ7zldY3pYAQw4Rrw2tzvArpYiELTHXAD2PUi+p9r2Skl5av+9xQxHH1AVI5HdxnRaKv5D4n9\\\\\\\/hNnqVjnx2+fe1qb3SgdXxhSUw7iblZVSzvskk6VIo37njEMke0qGwKL1rynkUkpZzRGfyGbBm6W25w4p9r33NM4MK8who1MiVTGLQK9tYM1RRy1n4DwbSLRth6S+p4XCnHnNzl\\\\\\\/t78VVPTqHlWjItNoVX9xPLQ8HumpptnGcpq0m8WysE+rmm8oY6v5oxutZxdIYqAjrWvraR7ZsdfUEzvu6uhOd8TzjkC7UYEFZSmUXe1Z7ecXV0WYRTW8ag8QXHMS4+NkphM7Po4R0HGgGnGW7xZjTecJ8mO6AxKtRCwyApIKDgr6dsqwRC9o57e4j16mwEqinFWKHt3gTj04+K2ZHzDr6jWh71aX39rCHM8KZoQNem6ENrwNTEgFX5hewmPHaoA6HhCRqpX+P9FsKCZl9mQ59NgwSHkqj12nSgIFnB0ylVSXbn6Y9+XziKkP7M2xQpw6j1l8jryaHQWWUQKP3HNN6caz0Gj48sQz9oR5HDrMcOmew1\\\\\\\/+1grx6Ci9+X2vWgBesnTUCbMWBupTJ1WbWW+VjtRn4ybQnJ64+eBErfp1B4v8n7ey4LiFV4O8xIczrejiDMcmKJoVqLHxfrayisadOWf1jUvY00NOhoaZFTC04K5rInpqWpETL9AoOxDzZ\\\\\\\/Gts\\\\\\\/gu6f4STNGMsIUYizam9j99cl+fX77j\\\\\\\/jJATsgaQOOBhJ2huZ9EH7gSmeDfL9LbPMNTSHIzj4In7MAces9O4uN6W5WprMsem+ukeEEcyxrCGG6Z7RdfOjGwn7iu+kLLV0lZWPXmLCO+512tOzR9DbaKk6sUTfafLvgfTU+5Meq0zXy7YcEi+rVKKDkfNKNqbX+Gi7i9PVb\\\\\\\/R1V5JQEFNhMqctYU5Y6grp8RbjngNcb\\\\\\\/4MU6EAjqlfPwcrQzkS8qAvSUfJ0fqfDFoWSP8skBq9f0nlj6D29LTudo2Qb7F6Ni\\\\\\\/ckWX0oIZpyTtHukYAHHNJtfCp28HF8mJ7LWHEcUG4jf4+A\\\\\\\/dBAO5xrzzWYWfPV9aKRVpjMmBvRrYTI3HjDF3eiiyaLLmVa77GKQO+ebYabWk6pswX3zoo6cUWXDMjqhsKd7doyzI4TZSBvvmHlJViUmilniWHigpa5MC0ippUZxKRHIRBr2tHR1Cs7FAIbiRRS871KPvbwT2eXO1Cqrxtcjn9XhcM3ie8DFnTzQ7WyabLRlAB7HF4bmkKCpZr5tMZUqA76z3F6h8B8GFi4wubret0sGe4OlnAliDe+svD42akjmJ4pzTmlwz7pALi7tVcnVqZWQOqoGgNHEN6BT9t9pqiib3xePdO1uq\\\\\\\/YpJaO+cBSJXf4B3DHnVZ3+7JayIFDPky+Ly7TsLpj7v4hJhPjRNsaFBpwALmjOm8jbFbkGbjti75dIoOmtKvHB06Cy\\\\\\\/DIt67aS\\\\\\\/GOpXjf2WyzMkKi7DsxyTXBPYzPnq+4CKtAzDZ7OwnZt4Fj9LlpeO+Wa7JA28GEFEARGAkk9Ss7VdS4sgZARltGviohBI2aHcPHLR2k8JrvEJctNyWIaG6zHnjYqBcw5xQf9Uqqi3Rw+\\\\\\\/Iat+kcE7eRWf\\\\\\\/+hwVR5VXUD9PxdvYBysnL2lH7Crm\\\\\\\/1CA5pkEvIFZ17qDiFf2\\\\\\\/lHexUmLOsO1g7r4EtGIUSfyrEr0jEb65rYACUc2O0kiD+NRBrksWQ2nihTzc\\\\\\\/UdgGNkV79k9GpE3+4GbBL6U0SwEydzsuDcI2oI7MXMSyYuX\\\\\\\/g4TakVYb1KQMSeP1coNB84XqbCquHKFmk49TQC6uuVYLGtFXMPOMuAoo1Z0au7Yz5e0oH\\\\\\\/8cpTgTEeN19VCykblPLDbG22x0Ewrc7GYy1Hh5um1+wyevXr8PBHdKH3Ft7Q0FlF9YZAD8CxYTcoZTAka2G5+4gkU6iuM3xk4WPgSQ0ns6T46MXMoxXGuGmUVYltpIfMwI4ioRRhDb90RDhCbOGyqcf+lwJz1zPk\\\\\\\/mAZ109rX1fzYzOG1TrF4B2AsczQUEq8C3uUVTRHkD14KWbfcQKpJXXjSEF6Wc\\\\\\\/flk6+FE9Cneem9czHTZiRitbyHW9udrTVz\\\\\\\/uPB0W3vsiRtY1vBEWrNos\\\\\\\/7g\\\\\\\/X\\\\\\\/nrivL6qP7m7pU\\\\\\\/okHE0dwl4iFxmn1sZsHOt+pckvJykmDHJt4pNTS9trzifdr4wGKxrUK4A6kFu8ZiXWf9dXSOO6cnZZzuBqOcRkeA+p7EZlToaJHatsv\\\\\\\/d7stxBjvkbDqtgeYjiYGzzHj2uVLwEXf42FBXXCdbKZu7IiL2bwh1csbTCsuzkj\\\\\\\/WGWJhBPS9vnfwR3OQCd4pLETRBuwSfTClwukPcz\\\\\\\/LJVSsE8A9cz+Muk\\\\\\\/Aba3geuInTO7RtRGPxv5H3M635hfebyqSFmFFPoY00rLopqxwHJcPVBbcXV7bH\\\\\\\/YBQgqYvn3\\\\\\\/JvtyAInUu5lT4M6EZ5I2vrBYRJsDddr0uWr\\\\\\\/CqAW1ZmlMYN0ays6JKSSgU3C9HZwBDy9IW0BR2yOtUn5a34E3kECiGBoDstD5CbKjACYGnGL5oVJ66nSTiUfKTiBd2\\\\\\\/0hFkmukJVg6OBX7lcL0QyA5aC0y5yvRDdpXXuNj3c3AyKtwfwGZFeb+rCMc7iDSbCzrPE499Ymp+fDe+uugcOm4VF39tjSuKZ7NMnZ+KJ4X01QD5Rta30HfxCchDzqvi9wpbPrxHr2Wie60F5yBXDnxnBObClgmL+OexA8D2KmPSel9fWWAmd309egrxuTBYVjaUjvXqRCwkhi+DgJ8Ftb9ACd9zQdG90b5sGDLpR0yigzxi0VlgWoAnheT8HJAZi2l+worFbErFxrGt9y7h3PhN3v39r74Vf\\\\\\\/88mjPnJh6mFlHh9e6IhogLIis5eVPIUZMrC7feA9gPdsiYrsGMGgWF2yEucaf5lq8krgYIZpFlT1pE0ZXOJe6a5+xk6q24pvLiDeuoZ127q8K2ExXwWHxSvlP66sJW8NlZ+TdOf8UuhTh5nbugiKY43unPRvuo3pOrlzbcvl75xogrRsxTtwcJOo0EmTryKbR+Ln3aQieraDvXP2twrscY1cBjeznNBxtrS\\\\\\\/ig5I\\\\\\\/E3ziFZlqwIRQ0Z6QZyzbO7NC9Kty5SSkZABPMOygU96AxB0yXyy1WGSspsbr7Czvh6xpd7hMMYStcfV2+SJFcRmfCfGI2fnXSW1epi1\\\\\\\/GrSvIkH7R6UAJY9XLHyz2JsjCgzYGWSbwj95I5kAReF5yiqvrQkBHWvwa9r\\\\\\\/Lgxj7gZFzigbd3RDjSFFakeEbG9NA3B\\\\\\\/HyVZlLt0ZY4M7\\\\\\\/EbSYVaW+6mI381nUaKLNnJ5NJl1vexDJkcWvOxy9Q97hLJgRFfEx7WZkz0mb2\\\\\\\/kzmSHI3+91WOoQrIR9FwCpyUT4svGBQqOg+CcPVstGmPPHzE9SdgKJMPlCKMPRjiH1OOnn9KjqKx5NPlIF6p1NaWpaCFicWOYb2zNYndnAdPnrJiUaqI3jZpvygGhujVlSY9rJRklokrVICTL4z9KA4Ks11MXwPMCYkU0Y0Us\\\\\\\/8bQkbc4hovr8E47QyJBWEQSkzPT1pSp\\\\\\\/WKskSeTwk+4pPILq0Oqsop4cIsNRUAuBBIa5TtqbQdOZ\\\\\\\/BetXSg\\\\\\\/n6HvT3C2jir8Qsjefz4NjLxvLEnU8YK6P9K+jBOJ+yhcl1bf\\\\\\\/Byoj+UT+PaWnci7n1ckK0Uoc+ks6iHMRAeOmk+7jmTDZWTcau3I1DUzZ4Kx\\\\\\\/Ccck8NEWld0zeTG677Bkgw+Wp3ihKyAsxmBFXgf0Zbo7WLQeRsDBhVhOJmNe5XlDB0PFnBkwVCD8G5BtARSx3jRAjfh\\\\\\\/YQobGHRjmghS8k1I2TTU0bkiQUO3zvHS1DjdzzguwTxIQ4TGsYwjAw7XjMj0MWMro4a7PFlkCfqAzilmfl+QEDtqesNjhFkYOQVjobwCXzEfy983SqKPDQBjfgYcFB52C7kZ3kvSO732YKCI3HfF9E5mkgTyHDURCs5T2WnBy4G895g4edddjZS\\\\\\\/us4UVDQtsuTV3uplqIqgfrvbJR2I+6bxRSpitSPg+u2wB1Awiy9cicneY1JhUvpQ2974h7EFIkqbFhLbOLOGGGY0o1+tfaJWWDFZoH7uVax3\\\\\\\/BR1tupMLuj3eaKiBoVun1WGVLN6Q5dPenQNzsySAmXImeln4Ft13Jvy3sg\\\\\\\/mx9sHKp1GZ5JgbBa00E6GPkF1FXb8QXMb+TRZkAodrxZTBG3ZjYVG8KLL5\\\\\\\/8xOWZMJZkFX78yzhuPU4gD0Zs5pBtDbrR80fIbDJ0gyHXAJ9gPR\\\\\\\/\\\\\\\/KONVQsd7dopaUTaI6+S1oMM5wLVxweTzloTgAdp3sc\\\\\\\/CSlXhScUafHtqxDBRbN6oTdy\\\\\\\/1x7FsGJIqxhF1tMTZAbbN6b1nW3h1lVGSJxHtWUUHlnpKevF4zDBSXrmKAothHbmDCnAjV1seXXvap1U2uMi8aUkzjgefAxMCsUNwEGVURkkrYvJgT\\\\\\\/4oSOsWRHu6YtOIwYFwMixMHcB+JMK0WZekwha8k6C\\\\\\\/Ys4k0iAoC1ikUOBeewLtTioqGT2sj\\\\\\\/P\\\\\\\/HH95hGeWj\\\\\\\/ds1kxQXWBLbtrtnZIq+x0RunDinYW15rmy2J+POlxZCUcS9Oj0CuJCrE\\\\\\\/kemtyj7I+Shknx2t9hwGmyZ7xQqryOlUtEd6Uddl0AAzBRE4Lz9G+nMPq3eifpr4Avrz5DkIMjhBYgrBBLy2hy0gMjB1z4GK7Jk\\\\\\\/dn53Ka18VP6LM1dCWHOBFHNiXMlzGRWdllxN6hfh0Qk7dV7Hspy5S72ZXCEI3gAw4g9DQv83cSEoaMdhUUstHVLe8S6RM\\\\\\\/Q+39+N2k2eqf3iplFyRM9vEIlBNaMsIZD7K8priAgNV9zrSZZepg0ILHDEVW8r65gpnQnMk46D5dsWNP2pLDmY86KCSY+0h2+wEkkfj75ZDz8tHopUJvKPBDsWU4FianT1E7iglAFpa+kEwX0LUfLXaLDktvmXh2HQe2CiDe8JOAQzeI4hSYFiSApAmhAPkxQrwjJD9nW4tCAH1GESujYF9oFEmJaYdTi9rbq4RcqYZY6iRCklMtcsh1m+9P4wvZHWr+Kl4yChKb1BdYCNKDxkhUIugk0jhhK3t2A6ZfzaKCejpcFJDZ2Dpjigk5ho\\\\\\\/F1svwF5X5BqePe+3jR5umjQgRATxzvOYPLlR6KhptnJNSdINxtYvCzK5gMbJwoA2S9Ny5IRF5iwPEVCmcZg3tZ5RM6bhXvGnHJrtYBQtAPt4dnz7DFOSkyI7lkpLxxaiQglPbu\\\\\\\/1QFZuLL0BWeyQrk7g\\\\\\\/cA1tQkq+34oD98yiwi3kUoG9rq9dUxO4g5CSBo3zcCcM+Bo5O+TEMXodtUVXra6Eq\\\\\\\/mVGt1FdcmS4S8XuSePEfNoQDe0SOiKWKGsQq6DKKTrxfoSSRgcxs8YfzSdh7DeHhaon+fYCrB5QC3SmEJGQkCEdZgNkVCXNeCE1t\\\\\\\/xdF7lT7+cHzNbSJCHi5K\\\\\\\/UyuDrzSw\\\\\\\/JXGk9VeBIIx1yh0u3fbZMMfCgbVGqzBxJ+YW3M9XBBCxnpNUbYE1GPRVGatCusH6R4upTQSJNKBgEyM8\\\\\\\/+hatuLR\\\\\\\/OAXdsiUfVtxAf6OE23aTKTXsw58phRUX1uja5VaGtD7JyB2AIVuEAzSNCrVU8jeNhzzRc+JH2AnMQphvSAGjurh\\\\\\\/a2wcO9iW0mTQS8uoiHM9JTRfnLL7JK5AIg3TigHKI0mWazxEdtoo0E\\\\\\\/Hz+w\\\\\\\/KPejeTOArX9J1raYdo3yDTKGVL+n\\\\\\\/WkmX4Bo1fYTcF16eUleUkeaS3GE5e15LnFDpwc8+hNYpQRPJUByLTvUSCLXE1mCmJT4vnKLpEqQ9+yEYSxtcYYpWtpjL5xAzYhB93Qt6I+7kngrk8krXdUWwSIoTp2Bvl1wlwEy4262LNfHKwWOWMzgcBdoFjptsmd7QS3\\\\\\\/YgHjTu7S4DD25bZUbxp2L\\\\\\\/eBx6TGUYPIVo9wh7gUXLcQnwOpaSiPXY7r+1yPCllsrXm+uY3rxv+yKvaKcYUEK2FJSr2zx52tRL9J47zz5VN6eWKJEamEJJioVh9JocGoFapKdkUaHcQHt5yiKNcRQP+AFLXVYlO7P8AAuTuHyRCbbhEryLp7syTuvPPA8CKfEu75BMyZxMx6DA5QGN73bjeX029JLlaRVVmXGQjItAU2CmfaybeOa\\\\\\\/hG7RO6JVbmIMUvq1tZiQW2p\\\\\\\/wiTlP9Yimd4Yl7KKJEjUgJHwu4Jy0n6NkROxiI1OAhhjaSLuUnPP9UBPCPRRfIMC0TYr041JHbdgdeew1AzIiQE+bUcnQ3vjd9cL7CHm2XO4HNi13YWpeG300NQzw9zbuzIcKnMlcEYvYmx\\\\\\\/f8ztOz4zr8\\\\\\\/QOZZB+TkNpDvV2qhz\\\\\\\/XBC3CGw6l+PdOasRGJoqmCOrD3\\\\\\\/AZTBUK5I9wlwsTRLx97E+Lh7mT9N+4ROMb\\\\\\\/3\\\\\\\/VZJpBxffvEf1etVZ88fXWTVtdcHXEohj3AUldE82p7ncWmDUNNWUjh\\\\\\\/HKzWF8j1Ze95bG523oaZYTlY36P4VoVAtz9staFckddG1XbPPqFtIKENA6IAXYPjBxW9WIcN16gLsep2l6x8DtxjlriIxS1iTrG3Btkf3nCXQ0kgRfnFEHyGLxrLUpfUYZQAt+JWVITBg6BqpWhrWqoy9mMBIHkgQZFFXFKO\\\\\\\/iahQWIQ\\\\\\\/\\\\\\\/cYJhhqnM3htlFoXVej8zTzzK3iNhqpyDyTVLj2GPiJHZVh\\\\\\\/VYbDlMpTZ64w5UkLWGFjgjPwqNFsUEkIMeStFD\\\\\\\/jbrcnp\\\\\\\/2rAsSjYztwgV4\\\\\\\/0IBwhsi3jRM3eyQMqmD86OV+gzmCc5sZbVcBJteTT5RwcUe1vtOFuAHJys2qc\\\\\\\/\\\\\\\/y43DeI1pY3c8IFNlaSX7lgFUMid7Bg0BCmuo1XuTGdjiStvhgU4ChZ0oCfP\\\\\\\/yPrVKAZvYk1NxdoSVvcS36c0fjfUTMhPDpilBw2hO8Qejemqx\\\\\\\/RXDgT3O0KDCQDZx3G+R6jklmBZmROD2aP4ytIdFXW9CZN89bvdWi5qvgsTLjUbgadUDVIu8oNqbh1yvK16cr1cHKctoNaytXP5Cwx0+o3GicRQlxZwjOM3+FT7NthlXiVWvcVZU1Lebj5u8ieWZyEx4J38\\\\\\\/mW8gh9zRCKYpP3UWiytiIBR9hPKblB3ocMq0Y31MFiuulFETqzvnr6gGXSWgfkuep8QkjiZp3uR6en4YsB0z2QEw2+3jci8JJprYgzg6ALP8FbObc5qD\\\\\\\/wZxwFX8xYqlxQs592fr3l5sX4Mh7o3YAseIPxsE8tMAlBivDwroppmSYqx8lqhTkD\\\\\\\/2ScFqxxggB1GJftOEVJLS4PGhWsexakD\\\\\\\/yZWGXJR92g33\\\\\\\/mdpz4t9Iazo\\\\\\\/HeU7FmKXhq+u\\\\\\\/Tw9Enh+JqZwBua12Bw6DCsKme7U4mVnhm1sVFJhxCo373Aqv8Z5O9lxl0z6cq+udzuonhxo+Q+oytlhXm+EZb4HtZsD3X3trMpEyAelM72\\\\\\\/HHU4EsFIb5xXP6vSW\\\\\\\/GkLyHpAND1wBkSAlucCsDsQxvagCTCAOWwJYv0lgEmU+7+h9x3z0Or7\\\\\\\/V4TWqN87IPde6LADQdVvhi0wwbGnNYVVsRwRjcIB6j3H6000dHGhhnRhIJmQaZRlMB\\\\\\\/fE+otBWy8nQz8AEcTu1kAFuMDZq6SCos\\\\\\\/sZ8y9lJrN7UW5MZJE5mS9kwpJBVX2LuDryYlQbEiCggI8GHDww6oA9PmRUf2NUdq1zYewWnFg8ujgLLWtBFAbZT\\\\\\\/\\\\\\\/NjaWNCROiYcsSfyvQ45CWfa2YVSywt5aa7DMIjKuHquZo2k8S1oedz\\\\\\\/97p\\\\\\\/GPXbYRrTJF1eE5hMTw0awUKbuo50Qo0cCiTLih9oIuIDC9fVIhkXtTu03XRVSXDE\\\\\\\/eLBkhoXxAixJ5Liv71ssVmeOHE8nW8plhWB9zSpucoyJmEg7CCTFn8ayHDkGGPW+GqOJoq3atLI00a\\\\\\\/UOpdhR5RR9yoJNx3bbp19ncxHRX+5WS758GbmPzQUjzIVofdAWQVrjt88rVbg9iAocC93YNmg+ZnejfRQ6IEEgi+NN2ppMAskEhVrKRK49Ttb9yDfl9MOqOMVu+mLb+AKGzuALALMGM0MM8DcPD849843HvxvhmK4GtABWIgrukiPmnMRa3BVztDPpXi\\\\\\\/M1ya+gqyC3jZ+JdyW5rSCJ7bf9H8VyG\\\\\\\/KulyjSUSlZTQIZIY\\\\\\\/LJLSBnExXwe6F6+ag5t5U58pSrG6jI2t5lvYto75v474syZcUQszATW5IJLoHUshgSZNTKpZ231\\\\\\\/hr0Odqi8+E5wsjojL1P2ek6lfNC5tW55csUuxSn3PkyDvG7nmFt+zn6b3W7YsRkFhS2mbXwt\\\\\\\/HQxM8kMDPkVuwAxX7UWhqoZBM\\\\\\\/qe\\\\\\\/2z1IO073rQua3tY8fKckKhEdR8wdWbqMW4CvEotMg+Dcu1WukWiBRdN8QLbCiPnkNoI0LvU+nu+w8Hngd+IdNTmJ\\\\\\\/jTSLvoowTZgzcrCSzstpwMu8UGjfbb\\\\\\\/W\\\\\\\/gJa2wzISuAVhrcno8Z8vArRMtrqOfZ9osHvdluUQ0+T91Z6UQ9veahSyTeqyDyv40a9nEYpAX0RymkDGPyvpTiLOo8aAy0Ed6t9tPuo72r7VJQmlJEmeV5nv7ahAR+WD1IlSAterPKmhobBPbV8ZWuDIavsWjMq\\\\\\\/DfqBbGZzgs3ecmcgcvcJQY1WdCkX0bWRlDqxyOyKVlJNU1U2AW4tGqB2AAW7N4biz35t1EtepjCdMSlVaos9lOorJYXWiYGizjGQctqtszdqf\\\\\\\/XHOeitqMy9dITnUn+ILzVY7Cbnt39Lw5HycJ3GZhA5RpKoA4K0fWD5AV6cGnDzMhS90ypZEJaEx40cy6rnnPvFp0a\\\\\\\/L3inDxPN+kE4ckpY8PWNkVLL381WjKXgUhAvZWuIA3+xprTqoqBM7pWFij01u9POtgvsppMfv2fMSYJFLe009cDv9Mhkcs2R3\\\\\\\/W6\\\\\\\/NRP5yN2KP2DBFGCivY3CdPXrT6\\\\\\\/avfaXR0vqGoeuf5Gd\\\\\\\/aZB7Ztjsank+yqPu2Y04B7gNK\\\\\\\/AmS3XESWM2bU8WzGcK6LCfaYTyaV4C73pDHILSuspJ35RbY7DCAL9g+ODcLrpkXvwgBDjsUd+qbPRHS8A4QyuPuHSI2x2U8uhGCb0dyYDs5Ao2DqPMigb0DYwjr0Q54R3uCrKv4QagdlBlDNg3usJIXdicdBDUqFZCUqWs1iHk6Ptkl67LziLg0rp\\\\\\\/h1KhXmpEc43Uj6cdNKqHUL2DytbM3qFbQQygCAMFtlFn5JPL6QBaD1WyHmjyLOTbLa7KQ\\\\\\\/qD\\\\\\\/OPxRTbiyDaoA\\\\\\\/YbQMYxomjg6CtieQhfc123pgfZvkCRfEFT1JUExFTbBylMOxa\\\\\\\/J2pn1jIeTBQrqR8ZIzvXYwj2zhBye34UYGz4CoRjo4nK+HlfQh7vxuXc7pJc+lBo\\\\\\\/C+0zbfN7cPL1VtnF2WrMSgZNsVcyzciA51yRjqCa0v56O2gmh7H2obgMdD4uUhLKFni+nelI7PYJigP40PWokjMS1x6eBaWX8fvC5KG3iWmB+zW\\\\\\\/w9ih\\\\\\\/GSHh42KvcBLIF6jYEb3DC1DduvuQAtnhuiO+rzynDgLV7ZcNydDg+T5UM\\\\\\\/j8wSt3hZ9CBdYthmf9eopUEJ1xIstGxKUawtYJ6DXHhHUHpaNqln2gFpUiBFvNgnc0bnQK4xD+Ffev9kQ1Ff6JZFmgpQvhWK+IJ6eXvkfP2eym1lW6tVl13C67jB0xOZlc0q\\\\\\\/KMdz7TrG6KIxgH3wGgW+kQY+bYnp4pk3lUvrR7aMgaLa\\\\\\\/oyygUS9L1+J8I6szEbrEUVkyHQrZMbHzpxjSON8KHci+v20AzoKo2dViN5wpbAih8msJnkIBTAK4MjptPKfiDDtRavubY8wFHqNn9wbJiZlvI8kwXv4MrQF2xP42Ek8rfx3oJHgEAbOHfsejqJj\\\\\\\/dKpZV8inSpdveBtY7lngHMwkax3GalHjsBFnKvpniOBD8+ZBqxAkqAuvNxXjRcP476ainbMiFylemyw9sOdUMAqWphpTiFtmf9abwneh9rJ5FcdkuVN5hBPhZYEkNTLrE4Cc6hOqlakbUshPuLvQTfYljjmOO4L2EySL00mUjl7EEOP7oroktzGfcqZTHiu\\\\\\\/8kHVEEnSqQDtDyiPp+f\\\\\\\/gmyZc\\\\\\\/IU1WM9lGxW4I8ExWJKDxfZmUXQlEIUgl\\\\\\\/bgBeLFPFXs5lsTy1c8i\\\\\\\/pU20YctMw\\\\\\\/7q3NbeENdX4Azp7ODXWtgkWTipPAqVBtLf3QJROlTusT45yeSntdAocMIlootMHq\\\\\\\/L5mI+5CEi9piJFalGnxzAXa\\\\\\\/sMaxkZbvffklEZM5RlE+t7hMTSbxve\\\\\\\/7VVT13vypeoV+cyA596Oq9C5E7WvHrSqMwZiVSIJFlBaQGHKpmHFKgEttOnci8HnjFNS6aRLa6jIUqo1uTWbmQVJhkjgoRAfwUgJLT4Zu3J9PoI90w+micSqzc0gjiU5g2Vv9HMLCUr+z7BJ8IwRBNqlLbIBTsCLQUyxfLTYW724C50xCDkU+oDPahQkZOHJA7BViO1eAlJ8lO\\\\\\\/nqHLjVKPbgpU0mgugamrpbWJ4KOGQ\\\\\\\/BlIv5864rRddflE0IMNPExjJT2Guk+LTzKJ+j5kKLlQpoeNqTpJyxRdAe4prsZjGeO88o0Jk\\\\\\\/MTcQIDYhdH\\\\\\\/e0ZRGf\\\\\\\/CDHcd9mcBKoEV16YKclwJssj5PKRU0rfd22GBT8CRXnsT8XFRbLWAqVLH+Z+3kJqvAd+aikA3LW5hLOkkSfY+6crGDOmVBYxXevgZqxxedCTDDOEPYqyCwaUhseOTLj0p6isGuQkSkb0hS2pAUPyFm4J2E6EbrpyKFHFtfynwRXU5h0HaH0hm28QSncMwt+kB5LapiYgWGcLI6qqoczXxVo4Zgv9GsSTRiSkmFKB0lPbCWdP2al8UeBAQTgmm2V1tFg3IOq5hWPbsXKLH1LJTsuLFqGmdzqck3gfPaxpG2zmwyLAceU+e07zcLKmS3pxh91yj5k6zC6IUTxqxilVD6fukKfNXL6fPKu3ql2gVwUZY\\\\\\\/daqhx26TLeuAEShdR4hlKIrtJjTBzZgCBe5TbxtsMXIELMWBE5lIrphc\\\\\\\/hmxvBpI5cjQd25eNtR8pDPV073Gxgem0OXDu\\\\\\\/SsmkXORtPkLLVOBkdAFdmpqRDkY\\\\\\\/Lc0CfqUNEpqNuGl8qbE\\\\\\\/ArL3YDS6tzOVpbh1SgTxjrL4Yy12HVwWhuieDNv2d2VwVnJCmY4Qrc1wr5De4ey0cXz8ZwpW36VMsD2ekM4rhdijxx55q\\\\\\\/9r2l1gop5F+thgRqvVEogX8JYhl6HUFBOkuBNcKH2EB7EsljKKWGK7UOZPkVlgQ3I6C8R\\\\\\\/sgdkJdm3ycX7rlY8OEhAzo6vAP7a58Y6WVxosoNwPsULF187GbHZMfeNoyXjJczFDlYp+jApkM5ApEw6zVxrtkAL53w56wZtH5Iaxte6GDzfyCjrs00pOwwcsgjo+8UtsEkCkaxHiT6gTKEZk43MxCctqm1iEbxR8JvraUVZwDkW3hS5GTsIIHC0e\\\\\\\/D\\\\\\\/Yn6FEcvEvoNorrMeocmaa2qKQCevoYHzWoYLe8CF0W0CB0ZcwYI4IVFgDpIezWIcAiSFWXBwXowO70wKf8B69nYkH4qlbPGglt4Mcu2sOpxX2J9VzqOnHA\\\\\\\/SrNGFGx+2AaB0DLLGwteMQiyxd6g535Z5u6Regn04tDqjQo0HhntC610vD79V6CP\\\\\\\/Q2Q2TU7OKlSL\\\\\\\/Uof2YTKAX\\\\\\\/+v075pqrsRZ7J50Browv7LydsjG8Fi3YsoKalWXSWETTbh5zAM9ztP3J+xw5sjxsU68WAv+wYWGRSeOLaMMzGujFaU4BCO+kxmnx7eQ7roskvmNAW9F8Kud1lrnWjJmkfZ21k8Pqrk+G7O\\\\\\\/Pfr\\\\\\\/NJLl0umitbUqvnBz8lG+z6HR1a6NvpeuuMI8RkP6CLOQPAP7GVH\\\\\\\/eFgYO7T0PcbT3z3nT8\\\\\\\/BgqWhJjrAUbkJdsjFSC7P7RyLqKj2bMB1kAgbCUzaY2\\\\\\\/pm8yTIZMj\\\\\\\/xFDkHvGBlQtXtA11+QStlxLCiSm7LgYjPKF1Aad6Xkocy7tYEJ+I8lSk2EkXyuDgEzhTUDXtvGjxi9FHa2x492YHxtcX0y\\\\\\\/YMcEEbX05yKK2sIih85jjO9TIv5H40Jq+ERkNZo8gLw3Qm5glhhzvBJ4DQ9Y\\\\\\\/qINsuDN7lv+53VJXhS\\\\\\\/UYP3d+H010Mk79Pb6n2ujMrwuEZDWeMO3dqkLnRZn\\\\\\\/tNzhb8vRHSK8srrosChThYLFj0rJjvZCTe\\\\\\\/TZhc3Nnj1u4SKWhxi9JN5ATIHIskQQwPqi\\\\\\\/WX93O+IzEFoosM610QAjOqYaC364DWl0mRCH14+KTDKh\\\\\\\/0VE4aNS9UBZL5g6tDz+eggZjL6AKUu5CyaL134ZNvgcpKqcHL\\\\\\\/p75Sl9N+uwaMixvDUq6SM6Rsp6vjsqV3z6533VpXB1\\\\\\\/fj0PgUFp73TrC2bhwfjm2k9\\\\\\\/7m6ztCE8MZi9jyeKT8BEzTqqEH6MZ3+NhxqmWpLGphWFnO\\\\\\\/RoPwAEd4BPetKNPNBHbsU2sMthrI1Rpjc4FvgYXxja1zXdhOw6yWAh5V2OazU0qO\\\\\\\/+ZKUeNftxHwl98b\\\\\\\/krCJD\\\\\\\/3YUaQqYbX8r\\\\\\\/gQHuQSbqjtgkwoNJE2tQV+uWxeFlOmnqWOmh5XQy6Oz53zKLdjLHPGLk5qA2lAFuOWcxKCnmCGp8RqkScok32+E1tkBexz1KtJ7GYZr3Y0biVulmKS9jNW4wcvcf6uVIrfSqfEQFHWFmPrzpmEaREcu3c3gjDx34Pfee8G5EXYOEvP\\\\\\\/pVlEXS5UtNdqpVAWUXDy5jH7jNcqtLagABwQiMR0fkBrXMygAJOchfcARvY7DHS8o7i7p+zTt9\\\\\\\/BOk6ZMsxA9JH8HmwpTTe2sPlwzNCAZQ5z2ZeSCz7ckX8skzN5zD54PTlzdNqk+jx6qJQHG3D1XrtSINv0gkoVCEK5onZF8o3cmc3iVr\\\\\\\/0t89K2KGF4QZXWwDVbuvcB3iUkPStpuXfFeUsisE3JGAFQSaM9dCTKntEGV6w50h\\\\\\\/uKUvCgYA8seKTB9WwZLczVMix2N4Q1dDoNqY5VpLzql7KpfQ8jYwF9TASTc5OHZasI3dJUdIc+cTr44ZFTCC83yFFWwp3fdyjAlLRcywvwnasL05kv8x93OQCd9cmirl8848VeGFeM3\\\\\\\/Ty6iDPOHOuIxXj6ASq4CFJsnfouCSNeOesBoVlLPyHm9MrUVxrGFt5YeWx2yune7xB+fgFh6Bg0fBnCvBdQ5nzmeK5KybWO9e0rqIq2yA6R\\\\\\\/zZTFiOudD4JxaEZ6QAg0twuc+mZlwNFjoFlp2VdAg6vKEdWJxxu9be5U+BSsl\\\\\\\/WX5RxWwcAL7Po5ODt6wy0UJ7AK2Mr3gWSu3XhdVRTd8q6oEYVW2DJy+HzzfI5q5J+K3GaMByWUoO\\\\\\\/zNgVcqsAlOzd038aNH7Eq+pUJlUp\\\\\\\/vvCS6mFu\\\\\\\/OfldyNEbyWivmpx2gNdcltKmqMAkwozJJJ0US2aSmhCNQWkwu3wTozSQI1lNrpdlDLd0n+uFZdlX3YPGRijx91MAlLbO+7vj4tmp1bYekyx0B2eqkpz8jTVl5SKlcnq68JAL21RRpu80U2mf0R+iPgTAuS9prbusNzZnzJ2IhqTcYFCFaC1pYkaGGt2Ab4c0sxK32C7LqaLyp786xX14pKTS6PCe3S6GJ2aGFGV+\\\\\\\/BL7ILFTQufBqrddFxIK+dg6BN4JVBgTj8SlkCZg3RqHW7H3bQdDvKQagHAXFXavtcDG0Y2SjsfEW+hTO3lpP3Kho8S+TpUO9xsWd5J5De\\\\\\\/c\\\\\\\/T9k9rp4FFTwVl8\\\\\\\/5qDD4xPsTJ+bt994ALgj8ThWlKji7cOFAJ6Uebb6A5\\\\\\\/jkXca0vAIydJHC9+\\\\\\\/2NcQx853Eb8caEMVYYmabjQoOkZrYYvXrrBzEaMS0I9rk+U7GGhX8ZfdavZp8q03SAQpdZ0po40xfHya01IPDVA086KI+VC8jxv95rFqi5y8\\\\\\\/Bb5Dca7bE5DHD\\\\\\\/RzQFFylqrS8F4ncR7SjjUmtImeHJ+iRlZUd16DKumISK36jh6h2w\\\\\\\/HPAGxvRZn+5QM6P7UQa+4JbQnWM79EornSa9WG5ysOHESrGMqJIpkHLVELEYbsInce7XkJQIW8j4zBpNn1GkiVGN82srQkidH27hAl3gswepurxlNZ\\\\\\\/s3+7c9H2NUV1I0zkmk0iOf81\\\\\\\/Xwzoemt93Qzc189OLg+oEjz4qJptyD7O58Q05rQNr5mINc0Rbm43LqurTjdY9yBBjqQkh7FskFoaI9\\\\\\\/sO5ReyPX8M6a0j6DH9tndVB3zdCiexJXJNHcs3080eYSHXkgqFD4fyVJm8HhCLM9W+fPj0ZU0Y2MrWD8WIo4cEs744WVPYy01os3AuFUJHsgKxsy6uKJiEdmbSMBaM8QNYW2E60YQMkBMVHIqwOjkcdGZ3k1qP8g1Jn\\\\\\\/ebGsAUZi0t1HDLB8Zv36yTp3xE+aPawXNgb7dUbakCt594VSXU6e1l4MgO0CcguAmOwQ8mie4u6iwidUy4fn0r4s2x9UauMh6uutmV6eaGgRUoDgiXnBzu1czrwJGUAGm5alncayqZlG6p+NrCbvky7\\\\\\\/Ha\\\\\\\/DAXs3pUtPEChOnbb+Bd0v5hCA2GgXqarJocDNImBDza1IhcLEhCqsbONf9ErCkiZyTY1+hroTdjq0ebUEHWwG6D9Se+\\\\\\\/qTGo6vCk3BFDpPKph0+TGQOV9zS+k5KHdXvdnvOr8yTR1GvCUi\\\\\\\/MlJkBm6yrCTXxyKNQPqEcYw19I\\\\\\\/WeZB6SC2tGIVMnIAXdpvD1BQGujyP+XYs0Q3B0VHfA+iuCHo2RRnogWXf3316rDDzpEZudCmwYH66p0PQXkG3Z\\\\\\\/6I1KtxigK\\\\\\\/5ytXk8muh+X4EPC6CBhduNvbymJxxwsjp80FBa53SGNoCl1mZiEEVSpIQXmtwa\\\\\\\/iLqwcAhgFcM7ly4V\\\\\\\/8dFWgfSjTwl4EVV0tYRa9V1IzCv50NwK9qGuqIrHfruC+85Glg9EJ\\\\\\\/+17L3C9NpTjrcAmtaviecgj\\\\\\\/jPgMwJBlMgtD7IAwdDpIO7\\\\\\\/\\\\\\\/EbSiLDZoRzql\\\\\\\/yHnsgEioMDbQ6DyxEZnNI8LjfxhF9vMTKPtflZVSeKGryTkTl3VdS3jhklgrYIQAaAt7MDQNox603Rx8rad85HuBeQ0Yoy9550LlZXi8GGl9zZIVQ5d0u0sQd0J9hgglDhe0FcSIO7XRogWOBq49zd8JCWivJxDiDKIl0tzyVycNpKdKg6kjngmQvkCk36WlOkZXr6Gm5Eggc8OtMt79fRnNfXu48TOceqQiYDHlHEVhLVzA+tEx8CSebranxFx+FIuMEgSt34BG9uBRQS+P5b1Em5N7T\\\\\\\/dWsRRUmlfeenDcedEC\\\\\\\/qSs6r4VyZy6wUBPsWWu4xCrLLRhNRnnOZ6sG8okGrijV342noD3KqDK\\\\\\\/85m1jp\\\\\\\/UfmjogKZUZBPTsOVV7jHot1U2QcSiBDEoDoceXfkDybPu3HtEjnaHaPRfRx\\\\\\\/d5+ftRcIi5FHzLvLTfIwThJTxnPKV7gJ9e43le7Zpd7f6Qr0997zBvvnWrMoPMtkXVmHbK6cnkXPhheSl0Zecdst7e538kZGIK+NK2YgPTw6l8IifySaB36\\\\\\\/0QJiUJjUMocqVeKBGMyx6JPCl1wgnLKKVK0ZN2N7wkhsLdODVongu9dkT+jChsqf6Xlfvmj1HCGAHoLtmVtHy8zDilTJOwcird8jVCZHxr0qTaQErG7U2vlw0238OoaPIIdSsz9ei2\\\\\\\/qbJzrKakwlUW9VTbBkDno9Uj\\\\\\\/lDq6obyXzUxezef02zq3ka0OZWjoaUygSbmLOaDfJchz42VTVaBUIk1SE9Q6OGW6WTkWZFock72w7e\\\\\\\/o3T2EOpfph7YkVCNV3BEgbwJkdW+FCcmXpOTi7BEA0\\\\\\\/TPKlbZZVi7EvjBJn1eN9yGhVEqfsr1GJuWowB5jzqvjY1kpRF016mhbyHtXz2nFOTJiJfywQsz8kGr\\\\\\\/tZVLZCWOtGLaYZYjOgkugVHCB\\\\\\\/nF7LfvranX4rabShphtz2Wj7LoWcYn2nHLSD1V+np9Rp67VJjoNsVr56IqyIespWGQmk+ZF6Io9vvuI9y76JMWsB9Ai8NxmOSrYUYvJiYYuRx43IYUcxTUGTIpCDSvluDgQ1RdCLs7ayjPV18TNI4FFbA8LCCSZevMcdn4rY3qwq3yeTvRUYh8kVFkEO4EMmyNHNRiyBh49sYBVTfyMjqiNN6xYfbP5uXNNDjX682S8EdmjG8PYtDUL9if5GxqeS5IG8bGXurxFQWnHAR5TUzIBJ2CXoDCw9bSoTEjZD1uLQzFEymu1RMxhpCcRaBHR7Wbm2tzLNniAsKYVLNaEGfZttGlFRF9y5S9xMbatBmkd3W6MAVYpb\\\\\\\/a\\\\\\\/Ey0Qfnl91lfg47dJRL4Jd5BzWPZ\\\\\\\/4uhaGnnduON1f8bgHQQEOa+9eHwiTh2H8DbCOCbF7gyahyj40Zt91MUh9h1KStJgbRxPXpC4QnW0M7eAawE9\\\\\\\/oYxx8bi1aHckvzJwgHKlA5wg35lm5Clb7pwqPgz1\\\\\\\/fUvwIqyM0qjf5c53ug42w1J6cbsCnkomr\\\\\\\/vvoprIdTYdZxtPsQ8z6nqbBG4RmMgQedvBbDVRCX9ftGUDTQ8J6o\\\\\\\/QcNljkDdADVA4PrDEabfy1ofT6Api1g\\\\\\\/Jn3qXoLueOVlTrsFCdQ4hti9IkrOaaycvo11khzRM2\\\\\\\/VcCHH2U0+yzftqTjnooXhxOU+DEdwf2D8lmLUOZ3RAjxlVeszcsf5V\\\\\\\/\\\\\\\/9J26LvwMo8ZUxlZre6KhtglFfrKGsFzrpbC8uXc1XQ8XTbVWFhJyzjuSWJGsX9GijlL7hxxRQJibAwHV9KqPP2XCcfUgjzZCuPm5ngw38fBCd4iDkz6EaZIkLVTlRupfuMnCCE8uaAvgBDOYr2SzoCYWvzZwigImRgvCuDUWAs4jKY5ZDmWZXFU6jL\\\\\\\/stc\\\\\\\/iMfAjqwuHRX023gHB571mlYKa0GE\\\\\\\/L4AFEdoF3IysrGYv87ZNUfzRHjpClQ5hL7jMe5MFstF56ElCAj7EBe0U0HK38HcqD8tjkvDi39vFrblg8cLW7lpwE8ZL3+z5AoU8xcbnCWViS7xXQT4PCGxuUi61yAbbl3x9HaGSJj4r1zoZOBf2gHs2SStPrk7CZR4c\\\\\\\/IIxyvK3P\\\\\\\/dhyeOIAl9u3dC9Lpc3a2IpukeV0Mumw5Z3NRkdbUQAoZ4ghypV0kZIbTyLOBggk4HtAnZOeYbvZGD6Pf9h\\\\\\\/i6JdOHEHLJ86tF8DRsR4Mt7MpOBTiBTgtk7YSs4qcu24VAGFjiytcSmIxpzRXPsReAT7yktAh\\\\\\\/O4oaRzCa1AYKhVoNzfk2juYJV7B+jgnETQ4+LONwXrJaIxP6ulbJppCfYtzLUA76EM+ZmbBcISb0ArN5S9hadT69wt4g+jo\\\\\\\/JrE287o9m3JIyD6NTKrlV49Qvv598ZavYNxcgiiBAKKLLo+RjwnUSqgRCWDEl7YdtXDyuL8KJYyOYBwyYl1kLtiUQc3W4Q91r00cAmkQlf31vbxbyFPqgmluKsczE\\\\\\\/J9cm4HJ3TDoreunsMme+7\\\\\\\/m1iYqRiLl1JoFp+vfdpUrIqwQHw\\\\\\\/qH\\\\\\\/aZKtU3OOv7oLSupBqSiuaO14iwRnGemwB08Z6Qz2Z1+vHrTA4jJJFNWf22oAmgIWWCeK5iDjV0oCieqzmahYatPWmnOCj+VQDSB+GiRRCl1JODHLcanmPC1ycsHAsw9QhYEsZqOdJbBp15SVR+slvkOftiXxZxW8x+v+o\\\\\\\/7Zyup4zMfkxl1Qb+jvqekzk5utqtANtGEAxqbiAKEca\\\\\\\/imI5IuhXvV0Z8Hi+0gjqnLmCIEeueeywAjiMFhT5E9j8XI9NRBZJCkgolNomi7WSLdY+rErOj0+K4L4AMAAWvUc+z26Q9y+DuLc24mSITNa0fHEK6znIiH3n1auzviTU6CZNvb2sXz271foChuLlvfJMvc5WrsU4j4ah0g1xlr6MhbYt5M0XvXRRZi2cgQnf39R7+r1OqVB1TdMGu7ntQpR2l3gb9DJlvtaXgpxOVLWrTDJtjreQXLcq5IxD0QBJ4wQ6GnM1rBvB76PeEcRMpoce6q\\\\\\\/oG88hJWOXBtIraJyFXvtvNAhIaCYL+hUe6DKYSNXxFxWskbdoMLRASzt5XGLwZWONzPgl2ZlR2DGY0\\\\\\\/mJIO4tkk7w9UKTnC3ro2x8G2HiAXrqNtadAoHET5ga5x\\\\\\\/CaDRFnmrAFc7ebGhB6YwDqKMepvw+mm6jm5Xtz0sauunown+ur5u3vxXO1nswlG+sSxwhMMJSRQ7i95B2ODOd2y7KBHUuNG9HX0dRbY92qOWdSFbEZm6ivX8cx\\\\\\\/R7jetgiLe+9x3lHLlz848wUAJEs7OPimxOulKgKu39V0WO1l\\\\\\\/zXJ4KHh6vgN8B4Eu\\\\\\\/PcWINh3pvDo36YCIhLKIM6ePlfYypJSzLfRjq3kUJKztgK6HISGg7bw4jdsQPwC1kdNE78qOtxEaXh7I1y+h6fHsqqGOf5JWy70XmvYnpnuTLvrPgZrbtxHISWUgO458lPP2EbMxNoS0rssv7ymW87h0pCtIeJgs3ODVF9NNskRZa5R5nW+9Kafafw9HDvQ0PjL7rFM+\\\\\\\/JXlvnq3cg5VlkwIiBgmgHFTfQVm9juMfF0n2h5F\\\\\\\/gH3y08s6nSW61ULNCgfir\\\\\\\/Qu2+PReyaAtXY+aR0OgezgTvXO1Ykz06CoA7aRgCyz+Sjcja1oWuXa\\\\\\\/qhTx+ekI3UjmSSaWIZtof3ZmCySKg1eMi1xA8gt+dOLOEYLm0FpJSImgVFPmrq4gcXR7utUGQM1c6ygSCJG5\\\\\\\/a5AlwD5t4c+iyG4Z7gRdwRL0jOiAlRtckeM7lDM45iSrkovtv\\\\\\\/jmcHZtQjS24gU4FW2xnHWTEQN5PtKiiQLuKwGIbu9JpJoW\\\\\\\/dyQQr1pp71utVqEnkU9h9ZKx6LlNKBuJ5TAVm2EMdLInarCUJh5H+AQYnsFD8y9iYusyYOKm77vJLpA9KvOla2tHvXzc7aiEKedEhbK+4j7pVOfL+dEH3uPT6xzVd4sGywxFk4Emfcpx0+jZkxEev5+WsfNONqJGhR1fM80rsgQZyLFL+H1ZZYiM131JdJQ1sfmUfohQ7+sAfapFe8oeWxQ3P2\\\\\\\/Y1\\\\\\\/Vx2WXMDFW5s7OSuuVp67r1l4hEKs58TzEX6YpqhPwMqlVTGE\\\\\\\/a\\\\\\\/sLEhDF1nVk\\\\\\\/EDXygL49Y3L+egj3C80xsdyqmxDWqqIrMsma5IRYKC0jy8r1HHLkdOQCvMFkxoUrPK7xyBsIm56lXRWHu0aXzdR+\\\\\\\/BvfuRReR20TTcE2MUSsqQdcCofkr3md3X2S2ZTD60bFscbbiFKnkWNMq2OtQbFMM+HLlgnmQGKjgZ1MQcU8OzxxL+gAUZsJ9MVHSI85zEpxw1HyHSRSF9f9cXiDbUwDTAgLZypELJOYVCcmAyXvI+hpM1OJJguHdgrqR\\\\\\\/vCJz61XEowoaIpQDyMOdCDKT\\\\\\\/eOlSI0we\\\\\\\/6v7GL+lwMjIrMi4nWkHNgdd+pDKznK5oKMQLWh5Q92W1ckHTGDdGpve\\\\\\\/MI6fJZ6Kp3oChNR16azNbQ6yeOdQTuVnWSS5axsTA6LsF8OYwM8D2Le1NUseAd0wLPUvGcglTTvRapFjRvWLC1fxdxJnJsnncU8V0lipZn0kWAM7QLco1yk6FPvpIT44aBQ89qyYxLsKhocR4TeLBdZSh66uUdSTXUdBNFiOBcNz7EgdhjogjNv4UfEiWopBMY3gtmo2N26vXfWAn+YltpMMfs0njnDqLTSDs13ruar8eTRYaP5Ay5NJu360aavU9QUYStMu0ZNWWu5qS8oV3tCoIOS5ZQ2v6OGh2P82eL4NlryGr6\\\\\\\/l1ml0Jt\\\\\\\/ZBXN30vLM8dZv7AnX0rG\\\\\\\/qW0qCGHmFOCT8GQ+he24Hyi3JbLwD90zdNgRozym1TmuJVd2raYU6qxUd1pBDdS0vrsb\\\\\\\/GIph48WM9C+pZyzYUlCMwEKB5DjRkJ3Je0NDbaKC2HOSH5bONGamdSdpD0hmkorVzifJ9zAQeygb7rO41QBpIz7ji3cp1\\\\\\\/j5wxrLJWm0hJX3XXg21FTgnJ01piPelA\\\\\\\/hyBCcWF9iuXphSLljkBc531WFWaS2k89g5mTWalJdwTpCapobBUXb\\\\\\\/c0pqVGRvyZl1pR\\\\\\\/iocipDKN8GPMrfusu0MB2LtZIOTvq+2EOhYanY4OUL6Hzm1yENZvJjvtj2NiceHiaMeWFH\\\\\\\/D4ph76Ia42cBq394pKdFHfoYPd3Vae+HSHQfevSTcg9dIqg6I6hZlMYErqK6Ykvlrk9XyDnvnSrp+xaeJcJZeJLh\\\\\\\/dI7QW5iXvP5tFRERLpCPLzxTd05RhEv69LhKFw\\\\\\\/ZOI\\\\\\\/rsi12d67mFUeiB+CrntoynZgrqrR94F8+D1pUXKYSQcyD7dQ0IO2zU0YRdR66P+rYzkruuvvFlsLwHuMdARSUVdgQ3mqsmvHWXtnDiEOfX7GPSN\\\\\\\/v7iCmB24bAL6tL\\\\\\\/lRmQ12nE\\\\\\\/HcI3v0wORPiXIcNtADk7jD8GKMEHqJXY6ytcxCyCxoq6LSCG1NmEsawGsfgWuSeZ0J1fBM55Q0NmO+\\\\\\\/LI\\\\\\\/TZaw6s5JEkFCtAnZfOlvNMZlfRVCsBPKGuDPtdY\\\\\\\/RikRS7P85KrXoET1NEKFinRlyJCEet2GK3wPjQxrUwqm4asl8G8iObsZlwRKsCtkPRZBbYhldpwXRH0g4CJieKPuBMSswAVKqG0BsF40v0uHWleZXzGKVnnD0r2gfsM0OnqEoxAv+8OZL8X4YREMc+OXm03+ZyvrWhOoNK58y8r6jG3jDqxyCAskAojX43jl8w+im8tIzKv+XcDitMLCf7I5y7M4lus8\\\\\\\/06eP6+cT1E+JgWSO4H7A6TC7XJym1ng5v3dp8t7Ltp6onw4xAbW\\\\\\\/\\\\\\\/qpC4JXXLsP5lLsSPQcH+++kOT38hb1QyQ7VbSOS0awxsajUlYVxG3pzxa0xAteWl6QEgynWoA\\\\\\\/aqTz0RIxFDZ5DJZpy1GHw9fr5F7ECGHl7oE1Eek7FNlhhQegf2NauaNMEHgk0umqafxVpcgm4cEPqm5kfuX+ytZgspK0qLkIoeLMf5UUWahP7oXDqTYWBS8ri3EXzskKhiMLtPdRFHu7wj\\\\\\\/x+z7H2XOv3N7sANP0n5LbUkB9pyTNElaSnG17MfikHmH+9F+cWlksRHtAvt\\\\\\\/TGCWdazrKIq2z0w4kdy7EuIXquooIKALeNulOFa9\\\\\\\/ddb+oPAiUIOa3t45QafU+Q5XIFmaaqfT45OAMb2QBaKysVHV6oA03qtU5BfsRstJbOc2Ffy6HKL7yWPGFWUcejqfEhloxczs3C+C8xejxVoxZSxogE0jbKANXXzA3GGUzIjyWKAVPdyTJS6TtJnRBzb4re9U+LkL2M\\\\\\\/pjjlyV9di+Zt4dDprvSa+kh4sA89O0IDegxUDc2OMGgFy4ejs0bKPVDJew+Tt3dxZsuE4bPPkpUvixSJMeYsPePiSQ7W9IeozFLMogVhKAUbvBUN1z6xKbsBfRBD\\\\\\\/cd4bGHoqVOohHP0MZtHk3Q40yWbKMytAG+cgEq5AvURxnsm4hYfSunJRbDSTWi3A8ly41aqgwFCRmH+B\\\\\\\/AvhNccMsKSN35ORBFjwbkWLPu2nV1xmw9n16ClGbWDaEe7S61zElkd9qc6oOyii\\\\\\\/sbOWoPOn+KC6KxAQ6jtMoAHZmGpoqz+kB3I++TflimOKteQufjQmtoZ5KlTOl\\\\\\\/iU1zp2J2DQ6usgaxMcpZfE8FhSQBCnE95NAD6Iuvuek1uPrvsjtaZuELgbNPcRrKUDT\\\\\\\/OD\\\\\\\/Dw1NrDv5meT46mA0IHc0t03oUWNRpkqcUkJC5F8Jgmx+\\\\\\\/a+ZOl+JtMiLVmbTKhmiMxGFX4aB+vgw9BigmhZgNsXoQ7dyToPdYPrkVmZgyCEUW+fjvL6+TKPFWvQ0XQVOh2vQg7poYnTgPCobzKjJtGkzAuDYmFIUpzc8CMujdoo9\\\\\\\/ZSisHto1OE8E8bndXKTwz6nmOSjguGrHmn1RYD11dW5s9jYsBwYItu5w3pIjsEgpY2HlSOQmYTkKwUalvD3Z3RwkmbU17QFLkQhs4T6VtbXdOdTgDv78GQCDvW7+Mr5X18\\\\\\\/gmswV4K2zkdc88TC9eCTBBgKCayEbCPQbIHr7JtD3lRD4KfarHGRapexbCPT5ZZHbKWGEhvb\\\\\\\/tzFplgqMOoNbxcAG1q+iBJ5UY2AbzSN2jaK\\\\\\\/ebbIPf2WqgHafTvCvVv3lfXgYZMol8f7it50mx1gpSfyDclgFPVF7JX0+ULWQ+7OUw0fTPFaY+BPzEVPkMFrYUiGE1JEkIZIJewajddEJne4C2IYMXc8UFVeUhJ47oFPaPG8piJwr5k1PEQqyx1Ayca5PrPDPG2gww51SB6S8X135GBxo9rzLJZMXp2f5WfPQDGBzZKXmKB9kqYwh6hR\\\\\\\/vlf59NSu+jmwVWPwzvoNE8ESr6lYx\\\\\\\/NK7VzJJb\\\\\\\/apjAh2FlPZWGNGGY1DxcjWRzLjn98Vj1W8O\\\\\\\/WYoQULczsSWNQ1+t\\\\\\\/jXXorv11m6Zu74V\\\\\\\/8FriFgfuiOHZz\\\\\\\/vxguy4F0L3AcmyVYkBGCQC\\\\\\\/b7LE63NeGwOTPpKRZ4EQPfZKMyoDArIbX+5nqIA3vYB6aQiaHgjVIS6\\\\\\\/i\\\\\\\/c6O6j3H7PaGBGpOV4TJ7R+Qc6yBC0Nj54V16GBetTdUFu2A7GdCR+8UIL0lxbpAuk+UIheGTJFVDWVML7DQYicXZQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00225bEc57cPIvN6hIG20Ox\\\\\\\/2A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ac2f0257bf70a799341f54f2d16848aa46510a600535ef3362663e3b7dc5f118\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226ZCAGnqexTrYJMpwFLu4rOau2t8OTUdVHyBfHOmJzNUy4Hm3SO2YJY2j+iC5aJ+GTNeNY3HOBT9akBcsKyri5wEcRixuVMc00p6XjnO0Bc+8UKhqDmpyT\\\\\\\/YloQk3wkZ93nZvu+HOFb+KpWDRnBPdvGBEeOp52+r7nbHeamObmfpxkc+7UZWYWN+z8SC8pZwYu8KpbIf0T4H0e+5LrH2448R0fuOQMUG85jtz9OAqXIqGa4sMDnvb\\\\\\\/TktM09TZPGK9Q6SOQpaDIpBQ3c+bi\\\\\\\/dMDxADJvEw2y0ndMIYtbYHpyjLsSgeP4V5yl+wH1rPaVeV9El5b9JwOf5vN9XJ9nMO5DBIpJQCaN5b3FGWvdXmRxf3Mk5vYomGFxR1NMrYnq3OI9J3JQs3wuFffRyNpxQaH9Tk9Sv09pQq1sYizouJR6a1o1mXgOcS2E2CefqSFj29v2gaBEeYZWV4eVf0UXbfx867cUyBrXhiEUIrflF1v1s62Gf\\\\\\\/jVTPGrll11TbdI1C5k00QoAdgBVOSAV0PVP8MU64DsCVdKPoG4qX5zABVdaDm0JNFe\\\\\\\/6n+LOF9ARtI5DaO4NhL9+3JJo9sVYGP5u+cRi85Ks0uhaS6\\\\\\\/qk4zONHAMSulZ0tWHBp5ejalj2yTPTxyna2zR5cvB95m6f\\\\\\\/u3WzkeI1nvg+JQ8Ms5iE2Qk\\\\\\\/D8gCPfen82ZqQDjM7vbh4SOVmj\\\\\\\/qfWqwJpNM+k\\\\\\\/LbjJLZvSdomST3gwo8GblUTk6EIPwme2Upct3ZgD3ewds4P3R27ipdyU1uSAMPXa8XxFCBsq388YhjNu\\\\\\\/rVCsjgVyy7wDuz1PifclMytf+dNY+Ouzjbu7N+2PkbNKqlb6WtluY0++N7MVek3okIp5orgApHTI5DTfIP8YZsjoP8r8oG1iEg7Y1tZeiqmNwNJ9Qlpv74pqleFsydGn0dazC\\\\\\\/vOO2vC9xtLT5ll40R6iNRgLE6QJcdnVASBMxcZ64z0HmPo6Si3DEPBgvWmAtc8FHfTGVQehyXJkxmqlcxHXGj0NINE1KLnqKvCVJbsVCjsSA1R\\\\\\\/tcVBNFPCYWejfRINUm7ifdTK6ISv4ND90AjSMXVHreHIG3\\\\\\\/MvBVoKbVIV7bljISxIz6TRyn52UPRed16eBLfVfrPT9tyXdFKvfDe9p9yOKWPVzHFU7iYSX+WZ9ElYc5UGHdEcnpYquIgehBHa5TsMv0q4XMt65PoVk275fjNQHDCvCFwD8DuXvSI5OthB5Au\\\\\\\/t8kddi0GW1s2u4zGo3ghjuqMeitnxJ377iIjFYZkh3pw4NLiz7ys4MBMmpn5yR2r9n2Tqcfrva68vwV1JuB3ovMju0P7Ypk6KVZ+LzbiNLceN+O7GDRxjCc73Fv5ItllMa530Vtsew6k0wHgUh1P0AoWwMhHU+AHXFlRB5il7Om\\\\\\\/9AYTSUpKv2H4S1IlahnGcxzhqd03d+Ezr6FMcxpvovX7KHtFwC09W\\\\\\\/V8XkRBZUvdkYuZ9H4KbMpHsRrTv9AwlXcHcRbzuBaU+PxPfmkVUY12VDmI3vg8C2k4BunreHpQ1+nStD4bAM8W15Xc17O8Q7uJ53wmVaastD4ZGv+\\\\\\\/uEGumLZt\\\\\\\/ydblJmVFW2SPHkpcIIXmEVaf18JgJc+MqktoLJKgngTtEpmYukbKg8gM\\\\\\\/429lks7PDaVSvJaWfs2sJ15W\\\\\\\/oqArKjiS6kdM4WA2LJuSiwA8Yff8RnhPo0GKarW6DkXexRoqI1BNg5mB8sa+YV+B7Judc3ptC4NXi18k1rCvX\\\\\\\/eD2xMkydHQx3kIYvzrNQODmLBgI14ak1jYpICif+jvXMEXzuV06KezpNlqAqkBL\\\\\\\/M0xTzTJ+gwgA9wfCdtE1UGUipa9duFwszlaWcOEUKMzO7y98WvwczaiJj4EfPOJ\\\\\\\/8p9r0DRgTYdnJjrymEEoeU8YUaW8vm3Ve0OKcFXx0RTBLX5MrC0lFyQLRJt6BIp8pNnR9QBzUXTcUQaMeaLR0KGcO4JXJmxo1s12nsBx+6d9wJTky\\\\\\\/+mURmw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022z6x9gKbfyr+SrEQ6JWSKyw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ee1b3fe7bff51120734f2d377911867ea49d2b1de5e6dfed7bb42d234f504501\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459239 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-mouse-genetic-resources">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/2/175" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6Ww48e_SZJNMw1bA0onutkg6-_VsQqKCwFEWzu6Mn4k" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/2/175" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Z1Aj5lY5IrRzC1R4OiIUXVHZC1RgD7TLBWvdDzveQQc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/2/167" title="Genome-Wide Association Mapping of Quantitative Traits in Outbred Mice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/2/191" title="Accelerating the Inbreeding of Multi-Parental Recombinant Inbred Lines Generated By Sibling Matings" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459239" id="node459239" data-pisa="ggg;2/2/175" data-pisa-master="ggg;2/2/175" data-apath="/ggg/2/2/175.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">HTreeQA: Using Semi-Perfect Phylogeny Trees in Quantitative Trait Loci Study on Genotype Data</h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhaojun Zhang</span>, <span class="highwire-citation-author" data-delta="1">Xiang Zhang</span> and <span class="highwire-citation-author" data-delta="2">Wei Wang</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">175-189;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001768 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459239" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhaojun Zhang</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Computer Science, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhaojun%2BZhang" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhaojun%2BZhang" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Xiang Zhang</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Electrical Engineering and Computer Science, Case Western Reserve University, Cleveland, Ohio 44106</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiang%2BZhang" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiang%2BZhang" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Wei Wang</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Computer Science, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BWang" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWei%2BWang" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:weiwang@cs.unc.edu" class="" data-icon-position="" data-hide-link-title="0">weiwang@cs.unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/2/175" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459239" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459239/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/2/175.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459239" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459239/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/2/175.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459239" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459239/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/2/175.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459239" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459239/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/2/175.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459239" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459239/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">With the advances in high-throughput genotyping technology, the study of quantitative trait loci (QTL) has emerged as a promising tool to understand the genetic basis of complex traits. Methodology development for the study of QTL recently has attracted significant research attention. Local phylogeny-based methods have been demonstrated to be powerful tools for uncovering significant associations between phenotypes and single-nucleotide polymorphism markers. However, most existing methods are designed for homozygous genotypes, and a separate haplotype reconstruction step is often needed to resolve heterozygous genotypes. This approach has limited power to detect nonadditive genetic effects and imposes an extensive computational burden. In this article, we propose a new method, HTreeQA, that uses a tristate semi-perfect phylogeny tree to approximate the perfect phylogeny used in existing methods. The semi-perfect phylogeny trees are used as high-level markers for association study. HTreeQA uses the genotype data as direct input without phasing. HTreeQA can handle complex local population structures. It is suitable for QTL mapping on any mouse populations, including the incipient Collaborative Cross lines. Applied HTreeQA, significant QTLs are found for two phenotypes of the PreCC lines, white head spot and running distance at day 5/6. These findings are consistent with known genes and QTL discovered in independent studies. Simulation studies under three different genetic models show that HTreeQA can detect a wider range of genetic effects and is more efficient than existing phylogeny-based approaches. We also provide rigorous theoretical analysis to show that HTreeQA has a lower error rate than alternative methods.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/phylogeny" class="hw-term hw-article-keyword hw-article-keyword-phylogeny" rel="nofollow">phylogeny</a></li><li class="kwd"><a href="/keyword/quantitative-trait-loci-qtl" class="hw-term hw-article-keyword hw-article-keyword-quantitative-trait-loci-qtl" rel="nofollow">quantitative trait loci (QTL)</a></li><li class="kwd"><a href="/keyword/mouse-collaborative-cross" class="hw-term hw-article-keyword hw-article-keyword-mouse-collaborative-cross" rel="nofollow">Mouse Collaborative Cross</a></li><li class="kwd"><a href="/keyword/mouse-genetic-resource" class="hw-term hw-article-keyword hw-article-keyword-mouse-genetic-resource" rel="nofollow">Mouse Genetic Resource</a></li></ul><p id="p-4">The goal of quantitative trait locus (QTL) mapping is to find strong associations representing (genomically proximal) causal genetic effects between observed quantitative traits and genetic variations. There are several mouse resources such as the Collaborative Cross (CC) (The Complex Trait Consortium 2004; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Collaborative Cross Consortium 2012</a>), Heterogeneous Stock (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Valdar <em>et al.</em> 2006</a>), and Diversity Outbred (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Collaborative Cross Consortium 2012</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Svenson <em>et al.</em> 2012</a>) for large-scale association study of complex traits, among which the CC captures the most genetic and phenotypic diversity (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Roberts <em>et al.</em> 2007</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aylor <em>et al.</em> 2011</a>).</p><p id="p-5">Many previous QTL mapping methods consider each genetic marker independently (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akey <em>et al.</em> 2001</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Thomas 2004</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Pe'er <em>et al.</em> 2006</a>). Standard statistical tests (such as the <em>F</em>-test) are used to measure the significance of association between a phenotype and every single nucleotide polymorphism (SNP) in the genome. These single marker−based methods usually do not consider the effects of (both genotyped and ungenotyped) neighboring markers and hence may fail to discover QTL for complex traits. To address this limitation, cluster-based methods, such as HAM (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Mcclurg <em>et al.</em> 2006</a>), QHPM (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Onkamo <em>et al.</em> 2002</a>), and HapMiner (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Li and Jiang 2005</a>), have been developed. Typically the genome is partitioned into a series of intervals. For each interval, these methods first cluster samples based on the genotypes within it and then assess the statistical correlation between the clusters and the phenotype of interest. The result is sensitive to the granularity of the partition, the definition of genotype similarity, and the choice of clustering algorithms. More importantly, these methods tend to emphasize mutations as the major events that cause the differences in the DNA sequences of the samples. This may not fully represent the genetic background underlying the differences.</p><p id="p-6">Phylogeny trees have been widely used to model evolutionary history among different species, subspecies, or strains (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Yang <em>et al.</em> 2011</a>). Their application in association study requires inferring an accurate global phylogeny tree from the DNA sequences (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Larribe <em>et al.</em> 2002</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Morris <em>et al.</em> 2002</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Minichiello and Durbin 2006</a>). This may not be feasible for the high-density markers in current QTL analysis. Some recent methods, such as Genomic Control (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Devlin and Roeder 1999</a>), EIGENSTRAT (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Price <em>et al.</em> 2006</a>), and EMMA (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Kang <em>et al.</em> 2008</a>), build global models to account for genetic effects. EMMA computes a kinship matrix to correct the effect of the population structure. Genomic Control estimates an inflation factor of the test statistics to account for the inflation problem caused by unbalanced population structure. EIGENSTRAT performs an orthogonal transformation on the genotypes using principal component analysis and then conducts the association study in this transformed space. However, the genetic background of the samples may not always be adequately captured by a global model. This is particularly true for the incipient Collaborative Cross population (PreCC). There is no significant global population stratification among the PreCC lines because each of the eight founders contributes roughly one-eighth of their entire genome (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Aylor <em>et al.</em> 2011</a>). This unique design removes the need for global population structure correction in QTL mapping.</p><p id="p-7">However, <em>local</em> population structures may still exist. Because of the limited number of recombinations occurred since the founder generation, the genome of each CC line is a coarse mosaic of composed segments from the eight founders. In a genomic region, a CC line may be determined by the contribution from a single founder and none from the rest. Because the eight founders are from three subspecies, local population structure may exist in these CC lines. We have observed uneven genetic background at the chromosome level in the 184 genotyped PreCC lines, and such pattern becomes stronger when we examine at finer resolutions. (Please see <em>Results and Discussion</em> for further discussion of the local population structure in the PreCC lines.)</p><p id="p-8">Local phylogeny becomes a natural choice for capturing this type of effect. Several recent methods [<em>e.g.,</em> TreeLD (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Zöllner and Pritchard 2005</a>), TreeDT (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Sevon <em>et al.</em> 2006</a>), BLOSSOC (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Mailund <em>et al.</em> 2006</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Besenbacher <em>et al.</em> 2009</a>), and TreeQA (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Pan <em>et al.</em> 2008</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">2009</a>)] have adopted local perfect phylogeny trees to model the genetic distance between samples. These methods examine possible groupings induced by each local phylogeny and report the ones showing strong statistical associations with the phenotype. Because these methods require a large number of statistical tests and their results are often corrected by large permutation tests, they are prone to multiple testing errors and incur significant computational burden. TreeLD and TreeDT can handle only a very small number of SNP markers and thus they are not suitable for large-scale QTL mapping. BLOSSOC is more efficient and can process the entire genome but still needs days to perform a large number of permutation tests. The recently proposed TreeQA algorithm uses several effective pruning techniques to reduce computational burden and is able to finish large permutation tests in a few hours.</p><p id="p-9">A common limitation shared by all of these local phylogeny-based methods is that the perfect phylogeny trees can be only constructed from haplotypes. These methods either assume that samples are purebred (<em>i.e.,</em> no heterozygosity), which is not true for many large mammalian resources, including the PreCC lines, or that a preprocessing step <em>phases</em> each genotype into a pair of haplotypes. However, haplotype reconstruction itself is a nontrivial process that is both time-consuming (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Scheet and Stephens 2006</a>) and error-prone (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Ding <em>et al.</em> 2008</a>). Even if haplotypes are phased accurately, the two haplotypes of the same sample may be located at different branches of a phylogeny tree and will be treated as if they were independent samples in subsequent statistical tests. This may create a bias favoring additive effects and lead to spurious results. For example, consider a recessive phenotype, we use <em>A</em>/<em>a</em> to represent the majority and minority alleles at the causative locus. The local phylogeny tree built from the surrounding region has an edge corresponding to the causative SNP that separates the samples into two groups carrying <em>A</em> and <em>a</em> alleles, respectively. Each heterozygous <em>A</em>/<em>a</em> sample is phased into two haplotypes, each belonging to a different group. The group having allele <em>a</em> would have mixed phenotypes. This may weaken the power of any statistical tests and fail to detect the causative edge (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Wang and Sheffield 2005</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Lettre <em>et al.</em> 2007</a>). The scenario may become even worse for phenotypes having overdominant effects on heterozygous samples.</p><p id="p-10">Therefore, a natural question to ask is whether we can design a phylogeny-based QTL mapping that can be applied to unphased genotypes directly. In this article, we introduce the model of tristate semi-perfect phylogeny tree directly built from unphased genotype data and explore its utility in QTL study. Our method, HTreeQA, has the advantages of phylogeny-based methods but does not require a separate phasing step. We demonstrate via simulation studies that HTreeQA can detect a wider range of genetic effects than other alternative methods.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials</h2><div id="sec-2" class="subsection"><h3>Collaborative Cross</h3><p id="p-11">We use the genotypes of 184 partially inbred mice from the CC lines (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Aylor <em>et al.</em> 2011</a>). On average, these mice have undergone 6.7 generations of inbreeding and have 16% heterozygosity. The genotypes at approximately 180K SNPs are collected using the mouse diversity array (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Yang <em>et al.</em> 2009</a>). The data can be accessed through the CC status website (<a href="http://csbio.unc.edu/CCstatus/index.py">http://csbio.unc.edu/CCstatus/index.py</a>). We study two phenotypes. One is the white head spot, which was originally observed on one of the CC founders, WSB/EiJ. Because there are no white head-spotted mice found in F1 crosses of the CC founders, the phenotype is believed to be a recessive trait. Among the 184 mice, there are four with white head spot. Another phenotype we study is the average daily running distance for mice of 5 to 6 days old. This is a typical measurement for mouse activity. The phentotypes are supplied as supporting information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001768/-/DC1/FileS1.zip">File S1</a>.</p></div><div id="sec-3" class="subsection"><h3>Synthetic data sets</h3><p id="p-12">The phenotype was simulated using three different models of genetic effects: additive, recessive, and overdominant (a special case of epistasis effect) models. We include the overdominant model because we observe that heterozygous individuals sometimes exhibit extreme phenotypes. This phenomenon cannot be captured by an additive or recessive model.</p><p id="p-13">To simulate phenotypes, we adopt the method used in <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Long and Langley 1999</a>. To simulate an additive phenotype for a given SNP, we use the following formula:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-1.gif"/></noscript></span></span>where π is the percentage of the variation attributable to the quantitative trait nucleotide, <em>N</em>(0, 1) is the standard normal distribution, and <em>p</em> is the minor allele frequency. In the additive model, <em>Q<sub>i</sub></em> takes values −1, 0, and 1 for homozygous wild-type, heterozygous type, or homozygous type, respectively. For recessive and overdominant models, we use<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-2.gif"/></noscript></span></span>where <em>p</em>′ is the fraction of individuals that are homozygous mutants. In a recessive model, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-3.gif"/></noscript></span></span> is 1 for homozygous mutant and 0 otherwise. In an overdominant model, <em>Q<sub>i</sub></em> takes 1 for heterozygous mutant and 0 otherwise. All causative SNPs are removed from the genotypes before analysis. We represent results of a wide range of realistic contributions of genetic variations by testing five genetic variation settings of π: 0.05, 0.1, 0.15, 0.2, and 0.25.</p><p id="p-14">We simulated genotypes of 170 independent individuals. Under each genetic effect model, we generated 100 independent test cases under each setting. In each case, there are 10,000 SNPs and one causative SNP is randomly picked among the SNPs with minor allele frequency greater than 0.15.</p></div></div><div class="section methods" id="sec-4"><h2 class="">Methods</h2><div id="sec-5" class="subsection"><h3>Notations</h3><p id="p-15">We follow the convention of using primed notation for unphased genotype data. Suppose that there are <em>m</em> individuals and <em>n</em> SNPs. We use <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-4.gif"/></noscript></span></span> to represent the unphased SNPs and {<em>S</em><sub>1</sub>, <em>S</em><sub>2</sub>, …, <em>S<sub>n</sub></em>} to represent the phased SNPs. The unphased genotypes can be represented as an <em>m</em> × <em>n</em> matrix <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-5.gif"/></noscript></span></span>, where the <em>k</em>-th row corresponds to the genotype of the <em>k</em>-th individual and the <em>l</em>-th column corresponds to the <em>l</em>-th SNP marker <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-6.gif"/></noscript></span></span>. Similarly, the 2<em>m</em> haplotypes can be represented as a 2<em>m</em> × <em>n</em> matrix <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-7.gif"/></noscript></span></span>, where the 2<em>k</em>-th and (2<em>k</em> + 1)-th rows correspond to the haplotypes of the <em>k</em>-th individual. In the haplotype matrix <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-8.gif"/></noscript></span></span>, we use 0 and 1 to represent the major allele and the minor allele of a SNP respectively. In the genotype matrix <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-9.gif"/></noscript></span></span>, we use 0, 1, and H to represent the homozygous major allele, the homozygous minor allele, and the heterozygous allele of a SNP, respectively. <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1A</a> shows an unphased genotype matrix, and <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1B</a> shows a phased haplotype matrix.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461237/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461237/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 </span> <p id="p-16" class="first-child">An example of unphased data (A), its phased data (B), and its transformed result (C)</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Perfect phylogeny tree</h3><p id="p-18">An <em>interval</em> along the genome consists of a set of consecutive SNPs. It corresponds to a submatrix C<sub><em>u</em>,<em>v</em></sub><span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-30.gif"/></noscript></span></span> of <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-31.gif"/></noscript></span></span> that contains all columns between the <em>u</em>-th column and the <em>v</em>-th column. A <em>perfect phylogeny tree</em> is the tree representation of the evolution genealogy for an interval in the genome (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gusfield 1991</a>).</p><div id="sec-7" class="subsection"><h4>Definition 1:</h4><p id="p-19"><em>Given an interval Cu,v<span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-32.gif"/></noscript></span></span> of 2m haplotypes and n SNPs, a perfect phylogeny tree is a tree, in which the haplotype sequences are the leaves and SNPs are the edges. Given an allele of any SNP, the subgraph induced by all the nodes that carry the same allele is still a connected subtree.</em></p><p id="p-20">The perfect phylogeny can be treated as an evolutionary history for the interval. Each edge represents the mutation event that derives two alleles of the corresponding SNP. All the haplotypes can be explained by the the evolutionary history without any recombination event. For example, <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a> shows the perfect phylogeny tree built from the haplotypes in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1B</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/175/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) is the perfect phylogeny tree generated on the phased haplotypes in Table 1B. Each node is labeled by its haplotype ID, followed by the corresponding phenotype value. (B) is a tristate semi-perfect phylogeny tree generated on the unphased genotypes in Table 1A. Each node is labeled by its sample ID followed by the corresponding phenotype value. (C) is the corresponding perfect phylogeny tree by deleting  and  in Table 1A, and (D) is the corresponding perfect phylogeny tree by deleting samples C and D in Table 1A." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1944878361" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;(A) is the perfect phylogeny tree generated on the phased haplotypes in Table 1B. Each node is labeled by its haplotype ID, followed by the corresponding phenotype value. (B) is a tristate semi-perfect phylogeny tree generated on the unphased genotypes in Table 1A. Each node is labeled by its sample ID followed by the corresponding phenotype value. (C) is the corresponding perfect phylogeny tree by deleting &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-33&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-34&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; in Table 1A, and (D) is the corresponding perfect phylogeny tree by deleting samples &lt;em&gt;C&lt;/em&gt; and &lt;em&gt;D&lt;/em&gt; in Table 1A.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/175/F1.medium.gif" width="440" height="326"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/2/175/F1.medium.gif" width="440" height="326"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/175/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/175/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461347" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-21" class="first-child">(A) is the perfect phylogeny tree generated on the phased haplotypes in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1B</a>. Each node is labeled by its haplotype ID, followed by the corresponding phenotype value. (B) is a tristate semi-perfect phylogeny tree generated on the unphased genotypes in <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1A</a>. Each node is labeled by its sample ID followed by the corresponding phenotype value. (C) is the corresponding perfect phylogeny tree by deleting <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/F1/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/F1/embed/mml-math-33.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/F1/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/F1/embed/mml-math-34.gif"/></noscript></span></span> in <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1A</a>, and (D) is the corresponding perfect phylogeny tree by deleting samples <em>C</em> and <em>D</em> in <a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1A</a>.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-8" class="subsection"><h3>Compatible interval</h3><p id="p-22">An interval C<sub><em>u</em>,<em>v</em></sub><span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-35.gif"/></noscript></span></span> is a <em>compatible interval</em> if every pair of SNP markers in the interval pass the four-gamete test (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hudson and Kaplan 1985</a>). That is, at most three of the four possible allele pairs {00, 01, 10, 11} appear in each pair of SNPs in the interval. This implies the existence of an evolution genealogy that can explain the evolutionary history of these two markers without recombination events, given the assumption of an infinite site model (<em>i.e.,</em> no homoplasy). For a given interval, a perfect phylogeny exists if and only if the interval is a compatible interval. If a compatible interval is not a subinterval of another compatible interval, it is called a <em>maximal</em> compatible interval.</p></div><div id="sec-9" class="subsection"><h3>Tristate semi-perfect phylogeny tree</h3><p id="p-23">The multistate perfect phylogeny tree (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Gusfield 2010</a>) is a natural extension of the perfect phylogeny tree discussed previously. It was originally proposed to model the rare events having multiple mutations at a single locus. Because the perfect phylogeny cannot handle heterozygous site properly, we propose a novel utility of the multistate phylogeny in modeling heterozygosity in QTL mapping. By treating the heterozygous allele as the third status, a tristate phylogeny tree can be generated from a set of unphased genotypes. Because this third state is not a result of a single mutation, the tristate phylogeny tree is a relaxation of a perfect phylogeny tree.</p><div id="sec-10" class="subsection"><h4>Definition 2:</h4><p id="p-24"><em>Given an interval <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-36.gif"/></noscript></span></span> of m genotypes and n SNPs, a tristate semi-perfect phylogeny tree is a tree in which the genotype sequences are the leaves and SNPs are the edges. A SNP corresponds to an edge if only two of the three possible alleles are observed and corresponds to two edges if all three alleles are observed. Given an allele of any SNP, the subgraph induced by all the nodes that carry the same allele is still a connected subtree.</em></p></div></div><div id="sec-11" class="subsection"><h3>Compatibility test on genotype data</h3><p id="p-25">Given an interval <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-37.gif"/></noscript></span></span> in the genotype matrix, we construct a binary matrix <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-38.gif"/></noscript></span></span>. Each column <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-39.gif"/></noscript></span></span> in <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-40.gif"/></noscript></span></span> corresponds to three binary columns <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-41.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-42.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-43.gif"/></noscript></span></span> in <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-44.gif"/></noscript></span></span>. <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-45.gif"/></noscript></span></span> is generated from <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-46.gif"/></noscript></span></span> by replacing every ‘H’ in <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-47.gif"/></noscript></span></span> by ‘1’. <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-48.gif"/></noscript></span></span> is generated from <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-49.gif"/></noscript></span></span> by replacing every ‘H’ in <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-50.gif"/></noscript></span></span> by ‘0’. <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-51.gif"/></noscript></span></span> is generated from <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-52.gif"/></noscript></span></span> by replacing every ‘H’ in <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-53.gif"/></noscript></span></span> by ‘1’ and ‘0’ and ‘1’ in <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-54.gif"/></noscript></span></span> by ‘0.’ This is equivalent to representing the ‘0,’‘1,'and ‘H’ alleles in the heterozygous <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-55.gif"/></noscript></span></span> by triplets (0,0,0), (1,1,0), and (1,0,1), respectively. For example, <a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1C</a> shows the generated binary matrix <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-56.gif"/></noscript></span></span> for the genotype matrix C<sub><em>u</em>,<em>v</em></sub><span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-57.gif"/></noscript></span></span> in <a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1A</a>. Note that all states in <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-58.gif"/></noscript></span></span> are identical to that in C<sub><em>u</em>,<em>v</em></sub><span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-59.gif"/></noscript></span></span> except the ‘H’ alleles and <em>S</em>′(<em>H</em>) columns. Given an interval, the following theorem states the necessary and sufficient condition for the existence of a tri-state semi-perfect phylogeny (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dress and Steel 1992</a>).</p></div><div id="sec-12" class="subsection"><h3>Theorem 1:</h3><p id="p-26"><em>Given an interval <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-60.gif"/></noscript></span></span> in the genotype matrix, there exists a tristate semi-perfect phylogeny, if and only if there exists a submatrix S formed by selecting two of the three columns in <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-61.gif"/></noscript></span></span> for each SNP marker, and any pair of columns in S pass the four-gamete test.</em></p><p id="p-27">An integer linear programming approach (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Gusfield 2010</a>) can be used to determine whether an interval is compatible and to compute the submatrix S. For example, in the matrix <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-62.gif"/></noscript></span></span> shown in <a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Table 1C</a>, the columns selected for <em>S</em> are boldface. Once S is computed, a tristate semi-perfect phylogeny tree can be constructed by applying any standard perfect phylogeny tree algorithm on S. For example, <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a> shows the tristate semi-perfect phylogeny tree constructed from the matrix <em>S</em> in <a id="xref-table-wrap-1-11" class="xref-table" href="#T1">Table 1C</a>.</p><p id="p-28">If there is no heterozygous allele, each genotype will be composed of two identical haplotypes; the tristate semi-perfect phylogeny tree is identical to the perfect phylogeny tree constructed on the haplotypes. If there are some heterozygous genotypes, removing the rows or columns in the matrix containing the heterozygous alleles does not affect the remaining part of the phylogeny tree. The tree in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a> shows the perfect phylogeny tree constructed on <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-63.gif"/></noscript></span></span> in <a id="xref-table-wrap-1-12" class="xref-table" href="#T1">Table 1A</a>, which can also be derived by collapsing the three edges labeled by <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-64.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-65.gif"/></noscript></span></span> in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>. If we remove nodes C and D (that have heterozygous genotypes) in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a>, the resulting tree is also identical to the perfect phylogeny tree constructed on A, B, E (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1D</a>). We observe that any heterozygosity only introduces local variations in a phylogeny tree.</p><p id="p-29">Another important observation can be made by comparing the perfect phylogeny tree constructed on the haplotypes to the genotype matrix. When the genotype matrix contains a small percentage of heterozygosity, the tristate semi-perfect phylogeny tree shares a substantial common structure with the perfect phylogeny tree on the haplotypes. <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1A</a> shows the perfect phylogeny tree constructed on the haplotypes in <a id="xref-table-wrap-1-13" class="xref-table" href="#T1">Table 1B</a>. Note that the two haplotypes (<em>e.g.,</em> D1, D2) of the same genotype (<em>e.g.</em> D) may be associated with different nodes in the tree. We will show later that this decoupling will weaken the power of detecting nonadditive genetic effects. However, this tree shares common induced subtrees with the tristate semi-perfect phylogeny tree. Removing the nodes associated with the decoupled haplotypes will result in <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1D</a>, whereas collapsing edges connecting these nodes will result in <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1C</a>.</p></div><div id="sec-13" class="subsection"><h3>Phylogeny tree−based test</h3><p id="p-30">An edge in a phylogeny tree connects two disjoint subtrees. Removing <em>x</em> edges partitions the tree into <em>x</em> + 1 subtrees. For example, removing the two edges labeled with <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-66.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-67.gif"/></noscript></span></span> in <a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1B</a> partitions genotypes into three groups {A, B, D}, {C}, and {E}.</p><p id="p-31">The statistical correlation between a partition and the phenotype can be examined by the F-statistics. Assuming that for a total of <em>t</em> individuals, we have <em>p</em> groups, and the <em>i</em>th group contains <em>t<sub>i</sub></em> individuals. We use <em>X<sub>ij</sub></em> to represent the <em>i</em>th element in the <em>j</em>th group, <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-68.gif"/></noscript></span></span> to represent the mean of the <em>j</em>th group, and <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-69.gif"/></noscript></span></span> to represent the overall mean value. Given such a grouping of phenotype values, <em>G</em>, the F-statistics is defined as<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-70.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span></p><p id="p-32">The corresponding <em>P</em>-value of <em>F</em>(<em>G</em>) can be calculated in the following way. If the phenotype values from each group follow a normal distribution, an <em>F</em>-test is applied to obtain the corresponding <em>P</em>-value. Otherwise, a permutation test is needed. The <em>P</em>-value is defined as <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-71.gif"/></noscript></span></span> where <em>nPerm</em> is the number of permutations and <em>n</em> is the number of times when the F-statistics of the permuted phenotype is larger than <em>F</em>(<em>G</em>).</p><p id="p-33">We examine all possible partitions generated by removing edges in the tree. The partition that generates the most significant <em>P</em>-value is reported. The corresponding <em>P</em>-value is used as the nominal (uncorrected) <em>P</em>-value of the association between the compatible interval and the phenotype.</p></div><div id="sec-14" class="subsection"><h3>Permutation test for family-wise error rate (FWER) controlling</h3><p id="p-34">Appropriate multiple testing correction is crucial for QTL studies. In HTreeQA, we apply the widely used permutation test to control family-wise error rate (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Westfall and Young 1993</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Churchill and Doerge 1994</a>). In each permutation, the phenotype values are randomly shuffled and reassigned to individuals. For each permuted phenotype, we repeat the previously described procedure and find the smallest <em>P</em>-value. The corrected <em>P</em>-value is the proportion of the permuted data whose <em>P</em>-values are more significant than that of the original data. We refer to such a corrected <em>P</em>-value as the permutation <em>P</em>-value. The basic routine of HTreeQA is summarized in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/175/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The workflow of HTreeQA. The inputs are the genotype and phenotype data. The output is a list of phylogenies and their P-values for measuring the association with the phenotype, and a threshold of P-value representing the 5% FWER." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1944878361" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The workflow of HTreeQA. The inputs are the genotype and phenotype data. The output is a list of phylogenies and their &lt;em&gt;P&lt;/em&gt;-values for measuring the association with the phenotype, and a threshold of &lt;em&gt;P&lt;/em&gt;-value representing the 5% FWER.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/175/F2.medium.gif" width="440" height="342"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/2/175/F2.medium.gif" width="440" height="342"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/175/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/175/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461316" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-35" class="first-child">The workflow of HTreeQA. The inputs are the genotype and phenotype data. The output is a list of phylogenies and their <em>P</em>-values for measuring the association with the phenotype, and a threshold of <em>P</em>-value representing the 5% FWER.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Comparison between TreeQA and HTreeQA</h3><p id="p-36">We outline two alternative approaches for local phylogeny-based QTL mapping methods and discuss their pros and cons.</p><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-38">HTreeQA: We compute compatible intervals by using integer linear programming and construct a tristate semi-perfect phylogeny tree for each compatible interval. Then we follow the procedure described above to find significant associations.</p></li><li id="list-item-2"><p id="p-39">Running TreeQA on phased data: We first phase the genotypes using any standard phasing algorithm and then apply TreeQA on the resulting haplotypes. Each haplotype is assumed to have the same phenotype value as the original genotype.</p></li></ul><p id="p-40">The second approach has an inherent drawback. It decouples the two haplotypes of the same genotype. As a result, the two haplotypes may reside in remote branches of the tree, which limits the ability to test certain genetic effects in QTL mapping. For example, the phenotype in <a id="xref-table-wrap-1-14" class="xref-table" href="#T1">Table 1A</a> follows a recessive model defined on <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-72.gif"/></noscript></span></span> : the phenotype is 2 for samples (C, E) having minor allele (‘1’) and is 10 for the remaining samples A, B, D (with alleles ‘0’ or ‘H’). There does not exist a set of edges in <a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1A</a> that can perfectly separate these two groups. (The haplotype D2 will always be in the same group as C1, E1, E2.) In contrast, the tristate semi-perfect phylogeny tree has an edge <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-73.gif"/></noscript></span></span> that perfectly separates A, B, and D from C, E. Therefore, the tristate semi-perfect phylogeny tree is more suitable for handling heterozygosity in association studies. We provide a theoretical comparison of these two approaches in <a id="xref-app-1-1" class="xref-app" href="#app-1">Appendix 1</a>.</p></div></div><div class="section results-discussion" id="sec-16"><h2 class="">Results and Discussion</h2><div id="sec-17" class="subsection"><h3>Population structure in the PreCC lines</h3><p id="p-41">Population stratification is an important issue in QTL analysis. Spurious associations may be induced by the stratification if it is not addressed properly (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Kang <em>et al.</em> 2008</a>). The combinatorial breeding design of the CC yields genetically independent incipient CC lines, which ensures balanced contributions of all eight founder strains without noticeable global population stratification (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Aylor <em>et al.</em> 2011</a>). <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a> shows a global phylogeny tree of 43 randomly selected PreCC lines. The balanced tree structure illustrates that these mice are genetically diverse and equally distant from each other. This observation is further confirmed by the kinship matrix in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a> used by EMMA for modeling genetic background (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Kang <em>et al.</em> 2008</a>). In <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>, each row (column) of the kinship matrix corresponds to a CC strain. Each entry in the matrix is the kinship coefficient that represents the genetic relatedness between the two mice. We can observe that all off-diagonal entries in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a> have almost identical values (around 0.8), which suggests that no significant global population stratification exists in these PreCC mice. (In <a id="xref-app-2-1" class="xref-app" href="#app-2">Appendix 2</a>, we provide a statistical analysis that EMMA degenerates to a standard linear model when applied to the CC lines.)</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/175/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Four phylogenies of 43 randomly selected (from a total of 184) PreCC mice. The sum of the edge depth between a leaf and the origin represents the genetic distance of the corresponding mouse from the common ancestry of the 43 mice. The mice with white head spot are highlighted in red. Their nearest common ancestor is indicated by a circled &#x201C;A&#x201D; in each figure. In (A), the global phylogeny is balanced, and all mice are almost equally distant from each other. The phylogenies in (B) and (C) are no longer balanced, with several deep branches. The local population structure is a confounding factor that complexes the QTL analysis. The tristate semi-perfect phylogeny in (D) has the simplest structure, with an informative branch that contains all four white spot mice." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1944878361" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Four phylogenies of 43 randomly selected (from a total of 184) PreCC mice. The sum of the edge depth between a leaf and the origin represents the genetic distance of the corresponding mouse from the common ancestry of the 43 mice. The mice with white head spot are highlighted in red. Their nearest common ancestor is indicated by a circled &#x201C;A&#x201D; in each figure. In (A), the global phylogeny is balanced, and all mice are almost equally distant from each other. The phylogenies in (B) and (C) are no longer balanced, with several deep branches. The local population structure is a confounding factor that complexes the QTL analysis. The tristate semi-perfect phylogeny in (D) has the simplest structure, with an informative branch that contains all four white spot mice.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/175/F3.medium.gif" width="435" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/2/2/175/F3.medium.gif" width="435" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/175/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/175/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461280" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-42" class="first-child">Four phylogenies of 43 randomly selected (from a total of 184) PreCC mice. The sum of the edge depth between a leaf and the origin represents the genetic distance of the corresponding mouse from the common ancestry of the 43 mice. The mice with white head spot are highlighted in red. Their nearest common ancestor is indicated by a circled “A” in each figure. In (A), the global phylogeny is balanced, and all mice are almost equally distant from each other. The phylogenies in (B) and (C) are no longer balanced, with several deep branches. The local population structure is a confounding factor that complexes the QTL analysis. The tristate semi-perfect phylogeny in (D) has the simplest structure, with an informative branch that contains all four white spot mice.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/175/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Three kinship matrices represent the genetic relatedness over the entire genome between any pair of the 184 CC mice based on the whole genome (A), the chromosome 10 (B), and the 20-Mbps interval in Chromosome 10 (C) respectively. The mice are arranged in the same order in both x and y axes. In (A), all off-diagonal entries have almost identical values, suggesting that there is no global population structure. In (B) and (C), the mice are arranged in the order of their genetic relatedness, genetically similar mice are near each other." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1944878361" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Three kinship matrices represent the genetic relatedness over the entire genome between any pair of the 184 CC mice based on the whole genome (A), the chromosome 10 (B), and the 20-Mbps interval in Chromosome 10 (C) respectively. The mice are arranged in the same order in both x and y axes. In (A), all off-diagonal entries have almost identical values, suggesting that there is no global population structure. In (B) and (C), the mice are arranged in the order of their genetic relatedness, genetically similar mice are near each other.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/175/F4.medium.gif" width="440" height="119"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/2/2/175/F4.medium.gif" width="440" height="119"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/175/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/175/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461240" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-43" class="first-child">Three kinship matrices represent the genetic relatedness over the entire genome between any pair of the 184 CC mice based on the whole genome (A), the chromosome 10 (B), and the 20-Mbps interval in Chromosome 10 (C) respectively. The mice are arranged in the same order in both x and y axes. In (A), all off-diagonal entries have almost identical values, suggesting that there is no global population structure. In (B) and (C), the mice are arranged in the order of their genetic relatedness, genetically similar mice are near each other.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">Although the genome of each CC line receives a balanced contribution from each founder strain, the founder contribution is not uniformly distributed along the genome because of the small number of recombination events undergone by each CC line. The genome of a CC line is essentially a mosaic of a small number of founder haplotype segments. On average, Pre-CC autosomal genomes had 142.3 segments on average (SD = 21.8) with a median segment length of 10.46 Mb (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Aylor <em>et al.</em> 2011</a>). As a result, some local subpopulation structure may be observed because the eight founder strains are not equally distant from each other (<em>i.e.,</em> three of founders are wild strains). The subpopulation structure is visible at the chromosome level. For example, there are several deep branches in the phylogeny tree of the selected PreCC mice built on Chromosome 10 (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). The corresponding kinship matrix in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a> shows that there are at least three subpopulations. The subpopulation structure is more evident if we narrow down to a 20 Mbps interval from 85 Mbps to 105 Mbps on Chromosome 10. The phylogeny tree in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a> becomes more skewed, and the corresponding kinship matrix in <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4C</a> also exhibits more pronounced structural patterns.</p></div><div id="sec-18" class="subsection"><h3>Selected methods for comparison</h3><p id="p-45">We compare our algorithm HTreeQA with existing methods: TreeQA (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Pan <em>et al.</em> 2008</a>, <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">2009</a>), BLOSSOC (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Mailund <em>et al.</em> 2006</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Besenbacher <em>et al.</em> 2009</a>), EMMA (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Kang <em>et al.</em> 2008</a>), and HAM (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Mcclurg <em>et al.</em> 2006</a>) using both real and simulated data sets. Some other methods, such as HapMiner (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Li and Jiang 2005</a>) and TreeLD (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Zöllner and Pritchard 2005</a>), are too slow to process large data sets. For comparison purposes, we also implemented two other methods: SMA (single marker association mapping) and HAM (haplotype association mapping). In SMA, each SNP marker partitions samples into groups on the basis of the alleles. Analysis of variance is used to evaluate the significance of the partition. In HAM, a sliding window of three consecutive SNP is used to group samples on the basis of their sequences, and an analysis of variance is conducted to test the association between the phenotypes and the grouping. FastPhase (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Scheet and Stephens 2006</a>) is used to reconstruct haplotypes from the genotypes for the methods that require haplotype data (TreeQA and BLOSSOC).</p><p id="p-46">Note that BLOSSOC, TreeQA, and HTreeQA are phylogeny-based methods. SMA, HAM, and EMMA are nonphylogeny-based methods. Although EMMA offers an option to use global phylogeny to estimate the kinship matrix, it does not test the associations between the phenotype and the phylogenetic trees. <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> shows the selected methods for comparison.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461295/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461295/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <p id="p-47" class="first-child">Selected methods for comparison</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Performance comparison on the white head spot phenotype</h3><p id="p-48">The white head spot is known as a recessive trait carried by WSB/EiJ (<a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Aylor <em>et al.</em> 2011</a>). We apply the selected methods to the white head spot phenotype. A permutation test is applied to control the FWER (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Westfall and Young 1993</a>, <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Churchill and Doerge 1994</a>). With FWER = 0.05, all the selected methods except HAM identify a QTL, which is approximately 100M bps in Chromosome 10 (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). This QTL is close to a gene named <em>kit ligand</em> known to be controlling white spotting (<a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">Aylor <em>et al.</em> 2011</a>). HAM fails to detect the QTL because it does not consider the compatibility between consecutive SNPs. The incompatibility between two consecutive SNPs suggests a high possibility of having a historical recombination event between them. Treating an interval containing incompatible SNPs as a single locus may lead to spurious results. The phylogeny-based methods, including HTreeQA, can avoid this problem by only examining phylogeny trees constructed from compatible intervals.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/175/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="QTL mapping of the white head spot phenotype. Only the SNPs that have top 0.5% -log(p-value) or BLOSSOC score are plotted. One QTL is detected by HTreeQA, which is near the location of gene kit ligand. The remaining methods except HAM have similar results to that of HTreeQA. The dashed line is the significance level with FWER = 0.05. (A) Result from HTreeQA. (B) Result from TreeQA. (C) Result from EMMA. (D) Result from BLOSSOC. (E) Result from HAM. (F) Result from SMA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1944878361" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;QTL mapping of the white head spot phenotype. Only the SNPs that have top 0.5% -log(p-value) or BLOSSOC score are plotted. One QTL is detected by HTreeQA, which is near the location of gene &lt;em&gt;kit ligand&lt;/em&gt;. The remaining methods except HAM have similar results to that of HTreeQA. The dashed line is the significance level with FWER = 0.05. (A) Result from HTreeQA. (B) Result from TreeQA. (C) Result from EMMA. (D) Result from BLOSSOC. (E) Result from HAM. (F) Result from SMA.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/175/F5.medium.gif" width="440" height="321"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/2/2/175/F5.medium.gif" width="440" height="321"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/175/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/175/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461339" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-49" class="first-child">QTL mapping of the white head spot phenotype. Only the SNPs that have top 0.5% -log(p-value) or BLOSSOC score are plotted. One QTL is detected by HTreeQA, which is near the location of gene <em>kit ligand</em>. The remaining methods except HAM have similar results to that of HTreeQA. The dashed line is the significance level with FWER = 0.05. (A) Result from HTreeQA. (B) Result from TreeQA. (C) Result from EMMA. (D) Result from BLOSSOC. (E) Result from HAM. (F) Result from SMA.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">In each panel of <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3, A−D</a>, the nearest common ancestor of the four white head spot mice (highlighted in red) is marked by a circled “A.” We observe from <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3, A−C</a> that the distance between the common ancestor and the four mice becomes smaller when the interval on which the tree is built becomes shorter. It is evident that the four white spot mice are clustered in the phylogeny tree built over the 20 Mb region in <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3C</a>, despite the local population structure. This becomes clearer in <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3D</a>, where the four white head spot mice having white head spot located on the same branch of the tristate semi-perfect phylogeny tree built on the compatible interval at the QTL. This demonstrates the effectiveness of the proposed model.</p></div><div id="sec-20" class="subsection"><h3>Performance comparison on the mouse running distance phenotype</h3><p id="p-51">We apply the selected methods on the phenotype “Mouse Running Distance at day 5/6.” With FWER = 0.05, all the methods except SMA identified a QTL at 169 to 169.2 Mbp (89 cM) on Chromosome 1 as shown in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>. The QTL falls into the previously reported <em>cplaq3</em> region (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Mayeda and Hofstetter 1999</a>). A later study also confirmed this QTL (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hofstetter <em>et al.</em> 2003</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/175/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="QTL for mice daily average running distance. Only the SNPs that have top 0.5% -log(p-value) or BLOSSOC score are plotted in the figure. The dashed line is the significance level with FWER = 0.05. (A) Result from HTreeQA. (B) Result from TreeQA. (C) Result from EMMA. (D) Result from BLOSSOC. (E) Result from HAM. (F) Result from SMA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1944878361" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;QTL for mice daily average running distance. Only the SNPs that have top 0.5% -log(p-value) or BLOSSOC score are plotted in the figure. The dashed line is the significance level with FWER = 0.05. (A) Result from HTreeQA. (B) Result from TreeQA. (C) Result from EMMA. (D) Result from BLOSSOC. (E) Result from HAM. (F) Result from SMA.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/175/F6.medium.gif" width="440" height="318"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/2/2/175/F6.medium.gif" width="440" height="318"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/175/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/175/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461264" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 </span> <p id="p-52" class="first-child">QTL for mice daily average running distance. Only the SNPs that have top 0.5% -log(p-value) or BLOSSOC score are plotted in the figure. The dashed line is the significance level with FWER = 0.05. (A) Result from HTreeQA. (B) Result from TreeQA. (C) Result from EMMA. (D) Result from BLOSSOC. (E) Result from HAM. (F) Result from SMA.</p><div class="sb-div caption-clear"></div></div></div><p id="p-53">Among the selected methods, only HTreeQA identified another QTL with FWER = 0.05, in the region of 16 M to 25 Mbps (8-12.5 cM) on Chromosome 12. The QTL falls into an unnamed QTL region at 11 cM on Chromosome 12 reported in (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Hofstetter <em>et al.</em> 2003</a>). The reason that many methods fail to report this QTL is that these methods have limited power in detecting non-additive effects. This result demonstrates that HTreeQA can detect more types of effects than the other methods.</p></div><div id="sec-21" class="subsection"><h3>Simulation study</h3><p id="p-54">To examine the performance of HTreeQA in a controlled environment, we simulated three different types of effects: additive, recessive, and overdominant. For each selected method, only the SNPs with significance level FWER = 0.05 are reported as QTL. Because we remove the causative SNPs in the simulated data before we run QTL analysis, to measure the accuracy of the result, we considered a reported QTL a true positive when it was located within 50 SNPs from the causative SNP. We used three measurements to estimate the performance of each method: <em>precision</em>, <em>recall</em>, and <em>F1 score</em>. Precision is defined as the ratio between the number of true QTL that are detected and the total number of detected QTL. Recall is defined as the ratio between the number of true QTL that are detected and the total number of true QTL that are simulated. The F1 score is the harmonic mean of precision rate and recall rate, and is defined as follows:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-74.gif"/></noscript></span></span></p><p id="p-55"><a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> compares selected methods. HTreeQA shows comparable performance to that of other methods in the additive model. In the recessive model and the overdominant model, HTreeQA demonstrates significant advantage over other methods. Because HTreeQA does not have any assumption of the type of genetic effect, it offers consistent power for detecting any effect. Other methods except HAM implicitly assume the additive model.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/175/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of HTreeQA, TreeQA, SSA, BLOSSOC, EMMA, and HAM under different genetic models. (A), (D), and (G) are under additive models; (B), (E), and (H) are under recessive models; (C), (F), and (I) are under overdominant models." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1944878361" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of HTreeQA, TreeQA, SSA, BLOSSOC, EMMA, and HAM under different genetic models. (A), (D), and (G) are under additive models; (B), (E), and (H) are under recessive models; (C), (F), and (I) are under overdominant models.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/175/F7.medium.gif" width="440" height="435"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7&#x2002;" src="https://www.g3journal.org/content/ggg/2/2/175/F7.medium.gif" width="440" height="435"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/175/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/175/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461214" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7 </span> <p id="p-56" class="first-child">Comparison of HTreeQA, TreeQA, SSA, BLOSSOC, EMMA, and HAM under different genetic models. (A), (D), and (G) are under additive models; (B), (E), and (H) are under recessive models; (C), (F), and (I) are under overdominant models.</p><div class="sb-div caption-clear"></div></div></div><p id="p-57">The phasing step required by the phylogeny-based methods BLOSSOC and TreeQA (for handling heterozygosity) will impair their ability in detecting associations between the phylogeny and the phenotype. The extent of its effect varies for different genetic models, especially with regard to heterozygous samples. It affects the additive model the least and overdominant model the most. For a homozygous sample, the nodes corresponding to the two haplotypes carry the same allele, and thus their phenotypes always belong to the same allele group. This may cause minor inflation of the QTL signals because the two haplotypes are treated as independent samples by these methods. For a heterozygous sample the two haplotypes carry different alleles and therefore their corresponding nodes and phenotype are in two allele groups. Under the additive model assumption, one allele group contains all homozygous samples with high phenotype values, and the other contains all homozygous samples with low phenotype values. The heterozygous samples have medium phenotype values, which are added to both allele groups. This may cause minor deflation of the QTL signals. This is why all selected methods have comparable performance. TreeQA slightly outperforms others because its local phylogeny trees can well model the local population structure and separate QTL signals from genetic background.</p><p id="p-58">However, under the assumption of overdominant model, heterozygous samples may have extreme phenotype values (beyond the range of phenotype values of the homozygous samples). These extreme phenotype values will always be in both allele groups; therefore, the phylogeny representation for phased data cannot explain the overdominant effects at all. This is why the traditional phylogeny-based methods like BLOSSOC and TreeQA fail under such a model. Note that HTreeQA does not require phasing. The tristate semi-perfect phylogeny tree has a partition that separates the heterozygous samples from the homozygous samples and thus it is able to detect an overdominant effect. Under the recessive model assumption, the heterozygous allele carries the same effect as one of the two homozygous alleles. Thus, the impact of assigning haplotypes of the heterozygous samples to the two allele groups is greater than that under the additive model and is not as great as that under the overdominant model. Again, this does not affect HTreeQA. Overall, HTreeQA has the best performance in recessive models and overdominant models.</p></div><div id="sec-22" class="subsection"><h3>Running time comparison</h3><p id="p-59">We present the running time for each selected method on a machine with Intel i7 2.67-GHz CPU and 8-G memory. We tested all methods using a dataset containing 180K SNPs and 184 individuals. <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> shows the running time of these methods. If phasing is required, this step usually takes more than 40 hr and dominates the running time. HTreeQA demonstrates a great advantage by completely avoiding haplotype reconstruction. It is more than 600 times faster than the other methods that require haplotype data. HTreeQA is 15 times faster than EMMA because it does not need to explicitly incorporate the effect of global population structure as EMMA does. The running time of HTreeQA is comparable with that of SMA and HAM, the simplest models for QTL studies. They are not as effective as HTreeQA, as demonstrated in the real phenotype and simulation studies.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461242/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461242/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <p id="p-60" class="first-child">Running time comparison of the selected methods</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-23" class="subsection"><h3>The choice between HTreeQA, TreeQA, and EMMA</h3><p id="p-62">HTreeQA is proven to have an overall lower error rate than TreeQA and other similar approaches (in <a id="xref-app-1-2" class="xref-app" href="#app-1">Appendix 1</a>). It can handle heterozygous genotype properly. It is suitable for genome-wide association studies on any populations, including the incipient CC lines, Heterogeneous Stock, Diversity Outbred, and Recombinant Inbred Crosses of CC lines. TreeQA is the best choice if one focuses on the additive effects. EMMA can correct for global population structure but is not able to address any local population structure. It degenerates to a simple linear model when applied to CC population with an evenly distributed global population structure as shown in <a id="xref-app-2-2" class="xref-app" href="#app-2">Appendix 2</a>. This represents a limitation of EMMA because local population structures exist in every mammalian resource, even though we only show the results on the CC population in this article.</p></div></div><div class="section conclusions" id="sec-24"><h2 class="">Conclusions</h2><p id="p-63">We propose a novel approach for local phylogeny-based QTL mapping on genotype data without haplotype reconstruction. We analyze the incipient CC and show that there is no significant global population structure but visible local population structure. Such local population structure may bias the QTL mapping if it is not addressed properly. The notion of a tristate semi-perfect phylogeny tree is introduced to represent accurate genetic relationships between samples in short genomic regions. As a generalization of the perfect phylogeny tree (defined on haplotypes), a tristate semi-perfect phylogeny tree treats the heterozygous allele as the third state. It provides the power of modeling a wide range of genetic effects and delivers unbiased and consistent performance. It also guarantees a lower theoretical error rate of statistical tests than the perfect phylogeny based approach. This is a significant advantage over any previous methods that have strong bias toward an additive model. It is also worth noting that HTreeQA is much more computationally efficient than any alternative approach.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-64">We thank William Valdar, Gary Churchill, Leonard McMillan, and the members in UNC Computational Genetics Group for valuable discussions. We thank Gary Churchill, Fernando Pardo-Manuel de Villena, and Daniel Pomp for providing the genotype and phenotype data used in this study. This work was partially supported by National Science Foundation awards IIS0448392, IIS0812464, and National Institutes of Health awards U01CA105417, U01CA134240, and MH090338.</p></div><div class="section app" id="app-1"><h2 class="">Appendix 1</h2><div class="section" id="sec-25"><h3>Theoretical Analysis on HTreeQA and TreeQA</h3><p id="p-65">In this section, we present the theoretical analysis of HTreeQA and TreeQA under different genetic models. It can be shown that HTreeQA has a theoretical advantage over the general phylogeny-based approach using phased haplotypes. We first prove that testing single SNPs on genotypes data has lower error rate than on phased haplotype data. We then analyze its potential effect on these two different phylogeny approaches.</p><p id="p-66">We assume that the causative SNP contains <em>n</em><sub>1</sub> homozygous subjects, <em>n<sub>h</sub></em> heterozygous subjects, and <em>n</em><sub>0</sub> homozygous wild subjects. We also assume that the phenotypes can be approximated by a normal distribution, which is a reasonable assumption in most cases. We use <em>X<sub>i</sub></em><sub>1</sub>, <em>X<sub>ih</sub></em>, and <em>X<sub>i</sub></em><sub>0</sub> to model each subject in these three groups:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-75.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-76.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-77.gif"/></noscript></span></span></p><p id="p-67">Without loss of generality, we assume the samples are independent and follow three normal distributions with different means and variances for each group. If <em>m<sub>h</sub></em> equals 0 or 1, it is a recessive model. If <em>m<sub>h</sub></em> is between 0 and 1, it is an additive model. Otherwise it is an overdominance model. If we use a phylogeny-based approach on phased haplotypes, each homozygous subject has a duplicate homozygous subject, and each heterozygous subject is treated as two different homozygous subjects. Thus we could use two groups to represent the partition of this SNP, {<em>X</em><sub>11</sub>, …, <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-78.gif"/></noscript></span></span>, <em>X</em><sub>11</sub>, …, <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-79.gif"/></noscript></span></span>, <em>X</em><sub>1<em>h</em>,</sub> …, <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-80.gif"/></noscript></span></span>} and {<em>X</em><sub>10</sub>, …, <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-81.gif"/></noscript></span></span>, <em>X</em><sub>10</sub>, …, <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-82.gif"/></noscript></span></span>, <em>X</em><sub>1<em>h</em>,</sub> …, <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-83.gif"/></noscript></span></span>}. If we use HTreeQA, which is directly applied on genotype data, there are three groups based on the allele of each subject, {<em>X</em><sub>11</sub>, …, <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-84.gif"/></noscript></span></span>}, {<em>X</em><sub>1<em>h</em>,</sub> …, <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-85.gif"/></noscript></span></span>}, and {<em>X</em><sub>10</sub>, …, <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-86.gif"/></noscript></span></span>}.<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-87.gif"/></noscript></span><span class="disp-formula-label">(A1)</span></span><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-88.gif"/></noscript></span><span class="disp-formula-label">(A2)</span></span><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-89.gif"/></noscript></span><span class="disp-formula-label">(A3)</span></span><span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-90.gif"/></noscript></span><span class="disp-formula-label">(A4)</span></span><span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-91.gif"/></noscript></span><span class="disp-formula-label">(A5)</span></span><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-92.gif"/></noscript></span><span class="disp-formula-label">(A6)</span></span><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-93.gif"/></noscript></span><span class="disp-formula-label">(A7)</span></span><span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-94.gif"/></noscript></span><span class="disp-formula-label">(A8)</span></span><span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-95.gif"/></noscript></span><span class="disp-formula-label">(A9)</span></span><span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-96.gif"/></noscript></span><span class="disp-formula-label">(A10)</span></span></p><p id="p-68">Following <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">Equation 1</a> in the <em>Methods</em> and A1 to A10, we define <em>F<sub>Haplotype</sub></em> and <em>F<sub>Genotype</sub></em> to represent the <em>F</em>-statistics of these two different groupings respectively.</p><span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-97.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-98.gif"/></noscript></span></span><p id="p-69">For the following analysis we assume that <em>n</em><sub>1</sub>, <em>n<sub>h</sub></em>, and <em>n</em><sub>0</sub> are large numbers, and we use ‘<em>a</em> ∼ <em>b</em>’ to denote <em>a</em> and <em>b</em> are asymptotically equal when the sample size approaches infinity. Here <em>b</em> is a number instead of a distribution. Similarly, we use ‘≲’ and ‘≳’ to represent asymptotically less than and greater than relationship respectively. Next, we prove that directly testing associations between a phenotype and the genotypes has a lower error rate than testing the association between the phenotypes and phased haplotypes when the sample size is large.</p><p id="p-70">First, for large sample sizes, we have the following lemmas as an immediate consequence of the Weak Law of Large Number Theorem,</p></div><div class="section" id="sec-26"><h3>Lemma 1</h3><span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-99.gif"/></noscript></span></span></div><div class="section" id="sec-27"><h3>Lemma 2 S<sub><em>Haplotype</em></sub> ≲ 2S<sub><em>Genotype</em></sub></h3><div id="sec-28" class="subsection"><h4>Proof</h4><p id="p-71">Sketch: The asymptotic values for variables in <a id="xref-disp-formula-14-1" class="xref-disp-formula" href="#disp-formula-14">Equations A7 and A9</a> are determined by Lemma 1. And the expanded form of <em>S<sub>Haplotype</sub></em> − <em>2S<sub>Genotype</sub></em> is a quadratic function of <em>m<sub>h</sub></em><sub>,</sub> and its discriminant is smaller than 0.</p></div></div><div class="section" id="sec-29"><h3>Lemma 3</h3><p id="p-72"><em>N random variables Y<sub>i</sub></em> <em>are independent and identically distributed, with mean value μ and finite variance ϕ. For any real number γ ≠ μ, when N → ∞, we have</em></p><span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-100.gif"/></noscript></span><span class="disp-formula-label">(A11)</span></span><div id="sec-30" class="subsection"><h4>Proof</h4><p id="p-73">Without loss of generality, we assume μ − γ &gt; 0,</p><span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-101.gif"/></noscript></span></span></div></div><div class="section" id="sec-31"><h3>Lemma 4 T<sub><em>Haplotype</em></sub> ≲ 2T<sub><em>Genotype</em></sub></h3><div id="sec-32" class="subsection"><h4>Proof</h4><p id="p-74"><span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-102.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-103.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-104.gif"/></noscript></span></span> converge to the mean of <em>X<sub>i</sub></em><sub>1</sub>, <em>X<sub>i</sub></em><sub>0</sub> and <em>X<sub>ih</sub></em> by Lemma 1, but <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-105.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-106.gif"/></noscript></span></span> converge to two different values as shown in Lemma 1. Lemma 4 follows directly from Lemma 3.</p></div></div><div class="section" id="sec-33"><h3>Theorem 2 <em>F<sub>Haplotype</sub></em> ≲<em> F<sub>Genotype</sub></em></h3><div id="sec-34" class="subsection"><h4>Proof</h4><p id="p-75">This can be directly proved from Lemmas 2 and 4.</p><p id="p-76">We use <em>F<sub>Null</sub></em> to represent the statistics of testing non-causative partitions from either a semi-perfect phylogeny tree or a perfect phylogeny tree. Because phenotype values can be approximated by a normal distribution, the distributions of <em>F<sub>Null</sub></em> using these two approaches converge to the same distribution. Although it is unlike that the causative SNP is genotyped in real situation, by linkage disequilibrium, there exists a partition in the semi-perfect phylogeny tree or the perfect phylogeny tree based on neighboring SNPs that is very similar to the partition of the causative SNP. Therefore, we have the following theorem.</p></div></div><div class="section" id="sec-35"><h3>Theorem 3 P(F<sub><em>Null</em></sub> &gt; F<sub><em>Haplotype</em></sub>) ≳ P(F<sub><em>Null</em></sub> &gt; F<sub><em>Genotype</em></sub>)</h3><p id="p-77">The probabilities in the Theorem 3 are the error rates of TreeQA on phased haplotypes and HTreeQA on genotypes.</p></div></div><div class="section app" id="app-2"><h2 class="">Appendix 2</h2><div class="section" id="sec-36"><h3>EMMA will degenerate to standard linear model in Collaborative Cross</h3><p id="p-78">First, we define a new class of matrix named <em>K<sub>uniform</sub></em>(<em>D</em>, <em>S</em>),</p><span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-107.gif"/></noscript></span><span class="disp-formula-label">(A12)</span></span><p id="p-79">where D represents the diagonal entries and S represents the off-diagonal entries in the matrix.</p><p id="p-80">Assume that <strong>y</strong> is a vector of phenotypes, <strong>X</strong> is a vector of fixed effects from a SNP, and <strong>e</strong> is a vector of residual effects for each individual. We omit the indicator matrix <em>Z</em> used in original EMMA model, because in the CC data, <em>Z</em> is an identity matrix. The EMMA model is presented in the following form:</p><span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-108.gif"/></noscript></span><span class="disp-formula-label">(A13)</span></span><span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-109.gif"/></noscript></span><span class="disp-formula-label">(A14)</span></span><span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-110.gif"/></noscript></span><span class="disp-formula-label">(A15)</span></span><span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-111.gif"/></noscript></span><span class="disp-formula-label">(A16)</span></span><p id="p-81">where <em>MVN</em> represents a multivariate normal distribution. <em>K<sub>emma</sub></em> is the kinship matrix inferred by the EMMA package.</p><p id="p-82">Similarly, a standard linear model is in the following form:</p><span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-112.gif"/></noscript></span><span class="disp-formula-label">(A17)</span></span><span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-113.gif"/></noscript></span><span class="disp-formula-label">(A18)</span></span><span class="disp-formula" id="disp-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-114.gif"/></noscript></span><span class="disp-formula-label">(A19)</span></span><p id="p-83">Assuming the samples of a population have exactly the same relatedness <em>S</em>:</p><span class="disp-formula" id="disp-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-115.gif"/></noscript></span><span class="disp-formula-label">(A20)</span></span><span class="disp-formula" id="disp-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-116.gif"/></noscript></span><span class="disp-formula-label">(A21)</span></span><span class="disp-formula" id="disp-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-117.gif"/></noscript></span><span class="disp-formula-label">(A22)</span></span><p id="p-84">Thus, if <em>K<sub>emma</sub></em> = <em>K<sub>uniform</sub></em>(1, <em>S</em>), by re-factorization of the random effects in the EMMA model, we have</p><span class="disp-formula" id="disp-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-118.gif"/></noscript></span><span class="disp-formula-label">(A23)</span></span><span class="disp-formula" id="disp-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-119.gif"/></noscript></span><span class="disp-formula-label">(A24)</span></span><span class="disp-formula" id="disp-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/175/embed/mml-math-120.gif"/></noscript></span><span class="disp-formula-label">(A25)</span></span><p id="p-85">This has the same form of a standard linear regression model. In CC, the kinship matrix can be represented by a <em>K<sub>uniform</sub></em> matrix with tolerable numerical error. This suggests that there is no significant difference between EMMA and the standard linear regression model when these two methods are applied to Collaborative Cross data.</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001768/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001768/-/DC1</a></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-10-01"><span class="received-label">Received </span>October 1, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-10-22"><span class="accepted-label">Accepted </span>October 22, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Zhang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.1" data-doi="10.1038/sj.ejhg.5200619"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Haplotypes <em>vs.</em> single-marker linkage disequilibrium tests: what do we gain</span>. <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bhuman%2Bgenetics%2B%253A%2B%2BEJHG%26rft.stitle%253DEur%2BJ%2BHum%2BGenet%26rft.aulast%253DAkey%26rft.auinit1%253DJ.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D291%26rft.epage%253D300%26rft.atitle%253DHaplotypes%2Bvs%2Bsingle%2Bmarker%2Blinkage%2Bdisequilibrium%2Btests%253A%2Bwhat%2Bdo%2Bwe%2Bgain%253F%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.ejhg.5200619%26rft_id%253Dinfo%253Apmid%252F11313774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.ejhg.5200619&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11313774&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F175.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168112200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.2" data-doi="10.1101/gr.111310.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulds-Mathes</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buus</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verdugo</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic analysis of complex traits in the emerging Collaborative Cross</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1213</span>–<span class="cit-lpage">1222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.111310.110%26rft_id%253Dinfo%253Apmid%252F21406540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTIxMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8xNzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Besenbacher</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mailund</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schierup</span>  <span class="cit-name-given-names">M. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Local phylogeny mapping of quantitative traits: higher accuracy and better ranking than single-marker association in genomewide scans</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">747</span>–<span class="cit-lpage">753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D181%26rft.spage%253D747%26rft_id%253Dinfo%253Apmid%252F19064712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19064712&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F175.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Empirical threshold values for quantitative trait mapping</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D963%26rft.epage%253D971%26rft.atitle%253DEmpirical%2Bthreshold%2Bvalues%2Bfor%2Bquantitative%2Btrait%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F7851788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzMvOTYzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8yLzE3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.5" data-doi="10.1534/genetics.111.132639"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Collaborative Cross Consortium</span>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genome architecture of the Collaborative Cross Mouse Genetic Reference Population</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D190%26rft.spage%253D389%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132639%26rft_id%253Dinfo%253Apmid%252F22345608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.111.132639&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22345608&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F175.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300621200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.6" data-doi="10.1111/j.0006-341X.1999.00997.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devlin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Genomic control for association studies</span>. <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">997</span>–<span class="cit-lpage">1004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D55%26rft.spage%253D997%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0006-341X.1999.00997.x%26rft_id%253Dinfo%253Apmid%252F11315092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0006-341X.1999.00997.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11315092&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F175.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084218000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.7" data-doi="10.1093/bioinformatics/btn406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mailund</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient whole-genome association mapping using local phylogenies for unphased genotype data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2215</span>–<span class="cit-lpage">2221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn406%26rft_id%253Dinfo%253Apmid%252F18667442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjQvMTkvMjIxNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8xNzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dress</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steel</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Convex tree realizations of partitions</span>. <abbr class="cit-jnl-abbrev">Appl. Math. Lett.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BMath.%2BLett.%26rft.volume%253D5%26rft.spage%253D3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.9" data-doi="10.1002/net.3230210104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gusfield</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">An efficient algorithms for inferring evolutionary trees</span>. <abbr class="cit-jnl-abbrev">Networks</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNetworks%26rft.volume%253D21%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fnet.3230210104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/net.3230210104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1991EQ55800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.10" data-doi="10.1089/cmb.2009.0200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gusfield</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Multi-State Perfect Phylogeny Problem with missing and removable data: solutions via integer-programming and chordal graph theory</span>. <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">383</span>–<span class="cit-lpage">399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DGusfield%26rft.auinit1%253DD.%26rft.volume%253D17%26rft.issue%253D3%26rft.spage%253D383%26rft.epage%253D399%26rft.atitle%253DThe%2BMulti-State%2BPerfect%2BPhylogeny%2BProblem%2Bwith%2Bmissing%2Band%2Bremovable%2Bdata%253A%2Bsolutions%2Bvia%2Binteger-programming%2Band%2Bchordal%2Bgraph%2Btheory.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2009.0200%26rft_id%253Dinfo%253Apmid%252F20377452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2009.0200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20377452&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F175.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.11" data-doi="10.1177/0748730403259468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hofstetter</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trofatter</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kernek</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nurnberger</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayeda</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">New quantitative trait loci for the genetic variance in circadian period of locomotor activity between inbred strains of mice</span>. <abbr class="cit-jnl-abbrev">J. Biol. Rhythms</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">450</span>–<span class="cit-lpage">462</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BRhythms%26rft.stitle%253DJ%2BBiol%2BRhythms%26rft.aulast%253DHofstetter%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D18%26rft.issue%253D6%26rft.spage%253D450%26rft.epage%253D462%26rft.atitle%253DNew%2BQuantitative%2BTrait%2BLoci%2Bfor%2Bthe%2BGenetic%2BVariance%2Bin%2BCircadian%2BPeriod%2Bof%2BLocomotor%2BActivity%2Bbetween%2BInbred%2BStrains%2Bof%2BMice%26rft_id%253Dinfo%253Adoi%252F10.1177%252F0748730403259468%26rft_id%253Dinfo%253Apmid%252F14667146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToic3BqYnIiO3M6NToicmVzaWQiO3M6ODoiMTgvNi80NTAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMTc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">N. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Statistical properties of the number of recombination events in the history of a sample of DNA sequences</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D111%26rft.spage%253D147%26rft_id%253Dinfo%253Apmid%252F4029609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=4029609&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F175.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985APL1600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.13" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckerman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient control of population structure in model organism association mapping</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1709</span>–<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D178%26rft.spage%253D1709%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F18385116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.107.080101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18385116&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F175.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254921600050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.14" data-doi="10.1006/tpbi.2002.1601"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larribe</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lessard</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schork</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Gene Mapping via the Ancestral Recombination Graph</span>. <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DLarribe%26rft.auinit1%253DF.%26rft.volume%253D62%26rft.issue%253D2%26rft.spage%253D215%26rft.epage%253D229%26rft.atitle%253DGene%2Bmapping%2Bvia%2Bthe%2Bancestral%2Brecombination%2Bgraph.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.2002.1601%26rft_id%253Dinfo%253Apmid%252F12167358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/tpbi.2002.1601&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12167358&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F175.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177739700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lettre</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirschhorn</span>  <span class="cit-name-given-names">J. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic model testing and statistical power in population-based association studies of quantitative traits</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">362</span>: <span class="cit-fpage">358</span>–<span class="cit-lpage">362</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D362%26rft.spage%253D358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.16" data-doi="10.1093/bioinformatics/bti732"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Haplotype-based linkage disequilibrium mapping via direct data mining</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">4384</span>–<span class="cit-lpage">4393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti732%26rft_id%253Dinfo%253Apmid%252F16249262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjEvMjQvNDM4NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8xNzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The power of association studies to detect the contribution of candidate genetic loci to variation in complex traits</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">720</span>–<span class="cit-lpage">731</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DLong%26rft.auinit1%253DA.%2BD.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D720%26rft.epage%253D731%26rft.atitle%253DThe%2BPower%2Bof%2BAssociation%2BStudies%2Bto%2BDetect%2Bthe%2BContribution%2Bof%2BCandidate%2BGenetic%2BLoci%2Bto%2BVariation%2Bin%2BComplex%2BTraits%26rft_id%253Dinfo%253Apmid%252F10447507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjkvOC83MjAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMTc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.18" data-doi="10.1186/1471-2105-7-454"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mailund</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Besenbacher</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schierup</span>  <span class="cit-name-given-names">M. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Whole genome association mapping by incompatibilities and local perfect phylogenies</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DMailund%26rft.auinit1%253DT.%26rft.volume%253D7%26rft.spage%253D454%26rft.epage%253D454%26rft.atitle%253DWhole%2Bgenome%2Bassociation%2Bmapping%2Bby%2Bincompatibilities%2Band%2Blocal%2Bperfect%2Bphylogenies.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-7-454%26rft_id%253Dinfo%253Apmid%252F17042942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-7-454&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17042942&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F175.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.19" data-doi="10.1023/A:1021639901679"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mayeda</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hofstetter</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">A QTL for the genetic variance in free-running period and level of locomotor activity between inbred strains of mice</span>. <abbr class="cit-jnl-abbrev">Behav. Genet.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehavior%2Bgenetics%26rft.stitle%253DBehav%2BGenet%26rft.aulast%253DMayeda%26rft.auinit1%253DA.%2BR.%26rft.volume%253D29%26rft.issue%253D3%26rft.spage%253D171%26rft.epage%253D176%26rft.atitle%253DA%2BQTL%2Bfor%2Bthe%2Bgenetic%2Bvariance%2Bin%2Bfree-running%2Bperiod%2Band%2Blevel%2Bof%2Blocomotor%2Bactivity%2Bbetween%2Binbred%2Bstrains%2Bof%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1021639901679%26rft_id%253Dinfo%253Apmid%252F10547923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1021639901679&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10547923&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F175.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083260600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.20" data-doi="10.1186/1471-2105-7-61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McClurg</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pletcher</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiltshire</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">A. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Comparative analysis of haplotype association mapping algorithms</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DMcClurg%26rft.auinit1%253DP.%26rft.volume%253D7%26rft.spage%253D61%26rft.epage%253D61%26rft.atitle%253DComparative%2Banalysis%2Bof%2Bhaplotype%2Bassociation%2Bmapping%2Balgorithms.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-7-61%26rft_id%253Dinfo%253Apmid%252F16466585%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-7-61&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16466585&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F175.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.21" data-doi="10.1086/508901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Minichiello</span>  <span class="cit-name-given-names">M. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mapping trait loci by use of inferred ancestral recombination graphs</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">910</span>–<span class="cit-lpage">922</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DMinichiello%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D79%26rft.issue%253D5%26rft.spage%253D910%26rft.epage%253D922%26rft.atitle%253DMapping%2Btrait%2Bloci%2Bby%2Buse%2Bof%2Binferred%2Bancestral%2Brecombination%2Bgraphs.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F508901%26rft_id%253Dinfo%253Apmid%252F17033967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/508901&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17033967&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F175.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241667400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.22" data-doi="10.1086/339271"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whittaker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balding</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Fine-scale mapping of disease loci via shattered coalescent modeling of genealogies</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">686</span>–<span class="cit-lpage">707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DMorris%26rft.auinit1%253DA.%2BP.%26rft.volume%253D70%26rft.issue%253D3%26rft.spage%253D686%26rft.epage%253D707%26rft.atitle%253DFine-scale%2Bmapping%2Bof%2Bdisease%2Bloci%2Bvia%2Bshattered%2Bcoalescent%2Bmodeling%2Bof%2Bgenealogies.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F339271%26rft_id%253Dinfo%253Apmid%252F11836651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/339271&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11836651&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F175.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173880000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.23" data-doi="10.1046/j.1469-1809.2002.00131.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Onkamo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ollikainen</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sevon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toivonen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mannila</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kere</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Association analysis for quantitative traits by data mining: QHPM</span>. <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">429</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DOnkamo%26rft.auinit1%253DP.%26rft.volume%253D66%26rft.issue%253DPt%2B5-6%26rft.spage%253D419%26rft.epage%253D429%26rft.atitle%253DAssociation%2Banalysis%2Bfor%2Bquantitative%2Btraits%2Bby%2Bdata%2Bmining%253A%2BQHPM.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1469-1809.2002.00131.x%26rft_id%253Dinfo%253Apmid%252F12485474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1469-1809.2002.00131.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12485474&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F175.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pardo-Manuel De Villena</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Threadgill</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">TreeQA: quantitative genome-wide association mapping using local perfect phylogeny trees</span>. <abbr class="cit-jnl-abbrev">Pac. Symp. Biocomput.</abbr> <span class="cit-vol">426:</span> <span class="cit-fpage">415</span>–<span class="cit-lpage">426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPac.%2BSymp.%2BBiocomput.%26rft.volume%253D426%253A%26rft.spage%253D415%26rft.atitle%253DTREEQA%253A%2BQUANTITATIVE%2BGENOME-WIDE%2BASSOCIATION%2BMAPPING%2BUSING%2BLOCAL%2BPERFECT%2BPHYLOGENY%2BTREES%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-2.2.175.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pardo-Manuel De Villena</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Threadgill</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-comment">2008 Quantitative Association Analysis Using Tree Hierarchies. In <em>Proceedings of the 2008 Eighth IEEE International Conference on Data Mining</em>, Washington, DC, pp. 971–976. IEEE Computer Society</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.26" data-doi="10.1038/ng1816"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pe'er</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Bakker</span>  <span class="cit-name-given-names">P. I. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yelensky</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evaluating and improving power in whole-genome association studies using fixed marker sets</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">663</span>–<span class="cit-lpage">670</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DPe%2527er%26rft.auinit1%253DI.%26rft.volume%253D38%26rft.issue%253D6%26rft.spage%253D663%26rft.epage%253D667%26rft.atitle%253DEvaluating%2Band%2Bimproving%2Bpower%2Bin%2Bwhole-genome%2Bassociation%2Bstudies%2Busing%2Bfixed%2Bmarker%2Bsets.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1816%26rft_id%253Dinfo%253Apmid%252F16715096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1816&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16715096&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F175.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237954800020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.27" data-doi="10.1038/ng1847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plenge</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinblatt</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shadick</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Principal components analysis corrects for stratification in genome-wide association studies</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">904</span>–<span class="cit-lpage">909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTo%26rft.auinit1%253DM.%2BD.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D904%26rft.epage%253D909%26rft.atitle%253DPrincipal%2Bcomponents%2Banalysis%2Bcorrects%2Bfor%2Bstratification%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1847%26rft_id%253Dinfo%253Apmid%252F16862161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16862161&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F175.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.28" data-doi="10.1007/s00335-007-9045-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pardo-Manuel de Villena</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Threadgill</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The polymorphism architecture of mouse genetic resources elucidated using genome-wide resequencing data: implications for QTL discovery and systems genetics</span>. <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">473</span>–<span class="cit-lpage">481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DRoberts%26rft.auinit1%253DA.%26rft.volume%253D18%26rft.issue%253D6-7%26rft.spage%253D473%26rft.epage%253D481%26rft.atitle%253DThe%2Bpolymorphism%2Barchitecture%2Bof%2Bmouse%2Bgenetic%2Bresources%2Belucidated%2Busing%2Bgenome-wide%2Bresequencing%2Bdata%253A%2Bimplications%2Bfor%2BQTL%2Bdiscovery%2Band%2Bsystems%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-007-9045-1%26rft_id%253Dinfo%253Apmid%252F17674098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-007-9045-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17674098&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F175.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249125700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.29" data-doi="10.1086/502802"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">629</span>–<span class="cit-lpage">644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DScheet%26rft.auinit1%253DP.%26rft.volume%253D78%26rft.issue%253D4%26rft.spage%253D629%26rft.epage%253D644%26rft.atitle%253DA%2Bfast%2Band%2Bflexible%2Bstatistical%2Bmodel%2Bfor%2Blarge-scale%2Bpopulation%2Bgenotype%2Bdata%253A%2Bapplications%2Bto%2Binferring%2Bmissing%2Bgenotypes%2Band%2Bhaplotypic%2Bphase.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F502802%26rft_id%253Dinfo%253Apmid%252F16532393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/502802&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16532393&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F175.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236755900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.30" data-doi="10.1109/TCBB.2006.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sevon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toivonen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ollikainen</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">TreeDT: tree pattern mining for gene mapping</span>. <abbr class="cit-jnl-abbrev">IEEE/ACM Trans. Comput. Biol. Bioinf.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">174</span>–<span class="cit-lpage">1085</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%252FACM%2BTrans.%2BComput.%2BBiol.%2BBioinf.%26rft.volume%253D3%26rft.spage%253D174%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTCBB.2006.28%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/TCBB.2006.28&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.31" data-doi="10.1534/genetics.111.132597"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">High-resolution genetic mapping using the mouse diversity outbred population</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSvenson%26rft.auinit1%253DK.%2BL.%26rft.volume%253D190%26rft.issue%253D2%26rft.spage%253D437%26rft.epage%253D447%26rft.atitle%253DHigh-Resolution%2BGenetic%2BMapping%2BUsing%2Bthe%2BMouse%2BDiversity%2BOutbred%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132597%26rft_id%253Dinfo%253Apmid%252F22345611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNDM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8yLzE3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.2.175.32"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">The Complex Trait Consortium</span>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The Collaborative Cross, a community resource for the genetic analysis of complex traits</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1133</span>–<span class="cit-lpage">1137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D36%26rft.spage%253D1133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-2.2.175.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-source">Statistical Methods in Genetic Epidemiology</span>, <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.34" data-doi="10.1038/ng1840"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solberg</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gauguier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burnett</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klenerman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome-wide genetic association of complex traits in heterogeneous stock mice</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">879</span>–<span class="cit-lpage">887</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DValdar%26rft.auinit1%253DW.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D879%26rft.epage%253D887%26rft.atitle%253DGenome-wide%2Bgenetic%2Bassociation%2Bof%2Bcomplex%2Btraits%2Bin%2Bheterogeneous%2Bstock%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1840%26rft_id%253Dinfo%253Apmid%252F16832355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1840&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16832355&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F175.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.35" data-doi="10.1086/497434"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheffield</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A constrained-likelihood approach to marker-trait association studies</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">768</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DK.%26rft.volume%253D77%26rft.issue%253D5%26rft.spage%253D768%26rft.epage%253D780%26rft.atitle%253DA%2Bconstrained-likelihood%2Bapproach%2Bto%2Bmarker-trait%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F497434%26rft_id%253Dinfo%253Apmid%252F16252237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/497434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16252237&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F175.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232569600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-book" id="cit-2.2.175.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Westfall</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">S. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-source">Resampling-Based Multiple Testing: Examples and Methods for P-Value Adjustment</span>, <span class="cit-publ-name">Wiley</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.37" data-doi="10.1038/nmeth.1359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hutchins</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szatkiewicz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A customized and versatile high-density genotyping array for the mouse</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">663</span>–<span class="cit-lpage">666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253D663%26rft.epage%253D666%26rft.atitle%253DA%2Bcustomized%2Band%2Bversatile%2Bhigh-density%2Bgenotyping%2Barray%2Bfor%2Bthe%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1359%26rft_id%253Dinfo%253Apmid%252F19668205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1359&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19668205&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F175.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269382900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.38" data-doi="10.1038/ng.847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buus</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Subspecific origin and haplotype diversity in the laboratory mouse</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">648</span>–<span class="cit-lpage">655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D43%26rft.issue%253D7%26rft.spage%253D648%26rft.epage%253D655%26rft.atitle%253DSubspecific%2Borigin%2Band%2Bhaplotype%2Bdiversity%2Bin%2Bthe%2Blaboratory%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.847%26rft_id%253Dinfo%253Apmid%252F21623374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21623374&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F175.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.175.39" data-doi="10.1534/genetics.104.031799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zöllner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Coalescent-based association mapping and fine mapping of complex trait loci</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1071</span>–<span class="cit-lpage">1092</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.031799%26rft_id%253Dinfo%253Apmid%252F15489534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS8yLzEwNzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMTc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/2/175.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/2/167" title="Genome-Wide Association Mapping of Quantitative Traits in Outbred Mice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/2/191" title="Accelerating the Inbreeding of Multi-Parental Recombinant Inbred Lines Generated By Sibling Matings" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/2"><span>Volume 2 Issue 2, February 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/2.cover.gif?itok=i7gjzosG" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/2/175" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UKqXCh3fHIvqYAEff6K1hXf8jC8SDrx9vcus9rLFFmA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/mouse-genetic-resources" class="" data-icon-position="" data-hide-link-title="0">Mouse Genetic Resources</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/2/175" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/2/175" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/2/175" class="active" data-icon-position="" data-hide-link-title="0">HTreeQA: Using Semi-Perfect Phylogeny Trees in Quantitative Trait Loci Study on Genotype Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459239" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-1NelaGScfegjioJ_tgg06-dqUmJluoWk6QkRbp5geyU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/2/175.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/2/175" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-xU75daxHTrC_vdL5G_-XJGc3qmXPp-AeUQjPn2hPQJ4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/2/175" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/2/175.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459239" id="node459239--2" data-pisa="ggg;2/2/175" data-pisa-master="ggg;2/2/175" data-apath="/ggg/2/2/175.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">HTreeQA: Using Semi-Perfect Phylogeny Trees in Quantitative Trait Loci Study on Genotype Data</h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhaojun Zhang</span>, <span class="highwire-citation-author" data-delta="1">Xiang Zhang</span> and <span class="highwire-citation-author" data-delta="2">Wei Wang</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">175-189;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001768 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459239--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhaojun Zhang</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Computer Science, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhaojun%2BZhang" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhaojun%2BZhang" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Xiang Zhang</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Electrical Engineering and Computer Science, Case Western Reserve University, Cleveland, Ohio 44106</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiang%2BZhang" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiang%2BZhang" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Wei Wang</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Computer Science, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BWang" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWei%2BWang" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:weiwang@cs.unc.edu" class="" data-icon-position="" data-hide-link-title="0">weiwang@cs.unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/2/175" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/2/175&amp;title=HTreeQA%3A%20Using%20Semi-Perfect%20Phylogeny%20Trees%20in%20Quantitative%20Trait%20Loci%20Study%20on%20Genotype%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/2/175&amp;title=HTreeQA%3A%20Using%20Semi-Perfect%20Phylogeny%20Trees%20in%20Quantitative%20Trait%20Loci%20Study%20on%20Genotype%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/2/175&amp;title=HTreeQA%3A%20Using%20Semi-Perfect%20Phylogeny%20Trees%20in%20Quantitative%20Trait%20Loci%20Study%20on%20Genotype%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/2/175&amp;text=HTreeQA%3A%20Using%20Semi-Perfect%20Phylogeny%20Trees%20in%20Quantitative%20Trait%20Loci%20Study%20on%20Genotype%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/2/175&amp;title=HTreeQA%3A%20Using%20Semi-Perfect%20Phylogeny%20Trees%20in%20Quantitative%20Trait%20Loci%20Study%20on%20Genotype%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/2/175&amp;t=HTreeQA%3A%20Using%20Semi-Perfect%20Phylogeny%20Trees%20in%20Quantitative%20Trait%20Loci%20Study%20on%20Genotype%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/2/175&amp;title=HTreeQA%3A%20Using%20Semi-Perfect%20Phylogeny%20Trees%20in%20Quantitative%20Trait%20Loci%20Study%20on%20Genotype%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/2/175&amp;title=HTreeQA%3A%20Using%20Semi-Perfect%20Phylogeny%20Trees%20in%20Quantitative%20Trait%20Loci%20Study%20on%20Genotype%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459239/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459239" id="node459239--2" data-pisa="ggg;2/2/175" data-pisa-master="ggg;2/2/175" data-apath="/ggg/2/2/175.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">HTreeQA: Using Semi-Perfect Phylogeny Trees in Quantitative Trait Loci Study on Genotype Data</h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhaojun Zhang</span>, <span class="highwire-citation-author" data-delta="1">Xiang Zhang</span> and <span class="highwire-citation-author" data-delta="2">Wei Wang</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">175-189;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001768 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459239--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhaojun Zhang</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Computer Science, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhaojun%2BZhang" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhaojun%2BZhang" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Xiang Zhang</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Electrical Engineering and Computer Science, Case Western Reserve University, Cleveland, Ohio 44106</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiang%2BZhang" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiang%2BZhang" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Wei Wang</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Computer Science, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BWang" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWei%2BWang" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:weiwang@cs.unc.edu" class="" data-icon-position="" data-hide-link-title="0">weiwang@cs.unc.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=175&amp;rft.epage=189&amp;rft.atitle=HTreeQA%3A%20Using%20Semi-Perfect%20Phylogeny%20Trees%20in%20Quantitative%20Trait%20Loci%20Study%20on%20Genotype%20Data&amp;rft.volume=2&amp;rft.issue=2&amp;rft.date=2012-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zhang%2C+Zhaojun&amp;rft.au=Zhang%2C+Xiang&amp;rft.au=Wang%2C+Wei"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459239/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459239/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459239/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459239/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459239/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459239/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459239/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459239/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459239/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459239/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459239/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459239/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459943" id="node459943" data-pisa="ggg;4/8/1429" data-pisa-master="ggg;g3.114.011759" data-apath="/ggg/4/8/1429.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/4/8/1429" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genomic Profiling of Collaborative Cross Founder Mice Infected with Respiratory Viruses Reveals Novel Transcripts and Infection-Related Strain-Specific Gene and Isoform Expression</span></a></div>
</cite>
</div></li><li class="last even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459240" id="node459240" data-pisa="ggg;2/2/203" data-pisa-master="ggg;2/2/203" data-apath="/ggg/2/2/203.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/2/2/203" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Transcriptome Atlases of Mouse Brain Reveals Differential Expression Across Brain Regions and Genetic Backgrounds</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Mouse%20Genetic%20Resources" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Mouse Genetic Resources</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/2/175" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials</a></li><li class="odd"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="odd"><a href="#sec-24" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix 1</a></li><li class="even"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix 2</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/2/175.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/2/175.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/2/175.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/2/175.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2019 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
  </body>
</html>
