<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/2/261" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/2/261.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/2/261.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/2/261.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2.cover.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/2/261.atom" />
<meta name="HW.pisa" content="ggg;2/2/261" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Whole-Genome Sequencing of Sordaria macrospora Mutants Identifies Developmental Genes" />
<meta name="DC.Identifier" content="10.1534/g3.111.001479" />
<meta name="DC.Date" content="2012-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Nowrousian et al. by the Genetics Society of America. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The study of mutants to elucidate gene functions has a long and successful history; however, to discover causative mutations in mutants that were generated by random mutagenesis often takes years of laboratory work and requires previously generated genetic and/or physical markers, or resources like DNA libraries for complementation. Here, we present an alternative method to identify defective genes in developmental mutants of the filamentous fungus Sordaria macrospora through Illumina/Solexa whole-genome sequencing. We sequenced pooled DNA from progeny of crosses of three mutants and the wild type and were able to pinpoint the causative mutations in the mutant strains through bioinformatics analysis. One mutant is a spore color mutant, and the mutated gene encodes a melanin biosynthesis enzyme. The causative mutation is a G to A change in the first base of an intron, leading to a splice defect. The second mutant carries an allelic mutation in the pro41 gene encoding a protein essential for sexual development. In the mutant, we detected a complex pattern of deletion/rearrangements at the pro41 locus. In the third mutant, a point mutation in the stop codon of a transcription factor-encoding gene leads to the production of immature fruiting bodies. For all mutants, transformation with a wild type-copy of the affected gene restored the wild-type phenotype. Our data demonstrate that whole-genome sequencing of mutant strains is a rapid method to identify developmental genes in an organism that can be genetically crossed and where a reference genome sequence is available, even without prior mapping information." />
<meta name="DC.Contributor" content="Minou Nowrousian" />
<meta name="DC.Contributor" content="Ines Teichert" />
<meta name="DC.Contributor" content="Sandra Masloff" />
<meta name="DC.Contributor" content="Ulrich Kück" />
<meta name="article:published_time" content="2012-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Whole-Genome Sequencing of Sordaria macrospora Mutants Identifies Developmental Genes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The study of mutants to elucidate gene functions has a long and successful history; however, to discover causative mutations in mutants that were generated by random mutagenesis often takes years of laboratory work and requires previously generated genetic and/or physical markers, or resources like DNA libraries for complementation. Here, we present an alternative method to identify defective genes in developmental mutants of the filamentous fungus &lt;i&gt;Sordaria macrospora&lt;/i&gt; through Illumina/Solexa whole-genome sequencing. We sequenced pooled DNA from progeny of crosses of three mutants and the wild type and were able to pinpoint the causative mutations in the mutant strains through bioinformatics analysis. One mutant is a spore color mutant, and the mutated gene encodes a melanin biosynthesis enzyme. The causative mutation is a G to A change in the first base of an intron, leading to a splice defect. The second mutant carries an allelic mutation in the &lt;i&gt;pro41&lt;/i&gt; gene encoding a protein essential for sexual development. In the mutant, we detected a complex pattern of deletion/rearrangements at the &lt;i&gt;pro41&lt;/i&gt; locus. In the third mutant, a point mutation in the stop codon of a transcription factor-encoding gene leads to the production of immature fruiting bodies. For all mutants, transformation with a wild type-copy of the affected gene restored the wild-type phenotype. Our data demonstrate that whole-genome sequencing of mutant strains is a rapid method to identify developmental genes in an organism that can be genetically crossed and where a reference genome sequence is available, even without prior mapping information.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/02/01" />
<meta name="citation_mjid" content="ggg;2/2/261" />
<meta name="citation_id" content="2/2/261" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/2/261" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/2/261.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/2/261.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/2/261.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.001479" />
<meta name="citation_pmid" content="22384404" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="2" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="261" />
<meta name="citation_lastpage" content="270" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Minou Nowrousian" />
<meta name="citation_author_institution" content="Lehrstuhl für Allgemeine und Molekulare Botanik, Ruhr-Universität Bochum, 44780 Bochum, Germany" />
<meta name="citation_author" content="Ines Teichert" />
<meta name="citation_author_institution" content="Lehrstuhl für Allgemeine und Molekulare Botanik, Ruhr-Universität Bochum, 44780 Bochum, Germany" />
<meta name="citation_author" content="Sandra Masloff" />
<meta name="citation_author_institution" content="Lehrstuhl für Allgemeine und Molekulare Botanik, Ruhr-Universität Bochum, 44780 Bochum, Germany" />
<meta name="citation_author" content="Ulrich Kück" />
<meta name="citation_author_institution" content="Lehrstuhl für Allgemeine und Molekulare Botanik, Ruhr-Universität Bochum, 44780 Bochum, Germany" />
<meta name="citation_author_email" content="ulrich.kueck@rub.de" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Balzer;citation_author=K. Malde;citation_author=I. Jonassen;citation_title=Systematic exploration of error sources in pyrosequencing flowgram data;citation_volume=27;citation_year=2011;citation_pmid=21685085;citation_doi=10.1093/bioinformatics/btr251" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=E. Bautz;citation_author=E. Freese;citation_title=ON THE MUTAGENIC EFFECT OF ALKYLATING AGENTS;citation_pages=1585-1594;citation_volume=46;citation_year=1960;citation_issue=12;citation_pmid=16590785;citation_doi=10.1073/pnas.46.12.1585" />
<meta name="citation_reference" content="citation_journal_title=Annual review of phytopathology;citation_journal_abbrev=Annu Rev Phytopathol;citation_author=A. A. Bell;citation_author=M. H. Wheeler;citation_title=Biosynthesis and Functions of Fungal Melanins;citation_pages=411-451;citation_volume=24;citation_year=1986;citation_issue=1;citation_doi=10.1146/annurev.py.24.090186.002211" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=S. Bloemendal;citation_author=K. M. Lord;citation_author=C. Rech;citation_author=B. Hoff;citation_author=I. Engh;citation_title=A mutant defective in sexual development produces aseptate ascogonia;citation_volume=9;citation_year=2010;citation_pmid=20952581;citation_doi=10.1128/EC.00186-10" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=C. Borghouts;citation_author=E. Kimpel;citation_author=H. D. Osiewacz;citation_title=Mitochondrial DNA rearrangements of Podospora anserina are under the control of the nuclear gene grisea;citation_pages=10768-10773;citation_volume=94;citation_year=1997;citation_issue=20;citation_pmid=9380708;citation_doi=10.1073/pnas.94.20.10768" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. U S A.;citation_author=H. V. Colot;citation_author=G. Park;citation_author=G. E. Turner;citation_author=C. Ringelberg;citation_author=C. M. Crew;citation_title=A high-throughput gene knockout procedure for Neurospora reveals functions for multiple transcription factors;citation_volume=103;citation_year=2006;citation_pmid=16801547;citation_doi=10.1073/pnas.0601456103" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. F. Conrad;citation_author=C. Bird;citation_author=B. Blackburne;citation_author=S. Lindsay;citation_author=L. Mamanova;citation_title=Mutation spectrum revealed by breakpoint sequencing of human germline CNVs.;citation_pages=385-391;citation_volume=42;citation_year=2010;citation_issue=5;citation_pmid=20364136;citation_doi=10.1038/ng.564" />
<meta name="citation_reference" content="citation_journal_title=Annals of the New York Academy of Sciences;citation_journal_abbrev=Ann. N. Y. Acad. Sci.;citation_author=J. W. Drake;citation_title=The Distribution of Rates of Spontaneous Mutation over Viruses, Prokaryotes, and Eukaryotes;citation_pages=100-107;citation_volume=870;citation_year=1999;citation_issue=1;citation_pmid=10415476;citation_doi=10.1111/j.1749-6632.1999.tb08870.x" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Lett.;citation_author=I. Engh;citation_author=M. Nowrousian;citation_author=U. Kück;citation_title=Regulation of melain biosynthesis via the dihydroxynaphtalene pathway is dependent on sexual development in the ascomycete Sordaria macrospora;citation_volume=275;citation_year=2007;citation_pmid=17681008;citation_doi=10.1111/j.1574-6968.2007.00867.x" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=I. Engh;citation_author=C. Würtz;citation_author=K. Witzel-Schlömp;citation_author=H. Y. Zhang;citation_author=B. Hoff;citation_title=The WW domain protein PRO40 is required for fungal fertility and associates with Woronin bodies;citation_volume=6;citation_year=2007;citation_pmid=17351077;citation_doi=10.1128/EC.00269-06" />
<meta name="citation_reference" content="citation_journal_title=European journal of cell biology;citation_journal_abbrev=Eur J Cell Biol;citation_author=I. Engh;citation_author=M. Nowrousian;citation_author=U. Kück;citation_title=Sordaria macrospora, a model organism to study fungal cellular development.;citation_pages=864-872;citation_volume=89;citation_year=2010;citation_issue=12;citation_pmid=20739093;citation_doi=10.1016/j.ejcb.2010.07.002" />
<meta name="citation_reference" content="citation_title=Cryptogams—Cyanobacteria, Algae, Fungi, Lichens;citation_author=K. Esser;citation_year=1982" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Flibotte;citation_author=M. L. Edgley;citation_author=I. Chaudhry;citation_author=J. Taylor;citation_author=S. E. Neil;citation_title=Whole-genome profiling of mutagenesis in Caenorhabditis elegans;citation_pages=431-441;citation_volume=185;citation_year=2010;citation_pmid=20439774;citation_doi=10.1534/genetics.110.116616" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=K. H. Han;citation_author=K. Y. Han;citation_author=J. H. Yu;citation_author=K. S. Chae;citation_author=K. Y. Jahng;citation_title=The nsdD gene encodes a putative GATA-type transcription factor necessary for sexual development of Aspergillus nidulans.;citation_pages=299-309;citation_volume=41;citation_year=2001;citation_issue=2;citation_pmid=11489119;citation_doi=10.1046/j.1365-2958.2001.02472.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. V. Irvine;citation_author=D. B. Goto;citation_author=M. W. Vaughn;citation_author=Y. Nakaseko;citation_author=W. R. Mccombie;citation_title=Mapping epigenetic mutations in fission yeast using whole-genome next-generation sequencing;citation_volume=19;citation_year=2009;citation_pmid=19423874;citation_doi=10.1101/gr.089318.108" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=V. Klix;citation_author=M. Nowrousian;citation_author=C. Ringelberg;citation_author=J. J. Loros;citation_author=J. C. Dunlap;citation_title=Functional characterization of MAT1–1-specific mating-type genes in the homothallic ascomycete Sordaria macrospora provides new insights into essential and non-essential sexual regulators;citation_volume=9;citation_year=2010;citation_pmid=20435701;citation_doi=10.1128/EC.00019-10" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. O. Korbel;citation_author=A. E. Urban;citation_author=J. P. Affourtit;citation_author=B. Godwin;citation_author=F. Grubert;citation_title=Paired-end mapping reveals extensive structural variation in the human genome;citation_volume=318;citation_year=2007;citation_pmid=17901297;citation_doi=10.1126/science.1149504" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=U. Kück;citation_title=A Sordaria macrospora mutant lacking the leu1 gene shows a developmental arrest during fruiting body formation.;citation_pages=307-315;citation_volume=274;citation_year=2005;citation_issue=3;citation_pmid=16133166;citation_doi=10.1007/s00438-005-0021-8" />
<meta name="citation_reference" content="citation_title=The Mycota XV, Physiology and Genetics;citation_title=Sordaria macrospora, a model system for fungal development;citation_author=U. Kück;citation_author=S. Pöggeler;citation_author=M. Nowrousian;citation_author=N. Nolting;citation_author=I. Engh;citation_pages=17-39;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. A. Lee;citation_author=C. M. B. Carvalho;citation_author=J. R. Lupski;citation_title=A DNA replication mechanism for generating nonrecurrent rearrangements associated with genomic disorders.;citation_pages=1235-1247;citation_volume=131;citation_year=2007;citation_issue=7;citation_pmid=18160035;citation_doi=10.1016/j.cell.2007.11.037" />
<meta name="citation_reference" content="citation_journal_title=Bioinf.;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows–Wheeler transform;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinf.;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=New England Journal of Medicine;citation_journal_abbrev=NEJM;citation_author=J. R. Lupski;citation_author=J. G. Reid;citation_author=C. Gonzaga-Jauregui;citation_author=D. Rio Deiros;citation_author=D. C. Chen;citation_title=Whole-genome sequencing in a patient with Charcot-Marie-Tooth neuropathy.;citation_pages=1181-1191;citation_volume=362;citation_year=2010;citation_issue=13;citation_pmid=20220177;citation_doi=10.1056/NEJMoa0908094" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Masloff;citation_author=S. Pöggeler;citation_author=U. Kück;citation_title=The pro1+ gene from Sordaria macrospora encodes a C6 zinc finger transcription factor required for fruiting body development;citation_pages=191-199;citation_volume=152;citation_year=1999;citation_pmid=10224253" />
<meta name="citation_reference" content="citation_journal_title=G3: Genes, Genomes, Genetics;citation_author=K. McCluskey;citation_author=A. E. Wiest;citation_author=I. V. Grigoriev;citation_author=A. Lipzen;citation_author=J. Martin;citation_title=Rediscovery by whole genome sequencing: classical mutations and genome polymorphisms in Neurospora crassa;citation_pages=303-316;citation_volume=1;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. L. Metzker;citation_title=Sequencing technologies - the next generation.;citation_pages=31-46;citation_volume=11;citation_year=2010;citation_issue=1;citation_pmid=19997069;citation_doi=10.1038/nrg2626" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=M. Nowrousian;citation_title=Next-generation sequencing techniques for eukaryotic microorganisms: sequencing-based solutions to biological problems;citation_volume=9;citation_year=2010;citation_pmid=20601439;citation_doi=10.1128/EC.00123-10" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=M. Nowrousian;citation_author=S. Masloff;citation_author=S. Pöggeler;citation_author=U. Kück;citation_title=Cell Differentiation during Sexual Development of the Fungus Sordaria macrospora Requires ATP Citrate Lyase Activity;citation_pages=450-460;citation_volume=19;citation_year=1999;citation_issue=1;citation_pmid=9858569;citation_doi=10.1128/MCB.19.1.450" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=M. Nowrousian;citation_author=C. Ringelberg;citation_author=J. C. Dunlap;citation_author=J. J. Loros;citation_author=U. Kück;citation_title=Cross-species microarray hybridization to identify developmentally regulated genes in the filamentous fungus Sordaria macrospora.;citation_pages=137-149;citation_volume=273;citation_year=2005;citation_issue=2;citation_pmid=15778868;citation_doi=10.1007/s00438-005-1118-9" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=M. Nowrousian;citation_author=S. Frank;citation_author=S. Koers;citation_author=P. Strauch;citation_author=T. Weitner;citation_title=The novel ER membrane protein PRO41 is essential for sexual development in the filamentous fungus Sordaria macrospora.;citation_pages=923-937;citation_volume=64;citation_year=2007;citation_issue=4;citation_pmid=17501918;citation_doi=10.1111/j.1365-2958.2007.05694.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. Nowrousian;citation_author=J. E. Stajich;citation_author=M. Chu;citation_author=I. Engh;citation_author=E. Espagne;citation_title=De novo assembly of a 40 Mb eukaryotic genome from short sequence reads: Sordaria macrospora, a model organism for fungal morphogenesis.;citation_pages=e1000891-e1000891;citation_volume=6;citation_year=2010;citation_issue=4;citation_pmid=20386741;citation_doi=10.1371/journal.pgen.1000891" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=S. Pöggeler;citation_author=U. Kück;citation_title=A WD40 Repeat Protein Regulates Fungal Cell Differentiation and Can Be Replaced Functionally by the Mammalian Homologue Striatin;citation_pages=232-240;citation_volume=3;citation_year=2004;citation_issue=1;citation_pmid=14871953;citation_doi=10.1128/EC.3.1.232-240.2004" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=S. Pöggeler;citation_author=U. Kück;citation_title=Highly efficient generation of signal transduction knockout mutants using a fungal strain deficient in the mammalian ku70 ortholog;citation_pages=1-10;citation_volume=378;citation_year=2006;citation_pmid=16814491;citation_doi=10.1016/j.gene.2006.03.020" />
<meta name="citation_reference" content="citation_journal_title=J. Microbiol. Methods;citation_author=S. Pöggeler;citation_author=M. Nowrousian;citation_author=S. Jacobsen;citation_author=U. Kück;citation_title=An efficient procedure to isolate fungal genes from an indexed cosmid library;citation_pages=49-61;citation_volume=29;citation_year=1997;citation_doi=10.1016/S0167-7012(97)00018-3" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=S. Pöggeler;citation_author=S. Masloff;citation_author=B. Hoff;citation_author=S. Mayrhofer;citation_author=U. Kück;citation_title=Versatile EGFP reporter plasmids for cellular localization of recombinant gene products in filamentous fungi.;citation_pages=54-61;citation_volume=43;citation_year=2003;citation_issue=1;citation_pmid=12684845;citation_doi=10.1007/s00294-003-0370-y" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=K. R. Pomraning;citation_author=K. M. Smith;citation_author=M. Freitag;citation_title=Bulk segregant analysis followed by high-throughput sequencing reveals the Neurospora cell cycle gene, ndc-1, to be allelic with the gene for ornithine decarboxylase, spe-1;citation_volume=10;citation_year=2011;citation_pmid=21515825;citation_doi=10.1128/EC.00016-11" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=X. S. Puente;citation_author=M. Pinyol;citation_author=V. Quesada;citation_author=L. Conde;citation_author=G. R. Ordonez;citation_title=Whole-genome sequencing identifies recurrent mutations in chronic lymphocytic leukaemia.;citation_pages=101-105;citation_volume=475;citation_year=2011;citation_issue=7354;citation_pmid=21642962;citation_doi=10.1038/nature10113" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. C. Roach;citation_author=G. Glusman;citation_author=A. F. A. Smit;citation_author=C. D. Huff;citation_author=R. Hubley;citation_title=Analysis of genetic inheritance in a family quartet by whole-genome sequencing;citation_volume=328;citation_year=2010;citation_pmid=20220176;citation_doi=10.1126/science.1186802" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=S. Sarin;citation_author=S. Prabhu;citation_author=M. M. O'meara;citation_author=I. Pe'er;citation_author=O. Hobert;citation_title=Caenorhabditis elegans mutant allele identification by whole-genome sequencing.;citation_pages=865-867;citation_volume=5;citation_year=2008;citation_issue=10;citation_pmid=18677319;citation_doi=10.1038/nmeth.1249" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=K. Schneeberger;citation_author=S. Ossowski;citation_author=C. Lanz;citation_author=T. Juul;citation_author=A. H. Petersen;citation_title=SHOREmap: simultaneous mapping and mutation identification by deep sequencing.;citation_pages=550-551;citation_volume=6;citation_year=2009;citation_issue=8;citation_pmid=19644454;citation_doi=10.1038/nmeth0809-550" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. Shendure;citation_author=H. Li;citation_title=Next-generation DNA sequencing.;citation_pages=1135-1145;citation_volume=26;citation_year=2008;citation_issue=10;citation_pmid=18846087;citation_doi=10.1038/nbt1486" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=E. Szewczyk;citation_author=S. Krappmann;citation_title=Conserved regulators of mating are essential for Aspergillus fumigatus cleistothecium formation;citation_volume=9;citation_year=2010;citation_pmid=20348388;citation_doi=10.1128/EC.00375-09" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=M. Walz;citation_author=U. Kück;citation_title=Transformation of Sordaria macrospora to hygromycin B resistance: characterization of transformants by electrophoretic karyotyping and tetrad analysis.;citation_pages=88-95;citation_volume=29;citation_year=1995;citation_issue=1;citation_pmid=8595663;citation_doi=10.1007/BF00313198" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=F. Zhang;citation_author=M. Khajavi;citation_author=A. M. Connolly;citation_author=C. F. Towne;citation_author=S. D. Batish;citation_title=The DNA replication FoSTeS/MMBIR mechanism can generate genomic, genic and exonic complex rearrangements in humans.;citation_pages=849-853;citation_volume=41;citation_year=2009;citation_issue=7;citation_pmid=19543269;citation_doi=10.1038/ng.399" />
<meta name="citation_reference" content="citation_title=The Mycota I. Growth, Differentiation and Sexuality;citation_title=Meiosis in mycelial fungi;citation_author=D. Zickler;citation_pages=415-438;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Zuryn;citation_author=S. Le Gras;citation_author=K. Jamet;citation_author=S. Jarriault;citation_title=A strategy for direct mapping and identification of mutations by whole-genome sequencing;citation_pages=427-430;citation_volume=186;citation_year=2010;citation_pmid=20610404;citation_doi=10.1534/genetics.110.119230" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Whole-Genome Sequencing of Sordaria macrospora Mutants Identifies Developmental Genes" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2.cover.gif" />
<meta name="twitter:description" content="The study of mutants to elucidate gene functions has a long and successful history; however, to discover causative mutations in mutants that were generated by random mutagenesis often takes years of laboratory work and requires previously generated genetic and/or physical markers, or resources like DNA libraries for complementation. Here, we present an alternative method to identify defective genes in developmental mutants of the filamentous fungus Sordaria macrospora through Illumina/Solexa whole-genome sequencing. We sequenced pooled DNA from progeny of crosses of three mutants and the wild type and were able to pinpoint the causative mutations in the mutant strains through bioinformatics analysis. One mutant is a spore color mutant, and the mutated gene encodes a melanin biosynthesis enzyme. The causative mutation is a G to A change in the first base of an intron, leading to a splice defect. The second mutant carries an allelic mutation in the pro41 gene encoding a protein essential for sexual development. In the mutant, we detected a complex pattern of deletion/rearrangements at the pro41 locus. In the third mutant, a point mutation in the stop codon of a transcription factor-encoding gene leads to the production of immature fruiting bodies. For all mutants, transformation with a wild type-copy of the affected gene restored the wild-type phenotype. Our data demonstrate that whole-genome sequencing of mutant strains is a rapid method to identify developmental genes in an organism that can be genetically crossed and where a reference genome sequence is available, even without prior mapping information." />
<meta name="og-title" property="og:title" content="Whole-Genome Sequencing of Sordaria macrospora Mutants Identifies Developmental Genes" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/2/261" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The study of mutants to elucidate gene functions has a long and successful history; however, to discover causative mutations in mutants that were generated by random mutagenesis often takes years of laboratory work and requires previously generated genetic and/or physical markers, or resources like DNA libraries for complementation. Here, we present an alternative method to identify defective genes in developmental mutants of the filamentous fungus Sordaria macrospora through Illumina/Solexa whole-genome sequencing. We sequenced pooled DNA from progeny of crosses of three mutants and the wild type and were able to pinpoint the causative mutations in the mutant strains through bioinformatics analysis. One mutant is a spore color mutant, and the mutated gene encodes a melanin biosynthesis enzyme. The causative mutation is a G to A change in the first base of an intron, leading to a splice defect. The second mutant carries an allelic mutation in the pro41 gene encoding a protein essential for sexual development. In the mutant, we detected a complex pattern of deletion/rearrangements at the pro41 locus. In the third mutant, a point mutation in the stop codon of a transcription factor-encoding gene leads to the production of immature fruiting bodies. For all mutants, transformation with a wild type-copy of the affected gene restored the wild-type phenotype. Our data demonstrate that whole-genome sequencing of mutant strains is a rapid method to identify developmental genes in an organism that can be genetically crossed and where a reference genome sequence is available, even without prior mapping information." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2.cover.gif" />
    <title>Whole-Genome Sequencing of Sordaria macrospora Mutants Identifies Developmental Genes | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q0wst1" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__KPgiwEdjo5O9HdRryHqQs1TpsV-PeBb9B7sGBvwM9FQ__opzZWgPNgQzoEgfATRS5qCKwBw_6XcNjDHoDVOzQv8M__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__YUqwBlv6bNoCToiBWgAUSPtppdS8sbGfzmAH3qEDcI0__Mvk93h3ScFH5Xi9WhibVKlf368fHw-V7HWqWX2QMvtw__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XNWYaQubFGWYWZNytUHkWs-b34LV21DsB4s_6gchI4k__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__7cpBA0GVdrMqkZGGZEW-_SPF2HrbTCmVg9-lrP5l_AQ__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__TV2D4R18tPiSIOmeQ3vyMqHObN_FDmQ6EEIrfE5sAoA__c9BBZFy1BrgGgF_Auu9Fmysb3Z7J56eJGYvMcLQrLks__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q0wst1"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__OaHnYyci68QTwctdRYR30kEfu0eYWBrLyMqy8THv5Ik__IyfJQYM1q1QEPwYsK_Eslh2RQkxpy6yo-nR0p4DWPQQ__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__fhq_kmKS3zjIxlsAa3GE1ubLajyXUgXXTXULzAtiqfg__Lm1_Ls-bdpbWRyl0q4o0mmHyxB-139AJ68dE4zOskLQ__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"q5YMscwVm_iLma-YwV3KHYc04nAEYBUkFkuymvEjJ60","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459244","apath":"\/ggg\/2\/2\/261.atom","pisa":"ggg;2\/2\/261","ac":{"\/ggg\/2\/2\/261.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/2\/261.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/2\/261"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/2\/261"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/2\/261":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022FAR5HoqbWlkA6jxtsLoQnqyjlg5QWOM5oUDEG+03TN1ZdLe+Oup\\\\\\\/PLjM+2m8cYhPMNkug9kYVhKFvQO81l1NyQebCnlZ\\\\\\\/PwKEz8uSNfntZZvvOSTL1+M0f9B6BSduZph+FuH0hQ8hLOLfbXaXw7p2MtKTX\\\\\\\/HmIayOJFOZCpNaIUqcxI1XmmckopCTtDBdQxl1DTvtE9gyst\\\\\\\/zGWbiUtHeIcUxbX2yktOpv4Ec1+wtTg4Iemo30fvlZMDPgKaK51Kzr7DnKdRH60gAi5n\\\\\\\/Ydvaftv5S0UC+AbW2E+cCbOVqhbkDNpDVmik5JkVW3ThsDQ8CE4DYfcz\\\\\\\/Jx0WUd+oYUOw7uVTY1CuTy3i3ibPyFv0gbnSZ4DONvLWXjMRn\\\\\\\/JApEr6mVfcsg2DY+fsdRNtmNrsv13m56PvUfV3rk97yTr\\\\\\\/VcrBowclnOobkd4t3PQLHbjEBXKc7j0jHCKmhN64vFNCBqKQEXC3yP29L3W\\\\\\\/arui1TbbNip963vDbFxlrN0dVBGaNILz3Un4TBrSkbco+6jFg5uVN52CV+CNaWqzFhgtWRHA4oJi\\\\\\\/Sn4R+Px7kAQ3KF99+KTa+LdDljD5MHySSxDfCey9isl10zLbTUmq9yCDE7TG6pLXUrx+xnXjg9GzXBVSStbhaXPDRbkNaxcPhTxyMpx13Jp8vXSFgrdyz66mxHsqBSentWM+H487JO50RgdFR+JwV8W8QMhMasx059CZKrLPsUO9DaD\\\\\\\/6F6BdUDDFZdcZQ1KxbGBFhc4qcMciA4sjn+SQuHUqkCQn3beCZVbzY104Y8bt8Ior9ncP9WCxQavkuJKXiGVi4\\\\\\\/CtwlrOmgm2M9wXOCViDu3e2EKCam7vexINEzh1ITw0RNDEYYM\\\\\\\/aeH4Qrk5zjaJnFhYyLSghJdh1bDD4gKWVwli5PSjHglXrDwsM7cSErxFWsVpkwUiFeqk\\\\\\\/r7UcG\\\\\\\/zIO7TyKrX3uiLSUEhMQh29B20l1BnaGMxEKrgjQIeosj3Mz3Zein18nMJypE0b6l3hv8D6lUX9jNP8WNbCFMM8GezKszOIVs7D1sCJJJpOYSf1B4PPffEeyJB6n2S5L6LIXrzm8ZEPJBE9AqCkyIHFeB9EnfoDm0WHBEi3qlUqu\\\\\\\/j088tvaBWIbaOIYZXg9UrO6wZzRxfUyCML4\\\\\\\/jXiKJub0r54qvVil5ppW3UFDxu0iabg8rycFo7V\\\\\\\/+M9grOO\\\\\\\/CRydwP7KvIYKV3m0Bp8Itb7WJU1TpFUd18x1KJe7Yye7SGxDEYdo6A9\\\\\\\/G2TOvs7nefdd0sLVP\\\u0022,\\\u0022iv\\\u0022:\\\u0022jX0P+NE+Kcasy4mVE8gZCQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222acb9c28d80392760ce822ad4a926d0448561aa224816e9a26e95b69a49c27ad\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Bcmv8paXgk4f3m+CExdgirJHKUjZ8jTmHIvO5fKhyEMqIlqrJSvaxRMlEA19hFp+dF6UFaTmdFKuIXZ7YdCihN3OK\\\\\\\/LM\\\\\\\/\\\\\\\/GURhLiCxW3yq+Cv+hE5PAC1yHA18yVIixaABeciDnFnId7Hz5hhx2MSs7qx0TawPrUBUFUh\\\\\\\/qCI9NkPqBlBSpr0GDCK2XdJTnzhfS1PnxDuJgItJn4haZTJjeUaoNGPFFd\\\\\\\/oooJ1Bp2NySZaW+iREI1NtoUytr9LdJuV2EZ93vI21Py3F6laUZ8oO3LEqjsWkPBlFIcXe805RBWCSgjlaKOMj3VHGneouqEYI3KyR7BozJ+4UcN5SLOU+0mH+8dOwjUldL6Y9Xte1SN7rjdakY50XXHzTtWadE4uop8QxbL6D3UoOyx4+IMg8NlDRxwPKmbjz9nHrzOgupx4cr\\\\\\\/Ias2WLbHi\\\\\\\/Pu0uJZ0KJK993f++JxxwfS8m478TH3MfaSWmOBpHf5WI0T+H0nsVDFoTfuW3f+6y069Z0ER1muAfXRfUyY3fx8CcEPP1NdYBXTS53G+HU9A\\\\\\\/JCPhR3C6EsUaKKGRDPNneeaUBFp9Mwfih6t+zZFmpesvhcvgYuCOBc6B7i\\\\\\\/XYqXTRBeQjiQN3LLZEegseXucP6FwXFVRdvp9HiB6qWLS\\\\\\\/XvL6gTYOn7kHaX1zGVaJz5s4p+NzhCcV217EBm3j9V1Q+lxMwvD9rQf+kqMGw6QXXqvlMQdi5QgSRXO02hRAVmkW+0HcQQTAddFc+7Aso7ohqS0A0UB13V\\\\\\\/mry+5YqKJ18onazcnasKvLPatYwm5PvkkhoCm2w2kMSqLA1wo6ANvYitSqGn+CsCDlZrCcbYOulUuJsiBIBZFDAULVKhvdmGi2DKtJwIMo1lRYrA4IMwchNZf+cn2rnlu8qtM1k9kc7Wpq1C1ysUeziqwbbmnrn3WnCKPYVh2DbUs2hF0hh0NUylG1NctRgjbZM92pDgTDqf5DXBVSJZQM8UoIdtcXt8zEeoSfEJLvJcg7Z1Bh7vdLLbjwYw7jzHiB0\\\\\\\/8fai+2ISCYIttq4r6hqUpOwCOtCBoehe9ap\\\\\\\/RlQi6CuZOc2nOABUS7eaY3jQTdQOeQqEYKbxFXsldcvnH5kQyCvu+86zxC42BCVVsncnIoyl0DyEz8I87lgrTj5dqR7JlEgJFia9wrW\\\\\\\/6rFUxu8RSNRpp09mePqft5b6w9LmDRaXz698VUqJnv0jvGOG2J1ZQ2N6\\\\\\\/M4YSBiskjJr2P1Kd1F6Q5SxkEClzoyJY1\\\\\\\/C5zOHz4ezoXa\\\\\\\/eusu\\\\\\\/EJeI6spB8fNjkBryAM\\\\\\\/gkVms+vYc2wOspIsB\\\\\\\/OL\\\\\\\/a5S+CdTejGTPma5hBf75IKDzxJFwJTXyE6qXeoTspiRy5\\\\\\\/gp0jgO3XAI3JxZgBC2nK1uLwLrhm88QoxeUEt0M3V\\\\\\\/D1cm5DC4nzhKxZIPQHLp5Cx\\\\\\\/bxIPC\\\\\\\/CZn0TaCKbAzCrbz6azm0QUKWkp4D8mGyp8l7UnI994lX99xzHMB6Fcs8FoEwxTfkSf5zQFYh\\\\\\\/osD+68WFMJNNvulPERrMpnLvwHAIRE+ClTrjIh7icgEjNt891yWriXtQgcqQPKhB4+Wcj9FvxEnJjnPrD5Q0Am1x0bfRPiieJzj9zQ868bcGqOS+a3Oxwp4ioIcIQjGViqU0mYAH6XIfkmudPM8QyHnLhjADQZc5LKVhvGPqZ23A\\\\\\\/NnqQX\\\\\\\/FptFcSRiQYj+5KqF1V1+NN+e1dOf+C2mzr14wv6EeuGTTGwi9rBDO9PQoTy0V\\\\\\\/AqH\\\\\\\/GnLZ+34cEenumA3P8J3pE1zshp0e8ApaNxiYdvuK8Nl7sJB1tt71j83ICR8reup0qEEK+FguyNwuauhSZ9cGz0oo1JdFemJZeCIbkJThuMUknmj6jSM3AejzqYJG3xmtEOfHKTcSf+G0Yo\\\\\\\/yu7QvKC4cxAt+I6kOTFQHlpIyRpcTTdKjx4T5t\\\\\\\/6Wb2qZuidIfbVbxcb\\\\\\\/m4IPd57GjnE8elds7nSNOnLpsAFf4J6dxq7VKCCfjUXsg3SRRc0n7fO1r0r1v2f6Ouyymqckox6tjhqcHOgJJEkbbaN2Quv+gQJ865n0RIRc0+7cqi8EI6nKpHI6n9IJhpvSUIr+i3xoX250remH0bo\\\\\\\/g3SyxTIz8OQmuTQgu69Z4OvdO\\\\\\\/B+jHcj+JzMhqmN9brpN9kQaf9EnEllktpHJ4FPXJQycmHWdwwp8y97Agu+NfTUWg43i7FFuBRqRrizCdRY0bT3yYE28ujolG++H5p0h96CqL28vemrsLbocs0RwUc27HPuKVi8lykI6hcpRNQ+ccxphfOyW+ShKNmXtef0+WGs7sIw2h6YMhLthM8thd3hRYJfAVPH\\\\\\\/N2VcBnKLIzExRs6CbIrn2tM4vxhJF1ogaJZoF3jKjpemMQlFvGd0X0QHBOvRFm088vbRUCugYCydZWpmNE3vLyK8pUHIcn5fnFOeu7QWvKQUn6Mrpd6Q60Lx0SIHGiK8nOL+o37o60J\\\\\\\/45zmVWcRVjb61xiGH4xvBH+6R+UIHOMtqCao6MjA19WruYdwgum3P4i3OkWjxDMzHyq33ztTO1KdTG7a+\\\\\\\/cT9cxbAy73LTM36xuQWCgC5B7SHNT75cyGgH6lRDsl19TNuHKCYPWDd390Ny9iZxTCG7uMwE9ErI63wNLeG1LG6Xu5MRA1FglyoaCxfK6dhCdhy6oZVKa2bbQIJhclV4ZgJsIrS3rd8gkW9j9LRI9QEF0hIRA\\\\\\\/NE4UzxBofHcCpgvJCuXFgv4EQSS\\\\\\\/H8SuEy3NMy8G9Q6GANpqgQPYaKIk\\\\\\\/jZMkJ0jfbdlzfsN\\\\\\\/aNakquhU7MJSBSxqykLmakU8gfTNeo1pmBS0s5CXnUlne93aYz63B4HP7BVR8WR4zvXjNHnmgpMehiDWYlEwsvdvYLhic9RKwNgpPgDuR1uf7edmNodl1wPUw6dr02zD4bpmJJTgQL+gt01lndcli2Sov9oMJjSX9NXHrkEzCqTSgIlzBOCw5Yz7qRZuNkuE+a+qq10ulEZ7mk58xPWFCd7bbSchfwfrF9qjjBXmn+gaAKX+rEAGCfcoPS6owJpmfz3y6VXtVlJYU3W\\\\\\\/UecdmF4N2zUREL4x1fMcJM7qShYibH8xSAlXK9Lri4e3WV5MLHxSzaY2zMXpUN9nyRVuLfRUx1T6Rt9jcBXq2KUZNrtjZsD+LNf6dNkh+vmSqnS5Ec9Swp6tpOuzBaBMM7rpYKZPhErhh2qUdyVNJd9cUcvg5rPF1WQ\\\\\\\/fZBvVctf81jezaA2Wi53IehyJGgMPI+0YVqLCYtYCZBngyFERJhRatTVokyJ5Vv807gBgvTmQxOe15U87wkd8o9uacXmGH4dOvjTfJJKLjiG7PiWAKEdCsD037Q5nrZxgc9SNJhLZE7m5D6oyuVAPPy\\\\\\\/ovNac+DVbQ5rvLwE2lvNyyxB5kS74DDpU8LVLlc9L+Y8coTQ16G0T78VrPBr9p\\\\\\\/3yhYeOlGfJUNfRIhlgfLLKseaSkjmBLkWOrXxw+LaQwsGr6ALvVCjeX+\\\\\\\/8AQS5k4BIoXUuzZm5oWSu09y39oR+\\\\\\\/E+i7PmMBngXydkkrW6Fhp1vMW08JWAw1ynoEYLZ2tVT9Pg81Q9lWG\\\\\\\/SuuYm3KpTeDi0ylQHO1gu9iYHohBj0qzOez1K1d4f6sxE10rXoBilTML4SiS6YCPMprMmHMoQG\\\\\\\/LRe0lehVJ9VxvD91kGQJfMheImrvKe7RJU0eR\\\\\\\/jnsSSqCRSRtMcT7ETBoX3U9UHULUWc8XVSlnSl6z+ayfhq4C2C2h\\\\\\\/GySt6fajYOQalINuJ3RfvfhVyswIK6fNC4vHKiKa0Hoh4J+dAv9ixVHER+HQ\\\\\\\/rQYM+Is6mpDSO3O1SuGo\\\\\\\/QN70KvVMIO1iy6a0heO7LHCGWdY2BPcyWiwoSYCGNcngvUe\\\\\\\/2NDMabt7e8uh3DjbW239XAS8PWIFQ5GgJABuoZh7mFUKXxKgHmbWli398JKFhK3TJzWIeiR6GwefNBVXwQkP2C3aTurCuHh540sx2mjBbXHCvtEmkOuadm99tx2sTSqMVPWIhwNY5O9yB1tiQWQfVdgP1G7p03Z16WIRCsFZPqHwbTGmLTCBR2z8eUSVOE6j+2pCeZKf4Thrkjq6GcGG2ePxHw+DKJmmDB5MSXWxDHGqLnv0mhrytSFZRGO6uPPvQXB+4i70TFfm9lcVphABTSDvQVC3v6VlO+ytQOXUHw30rONrgloBE3rvvw51Y9S8b7+CfHRbHYYg3HS5LceTNqik0CKryn\\\\\\\/h9Dupk74VvSTZCJw43enjt85\\\\\\\/WGihr2p\\\\\\\/T7n1jqPP\\\\\\\/F4joHCDfvhi0GXnqSEpRBR5gxKC+Py3QXBRjkXpnGz3wOfJefYgW6SUhlLSaBm3l5lSeM5LcPzdvqRbNO4YK1DusfVv5ji52W4EJ7wQBVuHZMn2x3hDHb\\\\\\\/AK9nGL5o1EotToAdr3vtk0V7ECoFi5J5qXg5yUGCJlfNwRtl9X6O6\\\\\\\/KbCkH\\\\\\\/2ZI52clJsbKEhKqvQPrIZDkOOJ5vZ59cifVJMyyQJnlG+w\\\\\\\/mw67flEkwNsXJfh003Mtes20cIVs1Jb\\\\\\\/F1YZAAFq\\\\\\\/9PRNuHfXqGmG\\\\\\\/6ci3nbuFcRzRH5LGrHf0zNzKWAwvVzAOLeP76TYQYI78Y5GPbWgoFx1zugXe3\\\\\\\/J4vvvkUFCqf4fm5ipcFXjqDzr3frhCeiD4td1wurYoHLgINBRB45gYDJbDzJSirpBrPxR4SacDpWVSDlmb1vZwdjtXq5PfckEMsOHb5gAwIluFZRhFmx5cpH74XBI3fK2lnmFAZ1Wk9OQ8grlh+T4TcCXnD2ROUZbEh818HBGRYewRYNJJrRv5c5sRkYjLCvYLKSp7K\\\\\\\/k5gR8qPw4TAhu90ifxHvltgAMTdh2bawfHayJwsxKSVJWUYj8YqhhdiQvvzoYqsTS0VYKIsv+0vP+eJDLXQ076awH9+75z\\\\\\\/QBNW+jm6LN5CULLljLbgi9Cfnjfoo+KbbHo85eGFplsnYRoL7HXz3Cgn3NXc6+4TA8Ym2WnLROQdFAdIZkoTuC3ULAeeVo4\\\\\\\/xD6aw7X72CPqNHAO3APuy3+B3UacDghaj8dCYWKlLroYSAEq4IQgAiSM3++Ba8BhOMRzdnn7JNXPzBRLjhP2Dr+e1dyqI\\\\\\\/TDDUeYCNJRMf\\\\\\\/zpaGrTndu1E0PI+F3R9VyJxJu1HRIBWQ0+dmngv170QlBqGVGdsNZ6UENgEQUx4UKD1uzLfN8pW4kUjiv91QYYjvhuILispeSqTyExpY0SJLVumDuB9M3YA2HiB8KHTpnaCvrtIVtyr\\\\\\\/eNBGcsXLn+flFcjFL4gx4VW58qxbcqwPyUa+G1vGRN\\\\\\\/eIYbuWqqnCsGVoRtALHOgdtPsbxPEYnOUhdvEHXB9lP8ImGhn\\\\\\\/XpsVxMH1oVDQ0X0zcJ0E4S9FpGcAc4Adxd8U7cDy0fVOgVWS8LMQJQ9o9xc8Z0+2tdprTwOw7089CKjz9hENMRTRThmxElOcBy4j4N4KvjSFXEoYDOc0gQuH93qQ925OCtAl\\\\\\\/7e\\\\\\\/956uwPdb+yvirQxfQbj2YvgEPqMyYTy6W8IIZvBt6RXTH8xYQeOnhWxTArWisjpTJrq57ziJbS+lO5UBkHOuBLu\\\\\\\/rRbj+AQvHYhVE2gvxUAVoraqkUxHc\\\\\\\/Kq2HGHYPfO7z\\\\\\\/kbis4SU9T3V3sxifZNOFQ\\\\\\\/0IB+4FuNvjeXVf9u9MFXNckmH3D+btWfnaJjz+WZufpo934rEh62NJLnwvb\\\\\\\/ksw5WOH+ynCJkAwDWbNp1Y6Kb1GCDIDQ7UXH\\\\\\\/kjrcKeA043bUhKz6aJQTycZ8inr2iXC3H+I3fT2zqkRn0KJNJZuap2ue8raF3izW1qrfMa4a1U46OW2VA1Vcjs\\\\\\\/pBfIQ6r13w8598vofhhZdW7RVX7k09v8oyPv1pnfEzO5GJ749J5iU1pOFqZL8mzKxEGg5qtPmjZmJidcvoaZbpvFIkYRG\\\\\\\/b5JTXsswlS4QO8oe0qiAFznQG4JTxMeeqp+WNdGlgp1Fr0J0\\\\\\\/NgieskBDoHtzRvpECqJY2igjHNXF1er9OhINU167CZ692Gw4DihMhlyhVZ7RMxJr\\\\\\\/mgJL8XIqCleMpNOCY9e+fmXNYX\\\\\\\/XYLEjChDL0kGcSozSDzEuB3+r0SNpWGOBnhJ22qy+BqY+rPyY8qSeIGj5xrUwtyR4XT+XnWTQW5LePOlBZZOOPNvoJ7m5dYO\\\\\\\/uHXgkK44dD\\\\\\\/qAKtmC3RinW\\\\\\\/N5FjQ3I7PVdqJhjlP9vl0GAcSHsWpcfyx+Ry1EaVZg0dsJmY8+XaXHu9E2cmPJRCODhHfIi9y7Gl7FByPMMFkPooLMPmbQ9\\\\\\\/9yo4TUoXzkjdqzM5sj\\\\\\\/lE1PzReJgFZBEfVhM7ynNA18fN2vUJ6UNw8Rhg0InWyNKK1l7St9v\\\\\\\/qg5iBIj78jn49ZviXfvj+epeNGxf2iklo5wFEgbmjo2jbQ1JIKGjucu4Dw6sSHF9G7dqbebp9ahCutjIATE83jOPNbazyn6Q8q0UytUGa58Vng\\\\\\\/Q\\\\\\\/IEXUfcO73sNr2Gb3Sq7Og4nEEHu9mbEOtyHGTlnlozsLTj5+PuiEom2QAqc0dpvagytQXQFIOPm860bml6y6NQOIJdX682iZk6r24lRoy8KDC30qfmGtJ9wpgSlX4ocuiYkjtswGUkGoJEikX6qtBwH4jMPPcPzUf7mrhQpCGA\\\\\\\/ojUN797Tj+KXa0v73N3yaOiNVIaHG7RK7yxyI1JYKH39PpXfo8oqYn+oOQI0E60BriCB4N7KkM+i1anOBovkpa2ecpRpq1s2JCGax9WKBhMlMMEofVMtn+4TW2WZacFT5KwWG7iz7PraNneGuI+pRw8fgU0e+fZF+8xA1sgWbMR0FaO\\\\\\\/KWr5OWK3fSuQJ2hKk8qkQWENZ3o9wt9PzAa6KH2oRtncFJ5LDhyBtnYF6nFcNDEP82I9WdECB0j+qSCLNhmI8DnBMnYjyUDHi7OVICh9e6qSkvVL7FyG98g7jLLx7Yba4zDbgGqIHtra5hSlN5lZJWdpZvl5LqdUfneihllpnTeqeBcaKtXUAgXKjCQlO+rxdLusZUUDUdgawkX2bw4posrnnShVo1\\\\\\\/FNM5j3Tsb4Xm9GZ4ciGuFwFnBKbyK4AoOJ6zDCnzUB6jUzb4jDqMNU7+epvRs5iEcbCBmswNMkCXzymQ8RL9Ywe9mzZSG3sBMIdl8FPAYCnLCfPFEnBvOGdTlDkfZjqs6w6bWoOKEaY3FW8fXeFHMLHki1J+yYxXanqUquECTP1LzZ2yikg4+8a+W8vpWTaZKN8D\\\\\\\/QIMN5PBQV4CD4bf7WllCGNBwV+MudVzLjcOrDDj7zirg4absaU6nHPK\\\\\\\/nVZCAs\\\\\\\/Z3liZ2NXnrVqiYBvgJjFYKYRyPpOf0uuU5cRRSv0rRvZGtcZ7vgmmvE9xCvoKDu2EKs+U+APtwY4goIeOHQSHRQc5Ha\\\\\\\/6DyvzBvInn36NhOgUp39GJ7DzNji9BYE80aLu55thnmqY9J5TGKExVpbXX08qChix1jBIrhkQ7rZyfrPNcngKagRFcbQFGQiwjuw8lhUvBW5aKQBLzGKwUFe\\\\\\\/FywyiI4GxIJBy8YtbtkKYwIjVJByFLgp57CnQ7KEJPgRZac4L9CZQr9D6WqiK9omx52gI8XCoG8N24FWbdBp3VLIT1FU0uJ2lP8srSCjB3kA4oN3Bs0FEMtxxislzhD+EGjfhtS3141m3qAL8fRSQyVJyedpk96GGIgOtJHprpish85zLHXzMOKOzFuO\\\\\\\/Knrs++94D7Y9fLP8ijt+gJ9s2Izsx6vao3+CSxH60eSrZ2mytCQf4DxVZPlDgEm1YXPRL1qx9FoANVnJEEn1NNu9ptChwHcszik8uXQBt5S5kSKZzGy2lS4FHvZ1HBnpJvD15Mjp5E+iykSk8Hg9Zoq6fL4UkhFRrnrfDqHUlm4\\\\\\\/vcjr\\\\\\\/AEoDXXQjdLOv42PaqQsZkrLr26MnVnQPSNIIbFAnKU8MQUjcKp5tNnXcyzP4chB\\\\\\\/aLzxIEhXeoiOZvL2h25znbw03\\\\\\\/VQjK2aDnE3sgHNVaMqSyyDP0sswQzYX1pEWmOdtLlNZAdbmWoZ2d6cIrth5uCFmzKExJxC4vPUZahgYhFMiuKVdOSTIalHTv3fhxxy6SvExxfxgLUmOEna42jIDVx00ClAO96E6i3Ibu4ce9R8qTU+ukf1E9cvkJxMlmRzTdXksOPSN2sNpTuKp\\\\\\\/PYIfvwCfxCNIaq31TFtzOFdeImZGxkpMOGm\\\\\\\/0n2PCgAIEx05chMwtfYOg5xfw9kmSpXCR4E27jrbeQS+P8Kb5caIlf\\\\\\\/4iXofMOXWrkB\\\\\\\/fxLftPAmB+IaOMzSFU+6tZuRu6o\\\\\\\/naN24p3i7wr2XrV\\\\\\\/P3INstruPwenwcbXZZwV8LQDmeBeeV\\\\\\\/LoDlFqEf1b\\\\\\\/ScRA9YDCQ1XnG+YDRn6uXO9xGGqexke\\\\\\\/fau6yXj8yFDueMAD8fnvkv6GQMRcZrFONPcqxNvGE7nzNiQBAu6dbxXjaDADToGjyuVDZKBut+vyIGYlN98i1CxvjdTGyleV3wFe6cZTFnEPkdJv9f77k8yNfJfBUgzXLvu4tJYUSS328Lhk2PBl1\\\\\\\/WVgiSeKmEzR39cwkDwdq6p9rT1tPXMKTUk89ZdtxnNoKNmozoj98x0H8c2nrSZMrs925yptyzrDq4SB3\\\\\\\/GI6NusoFlgqg3vdzgKoXwaCNO2\\\\\\\/6XPrY1HnyILO+8sY6N++pZBYd81ai41VoPEqI5kjnzeBrwk5t0BwMQsbo\\\\\\\/br8h8uMcJPObzsT16x\\\\\\\/nYBda1f\\\\\\\/\\\\\\\/ZZJG99PM+\\\\\\\/2zGYvFsj\\\\\\\/GehJbnlhcqdMzk9Hov6YQ+EruCC2xkebSDANnoNBgKrvWB52EzyNh855w8\\\\\\\/DtSYUPP7Xav9U3XtawjuH+yHTLZToaZIKbMbEmqgD03Wl9GUASDqP9RfoDF8RwHevPcXBFgdbPm6xwTsWTmbev1Dg9C80SW1xPOazWwSzaaGB5A5t\\\\\\\/0Bw18+AAEITSTTqv+trU+wurfXvMahcS+90MKAFj+sdioE4CPd+8RyUCPAoA2zHMW2RUKsw7d9Fr3s7IWnfdIvreI\\\\\\\/ptW3BFdEcj0ppL16PqFCrrrk7ZFS1piYVnJwt7ceW8Qvyg+2aOi9VQcKDgLoJoICtxWEBwTYenap8kL2t\\\\\\\/mdyB72TSwIaquN379xRAPtPCTDyw1q9VjiJbqHWNlolLoij57p5A9X7EnvJAofZBCCJWIe8V7Stx+4RivaJqALniGRl+9K3VrvF1LWiiF0MaIKHJBWu\\\\\\\/5SKdx7vuKxawmeciVwwCkWpm1H+rmeGusx+Y1h+MNtUn13NsOtEnvIdQ5Db5UFZMTVKXeQTWidZ\\\\\\\/x2dFs74zUSHWJs8CO\\\\\\\/9z4C\\\\\\\/MVzd6W8cYSiCAm3gvjNTc0Vk4pN2y\\\\\\\/nL4cuaQGtkuAIX\\\\\\\/\\\\\\\/tQ+OYMYddGYo00SY0y23ijTm7aw4JzfRK7Hlr+UzStIuE10AGaY\\\\\\\/V9yy9G87B2rDV5PrcyZDQ7y4Chvv7yewPXL9oMJdf356x4O+dlWIZedfjTHZjFDntg8fn0fXGGhxXVTpqiFKKuqrrTU8+HpuNvE1xIwDV8XRVZx1WSW6blcHD3RAEW9\\\\\\\/ltXIgb9eY5ftl0NUubM6Ad9dfxcOgr0TDurHflK7S8Q6OTacE1YuCyDFkSq2V2ahlstzlEpNA6\\\\\\\/GLziqfL1R\\\\\\\/U8A5AbrXAEbmu5c5X2NKi6W7aJpeQKOmm+yZJGXvDAwfbSqcI4aHz9AFmaIJmK2qNMciRDBXNfNPp7IswO24CrwRekt35\\\\\\\/GWdaequ4XK+Utv2oL7AR+3Mj8w2EZ67Qvqe+FaPLzqIMy+SQVuLxo71gCDST8i+IUfl9JvfWIzDmRxeQISf+s9WJAUAXUCHI+pyZv0QfPuPEukutzbnU+VDRLoFPYO9ctKMECBbjUpKSDLhjF1MeRfWOxcDfQgRp97VrXmsNZDEV7B1vfIeYymdYM9VYpqSlcjHDkHiZKVOXEgxGbldPsAdVdgBYQRle6laKQhhmT7V3Uf\\\\\\\/Rsi3FNXOEPzBERjdFrbB7KbO9abfNfZ+S4399j\\\\\\\/7\\\\\\\/nA8aEOkhTCqTCsDWoX2vYuJECSvv1Oi6atRmjbChHTBKRtVHfaQZyMBPdRQ\\\\\\\/sQMTwG0R184AzE7CYD02CQ6y8XxY0+54Kl\\\\\\\/JevyTeDnKNwN7W1+iPE6GM8TjC\\\\\\\/x1Ff2tUma4\\\\\\\/p3CkWoWphUfDcfOfsc+tRJapQ8mcEJF4hDtWdMuSlM04Rreow6O1f2xO4N1e6yNt6f+\\\\\\\/Ckc\\\\\\\/2DsL88X\\\\\\\/n9PwgAxDSNHd9S+6\\\\\\\/eGp1NVuj+WqQ6fuKN6NS6IBmvRwhk7P\\\\\\\/dAEERMSyfTCSM0MoqHThA7NxeeXoEhiPxS8q6Rt4+r\\\\\\\/frcQ\\\\\\\/7S56\\\\\\\/M61mUt1ecrpuPzrBr8b43hETwcZcvAVGgP4J7r9RV5I+jkdFAPnzn6XvcelIRVHqiowbcTo3NdCAcGoED4nKIJitGzmE1V\\\\\\\/Uorqx4QNsD8EvCrvhdgPU\\\\\\\/IDkGtVUrywg1lXoCK5KUyVPJgcI\\\\\\\/+WFgF10GfNsp0YNN7n2+Ocd7j3DZFofoZoN57jh2F8dLs8p0yN3t9OPreWDUdtujVvZhlcgTWfVVYRdtBODxolj6ijqIEGplm7B52XLljE3xhpAtJ+k3RLuna9C1vRrzdXwESNQDsH3NcWfcV6Ep4izZmwVBooUcCwjSEFdRSzWj8vbhEXz6w1y6zcbgtXA\\\\\\\/HLzYet2OAeU43fKEhdpYdwKeFMGA99ltzOZB+Ep+o+EL9ZsCRgLRZOvaR6oynBQHDIYcwsysD9QzSnztYLL3S71Bxii8XLnipZywjboLR8v57L5FJkCdM4Yq6vtyDDvCoPfyk\\\\\\\/glxWFhFjLw5jFmQJrlgyepH4ChY5KF2fAkNGrEqMTli2s2XbiANMLliouxbPHaaNgrWuwG41fPw5Df4zUPyQmOpknUfu0PfOHhNtUhIrN0zuX82huL0g8uMMCzzhn1QYf4vBxmi592s9jInjqYAmhRMNGPzRneuNkAG00Xy0sXpv\\\\\\\/l\\\\\\\/wa7xqsoIONx9FWIaNzYp+6szuaeW0G\\\\\\\/TwJN4gQOpqf+hdClcstx0UWM9jELPfouRJeUhdKNmicAhi+V4OcY1SF6O4FvVI4TmdojuLW7QV51HPdxTPuvsVhEsYF3P\\\\\\\/97tz1WuEDa\\\\\\\/+PJwFxd0RKyXg9swOj62fRVpZ+gcHv7B5V16OieekaPgesXg1zBEDg6lIzkrHAwqfcE0CqDe+QN58LBCEC+zXalDYqvGzFBrh7r1a\\\\\\\/SxUlhkEW9iDMhz9fnGrV+8m+t8c8+fphxuQPg5KzC9CMIV0TaAm7WBYh9+ABEhCSBRtAEznulGX7q8YI4ny5JtcQWB2Av847TW30Z0T8qsIekF+UONleC2yyKTG70\\\\\\\/AUDRQbenDvd5bXbwydeqNUlWHB0uSlXfj4mXKSTC6xux\\\\\\\/+9hr00gm\\\\\\\/QNQdcl0vgrih7S1+cjvrB57DHgzazpOZgJIXcbGPUTubbpi2SCkhVkeyuVQaxQlT0deBO1MoSx7WqIwG1aXnShhYEwhLk\\\\\\\/v+aq8rz1dnQagVad6WaHwqpwIaZcz+ojHNvc5THCJZ5bmX24AwWbmsKHpy1VyQRXK+Fjev22\\\\\\\/nYoVXaqbITru1zIv7E4baaLXyvwftWKC7UNr2oZtv0dDV5CSN+8Qg4ZuzqoRlz6pELkUdRGN85JaHG5df33QIlCBcwawAI4FJ8+UCwYjqiQ9zs3F54OTuofN8CqHAl3\\\\\\\/\\\\\\\/HxUag0etRbbSHYBCsw+mPavAJ1Oo0XVH\\\\\\\/WLgCzQmZ8xszRPTzxt8MqY2Bwmqd7yivYqWuz78hPe+xrt5JWBu\\\\\\\/qCcIE8oLzn78C89yyTiZqpPcTXT4DwGJnwKuK3eRZL+NLRolbgria10MIVrP4m7+mxzm47wj0lG4r8Hm9fXcgA23IirS2ZF185p9vURLVMiJIlDLY4ImKLNEiV+DfOQ81SI2xEqWRO8XAZg9pifsdGh6UvU+Du9XGSxP29pRWWDDM5+h5RGOfmFTpuLhflFXKAKZO4QTmjOC+RaNPIOI+CdDPdtRqK0qHlCEA1dEcaKTcv5jEbMyAo+bF8zVWFv6kZGsYSW15zDWHk09dInIHazczJ6ucewuQR6ndJrFT\\\\\\\/v\\\\\\\/ZM+ar8sSughGdAe+k9V789OxRernxDERtnAdZf8NJ0nfrPAI6DDBZS4Ql9BV\\\\\\\/WVO\\\\\\\/odpjzIUZ4wa\\\\\\\/kSyBPF2JONeSAVIDHpBdfcufytlntrxTjPt+JZjv6t6a9+7OoRCDq1ZY6I4W1I3rHNoCZ\\\\\\\/2VzqLiek4B8Pzcvy3Tv0PF1dFmJWlEItAJ4A0\\\\\\\/Dpkpvsl4BQrWVqiv8oA23WPzrVYJ17DBVQG0XO4dnhD6YrS2W1tj9yi1li2VLV3SUeCOJ0+Vlmfg8F586rwWtMnl798MvKsLzmooIPzYKDLYXSGXUp8m2E2mFuxoTfWVZA8Q1DoscY5RBiYuNxwgTA8XCdG0VTI5Y+w\\\\\\\/UJDfzJ1UVWJw5U7sIg7S1H7rmFbPE5YvLy6WL+SZyQJ751Q72uy\\\\\\\/tSiOAkCQdoOcNvkWMURr8Y8H6Pak7KtW8RYtwTvMOffYvcboSO+vDXN\\\\\\\/PLSJGh+E6J7HaGYESG3DWc6qy1rOEIWxPeL3tk6A7qibGEgwAl2GnAYw0moaBGyfjhiwyD4wApqMJpymjkBxWU55wKnnNvLZcEYoG7teOpBAspO03fkcSbSj0+bzj2qMGpq+PNgHCSrlmGo+iFg3fCdEoGj79iRAA98U6VebdbhJfAnFJstWQ0vTHDr5+dwPJwWkXcGhCZW1Rau8QnakCamRmZH+wM4kShJno\\\\\\\/jdqHPAz9cn6iI5K0raRpnHzhBjvLS00Ac5ytIbQSA7H9xNrYOvgmoSCo\\\\\\\/HkH4uLnAtUbMOSH8Za5w3zZvPXM18JyO78amFRqN2dqvizuxJIuDBc5HjdZDfCxRTRdmDj0BJmoZ0OxyKmu5bDfNskrQ6cERb\\\\\\\/\\\\\\\/R3VbIbJ0G0IKBRhyYU0eJ6MLZRDDXGy\\\\\\\/3wbTi2sboLe8RZ+KJYLvYKbQKSgwBvHFjDrrGy5Nk0m9SwnSZiD7tE1XEiyV7GCnH5beMUiOwF\\\\\\\/CRnA2ELWclaxYUKGKs1uHH3md4st\\\\\\\/WdYspxiy\\\\\\\/gtGs9cmOrSUE0mvLM+x\\\\\\\/\\\\\\\/ExBbV1IqjnZjp8qenMYkHr53+p5pC60f8YVnOOru0YXfeeXDnSxA+hc+OWTr+khtMlyUSwc38STnUqkrOz+5KjR3aNfo9R65BwJCH0f+m6GEusJ3OGQteuEZv1bq3MJwtPkX8CpxUoSA3g9O8p10Pj5Q+0oYxKngw7wBaeDdeh2XWTF4EF2TkZJapeeK73o33511p0QPtn7L8r7XWHOEX2sxDnEOdcBLPNRoAw3qRfaTMMhcdcVSNbsphYYqP6b8NRJ7A8iH44cAqJ7XC3wGYvJldglTBC2K3fIUQs7M7Oq9vOa1+tGqrpIEXI12Y0Np+qivjKKBjGAzMoKsyk5vApvxa\\\\\\\/Y7W8jH78LFliI73fF1nWNb5VWLdZsZjatRNKCfsCHmdQxC0gCihVZfdUy9MuJ1XZSfWxv0YJjavEagAOaeJ3jBWci9egcXZqkrsSxwBHzikMcZosisMU+5R9iL7MXNfF8ArONMLqzL+ROuV5PstHKmWE7mTOxsR4QVGTLLU3JKAc1cr4O4CHzwie4Y\\\\\\\/MAxm1SEP7WZi\\\\\\\/FsTxuj6yJZrwwCI95eDV5wGnwDkJlfOWA9tAY4jjoGkSottFmM7uKTeRpf9DtDVr67ZcAkQPGyeAn9FIW0F3ReWyZEtC\\\\\\\/hHaz48Nwhbhz4Pc6l9DQszsfjJSy7xOJiqgB92lIgLOL6Kyo5vmbYKICBxQrTkWUEUxIAZRVHwqgsjJeCp9RUZNwvs52DL\\\\\\\/+RU6UUIEwja2CnTv1+wkwIDuVRbVdTo2VBajuyKxQXHun3oRj5CnvzrNrdP\\\\\\\/Oyq9pfA\\\\\\\/cPZzVfUpn3ITnxXrWvaq+F2OIhgkAqlMV0DKdG429pCUiidQGHudkgkKsmplYbLg+EBfsTvBXyZZ+q\\\\\\\/1RrHXoLVN98PZl5aDoWJAVG5wLSz6ncQLuwMwCxd3Oe06LofCxQ5A0x4UUsCYWiAYkRhkr66Gdx+QdEG4xx\\\\\\\/Vy7PxAgWdE0s9DoxK4SC9oaZjITsqjP9oHKd\\\\\\\/TvIClJE0u2x\\\\\\\/88Jem23c5jqjwVHuVtnJQDoPzdbgWynEZe8UXeUR\\\\\\\/bfs2nIMvCFUoWvCDVzjIVoRSWfth8iB3DHGRhoXHWaMaa1AOHmzRpuTGORYpIj9Q6v4tM9n7uRJHqL1LKS\\\\\\\/oN7GABXrR4ndtcBwiFNXzNN5CDpDRpK+Do8Rx4FAYvoNjZLbdjV69u\\\\\\\/UVouw1gIu5+UUpfy\\\\\\\/BomfUOH0l+Y5WzHp5da\\\\\\\/\\\\\\\/DRyVQ8dxDrMU850nhROOW6TOSARu4QIoNRiTX4246CcZHvp6oyCKQS+38VEhLx1pt\\\\\\\/Eikmqa+wRCU2IL\\\\\\\/bq\\\\\\\/gSsxrvWZzudz3\\\\\\\/iiU3ZWSRcdJMtJNYS3vvubxAEPho2fBQG7\\\\\\\/cj6dAbgLXa72oo3W3vghb0lhRsDRhaCMcr2Iqg1fJoc8AD2TAacb9Qit5tfFDlhbgN8Y\\\\\\\/xOfURIpyGRuZ0LsId8XavD4jIXz+GC2oyyEjBOutub8WOLBWN7KkwHRtz2MDlYCvHQgafnfcIRhndZio8Vc2l4kZ62VAYpt3hY6wGwfXzPkQoF7lTHOZXY6MORQdqaYEZT8A7NpvYLQn4ahapETt2JyWZVy\\\\\\\/bSEQY8tHDnCXMHe3QpqQL4pybzItda8bjJjm\\\\\\\/xRZtJc0NHsJsaQc0KIhO7ARSIclxl2FOJ9seKOeMuP9CbxvGPAhX1j1Gt+ZfRMDj5JD+Znp2dTm1+5gV29crCHywe7YkN5p9+DQqHab3Nr9QuikrwZjX3JdlEYJ22MLmXluGXXoq3oJlBdJMW4ze+u+feI+hnhyAa+aTeukhlqTmRRv7wX2U\\\\\\\/g9WIrfQdQCePC5gMxmnA8MTBLxGxKK5DlfjhJFlussJpYfmXFRV81PmsTt7RNX15g+rkLqG9s4bnxfS3ZDjGX1uqzzyU8q73NSTfiiAE19CxW4XNf1t4AhY88a5BJUhsRI9xeUfIXkH+dBZaAdtpRuZKLU8aiku9FP07E2AZGkoO6JOLkVgrbekbKiZsaM1c3LxWvZhuu2erCU1dp6h+F5gGhRM7Obrt78hVJVIvPAuflLfEBUOWRBwu7g4YSkv5kKB091rmh7qC4pxUY0S+qHyAE598RWQ9vAZ\\\\\\\/CvNqSu8+3ZUqPyRTBBuJNxniDhtX8lJqBJ1ymo3tafNSzIA24bgBbaT2vWd4klzV\\\\\\\/AD9I5RSePWugdaIERD3ftOiwoWZwu5mO1GtJ2pYvNK4xh6za7NaWR65uIkEwMFk6QEzlBvCcXye9p4VqmqfhAHtITgb7PSDetN8ewXktYgnMclM6sCp9YUGVMaoGYNBPT0ur1zpbc\\\\\\\/3XZWIBgVGO7SDotB4+e19JDIXgT+vaDMcXae+vrAhTG8aFyIOMmPAKg7NMvsA3HDhXy0WYwnD00xkqkuUqhciLxM9XZOK7GNLLcDBNU\\\\\\\/qEGY0n+DMDGZjb2F3om2vXyCvHmR+l1bMfOi\\\\\\\/nJ2CAXl0GbX2\\\\\\\/GX91RRs04KvW9NSEefdupah6eHri+WDLu6jIegoi5LjUO63x33WsGcIoQvvmgI1TFe946Dyj4MhH\\\\\\\/jCiyYmSTRtRbIO0Hbrfa2PtKiBQPj02Lzqhci+ArSNDdk4aCHU803JTXt1hIfd86f1WD2hWLwcG+PZC7qcxnU+T9cZqRP0wVA4z5lvZUJRq1zJ0v9EQxqGlsmAbozIB7p9j+BW5AG2clqASSVoLU4Ld71cK8cOPmMBSaHLKGt45dfb7F99YD48xh5+tafS86bV+o7bqNfy2but\\\\\\\/wNvpETqAwvHZooiOu32KftAOvJfL2r\\\\\\\/rv0axoqPdEMAxv7q8TIJUWyFG1gIeNuAvkZztyESqNM4Py6La4RBHN15vHiBawEyVBG2nfLrWoYYfjNW8IFehrRJjsDU1pjQiKhfHudrY+S49Q+3culuiZpONcg9EffYeJJ2NlZMLoksF2BlgvvShHyf1rg\\\\\\\/brjFazd2G+ISXWbtVCNb69ej+joPyaXVj+\\\\\\\/PDnGZ7\\\\\\\/Y430T19x9sGPy75mD+L3EBlUNOPo8VXwvuEHKXhgIFD8eiBekclzPmera1kfERicg3nbHER+72IL8XQ+lmU8S8dj1Y5pio5MzN4XFjDesTNg7Xsr+nHWdQ9iEP001LA0oJ7XgBc3x4BJIiCHe+vYm\\\\\\\/RBgB+JPqQjeudCgHvul2tIty0F8ertM5xYAeCrPQ2lj1J9SLJaVfnCMHPVLkAw1FNw4Zjsjl94VsbNq+9cAT7KDI5a7oTEapr5V0k2OxHjbGg\\\\\\\/K92EL2nIAtHrtdIB7Rx+t\\\\\\\/xv\\\\\\\/BnTfottV4GbkBhNK2qVc85AR2aoD9Lidzsrejt4vdtI45yQyuBDu4C+DQF3rIjkk8W88nJB669UYU1sePvL5AFxlo\\\\\\\/UbMtPwvszJQAl\\\\\\\/5uG3RzB\\\\\\\/P9rspUrB6yrnIAy+GiZ0xzuGOMvbjnwN03zL0CeVyTh6Oha+l2ldJOS8NBl85xUde66qZaBeiuy2G\\\\\\\/bY6QxUNs+AKraGtYTL+DrveCtSbeWeRolTiBb7I4AI6UwSiYJs6bvnJTsy5wRzKxm5lc+2hwppmtK3uRcXDmfabCbuqmi+uhU0Z7ffJEp\\\\\\\/qSkw2+nzRSDv6yQZf9UDs1okRisY44ESqpXJZeAcpuVA\\\\\\\/crnivGtECPbvsCVXqJwnQYTKNfVnQ65bCR6v97cuSXklTbVwSJTRm2VezNUWW1GaKeIltIDHAammyHQ7VCBApvunz4s35Xk6dpXZxwnQUpI8RbxQXhZCX4KW1iNUUWj3Q1kG3P+fCk5S\\\\\\\/jqiJPDl82FCP\\\\\\\/5e2BjsFqGICxx1Wk4f7Omi8xINcSml5EjOU2SV\\\\\\\/oskElI9dH0GOhirnjppq5kaUAHhXbDfPDLzXbOW5YvLcGNLmBIu2b6L6mz+AE3CMVBSc9G9PhmB9hd0+j65QzeMG9QhbLdX3BKeaRahZjhwb6xi0Lqr92FKY1kH+ZNIbP3n11hh3qtA5OVninziUR2bnf31HtOe7NtrqTN5ol3QqAZpFLzR\\\\\\\/+pdYkkQlK9xLgwGU4y1hYpqzof1LONKBW3vTvpBSaNpKVKRjx5m+XavY+9L5j5RFh3nDR+8LS2X+rtQPoRvEHXreBvnGDLrJK619BFHOOhL+1HQ2PVe96KEjYJdgBioXrofnO8PANf5D4PSzY7ZxTWimo5ThDF89Uhryjpak\\\\\\\/WE0vPhrgYgyHd9HyNVJ\\\\\\\/OF9e1r1ueVT+lFZDHI1y7xJ1L7El72K4XvBbnUfl359J0mFK+Y2jG3dcp3jfSljdwCvuqG5iNbivtl2hxv7gBga4I76eHglBj4jOOQuhRFYCV\\\\\\\/KFCdzC\\\\\\\/ZYYmBK6lV45dJ6WGaT1pYdjGGik5i9y0QlSQLfWOICvB+cBC8d1JjWMPop+HnkLzmEGQgkgpulC2O2uqDTmzU9GkptWESCvzsM+k1Gpxp7tfyb7Atskwzi489M365T35bYYpq2w2p4610RIkbxDhkjOp+ocojtAgDA144Ymdc2XoOz3xumcyodLKTINfCX5j4sJiK5E9dVkP+oykbjq36zn5hJVd+X2QuvKhxcbManbMAEfvDLKU71P32dnJuh+rA8hv9YFfFiOsFXPgnCIjLjRUgaUcj8Zl84fGL7FWVcVdL3eWaLnUCt6DDJZ8GL0RfUKd3U8PrUTI+U5U8WLBMq94t0\\\\\\\/V9xfukWAmvnrF50k\\\\\\\/j357PdjED\\\\\\\/eHh21ubxJMfaGY\\\\\\\/pvK0Xg5YzyfqxHGGvrlo18+vyCUSkgLINwd3XAHoVisx1Tkbu6yOHsQPTdr8PGAFH8WBeTqJwHYOFQ9sIf2tslLag6MbWXyvAIbRc49CtKhCnNke+4g+DZLni9vjAR\\\\\\\/0IaUFO0clG\\\\\\\/pPeYlN\\\\\\\/D5HFh7HkobRw09Sd7Ss625qDtlqcjIP9yypueqgxyASaznTaiiHQVJprg\\\\\\\/NGDnu+1XAHlwr6WDHvRoQ62DRVDhmCDRHmMCH9WNcxnubuiZM4Yvd3FDGl7y3E8u+nmssg4HEqf6xSI2ciCpLPYBIu1aqSGeBpvAeHVk+LFGy8LK57C9YiikEIArrhGUvTmkv1ElaWnGqRhBdyQ3zOsgyD6cW7rK6bGhgaW63UcZM24J4Rucz9DUZQ9o++Fxs2KJG7lSEJ9tPsTaRx1pTEWCCQlUU2nKCr5s7UHvZX1xHP0O4\\\\\\\/15nPEviGfUiCTiQmoT9vOmsMc0QxJ76IMU+2SwGcI0lh3rfj2rGLxWfbnXdxFozxhZT5Uh1UOCUBoNvGRrOrPPRhHIKwAapOW+AsatJbTXrlIlhtWX76CbKUGDBvSVso49HZr6UOBaN1YQmQIJPLb5qNQem9Ty4A8r9r3TnHMwvZKQiqn7l6osY7kldTrwMQH0BNduxXVFpZ7ubHEux+ZiaS2yE6vTHvVoIKKnwdu4lXJ7j8tCGpfk6Zr6j+dXoHM0ajhqixE+jYSbT0b2RDmBPc6cMNazo8H+a4DqH\\\\\\\/0UELPeNB5OcIUt5Ljc5W\\\\\\\/JAUM+ZBBjfFmDMdwoNpFbB9oPz1pqUqrPL6BugYrq43fkptK7a6o5Spo13ljlFyvoEGHAfQbXJLiywBEYFtYHwRGV9SVNTKSdfg3dbe6cmn7L7qHe2tszTGlKvTYj4f5XdZ96Lk\\\\\\\/sW9rI\\\\\\\/cgyWs\\\\\\\/TmLL04OuySk3Sypuj1TNI4xIlhOzl8PsNiliwIZsP6Ya0e5H2NpVtlfzOSUa4acDtuhHc3epznati8aKoWdlOGa1DC+BWeEfwmtA8iIJ6w1eTxUQCxwHYEkLQPrIrjfFSv6sXvJuUTUllV7MX0nCicN3ItR9slATcW\\\\\\\/2Kly9F4v1yBKL2khH898GNeHW8tPDhMXCo0XDIPLrK59hcvUCYF89lhogjzH\\\\\\\/baqEYSl\\\\\\\/GsN8MDwGu9demrsqG1zYzD2qgnQLznPh1wfUi9Jt3Kv1voxrwigJbII7q\\\\\\\/3iYwjXCJRF9GD3nXbiV56Gz2ClR4fuZEX6zz+kHPDRflh6kii03LjgZ4U7G77fgIuvYKezxKrXNJtTeSQtIPQOqGXSfUf18+D4EnK8oJmfMkfxkW1oFssf4wsahqKI1b63l4siXiuZBdB\\\\\\\/t8vSs4g9ZucXSMUjkyG0qhwyUxqpTJ3wXgmtsf\\\\\\\/1QnMxpxwbK7VSOXjSpVd\\\\\\\/ZaiBNJe8QtgiBBcXFFglaxPpF15+71I+r58iBF7O6i1u7\\\\\\\/jpJVsNrG7Iv01JAh9pFnsldW2CqHQ5sSshroY\\\\\\\/p9jEL3Bk5seuurYDJsp71XL3UoT\\\\\\\/FqBJFfY6kYiSEjbDGowvIG8c8ayrACDuRr7iZO7apBNHrKSIutoB3SEvPRcGBjPPHl46JLkTyvo1u0Gw6Zthvyk0PU6NIA6F7RuoK+8FPvmwRMJIEGqWZdEcqbs0QPvQAKS0bILQjPQEf0+G2M7z6sogE3syttKaN3hks0RCylHRmg8WBqHA5STAVuW0T6\\\\\\\/w2yaU4Egyu3EZduDvTGp6PLo+yQYh9pl\\\\\\\/m17du9FVJJeZ8Zyl+HQmNZv4nrnrVfqgazUPEJ5a4ZAtL9baZCOCGp6L9WSFhpO\\\\\\\/TGgyfhUuNkos3K5FgsENeCyOavdQMKRYEw3KXbFah96qXOz7kTM8DV82aHx\\\\\\\/tP\\\\\\\/muu2vet5+vI3nfk62KhKFdVPGNZ7yExe\\\\\\\/lpzOY0d1vKna0cA8gyU3kxCVD42gbGiClwzsx040O\\\\\\\/VEWqBOTrHYTP\\\\\\\/15OdgP2fximHoms\\\\\\\/iIjRcmig2bgYSsOAQZ+uIcE9fd1zvGTEplQ7MtEqMVUlrGAwG3A4ZRDCPVoihougO1AyfDX99hxlCiPjHhwFStJHK1ySMprRinsufpGcBbCU7cEOLC9rhS0W6KwDcLtQe67vMSRzrPmOQrsFeeKbStf\\\\\\\/54jzGShhwksuVvtL9S5gJ5Y9Zd74YkFgDV41ZecmSGy8VdMS1PLgqMMrlbSjRGtPvOOq92CwPmY+pUUX7d4pXfHupN6V3sIC+pKOXUen1qIqvv\\\\\\\/mNqnMN2NGmxIA+PqrpcCb9Xf9ojLb9DcuqhiMK4zRK8Dnllrw6y5Sg9rVAxJ0tTfJii5s1O7riW3WznktVRLtvJxsgzB0TOa8DUnfkB46xa2SLeUnRIGgWdSF2FhrTBb\\\\\\\/iOaIPFr0fm3vjEu1DKXKffb3o76SkSF1suEG8lPfR5gH3lOnOUajduP9Kpw68X2I47lHJSn2DmxX32EqYxzYtR2me4+znqbPW4cqmuz5pcPkJ7d0ia45dQwyuG0pyM4QimNRVeVT6S3Y395NKOBvPDgRCmP1VKQf0J+PfkyeL\\\\\\\/QeafUDlA8OXJhTpAoKzDlJxPIft1jsfasjdcQp\\\\\\\/Uz\\\\\\\/BmdmbY4rF7YuUBJJ8B2obhOXyJrGY9wLgufSDh84dvmaU3uIZpvR6Y\\\\\\\/mgLxrgghCBJpgxfnpwWAWOLk8c4UOKrv13K+0Gz36HvSN8xmtz4UsXYJNzFfy6Th7C\\\\\\\/2ZCU9eSLtQ1SkxH+punIo+a\\\\\\\/iNSllg34ZjGNmp5UZjygNbVJmTb919\\\\\\\/ZUB1miASjyR6jlfqtl2be2zxLXFSUyAckv4\\\\\\\/+IxAHEeFjysBxc1FByi0LM6\\\\\\\/fnJXYAXnlmYgNgL+Y9hH3MLodqlYfuuD0XeYio0bZtCEcxdYNDOKZbn\\\\\\\/ffv7H8ukII+D\\\\\\\/m\\\\\\\/95kJ2\\\\\\\/6AXdguymnVS13C5sgk8QGbvhAkjbbAnZSfrcFEzI3EQ46ltcs\\\\\\\/owQLzPf\\\\\\\/WCUWor3VNlX0Pml+mVuEZmRVPTAWwDi5idsFQJ5cRjtNM5foWoBiiPnatEZpaPl56Jlgp3FkoN8eheyQB8P207nDeHUcJC+PK+W19OUeOBkON3GLSjhHvHcwVY8kvGL9PmhaoQlOmYklr7W4IBJz1UrM0AwQRfAajymkO625SUGdVIcTiy35T\\\\\\\/ympG9A0S0QP+3w9pFjRWMMhsyZ8lqP1X3TX9GJe2vUGDzDBoKfl5Pnp6GCAv4W8QNBU80sQ2DZvNdzmtfo0kl0xUoEIFsQV01ghwQEvlLhsYpeIFsZe\\\\\\\/+mpo7uRCHwOoCZ\\\\\\\/EqPdPGcSVXv33+xie7a\\\\\\\/wpMaH2c1l6kUOzOaDDIPRUguWJBncnN4biYOWw6s26d8FzXcn3S8JOzKtJC6LXl7ndVosJ7P+a1mfwK0RoOV2T58GFaZ1sjvMeztHO3dWJFeQcOFH5ZX+Kx1Wpk6tnasCiu0I9WB21Qrnr2pnf2+q1zvgGn4DDIyG2uSVCYX7iLoWEW+VKogkgz1o9XB\\\\\\\/\\\\\\\/vJZ7ZUje+CnZVoIqNPDZA84igqcb3Lml47r2LpBCuOqbNPvgr+Mu8mvQt6GmQH8tkw9BM6E3pRinr6e5I3woXhMkZFlxuNaBix8h5zNrETlS9LLJd\\\\\\\/ALmPZJa9rTqTsDxRzLxHPXesgsX5gfkStGdlujm19ahBTwy4eoYEHTa1MQ\\\\\\\/BDJWMrTegcdULQ6KZYdOMSdX0IhC3V5aBkPph07NrAkirhRRJ\\\\\\\/Fby9qZMVZVCyWCRc6FHDJrXWgirV2OFbWonEHQIv7LSR33vBbejATiktJrKm73wift7fetIqPa9708+ssEYHd6LPlJtQeX+CK\\\\\\\/vUB0I8Z6F7R5Q8VeFxoClbWmTc\\\\\\\/ucipv6D2tcZscbiXfAwxPqqbV6Gm\\\\\\\/d6Pj2sDxGVT1xzpHwoEm5MrTp5OBoOVXd8M0uccRw1QO4qHQ4AAtorenljPThDI5tNMet9E3vEhk4yOhk+GpsqXjknbBvoPOcjLC8ZEYCMeykd\\\\\\\/sS+7ySg3uLnk1n2b23HA6a02S7LuZmVatNyizVAji0GmghODyPaDOsAihE6\\\\\\\/xKwt0NYpYk0vBQAyk\\\\\\\/aut1isplLSNDVkaT6a6y6ezYpmxnXYuNTJi78Xmn0qPOC29ksB8ChVerP7C9ekO4NA4rpO6ApCf4jOL152OrkzyavbB9jb4rI0bF9UWqbgR5QN8VXTUDBWzFi5DYUQxCXhk+MBcDQuaiQ\\\\\\\/X6Z5cdoLvSzG\\\\\\\/0RQ7R\\\\\\\/Gr+NxImFoJRBd11EhCdpA3ATiOm9UcGIjAiUE2\\\\\\\/j8dJ5udgYEqHqvabPz5ACuQ3aL0gK2M7XtuE2egqjeEm5c4cjy7ZeoDkXFR0xQoYzJOhsPuVPjGIRvDYrJguUkA+CfrbCImC2E7FU2dhXId\\\\\\\/77K6LxdXlryoBOgft64fIRNF1tc0ESsZ5kyu3FiYpVoorJMU7Ax5FU+soT\\\\\\\/2Jjb7wT0epoOkT26KrZ6ce1UtH9JU6+8EXBapfd0ktkk6ezGYHzvzAjSe+qDjmA5+AL\\\\\\\/azfsxdCSu\\\\\\\/E35Pcky1LMSCkwL2Q+y87ld953bSqla9PjaXOfA4eGuQbeVj9trasE3Z81svUuN33e0kmgBi7Hgve9yDTl514XnaqkUKz4edbXEQRS62\\\\\\\/Wwcwp2wjxQZ7NExVpGA+vrO+HqeuobsdOlxri\\\\\\\/y6nBQOrjD3qTSETYpMgDuOjAy531wlY17t6Kc6DdBJ7LE5n8p\\\\\\\/ZXi+kFObsj9Kpy9RHYCdv+9dQRlul1OSjlNcSgS6qCWt2s3eOt+ugTuElevzO6GpN8JgV1luctNI+Ab6PCki4u3tmsVPjRU6UcspHshnZTdLjcWs374No\\\\\\\/6k+MAzNDb0bIgzIMEh+EDzTkm5gyqHQ\\\\\\\/oOGa1a\\\\\\\/WZMiq7AoOoKGrTvxhKvYlYQ9WOqVDjXr03nlLnKNaUD24cSTSoV4M0sSU3\\\\\\\/Doe5PKa4cFmcLs7yjUC0WpdvKb\\\\\\\/0Xua6trDsepREtQLZG4NivOFJdktUVD5tcLAqI4Py+c0L2TmxXOgCZOYu3KWavTbpeA2euUhpRZNS5PxYnErkHHgqgHJyhBk5Mqdd3l\\\\\\\/WUnAP8zmJSJKRK55oJlooEhmcwc42MfVvnh1Zbk9shUEzVOuDsa0UwJ9Gy5MyjGN3AU+4u4SvGSCGAj9cZKKqF4wyyPkoH3ulctKmI+RF+oCCLljGUU4rujf7ImZ77K\\\\\\\/UahIgf39MKOq+5D5RpQ9pJKbCbJbrcXdzHJHdoW76SUpp9Nb3V02MLR4\\\\\\\/9U7ezkGY1u1uJ7Q4MwllOMcL7nrBJIdqGxIMAO8chXEy\\\\\\\/b7a3NeFGUwUJhXYpiDtnkux96gP7wV5oa8YZQ6BHqcXhXe\\\\\\\/wjvjF6CxHBGIXsNtVewxE3EG0woMItb52fAT96tmnCUBZ\\\\\\\/9cuYi7h4VKUPmHSPMQKpPwlgNxcCA5aOb+Cax+ngEaFOtDDIXCUo23abVwLn5FSduMilOue9fcn2slW1baNJRr1DI9HiFMnpD6vq9NQKxp+Y7jy0Kcs1VPFNE+ZNRmKss+PvzXfeFn7S+unJnTeUdWwOgjxWXh8vxVJcyXRirE1angZSWHul+UZsTnUWh1eiArlmXmX\\\\\\\/rsxj6omdI8MUxGaBm8UlTq\\\\\\\/gNpYWC\\\\\\\/GT3srMH\\\\\\\/5Ll6gJkUFwSAb\\\\\\\/RhMCRSlSkwbE1H7y0uOukH0LqQ6kprxwfPuMep52LVbeTv3ZOYR9GGrjI8T2qF+yW22IvG3P3XzqH1qYw1UPrAwVVosatU93I1yVklVF4PEef8sgcSlV4mLgahX+v8N7eDqeb\\\\\\\/HbE0ZDfzVIbkqrbWdxLeUmCMGMbTS4OPnegdlDyC0yapL9qNKSOrYIYDf+S2q2tz\\\\\\\/d+aIyoxqkddrvLVuHsiCQJ2d5dECaf9phaZWIHw1BCc4CpKgaLlaLgvPQtQ+bdR9pmp\\\\\\\/rvVIypdr9yVy9sEfwjHAGc3xoPr79Ypf\\\\\\\/7VmCk4Wttst1y+InPVWmVdX8R3J2hM0+Z8UkB6jaVEs+1oXgL6iMC4wQJEl8WtA7BzVJr5x0KvfOyQOmMHtwMXVdWB5TeChMQWvYAxswilmwpEE2NvP2qW7m91ExdSdTKqBzxVscRgoYIvO714kC65BkJMZPmIqCMnehjZJDWb889mcvu5CQfc8jQo0cNOuKgQ7V9uyfT8nfNYylePuIa+\\\\\\\/+28q7yyOZJP0YCms2eKeHJ3rYMuQ5UMzRVVAtUPQK4v3HJ1v\\\\\\\/AymkAZsXwa9S4TGfX2wwDa6mTVs2Pf10IRPs5gwJj7Z2e0IENCMI+cxo+LQ5LUHS2fKRLRx5WEzzsL\\\\\\\/5qa8v4it65VybhTpVcaX9fHkcK0Cae6rSfVNDDTttzmgo+97Bbq7hpPJkUIrpluDnNnuxwLYpZchLbPCpb3RAcClGwewVn0f3krF6VL1QLakhicVccRdQhiadNffr3lCA0za07GzV\\\\\\\/0DExPXVq4rMwQaer5JoZgch8ZCDqGE\\\\\\\/QPA36Y4\\\\\\\/0C6CrDI3EbP6vSD1gHv0w56aUHz0AEdKF79FDRUXoV6JOiGICWhkNFM7EIZNsZJFemxysLtVbVVLQi6Vl3KrqkTaRzEB9C1uiQsUQptCyyhWrGP7MmH3hPKyef194Q83ESL7FWqgXnAZbKwKV4A8TE519ipwNdt+74JLY+n28KN\\\\\\\/7X+zgYRlCDFCxiU1xPbob6Pc6Vp6CSqbfAagyai+YBqV7jSIqXnpEm8MDH37V8IDAV5KQmEccHlJqSlU59cUA8ZmRLc1lTrq21nJTnZlgrQmW1dEx\\\\\\\/+OQ5MI2Mt623T+grPRz25QfK1K52GJb8GevwPh+7ct80xQZ56j+lJPMcV2nG1cp7f0WMsKlpZ5uO5uAVwHjolSlHnSi\\\\\\\/xSF1wqa8M1NnxaQsMOuyxhXaC8KqKgvKz5\\\\\\\/EslRAN0X\\\\\\\/+O3Aasmj9uFL2f5gzOjCz49kiMA8+paFNgso0pxss+lbII3HWUKzRw3mYLu350pFpl33RlWAQXqgz3gqQ31TT11piSB9U5OenC8kn8UNqifFlacX+AR8vwfgDo0+ld2cSiDJtJagEWjUFs6+O1C\\\\\\\/qtISVNxTq6po42xtl\\\\\\\/JxpqVvLarnafxiGDQMagdx5RzcEQHDbSxNNhaScgtcKxU2x5MMi2ndFTTZb4vp4hLjZvL8hwf8dY1sj1OOnjPXd6jgdXnTXJwLaxlCWP6cLVl5Pmecl6ARZR9uvFrvpJ4Mggh6ob61Kg8D1SYQvO\\\\\\\/wtjOoKAI+3Yie6LYYGYkhJVSA1il87bgvqyssdC9v\\\\\\\/eNKkcW\\\\\\\/hjFLJy530U\\\\\\\/5Hl8ZX94+BSAByB+1Yo2\\\\\\\/n2uAmhzzY4O15wEEM125MEo\\\\\\\/jxhliRWqondTDniVFu9tcx5ymgdpGLuP7Fq775PdmRGMEnoq2j8Za6LFFFrYIRiT620btWkV0MXNrWwWD2UHfuddQL+QL1y\\\\\\\/5AJSUqiVsFgJ4wx077CvruOnJtxLrPZ9x\\\\\\\/CrL8TMWKcq9jsk2yuQpxxHo6+QLmgCV2zweHLpe95vTXz0t5bVusNtfxw13T+LanZzKGo2IVM8wEH1UPaGXh\\\\\\\/wgo0hD0R0rOTBy+d6GeqTMqbnaHVswLBFWrT+gynlqmeAFy58JkXkYHYtHyXij\\\\\\\/GCox\\\\\\\/lYelDd7POEqregv2\\\\\\\/mQra9AwdGzX74t1kjCZWLbwCqNBfmDDCPx3P9R2Ndke0tTnnmJ8AAndoNn8J7dueSa92n7Gduyy\\\\\\\/LwI7XqyPH89coopG2Vw0Urkz6eEGC5gHrDlLcPt7P3siD9XSm6xK7KaWdHVXm6s\\\\\\\/PSWlzummxwlWUqn9yVyUE53bxf\\\\\\\/DArq38Flt9B\\\\\\\/EILvTNWEVAIRDtKULmYgrDxaB5Ph9pv+MdW2VIJRx2JjKdw6t9yvkNarS5Pr2xuwZgFiDrSmJl4hoCgVgfCU9s5\\\\\\\/aaidSoGGFIcDpXSTWxIhj9XT9Bpsqv1XKs7p44iqZ71cSIxkRMvJTuzg78aYD0NoYHUmAJ3\\\\\\\/PlWZ5LOoiYMjWL0fZ3w3wmD6G1SCHooLQyL0I8h2eWs4G7tH0Im+EgqoVoTkKBBA53Jr5L8jdO9jB1b8Gr8pvC+bPDYpUGvY8lgJBP138eXpg4K8kNoF\\\\\\\/SNZQVuu9KWMniRoNJvfBDMhAdK7I44BOEu0B+bhX2q2hp0fOpUarKa15qZZAvSfw2JiryhKR48Bmz0ZLPTxa7wFsemrdb+aD5MDPPALEQ6kBDhT3HFV2Hck\\\\\\\/M21FGGtja\\\\\\\/uu4BGKLlUsWF2r5tZ2RNFwuIBmy6fbkivSY7GNCvU4nn3eZn\\\\\\\/rib\\\\\\\/GjQqEnajD0Yak1JHDm\\\\\\\/w7ACHJGomo7tj3xvayF1UurRN2id4xKLDItzw62n1WeEvJJN2sINQ1YVlxEaNKc\\\\\\\/Wun9QBRchbtX3j4JMO93IUxIHiderscmMIjW8MoTEbYDo4xo9DsLcB3qZyiy4MI5nyqbZsy8JeRwP13nxpRMKUdWEHeFRPqs7kmbQXX2e2qkDIO5Z9djJu8dA8UmbI8n2xFkmFkJsfRCrMS9\\\\\\\/K2MZTDYvOB6PzJlya1PMBNFIH2smHmtIkRqeVWHpfo97VP7R6OSOvR\\\\\\\/r9PJFhm6i7Y5jgIzJM4ZilHXZ\\\\\\\/hM\\\\\\\/3\\\\\\\/1vl9Oc+vgdny\\\\\\\/TvzaLw9K8+lsitKo\\\\\\\/2BVj75ecPJhZLOeWlX6GvBX+eiYR9\\\\\\\/h04atgRUjnISy9\\\\\\\/whcaY6QICPWJC2hBcrrbka6AzLX37xNfYXYkN+zxXGWCqptigp3ikajpGUEFEC4+qdjet0LJmwWvoTfSihN7tZUer0ML\\\\\\\/t3P8Wtkhu7qUqkwqTLvIfkTDaqgArok4WZirer32UlkKN7+lae8lUExGBOqLDmwXM4JqblIppE1d4O6s9dbV1Tr2fQJK0iiyu8mlRKKVqKAn4Sn\\\\\\\/wjJs5iDn1UoH++1u3AmRUay3sj5ODvpPN4eQ5GVJUi\\\\\\\/VVW63d5WYPazaMjZrTNiqn4dFoDvGtXcEpNGnrBbCB6ouSEIKxdNEPRceqGiQYj1nVvmU8Y7xxd1t80Y10g\\\\\\\/4vYmbYW4Nmo+HcUnUGchPmx5seVfGtrnAuu+L+6SdH0wXV7QKpCiLWloop3\\\\\\\/VI+mYh4S7t+AAwPIB5\\\\\\\/SiMK2Yp5RETaVBnP55MeXE6FK5BiyZxCf04NqgwYHVeGv1O4bx6DamPYgcrx48enBV7dWBHhImITt4\\\\\\\/LctWFwaYaLpOP0t2COgg6rQ2o0SH6ztRSUcCicklcikqmGJRKva6JE+v4ayDtQQSoG5ojwkNZsAiyN0ov4Mq\\\\\\\/O9VHKdM35tc7hdAh+GeIZ3aX+bxG\\\\\\\/0bqyQhBQirnKuPd5FMLm65V9ufa9eloDvh4ehPe7mENn5\\\\\\\/Us1ZiuNYe72VJaCvJPtPVfAqzg4fmj8qZHnt\\\\\\\/9tCvhteC\\\\\\\/NY1j4xPtrC4dEZh1bP5NIlfFx6CW\\\\\\\/YKAINDzXPdFrCIg9RaVZ5lJm1HYhl4meBzAZLtVwJPtttSp7kAndBkxQquAPAUfKpJcr8Y+RaDf7+2we9gPW2geQn92UYafT\\\\\\\/52S0\\\\\\\/MI2c55Xvau\\\\\\\/iB\\\\\\\/uNk5SrBs7lDtgCPzN6Z1gr03CpvjTdKljUTnoeJCCaBAJIkesvDI10pV88fVO\\\\\\\/uRqg8uwxkMxQdeJMkyhAKbIv0C3vcCDUk2BrLQfHSq20R\\\\\\\/li0csq9uohmkHg6pylWiijyk19Nz3XSjcCTWdpm4iT5aKJqDi7PtqAy2zv+3kESjNoAWdgT8++VtRa4bnZxG9b7VZDJY52WgfJS5B0IvozIQna5lLoYN9TFLC3fMqzk5Ahcj8YZUh34v+T6TFCb+uvXOLvmFi5o9AG4cFtJHqabbJ+P3U1czYdrfL1Ywf7T6fASo5nP58K+hYm0DyJfolhkeE\\\\\\\/zdflzklju\\\\\\\/2hM1wQssptNI4B08AanBRE7tmxd6T5GBzOR8mg2eA7hAGzyBX6chIIha5DbBrP7aeJPrnYwfi0blQbGScA8A0cSmgGDZpP7yenUtah56qt2AVxFuqZcB0eLAO780JSzmRG5vvmru0j6qnUnU69IJTJxB4De+tI\\\\\\\/UvzC9B1RDUNLhAyGw13sf+kbSB988UbTEEA1xUu6Gr9NLjBxs9IBtXs9w6KIIQYzOUvml6qLmaW6dRqXnFLIp8QNCQOJuIv2Wdhi4s9sZmnwdfMFnff1bJ5cstFDhaSZjYB\\\\\\\/LfFJNBuphchquJnsfhbMUBBabJa+2hwz6G074hq3r4NRhMGZ9EQvS73lkkFM3ov26Hfeu3jPbPe53DFcTV\\\\\\\/ZidufrWcaETNpwN7VSq3V1ZF\\\\\\\/S9Vw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022zVoXksUV\\\\\\\/wtw5n5b0TydFw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d043a3135707b692376a7284d28dff76642310d7cbaf6a2ec565fa79267ec5b6\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Cc3NzHdQUMKaJwtX73hReOOrodjLyV1guiqNZlVHYwiLKl0kqdYAutOMfSYuqgsb9pZTifWJHQdjsTpElmWPkT331Pj5+2o247UHD5kb4Euz3AjIN7Xgor\\\\\\\/+FrgRuouZ0RCCxDMLUMDhYh+JFz2XrZnIyOuu8BfYeu74dqAf35aYT+vUMatRUSkAGEQtYevC8t\\\\\\\/cKYmKhxUYPsuBibfDtJzgFb1eEpH6I2Zz0SAmMsubNwYxGoccpgxJyjLgXY\\\\\\\/jxRwaGCVooDXquPTUdlu07D\\\\\\\/ak7aCm2c8Hj\\\\\\\/dmf3KGazG7cN6dc2y2whhAYczkRZmw1lQ+OS+cflw4CM9RDehxJh2bn++7GBJXiTDd2G4ERcs82BThS9k\\\\\\\/vTOu024Fj9HNlrbUmGRTu3VIOWbrGVSBhlpv\\\\\\\/Q781C\\\\\\\/Frt4KUcTj9ZhsM8Bky47BL54FLMCeP\\\\\\\/lH\\\\\\\/ziQ640kscniROjQDXjvsCVXKaQsiRRorHmA2JxB5zowZoC5iRX9w0HcSioht13FlWtpgIPhFBhAULs4pL5iN3N\\\\\\\/jvDYKokWlwkV+Ra\\\\\\\/0Wt2YPU6qpCJbaGp12phumogW6kVnikCo4GPry9C8y20m4j6Ab+Grir4FHFo1vkTDU3DFEEPiDjhJjYtrHbMhDqiDCl+NrqfqhwlqoGgo5C6Slfb2\\\\\\\/6D+PzAeX4FcDHTo5b7H\\\\\\\/5gQ9uZMYFN0s4HmQQf+oJTsvdjxZMo7Ou1hez0xchhEVMOyWDxt0kWi6nX1uQMDrBxPc11CyGf4syjo2haiaV8KvivL12f5rz247iatF15b4ivCLIRkKlWcOqT+5RNg2BShDiic0ZyOAQBsF5RiBB35cvYLRwvcod6tBDiXhHz4MwFxE3bP+z1f4mH85HomiUVhAQ4Z+9mQUdHu1ncB6eBnQuFW4AJLWh8n0id3F\\\\\\\/nB\\\\\\\/ic+NNTM4Z9chnNMLo1MVhxFOGSkOdQv+yI43dv6xQv+bov1VdvIoLuxfQWKgMor8p\\\\\\\/1eDFtzicWrMj2HOZ8kbh8XHSBkoCP9SGRo\\\\\\\/JnODtYC7Sx5tbLy78wnG5yvoigK4IWXjzgrEsVgDRvwq8PU6xVvHtHItTcSzfTznMo\\\\\\\/eANKK89dSwCTd1hmtizt0AaEaEKLvfLyd0cwXrdApmZJxV2tvnlj2Cm03vadZScomXmPxT+OYlullNHNyomzyVR35kiTH0XSGsxVb0RTQ4m0sbslx0S18ZpvxBDBd5S1aOgP3dDxqTzzeHqnJ5H5z9txHioONjx8Vl+yj\\\\\\\/ESwCxJoO1eC1m51kKLnZSqumrw\\\\\\\/eismeTj7DooNr79bpFsd7Ysc48IdE43UNympPoi0cdNEEGBOpsJHcsLh6rZlJLb\\\\\\\/rOOJu9+DYAohtYVsiJPVFu9eKc+HCmDeou1Q3aqd1yZSYaDaV4cZvSaYnASiev0ZG81cIqAhhl2vr0LahU+NSg+dCP2ZTXxfd38qiwVyn99KiuPUEGRvggd1IFHoIOucjtQyFhL4z88hgyNUAs9EG4O661qQm7BX7Ic5QRZiVZZsb+rExvLOT5y9sKi\\\\\\\/b1B2pGIvU6DKcr8OtGxOUM3Qg0C5d7hKTRbjVuizUl+38vn2mmcsEv8m2YWnYX8qOfZs3hYwhr+P+So7hTAGBvANjOFLRmu9FifgxUFnvSJvtb5l4ogQxerDqwdpi4OfwGvk92c3ad3z8o\\\\\\\/IrNEUKpnTesS5uhtlDFGeAsmgktwyKrEkpgXkiV9ChHcRICsKpkxIPn07Jl2lpvRjjUD\\\\\\\/8tLtY\\\\\\\/53foB108vv8zvG1egHIFRqozM4AixGPWUKTVP4Y9AGaj5g24lIWyypw17d36iDEruXr15bRtFvWTtrjY\\\\\\\/8+Hyim3NkaauPwkiCbjG8u0E3D2M4TvtGKqgW0nxsPV\\\\\\\/YQvGS4+Cqp+8rb2JwZpmqJkXF5TCXGblssBt+SoSWVxJV20xqIsYhKRm78keNcG+drzCByt2kvQfSn6wpsxujkxlTinpnYV9SntlsxLwCLB+oEilK93sibB5j8RVyYg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022EfjrCE0PEeKPmo9pAZ4OpQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220c4e5f2681c855af59c9f363fd8ae8cee3a370554dbe850263ec18293a0855f8\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459244 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/2/261" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-HWDuR1pvdnB4QmxobI_hJ58wvOeAOsFvXUbjzsOtl9U" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/2/261" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-IlDDG1Us4D4iErE9cePLgHv9PmpLvenh1GrzBUDnxAQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/2/249" title="A Whole-Chromosome Analysis of Meiotic Recombination in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/2/271" title="Narrowing Down the Mapping of Plant Sex-Determination Regions Using New Y-Chromosome-Specific Markers and Heavy-Ion Beam Irradiation-Induced Y-Deletion Mutants in Silene latifolia" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459244" id="node459244" data-pisa="ggg;2/2/261" data-pisa-master="ggg;2/2/261" data-apath="/ggg/2/2/261.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole-Genome Sequencing of <em>Sordaria macrospora</em> Mutants Identifies Developmental Genes</h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Minou Nowrousian</span>, <span class="highwire-citation-author" data-delta="1">Ines Teichert</span>, <span class="highwire-citation-author" data-delta="2">Sandra Masloff</span> and <span class="highwire-citation-author" data-delta="3">Ulrich Kück</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">261-270;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001479 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459244" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Minou Nowrousian</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Lehrstuhl für Allgemeine und Molekulare Botanik, Ruhr-Universität Bochum, 44780 Bochum, Germany</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Minou%2BNowrousian" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nowrousian%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMinou%2BNowrousian" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ines Teichert</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Lehrstuhl für Allgemeine und Molekulare Botanik, Ruhr-Universität Bochum, 44780 Bochum, Germany</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ines%2BTeichert" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Teichert%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AInes%2BTeichert" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sandra Masloff</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Lehrstuhl für Allgemeine und Molekulare Botanik, Ruhr-Universität Bochum, 44780 Bochum, Germany</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandra%2BMasloff" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Masloff%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandra%2BMasloff" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ulrich Kück</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Lehrstuhl für Allgemeine und Molekulare Botanik, Ruhr-Universität Bochum, 44780 Bochum, Germany</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ulrich%2BK%C3%BCck" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=K%C3%BCck%20U&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AUlrich%2BK%25C3%25BCck" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ulrich.kueck@rub.de" class="" data-icon-position="" data-hide-link-title="0">ulrich.kueck@rub.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/2/261" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459244" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459244/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/2/261.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459244" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459244/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/2/261.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459244" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459244/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/2/261.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459244" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459244/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/2/261.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459244" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459244/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">The study of mutants to elucidate gene functions has a long and successful history; however, to discover causative mutations in mutants that were generated by random mutagenesis often takes years of laboratory work and requires previously generated genetic and/or physical markers, or resources like DNA libraries for complementation. Here, we present an alternative method to identify defective genes in developmental mutants of the filamentous fungus <em>Sordaria macrospora</em> through Illumina/Solexa whole-genome sequencing. We sequenced pooled DNA from progeny of crosses of three mutants and the wild type and were able to pinpoint the causative mutations in the mutant strains through bioinformatics analysis. One mutant is a spore color mutant, and the mutated gene encodes a melanin biosynthesis enzyme. The causative mutation is a G to A change in the first base of an intron, leading to a splice defect. The second mutant carries an allelic mutation in the <em>pro41</em> gene encoding a protein essential for sexual development. In the mutant, we detected a complex pattern of deletion/rearrangements at the <em>pro41</em> locus. In the third mutant, a point mutation in the stop codon of a transcription factor-encoding gene leads to the production of immature fruiting bodies. For all mutants, transformation with a wild type-copy of the affected gene restored the wild-type phenotype. Our data demonstrate that whole-genome sequencing of mutant strains is a rapid method to identify developmental genes in an organism that can be genetically crossed and where a reference genome sequence is available, even without prior mapping information.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/next-generation-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-next-generation-sequencing" rel="nofollow">next-generation sequencing</a></li><li class="kwd"><a href="/keyword/developmental-mutants" class="hw-term hw-article-keyword hw-article-keyword-developmental-mutants" rel="nofollow">developmental mutants</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/sordaria-macrospora" class="hw-term hw-article-keyword hw-article-keyword-sordaria-macrospora" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Sordaria macrospora</em></a></li></ul><p id="p-6">The analysis of mutants with a distinct phenotype that were generated, for example, by random mutagenesis, is a powerful approach to identify factors essential for biological processes. This so-called forward genetics approach is important both in basic research as well as for medical applications, <em>e.g.</em> for the identification of mutations that underlie diseases. However, identifying the mutation that causes a phenotype is often time-consuming and requires additional resources, <em>e.g.</em> previous knowledge about genetic/physical markers for genetic linkage analysis or genomic/cDNA libraries for complementation experiments. In recent years, so-called next-generation sequencing techniques were developed that allow a massively increased sequence throughput at greatly reduced costs (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Metzker 2010</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Nowrousian 2010</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Shendure and Li 2008</a>), which now enable a different strategy, namely the sequencing of mutant genomes to identify affected genes. Strategies for mutation identification through whole-genome sequencing have been used for several model organisms, including <em>Schizosaccharomyces pombe</em>, <em>Caenorhabditis elegans</em>, <em>Arabidopsis thaliana</em>, <em>Neurospora crassa</em>, the rodent malaria parasite <em>Plasmodium chabaudi</em>, and humans (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Flibotte <em>et al.</em> 2010</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Irvine <em>et al.</em> 2009</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lupski <em>et al.</em> 2010</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">McCluskey <em>et al.</em> 2011</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Pomraning <em>et al.</em> 2011</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Puente <em>et al.</em> 2011</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Roach <em>et al.</em> 2010</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Sarin <em>et al.</em> 2008</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Schneeberger <em>et al.</em> 2009</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Zuryn <em>et al.</em> 2010</a>). This indicates that this approach is feasible even for large eukaryotic genomes; however, the strategies used in these studies often included previous mapping of the mutation to a small genomic region through the use of single nucleotide polymorphism (SNP) markers, crosses of the mutant strain to a strain with a different genetic background, or resequencing of large numbers of candidate mutations to distinguish between true mutations and errors in the sequenced or the reference genomes. Therefore, the studies have been largely restricted to organisms in which genetic and/or genomic resources that assist in mapping are already available.</p><p id="p-7">The aim of our study was to determine whether whole-genome sequencing of mutant strains is a useful approach to identify developmental genes in a filamentous fungus for which such resources are not available. Our experimental system is <em>Sordaria macrospora</em>, a model organism for the analysis of fungal multicellular development and meiosis (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Engh <em>et al.</em> 2010</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kück <em>et al.</em> 2009</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zickler 2005</a>). A set of genetic and molecular tools has been established for this fungus (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Nowrousian <em>et al.</em> 2005</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Pöggeler and Kück 2006</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Pöggeler <em>et al.</em> 1997</a>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">2003</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Walz and Kück 1995</a>), and more than 100 developmental mutants were generated through ultraviolet or ethyl methanesulfonate (EMS) mutagenesis (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Kück <em>et al.</em> 2009</a>), seven of which have already been characterized using classical complementation analysis, leading to the identification of novel developmental genes (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bloemendal <em>et al.</em> 2010</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Engh <em>et al.</em> 2007b</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kück 2005</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Masloff <em>et al.</em> 1999</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Nowrousian <em>et al.</em> 1999</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">2007</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Pöggeler and Kück 2004</a>). Recently, the <em>S. macrospora</em> genome was sequenced and assembled from next-generation sequence reads (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Nowrousian <em>et al.</em> 2010</a>); however, SNP maps, other large-scale physical or genetic mapping data, or characterized strains with different genetic background are not available for <em>S. macrospora</em>.</p><p id="p-8">In this study, we describe the identification of the causative mutations in three developmental mutants via a strategy involving whole-genome sequencing of pooled DNA from 40 single spore isolates per strain of three mutants and the wild type, in addition to subsequent bioinformatics analyses. The data presented here demonstrate that this strategy allows rapid identification of the developmental genes affected in the mutant strains, is feasible for an organism that can be genetically crossed and for which a reference genome is available, and does not require prior mapping information.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains, culture conditions, mutagenesis, and genetic crosses</h3><p id="p-9"><em>S. macrospora</em> strains (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) were grown on corn meal agar as described previously (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Esser 1982</a>). Generation of developmental mutants by EMS mutagenesis was as described by <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Pöggeler and Kück (2004)</a>. For crosses, strains were inoculated on opposite sides of a Petri dish, and asci were collected from perithecia formed in the contact zone of the two mycelia. A complete crossing history of the mutants used in this study is presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001479/-/DC1/001479SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001479/-/DC1/FigureS1.pdf">Figure S1</a>. For transcript analyses, strains were grown in minimal medium as described (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Nowrousian <em>et al.</em> 2005</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461185/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461185/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 </span> <span class="caption-title"><em>S. macrospora</em> strains used in this study</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>DNA preparation and Illumina/Solexa sequencing</h3><p id="p-11">For the preparation of pooled DNA from 40 single spore isolates, all strains were grown separately, then equal amounts of mycelium (equal weight after removal of medium by filtering) were combined, frozen in liquid nitrogen, and pulverized for subsequent DNA extraction as described previously (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Nowrousian <em>et al.</em> 2010</a>). Genomic DNA (5 µg) was used for Illumina/Solexa sequencing with a GAII or HiSeq at GATC Biotech (Konstanz, Germany). Reads of 50 or 76 bases were obtained from both ends of 2-kb fragments using a mate-pair sequencing strategy (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Korbel <em>et al.</em> 2007</a>). For DNA from spores representing the wild type and pro23/fus, respectively, one GAII lane of mate-pair reads was obtained for each sample. For DNA from spores representing fus, one GAII lane of single reads and two lanes of mate-pair reads were obtained, and for DNA from pro44, one HiSeq lane of mate-pair reads was obtained (for overview of read numbers, see <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461244/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461244/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2 </span> <span class="caption-title">Summary of sequence reads generated from mutant and wild-type samples</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Cleaning of raw sequence reads, mapping of reads to the reference genome, and analysis of sequence variants</h3><p id="p-13">Sequence reads that contained undetermined bases (“N”) were removed using custom-made Perl scripts. Reads lacking a proper mate after the cleaning process were collected in separate files and treated as single reads in the subsequent mapping. The cleaned reads were mapped to the <em>S. macrospora</em> reference genome (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Nowrousian <em>et al.</em> 2010</a>) using the Burrows-Wheeler Alignment tool (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Li and Durbin 2009</a>). The resulting SAM files were processed using the SAMtools pileup and filtering functions (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Li <em>et al.</em> 2009</a>). For a detailed description, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001479/-/DC1/FileS1.pdf">File S1</a>. To identify putative mutations that were present in one or two strains only, pileup results were processed with custom-made Perl scripts. To analyze regions present in the reference genome but not covered in the sequenced mutant strains (putative deletions), a consensus sequence for the sequenced strains was extracted with pileup (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Li <em>et al.</em> 2009</a>), and subsequently screened for regions not covered in the mutant strains using custom-made Perl scripts. Another approach to test for putative insertions, deletions, or inversions is to search for deviations from the expected distance of paired reads. This was performed using the SAM files from BWA and custom-made Perl scripts (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001479/-/DC1/FileS1.pdf">File S1</a>).</p></div><div id="sec-5" class="subsection"><h3>RNA preparation and expression analysis</h3><p id="p-14">Strains were grown under conditions that promote sexual development. RNA was prepared and reverse transcription was performed as described previously (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Nowrousian <em>et al.</em> 2005</a>).</p></div><div id="sec-6" class="subsection"><h3>Cloning of plasmids for complementation of mutant strains</h3><p id="p-15">Plasmids for complementation of mutant pro44 were generated by standard cloning techniques, or by homologous recombination in yeast as described by <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Colot <em>et al.</em> (2006)</a>. Plasmid pIG3146-37 contains the <em>pro44</em> (<em>SMAC_03223</em>) open reading frame (ORF; including introns), and 1.1 kb of upstream as well as 0.8 kb of downstream genomic regions in pDrive (QIAGEN, Hilden, Germany). Plasmid pIG3147-1 contains the <em>pro44</em> ORF under control of the <em>A. nidulans gpd</em> promoter and <em>trpC</em> terminator in vector pEHN8, which contains a hygromycin B resistance cassette for selection in <em>S. macrospora</em> (I. Teichert and U. Kück, unpublished data). Plasmid pRSnat-pro44-NA contains the <em>pro44</em> (<em>SMAC_03223</em>) ORF (including introns), and 1.5 kb of upstream as well as 1.5 kb of downstream genomic regions in pRSnat, which contains a nourseothricin resistance cassette for selection in <em>S. macrospora</em> (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Klix <em>et al.</em> 2010</a>). Plasmid pRSnat-pro44-OE contains the <em>pro44</em> ORF under control of the <em>A. nidulans gpd</em> promoter and <em>trpC</em> terminator in vector pRSnat. Plasmid pEHN5-tih for the fus mutant complementation contains the <em>tih</em> ORF under control of the <em>A. nidulans gpd</em> promoter and <em>trpC</em> terminator in plasmid pEHN5 carrying <em>hph</em> as a selectable marker (Teichert and Kück, unpublished).</p></div><div id="sec-7" class="subsection"><h3>Transformation of <em>S. macrospora</em></h3><p id="p-16"><em>S. macrospora</em> was transformed as described (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Nowrousian <em>et al.</em> 1999</a>) with the following modifications: Protoplasts were generated from vegetative mycelium using 40 mg/ml Glucanex 100 G (Novozymes A/S, Bagsvaerd, Denmark) and 1.4 U/ml chitinase (ASA Spezialenzyme GmbH); transformants were selected on 80 U/ml hygromycin B or 50 U/ml nourseothricin. Plasmid pIG3146-37 does not contain a resistance marker for selection in <em>S. macrospora</em> and was therefore co-transformed with plasmid pEHN1 that contains a Hygromycin B resistance cassette (Teichert and Kück, unpublished).</p></div><div id="sec-8" class="subsection"><h3>Accession numbers</h3><p id="p-17">The sequence reads that were generated in this study were submitted to the NCBI sequence read archive (accession number SRA026818.1). The updated <em>S. macrospora</em> genome assembly (version 02) was submitted to the EMBL database (ENA) and is available under accession numbers CABT02000001-CABT02001583 and at <a href="http://c4-1-8.serverhosting.rub.de/public/">http://c4-1-8.serverhosting.rub.de/public/</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Sequencing of pooled genomic DNA and mapping to the reference genome</h3><p id="p-18">The pro23, pro44, and fus mutant strains used in this study were generated in a previous mutagenesis program (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Kück <em>et al.</em> 2009</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Masloff <em>et al.</em> 1999</a>). The mutants pro23 and pro44 are sterile; they form only fruiting body precursors (protoperithecia) but no mature perithecia or ascospores. Mutant fus (<em>fusca</em>, Latin for brown) is fertile but generates light-brown instead of black ascospores and secretes a red-brown pigment into the medium. During the initial mutant screen, mutant phenotypes were tested for Mendelian segregation via backcrosses against the wild type, and single spore isolates were used for further analyses. When we initiated this study, each mutant had been backcrossed several times (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001479/-/DC1/FigureS1.pdf">Figure S1</a>), and the successive rounds of crosses should already have reduced the mutational load acquired during mutagenesis.</p><p id="p-19">However, to ensure that the genomic DNA for whole-genome sequencing was relatively homogenous, and that we would be able to distinguish between true mutations and sequencing errors, we used the following strategy (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>): Mutants pro23 and pro44 were crossed against the fus mutant (sterile mutants of the homothallic <em>S. macrospora</em> are usually fertile when outcrossed against a strain carrying the wild type-allele of the affected gene). Recombinant asci from these crosses can be readily distinguished because they contain four black and four light-brown ascospores. From each cross, 40 single spore isolates with the phenotype “light-brown spores, sterile” corresponding to the respective genotypes pro23/fus and pro44/fus were isolated. In addition, 20 single spores exhibiting a “black spores, fertile” phenotype corresponding to the wild-type genotype were isolated from each cross. Pooled genomic DNA from the 40 spore isolates from each of the three genotypes was used for Illumina/Solexa sequencing.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/261/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Strategy for whole-genome sequencing of pooled DNA from mutants and wild type. Mutants pro23 and pro44 were crossed against the spore color mutant fus. Single spore isolates derived from black and light-brown ascospores were screened for fertility and color; for both crosses, 40 spore isolates with a sterile/light-brown phenotype were chosen to represent double mutants pro23/fus and pro44/fus, respectively. For the resequencing of the wild type, 20 isolates with a fertile/black phenotype were chosen from each cross. The pooled DNA from 40 spore isolates for each genotype was used for sequencing. As it turned out during the analysis, mutant pro44 had acquired a second, unrelated mutation that also led to sterility. Therefore, the mutant name is given in brackets to indicate that this sample was no longer used for the analysis of pro44 but only for the analysis of the fus mutation. For the pro44 analysis, the mutant was outcrossed again to obtain a clean pro44 strain; this strain was used for crossing to obtain single spore isolates for sequencing according to the same principle (Figure S2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1958445924" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Strategy for whole-genome sequencing of pooled DNA from mutants and wild type. Mutants pro23 and pro44 were crossed against the spore color mutant fus. Single spore isolates derived from black and light-brown ascospores were screened for fertility and color; for both crosses, 40 spore isolates with a sterile/light-brown phenotype were chosen to represent double mutants pro23/fus and pro44/fus, respectively. For the resequencing of the wild type, 20 isolates with a fertile/black phenotype were chosen from each cross. The pooled DNA from 40 spore isolates for each genotype was used for sequencing. As it turned out during the analysis, mutant pro44 had acquired a second, unrelated mutation that also led to sterility. Therefore, the mutant name is given in brackets to indicate that this sample was no longer used for the analysis of pro44 but only for the analysis of the fus mutation. For the pro44 analysis, the mutant was outcrossed again to obtain a clean pro44 strain; this strain was used for crossing to obtain single spore isolates for sequencing according to the same principle (Figure S2).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/261/F1.medium.gif" width="440" height="374"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/2/261/F1.medium.gif" width="440" height="374"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/261/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/261/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461301" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-20" class="first-child">Strategy for whole-genome sequencing of pooled DNA from mutants and wild type. Mutants pro23 and pro44 were crossed against the spore color mutant fus. Single spore isolates derived from black and light-brown ascospores were screened for fertility and color; for both crosses, 40 spore isolates with a sterile/light-brown phenotype were chosen to represent double mutants pro23/fus and pro44/fus, respectively. For the resequencing of the wild type, 20 isolates with a fertile/black phenotype were chosen from each cross. The pooled DNA from 40 spore isolates for each genotype was used for sequencing. As it turned out during the analysis, mutant pro44 had acquired a second, unrelated mutation that also led to sterility. Therefore, the mutant name is given in brackets to indicate that this sample was no longer used for the analysis of pro44 but only for the analysis of the fus mutation. For the pro44 analysis, the mutant was outcrossed again to obtain a clean pro44 strain; this strain was used for crossing to obtain single spore isolates for sequencing according to the same principle (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001479/-/DC1/FigureS2.pdf">Figure S2</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">As it turned out during the analysis, mutant pro44 had spontaneously acquired a second, unrelated mutation also resulting in sterility. Spontaneous mutations occur with varying frequencies in fungi (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Drake 1999</a>). Therefore, the pro44/fus sample was analyzed only with respect to the fus genotype, and is from this point denoted as “fus.” For the pro44 mutation analysis, pro44 was outcrossed again to obtain a clean pro44 strain, and this strain was used in a subsequent cross to obtain single spore isolates for sequencing, according to the same principle as described above (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001479/-/DC1/FigureS2.pdf">Figure S2</a>). Consequently, the sequenced DNA pools correspond to the four samples wild type, pro23/fus, fus, and pro44 (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-22">For each of the four genotypes, one or two lanes of mate-pair reads were obtained, corresponding to a 40- to 100-fold coverage of the 40 Mb <em>S. macrospora</em> genome (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Mate-pair sequencing was used to facilitate detection of larger insertion/deletion events (indels) by analyzing the distance between mapped reads of a pair (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Korbel <em>et al.</em> 2007</a>). We chose this sequencing strategy because our previous analyses of strains from EMS mutagenesis showed that the causative mutations range from point mutations (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bloemendal <em>et al.</em> 2010</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Engh <em>et al.</em> 2007b</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Pöggeler and Kück 2004</a>) to deletions of several kilobases (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Nowrousian <em>et al.</em> 2007</a>). Before mapping, reads with ambiguous bases were removed, which left 23 to 86 million usable reads per sample (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). Reads were mapped to the <em>S. macrospora</em> reference genome using BWA (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Li and Durbin 2009</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001479/-/DC1/FileS1.pdf">File S1</a>). From each sample, 60% to 87% of the reads mapped to the genome (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>), consistent with results in other systems, <em>e.g.</em> 71% in a <em>C. elegans</em> mutant sequencing project (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Sarin <em>et al.</em> 2008</a>).</p></div><div id="sec-11" class="subsection"><h3>Identification of mutations in the sequenced strains through bioinformatics analyses</h3><p id="p-23">The four sequenced DNA samples correspond to the pro23/fus, fus, pro44, and wild-type genotypes. Therefore, the causative mutation for the pro23 phenotype should occur with 100% penetrance, <em>i.e.</em> present in all reads covering the corresponding position(s), in the pro23/fus sample only, but not in the pro44, fus, or wild-type samples. The same is true for the causative mutation in pro44, whereas the mutation causing the fus phenotype should be present with 100% penetrance in the pro23/fus and fus samples, but not in the wild-type or pro44 sample. Deviations from the reference sequence that are present in only a few reads of one or more strains are most likely caused by sequencing errors or represent mutations that are present in the genomes of some of the single spore isolates used for sequencing but are not the causative mutations for pro23, pro44, or fus. Therefore, we analyzed the mapping results for deviations from the reference sequence in only one sample, in combinations of two samples, or in all samples (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001479/-/DC1/FileS1.pdf">File S1</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461233/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461233/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3 </span> <span class="caption-title">Summary of small sequence variants detected for the sequenced genotypes when compared with the reference genome</span><div class="sb-div caption-clear"></div></div></div><p id="p-25">In principle, mutations can be caused by a number of genomic differences, <em>e.g.</em> SNPs and small indels of a few bases, or structural variants. The latter include large indels, inversions, more complex rearrangements involving combinations of insertions, deletions and inversions, and translocations. Different forms of aneuploidy can also cause mutant phenotypes, but many forms of aneuploidy as well as translocations of chromosome arms result in problems during meiosis. Therefore, it seemed unlikely that these types of mutations would be the underlying mutations here because all of our mutants form normal asci when outcrossed indicating meiosis is unaffected (data not shown). Thus, we looked for the following types of variants: SNPs and small indels of a few bases, and larger insertions or deletions in the range of several kilobases.</p><p id="p-26">To identify mutations caused by small sequence variants, sequence data were screened for SNPs and indels of &lt;4 bases that had a read coverage of at least 40% of the average coverage for the specific strain, and occurred in one sample only, or in the case of the fus mutant, in two samples (pro23/fus and fus; <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). The coverage threshold was applied because random verification of several putative mutations as well as coverage and variant frequency analysis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001479/-/DC1/FigureS3.pdf">Figure S3</a>) indicated that regions with low sequence coverage did not allow for reliable detection of variants; these regions are probably difficult to sequence, <em>e.g.</em> because of repetitive sequences, or the formation of secondary DNA structures. A similar strategy was used successfully in a <em>C. elegans</em> mutant sequencing project (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Flibotte <em>et al.</em> 2010</a>). The resulting putative sequence variants were screened for 100% penetrance, <em>i.e.</em> all the reads in the sample contained the SNP/indel, and none of the reads in a nonrelated sample carried this specific mutation. In the case of the fus mutant, exactly one mutation was found that satisfied these conditions, and this turned out to be the causative mutation (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>; see section <em>Analysis of the causative mutation in mutant fus</em>). For the pro44 mutant, seven mutations were detected that satisfied these conditions, and only one of these changed the coding region of a gene and was established as the causative mutation (see section <em>Analysis of the causative mutation in mutant pro44</em>).</p><p id="p-27">The low number of reliably detected SNPs and small indels was not the result of insufficient coverage or lack of sequencing quality because we detected 1193 variants present in all sequenced samples that presumably represent errors in the reference sequence. Most of these variants were single-base indels within sequence regions consisting of repeats of one base, and therefore they most likely represent homopolymer errors introduced through 454 sequence reads (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Balzer <em>et al.</em> 2011</a>) that were used for the assembly of the reference genome (<a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Nowrousian <em>et al.</em> 2010</a>). Polymerase chain reaction (PCR) amplification and sequencing of several cases confirmed the sequence error in the reference genome (data not shown). The 1193 errors were corrected; and in addition, the wild type mate-pair information was used to connect contigs, reducing the contig number in the <em>S. macrospora</em> genome from 4781 to 1583. The improved genome assembly was released as genome version 02 (acc. no. CABT02000001-CABT02001583, <a href="http://c4-1-8.serverhosting.rub.de/public/">http://c4-1-8.serverhosting.rub.de/public/</a>).</p><p id="p-28">Because our small variant analysis did not detect the causative mutation for pro23, we searched for larger indels in the sequenced samples. We used the following two strategies for this purpose: (1) we identified regions that were not covered by sequence reads in the pro23/fus sample but were covered in the other samples; and (2) we identified genomic regions in which the predicted insert size between the two reads of a mate-pair from the mapping data deviated significantly from the expected insert size of the sequenced library (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001479/-/DC1/FileS1.pdf">File S1</a>).</p><p id="p-29">The coverage analysis resulted in the detection of a 1.1-kb region not covered in the pro23/fus sample, but well-covered in the other samples. This deleted region contained part of the open reading frame (ORF) of the developmental gene <em>pro41</em>, and the variant was established as the causative mutation of mutant pro23 (as mentioned the section <em>Analysis of the causative mutation in mutant pro23</em>). Interestingly, analysis of mate-pair insert sizes did not detect the deletion. The reason for this turned out to be that this region in mutant pro23 apparently underwent complex rearrangements that resulted in the substitution of a 1.1-kb region that included part of <em>pro41</em> by another genomic region of similar size, but unrelated sequence (see below). Thus, searching for noncovered regions and analyzing mate-pair information does not necessarily give the same results; instead, the two analyses can be complementary. Overall, we were able to identify two point mutations and one insertion/deletion event as the causative mutations for the three mutants.</p></div><div id="sec-12" class="subsection"><h3>Analysis of the causative mutation in mutant pro23</h3><p id="p-30">The analysis of genomic regions not covered by sequence reads from sample pro23/fus, but covered by reads in the other samples yielded a 1.1-kb genomic region that contained part of the <em>pro41</em> ORF. Southern blot analysis using part of the region that is deleted in mutant pro23 as a probe showed that indeed this region is present in the wild-type and mutant pro44, but absent in mutant pro23 (data not shown). The region around the <em>pro41</em> gene was PCR-amplified from genomic DNA of mutant pro23 and resequenced to confirm the genomic structure in the mutant strain. Interestingly, the genomic region that contains <em>pro41</em> is not only partly deleted in mutant pro23 but carries a complex set of rearrangements involving duplications and inversions of several adjacent regions that occupy the space of the deleted 1.1-kb fragment (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/261/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic structure of the pro41 gene locus in the wild-type and mutant pro23. Numbers in the wild type correspond to nucleotides in contig 2.1 (S. macrospora genome version 02). An approximately 1.1-kb region (green in the wild type) is deleted in mutant pro23. Several other regions (orange, red, and light and dark blue) adjacent to the deleted region are duplicated in the mutant strain and occupy the position of the deleted region. In most cases, the orientation of the duplicated regions is the same as in the wild type, with the exception of one case in which a small region is inverted in pro23 (indicated by an arrowhead within the duplicated region). The deleted region comprises the C-terminal part of pro41 (black arrow)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1958445924" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genomic structure of the &lt;em&gt;pro41&lt;/em&gt; gene locus in the wild-type and mutant pro23. Numbers in the wild type correspond to nucleotides in contig 2.1 (&lt;em&gt;S. macrospora&lt;/em&gt; genome version 02). An approximately 1.1-kb region (green in the wild type) is deleted in mutant pro23. Several other regions (orange, red, and light and dark blue) adjacent to the deleted region are duplicated in the mutant strain and occupy the position of the deleted region. In most cases, the orientation of the duplicated regions is the same as in the wild type, with the exception of one case in which a small region is inverted in pro23 (indicated by an arrowhead within the duplicated region). The deleted region comprises the C-terminal part of &lt;em&gt;pro41&lt;/em&gt; (black arrow).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/261/F2.medium.gif" width="440" height="108"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/2/261/F2.medium.gif" width="440" height="108"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/261/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/261/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461204" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-31" class="first-child">Genomic structure of the <em>pro41</em> gene locus in the wild-type and mutant pro23. Numbers in the wild type correspond to nucleotides in contig 2.1 (<em>S. macrospora</em> genome version 02). An approximately 1.1-kb region (green in the wild type) is deleted in mutant pro23. Several other regions (orange, red, and light and dark blue) adjacent to the deleted region are duplicated in the mutant strain and occupy the position of the deleted region. In most cases, the orientation of the duplicated regions is the same as in the wild type, with the exception of one case in which a small region is inverted in pro23 (indicated by an arrowhead within the duplicated region). The deleted region comprises the C-terminal part of <em>pro41</em> (black arrow).</p><div class="sb-div caption-clear"></div></div></div><p id="p-32"><em>pro41</em> encodes an endoplasmic reticulum membrane protein that is essential for sexual development of <em>S. macrospora</em> (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Nowrousian <em>et al.</em> 2007</a>). The deleted region in mutant pro23 contains part of the <em>pro41</em> ORF (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>), and it was shown previously that this part is essential to complement mutant pro41, which carries a deletion comprising the complete <em>pro41</em> ORF (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Nowrousian <em>et al.</em> 2007</a>). Therefore, it seemed likely that the absence of this part of the <em>pro41</em> gene causes sterility in mutant pro23. To verify this hypothesis, the mutant was transformed with a construct containing the <em>pro41</em> ORF, previously shown to complement mutant pro41 (<a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Nowrousian <em>et al.</em> 2007</a>). The resulting transformants were fertile, producing mature fruiting bodies (perithecia) similar to the wild type (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). These results demonstrate that the lack of a functional <em>pro41</em> is the causative mutation in mutant pro23.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/261/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Complementation of mutant pro23 with the pro41 ORF. Mutant pro23 was transformed with plasmid pE3-5Mr that carries the pro41 ORF under control of the respective gpd and trpC promoter and terminator sequences of A. nidulans. The figure shows the wild-type (forms perithecia; black dots in the photographs of the petri dishes), mutant pro23 (sterile, does not form perithecia), and two complemented transformants (produce perithecia). Strains were grown on corn meal agar, and photographs were taken after 6 d (A) or 8 d (B). Scale bar in (B) indicates 2 mm." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1958445924" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Complementation of mutant pro23 with the &lt;em&gt;pro41&lt;/em&gt; ORF. Mutant pro23 was transformed with plasmid pE3-5Mr that carries the &lt;em&gt;pro41&lt;/em&gt; ORF under control of the respective &lt;em&gt;gpd&lt;/em&gt; and &lt;em&gt;trpC&lt;/em&gt; promoter and terminator sequences of &lt;em&gt;A. nidulans&lt;/em&gt;. The figure shows the wild-type (forms perithecia; black dots in the photographs of the petri dishes), mutant pro23 (sterile, does not form perithecia), and two complemented transformants (produce perithecia). Strains were grown on corn meal agar, and photographs were taken after 6 d (A) or 8 d (B). Scale bar in (B) indicates 2 mm.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/261/F3.medium.gif" width="440" height="255"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/2/2/261/F3.medium.gif" width="440" height="255"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/261/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/261/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461314" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-33" class="first-child">Complementation of mutant pro23 with the <em>pro41</em> ORF. Mutant pro23 was transformed with plasmid pE3-5Mr that carries the <em>pro41</em> ORF under control of the respective <em>gpd</em> and <em>trpC</em> promoter and terminator sequences of <em>A. nidulans</em>. The figure shows the wild-type (forms perithecia; black dots in the photographs of the petri dishes), mutant pro23 (sterile, does not form perithecia), and two complemented transformants (produce perithecia). Strains were grown on corn meal agar, and photographs were taken after 6 d (A) or 8 d (B). Scale bar in (B) indicates 2 mm.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Analysis of the causative mutation in mutant fus</h3><p id="p-34">Bioinformatics analysis of small variants yielded a single point mutation (G to A transition) that was present in pro23/fus and fus but not in the wild type and pro44 samples. Analysis of annotated features at the position of the mutation showed that the mutation alters the first nucleotide of the second intron of the <em>tih</em> gene encoding trihydroxynaphtalene reductase (locus tag <em>SMAC_05650</em>). TIH was predicted to be part of the melanin biosynthesis pathway in <em>S. macrospora</em> (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Engh <em>et al.</em> 2007a</a>). Therefore, it seemed likely that the G to A transition in the <em>tih</em> gene was the causative mutation in the spore color mutant fus, and that the transition might lead to a splice defect of the second intron.</p><p id="p-35">To confirm this hypothesis, we first sequenced a PCR amplicon of part of the <em>tih</em> gene from genomic DNA of the fus strain. As expected, the first base of the second exon exhibited a G to A transition in the mutant strain (data not shown). We subsequently tested whether this mutation leads to a splice defect (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). RT-PCR and sequencing of the PCR fragments showed two introns in the <em>tih</em> gene; the first is spliced correctly in both the wild type and the fus mutant strain, whereas the second intron is either not spliced at all in the mutant, or an incorrect 5′ splice site is used leading to a shorter transcript. In both cases, the predicted TIH polypeptide is shorter and carries incorrect C-terminal sequences as the result of frame shifts in the non-spliced or misspliced transcripts. This makes it probable that the fus mutant does not generate functional TIH and therefore cannot produce melanin. To further verify that the mutation in the <em>tih</em> gene is the causative mutation, we transformed the fus mutant with a construct expressing the wild type <em>tih</em> gene. The resulting transformants carrying the wild-type <em>tih</em> exhibited the wild-type phenotype and produced correctly spliced transcript, in addition to the misspliced versions (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). These results demonstrate that indeed the fus mutant phenotype is caused by a mutation in the second intron of <em>tih</em>, resulting in a splice defect in the mutant strain.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/261/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A point mutation in the tih gene causes a splice defect in the fus mutant strain. (A) PCR and reverse-transcription PCR analysis of the tih gene in the wild type and fus mutant. Small arrows in (B) indicate the oligonucleotide primers used to amplify a tih gene fragment. &#x201C;+&#x201D; and &#x201C;&#x2212;&#x201D; indicate experiments with or without reverse transcriptase, respectively. Each sample was analyzed in duplicate with two independent biological replications. As expected, a 1.1-kb fragment was amplified from genomic DNA of both strains. From wild-type cDNA, the expected 0.6-kb fragment (after splicing of two introns) was amplified. PCR from cDNA of the fus strain yielded two fragments of 0.7 and 0.3 kb. (B) Two misspliced tih transcripts were identified in the fus mutant strain. The wild-type tih gene is shown as a white arrow with introns indicated in gray. Below are the derived coding regions, and above predicted peptides from the wild type and mutant fus. Sequence analysis of the PCR fragments from (A) confirmed the point mutation in the first base of the second intron in the fus mutant (nt 1040, G to A transition). The mutation leads to a splice defect of the second intron resulting in two transcripts in the fus strain (fus-T1 and fus-T2). In the first transcript (fus-T1), the second intron is not spliced, leading to a longer transcript represented by the 0.7 kb RT-PCR fragment in (A), and a premature stop codon, and therefore a shorter coding sequence, and a frame-shifted, shorter peptide (indicated in light blue in peptide fus-P1). In the fus-T2 transcript, a second intron is spliced using an incorrect 5&#x2032; splice site at nt 718, which results in a shorter transcript represented by the 0.3 kb RT-PCR fragment in (A), and a shorter, frame-shifted peptide (indicated in dark blue in peptide fus-P2). The lengths of the coding sequences of the wild-type and the two fus transcripts are given on the right; adh_short indicates the alcohol dehydrogenase domain within the predicted TIH peptide which is not fully present in the derived peptides fus-P1 and fus-P2 of the mutant strain." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1958445924" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A point mutation in the &lt;em&gt;tih&lt;/em&gt; gene causes a splice defect in the fus mutant strain. (A) PCR and reverse-transcription PCR analysis of the &lt;em&gt;tih&lt;/em&gt; gene in the wild type and fus mutant. Small arrows in (B) indicate the oligonucleotide primers used to amplify a &lt;em&gt;tih&lt;/em&gt; gene fragment. &#x201C;+&#x201D; and &#x201C;&#x2212;&#x201D; indicate experiments with or without reverse transcriptase, respectively. Each sample was analyzed in duplicate with two independent biological replications. As expected, a 1.1-kb fragment was amplified from genomic DNA of both strains. From wild-type cDNA, the expected 0.6-kb fragment (after splicing of two introns) was amplified. PCR from cDNA of the fus strain yielded two fragments of 0.7 and 0.3 kb. (B) Two misspliced &lt;em&gt;tih&lt;/em&gt; transcripts were identified in the fus mutant strain. The wild-type &lt;em&gt;tih&lt;/em&gt; gene is shown as a white arrow with introns indicated in gray. Below are the derived coding regions, and above predicted peptides from the wild type and mutant fus. Sequence analysis of the PCR fragments from (A) confirmed the point mutation in the first base of the second intron in the fus mutant (nt 1040, G to A transition). The mutation leads to a splice defect of the second intron resulting in two transcripts in the fus strain (fus-T1 and fus-T2). In the first transcript (fus-T1), the second intron is not spliced, leading to a longer transcript represented by the 0.7 kb RT-PCR fragment in (A), and a premature stop codon, and therefore a shorter coding sequence, and a frame-shifted, shorter peptide (indicated in light blue in peptide fus-P1). In the fus-T2 transcript, a second intron is spliced using an incorrect 5&#x2032; splice site at nt 718, which results in a shorter transcript represented by the 0.3 kb RT-PCR fragment in (A), and a shorter, frame-shifted peptide (indicated in dark blue in peptide fus-P2). The lengths of the coding sequences of the wild-type and the two fus transcripts are given on the right; adh_short indicates the alcohol dehydrogenase domain within the predicted TIH peptide which is not fully present in the derived peptides fus-P1 and fus-P2 of the mutant strain.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/261/F4.medium.gif" width="440" height="338"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/2/2/261/F4.medium.gif" width="440" height="338"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/261/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/261/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461253" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-36" class="first-child">A point mutation in the <em>tih</em> gene causes a splice defect in the fus mutant strain. (A) PCR and reverse-transcription PCR analysis of the <em>tih</em> gene in the wild type and fus mutant. Small arrows in (B) indicate the oligonucleotide primers used to amplify a <em>tih</em> gene fragment. “+” and “−” indicate experiments with or without reverse transcriptase, respectively. Each sample was analyzed in duplicate with two independent biological replications. As expected, a 1.1-kb fragment was amplified from genomic DNA of both strains. From wild-type cDNA, the expected 0.6-kb fragment (after splicing of two introns) was amplified. PCR from cDNA of the fus strain yielded two fragments of 0.7 and 0.3 kb. (B) Two misspliced <em>tih</em> transcripts were identified in the fus mutant strain. The wild-type <em>tih</em> gene is shown as a white arrow with introns indicated in gray. Below are the derived coding regions, and above predicted peptides from the wild type and mutant fus. Sequence analysis of the PCR fragments from (A) confirmed the point mutation in the first base of the second intron in the fus mutant (nt 1040, G to A transition). The mutation leads to a splice defect of the second intron resulting in two transcripts in the fus strain (fus-T1 and fus-T2). In the first transcript (fus-T1), the second intron is not spliced, leading to a longer transcript represented by the 0.7 kb RT-PCR fragment in (A), and a premature stop codon, and therefore a shorter coding sequence, and a frame-shifted, shorter peptide (indicated in light blue in peptide fus-P1). In the fus-T2 transcript, a second intron is spliced using an incorrect 5′ splice site at nt 718, which results in a shorter transcript represented by the 0.3 kb RT-PCR fragment in (A), and a shorter, frame-shifted peptide (indicated in dark blue in peptide fus-P2). The lengths of the coding sequences of the wild-type and the two fus transcripts are given on the right; adh_short indicates the alcohol dehydrogenase domain within the predicted TIH peptide which is not fully present in the derived peptides fus-P1 and fus-P2 of the mutant strain.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/261/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Complementation of the fus mutant with a wild-type copy of the tih gene. Mutant fus was transformed with plasmid pEHN5-tih that carries the tih ORF under control of respecitve gpd and trpC promoter and terminator sequences of A. nidulans. Single spore isolates from three independent complemented transformants (R4557, R7055, and R7111) were analyzed for phenotype and expression of the tih gene. (A) Strains were grown on corn meal agar, and photographs of ascospores were taken after 9 d. The fus mutant has light-brown ascospores whereas the wild type and the complemented transformants have black ascospores. Scale bars indicate 20 &#x3BC;m. (B) RT-PCR analysis of tih expression. The three complemented strains express the spliced wild-type transcript of the tih gene (0.6-kb fragment with primers tih-1/2, position of primers indicated in panel C), whereas in the fus recipient strain, only the misspliced/non-spliced (0.3 kb and 0.7 kb, respectively) fragments were detected. To demonstrate that the complemented transformants still produce the misspliced transcripts (in addition to the complementing wild-type copy), RT-PCR analysis was performed with oligonucleotides tih-1/5 that specifically amplify the misspliced transcript. The corresponding fragment (0.3 kb) can be amplified from the fus strain and the transformants, but not from the wild type. (C) Position of the oligonucleotide primers and the spliced introns in the wild type transcript, and the two transcripts from the mutant strain." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1958445924" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Complementation of the fus mutant with a wild-type copy of the &lt;em&gt;tih&lt;/em&gt; gene. Mutant fus was transformed with plasmid pEHN5-tih that carries the &lt;em&gt;tih&lt;/em&gt; ORF under control of respecitve &lt;em&gt;gpd&lt;/em&gt; and &lt;em&gt;trpC&lt;/em&gt; promoter and terminator sequences of &lt;em&gt;A. nidulans&lt;/em&gt;. Single spore isolates from three independent complemented transformants (R4557, R7055, and R7111) were analyzed for phenotype and expression of the &lt;em&gt;tih&lt;/em&gt; gene. (A) Strains were grown on corn meal agar, and photographs of ascospores were taken after 9 d. The fus mutant has light-brown ascospores whereas the wild type and the complemented transformants have black ascospores. Scale bars indicate 20 &#x3BC;m. (B) RT-PCR analysis of &lt;em&gt;tih&lt;/em&gt; expression. The three complemented strains express the spliced wild-type transcript of the &lt;em&gt;tih&lt;/em&gt; gene (0.6-kb fragment with primers tih-1/2, position of primers indicated in panel C), whereas in the fus recipient strain, only the misspliced/non-spliced (0.3 kb and 0.7 kb, respectively) fragments were detected. To demonstrate that the complemented transformants still produce the misspliced transcripts (in addition to the complementing wild-type copy), RT-PCR analysis was performed with oligonucleotides tih-1/5 that specifically amplify the misspliced transcript. The corresponding fragment (0.3 kb) can be amplified from the fus strain and the transformants, but not from the wild type. (C) Position of the oligonucleotide primers and the spliced introns in the wild type transcript, and the two transcripts from the mutant strain.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/261/F5.medium.gif" width="379" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/2/2/261/F5.medium.gif" width="379" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/261/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/261/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461266" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-37" class="first-child">Complementation of the fus mutant with a wild-type copy of the <em>tih</em> gene. Mutant fus was transformed with plasmid pEHN5-tih that carries the <em>tih</em> ORF under control of respecitve <em>gpd</em> and <em>trpC</em> promoter and terminator sequences of <em>A. nidulans</em>. Single spore isolates from three independent complemented transformants (R4557, R7055, and R7111) were analyzed for phenotype and expression of the <em>tih</em> gene. (A) Strains were grown on corn meal agar, and photographs of ascospores were taken after 9 d. The fus mutant has light-brown ascospores whereas the wild type and the complemented transformants have black ascospores. Scale bars indicate 20 μm. (B) RT-PCR analysis of <em>tih</em> expression. The three complemented strains express the spliced wild-type transcript of the <em>tih</em> gene (0.6-kb fragment with primers tih-1/2, position of primers indicated in panel C), whereas in the fus recipient strain, only the misspliced/non-spliced (0.3 kb and 0.7 kb, respectively) fragments were detected. To demonstrate that the complemented transformants still produce the misspliced transcripts (in addition to the complementing wild-type copy), RT-PCR analysis was performed with oligonucleotides tih-1/5 that specifically amplify the misspliced transcript. The corresponding fragment (0.3 kb) can be amplified from the fus strain and the transformants, but not from the wild type. (C) Position of the oligonucleotide primers and the spliced introns in the wild type transcript, and the two transcripts from the mutant strain.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Analysis of the causative mutation in mutant pro44</h3><p id="p-38">Bioinformatics analysis of small variants yielded seven point mutations present in pro44 but not in the other samples. Among the mutations, only one changed a coding region: A mutation within the stop codon (A to G) of <em>SMAC_03223</em> turns the stop codon into an amino acid-encoding codon resulting in a 107 amino acid extension of the predicted protein (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). To demonstrate that this mutation is the causative mutation leading to sterility in mutant pro44, the mutant was transformed with plasmids encoding the <em>SMAC_03223</em> ORF under control of its own regulatory regions, or under control of the <em>A. nidulans gpd</em> promoter and <em>trpC</em> terminator. All constructs were able to complement the mutant to fertility, confirming that the sterile phenotype of mutant pro44 is attributable to the defective <em>SMAC_03223</em> gene (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). Therefore, <em>SMAC_03223</em> was assigned the name <em>pro44</em>.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/261/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A point mutation in the pro44 gene (SMAC_03223) shifts the stop codon in the mutant strain. Schematic representation of the 2.4-kb pro44 mRNA and the derived wild-type and mutant proteins (above). The A to G mutation in the wild-type stop codon leads to the formation of a longer polypeptide in the mutant strain. The additional 107 amino acids are shown in blue; the predicted GATA zinc finger domain is shown in black." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1958445924" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A point mutation in the &lt;em&gt;pro44&lt;/em&gt; gene (&lt;em&gt;SMAC_03223&lt;/em&gt;) shifts the stop codon in the mutant strain. Schematic representation of the 2.4-kb &lt;em&gt;pro44&lt;/em&gt; mRNA and the derived wild-type and mutant proteins (above). The A to G mutation in the wild-type stop codon leads to the formation of a longer polypeptide in the mutant strain. The additional 107 amino acids are shown in blue; the predicted GATA zinc finger domain is shown in black.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/261/F6.medium.gif" width="440" height="116"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/2/2/261/F6.medium.gif" width="440" height="116"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/261/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/261/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461321" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 </span> <p id="p-39" class="first-child">A point mutation in the <em>pro44</em> gene (<em>SMAC_03223</em>) shifts the stop codon in the mutant strain. Schematic representation of the 2.4-kb <em>pro44</em> mRNA and the derived wild-type and mutant proteins (above). The A to G mutation in the wild-type stop codon leads to the formation of a longer polypeptide in the mutant strain. The additional 107 amino acids are shown in blue; the predicted GATA zinc finger domain is shown in black.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/261/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Complementation of mutant pro44 with constructs containing SMAC_03223. Mutant pro44 was transformed with plasmid pIG3146-37 that carries the SMAC_03223 ORF under control of its own promoter and terminator sequences (transformant pro44::SMAC_03223_native) or plasmid pIG3147-1 that carries the SMAC_03223 ORF under control of the respective gpd and trpC promoter and terminator sequences of Aspergillus nidulans (transformant pro44::SMAC_03223_constitutive). The figure shows a side view (longitudinal section) of the region comprising the agar/air interface from cultures of the wild type, the sterile mutant pro44 and two complemented transformants. The wild type forms mature perithecia at the agar/air interface, whereas mutant pro44 only forms protoperithecia that are submerged in the agar (greater magnification shown in small inserted photograph, arrowheads indicate protoperithecia). Complemented transformants produce mature perithecia at the agar/air interface like the wild type; however, some protoperithecia and even mature perithecia are still formed submerged in the growth medium. Strains were grown on corn meal agar; photographs were taken after 8 d; scale bar indicates 1 mm, and 200 &#xB5;m in the inserted pro44 photograph." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1958445924" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Complementation of mutant pro44 with constructs containing &lt;em&gt;SMAC_03223&lt;/em&gt;. Mutant pro44 was transformed with plasmid pIG3146-37 that carries the &lt;em&gt;SMAC_03223&lt;/em&gt; ORF under control of its own promoter and terminator sequences (transformant pro44::SMAC_03223_native) or plasmid pIG3147-1 that carries the &lt;em&gt;SMAC_03223&lt;/em&gt; ORF under control of the respective &lt;em&gt;gpd&lt;/em&gt; and &lt;em&gt;trpC&lt;/em&gt; promoter and terminator sequences of &lt;em&gt;Aspergillus nidulans&lt;/em&gt; (transformant pro44::SMAC_03223_constitutive). The figure shows a side view (longitudinal section) of the region comprising the agar/air interface from cultures of the wild type, the sterile mutant pro44 and two complemented transformants. The wild type forms mature perithecia at the agar/air interface, whereas mutant pro44 only forms protoperithecia that are submerged in the agar (greater magnification shown in small inserted photograph, arrowheads indicate protoperithecia). Complemented transformants produce mature perithecia at the agar/air interface like the wild type; however, some protoperithecia and even mature perithecia are still formed submerged in the growth medium. Strains were grown on corn meal agar; photographs were taken after 8 d; scale bar indicates 1 mm, and 200 &#xB5;m in the inserted pro44 photograph.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/261/F7.medium.gif" width="440" height="332"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7&#x2002;" src="https://www.g3journal.org/content/ggg/2/2/261/F7.medium.gif" width="440" height="332"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/261/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/261/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461318" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7 </span> <p id="p-40" class="first-child">Complementation of mutant pro44 with constructs containing <em>SMAC_03223</em>. Mutant pro44 was transformed with plasmid pIG3146-37 that carries the <em>SMAC_03223</em> ORF under control of its own promoter and terminator sequences (transformant pro44::SMAC_03223_native) or plasmid pIG3147-1 that carries the <em>SMAC_03223</em> ORF under control of the respective <em>gpd</em> and <em>trpC</em> promoter and terminator sequences of <em>Aspergillus nidulans</em> (transformant pro44::SMAC_03223_constitutive). The figure shows a side view (longitudinal section) of the region comprising the agar/air interface from cultures of the wild type, the sterile mutant pro44 and two complemented transformants. The wild type forms mature perithecia at the agar/air interface, whereas mutant pro44 only forms protoperithecia that are submerged in the agar (greater magnification shown in small inserted photograph, arrowheads indicate protoperithecia). Complemented transformants produce mature perithecia at the agar/air interface like the wild type; however, some protoperithecia and even mature perithecia are still formed submerged in the growth medium. Strains were grown on corn meal agar; photographs were taken after 8 d; scale bar indicates 1 mm, and 200 µm in the inserted pro44 photograph.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">Interestingly, the complemented transformants, although again producing fully fertile fruiting bodies, retained some mutant characteristics, namely the production of sexual structures within the agar medium. Whereas the wild type forms protoperithecia and later mature perithecia at the agar/air interface, mutant pro44 only forms protoperithecia, and these are completely embedded in the medium (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>). The complemented transformants form mature perithecia not only at the agar/air interface as in the wild type but also within the medium. Thus, the fruiting body maturation process was fully restored in the complemented transformants, whereas the correct distribution of sexual structures was not restored in all cases. However, it must be considered that the transformants still produce the mutated version of the protein that could exert aberrant functions, in this case resulting in mislocalization of fruiting bodies.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><div id="sec-16" class="subsection"><h3>Whole-genome sequencing of mutant strains is a rapid method to identify causative mutations even without prior mapping information</h3><p id="p-42">Advances in sequencing technologies and bioinformatics now make it possible to sequence even eukaryotic genomes quickly and at relatively moderate costs. Therefore, many genomes from a single species can be sequenced, and recent reports for several species demonstrate that mutations can be identified that cause specific phenotypes or diseases (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Irvine <em>et al.</em> 2009</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Lupski <em>et al.</em> 2010</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Pomraning <em>et al.</em> 2011</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Roach <em>et al.</em> 2010</a>; <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Sarin <em>et al.</em> 2008</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Schneeberger <em>et al.</em> 2009</a>). However, the strategies applied in these studies often included previous mapping of the mutation to a smaller genomic region and/or resequencing of large numbers of candidate mutations to distinguish between true mutations and sequence errors. Recently, two studies were performed in <em>S. pombe</em> and <em>C. elegans</em> to identify mutations through whole-genome sequencing without previous mapping and without crossing of strains with different genetic backgrounds (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Irvine <em>et al.</em> 2009</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Zuryn <em>et al.</em> 2010</a>). Similar to our work, in the <em>C. elegans</em> study mutants were backcrossed against the wild type, and more than one mutant genome was sequenced simultaneously. This strategy allows for easier elimination of sequence deviations caused by sequencing errors in the sequenced or the reference genomes. In our study, the robustness of the assay was increased further by using pooled DNA from 40 single spore isolates with the same phenotype instead of sequencing individual genomes of single progeny. In contrast to the analysis in <em>C. elegans</em> (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Zuryn <em>et al.</em> 2010</a>), we could not use an accumulation of mutagen-induced variants to indicate the region carrying the mutation in the progeny of backcrossed mutants, because the mutant strains used here had already been backcrossed several times to reduce the mutational load induced during mutagenesis. However, this fact and the use of pooled DNA from progeny with the same phenotype ensured that overall, the number of true sequence differences in comparison with the reference sequence was small. The true differences mostly consisted of sequencing errors in the reference genome that were easily identified because they occurred in all sequenced samples, including the resequenced wild type.</p><p id="p-43">Prior to our study, only two mutant sequencing approaches to identify mutations responsible for a specific phenotype were reported for filamentous fungi, both of them for <em>N. crassa</em> (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">McCluskey <em>et al.</em> 2011</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Pomraning <em>et al.</em> 2011</a>). <a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Pomraning <em>et al.</em> (2011)</a> crossed a mutant with a wild-type strain with different genetic background; <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">McCluskey <em>et al.</em> (2011)</a> used previous mapping information to narrow the genomic regions to be screened for mutations. However, such resources are not available for the majority of filamentous fungi, including <em>S. macrospora</em>. In the analysis described here, we used a different approach to show that identification of mutations through whole-genome sequencing is possible, even in the absence of prior mapping information, or strains with different genetic backgrounds.</p></div><div id="sec-17" class="subsection"><h3>Sequenced mutant genomes carry mutations in three developmental genes</h3><p id="p-44">Although the mutants that were used in this study were all generated by EMS mutagenesis, the causative mutation in pro23 is quite different at the molecular level from those in pro44 and fus. The mutations in mutant fus and pro44 are transitions, which are typical for EMS-induced changes (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bautz and Freese 1960</a>). In contrast, the mutation in pro23 consists of a combination of deletion, duplication, and inversion (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>) at the <em>pro41</em> locus. This type of mutation is not typical for EMS mutagenesis; however, our previous studies of mutant pro41 (also a product of EMS mutagenesis) showed that this mutant carries a 4-kb deletion that includes the <em>pro41</em> ORF (<a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">Nowrousian <em>et al.</em> 2007</a>). This finding indicates that the <em>pro41</em> gene locus might be prone to this type of mutation. Another possibility is that the rearrangements detected in mutant pro23 arose from a replication-based mechanism recently described to explain complex structural variants in the human genome (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Conrad <em>et al.</em> 2010</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lee <em>et al.</em> 2007</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Zhang <em>et al.</em> 2009</a>). It was proposed that when the replication fork stalls, <em>e.g.</em> due to single-strand breaks or single nucleotide deletions, repeated template switching to nearby genomic regions can occur, leading to apparent deletions when parts of the template are skipped, and to duplications when the replication machinery “jumps back”. One consequence of the fact that the mutations found after EMS mutagenesis are apparently not restricted to point mutations is that both the sequencing strategy, and the subsequent bioinformatics analyses should be optimized not only to find small variants but also larger structural variations in the range of several kilobases.</p><p id="p-45">The rearrangements in mutant pro23 within the genomic region comprising the <em>pro41</em> gene lead to a defective <em>pro41</em> as demonstrated by the fact that the wild-type copy of <em>pro41</em> is able to rescue the pro23 mutant phenotype. The developmental gene <em>pro41</em> was first identified through the analysis of the pro41 mutant (<a id="xref-ref-30-7" class="xref-bibr" href="#ref-30">Nowrousian <em>et al.</em> 2007</a>). The identification of an allelic <em>pro41</em> mutation as the causative mutation in the independent mutant pro23, which originated from a different round of EMS mutagenesis, indicates that our mutant collection of ∼100 sterile mutants might reach saturation (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Kück <em>et al.</em> 2009</a>).</p><p id="p-46">The identification of a point mutation leading to a splice defect in the <em>tih</em> gene is the first case of a mutation within an intron sequence that results in a mutant phenotype in <em>S. macrospora</em>. The mutants characterized previously either carry deletions that cover complete ORFs (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Masloff <em>et al.</em> 1999</a>; <a id="xref-ref-30-8" class="xref-bibr" href="#ref-30">Nowrousian <em>et al.</em> 2007</a>), or point mutations within exons that lead to amino acid exchanges or premature stop codons (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Bloemendal <em>et al.</em> 2010</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Engh <em>et al.</em> 2007b</a>; <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Nowrousian <em>et al.</em> 1999</a>; <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Pöggeler and Kück 2004</a>). A similar mutation of G to A at the 5′ end of an intron was described in the developmental mutant grisea of <em>Podospora anserina</em> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Borghouts <em>et al.</em> 1997</a>).</p><p id="p-47">The mutation in the <em>tih</em> gene leads to a shorter/altered C-terminus in the derived TIH protein that is apparently not functional in the fus mutant, evidenced by brownish pigments instead of black melanins. TIH is predicted to catalyze the fourth enzymatic step in the biosynthesis of DHN (dihydroxynaphtalene) melanins (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bell and Wheeler 1986</a>). <em>tih</em> and three other genes (<em>pks</em>, <em>sdh</em>, and <em>teh</em>) involved in melanin biosynthesis are transcriptionally up-regulated during sexual development, and down-regulated in mutants blocked early during this process (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Engh <em>et al.</em> 2007a</a>; <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Nowrousian <em>et al.</em> 2005</a>). <em>pks</em> and <em>sdh</em> were already shown to be essential for melanin biosynthesis, and this study demonstrates that <em>tih</em> is also required for the melanin formation that gives rise to the black color of the <em>S. macrospora</em> ascospores.</p><p id="p-48">In mutant pro44, a mutation within the stop codon of the <em>pro44</em> gene leads to a predicted protein with an additional 107 amino acids at the C-terminus. The wild type C-terminus of PRO44 is the most conserved part of the protein and comprises the predicted DNA binding domain of GATA-type transcription factors. Therefore, it seems likely that the additional amino acids derived from a usually noncoding genomic region cause structural changes that prevent function of the mutant protein. <em>pro44</em> orthologs have been characterized in <em>A. nidulans</em>, <em>N. crassa</em>, and <em>A. fumigatus</em>, where they were shown to be essential for fruiting body formation (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Colot <em>et al.</em> 2006</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Han <em>et al.</em> 2001</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Szewczyk and Krappmann 2010</a>). Furthermore, the <em>N. crassa</em> deletion strain of the <em>pro44</em> ortholog <em>sub-1</em> produces protoperithecia that are submerged in the agar medium as opposed to the production of protoperithecia at the agar/air interface in the wild type (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Colot <em>et al.</em> 2006</a>). This phenotype is similar to what we observed in pro44 (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>). Thus, <em>pro44</em> encodes a conserved transcription factor essential for sexual development in filamentous ascomycetes. <em>pro44</em> is the second transcription factor-encoding gene that was identified from the pro mutant-screen; the first being the zinc cluster-type transcription factor encoded by <em>pro1</em> (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Masloff <em>et al.</em> 1999</a>).</p></div><div id="sec-18" class="subsection"><h3>Conclusions</h3><p id="p-49">The results presented here demonstrate that for small eukaryotic genomes, whole-genome sequencing of mutant strains is a fast and efficient means to discover mutations that are responsible for specific phenotypes. The strategy used here requires that the mutants can be genetically crossed against other strains, but it does not require any prior mapping information, or strains with different genetic backgrounds. Therefore, this method will be useful for the growing number of species that are used as experimental systems to study a wide range of biological phenomena, but for which only a small range of genetic and molecular resources is available.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-50">We thank Ingeborg Godehardt, Susanne Schlewinski, Swenja Ellßel, Kerstin Kalkreuter, Silke Nimtz, and Regina Ricke for excellent technical assistance, and Dr. Michael Freitag (Oregon State University, Corvallis, OR) for helpful discussions. This research was supported by a grant from Protein Research Department of the Ruhr-Universität Bochum (U.K. and M.N.) and a grant from the German Science Foundation (Deutsche Forschungsgemeinschaft, grant FOR 1334) (U.K.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001479/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001479/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">The Illumina/Solexa sequence reads that were generated in this study have been deposited with the NCBI sequence read archive under accession number SRA026818.1. The updated <em>Sordaria macrospora</em> genome assembly (version 02) has been deposited with the EMBL database (ENA) under accession numbers CABT02000001-CABT02001583.</p></li><li class="fn" id="fn-7"><p id="p-51"><em>Communicating editor: D. J. Gresham</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-10-21"><span class="received-label">Received </span>October 21, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-12-04"><span class="accepted-label">Accepted </span>December 4, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Nowrousian <em>et al</em>. by the Genetics Society of America</li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.1" data-doi="10.1093/bioinformatics/btr251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balzer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malde</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jonassen</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Systematic exploration of error sources in pyrosequencing flowgram data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">i304</span>–<span class="cit-lpage">i309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr251%26rft_id%253Dinfo%253Apmid%252F21685085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTMvaTMwNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8yNjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.2" data-doi="10.1073/pnas.46.12.1585"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bautz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freese</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1960</span> <span class="cit-article-title">On the mutagenic effect of alkylating agents</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U S A.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1585</span>–<span class="cit-lpage">1594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBautz%26rft.auinit1%253DE.%26rft.volume%253D46%26rft.issue%253D12%26rft.spage%253D1585%26rft.epage%253D1594%26rft.atitle%253DON%2BTHE%2BMUTAGENIC%2BEFFECT%2BOF%2BALKYLATING%2BAGENTS%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.46.12.1585%26rft_id%253Dinfo%253Apmid%252F16590785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiNDYvMTIvMTU4NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8yNjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.3" data-doi="10.1146/annurev.py.24.090186.002211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">M. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Biosynthesis and functions of fungal melanins</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Phytopathol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">411</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bphytopathology%26rft.stitle%253DAnnu%2BRev%2BPhytopathol%26rft.aulast%253DBell%26rft.auinit1%253DA.%26rft.volume%253D24%26rft.issue%253D1%26rft.spage%253D411%26rft.epage%253D451%26rft.atitle%253DBiosynthesis%2Band%2BFunctions%2Bof%2BFungal%2BMelanins%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.py.24.090186.002211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.py.24.090186.002211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1986D802500022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.4" data-doi="10.1128/EC.00186-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bloemendal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lord</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rech</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoff</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engh</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A mutant defective in sexual development produces aseptate ascogonia</span>. <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1856</span>–<span class="cit-lpage">1866</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00186-10%26rft_id%253Dinfo%253Apmid%252F20952581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo5OiI5LzEyLzE4NTYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMjYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.5" data-doi="10.1073/pnas.94.20.10768"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borghouts</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimpel</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osiewacz</span>  <span class="cit-name-given-names">H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Mitochondrial DNA rearrangements of <em>Podospora anserina</em> are under control of the nuclear gene grisea</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U S A.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">10768</span>–<span class="cit-lpage">10773</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBorghouts%26rft.auinit1%253DC.%26rft.volume%253D94%26rft.issue%253D20%26rft.spage%253D10768%26rft.epage%253D10773%26rft.atitle%253DMitochondrial%2BDNA%2Brearrangements%2Bof%2BPodospora%2Banserina%2Bare%2Bunder%2Bthe%2Bcontrol%2Bof%2Bthe%2Bnuclear%2Bgene%2Bgrisea%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.20.10768%26rft_id%253Dinfo%253Apmid%252F9380708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTQvMjAvMTA3NjgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMjYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.6" data-doi="10.1073/pnas.0601456103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colot</span>  <span class="cit-name-given-names">H. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ringelberg</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crew</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A high-throughput gene knockout procedure for Neurospora reveals functions for multiple transcription factors</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U S A.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">10352</span>–<span class="cit-lpage">10357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU%2BS%2BA.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0601456103%26rft_id%253Dinfo%253Apmid%252F16801547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzI3LzEwMzUyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8yLzI2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.7" data-doi="10.1038/ng.564"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bird</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackburne</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindsay</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mamanova</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mutation spectrum revealed by breakpoint sequencing of human germline CNVs</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">385</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DConrad%26rft.auinit1%253DD.%2BF.%26rft.volume%253D42%26rft.issue%253D5%26rft.spage%253D385%26rft.epage%253D391%26rft.atitle%253DMutation%2Bspectrum%2Brevealed%2Bby%2Bbreakpoint%2Bsequencing%2Bof%2Bhuman%2Bgermline%2BCNVs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.564%26rft_id%253Dinfo%253Apmid%252F20364136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.564&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20364136&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277179500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.8" data-doi="10.1111/j.1749-6632.1999.tb08870.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The distribution of rates of spontaneous mutation over viruses, prokaryotes, and eukaryotes</span>. <abbr class="cit-jnl-abbrev">Ann. N. Y. Acad. Sci.</abbr> <span class="cit-vol">870</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bthe%2BNew%2BYork%2BAcademy%2Bof%2BSciences%26rft.stitle%253DAnn.%2BN.%2BY.%2BAcad.%2BSci.%26rft.aulast%253DDRAKE%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D870%26rft.issue%253D1%26rft.spage%253D100%26rft.epage%253D107%26rft.atitle%253DThe%2BDistribution%2Bof%2BRates%2Bof%2BSpontaneous%2BMutation%2Bover%2BViruses%252C%2BProkaryotes%252C%2Band%2BEukaryotes%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1749-6632.1999.tb08870.x%26rft_id%253Dinfo%253Apmid%252F10415476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1749-6632.1999.tb08870.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10415476&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081585400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.9" data-doi="10.1111/j.1574-6968.2007.00867.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Engh</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nowrousian</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kück</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">a</span> <span class="cit-article-title">Regulation of melain biosynthesis via the dihydroxynaphtalene pathway is dependent on sexual development in the ascomycete <em>Sordaria macrospora</em></span>. <abbr class="cit-jnl-abbrev">FEMS Microbiol. Lett.</abbr> <span class="cit-vol">275</span>: <span class="cit-fpage">62</span>–<span class="cit-lpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6968.2007.00867.x%26rft_id%253Dinfo%253Apmid%252F17681008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc2xlIjtzOjU6InJlc2lkIjtzOjg6IjI3NS8xLzYyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8yLzI2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.10" data-doi="10.1128/EC.00269-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Engh</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Würtz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witzel-Schlömp</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoff</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">b</span> <span class="cit-article-title">The WW domain protein PRO40 is required for fungal fertility and associates with Woronin bodies</span>. <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">831</span>–<span class="cit-lpage">843</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00269-06%26rft_id%253Dinfo%253Apmid%252F17351077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI2LzUvODMxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8yLzI2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.11" data-doi="10.1016/j.ejcb.2010.07.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Engh</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nowrousian</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kück</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title"><em>Sordaria macrospora</em>, a model organism to study fungal cellular development</span>. <abbr class="cit-jnl-abbrev">Eur. J. Cell Biol.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">864</span>–<span class="cit-lpage">8727</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bcell%2Bbiology%26rft.stitle%253DEur%2BJ%2BCell%2BBiol%26rft.aulast%253DEngh%26rft.auinit1%253DI.%26rft.volume%253D89%26rft.issue%253D12%26rft.spage%253D864%26rft.epage%253D872%26rft.atitle%253DSordaria%2Bmacrospora%252C%2Ba%2Bmodel%2Borganism%2Bto%2Bstudy%2Bfungal%2Bcellular%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ejcb.2010.07.002%26rft_id%253Dinfo%253Apmid%252F20739093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ejcb.2010.07.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20739093&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-2.2.261.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Esser</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-source">Cryptogams—Cyanobacteria, Algae, Fungi, Lichens</span>, <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.13" data-doi="10.1534/genetics.110.116616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flibotte</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edgley</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaudhry</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neil</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Whole-genome profiling of mutagenesis in <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D185%26rft.spage%253D431%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116616%26rft_id%253Dinfo%253Apmid%252F20439774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.110.116616&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20439774&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281905200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.14" data-doi="10.1046/j.1365-2958.2001.02472.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">K. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chae</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jahng</span>  <span class="cit-name-given-names">K. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The <em>nsdD</em> gene encodes a putative GATA-type transcription factor necessary for sexual development of <em>Aspergillus nidulans</em></span>. <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DHan%26rft.auinit1%253DK.%2BH.%26rft.volume%253D41%26rft.issue%253D2%26rft.spage%253D299%26rft.epage%253D309%26rft.atitle%253DThe%2BnsdD%2Bgene%2Bencodes%2Ba%2Bputative%2BGATA-type%2Btranscription%2Bfactor%2Bnecessary%2Bfor%2Bsexual%2Bdevelopment%2Bof%2BAspergillus%2Bnidulans.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2001.02472.x%26rft_id%253Dinfo%253Apmid%252F11489119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2001.02472.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11489119&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.15" data-doi="10.1101/gr.089318.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Irvine</span>  <span class="cit-name-given-names">D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goto</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaughn</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakaseko</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mccombie</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mapping epigenetic mutations in fission yeast using whole-genome next-generation sequencing</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1077</span>–<span class="cit-lpage">1083</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.089318.108%26rft_id%253Dinfo%253Apmid%252F19423874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTA3NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8yNjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.16" data-doi="10.1128/EC.00019-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klix</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nowrousian</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ringelberg</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loros</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunlap</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Functional characterization of <em>MAT1–1</em>-specific mating-type genes in the homothallic ascomycete <em>Sordaria macrospora</em> provides new insights into essential and non-essential sexual regulators</span>. <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">894</span>–<span class="cit-lpage">905</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00019-10%26rft_id%253Dinfo%253Apmid%252F20435701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI5LzYvODk0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8yLzI2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.17" data-doi="10.1126/science.1149504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korbel</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urban</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Affourtit</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Godwin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grubert</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Paired-end mapping reveals extensive structural variation in the human genome</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">318</span>: <span class="cit-fpage">420</span>–<span class="cit-lpage">426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1149504%26rft_id%253Dinfo%253Apmid%252F17901297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTgvNTg0OS80MjAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMjYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.18" data-doi="10.1007/s00438-005-0021-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kück</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A <em>Sordaria macrospora</em> mutant lacking the <em>leu1</em> gene shows a developmental arrest during fruiting body formation</span>. <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">307</span>–<span class="cit-lpage">315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DK%25C3%25BCck%26rft.auinit1%253DU.%26rft.volume%253D274%26rft.issue%253D3%26rft.spage%253D307%26rft.epage%253D315%26rft.atitle%253DA%2BSordaria%2Bmacrospora%2Bmutant%2Blacking%2Bthe%2Bleu1%2Bgene%2Bshows%2Ba%2Bdevelopmental%2Barrest%2Bduring%2Bfruiting%2Bbody%2Bformation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-005-0021-8%26rft_id%253Dinfo%253Apmid%252F16133166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-005-0021-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16133166&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-2.2.261.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kück</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pöggeler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nowrousian</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolting</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engh</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title"><em>Sordaria macrospora</em>, a model system for fungal development</span>, pp. <span class="cit-fpage">17</span>–<span class="cit-lpage">39</span> <span class="cit-comment">in</span> <span class="cit-source">The Mycota XV, Physiology and Genetics</span>, <span class="cit-comment">edited by</span> <span class="cit-ed"><span class="cit-name-surname">Anke</span>  <span class="cit-name-given-names">T.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">D.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin, Heidelberg</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.20" data-doi="10.1016/j.cell.2007.11.037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carvalho</span>  <span class="cit-name-given-names">C. M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lupski</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A DNA replication mechanism for generating nonrecurrent rearrangements associated with genomic disorders</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">1235</span>–<span class="cit-lpage">1247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLee%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D131%26rft.issue%253D7%26rft.spage%253D1235%26rft.epage%253D1247%26rft.atitle%253DA%2BDNA%2Breplication%2Bmechanism%2Bfor%2Bgenerating%2Bnonrecurrent%2Brearrangements%2Bassociated%2Bwith%2Bgenomic%2Bdisorders.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.11.037%26rft_id%253Dinfo%253Apmid%252F18160035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.11.037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18160035&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252217200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-2" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.21" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows–Wheeler transform</span>. <abbr class="cit-jnl-abbrev">Bioinf.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinf.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8yNjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.22" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools</span>. <abbr class="cit-jnl-abbrev">Bioinf.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinf.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8yNjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.23" data-doi="10.1056/NEJMoa0908094"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lupski</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzaga-Jauregui</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rio Deiros</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Whole-genome sequencing in a patient with Charcot-Marie-Tooth neuropathy</span>. <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">362</span>: <span class="cit-fpage">1181</span>–<span class="cit-lpage">1191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.volume%253D362%26rft.issue%253D13%26rft.spage%253D1181%26rft.epage%253D1191%26rft.atitle%253DWhole-genome%2Bsequencing%2Bin%2Ba%2Bpatient%2Bwith%2BCharcot-Marie-Tooth%2Bneuropathy.%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa0908094%26rft_id%253Dinfo%253Apmid%252F20220177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa0908094&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20220177&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276239500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Masloff</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pöggeler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kück</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The <em>pro1+</em> gene from <em>Sordaria macrospora</em> encodes a C<sub>6</sub> zinc finger transcription factor required for fruiting body development</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">191</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D152%26rft.spage%253D191%26rft_id%253Dinfo%253Apmid%252F10224253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10224253&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCluskey</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiest</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grigoriev</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipzen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Rediscovery by whole genome sequencing: classical mutations and genome polymorphisms in Neurospora crassa</span>. <abbr class="cit-jnl-abbrev">G3: Genes, Genomes, Genetics</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">303</span>–<span class="cit-lpage">316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%253A%2BGenes%252C%2BGenomes%252C%2BGenetics%26rft.volume%253D1%26rft.spage%253D303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.26" data-doi="10.1038/nrg2626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Metzker</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Sequencing technologies - the next generation</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMetzker%26rft.auinit1%253DM.%2BL.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D46%26rft.atitle%253DSequencing%2Btechnologies%2B-%2Bthe%2Bnext%2Bgeneration.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2626%26rft_id%253Dinfo%253Apmid%252F19997069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2626&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19997069&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272867700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.27" data-doi="10.1128/EC.00123-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nowrousian</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Next-generation sequencing techniques for eukaryotic microorganisms: sequencing-based solutions to biological problems</span>. <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1300</span>–<span class="cit-lpage">1310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00123-10%26rft_id%253Dinfo%253Apmid%252F20601439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiI5LzkvMTMwMCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8yNjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.28" data-doi="10.1128/MCB.19.1.450"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nowrousian</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masloff</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pöggeler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kück</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Cell differentiation during sexual development of the fungus <em>Sordaria macrospora</em> requires ATP citrate lyase activity</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">450</span>–<span class="cit-lpage">460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DNowrousian%26rft.auinit1%253DM.%26rft.volume%253D19%26rft.issue%253D1%26rft.spage%253D450%26rft.epage%253D460%26rft.atitle%253DCell%2BDifferentiation%2Bduring%2BSexual%2BDevelopment%2Bof%2Bthe%2BFungus%2BSordaria%2Bmacrospora%2BRequires%2BATP%2BCitrate%2BLyase%2BActivity%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.19.1.450%26rft_id%253Dinfo%253Apmid%252F9858569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjE5LzEvNDUwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8yLzI2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.29" data-doi="10.1007/s00438-005-1118-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nowrousian</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ringelberg</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunlap</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loros</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kück</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Cross-species microarray hybridization to identify developmentally regulated genes in the filamentous fungus <em>Sordaria macrospora</em></span>. <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">273</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DNowrousian%26rft.auinit1%253DM.%26rft.volume%253D273%26rft.issue%253D2%26rft.spage%253D137%26rft.epage%253D149%26rft.atitle%253DCross-species%2Bmicroarray%2Bhybridization%2Bto%2Bidentify%2Bdevelopmentally%2Bregulated%2Bgenes%2Bin%2Bthe%2Bfilamentous%2Bfungus%2BSordaria%2Bmacrospora.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-005-1118-9%26rft_id%253Dinfo%253Apmid%252F15778868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-005-1118-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15778868&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228729400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.30" data-doi="10.1111/j.1365-2958.2007.05694.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nowrousian</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frank</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strauch</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weitner</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The novel ER membrane protein PRO41 is essential for sexual development in the filamentous fungus <em>Sordaria macrospora</em></span>. <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">923</span>–<span class="cit-lpage">937</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DNowrousian%26rft.auinit1%253DM.%26rft.volume%253D64%26rft.issue%253D4%26rft.spage%253D923%26rft.epage%253D937%26rft.atitle%253DThe%2Bnovel%2BER%2Bmembrane%2Bprotein%2BPRO41%2Bis%2Bessential%2Bfor%2Bsexual%2Bdevelopment%2Bin%2Bthe%2Bfilamentous%2Bfungus%2BSordaria%2Bmacrospora.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2007.05694.x%26rft_id%253Dinfo%253Apmid%252F17501918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2007.05694.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17501918&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.31" data-doi="10.1371/journal.pgen.1000891"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nowrousian</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stajich</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engh</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Espagne</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title"><em>De novo</em> assembly of a 40 Mb eukaryotic genome from short sequence reads: <em>Sordaria macrospora</em>, a model organism for fungal morphogenesis</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000891</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DNowrousian%26rft.auinit1%253DM.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De1000891%26rft.epage%253De1000891%26rft.atitle%253DDe%2Bnovo%2Bassembly%2Bof%2Ba%2B40%2BMb%2Beukaryotic%2Bgenome%2Bfrom%2Bshort%2Bsequence%2Breads%253A%2BSordaria%2Bmacrospora%252C%2Ba%2Bmodel%2Borganism%2Bfor%2Bfungal%2Bmorphogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000891%26rft_id%253Dinfo%253Apmid%252F20386741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000891&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20386741&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.32" data-doi="10.1128/EC.3.1.232-240.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pöggeler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kück</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A WD40 repeat protein regulates fungal cell differentiation and can be replaced functionally by the mammalian homologue striatin</span>. <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">232</span>–<span class="cit-lpage">240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DPoggeler%26rft.auinit1%253DS.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D232%26rft.epage%253D240%26rft.atitle%253DA%2BWD40%2BRepeat%2BProtein%2BRegulates%2BFungal%2BCell%2BDifferentiation%2Band%2BCan%2BBe%2BReplaced%2BFunctionally%2Bby%2Bthe%2BMammalian%2BHomologue%2BStriatin%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.3.1.232-240.2004%26rft_id%253Dinfo%253Apmid%252F14871953%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiIzLzEvMjMyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8yLzI2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.33" data-doi="10.1016/j.gene.2006.03.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pöggeler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kück</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Highly efficient generation of signal transduction knockout mutants using a fungal strain deficient in the mammalian <em>ku70</em> ortholog</span>. <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">378</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D378%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2006.03.020%26rft_id%253Dinfo%253Apmid%252F16814491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2006.03.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16814491&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.34" data-doi="10.1016/S0167-7012(97)00018-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pöggeler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nowrousian</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobsen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kück</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">An efficient procedure to isolate fungal genes from an indexed cosmid library</span>. <abbr class="cit-jnl-abbrev">J. Microbiol. Methods</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMicrobiol.%2BMethods%26rft.volume%253D29%26rft.spage%253D49%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0167-7012%252897%252900018-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0167-7012(97)00018-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XG53000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.35" data-doi="10.1007/s00294-003-0370-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pöggeler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masloff</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoff</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayrhofer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kück</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Versatile EGFP reporter plasmids for cellular localization of recombinant gene products in filamentous fungi</span>. <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">54</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DP%25C3%25B6ggeler%26rft.auinit1%253DS.%26rft.volume%253D43%26rft.issue%253D1%26rft.spage%253D54%26rft.epage%253D61%26rft.atitle%253DVersatile%2BEGFP%2Breporter%2Bplasmids%2Bfor%2Bcellular%2Blocalization%2Bof%2Brecombinant%2Bgene%2Bproducts%2Bin%2Bfilamentous%2Bfungi.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00294-003-0370-y%26rft_id%253Dinfo%253Apmid%252F12684845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00294-003-0370-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12684845&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.36" data-doi="10.1128/EC.00016-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pomraning</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freitag</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Bulk segregant analysis followed by high-throughput sequencing reveals the Neurospora cell cycle gene, <em>ndc-1</em>, to be allelic with the gene for ornithine decarboxylase, <em>spe-1</em></span>. <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">724</span>–<span class="cit-lpage">733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00016-11%26rft_id%253Dinfo%253Apmid%252F21515825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiIxMC82LzcyNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8yNjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.37" data-doi="10.1038/nature10113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Puente</span>  <span class="cit-name-given-names">X. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pinyol</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quesada</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conde</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ordonez</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Whole-genome sequencing identifies recurrent mutations in chronic lymphocytic leukaemia</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">475</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPuente%26rft.auinit1%253DX.%2BS.%26rft.volume%253D475%26rft.issue%253D7354%26rft.spage%253D101%26rft.epage%253D105%26rft.atitle%253DWhole-genome%2Bsequencing%2Bidentifies%2Brecurrent%2Bmutations%2Bin%2Bchronic%2Blymphocytic%2Bleukaemia.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10113%26rft_id%253Dinfo%253Apmid%252F21642962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21642962&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292461300053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.38" data-doi="10.1126/science.1186802"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roach</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glusman</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smit</span>  <span class="cit-name-given-names">A. F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huff</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubley</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Analysis of genetic inheritance in a family quartet by whole-genome sequencing</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">328</span>: <span class="cit-fpage">636</span>–<span class="cit-lpage">639</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1186802%26rft_id%253Dinfo%253Apmid%252F20220176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjgvNTk3OC82MzYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMjYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.39" data-doi="10.1038/nmeth.1249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sarin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prabhu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O'meara</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pe'er</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hobert</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title"><em>Caenorhabditis elegans</em> mutant allele identification by whole-genome sequencing</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">865</span>–<span class="cit-lpage">867</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSarin%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253D865%26rft.epage%253D867%26rft.atitle%253DCaenorhabditis%2Belegans%2Bmutant%2Ballele%2Bidentification%2Bby%2Bwhole-genome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1249%26rft_id%253Dinfo%253Apmid%252F18677319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18677319&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259650100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.40" data-doi="10.1038/nmeth0809-550"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ossowski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juul</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petersen</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">SHOREmap: simultaneous mapping and mutation identification by deep sequencing</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">550</span>–<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSchneeberger%26rft.auinit1%253DK.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253D550%26rft.epage%253D551%26rft.atitle%253DSHOREmap%253A%2Bsimultaneous%2Bmapping%2Band%2Bmutation%2Bidentification%2Bby%2Bdeep%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth0809-550%26rft_id%253Dinfo%253Apmid%252F19644454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth0809-550&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19644454&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268493700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.41" data-doi="10.1038/nbt1486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shendure</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Next-generation DNA sequencing</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1135</span>–<span class="cit-lpage">1145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DShendure%26rft.auinit1%253DJ.%26rft.volume%253D26%26rft.issue%253D10%26rft.spage%253D1135%26rft.epage%253D1145%26rft.atitle%253DNext-generation%2BDNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1486%26rft_id%253Dinfo%253Apmid%252F18846087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1486&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18846087&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259926000028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.42" data-doi="10.1128/EC.00375-09"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szewczyk</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krappmann</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Conserved regulators of mating are essential for Aspergillus fumigatus cleistothecium formation</span>. <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">774</span>–<span class="cit-lpage">783</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00375-09%26rft_id%253Dinfo%253Apmid%252F20348388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI5LzUvNzc0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8yLzI2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.43" data-doi="10.1007/BF00313198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kück</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Transformation of <em>Sordaria macrospora</em> to hygromycin B resistance: characterization of transformants by electrophoretic karyotyping and tetrad analysis</span>. <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">88</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DWalz%26rft.auinit1%253DM.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D88%26rft.epage%253D95%26rft.atitle%253DTransformation%2Bof%2BSordaria%2Bmacrospora%2Bto%2Bhygromycin%2BB%2Bresistance%253A%2Bcharacterization%2Bof%2Btransformants%2Bby%2Belectrophoretic%2Bkaryotyping%2Band%2Btetrad%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00313198%26rft_id%253Dinfo%253Apmid%252F8595663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00313198&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8595663&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.44" data-doi="10.1038/ng.399"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khajavi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connolly</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Towne</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batish</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The DNA replication FoSTeS/MMBIR mechanism can generate genomic, genetic and exonic complex rearrangements in humans</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">849</span>–<span class="cit-lpage">853</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DF.%26rft.volume%253D41%26rft.issue%253D7%26rft.spage%253D849%26rft.epage%253D853%26rft.atitle%253DThe%2BDNA%2Breplication%2BFoSTeS%252FMMBIR%2Bmechanism%2Bcan%2Bgenerate%2Bgenomic%252C%2Bgenic%2Band%2Bexonic%2Bcomplex%2Brearrangements%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.399%26rft_id%253Dinfo%253Apmid%252F19543269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.399&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19543269&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267786200019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-book" id="cit-2.2.261.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zickler</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Meiosis in mycelial fungi</span>, pp. <span class="cit-fpage">415</span>–<span class="cit-lpage">438</span> <span class="cit-comment">in</span> <span class="cit-source">The Mycota I. Growth, Differentiation and Sexuality</span>, <span class="cit-comment">edited by</span> <span class="cit-ed"><span class="cit-name-surname">Kües</span>  <span class="cit-name-given-names">U.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">R.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin, Heidelberg</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.261.46" data-doi="10.1534/genetics.110.119230"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuryn</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Gras</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jamet</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarriault</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A strategy for direct mapping and identification of mutations by whole-genome sequencing</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D186%26rft.spage%253D427%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.119230%26rft_id%253Dinfo%253Apmid%252F20610404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.110.119230&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20610404&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281950600036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/2/261.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/2/249" title="A Whole-Chromosome Analysis of Meiotic Recombination in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/2/271" title="Narrowing Down the Mapping of Plant Sex-Determination Regions Using New Y-Chromosome-Specific Markers and Heavy-Ion Beam Irradiation-Induced Y-Deletion Mutants in Silene latifolia" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/2"><span>Volume 2 Issue 2, February 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/2.cover.gif?itok=i7gjzosG" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/2/261" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-W2DfLO6OeLEFIDPFMtqxZ4UNjiQeJwt_0EnlAIrF3vc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/2/261" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/2/261" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/2/261" class="active" data-icon-position="" data-hide-link-title="0">Whole-Genome Sequencing of Sordaria macrospora Mutants Identifies Developmental Genes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459244" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-_tR4o9i4x-3wEm4MrRcBNLyBqK6F_4g4LVMgiT1Hfyk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/2/261.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/2/261" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-NSPjL0iW9a5sLzHCd-LhssjrUjaC_6d19qwLMHhqQRk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/2/261" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/2/261.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459244" id="node459244" data-pisa="ggg;2/2/261" data-pisa-master="ggg;2/2/261" data-apath="/ggg/2/2/261.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole-Genome Sequencing of <em>Sordaria macrospora</em> Mutants Identifies Developmental Genes</h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Minou Nowrousian</span>, <span class="highwire-citation-author" data-delta="1">Ines Teichert</span>, <span class="highwire-citation-author" data-delta="2">Sandra Masloff</span> and <span class="highwire-citation-author" data-delta="3">Ulrich Kück</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">261-270;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001479 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459244" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Minou Nowrousian</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Lehrstuhl für Allgemeine und Molekulare Botanik, Ruhr-Universität Bochum, 44780 Bochum, Germany</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Minou%2BNowrousian" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nowrousian%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMinou%2BNowrousian" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ines Teichert</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Lehrstuhl für Allgemeine und Molekulare Botanik, Ruhr-Universität Bochum, 44780 Bochum, Germany</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ines%2BTeichert" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Teichert%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AInes%2BTeichert" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sandra Masloff</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Lehrstuhl für Allgemeine und Molekulare Botanik, Ruhr-Universität Bochum, 44780 Bochum, Germany</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandra%2BMasloff" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Masloff%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandra%2BMasloff" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ulrich Kück</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Lehrstuhl für Allgemeine und Molekulare Botanik, Ruhr-Universität Bochum, 44780 Bochum, Germany</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ulrich%2BK%C3%BCck" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=K%C3%BCck%20U&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AUlrich%2BK%25C3%25BCck" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ulrich.kueck@rub.de" class="" data-icon-position="" data-hide-link-title="0">ulrich.kueck@rub.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/2/261" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/2/261&amp;title=Whole-Genome%20Sequencing%20of%20Sordaria%20macrospora%20Mutants%20Identifies%20Developmental%20Genes" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/2/261&amp;title=Whole-Genome%20Sequencing%20of%20Sordaria%20macrospora%20Mutants%20Identifies%20Developmental%20Genes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/2/261&amp;title=Whole-Genome%20Sequencing%20of%20Sordaria%20macrospora%20Mutants%20Identifies%20Developmental%20Genes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/2/261&amp;text=Whole-Genome%20Sequencing%20of%20Sordaria%20macrospora%20Mutants%20Identifies%20Developmental%20Genes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/2/261&amp;title=Whole-Genome%20Sequencing%20of%20Sordaria%20macrospora%20Mutants%20Identifies%20Developmental%20Genes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/2/261&amp;t=Whole-Genome%20Sequencing%20of%20Sordaria%20macrospora%20Mutants%20Identifies%20Developmental%20Genes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/2/261&amp;title=Whole-Genome%20Sequencing%20of%20Sordaria%20macrospora%20Mutants%20Identifies%20Developmental%20Genes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/2/261&amp;title=Whole-Genome%20Sequencing%20of%20Sordaria%20macrospora%20Mutants%20Identifies%20Developmental%20Genes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459244/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459244" id="node459244" data-pisa="ggg;2/2/261" data-pisa-master="ggg;2/2/261" data-apath="/ggg/2/2/261.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole-Genome Sequencing of <em>Sordaria macrospora</em> Mutants Identifies Developmental Genes</h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Minou Nowrousian</span>, <span class="highwire-citation-author" data-delta="1">Ines Teichert</span>, <span class="highwire-citation-author" data-delta="2">Sandra Masloff</span> and <span class="highwire-citation-author" data-delta="3">Ulrich Kück</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">261-270;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001479 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459244" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Minou Nowrousian</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Lehrstuhl für Allgemeine und Molekulare Botanik, Ruhr-Universität Bochum, 44780 Bochum, Germany</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Minou%2BNowrousian" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nowrousian%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMinou%2BNowrousian" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ines Teichert</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Lehrstuhl für Allgemeine und Molekulare Botanik, Ruhr-Universität Bochum, 44780 Bochum, Germany</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ines%2BTeichert" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Teichert%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AInes%2BTeichert" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sandra Masloff</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Lehrstuhl für Allgemeine und Molekulare Botanik, Ruhr-Universität Bochum, 44780 Bochum, Germany</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandra%2BMasloff" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Masloff%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandra%2BMasloff" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ulrich Kück</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Lehrstuhl für Allgemeine und Molekulare Botanik, Ruhr-Universität Bochum, 44780 Bochum, Germany</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ulrich%2BK%C3%BCck" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=K%C3%BCck%20U&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AUlrich%2BK%25C3%25BCck" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ulrich.kueck@rub.de" class="" data-icon-position="" data-hide-link-title="0">ulrich.kueck@rub.de</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=261&amp;rft.epage=270&amp;rft.atitle=Whole-Genome%20Sequencing%20of%20Sordaria%20macrospora%20Mutants%20Identifies%20Developmental%20Genes&amp;rft.volume=2&amp;rft.issue=2&amp;rft.date=2012-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Nowrousian%2C+Minou&amp;rft.au=Teichert%2C+Ines&amp;rft.au=Masloff%2C+Sandra&amp;rft.au=K%C3%BCck%2C+Ulrich"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459244/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459244/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459244/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459244/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459244/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459244/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459244/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459244/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459244/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459244/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459244/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459244/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494625" id="node494625" data-pisa="ggg;9/11/3891" data-pisa-master="ggg;g3.119.400723" data-apath="/ggg/9/11/3891.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3891" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Mouse Mutation That Dysregulates Neighboring <em>Galnt17</em> and <em>Auts2</em> Genes Is Associated with Phenotypes Related to the Human AUTS2 Syndrome</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494624" id="node494624" data-pisa="ggg;9/11/3781" data-pisa-master="ggg;g3.119.400647" data-apath="/ggg/9/11/3781.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3781" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Transgenic Analyses in Drosophila Reveal That mCORL1 Is Functionally Distinct from mCORL2 and dCORL</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494622" id="node494622" data-pisa="ggg;9/11/3843" data-pisa-master="ggg;g3.119.400363" data-apath="/ggg/9/11/3843.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3843" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Analysis of Cell Cycle-Regulating Genes in the Symbiotic Dinoflagellate <em>Breviolum minutum</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/2/261" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/2/261.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/2/261.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/2/261.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/2/261.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2019 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
  </body>
</html>
