<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/8/853" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/8/853.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/8/853.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/8/853.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/2/8/853/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/8/853.atom" />
<meta name="HW.pisa" content="ggg;2/8/853" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome-Wide Association Mapping in Tomato (Solanum lycopersicum) Is Possible Using Genome Admixture of Solanum lycopersicum var. cerasiforme" />
<meta name="DC.Identifier" content="10.1534/g3.112.002667" />
<meta name="DC.Date" content="2012-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Ranc et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genome-wide association mapping is an efficient way to identify quantitative trait loci controlling the variation of phenotypes, but the approach suffers severe limitations when one is studying inbred crops like cultivated tomato ( Solanum lycopersicum ). Such crops exhibit low rates of molecular polymorphism and high linkage disequilibrium, which reduces mapping resolution. The cherry type tomato ( S. lycopersicum var. cerasiforme ) genome has been described as an admixture between the cultivated tomato and its wild ancestor, S. pimpinellifolium . We have thus taken advantage of the properties of this admixture to improve the resolution of association mapping in tomato. As a proof of concept, we sequenced 81 DNA fragments distributed on chromosome 2 at different distances in a core collection of 90 tomato accessions, including mostly cherry type tomato accessions. The 81 Sequence Tag Sites revealed 352 SNPs and indels. Molecular diversity was greatest for S. pimpinellifolium accessions, intermediate for S. l. cerasiforme accessions, and lowest for the cultivated group. We assessed the structure of molecular polymorphism and the extent of linkage disequilibrium over genetic and physical distances. Linkage disequilibrium decreased under r 2 = 0.3 within 1 cM, and minimal estimated value (r2 = 0.13) was reached within 20 kb over the physical regions studied. Associations between polymorphisms and fruit weight, locule number, and soluble solid content were detected. Several candidate genes and quantitative trait loci previously identified were validated and new associations detected. This study shows the advantages of using a collection of S. l. cerasiforme accessions to overcome the low resolution of association mapping in tomato." />
<meta name="DC.Contributor" content="Nicolas Ranc" />
<meta name="DC.Contributor" content="Stephane Muños" />
<meta name="DC.Contributor" content="Jiaxin Xu" />
<meta name="DC.Contributor" content="Marie-Christine Le Paslier" />
<meta name="DC.Contributor" content="Aurélie Chauveau" />
<meta name="DC.Contributor" content="Rémi Bounon" />
<meta name="DC.Contributor" content="Sophie Rolland" />
<meta name="DC.Contributor" content="Jean-Paul Bouchet" />
<meta name="DC.Contributor" content="Dominique Brunel" />
<meta name="DC.Contributor" content="Mathilde Causse" />
<meta name="article:published_time" content="2012-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genome-Wide Association Mapping in Tomato (Solanum lycopersicum) Is Possible Using Genome Admixture of Solanum lycopersicum var. cerasiforme" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genome-wide association mapping is an efficient way to identify quantitative trait loci controlling the variation of phenotypes, but the approach suffers severe limitations when one is studying inbred crops like cultivated tomato (&lt;i&gt;Solanum lycopersicum&lt;/i&gt;). Such crops exhibit low rates of molecular polymorphism and high linkage disequilibrium, which reduces mapping resolution. The cherry type tomato (&lt;i&gt;S. lycopersicum var. cerasiforme&lt;/i&gt;) genome has been described as an admixture between the cultivated tomato and its wild ancestor, &lt;i&gt;S. pimpinellifolium&lt;/i&gt;. We have thus taken advantage of the properties of this admixture to improve the resolution of association mapping in tomato. As a proof of concept, we sequenced 81 DNA fragments distributed on chromosome 2 at different distances in a core collection of 90 tomato accessions, including mostly cherry type tomato accessions. The 81 Sequence Tag Sites revealed 352 SNPs and indels. Molecular diversity was greatest for &lt;i&gt;S. pimpinellifolium&lt;/i&gt; accessions, intermediate for &lt;i&gt;S. l. cerasiforme&lt;/i&gt; accessions, and lowest for the cultivated group. We assessed the structure of molecular polymorphism and the extent of linkage disequilibrium over genetic and physical distances. Linkage disequilibrium decreased under &lt;i&gt;r&lt;/i&gt;&lt;sup&gt;2&lt;/sup&gt; = 0.3 within 1 cM, and minimal estimated value (r&lt;sup&gt;2&lt;/sup&gt; = 0.13) was reached within 20 kb over the physical regions studied. Associations between polymorphisms and fruit weight, locule number, and soluble solid content were detected. Several candidate genes and quantitative trait loci previously identified were validated and new associations detected. This study shows the advantages of using a collection of &lt;i&gt;S. l. cerasiforme&lt;/i&gt; accessions to overcome the low resolution of association mapping in tomato.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/08/01" />
<meta name="citation_mjid" content="ggg;2/8/853" />
<meta name="citation_id" content="2/8/853" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/8/853" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/8/853.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/8/853.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/8/853.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.002667" />
<meta name="citation_pmid" content="22908034" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="853" />
<meta name="citation_lastpage" content="864" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Nicolas Ranc" />
<meta name="citation_author_institution" content="INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France" />
<meta name="citation_author" content="Stephane Muños" />
<meta name="citation_author_institution" content="INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France" />
<meta name="citation_author" content="Jiaxin Xu" />
<meta name="citation_author_institution" content="INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France" />
<meta name="citation_author_institution" content="Northwest A&amp;F University, College of Horticulture, Yang Ling, Shaanxin, 712100, People's Republic of China, and" />
<meta name="citation_author" content="Marie-Christine Le Paslier" />
<meta name="citation_author_institution" content="INRA, UR1279, Unité Etude du Polymorphisme des Génomes Végétaux, CEA-Institut de Génomique-CNG, Evry, 91057, France" />
<meta name="citation_author" content="Aurélie Chauveau" />
<meta name="citation_author_institution" content="INRA, UR1279, Unité Etude du Polymorphisme des Génomes Végétaux, CEA-Institut de Génomique-CNG, Evry, 91057, France" />
<meta name="citation_author" content="Rémi Bounon" />
<meta name="citation_author_institution" content="INRA, UR1279, Unité Etude du Polymorphisme des Génomes Végétaux, CEA-Institut de Génomique-CNG, Evry, 91057, France" />
<meta name="citation_author" content="Sophie Rolland" />
<meta name="citation_author_institution" content="INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France" />
<meta name="citation_author" content="Jean-Paul Bouchet" />
<meta name="citation_author_institution" content="INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France" />
<meta name="citation_author" content="Dominique Brunel" />
<meta name="citation_author_institution" content="INRA, UR1279, Unité Etude du Polymorphisme des Génomes Végétaux, CEA-Institut de Génomique-CNG, Evry, 91057, France" />
<meta name="citation_author" content="Mathilde Causse" />
<meta name="citation_author_institution" content="INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France" />
<meta name="citation_author_email" content="Mathilde.Causse@avignon.inra.fr" />
<meta name="citation_reference" content="citation_title=Nat. Rev. Genet.;citation_author=K. G. Ardlie;citation_author=L. Kruglyak;citation_author=M. Seielstad;citation_title=PATTERNS OF LINKAGE DISEQUILIBRIUM IN THE HUMAN GENOME;citation_pages=299-309;citation_volume=3:;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot. (Lond.);citation_author=Y. Bai;citation_author=P. Lindhout;citation_title=Domestication and breeding of tomatoes: what have we gained and what can we gain in the future?;citation_volume=100;citation_year=2007;citation_pmid=17717024;citation_doi=10.1093/aob/mcm150" />
<meta name="citation_reference" content="citation_journal_title=J. Educ. Behav. Stat.;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=On the adaptive control of the false discovery rate in multiple testing with independent statistics;citation_pages=60-83;citation_volume=25;citation_year=2000;citation_doi=10.3102/10769986025001060" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. Bradbury;citation_author=Z. Zhang;citation_author=D. E. Kroon;citation_author=T. M. Casstevens;citation_author=Y. Ramdoss;citation_title=TASSEL: software for association mapping of complex traits in diverse samples;citation_volume=23;citation_year=2007;citation_pmid=17586829;citation_doi=10.1093/bioinformatics/btm308" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=I. Buckler;citation_author=S. Edward;citation_author=J. M. Thornsberry;citation_title=Plant molecular diversity and applications to genomics.;citation_pages=107-111;citation_volume=5;citation_year=2002;citation_issue=2;citation_pmid=11856604;citation_doi=10.1016/S1369-5266(02)00238-8" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. S. Caldwell;citation_author=J. Russell;citation_author=P. Langridge;citation_author=W. Powell;citation_title=Extreme population-dependent linkage disequilibrium detected in an inbreeding plant species, Hordeum vulgare;citation_volume=172;citation_year=2006;citation_pmid=16219791;citation_doi=10.1534/genetics.104.038489" />
<meta name="citation_reference" content="citation_journal_title=Journal of Experimental Botany;citation_journal_abbrev=J Exp Bot;citation_author=M. Causse;citation_author=V. Saliba-Colombani;citation_author=L. Lecomte;citation_author=P. Duffe;citation_author=P. Rousselle;citation_title=QTL analysis of fruit quality in fresh market tomato: a few chromosome regions control the variation of sensory and instrumental traits;citation_pages=2089-2098;citation_volume=53;citation_year=2002;citation_issue=377;citation_pmid=12324532;citation_doi=10.1093/jxb/erf058" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. J. Daly;citation_author=J. D. Rioux;citation_author=S. F. Schaffner;citation_author=T. J. Hudson;citation_author=E. S. Lander;citation_title=High-resolution haplotype structure in the human genome.;citation_pages=229-232;citation_volume=29;citation_year=2001;citation_issue=2;citation_pmid=11586305;citation_doi=10.1038/ng1001-229" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Doebley;citation_author=A. Stec;citation_author=L. Hubbard;citation_title=The evolution of apical dominance in maize.;citation_pages=485-488;citation_volume=386;citation_year=1997;citation_issue=6624;citation_pmid=9087405;citation_doi=10.1038/386485a0" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. Drouaud;citation_title=Variation in crossing-over rates across chromosome4 of Arabidopsis thaliana reveals the presence of meiotic recombination hot spots;citation_volume=16;citation_year=2006;citation_pmid=16344568;citation_doi=10.1101/gr.4319006" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=G. Evanno;citation_author=S. Regnaut;citation_author=J. Goudet;citation_title=Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study.;citation_pages=2611-2620;citation_volume=14;citation_year=2005;citation_issue=8;citation_pmid=15969739;citation_doi=10.1111/j.1365-294X.2005.02553.x" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Plant Biol.;citation_author=S. A. Flint-Garcia;citation_author=J. M. Thornsberry;citation_author=S. Edward;citation_author=I. V. Buckler;citation_title=Structure of linkage disequilibrium in plants;citation_volume=54;citation_year=2003;citation_pmid=14502995;citation_doi=10.1146/annurev.arplant.54.031902.134907" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=S. A. Flint-Garcia;citation_author=A.-C. Thuillet;citation_author=J. Yu;citation_author=G. Pressoir;citation_author=S. M. Romero;citation_title=Maize association population: a high-resolution platform for quantitative trait locus dissection.;citation_pages=1054-1064;citation_volume=44;citation_year=2005;citation_issue=6;citation_pmid=16359397;citation_doi=10.1111/j.1365-313X.2005.02591.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Frary;citation_author=T. C. Nesbitt;citation_author=A. Frary;citation_author=S. Grandillo;citation_author=E. D. Knaap;citation_title=fw2.2: A Quantitative Trait Locus Key to the Evolution of Tomato Fruit Size;citation_pages=85-88;citation_volume=289;citation_year=2000;citation_issue=5476;citation_pmid=10884229;citation_doi=10.1126/science.289.5476.85" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=P. K. Gupta;citation_author=S. Rustgi;citation_author=P. L. Kulwal;citation_title=Linkage disequilibrium and association studies in higher plants: present status and future prospects.;citation_pages=461-485;citation_volume=57;citation_year=2005;citation_issue=4;citation_pmid=15821975;citation_doi=10.1007/s11103-005-0257-z" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Notes;citation_author=O. J. Hardy;citation_author=X. Vekemans;citation_title=spagedi: a versatile computer program to analyse spatial genetic structure at the individual or population levels;citation_pages=618-620;citation_volume=2;citation_year=2002;citation_doi=10.1046/j.1471-8286.2002.00305.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. L. Hyten;citation_author=I.-Y. Choi;citation_author=Q. Song;citation_author=R. C. Shoemaker;citation_author=R. L. Nelson;citation_title=Highly variable patterns of linkage disequilibrium in multiple soybean populations;citation_pages=1937-1944;citation_volume=175;citation_year=2007;citation_pmid=17287533;citation_doi=10.1534/genetics.106.069740" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Innan;citation_author=W. Stephan;citation_title=Distinguishing the hitchhiking and background selection models;citation_pages=2307-2312;citation_volume=165;citation_year=2003;citation_pmid=14704207" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=J. Jimenez-Gomez;citation_author=J. Maloof;citation_title=Sequence diversity in three tomato species: SNPs, markers, and molecular evolution.;citation_pages=85-85;citation_volume=9;citation_year=2009;citation_pmid=19575805;citation_doi=10.1186/1471-2229-9-85" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=H. Kaya;citation_author=K.-i. Shibahara;citation_author=K.-i. Taoka;citation_author=M. Iwabuchi;citation_author=B. Stillman;citation_title=FASCIATA genes for chromatin assembly factor-1 in arabidopsis maintain the cellular organization of apical meristems.;citation_pages=131-142;citation_volume=104;citation_year=2001;citation_issue=1;citation_pmid=11163246;citation_doi=10.1016/S0092-8674(01)00197-0" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. Kim;citation_author=V. Plagnol;citation_author=T. T. Hu;citation_author=C. Toomajian;citation_author=R. M. Clark;citation_title=Recombination and linkage disequilibrium in Arabidopsis thaliana.;citation_pages=1151-1155;citation_volume=39;citation_year=2007;citation_issue=9;citation_pmid=17676040;citation_doi=10.1038/ng2115" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=J. A. Labate;citation_author=A. Baldo;citation_title=Tomato SNP discovery by EST mining and resequencing;citation_pages=343-349;citation_volume=16;citation_year=2005;citation_doi=10.1007/s11032-005-1911-5" />
<meta name="citation_reference" content="citation_author=J. A. Labate;citation_author=S. Grandillo;citation_author=T. Fulton;citation_author=S. Muños;citation_author=A. L. Caicedo;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. Labate;citation_author=L. Robertson;citation_author=F. Wu;citation_author=S. Tanksley;citation_author=A. Baldo;citation_title=EST, COSII, and arbitrary gene markers give similar estimates of nucleotide diversity in cultivated tomato (Solanum lycopersicum L.).;citation_pages=1005-1014;citation_volume=118;citation_year=2009;citation_issue=5;citation_pmid=19153710;citation_doi=10.1007/s00122-008-0957-2" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=L. Lecomte;citation_author=V. Saliba-Colombani;citation_author=A. Gautier;citation_author=M. C. Gomez-Jimenez;citation_author=P. Duffé;citation_title=FINE MAPPING OF QTLS OF CHROMOSOME 2 AFFECTING THE FRUIT ARCHITECTURE AND COMPOSITION OF TOMATO;citation_pages=1-14;citation_volume=V13;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Liu;citation_author=J. Van Eck;citation_author=B. Cong;citation_author=S. D. Tanksley;citation_title=A new class of regulatory genes underlying the cause of pear-shaped tomato fruit;citation_volume=99;citation_year=2002;citation_pmid=12242331;citation_doi=10.1073/pnas.162485999" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=A. D. Long;citation_author=C. H. Langley;citation_title=The Power of Association Studies to Detect the Contribution of Candidate Genetic Loci to Variation in Complex Traits;citation_pages=720-731;citation_volume=9;citation_year=1999;citation_issue=8;citation_pmid=10447507" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. Manning;citation_author=M. Tor;citation_author=M. Poole;citation_author=Y. Hong;citation_author=A. J. Thompson;citation_title=A naturally occurring epigenetic mutation in a gene encoding an SBP-box transcription factor inhibits tomato fruit ripening.;citation_pages=948-952;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16832354;citation_doi=10.1038/ng1841" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=S. Muños;citation_author=N. Ranc;citation_author=E. Botton;citation_author=A. Bérard;citation_author=S. Rolland;citation_title=Increase in tomato locule number is controlled by two key SNP located near Wuschel;citation_volume=156:;citation_year=2011;citation_pmid=21673133;citation_doi=10.1104/pp.111.173997" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. C. Nesbitt;citation_author=S. D. Tanksley;citation_title=Comparative sequencing in the genus Lycopersicon: implications for the evolution of fruit size in the domestication of cultivated tomatoes;citation_pages=365-379;citation_volume=162;citation_year=2002;citation_pmid=12242247" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Nordborg;citation_title=Linkage disequilibrium, gene trees and selfing: an ancestral recombination graph with partial self-fertilization;citation_pages=923-929;citation_volume=154;citation_year=2000;citation_pmid=10655241" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. Nordborg;citation_author=S. Tavare;citation_title=Linkage disequilibrium: what history has to tell us.;citation_pages=83-90;citation_volume=18;citation_year=2002;citation_issue=2;citation_pmid=11818140;citation_doi=10.1016/S0168-9525(02)02557-X" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. Nordborg;citation_author=J. O. Borevitz;citation_author=J. Bergelson;citation_author=C. C. Berry;citation_author=J. Chory;citation_title=The extent of linkage disequilibrium in Arabidopsis thaliana.;citation_pages=190-193;citation_volume=30;citation_year=2002;citation_issue=2;citation_pmid=11780140;citation_doi=10.1038/ng813" />
<meta name="citation_reference" content="citation_journal_title=Planta;citation_author=C. Paulus;citation_author=B. Köllner;citation_author=H.-J. Jacobsen;citation_title=Physiological and biochemical characterization of glyoxalase I, a general marker for cell proliferation, from a soybean cell suspension;citation_pages=561-566;citation_volume=189;citation_year=1993;citation_pmid=7763577" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. L. Price;citation_author=N. J. Patterson;citation_author=R. M. Plenge;citation_author=M. E. Weinblatt;citation_author=N. A. Shadick;citation_title=Principal components analysis corrects for stratification in genome-wide association studies.;citation_pages=904-909;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16862161;citation_doi=10.1038/ng1847" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of population structure using multilocus genotype data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_pmid=10835412" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=N. A. Rosenberg;citation_author=P. Donnelly;citation_title=Association mapping in structured populations.;citation_pages=170-181;citation_volume=67;citation_year=2000;citation_issue=1;citation_pmid=10827107;citation_doi=10.1086/302959" />
<meta name="citation_reference" content="citation_journal_title=Plant, Cell, and Environment (Print);citation_journal_abbrev=Plant, Cell, and Environment (Print);citation_author=M. Prudent;citation_author=N. Bertin;citation_author=M. Génard;citation_author=S. Muños;citation_author=S. Rolland;citation_title=Genotype-dependent response to carbon availability in growing tomato fruit.;citation_pages=1186-1204;citation_volume=33;citation_year=2010;citation_issue=7;citation_pmid=20199617" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=A. Rafalski;citation_title=Applications of single nucleotide polymorphisms in crop genetics.;citation_pages=94-100;citation_volume=5;citation_year=2002;citation_issue=2;citation_pmid=11856602;citation_doi=10.1016/S1369-5266(02)00240-6" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=N. Ranc;citation_author=S. Muños;citation_author=S. Santoni;citation_author=M. Causse;citation_title=A clarified position for Solanum lycopersicum var. cerasiforme in the evolutionary history of tomatoes (solanaceae).;citation_pages=130-130;citation_volume=8;citation_year=2008;citation_pmid=19099601;citation_doi=10.1186/1471-2229-8-130" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=K. Ritland;citation_title=Estimators for pairwise relatedness and individual inbreeding coefficients;citation_pages=175-185;citation_volume=67;citation_year=1996;citation_doi=10.1017/S0016672300033620" />
<meta name="citation_reference" content="citation_journal_title=pp. 365–386 in Bioinformatics Methods and Protocols: Methods in Molecular Biology, edited by S. Krawetz and S. Misener. Humana Press, Totowa, NJ;citation_title=Primer 3 on the WWW for general users and for biologist programmers,;citation_author=S. Rozen;citation_author=H. Skaletsky;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=V. Saliba-Colombani;citation_author=M. Causse;citation_author=D. Langlois;citation_author=J. Philouze;citation_author=M. Buret;citation_title=GENETIC ANALYSIS OF ORGANOLEPTIC QUALITY IN FRESH MARKET TOMATO. 1. MAPPING QTLS FOR PHYSICAL AND CHEMICAL TRAITS;citation_pages=259-272;citation_volume=V102;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Journal of bioinformatics and computational biology;citation_journal_abbrev=J Bioinform Comput Biol;citation_author=M. Takahashi;citation_author=F. Matsuda;citation_author=N. Margetic;citation_author=M. Lathrop;citation_title=Automated identification of single nucleotide polymorphisms from sequencing data.;citation_pages=253-265;citation_volume=1;citation_year=2003;citation_issue=2;citation_pmid=15290772;citation_doi=10.1142/S021972000300006X" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. D. Tanksley;citation_author=M. W. Ganal;citation_author=J. P. Prince;citation_author=M. C. de-Vicente;citation_author=M. W. Bonierbale;citation_title=High density molecular linkage maps of the tomato and potato genomes;citation_pages=1141-1160;citation_volume=132;citation_year=1992;citation_pmid=1360934" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=A. Tenesa;citation_author=A. F. Wright;citation_author=S. A. Knott;citation_author=A. D. Carothers;citation_author=C. Hayward;citation_title=Extent of linkage disequilibrium in a Sardinian sub-isolate: sampling and methodological considerations;citation_volume=13;citation_year=2004;citation_pmid=14613964;citation_doi=10.1093/hmg/ddh001" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. M. Thornsberry;citation_author=M. M. Goodman;citation_author=J. Doebley;citation_author=S. Kresovich;citation_author=D. Nielsen;citation_title=Dwarf8 polymorphisms associate with variation in flowering time.;citation_pages=286-289;citation_volume=28;citation_year=2001;citation_issue=3;citation_pmid=11431702;citation_doi=10.1038/90135" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. van Berloo;citation_author=A. Zhu;citation_author=R. Ursem;citation_author=H. Verbakel;citation_author=G. Gort;citation_title=Diversity and linkage disequilibrium analysis within a selected set of cultivated tomatoes.;citation_pages=89-101;citation_volume=117;citation_year=2008;citation_issue=1;citation_pmid=18389208;citation_doi=10.1007/s00122-008-0755-x" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. Van Deynze;citation_author=K. Stoffel;citation_author=C. R. Buell;citation_author=A. Kozik;citation_author=J. Liu;citation_title=Diversity in conserved genes in tomato.;citation_pages=465-465;citation_volume=8;citation_year=2007;citation_pmid=18088428;citation_doi=10.1186/1471-2164-8-465" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=M. L. Verbsky;citation_author=E. J. Richards;citation_title=Chromatin remodeling in plants.;citation_pages=494-500;citation_volume=4;citation_year=2001;citation_issue=6;citation_pmid=11641064;citation_doi=10.1016/S1369-5266(00)00206-5" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. L. Weber;citation_author=W. H. Briggs;citation_author=J. Rucker;citation_author=B. M. Baltazar;citation_author=J. de Jesus Sanchez-Gonzalez;citation_title=The genetic architecture of complex traits in teosinte (Zea mays ssp. parviglumis): new evidence from association mapping;citation_pages=1221-1232;citation_volume=180;citation_year=2008;citation_pmid=18791250;citation_doi=10.1534/genetics.108.090134" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=W. Yang;citation_author=X. Bai;citation_author=E. Kabelka;citation_author=C. Eaton;citation_author=S. Kamoun;citation_title=DISCOVERY OF SINGLE NUCLEOTIDE POLYMORPHISMS IN LYCOPERSICON ESCULENTUM BY COMPUTER AIDED ANALYSIS OF EXPRESSED SEQUENCE TAGS;citation_pages=21-34;citation_volume=V14;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yu;citation_author=G. Pressoir;citation_author=W. H. Briggs;citation_author=I. Vroh Bi;citation_author=M. Yamasaki;citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.;citation_pages=203-208;citation_volume=38;citation_year=2006;citation_issue=2;citation_pmid=16380716;citation_doi=10.1038/ng1702" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=L. Zhang;citation_author=S. Marchand;citation_author=N. Tinker;citation_author=F. Belzile;citation_title=Population structure and linkage disequilibrium in barley assessed by DArT markers.;citation_pages=43-52;citation_volume=119;citation_year=2009;citation_issue=1;citation_pmid=19343317;citation_doi=10.1007/s00122-009-1015-4" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=K. Zhao;citation_author=M. J. Aranzana;citation_author=S. Kim;citation_author=C. Lister;citation_author=C. Shindo;citation_title=An Arabidopsis example of association mapping in structured samples.;citation_pages=e4-e4;citation_volume=3;citation_year=2007;citation_issue=1;citation_pmid=17238287;citation_doi=10.1371/journal.pgen.0030004" />
<meta name="citation_reference" content="citation_journal_title=The Plant Genome;citation_journal_abbrev=The Plant Genome;citation_author=C. Zhu;citation_author=M. Gore;citation_author=E. S. Buckler;citation_author=J. Yu;citation_title=Status and Prospects of Association Mapping in Plants;citation_pages=5-20;citation_volume=1;citation_year=2008;citation_issue=1;citation_doi=10.3835/plantgenome2008.02.0089" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome-Wide Association Mapping in Tomato (Solanum lycopersicum) Is Possible Using Genome Admixture of Solanum lycopersicum var. cerasiforme" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8.cover-source.jpg" />
<meta name="twitter:description" content="Genome-wide association mapping is an efficient way to identify quantitative trait loci controlling the variation of phenotypes, but the approach suffers severe limitations when one is studying inbred crops like cultivated tomato ( Solanum lycopersicum ). Such crops exhibit low rates of molecular polymorphism and high linkage disequilibrium, which reduces mapping resolution. The cherry type tomato ( S. lycopersicum var. cerasiforme ) genome has been described as an admixture between the cultivated tomato and its wild ancestor, S. pimpinellifolium . We have thus taken advantage of the properties of this admixture to improve the resolution of association mapping in tomato. As a proof of concept, we sequenced 81 DNA fragments distributed on chromosome 2 at different distances in a core collection of 90 tomato accessions, including mostly cherry type tomato accessions. The 81 Sequence Tag Sites revealed 352 SNPs and indels. Molecular diversity was greatest for S. pimpinellifolium accessions, intermediate for S. l. cerasiforme accessions, and lowest for the cultivated group. We assessed the structure of molecular polymorphism and the extent of linkage disequilibrium over genetic and physical distances. Linkage disequilibrium decreased under r 2 = 0.3 within 1 cM, and minimal estimated value (r2 = 0.13) was reached within 20 kb over the physical regions studied. Associations between polymorphisms and fruit weight, locule number, and soluble solid content were detected. Several candidate genes and quantitative trait loci previously identified were validated and new associations detected. This study shows the advantages of using a collection of S. l. cerasiforme accessions to overcome the low resolution of association mapping in tomato." />
<meta name="og-title" property="og:title" content="Genome-Wide Association Mapping in Tomato (Solanum lycopersicum) Is Possible Using Genome Admixture of Solanum lycopersicum var. cerasiforme" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/8/853" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genome-wide association mapping is an efficient way to identify quantitative trait loci controlling the variation of phenotypes, but the approach suffers severe limitations when one is studying inbred crops like cultivated tomato ( Solanum lycopersicum ). Such crops exhibit low rates of molecular polymorphism and high linkage disequilibrium, which reduces mapping resolution. The cherry type tomato ( S. lycopersicum var. cerasiforme ) genome has been described as an admixture between the cultivated tomato and its wild ancestor, S. pimpinellifolium . We have thus taken advantage of the properties of this admixture to improve the resolution of association mapping in tomato. As a proof of concept, we sequenced 81 DNA fragments distributed on chromosome 2 at different distances in a core collection of 90 tomato accessions, including mostly cherry type tomato accessions. The 81 Sequence Tag Sites revealed 352 SNPs and indels. Molecular diversity was greatest for S. pimpinellifolium accessions, intermediate for S. l. cerasiforme accessions, and lowest for the cultivated group. We assessed the structure of molecular polymorphism and the extent of linkage disequilibrium over genetic and physical distances. Linkage disequilibrium decreased under r 2 = 0.3 within 1 cM, and minimal estimated value (r2 = 0.13) was reached within 20 kb over the physical regions studied. Associations between polymorphisms and fruit weight, locule number, and soluble solid content were detected. Several candidate genes and quantitative trait loci previously identified were validated and new associations detected. This study shows the advantages of using a collection of S. l. cerasiforme accessions to overcome the low resolution of association mapping in tomato." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8.cover-source.jpg" />
    <title>Genome-Wide Association Mapping in Tomato (Solanum lycopersicum) Is Possible Using Genome Admixture of Solanum lycopersicum var. cerasiforme | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q0wst1" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__KPgiwEdjo5O9HdRryHqQs1TpsV-PeBb9B7sGBvwM9FQ__opzZWgPNgQzoEgfATRS5qCKwBw_6XcNjDHoDVOzQv8M__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__YUqwBlv6bNoCToiBWgAUSPtppdS8sbGfzmAH3qEDcI0__Mvk93h3ScFH5Xi9WhibVKlf368fHw-V7HWqWX2QMvtw__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XNWYaQubFGWYWZNytUHkWs-b34LV21DsB4s_6gchI4k__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__7cpBA0GVdrMqkZGGZEW-_SPF2HrbTCmVg9-lrP5l_AQ__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__TV2D4R18tPiSIOmeQ3vyMqHObN_FDmQ6EEIrfE5sAoA__c9BBZFy1BrgGgF_Auu9Fmysb3Z7J56eJGYvMcLQrLks__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q0wst1"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__OaHnYyci68QTwctdRYR30kEfu0eYWBrLyMqy8THv5Ik__IyfJQYM1q1QEPwYsK_Eslh2RQkxpy6yo-nR0p4DWPQQ__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__fhq_kmKS3zjIxlsAa3GE1ubLajyXUgXXTXULzAtiqfg__Lm1_Ls-bdpbWRyl0q4o0mmHyxB-139AJ68dE4zOskLQ__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"HzPOXarGSk0DHPjohTtiBKPWgAfPHON3yZ9OA9tw2D4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459304","apath":"\/ggg\/2\/8\/853.atom","pisa":"ggg;2\/8\/853","ac":{"\/ggg\/2\/8\/853.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/8\/853.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/8\/853"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/8\/853"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/8\/853":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NZX6dO0+6UzcHZUHV0E90aY9DncEk2dIf0enoxHsW+0DZBYoglD5BoWtGilolYTgpdrWBlzqLZxnpK0ecR7TXkn6OXuO+JhNUIoMs0TS71Lba91zHgKSWdkcM2GRqe3WPrUzO8XaYRXRi0rAWTW5mD+D1pJbJfj0fLErIVIPHpJowzDe7Cu2TQX9s2WRCvrQpp\\\\\\\/YpzRiCj8nw3pdNKBamg1D\\\\\\\/Uw6zEcWypB\\\\\\\/\\\\\\\/UaCS7EWSCPuUtCy9COJZuZZ73w9ZW7JPo6SEIfKOKHpC0mwPFB7reH9ZBMSKrR5qjoQX146QB1g3I1i2GHCy1XXOdEqSYwzLBl0pvzcN8yV2GNiSFS9Df\\\\\\\/WeoqEPky0CYKe+L1KT+FNdqs4Jxx+bgU3BakqdPp\\\\\\\/NDbAnfAifwbqiLlc0SEePceuoL0XUAgcPPq7OT3ji\\\\\\\/l6X6zx9dLMo3KDt3TzgdSpkY\\\\\\\/KGerDqy5ni\\\\\\\/lXuoWwtyVN0R7\\\\\\\/y9Z7UgbrzinnxIOCLW\\\\\\\/LsFLNj4+DoMtzNz1F6XW7z+ly4JPcsTU1fAYfefqGKpDTT6owjIHrE82olABebdTi\\\\\\\/HV8nwGvymQcHrfmkbaA9LuavOZP0wRoTlSFHKm8m7GiYqNLhgX4zZmgPDBua+g7UC4vJcS8Yrvqv1bfrba5iYXyoptFlfdJgR6sR1NGEZfM91tJ5KdZ9uaQRFBVEs77i6ASUHJtBVE1mstxn\\\\\\\/qnS49wD1\\\\\\\/0ZkqQqjen8EjzpWb1ltfrCnt0GI+tKv1\\\\\\\/6jpRoyHxcpr8kx+IheJBMSmFiRnkqQuVIV\\\\\\\/uexy9NhL2n0vYN2GE3OHvD1mbJ0RAP68XoINP9cGI0R80ihUHqQJojy2wEHKwgBIIHcFqRjO8SYOyB33IDnWh50x9laE8PU2aRlUYDV6CkJecAoC2nvL0nmrjKiawRPDmkWwYGzVufZx+CjMe9gd3kRxFAMqkaYof9msgIUZjSdwtt5BGvy1cMkPIc1XP9h+BDndRFROofHzTQ+MlCumhBc81KDWLm6UB91eZTIzNyMDRh7brqXaEH9TIeO+0G2wdBK9\\\\\\\/qhTPmoGHNyH1ttPq98\\\\\\\/6w1IhEJQd+\\\\\\\/7VSH1BJcnScG0dU42Bm3i0Z7DkYVWvLBC2ADKJtfc09OKRilROfuG3POJd4zVp6kGoywS3UOS3KxQ1tL\\\\\\\/jGp4NPc\\\\\\\/19yqG8IQTOrrDoYP2Vu0iy5AZhcoufw0lmuOPL1zHjxYLV2H+QmPrb08ljrAQwkoegCb\\\\\\\/oZkJiWD\\\\\\\/8kjrDibk2UdnT\\\\\\\/XnZ\\\\\\\/pbi48eyZsFJn6+Qj\\\\\\\/C\\\u0022,\\\u0022iv\\\u0022:\\\u0022w05EH2HXYjUcHW5mtT3A4g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226b8752c266f873c2d375cbc2e75818cd3a29237e29babaac477819888bbaa924\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xFWVVowbW594SL\\\\\\\/0rQafXozp6u6ZiAXkoZ3BJMdiGhyW7uH7vNuyjdola0cwSHNuiIwnNLalnh4YoDPkXg1gknZylLn20SKdjdvrMdeHR1VNAckkrRjJobQWz6hfxucxpIP99ScwJLmrRBtSSMTQyFDFC4qyzKHzdzqe\\\\\\\/MBOCXiDQ5rXth5xeYb8X\\\\\\\/Dh0bCe+ZWkJJU9jxKz7kAvuCf+z1TImWxvfT4UT60NI7icwu9d7cxYy\\\\\\\/f+bUHgAl0Ji26sCjoYucWm5taa0IUgO7h3SfxhcxWkBw6ZaRyosuYytxbcrdosxXahQmhqJcqsUijFgrPzaq63rE\\\\\\\/wRti4Eao1HOzpv0AfUQGN6iR3Lt7Ti6XhmLPTtUuMz+\\\\\\\/VU1wXfmY67nMsrGxEzRiOTi+EuEvCkqkAvvyl3+Tc+XPmUr2j5hhI\\\\\\\/IScD6EWcWDZ1yVp+GUoui5d3uoRnwI9dGWvmHvsKSbMWoESNu+7r8Ljjymb+wNje9opM60ePs+twyt6X+1ybbgBnMq\\\\\\\/JSjloMeSXx1LDrRA4HuytiIuCZ85PsQZesRAC8MkSzkXaWxGvWbCvlqsfBV1obKCbWY8Ro7W+imMj0rKARg3XSvV2P5Meyo5b30R9559bqN7tWwQnVc9IyxH39cMWVQAuML8sDT6wt\\\\\\\/dbjiCUo0eg04brB\\\\\\\/cuxoZ2auNGgX\\\\\\\/333Rywp7\\\\\\\/Ok\\\\\\\/hqXaio8HXlDUPik7UtrQHiWjLlmDqwF2AWeR5LXHQYUnKsjDn5iLH7+PJA7cfBZphx1m0HzW5dZ6aibnnhy2Q2nVg33HLaXYRFmXS2PVHPYVNJvgTl1ET0qIPZ4eeWIhVf2m2aO\\\\\\\/55l1cx2HsyuYMPky1pdCs+FE2msF\\\\\\\/M6ofrOSn00wPNLMZiGxiiPErDL4TZXLZBHH9mb6AMpjEOPv9wro+b84YUzuh1a0j\\\\\\\/\\\\\\\/XIVvn7lMqNDBWKB6kbuOi+N0okGvk3eKWFtJJA2yj5QMjbeK\\\\\\\/sHn52jTPWFbViWRQoSydnnnJv8u1QXNOlxC4zyEXZaIM5xckbjCA86WoSOG8Sb0+RtlkjhZPiOK8DwMB\\\\\\\/ctejAaOAYlnU4YBywMfisn9ufJvVxb1I7nfniSPYwZ+SLKWbsNW6kY3n19ooQsrLu0AbV04c5uiT4CY4nVAWBWduYTpzPyvYKfo6DTWCqSaX19Kv\\\\\\\/Ld6gpDG1+CcyU2R+rPmdqSUuHAiRxDvD46k4Dap7F700XYFTYjd2rbMepyrZsdpyyJ9yMeKotnGHfJKOJZf0P0GBz0Dsa+cK0aKMkJu6qZvlZ8JiLEWWDIUMxPVjqkYv4wzbhpQE03w7PMbwWJ2lcSQP0CKcoTUNCP8cp86BLWrtu8nLlueu8pQw6YZ3AzMPpLVXPMDx\\\\\\\/630YHuMZ9Zv+dwLTm50se5uigft1mwFALtIQJ+oF1pYMWX\\\\\\\/4Q3RAnNf2dR1ZbrbE0il0+FesWcFwnKeK\\\\\\\/k71p43DlWCfSrlGwzwGs2hQncTJON2uRxUnDpGKhDcutBMe98pyn5fVJBKYfzPVBWYaw1uyuqa0+mQMWZl97Oudy+xJSDLr0lCnHURKXgTNjg9Mkqjk\\\\\\\/QVFFWekF5gqWuPjQH0+V7kXoCzWQIybsLf2fgQql87FBrCqUKzQFXqKvuLMHV7GWWTa2WHvuO901EDYLZHZmBm1kP8wjdHdLv4h4mHd9bwRbaHIPJhg2GycItsbfMFs3lpH2\\\\\\\/UImzmxiRAJ1AY3MYcl5VIISyyJM7LQsT82\\\\\\\/3v0s1NL5eXZW2k\\\\\\\/Hn\\\\\\\/TXPygnxcBnKN7zyXckCkPOo1cCA3lZ4ygMk6HmtVZP98aOECaUFPNd3Eh6hP7YfzjI6\\\\\\\/VHRLKRmCPgVB0MgldNLVEPrqlsQnlb1QlJOBfqDO9UJ2Oyq2jxDYGdJ30qbETNMaw4HZfNT4kpxmK8W2zTwSPSe4mHWyeVeexrMNSfuU1Ol3kyxfr+TQnH0GDVab+2coP\\\\\\\/IktP8mo+ega3yKjbaOSPCZ4m8ZJMES30zlcqhnthYaYnu5dd+pTQOxewJ9G7vquT81alTAezc4e8yBZAs7+oN7k2sicJ1UUc0wtKXTXGumNaB0DlIJjTwxdcfZgAIVHErC0U+zoyrhBRfSujAR7\\\\\\\/3rELrHZJHk8qtTNCGqvc62ogvynqBIHdES2lYTBSlqC5oRTaif19\\\\\\\/tLVizOuEXqsjt2OrcBh8Jgg6wEv5e35gXMVUqnzQwfDIoJI5+jqvlzX62FkzEFumAogQDzdkkPs28LcDG0meqGB0ayamvG9Ua+5Tel+0NTWTPWAe2FTrBE7PWS3L8S7OOF2H6yJOOlEnYUdnHaTz1SBKzh1Y\\\\\\\/933nlBEG99vmiL6TBg0sK2EWlJrWaocc0yZQKVoFxM12VWUgPPGL42xzev2TR98DpPTLHaMSa4xetqMPbGEjrTEwASl38Zibl\\\\\\\/nPXYSZwSRVixG88ieHeIybChHhZk2vxpWlY0jjrXz\\\\\\\/X7zRdDlbJstHmJ4m8i0yluh0dvuu6k5E\\\\\\\/MUdMi3YnODdGNUJnUAO\\\\\\\/WU9zlwTgKbGIOwCj\\\\\\\/HwmA9hYLamTkRXtCwSYIvSFZcu3vISxOLmMlhBPBLv3pW9590Q1lL16ElH7ulWcLMiGdnuz5G4UoKsczlXSaY7f5sLD\\\\\\\/PNowO93QnnEkyPRffKWxeROL3\\\\\\\/LK5EZKbaFfPoZ6HwTnqbouSGYw+wg7X7f7vDmr8rGFT1AOqKy6f\\\\\\\/Tv2c\\\\\\\/rU2RfQ6jku0ZltB\\\\\\\/262OWyqCOlc8IPwrh5YzcsVwcTDPrthV6\\\\\\\/Lxom8CarKTXnlV3MoWcYqvL\\\\\\\/1+t3BdXHXsyhYbLa661LbKictFBaZGojQkyqW2R\\\\\\\/L8fzOWSTsKYZs6s14gawwWSAGKoQ\\\\\\\/JpdZ1aNY6443oNZ1pKw6iRrOvpFWCuBge2u8wmPV0lzhzyaPPQhbArWn87uQdR+ozpI7\\\\\\\/aSozXsee1wgri17WnImjsL\\\\\\\/Vo0LWjoqb1tmq9fD7UwG\\\\\\\/1hG7UhQd8OBRChuIF5gTinbnRrNdXntL4fMrsDmtg6jcHt2tI3k+9mWQgvz7IEt83+jbqbeZhubBAUTRrrSpaL2Ce7MTbF8DxMyBDMAgHxzvXb4jXSGXD8j0NWDb5zaBqg16cpJCDHslRxnl1UZPO0nOsLtjE1v3WmRlURMlXEaBZxnK2kG6ATAUJDAzrhlmBGEmfAjhHN\\\\\\\/59vZNo6dZN2sGJfedLo3V\\\\\\\/yQxYKgK9oxWIH\\\\\\\/xiou5L02HZ+0FljlgBC7iXz+7zCTK8X2xhNmGNXxdNvFJEBOSk62FQblyPYAp4k7DP\\\\\\\/RNyMTz2H1JD2CWzQDYG9pxDCKUzGYyNjYryliHJuMOCLaL1klj0qa5d1i8U\\\\\\\/7eSYbVKIFwxwLHs4EAc2glm2RVudUBADCWbzYg8WHwdWLR6EK4KCRY4743MNhvsSSjNJxsYzwmwDd484cOpOsDL9ZJkBBZ7Q3YJYxJFxt2YrrNybbAoyMzCHcUXFzY\\\\\\\/qtDoTfcIXo7RT1zn6SjOQsJx7u7DBpGpyw3ohpque4HnFGNRF9f1n0Vi8T82KzAIcz54L04WRn0Ak77njGJRa5pcfdSwmWx5WVJcaDRhHlKfei\\\\\\\/EozgiXB0KZaPwdi5ub4K7Jcq5njoD4ZzsT\\\\\\\/4UMhkfDjcUVFUpNketVzKN8jTbui\\\\\\\/Vnzdwj+Xugf0LsoY3Ny1Frp2r0cMoS\\\\\\\/zoR2SjYgXhAr+aB8I+y6KeDrw0vN2Pw3N\\\\\\\/tkb9uNIbOmokmRfBkoPIwtdQ94y4KQy4ecJKWHkkIlsG13sYapwPHTP1\\\\\\\/+YGI0MGBP77AG7CVoFU9\\\\\\\/WTrqzTz2HM3hacc23TXbmsaJ+5qQdpt4IxziPtzL8hLtLT\\\\\\\/jkhtmH8v2KqV6KfR+Fr0g2Q9luGlLt1FqF132xmdkHiEbe1O635LhmMYQVukBHKjYhiE+3DYU3iIXo\\\\\\\/AOvo2vjI173WS6AjlwNIkQvI1mGW7+Pt1mkg9cfWrAPP0SQd342gkyo0iYhi5WsVi4Pq1ivmOrr7I1tdpkq2A0zl0t4BOyNSAGANJ0j+A+suIEBC56VO99lpMSHutqIXA+AuH\\\\\\\/APuaY\\\\\\\/agLRlT4\\\\\\\/jquWZcwCH5LMTCJRFE7S7hm+6IKBZi5U9\\\\\\\/BLq7sa1J6cHKm1pOh0+X\\\\\\\/3cL9mADKp7v3nt+u6ydGdrLmyuOV57iCEfEckrjqdoVEh5k+sJdlMAukeJOF5JVK6OVjur1ozICkbG9JepKpfQvF3KcLwxx3KwGzwQngknD8QRiKEKQ3GmCtl6kZhL43mWwBQwk4Esr6QdRiXWBDvOB8QWlpXzgaFsoWh4dKIHUgREcvsJM62ad6U8TUCtEXnECpk+2pB20dvbOeSmzVLYtFp9\\\\\\\/zv1ikiTgdJ2dSOsS\\\\\\\/jl6WHBRzaj\\\\\\\/bXEWvd+Nh3Gu4ejqJ96819R1doLucJBItt6hhAZij8sowxQigfBJdPxLFiNNHWrFZ3Eld83Ice2jR2ke5vBqzmlyOXJ5gMSeWA2ZcAfnNoWlMaNmlMwEYeZwtaATahqhYjdi3Ae7qWTZg+I12UazvU2uPkG5VO99r79UHEh8mQZ4pYOqBgn0rfzaxFBwzyvRSfPrcQHA2hjmkJdPF0Da95jsEndfomCZdrORY5OlgnXMdFBZzRBnKhgQjWosec0OlFrN7BOwEcJ4n9r\\\\\\\/VO1Cn8NPVy7KKsDlBxIOKo7n6bzzdL3\\\\\\\/rLnJ98YkIm5C3s+bn+5kDorvxRz19ISYPLKE5grNvE+t88WDKzMPSTq08fPY7aUnaTcffr0WAFe\\\\\\\/9XwHUM35wG7fA64PdNR48FqRBZsc0zp8f+HZdBzPAQ0eEYvtkFQB96Wth3AHlJYpcqOqO4wZy\\\\\\\/DVQcYeS6wjmg3ka3ev+oRg4jH2GAQrfCOFxh3EZycSQF1QBkbEuFXIW1YdlHLsXUtMuRr8IIebZE1tdHvrYWUCeP+vh7ehfxwSBgnYPWMROBP52HmEfT5b8ufWbD8KG4JEAn\\\\\\\/+wF3ZSeBeIhBkGKgzMf60Z24Le7W9rLgK1x8YP9a\\\\\\\/TlVKOEQBRtsgHzs0GWPDZi7jeSOEtciyzzMwK+R5G9P4iNb4hgrbHRu\\\\\\\/oOXIiDSZieIWCJDuPfmvsBT2eRibLnhZLDwrwuMSnm5JB6JgyrWgdIV332HKS89We44vHHxvJJixwCNOJ1GskYGuNuoapegDdfUIRjCVrAG8ItALnDY3QHm3rjbJR2qh+Hh9enNgMSijCXmjph4VsMt5VT1AjNlXu+Yi371uMBfCuRGiElkP0\\\\\\\/6b8SmBohILk9h6a5lkk\\\\\\\/4OYSdNjf899wTP7uqY3cwoOCiAVqDrxA\\\\\\\/HxtPhpAriSpOGgfrijD0s3EqoMzryRs1C4f0zgBhQZI+0rR\\\\\\\/XTtBjuIhM4BjfcrSZLrC9w\\\\\\\/4SZWJSPO6PNDEYipUMgR3z\\\\\\\/HaRC2\\\\\\\/FlznR8kcwC9ilq4IsZf2YMCJlY70rUpNiP8niIY8yrbtAin8wiQFaWQPRQnuswSx6sJOZrdiIOsuMvUnME+qx6k7HESe\\\\\\\/BiNaPx7HSqfcuy9+SrwSV3wWmp0l5SlFHuJJ8ZxFQylqdL+QY97d\\\\\\\/kOVXKWWHBJ0F3o\\\\\\\/9UKFcwsPm7q3\\\\\\\/MxjeLvQp3Qu\\\\\\\/JsAVTCaQyXKm02VZCKeLXeDhpNkutEzAoCEMlr1oFs5v6BVTg1ebxOScRKCb2dg\\\\\\\/ouPxh6tuEpFBLbYa2xhk2o8C0WzmOa0JvMZpG1lqvOY9n09\\\\\\\/akcpRYYMjwBpijeS0doCSVxDyj5SI2xi2pLzHlJI7VDGsWfrSd2eRDYxZFxp7ikyb\\\\\\\/C2wbWnOQqnhCC8amjcTTDDnOhwU6o+ZS1gPOIJEb+IV7jgJexH2j\\\\\\\/KL5zIRBvjwglmX\\\\\\\/oCrvG73f+wI3Ybl\\\\\\\/Safn99OFNJd2Z\\\\\\\/wW93hWWPJRXV0G6NjF2wIc2bGHH+URQA9S8DIONgEaJNMKYi5nYIGBE9hpfxcQHiE+TP+sMb\\\\\\\/s4Z7iyrYixbwi7EMCm+dyJ5olg7t7s65jzn0H3FPvXHM1rdvb1ZeKHKk102nLIdY\\\\\\\/r0d8S9Yg8MBJ6WGfUg6ySMBGLyPN9KA6u4\\\\\\\/LDA928bYCzxnqkO+UGYXZhQ6T4vSVf2Tx+7nGbDGKJCJ7zU8N\\\\\\\/YqA2IgL5Odj\\\\\\\/oz45o\\\\\\\/+HmP1z5ixNLqF0VIiIELivF6fPWQXAEi9oMBuuffpPfwO39UIZxScyseziB+RCdwWSDVvgjshfFUqCgZQ0XwoLuFDvfX3VEIIWAoH5CfVx5GbZAwQL8H4hlirF9T5S2Q+ZAk3etqZDPmK2adFotx3K03NZ34s6mc8GCS\\\\\\\/GOYKfC6aI5ge8WlXbbZvPalSsfNyMh31seBF2SeI0hd0xJ8\\\\\\\/H4dghCYd03lg9AKChA1OFjJ0Vg74DS08MVBaAMqGKj1iHUvvQoYbLOEB4sXPrnfOoPHUKexL5qJxepZAXjvKkTFMp2tHOI924rMXY8nETUPX\\\\\\\/xDm9DAHlNkfv4FqTIto8Qs\\\\\\\/jpVGVNajseicF1hnabO+\\\\\\\/YOIGCkx21qjx1MwyugD4pc0OaDwa3EKAsf\\\\\\\/UNGoavGdKa5mrlhnMkbTNkNqTadqbadaxd6L1X8JmxIT+C3k2z2oOyQ+1OkzPEA9vL\\\\\\\/kQRXozHKZR28AjL\\\\\\\/ZYqyvjIWSR3lK4wWd+5wncPCqzWMXOxMwqwolqfd5ZqJU1hgP8ocN1MydnVVHwdyyYtWQW\\\\\\\/NN6WLt4D6ZRRA9p73ZjBp6gZDj3\\\\\\\/YzYEiAMNXDLNj93ujdTwmOiU1LNWKG\\\\\\\/uzpI4+5nUkfDXZkqwjpEM8Xz\\\\\\\/m5JTOFur9vcTIpPiWecs0cFzTGmkGiD7aAwOfU8MzknPbiJE0yQbEKNMnPue0mjhosq1Jcnesa3VluIs7o3zFsSTKN358FoAnDABBYqMDOrU4tSKsJTm4fd3FEN0BLHP56iz6EvfSeHcmZUX0cWPcM5joafczDmeLOYB\\\\\\\/TYRROjUlggROT\\\\\\\/Q925Eh7rFo\\\\\\\/rohRJ5EIMeU\\\\\\\/CmHBdZAJiw+uUzvojrEhHvMS6SCqBcY2tlWVPQ36w6aPOkX2IDe\\\\\\\/L9e2lQzHGuQUgT\\\\\\\/XUFtnKb7s1Nc4A0PxQJmrpiy6CP7zC4NcbtWVyCXJusAWmYtHmmyoKYB7CAxYOROx\\\\\\\/bHweHvDuu47iEVwpfRjBYtqUwzWVhKO8QO\\\\\\\/HqTmpMncyvuG2uS9PCd9GyENaVWGRo082itn\\\\\\\/pQQMGtGumDlfsquITzR2eBcU6ZahLcY+vJeso1VPpOts9JSzae0rQdI30k9scH7P0PLm2a4zpTBrIxJ7QIpLVkWQ8gxKSpbEv1loUAYx1zuKI0nVaNF6c5UDihAnm1ycWNXvyWIRFJmljfIUyBnte9h04Nx+JDZcS0B\\\\\\\/5FYE0fXPBzO7aKxT2R2xn\\\\\\\/M9+eZEzo+8nhWlPwKN0b3iGN7fiNBqsteQFMpfyvJx1kbLEbefIaGY0Iw2b4alq1BOkkzVTcuR8sGJa57Sjcbj266cj93cDtYl++\\\\\\\/y0gj27IysiLsx4014L\\\\\\\/HLjKF9R5LRrjgG69XSkf7C4tzy+JV2KqQxM2W03YguexF3eJhoxBrlU4QbjLBbMxycnHta57cw3y7SKCR5G56Splr+kGkEI1D1F5ie1rZT1uO+hKWsWnrC920aiyXhXbEBlQvnGz7AtUa5X55XLta5apvmnSeXuBktpU60d7suojdQYzy2UMLHOf6itm1+YI7AZ08Csxe52dPYi264SHe9psddqurkgIbXlLTpSPxUQn+3IIYXnvVmwU0yKiNLQHkfnybQUFNiRMC7Yz2FEp84woR\\\\\\\/1i57b0bFbdj9vcBF7UInZ3\\\\\\\/dL3Dyx59rL\\\\\\\/Uymr6K67inRq4ArvvOtc0z1bVx2mAy3xHi1B6IjZ7xSmDSkSGTpixaKWF3ID5KRmR+T5nq7Ulna3UvR3K2nfmtJ8SCHU1rKR3\\\\\\\/SAiNvVtROQsQtlbQagYjjiCIfHM1yz7U8uZTLuxY9FduYrxliA4ms191UdC2HDg2AZpqXS\\\\\\\/NWvlHXP10RoVxT4bzebifZXd4+XQW7ABWYjzIN4m1p4iPv+MZbVv72\\\\\\\/QVdCJYU1aHgLf6MXnlM9FFFOh2nYNVtPBIpAlenPGtfEgE309eqDH3HkBx3WBbetC3EBAC483b5D0jqrZp901nz+sprGUw\\\\\\\/HfIbj\\\\\\\/MhDhykw1gjuehTEdYaYmUsx1pzIkcuYoKk6oJ4di7pGroAO0v3G55rIM8DpE\\\\\\\/5C0lUGfPjMNmlCRz0vRFa\\\\\\\/kSGR\\\\\\\/99szLQ2sXKF7hgQ3bJyWoig7UIw+VwcjtaKk4uxVERt5\\\\\\\/rqCapckcvDCnvoFwj382+SkCEgsfbPL+8M0\\\\\\\/PTjFT1C8cIc\\\\\\\/dH7j++SPgJf41lLBWJMV8Eet9Guhrw1f+qpYh6jc0VT0sg+C2DOftpvbQXMU6i\\\\\\\/l3nLvl2vaHSaGVj6gM1A5iTH9qaH1Tdlg2QDTUDkNkQzy+JHclT\\\\\\\/9EjCiMZYtM96EVL2TU8HVy\\\\\\\/QeFoEJqABBcOkT48yJA\\\\\\\/jZCieq9iVAA9SPeDfUeOFVA0gwXJJVIhA3i1EAfEot5S\\\\\\\/BuyCQpj8\\\\\\\/OSVQlbPJNyx\\\\\\\/OOvgiq\\\\\\\/4E9+IFRjUjHl4W71eB8NRXtXXxZAQI8H6bZvbk1cKkYpj7xJuN+SzKcIrwNB77rPACO\\\\\\\/4ssU3cv2iCoF+WJHigps8mZNkyADeOGR+S9EA2RfDJbt\\\\\\\/DZc6L1y\\\\\\\/q9nPssOrMYmdvEBg+8vhq6yy1GtraOjOHKzbhGi+wdpI4g\\\\\\\/h+fMv4ewwImwLoUC0ES5hlQ\\\\\\\/rxViEr8rzCLQ6qLSzIwbVuvj5D6kmOIIkDHVHGbXbQuWrgV6qNDvpHjwrd8tJY7i8cCFHLwoMEyiq7d89MXpEM5U6N+Rry\\\\\\\/76f6XrEdlueEwGRDVnxvbbN5q1CG1+AT+HDhbL+FGT3O0GNqDBTChBVjCpzR+jbq3Q9yHiHLGC6rDXOFu\\\\\\\/fhYAXU0\\\\\\\/SF5QMTA59TVi+9hBRcHScnWfF83WXWdKbbOrtRbk\\\\\\\/qHZmLI74uGik1duHDTqDtWX3+7AiXFgIxUnOfCmzX6+ATuQXyixZR6bV56F+vbMmjoEpw6qT\\\\\\\/GxAKyB+rRcMLrJnx\\\\\\\/IpVnjVheLTQ6TwtBYPgbJNbTMEcyYoQ+5vZjyOyZ7\\\\\\\/G61UafmQi9ptTMLlQ5rE0VEBu\\\\\\\/wyAqEGUIwUXXzuOfN3zCfqLZUD7vDQcwV\\\\\\\/pf5MDn\\\\\\\/AmSXPRB7KlvDziDog2pczEbpSRjTYqF7nVt8TMD8H3bIfs+xQtPYw181LHf9LQfmqxFO+191ZDXXmDJFbPthZHbOxg2xKLl1VtQJNoTxSAi5kzXYBvGZxeBA3ZK6pDaDCR5y44t6zqelsDxLpTKlzFSHBMuQT7cG4SnSIrJlcORD38FciCnSHZljYMB\\\\\\\/iW0dQG0VGxrluBwV8q78mJdGs1DNhcD4026vqLqxciO\\\\\\\/T5H7uOCvkEVuL5QWpzHLX3lcbMuuSVm6oOGxjbX\\\\\\\/WdWpfVPf0K31TdQiqagrp8xBo9tyTZP\\\\\\\/X\\\\\\\/1WjQJPqHL\\\\\\\/vJfSn29MYLxvQIChUsBVyfQWdWqKuvoHv1NuRbDx72TcKcgiW7Ls9sbEeQud6ICOUuQeJ+7BoihOSzVFAkPitefHjtv+BO0UoxJm+YJv04mb+m\\\\\\\/J\\\\\\\/EEBAWH7RoTnizAv8z+00obhh1jBP83mRlO54J29E1ydDCf4v1FssFONPkvwYwK8QR5hGyv+onzqJaHDAF5O9ay+nNUp88sti\\\\\\\/GzV8uPOeoNhhNH67t5KHOYUJiY4++\\\\\\\/AbXixQ6OBX+khn+jly+YhcIBNcMdkIntmC4ZkXIEd14bRTbYKh\\\\\\\/QDGovofAx6EE8CJFzrgHAx7fZWf2cdoqBJcyc9HtLFxB3J1h3pkJI2CjR3iiaCGNl8qJnFcJMGNlVFk\\\\\\\/cF1hw5My0q6y8pVNCGQ\\\\\\\/WgPiYWvoCXyncVKL3RuLPmxO+DmUZ9VBDcoAUxUdo80yMG+Fn8l\\\\\\\/J8ZB16RYTbx0Td3nsDcJ86x5GClhJh+Z+P8iMvE2hsWxOTiunhUJO1K1FONvOo4RZLie+jGIwOcWmYtoNM2AFCDgygKahDLflEBsDwKoeLKNBM3\\\\\\\/KAlDeRT\\\\\\\/MXYpzsLuMeqvZvi4NjSDLtpxNLceMGh8jqxmmnFr17Uo9jewNBllhiuef\\\\\\\/k9Kid5QWUA6M8P9EY0v6IuWE99yzKpYt2gmc18Cy8cxugxe70pj3Tsh8xoPtjZCJv1bqETwQlP8PM1oW8Oes4yE4C0sVKf\\\\\\\/jYuVyB9bFTybGruOpuLJBjWIKDrMzhG3NgJ0GLP8wURRSGP067JnCGeamk9NfYREI+tNQK7zbQ4ftQEDqEz6ML8TtTDshhlWYJ4LTxdxyu3xejbzrzbwZSGul96rUTGEE\\\\\\\/DcxOpyQ3wggRvM08W\\\\\\\/BOaEuXsfDgkRsX2EwqFQsyzJybdv2mzCTj084BZu0bLq0Kk64KMes1\\\\\\\/yBgLJrV3Bp8aghsoOx6YKFsCsPXLCkiDv\\\\\\\/G6WvR3vH4eTRy7GUhJD9bZ3NRiK+BgJDSyJSsNDOm\\\\\\\/UUweVNdqXNSwSK9qmlD1zwYQ9E19po1z7BseOZDXJRm4nViH1XmSJ8sWQkb23qxVOs9dq8g9t8zjbsTj7GnDKBbJkuRy7Le6BBRtbhroFiW2+cQZZik04WDSwZL+UNx73CI7EfhaDMt7hm5v290eUHm\\\\\\\/gUVQVJQ68cYI+u37t6LNFTMaV923mmxPDzC3s\\\\\\\/QdLFX8LFmq8wxNoNDLea7alP2nTnWeF+V4G7JVX8IEKobXsQlB\\\\\\\/VuzeKHWYfx\\\\\\\/NHB1k2THI\\\\\\\/fjub8OLhUW9s4vdhUuR67StHowdhXAoz21f9fsUB+yj\\\\\\\/01wTTzCB0xTKyeMo2vmDlf+NZGzK4hPIbO1pHDkMYWv5qq8\\\\\\\/SES0tHwKI7mnbHw92b5IoLbt9wSD6PLAKwtFqV30nqrY5TRpF5uqOQlVqPUWV7JhpdR0rM\\\\\\\/Kzh2qXHhDMzzTQryFCySsdk1Yy\\\\\\\/6yi71fX2ZG6Z1YA+H2HlWX1G7K+AnvjLS\\\\\\\/zot\\\\\\\/lKDLtC4Y3hPbFNhRnC4XQk3RpdY8\\\\\\\/7+XGrJA8\\\\\\\/DopOL7wAlbx3gxtDTTNCrJ4x5fcGBV0Fv+HnIiiykfgO2BulfAhHKstxPg3OKo1wVpURPhTIoM+7dagEohNnxyJYlW8Fz6LG7dpfEtuiph3KRKZnu3i\\\\\\\/M0GZ\\\\\\\/zxpknoy4yV6XMRDmydvz07yyiv38FcZ2lGsQin9tG5pI9MvqMbSSoQDgxRQsvIkFSf8lCrxUMf0+v0m+lT9HkX+gk3A2asGppygyZabuI9oCmb7rTRflZBqfH85EBDMKHonDvzxuFm+2TGRUsKb8UVE2GHg43LZgUbbGg17BzFjxTpkoUvs8VcQtQgglxjEGaVXpNO7wrT+SYpu2h0cxpz2cBPhTjkNyoKMIwzxLt6odKwAHZpv9I6gxWvVyaGRAkYakb4oNELNNJDQbFGlarSQHlsK1lRSntrTY0y\\\\\\\/PFFQD3giQyipH6gZvuvKkTJRzjtWa6ZvUyj17vR5K848\\\\\\\/vipUXQvDFswhv\\\\\\\/Evc6Gg938VzKbizdwKVXlmtiJZDkmp1pOhpVRBP5GeSrjcokvEoWKdXqBswtu82vihcQ3+gw6evoUH2YX+AJZ+mnftKjik0J1eVTrWBqjmJbu5IEBbl2UHKRZ\\\\\\\/iNhCrNiGCNxDL17NzxJ0iJrA7iPG9hN\\\\\\\/001ULG21R18USr\\\\\\\/zl9SKvevNWMjbhHQHt3MkOn5asFgB9+02nzUj+nF2nz4tFW6kEsEhLahhLLIzA3OjUuHymn4Cj8RGlbllxwQr26fp7S3+4yzE0O4sM0L+ZhKyY+pXMmN8iLdpfeb8fgK20lJZjdiAIT+0Dp8fvqtmsne48jk+3ZHh+eZadxv+Ct6lmIZDa5\\\\\\\/fBYmjVeqh25p5urt2B4oPaIP1ifMi9H9vb20Zck9OnahHjG985y4sTaFHzY+N7xXt0syKtuxxcGcmCIMIL681FoLcTNpVazNC\\\\\\\/Do4T3bL53ummta3pw+onZh3Jt2zwaSvxSAHGtfCfpPPUtor8PwwSsdkA7b7\\\\\\\/2uGSW91kPENFHlyG0v3O8Iqbk7Fwd7KU3700mCWaCf4tEZPpmtpV3UYY5\\\\\\\/Cy9S1GjXz+TljWeI\\\\\\\/oFfoMVPVepHKDOqLk2oKE9MeWh857+2rhTX\\\\\\\/jEN2P3k9A7srnAHf5WBWAmQpyIcUvd3B1uKoQR4brF9iLr5QMVk3lLbhl307fas0dNzTaQ6yLHNCFaWefgG0hPQlN6ie8mMU\\\\\\\/f3waZcYOyxCEbArhk5H25Zb3\\\\\\\/V5xiUVlG\\\\\\\/cWxy1PMAP33lL2EJFfc7N7D06qIRt\\\\\\\/LGgtVWsnZL0qX+PL\\\\\\\/0xvwYqyqa18CS3Ne7nd8i9DT\\\\\\\/BuzRgoL9+4xgQ8VpDjLaFNr0Stw\\\\\\\/gWo6KZxOjVVoDr0UfSyuu0lmqKdbPNBiDvv74FVH27QWZWJvZ4t1qKmyetTNlTk388gsdfC888z7N4BWctfwuPvsx9gXeU3deH2YM49U67bLqyQmwYOC4UGpGLLe+d52xiuR1+gdG3kbY6hnF+6mAXYehv83V7ghwRPWC7YZvj89jm20hFpUdB2pCweYKg+MzDBpZIwjj0mTOHg5hwlfFPdC045l07JMSg8v6I39sZe7EVTu20+W6GELJwI+VB0X77XrUh9wLuPKDfNzDSlJo36OlB97iYGoCLSdKrDD0tE\\\\\\\/ix3shkNg6YNOUBRwwMzBP14ofiYfxNRCE15qwNbysLnvCPs96vUGhvHF62gsy8IAyNJVg73orLz86r31WS2crzbxb0licKpVmQplFoIf\\\\\\\/KO4fkm19e7s8Q0ww8kJrmium02AKIEH+Bk10LIUVDAJg81NVMpw3zgS9uJvlgLFvsi4UGNZHvvsBL2TmQDocpkv48Ux3SjJNPikZXHhOaRxzWcqipES9AWXphl8VMn6zbH2WVIlH0lbF7XG7KSFzqC9hFN588CsfcSu+bLBDt+7fN5q4gwAt23u5blxWYhRWtxxd+pgZnQY0xI7nYDIMAwhJfnUfNAIXPRPTOHzfrfYFHkdDJBfUPURXqniR86OPlj94Qcw04zXHXewYcZORYFVa3OLcJHRN0pvg0u3Lxo5\\\\\\\/25QTfgdwLRr9wBVMm5x8jds4RotujwjPG9rorDO8aRkGWcN7JuDFBfmkvdL2pniTvC1RmkkXS7w+gdJiJtV0Hdq3LLAWWVwvkQwihEd4qx1v+bPHqdgIvCiXvuwTE8dom87Z\\\\\\\/QeLNp8BrBHfoumi3HcnE30+Rdh53FnAtpTnuc\\\\\\\/nIbfer9zJJCVF5nImi\\\\\\\/RgFHsjU8BG3rIYkx2EiFKjzAs0aeHucApoo+It2BazObip0R0SrS6UwJMmMLwBPV5qiPGKToOPc+5FSfQUzjy5bL8SHiO4gmMTOzG7RRjOobl9UclMZ6TYJION1Qs8OOfB4A4YoVbeW8ZqoXd1zr+UPk1PxubbaVe2uwDJVdnJFBU2luBjhWYKO5mE1et2DgphpDMup2Vq\\\\\\\/zqrzzmk+4\\\\\\\/kxS04LoSXxaomtMCm\\\\\\\/OwmaCS9bQO2ZdyAdEK6Rno3sORYFQsxMQe2LqJf5ih9mou9e64UzNPEG0rmrjnmJms+LevYoi5tD27F141NLQPH5Spi65tlzEWAoWbTH5+nINxr7TURUZXOPkG9SLSmk+4BS+NQZq2JLXz5bQq7K04Z2VLnADGVsTyVVgZAEdo03SgW1bmMPtMmu8r2K83zbN2nWQE9cMsMrLTS6iChuVIARYBYXzKvapEofYBVr22mfqcfnT\\\\\\\/9XIzYgc8PI7LwLBaYZh6ZxZEtZCjn0YRHi23JWyiFuql88C5rDaDgfbiAYYxQRe059ELVfK\\\\\\\/zBXA1H0Zee0kTRGkE4bUP7cY1cB+Pjye3o8hCclyp4XMKGoXNpPEihjjHaPhwsl\\\\\\\/Hl9NV7udlDSGYdvvnqUIQC735kg7z5ctTzTuHK7h7\\\\\\\/x8XRx6IhoLDwRBhOeFu9hdbKsfBZoW2N7xnuNTNiwi4IB67dgtolfKyihirDVh8vZwH0uNwuglvqdnfzV6wydgJe4Rh\\\\\\\/UP1J2gY0I5mp8Fx93BJkaQ0fNiFmYlt6GJ0kAKBhp4MgML+gEhUnegTvQ5HJ+D9DYLJhvSd8\\\\\\\/RMnMApvAFN74+TTthO6D298s5NIuI3TmVU2Bq6V5CpKfu1Y9KEGi3b8L+zzvn1HkPx8\\\\\\\/rtguP5n8v37mO6KAVDjGXwKITh912UHkZHY3fDwnPEHUul8aPtEdpkFBCF8jBD8vBJRTGQv1sUpf+AdK4Ld2H+T3iVtn\\\\\\\/SRmKkozN5joPIxE\\\\\\\/ryvGKUNY0pdSIkGeBB0eVp2dnWVhhbFeKi6hahne5CArqBlM6su9EXuIIK7jdZBh6ryK+Hnq0QgQeD9ZeEBMAZpSNyj6bgFDeozkmJy2q0sXShsNXkJ6+Z8bs6e0TDDu3x58GvnFfKpkrdoREbhyj5nj9dUhgZEnqva\\\\\\\/2XV9tNxGDpWWh5jDshRvfxfATOLeYRa5BaPuna+8j\\\\\\\/a4k0rXO2GIeZUSzeQbxkpM6VBJ8HWEp4DfDr\\\\\\\/I+r1XQ4\\\\\\\/wDQTVDKb55MHYI6ODrsieMDV72EkwyHauGP1kRV+L4V\\\\\\\/q76Fl9GvXs\\\\\\\/idzJdaUuJRVa2O\\\\\\\/Rc4G5fI0ASra+mZixkr5WrAKFPad0hbXFwtgKb6xDA1TEgBQAhPm5OCAxloAL+PsTgbLr3+tu8s\\\\\\\/J57HUlw6\\\\\\\/ZfRLiB67y9yYtU6f23QyV+Y4mH3+B7vn+LguKMitJOnpyjrcJbd\\\\\\\/OQC67OvcuBmdJVnibrcBxI7MtCRtbxEbxK8Jcajx2lYhZ9zobTLQpoPzrJdkGx4W32sJiMyE9WqbkGI\\\\\\\/7UAZFVCdumXOnhb+D9cRzY+83FKJsTbeLwsto9DQTJHKFWSzE\\\\\\\/erMKo1ynmz2Sc\\\\\\\/nqKkhIlKp+tvTGyWqsYUyjWDkUHRD\\\\\\\/+6IYhJQv\\\\\\\/dG+\\\\\\\/CTAXO68hU9qkdatbdgMU9cOzsYvDXc2KICql6\\\\\\\/SurbaCerqT9ZNceYQU1MMeMghgY6ysM59w\\\\\\\/DK\\\\\\\/jWsznQDbPbfCrTG84Tc7dquaFUtNMuYa7XT3Nu8odQzNmu0b8pbWVlh4PjcQR1aZ9nPCtu20473ZMQWC0TFzlKdIHaMksNI7MrGf4niLDT16wKJJdMUeSmyiw01x976MihEnjx0WhTqcbPNoXv7M9KLLA0X4MKl6GSIYRaUpEdIOBaUW5B7XrOmB+VSbuFxSh4GIazWZ+WMyfFozNuSgaE9umSb0G6IwPLVzDf2WozA1sI5xZ9a8UM+Wb+LHzNbG1Sl7B3a6DktK6EW022L05UDy8NWN563MvBb9TLCGOB\\\\\\\/VwsZzehrf4hrz7PCW\\\\\\\/KMhGKHUy65HtqovdulR968lzUDfN1tUFYoiLsAdRSYc27ZNvNqhsUI\\\\\\\/M32tXa7fOF2TzfR+cPEZB8N4\\\\\\\/2dWvHb5fcYV+f6RmlGEhmTfll0HR065LHXZchUY1gTmorslPVB2k6Yhds99aby4hT5c3KHDuinH43WjDUruMKZRyjCqhJdaNxWHkDTcoolIsxGmhueOrMiL44PyfjIdtUhj0velTXz9PhgVvQEhCdGj4Sy6HIV5D2FETCpota\\\\\\\/V8OZ2hQnEKLvsmpeG6Le2QCSH2ZEf8ELLsaAVxjFQyeamaBzifdfM4AP0UlgtO8k775bi2q0\\\\\\\/FPiHEnQVihDcXSe43E1dutywQHrmN+e6yf\\\\\\\/dNFbcnSt1v7RQplaYZUYgrz4CeI8mLyE+XnDV1ZPjd8bmanO8A2VfSKpVWM3bqXBzb7U4TORaMBkZxjuFlOb4+P0XvlOqAURYVd7UC8L84d9XSNUPCd6rbqGZX\\\\\\\/lHTfMWgNY2QrQnaO0FerfxyajbOdGROfqqcdqoB4dy1x58dkP85MLL5\\\\\\\/A\\\\\\\/VhfAYRXZfqLNg5e0mR91RT8g1CApyz8DdOA83TWERoMP8vxSjOgtHV7x8INJIOQCKZeuMoqryK6TvgLmQE627+xPh7jEAxqrc2mci6QHlgNQO9Qqgs\\\\\\\/UYzkEjoELbvnCuie3mDCBI+8+ixIvdyqs+kmQbgxp\\\\\\\/Yw9kX39k\\\\\\\/vjZZTt\\\\\\\/SIvQ4J6UtvNCV1mQ9u9TQdZ6YidkXtvi3Q3OfCd6i3J\\\\\\\/heKpMn5jyRvaG3onqAV1SgU6zJUdYZrA77aRsSgNJWTKLOe5tUpfOX\\\\\\\/v6R8jXMCGlBmS4PmPka0fc0EmSgz+VrHmNFS+0tVjhX4UBJIMX1sI4Yvjf\\\\\\\/5yxzHsTZ5ArCzFSqIt9gN2dLntx\\\\\\\/a3mpFaLEfldpnyuVIf0sq6Ca3k+y02piOoXUOYWm6V06jsKAMJdfKh4SaF4Jhywcftozszo3myzvIDG9kAEp+KimrgNDsgmEYO3l1V9mNxJ\\\\\\\/Ax\\\\\\\/EA4CTns48adPyYDHtXDQ3k6ROVrcVb21ZAHiS+R5XlmN8eXVts0L2n4UjX6Ir55E4S+ZcPI+Z7eb\\\\\\\/lBtw6XqcfHLU9tB4FI7AVaiZIMYGTi7K0fRknqEGtBCE9vDuuQXxTWh1dh+e6NQJHo\\\\\\\/hPfyj05Z72OY036O5Sy5AK5zYjCH8oCScvRTZbeRA7dy1jhCYxKI7jNzAt1d7v0kGolCXpi2hNw+oYGmksH9URdTCIzgguIoZ5pRSEWQHkz0Jorl76lt2IsQdvZfRHgCozIaCK3u3fbWIvZzshltwAfw8M\\\\\\\/0yefKLJJi7mGj6qip5dAajzdKJPHmTg8AaLGR2gxb4gDQ9cG77v9rgYs1cKqdzPJnnngGDixkBbw\\\\\\\/ZkqNb2\\\\\\\/+7fNgZjd9GVvdXviaC1\\\\\\\/W9+yWtj3MSXe17AxHpJUIK1rzH\\\\\\\/uEOcpJUeclEsKkrArQkoMSbeHMkmgXtX09MfvRFPtuYKPB5XSSnzEeunTo9lbM3k3mw\\\\\\\/zq8i4xukJA\\\\\\\/IhvuwNh2iSmkfye1+hYfcdB6kYgWLMK9WM4HZ7QunqjtJnlVEFoQXcE32p0EolNil93Mp5epLt\\\\\\\/LbQlA2Nu6ROAj1PopvbsgE5JBCsXSD+HI50kT5fxPsLkpUBK3CfPqoMCaOLOx9Fc2s\\\\\\\/vxT4oCGNt56A5CxPUHA1Y8CX8zb45n1xGAJdc1rWLOgopLRTXjN6l6VBSbuzW48N41mMaiOj8FOK2bKAb+4bLP35Ammi+mepQxhFT36q0UYdneliQExXwO0+JB4nlSi+GKzzpz8Etk8ZL\\\\\\\/lIozRAhpRZut8BNUTQqCt1Q0mNLp+3vpHTyXeKpAwnmW3Hnp3YgQrWfIZIm243KoTzWA2BnGcXoEcrj7ptA5E4htOSgL\\\\\\\/\\\\\\\/MLQIfciP2X21brawnQm\\\\\\\/hV8HavUvqBsNLOjqlReFSjniyH\\\\\\\/ieCq2Z1wSkboIC1iRad8788bWLJf298C+mx57e5E7nXEYDsDBW3KM3FC74Zmu1f\\\\\\\/h5\\\\\\\/cg+o\\\\\\\/B1HF2UWfe2EdLveqwKay+aET4DyDMDOrfPJw+TrYI9g6fy1iyxGyzY7FgiVTJwDEXSeiRcu4dWhCGyZ0uKsjf9Xo6+8B\\\\\\\/wMuo2kEumRBvPcd5E4tU\\\\\\\/+E0JWgVy9KmTKz4XQ0uTKW+I8CnZD11wcfx+YaXNCHJeYl9eNvm5OR5QZEqMYx\\\\\\\/YPQRPfJodWFk\\\\\\\/vtsANX3datq8kNrXj+wVsuGmmg4GUPUFP1hHph0jamGhj4DaEOnBmgeHzc+nKL1tY43x241s8LP10T7DDHRHU+OsSzuZHlqGYtzmBpSsf1wl+9bkcAJ79lrnfwM3O8KXFqHRZ7s6tA\\\\\\\/voWK6gJJsbW0kCkDfobu\\\\\\\/hlXideLYrVhYtOM2htRxaq3Ww2Q0T805ynzBQeFrdIE3ZCvlW2DcBivWcQJz33T9OTHb82YFDyEy54cccCxNBDBm0npX7dFNfgR2sumGtDZvUnHBY87ZI73SicRhI1d8Lzm1bQ+WKj\\\\\\\/CRkgl24sNCuIR49JXAvB4g0x1c7pIOdkrwREWmPMhgUnMJpZTPW8mUll0q1okfy4zy\\\\\\\/2y0Y3EdmqZQArLGiVT9uvngtut9IMZeyCDufUHtLGvW2SaWKd6nX5I8HcjrbyisU0MoSoQg\\\\\\\/u3GADzpEJPkvKDdGysg5M6htWc2xa0Z79qc9btQgsoq4oP7WvmrALLDHkj6xL\\\\\\\/lIhjzYbYH9HH183K0U7n9Cn5cgGJmTOZCiz\\\\\\\/hDom8jTclrrrGZCUEzjR7es7PHqVzHkPnbu8Mz\\\\\\\/3b37riWSJnX\\\\\\\/9fQgeIHhB1dv3PGoTHqJnu58pK3xj6wvtPnrFvsijA482GfPW4Csaq9xyremdB\\\\\\\/2a78QQSbIDYqe6a4QuacbGbootKDZ3J5avE\\\\\\\/JEr4AEXsRUZ2z1ZXLz0tNzNlXHd4B5nhDkMMF+xR02NifCReyilnp+Nxr5Onx1Xdrz0U\\\\\\\/ggs5SDDb\\\\\\\/JgrCuIaM\\\\\\\/blmC3J6ENJ1J4BFvuKT4TqVXiOGDGqgqICA8nDUzfsRwV2eku1pX8tIUHd46RQYSneQMQUbNAD6I0fsMInTwTSQv1VoNFF+pK2QC6v9Bw6pVL3raQCcxe0BIeIwYS7SRjdmdYcjDZz13+9wWo7bsRWeV6n5yumRXiK5wE2lkZxQMN3uNZkSfVVAleW75Gsk9ppDI96ByoFRzsgi9syTb6ekGbLIC78fi1eOthNveK0Tu68cXa3kWP8GVY\\\\\\\/Y2WlrzzHvH7DddyM8OKxMcDLQfSsJ6ma4SxcthFHhZfeI2EdRbqD4wHwY\\\\\\\/RRmoU8RjV6ZpmfaC6s\\\\\\\/YI2376nBSBM0quhIXEWSxG7Bd81kw+Iw7xClcjSClC+zOo\\\\\\\/LGaXeNo64pd0DRW2kRqRujfBSJTsV+a0EWw\\\\\\\/18ETXAr7MKnlv5h58pk7Q+mYJfM85j62YsgpPG6MM9bf\\\\\\\/DdPQrvwXIz0KDn4Tyh5RMPZgOXJVE\\\\\\\/pVXbvktvRsayviQ6R\\\\\\\/xc+43IpgwG+9Y7NWTO3Sbm4EAAKlOozY05x+F+ScppKJ2jXZ3f1Yg0A6s69bo954umTDwVCZolwwhY9ppi+cNC3y+DFdGjAaefxLaXRExlXceKQK48epUqkbJjUvBl\\\\\\\/R28FW9thTqqlT+0bmL5YylJPYfnmlDAbNqUuOHJhEkwOjXc78LtUzS0xEIEbJRY9OiVW2ZMCD5tH7P+lPD\\\\\\\/5lRtW\\\\\\\/egdv+g+uUQA4w\\\\\\\/x9rRWvSVYqk7UZgED6kfMIH\\\\\\\/ZKHTPdnJ0AJOdeD\\\\\\\/wVQvj8k4K4+R4KU2iaf0vBUfNgHvjvErvQ4DK\\\\\\\/TAWkEZ65pSqVtdbOniXcUtmCJdZdS5zwz8EEFpbz29Y9fToQ7eBkILEaw7gUgmUkWveyUMjRnLaZnmtLBF7h+tvdklbE36Fd6WFGcuKgvMRu8qOzLLnXvtZn1+TsyipkwH6waXOPRAm3mMxMoV5Mnx219desfrprlQkw4oMmtY6iw0JK9Cmy2dcLB\\\\\\\/py6R1dpjMEHw1JdDCKbllI4Xv5Lwp9eyGrMbO0TcspjrrHtWIK7yTvFdxiNipByFJb\\\\\\\/a3BtzjEDlE2JpsUjcGh9BVevY0s9Oth1g3k4PyYCQLsq8WeJ2z6si4d9L7eBrWoB5xWHPpRST0nbtKKmC7CeUpO+CF7oIyU\\\\\\\/saSS1oMvt6afPY49giXQKuG42iq65U6gIeKvC33DzK+eDrqIpcDnk7hRBs9sVJo4fXhRBVdLZNrJSYl7hWh+JgsphPicJ2na4Ia4usQwidWBFnKuHHRfwLGqDkHYX4hSqoHaVMEdGIoHYvvyQYuKQ4iTxFKmeqWOn\\\\\\\/HZyY9QSEBhPfomznVaso412fj5wd4EhZMmlwCrKNP05rbG4M0VzTGL7g+UFPd5ItEyuTi1L7DAJzd4CmOlmKU1IQ3irQv2UtqxXFvw7VgGKBeEgxQCjjd0tsbJ2tSkFBIDTX114VXe5ByIgB0xSzvbBqupGOCuv8XSLSVtpxU4M\\\\\\\/STYarv2PTgft6CTh3i56dsxKJIvOgahvRT+RI8nOkZa9oH8mUroETBbHah+pffgNhCMcB0ViWYR1o7syBmvE5Yal1IaH1pU0zFT33ODRogjBfAx\\\\\\\/gTdkurqi+EtcA3b3JWitMeUZAxFZvow2mtbuqRwUVmg4D7EzBi1c2de0BhflOAPaFvVtwE1UfNM4OCifhBA+suXw0FQWnYda3nNcP7JTbogbWytqsGdCiukfruz4eBPctX3SWNt0UV4JpKYbIUFF7BgpdhZThj5wBeDXDR2nHtMTnn7mAXLwO3E1FmOyv7HWZxKaYX\\\\\\\/2sjX2Aotf5Xo36+duXkRUTGMQrEEkOc5GqZz0qUTIFN0FM1zvanq1O5D5g6u8OuFbCBW+Olm6+8ketMkOv2xpkO5v9v0bM3uoRV+lZlfYQ+vrzsXIb\\\\\\\/XcjTP9Vevjd6R2R8pT0mbLJA59RAc9oYEYBv4XpeI0xcpBMjOPJcpjGTuZSYvM4t+YNqOI6RbomK\\\\\\\/vGSQz0p365EPA3OCZjghP33kQBBsim\\\\\\\/1ls48Vu2BqoVwj4Qrzzii1oGriBCW8l\\\\\\\/yqqobN2Rku4rU8t71ORAnLSVwz7bmmxecz2JA49feirbm8ob9C433YcC+G\\\\\\\/vrTcVkLt3Sf8+6vQ6NKtUQm78VHlisXNspZZQw7xfPUmNMfIKG8FLAMcbUf7D2QWakgXu5GMSriVwmx2XcaaWlYBV4tTy8fuC8PvpKwWwxiNaVeETrkkMkf4qG6c\\\\\\\/hW\\\\\\\/IgsX8lJ2u89MEjDeIsDCseQjUQocUd2z\\\\\\\/GpnG6L6zJbMnowDzQstXGA+l0uIcwW+c4ECfZBuGxem909MCSfN0x1zTSPTnNwcJxUaYth4rzYhAEFAOdC\\\\\\\/UfQ\\\\\\\/bBaCFWYiRl0XXgaWGb6DWXu61lzZEnCEVuv\\\\\\\/cQqZDyYeduNUs0Y3SdbIr2qiREOX8m2OlRWcqAHwsC3Lv79hEv5ZEw0rO0+dbXYd1VrKEAMege4WO0CcvmVSWHhBZXTIa9P+Pfu3ouHl0c1SjziDzFX5wIJ1CDH4gmrVHmZg+CO07mJQBhiS4gnF0gMciLAvNw8Wi9yoziGcqgFonPi5rZizQwoGePnoOCzdxhVeuskkyuAQpKHtF77p454zEFh3ZKZKtuaUWQ5qgtAkQEBMEa6Q3g1Fn6hxXneSEE7HR25X8C\\\\\\\/HUYXcRbpCIEEqQv5N1maq9RRuu8FjqreDoo0qLzcc77COSSNxQsTRB352lJppkpKxY79XWzH846hcHHkR8iSi\\\\\\\/gWhT+xtiJrHOJjxBkBG6fVyVRCqfxv4OTWp6xLHS6nA8pxgoEP9WHgCZc\\\\\\\/JWBAMly5MPO0S2p3FvU7yuOqGx4ca+0pWCsdprv7XYY2L7ZnUpINL4MUVLUDio5ftXCcLkDL4rEK4cM1q25TfDiM2Lbrf58WnHW80nn2tww6qxh+qWJsMA4\\\\\\\/FUMhMVAFA8N1zF\\\\\\\/4Ft9iYZbewnxpr+B3BQG6E7Z3nqG5a3FHGbarVO3nUmDzZPsiqdvGx5eIFSlByxjtQM6GJQvaTI6Ca0KKLr4U9Ks+Mx\\\\\\\/DxMtRzqcYzoFu2nBzLZi+NnCp5ZExHk5WGTeDNEgn\\\\\\\/hz1fuSLRwA0IAZKaXuouvSLVY81coiFQRdHZgWpQiRrfP7jea+6QLCbnfGT701HV6o+kujmYH2a9HIGxMUwn1zOyF+sxhuZnRDOCyHK911p6gXiDl9J8MpVDP+RVLUnXVBYHLKAaGp8P2cKKEBONYwh10A4vaHLYD2QnTi3H8G4kLsvFuR7ZrpfaleqDZkjoOpNDAZlTx6an\\\\\\\/NVJh4oPYlKOfiAJUpOqgoDTAO+\\\\\\\/dxEW4DaztKOKkc20ATCLUyiaj825OUTk4Af\\\\\\\/isr+4lwTWMFamZmQepA+RDFrdFANRRVwVF0wKtZQ1hAkEUm4ei4N46ll3oXTqRKtKM7a3KbwqqLs9Leb962kDnIvbh2ftJ7WROAfLnQ6atxCaMTAzzQxpA\\\\\\\/YgTLEtmJtqCMqPI0h5zLwuHWnKFfiF2RuOUge0uH5+EOPRAXNXNIuo0bAy3JPqvJ0nnHfCjwKnRCTs4vXFTAgx2l6S3G2n63ZOzuHxMwG3xbdSVVtikcPT0EeAyaGbS+X5uWiORCblgQHyQnfOg4yhuuU4bfdIwB\\\\\\\/2K1YKFfMBw3yg\\\\\\\/7F5Vv05D99nMWyKfMeIG7nx2jRE69hCTR146qBARrzlB1D9l2xLvXScFCimcK3rlHcbEkWqC1xStIFiv71wmKdtn4r60d1RWTrJZ2pAoAuJNsYP\\\\\\\/\\\\\\\/Jygih8OhhO4zCWE\\\\\\\/3AAYrRbGFaHa4gwupQGJYFB\\\\\\\/qp6uzhcyqv6PAFQZHKWTUoLBRWlTNTU3cBywLyce3nZohXQTtKOkAVwaPkNEoIrjCTVBS64TSEyfX4U3mTQM3XdPEOlYtYSbFkIhnj9iDA2xYML59YfjtLX4RId7RRA9S94Rp7d+MlPBzx0paIbq2nGiNfhEqU+5oAbu7rGhvcS\\\\\\\/aD2bi6ouQv31kYoTn7jpMh7m++6WfrV75KoX9S5ZqeafbqjxAyT6nPlSnW\\\\\\\/KMvhLQvra52otwuTFcmX8qCNa4VR3yPve6NlOvgW64PLeMQmAMl5WXLPfzLgvdUeGdp7h5Yda\\\\\\\/qAhyuGENL3itTMNKHVKl2wX8t5OgEZ7gBbtz8G6+X4u+dmEo7UE\\\\\\\/MxSgktwYYWCBrJqljRsXpRF8ELUcow\\\\\\\/jf\\\\\\\/EUkvov8KDdEoxuZr7Rc0NaH4zcPJfj7JcNjg\\\\\\\/c1vkes3cM5wY+PppTL31m5gMhQKU8qHLw3\\\\\\\/ZH4EAnYmkYqJ9MUiZtr5KBDWYedvK\\\\\\\/ZouovyKD+XBUJ0rtGwbcxqbSz0X0BZCdHsfxVviOQM883PIx6tda1nktILxrotWtqMsKFkeLe8eNFH9Z5Tl7QyEDrzlHOpkV30\\\\\\\/DQUfzNOv+FkXpSIs7FZrKQbQvzMKrh7aQUOh5XNijuSnSxwkIjU9cNVzqTetGmfGlzumY4tCcFden8ewdP8KMA0HdAkV7D0nziqlYhfW0F4GMLBFcBR5E3qhwkd3hKqB03Ny7LbseIUSQsMP2WxN\\\\\\\/issvpHxqKMQANX9ZBADMjnqetUDPcyPio0mXsfjJJhUpcAj1zbKaitCE4V+vYxadIwEwMOIYH8zOtaaXZXnfP9f5OvW48sJxSfHWM06klLaj7cHGeasEeSiNBLHKjveMzosRloCzl7+bFUHlBoDjQdkD9v8il0geHaj3DiIhop44TCuhba9\\\\\\\/wZD3iXkV0xdKq0v4FWdnf90w\\\\\\\/fMGaH4nLbAoZLPdzabZKEet\\\\\\\/OT7X9+yrVlWunViLnISbVfYNv2S2Sj8NHnaY\\\\\\\/Yb21kktU3I3M4a+sSWNYTfJMokJluCANxaiHVcm3TaxFpPBedm86RAnMzrehIzPapx+Hx09ESZDNAy+uttO8pufUSpDlon0Q6BLOIyP7YOl71yEZAgsl2UEFjlZYxlFlk6F2rXzAhqYsmzWxxB3XaSBwHyvauvF5xYygjClb5aY+bNj\\\\\\\/j3QwSoqtDnlyiebSXTqcfcaId1ZFJVAa0zHMOjhexn4B6yH1Ty4WnCEETfovCDPObE5mPHLVtiRSQ+g3QtdTCc1fznu89D08k1bIV69NiGNZuko0FDLGgxoXzU6J5xKdI1nhjCY+2QXseQlV7OCTgfa5im414GugOd8KWF\\\\\\\/ewrWFGygMepbL0e\\\\\\\/\\\\\\\/LZSgMIreWFMoK\\\\\\\/hDA+AZGknqOhgMurYurnnMvGM11RF47PNeG0vLEx1qLuJiNZBsHsgz8+qqv0JZPgRzdUNQALlgvzOI3E2TSUOkMk79+VSZOd5sBMVE9\\\\\\\/pDy8gZromQnya0DzCxxF1zx53h6Ek84Fuys1MQufTr60K6j4cEbAPg\\\\\\\/Cmcwv3WD1WFjZxlEVbucNqAUykIfbGXHTHjbptuzBJohFhJLHW6vbwp2DJu\\\\\\\/IQWtp7TKG239rVmF\\\\\\\/GB6uT+VGVCc22mzJrUzVDMerH+mrRraJ3AvMZxxcknYngAlacDD1TTv9taWS+kfgDhwz\\\\\\\/wh8WhKUFISaW0yteeMpdEoCcWMRD77noE+7T7kC0HAe8w++AMmfExVeV1+D\\\\\\\/SZAIOM8rG7cw3RtsyyCPQ9JfOYLYstsDiscVT+haqm\\\\\\\/jtfrvE266tjHZhURG\\\\\\\/c\\\\\\\/DWYYgWGfTVFv2Ji+dUVEFnGMgjBjiN1+5tjOOLx\\\\\\\/bXEMEfLU0Oqv4QpkIYcUVbUYZbsWBEgNxDP1ibv8zjAP77QACw3lQ2\\\\\\\/GDt21se+QQ9k\\\\\\\/M8GPr6WrJsr0fms4TwiU+T9Eycgj63x4xCqqseoSWOi\\\\\\\/vAh3jdulnPDjEszI5bLKqaAA0uRITQxAtGJSdz4wasaNcL4EzTV1\\\\\\\/GPTAu7\\\\\\\/qqPb+9FExt3PETCkTdOh+efNKPEYagu9Xsqy18uG2dA924Rx8rSsl5NH2IqVpahuBQKGyjqHpG9rasrb97X4TJ35GLES9SseTULAk7AnOb7LimDUG9JwuSGoG5LuHTz1wQ9sbIxtUN2a6loYB\\\\\\\/L4WKyyuA1L\\\\\\\/6HTivs5lr\\\\\\\/Vocr9q2f10BEcG\\\\\\\/TYQbF75UstEWvycmcHbRvLs+jRdbHDhyFCs1BwoRwOoSj0ccQoW6sbTCsEx+QmslReBizH2iUZJu14C2faLPNkPYUIvIYZQ\\\\\\\/bhGLC7WhGFppHj+bf3KLRkaqIf7a1flge0qJd7Fq\\\\\\\/rqfnxthUDkHU8sO2g4s2+GTKYKTCE9uIWJv\\\\\\\/HPQ5MLcPPkBIUFlr6EJnb9zfHbi8l1mCGGJw7qbFJaXdabsbXK2VuftLaKXDJKvHVYDWuMSp\\\\\\\/psZOvvWGSthy1nSrRNbqPfMxDjN5fQIBTPVdnIOCw9e7krKMr8R1gIMXUZnw4UYGfqES31dWa1Wc0C1pOB86V0uwZYl5aw5Js5+wjrzDCKoB8D4PqxokR1C3Ri+1gEqN8YiQik351zIuctkHcc6fVbh1t6uLmIi4TT0fzdxwiDoQYxdZ4RsyLQHm4qn88\\\\\\\/O22HbyygvY7bv6lx2VfwNtjIhhGp6iDqatpMzuBdD4n+etErHa7O4sm+nTkNWJZ8w5YRgMk9jXz9D7LLVFm1YdZk6amNKoHcqds50mGFp6za1COhJfsp9tW8qLMqndhHJcFPL8YlO4JjAwbnlECU2IshT3LNwDHB1bcS1dkcWLa4bqyBgJ7FV60ScKtcRObdMq7O0txDinTSVhrOa8DXIOSGGEyt7LNusoUd6Ndx076if6c41F+5\\\\\\\/2O7w3pMmnOwBB6QOX0YfuMti8PZO1xrjgUfTRYjJW63KSyOa+x\\\\\\\/XMC658YNvli9h2VLueIqRidxwyDJTzyFGbRkx\\\\\\\/KpWk3+qiKejbGJOwsa1RMOirKwzAGe5ZmOj5lIS9uOtvBmXO5QjAwHbR83zLXl9WfslHnppd7ah1iZ3j0EjtWfa15D1PVHOOz7mfUjWjO4J7nEezYEAWKxWCA06hxEK5YEWPRncrvPVrXCf+2SIRFDZqR+VZStRr9v6T7Pouh1eOObnfoVfgZ0uh5phXQzbsE0tTv0wBqd+XpfmHzjaAgWtIqiyOCtH7bRg5WwKe6+i2lISz+OLnii7Vh\\\\\\\/F9xjRAD9EwS1IxBN+eeJYHV8t3PQqdoyWVfAt+v0KRAHdoonzfTTGRwP9YpzGEJ3CW48T5CHF2eWVPUNuzck0MO2IcJhDPqIF7s4wVup9\\\\\\\/6Z+SG0w+pXh0PxqZEjy9WjHqeG5jP8R0tss\\\\\\\/P0jxroLtNrieZRTvp41miLv16COk7m0vgH7oI07YPi6REKWlowX5cIIiFDfbt0W4PMP7\\\\\\\/03oMwC2A5y1rRnsynICusK3rVPtZDg2UaGecqZELF5hFttAfvzJVq8vNJYc8+nw92ILWDqVh7HbpQNT6BGY5My7vcRYlGW6tsWtpGdQ+MfR4izCYQZ9iMQ9ghAa4em6NkZdWB3EMhzNYKaUZ8oBo9IN9\\\\\\\/SZoVJFy1fw2V2AiB7Tv1Riw3qcPZfiqtyPKL3OY83dXRM+0Jhq+Y+BZGehP5c1IlGlaXXoKINZbRwajTUOznyOu2FYV05PaXDRj6t7ZGB3iFe+KLB6wARMmHTCTMxcGFZdYsOO3D81SzUAb7HnnTESqiGtqFkr3QMv3gzNZMiqP\\\\\\\/qe1AmFsoRfSwE2yZcWIVuTn0St7UGHwcPLRquGkKC016mD47j4YAs+In\\\\\\\/K5jbFTtbKMAZGGfRgiD5JWn9ypXfrm5+idmlKCoA9uO1sFdOYJqcvxE\\\\\\\/nS\\\\\\\/PGCkXdY3m3EoFCkjgdtXNScAFMvZjzwWPu065vO8t1qUp0xn8pkZAO7pZEWE44ZMfPRRiShQD6b1ervegMR09oTAEADnIvOjrtP9qI1UJVobVp8hNvyAkEfIDLdcnv7okSLGBLNdi1QB+IUtw0lgO7mvkIwvcoy8rKzQYPeZEcuVos1\\\\\\\/rsDUVSVDlAEfiI9ihv9SiWYIaqHJmBcBICEAqZPUreBhd+Bodka12EBtWJ5zloXgVvc9AeDjmbftd0vvyKmy2qhPXiUHRvFb4vxAOTJnz4mfNOY\\\\\\\/8lWiQ+ahlvUoe8JYQuRM4HLaZq5PyHOeV4XSy960AzW1kd8dXMDo9LT7MGtpfULZWdKXRtHAN54Q+9UEg4t\\\\\\\/hTg3qLaJIE6w0KQUp1iHO\\\\\\\/aOkHWiTfoI30T0w\\\\\\\/6LTzPegU7xBAogDVOJSyo24PvDzGRPN35HMUDOXiLe1bBDMoXQO+t4eD5NtGb+yxnObW17NSN3tU4B8Jrejyxn\\\\\\\/x+l2MIqrBQH1l61gmZtoND\\\\\\\/0wi6RIo2Hl+Gpl\\\\\\\/M9ef++W03sOLR35xpSaGpOxthvQAlx9qFg\\\\\\\/nlQCGfwjRUmecBKwgnoBeviHfHzEcKKDGRoIuuE+dQAk+gNrJAOyeHOAXwTeVbUqUY06BAEkW6o0fQhCIawQbdPSMFeu13uuie6HHHzf6pZ9cDk3g4g2eJwEUWMCZl1GStEseJAG6yHeRQl+i+qOUtp\\\\\\\/lp1jr2lpX1rkwN+BxhRCINROPyWeVF9ue0NV2v4bP61Uxpfk9zgHxPwdaMeYo8YK\\\\\\\/nQsDJeIZlT5wQ9bbAp7blrnyjKcJtEoPYW6k4ZMDK1T6gcU\\\\\\\/CnCc4AnuuO1LtWUsxqr1eQfnrxahw5PgUvQatyly\\\\\\\/CRazzT2FAEauR0+BLdFmK355dxJsyD4RLFm\\\\\\\/9VR8YuiZtts0a+zQG0tyNH6PBpl\\\\\\\/OZYrK3xJGJolZ7WUHxuPfVpsMPQAvSiDRo+CZPlclYMSuZqcju6HW0zfhCbNTEFO1GFwMMqZAbmSdO5G8HSyIVQf\\\\\\\/8wxqRhV+sZQ7XTLEsLDsqqmWgDNQ9Yil+AQer3YKwM+gLtZb7faB4MWewm+DirPQYkCS9vJYJsRbo8yLlbuMu5YJVBX+pVEBTdyJfYdtZp5qMKwAyLL6DA+KHSSZjay9CJYAVJvlO1Vil9JPLDbSMtx+TbxYoSiVmaycv3cLSyx3EwYOaiwsNzXQTofwCBUpJM9oEqAWfumFBP\\\\\\\/I5PSs+AUUPDH4IDAlzCXFGxH\\\\\\\/cRJXq\\\\\\\/LQwd6Flnpa04828MQ4gWHyUFWFzgf67D+SDCxQtDI41w809VU0IpOuTIBJwVcLgMuvnXcfwQqTciPmCAHQtOyqnvjLpESoOM+79VYY8AcTOSnbqpPS955Ytq5kO8kHcNpqqe5f659sL6KHZsQiOyp+FwaTuHN6fV\\\\\\\/2SCvJJeoN6fw3WOfKNBFRYz5f\\\\\\\/T64hoae7+6xHpWPpq\\\\\\\/3g5KZBvCCC9MyEHH5unOZNSTP2oP+x85BF9rLw3zXefhbVMfFqf59zjw6aZIj9InW0Bx3HdTt6+OX9T+CH0V3530mf8EUlo8zukz754kSHJhgTlYq7GO\\\\\\\/fZgRE04K5tAKUskGtYHQQPsaLCr6Ct8GhO26DOu+pCnuy62HJCr\\\\\\\/gZlMjEx2d0vQsCe3U0r0Y9pC4GU42OyfhhNrOpUmclInXsLKhCZhhng3XaPSpDgodLFdd8Hu0IzgH9qUMVA1yT3tLokTKTVde7hVW0Vco\\\\\\\/PPyv3YOUgivCrOo3kq29hCY3MS\\\\\\\/wVOXo16a\\\\\\\/8SylCQPvR8n9xDhUaGClF8x+vXADd3LxRAyiCxWEPWJhAKQ+B2kdugEmhoC3RjxsYs9vG9ZpZct84OPYnjgiG3\\\\\\\/pSyMglxxA5P7RfJDbuxcQmX3aSUUA9QMQR7ye\\\\\\\/Bzl2yjGwJmTTQ80OMurRFc04Vup\\\\\\\/9MsvqZ6PtyWsx7JQkJXNIBXjRY89z5KGfpJjknwBJRF8jh47p2GrQy4o4Jp5vYrlwMHE\\\\\\\/EQxluNt3ZPsErw7MJyAYv+W9uFUccgA==\\\u0022,\\\u0022iv\\\u0022:\\\u00225yBOmE6xWXZlj1t+AyKhyw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002230409855c26350e1e57054e05a02bd2c0ef400d3e3ea30be593cb9201cac6b5e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022KkfpbhTr2VMb2ARNdr2YMZjNfvZmM2EIuMohl\\\\\\\/j74UxAq4i3FMlkTIis+Qp\\\\\\\/152xYLNNbvt79nI5lZQ2D\\\\\\\/4aVzCPqEl5GNOWM4jUhRffCVZRes4x7w6gOH49YmsK\\\\\\\/BRC5T5eU9+kYp57kr759toqveDjMYyrCbYzOcj1MU5MRf9dAp9Alt1tnASO9ji\\\\\\\/aEs1qiVb4KoBl7rg4luo8gF\\\\\\\/cPuMSmPc1fXmDNrCba3oYESpTiQ6X+akFJQ1k3lSnbb0YyGAlV7yf\\\\\\\/BuID3dzhFwQHdWre5mlFrpOzWYGbxNDQ1sf6kE35OgHzkvZjLQSLdBSbnjwoQjNN\\\\\\\/ft+XqXN2MWtwEHNfwNjLSAsSKhKO\\\\\\\/sxOgtFSa7V8NT89XjhAXjR6CgBieBT8++iyxjH3fNB+Y\\\\\\\/bVqY+i6d56TAERZEoPQp\\\\\\\/SkB9XZ68M3NAOLp95Pu+Fhmii9ttFbPo711C77I4Jn2criVzejKKansz9KdUHuC35BESsxUvgdEQaxDNQXE2CesvIZD2rqWJbnFSH6SLh20+sl7gpeypM1xax3r48wNrQnh5O\\\\\\\/ypvmwzYDf6gXtAj3unFFvfEFbIVZ5kaqLjWn95Yk7UmkCEY8xwuzmYd+MyB0kVJvyqSaeoHP3ZDUTpBX+98ZRTMEh4y09Uw3ugz7sy\\\\\\\/dzZaEpYmTzcfogBDLsF\\\\\\\/+\\\\\\\/\\\\\\\/4Q6yE+V8lFlEmD1WsnOu1\\\\\\\/P7jq1odZAvnB0Efb9Q6MkIJ539aq97cZYuMVPyMQt+75V3KtEWf2b9RU8ogMo9R+ye59akakhSCyPM9xRraDq+DabA5hWmeYKLG0lZe4tfof4vzRSgr+joWIPRIJH+OQoAbP4e9gC4mpU0Uw4mNi8YHaHV\\\\\\\/IRXd1yYtTplHFEFPaMJUTQyn4tjbSdCblV+hFbTAzi9KW99+TUBKlc+z\\\\\\\/G1IHA+51y3O\\\\\\\/lB4\\\\\\\/ylrWgujzSN4Sk2YKuZidYBPOmB5sEOuC9ebvZu7BfgLFDGIU7LgEtT5eIUtvTZzRi\\\\\\\/3\\\\\\\/rg4X3DtVFtUXLygiBSPjp41oAMwta2pyQRdPA1y5g\\\\\\\/CfvfvjxC0Dh69pbYcBIJT7aObGdO2JwfmZ3gvyaPTGaHqAomcU1c3jzhF9Q1jUu17H12RY5RDuGjIpMlHCUOaWgTw4g2FfjPJ9lzKFHcuzacTbs6jVVLxyu9Z0otqaeV028FSXIgsOUHWnfT7oK1\\\\\\\/mO0SYdp3v6KxAlxnL3E3g7+m7khu7qbGO6j8oAz6IzAChGaEgXs66BgiCoc5TSgKc054J0XIPMrDAU4uCT7oLkyi3XCl1MrkfwFs4E47oXGipEXIM+Uw9Cx65o3udcXJke1i\\\\\\\/u93Gt+QAKW8aWAiEDzkX\\\\\\\/O8B7m3Xx\\\\\\\/ry4DyPQNRssvTCG2mFSKdbmlLRhPixgS5QbWooXHF2PyE43KZwJdVkFpTLsAqoNYlMPjq6jj0F4cqzZ7zRWGFhTAZVW1Ix2oYGJ0srbeIEncv+tM6hbIauOZXA2eopvs7AH7Qz1AlFyHw6Nnu4ig1kxfGKRHjEqi+TENbW6LysUxcfYo7IQdJvQejHZVhbokU6i6Sm27hwogPSriYizasERqyzPLSPfqTKWdggmc\\\\\\\/96kiq0LvuMIXI\\\\\\\/\\\\\\\/zG7Xu6O9A+QkvBAW61ojU1OJbNBLeBPwgV8H0TRCWrpNmLR5H+qQrfjCnuPZdiKg7j+5I4BmMO8dVAnypfqaVmGp+CKxWCWVkQB8P7G2qT49f83thOnGZXaCTFs7ZAfCC6jDjs2SVxZPKN5w1rKJ4jLUJ2tqmyU4Ve32faROZG\\\\\\\/rOx\\\\\\\/LXo25uhfmkhiX6FlfR+VrqxSlhiRyBRtkalIfZz5tTTAzaZyeZhbQP4O7J8mZGLTeCUxcXgRBVPmqqD00Yy+4nbC9R9jqimeO7BSpR60jFYBx\\\\\\\/38ZkoLi0OD\\\\\\\/n4u4i0IEwKOz3Rd53Sy2W4VocDBpbR9i8DvkIs\\\\\\\/IHGNZ3FA7KsRvUr6DXH6Z8P+5+cHhEsNG\\\\\\\/vr2X8P1eDgKZhdP96smFE0CJlLaeuYQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Zqn1AFTOopkbzym3q1mrUA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002216d4307a91ca77c3bc41d0fafcb94a24b1b53a5fa06081a5d198305887e61363\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459304 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/8/853" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-v3lKbVLFMIT3nXfab7OTRX1s41ED4I47Y1CngfB4rCU" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/8/853" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-35QIBF3j7Acm9mcxjY-g53zsWggXcIxrMnvV2SIRbGs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/8/843" title="Expression Profiling of Attenuated Mitochondrial Function Identifies Retrograde Signals in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/8/865" title="Proper Cyclin B3 Dosage Is Important for Precision of Metaphase-to-Anaphase Onset Timing in Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459304" id="node459304" data-pisa="ggg;2/8/853" data-pisa-master="ggg;2/8/853" data-apath="/ggg/2/8/853.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Association Mapping in Tomato (<em>Solanum lycopersicum</em>) Is Possible Using Genome Admixture of <em>Solanum lycopersicum</em> var. <em>cerasiforme</em></h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nicolas Ranc</span>, <span class="highwire-citation-author" data-delta="1">Stephane Muños</span>, <span class="highwire-citation-author" data-delta="2">Jiaxin Xu</span>, <span class="highwire-citation-author" data-delta="3">Marie-Christine Le Paslier</span>, <span class="highwire-citation-author" data-delta="4">Aurélie Chauveau</span>, <span class="highwire-citation-author" data-delta="5">Rémi Bounon</span>, <span class="highwire-citation-author" data-delta="6">Sophie Rolland</span>, <span class="highwire-citation-author" data-delta="7">Jean-Paul Bouchet</span>, <span class="highwire-citation-author" data-delta="8">Dominique Brunel</span> and <span class="highwire-citation-author" data-delta="9">Mathilde Causse</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">853-864;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002667 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459304" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicolas Ranc</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BRanc" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ranc%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicolas%2BRanc" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Stephane Muños</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephane%2BMu%C3%B1os" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mu%C3%B1os%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephane%2BMu%25C3%25B1os" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jiaxin Xu</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France</span><span class="nlm-aff">Northwest A&amp;F University, College of Horticulture, Yang Ling, Shaanxin, 712100, People's Republic of China, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiaxin%2BXu" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiaxin%2BXu" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marie-Christine Le Paslier</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1279, Unité Etude du Polymorphisme des Génomes Végétaux, CEA-Institut de Génomique-CNG, Evry, 91057, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-Christine%2BLe%2BPaslier" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Le%20Paslier%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie-Christine%2BLe%2BPaslier" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Aurélie Chauveau</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1279, Unité Etude du Polymorphisme des Génomes Végétaux, CEA-Institut de Génomique-CNG, Evry, 91057, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aur%C3%A9lie%2BChauveau" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chauveau%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAur%25C3%25A9lie%2BChauveau" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Rémi Bounon</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1279, Unité Etude du Polymorphisme des Génomes Végétaux, CEA-Institut de Génomique-CNG, Evry, 91057, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R%C3%A9mi%2BBounon" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bounon%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR%25C3%25A9mi%2BBounon" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Sophie Rolland</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sophie%2BRolland" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rolland%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASophie%2BRolland" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jean-Paul Bouchet</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Paul%2BBouchet" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bouchet%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Paul%2BBouchet" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Dominique Brunel</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1279, Unité Etude du Polymorphisme des Génomes Végétaux, CEA-Institut de Génomique-CNG, Evry, 91057, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominique%2BBrunel" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brunel%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADominique%2BBrunel" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Mathilde Causse</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mathilde%2BCausse" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Causse%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMathilde%2BCausse" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Mathilde.Causse@avignon.inra.fr" class="" data-icon-position="" data-hide-link-title="0">Mathilde.Causse@avignon.inra.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/8/853" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459304" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459304/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/8/853.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459304" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459304/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/8/853.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459304" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459304/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/8/853.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459304" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459304/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/8/853.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459304" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459304/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-6">Genome-wide association mapping is an efficient way to identify quantitative trait loci controlling the variation of phenotypes, but the approach suffers severe limitations when one is studying inbred crops like cultivated tomato (<em>Solanum lycopersicum</em>). Such crops exhibit low rates of molecular polymorphism and high linkage disequilibrium, which reduces mapping resolution. The cherry type tomato (<em>S. lycopersicum var. cerasiforme</em>) genome has been described as an admixture between the cultivated tomato and its wild ancestor, <em>S. pimpinellifolium</em>. We have thus taken advantage of the properties of this admixture to improve the resolution of association mapping in tomato. As a proof of concept, we sequenced 81 DNA fragments distributed on chromosome 2 at different distances in a core collection of 90 tomato accessions, including mostly cherry type tomato accessions. The 81 Sequence Tag Sites revealed 352 SNPs and indels. Molecular diversity was greatest for <em>S. pimpinellifolium</em> accessions, intermediate for <em>S. l. cerasiforme</em> accessions, and lowest for the cultivated group. We assessed the structure of molecular polymorphism and the extent of linkage disequilibrium over genetic and physical distances. Linkage disequilibrium decreased under <em>r</em><sup>2</sup> = 0.3 within 1 cM, and minimal estimated value (r<sup>2</sup> = 0.13) was reached within 20 kb over the physical regions studied. Associations between polymorphisms and fruit weight, locule number, and soluble solid content were detected. Several candidate genes and quantitative trait loci previously identified were validated and new associations detected. This study shows the advantages of using a collection of <em>S. l. cerasiforme</em> accessions to overcome the low resolution of association mapping in tomato.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/tomato-solanum-lycopersicum" class="hw-term hw-article-keyword hw-article-keyword-tomato-solanum-lycopersicum" rel="nofollow">tomato (<em xmlns:default="http://www.w3.org/1999/xhtml">Solanum lycopersicum</em>)</a></li><li class="kwd"><a href="/keyword/admixture" class="hw-term hw-article-keyword hw-article-keyword-admixture" rel="nofollow">admixture</a></li><li class="kwd"><a href="/keyword/association-mapping" class="hw-term hw-article-keyword hw-article-keyword-association-mapping" rel="nofollow">association mapping</a></li><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">linkage disequilibrium</a></li></ul><p id="p-7">Linkage mapping has proved its usefulness in detecting important qualitative and quantitative loci in crops (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Doebley <em>et al.</em> 1997</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Frary <em>et al.</em> 2000</a>). Linkage mapping strategies are limited in detecting loci underlying quantitative traits (QTL) because, commonly, only two extreme parents are used for generating the segregating population, and only a few recombination events are studied (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Flint-Garcia <em>et al.</em> 2005</a>). Furthermore, the discovery of new genes underlying the variation of phenotypic traits is limited to those having a large effect on phenotypic variation (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Buckler <em>et al.</em> 2002</a>). Genetic resources consist of a large number of accessions with different histories, mutations, and recombination events and may represent a large reservoir of phenotypic and molecular diversity. The association mapping strategy has been proposed to identify polymorphisms involved in phenotypic variations and may prove useful in identifying interesting alleles for breeding purpose.</p><p id="p-8">Recently, the value of association mapping in genetic studies has been described (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gupta <em>et al.</em> 2005</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Zhu <em>et al.</em> 2008</a>). New statistical methods have been developed to analyze structured samples (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Pritchard <em>et al.</em> 2000b</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Price <em>et al.</em> 2006</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Yu <em>et al.</em> 2006</a>), and these methods have been efficiently applied to plants (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Thornsberry <em>et al.</em> 2001</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Flint-Garcia <em>et al.</em> 2005</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Zhao <em>et al.</em> 2007</a>). One of the most important parameters in association mapping is the intensity of linkage disequilibrium (LD) over the genome. LD is defined as nonrandom association of alleles, and its intensity determines the resolution of association mapping (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Rafalski 2002</a>). When LD extends within several hundreds of base-pairs (bp), a large number of markers is necessary to cover the whole genome, and alleles at selected candidate genes should be tested for association. If it extends over greater distances, the whole genome may be scanned with a lower density of markers to identify polymorphisms associated with phenotypic variation. The extent of LD over the genome is expected to vary according to the species, genome region, and population under study (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Nordborg and Tavare 2002</a>). LD is expected to be stronger in inbred than outbred species because recombination is less effective in selfing species, where individuals are more likely to be homozygous at a given locus, than in outcrossing species (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Flint-Garcia <em>et al.</em> 2003</a>). Moreover, reduction in population size (bottleneck) increases the drift effect and, consequently, LD within and between chromosomes. Thus, inbred crops are theoretically less suitable for high-resolution association mapping because of their low level of molecular diversity and high overall genomic LD.</p><p id="p-9">The cultivated tomato (<em>Solanum lycopersicum</em> var. <em>esculentum</em>, formerly <em>Lycopersicon esculentum</em>) is a diploid plant that is predominantly selfing and highly inbred. The tomato was domesticated from its wild relative, <em>S. pimpinellifolium</em>, with the first domesticated form presumably represented by <em>S. lycopersicum</em> var. <em>cerasiforme</em> (<em>i.e.</em>, the cherry tomato). The modern cultivated tomato accessions exhibit a low level of genetic diversity compared with their wild relatives as the result of several bottlenecks that occurred during domestication, migration, and selection; this low level of genetic diversity is exacerbated by the autogamous nature of this species (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Yang <em>et al.</em> 2004</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Van Deynze <em>et al.</em> 2007</a>). As expected, LD extends through long genetic distances in the cultivated accessions (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">van Berloo <em>et al.</em> 2008</a>). Part of the <em>S. lycopersicum</em> var. <em>cerasiforme</em> (<em>S. l. cerasiforme</em>) accessions display a genetic admixture pattern between cultivated and wild tomato accessions (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Ranc <em>et al.</em> 2008</a>). Such an admixture population could be compared with advanced intercrossed lines (<em>i.e.</em>, populations derived from two inbred strains that were randomly intercrossed for several generations). As a consequence, cherry-type tomatoes have a greater level of genetic diversity than <em>S. l. esculentum</em> and a greater phenotypic diversity than <em>S. pimpinellifolium</em>, which offers interesting properties for association mapping.</p><p id="p-10">Association mapping has rarely been used to identify the molecular bases of QTL in the tomato, with the exception of analysis of two regions encompassing map-based cloned genes. Recently, association mapping was shown to be relevant in identifying quantitative trait nucleotides (QTN) responsible for locule number (LCN) differences between <em>S. l. cerasiforme</em> and <em>S. l. esculentum</em> (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Muños <em>et al.</em> 2011</a>). A sequence of 1800 bp containing the QTL <em>lcn2.1</em> was identified by map-based cloning. LD mapping detected two SNPs within this sequence that show highly significant associations with phenotypic variation. Previously, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Nesbitt and Tanksley (2002)</a> failed to find any association between fruit size and genomic sequence of the <em>fw2.2</em> region, which carries a QTL for fruit size; however, they studied only 39 cherry tomato accessions.</p><p id="p-11">The objectives of the present study was to define the optimal conditions for whole-genome association in the tomato by using cherry tomato accessions and to assess the marker density needed to perform association mapping in this crop. This pilot study focused on chromosome 2 because several clusters of QTL for fruit morphology and quality traits have been mapped on this chromosome (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Causse <em>et al.</em> 2002</a>). Four genes underlying these QTL have been cloned: <em>fw2.2</em>, which is responsible for fruit weight (FW) variation (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Frary <em>et al.</em> 2000</a>); <em>Ovate</em>, which causes pear-shaped tomato fruit (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Liu <em>et al.</em> 2002</a>); <em>Cnr</em>, which causes nonripening fruit (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Manning <em>et al.</em> 2006</a>); and <em>lcn2.1</em>, responsible for LCN (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Muños <em>et al.</em> 2011</a>).</p><p id="p-12">We genotyped a core collection of 90 accessions mainly composed of <em>S. l. cerasiforme</em> accessions by Sanger sequencing of DNA fragments. We sequenced 81 fragments mapped on chromosome 2 and spread over three different mapping densities: (1) a whole chromosome density (1 fragment/5 cM); (2) a fine mapping density (1 fragment/cM) and (iii) a physical mapping density (1 fragment/100 kb). For physical mapping density, we focused on regions in which QTL were previously fine mapped (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lecomte <em>et al.</em> 2004</a>). In this study, we describe the level of molecular polymorphism detected. The extent of LD was assessed over the entire chromosome and over physical distances. Finally, association tests regarding FW, LCN, and soluble solid content (SSC) phenotypes were performed.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant material</h3><p id="p-13">The accessions were sampled in a germplasm collection that is maintained and characterized at the Institut National de la Recherche Agronomique (INRA) in Avignon, France. These accessions are part of a core collection drawn from 380 accessions that maximizes both genetic and phenotypic diversity (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Ranc <em>et al.</em> 2008</a>). A set of 90 tomato accessions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002667/-/DC1">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002667/-/DC1/TableS1.pdf">Table S1</a>) was used for sequence analysis. This sample was composed of 63 cherry type tomato accessions (<em>i.e.</em>, <em>S. lycopersicum</em> var. <em>cerasiforme</em>, hereafter named <em>S. l. cerasiforme</em>), 17 large fruited accessions (<em>S. lycopersicum</em> var. <em>esculentum</em>, hereafter named <em>S. l. esculentum</em>), and 10 <em>S. pimpinellifolium</em> accessions. Accessions were derived from French researchers' prospecting, breeders' collections, the Tomato Genetics Resource Center (Davis, CA), the Centre for Genetic Resources (Wageningen, The Netherlands), the North Central Regional Plant Introduction Station (Ames, IA), and the N.I. Vavilov Research Institute of Plant Industry (St. Petersburg, Russia).</p></div><div id="sec-3" class="subsection"><h3>Tomato phenotyping</h3><p id="p-14">The accessions were grown during 2007 and 2008 summers in Avignon. Four plants per accession were bred in plastic greenhouse. Three harvests of 10 ripe fruits were done for each accession and were used as repetition in the phenotypic analysis. The 10 fruits were phenotyped for FW, LCN, and SSC. Year and accession effects were assessed by two-factor analysis of variance with [R] software (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">R Development Core Team 2005</a>). Heritability estimations were calculated as following: <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/853/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/853/embed/mml-math-1.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/853/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/853/embed/mml-math-2.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/853/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/853/embed/mml-math-3.gif"/></noscript></span></span> the genetic and residual variance, respectively. <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/853/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/853/embed/mml-math-4.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/853/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/853/embed/mml-math-5.gif"/></noscript></span></span>were estimated by (MSc-MSe)/89 and MSe, respectively. MSc and MSe represent the mean squares of cultivar and residual effects, respectively. Because genetic effect over the two years was much significantly greater than year effect, we calculated associations by using accession adjusted means over years. FW and LCN were log transformed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002667/-/DC1/TableS1.pdf">Table S1</a>). Pearson correlations were assessed among traits.</p></div><div id="sec-4" class="subsection"><h3>DNA fragments sequenced</h3><p id="p-15">The positions of the sequence tag sites (STS) along the chromosome 2 are shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. We used Primer3 (Rozen and Skaletsky 2000) to design pairs of primers for each STS based on sequence data of genes and markers mapped on chromosome 2 (<a href="http://solgenomics.net/">http://solgenomics.net/</a>). These fragments were chosen to cover the entire chromosome with three different densities: (1) fragments every 5 cM chosen to cover the whole chromosome (2); fragments every cM chosen to cover the middle of the chromosome; and (3) fragments every 100 Kb chosen to cover five physical contigs representing candidate regions for fruit quality QTL: Contig1 (SL2.40ch02: 41129698.. 41563558), mapped around a sugar content QTL (sugs2.1); Contig2 (SL2.40ch02: 41752714.. 42140082), mapped in an LCN QTL (lcn2.1); Contig3 (SL2.40ch02: 42664935.. 43230501), mapped around a SSC QTL (ssc2.2); Contig4 (SL2.40ch02: 46744832.. 46893523), mapped in around FW QTL (fw2.2); and Contig5 (SL2.40ch02: 47342796.. 47472243), mapped around a sugar content QTL (sugs 2.2). Because of a low level of polymorphism previously described in <em>S. lycopersicum</em>, we targeted fragments for sequencing on intronic or intergenic regions. For a specific unigene, intron localization was predicted with tblastx on <em>Arabidopsis thaliana</em> genomic sequence and primers were designed on exonic sequence surrounding introns. The characteristics of the STS are presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002667/-/DC1/TableS2.pdf">Table S2</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/853/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic and physical location of the polymorphic fragments sequenced on chromosome 2. Genetic distances on the EXPEN2000 reference map are indicated on the left of the chromosome. Physical contigs are drawn on the right of the scheme. Cloned QTL are indicated on the left of the chromosome. Gray shaded area indicates homology of contigs on chromosome 2 pseudo-molecule. Numbers of polymorphisms (SNPs and indels) found in noncoding and coding regions are indicated within bracket in the first and second position, respectively. Markers in italics show high LD when compared together." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1747539730" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genetic and physical location of the polymorphic fragments sequenced on chromosome 2. Genetic distances on the EXPEN2000 reference map are indicated on the left of the chromosome. Physical contigs are drawn on the right of the scheme. Cloned QTL are indicated on the left of the chromosome. Gray shaded area indicates homology of contigs on chromosome 2 pseudo-molecule. Numbers of polymorphisms (SNPs and indels) found in noncoding and coding regions are indicated within bracket in the first and second position, respectively. Markers in italics show high LD when compared together.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/853/F1.medium.gif" width="381" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/853/F1.medium.gif" width="381" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/853/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/853/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461711" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-16" class="first-child">Genetic and physical location of the polymorphic fragments sequenced on chromosome 2. Genetic distances on the EXPEN2000 reference map are indicated on the left of the chromosome. Physical contigs are drawn on the right of the scheme. Cloned QTL are indicated on the left of the chromosome. Gray shaded area indicates homology of contigs on chromosome 2 pseudo-molecule. Numbers of polymorphisms (SNPs and indels) found in noncoding and coding regions are indicated within bracket in the first and second position, respectively. Markers in italics show high LD when compared together.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Fragment sequencing and analysis</h3><p id="p-17">Genomic DNA was isolated from 100 mg of frozen leaves using the DNeasy Plant Mini Kit (QIAGEN, Valencia, CA) according the manufacturer's recommendations. Amplification reactions were performed in a final volume of 5 µL in a reaction mix composed of 2.5 ng of template DNA, 0.4 pmol of each primer, 0.05 mM concentration of each deoxynucleotide, 2 mM MgSO<sub>4</sub>, 1X <em>Taq</em> polymerase buffer P, and 0.03 units of Platinum <em>Taq</em> HiFi (Invitrogen, Carlsbad, CA). After 5 min of denaturation at 94°, 30 cycles were performed of 20 s at 94°, initial denaturation during 20 s at 55°, annealing during 2 min at 68°, followed by a final extension step of 5 min at 68°. Pairs of primers revealing single-band polymerase chain reaction (PCR) product were chosen for sequencing. PCR products were purified using the ExoSAP method with Exonuclease I (NEB, Beverly, MA) and Shrimp Alkaline Phosphatase (USB, Cleveland, OH). Fragments were sequenced with SP6 universal primer in an adapted 5-µL reaction volume method using BigDye terminator V3.1 and analyzed on an ABI 3730 xl sequencer (Applied Biosystems, Foster City, CA). Sequence alignment and SNP detection were performed using Genalys software available at <a href="http://software.cng.fr/">http://software.cng.fr/</a> (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Takahashi <em>et al.</em> 2003</a>). Sequences of <em>lcn2.1</em>, previously obtained for this core collection (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Muños <em>et al.</em> 2011</a>), were added in this study (embl accession number <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=JF284938&amp;atom=%2Fggg%2F2%2F8%2F853.atom">JF284938</a> and <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=JF284939&amp;atom=%2Fggg%2F2%2F8%2F853.atom">JF284939</a>). Genotype data are provided in <a href="http://www.g3journal.orglookup/suppl/doi:10.1534/g3.112.002667/-/DC1/TableS3.xlsx">Table S3</a>.</p></div><div id="sec-6" class="subsection"><h3>Linkage disequilibrium</h3><p id="p-18">The molecular diversity was estimated by Watterson's θ. The LD parameter <em>r<sup>2</sup></em> was estimated among loci with <span class="sc">tassel</span> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bradbury <em>et al.</em> 2007</a>), and the comparison-wise significance was computed by 1000 permutations. We compared different strategies for analyzing LD decay over genetic distances. We examined pairwise LD values, analyzing all polymorphisms with minor allele frequency (MAF) greater than 5% or only one polymorphism by fragment with the greatest heterozygosity index. We also compared pairwise LD decay between polymorphisms assessed in the whole population (N = 90) or only in the <em>cerasiforme</em> subset (n = 63). Pairwise <em>r<sup>2</sup></em> were plotted according to genetic distance between two loci, and nonlinear regression fitted the decay of LD over genetic or physical distance. The decrease of LD over genetic distance was fitted by the equation: <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/853/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/853/embed/mml-math-6.gif"/></noscript></span></span> using nonlinear regression, where y represents <em>r<sup>2</sup></em> and x represents the genetic or physical distance in cM or kb (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Tenesa <em>et al.</em> 2004</a>).</p></div><div id="sec-7" class="subsection"><h3>Association analysis</h3><p id="p-19">An association study was performed with the set of 90 accessions. Several statistical models were tested: (1) the Simple general linear model (GLM); (2) the structured association model (Q model), taking into account only the structure of the collection; and (3) the mixed linear model (K+Q or MLM model), taking into account both kinship and structure, as described by <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Yu <em>et al.</em> (2006)</a>. The significance of associations between traits and markers was estimated with T<span class="sc">assel</span>. Population assignment of individuals was inferred by Structure 2.1 software (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Pritchard <em>et al.</em> 2000a</a>) based either on 20 simple sequence repeat (SSR) markers spread throughout the genome (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Ranc <em>et al.</em> 2008</a>) or on the genotypes of all the STS markers or just a subset of these markers. For inferring the most likely number of population, the <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Evanno <em>et al.</em> (2005)</a> transformation method was used. The Ritland's matrix of relative kinship coefficients (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Ritland 1996</a>), implemented in the mixed linear model, was estimated using SPAGeDi (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hardy and Vekemans 2002</a>) based on the set of SSR markers. According to <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Yu <em>et al.</em> (2006)</a>, the diagonal of the matrix was set to 2.0, and the negative values were set to 0. To deal with multiple testing, we computed adjusted <em>P</em> values using <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Benjamini and Hochberg (2000)</a> procedures to control for the false discovery rate. Associations with an adjusted <em>P</em> value less than 0.005 were declared significant. For markers that were significantly associated with a trait, a general linear model with all fixed-effect terms was used to estimate R<sup>2</sup>, the amount of phenotypic variation explained by each marker. The standardized effect of each marker was also calculated by dividing the difference of average values of the two homozygous classes by the phenotypic standard deviation for the trait (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Weber <em>et al.</em> 2008</a>). The accession used for tomato genome sequencing, Heinz 1706, was used as a reference for allele effect calculation.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Identification of polymorphisms on chromosome 2</h3><p id="p-20">Eighty-six pairs of primers, corresponding to 86 loci on chromosome 2, revealed a unique PCR band and were chosen for forward sense sequencing of the 90 tomato accessions. Five fragments were not readable because of heterozygous signals, probably due to the amplification of paralogous sequences. The 81 remaining fragments (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002667/-/DC1/TableS4.pdf">Table S4</a>) had an average size of 542 bp. Noncoding regions represented almost 69% (30,396 bp) of the total length sequenced (44,223 bp). Eleven fragments (13%) were monomorphic among the 90 accessions. <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> shows the location and polymorphism content of the 70 polymorphic STS. A total of 300 single-nucleotide polymorphisms (SNPs) and 52 insertion-deletions (indels) were detected among 90 accessions. Only polymorphisms with MAF values greater than 5% were taken into account in the following description. Polymorphisms were analyzed according to species membership of accessions (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). SNPs and indels were more frequent in noncoding regions, with an average of 8.7 polymorphisms per 1000 bp, than in the exonic parts of genes (average of 5.4 polymorphisms per 1000 bp). The molecular diversity decreased from wild to cultivated groups, whereas the number of polymorphisms dropped only for <em>S. l. esculentum</em> (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). <em>S. l. cerasiforme</em> shared polymorphisms with both cultivated and wild accessions. <em>S. l. cerasiforme</em> had only five specific polymorphisms, and 344 polymorphisms shared with one of the two other species (187 with <em>S. pimpinellifolium</em>, 11 with <em>S. lycopersicum</em>, and 146 with both species). Fifty-four percent of overall polymorphisms identified in <em>S. l. esculentum</em> corresponded to singletons within this group. Most of these polymorphisms were carried by two accessions (LA0409 and Stupicke Polni Rane).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461639/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461639/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Distribution and frequencies of polymorphisms (SNP and indel) across species and ratio of polymorphism in coding and noncoding region</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/853/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Molecular diversity of the three groups of tomato based on 352 polymorphisms. Molecular diversity was estimated by Watterson's &#x3B8; and compared with the total number of polymorphisms (S) for S. pimpinellifolium, S. l. cerasiforme, and S. l. esculentum." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1747539730" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Molecular diversity of the three groups of tomato based on 352 polymorphisms. Molecular diversity was estimated by Watterson's &#x3B8; and compared with the total number of polymorphisms (S) for S. pimpinellifolium, &lt;em&gt;S. l. cerasiforme&lt;/em&gt;, and &lt;em&gt;S. l. esculentum&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/853/F2.medium.gif" width="440" height="242"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/853/F2.medium.gif" width="440" height="242"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/853/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/853/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461704" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-23" class="first-child">Molecular diversity of the three groups of tomato based on 352 polymorphisms. Molecular diversity was estimated by Watterson's θ and compared with the total number of polymorphisms (S) for S. pimpinellifolium, <em>S. l. cerasiforme</em>, and <em>S. l. esculentum</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">The ratio of polymorphisms in noncoding regions to coding regions is similar in <em>S. pimpinellifolium</em> and <em>S. l. cerasiforme</em> but is strikingly higher in <em>S. l. esculentum</em> (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). <em>S. l. esculentum</em> also showed an excess of low frequency polymorphisms, as did <em>S. l. cerasiforme</em>, although to a lesser extent (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/853/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of polymorphism MAFs among tomato species. S. l. cerasiforme (n = 63) is represented in black, S. l. esculentum (n = 17) in dark gray, and S. pimpinellifolium (n = 10) in light gray. Polymorphisms with overall species MAF lower than 0.05 were previously discarded (see Materials and Methods)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1747539730" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of polymorphism MAFs among tomato species. &lt;em&gt;S. l. cerasiforme&lt;/em&gt; (n = 63) is represented in black, &lt;em&gt;S. l. esculentum&lt;/em&gt; (n = 17) in dark gray, and &lt;em&gt;S. pimpinellifolium&lt;/em&gt; (n = 10) in light gray. Polymorphisms with overall species MAF lower than 0.05 were previously discarded (see &lt;em&gt;Materials and Methods&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/853/F3.medium.gif" width="440" height="248"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/853/F3.medium.gif" width="440" height="248"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/853/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/853/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461716" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-25" class="first-child">Distribution of polymorphism MAFs among tomato species. <em>S. l. cerasiforme</em> (n = 63) is represented in black, <em>S. l. esculentum</em> (n = 17) in dark gray, and <em>S. pimpinellifolium</em> (n = 10) in light gray. Polymorphisms with overall species MAF lower than 0.05 were previously discarded (see <em>Materials and Methods</em>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Linkage disequilibrium</h3><p id="p-26">We compared LD decay over genetic distances in different samples. LD decreased over shorter genetic distances when all polymorphisms per sequence were taken into account than when using a single polymorphism per fragment. Minimal difference was observed when only the <em>cerasiforme</em> subset was analyzed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002667/-/DC1/FigureS1.pdf">Figure S1</a>). LD was likely overestimated in the whole sample because of the genetic structure with both cultivated and wild accessions added to the <em>cerasiforme</em> subset. For further LD analysis, we focused on the 63 <em>cerasiforme</em> accessions. Based on the regression of LD over distances, LD decay reached <em>r<sup>2</sup></em> = 0.3 for a genetic distance of 1 cM, and the minimal value of <em>r<sup>2</sup></em> = 0.09 was obtained for distances of 13 cM (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>A). Nevertheless, high <em>r<sup>2</sup></em> (reaching the maximum value <em>r<sup>2</sup></em> = 1) remained even within a distance of 60 cM, but only 28 sites of 340 (corresponding to 12 STS spread over chromosome 2) were responsible for these high pairwise LD values.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/853/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimates of LD (r2) vs. genetic and physical distance on chromosome 2 for the 63 S. l. cerasiforme accessions. Only polymorphic sites having MAF greater than 5% are indicated (see Materials and Methods). (A) Decay of r2 over genetic distance on chromosome 2. Plot of r2 over distance was fitted by nonlinear regression (red curve). (B) Decay of r2 over physical distance on the five major contigs. Plot of r2 over distance is fitted by nonlinear regression (red curve). The inset shows a more detailed view of the LD decay curve for markers located less than 20 Kb apart. (C) Matrix of pairwise LD P value between and within physical contigs. P values were calculated with 1000 permutations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1747539730" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Estimates of LD (&lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt;) &lt;em&gt;vs.&lt;/em&gt; genetic and physical distance on chromosome 2 for the 63 &lt;em&gt;S. l. cerasiforme&lt;/em&gt; accessions. Only polymorphic sites having MAF greater than 5% are indicated (see &lt;em&gt;Materials and Methods&lt;/em&gt;). (A) Decay of &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; over genetic distance on chromosome 2. Plot of &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; over distance was fitted by nonlinear regression (red curve). (B) Decay of &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; over physical distance on the five major contigs. Plot of &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; over distance is fitted by nonlinear regression (red curve). The inset shows a more detailed view of the LD decay curve for markers located less than 20 Kb apart. (C) Matrix of pairwise LD &lt;em&gt;P&lt;/em&gt; value between and within physical contigs. &lt;em&gt;P&lt;/em&gt; values were calculated with 1000 permutations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/853/F4.medium.gif" width="239" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/853/F4.medium.gif" width="239" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/853/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/853/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461750" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-27" class="first-child">Estimates of LD (<em>r<sup>2</sup></em>) <em>vs.</em> genetic and physical distance on chromosome 2 for the 63 <em>S. l. cerasiforme</em> accessions. Only polymorphic sites having MAF greater than 5% are indicated (see <em>Materials and Methods</em>). (A) Decay of <em>r<sup>2</sup></em> over genetic distance on chromosome 2. Plot of <em>r<sup>2</sup></em> over distance was fitted by nonlinear regression (red curve). (B) Decay of <em>r<sup>2</sup></em> over physical distance on the five major contigs. Plot of <em>r<sup>2</sup></em> over distance is fitted by nonlinear regression (red curve). The inset shows a more detailed view of the LD decay curve for markers located less than 20 Kb apart. (C) Matrix of pairwise LD <em>P</em> value between and within physical contigs. <em>P</em> values were calculated with 1000 permutations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">We assessed the extent of LD over physical distances within the five physical contigs covering a total of 1.86 Mb (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>B). The minimal estimated <em>r<sup>2</sup></em> fitted value of 0.13 was obtained within 20 kb, but high pairwise LD persisted within 400 kb. <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>C shows the matrix of LD between polymorphic sites of the physical contigs. The pattern of LD intensity over physical distances was heterogeneous. In Contig1, polymorphisms within STS formed blocks with high LD. In Contig2 and the first part of Contig3, STS did not form LD blocks. High LD between and within STS was interrupted by polymorphisms showing low LD with other polymorphisms. A striking break in the LD pattern over physical distance appeared in the middle of Contig3, where strong intrafragment blocks of LD but low LD between fragments were observed. To check whether this region corresponds to a hotspot of recombination, we used the tomato genome sequence to assess the physical positions of STS and the reference genetic map (EXPEN2000, <a href="http://solgenomics.net">http://solgenomics.net</a>), and we calculated the ratio of physical to genetic distances among STS. The genetic <em>vs.</em> physical distance ratios in Contig3 were unevenly distributed with 136 kb/cM between TD140 and TD055 and 20 kb/cM between TD109 and TD106, suggesting the presence of a hotspot of recombination. The difference in LD behavior between and within contig clearly appears on graphical haplotypes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002667/-/DC1/FigureS2.pdf">Figure S2</a>).</p></div><div id="sec-11" class="subsection"><h3>Association mapping</h3><p id="p-29">The genetic structure of 90 tomato accessions was first estimated using 20 SSR markers spread over the genome. The most probable number of subpopulations in the sample was two (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002667/-/DC1/FigureS3.pdf">Figure S3</a>). A subdivision in four populations was also detected, as previously shown with 318 accessions (<a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Ranc <em>et al.</em> 2008</a>). Twenty-six cherry tomato accessions were not clustered with high probability (Q &gt; 0.8) within one structure group and were thus classified as an admixture between the two major groups (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002667/-/DC1/TableS1.pdf">Table S1</a>). The same trend of structure with only two populations was observed when estimating the structure with all the STS markers on chromosome 2.</p><p id="p-30">FW and LCN were log-transformed to fit a normal distribution graphically, but LCN fitted a Poisson distribution. The three traits were correlated together (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002667/-/DC1/FigureS4.pdf">Figure S4</a>). Broad-sense heritabilities were high: 0.94, 0.96, and 0.95 for SSC, FW and LCN, respectively. Genetic structure assessed by SSR markers had a significant effect on FW and SSC with R<sup>2</sup> values of 0.24 and 0.12, respectively, whereas population structure accounted only for 5% of the LCN variation. For association mapping, the mixed model taking into account both genetic structure assessed with all STS (Q<sub>STS</sub>) and coancestry matrices (K+Q model) resulted in the best approximation of the expected cumulative distribution of <em>P</em> values, followed by the K+Q model with Q assessed with SSR markers (Q<sub>SSR</sub>), then the structured association model (Q model) and the simple model (GLM; <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/853/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cumulative density functions (CDF) using several alternative models of association. Model comparisons are performed for FW (A), LCN (B), and SSC (C). Associations are tested for all polymorphic sites with MAF &gt;5% on 90 individuals. Naive GLM (black diamond) and K+Q models, with structure based on SSR markers (white squares), on 4 PCA axis (white circles) and on all STS markers (black squares) were tested. The diagonal indicates uniform distribution of P values under the expectation that random SNPs are unlinked to the polymorphisms controlling these traits (H0: no SNP effect)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1747539730" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Cumulative density functions (CDF) using several alternative models of association. Model comparisons are performed for FW (A), LCN (B), and SSC (C). Associations are tested for all polymorphic sites with MAF &gt;5% on 90 individuals. Naive GLM (black diamond) and K+Q models, with structure based on SSR markers (white squares), on 4 PCA axis (white circles) and on all STS markers (black squares) were tested. The diagonal indicates uniform distribution of &lt;em&gt;P&lt;/em&gt; values under the expectation that random SNPs are unlinked to the polymorphisms controlling these traits (H0: no SNP effect).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/853/F5.medium.gif" width="284" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/853/F5.medium.gif" width="284" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/853/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/853/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461680" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-31" class="first-child">Cumulative density functions (CDF) using several alternative models of association. Model comparisons are performed for FW (A), LCN (B), and SSC (C). Associations are tested for all polymorphic sites with MAF &gt;5% on 90 individuals. Naive GLM (black diamond) and K+Q models, with structure based on SSR markers (white squares), on 4 PCA axis (white circles) and on all STS markers (black squares) were tested. The diagonal indicates uniform distribution of <em>P</em> values under the expectation that random SNPs are unlinked to the polymorphisms controlling these traits (H0: no SNP effect).</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">We also tested alternative models to take the structure into account. Taking in the MLM model the four main coordinates of significant axes of principal components analysis provided almost similar results to the naïve model (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>) as well as using k = 4 structure model (data not shown). When we used Q<sub>STS</sub> in the MLM model, the probability plot was much closer to the diagonal (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>), suggesting that the correction for the structure was much better, and thus we present the associations obtained with this model with corrected <em>P</em> values less than 0.05 (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). With this model, we detected 14, 3, and 3 associations with FW, LCN, and SSC, respectively (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Using just a subset of 265, we found that STS avoiding the loci involved in the main regions where significant associations were detected provided the same results (data not shown).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461819/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461819/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Significant associations for fruit weight (FW), locule number (LCN), and soluble solid content (SSC) estimated with K+Q models on 90 accessions</span><div class="sb-div caption-clear"></div></div></div><p id="p-39">Because the correction for structure when using Q<sub>SSR</sub> was not fully satisfying with this model and many associations appeared significant, we only retained the most significant associations with adjusted <em>P</em> value lower than 0.005, although some association may still be false positive. With this model, we detected 37, 3, and 14 associations with FW, LCN, and SSC, respectively. Finally, taking into account both STS and SSR markers for the structure analysis gave the same results as the STS alone. For FW, LCN, and SSC, polymorphisms with the greatest <em>P</em> values explained a large part of the trait variation (22%, 44%, and 21%, respectively). As reference alleles are based on Heinz 1706, a genotype with large fruit and low SSC, allele effects were almost all positive for FW, whereas allele effects for SSC were all negative.</p><p id="p-40"><a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> shows the significant associations between the polymorphisms and the traits along the chromosome with both MLM models. Most of the polymorphisms found in association with one of the traits were part of a dense chromosome region. For FW, the two strongest associations involved TD380-526 (fragment TD380 polymorphic site at position 526) on Contig3 and TD387-452 on Contig2. The <em>r<sup>2</sup></em> value for LD estimation between these two SNPs is 0.41 in the whole accession sample (<a href="http://www.g3journal.orglookup/suppl/doi:10.1534/g3.112.002667/-/DC1/FigureS5.pdf">Figure S5</a>). Because other sites revealed similar level of LD and did not result in significant association, these two associations could correspond to two linked QTL on adjacent contigs. When only the 63 <em>S. l. cerasiforme</em> accessions were used for association analysis, TD387-452 was not associated with FW, but association with TD380-526 remained significant. A significant association for FW was detected with TD056-134, which corresponds to the 5′ region of the <em>fw2.2</em> QTL previously cloned by positional cloning (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Nesbitt and Tanksley 2002</a>). In addition, TD049-528 was associated with FW and colocated with FW2.1, a QTL for FW variation fine mapped in a biparental <em>S. l. esculentum</em> × <em>S. l. cerasiforme</em> progeny (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Lecomte <em>et al.</em> 2004</a>). Finally, we detected significant associations for FW with coding polymorphisms in the TD055 fragment corresponding to the <em>Ovate</em> gene.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/853/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of association P values over the chromosome 2. Associations are estimated for 90 accessions. K+Q model was used to screen for association between polymorphisms and (A) FW, (B) LCN, and (C) SSC. Stars indicate the associations detected with the structure assessed with all STS, and black dots the associations detected with 20 SSR markers. The upper part of each graph represents associations along genetic distance over the entire chromosome 2. The lower part shows associations for each physical contig. Arrows indicate the marker name of the most significant associations. Adjusted P values for multiple testing (see Materials and Methods) are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1747539730" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of association &lt;em&gt;P&lt;/em&gt; values over the chromosome 2. Associations are estimated for 90 accessions. K+Q model was used to screen for association between polymorphisms and (A) FW, (B) LCN, and (C) SSC. Stars indicate the associations detected with the structure assessed with all STS, and black dots the associations detected with 20 SSR markers. The upper part of each graph represents associations along genetic distance over the entire chromosome 2. The lower part shows associations for each physical contig. Arrows indicate the marker name of the most significant associations. Adjusted &lt;em&gt;P&lt;/em&gt; values for multiple testing (see &lt;em&gt;Materials and Methods&lt;/em&gt;) are shown.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/853/F6.medium.gif" width="385" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/853/F6.medium.gif" width="385" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/853/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/853/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461655" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 </span> <p id="p-41" class="first-child">Plot of association <em>P</em> values over the chromosome 2. Associations are estimated for 90 accessions. K+Q model was used to screen for association between polymorphisms and (A) FW, (B) LCN, and (C) SSC. Stars indicate the associations detected with the structure assessed with all STS, and black dots the associations detected with 20 SSR markers. The upper part of each graph represents associations along genetic distance over the entire chromosome 2. The lower part shows associations for each physical contig. Arrows indicate the marker name of the most significant associations. Adjusted <em>P</em> values for multiple testing (see <em>Materials and Methods</em>) are shown.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">For LCN, only three associations were significant (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>, <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>B). The greatest associations involved two SNPs that have been identified through map-based cloning as responsible for the LCN variation (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Muños <em>et al.</em> 2011</a>). LD between these two SNPs was extreme (<em>r<sup>2</sup></em> = 0.95). The other significant association implicated TD373-391 on the same contig. TD373-391 showed the greatest <em>r<sup>2</sup></em> with the <em>lcn2.1</em> SNPs (<em>r<sup>2</sup></em> = 0.47). This association may thus result from the LD with the functional <em>lcn</em>2.1 SNPs (<a href="http://www.g3journal.orglookup/suppl/doi:10.1534/g3.112.002667/-/DC1/FigureS5.pdf">Figure S5</a>).</p><p id="p-43">For SSC, the strongest associations were found with TD380-526 and TD387-452 loci, which were also significantly associated with FW. These results could be a consequence of the high negative correlation between FW and SSC (r = −0.66). Several of the other polymorphisms showing associations when using QSSR were in significant LD (<a href="http://www.g3journal.orglookup/suppl/doi:10.1534/g3.112.002667/-/DC1/FigureS5.pdf">Figure S5</a>).</p><p id="p-44">When we screened for associations using only the 63 cherry tomato accessions, a group of accession chosen to limit the population structure (<a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">Ranc <em>et al.</em> 2008</a>), the MLM model was very close to the naïve GLM model (<a href="http://www.g3journal.orglookup/suppl/doi:10.1534/g3.112.002667/-/DC1/FigureS5.pdf">Figure S5</a>). Many associations were no more significant whatever the model. For FW, significant associations were detected with loci that were also detected in the whole collection, TD380-526, TD056-134 (fw2.2), TD116-707, and TD117-219. A new association was detected with TD138-61. For LCN, the two SNPs in the <em>lcn2.1</em> locus remained significant. For SSC, the main association with TD380-526 was significant, as well as two with TD120 markers (<a href="http://www.g3journal.orglookup/suppl/doi:10.1534/g3.112.002667/-/DC1/TableS5.pdf">Table S5</a>).</p></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-45">To assess the genetic diversity among tomato accessions and analyze the extent of LD, we sequenced 81 DNA fragments, covering 44 kb, in 90 accessions of wild and cultivated tomatoes. We detected 352 polymorphic loci (SNP or indel). The extent of LD varied according to the regions, scales, and associations between phenotypes and polymorphisms that were successfully detected.</p><div id="sec-13" class="subsection"><h3>Power of <em>S. l. cerasiforme</em> for polymorphism discovery</h3><p id="p-46">The 63 <em>S. l. cerasiforme</em> accessions were previously sampled to maximize both genetic and phenotypic diversity. This sample captures 98% of SSR alleles identified in a larger sample of 144 cherry type accessions (<a id="xref-ref-41-6" class="xref-bibr" href="#ref-41">Ranc <em>et al.</em> 2008</a>). These accessions represent a large level of molecular variability that is almost identical to that of their wild progenitor, <em>S. pimpinellifolium</em>. In tomato, several studies were aimed at discovering SNPs and indels. <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Nesbitt and Tanksley (2002)</a> searched for molecular polymorphisms in the <em>fw2.2</em> region within a collection of <em>S. l. esculentum</em> (N = 4) <em>and S. l. cerasiforme</em> (N = 39) accessions. They found only one SNP per 7 kbp within <em>S. l. esculentum</em> accessions and one SNP per 340 bp within the <em>S. l. cerasiforme</em> sample. Mining or resequencing ESTs is another strategy to discover SNPs. Using this method, Yang <em>et al.</em> (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Yang <em>et al.</em> 2004</a>) detected one SNP every 8500 bp in coding regions. <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jimenez-Gomez and Maloof (2009)</a> found more than 15,000 intraspecific polymorphisms in a set of 223,000 ESTs in <em>S. lycopersicum</em>. However, most of these polymorphisms have low allelic frequency in cultivated tomato. (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Labate and Baldo 2005</a>) reported a greater amount of polymorphic ESTs, but the studied accessions were described as highly variable compared with other <em>S. lycopersicum</em> accessions because of introgressions from wild relatives. Among the 1487 SNPs detected by <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Labate <em>et al.</em> (2009)</a>, only 162 were polymorphic in <em>S. lycopersicum</em> breeding germplasm, and most of them had minor allele frequency below 10%.</p><p id="p-47"><a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Van Deynze <em>et al.</em> (2007)</a> increased the frequency of SNPs and indels compared with previous studies by focusing on gene introns. In the present study, the use of <em>S. l. cerasiforme</em> allowed us to detect 352 polymorphisms (SNP and indels) in 81 sequenced fragments. Four of the eleven monomorphic fragments (TD085, TD098, TD111, and TD384) contained only coding regions, which are less polymorphic. The difference in the polymorphism rate between species for noncoding regions may be a consequence of either (1) hitch-hiking of the region surrounding a selected polymorphism or (2) a demographic bottleneck during domestication associated with a reduction of the population effective size. <em>S. l. cerasiforme</em> suffered a decrease of its population effective size during domestication from <em>S. pimpinellifolium</em> (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bai and Lindhout 2007</a>). The lack of diversity differences between <em>S. pimpinellifolium</em> and <em>S. l. cerasiforme</em> could be due to the greater number of accessions sequenced for the latter. The theta diversity statistic corrected for the unbalance in sample size and highlighted a higher molecular diversity in the wild sample. Molecular polymorphism is linked to the population effective size by the Watterson's estimate of the scaled mutation rate (per site) <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/853/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/853/embed/mml-math-7.gif"/></noscript></span></span>, where <em>Ne</em> is the population size and <em>µ</em> is the mutation rate. The transfer of the tomato from Mexico to Europe during the 16th century greatly reduced the effective population size of the tomato and subsequently decreased the amount of molecular diversity in <em>S. lycopersicum</em>. A selection pressure that targeted coding regions could explain the higher ratio between noncoding and coding polymorphisms for <em>S. lycopersicum</em>. The reduction of diversity could arise on the fragment targeted by selection but also on the region suffering genetic hitchhiking or background selection (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Innan and Stephan 2003</a>). Thus, a less drastic reduction in population size and continuous inter-mating with <em>S. pimpinellifolium</em> shaped a higher level of molecular diversity for <em>S. l. cerasiforme</em>.</p></div><div id="sec-14" class="subsection"><h3>LD decay over genetic and physical distances</h3><p id="p-48">An ancient admixture increased the polymorphism level of cherry tomatoes and limited their overall LD. We reached minimal LD values (<em>r<sup>2</sup></em> &lt; 0.09) with distances greater than 13 cM, but extreme LD values were still found over 60 cM for a few marker pairs. Our results support those from <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">van Berloo <em>et al.</em> (2008)</a>, who described an LD extent ranging from 15 to 20 cM using AFLP markers in a cherry tomato sample (N = 18). <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Nesbitt and Tanksley (2002)</a> showed that LD in <em>S. l. cerasiforme</em> could be broken within 150 kb around <em>fw2.2</em>. With an average ratio of 750 kb/cM on the whole tomato genome (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Tanksley <em>et al.</em> 1992</a>), the results of LD decay over physical and genetic distances are not consistent. In our <em>S. l. cerasiforme</em> sample, some <em>r<sup>2</sup></em> values were still extreme over hundreds of kb, but the drop estimated by nonlinear regression indicated that minimal LD is reached over 20 kb. <em>Arabidopsis thaliana</em> also showed a large extent of LD over the <em>FRI</em> locus. LD extends to 200 kb, corresponding to one cM in this species (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Nordborg <em>et al.</em> 2002</a>). This estimate is locus-specific and when studies are performed on the whole genome, LD decays within 10 kb on average (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kim <em>et al.</em> 2007</a>). <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Nordborg (2000)</a> estimated from simulations that LD should vanish over a scale of 10 kb for inbred species. Our results in the cherry tomato support these simulations. The results of LD decay over genetic distances in the tomato are similar to the LD pattern assessed in barley, another highly inbred crop (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Zhang <em>et al.</em> 2009</a>). In barley, large differences are observed in the LD decay pattern among cultivated accessions, landraces, and wild accessions (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Caldwell <em>et al.</em> 2006</a>). The greater LD extent for the crop compared with the wild ancestor or to <em>A. thaliana</em> could be due to a major bottleneck that fixed large haplotypes during domestication.</p><p id="p-49">The LD pattern observed in the physical contigs is similar to haplotype blocks described in soybean landraces, which is also an inbred crop (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hyten <em>et al.</em> 2007</a>). It is also similar to haplotype blocks in <em>A. thaliana</em> (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Kim <em>et al.</em> 2007</a>) and in humans (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Daly <em>et al.</em> 2001</a>). For the tomato, this LD pattern could be due to recent mutations with low frequencies (more than 50% of polymorphisms had MAF &lt; 0.2). These polymorphism patterns may have evolved by lineage effects rather than by recombination and thus may decay in LD in a small region that is not correlated to distance. The high LD pattern described in the first part of Contig3 and Contig2 could have been shaped by selection. Clusters of QTL have been mapped in this region regarding LCN, fruit shape, FW, soluble solids, and sugar content (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Lecomte <em>et al.</em> 2004</a>). The selection of new advantageous mutations during domestication should have increased LD in domesticated accessions (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Nordborg and Tavare 2002</a>). In <em>A. thaliana</em>, LD blocks surrounding selected polymorphisms are significantly longer than blocks surrounding nonselected alleles (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Kim <em>et al.</em> 2007</a>). Finally, a recombination hotspot is likely responsible for the break in the LD pattern observed in Contig3. Mapping data offered direct confirmation of uneven distribution of recombination over Contig3, but the high density of polymorphisms detected in this study should be mapped on a large F2 population to confirm the presence of such a recombination hotspot (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Drouaud 2006</a>).</p></div><div id="sec-15" class="subsection"><h3>Candidate genes are validated by association mapping</h3><p id="p-50">Our approach using a core collection was efficient in detecting association in several candidate gene regions. Recently, <em>lcn2.1</em> was identified by the map-based cloning approach as a QTN responsible for variation in the tomato LCN (<a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Muños <em>et al.</em> 2011</a>). We used information on <em>lcn2.1</em> to highlight any possible effect of these two SNPs on FW and SSC. A significant association was found between these two SNPs and FW. <a id="xref-ref-29-6" class="xref-bibr" href="#ref-29">Muños <em>et al.</em> (2011)</a> highlighted the role of this locus in tomato domestication and further FW increase. This association was the only one with negative allelic effect. The reference genotype, Heinz 1706, has large fruits with only two locules, whereas almost all other two-locule genotypes carry small fruit. The large number of these small-fruit accessions in the reference group induced a negative effect for FW. <a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Nesbitt and Tanksley (2002)</a> could not detect any association in a <em>S. l. cerasiforme</em> sample between FW and polymorphisms in the <em>fw2.2</em> region cloned previously. These authors concluded that genes other than <em>fw2.2</em> are responsible for the variation of FW in cherry tomatoes. The number of accessions (39 <em>S. l. cerasiforme</em>, 4 <em>S. l. esculentum</em> and 3 <em>S. pimpinellifolium</em>) was the principal limitation of the study. Using 90 accessions selected to represent the diversity of a larger collection, we found a significant association with a polymorphic site located in the promoter of the gene. This polymorphism could be responsible for the phenotype variation or could be in LD with the responsible one. The entire cloned region should be sequenced and tested for association before concluding.</p></div><div id="sec-16" class="subsection"><h3>Association mapping for the discovery of new QTL and candidate genes</h3><p id="p-51">Many QTL related to fruit traits map to chromosome 2 (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Causse <em>et al.</em> 2002</a>, Labate <em>et al.</em>, 2007). These QTL and the QTL that were fine-mapped in the mapping population <em>S. l. cerasiforme</em> × S. <em>l. esculentum</em> for FW, LCN and SSC (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Lecomte <em>et al.</em> 2004</a>) were also identified by association mapping. The screening of polymorphisms on chromosome 2 with high-density markers allowed the detection of many new associations and identification of several putative new candidate genes. The number of significant associations found with FW can result from LD caused by strong selection on this phenotype (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bai and Lindhout 2007</a>). TD380-526 showed the most significant association with FW. This fragment STS matched a predicted gene, Solyc02g085390.1.1, which is homologous to <em>A. thaliana</em>'s SNF2-like protein (AT5G66750). This gene has been characterized as an ATP-dependent helicase with chromatin remodeling activity. Chromatin remodeling proteins reconfigure protein–DNA interactions that accompany or induce changes in genome activity, such as gene expression (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kaya <em>et al.</em> 2001</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Verbsky and Richards 2001</a>). The other highly significantly associated fragment, TD387, has homology with a <em>S. lycopersicum</em> unigene (SGN-U596069) and matches the <em>S. lycopersicum</em> annotation Solyc02g084070.1.1. This gene has no homology with any gene of known function.</p><p id="p-52">Another association was detected for FW with TD049-528. TD049 was tagged in the 3′ region of a gene coding for glyoxalase I (Solyc02g080630.1.1). This gene colocalizes with a QTL for FW variation in a mapping population derived from a <em>S. l. esculentum</em> x <em>S. l. cerasiforme</em> cross (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Saliba-Colombani <em>et al.</em> 2001</a>). Because of the putative impact of glyoxalase I protein on plant cell proliferation (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Paulus <em>et al.</em> 1993</a>), this gene represents a good candidate gene for FW variation. The two polymorphisms most significantly associated with FW were also associated with SSC. This could be due to the dilution effects of soluble sugars and acids according to fruit size (Prudent <em>et al.</em>, 2010). The two polymorphisms were no longer statistically associated with SSC when we added the FW effect as a covariate in the K+Q-model. We observed the same result for TD117 (Solyc02g091640.1.1, which codes for an Endoribonuclease E-like protein), which is genetically close to the <em>fw2.2</em> gene. The two other strongest associations, TD047 (promoter of Solyc02g083950.1.1, which codes for the WUSCHEL transcription factor) and TD133 (Solyc02g084030, which codes for a methionine sulfoxide reductase), are both located in the same region as TD120. Because TD047 and TD133 are separated by a distance of 2 cM, this region must be enriched in SNPs to locate precisely one or more responsible polymorphisms. TD055 mapped in a SSC QTL (brix2.2) described in the mapping population involving cherry tomato (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Saliba-Colombani <em>et al.</em> 2001</a>; Lecomte <em>et al.</em> 2004). TD055 was designed in the <em>Ovate</em> gene and showed association with SSC. <em>Ovate</em> is implicated in the modification of fruit shape, but no effect on SSC has yet been reported. This polymorphism could thus be in LD with the responsible polymorphism. SSC also showed significant association with TD140 (Solyc02g085100 0.1.1), which was identified as an aldose-1-epimerase. This enzyme catalyzes the transformation of alpha-D-glucose into beta-D-glucose and participates in glycolysis and gluconeogenesis. The aldose-1-epimerase thus represents a new candidate for SSC variation.</p></div><div id="sec-17" class="subsection"><h3>Optimal conditions for genome-wide association studies in tomato</h3><p id="p-53">We highlighted the greater efficiency of the K+Q-model in dealing with type I error rates for association mapping in the tomato. Information on the estimated familial relatedness in our sample did not influence the results for association with FW because most of the false positives are also corrected with genetic structure information. The K+Q-model may prove its power in a sample of increased size as well as broader allelic diversity (<a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Yu <em>et al.</em> 2006</a>). A greater number of markers to detect structure may also reveal a more subtle structure. Taking in the MLM model the structure in 4 subgroups did not change the associations, neither using the coordinates of the first four axes of principal components analysis. The departure from the distribution of <em>P</em> values under the expectation that random SNPs are not linked to the polymorphisms controlling FW, SSC, and LCN indicates that our analysis did not succeed in correcting for the whole genetic structure. However, the number of polymorphisms tested was too small and nonrandomly spread over the genome. We then decided to focus only on highly significant associations to reduce the acceptance of false-positive associations. When we used in the MLM model the structure based on the STS detected on chromosome 2 (all or a subset excluding the positions with the main effects), the correction was much better (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>), and many associations were no more significant (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>), confirming that many associations could be due to the structure. Nevertheless structure based on STS on chromosome 2 may capture a large part of the LD on that chromosome, and thus exclude interesting associations. Furthermore, the traits studied here have strongly evolved from wild to domesticated forms (as shown by the large part of variation explained by the structure). Correcting for the structure may thus hamper the discovery of relevant loci involved in domestication. For these reasons, we presented results of both models.</p><p id="p-54">The core collection may be efficient in detecting polymorphisms with large effects on trait variation, but it will suffer a decrease of statistical power when dealing with low effect variants. A larger collection is necessary to map such genes with low effect. A higher power may be achieved by increasing the sample size rather than by increasing the number of polymorphisms (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Long and Langley 1999</a>). The density of markers needed for association analysis is estimated by LD decay over genetic or physical distance (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Rafalski 2002</a>). An <em>r<sup>2</sup></em> value of 0.3 indicates a sufficiently strong LD to be useful for association mapping in human studies (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ardlie <em>et al.</em> 2002</a>). In <em>S. l. cerasiforme</em> accessions, LD estimated values decayed below this value within 1 cM. One SNP per cM could thus be valuable for medium resolution genome-wide association. Nevertheless, many associations may not be detected with such low number of markers as for physical distances, even if extreme LD is still found over hundreds of kb, an estimate of LD decay indicates that LD is minimal after 20 kb. With a genome size of 950 Mb, a minimum set of 48,000 markers would thus be necessary to have a physical resolution for genome-wide association in tomato. This is a minimal number, which should be probably doubled to tag common polymorphisms in all regions. To validate these estimations based on LD, we looked at the number of significant associations for different marker densities. As expected, the number of SNPs in association with traits increased with densification of polymorphisms. Significant associations (<em>p</em> &lt; 0.005) were found using a large mapping strategy (1 marker per 5 cM) for FW, but no association was found for LCN or SSC. The density of markers necessary for analysis will thus depend on the trait, the locus targeted, and the population studied. For example, it would not have been possible to physically map the <em>lcn</em>2.1 QTN using only LD because these two SNPs are in complete equilibrium with surrounding polymorphisms, except with TD373, which is located on the physical region of <em>lcn</em>2.1 (<a id="xref-ref-29-7" class="xref-bibr" href="#ref-29">Muños <em>et al.</em> 2011</a>).</p><p id="p-55">Our results suggest that genome admixture of <em>S. l. cerasiforme</em> provides an interesting source of molecular diversity for the domesticated tomato. The design of our core collection was efficient enough to detect associations in all the candidate regions where QTL have been previously mapped. We highlighted the greater efficiency of the K+Q model in dealing with type I error rate even in a relatively small sample. Association mapping validated the polymorphisms discovered by positional cloning (<em>lcn2.1</em> and <em>fw2.2</em>) or fine mapping (<em>fw2.1</em>). The screening of polymorphisms along chromosome 2 with a high marker density allowed the detection of many new associations that were confirmed in a larger sample. We identified several putative new candidate genes. If we extrapolate our results to the whole genome, at least 50,000 SNPs will be necessary for high-resolution mapping in such a collection and the double would be more realistic to avoid SNP with low MAF. Due to the recent advances in next-generation sequencing technologies, the development of genomic tools (<em>i.e.</em> SNP-chip) of high to very high density will allow screening of the whole tomato genome for association with traits of interest.</p></div></div><div class="section" id="sec-18"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-56">We thank Hélène Burck for characterizing and maintaining the INRA tomato Genetic Resources collection. We are grateful to Yolande Carretero, Esther Pelpoir, and Laure David for their help with growing and phenotyping cherry tomato accessions. We thank Aurélie Chauveau and Rémi Bounon for plant DNA sequencing. French INRA AIP BioRessources supported this work. The thesis of Nicolas Ranc, Ph.D., was financially supported by the EUSOL European project PL016214-2. Jiaxin Xu, Ph.D., was financially supported by the Chinese Scientific Council.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002667/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002667/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">Sequence data from this article have been deposited to the European Nucleotide Archive under accession nos. HE805129 to HE805210.</p></li><li class="fn" id="fn-13"><p id="p-57"><em>Communicating editor: D. Zamir</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-03-16"><span class="received-label">Received </span>March 16, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-05-24"><span class="accepted-label">Accepted </span>May 24, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Ranc <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-5">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-other" id="cit-2.8.853.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ardlie</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seielstad</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Patterns of linkage disequilibrium in the human genome</span>. <span class="cit-source">Nat. Rev. Genet.</span> <span class="cit-vol">3:</span> <span class="cit-fpage">299</span>–<span class="cit-lpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D3%253A%26rft.spage%253D299%26rft.atitle%253DPATTERNS%2BOF%2BLINKAGE%2BDISEQUILIBRIUM%2BIN%2BTHE%2BHUMAN%2BGENOME%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.2" data-doi="10.1093/aob/mcm150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindhout</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Domestication and breeding of tomatoes: what have we gained and what can we gain in the future?</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">1085</span>–<span class="cit-lpage">1094</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%2B%2528Lond.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcm150%26rft_id%253Dinfo%253Apmid%252F17717024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjEwOiIxMDAvNS8xMDg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84Lzg1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.3" data-doi="10.3102/10769986025001060"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">On the adaptive control of the false discovery rate in multiple testing with independent statistics</span>. <abbr class="cit-jnl-abbrev">J. Educ. Behav. Stat.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">60</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEduc.%2BBehav.%2BStat.%26rft.volume%253D25%26rft.spage%253D60%26rft_id%253Dinfo%253Adoi%252F10.3102%252F10769986025001060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3102/10769986025001060&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000087561900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.4" data-doi="10.1093/bioinformatics/btm308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kroon</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramdoss</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">TASSEL: software for association mapping of complex traits in diverse samples</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2633</span>–<span class="cit-lpage">2635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm308%26rft_id%253Dinfo%253Apmid%252F17586829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMTkvMjYzMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOC84NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.5" data-doi="10.1016/S1369-5266(02)00238-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edward</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornsberry</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Plant molecular diversity and applications to genomics</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DBuckler%26rft.auinit1%253DE.%2BS.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253D107%26rft.epage%253D111%26rft.atitle%253DPlant%2Bmolecular%2Bdiversity%2Band%2Bapplications%2Bto%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1369-5266%252802%252900238-8%26rft_id%253Dinfo%253Apmid%252F11856604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1369-5266(02)00238-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11856604&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174097000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.6" data-doi="10.1534/genetics.104.038489"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caldwell</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langridge</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Extreme population-dependent linkage disequilibrium detected in an inbreeding plant species, <em>Hordeum vulgare</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">557</span>–<span class="cit-lpage">567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.038489%26rft_id%253Dinfo%253Apmid%252F16219791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcyLzEvNTU3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84Lzg1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.7" data-doi="10.1093/jxb/erf058"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Causse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saliba-Colombani</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lecomte</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duffe</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rousselle</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">QTL analysis of fruit quality in fresh market tomato: a few chromosome regions control the variation of sensory and instrumental traits</span>. <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">2089</span>–<span class="cit-lpage">2098</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BExperimental%2BBotany%26rft.stitle%253DJ%2BExp%2BBot%26rft.aulast%253DCausse%26rft.auinit1%253DM.%26rft.volume%253D53%26rft.issue%253D377%26rft.spage%253D2089%26rft.epage%253D2098%26rft.atitle%253DQTL%2Banalysis%2Bof%2Bfruit%2Bquality%2Bin%2Bfresh%2Bmarket%2Btomato%253A%2Ba%2Bfew%2Bchromosome%2Bregions%2Bcontrol%2Bthe%2Bvariation%2Bof%2Bsensory%2Band%2Binstrumental%2Btraits%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferf058%26rft_id%253Dinfo%253Apmid%252F12324532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4Ym90IjtzOjU6InJlc2lkIjtzOjExOiI1My8zNzcvMjA4OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOC84NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.8" data-doi="10.1038/ng1001-229"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rioux</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffner</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">High-resolution haplotype structure in the human genome</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">229</span>–<span class="cit-lpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDaly%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D29%26rft.issue%253D2%26rft.spage%253D229%26rft.epage%253D232%26rft.atitle%253DHigh-resolution%2Bhaplotype%2Bstructure%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1001-229%26rft_id%253Dinfo%253Apmid%252F11586305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1001-229&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11586305&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171374100033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.9" data-doi="10.1038/386485a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doebley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stec</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubbard</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The evolution of apical dominance in maize</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">386</span>: <span class="cit-fpage">485</span>–<span class="cit-lpage">488</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDoebley%26rft.auinit1%253DJ.%26rft.volume%253D386%26rft.issue%253D6624%26rft.spage%253D485%26rft.epage%253D488%26rft.atitle%253DThe%2Bevolution%2Bof%2Bapical%2Bdominance%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F386485a0%26rft_id%253Dinfo%253Apmid%252F9087405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/386485a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9087405&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WR25600049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.10" data-doi="10.1101/gr.4319006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drouaud</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Variation in crossing-over rates across chromosome4 of <em>Arabidopsis thaliana</em> reveals the presence of meiotic recombination hot spots</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">106</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4319006%26rft_id%253Dinfo%253Apmid%252F16344568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzEvMTA2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84Lzg1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.11" data-doi="10.1111/j.1365-294X.2005.02553.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evanno</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regnaut</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Detecting the number of clusters of individuals using the software structure: a simulation study</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2611</span>–<span class="cit-lpage">2620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DEvanno%26rft.auinit1%253DG.%26rft.volume%253D14%26rft.issue%253D8%26rft.spage%253D2611%26rft.epage%253D2620%26rft.atitle%253DDetecting%2Bthe%2Bnumber%2Bof%2Bclusters%2Bof%2Bindividuals%2Busing%2Bthe%2Bsoftware%2BSTRUCTURE%253A%2Ba%2Bsimulation%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02553.x%26rft_id%253Dinfo%253Apmid%252F15969739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02553.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15969739&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229961500029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.12" data-doi="10.1146/annurev.arplant.54.031902.134907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint-Garcia</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornsberry</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edward</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">I. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Structure of linkage disequilibrium in plants</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BPlant%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.arplant.54.031902.134907%26rft_id%253Dinfo%253Apmid%252F14502995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.arplant.54.031902.134907&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14502995&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185094100014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.13" data-doi="10.1111/j.1365-313X.2005.02591.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint-Garcia</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thuillet</span>  <span class="cit-name-given-names">A.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romero</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Maize association population: a high-resolution platform for quantitative trait locus dissection</span>. <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1054</span>–<span class="cit-lpage">1064</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DFlint-Garcia%26rft.auinit1%253DS.%2BA.%26rft.volume%253D44%26rft.issue%253D6%26rft.spage%253D1054%26rft.epage%253D1064%26rft.atitle%253DMaize%2Bassociation%2Bpopulation%253A%2Ba%2Bhigh-resolution%2Bplatform%2Bfor%2Bquantitative%2Btrait%2Blocus%2Bdissection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2005.02591.x%26rft_id%253Dinfo%253Apmid%252F16359397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2005.02591.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16359397&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233776900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.14" data-doi="10.1126/science.289.5476.85"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frary</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nesbitt</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frary</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grandillo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knaap</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title"><em>fw2.2</em>: a quantitative trait locus key to the evolution of tomato fruit size</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">289</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFrary%26rft.auinit1%253DA.%26rft.volume%253D289%26rft.issue%253D5476%26rft.spage%253D85%26rft.epage%253D88%26rft.atitle%253Dfw2.2%253A%2BA%2BQuantitative%2BTrait%2BLocus%2BKey%2Bto%2Bthe%2BEvolution%2Bof%2BTomato%2BFruit%2BSize%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.289.5476.85%26rft_id%253Dinfo%253Apmid%252F10884229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIyODkvNTQ3Ni84NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOC84NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.15" data-doi="10.1007/s11103-005-0257-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rustgi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulwal</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Linkage disequilibrium and association studies in higher plants: present status and future prospects</span>. <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">461</span>–<span class="cit-lpage">485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DGupta%26rft.auinit1%253DP.%2BK.%26rft.volume%253D57%26rft.issue%253D4%26rft.spage%253D461%26rft.epage%253D485%26rft.atitle%253DLinkage%2Bdisequilibrium%2Band%2Bassociation%2Bstudies%2Bin%2Bhigher%2Bplants%253A%2Bpresent%2Bstatus%2Band%2Bfuture%2Bprospects.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11103-005-0257-z%26rft_id%253Dinfo%253Apmid%252F15821975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11103-005-0257-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15821975&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228284900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.16" data-doi="10.1046/j.1471-8286.2002.00305.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hardy</span>  <span class="cit-name-given-names">O. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vekemans</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">spagedi: a versatile computer program to analyse spatial genetic structure at the individual or population levels</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol. Notes</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">618</span>–<span class="cit-lpage">620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BNotes%26rft.volume%253D2%26rft.spage%253D618%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1471-8286.2002.00305.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1471-8286.2002.00305.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000179678400078&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.17" data-doi="10.1534/genetics.106.069740"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">I.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Highly variable patterns of linkage disequilibrium in multiple soybean populations</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">1937</span>–<span class="cit-lpage">1944</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D175%26rft.spage%253D1937%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.069740%26rft_id%253Dinfo%253Apmid%252F17287533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.106.069740&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17287533&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246448800033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Distinguishing the hitchhiking and background selection models</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">2307</span>–<span class="cit-lpage">2312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D165%26rft.spage%253D2307%26rft_id%253Dinfo%253Apmid%252F14704207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=14704207&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187805300054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.19" data-doi="10.1186/1471-2229-9-85"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jimenez-Gomez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maloof</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sequence diversity in three tomato species: SNPs, markers, and molecular evolution</span>. <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DJimenez-Gomez%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D9%26rft.spage%253D85%26rft.epage%253D85%26rft.atitle%253DSequence%2Bdiversity%2Bin%2Bthree%2Btomato%2Bspecies%253A%2BSNPs%252C%2Bmarkers%252C%2Band%2Bmolecular%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-9-85%26rft_id%253Dinfo%253Apmid%252F19575805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-9-85&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19575805&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.20" data-doi="10.1016/S0092-8674(01)00197-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shibahara</span>  <span class="cit-name-given-names">K.-i.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taoka</span>  <span class="cit-name-given-names">K.-i.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwabuchi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title"><em>FASCIATA</em> genes for chromatin assembly factor-1 in <em>Arabidopsis</em> maintain the cellular organization of apical meristems</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKaya%26rft.auinit1%253DH.%26rft.volume%253D104%26rft.issue%253D1%26rft.spage%253D131%26rft.epage%253D142%26rft.atitle%253DFASCIATA%2Bgenes%2Bfor%2Bchromatin%2Bassembly%2Bfactor-1%2Bin%2Barabidopsis%2Bmaintain%2Bthe%2Bcellular%2Borganization%2Bof%2Bapical%2Bmeristems.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252801%252900197-0%26rft_id%253Dinfo%253Apmid%252F11163246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(01)00197-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11163246&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166882300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.21" data-doi="10.1038/ng2115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plagnol</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toomajian</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Recombination and linkage disequilibrium in <em>Arabidopsis thaliana</em></span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKim%26rft.auinit1%253DS.%26rft.volume%253D39%26rft.issue%253D9%26rft.spage%253D1151%26rft.epage%253D1155%26rft.atitle%253DRecombination%2Band%2Blinkage%2Bdisequilibrium%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2115%26rft_id%253Dinfo%253Apmid%252F17676040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17676040&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.22" data-doi="10.1007/s11032-005-1911-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Labate</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baldo</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Tomato SNP discovery by EST mining and resequencing</span>. <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D16%26rft.spage%253D343%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-005-1911-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-005-1911-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000233739000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-2.8.853.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Labate</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grandillo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muños</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caicedo</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-comment">Tomato, pp. 11–135 in <em>Genome Mapping and Molecular Breeding in Plants</em>, Vol. 5, Vegetables, edited by C. Kole. Springer-Verlag, Berlin/Heidelberg</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.24" data-doi="10.1007/s00122-008-0957-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Labate</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baldo</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">EST, COSII, and arbitrary gene markers give similar estimates of nucleotide diversity in cultivated tomato (<em>Solanum lycopersicum</em> L.)</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">1005</span>–<span class="cit-lpage">1014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLabate%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D118%26rft.issue%253D5%26rft.spage%253D1005%26rft.epage%253D1014%26rft.atitle%253DEST%252C%2BCOSII%252C%2Band%2Barbitrary%2Bgene%2Bmarkers%2Bgive%2Bsimilar%2Bestimates%2Bof%2Bnucleotide%2Bdiversity%2Bin%2Bcultivated%2Btomato%2B%2528Solanum%2Blycopersicum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0957-2%26rft_id%253Dinfo%253Apmid%252F19153710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0957-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19153710&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263900200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lecomte</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saliba-Colombani</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomez-Jimenez</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duffé</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Fine mapping of QTLs of chromosome 2 affecting the fruit architecture and composition of tomato</span>. <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">V13</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253DV13%26rft.spage%253D1%26rft.atitle%253DFINE%2BMAPPING%2BOF%2BQTLS%2BOF%2BCHROMOSOME%2B2%2BAFFECTING%2BTHE%2BFRUIT%2BARCHITECTURE%2BAND%2BCOMPOSITION%2BOF%2BTOMATO%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.26" data-doi="10.1073/pnas.162485999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Eck</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cong</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A new class of regulatory genes underlying the cause of pear-shaped tomato fruit</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">13302</span>–<span class="cit-lpage">13306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.162485999%26rft_id%253Dinfo%253Apmid%252F12242331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTkvMjAvMTMzMDIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzgvODUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The power of association studies to detect the contribution of candidate genetic loci to variation in complex traits</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">720</span>–<span class="cit-lpage">731</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DLong%26rft.auinit1%253DA.%2BD.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D720%26rft.epage%253D731%26rft.atitle%253DThe%2BPower%2Bof%2BAssociation%2BStudies%2Bto%2BDetect%2Bthe%2BContribution%2Bof%2BCandidate%2BGenetic%2BLoci%2Bto%2BVariation%2Bin%2BComplex%2BTraits%26rft_id%253Dinfo%253Apmid%252F10447507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjkvOC83MjAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzgvODUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.28" data-doi="10.1038/ng1841"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manning</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tor</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poole</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A naturally occurring epigenetic mutation in a gene encoding an SBP-box transcription factor inhibits tomato fruit ripening</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">948</span>–<span class="cit-lpage">952</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DManning%26rft.auinit1%253DK.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D948%26rft.epage%253D952%26rft.atitle%253DA%2Bnaturally%2Boccurring%2Bepigenetic%2Bmutation%2Bin%2Ba%2Bgene%2Bencoding%2Ban%2BSBP-box%2Btranscription%2Bfactor%2Binhibits%2Btomato%2Bfruit%2Bripening.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1841%26rft_id%253Dinfo%253Apmid%252F16832354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1841&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16832354&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.29" data-doi="10.1104/pp.111.173997"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muños</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranc</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botton</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bérard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rolland</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Increase in tomato locule number is controlled by two key SNP located near Wuschel</span>. <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">156:</span> <span class="cit-fpage">2244</span>–<span class="cit-lpage">2254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.173997%26rft_id%253Dinfo%253Apmid%252F21673133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTU2LzQvMjI0NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOC84NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nesbitt</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Comparative sequencing in the genus <em>Lycopersicon</em>: implications for the evolution of fruit size in the domestication of cultivated tomatoes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">365</span>–<span class="cit-lpage">379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D162%26rft.spage%253D365%26rft_id%253Dinfo%253Apmid%252F12242247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12242247&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178363400030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Linkage disequilibrium, gene trees and selfing: an ancestral recombination graph with partial self-fertilization</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">923</span>–<span class="cit-lpage">929</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D154%26rft.spage%253D923%26rft_id%253Dinfo%253Apmid%252F10655241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10655241&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085178700037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.32" data-doi="10.1016/S0168-9525(02)02557-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavare</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Linkage disequilibrium: what history has to tell us</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DNordborg%26rft.auinit1%253DM.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D83%26rft.epage%253D90%26rft.atitle%253DLinkage%2Bdisequilibrium%253A%2Bwhat%2Bhistory%2Bhas%2Bto%2Btell%2Bus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252802%252902557-X%26rft_id%253Dinfo%253Apmid%252F11818140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(02)02557-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11818140&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173596300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.33" data-doi="10.1038/ng813"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borevitz</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergelson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berry</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chory</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The extent of linkage disequilibrium in <em>Arabidopsis thaliana</em></span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">190</span>–<span class="cit-lpage">193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DNordborg%26rft.auinit1%253DM.%26rft.volume%253D30%26rft.issue%253D2%26rft.spage%253D190%26rft.epage%253D193%26rft.atitle%253DThe%2Bextent%2Bof%2Blinkage%2Bdisequilibrium%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng813%26rft_id%253Dinfo%253Apmid%252F11780140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng813&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11780140&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173708700020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paulus</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Köllner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobsen</span>  <span class="cit-name-given-names">H.-J.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Physiological and biochemical characterization of glyoxalase I, a general marker for cell proliferation, from a soybean cell suspension</span>. <abbr class="cit-jnl-abbrev">Planta</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">561</span>–<span class="cit-lpage">566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlanta%26rft.volume%253D189%26rft.spage%253D561%26rft_id%253Dinfo%253Apmid%252F7763577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7763577&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.35" data-doi="10.1038/ng1847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plenge</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinblatt</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shadick</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Principal components analysis corrects for stratification in genome-wide association studies</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">904</span>–<span class="cit-lpage">909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTo%26rft.auinit1%253DM.%2BD.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D904%26rft.epage%253D909%26rft.atitle%253DPrincipal%2Bcomponents%2Banalysis%2Bcorrects%2Bfor%2Bstratification%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1847%26rft_id%253Dinfo%253Apmid%252F16862161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16862161&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span><span class="cit-comment">a</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D155%26rft.spage%253D945%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10835412&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087475100039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.37" data-doi="10.1086/302959"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span><span class="cit-comment">b</span> <span class="cit-article-title">Association mapping in structured populations</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">170</span>–<span class="cit-lpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D67%26rft.issue%253D1%26rft.spage%253D170%26rft.epage%253D181%26rft.atitle%253DAssociation%2Bmapping%2Bin%2Bstructured%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F302959%26rft_id%253Dinfo%253Apmid%252F10827107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/302959&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10827107&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088926900019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prudent</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertin</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Génard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muños</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rolland</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genotype-dependent response to carbon availability in growing tomato fruit</span>. <abbr class="cit-jnl-abbrev">Plant Cell Env.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1186</span>–<span class="cit-lpage">1204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%252C%2BCell%252C%2Band%2BEnvironment%2B%2528Print%2529%26rft.stitle%253DPlant%252C%2BCell%252C%2Band%2BEnvironment%2B%2528Print%2529%26rft.aulast%253DPrudent%26rft.auinit1%253DM.%26rft.volume%253D33%26rft.issue%253D7%26rft.spage%253D1186%26rft.epage%253D1204%26rft.atitle%253DGenotype-dependent%2Bresponse%2Bto%2Bcarbon%2Bavailability%2Bin%2Bgrowing%2Btomato%2Bfruit.%26rft_id%253Dinfo%253Apmid%252F20199617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20199617&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278624200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-other" id="cit-2.8.853.39"><div class="cit-metadata"><cite><span class="cit-comment">R Development Core Team, 2005 <em>R: A Language and Environment for Statistical Computing</em>, reference index version 2.2.1., edited by F. F. S. Computing, Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.40" data-doi="10.1016/S1369-5266(02)00240-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rafalski</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Applications of single nucleotide polymorphisms in crop genetics</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">94</span>–<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DRafalski%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253D94%26rft.epage%253D100%26rft.atitle%253DApplications%2Bof%2Bsingle%2Bnucleotide%2Bpolymorphisms%2Bin%2Bcrop%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1369-5266%252802%252900240-6%26rft_id%253Dinfo%253Apmid%252F11856602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1369-5266(02)00240-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11856602&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174097000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.41" data-doi="10.1186/1471-2229-8-130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ranc</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muños</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santoni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Causse</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A clarified position for <em>solanum lycopersicum</em> var. <em>cerasiforme</em> in the evolutionary history of tomatoes (solanaceae)</span>. <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DRanc%26rft.auinit1%253DN.%26rft.volume%253D8%26rft.spage%253D130%26rft.epage%253D130%26rft.atitle%253DA%2Bclarified%2Bposition%2Bfor%2BSolanum%2Blycopersicum%2Bvar.%2Bcerasiforme%2Bin%2Bthe%2Bevolutionary%2Bhistory%2Bof%2Btomatoes%2B%2528solanaceae%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-8-130%26rft_id%253Dinfo%253Apmid%252F19099601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-8-130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19099601&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.42" data-doi="10.1017/S0016672300033620"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ritland</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Estimators for pairwise relatedness and individual inbreeding coefficients</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D67%26rft.spage%253D175%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300033620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300033620&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996UK49900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.8.853.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rozen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skaletsky</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Primer 3 on the WWW for general users and for biologist programmers,</span> <abbr class="cit-jnl-abbrev">pp. 365–386 in Bioinformatics Methods and Protocols: Methods in Molecular Biology, edited by S. Krawetz and S. Misener. Humana Press, Totowa, NJ</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saliba-Colombani</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Causse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langlois</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Philouze</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buret</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Genetic analysis of organoleptic quality in fresh market tomato. 1. Mapping QTLs for physical and chemical traits</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">V102</span>: <span class="cit-fpage">259</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253DV102%26rft.spage%253D259%26rft.atitle%253DGENETIC%2BANALYSIS%2BOF%2BORGANOLEPTIC%2BQUALITY%2BIN%2BFRESH%2BMARKET%2BTOMATO.%2B1.%2BMAPPING%2BQTLS%2BFOR%2BPHYSICAL%2BAND%2BCHEMICAL%2BTRAITS%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.45" data-doi="10.1142/S021972000300006X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsuda</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Margetic</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lathrop</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Automated identification of single nucleotide polymorphisms from sequencing data</span>. <abbr class="cit-jnl-abbrev">J. Bioinform. Comput. Biol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">253</span>–<span class="cit-lpage">265</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbioinformatics%2Band%2Bcomputational%2Bbiology%26rft.stitle%253DJ%2BBioinform%2BComput%2BBiol%26rft.aulast%253DTakahashi%26rft.auinit1%253DM.%26rft.volume%253D1%26rft.issue%253D2%26rft.spage%253D253%26rft.epage%253D265%26rft.atitle%253DAutomated%2Bidentification%2Bof%2Bsingle%2Bnucleotide%2Bpolymorphisms%2Bfrom%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1142%252FS021972000300006X%26rft_id%253Dinfo%253Apmid%252F15290772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1142/S021972000300006X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15290772&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prince</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de-Vicente</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonierbale</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">High density molecular linkage maps of the tomato and potato genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">1141</span>–<span class="cit-lpage">1160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D132%26rft.spage%253D1141%26rft_id%253Dinfo%253Apmid%252F1360934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=1360934&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JZ83200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.47" data-doi="10.1093/hmg/ddh001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tenesa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carothers</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayward</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Extent of linkage disequilibrium in a Sardinian sub-isolate: sampling and methodological considerations</span>. <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddh001%26rft_id%253Dinfo%253Apmid%252F14613964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjc6IjEzLzEvMjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzgvODUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.48" data-doi="10.1038/90135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornsberry</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodman</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doebley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kresovich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Dwarf8 polymorphisms associate with variation in flowering time</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">286</span>–<span class="cit-lpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DThornsberry%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D28%26rft.issue%253D3%26rft.spage%253D286%26rft.epage%253D289%26rft.atitle%253DDwarf8%2Bpolymorphisms%2Bassociate%2Bwith%2Bvariation%2Bin%2Bflowering%2Btime.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F90135%26rft_id%253Dinfo%253Apmid%252F11431702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/90135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11431702&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169656400024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.49" data-doi="10.1007/s00122-008-0755-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Berloo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ursem</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbakel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gort</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Diversity and linkage disequilibrium analysis within a selected set of cultivated tomatoes</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253Dvan%2BBerloo%26rft.auinit1%253DR.%26rft.volume%253D117%26rft.issue%253D1%26rft.spage%253D89%26rft.epage%253D101%26rft.atitle%253DDiversity%2Band%2Blinkage%2Bdisequilibrium%2Banalysis%2Bwithin%2Ba%2Bselected%2Bset%2Bof%2Bcultivated%2Btomatoes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0755-x%26rft_id%253Dinfo%253Apmid%252F18389208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0755-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18389208&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256260900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.50" data-doi="10.1186/1471-2164-8-465"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Deynze</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buell</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Diversity in conserved genes in tomato</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DVan%2BDeynze%26rft.auinit1%253DA.%2BE.%26rft.volume%253D8%26rft.spage%253D465%26rft.epage%253D465%26rft.atitle%253DDiversity%2Bin%2Bconserved%2Bgenes%2Bin%2Btomato.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-8-465%26rft_id%253Dinfo%253Apmid%252F18088428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-8-465&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18088428&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.51" data-doi="10.1016/S1369-5266(00)00206-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verbsky</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Chromatin remodeling in plants</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">494</span>–<span class="cit-lpage">500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DVerbsky%26rft.auinit1%253DM.%2BL.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253D494%26rft.epage%253D500%26rft.atitle%253DChromatin%2Bremodeling%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1369-5266%252800%252900206-5%26rft_id%253Dinfo%253Apmid%252F11641064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1369-5266(00)00206-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11641064&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171822700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.52" data-doi="10.1534/genetics.108.090134"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rucker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baltazar</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Jesus Sanchez-Gonzalez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The genetic architecture of complex traits in teosinte (<em>Zea mays ssp. parviglumis</em>): new evidence from association mapping</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">1221</span>–<span class="cit-lpage">1232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D180%26rft.spage%253D1221%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.090134%26rft_id%253Dinfo%253Apmid%252F18791250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.108.090134&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18791250&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260284400042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kabelka</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eaton</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamoun</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Discovery of single nucleotide polymorphisms in Lycopersicon esculentum by computer aided analysis of expressed sequence tags</span>. <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">V14</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253DV14%26rft.spage%253D21%26rft.atitle%253DDISCOVERY%2BOF%2BSINGLE%2BNUCLEOTIDE%2BPOLYMORPHISMS%2BIN%2BLYCOPERSICON%2BESCULENTUM%2BBY%2BCOMPUTER%2BAIDED%2BANALYSIS%2BOF%2BEXPRESSED%2BSEQUENCE%2BTAGS%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.54" data-doi="10.1038/ng1702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vroh Bi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D203%26rft.epage%253D208%26rft.atitle%253DA%2Bunified%2Bmixed-model%2Bmethod%2Bfor%2Bassociation%2Bmapping%2Bthat%2Baccounts%2Bfor%2Bmultiple%2Blevels%2Bof%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1702%26rft_id%253Dinfo%253Apmid%252F16380716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16380716&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234953200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.55" data-doi="10.1007/s00122-009-1015-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchand</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tinker</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belzile</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population structure and linkage disequilibrium in barley assessed by DArT markers</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DL.%2BY.%26rft.volume%253D119%26rft.issue%253D1%26rft.spage%253D43%26rft.epage%253D52%26rft.atitle%253DPopulation%2Bstructure%2Band%2Blinkage%2Bdisequilibrium%2Bin%2Bbarley%2Bassessed%2Bby%2BDArT%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1015-4%26rft_id%253Dinfo%253Apmid%252F19343317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1015-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19343317&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266666500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.853.56" data-doi="10.1371/journal.pgen.0030004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aranzana</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lister</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shindo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">An <em>Arabidopsis</em> example of association mapping in structured samples</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DZhao%26rft.auinit1%253DK.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253De4%26rft.epage%253De4%26rft.atitle%253DAn%2BArabidopsis%2Bexample%2Bof%2Bassociation%2Bmapping%2Bin%2Bstructured%2Bsamples.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030004%26rft_id%253Dinfo%253Apmid%252F17238287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17238287&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.8.853.57" data-doi="10.3835/plantgenome2008.02.0089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Status and prospects of association mapping in plants</span>. <abbr class="cit-jnl-abbrev">The Plant Genome</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BGenome%26rft.stitle%253DThe%2BPlant%2BGenome%26rft.aulast%253DZhu%26rft.auinit1%253DC.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D5%26rft.epage%253D20%26rft.atitle%253DStatus%2Band%2BProspects%2Bof%2BAssociation%2BMapping%2Bin%2BPlants%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2008.02.0089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2008.02.0089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/8/853.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/8/843" title="Expression Profiling of Attenuated Mitochondrial Function Identifies Retrograde Signals in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/8/865" title="Proper Cyclin B3 Dosage Is Important for Precision of Metaphase-to-Anaphase Onset Timing in Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/8"><span>Volume 2 Issue 8, August 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/8.cover-source.jpg?itok=etPFDPKt" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/8/853" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jpkzexUev7euUxbNFL6widuKvpwra1HdDMR6xvPsTR4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/8/853" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/8/853" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/8/853" class="active" data-icon-position="" data-hide-link-title="0">Genome-Wide Association Mapping in Tomato (Solanum lycopersicum) Is Possible Using Genome Admixture of Solanum lycopersicum var. cerasiforme</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459304" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-joqVgLdeGpJfjP7nIIqUQ0jme51KH4KhGj5fRZrPJJo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/8/853.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/8/853" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-_dIv5MN01biLxXWt2sZWOgNdJ5KFcuxjZobfUoy7LMA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/8/853" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/8/853.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459304" id="node459304--2" data-pisa="ggg;2/8/853" data-pisa-master="ggg;2/8/853" data-apath="/ggg/2/8/853.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Association Mapping in Tomato (<em>Solanum lycopersicum</em>) Is Possible Using Genome Admixture of <em>Solanum lycopersicum</em> var. <em>cerasiforme</em></h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nicolas Ranc</span>, <span class="highwire-citation-author" data-delta="1">Stephane Muños</span>, <span class="highwire-citation-author" data-delta="2">Jiaxin Xu</span>, <span class="highwire-citation-author" data-delta="3">Marie-Christine Le Paslier</span>, <span class="highwire-citation-author" data-delta="4">Aurélie Chauveau</span>, <span class="highwire-citation-author" data-delta="5">Rémi Bounon</span>, <span class="highwire-citation-author" data-delta="6">Sophie Rolland</span>, <span class="highwire-citation-author" data-delta="7">Jean-Paul Bouchet</span>, <span class="highwire-citation-author" data-delta="8">Dominique Brunel</span> and <span class="highwire-citation-author" data-delta="9">Mathilde Causse</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">853-864;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002667 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459304--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicolas Ranc</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BRanc" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ranc%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicolas%2BRanc" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Stephane Muños</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephane%2BMu%C3%B1os" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mu%C3%B1os%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephane%2BMu%25C3%25B1os" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jiaxin Xu</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France</span><span class="nlm-aff">Northwest A&amp;F University, College of Horticulture, Yang Ling, Shaanxin, 712100, People's Republic of China, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiaxin%2BXu" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiaxin%2BXu" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marie-Christine Le Paslier</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1279, Unité Etude du Polymorphisme des Génomes Végétaux, CEA-Institut de Génomique-CNG, Evry, 91057, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-Christine%2BLe%2BPaslier" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Le%20Paslier%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie-Christine%2BLe%2BPaslier" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Aurélie Chauveau</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1279, Unité Etude du Polymorphisme des Génomes Végétaux, CEA-Institut de Génomique-CNG, Evry, 91057, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aur%C3%A9lie%2BChauveau" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chauveau%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAur%25C3%25A9lie%2BChauveau" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Rémi Bounon</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1279, Unité Etude du Polymorphisme des Génomes Végétaux, CEA-Institut de Génomique-CNG, Evry, 91057, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R%C3%A9mi%2BBounon" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bounon%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR%25C3%25A9mi%2BBounon" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Sophie Rolland</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sophie%2BRolland" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rolland%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASophie%2BRolland" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jean-Paul Bouchet</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Paul%2BBouchet" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bouchet%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Paul%2BBouchet" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Dominique Brunel</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1279, Unité Etude du Polymorphisme des Génomes Végétaux, CEA-Institut de Génomique-CNG, Evry, 91057, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominique%2BBrunel" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brunel%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADominique%2BBrunel" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Mathilde Causse</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mathilde%2BCausse" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Causse%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMathilde%2BCausse" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Mathilde.Causse@avignon.inra.fr" class="" data-icon-position="" data-hide-link-title="0">Mathilde.Causse@avignon.inra.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/8/853" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/8/853&amp;title=Genome-Wide%20Association%20Mapping%20in%20Tomato%20%28Solanum%20lycopersicum%29%20Is%20Possible%20Using%20Genome%20Admixture%20of%20Solanum%20lycopersicum%20var.%20cerasiforme" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/8/853&amp;title=Genome-Wide%20Association%20Mapping%20in%20Tomato%20%28Solanum%20lycopersicum%29%20Is%20Possible%20Using%20Genome%20Admixture%20of%20Solanum%20lycopersicum%20var.%20cerasiforme" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/8/853&amp;title=Genome-Wide%20Association%20Mapping%20in%20Tomato%20%28Solanum%20lycopersicum%29%20Is%20Possible%20Using%20Genome%20Admixture%20of%20Solanum%20lycopersicum%20var.%20cerasiforme" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/8/853&amp;text=Genome-Wide%20Association%20Mapping%20in%20Tomato%20%28Solanum%20lycopersicum%29%20Is%20Possible%20Using%20Genome%20Admixture%20of%20Solanum%20lycopersicum%20var.%20cerasiforme" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/8/853&amp;title=Genome-Wide%20Association%20Mapping%20in%20Tomato%20%28Solanum%20lycopersicum%29%20Is%20Possible%20Using%20Genome%20Admixture%20of%20Solanum%20lycopersicum%20var.%20cerasiforme" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/8/853&amp;t=Genome-Wide%20Association%20Mapping%20in%20Tomato%20%28Solanum%20lycopersicum%29%20Is%20Possible%20Using%20Genome%20Admixture%20of%20Solanum%20lycopersicum%20var.%20cerasiforme" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/8/853&amp;title=Genome-Wide%20Association%20Mapping%20in%20Tomato%20%28Solanum%20lycopersicum%29%20Is%20Possible%20Using%20Genome%20Admixture%20of%20Solanum%20lycopersicum%20var.%20cerasiforme" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/8/853&amp;title=Genome-Wide%20Association%20Mapping%20in%20Tomato%20%28Solanum%20lycopersicum%29%20Is%20Possible%20Using%20Genome%20Admixture%20of%20Solanum%20lycopersicum%20var.%20cerasiforme" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459304/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459304" id="node459304--2" data-pisa="ggg;2/8/853" data-pisa-master="ggg;2/8/853" data-apath="/ggg/2/8/853.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Association Mapping in Tomato (<em>Solanum lycopersicum</em>) Is Possible Using Genome Admixture of <em>Solanum lycopersicum</em> var. <em>cerasiforme</em></h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nicolas Ranc</span>, <span class="highwire-citation-author" data-delta="1">Stephane Muños</span>, <span class="highwire-citation-author" data-delta="2">Jiaxin Xu</span>, <span class="highwire-citation-author" data-delta="3">Marie-Christine Le Paslier</span>, <span class="highwire-citation-author" data-delta="4">Aurélie Chauveau</span>, <span class="highwire-citation-author" data-delta="5">Rémi Bounon</span>, <span class="highwire-citation-author" data-delta="6">Sophie Rolland</span>, <span class="highwire-citation-author" data-delta="7">Jean-Paul Bouchet</span>, <span class="highwire-citation-author" data-delta="8">Dominique Brunel</span> and <span class="highwire-citation-author" data-delta="9">Mathilde Causse</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">853-864;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002667 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459304--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicolas Ranc</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BRanc" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ranc%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicolas%2BRanc" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Stephane Muños</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephane%2BMu%C3%B1os" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mu%C3%B1os%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephane%2BMu%25C3%25B1os" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jiaxin Xu</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France</span><span class="nlm-aff">Northwest A&amp;F University, College of Horticulture, Yang Ling, Shaanxin, 712100, People's Republic of China, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiaxin%2BXu" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiaxin%2BXu" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marie-Christine Le Paslier</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1279, Unité Etude du Polymorphisme des Génomes Végétaux, CEA-Institut de Génomique-CNG, Evry, 91057, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-Christine%2BLe%2BPaslier" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Le%20Paslier%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie-Christine%2BLe%2BPaslier" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Aurélie Chauveau</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1279, Unité Etude du Polymorphisme des Génomes Végétaux, CEA-Institut de Génomique-CNG, Evry, 91057, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aur%C3%A9lie%2BChauveau" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chauveau%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAur%25C3%25A9lie%2BChauveau" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Rémi Bounon</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1279, Unité Etude du Polymorphisme des Génomes Végétaux, CEA-Institut de Génomique-CNG, Evry, 91057, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R%C3%A9mi%2BBounon" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bounon%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR%25C3%25A9mi%2BBounon" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Sophie Rolland</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sophie%2BRolland" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rolland%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASophie%2BRolland" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jean-Paul Bouchet</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Paul%2BBouchet" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bouchet%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Paul%2BBouchet" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Dominique Brunel</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1279, Unité Etude du Polymorphisme des Génomes Végétaux, CEA-Institut de Génomique-CNG, Evry, 91057, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominique%2BBrunel" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brunel%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADominique%2BBrunel" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Mathilde Causse</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, UR1052, Unité de Génétique et Amélioration des Fruits et Légumes, Avignon, 84143, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mathilde%2BCausse" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Causse%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMathilde%2BCausse" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Mathilde.Causse@avignon.inra.fr" class="" data-icon-position="" data-hide-link-title="0">Mathilde.Causse@avignon.inra.fr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=853&amp;rft.epage=864&amp;rft.atitle=Genome-Wide%20Association%20Mapping%20in%20Tomato%20%28Solanum%20lycopersicum%29%20Is%20Possible%20Using%20Genome%20Admixture%20of%20Solanum%20lycopersicum%20var.%20cerasiforme&amp;rft.volume=2&amp;rft.issue=8&amp;rft.date=2012-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ranc%2C+Nicolas&amp;rft.au=Mu%C3%B1os%2C+Stephane&amp;rft.au=Xu%2C+Jiaxin&amp;rft.au=Le+Paslier%2C+Marie-Christine&amp;rft.au=Chauveau%2C+Aur%C3%A9lie&amp;rft.au=Bounon%2C+R%C3%A9mi&amp;rft.au=Rolland%2C+Sophie&amp;rft.au=Bouchet%2C+Jean-Paul&amp;rft.au=Brunel%2C+Dominique&amp;rft.au=Causse%2C+Mathilde"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459304/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459304/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459304/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459304/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459304/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459304/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459304/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459304/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459304/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459304/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459304/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459304/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494625" id="node494625" data-pisa="ggg;9/11/3891" data-pisa-master="ggg;g3.119.400723" data-apath="/ggg/9/11/3891.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3891" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Mouse Mutation That Dysregulates Neighboring <em>Galnt17</em> and <em>Auts2</em> Genes Is Associated with Phenotypes Related to the Human AUTS2 Syndrome</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494624" id="node494624" data-pisa="ggg;9/11/3781" data-pisa-master="ggg;g3.119.400647" data-apath="/ggg/9/11/3781.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3781" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Transgenic Analyses in Drosophila Reveal That mCORL1 Is Functionally Distinct from mCORL2 and dCORL</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494622" id="node494622" data-pisa="ggg;9/11/3843" data-pisa-master="ggg;g3.119.400363" data-apath="/ggg/9/11/3843.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3843" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Analysis of Cell Cycle-Regulating Genes in the Symbiotic Dinoflagellate <em>Breviolum minutum</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/8/853" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/8/853.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/8/853.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/8/853.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/8/853.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2019 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
  </body>
</html>
