<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/7/753" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/7/753.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/7/753.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/7/753.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/7/753.atom" />
<meta name="HW.pisa" content="ggg;2/7/753" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Noncomplementation Screen for Quantitative Trait Alleles in Saccharomyces cerevisiae" />
<meta name="DC.Identifier" content="10.1534/g3.112.002550" />
<meta name="DC.Date" content="2012-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Kim et al." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Both linkage and linkage disequilibrium mapping provide well-defined approaches to mapping quantitative trait alleles. However, alleles of small effect are particularly difficult to refine to individual genes and causative mutations. Quantitative noncomplementation provides a means of directly testing individual genes for quantitative trait alleles in a fixed genetic background. Here, we implement a genome-wide noncomplementation screen for quantitative trait alleles that affect colony color or size by using the yeast deletion collection. As proof of principle, we find a previously known allele of [CYS4][1] that affects colony color and a novel allele of [CTT1][2] that affects resistance to hydrogen peroxide. To screen nearly 4700 genes in nine diverse yeast strains, we developed a high-throughput robotic plating assay to quantify colony color and size. Although we found hundreds of candidate alleles, reciprocal hemizygosity analysis of a select subset revealed that many of the candidates were false positives, in part the result of background-dependent haploinsufficiency or second-site mutations within the yeast deletion collection. Our results highlight the difficulty of identifying small-effect alleles but support the use of noncomplementation as a rapid means of identifying quantitative trait alleles of large effect.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003387
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003320" />
<meta name="DC.Contributor" content="Hyun Seok Kim" />
<meta name="DC.Contributor" content="Juyoung Huh" />
<meta name="DC.Contributor" content="Linda Riles" />
<meta name="DC.Contributor" content="Alejandro Reyes" />
<meta name="DC.Contributor" content="Justin C. Fay" />
<meta name="article:published_time" content="2012-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Noncomplementation Screen for Quantitative Trait Alleles in Saccharomyces cerevisiae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Both linkage and linkage disequilibrium mapping provide well-defined approaches to mapping quantitative trait alleles. However, alleles of small effect are particularly difficult to refine to individual genes and causative mutations. Quantitative noncomplementation provides a means of directly testing individual genes for quantitative trait alleles in a fixed genetic background. Here, we implement a genome-wide noncomplementation screen for quantitative trait alleles that affect colony color or size by using the yeast deletion collection. As proof of principle, we find a previously known allele of &lt;i&gt;CYS4&lt;/i&gt; that affects colony color and a novel allele of &lt;i&gt;CTT1&lt;/i&gt; that affects resistance to hydrogen peroxide. To screen nearly 4700 genes in nine diverse yeast strains, we developed a high-throughput robotic plating assay to quantify colony color and size. Although we found hundreds of candidate alleles, reciprocal hemizygosity analysis of a select subset revealed that many of the candidates were false positives, in part the result of background-dependent haploinsufficiency or second-site mutations within the yeast deletion collection. Our results highlight the difficulty of identifying small-effect alleles but support the use of noncomplementation as a rapid means of identifying quantitative trait alleles of large effect.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/07/01" />
<meta name="citation_mjid" content="ggg;2/7/753" />
<meta name="citation_id" content="2/7/753" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/7/753" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/7/753.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/7/753.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/7/753.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.002550" />
<meta name="citation_pmid" content="22870398" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="753" />
<meta name="citation_lastpage" content="760" />
<meta name="citation_author" content="Hyun Seok Kim" />
<meta name="citation_author_institution" content="Department of Genetics" />
<meta name="citation_author_institution" content="Computational Biology Program, and" />
<meta name="citation_author" content="Juyoung Huh" />
<meta name="citation_author_institution" content="Department of Genetics" />
<meta name="citation_author" content="Linda Riles" />
<meta name="citation_author_institution" content="Department of Genetics" />
<meta name="citation_author" content="Alejandro Reyes" />
<meta name="citation_author_institution" content="Computational Biology Program, and" />
<meta name="citation_author" content="Justin C. Fay" />
<meta name="citation_author_institution" content="Department of Genetics" />
<meta name="citation_author_institution" content="Computational Biology Program, and" />
<meta name="citation_author_institution" content="Center for Genome Sciences and Systems Biology, Washington University, St. Louis, Missouri 63108" />
<meta name="citation_author_email" content="jfay@genetics.wustl.edu" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=N. Ahituv;citation_author=N. Kavaslar;citation_author=W. Schackwitz;citation_author=A. Ustaszewska;citation_author=J. Martin;citation_title=Medical sequencing at the extremes of human body mass.;citation_pages=779-791;citation_volume=80;citation_year=2007;citation_issue=4;citation_pmid=17357083;citation_doi=10.1086/513471" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. Altshuler;citation_author=M. J. Daly;citation_author=E. S. Lander;citation_title=Genetic Mapping in Human Disease;citation_pages=881-888;citation_volume=322;citation_year=2008;citation_issue=5903;citation_pmid=18988837;citation_doi=10.1126/science.1156409" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=G. Ben-Ari;citation_author=D. Zenvirth;citation_author=A. Sherman;citation_author=L. David;citation_author=M. Klutstein;citation_title=Four linked genes participate in controlling sporulation efficiency in budding yeast.;citation_pages=e195-e195;citation_volume=2;citation_year=2006;citation_issue=11;citation_pmid=17112318;citation_doi=10.1371/journal.pgen.0020195" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. C. Cohen;citation_author=R. S. Kiss;citation_author=A. Pertsemlidis;citation_author=Y. L. Marcel;citation_author=R. McPherson;citation_title=Multiple Rare Alleles Contribute to Low Plasma Levels of HDL Cholesterol;citation_pages=869-872;citation_volume=305;citation_year=2004;citation_issue=5685;citation_pmid=15297675;citation_doi=10.1126/science.1099870" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. A. Coyne;citation_author=S. Simeonidis;citation_author=P. Rooney;citation_title=Relative Paucity of Genes Causing Inviability in Hybrids Between Drosophila melanogaster and D. simulans;citation_pages=1091-1103;citation_volume=150;citation_year=1998;citation_issue=3;citation_pmid=9799261" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. M. Deutschbauer;citation_author=R. W. Davis;citation_title=Quantitative trait loci mapped to single-nucleotide resolution in yeast.;citation_pages=1333-1340;citation_volume=37;citation_year=2005;citation_issue=12;citation_pmid=16273108;citation_doi=10.1038/ng1674" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Diezmann;citation_author=F. S. Dietrich;citation_title=Oxidative stress survival in a clinical Saccharomyces cerevisiae isolate is influenced by a major quantitative trait nucleotide;citation_pages=709-722;citation_volume=188;citation_year=2011;citation_pmid=21515583;citation_doi=10.1534/genetics.111.128256" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. W. Doniger;citation_author=H. S. Kim;citation_author=D. Swain;citation_author=D. Corcuera;citation_author=M. Williams;citation_title=A catalog of neutral and deleterious polymorphism in yeast.;citation_pages=e1000183-e1000183;citation_volume=4;citation_year=2008;citation_issue=8;citation_pmid=18769710;citation_doi=10.1371/journal.pgen.1000183" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. D. Dowell;citation_author=O. Ryan;citation_author=A. Jansen;citation_author=D. Cheung;citation_author=S. Agarwala;citation_title=Genotype to Phenotype: A Complex Problem;citation_pages=469-469;citation_volume=328;citation_year=2010;citation_issue=5977;citation_pmid=20413493;citation_doi=10.1126/science.1189015" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=N. S. Fearnhead;citation_author=J. L. Wilding;citation_author=B. Winney;citation_author=S. Tonks;citation_author=S. Bartlett;citation_title=Multiple rare variants in different genes account for multifactorial inherited susceptibility to colorectal adenomas;citation_volume=101;citation_year=2004;citation_pmid=15520370;citation_doi=10.1073/pnas.0407187101" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Gerke;citation_author=K. Lorenz;citation_author=B. Cohen;citation_title=Genetic Interactions Between Transcription Factors Cause Natural Variation in Yeast;citation_pages=498-501;citation_volume=323;citation_year=2009;citation_issue=5913;citation_pmid=19164747;citation_doi=10.1126/science.1166426" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. Gerke;citation_author=K. Lorenz;citation_author=S. Ramnarine;citation_author=B. Cohen;citation_title=Gene-environment interactions at nucleotide resolution;citation_pages=e1001144;citation_volume=6;citation_year=2010;citation_pmid=20941394;citation_doi=10.1371/journal.pgen.1001144" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Giaever;citation_author=A. M. Chu;citation_author=L. Ni;citation_author=C. Connelly;citation_author=L. Riles;citation_title=Functional profiling of the Saccharomyces cerevisiae genome.;citation_pages=387-391;citation_volume=418;citation_year=2002;citation_issue=6896;citation_pmid=12140549;citation_doi=10.1038/nature00935" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=A. L. Goldstein;citation_author=J. H. McCusker;citation_title=Three new dominant drug resistance cassettes for gene disruption in Saccharomyces cerevisiae.;citation_pages=1541-1553;citation_volume=15;citation_year=1999;citation_issue=14;citation_pmid=10514571;citation_doi=10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K" />
<meta name="citation_reference" content="citation_journal_title=Biochemical and biophysical research communications;citation_journal_abbrev=Biochem Biophys Res Commun;citation_author=C. M. Grant;citation_author=G. Perrone;citation_author=I. W. Dawes;citation_title=Glutathione and catalase provide overlapping defenses for protection against hydrogen peroxide in the yeast Saccharomyces cerevisiae.;citation_pages=893-898;citation_volume=253;citation_year=1998;citation_issue=3;citation_pmid=9918826;citation_doi=10.1006/bbrc.1998.9864" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=B. Grünenfelder;citation_author=E. A. Winzeler;citation_title=Treasures and traps in genome-wide data sets: case examples from yeast.;citation_pages=653-661;citation_volume=3;citation_year=2002;citation_issue=9;citation_pmid=12209140;citation_doi=10.1038/nrg886" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. E. Hillenmeyer;citation_author=E. Fung;citation_author=J. Wildenhain;citation_author=S. E. Pierce;citation_author=S. Hoon;citation_title=The Chemical Genomic Portrait of Yeast: Uncovering a Phenotype for All Genes;citation_pages=362-365;citation_volume=320;citation_year=2008;citation_issue=5874;citation_pmid=18420932;citation_doi=10.1126/science.1150021" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=T. R. Hughes;citation_author=C. J. Roberts;citation_author=H. Dai;citation_author=A. R. Jones;citation_author=M. R. Meyer;citation_title=Widespread aneuploidy revealed by DNA microarray expression profiling.;citation_pages=333-337;citation_volume=25;citation_year=2000;citation_issue=3;citation_pmid=10888885;citation_doi=10.1038/77116" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. S. Kim;citation_author=J. C. Fay;citation_title=Genetic variation in the cysteine biosynthesis pathway causes sensitivity to pharmacological compounds;citation_volume=104;citation_year=2007;citation_pmid=18042712;citation_doi=10.1073/pnas.0708194104" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=H. S. Kim;citation_author=J. Huh;citation_author=J. C. Fay;citation_title=Dissecting the pleiotropic consequences of a quantitative trait nucleotide;citation_volume=9;citation_year=2009;citation_pmid=19456872;citation_doi=10.1111/j.1567-1364.2009.00516.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Li;citation_author=M. A. Lyons;citation_author=H. Wittenburg;citation_author=B. Paigen;citation_author=G. A. Churchill;citation_title=Combining data from multiple inbred line crosses improves the power and resolution of quantitative trait loci mapping;citation_pages=1699-1709;citation_volume=169;citation_year=2005;citation_pmid=15654110" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=A. L. Linderholm;citation_author=C. L. Findleton;citation_author=G. Kumar;citation_author=Y. Hong;citation_author=L. F. Bisson;citation_title=Identification of genes affecting hydrogen sulfide formation in Saccharomyces cerevisiae;citation_volume=74;citation_year=2008;citation_pmid=18192430;citation_doi=10.1128/AEM.01758-07" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Enology and Viticulture;citation_journal_abbrev=Am. J. Enol. Vitic.;citation_author=A. L. Linderholm;citation_author=T. L. Olineka;citation_author=Y. Hong;citation_author=L. F. Bisson;citation_title=Allele Diversity among Genes of the Sulfate Reduction Pathway in Wine Strains of Saccharomyces cerevisiae;citation_pages=431-440;citation_volume=57;citation_year=2006;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Liti;citation_author=D. M. Carter;citation_author=A. M. Moses;citation_author=J. Warringer;citation_author=L. Parts;citation_title=Population genomics of domestic and wild yeasts.;citation_pages=337-341;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212322;citation_doi=10.1038/nature07743" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=T. F. C. Mackay;citation_title=The genetic architecture of quantitative traits: lessons from Drosophila.;citation_pages=253-257;citation_volume=14;citation_year=2004;citation_issue=3;citation_pmid=15172667;citation_doi=10.1016/j.gde.2004.04.003" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=R. K. Mortimer;citation_title=Evolution and Variation of the Yeast (Saccharomyces) Genome;citation_pages=403-409;citation_volume=10;citation_year=2000;citation_issue=4;citation_pmid=10779481;citation_doi=10.1101/gr.10.4.403" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. O. Perlstein;citation_author=D. M. Ruderfer;citation_author=D. C. Roberts;citation_author=S. L. Schreiber;citation_author=L. Kruglyak;citation_title=Genetic basis of individual differences in the response to small-molecule drugs in yeast.;citation_pages=496-502;citation_volume=39;citation_year=2007;citation_issue=4;citation_pmid=17334364;citation_doi=10.1038/ng1991" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. K. Pritchard;citation_title=Are rare variants responsible for susceptibility to complex diseases?;citation_pages=124-137;citation_volume=69;citation_year=2001;citation_issue=1;citation_pmid=11404818;citation_doi=10.1086/321272" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=M. V. Rockman;citation_title=The QTN program and the alleles that matter for evolution: all that's gold does not glitter.;citation_pages=1-17;citation_volume=66;citation_year=2012;citation_issue=1;citation_pmid=22220860;citation_doi=10.1111/j.1558-5646.2011.01486.x" />
<meta name="citation_reference" content="citation_journal_title=Biotechniques;citation_author=A. I. Saeed;citation_author=V. Sharov;citation_author=J. White;citation_author=J. Li;citation_author=W. Liang;citation_title=TM4: a free, open-source system for microarray data management and analysis;citation_pages=374-378;citation_volume=34;citation_year=2003;citation_pmid=12613259" />
<meta name="citation_reference" content="citation_journal_title=Science's SAGE KE;citation_journal_abbrev=SAGE KE;citation_author=P. M. Service;citation_title=How Good Are Quantitative Complementation Tests?;citation_pages=pe13-pe13;citation_volume=2004;citation_year=2004;citation_issue=12;citation_pmid=15044758;citation_doi=10.1126/sageke.2004.12.pe13" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=H. Sinha;citation_author=B. P. Nicholson;citation_author=L. M. Steinmetz;citation_author=J. H. McCusker;citation_title=Complex genetic interactions in a quantitative trait locus.;citation_pages=e13-e13;citation_volume=2;citation_year=2006;citation_issue=2;citation_pmid=16462944;citation_doi=10.1371/journal.pgen.0020013" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=L. M. Steinmetz;citation_author=H. Sinha;citation_author=D. R. Richards;citation_author=J. I. Spiegelman;citation_author=P. J. Oefner;citation_title=Dissecting the architecture of a quantitative trait locus in yeast.;citation_pages=326-330;citation_volume=416;citation_year=2002;citation_issue=6878;citation_pmid=11907579;citation_doi=10.1038/416326a" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=K. H. Takahashi;citation_author=Y. Okada;citation_author=K. Teramura;citation_title=Genome-wide deficiency mapping of the regions responsible for temporal canalization of the developmental processes of Drosophila melanogaster;citation_volume=102;citation_year=2011;citation_pmid=21525178;citation_doi=10.1093/jhered/esr026" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=M. K. Vorachek-Warren;citation_author=J. H. McCusker;citation_title=DsdA (D-serine deaminase): a new heterologous MX cassette for gene disruption and selection in Saccharomyces cerevisiae.;citation_pages=163-171;citation_volume=21;citation_year=2004;citation_issue=2;citation_pmid=14755641;citation_doi=10.1002/yea.1074" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=A. Wach;citation_author=A. Brachat;citation_author=R. Pöhlmann;citation_author=P. Philippsen;citation_title=New heterologous modules for classical or PCR-based gene disruptions in Saccharomyces cerevisiae.;citation_pages=1793-1808;citation_volume=10;citation_year=1994;citation_issue=13;citation_pmid=7747518;citation_doi=10.1002/yea.320101310" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=W. Y. S. Wang;citation_author=B. J. Barratt;citation_author=D. G. Clayton;citation_author=J. A. Todd;citation_title=Genome-wide association studies: theoretical and practical concerns.;citation_pages=109-118;citation_volume=6;citation_year=2005;citation_issue=2;citation_pmid=15716907;citation_doi=10.1038/nrg1522" />
<meta name="twitter:title" content="A Noncomplementation Screen for Quantitative Trait Alleles in Saccharomyces cerevisiae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7.cover-source.jpg" />
<meta name="twitter:description" content="Both linkage and linkage disequilibrium mapping provide well-defined approaches to mapping quantitative trait alleles. However, alleles of small effect are particularly difficult to refine to individual genes and causative mutations. Quantitative noncomplementation provides a means of directly testing individual genes for quantitative trait alleles in a fixed genetic background. Here, we implement a genome-wide noncomplementation screen for quantitative trait alleles that affect colony color or size by using the yeast deletion collection. As proof of principle, we find a previously known allele of [CYS4][1] that affects colony color and a novel allele of [CTT1][2] that affects resistance to hydrogen peroxide. To screen nearly 4700 genes in nine diverse yeast strains, we developed a high-throughput robotic plating assay to quantify colony color and size. Although we found hundreds of candidate alleles, reciprocal hemizygosity analysis of a select subset revealed that many of the candidates were false positives, in part the result of background-dependent haploinsufficiency or second-site mutations within the yeast deletion collection. Our results highlight the difficulty of identifying small-effect alleles but support the use of noncomplementation as a rapid means of identifying quantitative trait alleles of large effect.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003387
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003320" />
<meta name="og-title" property="og:title" content="A Noncomplementation Screen for Quantitative Trait Alleles in Saccharomyces cerevisiae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/7/753" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Both linkage and linkage disequilibrium mapping provide well-defined approaches to mapping quantitative trait alleles. However, alleles of small effect are particularly difficult to refine to individual genes and causative mutations. Quantitative noncomplementation provides a means of directly testing individual genes for quantitative trait alleles in a fixed genetic background. Here, we implement a genome-wide noncomplementation screen for quantitative trait alleles that affect colony color or size by using the yeast deletion collection. As proof of principle, we find a previously known allele of [CYS4][1] that affects colony color and a novel allele of [CTT1][2] that affects resistance to hydrogen peroxide. To screen nearly 4700 genes in nine diverse yeast strains, we developed a high-throughput robotic plating assay to quantify colony color and size. Although we found hundreds of candidate alleles, reciprocal hemizygosity analysis of a select subset revealed that many of the candidates were false positives, in part the result of background-dependent haploinsufficiency or second-site mutations within the yeast deletion collection. Our results highlight the difficulty of identifying small-effect alleles but support the use of noncomplementation as a rapid means of identifying quantitative trait alleles of large effect.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003387
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003320" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/7.cover-source.jpg" />
    <title>A Noncomplementation Screen for Quantitative Trait Alleles in Saccharomyces cerevisiae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q0wst1" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__KPgiwEdjo5O9HdRryHqQs1TpsV-PeBb9B7sGBvwM9FQ__opzZWgPNgQzoEgfATRS5qCKwBw_6XcNjDHoDVOzQv8M__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__YUqwBlv6bNoCToiBWgAUSPtppdS8sbGfzmAH3qEDcI0__Mvk93h3ScFH5Xi9WhibVKlf368fHw-V7HWqWX2QMvtw__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XNWYaQubFGWYWZNytUHkWs-b34LV21DsB4s_6gchI4k__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__7cpBA0GVdrMqkZGGZEW-_SPF2HrbTCmVg9-lrP5l_AQ__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__TV2D4R18tPiSIOmeQ3vyMqHObN_FDmQ6EEIrfE5sAoA__c9BBZFy1BrgGgF_Auu9Fmysb3Z7J56eJGYvMcLQrLks__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q0wst1"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__OaHnYyci68QTwctdRYR30kEfu0eYWBrLyMqy8THv5Ik__IyfJQYM1q1QEPwYsK_Eslh2RQkxpy6yo-nR0p4DWPQQ__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__fhq_kmKS3zjIxlsAa3GE1ubLajyXUgXXTXULzAtiqfg__Lm1_Ls-bdpbWRyl0q4o0mmHyxB-139AJ68dE4zOskLQ__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"IRAkflyjPLm3zEQ0ByK-84XQpJZjQ6gNq_GRjDVEUP8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459294","apath":"\/ggg\/2\/7\/753.atom","pisa":"ggg;2\/7\/753","ac":{"\/ggg\/2\/7\/753.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/7\/753.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/7\/753"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/7\/753"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/7\/753.abstract":true,"\/content\/2\/7\/753":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+wVzVT2ubiEsAaaNzndtg2vefoNN7S0uI6D6URkKAo4RP25GiHGGNDST7xzLvz8Ye0MA+SpkxPas81TYWjyWJL9cb7sfCAq+u1zLDRPoWByUjjAyEx1V\\\\\\\/M1aOB5jlkhEsGeGANMJvKOniNh9VhMkDxCK0q8vYkeLEkpUi0QLrjmV6SzpbiJ5qbSRm0JF2aT5i4St2uTq4fkWnXgcR74W01IDvWaV7VxNMK50tw7IY\\\\\\\/rzax0E8Ef6x9RCM8zRWJq9xc1UNO2XvvqVCvOq8LAFFm4VF03M8lZ3XgnU1P4IcIFmGnb0qBXrcE3cBFRzRgoBum\\\\\\\/WK48Tqj0MzLIxknUTBkftP76kJLhNkHueihdajWyZ23j54N75G9XjJMY+TE18W4BUcm09F1lFZled2vkm3UInYbMR7gIBhpJFqwtOaApgYn5D4IsQsDwJe0Smz\\\\\\\/UnSAEIw+2TcLmfY2Mz+5lbkdrzkDzLlW4R2ypb78TnOvilPM\\\\\\\/PrTl+rR2zX5I5yfdi9qIikjzw9EsLnz5IuHA8NVXWuEwGtHtoo4ajalDjXT7ag+vsvyM8zccj7Gvb2kmEftAsU31VVRyVkayIFG8sqZ0efC\\\\\\\/dQ\\\\\\\/VhJmJ+z8TyQFAawiyB4lfrwTJjatEFqfAPvAJLuGgmvdejcKyavui4jt3lJ1vkX4MoyfT+TB6yEEDccSIlsq\\\\\\\/tY1HnfXGqTOCCLgl7OdeBBh1Pb2OkJmqChAT7YfA3s7GxeYay8fV040TOBKljL7t9QeXCXrtsGeqA3G\\\\\\\/1SRBYUISzxmj1L0Gt7nLVk0Tu07mnUI8YyLevVkWlAoCyY8DBQd3zn67IwRog+GJ+gRQXGxam9JW3pn4J2TyrIfhSBF2TzP0ZGg0FjnsswIoNaBVQnIElLz7FovU8n749xsz51UF7CJGU58F7rqMsmlJ4Zs4KtZXSXngojf\\\\\\\/qR4kS0PdhDZeh0v359kaC0kN7+6f67cByKq6bxyDr6\\\\\\\/X7MJc5Fy\\\\\\\/WtJV5W06lh+A4Jyo5FTxxf2Uh8x9x4aVO53L6V3vZUKrKCwjeeycCxhSebhmHHVqpHmHYzuTp3+I4x\\\\\\\/gbi7EbJOu9tQB9uY7tE\\\\\\\/MXEFxKe1ioJaER2xP+eg+PKtWuw37vg63OCRSHmI+wz1+Bzut2IZISUpgVtOxXEEPzUCxvENGduGYYZiFKeV9INs3wZJXZ7hIDgdBGylLarFgyZa1LUUEl8z434IWuu64a96UXVkVutplagfYcMYEhA0en65gHd6olhPRsdMHggxI71dI28wmClS03wUtv\\\u0022,\\\u0022iv\\\u0022:\\\u0022skLPG1mOca+bmN1hwBN96w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d79c997120aae3ce5996c4c143253d95465eca689b60e4b366841ce0abd347c1\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022S+Q9nH6fpFpgWDyq1q2UoRLFm55yK7Aj+Hg\\\\\\\/UJfTxtwXQG2oJieaaU2KTeooYjMuLUh8jW7oRu7vdWW5xNktDjNBvJC7NfT+24g0WAwUZvUighUsnRNTfaGbAzQdlExE4C1yjp+F9cbUvodSP5hyW97rQMJ6\\\\\\\/mKPMcflhbnf\\\\\\\/ZpswoQI4bp5+wu\\\\\\\/PcaBXBfCLNbirlFjiBjglexK4yH9p+83pa+v5n6qn8AXAYaRpbX\\\\\\\/xh5\\\\\\\/tQWL7LY3Se7Y+k6Izvhec4AvC4sXEd\\\\\\\/enRcV3blsd5F12VEqh9j+Zy3a+xiyt3JaePebCtkaxvzLb6CTk30\\\\\\\/VcpwNgVs0HBm+zg3JlvY34ABHsYRwDJgfttgpu0r5h4i3v3X5xUfq\\\\\\\/Nin1eRy3tstwgL81tEEADTjYkakyk7MNMnqghoMKmzkocxjSzmdT3r4zadT5Q1WxgEL2YIpF6UPYmQdhY\\\\\\\/A6j9dFX9snllcsS\\\\\\\/B9+W4UJXxxsmg1GL2jvTKMX\\\\\\\/uAmMsVi7TIzulgyuZd7vOSDr+Efwzbu5j1dXXwQAEljLQgnkKhAOjrpA0EnXycMTr\\\\\\\/k86WpMtH9eCaPRlxlUuqoinJX4elItQZMdO5zbG60QxE\\\\\\\/mcPXJBqdwgRBGODQywm7GY6EIpCB\\\\\\\/KlR1VvOSsb9hhn+hjKXOefd0NuaCrIVujhHDuGCse4zvDeGVppPRltJLvx\\\\\\\/geQ7x2iRtMU3zqQtgymnC66A5thY\\\\\\\/JBpkSvZQuC7rINZZEqlpYu1n9IXXbIoJeAF66o1GIegj4CnOViMEBOFh6PIEOTYXywg2Wsbz8FULabVbCsv+652HNJKZYEMv1bcqcyAIPrKCy\\\\\\\/pndXkqbYKcPpphLMwhARVBkK7sd8HAb6NDsGEi7Th0N57EbfkdJp1vYn46srQPhXYZddqs4xuNBO+axL\\\\\\\/yJgikn4Qv7w5gPTe9f2u8IdkI4S11w\\\\\\\/XA816KtF8M3JAbdRPAciS4OwPoIn64qLYkVoBoT7G4LYKAxn2KeO+slO6d1M9c2eiFMx44aEsBGk87mWrIryqAhd9B9i9iSP9KyXMLDgGGhM2eUC6DxtopvGSknyxsQoQiRgnO8xEEyXVvarQbFymcla+c4B+ziNyLoBs0qTTjQ5sOjoanEto2Wjo+fTUTxm2tjMjNFSQIuYOQqnwqU249ITlB\\\\\\\/SUQKkPvD+WmRbPUlBSFmWH9uYeiTJHWusqJra80XlR\\\\\\\/Y+DXJOybYUESMa5wSn4jEH+rgfTIm4aps1VeIzWd2pEh41Ec82H1FAPKIB4cnolmnPN84GBc\\\\\\\/8x51emgUb0\\\\\\\/Zgv4ncThMkO1CeZ1\\\\\\\/5qXgAPmSFgRA5KKiUttQcLSCNNa7XERcCEWdRonTW1K\\\\\\\/d5HuFYBEroKtJfnjcxERGmlqPYRSRrYqtAGK1pZGFiOq4B8acRN+8+BYWryHLf2cV9NBftr22iJXPnSBLneZXgMRQi4eoxJlbWatSOtw3mmlfDsaUIwbRbOBeAJ8gEdl67EueWLer1e45DmXDTGLXybAAVfahr8p4yPcd3G0Xz3DyeTNZ4U+wDpjMcofqCZbG3o4PDxUNsf6W6yCB0EHcAdEVz8wcccAcPQcrJDd8Rb\\\\\\\/NA6HtXnPkwneQ0sww\\\\\\\/sI8OnlEBMb8siLIQJazlvovVQPVsLJtVgGUMw\\\\\\\/2sypQsJ3KcZqeZ0uCEYvjpv250p3UPs6oZe5Ly+d8iNWB+pzUE9n2uCZxII+UDHMY6unefZpahrU4M\\\\\\\/71iPTuVIIR0AryrZUodYG4hJwQtGnSrCCCCs6tApJjfOp8xq4mneIyOvKZFnvLAxWWBSk97IEb1llJt8k0xbAFUdfItpheFJdugQlqioZQjksainXOXu7Q+u0m7OZArJeN\\\\\\\/u\\\\\\\/0FxOpYLK+zVJtX099WAubYvR55FJtzvXByFYJ+YBge4Yyfpv2mKZCoZA02xXnZExoUhkqF4FWRZqpfhuuFQX\\\\\\\/V6PDtnZ84\\\\\\\/e4AwlyaVsWqNuqSFxE6qjtrGqWpsATsH9bDjP41AEWOwYB73pDQiSFv8IZSQKeAuqD1TH9OiFYFvkiCVIt9AoikI+JBhOHG9qcCUTCPD\\\\\\\/Vu1r+ViY+z8NVuqnpubruVoObCqGho0NyvPXt\\\\\\\/I2iWxVmT7BvBwQ4Awg7tLbzadSwKfowrHGX0wHQgr4XHDfp4SNfjviAcrFCe3dS6hgGo8tIaP9tFH8+AahcMkwh\\\\\\\/QznDVzFMEG5ycxXW11FHe7Vxo1yol4K016vrA7fGCQx\\\\\\\/jdqQ+IHmP\\\\\\\/fbhXrIbEyzW0cvReqOgNqa2O7aOt7mRkCrBXNqMmYyoaW7ksZJbmbgN8K1OHmiNwXlG0C7RfwQJYeEXGsvhZXVEyZ3e8orhnHzw0QoJRfxBQw0+VWYZ7vnjrebd\\\\\\\/Qo4cQkH5gNxHZhyJ5NGd61\\\\\\\/Jvs324VO88BU2AyasHGprlphxaNgpNooNF8jbSaAwaVrRgsOUIvhC7Zkc3w2PJO\\\\\\\/lkB4mjJ2GZ\\\\\\\/EL7LF\\\\\\\/6Iqq5wIP3jUV1o3ZqpPJPrAZgkAj0GS\\\\\\\/MapTaafsIzVBic1\\\\\\\/nhOeKUopzXLpczqqTdZVx8Z8wyIcfJzplC+kDGG4H8BPbJurrH92wE\\\\\\\/bhaDrqzK+vaVtZK\\\\\\\/lQGhFE79aTw4ca1Ufr04tsOTN4HKmwAj7xIYu24NWzv3jXeoquTsqZyK3CbAs0u6J3paQFfDTwIjgg5XFwdp6qz4ZjUhP0+N+7AZSg\\\\\\\/ydaocQ\\\\\\\/HnzbgQNeirHIg0M3GUi49wrxiTdKXEMoOh+lwH2CUzjSqqRIa07K017g7PA6aqtCnX4zw0OtO6It5+3gFX8cXvpMQ8YdrYqCqcwWpEXoA3Ar9KbtpfX091YuO8Nq2L4GNk9ovvDOOUpeauvbqBoGaR3sWdGCO6Gsra+\\\\\\\/isCWu81EWi281v9fkUeg4o180aTzxkOXtcGRKcqZIJW8EjTtGUF+z97euvlgyRInGidII1PUSjLcCkXE7T1LpTFoADYjRh0F15UFfVvZZX1SyICCggETUGUmBJa8osqNf+1BkMXW8ECijx+gwMRhjlmlNalv8hLEwrGq1Qzz65jdY9OvNi9D64qV07iSTkLV9qO2\\\\\\\/RZ3smR7F+p5k+0DhN+MghYdhJVXPx2du4Kp1Y\\\\\\\/MoXQTdKoMSWSHaRj0C1UnsLJHB3f6vqmNg6TWhe+MlIHMo5oFZsnenMHJV4VQGpcd1wlVV35wnf0tY4Uu3N1e1LYPlPMrxefRSKOOJtFLdl1OGNSIwn5tohNb4y7t9l6I1ZSY77s0HOMoHI5FRFbieSMSCERCtPVdKnAFbSVrpbuvIe\\\\\\\/po5KayyHJdWBonPT7xz9KwQ6hBQDSCudISjSr8WHq70sDTuANV7TYhLjH5aSUQdqCkceQOuEaJLL9RLu2YQllOgnbPSHAIPQAY0DhdmcwYFV31ebb\\\\\\\/aK4xQ00bn7233yV5xiT0VbbA+VFyPVFTpfbpo9Bk157BtmTsCwJ5GDCzzV4cQahpwTgrmHFh41a0yN77\\\\\\\/mhb7THsKKQ\\\\\\\/oU26yocmybMcDaGnJ5C3FnRpc9SOTFCUYVoAotEKJwQpxZEDB0KSCEKQmup\\\\\\\/Z8OIxrKeSDS09b9RyF\\\\\\\/lq81BC\\\\\\\/IHK77n6vcSVO7XKwwFL8dOWj\\\\\\\/d8kv7UGjrQaWQT5EgRRCEs6RkCbdEYDGrsK3RhpVsPZeXxyS\\\\\\\/5NnfX2RkiEK925ygYfPjFu+\\\\\\\/5MeBmiiStMmJw2ekzVt5g+gUK\\\\\\\/cPDL9L9\\\\\\\/2wdQotKz1lgXunnLgspPshv9aW96rl3sxv3ftooQ1F1eOpH0depVGPNCB0aNYy8M8U1hHYLQ0DIuxNN9jH5No1MKfqE3oFe1EFoim3zRj1lRO+LW3Q2tkrovUmOZ2pxhY9U6fEOMAXQq0bZe8UjTgIVMLb6KH0NkuzaDCPfsnIjyQrVIRhC+os4YCI\\\\\\\/ZDjBvVsLHpwUPY81TmFLupUTvm+2h11lHcqyWsrEIDK02KxWTV2UZx4KBktui5I6kjyE\\\\\\\/W4Y1MqwsAhpUq1u4rfBC3Kuau9XkAOiHc8n1qPqO0+UTidkrNFVGl0wSPMxuhxHuQ5GqsiETl39tEIP\\\\\\\/B3GHTC3j09M5qRWAzgNRBy\\\\\\\/ViKgwzhAyg9+eq2AcQYDr3K2U\\\\\\\/2TVu4LOEuJCTLUiZ1s07XcIC5bkvSPCrCplmetNKKHFFJYty+L+bqTR1p9t\\\\\\\/fzy6HUocXMYWPkpgNoYRpLdvFZQalg3cA0Ubl5INOSqjOyiENTe+QQNnhyFU6zt+BUKHK2J\\\\\\\/UsrnTKfZRhJ7SCY83hniZtKs89drqq\\\\\\\/6PehXJLkcRiHh1T6gaWBa5edmhUsqhPSKR0zjQouwfKu8ZyarZMmhEO\\\\\\\/gfQhkYOoahg14MeGnu6q8W9AZuiC0L9aC\\\\\\\/LnHOtTo+kjnThZaVUCs+7ynXvauH60nML1paGdphB5B7B\\\\\\\/lxyrfua\\\\\\\/hUL8m0+34rYTcSAT6zD\\\\\\\/VIphFOpeZGwz4l6NUiXwt3qYkipIsSPln8MUvCTdBJI\\\\\\\/mVWTVKdik5hPK+\\\\\\\/8iGVQIWpDkddoVgPT2I\\\\\\\/bAO1pJnPt9GmSlGeoCXHPr046iUv+uAsMl5prLuvlJiekfdYGP8XOP38HaXaQKOCV6J+nyfUhLqKPpOWvolrKvSQErgunSZvh+l4U\\\\\\\/djIkI8Ynu1Z3a\\\\\\\/2nau20KwCBCpU22BhxEZvFiH1FTgbM\\\\\\\/CYQ\\\\\\\/+rFxTySRV8ImK\\\\\\\/l4lZj6gc6OOp3Hm3JTMR8kvevuJrx69Y2KyMraN84cG2qilWRKfyoFyEHETKrBaV1jj7j1RURA2CnkA1poHSUCtmQ7+OOhpqLwqXE9Vcj95vhXOBX4FlnRRLHs23KvEidxDqhAvZxLKWMi5\\\\\\\/12PTubtvwaaFH+DpNdJw2qK2zfC0kCdupAGfXbYcPZscNA8pvVbyctRS9BFse3XIJYbLlCNVzt7lAgIOS024hJ2F4lCXgfjVYv1gbWlYkfbU6PI\\\\\\\/QAEYGzHXXI8axpQ496tEcaXh3O0KufwshXe4IhszPan\\\\\\\/XA7f3d35Viq7nVQEezoe+uq3Uv3\\\\\\\/MLBcCFw81xDt1BDMzpLauiFYPM2H8\\\\\\\/eBsjRIVvjTCuQ9SOiPzY\\\\\\\/XVQpglwctIWwEHyUltscKeBKD6LG0\\\\\\\/JOdLaUNZjOuBWuFDp69rbnNUAzVOtpzIx0jbeOQcaXlgZ3RiA4y3vVvA4wg9U\\\\\\\/JgA+J3uNmvhWScnSW8dMlFp\\\\\\\/N5Yrq0QLTIwSZa76YgjOdzsvq68oJMk71M4a5vt3l+jsxZDPHgGod2XothnWvl8pVkCl6iVk+zFDsFvJmfVCITRM8eePrLVcqb70o2Qa2G2yiRBfeF4DotTF3ZdXxgChT7mIOLveclM9GLsBswL5JMIH5Wd+h0oFZWb7W+gjxBvMsqseO5UBFoGn71IZFhho77PMIDDD4bxKqfpvRFTXfrI4prCzpgPLsme4s4dh\\\\\\\/PI4AO8wjTHlnqOIEVKdmfWUJPwsM06iY\\\\\\\/A1CAxRrTMIwv\\\\\\\/OuwkoPh8WcK6iW8fQU4qh4YVfsGwFxCjgH0YSWqNffx1ZiZ72DyQ1062vp5xy6ydHaFosBR8uolMdjLipuJG+Z8rx2iThJakYmdXCtTMMgTy4rvEyRzG9HzMzBImucqvtZU5Ttrxinr\\\\\\\/l4s8l70OvhYcLvp6Ywqt6MqlbwdAGdTssh3\\\\\\\/wezq\\\\\\\/BYI+XKIIVHZ7Zx10s6sQotuDr0k7BrTaun6vCzYEkcQ6BDMCS8821++mFqXKYheNCTq06P3z7P\\\\\\\/uuBFDea4IjzBBwfgVBPi0qaiGbtUaJYyCOlSW2ygoGr2W0lAAMi3NfEXRouVUp36euZRF\\\\\\\/\\\\\\\/7gAmRlnaCnKUVVV\\\\\\\/ENg5rOJdb4VhqakOXMJd7LYGhXq7N6EAM2wJa75OuiQ4qFp0rSQOPIYs7EF7MDE2KerJghH92KLT8OL5404NBnvpaZ0CIAD+HCnaQyHoTlcmWqc7XTzcvt0OAFYTlC\\\\\\\/YwsqtBOBr9NkFb24ygaGiDO39TNX5qeT8Cb2\\\\\\\/+cKXAp6d2bMOW\\\\\\\/BL1UBEY+j7MH19GRcxp8lwFh\\\\\\\/TkgDI4RbKn2hwddcLdDmzs60p4HQJGJ\\\\\\\/NLnrz8Q8NEQIk8WiFMbl4LPHRKy5D9GDGshdxyFiE+bPZaz+5PR5xF4VC9A+fC51euqiBxz5Hz8s4vChnwxbteRRDYnuGtdpuPc7YqD562iNqjN3NvEbWltj8PUtROHJKQBJVHJHiUTb1TeHCruTZvS8EIgeemUI0RJPUlsIuccCXHghJjyvWCYEtzuB+LHzUEXn+ItutbENjK8lEIM5P83QMVeqq8mycKUbLt\\\\\\\/3CU9aHzneV++sCHc8ytK+fCejCawrw1OmOGzOFSGoq28SlOap9LMKoboZmb2gQ21wS0nPeFxkgAjwEipdWgYHKWB91mDDiInL0TugcLiP\\\\\\\/mdo\\\\\\\/22Omcs8gShQrxhx9jEkXI56\\\\\\\/oBUbBMg489ZDuYMWHwD4iOoq9cqix0iWiA9KkGHbHOPqRpTCgh02lln5xsd1WXMc4rPcT1hNx0aF4fYnOZxTOvNnVxgPn0uvzm4oO50RXVreRbQoocJ9RxgiY0KoWDCtnA3vHc8jIi73LEn750ZlQUO\\\\\\\/qsw\\\\\\\/\\\\\\\/fVCygHS6q7ZJPgzt07weWIk3PkdO40DlPjJtO4Awt1FkE4kDPpgqRinTP5GUzrJhlE8S7ABlqPtE+gUO1GM2Yrh6nCA16pNFw3myBaeJPIAjYybVXFlu0m4QapYFGgmKNRHhqkzsH5Xcmg8FJktPaNTAdpVSqrmyS2dZ2V2PH8k7gXyJUTpZalIJhWAvEal7pC+qnoaeRPo49eEAZq0fQAw+gh+H7rln1OCdceIfacNWYEfzMl0EQ\\\\\\\/XhVA73baQLoXRQ7IXAez7vhZjXF+8E8RhXzFGvRePpx8cc1Oqv6b8bTQisBYRAw\\\\\\\/kWsSIowObxBkrJEP9USqOTSIKgBulsb2rk3\\\\\\\/4IKCaehDsn8682Dm\\\\\\\/2Xx1VbsfBbSYAhSACMdi9Hraov8q6i8z9M4zFH5fSzFiUlLNp2D2dwxkh4ceWeSbd2DO5F+eYSmn6SMHJRe0VaBr1vawWVg77NS5jcor\\\\\\\/mqdyFDH2Bdmn7DetAnFDXRiA3ImdHGYZwhys8aCOVTqkYVRYVL\\\\\\\/PkOf6+Zuv6DNNlMwJyIPWAJHGTokJFjNIXLeAsbvmhWa4j0tT5hsFiOzLr72DAgXpKy9lH\\\\\\\/lu6f7mmx4AZVwgq28sUqjUvINqUiLXMHp6iVg8EYH\\\\\\\/SXHeVoWp7pii56DjkOwJlMXPYtJ4H6J58ktwYmh+sw68u5SPcdxNylY1vBkpwUIztDzEFOCoeX9gV4+t9FcYZIC4NpO0pe\\\\\\\/9yYfURZTnjkc1rumQt0M2Us\\\\\\\/7zMSiR3uDH4\\\\\\\/rX55fAZyKjfk1o2f9xKa8EAWEreA2TYbIoxfaH8f02Yr1gisl8MTY\\\\\\\/0IKByeYZuzP3pB88Rti6YLvBnFepW30Dbs4d7W+NlzXdF+kNWzcdc1p3zkDM7RfLmaIqN5L43VSGZIOdBBQ9Lq\\\\\\\/aclf8I7GEIfJCBYsqNNaMH15FhmtH7sfTHjj+cYn1ejLn24fbJQl8oFiVw0qcVXHhPp3GYK3CNwdpqDF4KNI4qV+Ie44uuFMdNyIMZQYdkPOGZzWFcjlgFoEjl+1VGaH6bG7EF5Lkq6mOk2vmdfg\\\\\\\/ueZmXWRDrVk8EGfiyLRdoIq1oyP2s8BSbNy08WlszUzec0qgyO0XhQN+UjfK8mV8V1rT996e2O7+iq19MLg1FpEqLD7ts6eIAO2WVlyVvy0penNRuOj+PBS+R5h04qCBVOB+gHm6Nc553i7kIzZuepAG6VCv8XmaeEaPtg8HM601o5cjCVIMoLST\\\\\\\/fD3EpR2820gtiujwiWbGLI+ZuO1ZZ4\\\\\\\/db0OyDwOr1w743AwRZlnl08rIViCkIDx6hHbyWWtodl84C6be4bxR8vQr3c2k2JT8\\\\\\\/jm3rLfRPQKyzde2cl97Aw5u7KKEtAsjR403OHV65X2U5MYGD2RGbu1ZitK0FxFhrWLu7PiD4wRqDiLcXv6L4xt01a1IC\\\\\\\/bQHDduKHihooJtKfqEQT\\\\\\\/Qptojq4cswB5sh6OqecSHY\\\\\\\/W0muXF3\\\\\\\/sXK8bkWs+BHnfwVlk7f3Mq5mtov4R2d95a6XDetRiSMIKsPWSEYvRRybRze9mH0CRQ+nCJ1M9hXGdk7DZ5xpYfuDW6SV+SUcfmdoRr3xwtJrjQ+HwGijDYDJcbcpZOe5B3oJUEO06Ff9buy1gDPFzL9Ph\\\\\\\/XwNu0KxdtUVWdOmjsrG9qrrvCKEYJY6JDqNGL2YW18OvRKaJtqokLnSsAORMbq0jcmISBSB0pNpGFTSitHxJT7ZoIpk9Sz6wyql+sLPZyf9PoNy3VKoydvjUoHMODSqb8Ee8mZRfr8sMsz5Gqzi+skT7SDrUeXVtNIQqFBVW7D1pHiV2poCLkqwYpSB9PM4r3fk1WIyMouw\\\\\\\/pR1dKBgE6e8jbJIo+20vwpng2lwm55eOl54ll9YHJLK1HIqeoDXqZ6vfNcZYuM72fN2nFAhGf26+Yb0M2Lpib19JH77oP5STRM0MO3vDcQo8090\\\\\\\/Rd2joUOO+G6+yakrDOuRrVzS5b83rhlKyJj\\\\\\\/tVac6h+wnFcUkDV3\\\\\\\/sU0sdMMLo78djgJnAiBP\\\\\\\/qV3X+gJLH0n1CiNmY2hefjSdaqhJ7BKS2pjfA7vs+13fT+ckyu5pZ4pUkGGQzQPW7edCTL2L16xKe1MkKSJmPbCPn96PqQNEjUHOyG890iiZBIESSx6DBuo1g1eoD2On6oBKHe\\\\\\\/zcQCZMCj6CNxHLSGGgE9lWg3iIKd3AjCFFDPQA9JAtIEuPLr13r3BYR5hFtiraOVbYdDk9\\\\\\\/B9HXDZl\\\\\\\/oUY5cHuewbmzG\\\\\\\/q\\\\\\\/pEMPxn10q\\\\\\\/CdCKBUtGLYQjfYjH0bzVW5ymYkCwiYsMXXbafzKqybUqlH3SAm+ubc4LOWOFRamVIkZxN5E5Eyqc0KyJ4Ho+V11bYfb5UOHCw2Osu86QFZCJnWRemV6zX9XxDXEfSGZM9Z7XOj7Epb07mKPgkprrB6Buo+MRtPswErs8rXB8aMqNfKSheOH4W9sPc2bSDtYz60htK+8OLbAsyUu5VGAg1hmV3MupRfSrtMLZ9cTA8cEC5IaZCjto0s5H6XU0QKLZWwBkuD\\\\\\\/h75t6MIpb3gaIMlvsDJcjW4j7NQHYqm8\\\\\\\/ywPgMPVU8mWhmLfPRJkjJUHsvAL8LZV82aax9Zvfbs9xxpQYkFS74dgqnBnadvW8muXZikjSUxuOBOG0rmULnzz3mabr2EVEB54oURqpFtD5Jvr8\\\\\\\/TiBTBIpmxDRkqZRrnbF2ls2JmBOYdhhc4T4wru5kSWED\\\\\\\/Pwq4vAaFimow08abeQUDuJVJ8HkART0EOV6UEjUgQdS1z03fZUNux8TbpOQhA1pVrIbhI0nvDSppZwSn0VTltLFCKQ2Q1l1RaDgyx2eQgzajqOrhKj4\\\\\\\/r\\\\\\\/XZGu87L2Z9MpQFCyzgEXLH0mPhAv9\\\\\\\/P1vNngm\\\\\\\/lEN1I0SoTDWYFgheRcakq5C3iBjXH3PDdbPHZepdoR7faQqMTa1h0BR4aGROdXQPldMWcEd2RcwvNyvR+8WLYDJiQo++0J1Y6O5+z7KCW7o+rhBjldzyLDlL+t+k\\\\\\\/RlgaUOrgTxnQW6vy3pvpqHAAByMTz7jB60pIuro3fj1ENER19AQ9JJh+ZlLDTM\\\\\\\/R5sKPvtapX9\\\\\\\/yxvMNj8X7GmulIwrM7Az8YlGZ08\\\\\\\/YKxsJcbyxy\\\\\\\/QaPCXA8KXIzoK\\\\\\\/dyRNiK362DciV3RFwNPIbnCEmrsI5L5aEutlsG074ETWomjdTLOvUYWTxlxtX1sq1yHhYHS4yu+eIRB898R1tm9JGTo8IGBhiJt\\\\\\\/X4IRujUUWcwACVG4Yz2y3oRG6lD0+07Bh7CtQJ4cHs3knrPhp1bjxQstnJ41EAjLhgYCZIbtlmj0khEwU8MN0UMtJ00i3CGy6pLbdYKrqxFBBbhN8e2ufr5p\\\\\\\/LdfqRBRVcaVef9THCc6ywAJJunjDRuyBG0VaL9Z6cUL\\\\\\\/c3mCJ\\\\\\\/XrYZAmnE6FVOM9jw\\\\\\\/XePkJB\\\\\\\/Mjwky2xYXDBRzv0DcRAXA2jqA+eSrvxIgThGO99ABZyS5swaAHgaObS6wLBvShseJf61PDlIfDFBvuv7Vv+\\\\\\\/q0XAUlai3nY2EnoLEsAFgFTUsA45kvimzw3cPlTzWBPhdEyryrsFQAdPhYiS0A+cvPMrG1RZf1YvMdCz8m5zkejkymscQFJ9r+7ftHb7s+Phronz+30yQS6jdZ2TUmxb4vPVhOsEAW8OW4\\\\\\\/f5Ly5D\\\\\\\/RcMTsASIS0cicoWXbqIIteWt5WIStp73j5X0uhWnK8Ihvrjz+Qr6KRok+\\\\\\\/b31B+bp9QVAXuHA50nm+22Rua131Xn9z1Ty9quZXbjVKYs+NKOcfxbow29PRCNghXwtDbEnpZZMzMJEOYk2si8HyDefN40tynONhiabKwM2GsA96DK4x84cmCaGvQUQDjR7C147jocS\\\\\\\/sroZiU4Z62lAfz69cjgwpi9F1rQ1AZ7jgtHjFQLKptVxHKTbzUGJNuRQQh8KNhm6AujcemjlgR5wy1u4dr7HjmPkb+XV9C6szpK9Cn\\\\\\\/UvAN0keQ8hGhSO+iPQGOdu4lRvRDDDsMqNaUWZXKBMOYDqkEDTLbIsbqUWNOjln3h3EB0xWKKBNL4iSs6PKm4RvgT2QaYVlW6svMb5g7Q\\\\\\\/eld5BLPnJQJS1ECtCDCurfEVc94AZTC1oCqsmNmwa3h9T875codIWYnVTOL8bFAe2oPdfLHwYWD2leGgkGhO4Uj2NTd872c+9VxjKZH+ff\\\\\\\/59xsKElG3aA15TC65cJe6zRTwYlYbw8RGx0TxEMlhPlXodaqR4DYrhboE3a+oYOZRMVs+vilKdua9QBra9U187CLuZwGb\\\\\\\/dK6FL72V4fSGrUQRfOoJFQD8WNSEC2JfbcKj1+o9Rap\\\\\\\/6HmeYalHt+dy04TKCfgLG+VizXYr8GqtCFnoL3r+7yBWOeswCtVWy2pbfTxRKlsr\\\\\\\/1Hw9Or6HdSZDENP8NQfM7wTXyg1\\\\\\\/8eG6iwqZypvRZwtl0vtboIyQoBxg7UIMoEpP9jzMT2oT5OFzdruujv90SRK1xqo4aY10TIs5dKiMJF1t+4XCPEyGvI0pagwth6\\\\\\\/umGR2zO9KQgLRU77U\\\\\\\/jD9pX9LplWN0syEuwlL14V+7TShd0\\\\\\\/PD6UDjM2VXDg6tkZnjl\\\\\\\/iNMHLyA7l2TQMfmWyktw0CTF9+XnOEdWbsIFMnwa0SBj9UrVYgvWTtRhAnLUXY4n03KzHG3DPT2MzdqFctpg3yw9qBd8wCsamz8QbeCArWafkaTLLMf6nhwH9C3Gh9u5o+kES\\\\\\\/b5\\\\\\\/fRPGpp4BAN5Zzj4Gh\\\\\\\/LBbMFG+heVXTeY1MIbaB93TY+c2\\\\\\\/6dFV6LMVOOQDiwN1gW5CwxZQYACwbTu19vR3QuB3M\\\\\\\/u3UxOe6MMmYLyNIiyfHyWyMaZe7aKyOmRQKkGI0yjXql9PPL+FNpiRFvje56MevW5dcfl0kUmKHGYgbyAnXasTfDRYSCMgWapyS\\\\\\\/iASFfLVOATQXvFQ2M04ZGqQlIqg983KNnfjTVHLbqE4dtIGenWX4lnB4VuF9IgG88u824YLIgWJ+vgDIQJ+t8CN+RZJcFr4npZXLbhBLEN\\\\\\\/WRNEMUqmucqBbTIyAj2KXMX8B1pdWu2hIVBfWIcy7bRRFUg++nZ\\\\\\\/QwiWKNqDKvWTyDw323HcnFhDmWQOwqzJVOCAsju6Wyxvh1Jgv5TfEqVOGwwctHYZUDrayXcMCOo4\\\\\\\/\\\\\\\/i8n3sU0evaT9bKwiTJcJMgInVJ6ZUbUs3LTkED0jEV\\\\\\\/KYCJgS0tAOPcIDxj7cV2ISJVLWMdi3+DPUdADW12EyjTLm0U6rkk9aVHwHfbGmRYc3jM4q\\\\\\\/bNXuTGQko9FKzdSEkOUV87z9907plcpB7D15erKjBE5q+fDMErScqtdRdTfP0lJMwgmV1WIWXTrt5b4o2bafi6Ada7TEYXQXq1oejh+DzocaY679gKmOuqoEJkRenMLiZSVkZLjDnCzzcDmMl43i0cHJAyhoKP36ByIW\\\\\\\/Si6mSH4XRyolGu0kMqZntq1owFwXgzaZyXzni65bwZ2W8SuTrOG1nPoyoZo2VprOq6sAaCJLYwmqVLxjNsYlBRccHHi5gI+f3DcF+KrIYgCihmcR61F6kTh9vDGAPxLPm7GNGDtfEgtP4WcEwbbDh1cEpM7D71EFyAJu0UFITuSyau\\\\\\\/SuV39Cek9dNocy\\\\\\\/wYDuvadOTg8hbNJTpVKu0Yhwi03xceSJdvr61Gq+Mqh8XVgeJ6Ng2TCFN8uAwuPnosW4S2gj+vxq\\\\\\\/eyHxcq3TMpcXC1VxZAOorFmcZe1XPelwcwgDpe83gArpWZaMQoCn3FNDxwaE1qMy6Ph4+MKwqGoGGJ7HjRzdXMpQwrVI+l850iVq2f4sfLOYXB17q5T5yf6X4pPkZyhNcufS0nVRIjuU\\\\\\\/8nfqZOtQO6iWbc3fBWWRkz4a89ugBqyhSb44sqD\\\\\\\/WcTf90t0oOPRL2pfbgMgd8OMYlG5JngNLcZbDSafxFeSu+2J9cCSf5lOjBR7Y2kGS8+SAcSZNCxVGm6cbxY+IPL6zjf05cxfrjCQ91+6O2v7AI1aRmSvVK\\\\\\\/JqCf8P3NO9Np4PnTzGsDLWGpAGnKJqlNwLaQsPizTv5pTrWmcmulVzE38uvsJhYQhmF9TlSQRnsLJsHme7agVGVtfi0M7+gUE3G5SB1obeBNNQPf\\\\\\\/K7DLZwiUzEIOnzALmETWDIAPJVOdaAVAEMWPxzEjC\\\\\\\/1nfTfAtXHDqLt2nkejkUfTkGIDyFis8T5MrTeabwx+qylZqF\\\\\\\/wEqyUXieHrs+wvXd4dfM71of2RGQJAhfY8FUj9DyMq1q\\\\\\\/2Q2tle24tyQJWsomskILtgQh6VL6+Rmse3APZ0aBXnqkjlXOGBEFNePn9XTw4A+RQbgpWAgjGskSS6dYsZ2YPg47ijfZlEIMSZ+rF9bkMOQ71zTYp5BgVdPGvK4vowz7EvbiqaKNWW8ccmBA2baqvjj8TK05U2syfyFZRUN3g7mFiPKngI75UihYTl7i2bUblO6mAOaF7LYLVSjrRYCOe6R0cayZeitI3mrTlwmySHp8+cfeEUPSAPN4x\\\\\\\/Mg7lSZw271ao4GfVmlbOVSQ\\\\\\\/NcaQaC82kT3cD3JMaiW+lJMgV5nc8CfXmc7yzKy5F52gk22cZE0GEsXCy6YSpKfspGEbJbORLTN2Gtrb1asZoQ8iBnjmuZVbK9BYa0m+Qz7AI3ekh91RG9pRrsOXAGHZ7y\\\\\\\/8+O2zLojx8jXszhZD9qVV\\\\\\\/e\\\\\\\/t2y8CUJ2tRE6VbDTcxBLk1aiL+QQbuk5gUcpd3NKeszKpUJAIWQzf6AuSrifD+gxPg\\\\\\\/XZNIQIRvSn2nJ9HOvZ6ZzAFU4yIZERsH2oBVHpAFmDWng8ZMGuH9y062L7NG6maOL+ZULUHV1lZ13rEg9HZMoHbjJ2gMt0p4H4X9creA6KTIa\\\\\\\/lwjQ4WqgGxo1\\\\\\\/wQo6BdVuk4Lv1UFl1JHNfkU+rLyUnOMVsf2ynOzVos\\\\\\\/2l6PtaEqfqMXyPufPO+ijfdIROmFaTaTxA225q+GO2ZOlaBUaArPnNwskXaz6CnJhBf0aBmBDSVi8qIYPeV989zeVGnkZNnjzqunOfe8c8R3IEzrLZIWCdaxDAHVutrhG5e3U0SKTqgB4E\\\\\\\/fvAwwf4Edlbvt6hdlxHxaZxJ89irfvHfj+h3YkD+DlD6TdFQ1uvWlmIf462xLJATPDcSv3T6mE+sGLmoDvcGiEnKHBmw56Mr1Y9WAguKihtaKbRnxf8lAN7xfnJ+OCRDnH1XepOR419yvGiP4jGLzEX1Sqec7uZXFWhh29kJtOVeFfKb2kOdSUoWv+96QZAgJZV5Eg3RXRXu7eBwnWgaRAJLgByfMi1QOupXFn6+6gAJh5QaybHbSLb+ffaxZwu16lKzfZU2RFSPbRmK+lhpm3BunxJFHOT+re8w3d0v7+oPjki\\\\\\\/qQSJe1CwCaCuYWjpcI0KrO2Dh8523HC23X8kbSrKtUcXQfdrp0m1f1r74HlWl+3oIb8b25zuB289qSCsbXiHU0aOVsCbmqa\\\\\\\/j1LGvGhxYZwVdXu4Sl0bwV6AJQQ0o4QSKjqofnD77Xb1YPc0Hi9MLCQJiy4CvBfM1PtqA4NekRGasdqQLTqlekMbZegGb8HqrxBQHWS3PEB9QDJWjp2j4TQt8gPT2AM+U6PB4GTHlLdUsTdgihNhbGXtfXnGVOJovQM62pWBPTqVkA5PSmInKkN4O4KWD7L5FPJ4ScC6\\\\\\\/D52yjzFMcnj6H\\\\\\\/TyiHNgcYor5Uy5deLpEUQDFin4GsjJeOY+w1lCcn+2dii0VlrcLb5q0TjqwQU5kjEjcLUq6HJAKNBJcTVr9xa5UuamyVke8KpEq9GuJ9VJL\\\\\\\/+aJhY0jm\\\\\\\/3rwQ3R9D3AoDWYXnbZHCpR33Tv68vcMVfpG65t1wZGZY0yvSCWIgwUIJP+2dJsbG90tTT9rryYDPKSKdkTwCOB6V9vOSVUb\\\\\\\/3eYNSTnK4Xt58UVdR0MW4d3jPNsn3lz\\\\\\\/rspZv5EeSedAdhA1Ysj\\\\\\\/jfw9Wd5A1O3sWk82\\\\\\\/jfsKqL40laXL10\\\\\\\/9Hq1bUg68dz8YutXFhMh7jWcbDepdEi9N0AzvJCnc4fya8zLvWThsg2uvUqYRYTr0Vo6gdKy8\\\\\\\/d3C2hTCWwnKsYueZgSLlVwhRfvevxwJqofmCvp3u+9zIojdGPL8xbqMMqt8MC6fIdnZvmnw6XQL97Wo8PbI8TNndbp4Y1o9RwECsfY6b8BCPt9m9vyLfGmzeB9KAnkbEXTPxrqVWN+hD5BbXQY2XH60kbXCby+KeMJ5LViLFaRLlqmkD81L8zynP098WEBo30O9N92svKRzxjNyt+mYXSMhImFC5PCq5npbXjFs9mn93o8zLVO\\\\\\\/XncuXZWIXo+qRkkaQRsL7\\\\\\\/4GFuhReNNoz8Y0A\\\\\\\/DsbxBscMXgUghzrmaHYB5197odwJeOZlWQFagKOcYcbdTiVT0OVyixNOuWfN2X\\\\\\\/xvg2l1dPEg2gN6steLEnaBlwOS3sNGro9\\\\\\\/rtHobjsXKzunGm6qqTfMzlHa5vbMnOEoiVTCRhSyHuhi2elzpY5Sh92nvPvh49G1Z3HhW71Y\\\\\\\/0cihiOeVBL1VzQHl6nl08YYncvT0\\\\\\\/mrbUiFR4LbvmXYVeD5zDuqGuaMRR5kpoRRdEKep\\\\\\\/WRqO7SryB68RAGSeWhEUxJUH69pJpW5ptCefJN+r\\\\\\\/C+Kruo977OGTPOWpgfi2APDhSqspXF6X2gan5SRg1r\\\\\\\/0xATnrBd1TEKBTOHe2IsKKY00hx\\\\\\\/4hFjZGeUUTNZDhGKRc39qV+rJobLMtvCOaAKqhJRTDy6Dnuzw7W9QYh2Y+FgnvTXn0NZql\\\\\\\/XNbKfdnCAq45MhD5qRJ7W1BCGyYApkgdlJgq23eCfhZFWmJyPcM2uGlGGXfJOiJUvjBmjAz8HMIrtitWySEfc\\\\\\\/M40WWGjVN\\\\\\\/RUFEzPENnQNwf9X2HMfyeF4EU5UsybieiS4Xsrsfu3q8WbFXLuskOjrXNnnOLky8OJiEI8LYVQUS8dL93rsMkyX9Cs9cDYEUSmjKjvD2qrrFd6oxZaDn\\\\\\\/xUlJpalJiaUBdZJYQEO5MXFMgB7az\\\\\\\/Cmw2bk0kFlsQVzIyfzpe9CN5EIeBMAoc99A2aQ5Ko08qWL\\\\\\\/CmbJ3sMP\\\\\\\/npSWeMCgml+vFODkFCT0bf0hf2qisJCwx0rqnGDY4YG0lkjz\\\\\\\/8sIGv176dnpujU4MqvTE\\\\\\\/\\\\\\\/DeiJyEKPZvVkiWkq0xzq9gqN\\\\\\\/8ukszXD+4pAKSib6k6YEpdNy\\\\\\\/HAR64Ao2dOy2QPnfpEztK9VfHBAHxbb8lOuNNyffDxFis\\\\\\\/75E\\\\\\\/dE6Zt660XDag8llTGneNw1PNPY1I0mYwL\\\\\\\/WLdIX7TFxLupGl7lH1AwGQJaySyCu4pHjq3JREWUkTE\\\\\\\/dUtMdL84IUa+55qx0N0cC76yah7B38O0843yxM+ZgaL9HxtqT9TFX7LJjZcZ9QIB8BdKaSrhH0zeqwrxFAs4+S18QiHRTWyL+rR9zx435eHWAvp6nHgabCvgSzIwi\\\\\\\/lDTx8enMp4SiUrURZXcdAL2IPsrlyoolZ0V5aCEEjUtYuJUpAnV2iHw7GpxIfKGdCE+N+MUT2yUfBI8lIvDmLckm833h5opqXkuU9YWsh1Bk2DSoi\\\\\\\/brkkqfLWuCqj9\\\\\\\/hYZMC1oAwn2jY7mybkin3MEk9P7HKfspVA2PHnflp8jr31PItWYmJyvx5sUIGU+tMUVPRntK2XTVBJrECWNhZSSd01g9vnjKv0IvJmZ3AXPr+DhJGRJGVF0maGFyntRRm3P40WWMqkX224+\\\\\\\/NTtFDlfS5qCLCeJVYIvNTRpCIB9n1I0ZmyszfacBWuyHmToCY0pr3+O3VJa\\\\\\\/\\\\\\\/7Nx6hBQEKdpLHWeuNzsWgOXDqjb3a0mMsjpSxmrtzP3eS0E3doIbQYVUXFuKrCrdwN1zQoblL6fN6y6lKOBVxI+bzBn008mZaSfYz9eAbSXCrvpeYz1RpdyYLLzN3vakXXPIwvIBlm\\\\\\\/2T0PWbZRvIPHk8VPXEVfGubyd11eWeJyExF30aYGot+u0wsrVuwQc9TZIAmAQnmCGk4BQrh9klRMj1OZq15U\\\\\\\/SAXQrIkrVEtakpmw9hIYCkveGUwwOq8SgQiXaPdH0KgVm0Mz6jqtY88TG4ckJReATKx3nDlmA6QKxtomBXdjfr62COIWCqf4r3Fjz0QMBzEsquttHWgWouYTb96IR\\\\\\\/BVrXV1GVe58OCihXUw6wpPTbRWnUaT2K3RsY2Nq4hrDMqx\\\\\\\/kzzAC0c+DXdGo6rd0Yo0RChT5U2I2XBYkkSbKrLhgXR+qEjsgLpeLMOjxlzI4rOhx5WDt\\\\\\\/vznnRl04HqVHbC1kJlXGyWSjHs7U9mvWs4ufmAWZcT9C7zuMEc6aYFEU3mn6zLiinQoyPPlESDdpVj0xV\\\\\\\/uY4OGHfOpuDiHEjH5APYVbqs+tP1DOqMpc5gS5g7UWx\\\\\\\/hohUxC0zCVFdSUo+xaIvGiBhXRhOwDCGDaENmGqvD517oWUEWrBMV7AvIjqATymp0K6gv+FyWTiLqQ7ohiONN9\\\\\\\/I\\\\\\\/nUPxu0So03C3+QIt72hHZ1xneV97cYA4O\\\\\\\/z0Z8jKUbxsrtgCDopd2zWwIBiNbbgpu\\\\\\\/uNOEB61L7E5lKKBx4+F72QQeyl6MS+KX5GHWNdpVDLv9KRtIW5QmQHUMZyr\\\\\\\/\\\\\\\/FzFsSxwLNqnW5oGRZB3hBNymDX9urRFyoN69\\\\\\\/Aih1kO0eCZm8nggQamRIVFqbgYobHlL2h4QaBQ7CKfxawbya0LrxWwQ7LSI2cJe4YYHR1ULD1S9Dn90io2lsQw7pM1uNX0QL5HOBQ64ajxHNwEesPOUVezQBD69fGsvcYI6CbfHf2+PNcA7ByGSCHh7jS3HjlzFob1+aEtd5rur8Wwc5YKjwyW7K6Gy5+\\\\\\\/jk2TGwuH6ll\\\\\\\/juSGdHad4gMZp9sHJD4779Hf\\\\\\\/4xjTnArSUpo+ri9374LJTnbkVs8l6qDpk2TAAgFE1oiFPnJ75ZReiZ7LPLKClOun9owm8DtYsbmVDkDrjL0Paug5e1XCQcfYz5\\\\\\\/QZ+aQyqZKYFmfKZSaALgRO3xlAGPbDqdp0yAPvAApEl\\\\\\\/afU4eBqUS9MmDz42DHd5QIY235tPYRaSDeITmWMHYkP+hpa1TySUGOx04bL09mcMt\\\\\\\/A45dwN8QULYhWLSPDxQ9uAicgRZTS5e1FWAlStLHQLcLSfI1+MlASGzj71M627KKm9Sy8epXAi4p28eDcpWzCF\\\\\\\/EjKJvjYIaw0Q2thcyFzTxgjAlOrPuVMTfYt9LV9P0NgM1Sz3OLBjwuiQuBeRGVhrgexa4CUrgTK8VT5E2c+iNE8CHIYTmzOfKNxeARKtMoYhkO5R5FzYTXEByEUdCtlWiRPFnlIUV\\\\\\\/vYloNdyVdQCVaxeiNTZVfLu23JLlJUfNjIcCdtsRyTeYM9e+9cowX8A6GZGhOISvT9\\\\\\\/DIy0XLHN7ALCzU7kRehnREVyJqh9sqFFgUHSFrBh9QTlaRcOskCaVbhvXVMl+HkWjQ+BgK3YwZzDxDmkBAo7DANMVNbEA0H3BjvDUuhUhHkAqDKNdbxKEsuQQUH1kmJyDD20\\\\\\\/RKeysUcw5XjrcFM6oWS1jeTntbNmW+e3G4jPq1o0kyqpqfTsgxxqEBUJR2P8Nn9fh+MfaNiW7URMUolWwboem0fJmDjC\\\\\\\/O2eKSMxdJ00kEMcgSU8G5n0zolMmQ5Nd4DC6v+7TG1FYGfV0T07dKoFfGNbZrH2Tgh33X2XLVgRojUg48X0eiQIgnF3pbilQLWXlXsMlX8l6+GBxxPi8T9bD8IlXHmMrH6\\\\\\\/ga7oiN0TRykIXAv7cN5nUaWgfYAW1Xfac2+QqcUvJoEydGwA3x9U866nOkSBmNs1FhuTEiQQWMqN00GJxoFopQ6NqkFZFwXtz0sM2uqsmlrjgMiidb2qFzJD23feJnIr9lXPhwuBQgsw\\\\\\\/XbiCmv90XNp+BWOgAoZKMg1tvNMSScrkCPD3P+KVbTT\\\\\\\/SI5m5gxb+Ztp+VuDa568nMAPBMM0XmYzkP16OTyAKimeQUSyRFKLr0pkT5d4R8OcST47LH0hLS1THaZy+o1e4drS8ybu\\\\\\\/jFmOxtaXiCv2Yj4xUKMAnVUPmxfjn5UsjM+TeR6k\\\\\\\/PQYkgh88AlFOcXW0ms0tBeF64z3DjcEARp5LkERzslEQ7YK1iZANlxQiucQdzUtPfJKqiR3rEWUlx8fZTJlD9gEHal+Fdw5Yxy6NyBfmYipSbx4fDVDQb4s8SqAHMLmYHjHYekSw1GkuC3rZUfXuHPP5iK61H6QfRjRdDlMNx5CbIgrjSSZIe4qy8zKWuNrU+9+gMgXDDF78hKN\\\\\\\/gvHklohduNDilP\\\\\\\/l7nO7YP5L2GZZ7LcyQSIL4V0eafjruvVa5gyiUa4a+TW7neyQbbCWvHAQqoKyX4SVPWSmoj26hZZXAUfpavgccDp7Qxcv1O88yhJpr2qctJOPghtu4r\\\\\\\/u+ueTfNnQc\\\\\\\/Z29V8KyI86oeZkEJoHKO2Q7W0slka8v1\\\\\\\/+MF7Na6+SqikZn9kA8xEGCd9o49BYq7LJWFG3HI40YavJhv3DuwTkfDT2mOTRZ7vKcSMhQbOdTT+gDcq98ssAan959sEwNehnBp3C4gqZPuYsWwqtY\\\\\\\/x\\\\\\\/SOWuY1CW6oe6fXQmCbsK8uRYrsLpK0Y3FaoxN6V48n6wGIjsnwhjdkSlz2I7CLYtNMEx+zdH1ykwEKYPVprA\\\\\\\/7TGAHpecjHd+3zoxumGJjKPIUdpODGtVuWXaJKAbw703wHbWlkonESWWRrXgQpNfxGSSVOuLKyVgty1Bg\\\\\\\/JJ0FgL95KgJhNoYM1bVMxD86WApbAy5yvbeHJ9pSmZUJl8hjlp\\\\\\\/hbpzCu8qq2qFfA0h5vzIFTAm7Ba2aCoVB61dpDS4lNnUaXhz4ZRlibykzSgglG4EhPABF7CY+KpV5RpMzbIr8N8KLS4IjGqst5TiJOlpkNEuk5ujQrw4faSTciSyUskGWTVXuT+7hJc92a\\\\\\\/D33ly9FT7VZdawMsK+eGvgiZIgcx6MTsilk0AKdYcWFAGjjzZPIOUl5kZgywdNadIAjF5cLo8hPFtxjEjoYFuppeQOIYbGgjyZWiWUQDnFTWfIiFU6MfT7CloPV6C9a5EgYe06tQit2ue1NStLQvYdRCFlaOGhfrxk6c3KvmiqDZHaaj6JFi6x5OytdAhrmxP8PxNWDRedoTiQPXpWIy6fgDEbVyNxYfOOklMjxuUu3yiQBq56PzKvDvE0nMdsO+C\\\\\\\/\\\\\\\/J6lwKkobGEXEFss2svZPNK5NWPyJMCg4SYXWcNsr5KATxJU5q8xWcLu6UoNUvQ4ioYfygSigq\\\\\\\/5UfaRq\\\\\\\/3vjncXjJAjUTsKxusX7RI4u1WP6HMlSv0MSVo7dwSG8UwwPzfLlm4VsdpuRc5t\\\\\\\/5kD5aaEJx9uZ7dVC1pckMheyVk3S5Yleii8IvPBUEZ\\\\\\\/CQ9xecrAXguETXYXtWUhSygSOCSdGqIDY6MvtiG3AP6H7tlTaJFdWX6XKG1NTGG9m9I5O8kazRxkp8SA5vMeaWkAgSZN\\\\\\\/zAxDIIDKnMAuZOJIGSSobMNBvx0v\\\\\\\/vGKSjVlDnWFuGo0ZdcX2FbCvUsM5L3E4H+EnHFsX9bOhmemtaueHU\\\\\\\/r8tkZDjHqSPNng43LeG+7tZW8Pbgic9hytg88dcpFv9mJE4E5Hoyl8eucIwPJXxx93PGiEsxcUmGTYq5CxKIrUxjnZ91S208T6oR8IDQzwat9o0hKV3gBb+exp\\\\\\\/2N9ComGOIH\\\\\\\/dbspM+5be54WqHzsx2O3Exual3Yp0zoicLhy+gGEepQ1G+AcZ5SoYLPDVSEK9bYbgpTivb48NrabSlDyscMZMzFHeTDDaBs0f3K5FtWBHSpG6+hSXWhbiju26AGhkAxZLYQs0lDwvZA7cdBC11B1PiOb8y8EhTD2kdqoUruX\\\\\\\/V6+GQNdq3woj2NSRvNW1zB4sHLlicv0HRHEDwAUPw4CNlLgslDAAAd04DDkW4or\\\\\\\/O90WignXwv384pdeayS6b6uvCH4bSOeqoBE2lctuQleeRWLaWYViO9sy7AxyLcG7rGIn2OejpOSGfD2pXuMY3JNFtUajTu0QMUn0MiHvjAYkp\\\\\\\/VGzxK3K0WHCEA1i71uG7TQwgevD+TZmVVGleleEAOFTvchta1hB2+yBdg4iZgp1BoiIl2dl+FvfKyTLQWEsCo1KOJ3qSFHHwArPmx5NfmB53otl1tGKuKi7CfDp9BDiDLaJgU1RbpIj\\\\\\\/Nd5yPhzkIpWCM0vfy0V8yUDXuP\\\\\\\/w4PBg2YbBQg1HTbCI7aEc1IihVADPjlWYVzh5FMPvmRnOzMaj1NBhvnob8CulQt5TWhxtUboj7G3qlzcoY\\\\\\\/QsqbxMotLHtT\\\\\\\/prQbxXwwAEaVp7\\\\\\\/AL0YB7te7HyFoRo+SyInqPRCc3qrVQRNXv7lTqIuhMx8ga5LO4Fmd4apgzCHy0C9RhMHS9phZz0iLJjn5iXOVH9Z65S42q9Lj\\\\\\\/FDTji5MRQGLnsPURtiWaMuZG0e2F0qqesUnWNIBLEwOwzgr60hNNzNmw0FAfivbtlkMh5GZ\\\\\\\/AID9ZorMGsj8W3wCfY\\\\\\\/QLdwnn8mKWw\\\\\\\/TJxg+OjqpF3xnGIs2d7+kG0J3g2P+JTzZmfTkMnnF7IcPNjrwh113tF2z\\\\\\\/V3HLM232fnFN\\\\\\\/BL61cMZbimzcMg0FBMjvjRGrTqYH2jThVybjmNT0xYI2tPn1T5J3CSLGEf+qV3kSSfz7CxZMUVtbdGa5obNEHmmuUjDDZ5Erxz6Tf80LoIR5QuWU9eJozr1eUFNE5JufACuTnLT21mj2EdT2dOCmuVy89IfhIFYqjb20esZEM+9CYCUrE88WRJtaGVXyQobSmKjBBw9J5zLhXfwGM9i5soPKD6jdkdTQQQWrmwTYDbbe3O2vjPg24PDOd3dSK439se0jDtGh6d\\\\\\\/oHEyAGK6FHcBViGfa7+uI9IaFYTGFud7I30BPY17AELTol7UxemT89GnxUqqdpBhzSk87gYLRrYsVXqZvkqCM8VeXm9eGYGk1IWjwWXpHad0oJbp85p+KtrsiH3EBk04jMLBMZWc3BWbdiBrbDmnasIRXhLYyunGOgl+yTmZkRfvlqpErVbFMKsPHVut40k8hskoVQ1eNsTs8Tv2DC+DHWIPOJ4w1p+yMSntFSN3Euh\\\\\\\/LTsyhR2kzkkNDeTLBbATOy8Gd7YLUM1XNlZGp6rK9HHiNr4yv\\\\\\\/2oJCuXqXjKsuqv1Ybgant4TB6XfHhPblNCUfxIkBEuC4pxeX27KB4YFRcYCGpIbCYz+o3\\\\\\\/AfutOHnVvEiMvMWhSWfYwwvI9UD2bbsyQfMIdIkaanJX6ocDGzsWud55bchznb47SDQK9bKz0G3X03P6RpqtxHHEiCvLYWY7H+wt+faDRZQlXU6MpdYOH40eCfvGGKVFzydZaPfdt+b9zKznw5ivLI4Pcs61dAB1xYp7Aw6TGUfEKhINE83QT1W9Q+g1y1Z8k52aJ91t4iBTaZIiQvG3TVjbgxzAodw68pb\\\\\\\/d7PvGLBqrm2C5XMDtXdoHHNncQH8piDyS8qk6Gq6N2eQ6vohJQsgUTtae9Iz+P82lMncP6ZYHEpUESQkYdEL7yEC7yzSSazBYJTreNB6aBFLuIF\\\\\\\/m7aQNJOtXEQxg3KXTHNPQVRbrT9mM8R6pw4MIZ+LrwWRAsKVxYc96pawqHSitR1rwTl04gtcQtMJo\\\\\\\/2wZcMTJbL5Zpq3rpQzyXnmOxeU3zqChsMArZURLibb6zFtg6+54UxSBb7jxrHQmnaMuhgTjthLJ2owZWtfdIk8JgAoG7X5wHPMuik4BslPmfxTkTeI9yr4TU0pY8b0wHEYBgtu8LU1qssMFgA8O2zaWECS0cYHtEERizR+q04khM9MV9xZcz1q5xpJnGNaSa3GCXG5E1i9alQjyijROcy5gsPGqpqnuVNqZvzk7PApRz3MCBOYZTBkg1XdD4m6DWtuvJaj59mHgiTuOpFScfY4jHzrIKCk9wDFY6TdwQ1DQ9bLpdNxKXzru83T5KUnLCrnkFPVU2vgyeHD4BZeJ6iC5B1O\\\\\\\/vkjIz1iBLELkDZmEZE83dfaUFrdOOK1X2vJq\\\\\\\/CL2\\\\\\\/xXyK4wgduOVqntYlUvI8NjTyQDcqkEDi4Co8tQ3zoxSSVzErxdcnPB\\\\\\\/mQ+eE3oGDnmS2g2CAWRxNlrzjrpLbwqCSkx\\\\\\\/tO566gl\\\\\\\/EwSfvSiXjRXkRVxHJtgzmu34Y4ZiN7ThhtWOOgEh8WIS2GSyV7snyjWq8YlEPxT+kWho00HlEaFgHDY1Z4NdzdtDIlUaaTOhB5go0YpaANVA+OXpdpNE6OgieS7LMcugdSS3XXRySv3td7L1MH2vG9ErfpXaEpa2AdtscMYOk1lQw6lAz7QsdrpFbyi8lRplrIzrMVhEoNzLJVZN58BZGNR2p5GwePoQ0T464S0tig+bqOKZOOCeDopMO83T9ZARpLo5z3XFfkmX+6TLIvJZZAtvV2ZUNWgTxyQ+vMLKDGDXsvilNe5JiGx4LfJl5yoIavFxIHfFYxdwk9iXZsodWF24dxDp64GYWxKkAbgp3VUcUcUka+ajFz8ujcFE1DnPgfVXMJ1adCZcSRc1yBJXwvauSIblDjqUwZIeuRR5eYR+eIc3bYujDGa3VxIBM8k+MKNwLp03hC4DE\\\\\\\/nUsHgDZU7NbcD\\\\\\\/ELntzwlk2YPr24xxOPZy4b66LUu4C8EOovMx6ZrgIiWeG35jwITinYHwhOXSTFfA8iQjs4nZ7YaaCnfNNYpyadZ+MOq913z20KXOS92XQJzy6QAEOJaS7idv0T8xKZLT8P3AHI6SVK0TnBcUyxceQ0ElaWY9TiCKgF79830tUt8FHPApZXRd8gaTdMAaOAxAIiv3Ke3\\\\\\\/KDfoI6ctMIEJu3CYXd06iLBQTDSZ6r8OOiCafmRXydJtYUrx3no70T13mGaruSnr4rIYmFP3PIqfhLCZyRUAgFPQrC5djj2OP5+GF7WUe\\\\\\\/v4SNOlqqNBOEWS8NfxDXnUCFx8W7b2ysuM\\\\\\\/JR7WjvrGK7bcIQrRmao2W6OtZz13PvGzU61u0dw47y2YFW598kPPsrSbPmKuE1eIB5D1NRod0QA\\\\\\\/9NeA1g9c37Cyz1IZqxxFBFlM4VHWmrISJ9dMBbmoF29NX43EHLYiDqUzbQEgraiZ7QohLFFrBRL1MhM9scBQmXo5HXaA6T4NPYxv3iMTP1ZAQXWlqNCFnXwSRhh012gW9yuk0UAS\\\\\\\/VqpY\\\\\\\/aLfMRayWoO9qMyE69jVVAycH1yF2Ew+qRtZJsa4C+7SuTNnoh+ht+mfTR1OQj5rC7yC714jGkEnR1LzDKuRqrm7d4MUgsU++W2ErwyNS6llCI0kaZ+4PpSiWQ\\\\\\\/\\\\\\\/qATcDKaj2l6ilEMiUJMEhNhx90Cc48IR4J4+0yjMyt+\\\\\\\/JRcNyhYfUm45k0suV87dFXvY32FJYOSkERZzMlbmyIjInqjuVv3Njugd2cWbw5o4Ujh41g6mMwsAuFBfeZq\\\\\\\/YJNrpfK9zvDYKQfkpxFK7hSB\\\\\\\/xmmy02oS6kpIS6\\\\\\\/nXfGLio483zd4J99BJ4QX6OiT26V6A84lmbAxk3Xbf41w5JKgPo1V576rd07bqck2\\\\\\\/9nPeiz2MMmSL1Zq1A1wukMTPicey\\\\\\\/un35pUKbQWcnnurEri9egvrnbNDctR4vTmeTJ+OQ1fW6oybNe4Cg3w\\\\\\\/iWER\\\\\\\/SZZfa6uBdOfSmNgSgo6iBWyyBGCfEqmnNQ40Xmo85U8sciHLdQW9O5RwzHk7O6HKmgRmrFnSbvxi7O5ffVfRmu50Xbo8I4bWR9IwALaekp06Ovq7byDmBO2gb9pAiPa1WcPt0djDz16nrIKLLH3sEJ58DQN5ECZa3e5JQQi+caTTWeGCeEmkaPMXqCvOIRHWHNmrFawiiLzU9KmJK1hy58kwlm1btYHChve1RMYDXH3iEnKImr2MpnK\\\\\\\/R8Kv3pZNKJmqR1JeleNfmhSzHOOHFOcwXFeu9J4hojGilvsJBd2VwfJhw\\\\\\\/gw3Mi\\\\\\\/4HJllhXf7ApRBAIiBTRGhb0DNUCk8FUmAomZ6iqG+SBYXGwTW0gSatOmt9yo3oELf6v+r7NgZr9hWcVCf3VibppFMP40MzRy0OKkndHF\\\\\\\/APjVbNE3nfo02ZBlVvzy0SQwu02iP6GFDwtrgn9riLqyTZc7cbf0sf84gOxgIJMZEBQQtk7AfBo6r9rwZXlkIOjsMW1K\\\\\\\/\\\\\\\/EPxS8BnF2rNFzNZK+7fa8OOwA8okuW9yb8GwmDzUePRIsp6r2+Rwgr9NS5VNHAkQUwIE7jTfvPmwI1PnfIe3qToeLtqJy76Mo5dDyj1yOScfozqSbyYMXy20j261T8LnTaqYSUzAdqqjEMKZm3YQFp8yK8n8lsXG3Io4cBu7q6fiT89+mbmRrgUFuOd+CpExbcnhzp6VIPMC1h+hMcHQ2xGBzG9P6ihfNDEEk+FxTUYMHumPfoAADpAjQC5OVNWcTSkbO063VxsrELu64Kk0WianwgFHYpwhao0z3lc+wfsg0OMKKZQtLe0SQuneaGFcXlW5UI3uZAwl\\\\\\\/jtnFEEYn0mCZSRlP7GFOemofnvh9M0uNPFI7P1pNR2kCh5VgUr2skyq9qg9sFxtLJfmlgU702k5jBnogPHTamJEp9dXo6ns0TLbjWGY4mUIHF2p4OXVi+Kw+\\\\\\\/6VuBMDIyxlfWXrjbsEdAnCAINakMbbKnIcoL1E5ynPwhIfvn4ZqR6V40amROMvKmX\\\\\\\/wYywb9t7sSUNZvfxnkDbuNpxt5E602\\\\\\\/YmFJc\\\\\\\/J6rpFcWP1fcGqEEj3L4Y5TJ39UNuybydN3mbIn7JuQUvIAsWZKAfWYLl\\\\\\\/\\\\\\\/YPvvtinb5HFArXUiwn7elEh2OFM+6UdwizDpDVeqe2+JPx0d7fa2myjEO9h3DAMSRbARUaJUvjSkQDcNgrszhKPwVko0wQ6X3QoFk7to1xxBH+isFKEblKif0N\\\\\\\/jsGg\\\\\\\/Vrnt9Y21W9OAapiJjwHx9yxDXV\\\\\\\/N7CuMwMfEPtPYVrWauAcmoqhMwO2aSb260ak9OLvSOF6CWr57WzZFPyR8x3TzXb2uWV2RNIFysVBiPA4pORQjzt+RAQuwW68mhtEuE04UZTdODSRa1WelrbwOloPl2uH\\\\\\\/3a83aUExtDwAzr3PYxT\\\\\\\/uN5xS7rnJ2icFSw1Wt80nEk\\\\\\\/6tMN37bCouP1vMCM0OhMcaMlAnHcWKkHMhvzsNfFldJYdPW1WgQw4tS1Q4Ky0262I84MMC0pP5lrFHVMLS1LAsKEdV2QuPyE7psPNMIa3a7mrVKXmBh6zVF2z9H9XFyvqJR2s1\\\\\\\/xIMTTyjfdCpViAEUUIvDaFTtCji2mR4Qgin3QVYU5+TYYhCPD\\\\\\\/8DqIiIPBciQsn82rmGcYdH2Lm0WolMPpS5BuikrWy3OgO3K8CrWvfhUP9nx94WmWfXXYQGc0+P5XowJkxgMRGFahIv4ZsLf8ovYdjniADXAiCzMbwt9rq24IzPSilt0kdEubPY+mlyvb3VwdtKTPeNP9Gp9e2Pn6+PYkzhrfrPRZTxy33s+q+j4WWfsCFvFk5dwXRswVO4GpnKJiWL6MhDGHFYxm9kFnNmiRQ0uwxlse8FLVp+AF0AJjRSJqIVYpp8Tz2aqOrXIXTveGU+NwNEBXIe4VwM4KPC7iW4o7gFD0yrij5f3to2zIUcMu3RgTFiOGLCjfeIXU3DEno7mqxRmg196Gv8maVFsR3PN\\\\\\\/UinopOr7WzDuKozLeLGY2KFu\\\\\\\/CfrRPp5sReRHRcyNCnvsCq+8c1wP2Osw4lk582Jd15\\\\\\\/fceBl5gIbeqiu8FfhHWLRkPv7fm5uX64L1cUrPlzO0Zb3mWJVLN1aBykwrtJAyz6iyk7YX1Tlkhf6PkkI3nlizzddKCSojuFbdMvPBPpS8xusDk3SEGP2E9c+Yqa9GuSojupMNzHw\\\\\\\/OTgTM0zMAT\\\\\\\/BJAwGuLWqEm1SjaoIdr1FZ4Lr5UUH4kqqtBJVMaI+m47cJcaQ3IP7vrhQvBM90YFaLSC0lDg\\\\\\\/4l3ksmbXvGL9VTZmmJFWLsM6ET0s30jSoiAl50xwxmkasDHUY8DRKARrsAtyKtpTOvAFTPyumlaYOSx50rw4p7RVV8LrBmg76\\\\\\\/eUj2Aw56WmNc4PpWLckGUHDRJfizU4Kna3X3P6tkDF0BjQTjFEse3NtH8YbMUlcSyreqVvq4biqEH\\\\\\\/Xds02NuAYUL\\\\\\\/7EyA0a9wfZGTrTo+vyWjYlWnjB+Om3PM0QmecT2fiUJChhbFPLsAUAzv8Lz09r5VGE+6HEQdxhGS\\\\\\\/nq+5cc6V0l9s9iSm8ar\\\\\\\/WBQlnUKhyoxsmnG2k4\\\\\\\/pH0PYX1YEVHcN63HMldXdEO1XoLatnvwZ+kcZ6MWN6VzeS0cIZgwFUOcaVJpjqUqMuFfWwfSozbrxrjAEtwpeZPGijDJ47G2vgoJmsGCKd7zlSjfMrPoHh0H0xOi0RNwNB7kU+8lj71jwTGimFlKGGv1Qd3SF4fMPuk5ZX8A+2rHj1t\\\\\\\/Wzesc3+EjvaF\\\\\\\/xsIeXJiNuhbp9d3mbvkfMqIQw47D\\\\\\\/VgsD+lbdDdtYf0NiVYDrKMwtLCSysRh8d8CrF1Pc0LGlA9HqmutRzTxIksv2yKHjzFWjKTxb9T5Snraa8sByZoUQPATPS8AC3TlkNdIOkDr6h8g9bszQOMHFbQnXP6UdhDXbq+ab5YpS9dA8eVxF0gnmryhcJuFomjK3J\\\\\\\/UhUZSUmxH1FZ3nv9CC2v86VZkQQdmTu7nQuVZBBTpHgNg1i1+oFmWltIGW3DGoIoRUx\\\\\\\/kQ4sKlTfvc2nu7mmm3CCqKTlq1EvxRilzdoCKnawrHAMs9SM39gUbVO3C2qiEyBQU6MBFQNNChBAVEg0TOZD9FG9vMu2DqLfLpZIJS+g8xtUTMWKS20ADFHkc+yhaBQ0hYssVqltgsLWCyjqQct2HqLBB036v7xrRsn0c+BfhhMWNanaFnfY6cDfF0YZsh1tq7MRynmR\\\\\\\/Fp9g0uCgSyoZjUm\\\\\\\/ZZl7fhdW5UuPDIoOz4BdzFuktRgcQrft0pavNZdJsaizO+WBFhwDg6iAqCutSBwN6YttrGRBmafFetO4d8S2Dq9R4osBD\\\\\\\/vijYFu+Nr25bUKPUeAROlUDsyQ+9pOS0c3pY8NFMdgY\\\\\\\/94\\\\\\\/QquwSvHDy0K64SiwvNSnhEbWKgM57tKR\\\\\\\/FBevoInPQ\\\\\\\/Qj897Keye6QeGI+\\\\\\\/pXWXQak8lWOSKzMnrPIn2ip6ebdjTeX59SnizRdgQTr0W4IMtDqiP\\\\\\\/CsH5RY+lFAe+k6GEhq3KvpXeLMiNzwf006jz7GveU3B0ttmoJeW8qX+a3MhnXGR5eqhtKzHdbPB+Uf5gHW6O8afxVe7EpdDxdbR7Dq2ZU\\\\\\\/DTXUokeD9XWoeeU\\\\\\\/GBRQezxmAI1fM0N69h\\\\\\\/ydNh4P3yJ6tY+TP3MgY1u2HASCGi0Gc9EJltKVgtgeT2Ua28CbbePvJWsxVOPxacVHDJsiNrtGsNVvaYJU1dusc228e9DSApyTv87Shp5hEg0sQb3mclR\\\\\\\/Kn2rjrKuUP9D6uzeW2tFSxA3ZYuG2Wmhoz+E6Sn6rVM0NHr\\\\\\\/B6om9vm3EtOEfKbCJPeH6o6Zur+qXfKMOnWkLmbNJdRn7i2cLIYVHsd6ll+\\\u0022,\\\u0022iv\\\u0022:\\\u0022XwHxoacdrJpGB\\\\\\\/lKpoutnw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229747ae313734ecffb8335de8f8a7a9963cfa60bd0dee2ab61f2f4c08b458bbff\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zYI9vdFlx2p+QtzmR4+6yaHYa5MVFGLHSdB1efOjU\\\\\\\/BX6wul5b+aNmeSfO9ZLUfhBgewz577+Wb+94dJ+HupTJxsiNGo9G7pBBZKjUTabBMNA8jzoaB697QcfZawGdoATX1JuqnGEIeU+dOg4kxU9qFoyP2ZGbJ9UpiwNaNxnptlVq935hx\\\\\\\/4DjYzOOvhosSCGmjFc2i3q+hcSFnX7ZzO54CGSvWsTfxPowlrY5JaCBdJVgzAnc3PEf9srGLG97m2w1NrZnX7V9lLJRtvXUDv+c5aw4FUWB+fvo63V8Mc9NTud\\\\\\\/eznThfv9Btrdch+kZdg6IelY7g5rYG3uClJRVFZLGZtkMHH5RRaVsyt3EHJrdxdvYHsGvmWe5tqK7FDcVeAQ3JpsJbDPCgTVpR9qM5AqNHWEslaj566vwiK7I7sn2rT7hU3ux7nu7XZGcwHYNLxTLYnJIdp24P2zQIY76W8o6f1IaooPSPHgSk1o7P4XiLdQm3v496HKsF9tvUEiWENx5ldz+Y\\\\\\\/ZHfOKqVWKhwgnSuR5HbFjsTtrhe1fpDEcodRRmzeWFG9aeC0xO5QPQyfxRBaQGjrMhDmNgRih9dMe071uoqLBqxWMiX93eC+nqsYh5QmFiLf0PxNzSpqmlJRLii8OD0Qqz9i33raR7lS0i\\\\\\\/M6Dh8C\\\\\\\/lmwnv9qhF1pizLNYInl6eaI+nZlkms4ogDasEmM51nepmHkpIsLgYw8VX4kCm7cY0+RNiyQQ1uBcEi+DYlY5Zql8J9vbk6\\\\\\\/wpl1zFwPFdQdqaN\\\\\\\/pIImep0T93Q6wE\\\\\\\/MURhW\\\\\\\/T5Qtf2UhWodqcAfqPaDAN4JLa6zMJHcW2jH9QKjV8\\\\\\\/jpFAQqKs2bJ4I3Bfj+bleCXhopj4pJcHZGPnCk4hMOnKrAvgPCt8tYFjWQwBKMBJ+xDFPZSF5ZxgQEH0gGrP+vm8nCCxJeNj6lgdl5RXafoY3z1R77SJ0c+UAemXqvYsf7FOjuCEAv58jFs7AHzhbKYgnaHqB\\\\\\\/UoEHp3ySMyx7zvWDCVzQjuJ1n7DJRx2GYNiv\\\\\\\/aQxdskD9VQrKWM1hUykyCe17pSPcyyYQzRtSDqmlh5SW6bebq9Gq9oakPsjTCP3ugSPaYHtCgfl9rmpGlNeHMAQwNp7DIjPJuy7Rd6PHW9CIuJuqfQajM+wx7\\\\\\\/199l6A9HcHbZFvO0TmBi3XFOOGsMKt35zfFa00dP2JTWhOg3HnAPgC1sGSyiX0j5A\\\\\\\/U8CCowCGxNVWdSWQrdNeCixzweljjdvPODgC4y0M4VJQFrRVlcoh0bErQfNbsz6\\\\\\\/LNCeFTxxzh12pgeVThLBfD899\\\\\\\/thQbYYW3HCGfiJ0+E7Om5omdicaSgcwvitYbl6ZZF\\\\\\\/g02wAEXehJ8njqED9h\\\\\\\/g5CVLYiOckPkrQuj0G\\\\\\\/SADJ9eUVzROVKotYS3e6ciNWXVvIf\\\\\\\/qm82v7vdwJ7Ih270gR0yXTo\\\\\\\/mARorrmRAm2I+K8XLBRp4bxYxBtSZrdMH7DuwPPX6ShGgHub9VIzICWG\\\\\\\/PRnPg5LUt3GkJ1woLD0vBtXbeUSogRTv+j8r2XH\\\\\\\/m8lTu18MlwV\\\\\\\/5QLkpmG7F0fpfmWUHVEXcaZZyKgUxocBMAaFgjti0ukvt4uJniotDVwT57Dgfdw9\\\\\\\/JsKu3SHysodQq\\\\\\\/1odvekSkiTAdyhGh3o8xg\\\\\\\/JB7n0tspmBFenE\\\\\\\/oNfI2JjDyQRAm59FvghH1iyXU1J8lrNwu2jI\\\\\\\/rI+za8l6k\\\\\\\/T6taP15FLHkbV6UAwDfuHbVcyaT+e\\\\\\\/kzMRWBHTBQGrDdN6UgVtUR5lPOKkapnS7RoQzDFfFUrtJ5ydA9teDUl4GL8CZJ3HBjOhFS6piHmHasuruUqS\\\\\\\/euyvVVOwmvatb9\\\\\\\/0bjgV4nkdDsMnekKWeyOjFkJT9VgzeFtG9A3scTq7+UxbsX273Ai4bSjMM\\\\\\\/CTJWVXStBVyVPVATyGC\\\\\\\/UeznC72zs6fxBCuCjp3av1M1dJ1gPQEsUSFJhQxjHyzzy4yFC5Bk0rjonTVgCedjwj60lo9Xh3KgNE9eATNS7C5EMXCek=\\\u0022,\\\u0022iv\\\u0022:\\\u00225FqkavQn2y415p70n8Onaw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226fe560c52a911f7eea0341ae710db6ec2958a3b0664e2abe90f8c530b0d987b5\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459294 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/7/753" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FgUcYnmY3mVbT7qxvqBPEXyYYWoQ8h_0Zx6rK6vto_E" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/7/753" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zkZrfaQcn3C6hMIQC0ypQXByfR5r72FNmeHfS93ltaw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/7/741" title="Mutations in the pqe-1 Gene Enhance Transgene Expression in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/7/761" title="Impact of Loci Nature on Estimating Recombination and Mutation Rates in Chlamydia trachomatis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459294" id="node459294" data-pisa="ggg;2/7/753" data-pisa-master="ggg;2/7/753" data-apath="/ggg/2/7/753.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Noncomplementation Screen for Quantitative Trait Alleles in <em>Saccharomyces cerevisiae</em></h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hyun Seok Kim</span>, <span class="highwire-citation-author" data-delta="1">Juyoung Huh</span>, <span class="highwire-citation-author" data-delta="2">Linda Riles</span>, <span class="highwire-citation-author" data-delta="3">Alejandro Reyes</span> and <span class="highwire-citation-author" data-delta="4">Justin C. Fay</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">753-760;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002550 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459294" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hyun Seok Kim</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Genetics</span><span class="nlm-aff">Computational Biology Program, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hyun%2BSeok%2BKim" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20HS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHyun%2BSeok%2BKim" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Juyoung Huh</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Genetics</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juyoung%2BHuh" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huh%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuyoung%2BHuh" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Linda Riles</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Genetics</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Linda%2BRiles" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Riles%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALinda%2BRiles" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Alejandro Reyes</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Computational Biology Program, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alejandro%2BReyes" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reyes%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlejandro%2BReyes" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Justin C. Fay</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Genetics</span><span class="nlm-aff">Computational Biology Program, and</span><span class="nlm-aff">Center for Genome Sciences and Systems Biology, Washington University, St. Louis, Missouri 63108</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BC.%2BFay" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fay%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJustin%2BC.%2BFay" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jfay@genetics.wustl.edu" class="" data-icon-position="" data-hide-link-title="0">jfay@genetics.wustl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/7/753" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459294" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459294/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/7/753.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459294" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459294/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/7/753.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459294" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459294/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/7/753.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459294" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459294/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/7/753.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459294" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459294/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Both linkage and linkage disequilibrium mapping provide well-defined approaches to mapping quantitative trait alleles. However, alleles of small effect are particularly difficult to refine to individual genes and causative mutations. Quantitative noncomplementation provides a means of directly testing individual genes for quantitative trait alleles in a fixed genetic background. Here, we implement a genome-wide noncomplementation screen for quantitative trait alleles that affect colony color or size by using the yeast deletion collection. As proof of principle, we find a previously known allele of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003387">CYS4</a></em> that affects colony color and a novel allele of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003320">CTT1</a></em> that affects resistance to hydrogen peroxide. To screen nearly 4700 genes in nine diverse yeast strains, we developed a high-throughput robotic plating assay to quantify colony color and size. Although we found hundreds of candidate alleles, reciprocal hemizygosity analysis of a select subset revealed that many of the candidates were false positives, in part the result of background-dependent haploinsufficiency or second-site mutations within the yeast deletion collection. Our results highlight the difficulty of identifying small-effect alleles but support the use of noncomplementation as a rapid means of identifying quantitative trait alleles of large effect.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/complex-trait" class="hw-term hw-article-keyword hw-article-keyword-complex-trait" rel="nofollow">complex trait</a></li><li class="kwd"><a href="/keyword/copper-sulfate" class="hw-term hw-article-keyword hw-article-keyword-copper-sulfate" rel="nofollow">copper sulfate</a></li><li class="kwd"><a href="/keyword/hydrogen-sulfide" class="hw-term hw-article-keyword hw-article-keyword-hydrogen-sulfide" rel="nofollow">hydrogen sulfide</a></li></ul><p id="p-4">Identifying genes responsible for phenotypic variation in natural populations is difficult because most traits are influenced by multiple genes and because the effects of each gene must be mapped within a heterogeneous genetic background. Both linkage mapping and genome-wide association studies overcome this heterogeneity by measuring the average effect of a gene over a large number of samples. However, the two approaches detect qualitatively different types of alleles. Linkage mapping often reveals alleles with large and in some cases epistatic effects that are rare in the general population (<em>e.g.</em>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Deutschbauer and Davis 2005</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ben-Ari <em>et al.</em> 2006</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Sinha <em>et al.</em> 2006</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Gerke <em>et al.</em> 2009</a>). In contrast, genome-wide association studies often identify small-effect associations with common alleles and find little evidence of epistasis (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altshuler <em>et al.</em> 2008</a>). Although many factors likely contribute to these differences (<em>e.g.</em>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gerke <em>et al.</em> 2010</a>), our understanding of quantitative trait alleles depends on both how they are mapped and our ability to map them (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Rockman 2012</a>).</p><p id="p-5">One particularly undersampled source of variation is rare alleles of moderate or small effect (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Pritchard 2001</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Wang <em>et al.</em> 2005</a>). Under a rare alleles model, alleles segregating in one cross are expected to be absent in other crosses because they are rare in the general population. Furthermore, most rare alleles are not detected by population association because power is a function of allele frequency. The larger number of rare missense or nonsense alleles in case compared with control samples supports the contribution of rare alleles to a number of complex human genetic diseases (<em>e.g.</em>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cohen <em>et al.</em> 2004</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fearnhead <em>et al.</em> 2004</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ahituv <em>et al.</em> 2007</a>). However, without a population-based screen for quantitative trait alleles that does not depend on their frequency, the amount of variation explained by rare alleles has been difficult to assess.</p><p id="p-6">Quantitative noncomplementation provides a means of identifying and measuring the effect of an allele. The idea is that the effect of a recessive or partially recessive allele will be revealed in the absence of a wild-type allele, whereas the effect of a dominant allele, typically wild type, will remain unchanged (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Quantitative noncomplementation has been predominantly used to fine-map quantitative trait loci (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Mackay 2004</a>). However, it can also be used to screen the genome when a large number of mutations are available (<em>e.g.</em>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Coyne <em>et al.</em> 1998</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Takahashi <em>et al.</em> 2011</a>). In the context of a genome-wide screen, quantitative noncomplementation offers two distinct advantages over linkage and association studies. First, it can be applied to multiple genomes, making it independent of allele frequency. Second, a fixed background can be used to make it more sensitive to alleles with small or epistatic effects, even when large-effect alleles are also present at other loci. However, quantitative differences in noncomplementation can also result from background-dependent dominance of the mutation being complemented (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Service 2004</a>). In yeast, the reciprocal hemizygosity test (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Steinmetz <em>et al.</em> 2002</a>) has been extensively used to control for any interactions with genetic background, including background-dependent dominance, by comparing two hybrid strains that only differ by the allele present at a single hemizygous locus (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/753/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Identification of quantitative trait alleles via quantitative noncomplementation and reciprocal hemizygosity analysis. (A) Noncomplementation is shown by a recessive quantitative trait allele (*) that has a different effect in the hemizygous (bottom left) compared with the heterozygous (top left) state. A wild-type allele (blue) is expected to have the same effect in both states (right), assuming no haploinsufficiency. Colors indicate two wild strain alleles (orange and blue) and a laboratory strain allele (black). Rounded rectangles show strains with different phenotypes (orange vs. black). Gene deletions are indicated by a cross within a white background. (B) A reciprocal hemizygosity test reveals a quantitative trait allele by differences between two reciprocal hemizygotes. No differences are expected as a result of interactions between the dominance of the deletion (haploinsufficiency) and the hybrid background because both hemizygotes have a fixed genetic background except at the locus of interest." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-962665190" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Identification of quantitative trait alleles via quantitative noncomplementation and reciprocal hemizygosity analysis. (A) Noncomplementation is shown by a recessive quantitative trait allele (&lt;sup&gt;*&lt;/sup&gt;) that has a different effect in the hemizygous (bottom left) compared with the heterozygous (top left) state. A wild-type allele (blue) is expected to have the same effect in both states (right), assuming no haploinsufficiency. Colors indicate two wild strain alleles (orange and blue) and a laboratory strain allele (black). Rounded rectangles show strains with different phenotypes (orange &lt;em&gt;vs.&lt;/em&gt; black). Gene deletions are indicated by a cross within a white background. (B) A reciprocal hemizygosity test reveals a quantitative trait allele by differences between two reciprocal hemizygotes. No differences are expected as a result of interactions between the dominance of the deletion (haploinsufficiency) and the hybrid background because both hemizygotes have a fixed genetic background except at the locus of interest.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/753/F1.medium.gif" width="440" height="300"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/2/7/753/F1.medium.gif" width="440" height="300"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/753/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/753/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461547" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child">Identification of quantitative trait alleles via quantitative noncomplementation and reciprocal hemizygosity analysis. (A) Noncomplementation is shown by a recessive quantitative trait allele (<sup>*</sup>) that has a different effect in the hemizygous (bottom left) compared with the heterozygous (top left) state. A wild-type allele (blue) is expected to have the same effect in both states (right), assuming no haploinsufficiency. Colors indicate two wild strain alleles (orange and blue) and a laboratory strain allele (black). Rounded rectangles show strains with different phenotypes (orange <em>vs.</em> black). Gene deletions are indicated by a cross within a white background. (B) A reciprocal hemizygosity test reveals a quantitative trait allele by differences between two reciprocal hemizygotes. No differences are expected as a result of interactions between the dominance of the deletion (haploinsufficiency) and the hybrid background because both hemizygotes have a fixed genetic background except at the locus of interest.</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">In this study, we used the <em>Saccharomyces cerevisiae</em> deletion collection (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Giaever <em>et al.</em> 2002</a>) to conduct a genome-wide noncomplementation screen for quantitative trait alleles underlying two copper-related traits. We mated nine diverse strains to both the <em>MAT</em>a and <em>MAT</em>α deletion collections and compared these with two control crosses involving the parents of the deletion collection. To control for interactions with the genetic background a subset of noncomplementing, candidates were tested for quantitative trait alleles by reciprocal hemizygosity analysis. Although we recovered a known mutation of large effect in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003387">CYS4</a></em> (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kim and Fay 2007</a>), as well as a novel allele of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003320">CTT1</a></em> that confers sensitivity to hydrogen peroxide, most of the candidate quantitative trait alleles identified in our screen appear to be false positives related to background-dependent dominance of the deletion or second-site mutations within the yeast deletion collection.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains and media</h3><p id="p-9">Rich medium (2% yeast extract, 1% peptone, 2% dextrose, 2% agar), G418 medium (rich medium, 200 mg/L G418), hydrogen peroxide medium (rich medium, 0.0038% hydrogen peroxide), copper sulfate medium (rich medium, 3 mM CuSO<sub>4</sub>), and PPG medium (rich medium, 1 mM CuSO<sub>4</sub>, 320 μM propargylglycine; PPG) were prepared at a volume of 50 mL/plate and dried for 12 hr at room temperature.</p><p id="p-10">Heterothallic strains were generated by deleting <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">HO</a></em> using <em>dsdAMX</em> (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Vorachek-Warren and McCusker 2004</a>), <em>natMX</em> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Goldstein and McCusker 1999</a>) for BC187, dissecting tetrads and selecting <em>MAT</em>a or <em>MAT</em>α haploids. Only <em>MAT</em>α haploids were obtained for YJM210, DBVPG1106, and UWOPS87 and only <em>MAT</em>a for Y12. For hybrid selection, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em> was deleted using <em>hghMX</em> (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Goldstein and McCusker 1999</a>) and <em>natMX</em> for UWOPS87. For control matings, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em> was deleted using <em>kanMX</em> in BY4741 (<em>MAT</em>a, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a></em>Δ<em>1</em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a></em>Δ<em>0</em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">met15</a></em>Δ<em>0</em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a></em>Δ<em>0</em>) and BY4742 (<em>MAT</em>α, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a></em>Δ<em>1</em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a></em>Δ<em>0</em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">lys2</a></em>Δ<em>0</em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a></em>Δ<em>0</em>). Yeast deletion collections within the BY4741 (<em>MAT</em>a, 4695 deletions), BY4742 (<em>MAT</em>α, 4680 deletions), and BY4743 (<em>MAT</em>a/<em>MAT</em>α <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a></em>Δ<em>1</em>/<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a></em>Δ<em>1 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a></em>Δ<em>0</em>/<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a></em>Δ<em>0 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">LYS2</a></em>/<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">lys2</a></em>Δ<em>0 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">met15</a></em>Δ<em>0</em>/<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">MET15</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a></em>Δ<em>0</em>/<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a></em>Δ<em>0</em>, 4670 deletions) background were provided by Linda Riles and Mark Johnston (Washington University). YJF173 (S288c derivative, <em>MAT</em>a, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">ho</a></em>-, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>-52</em>) and other strains backgrounds used in the reciprocal hemizygosity test are described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/002550SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS1.xls">Table S1</a>.</p></div><div id="sec-3" class="subsection"><h3>Hemizygous deletion collections</h3><p id="p-11">Five of the nine haploid strains were crossed to both the <em>MAT</em>a and <em>MAT</em>α deletion collections. Two control strains (BY4741 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">trp1</a></em>Δ::<em>kanMX</em> and BY4742 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">trp1</a></em>Δ::<em>kanMX</em>) were also mated with the deletion collection of opposite mating type. For each mating, the haploid strain was spotted on rich medium plates (384 strains/plate), the deletion collection was spotted on top using a Singer Rotor robot (Singer Instruments, Somerset, UK), and plates were incubated for 24 hr at 30°. Colonies, a mixture of hybrid diploid and haploids, were replica plated onto medium lacking tryptophan, uracil, leucine, and lysine. To test the fraction of diploid cells from a colony on selection medium, colonies were suspended in water, plated on rich medium, and 100 single colonies were transferred to selective media. For matings with two test deletions, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001926">MET10</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003387">CYS4</a></em>, all the colonies tested were diploid. In addition, hemizygosity was confirmed by polymerase chain reaction (PCR) amplification of two different-sized fragments at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001926">MET10</a></em> in all 14 hemizygous strain collections.</p></div><div id="sec-4" class="subsection"><h3>Phenotyping</h3><p id="p-12">A set of ~4.7K diploid hybrids for each of the 14 natural isolates (10 representing opposite mating types of five distinct strains), two heterozygous deletion controls, and the homozygous deletion collection were replica-plated onto rich medium, copper sulfate, and PPG medium and grown for 48 hr at 30°, a total of 240K colonies. Digital photographs were taken at 24 hr for copper sulfate sensitivity and 48 hr for colony color. Images were trimmed, and red and green channel images were saved using Photoshop (Adobe, San Jose, CA). Spotfinder (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Saeed <em>et al.</em> 2003</a>) was used to grid the colonies and measure colony size and color. The red channel was used to grid the colonies and measure colony size using the Otsu method, and colony color was measured using the green color intensity. To control for edge effects (colony color is darker at the edges of a plate), raw color values were divided by median color of the column or row depending on which one generated values closest to the overall median. Colony size on copper sulfate medium was divided by that on rich medium to control for copper independent growth differences. Each collection of strain phenotypes was divided by its median. Candidate genes were selected by those that passed a cutoff of less than 0.8 or greater than 1.2 for color and less than 0.8 or greater than 1.4 for size. These cutoffs represent phenotypes that were just large enough to see. Candidate genes were subsequently eliminated if the hemizygous controls showed effects less than 0.95 or greater than 1.05.</p><p id="p-13">Phenotype assays were replicated for 36 candidate genes that passed the same cutoffs in the homozygous deletion collection. The diploid hemizygotes were regenerated, and phenotypic differences between Strain/BYΔ and Strain/BY were compared with the difference between the controls, BY/BYΔ and BY/BY, using analysis of variance where Strain indicates one of the natural strains, BY indicates one of the parents of the deletion collection, and BYΔ indicates a deletion strain.</p></div><div id="sec-5" class="subsection"><h3>Interactions with genetic background</h3><p id="p-14">A small set of genes were further examined for differences between hemizygotes generated using the BY4741 (<em>MAT</em>a) and BY4742 (<em>MAT</em>α) deletion collections. First, the hemizygotes were tested for loss of heterozygosity at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">LYS2</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">MET15</a></em> for nine candidate genes (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002910">SAM2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004415">ATG17</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001732">DBP7</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003962">UBI4</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002610">RAV2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005121">MRPL22</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000870">MOT2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004747">RIM11</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005368">COQ10</a></em>) that showed effects in the BY4741 or BY4742 hemizygotes but not both. No loss of heterozygosity was found. Second, the effect of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">MET15</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">LYS2</a></em>, and <em>MAT</em> genotype was tested for five of the genes (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002910">SAM2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003962">UBI4</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002610">RAV2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005121">MRPL22</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005368">COQ10</a></em>). Homozygous diploid deletion strains (BY4741/BY4742) were sporulated, tetrad were dissected, and one or two spores from each of the four possible progeny, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">lys2</a>-/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">met15</a></em>-, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">LYS2</a>/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">MET15</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">LYS2</a>/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">met15</a></em>-, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">lys2</a>-/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">MET15</a></em>, were obtained for both mating types and mated to M22 (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002910">SAM2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003962">UBI4</a></em>), YPS163 (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002610">RAV2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005121">MRPL22</a></em>), or YJM326 (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005368">COQ10</a></em>). A significant difference in noncomplementation among hemizygotes with different genetic backgrounds was tested by analysis of variance using the model: <em>y<sub>i</sub></em> = <em>μ</em> + <em>MAT</em> + <em>MET</em> + <em>LYS</em> + <em>MAT<sup>*</sup>MET</em> + <em>MAT<sup>*</sup>LYS</em> + <em>MET<sup>*</sup>LYS</em> + <em>ε<sub>i</sub></em>, where <em>y<sub>i</sub></em> is the phenotype of strain <em>i</em>; <em>μ</em> is the average phenotype across all strains; <em>MAT</em>, <em>MET</em> and <em>LYS</em> are the effects of the <em>MAT</em> locus, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">MET15</a></em> deletion, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">LYS2</a></em> deletion, respectively, and <em>ε<sub>i</sub></em> is the error.</p></div><div id="sec-6" class="subsection"><h3>Reciprocal hemizygosity analysis</h3><p id="p-15">Deletions were generated for 18 of 25 candidate genes in each haploid strain as well as the progenitor of the deletion collection. Seven of the genes were not tested because of difficulties in generating either one or both deletions required for the reciprocal hemizygosity test. Deletions were generated by PCR amplification of the <em>kanMX</em> deletion cassette and lithium acetate transformation (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Wach <em>et al.</em> 1994</a>) and were confirmed by PCR. Two or more independent deletions were generated for each gene in each strain except for deletions of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000632">RBK1</a></em> in CLIB382 and Y12, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005364">SIN3</a></em> in Y12, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004888">BUL1</a></em> in BC187, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000659">BUD31</a></em> in UWOPS87, where only one independent deletion was generated. Strains with only one deletion (except <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004888">BUL1</a></em>) and strains showing inconsistent phenotypes between independent deletions, including deletions from the yeast deletion collection, were tested for linkage between the phenotype and the deletion by mating the deletion strain to the progenitor of the deletion collection, sporulating the diploids, dissecting three or more tetrads, and phenotyping. For an unlinked second-site mutation, the probability of cosegregation of G418 resistance and colony color or size in three tetrads is (1/6)<sup>3</sup> or 0.005. Using this method, we found <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004417">TUS1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000632">RBK1</a></em> to have effects caused by second-site mutations within the yeast deletion collection, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000504">YBR300C</a></em> was found to have effects caused by a second-site mutation within the newly generated <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000504">YBR300C</a></em> deletion strain.</p><p id="p-16">Phenotype differences between reciprocal hemizygotes were tested by analysis of variance using the model: <em>y<sub>i</sub></em> = <em>μ</em> + <em>H</em> + <em>R</em> + <em>ε<sub>i</sub></em>, where <em>y<sub>i</sub></em> is the phenotype of strain <em>i</em>, <em>μ</em> is the average phenotype across all strains, <em>H</em> is the type of hemizygote (Strain/BYΔ or StrainΔ/BY), <em>R</em> is for replicates obtained from independent deletions, and <em>ε<sub>i</sub></em> is the error.</p></div><div id="sec-7" class="subsection"><h3>Replication of reciprocal hemizygosity analysis</h3><p id="p-17">Gene deletions were generated as described previously, except that laboratory strain deletions were made in YJF173 rather than BY4741 or BY4742. Deletions were backcrossed to progenitors of the opposite mating type, progeny sporulated, and haploids of the desired genotype were selected by PCR of the mating type locus and assaying resistance to G418 as an indicator of the deletion. Segregants that did not exhibit a consistent deletion phenotype were backcrossed until phenotypic homogeneity was achieved. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003952">FRA1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000659">BUD31</a></em> from UWOPS87 were not included because of our inability to obtain 2:2 segregation of the deletion in the backcrossed progeny. Further problems arose when we remade the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003952">FRA1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000659">BUD31</a></em> deletions in heterothallic diploid versions of UWOPS87 and found that the resulting monosporic clones were diploid at the mating type locus. For <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006031">GDE1</a></em> we measured both colony color and size because it was identified in the size screen but showed color effects in the initial reciprocal hemizygosity analysis. However, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006031">GDE1</a></em> alleles did not affect colony color (see <em>Results</em>) or size (not shown). For <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005121">MRPL22</a></em> in BC187, we only measured effects in YJF173 <em>MAT</em>a x BC187 <em>MAT</em>α hybrids because we were not able to recover both deletions of the opposite mating types.</p><p id="p-18">Colony color and size were measured for each strain using the average of four colonies on the same plate and from two pictures of the same plate, one in the forward and one in the reverse direction. The position of each strain was randomized across the plate excluding border positions filled by YJF173. Pictures in two orientations were taken to control for a subtle trend in colony color across the plate as the result of variation in the amount of light reflected by each colony. Significant differences between reciprocal hemizygotes were tested using a linear model <em>y<sub>i</sub></em> = <em>μ</em> + <em>H</em> + <em>M</em> + <em>ε<sub>i</sub></em>, where <em>y<sub>i</sub></em> is the phenotype of strain <em>i</em>, <em>μ</em> is the average phenotype across all strains, <em>H</em> is the type of hemizygote, <em>M</em> is the cross direction (Lab <em>MAT</em>a x Wild <em>MAT</em>α or Wild <em>MAT</em>a x Lab <em>MAT</em>α), and <em>ε<sub>i</sub></em> is the error. If mating type was not significant, the term was dropped from the statistical analysis. To determine whether the significance of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005135">DUG3</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003320">CTT1</a></em> alleles depends on the assumptions of the analysis of variance we also used a Wilcoxon rank sum test and found significant results for both (<em>P</em> = 0.0062 and <em>P</em> = 0.014, respectively). Haploinsufficiency was tested using an analysis of variance term to distinguish the two hemizygous hybrids from the hybrid without a deletion.</p></div><div id="sec-8" class="subsection"><h3>Statistics</h3><p id="p-19">Unless otherwise noted, all <em>P</em> values were generated by fitting linear models using the “lm” package of R (<a href="http://www.R-project.org">http://www.R-project.org</a>) and testing for significant terms using analysis of variance.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Identification of <em>CYS4</em> and <em>CTT1</em> quantitative trait alleles</h3><p id="p-20">To evaluate the capability of a quantitative noncomplementation screen to identify quantitative trait alleles we examined colony color in M22. M22 colonies are rust-colored in the presence of copper sulfate because of a recessive, loss of function, nonsynonymous mutation in cystathionine beta-synthase (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003387">CYS4</a></em>) (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Kim and Fay 2007</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kim <em>et al.</em> 2009</a>). We mated M22 to the yeast deletion collection and examined colony color of the resulting collection of hemizygous strains. Only a single strain, hemizygous for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003387">CYS4</a></em>, showed a noticeable difference in colony color (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/753/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Identification of quantitative trait alleles of CYS4 and CTT1. (A) A noncomplementation screen of M22/BY4741 hemizygotes identifies the M22 allele of CYS4 as a colony color allele. Hemizygotes were generated by crossing M22 and BY4742 (control) to the BY4741 MATa deletion collection, and colony color was assayed on copper sulfate medium. The CYS4 hemizygote of M22 (M22/BY4741-CYS4&#x394;) and BY4742 (BY4742/BY4741-CYS4&#x394;) are shown along with strains at adjacent positions on the plate, which are hemizygous for different genes. (B) Reciprocal hemizygosity analysis shows that the YPS163 allele of CTT1 affects sensitivity to hydrogen peroxide. The YPS163 allele of CTT1 does not complement a CTT1 deletion (YPS163/BY4741-CTT1&#x394;), whereas the BY4741 allele of CTT1 does complement (YPS163-CTT1&#x394;/BY4741). Each photograph shows four replicate colonies of each strain." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-962665190" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Identification of quantitative trait alleles of &lt;em&gt;CYS4&lt;/em&gt; and &lt;em&gt;CTT1&lt;/em&gt;. (A) A noncomplementation screen of M22/BY4741 hemizygotes identifies the M22 allele of &lt;em&gt;CYS4&lt;/em&gt; as a colony color allele. Hemizygotes were generated by crossing M22 and BY4742 (control) to the BY4741 &lt;em&gt;MAT&lt;/em&gt;a deletion collection, and colony color was assayed on copper sulfate medium. The &lt;em&gt;CYS4&lt;/em&gt; hemizygote of M22 (M22/BY4741-&lt;em&gt;CYS4&lt;/em&gt;&#x394;) and BY4742 (BY4742/BY4741-&lt;em&gt;CYS4&lt;/em&gt;&#x394;) are shown along with strains at adjacent positions on the plate, which are hemizygous for different genes. (B) Reciprocal hemizygosity analysis shows that the YPS163 allele of &lt;em&gt;CTT1&lt;/em&gt; affects sensitivity to hydrogen peroxide. The YPS163 allele of &lt;em&gt;CTT1&lt;/em&gt; does not complement a &lt;em&gt;CTT1&lt;/em&gt; deletion (YPS163/BY4741-&lt;em&gt;CTT1&lt;/em&gt;&#x394;), whereas the BY4741 allele of &lt;em&gt;CTT1&lt;/em&gt; does complement (YPS163-&lt;em&gt;CTT1&lt;/em&gt;&#x394;/BY4741). Each photograph shows four replicate colonies of each strain.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/753/F2.medium.gif" width="440" height="229"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/2/7/753/F2.medium.gif" width="440" height="229"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/753/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/753/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461604" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">Identification of quantitative trait alleles of <em>CYS4</em> and <em>CTT1</em>. (A) A noncomplementation screen of M22/BY4741 hemizygotes identifies the M22 allele of <em>CYS4</em> as a colony color allele. Hemizygotes were generated by crossing M22 and BY4742 (control) to the BY4741 <em>MAT</em>a deletion collection, and colony color was assayed on copper sulfate medium. The <em>CYS4</em> hemizygote of M22 (M22/BY4741-<em>CYS4</em>Δ) and BY4742 (BY4742/BY4741-<em>CYS4</em>Δ) are shown along with strains at adjacent positions on the plate, which are hemizygous for different genes. (B) Reciprocal hemizygosity analysis shows that the YPS163 allele of <em>CTT1</em> affects sensitivity to hydrogen peroxide. The YPS163 allele of <em>CTT1</em> does not complement a <em>CTT1</em> deletion (YPS163/BY4741-<em>CTT1</em>Δ), whereas the BY4741 allele of <em>CTT1</em> does complement (YPS163-<em>CTT1</em>Δ/BY4741). Each photograph shows four replicate colonies of each strain.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">To further evaluate a quantitative noncomplementation screen, we examined sensitivity to hydrogen peroxide in YPS163. YPS163 is particularly sensitivity to hydrogen peroxide, but no genes have yet been found to be responsible for this sensitivity (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Kim and Fay 2007</a>). We mated YPS163 to the yeast deletion collection and found that only the strain hemizygous for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003320">CTT1</a></em> showed sensitivity to hydrogen peroxide similar to that of YPS163. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003320">CTT1</a></em> encodes a cytosolic catalase that protects cells from oxidative damage by converting hydrogen peroxide to water and oxygen (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Grant <em>et al.</em> 1998</a>). <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003320">CTT1</a></em> was deleted in YPS163 and BY4741, and the resulting strains were used to obtain two reciprocal hemizygotes, each carrying a different allele of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003320">CTT1</a></em>. The two hemizygotes showed allele-specific complementation; only the YPS163 allele of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003320">CTT1</a></em> failed to complement the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003320">CTT1</a></em> deletion (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). The results of these two preliminary screens motivated us to conduct a more comprehensive screen for quantitative trait alleles by crossing multiple strains to the yeast deletion collection and testing each hemizygote for non-complementation.</p></div><div id="sec-11" class="subsection"><h3>Large-scale screen for colony color and size alleles</h3><p id="p-23">To scale the noncomplementation screen to multiple strains, we examined two traits that vary among nine diverse yeast strains, colony size in the presence of copper sulfate and colony color in the presence of copper sulfate and PPG (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). PPG is an inhibitor of cystathionine gamma-lyase (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000010">CYS3</a></em>) and causes typically white-colored strains to become rust-colored in the presence of copper sulfate (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Kim <em>et al.</em> 2009</a>). Both traits may have ecological significance: rust coloration is dependent on hydrogen sulfide production (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Kim <em>et al.</em> 2009</a>), a significant problem in wine production (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Linderholm <em>et al.</em> 2008</a>), and resistance to copper sulfate has been hypothesized to be an adaptation to fungicides sprayed in vineyards (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Mortimer 2000</a>). The nine strains were selected on the basis of genome sequencing projects (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Doniger <em>et al.</em> 2008</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Liti <em>et al.</em> 2009</a>) and their phenotypic diversity within a larger collection of yeast strains (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/FigureS1.pdf">Figure S1</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/753/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of noncomplementation screen for colony color and size alleles. (A) Colony color (PPG medium) and size (copper sulfate medium) is shown for strains crossed to the MATa (BY4741) and MAT&#x3B1; (BY4742) deletion collections. Colony color and size of the BY4741 and BY4742 parental strains are shown as representatives. (B) The resulting collections of hemizygotes were phenotyped for colony color and size (see Materials and Methods). BY&#x394; indicates either BY4741 or BY4742 deletion strains. (C) Control matings of BY4742 (parent) to BY4741&#x394; (deletions) and the reciprocal set of crosses were used to eliminate candidates that exhibited haploinsufficiency. (D) Candidates that showed no effect in the homozygous deletion collection (BY4743&#x394;) also were eliminated. (E) Replicated measurements of colony color and size were obtained for manually generated hemizygotes. Reciprocal hemizygosity analysis was used to test the remaining candidates for quantitative trait alleles. The number of noncomplementing candidate genes indicates the remaining number after each filter." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-962665190" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Overview of noncomplementation screen for colony color and size alleles. (A) Colony color (PPG medium) and size (copper sulfate medium) is shown for strains crossed to the &lt;em&gt;MAT&lt;/em&gt;a (BY4741) and &lt;em&gt;MAT&lt;/em&gt;&#x3B1; (BY4742) deletion collections. Colony color and size of the BY4741 and BY4742 parental strains are shown as representatives. (B) The resulting collections of hemizygotes were phenotyped for colony color and size (see &lt;em&gt;Materials and Methods&lt;/em&gt;). BY&#x394; indicates either BY4741 or BY4742 deletion strains. (C) Control matings of BY4742 (parent) to BY4741&#x394; (deletions) and the reciprocal set of crosses were used to eliminate candidates that exhibited haploinsufficiency. (D) Candidates that showed no effect in the homozygous deletion collection (BY4743&#x394;) also were eliminated. (E) Replicated measurements of colony color and size were obtained for manually generated hemizygotes. Reciprocal hemizygosity analysis was used to test the remaining candidates for quantitative trait alleles. The number of noncomplementing candidate genes indicates the remaining number after each filter.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/753/F3.medium.gif" width="216" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/2/7/753/F3.medium.gif" width="216" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/753/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/753/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461623" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child">Overview of noncomplementation screen for colony color and size alleles. (A) Colony color (PPG medium) and size (copper sulfate medium) is shown for strains crossed to the <em>MAT</em>a (BY4741) and <em>MAT</em>α (BY4742) deletion collections. Colony color and size of the BY4741 and BY4742 parental strains are shown as representatives. (B) The resulting collections of hemizygotes were phenotyped for colony color and size (see <em>Materials and Methods</em>). BYΔ indicates either BY4741 or BY4742 deletion strains. (C) Control matings of BY4742 (parent) to BY4741Δ (deletions) and the reciprocal set of crosses were used to eliminate candidates that exhibited haploinsufficiency. (D) Candidates that showed no effect in the homozygous deletion collection (BY4743Δ) also were eliminated. (E) Replicated measurements of colony color and size were obtained for manually generated hemizygotes. Reciprocal hemizygosity analysis was used to test the remaining candidates for quantitative trait alleles. The number of noncomplementing candidate genes indicates the remaining number after each filter.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">For each of the nine strains, we generated heterothallic derivatives with a dominant selectable marker at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em> locus for mating. Five of strains (M22, YPS163, YJM326, BC187, and CLIB382) were mated to both the <em>MAT</em>a (BY4741) and <em>MAT</em>α (BY4742) deletion collections, and the other four were mated to only one. As a control, we also mated the parents of the deletion collection to the <em>MAT</em>a and <em>MAT</em>α deletion collections. Including the controls, the ~75K hemizygotes were phenotyped by robotic arraying of the strains in a 384-colony format and measuring colony color on PPG medium and colony size on copper sulfate medium using semi-automated image analysis (see <em>Materials and Methods</em>).</p><p id="p-26">A total of 595 genes showed noncomplementation for colony color or size in one or more strains (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). After eliminating genes that showed effects in the hemizygous controls, we found 366 noncomplementing candidates, an average of 25 and 39 genes per strain, for colony color and size, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS2.xls">Table S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS3.xls">Table S3</a>). Most of the genes, 65%, were only identified in one of the nine strains (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/FigureS2.pdf">Figure S2</a>).</p></div><div id="sec-12" class="subsection"><h3>Interactions with genetic background</h3><p id="p-27">Many of the noncomplementing candidate genes showed different effects in hemizygotes generated using the <em>MAT</em>a <em>vs.</em> the <em>MAT</em>α deletion collection (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Of the 268 genes that were identified in the five strains crossed to both the <em>MAT</em>a and <em>MAT</em>α deletion collection, only 19 were identified in both crosses. Some of these differences are likely due to measurement error and the cutoffs used to identify noncomplementing genes. However, differences could also result through interactions with the genetic background.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/753/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Noncomplementation phenotypes differ between the MATa and MAT&#x3B1; deletions collections. Colony size (circles) and colony color (squares) phenotypes are shown for hemizygotes generated using both the MATa and MAT&#x3B1; deletion collections (M22, YPS163, YJM326, BC187, and CLIB382). Noncomplementing candidates are shown in red and are defined by genes that exhibit an effect in one or more of the wild strain hemizygotes but not the hemizygous controls (BY4741 and BY4742 matings), and an effect in the homozygous deletion control (BY4743). Phenotypes are normalized to one by the mean of each hemizygous deletion collection (see Materials and Methods)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-962665190" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Noncomplementation phenotypes differ between the &lt;em&gt;MAT&lt;/em&gt;a and &lt;em&gt;MAT&lt;/em&gt;&#x3B1; deletions collections. Colony size (circles) and colony color (squares) phenotypes are shown for hemizygotes generated using both the &lt;em&gt;MAT&lt;/em&gt;a and &lt;em&gt;MAT&lt;/em&gt;&#x3B1; deletion collections (M22, YPS163, YJM326, BC187, and CLIB382). Noncomplementing candidates are shown in red and are defined by genes that exhibit an effect in one or more of the wild strain hemizygotes but not the hemizygous controls (BY4741 and BY4742 matings), and an effect in the homozygous deletion control (BY4743). Phenotypes are normalized to one by the mean of each hemizygous deletion collection (see &lt;em&gt;Materials and Methods&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/753/F4.medium.gif" width="440" height="399"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/2/7/753/F4.medium.gif" width="440" height="399"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/753/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/753/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461630" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-28" class="first-child">Noncomplementation phenotypes differ between the <em>MAT</em>a and <em>MAT</em>α deletions collections. Colony size (circles) and colony color (squares) phenotypes are shown for hemizygotes generated using both the <em>MAT</em>a and <em>MAT</em>α deletion collections (M22, YPS163, YJM326, BC187, and CLIB382). Noncomplementing candidates are shown in red and are defined by genes that exhibit an effect in one or more of the wild strain hemizygotes but not the hemizygous controls (BY4741 and BY4742 matings), and an effect in the homozygous deletion control (BY4743). Phenotypes are normalized to one by the mean of each hemizygous deletion collection (see <em>Materials and Methods</em>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Strains generated using the <em>MAT</em>a and <em>MAT</em>α deletion collection differ for two auxotrophic markers, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">MET15</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">LYS2</a></em>, origin of mating type, and any second-site (unintentional) mutations that occurred during the construction of the deletions. To test whether the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">MET15</a></em> deletion, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">LYS2</a></em> deletion, or the mating type locus could explain the difference between hemizygotes generated using the <em>MAT</em>a and <em>MAT</em>α deletion collection, we tested five genes with a large effect in one hemizygote but not the other. Differences in the genetic background were assorted using recombinant strains generated by sporulating a diploid hybrid of the two haploid deletion strains from the <em>MAT</em>a and <em>MAT</em>α deletion collection. Using hemizygotes generated with the recombinant strains, we found noncomplementation of two genes, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002910">SAM2</a></em> (<em>P</em> = 2.2 × 10<sup>−16</sup>) and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003962">UBI4</a></em> (<em>P</em> = 6.3 × 10<sup>−16</sup>), was dependent on the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">LYS2</a></em> deletion. For the remaining three genes, phenotypic variation among the hemizygotes was not associated with mating type or auxotrophies, raising the possibility that second-site mutations were present in one or both of the deletion strains. Regardless of the cause, our results suggest that auxotrophic markers and second-site mutations are not entirely recessive as is often assumed (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Grünenfelder and Winzeler 2002</a>).</p></div><div id="sec-13" class="subsection"><h3>Reciprocal hemizygosity analysis</h3><p id="p-30">Candidate genes may be false positives because of the dominance of the deletion (haploinsufficiency) or dominance of second-site mutations within the yeast deletion collection. Although both the deletion and any second-site mutations were also present in the hemizygous controls, their effects may depend on the hybrid background. We used reciprocal hemizygosity analysis (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Steinmetz <em>et al.</em> 2002</a>) to account for this possibility.</p><p id="p-31">Two filters were used to select noncomplementing candidates for reciprocal hemizygosity analysis. First, we selected those candidate genes that showed a significant effect in the homozygous diploid deletion collection. Interestingly, only 22 of 171 and 16 of 207 candidate genes showed phenotypic effects in the homozygous deletion collection for colony color and size, respectively. Candidate genes without effects in the homozygous deletion collection could be false positives but could also arise as the result of differences between the genetic background of the hybrid hemizygotes and the homozygous deletion collection. Second, we retested the resulting 36 candidate for noncomplementation by manually regenerating each hemizygous strain and obtaining replicated phenotype measurements. After replication, 25 of 36 candidate genes retained evidence of noncomplementation (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS4.xls">Table S4</a>). We were able to generate deletions in both the laboratory and wild strain(s) for 18 of the remaining 25 candidate genes. Because some genes were identified in more than one strain, we generated deletions that enabled us to test a total of 44 candidate alleles by reciprocal hemizygosity analysis.</p><p id="p-32">Of the 44 alleles in 18 genes that were tested, 14 alleles in 10 genes showed significant differences between the two reciprocal hemizygotes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS5.xls">Table S5</a>). For one of these, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006031">GDE1</a></em>, we found effects on colony color even though it originally only passed our screening filters for colony size. For the eight candidate genes that were not validated by reciprocal hemizygosity: <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003133">YGL165C</a></em> was discarded because it is a dubious gene and overlaps with another candidate, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003134">CUP2</a></em>; <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004015">SNF7</a></em>, showed evidence for haploinsufficiency; three genes <em>(<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000632">RBK1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004417">TUS1</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000504">YBR300C</a></em>) were false positives as the result of second-site mutations; and three genes had deletion phenotypes inconsistent with the deletion collection (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000664">ATG15</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000502">MAL31</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000683">YCR087W</a></em>).</p><p id="p-33">Of the 14 alleles identified by reciprocal hemizygosity analysis, most showed small, barely noticeable effects, much smaller than the effects of the haploid deletions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/FigureS3.pdf">Figure S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS5.xls">Table S5</a>). However, the effects of the haploid deletions were often smaller in the wild strain than in the laboratory strain, consistent with a recessive loss of function mutation in the wild strain. We sequenced all 14 alleles and found two of the alleles, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004415">ATG17</a></em> in M22 and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004888">BUL1</a></em> in BC187, had no mutations within either the coding or adjacent noncoding regions in comparison with the laboratory strain. Although we generated multiple independent deletions or observed cosegregation between the deletion and the phenotype of interest, the sequence analysis raised the possibility that second-site mutations occur commonly enough to generate small but consistent phenotype differences between the reciprocal hemizygotes. As such, we decided to replicate the reciprocal hemizygosity analysis using deletion strains that were backcrossed to their progenitor to eliminate any potential second-site mutations generated during transformation.</p></div><div id="sec-14" class="subsection"><h3>Replication of reciprocal hemizygosity analysis</h3><p id="p-34">To replicate the reciprocal hemizygosity test, we independently derived new deletions for each gene, backcrossed these strains to their parent, and selected 10 segregants with the deletion from each backcross for reciprocal hemizygosity analysis, five of each mating type. Rather than using the parent of the yeast deletion collection, which has a number of auxotrophies, we used a different version of S288c with only a single auxotrophy (YJF173, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS1.xls">Table S1</a>). <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003320">CTT1</a></em> was also included in the analysis because our previous reciprocal hemizygosity analysis showed large phenotypic differences and we wanted to ensure these differences can be attributed to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003320">CTT1</a></em>. However, we excluded the two genes without mutations, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004415">ATG17</a></em> in M22 and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004888">BUL1</a></em> in BC187, and two genes for which we were not able to obtain backcross progeny, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003952">FRA1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000659">BUD31</a></em> in UWOPS87.</p><p id="p-35">Out of the 11 alleles tested, three showed significant differences between reciprocal hemizygotes that depended on the cross direction, defined by either a <em>MAT</em>a or <em>MAT</em>α strain mated to the laboratory strain of the opposite mating type (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/FigureS4.pdf">Figure S4</a>). For <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003387">CYS4</a></em>, the dependency on the cross direction could be the result of differences between the YJM210 <em>MAT</em>a and <em>MAT</em>α strain; strains generated with YJM210 <em>MAT</em>a were consistently darker than those generated with YJM210 <em>MAT</em>α (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/FigureS4.pdf">Figure S4</a>). For <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002910">SAM2</a></em>, the two cross directions generated hemizygous effects in opposite directions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/FigureS4.pdf">Figure S4</a>). For <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005135">DUG3</a></em> in M22, one cross direction showed an effect consistent with DUG3 in CLIB382 and the other showed no effect.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/753/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Replication of reciprocal hemizygosity analysis for colony color and size alleles. (A) Colony color. (B) Colony size. The mean and standard error (bars) for four genotypes are shown for a hybrid between a laboratory strain (YJF173) and another strain (middle labels). The homozygous deletion (&#x394;/&#x394;), laboratory allele deletion (W/&#x394;), wild allele deletion (&#x394;/W), and no deletion (W/W) of the gene being tested (bottom labels), where &#x394; indicates a deletion and W indicates wild-type allele present in the strain." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-962665190" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Replication of reciprocal hemizygosity analysis for colony color and size alleles. (A) Colony color. (B) Colony size. The mean and standard error (bars) for four genotypes are shown for a hybrid between a laboratory strain (YJF173) and another strain (middle labels). The homozygous deletion (&#x394;/&#x394;), laboratory allele deletion (W/&#x394;), wild allele deletion (&#x394;/W), and no deletion (W/W) of the gene being tested (bottom labels), where &#x394; indicates a deletion and W indicates wild-type allele present in the strain.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/753/F5.medium.gif" width="385" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/2/7/753/F5.medium.gif" width="385" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/753/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/753/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461521" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-36" class="first-child">Replication of reciprocal hemizygosity analysis for colony color and size alleles. (A) Colony color. (B) Colony size. The mean and standard error (bars) for four genotypes are shown for a hybrid between a laboratory strain (YJF173) and another strain (middle labels). The homozygous deletion (Δ/Δ), laboratory allele deletion (W/Δ), wild allele deletion (Δ/W), and no deletion (W/W) of the gene being tested (bottom labels), where Δ indicates a deletion and W indicates wild-type allele present in the strain.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Two of the 11 alleles showed significant differences between reciprocal hemizygotes that did not depend on the cross direction (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>); <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005135">DUG3</a></em> in CLIB382 (<em>P</em> = 0.0024, <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>) and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003320">CTT1</a></em> in YPS163 (<em>P</em> = 0.0054, <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). The difference between the two <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005135">DUG3</a></em> alleles is 21% of the difference between the homozygous deletion (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005135">DUG3</a></em>Δ/<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005135">DUG3</a></em>Δ) and wild-type (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005135">DUG3</a></em>/<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005135">DUG3</a></em>) hybrid and 13% of the difference between the two haploid parent strains without the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005135">DUG3</a></em> deletion, CLIB382 and YJF173. However, the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005135">DUG3</a></em> hemizygotes generated using YJF173 had phenotypes that were inconsistent with the initial reciprocal hemizygotes generated using BY4741. Hybrids with a hemizygous YJF173 allele were darker than those with a CLIB382 allele of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005135">DUG3</a></em>, indicating that the YJF173 allele confers a darker colony color, rather than a lighter color as originally found (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/FigureS3.pdf">Figure S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/TableS5.xls">Table S5</a>). We observed a similar but insignificant trend for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005135">DUG3</a></em> in M22/YFJ173 hemizygotes (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1/FigureS4.pdf">Figure S4</a>). Finally, we found the YPS163 allele of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003320">CTT1</a></em> in the hemizygous state has a large effect on sensitivity to hydrogen peroxide, equivalent to a deletion (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>), and consistent with our previous reciprocal hemizygosity analysis.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/7/753/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Replication of reciprocal hemizygosity analysis for CTT1 and sensitivity to hydrogen peroxide. The mean and standard error (bars) is shown for four hybrids (YJF173/YPS163) with genotypes at CTT1 labeled on the bottom, (&#x394; = deletion). Sensitivity to hydrogen peroxide was measured by colony size." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-962665190" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Replication of reciprocal hemizygosity analysis for &lt;em&gt;CTT1&lt;/em&gt; and sensitivity to hydrogen peroxide. The mean and standard error (bars) is shown for four hybrids (YJF173/YPS163) with genotypes at &lt;em&gt;CTT1&lt;/em&gt; labeled on the bottom, (&#x394; = deletion). Sensitivity to hydrogen peroxide was measured by colony size.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/7/753/F6.medium.gif" width="440" height="397"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/2/7/753/F6.medium.gif" width="440" height="397"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/7/753/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/7/753/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461542" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-38" class="first-child">Replication of reciprocal hemizygosity analysis for <em>CTT1</em> and sensitivity to hydrogen peroxide. The mean and standard error (bars) is shown for four hybrids (YJF173/YPS163) with genotypes at <em>CTT1</em> labeled on the bottom, (Δ = deletion). Sensitivity to hydrogen peroxide was measured by colony size.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-39">Identifying quantitative trait alleles depends on our ability to map and resolve these alleles to individual genes. However, both linkage and linkage disequilibrium mapping require large sample sizes to detect the effects of a locus across a heterogeneous genetic background. Thus, many quantitative trait alleles may be missed if they only have effects in certain backgrounds (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Li <em>et al.</em> 2005</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Sinha <em>et al.</em> 2006</a>; <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Kim and Fay 2007</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dowell <em>et al.</em> 2010</a>). To compensate for this deficiency, we implemented a genome-wide noncomplementation screen for quantitative trait alleles using the yeast deletion collection. Paradoxically, we show that most of the candidate quantitative trait alleles we identified are likely false-positive results because of subtle genetic interactions between the locus of interest and the near-isogenic background. Compared with the identification of two large-effect quantitative trait alleles of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003387">CYS4</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003320">CTT1</a></em>, our results highlight the crux of investigating quantitative trait alleles with small and/or background-dependent effects.</p><p id="p-40">Using the noncomplementation screen, we were able to identify two large-effect quantitative trait alleles. The M22 allele of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003387">CYS4</a></em> was used as a positive control as it is recessive and causes an easily measured effect on colony color, nearly equivalent to that of a null allele (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Kim <em>et al.</em> 2009</a>). We also found a novel allele of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003320">CTT1</a></em> in YPS163 that causes sensitivity to hydrogen peroxide. In a previous study, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Diezmann and Dietrich (2011)</a> found no effects of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003320">CTT1</a></em> on sensitivity to hydrogen peroxide. However, they examined linkage in a cross between a laboratory strain, S288c, and a clinical isolate, YJM789, and YPS163 has four candidate amino acid polymorphisms in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003320">CTT1</a></em> not present in either S288c or YJM789 (F212Y, A255E, D431N, T522A).</p><p id="p-41">Many of the candidates identified in the initial noncomplementation screen were false positives. Although our high-throughput phenotyping assay was quite reproducible when applied to the same hybrid strains, we found 31% (11/36) of candidate alleles did not show replicable effects when hybrids were manually generated and tested for quantitative noncomplementation. Less definitive but still informative, we found only 7% (19/268) of candidates were identified using both the <em>MAT</em>a and <em>MAT</em>α deletion collection. Out these 19, 10 showed effects in the homozygous deletion collection, and four (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003134">CUP2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005135">DUG3</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000664">ATG15</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004888">BUL1</a></em>) of these were positive within our initial reciprocal hemizygosity test. Thus, there is no apparent benefit of only using candidates identified in both the <em>MAT</em>a and <em>MAT</em>α deletion collection, although these two collections were generated independently of one another.</p><p id="p-42">One source of false positives is likely the deletion collection itself. Within the yeast deletion collection, 8% of strains are estimated to carry chromosomal aneuploidies (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hughes <em>et al.</em> 2000</a>), and during the construction of the deletion collection, 6.5% of strains exhibited an overt phenotype that did not segregate with targeted deletion (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Grünenfelder and Winzeler 2002</a>), implying the presence of many other second-site mutations without an overt phenotype. Although we considered only investigating candidates with effects in both collections, we did not use both collections for all nine strains, and some of the differences between the two collections could be true positives that depend on auxotrophic differences. In particular, BY4741 carries a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">MET15</a></em> deletion and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">MET15</a></em> affects colony color (<a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Kim <em>et al.</em> 2009</a>). Instead, crosses between BY4741 and BY4742 deletion strains showed that noncomplementation depended on the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">LYS2</a></em> deletion for two of five genes tested. Thus, at least some of the differences between the two deletion collections can be attributed to allele differences that depend on the genetic background, as has been found in other quantitative genetic studies (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Perlstein <em>et al.</em> 2007</a>; <a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Kim and Fay 2007</a>).</p><p id="p-43">Another potential indicator of false positives was that only 10% (36/366) of the noncomplementing genes showed effects in the homozygous deletion collection. Although even essential genes can be nonessential in other strain backgrounds (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Dowell <em>et al.</em> 2010</a>), we decided to only apply the reciprocal hemizygosity test to the subset of candidates that also exhibited effects in the homozygous deletion collection. However, it is possible that some of the candidates that were not tested are true positives.</p><p id="p-44">We used reciprocal hemizygosity analysis as the primary means of eliminating false positives caused by to haploinsufficiency, second-site mutations within the yeast deletion collection, or other unknown causes. Using this approach we eliminated 8 of 18 candidate genes, a total of 30 of 44 candidate alleles. However, not all false positives were eliminated; two of the genes contained no mutations that could underlie the allele differences. One potential explanation is that second site mutations with subtle effects were frequently generated during transformation. Although we attempted to account for this possibility by generating at least two reciprocal hemizygotes using independently generated deletions, second-site mutations may be positively selected to compensate for the deletion, or they may tend to generate phenotypic effects in a consistent direction.</p><p id="p-45">To more carefully control for the effects of second-site mutations, we replicated the reciprocal hemizygosity test using deletion strains that were backcrossed to their parents. However, a number of observations obscure definitive conclusions on the presence or absence of quantitative trait alleles in these genes. Although we found that the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005135">DUG3</a></em> allele from YJF173 conferred a darker colony color than the CLIB382 allele, this effect was opposite of that observed in our original reciprocal hemizygosity analysis. The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005135">DUG3</a></em> allele of YJF173 also conferred a lighter color than that from M22, but this difference depended on the direction of the cross. Two other genes, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003387">CYS4</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002910">SAM2</a></em>, showed hemizygous effects that depended on the cross direction. Interestingly, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003387">CYS4</a></em> in YJM210 carries an amino acid mutation, S504N, that has been reported to affect fermentation rate and hydrogen sulfide production in a vineyard strain (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Linderholm <em>et al.</em> 2006</a>). Thus, there is evidence both for and against the presence of small-effect quantitative trait alleles in these genes. One additional factor that may be relevant is that four of the hybrids showed evidence of haploinsufficiency, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002910">SAM2</a></em> in M22 (<em>P</em> = 0.005), <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005135">DUG3</a></em> in CLIB382 (<em>P</em> = 5.8 × 10<sup>−4</sup>), <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005135">DUG3</a></em> in M22 (<em>P</em> = 1.5 × 10<sup>−5</sup>), and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003134">CUP2</a></em> in CLIB382 (<em>P</em> = 1.1 × 10<sup>−6</sup>). Haploinsufficiency may make these genes particularly susceptible to subtle differences in the genetic background.</p><p id="p-46">The noncomplementation screen is expected to miss certain quantitative trait alleles. Any alleles of essential genes would be missed because we only tested the ~4,700 nonessential genes within the haploid yeast deletion collection. Certain colony color alleles could have been missed if colony size were small as the result of simultaneous effects on copper sensitivity. Although this may have occurred in a few instances, the hybrids used for the colony color screen were fairly uniform in size. A potentially large source of false negatives was our stringent filter to eliminate any genes showing evidence of noncomplementation in the control crosses generated using the parents of the deletion collection. When sensitive phenotyping is used, most deletions exhibit at least some degree of dominance (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hillenmeyer <em>et al.</em> 2008</a>).</p><p id="p-47">In conclusion, application of a noncomplementation screen for quantitative trait alleles in yeast revealed a number of pitfalls but also insights that may help guide future studies. A primary pitfall is second-site mutations without overt phenotypes, as such mutations could be more common than is currently appreciated. However, the identification of two large-effect quantitative trait alleles indicates that quantitative noncomplementation may be well-suited to identifying alleles of large effect. Thus, quantitative noncomplementation could prove to be an excellent means of identifying alleles of large effect in any wild strain that can be mated to the yeast deletion collection without the need for extensive genotyping.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">We thank members of the Fay laboratory, S. Arur, R. Mitra, J. Gerke, C. Hittinger, Kim Lorenz, and B. Cohen for comments; E. Engle for assistance; and L. Riles and M. Johnston for providing the yeast deletion collection. This work was supported by grants from the National Institutes of Health (GM080669 and GM086412) to J.C.F.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></li><li class="fn" id="fn-2"><p id="p-2">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002550/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-49"><em>Communicating editor: F. P. Roth</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-03-15"><span class="received-label">Received </span>March 15, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-04-30"><span class="accepted-label">Accepted </span>April 30, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Kim <em>et al</em>.</li></ul><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.1" data-doi="10.1086/513471"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ahituv</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kavaslar</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schackwitz</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ustaszewska</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Medical sequencing at the extremes of human body mass</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">779</span>–<span class="cit-lpage">791</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DAhituv%26rft.auinit1%253DN.%26rft.volume%253D80%26rft.issue%253D4%26rft.spage%253D779%26rft.epage%253D791%26rft.atitle%253DMedical%2Bsequencing%2Bat%2Bthe%2Bextremes%2Bof%2Bhuman%2Bbody%2Bmass.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F513471%26rft_id%253Dinfo%253Apmid%252F17357083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/513471&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17357083&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245465200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.2" data-doi="10.1126/science.1156409"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic mapping in human disease</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">322</span>: <span class="cit-fpage">881</span>–<span class="cit-lpage">888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAltshuler%26rft.auinit1%253DD.%26rft.volume%253D322%26rft.issue%253D5903%26rft.spage%253D881%26rft.epage%253D888%26rft.atitle%253DGenetic%2BMapping%2Bin%2BHuman%2BDisease%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1156409%26rft_id%253Dinfo%253Apmid%252F18988837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjIvNTkwMy84ODEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzcvNzUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.3" data-doi="10.1371/journal.pgen.0020195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ben-Ari</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zenvirth</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klutstein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Four linked genes participate in controlling sporulation efficiency in budding yeast</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBen-Ari%26rft.auinit1%253DG.%26rft.volume%253D2%26rft.issue%253D11%26rft.spage%253De195%26rft.epage%253De195%26rft.atitle%253DFour%2Blinked%2Bgenes%2Bparticipate%2Bin%2Bcontrolling%2Bsporulation%2Befficiency%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020195%26rft_id%253Dinfo%253Apmid%252F17112318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17112318&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.4" data-doi="10.1126/science.1099870"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiss</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertsemlidis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcel</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McPherson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Multiple rare alleles contribute to low plasma levels of HDL cholesterol</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">305</span>: <span class="cit-fpage">869</span>–<span class="cit-lpage">872</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCohen%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D305%26rft.issue%253D5685%26rft.spage%253D869%26rft.epage%253D872%26rft.atitle%253DMultiple%2BRare%2BAlleles%2BContribute%2Bto%2BLow%2BPlasma%2BLevels%2Bof%2BHDL%2BCholesterol%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1099870%26rft_id%253Dinfo%253Apmid%252F15297675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDUvNTY4NS84NjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzcvNzUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coyne</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simeonidis</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rooney</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Relative paucity of genes causing inviability in hybrids between <em>Drosophila melanogaster</em> and <em>D. simulans</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">1091</span>–<span class="cit-lpage">1103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCoyne%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D150%26rft.issue%253D3%26rft.spage%253D1091%26rft.epage%253D1103%26rft.atitle%253DRelative%2BPaucity%2Bof%2BGenes%2BCausing%2BInviability%2Bin%2BHybrids%2BBetween%2BDrosophila%2Bmelanogaster%2Band%2BD.%2Bsimulans%26rft_id%253Dinfo%253Apmid%252F9799261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1MC8zLzEwOTEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzcvNzUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.6" data-doi="10.1038/ng1674"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deutschbauer</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Quantitative trait loci mapped to single-nucleotide resolution in yeast</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1333</span>–<span class="cit-lpage">1340</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDeutschbauer%26rft.auinit1%253DA.%2BM.%26rft.volume%253D37%26rft.issue%253D12%26rft.spage%253D1333%26rft.epage%253D1340%26rft.atitle%253DQuantitative%2Btrait%2Bloci%2Bmapped%2Bto%2Bsingle-nucleotide%2Bresolution%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1674%26rft_id%253Dinfo%253Apmid%252F16273108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1674&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16273108&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233657300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.7" data-doi="10.1534/genetics.111.128256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diezmann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dietrich</span>  <span class="cit-name-given-names">F. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Oxidative stress survival in a clinical <em>Saccharomyces cerevisiae</em> isolate is influenced by a major quantitative trait nucleotide</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">709</span>–<span class="cit-lpage">722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D188%26rft.spage%253D709%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.128256%26rft_id%253Dinfo%253Apmid%252F21515583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.111.128256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21515583&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.8" data-doi="10.1371/journal.pgen.1000183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doniger</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swain</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corcuera</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A catalog of neutral and deleterious polymorphism in yeast</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DDoniger%26rft.auinit1%253DS.%2BW.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253De1000183%26rft.epage%253De1000183%26rft.atitle%253DA%2Bcatalog%2Bof%2Bneutral%2Band%2Bdeleterious%2Bpolymorphism%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000183%26rft_id%253Dinfo%253Apmid%252F18769710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18769710&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.9" data-doi="10.1126/science.1189015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dowell</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryan</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agarwala</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genotype to phenotype: a complex problem</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">328</span>: <span class="cit-fpage">469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDowell%26rft.auinit1%253DR.%2BD.%26rft.volume%253D328%26rft.issue%253D5977%26rft.spage%253D469%26rft.epage%253D469%26rft.atitle%253DGenotype%2Bto%2BPhenotype%253A%2BA%2BComplex%2BProblem%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1189015%26rft_id%253Dinfo%253Apmid%252F20413493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjgvNTk3Ny80NjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzcvNzUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.10" data-doi="10.1073/pnas.0407187101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fearnhead</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilding</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winney</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tonks</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartlett</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Multiple rare variants in different genes account for multifactorial inherited susceptibility to colorectal adenomas</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">15992</span>–<span class="cit-lpage">15997</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0407187101%26rft_id%253Dinfo%253Apmid%252F15520370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzQ1LzE1OTkyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi83Lzc1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.11" data-doi="10.1126/science.1166426"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic interactions between transcription factors cause natural variation in yeast</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">323</span>: <span class="cit-fpage">498</span>–<span class="cit-lpage">501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGerke%26rft.auinit1%253DJ.%26rft.volume%253D323%26rft.issue%253D5913%26rft.spage%253D498%26rft.epage%253D501%26rft.atitle%253DGenetic%2BInteractions%2BBetween%2BTranscription%2BFactors%2BCause%2BNatural%2BVariation%2Bin%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1166426%26rft_id%253Dinfo%253Apmid%252F19164747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjMvNTkxMy80OTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzcvNzUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.12" data-doi="10.1371/journal.pgen.1001144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramnarine</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Gene-environment interactions at nucleotide resolution</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D6%26rft.spage%253De1001144%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001144%26rft_id%253Dinfo%253Apmid%252F20941394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001144&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20941394&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.13" data-doi="10.1038/nature00935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riles</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Functional profiling of the <em>Saccharomyces cerevisiae</em> genome</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">418</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGiaever%26rft.auinit1%253DG.%26rft.volume%253D418%26rft.issue%253D6896%26rft.spage%253D387%26rft.epage%253D391%26rft.atitle%253DFunctional%2Bprofiling%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature00935%26rft_id%253Dinfo%253Apmid%252F12140549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature00935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12140549&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177009700029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.14" data-doi="10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Three new dominant drug resistance cassettes for gene disruption in <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1541</span>–<span class="cit-lpage">1553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DGoldstein%26rft.auinit1%253DA.%2BL.%26rft.volume%253D15%26rft.issue%253D14%26rft.spage%253D1541%26rft.epage%253D1553%26rft.atitle%253DThree%2Bnew%2Bdominant%2Bdrug%2Bresistance%2Bcassettes%2Bfor%2Bgene%2Bdisruption%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%2528199910%252915%253A14%253C1541%253A%253AAID-YEA476%253E3.0.CO%253B2-K%26rft_id%253Dinfo%253Apmid%252F10514571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10514571&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083104100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.15" data-doi="10.1006/bbrc.1998.9864"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perrone</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawes</span>  <span class="cit-name-given-names">I. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Glutathione and catalase provide overlapping defenses for protection against hydrogen peroxide in the yeast <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">Biochem. Biophys. Res. Commun.</abbr> <span class="cit-vol">253</span>: <span class="cit-fpage">893</span>–<span class="cit-lpage">898</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2Bbiophysical%2Bresearch%2Bcommunications%26rft.stitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.aulast%253DGrant%26rft.auinit1%253DC.%2BM.%26rft.volume%253D253%26rft.issue%253D3%26rft.spage%253D893%26rft.epage%253D898%26rft.atitle%253DGlutathione%2Band%2Bcatalase%2Bprovide%2Boverlapping%2Bdefenses%2Bfor%2Bprotection%2Bagainst%2Bhydrogen%2Bperoxide%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fbbrc.1998.9864%26rft_id%253Dinfo%253Apmid%252F9918826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/bbrc.1998.9864&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9918826&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077846900062&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.16" data-doi="10.1038/nrg886"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grünenfelder</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winzeler</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Treasures and traps in genome-wide data sets: case examples from yeast</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">653</span>–<span class="cit-lpage">661</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DGr%25C3%25BCnenfelder%26rft.auinit1%253DB.%26rft.volume%253D3%26rft.issue%253D9%26rft.spage%253D653%26rft.epage%253D661%26rft.atitle%253DTreasures%2Band%2Btraps%2Bin%2Bgenome-wide%2Bdata%2Bsets%253A%2Bcase%2Bexamples%2Bfrom%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg886%26rft_id%253Dinfo%253Apmid%252F12209140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg886&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12209140&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177785700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.17" data-doi="10.1126/science.1150021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hillenmeyer</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fung</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wildenhain</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pierce</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The chemical genomic portrait of yeast: uncovering a phenotype for all genes</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">362</span>–<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHillenmeyer%26rft.auinit1%253DM.%2BE.%26rft.volume%253D320%26rft.issue%253D5874%26rft.spage%253D362%26rft.epage%253D365%26rft.atitle%253DThe%2BChemical%2BGenomic%2BPortrait%2Bof%2BYeast%253A%2BUncovering%2Ba%2BPhenotype%2Bfor%2BAll%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1150021%26rft_id%253Dinfo%253Apmid%252F18420932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjAvNTg3NC8zNjIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzcvNzUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.18" data-doi="10.1038/77116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Widespread aneuploidy revealed by DNA microarray expression profiling</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHughes%26rft.auinit1%253DT.%2BR.%26rft.volume%253D25%26rft.issue%253D3%26rft.spage%253D333%26rft.epage%253D337%26rft.atitle%253DWidespread%2Baneuploidy%2Brevealed%2Bby%2BDNA%2Bmicroarray%2Bexpression%2Bprofiling.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F77116%26rft_id%253Dinfo%253Apmid%252F10888885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/77116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10888885&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087920900025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.19" data-doi="10.1073/pnas.0708194104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic variation in the cysteine biosynthesis pathway causes sensitivity to pharmacological compounds</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">19387</span>–<span class="cit-lpage">19391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0708194104%26rft_id%253Dinfo%253Apmid%252F18042712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzQ5LzE5Mzg3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi83Lzc1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.20" data-doi="10.1111/j.1567-1364.2009.00516.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Dissecting the pleiotropic consequences of a quantitative trait nucleotide</span>. <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2009.00516.x%26rft_id%253Dinfo%253Apmid%252F19456872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjc6IjkvNS83MTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzcvNzUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyons</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittenburg</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paigen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Combining data from multiple inbred line crosses improves the power and resolution of quantitative trait loci mapping</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1699</span>–<span class="cit-lpage">1709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D169%26rft.spage%253D1699%26rft_id%253Dinfo%253Apmid%252F15654110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15654110&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.22" data-doi="10.1128/AEM.01758-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Linderholm</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Findleton</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bisson</span>  <span class="cit-name-given-names">L. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Identification of genes affecting hydrogen sulfide formation in <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">1418</span>–<span class="cit-lpage">1427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.01758-07%26rft_id%253Dinfo%253Apmid%252F18192430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6Ijc0LzUvMTQxOCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNy83NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Linderholm</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olineka</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bisson</span>  <span class="cit-name-given-names">L. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Allele diversity among genes of the sulfate reduction pathway in wine strains of <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">Am. J. Enol. Vitic.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BEnology%2Band%2BViticulture%26rft.stitle%253DAm.%2BJ.%2BEnol.%2BVitic.%26rft.aulast%253DLinderholm%26rft.auinit1%253DA.%2BL.%26rft.volume%253D57%26rft.issue%253D4%26rft.spage%253D431%26rft.epage%253D440%26rft.atitle%253DAllele%2BDiversity%2Bamong%2BGenes%2Bof%2Bthe%2BSulfate%2BReduction%2BPathway%2Bin%2BWine%2BStrains%2Bof%2BSaccharomyces%2Bcerevisiae%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYWpldiI7czo1OiJyZXNpZCI7czo4OiI1Ny80LzQzMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNy83NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.24" data-doi="10.1038/nature07743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population genomics of domestic and wild yeasts</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLiti%26rft.auinit1%253DG.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D337%26rft.epage%253D341%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bdomestic%2Band%2Bwild%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07743%26rft_id%253Dinfo%253Apmid%252F19212322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212322&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.25" data-doi="10.1016/j.gde.2004.04.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The genetic architecture of quantitative traits: lessons from <em>Drosophila</em></span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">253</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%26rft.volume%253D14%26rft.issue%253D3%26rft.spage%253D253%26rft.epage%253D257%26rft.atitle%253DThe%2Bgenetic%2Barchitecture%2Bof%2Bquantitative%2Btraits%253A%2Blessons%2Bfrom%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2004.04.003%26rft_id%253Dinfo%253Apmid%252F15172667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2004.04.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15172667&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222171600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.26" data-doi="10.1101/gr.10.4.403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mortimer</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Evolution and variation of the yeast (<em>Saccharomyces</em>) genome</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DMortimer%26rft.auinit1%253DR.%2BK.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D403%26rft.epage%253D409%26rft.atitle%253DEvolution%2Band%2BVariation%2Bof%2Bthe%2BYeast%2B%2528Saccharomyces%2529%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.10.4.403%26rft_id%253Dinfo%253Apmid%252F10779481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEwLzQvNDAzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi83Lzc1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.27" data-doi="10.1038/ng1991"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perlstein</span>  <span class="cit-name-given-names">E. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruderfer</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schreiber</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic basis of individual differences in the response to small-molecule drugs in yeast</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">496</span>–<span class="cit-lpage">502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DPerlstein%26rft.auinit1%253DE.%2BO.%26rft.volume%253D39%26rft.issue%253D4%26rft.spage%253D496%26rft.epage%253D502%26rft.atitle%253DGenetic%2Bbasis%2Bof%2Bindividual%2Bdifferences%2Bin%2Bthe%2Bresponse%2Bto%2Bsmall-molecule%2Bdrugs%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1991%26rft_id%253Dinfo%253Apmid%252F17334364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1991&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17334364&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245271200019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.28" data-doi="10.1086/321272"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Are rare variants responsible for susceptibility to complex diseases?</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">124</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D69%26rft.issue%253D1%26rft.spage%253D124%26rft.epage%253D137%26rft.atitle%253DAre%2Brare%2Bvariants%2Bresponsible%2Bfor%2Bsusceptibility%2Bto%2Bcomplex%2Bdiseases%253F%26rft_id%253Dinfo%253Adoi%252F10.1086%252F321272%26rft_id%253Dinfo%253Apmid%252F11404818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/321272&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11404818&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170108100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.29" data-doi="10.1111/j.1558-5646.2011.01486.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The QTN program and the alleles that matter for evolution: all that’s gold does not glitter</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DRockman%26rft.auinit1%253DM.%2BV.%26rft.volume%253D66%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D17%26rft.atitle%253DThe%2BQTN%2Bprogram%2Band%2Bthe%2Balleles%2Bthat%2Bmatter%2Bfor%2Bevolution%253A%2Ball%2Bthat%2527s%2Bgold%2Bdoes%2Bnot%2Bglitter.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2011.01486.x%26rft_id%253Dinfo%253Apmid%252F22220860%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2011.01486.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22220860&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298739100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saeed</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharov</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">TM4: a free, open-source system for microarray data management and analysis</span>. <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">374</span>–<span class="cit-lpage">378</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechniques%26rft.volume%253D34%26rft.spage%253D374%26rft_id%253Dinfo%253Apmid%252F12613259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12613259&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180892500018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.31" data-doi="10.1126/sageke.2004.12.pe13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Service</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">How good are quantitative complementation tests?</span> <abbr class="cit-jnl-abbrev">Sci. SAGE KE</abbr> <span class="cit-vol">2004</span>: <span class="cit-fpage">pe13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2527s%2BSAGE%2BKE%26rft.stitle%253DSAGE%2BKE%26rft.aulast%253DService%26rft.auinit1%253DP.%2BM.%26rft.volume%253D2004%26rft.issue%253D12%26rft.spage%253Dpe13%26rft.epage%253Dpe13%26rft.atitle%253DHow%2BGood%2BAre%2BQuantitative%2BComplementation%2BTests%253F%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsageke.2004.12.pe13%26rft_id%253Dinfo%253Apmid%252F15044758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic2FnZWtlIjtzOjU6InJlc2lkIjtzOjEyOiIyMDA0LzEyL3BlMTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzcvNzUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.32" data-doi="10.1371/journal.pgen.0020013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicholson</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Complex genetic interactions in a quantitative trait locus</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSinha%26rft.auinit1%253DH.%26rft.volume%253D2%26rft.issue%253D2%26rft.spage%253De13%26rft.epage%253De13%26rft.atitle%253DComplex%2Bgenetic%2Binteractions%2Bin%2Ba%2Bquantitative%2Btrait%2Blocus.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020013%26rft_id%253Dinfo%253Apmid%252F16462944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16462944&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.33" data-doi="10.1038/416326a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spiegelman</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oefner</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Dissecting the architecture of a quantitative trait locus in yeast</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">416</span>: <span class="cit-fpage">326</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSteinmetz%26rft.auinit1%253DL.%2BM.%26rft.volume%253D416%26rft.issue%253D6878%26rft.spage%253D326%26rft.epage%253D330%26rft.atitle%253DDissecting%2Bthe%2Barchitecture%2Bof%2Ba%2Bquantitative%2Btrait%2Blocus%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F416326a%26rft_id%253Dinfo%253Apmid%252F11907579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/416326a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11907579&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174482200044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.34" data-doi="10.1093/jhered/esr026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okada</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teramura</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide deficiency mapping of the regions responsible for temporal canalization of the developmental processes of <em>Drosophila melanogaster</em></span>. <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">448</span>–<span class="cit-lpage">457</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesr026%26rft_id%253Dinfo%253Apmid%252F21525178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjk6IjEwMi80LzQ0OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNy83NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.35" data-doi="10.1002/yea.1074"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vorachek-Warren</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title"><em>DsdA</em> (D-serine deaminase): a new heterologous MX cassette for gene disruption and selection in <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DVorachek-Warren%26rft.auinit1%253DM.%2BK.%26rft.volume%253D21%26rft.issue%253D2%26rft.spage%253D163%26rft.epage%253D171%26rft.atitle%253DDsdA%2B%2528D-serine%2Bdeaminase%2529%253A%2Ba%2Bnew%2Bheterologous%2BMX%2Bcassette%2Bfor%2Bgene%2Bdisruption%2Band%2Bselection%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1074%26rft_id%253Dinfo%253Apmid%252F14755641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1074&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14755641&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188977600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.36" data-doi="10.1002/yea.320101310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wach</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brachat</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pöhlmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Philippsen</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">New heterologous modules for classical or PCR-based gene disruptions in <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1793</span>–<span class="cit-lpage">1808</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DWach%26rft.auinit1%253DA.%26rft.volume%253D10%26rft.issue%253D13%26rft.spage%253D1793%26rft.epage%253D1808%26rft.atitle%253DNew%2Bheterologous%2Bmodules%2Bfor%2Bclassical%2Bor%2BPCR-based%2Bgene%2Bdisruptions%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.320101310%26rft_id%253Dinfo%253Apmid%252F7747518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.320101310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7747518&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994QJ42800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2.7.753.37" data-doi="10.1038/nrg1522"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W. Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barratt</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clayton</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genome-wide association studies: theoretical and practical concerns</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DW.%2BY.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D109%26rft.epage%253D118%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%253A%2Btheoretical%2Band%2Bpractical%2Bconcerns.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1522%26rft_id%253Dinfo%253Apmid%252F15716907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1522&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15716907&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F7%2F753.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226721300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/7/753.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/7/741" title="Mutations in the pqe-1 Gene Enhance Transgene Expression in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/7/761" title="Impact of Loci Nature on Estimating Recombination and Mutation Rates in Chlamydia trachomatis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/7"><span>Volume 2 Issue 7, July 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/7.cover-source.jpg?itok=tq8UCk63" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/7/753.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tUnxWxEPLf3eR8-NDE9GcpQDwwAVUEBvWoixqkXtXWY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/7/753" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/7/753" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/7/753" class="active" data-icon-position="" data-hide-link-title="0">A Noncomplementation Screen for Quantitative Trait Alleles in Saccharomyces cerevisiae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459294" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-3HScYLsjguxgPL9cAp2ovRqliQRUmz2TPnqgIekwrJU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/7/753.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/7/753" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-hoFoT-Yned1ubRLCvgLYfH8SXiei4AeR9HIttjCel-M" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/7/753" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/7/753.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459294" id="node459294" data-pisa="ggg;2/7/753" data-pisa-master="ggg;2/7/753" data-apath="/ggg/2/7/753.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Noncomplementation Screen for Quantitative Trait Alleles in <em>Saccharomyces cerevisiae</em></h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hyun Seok Kim</span>, <span class="highwire-citation-author" data-delta="1">Juyoung Huh</span>, <span class="highwire-citation-author" data-delta="2">Linda Riles</span>, <span class="highwire-citation-author" data-delta="3">Alejandro Reyes</span> and <span class="highwire-citation-author" data-delta="4">Justin C. Fay</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">753-760;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002550 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459294" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hyun Seok Kim</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Genetics</span><span class="nlm-aff">Computational Biology Program, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hyun%2BSeok%2BKim" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20HS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHyun%2BSeok%2BKim" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Juyoung Huh</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Genetics</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juyoung%2BHuh" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huh%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuyoung%2BHuh" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Linda Riles</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Genetics</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Linda%2BRiles" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Riles%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALinda%2BRiles" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Alejandro Reyes</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Computational Biology Program, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alejandro%2BReyes" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reyes%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlejandro%2BReyes" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Justin C. Fay</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Genetics</span><span class="nlm-aff">Computational Biology Program, and</span><span class="nlm-aff">Center for Genome Sciences and Systems Biology, Washington University, St. Louis, Missouri 63108</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BC.%2BFay" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fay%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJustin%2BC.%2BFay" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jfay@genetics.wustl.edu" class="" data-icon-position="" data-hide-link-title="0">jfay@genetics.wustl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/7/753" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/7/753&amp;title=A%20Noncomplementation%20Screen%20for%20Quantitative%20Trait%20Alleles%20in%20Saccharomyces%20cerevisiae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/7/753&amp;title=A%20Noncomplementation%20Screen%20for%20Quantitative%20Trait%20Alleles%20in%20Saccharomyces%20cerevisiae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/7/753&amp;title=A%20Noncomplementation%20Screen%20for%20Quantitative%20Trait%20Alleles%20in%20Saccharomyces%20cerevisiae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/7/753&amp;text=A%20Noncomplementation%20Screen%20for%20Quantitative%20Trait%20Alleles%20in%20Saccharomyces%20cerevisiae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/7/753&amp;title=A%20Noncomplementation%20Screen%20for%20Quantitative%20Trait%20Alleles%20in%20Saccharomyces%20cerevisiae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/7/753&amp;t=A%20Noncomplementation%20Screen%20for%20Quantitative%20Trait%20Alleles%20in%20Saccharomyces%20cerevisiae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/7/753&amp;title=A%20Noncomplementation%20Screen%20for%20Quantitative%20Trait%20Alleles%20in%20Saccharomyces%20cerevisiae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/7/753&amp;title=A%20Noncomplementation%20Screen%20for%20Quantitative%20Trait%20Alleles%20in%20Saccharomyces%20cerevisiae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459294/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459294" id="node459294" data-pisa="ggg;2/7/753" data-pisa-master="ggg;2/7/753" data-apath="/ggg/2/7/753.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Noncomplementation Screen for Quantitative Trait Alleles in <em>Saccharomyces cerevisiae</em></h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hyun Seok Kim</span>, <span class="highwire-citation-author" data-delta="1">Juyoung Huh</span>, <span class="highwire-citation-author" data-delta="2">Linda Riles</span>, <span class="highwire-citation-author" data-delta="3">Alejandro Reyes</span> and <span class="highwire-citation-author" data-delta="4">Justin C. Fay</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">753-760;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002550 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459294" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hyun Seok Kim</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Genetics</span><span class="nlm-aff">Computational Biology Program, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hyun%2BSeok%2BKim" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20HS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHyun%2BSeok%2BKim" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Juyoung Huh</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Genetics</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juyoung%2BHuh" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huh%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuyoung%2BHuh" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Linda Riles</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Genetics</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Linda%2BRiles" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Riles%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALinda%2BRiles" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Alejandro Reyes</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Computational Biology Program, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alejandro%2BReyes" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reyes%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlejandro%2BReyes" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Justin C. Fay</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Genetics</span><span class="nlm-aff">Computational Biology Program, and</span><span class="nlm-aff">Center for Genome Sciences and Systems Biology, Washington University, St. Louis, Missouri 63108</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BC.%2BFay" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fay%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJustin%2BC.%2BFay" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jfay@genetics.wustl.edu" class="" data-icon-position="" data-hide-link-title="0">jfay@genetics.wustl.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=753&amp;rft.epage=760&amp;rft.atitle=A%20Noncomplementation%20Screen%20for%20Quantitative%20Trait%20Alleles%20in%20Saccharomyces%20cerevisiae&amp;rft.volume=2&amp;rft.issue=7&amp;rft.date=2012-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kim%2C+Hyun+Seok&amp;rft.au=Huh%2C+Juyoung&amp;rft.au=Riles%2C+Linda&amp;rft.au=Reyes%2C+Alejandro&amp;rft.au=Fay%2C+Justin+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459294/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459294/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459294/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459294/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459294/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459294/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459294/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459294/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459294/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459294/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459294/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459294/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-ajax-pane-content pane-highwire-permission-link" >
  
        <h2 class="pane-title"><span class='panels-ajax-pane-title' data-pid='new-37'></span></h2>
    
  
  <div class="pane-content">
    <div class='panels-ajax-pane panels-ajax-pane-new-37' data-pid='new-37'></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494625" id="node494625" data-pisa="ggg;9/11/3891" data-pisa-master="ggg;g3.119.400723" data-apath="/ggg/9/11/3891.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3891" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Mouse Mutation That Dysregulates Neighboring <em>Galnt17</em> and <em>Auts2</em> Genes Is Associated with Phenotypes Related to the Human AUTS2 Syndrome</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494624" id="node494624" data-pisa="ggg;9/11/3781" data-pisa-master="ggg;g3.119.400647" data-apath="/ggg/9/11/3781.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3781" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Transgenic Analyses in Drosophila Reveal That mCORL1 Is Functionally Distinct from mCORL2 and dCORL</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494622" id="node494622" data-pisa="ggg;9/11/3843" data-pisa-master="ggg;g3.119.400363" data-apath="/ggg/9/11/3843.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3843" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Analysis of Cell Cycle-Regulating Genes in the Symbiotic Dinoflagellate <em>Breviolum minutum</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/7/753" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/7/753.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/7/753.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/7/753.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/7/753.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2019 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
  </body>
</html>
