<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/8/905" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/8/905.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/8/905.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/8/905.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/2/8/905/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/8/905.atom" />
<meta name="HW.pisa" content="ggg;2/8/905" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Resampling QTL Effects in the QTL Sign Test Leads to Incongruous Sensitivity to Variance in Effect Size" />
<meta name="DC.Identifier" content="10.1534/g3.112.003228" />
<meta name="DC.Date" content="2012-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Rice, Townsend. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Allelic effects at quantitative trait loci (QTL) between lineages are potentially informative for indicating the action of natural selection. The QTL Sign Test uses the number of + and − alleles observed in a QTL study to infer a history of selection. This test has been constructed to condition on the phenotypic difference between the two lines in question. By applying the test to QTL data simulated under selection, we demonstrate that conditioning on the phenotypic difference results in a loss of power to reject the neutral hypothesis and marked sensitivity to variation in locus effect magnitude." />
<meta name="DC.Contributor" content="Daniel P. Rice" />
<meta name="DC.Contributor" content="Jeffrey P. Townsend" />
<meta name="article:published_time" content="2012-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Resampling QTL Effects in the QTL Sign Test Leads to Incongruous Sensitivity to Variance in Effect Size" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Allelic effects at quantitative trait loci (QTL) between lineages are potentially informative for indicating the action of natural selection. The QTL Sign Test uses the number of + and − alleles observed in a QTL study to infer a history of selection. This test has been constructed to condition on the phenotypic difference between the two lines in question. By applying the test to QTL data simulated under selection, we demonstrate that conditioning on the phenotypic difference results in a loss of power to reject the neutral hypothesis and marked sensitivity to variation in locus effect magnitude.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/08/01" />
<meta name="citation_mjid" content="ggg;2/8/905" />
<meta name="citation_id" content="2/8/905" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/8/905" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/8/905.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/8/905.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/8/905.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.003228" />
<meta name="citation_pmid" content="22908039" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="905" />
<meta name="citation_lastpage" content="911" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Daniel P. Rice" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Yale University, New Haven, CT 06520" />
<meta name="citation_author" content="Jeffrey P. Townsend" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Yale University, New Haven, CT 06520" />
<meta name="citation_author_institution" content="Program in Computational Biology and Bioinformatics, Yale University, New Haven, CT 06520" />
<meta name="citation_author_email" content="Jeffrey.Townsend@Yale.edu" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=R. Albertson;citation_author=J. Streelman;citation_author=T. Kocher;citation_title=Genetic Basis of Adaptive Shape Differences in the Cichlid Head;citation_pages=291-301;citation_volume=94;citation_year=2003;citation_issue=4;citation_pmid=12920100;citation_doi=10.1093/jhered/esg071" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. Anderson;citation_author=M. Slatkin;citation_title=Orr's quantitative trait loci sign test under conditions of trait ascertainment;citation_pages=445-446;citation_volume=165;citation_year=2003;citation_pmid=14575044" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=N. Barton;citation_author=H. De Vladar;citation_title=Statistical mechanics and the evolution of polygenic quantitative traits;citation_pages=997-1011;citation_volume=108;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Bedford;citation_author=D. Hartl;citation_title=Optimization of gene expression by natural selection;citation_volume=106;citation_year=2009;citation_pmid=19139403;citation_doi=10.1073/pnas.0812009106" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=C. Boake;citation_author=S. Arnold;citation_author=F. Breden;citation_author=L. Meffert;citation_author=M. Ritchie;citation_title=Genetic tools for studying adaptation and the evolution of behavior;citation_pages=S143-S159;citation_volume=160;citation_year=2002;citation_pmid=18707473;citation_doi=10.1086/342902" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. S. Chang;citation_author=M. A. F. Noor;citation_title=The genetics of hybrid male sterility between the allopatric species pair Drosophila persimilis and D-pseudoobscura bogotana: dominant sterility alleles in collinear autosomal regions;citation_volume=176;citation_year=2007;citation_pmid=17277364;citation_doi=10.1534/genetics.106.067314" />
<meta name="citation_reference" content="citation_journal_title=Paleobiology;citation_journal_abbrev=Paleobiology;citation_author=B. Charlesworth;citation_title=Some quantitative methods for studying evolutionary patterns in single characters;citation_pages=308-318;citation_volume=10;citation_year=1984;citation_issue=3;citation_georef=1985039484" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. Grant;citation_author=B. Grant;citation_title=Unpredictable Evolution in a 30-Year Study of Darwin's Finches;citation_pages=707-711;citation_volume=296;citation_year=2002;citation_issue=5568;citation_pmid=11976447;citation_doi=10.1126/science.1070315" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Kimura;citation_title=On probability of fixation of mutant genes in a population;citation_pages=713-719;citation_volume=47;citation_year=1962;citation_pmid=14456043" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=C. Lai;citation_author=J. Leips;citation_author=W. Zou;citation_author=J. Roberts;citation_author=K. Wollenberg;citation_title=Speed-mapping quantitative trait loci using microarrays.;citation_pages=839-841;citation_volume=4;citation_year=2007;citation_issue=10;citation_pmid=17873888;citation_doi=10.1038/nmeth1084" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=R. Lande;citation_title=Natural-selection and random genetic drift in phenotypic evolution;citation_pages=314-334;citation_volume=30;citation_year=1976;citation_doi=10.2307/2407703" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=R. Lande;citation_title=Statistical tests for natural-selection on quantitative characters;citation_pages=442-444;citation_volume=31;citation_year=1977;citation_doi=10.2307/2407764" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=R. Lande;citation_author=S. Arnold;citation_title=The measurment of selection on correlated characters;citation_pages=1210-1226;citation_volume=37;citation_year=1983;citation_doi=10.2307/2408842" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Lexer;citation_author=D. Rosenthal;citation_author=O. Raymond;citation_author=L. Donovan;citation_author=L. Rieseberg;citation_title=Genetics of species differences in the wild annual sunflowers, Helianthus annuus and H-petiolaris;citation_pages=2225-2239;citation_volume=169;citation_year=2005;citation_pmid=15545657;citation_doi=10.1534/genetics.104.031195" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=M. Lynch;citation_title=The rate of morphological evolution in mammals from the standpopint of the neutral expectation;citation_pages=727-741;citation_volume=136;citation_year=1990;citation_doi=10.1086/285128" />
<meta name="citation_reference" content="citation_journal_title=Journal of mathematical biology;citation_journal_abbrev=J Math Biol;citation_author=J. R. Miller;citation_author=M. C. Pugh;citation_author=M. B. Hamilton;citation_title=A finite locus effect diffusion model for the evolution of a quantitative trait.;citation_pages=761-787;citation_volume=52;citation_year=2006;citation_issue=6;citation_pmid=16463184;citation_doi=10.1007/s00285-005-0368-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Orr;citation_title=Testing natural selection vs. genetic drift in phenotypic evolution using quantitative trait locus data;citation_pages=2099-2104;citation_volume=149;citation_year=1998;citation_pmid=9691061" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=H. Orr;citation_title=The evolutionary genetics of adaptation: a simulation study.;citation_pages=207-214;citation_volume=74;citation_year=1999;citation_issue=3;citation_pmid=10689798;citation_doi=10.1017/S0016672399004164" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Orr;citation_title=The distribution of fitness effects among beneficial mutations;citation_pages=1519-1526;citation_volume=163;citation_year=2003;citation_pmid=12702694" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. P. Rice;citation_author=J. P. Townsend;citation_title=A test for selection employing quantitative trait locus and mutation accumulation data;citation_volume=190;citation_year=2012;citation_pmid=22298701;citation_doi=10.1534/genetics.111.137075" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. Rieseberg;citation_author=A. Widmer;citation_author=A. Arntz;citation_author=J. Burke;citation_title=Directional selection is the primary cause of phenotypic diversification;citation_volume=99;citation_year=2002;citation_pmid=12221290;citation_doi=10.1073/pnas.192360899" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=J. R. True;citation_author=J. J. Liu;citation_author=L. F. Stam;citation_author=Z. B. Zeng;citation_author=C. C. Laurie;citation_title=Quantitative genetic analysis of divergence in male secondary sexual traits between Drosophila simulans and Drosophila mauritiana;citation_pages=816-832;citation_volume=51;citation_year=1997;citation_doi=10.2307/2411157" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=M. Turelli;citation_author=J. Gillespie;citation_author=R. Lande;citation_title=Rate tests for selection on quantitative characters during macroevolution and microevolution;citation_pages=1085-1089;citation_volume=42;citation_year=1988;citation_doi=10.2307/2408923" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Resampling QTL Effects in the QTL Sign Test Leads to Incongruous Sensitivity to Variance in Effect Size" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8.cover-source.jpg" />
<meta name="twitter:description" content="Allelic effects at quantitative trait loci (QTL) between lineages are potentially informative for indicating the action of natural selection. The QTL Sign Test uses the number of + and − alleles observed in a QTL study to infer a history of selection. This test has been constructed to condition on the phenotypic difference between the two lines in question. By applying the test to QTL data simulated under selection, we demonstrate that conditioning on the phenotypic difference results in a loss of power to reject the neutral hypothesis and marked sensitivity to variation in locus effect magnitude." />
<meta name="og-title" property="og:title" content="Resampling QTL Effects in the QTL Sign Test Leads to Incongruous Sensitivity to Variance in Effect Size" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/8/905" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Allelic effects at quantitative trait loci (QTL) between lineages are potentially informative for indicating the action of natural selection. The QTL Sign Test uses the number of + and − alleles observed in a QTL study to infer a history of selection. This test has been constructed to condition on the phenotypic difference between the two lines in question. By applying the test to QTL data simulated under selection, we demonstrate that conditioning on the phenotypic difference results in a loss of power to reject the neutral hypothesis and marked sensitivity to variation in locus effect magnitude." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8.cover-source.jpg" />
    <title>Resampling QTL Effects in the QTL Sign Test Leads to Incongruous Sensitivity to Variance in Effect Size | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q0wst1" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__KPgiwEdjo5O9HdRryHqQs1TpsV-PeBb9B7sGBvwM9FQ__opzZWgPNgQzoEgfATRS5qCKwBw_6XcNjDHoDVOzQv8M__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__YUqwBlv6bNoCToiBWgAUSPtppdS8sbGfzmAH3qEDcI0__Mvk93h3ScFH5Xi9WhibVKlf368fHw-V7HWqWX2QMvtw__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XNWYaQubFGWYWZNytUHkWs-b34LV21DsB4s_6gchI4k__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__7cpBA0GVdrMqkZGGZEW-_SPF2HrbTCmVg9-lrP5l_AQ__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__TV2D4R18tPiSIOmeQ3vyMqHObN_FDmQ6EEIrfE5sAoA__c9BBZFy1BrgGgF_Auu9Fmysb3Z7J56eJGYvMcLQrLks__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q0wst1"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__OaHnYyci68QTwctdRYR30kEfu0eYWBrLyMqy8THv5Ik__IyfJQYM1q1QEPwYsK_Eslh2RQkxpy6yo-nR0p4DWPQQ__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__fhq_kmKS3zjIxlsAa3GE1ubLajyXUgXXTXULzAtiqfg__Lm1_Ls-bdpbWRyl0q4o0mmHyxB-139AJ68dE4zOskLQ__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"-cIm1quONHwDacrXqSa0o2R8Bh-28XrYYcrw2VR8_hY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459309","apath":"\/ggg\/2\/8\/905.atom","pisa":"ggg;2\/8\/905","ac":{"\/ggg\/2\/8\/905.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/8\/905.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/8\/905"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/8\/905"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/8\/905":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222HxrvyGYnHU2aDsVhXuFaOflG7LF7YYKfOiTzZdX6Ats9GcaUiGtp1omFEadLAm2knPTwg9wuiw1uu8zQR1RdVA3fCABuaxMCZ9CmilECwSwSOWA26Ih\\\\\\\/wbvShdV3U9rm4np2PzqFCJXNUP7MEi\\\\\\\/uFAEOn4X+6zxm9XojtSLazuJiNW8pCtoBjBIx8jOHUg+rjOkgWqLM7oHWbZtcT8tTwwoiZVA3Rjs35j8+Tbe\\\\\\\/B1PhS5YK3I506npGBmCj\\\\\\\/zRXYMaeM\\\\\\\/zfhTGCRT3lTDvb\\\\\\\/3AQT3Cqav7DAmZNnOTZKiFsugOf95a2M1XbU3umyburtRYlGlHprVEB6lS0UedxV5qKjPXufCB0+Qt4DPE\\\\\\\/KeIjv3TTMw5tYae2WVS33caueH3HAptdp7ISzt+f3iT8EpmrFm1IpfDU\\\\\\\/J7eNPLAwpLaOs7pcxitM1pPirEZXng+wNqEuppInOsIxFKrS\\\\\\\/OOTAj\\\\\\\/mUZGG+GRSlOTkvUfFNuUOJlQUtRXj8xcnvbTZ2M6g10B7n2lxvU7rTqCg7rthm6xplJsLhwkoNbRj2Vi6nUXPH\\\\\\\/goM+XopFk4FfyJXFwygWGVnuoh9rW\\\\\\\/j3aNJhBMOHuPYYIZLeqgEDilddC7twWcv9nnw7DvY1EdK9SgQqlq0Ufw4CDBDizi45i5ye\\\\\\\/mFbk4ZvCAFpYoqpXaYgcNzozaTnRo9ap4TnskkzS5JwKrc0FtlvfP6GPJD3clSyr0M7hB8ClA\\\\\\\/Y3pNUZUxdUxmRzITk8NkEopmATfjO1B7rEQCzfYy7cJNkXaSTwRsSd33Qj3tXaasuNiJbDvl+DXx3\\\\\\\/SiFuSpX2d62ZahHITEie\\\\\\\/HfX\\\\\\\/dXbyvzf4pZiPgJBMdfFr6GnN7PJa5HoIACL6Rhm8UF5miE0QeA0XWN882FY5pKkrYGkvPR0p3YwZoO6RtQyS021RHZcNxRO6QRxl\\\\\\\/neCBJmnRtlRwD8FNwMqbznbP6+3VQO\\\\\\\/5qjwyXVvcWyb6BrqJMdT6s14D9Lf0+HpueA3AOUPCJCH1G8kbIExTWKl6O3Nof68UTa8u1MRko6NkXEZ+q5p3FxZbAfKK3NTdZocByeX4dTQdVPqEsPrLVfDfDcPKejbm3SquM1KYxa1HOE2oITeHVUEGKk5aSUfJwPqG8dX07nGKY30qgaZvWWGj0U2JVTJja6i614lL6CkFgo0YdllIy3rUovZjdHcVVmHIhxj3tY2EEWFyxMcUFz\\\\\\\/lPQ7lvtThNgvTo5T7xE3GNwmMvV0waXMrATuu7Ba6okwI3fbrpEEpER\\\\\\\/WQTnpP\\\u0022,\\\u0022iv\\\u0022:\\\u00225b4QIzVEKVWKj1Mnq\\\\\\\/jZGA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002282aedbd9513ddefc211206ee2609f77cb8a438ac3060319e8a9b4d59bd9ebdc8\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mjI4NxiSJwF9Ka8lTEM2PckGJwos7jZvFfCBROYhLDKxxF\\\\\\\/2ks8Lkw0rOh0vhx1hTP3QS0EI3h9XHNP\\\\\\\/1Z5Xltdh7LJ2n98sLGlSSxRFfjApYSLcSQByKU+TxbRUw0WUmmBWAi8pTSxY1KLz0AiOcom02uY5RbuOfRXGvccNQWPx2FVYCOThxDyNCPJaXfaBrhnaUjK7t0Z71o1Xgd1jU2odQkzEVDnzubp1P5OpFBouYiB22XAgxUO4+VTEP5CsdSMNFI\\\\\\\/keF4ntRIs90aKoh3efzKXcgHldHXwfzyFK6H3CEI7XyXiLXh65+FaQAYCj\\\\\\\/Uv8Rlfktym8A+00y0p8g6vF\\\\\\\/uCNWjwFzp15oJIcvLFMMwXMjqOij4uM+wd7eAdv7GtdLfV2sLvsbgTDnFpDWFcspyUh27YdHsLT4oaCJHo51JybY1Rt1wjZIXpldj9k+xw3E8Yhw6WumAEeRPsU+fFO1a7rXb\\\\\\\/Y5ag28AEGvN7Q55qGCH9mTnYkHLom0Q4\\\\\\\/NiGxiNPuIy0woT0zr4JlCsIY6uS\\\\\\\/LspZMtxPNANSlYqXVHxnYO\\\\\\\/lA8kqF4rw8M0nBcOenEVuxbBi+C78PGFMIl+yjN55yw7TTe6SfmQudmg00Uf0AlTgDPtiwU3JYNq4aCwjzAB7CfT+vTHDBy78v0Mo1CCKYDpaQpgOkTXah6bXkYAtr6yMGXmJ9hQJcEdWSDtQg1ic5+u8W\\\\\\\/iFv2P+YFDXBV1m2b6HaOwkWluiXftRsQaqdUVYItzmdTma35aGHQpk3xhD0jCva\\\\\\\/IFDwxu+Y3IlWCufkyHRfGVIbQHlWlqbD5Qi81l5dfm+IFfKce8X3eS+6+KGw8UW4V2Axwa+LlHcWa98DpJaNnyueyhMmOvo7YuaCQN4Dkgla5tSOc\\\\\\\/kkBj4RnI\\\\\\\/GcUfJR3t9qdHdbzRcjaANHXLKsECkcHh9ecFV9Ko2JxHJ6441sbI\\\\\\\/hxc3sT1HVFKI1\\\\\\\/RLLfb0se3LSTS\\\\\\\/G3dqVcmz4w0YDUPzTN7WDj0B46eLufZQTygHg39v4K48c8gDkUrKYj9FUxQYz2U+TfItFIkj+lEAYSs6mJj3fm1JpvWUfV9RGn2lvxgLuDqJHL1muW3ivfYMdBnFREWFBgsR0Qfe158+0+zKMiRrVGCiTNvi6HPtLsum1R7ZJ8BhdjZRwnTzypqefNwlymVpByOkWOfki50CiVj7CV4IJ4PmpDRXSo0T3qTkjWzeHHWb1rGUsZ+vy0UbQTV+YOtsxzIM4+IDPbrCJLAZVaSNVMjeB+ukvHZ2qHbs5Q2dQojI3eHuhWPUgU8gSUHpc70KpUpC3bCeAm32Mau911EG2wkiTwi34pcXsqjdfhmwAG1KPcSK+6LJU8sxh1ckA0MyQN2jaF05LbPB4bjzBxFmtT8ETT7Q8J5WN5V\\\\\\\/NdPIRWe\\\\\\\/9xI3+j7ZcnjjHv++BrcAkyKxxswzc1pv39U\\\\\\\/7mrK57zpzQCxSLI7sA8zINzIx0lulXk8yv1vpactfxv7b3e1uKe7rOvptG5itpZZ9nQAoEY9jNIL4q1+NZ8rdUshz6Y5BG5jGDFONJApSUUJFnnFGKV09xuLBd7rAtkRXpa9V3BcsQKLqY+g3fgosXDCrl0hmr3cnQtC6dSAyI5QoNRNiCNvtZrMeTo\\\\\\\/NP0yp7P1QTCvIQkm+rS2csWewjV4fWwKrpDq3TDnVsjLjFTyj1VjfqhWOVsVRFsU5DPD2a\\\\\\\/oYstzd\\\\\\\/tMBgAyBgo57Fsw7SqukXuJ3UAgS787pSdgLO7MoAy159eA6oeaS5sKqeWWKE4Q9A0nKV7cR1ThRm3hKslW0+0x3cLrWcW+nVh0WmYPTXjbekB\\\\\\\/LclLz9jNYSiASmnQiEumbTSyknQPNq9r53uBrYRANU14ivZ7eJCcHs2Vq66IZGQjWlnzg\\\\\\\/WKIv9zvS6AsvhoH1zaYaSDO4mgQyURA4Tl1TdvTq3+AdegMv9f1PMFUTcZH1F\\\\\\\/YOZ+vnIkLxqhjm7aluKUqQEZQCr03IadrIlE+i9vuFUxWIVAJIuo7oZ8U7QWX0Zn2MBZsbNBEy3Y3QTmxIRt4ITo5cba2xyD6RZPsyr3t4ttD67gle6SyGiSwWvhW6neY\\\\\\\/f+raoKFmxfPvBrfFZPQXTYchLNtpEMQoHumzsL3iKFydrvmJ6sAsr5f6gQAJ\\\\\\\/GA+eSS5+08iOJB\\\\\\\/AT9ff4BLopNfnlfCFC3JfURTf0n+wGbcDr95DHrYb8MmCsRCH8FY+LLhFJ2i3u1jXUNZXOwir19ScltoSTea7\\\\\\\/jSWHqKXCbhqQY1oFEKaVt9SmQpBmOyMJpcdpRC44lPlywEEY8QD+av4UhoMyKnsx5sglzrTwGAi5Pjq2jSBRyN3dZltZmrkQQ8b90kl7lTAAmlcB7zsNF2J5laUXPa5vXc7pRAuCxjjbEOSdzL8FcIGDUbZNvz0GSVA5t2gy1rnDwyLzDAbGIRma2lxaUhtbXLSQW9DTOhpwPpD1Zxllyz+0yMRmwQd6OskBBgZdOVkXv62J96cXYs34o2r2a3rs5P957elHqHEH1CSHOnEhQaQQSiYuMosydyJ7ckhIl6Ja5gpqc8lFnT2M574jegkzsROlWWLPdvxFqPsnqWlhPWRSX4dUUNTWQIcnlkbdTcJn4\\\\\\\/0HIwo3aNEpcVaJaheV2CWUaaHkyDAq6OZ\\\\\\\/lvW\\\\\\\/3P79eHPMWDpbwqd3T9b5Dgow4LG6kmk96u\\\\\\\/95hMMW3LajFoPfv03JHtr\\\\\\\/IdClfEKTDzm0Qg0U+VIIWSVe+tQd6+09yrYJJc5zuTiYH5t5j6gjsTw1otM47iCUDHtlsBrg4FIFFfohnufkjG2sD3Rr1R68AqX\\\\\\\/v7\\\\\\\/MaagxER36RujF5Hsm\\\\\\\/TtNDVWHcE60yuxbrm08+v9QIc5DAFQpEPZ7HLdDfqSip4C8BcjpsQMZFXv5FYpyTYsAKFgLxd++H9KZIPhBLovFMa9999elUo51cr4m8zO7bm8T3+9C9WSfqdx1Q9YaJpES7cb7jkebztEkTHfQzSlf4mDupnh6+RbmFKLXtQz1AJol9mzS9SRTk2KbHWFlb2fHDgT8Asy5G7JkxHvB4sH3TE7NjPc0ee8EABDacLu+eduvBUKzH6mQ5M5XTT\\\\\\\/l\\\\\\\/pbKaaWQDkDtmQqhaO8NUqanKIL\\\\\\\/4b\\\\\\\/85Nsdu3hijB9SAQwhg4g4zYvmxiIfvXNFdRU1SE+RUgc0eFY4Al5HiwBUjv69qJeobr4HzbtXtExNVCbBbBa60obMzhcuCrq6p3EJyJCfVtvx9fARVMi9uck3Y1tMl7PQGlS57lACMUqUkZ2hYn+2oOh\\\\\\\/zNTY5+G8tGvl6DpQF0bwVEMVLcwZTybLkmmMEeh1u2uqkobRn23CuWzpgvY7eiR1oIYyOTTD+r95Kj16hUjhFxW2HUd1YdbfuOuD1EANnf+cYgELjbz+9alYqnL5\\\\\\\/JNqP9J1b0Hy7t8+gb6RyGTorGgBltcMoV7Am2hPsiIPPmZ+Id80iPh\\\\\\\/lMfHG9XAhdX7hnQqToQIuAJItFo7Kva07Ek\\\\\\\/bTTZsH03z0reolyq1cgDmPZpbM+vYgXmM8SQsuYJJbutZlWqmTriJV\\\\\\\/AxGov2gwZo84ZUJY2yt7O9PqwqgprogUr3phRVaEFXVKUmsEmQHaYTYwHHfZq\\\\\\\/UgIQEpgtlXngNFXZw2uNH\\\\\\\/GWberZevmkHEWEd6OIBKymKYUsZAkVFRsFKRPdm+LLgATicHmpeGjK4RAIwT\\\\\\\/R9U76CX21FWeDdoqenD6hBqO5ri2Sq+g\\\\\\\/jPzYFpUh3+M3ArOKVTUWs+J+IrNrizYwF5LLLpbnDRrAmNZi89ag9obIw2kO1yqS+8SXnaw6dcQmb5K9xjuEX3Wvk2vqCWh0hEYOuKbnTz+XvMoxUtYHopm72rmzwMaP55CG6kncIRe1r5LjE4sD4mcsDXp7zCoFcH0DC+YcubFQbbq71iz5x58jczbcz8qUAC\\\\\\\/+poespFb6xqxro1c6trgbscKxleqdTCTPxHV70CSLGgrDvJzc3qs06NWZqPuizhQAmY85rcY5Pdhvcq\\\\\\\/Hl1GsrJty7jsQ9AvF4XX7dXfFm6S3VI\\\\\\\/d1c2u2vGH\\\\\\\/203oxHWo4w1xswuE1blPHsh5mqYH27AYrgXGat4BbYbM\\\\\\\/HTxtRJCG3aUCfT7qK40ZFP\\\\\\\/9KUZAlnDi4jNIjIV9kECZ9ykXZOqD7dVF5ASznHIBfKKZ492C+178iH0Q1FX59bL9trA11mS1SG62hQkvXYcDE4S1tlXaXkfb2KQT1mb30jq5RSVQx\\\\\\\/JNVKp9f7rQFyk7ROokzPXLF9brCarCtRXb7nEHqiZFrSFya77IP2bjG9b\\\\\\\/b2wwCrZ7WvT+qzEYyeU7qlev8VOV4wvNYg9aP1hFh8ZIvd4t46xrU2\\\\\\\/WxnkQeARQL4XBsB4rD3q4DnuyQsaKUSQAktFfPzoxHBjfwwlCr7C8QcXvzQmhqgdYbUDFu9y6nTPVKMDBE6knIykyUhK4btFtxkcJWwgi83hhYBUiHNOdrTBJr\\\\\\\/XugI5dVd42dRziDR0i\\\\\\\/yMDUn1W8yg30CmPswN\\\\\\\/zrCnQy\\\\\\\/HLeCxhv3lnQMd77fFXreYich9rf1rsBaNro+J44gYd0z3ahDzR7xucraZCO+MEchevcy7Wh0NbSObL9XgQkSxxaIQamm+sCTWs7KVJptt1GSWafnTTmovE\\\\\\\/OvnVmBtLqQ1bws3W7xmMIyax8C+ofStN3xBD0PiISQuItKNmTlFI+S18Cr6CGTAqUdS+NKH1aJpjSrnA6J6q\\\\\\\/JKDbmGaaqJl8JvrMkkfIfUBxG7AiPX\\\\\\\/zI9hdoHbUzb3CESnfRgtmn4bqTEaJBFbqd+utqgg12MliDKslCJ7RYdSStJxraMX9hsyXzNCM+7Po3u5lAb+t4nO2BcJcXFTiITP8YNe2xYBgUSVeTE7og9KeKvLUhX\\\\\\\/LbB+A86nhkvfhDpXpn9PemCqmYitj\\\\\\\/JPq8FRb1qv8SWCzNE7WldrK+KbiXx8UieNkzi8RCeXdM8vRAZbfTzX\\\\\\\/JLu2SOzX4KlD36NUWvunJqhvYaf4zZVydnF79BJBQ2IB42++Gm\\\\\\\/t+KLgKDfnJguRyvsQig9w\\\\\\\/CVbTHuZ\\\\\\\/hWIQOe1BRAKFcrYV1nk15iF8YgsblR+WGlgeut9Y0xwy4\\\\\\\/OTSNTnj3VF8diqnYesFG34yytlYIwcmtMpOKuRsxFN63\\\\\\\/0O2JouwU0diwpLt52j9tdu4r2QUD4Sud3rtlQehGZX6QtjKxNSfI0c8tEm2rlJ\\\\\\\/xo60lRLync+8usHBZYSZsp6NDdM7PKp+2DYS6qplfGqzI52hI73kNN14vvtRUpaSCgLuUZKcUmbgIgKrX4Ruv0jXlfLGQIyL72qFJQpx+Uo\\\\\\\/1MQ6zndv22JQNqK8WMrGiKAzi2heL6ftewHqQgbiw37ynVCgrB\\\\\\\/\\\\\\\/rEGwNTGhs+hscVzGjRYKros\\\\\\\/84l9PwEkcRksPbyzo4QXioaPT55+QTiyIpvvQv\\\\\\\/tMlR3w92I94uivZe1mx4iWgMFEQhh7\\\\\\\/lvRVCBIIr+him0FF3CctZ4pLrTE6n7oAI3MeAaL2y1Rmmq2mp+UWYwuPShzxrm8PAcizWXYWOh9SMu3ZI73D\\\\\\\/TPl8wSMIDTXLOkFh2p0QEvjTU8FcWPw0sdyY4add8TvPzI8cqUTgXVWYnryK0rs+75vaNAeqBLLIxLDA4xaAwgGhwKvNXcIv2et+R1Vu+h034e\\\\\\\/q9OP3j8YIA5QNPen0j7qSsYZh3WvFsg3eCg43ghvXVCQS3yn\\\\\\\/IJKSmpjFDMZ0EcDU\\\\\\\/U7KbobqWBzfrx\\\\\\\/fYdu78FGAsQOcZ04YPXG4PTnIvSWXMQySnuWcoKFNtmLaC+DwfZtQOdT1RdyA7KPaPxrI4S8nBlnWxyLzoR53MnFHTRMEhQFnBS4qAWob6JEeUqgtIjlF6XEq6BrvVhKsSWR5AvWBL+DjIvHwS96SpZ+EBQYPGhDQ6JTO3Gi1yLbjjK32zbDni7mJgTrYeh4ydpIZLwik13j2893W6jF9t9GeqBfrYGIM1yiNenPg0VhYNctCDddK9gpruRObEiF3wsRkIWkmqL7+TlJ5KyjxBPUL137HTs5Z4pGhA1hfat\\\\\\\/SZ7DdJ1uGCLU+ZXDTrZ8+YZhHeIiskGL0gABuv+EN4wqrIZPPJqKN6AiGLfCpI9MqscGRbq9\\\\\\\/NkAp1Pfa+uOU+5X9hKbqYp0mIHq4Vs7jBb8b0wvpYOcVUrWXJYbcKC0jJPRNhf+TofnJgMy99QzFd05RkzQ6rE\\\\\\\/\\\\\\\/K\\\\\\\/TZK7R0OcFw5B+q7hjdwAdULajHGLZlA+4F8vFV0ENcxu9DIrQiMXBS1hN1j6jTfSsygn2x7+RDutBWxSOiGXLNkmHeCJHBGH\\\\\\\/QzCFOuWWrdJZvEmQtPYnNJX9iOeTNubIQjueykxTP55EtJK6uZMpL0UNA\\\\\\\/quDbooaOAXQ+1axFuLEiV\\\\\\\/GaUGrUw45T5TOR0WGHoUt6vy1rZNlSMi\\\\\\\/fm6m5i6CjC573lawd7aaonngpXFhfM+bUiXoXX3gq4tFZfOpHwQUuhLwzKsqWzp\\\\\\\/UeaV3oxDkdOb\\\\\\\/nQ+P5Gn85yQ65HL+uX4KUP4+l485jlwu66crCEGiszlCb+k15WvCn9T9LK0NiRLonNBPP0l1usbUyN4F1ewY3Ryky5dPqMOPGBi8aSD9STJ6YIC3se\\\\\\\/kNP9Q7Ll4SsdPqtJZEl5f+\\\\\\\/+rdR31aohf+aphipRLVP6y3rtANjCAx\\\\\\\/B1JN6E\\\\\\\/jYf3fWQZQ0XwpNoZ1Ykt\\\\\\\/1XtQ1MCX7wVkHiLuW3pm8wxKiRAvEahKWZLt7b1Qt4rXK4B3KrumszpYKeBxmPhi3p+5UoTzA87BOImg\\\\\\\/bTzl8QJQTANUZl7\\\\\\\/7WqEYMHr1MEBzOOFthILep2Hb2WP2epN7DM82RaBXGEOTBUQUf4A\\\\\\\/aAoBOoFNltqjrr4WFo12RNkX8DGJeetHDATUC14ONzgPafSBqmn7lnHFJaZlw9QTP2DZoiyMell6Yz54GvOrTIXQnp7rzQOUK6YNOHihgtlgHZwCIi7Z0QT0rCGONUqx0raujs0lfWIVs020Hu56XwlE7KH89mM2eMYVltDbKOxiI+RWpnfCVN+Mz3Lwm9p12VN4Eg1xYrZIaXXj4OS6g16FZ7pkSPmDy5DnpphQtyfk3LsTcXd85X7CYexbT6dHIDDXTx\\\\\\\/WzUSrAtrYvtvPkWFAdXwkF66oNV0HlQqaUVTSGYf\\\\\\\/r6b0a7UVcnKzMps6un1uhLi3W1M7i8+8R5Iq0L4cbxk1RywNANRchenFVAFbfd2aazQBrdoCiGflm09fv1BNqwkKy+ILQKIKKbMBuaVVrlW3KrbSkuR2LBghdPEuR0XEheHYRPagN\\\\\\\/99Q38iEhDZKRCwcLsA+0+71BIdgwr5BLLiHKGtL72LSK\\\\\\\/9eo\\\\\\\/j5\\\\\\\/Gf+DPz57AdLgu8Gd\\\\\\\/cy1V9\\\\\\\/m52BS61M\\\\\\\/egP4XUNqCLFTtcpw+aldxZvFVLPux+FJhk\\\\\\\/4xx8FBAd59irtB6PpEHbfeuTERqCovIjoExU8aXUU4sD3+dB+\\\\\\\/riIjWGvsGCngJ3WGVrAk3ZGEtWC+urDEj\\\\\\\/JADLTdLVZdOm731I1IeJJdNCs\\\\\\\/BRfX5OKPBwKIje2fVuquC1+RsTOSiiTkJRceS+v8noMFsuv660H4yPpiqjX0LkqKU0GMjQcsnF4DngltmADUgH+SvJo5BpuvEhCu7fAu6J9HkWvdeq52GrsL\\\\\\\/B4W9yHW6nXhfJC9GQsEH3nAye4McyTW2jPQALGTuq2HyE5z\\\\\\\/suXe7RltTBK6chLW6z9UTEsvq560tIDArET8nSX7Sh0pVxvuqfejRMVdawq7xGq9AwxrebRrHp8C4xwFZgu8W7DRH5syLGc3raM0LRklMQE0NmZQB+CeLti00CdxnmKw0LqePd5RsiCd20IEgRWJYonYL+3M2cGeoNqj8f9KIpMDc\\\\\\\/Ur6z3RISdk0LCq6DaL1eVF48Zvv6PNB1ZVoAysf8f9o9UE1QWz009e6ep93hkmsKITg7o3\\\\\\\/ZyP2LdIanjr3rvYyQdW6BlmLjle9ij\\\\\\\/bLF80dw7OqkAmh880tu\\\\\\\/XwHUZJuObJ1a7RR03cqsd54XPBTGnP5pPX9C3UD6mfmH5RNibO3sX\\\\\\\/uqYT2B3t0LwIBv5NTGYrRv4cuw18vTmv41DseMmpmQWE2+HHX+wQF00ZMEu8ldn2fs\\\\\\\/tIh2Q3s1Qs2f2XJUjDcktpNvvI39X9dXtOpPh0hT0GSmSmHtq8pXwe6V7U8NZxivulk0pxxSaChgl6APWmEl+hYTLc+2u4Jic4G1mpLs65sGHsK23d0cNsm0jCL\\\\\\\/4vH5NuxqdBQXHBxi8s5tFbXaMycSpZWLhS6ojliKvqZAkYWlM2\\\\\\\/R8yV6PzruBOceQJz1fEgLxWKT+HO\\\\\\\/zvu5TB8rSOXsfBbr8wT\\\\\\\/uF3UaPVRYphDYRk1quTdgh5DBZxF9Pzrmt0ckPC68ggZw0X2NgeTk1GSdfYiKq8QJwekho9lN5J4XZpv5CfpO36G0ZrWcG3S5yW4iVlhtgM7fb\\\\\\\/fqcXJ4Vt2YJp5cfA3Pz6LYW5E0l5IFfiDmgVDgo1WY6WENjAa3IcDgRt5R7iuZoB4TBtplsNvbWD1ot673ZzYsSYQ8mkP3PQEi1g2Z4JSrGp+9hhk8jW80fg9PvlVZ6MKktUV2ZHuccGyr9cLrToJSLgWPbVjE9OdFVdutWX89ZEu7Si99n\\\\\\\/YjzCFxRHdMTpduzTDhpd4wAkCv+SjRBqME22WovdFtXz9PKjMZWhAP9hdCvqbBEEd6ybkKPxJBPLe2mzj6Sj1C+MA6RdUkYJo0Mi3qPCuBmFsgrL1i9XklxDzAGGR5zU7UtqFU9FIsV5kTo0wCO5Wrx6dehgX5f+FuWGCedRJIaDvVYBvTgRhSyyxn+umBeO3WyUDcGyCE5OceihE5PpsfFH9+gO7Whk\\\\\\\/+zlKNDwo7CwBWY1kfCRUn5LCrfdXzd3cjyuh51fpmEWb+Hf+lSbeFybFQhgFNNI2xTgyCunmDaMr\\\\\\\/XWyfGrWyCgFt4VNOBc112Fu1bjrIydIMdspHfzk39DziXge7QglxOJbyfj+uv5ISLhQeG3vXtXP9Hj3rG27bJ0TU4v2foZSEoRpHlcFbUd3qIhA09MIE3CfSdOu0uatkdyIfokcUeYqyelxL+J84s24m2Osb8di39QKnsoQt9qpEntZXoz\\\\\\\/NnjuJsflBkiQtKAWpsWEujVOSacYEZkDHKslOUZb1xKwYW2U9DM13nGJBoxmPOOx1OFDXEWb+B596+QrxIoqBFubwIubA4Yi08lef0p3Jp7VUiAXVm6rKohRJcEPetvDKXSkYofkpjgt3yU6Rd9PK3gAh56ycuCLbb+7jBo7JH0U7l3tEo+54Wj05jprR\\\\\\\/+Q0nN+nSVvLmM0c0PD5W+Ww1xuf6sQ05GiC6iprT8EMpWGQpjDry\\\\\\\/sp7QuDak+JIKiJJPYZE4oQ3cxiugZ62SDCarwGSqDCl9SjxGFtas0y5yP8SwnFtFa08UWC6g8FX6rCB9rCOmp1pRA5zlYYH\\\\\\\/\\\\\\\/YiGbVCA8bvM+D3Ja2t8j7q5gDWqRsrEcz+f8ESRtrTRG8wcfoosUEOOoL79gnWrRgYMuO1lf\\\\\\\/\\\\\\\/aD6VaQ\\\\\\\/YolOomJCNnwhc4o0zTjEzWpCqB3F4ILxCU+xVVT2QiYdr5VCVDQQOgCfJGeuMisG9OAmswDnBwiU3OZQk\\\\\\\/BCrbg1xM969xukZkKv8NBfi7tEFaICZ78YjG4sG51frf1e6uz8RT3fQ7u8bkriqGFiUvE2sfJnWMLPvvWGo+bOoB78tWU8ujNnux3TZpMQwtXasEcRyMm9skFqiTMRYu6kMqa0OZIUvmJNLtAuzfVRmsGOj179eN2PbTA79qg\\\\\\\/NvDWLgaZMIt5Y\\\\\\\/8g6bt8D5wBbTMDoYsQR5myLTY1mQyI9hFdc\\\\\\\/FP1tg46OAk5RDFy0Tc62zlDOzt5QX0RZyP6OegiPJ2AP1fFnRpHrlCatjkAWIb2WtrBTMnTmoDkYtcsQdi1Hgy31B5jXD38s1NgOworsfUnX7EQ8Klkq4N0xTdN96XUBnMHhGyKgMDqloXPXEiC59KqXPY4mN9CJGXihUzG32vVCvLfZxQB887r9qMY9kYM5xb3W0qUe9VAdU7zeX7c8NTExvSTNrx79hJM2lClJt0YKU50qj4rTXfRx+EZiUWdZh6a\\\\\\\/tEZgboJvO6W+4tszyBDAmms9ByNUnSaWD\\\\\\\/bK0bIH749MxBs7E9U0tuFWfh8CY2Yvxu1S+Y4LWCKaUUy4JofIVc8jB8Q7egdOj9jeCHPvKUDzQxPwblWCcPu2yB1ha1XqWAL9XUOKiJTVSkXwx99ZckaxLnLD4lUAnBSIgj+ZMJFCvVEIQ32Od0ws+kN+Tjq5ehl16x\\\\\\\/Z3c9+GqbRM+MsN3yzC8r8hvvaVTtsCCexV1fFW91jJJ333Yd1qRKzyhG\\\\\\\/dvs8j22CRmbHcnwEhS1oQHG91J77cfCHGaSjsvbhksurs2\\\\\\\/0CVsSXTc0ehoblaeyxeXXNtA5C3zVZlWS8MQIPes7TSxxAqNTPzE99CTqUOW+6V9IWh2s9aX3dGLnLPDvJL6n1AKZSvzAHrnHQ7XtkHFz4FouXCkJy4OzmsoHiAzU9gQbpe0sxK+aC1vRQJur470bff7fpustfxy3SBYJ4ciEUTRtR6jKI6X6\\\\\\\/37YZLRXL+m4jE2iWQ00Cme66eA4U8bsdDi6YGZ0T3Qp\\\\\\\/toAG8UfVamMcSXxj+IVtsLhciHu1EHn46FOVYmcmR7KYe+dAb0d\\\\\\\/GgcSTAgwe0t8IdZN2TpN8gbnb2wyJO0eBjIU\\\\\\\/BFCOVEgDRIP0f4Xu91MphIK2l1A+a40Igyj+u8spFWcFaW3LLG3JKaLjqNIHgdbezB2W4m+TUUFUKZGID4qYLF8Vbc8eKkgi33R4KMlNR99VMlQfINsl\\\\\\\/shxWFeCjvepanDsIcf4V1AW43F7S3L8ecOqvNnxisjD3wiPrBndSEVGnZXMl0gg4oOUaUZ12KK3pghw7OTwOTTmMlfw8MuPNY5fFjdoAzQ\\\\\\\/MpnmU+jJx7qZZsp5qBDPq6mG5jTbxwbtjf3fuI42vkqjrVPuoyjxxxZNSdEmyBdq6eaCtQnNplgouCiDRKtV7oXT71hl\\\\\\\/vMUot8KnAvB1Ai6BGQUqctaZP8WvKVKG0m7iSk26C\\\\\\\/I\\\\\\\/oTuhIbuE\\\\\\\/LMmepQiiMqtd0hiFxdReGvei90igXEWGquu\\\\\\\/68FYuwJKDKeMSx1EQYwQis5VwlW3wkh8ZENj9lyJkM6kgCPH0VuKftKmf\\\\\\\/85ZonBQ2fmB5KkU2yKMackzrhdy0s8w0xCZAgfMwvAvNHeygpvbC1Q\\\\\\\/YqhpIza03C5\\\\\\\/EWcEoEc8YeBioWyjNQLQmuebSV6VgPOSe5c\\\\\\\/atXW6tEb75Le74NTF9e0kdw46MbUJSDTe4Z+fTNtElNOgHdL7NLSzLk0Orp7AibjWx6XPu7HX9L6R5KpiExeqPRnwpzXu8+eMRJTYeAzMfixcTQQs8x+mrSjwYaJun9okadoSn9KDkLnDHrqWfuQafIeuDpi+3URrjk\\\\\\\/G1Vjw7T8qeKBdBnK4zHvldo702zBd\\\\\\\/dwlDPomF37JPdpyFJsTd78dIryeEmcW6RVjigiCGJ6DKgT7h5PvZ4gKVnjdtOCYoRo23t\\\\\\\/BBz8xINNXpcMttEgXdfbGqRuRWpdDcmfigmSaPRPBTAhrVbha+3qsORuhl8WJOEZ1mxUlU0ywH2n9AJR71fU2ycvZ0MGijjNjEnh5BIzb6drdLBaDfHFSgMuo0Do8ypeFqc3rLtdSRafIx6H4\\\\\\\/AHsN8gdyzsG0SLDoQkBX0qR4Q2CTgnEUnCCGRsdzn5QmfSeJ+\\\\\\\/YTdzTGHKuDe1y1w04cOU7qrRGqmoB2CFr017ThFDwqmjIRcpSooO81LkH97b6paCH2ErC5fu9g9Kk8PVUIk5vhGfgmef0u4SwEbx0ogRpZ4hgFK3xvpplEZKQKD8kf5f3\\\\\\\/xBewPnjPA7J7aQfUXFxdJxi3Ffheebg1+Xb8fZTby+68OEfPNziJDWR1UM\\\\\\\/Zs8IoQhrQdEKeuina23uQCYQIY6BNpj6q5Qc9jONvwtKQKgOcbI0iFBSvA7KMlrMnh\\\\\\\/Dijg9L8efjC3V9v3PD\\\\\\\/AO1OXAfnVR1zLfH1ZD8xJbCKgO2ocJpBZ\\\\\\\/jLMdC17GUXUdnG0TWx9pCOGZJNKdJAiNaYJ27QySJbUDmi5Y6Nox68L7xVC9ulJXlD4RC4q155MEuw0xj+awa4+WE9h\\\\\\\/OIR7UxLHijqOBZpOI0w2QAZIjQTwYXa+Z2Qty9bfsBOapFp8hyN8+06Ts0z3JdaL+jVe34zWmgU\\\\\\\/dE12TMFo7gZJZMX8+7MY25rS0+PEN19Bu4X5QmxnbofwS4ivvVfzDyukYzdUQ8d80U9bQgBvcbYfaXYW\\\\\\\/Y8MEfC279zVYYeHG0vSphpVrnOqZ4iqAXsVzokTI1NGawwwt9kdZnc+kPQiPi0bPW0v3U\\\\\\\/O\\\\\\\/6uH04bG1109ZnM6WNLECnaB7s0lSAGwwcfIPEAv7Ed5WbQGjaMCTv+L5SAATIS814M3dPTj6r78dIJ7OacAfDHe\\\\\\\/ntAuV5LXdNfbBLRrojyPgynRlEpIkuiJSXuoVdzHMjNMcZqTXQArUtBcPsKMNS0dRO4KwxWKEbZXW6b\\\\\\\/GgfIenKuskvEfQHyFrMRRRLHfmOunBAbkor\\\\\\\/3deggjoHEXbib4QLkksPbQVhgWpjYvbHw20807YDUGsiwsSlrOB0kvuXSzH015C863kCntK1IRH1J\\\\\\\/ZDwDbDE5szOx6iAVebYP+cRpzUIvWce7S5LOZAl5iPsmBAMx8bjnarvu49KqnOV4SJbS4MEwX33Z4g2Sk5T6GzpzcJAYtPdCpv2QzZZXCzbvP29l\\\\\\\/21t84u2KIL2pFM00ePvJg+8K7cmmTnbY6pncKGPDFfE8iAen571IR42qvLoAENRMsPr7MNqpLV7bxoa8ZiPBByUtBgqa5IwNxHVGi0aghuaYX+H682Wk\\\\\\\/AMfQOINCxocnyACXXr\\\\\\\/\\\\\\\/CFk7l\\\\\\\/AEI5kw6EWW4XRQbh2X0z1oYDVRslBq\\\\\\\/avxNR\\\\\\\/c7\\\\\\\/riPoruWbWLKrFWMdeBHE6HySEFRF6fgBODLBSqER+E6+rgRa+FJe2mAJf595X6YnAL3hHKLQZx3U11o6OpAZgUcM3vbu6oIK1vQTB+mTd1l\\\\\\\/IYqfYrIGGMBnZwPKQx4WSsKauuiL7jRAaSAXFl2ZcHsH14wa7sKxW+eIjSWCxh2SEcAGu4IA0jkdl9e+s79CL6yawZoVg\\\\\\\/QOzNxJG8uJxlzVNAbH\\\\\\\/KbqXMCdOgC0UjCCuXhV9k95ruYif0tQteTpW2QsSG01ZAZ26fV\\\\\\\/thVNbG8DK0PwUAkrFU\\\\\\\/DQzRJ6WcikCnM3jK5Rb+NQaqggAI3OdIqnTc7TUekZ4bqGRlo0M1Kv7z6HYkphRTR77E0dpLSr04C98x5RqgoJIPPvWhQg9IqwPqAZjF51ZQl\\\\\\\/DYldyeFn8+zdU9SBg+yOCa1t4cfLiwAsBnc8FDSrvgh9q+q7I24Tdlk0pb2M435t1U7pSsY0pOaHdbxYqlE5OYcBp8v72BQGsfB7vafeNlHYzXY2NLUUuSCPzy54fR5EOj3Kbyr4VnjvlvtAG\\\\\\\/W7GWX0+tYPcwDVJIyKlmRi\\\\\\\/GwBBpCJNlF4sO18DtAipDWHxgfCOPywYNRCPn0cjeyinUi1xqnl41hIX2vbvO8NdmYfBSVMBn7h3xqQ2h5G6ZjXOfSYpQbg18xYsajtwhaf2I+BcsJiTHCwutxhTxYxPxqCt6fF0rnKPKdF4fgVqzz3FvLs\\\\\\\/hTod7pyebzluf+n6CIcQvZoruhf31GeUgAM\\\\\\\/pejJA6tvieOtICSNtB3Giu6bZMMPyuGWa+W59cf7IkjzEEThnsQNtKxWpcuJ94ba+lExE+ME4tOB4HhnxIM+3W\\\\\\\/2xtBpDevHSirYWf1bVrPABbFhb9tW8Plggvj1RgttKJ03IQaskCGuvbnTehjlB8u6N6YJw8tkc6ECJbEsmPZb+kqsRejF5CYhy4rvXJaLlxYxaWBmDvM4+Hk6qWHXRw0fMm7QzdcJES4oPr9Njk1WK4H\\\\\\\/1BH9SoSm5np03UlpEjV2\\\\\\\/wGlhkijt\\\\\\\/QLGRxCuxRtSml9xmTWSYPIm7cjdYhuGMclhirBGvgdGeCTMtnxJlEfn\\\\\\\/hqJRUIZv8R0eNe6EnRN9GC3YgtZrtXKo3SDW5qh+e74JOp1jqn\\\\\\\/E9XhYHOjnzXpAeisdtJnHVUhsUdoeBFLKgPrvEghzcvQoO4qfDgyuzvikDoqpfcXodDnngU0P0oU9fyS7CjiFeW9RkDjljcDuiRUMVSMfgZccvrzluH552sycqRhK6EqhZV7fXKexjS9QF1tZbOdDOo5j5nuhdEKBh9YkzoNMdfz5uytaW0Cur1MDe8iAfvSQ5tc7CzQmSmwmG6M8umN5qnak21ucG4r31EJyCfQDyyQmJ56mbbb1VOu0dsuTEpupyx0w4Op7SEOKpCOl4r9ZMKP5TyYrn79PgPQeaYirewmRwEAZcxrODi3xpep5gHjgcRrZXbYx8ojRdiW9HJD\\\\\\\/qzkwXYRh0cGYWHBEIHf4QhlqwartTFOfh1JzWYUbX+J4PBh2WAPGW1l8FzTNfhikoyjTtnKQxASETom2G\\\\\\\/wlH+Poj+swwTdfm6ddXD5bGCRsH07Eb6W5LUuUlzB3C8Iq1AOyOhkATk1\\\\\\\/KUlXIb7Wm40S+jQnf\\\\\\\/nyhgnSsQaGg6HT2OatKkkg+E07SgFnYo0GHL2KyFR0MP60vrAi1A0s+pJkWcIbswFjahX2tIK67dbuHO6OAflKcW8ABStSwzzLjQRzbBc1dpI8D3Lecw3FdDh51UgBPKL0YQ755xfQdb1dX0ixfkApHb6Yd\\\\\\\/9ek6K5Ra3qou4p8xw+oXPRBfl1F6EuIzexO9P9Ngy7fBDEHXYYd1XR+EJQw5jqjBfGs4UjzYV4jbzAmoHtWvrmLJUv\\\\\\\/V7P6PRYMbEaiCwpzUBg8TAvsZo66O1T5TzI3MCLs5441DZNg2GCaCUcRZVB3pZnm\\\\\\\/uPuHu6dhbhWkG6Pu+qEnAV0wF0EAaacjkdWnlIiA3CDuEjvDn3QoSB\\\\\\\/MWsoXC1l2fVIXcKJ5sHEvlxPxjwbypbFWFgKSyDbvD4aZzkL7SbgebTJgjPMxy89bg\\\\\\\/JTxgo17Lx3LvsfkbmBzClENJOyj\\\\\\\/ZkuMk8EG99T4qrWePFiGU0GaUmKDe8CijoRaflILQULm3HjGM+6U29gnJ3KLe\\\\\\\/C4GIOrcI7NYJDdsa9\\\\\\\/8dr84iExz7KaNpebzvbWbW8XtD1tqY9eL5DGWTelA9VvvyJE6AxiUyL0LwDeQ77wkrNE1MtTQwN6Hn\\\\\\\/Ni1SkvOR71d7N8YPH97iyHI3pfN6L9L9o\\\\\\\/0COXxy+tsal2\\\\\\\/UQt\\\\\\\/v4yxh0Y2tTGSMcBhFAAF4\\\\\\\/iQ4mGtGK4kcLRAqMVhfsBELIRQkrxZwkZlokayDyrcNEfFigRJeXAUFKKMbrCwFsex9D8lARCBk38RSdk1T5+fhSxAYv8qRW1J0Dkz4SKlNv+lsHd\\\\\\\/2kKInGZGNuSJIjx7aocVcNEkS6W15XnFWjBD8uHmEiuGC21slsvtL1u5hyR9ziBrIEFliOvta3Rat3J\\\\\\\/mP8v9J3h\\\\\\\/xXon3yhWSFos+fjwcKwulcv0X\\\\\\\/+KKpYpnYR+DX6ZcHZzmrEtq9E96+7Bq+XmuC7LI1F1WnHIMZQ8HIcri8nueo54bYlhxH4PrleQ3wAzmZEN4veLhwne\\\\\\\/0yNT+V0TBU7oiAejnJdxS\\\\\\\/ot+pZem2elpL7KE9nJCcspiyWUvkUOZRiRfd4TIbZGCSIwVGFE7VlWDw1ciFm6lQun3GD9TSBUYjjPjnsUGP4XVNFBxObGs0y\\\\\\\/t0R5xWQtWrmGcV9CSDqY1RZmIhR3fxVUZMsjwwvmptUtoj3kAbDfuZSEDKUj9B1Cj\\\\\\\/m581RT\\\\\\\/mEZu3Ynyo1vHgvDNGHVGXnG6mh44HwkZkfAW9OpXK92numWDL0KYt\\\\\\\/3i7oi9mjUYHhEkfQbAZ6oaYmmmeYvxQtj5ue2sli5rAUAKmclizO7ubVc9BcWJ1ZRci5AdOJBK1luHdLbfmGP7Wm+JLs9DoN0u2JtMh7Mql+iz9zNjKaDKI4iw+l+mXI5xJgUPmdk+gEErGuWuzxKAnG7SGseEz+yu6efsLXUOYxcFcScT5oe1dsFlgG8HVoO0QCdMmhLLfkmOycMB0ilt+wQ78bw83EJW++yx1effNVC\\\\\\\/J+ColGpiAUnvKq8f+rIE2ytGLPYYRcf3NFxymsZrZsdj+treLPxgDBKQCe+pEGf7WHlC56ietSqp8QwwzA1LvXHMjtOvcaB7VqN4tlJbfQZ3jslQ0HX79PmWu92MuqvYB25JpH7s8Bqpadp7Yeaq18VUDCmPVtHil1xUveCNpp79gPnoDNo3AybzGXpF92lesfa+TzGPg965ZHq4e\\\\\\\/wWzInx5hnS6Nf+EuDyYP2oxIYrkbiw6CqwUAEeZgHktgdpMt7V2ZpsTfna5lmJGDLfKkZOHE9hTmbRjvm\\\\\\\/YXF3RnURMKXXqMp9kutYojpnZ+LnfGnCFv5oy6GJ7GPFOZOZK7Q9kkkALFkIBcNmhnUUC3UJFkU04B8gvt9XlcvHDlzTQ1mcPhU8cJcwHYkP3EnhgffOIASi1Ab6oS1y+1Wh69pWUymENx0m0DLGB0GvM9xmuK\\\\\\\/t9SoPQBNLPyn5oNgZeVhEUfeLJHZewid7DA3FRga051Zrwa+UO+7LVRMe3vyuGJm2loQ0WkHOzoPfWK15iVilP3PNp9f1gvUXLKYjmBDSXv3GNtEP\\\\\\\/y5+PdKL6MT8G00iWpWlSFzZOAVk7BLb61DTuWm6SAlJt4+0VCICRQEurFEtVLIliYb98++yBdsOzuzi8S2fPYMNIJZTwQUKUVXEfAfvpttXVKO1oWAaOM+NWEJ7phXKpYKvy3Jt1fKPN8XxV\\\\\\\/kffNB99XZj7YxCf+CxQ4wp2JFxdRXcO9oGG9AA6BfRVb1ydyN2OGjqXlYHWnzEqdeASciqti9UqaU75j5LpwVeZ79gcjX7AP5ekjZez0rqUNU8A6MuITC9gPYkcAu6AwJCdYrOa4q0Deb6Irt47iFW2HUgxqZninxDNlH6dawS18wJy7Pq8ON3lMkO8zlReW19pNq\\\\\\\/gd9CjC1SBL0W53A1nGV\\\\\\\/0useRwVjpAGxuI+fiJMsjx9wNnasIE1KxqbVYWyo4saLCh8wvzD6TdkZ6V8seupy6Z3nk15ERFkHVMIeNE8bVoCOt95LX33vVMAM2uOLO40MmMC3lGEsAgo9jsUc92n\\\\\\\/GQc2fOqQKFw8GFrDtFjF9mdwNAg7B9ZBVxAh4d4+FhgA0FA0H0FDCZ0hhYKBJKySXtZltsJTnfJZxW4Z42qZlbPraOCrGU\\\\\\\/s+d6gdza1s6jdYFO7AyppPVybypCSNeAn+oaQywZOOnTnX1z3GoQaFrZzSl2TnVYKLT1jZ70L4wS6D4pvrdm3AaKfBVtXn9omj7wd7LTUvpbFGAlMX24gvZYMNGReAgCqfJVa8jKPwffepwTl7Iho9LA2F9njOllqf27hWvbtovIwfE8fwQOUolJADxW6EI66pL\\\\\\\/zRqopIhVD5TycA1IfGa85SGWXx0yO\\\\\\\/LmOHdh1QJgdJJ1YRdmZ9KQHylb9Q4bWFJ4R4VKut6ZdzrJAWWdJBesntDQmecQSHU2oIQBOEq7MmC6P2Sr+Efsgy6U+LOX9m1DBudWFv8f5ZVBok2k2xhII30s48HaPGlQ\\\\\\\/RYop1hxqcT2GN05hpmqOcPZ4EDK9BDUHHdEe208WbTsV2CApwn0nkBsYqCMQRb6ZPmSiSXZ3ZkCY\\\\\\\/muiH3rSOCZQ0YnIGovTGeV78weyiDHT309ItXezfYAdv5Y4TCYWwcrG26+s0TRrwewTnqzD8DBgKQtF+yFZL9TNBGdb+thOc\\\\\\\/bHWYPLDKiT154n6i6RXLc4ur0YDguSp803Ipug2RUwvcgyCVwXU5Piz6K4yLw3Rq9oOgWw56\\\\\\\/F4Up2PLphhA+hdcu2Si3VBFKyD69ZRply8sNKt1k4PPHPyYe987wTlszWCfNE\\\\\\\/vAhkY6NDuHpz5SBt8RSgs9t73UAurchHc2yO8VoSHOUxa+VWOM\\\\\\\/51mLD6Hkelok\\\\\\\/EMw5RcnCYxzEtXosZzauiikmme180iu4xlrq82X1yiV\\\\\\\/OMSI78tRoGMFrsUzywMHh9AJIhAdUM9NryUiZCrQifSHJw4zTYZgP9W3ggcICJiKEgeZW\\\\\\\/p2RuET3JUxETJdli7XiYd0zj\\\\\\\/VjVqJDPbAfCXmh84+RHXVQuR+vpcMOCiaH+5igxG8Xg5ChmtDEQoeT+lcB4Gh7zpn8Mh8rZkQqNZIA+ICp4Dy1v9kTH+B8JiyMlThZe0uF9oEz9ulpFYymRjykHRaGZ1H\\\\\\\/cd\\\\\\\/irT1K1KoqiOvGU3ucY8NFmHGOU8ZXaezsFIPi7oOI4DhyzGCFfRH7wgj\\\\\\\/R3oVdeMEBu95gsTCFGgMWvSmhicBY+6oedkmnaU91ao5HsjnGadxgqVtRI47CP0\\\\\\\/8VI6q1Rxj3lshDiH5UoyrGi68i8GTABwJynqpVZqpOEaNctOZ2M8ESG6cMjnnw74DVBMt3K63njkzE7CtcjKFMi02uCE1F8CcPqVFdCtCYf+KSiS5uJLzy8E7SAAY+T6Vu6FrjipNJIHGK8oLoLroF\\\\\\\/wz9TQ5yvmMJhYUpILqYPWHtyLSKN7PkQtEnU1b4MF3UPAo3CS7ILpWsXM5D5fFUzBK+OuvErv4VtC8lm1cFfXe+QPstpTIBxpP4HLc5zj0ChaAZbmaOUjmn9tSgzM+5mvntqVYQQg5CjMN9mNfhAOtRrMv5KXWhRQmtlKQPcdTtQa9QHu6GrZi9AQYmyWIxHI1CNqdXHrD3E\\\\\\\/+u5w7FsR3Y5x2DuqC+Jqe7Yy2pexfDdZMwF0O8\\\\\\\/XnsWb3kI6Z8o6+VqnUf4akLUSl6ISoHE4XwujsFnl+olqyQhQCCdOGQcKVaa3PJg1E4UwnSQkfwiWyqR8IMwl61ECtlRq7MlCA6HIIAw4eO\\\\\\\/O86VdwtCvPMWe\\\\\\\/CJGPfBMDfQwjEJZNQ8O1ntJT\\\\\\\/7Cnnl61S5djLhTOqEEdcMVlf1MYtzD1nInXh8s9B\\\\\\\/ylmhYyVnnj4s4xRtMd3Xv2n1NYwzEUIslEgnDxeAsIwAyMguagPUE2hNWY6ngHcAmKJWy1LUfqASUiugjz4+kKWbrIF0Zzl3ET2QviYA5zwbfjmlZzl114llrcrOZKRZv37hWpwt3JGz3ih7uY0iDMXIa2O86J9AEFxWQgKA3Ti7LFfDRPfNZiqv6cic7b039HJpm8xUa77n+srlmgVID87LJZ1orTv+M9a7qGlAXVaKpn8RqODhYB7l9g1cGIIWWGOkPqbiWKtp5P\\\\\\\/8yGY3QaVXAYvdtRyMQDtqi4MjZY1YuSaaV6nhdvrd3+XKsa9NTfcge1VYjpnbiLEQieqFjws6OGzoxmcgltVrRuZH0voBwS4T3OF\\\\\\\/aBXoTNId4pqcCiid2wbY9bxpgxV84+Pw1Q2+c02EgdEreVH4qq8Wp304iFMQKZxOgJYT0BD8J85zKMpXNlkngfSK+hppA1cT3B4deFX3R+zSS8HINe1x9XYeqwwCda2XsveCyGOB2ToDDreMrNym7A0h5X4CqtWIlgl3xMZRbz9CIL9EFbf49OO9J6rsHFCOsFirH21sIPiBEIzNRXL78HraeQz5BuEmiNzQpGfqb7CQEsUr4jYdf9+cPgOuvCIO6gULAMnY9nHabJ9ApBnb72zsHeYwvm9Kub9IuEgZt7s7AFmZNf3\\\\\\\/RQh\\\\\\\/OewrkoxYlpN39s\\\\\\\/9OMdXotevf89Qa+23+V7+s4gsw9kPR7It8NHuPcl4\\\\\\\/5RZdztl0sV5zfNI9lPkSIXFR0JPioy\\\\\\\/qsZS68tQQ\\\\\\\/ioh1GSw1iy4rXNw6SAPLo2VoGdAI+u7mSrBQ9rQbvVTHHAG5GxHymZKOCC\\\\\\\/DEOgiiH+Rio9HBE8Ens5x++e3lSegstWqtQAH1OK+QBmwNCOapUFhsf5xqBksgwQ+4vxud2MTZBXOamnzxcYDOZIniAk\\\\\\\/YF\\\\\\\/p8e4ouHMYK+C81iTGEgI3rbCuTcMWZJHeKPYfkWnMCobhYD0bBD93BssmUzm+1dCCr5QLJ9\\\\\\\/LxkLkkC\\\\\\\/kO+mIbu0pR0qI+J8OF62\\\\\\\/pvhGZQMMUIY7HsZ3Bf1x7F\\\\\\\/wRiS8odcC+kALPIfQPIasDNCdQ4L5Yj9GaWK2ql6CShfwj4io1MV0+X2k8Oixak6+uXSzeLqRiDi+KRXsHhuFbE0rGk8txPJwp\\\\\\\/kLnwuNv+AoKPx\\\\\\\/Ndb1OY9uaD3lcOsjlKcIG1oYAMCyNSUg1zF67Z9yJPP8P+HpVwFqpn+W4QYfOLB979BmwGXqVQAZyr98OIrTAuJBe3EPUaFaQ5Bb0haifrl+aaSG7d3sU0MQvArGViNmdfBW+LteEcqCSXPecCMWzYbHzSmtl5K8VmCL2MtP2\\\\\\\/x08A+upQu\\\\\\\/dzXtfMJ4XqNejo+gwugVuasJYoiRIMprThsmsQLZeh+LV0TKDijqwU3qGu8iSwK2Z5g\\\\\\\/EAB721WHc\\\\\\\/CnERb0pC6cgV6g3fgT+joCR1PcNuwZwWPjGibEluZZqbk8YLu\\\\\\\/0CemaSM7MDBPe6qdXPzTWTtkHNzh4EcI\\\\\\\/WEwrCNPZDi7Cr+4\\\\\\\/+6O+KfdnDU06afJAXKZ2fCSbQGfoK3i6hWSTm9CXpbKzqdUUDLRX4PCnrR6tEozvraL3J\\\\\\\/WiMeCy6TKEwhTU+7PHHk7Cn9AyasZ2JbLi+O1bj\\\\\\\/kNHD390riG1zdJ4cwp5\\\\\\\/Nv6mkrSAChirL4mByUQHLhEAqc7icE2xBBaJwLieJM7du+HYyXuloSMja683DILiKvHQPPNmWunlw0bsPlFNmx9ZLi0AThNLpp+orwrdtldHGhn2N7LW0HwGb5LECwo2SKAuJsL1BwgkVpl8rH8Qtoy3UkAbTpxtnXomrqd8lqM0zEAJKwgcO1tC\\\\\\\/HueYbWbeU5TBS9h93enIVzXoIZYWE48uOaBDiFjoJeTjnImkPguo7VIQ9SG+EAwwpVmBKUOS7IcQU2AZ7vDJLE2SSn+2GyYdyQOhxiXVUMHBJiDNxTWSPQzx7ajZynyWObquLcDAxcgXA5zf3myP5GA7TgGrsu9ow\\\\\\\/bKcioqpf55kDdMkdugZeyIYHWcldXEEfqq1JrBJtDe+rIpnaNmPnM4lizA6zFcsqTItqZz9TATFF6Dv+5X0DQnujPCV4RKzK+r5AvLcLdSiJdul88RDRVkpTAZYl\\\\\\\/v1L2vd87hrXBq2+WTjH\\\\\\\/AneSjC3wHT\\\\\\\/Zxl8qTJQvv7kPhxjAWqxVpdO92a5KhP\\\\\\\/wruEuG8SYWHSqdFgcG2xu8V2FxTNfDbr+7YJYMcEnxJPurZdgeUnAb6qpHU4okchQa42gXxhJJkC7DBnLvGWaRzsqOZAGOAvODyVtvP3NOI582SkOQ7kBeY9Ew64boWnyuN7+trRIexGZ5QOzcMziM5tjJQG4WrZM+CefhRvmpP5WsP6ZWf\\\\\\\/5mGXMCdUmHVWghxKvYvWF2+9x6m8qYYIi1Cq6B\\\\\\\/4InAGcXewSIowtkI2TP\\\\\\\/\\\\\\\/KfvvotE6DcBNXj79yUmWAT68XcOaK82HDaQvbHjbBpdwB22QJYP\\\\\\\/+Wpp1agQxjjsUGmW3itiR1gcWTWjlkchC3yB9rRnlgK6BN6jJvVhjx2oGP1+1UtcLZSB95veGE0LGSBiAh6uyq7zJ0RYdZNJQi50doimAz2UhFA249KMbclOjZMaZlrjeg84MvWFL3a6QGvU3IU3srroFY534l1lOEnlUBadqkQrT70qDzrCC0Y7KExm79Z2mFaCxLEmh+XUTeLCcZEUikWn7gj5e8FBCjTIJgP1dgGRNIRzWP5Xq78Dfd6X7BJNEe7NrEZkaxLYj2UvRzITjdC+tjRc08\\\\\\\/VTmBG6fL\\\\\\\/SkR5uwmVtJls72iYHrxbXeCS7dfVJ6gxRKw753M7qUQi7z9jLUwrKP+MZEnR2GuyvR+22c4RcIX89Gryo85x17Kt6SiXBuwLUAMHRkZNUPwoKE0qnAAjbXKNXOMaPhgODU3UfIHEEb3wJxuDflV6MftlOx59Ek7FdIQJhZFjE9VOHMIbEOkuavgyv9x+xCsP\\\\\\\/YR1yFFjMbtmjM+JLHZW06ubmI\\\\\\\/gLRClQZiYS98tHWmf1\\\\\\\/zGdFQLnc\\\\\\\/5aTww9oPdzrOE5SKC+uFBPeYAJHIP+iWWg5W+r1OOffjofYbzuMrDxk\\\\\\\/m3oUmJBkFKFemzAVBk6z99pMf7o\\\\\\\/jwwXWRjOtyoJyuVAYYeII9sspuEPXdTjzq2DjLGbNEvvd4oBEiYcyJLZO+mdE0rvIuBmZMburjsFUrtFRj3gv94d4m1gatbLvelWmoj1JU2ZCUh8aZZQ1f8cPYMMjhudZlUp9R1BZSvMRFx\\\\\\\/MUTplo3yXAuSF7qsgiQMDDiQk6ODjTpWH22NtAd013ZQ73Z6g7Had90XxauUgNXgT459UE8e0YpN0rFjjQ9+3jH09O6I\\\\\\\/M0JaRkExxSnfsjjwSJkYX40tM9JQKuSQj9wI+9yrZdtl9dVEQkH96TWSJu+snYkACHq3PBtRV\\\\\\\/1PtRdtL8XH0Sytf8iM0pDCz67RuMPirEFGYMw4j0NKbTDBa5qDs1rkEwcsx73FqWRQFd5vyWT303ApaRK90bSFUTtM6ZgTLNEXuH860\\\\\\\/\\\\\\\/rCInP6McbFLId5YlKoBLYNKXb1z7IPRd1B17iIZH\\\\\\\/yf3UYp5ay+T7cBaR07fKMuQRvX8UJo4ooLUHKHMRkYrBpJV3nMVCjLKbJqNP0pWlodGazwH0gVGJ3xNrduhXA7vDzdjb\\\\\\\/BvmUbwJ6JzelNonqJ8iyvsucQ5fafsxYmsNi8CxWEsFeRtSr85IOo99UgubjSyg37UalKMq77tv1GvbLeuA6PFFJKGDJdfFxkm9\\\\\\\/PygL8BMBojiPn06SvJfMFlD5MC60CUbh9EOb6eV6Qa\\\\\\\/kimfRMWrfoWHVsWAZ2brYdKt5BK33Ey25OdLVINhNoKx\\\\\\\/K2CSHlzL7aGAxcrPu1X0gx79eFQ8YszOdMpl0ibznNEUUDNhzVGlstpjdVQJIs0Z+NZbx2VbHIzi3gfymQr7mkMfLt7F3VqbS3uM+26kwI54kHVxClFXPZy7LFe+G4\\\\\\\/LzJvFqgvluHckbmtDNGIKI6ctFaI\\\\\\\/678HEop9cX1lupkThIHk3I5PdenerlSRrMWZKeFjjjoJXMrH3ME8uF25+AxYxhPdcLv5ZPostG4axCNy3sMT6605jLHK5oka8Fz6WRkkIJMkvnZmcabEZKCgL\\\\\\\/v6AqEDnfrFhRZTKGfuYHZK73NH2C7\\\\\\\/WUufpVKJ23idahcj3Xt\\\\\\\/JgMyMalfLU7+MZy1fJ43gaJkORBfChkd7Qm3hz8asdvQhkztlZXFPff0Z6JimeNBwFQ41etmd7MTkSKIm6g6fK6cQKxHS2gNLUN4fFwNi+0cKGTLEerNlsW+2nspS6e+i4QE5QqLcLKcFr0TMNpZnZFE9QwBDKhNleC1BNf94m\\\\\\\/j\\\\\\\/mRHI8t4abkm8vYyw+6Qrk6uo7jb27PmbHHeetYZICb3i3ABaAQPL2\\\\\\\/xBcr+m8rgKyYGmqOATpfjixF7JRXB8WMjhg85WmX3lVKbEm1uULwTiFZJwiGMwBuc4L7Qz07P21515xBIqQdjcnQ1XuRAEywyujyFpI8FkFAUdyxVs55XIIKP5elBEbTEa3u0m66PaJfrt\\\\\\\/czJfE3Ufr\\\\\\\/be2XtZICWv2OroFGAJQotFkGZk9Gb3zSoxrf0v\\\\\\\/4iY4MyhCLnOvJmUAQLSWrdR2Y33hnpbII5U7q+V7VvXT9asTGkMqvt5Dw4xHe7p+pi4kfNJ0jWlvuczdjDgGYVjFlHWLig2XeWrwCv3Tw+ANdL3w825qGmILrbpR\\\\\\\/aXaCgkh+8JljLr35ALjLwFY9Kzua7sR96V2UOOLlM5YcuSf9o3x6E61ZnxhGM7bxC9Z5\\\\\\\/Jjm6p9tyXYJOgNKXIfeqXmlGeK9rgD4mCliVsp4GM51Nq6YA\\\\\\\/TM2Y29z7PtqIJGrDSumlaGIpuSpcypjCehJhRnP62eAmQQ1NT31kbbyvWVQOYyQuN818YeXuQGr2AOQTTDaC4AwYrCaZ9\\\\\\\/m5stH3AMxl6\\\\\\\/HmKyFQF24\\\\\\\/SaBymNwuVNQrFymRUUCGfkOIEt2\\\\\\\/Zz\\\\\\\/QxO48\\\\\\\/QgzEgBgKgbBZZT+T8jtI0G2GZyP0afCs6I3pYokaDod6ln0h5jOMllCg\\\\\\\/ZKHHsa22iItADg\\\\\\\/0PKAsG1jXelX5sD561zaEGnxNC46PizEhCkVPB3rw0DEFZeuEwaiAPDhMgVZz4PtY91KUj+OJXjRGber3fBUTdZHbtf4hEPsIjvXE91o5CuNA+zD7ol0awO+YTARIK5Hgd+iLHcGxSZrwF9mSTbS6Dlw0nM4syET3dMA6LpzdMjckzT5ZpburzTdSjgV9g5iafHg5UvNTf0rakC\\\\\\\/tKNjbyU2RgdILYsVMmeaoQRjbthS1VAloNwu22nLBuViLEJMxIBbbI0wcO6Nt2URRSnPmSz5tYSqYhFlzXCm2q8f917TUai5wtBpKiEOqE1XWLBGYeXQ2hHILaHKYxUDFfR5nhyxa2fmL\\\\\\\/wN+\\\\\\\/VVUd0Tnkwv+16ud1hAgGty1QJMyJETqigJORvXVpvmMa0QP96zDh4hCQgoIhw+IEZwDy40YSPA7sMgv9i\\\\\\\/0wG5QM\\\\\\\/MBF+NSmApaCKpJNcqok1FtaNsnp02XoqNUz4\\\\\\\/XJ3qW4mHAwqJJ9DMA2X7Rob5Q5a6DL9vi4uR\\\\\\\/QcLpj\\\\\\\/6x6tyqfSasd1jCJQWo4fQ1AQjfty4oHTDM+2nO\\\\\\\/\\\\\\\/lTPG13MoagClfclD4PR19eRt\\\\\\\/GOJExbSzKMhPPj2GBFPx3Z0s+62NCSTmz4fN3E1jRmHGKTLEnrdShtAh\\\\\\\/fneXgv9qvk1O5XBDTf0UtXlikyyTDRYA+Gu2bMdpFlYHK9iGqHokOHv90syfx\\\\\\\/zOHKljmM9EnMPWu1nHZvqa9O12n9znfiH8bpl3YdDGMUHpvoRT\\\\\\\/9IfAAHs0XGjpsjPvirmJbD\\\\\\\/e7OiMO+i7lZsDkpf\\\\\\\/ISHm+ZpDZgg5AvpB3VJBivSv\\\\\\\/twsFe30FWLwYVj+ZeGITp6+dSBI0rxXFCFzvut0g8RDVKc9YGNrE4+zRg\\\\\\\/ujuuWuPAW8OzCCwgoV6W7+\\\\\\\/hFad3\\\\\\\/osUmdWOVZ3fWVFIRDLheJfgUWXCgMIAJXxOM1ZyaAVt50jNKLHdlA65AdOZUfCYhe0a23ZrIb1A5JftiVavl4D1uJEsNOROEwtreeWAaLg6S7\\\\\\\/15pbk1RRlNqMNatn4i+M4oGP4vo2nPhW68ccpH8XGQpjmLZCyBu0P8OprG3sLqfO3rBonPikzKUj1ECsY\\\\\\\/xktghOn+J\\\\\\\/7Cw1hFw7+I+BOzYcs05zCBBsEr\\\\\\\/yuRGvxOWxcDCbLdFlTKfo85ETTEa70dy6Rf0bpFdYH3Sdx7LNimhMMBGjL0ujgce4nTIIPhdhMv06u0SYsc7zzn6ci8g3qgcCS4KKFurAP2GiaYd\\\\\\\/KNtv8o+ZvRe1m3894NMtvGWfZzKkxzhZMzRXN4GRVU8vmcuj7U+DyWPKFNEbVhCWbMgQ98zb2GLzNoy66VouMJudPEk+9aAeoZKbKufKCVvwVDyqQgfvCLsg19lz1DCMOZNTWZS27GlgkqnuF+cgqVvv2CWiGKxRXbW6cccBjPTVC3Tb0nrqXF+Uqebdc\\\\\\\/WOTU3Ra\\\\\\\/Ua73lGziH2JuSG2rmTux1IzbdBXaJTkYi+PWWxL2nz27JD9w+VuL8N3M+kuRsqNbZuPk4xH8zwxgK423pZR9M1GVNBFpO7Sq5Pu\\\\\\\/CFE02X6ZCV7dC7uOzT8rnwQGj3bqzpUk\\\\\\\/vau\\\\\\\/RIzk4Jii6uyyD4+tI40snsl76YICkZ2fids4BqT+9zCScrrDxAn8m58H\\\\\\\/dvSX2wFh3z8861n8ZLCLl7UI\\\\\\\/a6LvlUC0NoTPLle95QNtZvR7i304yhImTzSog228YtyjcIiLdhNFKlZfOiWmo7\\\\\\\/\\\\\\\/KlV8WGn+Ygkmd\\\\\\\/32Rr7zUKd8Tgl0BU114SqiBBRPI62krQROW3ns7LOBUMn8CXXPrdUBlhSM+pPD7ktGuLRK\\\\\\\/4dPywEty0OldpVj8tamEoTThl8JYay64JmyLnfzeWskSMn1Icdd9wsvg+CLwu0x1NQTkRi5JacLgmHqmbMxL9QKQpNc4BrLtnvTHP2lTyceheYaw0SC9ToLPtxPzNM2Fd+JfsIQAuju8ITl9Z5rLCMRfRnOI20VkAasFHe88yDjLXOx7gkDjeEP6puEYXSi5V+bIdxhXxeg3s2MHAHcNWsfUKHlNSIU5sUcqGL1Dxb1UMyfjop1Ze0vPCSqiAYin6F1iN+CPufLZRwj\\\\\\\/f+x5dM7ULG1SxZTlaidaj0H\\\\\\\/zuLtEJArXKYb\\\\\\\/fVo0TiTa4Sz2e3FqzjxQ5jqkB\\\\\\\/KS+5jSb4IQuymDrAIiMaLz5dr8g0kA5n02PTWvsiY9I\\\\\\\/cTSvPPFoSDgv0BwLLOfPGuZBJJD3uTVd0tBuFJ\\\\\\\/yUr45OC8KBVQKCUqq0O7hpvu0v9zmwJFuUDTMzV6GEqAi4FezKKM49+tSHS8gtBTOjg4Ym5KRKNFtYVsTYtTLABBNXBoW+QXG0j5axce3v9aw5OqupXPXcb57sJ0lbmAgirMMyh5CxyAjSqrjGj4munEo2fh6bIQ2Gukx6has\\\\\\\/IkEUiIqlMZpXdTusvo54vIm6Xpmvy8fHkyqwpnJuteAYccakkonwT0CTrrwteRUNUqBuclEKqvK3xNzD0fmB7yFKhSSIkHHY7eWPlN+7ZrU9DM86wDRHRF\\\\\\\/RRTsuikaJI+iPSmacyd1ht0eWYVVO76APfTW7wOZxiWk5rUfeOOu1nzBu2OYaL2DDHR7BgnYOvCfTZl+9f+xeoY7NzSbRyoyZ92M3L3aMi7E2iLWB0mZxyFJ60FiyszOO5C3jb57EkD\\\\\\\/lD73ejTitD6vMNT16DcGb73ieOt7vJLzGFRJQP9yGh+Z01M4fawM\\\\\\\/LOxuGuxMiKMz9PkPqclPgol47NxFkbR9OysQ\\\\\\\/EkeqIzBOW01lDX5MGzH7OxfyDEYcAhJiirAxlBGR4jJJE42Jq2UrmavQbsYHZE0j25YxqABthLdQrd0+5zGQVJz8aPilw+Z\\\\\\\/Mc9N9+NG6+Br2reBPuVMRBzmIYCZUNufT9F08KtRvPwgulA747eRaAzIZMrxVFnbS+NBcM5PnICiEvjZZa9SuKiOY8aHQRTQfHKj7Afjl7aXCzQwS9MbkYtYPhcGLKPHjJfKUC6yqHri9JyHIBhXePu+cJWurGQ+P5dinmLfZRON3zFZk3Xbv7mSTR64tb5MJ4B\\\\\\\/1e1EkiCaC0txVea\\\\\\\/vHwzVS3UCMnuCuAdFIgLRApMj9DGlrxYnRjmDlF+esJIFeDHLL5mGbl73yuW++\\\\\\\/uHTe1NB9g4DDXutH2K3JsDFo9nGSd9ykVBLxrAMcEZVhXZhRw4\\\\\\\/5TJfoR9TQc6UMbOcFOMthNd+kKffaAsM4JypUOL6MWarubskSJMwpYItxettBTjy5yuqpub7GCbT3v1Nq1iM0MRR1yjTyUeug1X8+5uA4+91Cr+\\\\\\\/ccfqXMyDSi4sY6TRFUQzkijg7Nc9FO3PcUcGrHvb0+BidDaCGvrgC7BQb4bA4Ele5A4WHeTE6RNLtcebBzpDBq3vnGUhcg1dbgDfemaNQ5EZM61RvQKACgh88gikgcvtxOHqK\\\\\\\/JwPuruwY6885dXH2\\\\\\\/Ofyc93XI9XNzPYW32pBZP\\\\\\\/m9Tn9upLSst6VcOiG\\\\\\\/Q8NxE17KfBhDvaCBZhs1noEj\\\\\\\/3SQWEiSHEXjD\\\\\\\/F2PdI+VZGKYp+4HIA\\\\\\\/9TZXbHraxbEo2CP16zCYsgnK0oaALEIqffNJ0fN4d2S6fPeIfZKEW17hl3LX1icAL6LbotTbKCQ2XUcHacDlMFEaFI7WsNIsujrceqBsN1tFI4pFY9+WQPnEM4\\\\\\\/vt8NaGadl6aoKxiKQ1Gww5fdZxa3FDJFosAuhRPZBmi9dLUCmzHzOLvnbRu8VyQQwInRtgqS+o\\\\\\\/TAyx2shZY3CNt9RBgxEHUS0ernzoLpodQVeiDzab8uzAPcYWnxJM5m3Iy2Ok8oMEvakL5YEjDjkEFPPYOOk+H+usrQ0m1+Dp3hCIzM4eXacrRWxqUNajUsf3vhveSESJ1MA1jx5ciUlkh85yeGtoLdedR19pHezxBs7A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022m5DVnICFHo8PGl1QGhYnjQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225001c4a480312388d75141d5b477cf49d6c21a8e4c3f2eb6b6c569a7722acba9\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022EJ8wTjjl4jaZWoaYOD+Nvbmerwm8mWvbeY8pLe8LpOSr54PYxcK7KUFlRDi5zZh4lJ\\\\\\\/y7aAoPzeKUf0rUCIvorxJPXc\\\\\\\/8yBlZgKnFwZuiEg+XfXExJjHMv7BI0F3zp065eX2CkzExAV2yC\\\\\\\/Btr+SlM1BAVUOas7wQZNreqEUhkqhLKbI+Ri05KV5BxVf6tv7JKF\\\\\\\/hYrA\\\\\\\/+2hEoYRwAJXC4t1724hfxqFYe44+acdmCN+tSAbL3ul8zBFXHtZ4bqeAlypeEnBCPKVEQKSGijlZxefq7w\\\\\\\/TntOAFb+JgkK0JQieFhCGK7kibjtA1pcvl0lIfF34QM\\\\\\\/yJV4xQ77dMjfjtiuzX8RVCwXWhvrLm07Nwa2cW9kZOXeTc33zA14ezGwiD2CZs0RhSC\\\\\\\/hy+bIpo9RQkvfUhONhy5RfnG1kmKMlAeMIVIaIy7gY4T1KzdS9Jrlon5EpvHKPDU36GB+MkJZ\\\\\\\/q7YWXM3VoRMfdZGLVoUZ2WyXIIE6UpBYzrCTQbM6Jw6fcsUeIE0XyUHPOtxvwF8G43HiNJTqfkq41uYZW7alQeGaYcN0doVmVH+TWMGya9qFW43AyeIDebHv6OeUpfURRpVDpFoLSmQQ58QFp8ywaPhb66s2bA2sMnix2jW+9DhSjKy+HRo9bPGc1SD7cjPD3EileJ2Vd+20PFAryWxjaCvLOzd63NLl676IUnflygUles1zXMw7F3xBMGiz2693TVMnf3dVjwQMZ0FtQMoOI9PQZ3MOJFw5YUs6vB2LbmH0CPdYgJsRH7ace4OHXpy4h274mM5suoxqq0XFSHTmxbzRohiEbJwR8x2CvbiZR6IX+DqGB81k8emXN45On8eXDdfombMCO5K6eKvvA5joMTd\\\\\\\/jDnT0NkPv5eRo8\\\\\\\/KUa6IdLQtNZmEa8LuTC0xDZ9+0a2c4ItB5ZBFeM3w3JDiPMXQszNOz3dY1c34EH4rR75p6jE+Bosefv4xJ5O3qSfjK8yvcgxQsOEQwJt20n0b1KqAS6s5G6qdArJTRjJrPm4kv6Gmmgg0NyD9tR1mNW3vTu4\\\\\\\/ww9kKLybalqO5nJhyGwE5+f7B9ctuWQqL\\\\\\\/rqC1rV+futlpByq9O+N4xQ44PZ1LUdnRT3AtvSAtjtVq3A86wunbFTlI0JpgjiXpLEO93pttqIksPqI4VsijbA\\\\\\\/dLMjF2izo6d8OkEjs7jb4PW501yJoW9cwNCdtBMfzKgO0dkK3NWhKYl6Ay41ACWFy24pkKGAmzBVfMjAAPZv4vuXBY9jWdB4fFQvGHraJGrDCUcdGYCmOpvMsdD0cb3BZBd2tjy2iOSWEMzPYsmO+Z4k9ouqOKbOATZrw9h\\\\\\\/5y3noxC85dJQfyXSuhi7ISq6tuZt+e3rVsivq9ECYWSHSM6Y+7\\\\\\\/+Ey7rDX0++N3YmYXac7bkAcG\\\\\\\/xZwfu\\\\\\\/W7ZOPuogelx8b7PSadwJlAuXPK865KjU8CfDOPgm43mGHJ\\\\\\\/\\\\\\\/NH36qF643R9FmoUWL059qnAW7sFcOhzJQnJKe7vEDq\\\\\\\/Qrw9EdXBnJwOu65aklEz7FG9sX7c4yH3QolqeFJn+q8X5eIgDVKzF7QVaEQjoGV1auL3ARVU6wziNN3IAxHaYOUl1z1GRDbOdhwi0yJBCmHeT4Ec13mldfog9Vkdlmg1R\\\\\\\/YN\\\\\\\/6WDDqWKPSPBwYIgDHxC4dfeXTTOvxkNunpgfaOairQYvA\\\\\\\/TFISLfik0tChxcEhEkve5dJ4KPzZ+hEpSIlSRUPxc5ZIAHOxhxLLDJBKznF5S1RrvN43ZIczc8Ht6E1yGAbIutgnhIu\\\\\\\/uKneDL7O+3OrdEB6j4yXCx6SuSCEXjVaEDis1LCKAp8cMs+VmqPdd1TsCTGxGjnsG2s31AH03465lPhQXW7fYSElf4A2U+N4ifCmuhE9Fuzw0LhcmPOkf05VolSlQYK1wAAiHbhrMgWw\\\\\\\/DTEP8d+nnBD3jD\\\\\\\/pBNcpodgdwUMOFgzlt9+HBk1pGfNWhpO9zhYbn\\\\\\\/ezZ3\\\\\\\/qQLTcOlCegofnsXBwav3t3g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022OzENIfNm1+ZX0vPLyDd8lw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fa97465272fc11b652b69dc810ff4f7045891ebabf3baabb77f268efcb7bfebc\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459309 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/8/905" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-LSbD1HbE-5zJak0ied6n1DzaeFOVJ76_hrsVQnK-Qeo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/8/905" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0J9rpAlUf8j7qWxyiv64gUPsTZX2ZEmycYqmUBWeTlM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/8/891" title="Incidence of Genome Structure, DNA Asymmetry, and Cell Physiology on T-DNA Integration in Chromosomes of the Phytopathogenic Fungus Leptosphaeria maculans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/8/913" title="Genetic Interactions Between Arabidopsis DET1 and UVH6 During Development and Abiotic Stress Response" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459309" id="node459309" data-pisa="ggg;2/8/905" data-pisa-master="ggg;2/8/905" data-apath="/ggg/2/8/905.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Resampling QTL Effects in the QTL Sign Test Leads to Incongruous Sensitivity to Variance in Effect Size</h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel P. Rice</span> and <span class="highwire-citation-author" data-delta="1">Jeffrey P. Townsend</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">905-911;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003228 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459309" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel P. Rice</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Ecology and Evolutionary Biology, Yale University, New Haven, CT 06520</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BP.%2BRice" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rice%20DP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BP.%2BRice" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jeffrey P. Townsend</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Ecology and Evolutionary Biology, Yale University, New Haven, CT 06520</span><span class="nlm-aff">Program in Computational Biology and Bioinformatics, Yale University, New Haven, CT 06520</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BP.%2BTownsend" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Townsend%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BP.%2BTownsend" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Jeffrey.Townsend@Yale.edu" class="" data-icon-position="" data-hide-link-title="0">Jeffrey.Townsend@Yale.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/8/905" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459309" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459309/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/8/905.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459309" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459309/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/8/905.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459309" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459309/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/8/905.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459309" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459309/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Allelic effects at quantitative trait loci (QTL) between lineages are potentially informative for indicating the action of natural selection. The QTL Sign Test uses the number of + and − alleles observed in a QTL study to infer a history of selection. This test has been constructed to condition on the phenotypic difference between the two lines in question. By applying the test to QTL data simulated under selection, we demonstrate that conditioning on the phenotypic difference results in a loss of power to reject the neutral hypothesis and marked sensitivity to variation in locus effect magnitude.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/phenotype" class="hw-term hw-article-keyword hw-article-keyword-phenotype" rel="nofollow">phenotype</a></li><li class="kwd"><a href="/keyword/selection" class="hw-term hw-article-keyword hw-article-keyword-selection" rel="nofollow">selection</a></li><li class="kwd"><a href="/keyword/statistical-tests" class="hw-term hw-article-keyword hw-article-keyword-statistical-tests" rel="nofollow">statistical tests</a></li><li class="kwd"><a href="/keyword/quantitative-trait-loci-qtl" class="hw-term hw-article-keyword hw-article-keyword-quantitative-trait-loci-qtl" rel="nofollow">quantitative trait loci (QTL)</a></li><li class="kwd"><a href="/keyword/neutrality" class="hw-term hw-article-keyword hw-article-keyword-neutrality" rel="nofollow">neutrality</a></li></ul><p id="p-4">Distinguishing traits subject to selection from traits evolving neutrally is a challenging and important biological problem (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Boake <em>et al.</em> 2002</a>). It has been difficult to link the microevolutionary forces studied in contemporary populations to inferences about historical phenotypic selection (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Grant and Grant 2002</a>), yet this task is central to evolutionary biology (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Rieseberg <em>et al.</em> 2002</a>). Accordingly, the genetic basis and evolution of quantitative variation are receiving renewed attention (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Lai <em>et al.</em> 2007</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barton and De Vladar 2009</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chang and Noor 2007</a>). Nonetheless, most studies have not exploited information about quantitative trait locus (QTL) effects to infer selective histories (but see <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Rieseberg <em>et al.</em> 2002</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Lexer <em>et al.</em> 2005</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albertson <em>et al.</em> 2003</a>).</p><p id="p-5">An innovative attempt (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Orr 1998</a>) to integrate QTL data and population genetic theory generated two tests for historical selection based on the proposition that selection generates a preponderance of QTL effects in the same direction (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">True <em>et al.</em> 1997</a>). One test, the QTL Sign Test with Equal Effect (QTLST-EE), rejects the hypothesis of neutrality if more + alleles are observed than would be expected by chance. This simple test can be applied in cases of low or zero QTL effect variance. However, it has been criticized for its high false-positive rate (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anderson and Slatkin 2003</a>). The other test, the QTL Sign Test (QTLST), attempts to compensate for ascertainment bias in the QTLST-EE by conditioning not just on a difference having been observed between the two lines but by the full observed difference having been observed. Orr argued that tests for selection are more likely to be performed on traits that exhibit a large phenotypic difference and that having observed a large phenotypic difference, we are more likely to detect an excess of + alleles. To this end, the QTLST conditions on the phenotypic difference by asking whether there is a preponderance of + alleles compared with randomly assigned QTL effects that result in a phenotypic difference at least as large as that observed. <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Anderson and Slatkin (2003)</a> showed that the QTLST indeed controls for false-positives caused by trait ascertainment but did not examine the true-positive rate based on levels of selection and the variance of QTL effects.</p><p id="p-6">Conditioning on the observed phenotypic difference has the consequence that the gross phenotypic difference itself has no effect on the inference of selection. In contrast, because selection acts directly on phenotype, phenotypic divergence is generally touted as evidence in favor of selection. Accordingly, in approaches testing for statistically significant evolution of phenotype by natural selection, gross phenotypic difference has historically been precisely the signal assessed (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Lande 1976</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">1977</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Lande and Arnold, 1983</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Lynch 1990</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Turelli <em>et al.</em> 1988</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Charlesworth 1984</a>). With the observed phenotypic difference conditioned away by the QTLST, the test would appear to be likely to exhibit very low power; yet it can still yield positive results on some data sets (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Lexer <em>et al.</em> 2005</a>). Here we first present an analytical example demonstrating the negligible power of the QTLST for data sets with low QTL effect variance. Second, we show that its power depends peculiarly on the variance of QTL effects. To demonstrate this, we simulate QTL data under a model of selection and use the simulated data to assess the test's sensitivity to selection. We characterize how the QTLST performs in comparison with a sign test by setting aside the information regarding the number of allelic effects in each direction and assessing whether the QTLST is more likely to detect selection under more selective conditions.</p><div class="section other1" id="sec-1"><h2 class="">Analysis</h2><p id="p-7">Conceptually, a test of neutrality based on QTL effects should reject neutrality whenever presented with a sufficient number of loci whose effect directions are aligned with the difference between lineages. In this section, we analytically evaluate the QTLST in the “equal effects” case when all QTL effects have the same sign and magnitude. We show that the hypothesis of neutral evolution remains as far from being rejected as possible, irrespective of the number of + alleles, a point that is not obvious in the original presentation of the test.</p><p id="p-8">Suppose that a trait is controlled by a number of loci (for example, 10), each of which may have + or − alleles, all of equal magnitude. Assume further that the trait is under such strong selection for enlargement that every QTL locus acquires the + substitution.</p><p id="p-9">From <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Orr (1998)</a>, the calculation of the QTLST <em>P</em>-value for rejection of neutrality is<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <em>i</em> is an index variable, <em>n</em> is the number of QTL loci observed, <em>n<sub>+obs</sub></em> is the number of + alleles observed, <em>R</em> is the actual observed phenotypic difference between populations, and 2∑<strong><em>G</em></strong><sub>1</sub> is the phenotypic difference obtained by resampling from the observed distribution of QTL effects between the populations. The vector <strong><em>G</em></strong><sub>1</sub> comprises the allelic effect values of the loci in the first of the two populations. In this model, the effects are additive. The two before the summation accounts for the fact that the high line allelic values are represented as <strong><em>G</em></strong><sub>1</sub> and the low line values as −<strong><em>G</em></strong><sub>1</sub>. With 10 loci,<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span></p><p id="p-10">Because in this example all 10 loci have a + allele in the “selected” population,<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-3.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span></p><p id="p-11">Because all loci have equal effect, there is only way to sample from the distribution and assign + or − alleles to each locus that results in a phenotypic difference between populations that is (a) in the correct direction and (b) as large as that observed. That way is for all 10 loci to have + alleles, so that 2∑<strong>G</strong><sub>1</sub> = <em>R</em>. Thus, the proportion of times that the observed number of + loci (<em>n</em><sub>+</sub>) is 10 when 2∑<strong>G</strong><sub>1</sub> ≥ <em>R</em> is 1.0. Therefore, <em>P</em> = 1.0 ≫ 0.05, and thus, the hypothesis of neutral evolution is as far from being rejected as is quantitatively possible. In fact, using this test, one fails to reject the null hypothesis of neutrality when effects are of equal size and in the same direction, irrespective of how many + alleles, are observed.</p><p id="p-12">Although this case of exactly equal effect sizes is not very likely, it serves two important purposes. First, it illustrates the loss of power inherent in the QTLST's conditioning on the phenotypic difference. In this extreme case, conditioning on this difference throws out the entire signal left by selection, leaving the test nothing to operate on. Second, because it associates zero power with zero variance in effect sizes, it implies that rejection of neutrality would become increasingly probable with increased variance of QTL effects. In contrast, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Miller <em>et al.</em> (2006)</a> demonstrated that the response to selection of a quantitative trait does not depend on this variance when mutation is negligible. The following section tests this sensitivity of the QTLST to the variance of QTL effects by simulation.</p></div><div class="section other2" id="sec-2"><h2 class="">Simulations</h2><p id="p-13">To determine generally whether the QTLST accurately and precisely detects selection, we mapped the probability of fixation to the level of selection on + or – alleles in the two lineages. We performed the mapping via a model of QTL evolution under selection that encompasses the original QTLST's putatively neutral model. For <em>n</em> loci, presuming no directionality to mutation,<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-4.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span></p><p id="p-14">A conceptually linked selective model should reduce to this case when <em>s</em>, the selection coefficient, is 0, such that the<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-5.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-6.gif"/></noscript></span></span> as <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-7.gif"/></noscript></span></span>.</p><p id="p-15">To characterize the probabilities of fixation given selection coefficients, we applied a Markov model of allelic state with transition probabilities characterized by the solution of selection-diffusion equations (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Kimura, 1962</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bedford and Hartl 2009</a>). Based on the Markov model, equilibrium solutions can be straightforwardly derived for the proportion of + alleles (see <a id="xref-app-1-1" class="xref-app" href="#app-1"><em>Appendix</em></a>). Substituting Equation A4 from the <a id="xref-app-1-2" class="xref-app" href="#app-1"><em>Appendix</em></a> for <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-8.gif"/></noscript></span></span> in <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">Equation 5</a> yields<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-9.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <em>N</em> is the population size.</p><p id="p-16">To determine the relationships between the power of the QTLST, the false-positive rate of the QTLST, and the variance of QTL effects, we simulated QTL data (<em>n</em> = 10) by binomial sampling (<a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">Equation 6</a>). We assigned a suite of selection coefficients that, in the context of our model of selection with N = 10<sup>6</sup>, produce fixation probabilities spanning from 0.5 (neutrality) to just below 1 (nearly assured fixation; <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Our suite of selection coefficients maps to the full range of probabilities of fixation because selection coefficients larger than <em>s</em> = 2 × 10<sup>−6</sup> under our model would produce asymptotically smaller increases in the probability of fixation and thus would generate results essentially equivalent to <em>s</em> = 2 × 10<sup>−6</sup>. We drew allelic effect sizes from a flexible empirically and theoretically supported gamma distribution (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Orr 1998</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">1999</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">2003</a>). The shape and scale parameters of the gamma distribution were set equal to each other to explore variances ranging from 0.06 to 3.8 while maintaining the same mean. We then applied both the QTLST and the QTLST-EE to 10,000 sets of simulated QTL for each value of <em>s</em> and variance of allelic effects, preserving the original C code for the QTLST from <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Orr (1998)</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461690/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461690/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Probabilities of being in the selected (+) allelic state given a range of selection coefficients and a population size (<em>N</em>) of 10<sup>6</sup> (Equation A4)</span><div class="sb-div caption-clear"></div></div></div><p id="p-18">We first simulated effect sizes and directions assuming that selection coefficient was independent of QTL effect magnitude. However, it may be more realistic to assume that the selective value is proportional to the phenotypic effect (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Lande 1976</a>). Introducing a correlation between QTL effect size and allelic state could affect the power of the test. To assess this effect, we also applied both tests to simulated QTL data in which the selection coefficient used to calculate the probability that a given locus had a + allele was<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-10.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>where <em>s′</em> is the selection coefficient used to calculate the probability of fixation, <em>z</em> is the phenotypic effect drawn from the gamma distribution, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-11.gif"/></noscript></span></span> is the mean effect, and <em>s</em> is a selection coefficient from <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>.</p></div><div class="section results" id="sec-3"><h2 class="">Results</h2><p id="p-19">When applied to simulated QTL data, the QTLST-EE exhibited a false-positive rate of 0.021 across all variances, whereas the false-positive rate of the QTLST rose from 0.001 when the standard deviation of effect sizes was 0.24 to 0.035 when the standard deviation was 1.95 (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/905/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="False-positive rates vs. standard deviation of QTL effect sizes. Circles represent the QTLST-EE, and crosses represent the QTLST." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-581730016" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;False-positive rates &lt;em&gt;vs.&lt;/em&gt; standard deviation of QTL effect sizes. Circles represent the QTLST-EE, and crosses represent the QTLST.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/905/F1.medium.gif" width="440" height="427"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/905/F1.medium.gif" width="440" height="427"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/905/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/905/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461805" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-20" class="first-child">False-positive rates <em>vs.</em> standard deviation of QTL effect sizes. Circles represent the QTLST-EE, and crosses represent the QTLST.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">In the simulations where the strength of selection was independent of the effect size of the mutation, the QTLST detected selection more often when the variance of effect sizes was high than when the variance was low for all levels of selection except <em>s</em> = 2 × 10<sup>−6</sup>, when there was little discriminating power because it nearly always detected selection (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). In contrast, the QTLST-EE showed no sensitivity to the variance of QTL effects (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/905/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportion of samples for which tests rejected neutrality vs. the standard deviation of QTL effect sizes when selection is independent of effect size. (A) Proportion of rejections of neutrality by the QTLST. (B) Proportion of rejections of neutrality by the QTLST-EE." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-581730016" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Proportion of samples for which tests rejected neutrality &lt;em&gt;vs.&lt;/em&gt; the standard deviation of QTL effect sizes when selection is independent of effect size. (A) Proportion of rejections of neutrality by the QTLST. (B) Proportion of rejections of neutrality by the QTLST-EE.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/905/F2.medium.gif" width="440" height="186"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/905/F2.medium.gif" width="440" height="186"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/905/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/905/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461718" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-22" class="first-child">Proportion of samples for which tests rejected neutrality <em>vs.</em> the standard deviation of QTL effect sizes when selection is independent of effect size. (A) Proportion of rejections of neutrality by the QTLST. (B) Proportion of rejections of neutrality by the QTLST-EE.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">In the simulations where the strength of selection was proportional to the effect size, the QTLST detected selection less often in lower variance QTL samples for <em>s</em> &lt; 10<sup>−6</sup> and less often for intermediate variance samples for <em>s</em> ≥ 10<sup>−6</sup> (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The QTLST-EE showed little dependence on variance of effect sizes for <em>s</em> &lt; 5 × 10<sup>−7</sup> but detected selection less often in higher variance samples for <em>s</em> ≥ 5 × 10<sup>−7</sup> (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/905/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportion of samples for which tests rejected neutrality vs. the standard deviation of QTL effect sizes when selection is proportional to effect size. (A) Proportion of rejections of neutrality by the QTLST. (B) Proportion of rejections of neutrality by the QTLST-EE." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-581730016" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Proportion of samples for which tests rejected neutrality &lt;em&gt;vs.&lt;/em&gt; the standard deviation of QTL effect sizes when selection is proportional to effect size. (A) Proportion of rejections of neutrality by the QTLST. (B) Proportion of rejections of neutrality by the QTLST-EE.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/905/F3.medium.gif" width="440" height="184"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/905/F3.medium.gif" width="440" height="184"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/905/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/905/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461817" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-24" class="first-child">Proportion of samples for which tests rejected neutrality <em>vs.</em> the standard deviation of QTL effect sizes when selection is proportional to effect size. (A) Proportion of rejections of neutrality by the QTLST. (B) Proportion of rejections of neutrality by the QTLST-EE.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">To better characterize these results, we plotted the results of the QTLST segregated by the number of + QTL detected. In the simulations where selection was independent of effect size, for a given number of + alleles, frequency of rejection increased with increasing variance of the distribution of effect sizes, but it did not vary with selection coefficient (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Accordingly, for each level of variance, the proportion of times the test rejected neutrality was approximately equal for all selection coefficients, including <em>s</em> = 0. This independence of the selection coefficient to rejection of neutrality manifested stochastically for 8 observed + QTL (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>), and for 9 observed + QTL (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). For all other numbers of observed + QTL, the proportion of times the test rejected neutrality was exactly equal for all selection coefficients, including <em>s</em> = 0: when fewer than 8 + QTL were observed, the test never rejected neutrality, and when 10 + QTL were observed the test always rejected neutrality, regardless of the selection coefficient and variance of QTL effects.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/905/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportion of samples for which QTLST rejected neutrality vs. the standard deviation of QTL effect sizes when selection is independent of effect size, broken down by number of + alleles detected. (A) Proportion of rejection when 8 out of 10 + alleles were detected. (B) Proportion of rejection when 9 out of 10 + alleles were detected. When fewer than 8 + alleles were found, the test never rejected neutrality. When 10 + alleles were found, the test always rejected neutrality. When s was set to a value of 2*10&#x2212;6, there were never fewer than 9 + alleles in our simulations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-581730016" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Proportion of samples for which QTLST rejected neutrality &lt;em&gt;vs.&lt;/em&gt; the standard deviation of QTL effect sizes when selection is independent of effect size, broken down by number of + alleles detected. (A) Proportion of rejection when 8 out of 10 + alleles were detected. (B) Proportion of rejection when 9 out of 10 + alleles were detected. When fewer than 8 + alleles were found, the test never rejected neutrality. When 10 + alleles were found, the test always rejected neutrality. When &lt;em&gt;s&lt;/em&gt; was set to a value of 2&lt;sup&gt;*&lt;/sup&gt;10&lt;sup&gt;&#x2212;6&lt;/sup&gt;, there were never fewer than 9 + alleles in our simulations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/905/F4.medium.gif" width="440" height="198"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/905/F4.medium.gif" width="440" height="198"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/905/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/905/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461785" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-26" class="first-child">Proportion of samples for which QTLST rejected neutrality <em>vs.</em> the standard deviation of QTL effect sizes when selection is independent of effect size, broken down by number of + alleles detected. (A) Proportion of rejection when 8 out of 10 + alleles were detected. (B) Proportion of rejection when 9 out of 10 + alleles were detected. When fewer than 8 + alleles were found, the test never rejected neutrality. When 10 + alleles were found, the test always rejected neutrality. When <em>s</em> was set to a value of 2<sup>*</sup>10<sup>−6</sup>, there were never fewer than 9 + alleles in our simulations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">When strength of selection was proportional to the effect size of mutations, for a fixed number of + alleles, frequency of rejection again increased with the variance of the distribution of effect sizes, but it was again largely independent of selection coefficient (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). This independence of selection coefficient manifested stochastically when 8 (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>) or 9 (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>) + alleles were observed. For both 8 and 9 + QTL with proportionality of the selection coefficient, <em>s</em> = 0 was more likely to lead to a conclusion that selection had been in operation than other selection coefficients, which were all otherwise equivalent. Simulations of selection coefficients smaller than 10<sup>−7</sup> showed increasing probability of rejecting neutrality with decreasing selection coefficients, reaching a plateau at the probability for <em>s</em> = 0. As in the case with no proportionality of selection, when fewer than 8 + alleles were observed, the test never rejected neutrality, and when 10 + alleles were observed, the test always rejected neutrality.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/905/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportion of samples for which QTLST rejected neutrality vs. the standard deviation of QTL effect sizes when selection is proportional to effect size. (A) Proportion of rejection when 8 out of 10 + alleles were detected. (B) Proportion of rejection when 9 out of 10 + alleles were detected. When fewer than 8 + alleles were found, the test never rejected neutrality. When 10 + alleles were found, the test always rejected neutrality." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-581730016" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Proportion of samples for which QTLST rejected neutrality &lt;em&gt;vs.&lt;/em&gt; the standard deviation of QTL effect sizes when selection is proportional to effect size. (A) Proportion of rejection when 8 out of 10 + alleles were detected. (B) Proportion of rejection when 9 out of 10 + alleles were detected. When fewer than 8 + alleles were found, the test never rejected neutrality. When 10 + alleles were found, the test always rejected neutrality.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/905/F5.medium.gif" width="440" height="196"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/905/F5.medium.gif" width="440" height="196"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/905/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/905/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461722" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-28" class="first-child">Proportion of samples for which QTLST rejected neutrality <em>vs.</em> the standard deviation of QTL effect sizes when selection is proportional to effect size. (A) Proportion of rejection when 8 out of 10 + alleles were detected. (B) Proportion of rejection when 9 out of 10 + alleles were detected. When fewer than 8 + alleles were found, the test never rejected neutrality. When 10 + alleles were found, the test always rejected neutrality.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section discussion" id="sec-4"><h2 class="">Discussion</h2><p id="p-29">We have shown that the QTLST is highly sensitive to the variance in QTL effects. Furthermore, our simulations demonstrate that for a given number of observed + QTL, the QTLST does not reject neutrality more often when the QTL it is applied to are generated under stronger selection (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figures 4</a> and <a id="xref-fig-5-4" class="xref-fig" href="#F5">5</a>). Instead, for a given number of observed + QTL, the probability of rejecting neutrality is independent of the selection coefficient used to generate the QTL (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>). In fact, once the number of + QTL is set, the power of the test to reject neutrality is determined by the variance in effect size among the QTL, a factor which we have shown has no bearing on whether the test should detect selection.</p><p id="p-30">When selection was proportional to effect size, the QTLST was more likely to reject neutrality when <em>s</em> = 0 than when <em>s</em> &gt; 0 (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>). This effect results from conditioning on the phenotypic difference. Under proportional selection, + alleles are more likely to fix at large-effect loci than at small effect loci. Therefore, we expect greater phenotypic differences in selected traits than in neutrally evolving traits with the same number of + alleles fixed. Under the test's null model, large numbers of + alleles are less likely given a small phenotypic difference than a large phenotypic difference. Thus, conditioning on the trait difference leads to smaller <em>P</em>-values and more frequent rejection of neutrality when <em>s</em> = 0.</p><p id="p-31">The QTLST is, appropriately, more likely to conclude that selection has occurred when there are more + alleles; thus, it does have some power to detect selection (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figures 2</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">3</a>). This power arises from the fact that resampling from a larger number of alleles with significant variance in effect is more likely to exceed the observed phenotypic difference. However, this power is causally unrelated to the fact that + alleles fix more frequently under positive selection than they do under neutrality. Accordingly, the QTLST yields plausible results when one examines the results comparing across the number of + QTL (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figures 2</a> and <a id="xref-fig-3-4" class="xref-fig" href="#F3">3</a>), but within a given number of + QTL, its results are strongly dependent on the variance of QTL effects (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figures 4</a> and <a id="xref-fig-5-6" class="xref-fig" href="#F5">5</a>). In fact, for QTL data sets with low variance of effect sizes, the power to detect selection asymptotes to zero. Accordingly, the false-positive rate of the QTLST also depends on the variance of effect sizes (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). For QTL samples with large variance, the QTLST has a higher false-positive rate than the QTLST-EE. For small-variance samples, the QTLST has a smaller false-positive rate than the QTLST-EE, but it also has a correspondingly low true-positive rate. Thus, the low false-positive rate arises at the expense of power. Note that the QTLST-EE's false-positive rate is less than 0.05 because the binomial distribution is discrete: with 10 QTL the test rejects neutrality when 9 + QTL are observed (<em>P</em> = 0.021) but not when 8 + QTL are observed (<em>P</em> = 0.11). Nine or more + QTL were observed 0.021 of the time in our neutral simulations, so that is the false-positive rate.</p><p id="p-32">The details of the simple model of selection that we employ are irrelevant to these conclusions. The purpose of the precise selective model in our simulations is only to establish a correspondence between strength of selection and the probability of the + allele fixing. For all plausible models of directional selection it will be true that (1) as selection increases, the probability of the QTL having the + allele goes to 1, and (2) for neutral traits, the probabilities of having the + and – alleles are equal. Our model of selection allowed us to define a range of selective strengths that generated probabilities of having the + allele ranging from 0.5 to slightly less than 1, encompassing the entire relevant range. Had we chosen to use a different model of selection, we would then have tested somewhat different selection coefficients to map to the same range of fixation probabilities, but the results of the simulations would have been identical.</p><p id="p-33">In conclusion, both the QTLST and the QTLST-EE are problematic when applied to certain types of data. <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Anderson and Slatkin (2003)</a> previously demonstrated that the QTLST-EE suffers from ascertainment bias when QTL data sets are selected for testing based on their large phenotypic differences. However, in attempting to correct for this bias, the resampling procedure of the QTLST introduces sensitivity to the variance of the QTL effects, a result which has no basis in the history of neutrality or selection. In fact, for the extreme case of zero variance, the test has no power at all. Therefore, researchers using these tests must carefully consider not only the possible ascertainment bias in their data but also the variance of QTL effects. A recent alternative approach to testing for selection with QTL data (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Rice and Townsend 2012</a>) avoids these issues by capitalizing on information about mutation effect distributions to construct more realistic neutral and selective models.Can't read 3B2 tag because stream don't exist.Tag: Eq_1</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-34">We thank Allen Orr and an anonymous reviewer for helpful comments on the manuscript. D.P.R. was partially funded by a Yale College Dean's Research Fellowship, a Yale–Howard Hughes Medical Institute Future Scientist Summer Fellowship, and National Science Foundation Graduate Research Fellowship DGE-1144152. This work was also facilitated by the Yale University Faculty of Arts and Sciences High-Performance Computing facility and staff.</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><div class="section" id="sec-5"><h3>Modeling Selection</h3><p id="p-35">To match the neutral model underlying the QTLST, we draw QTL allelic effect sizes from a distribution, and only two final states for the parental lineages are allowed: homozygous + and homozygous -. For the sake of intuitive clarity, we assume that the + allele is the one selected for, though of course in natural systems selection may favor phenotypes divergent in either direction. We assume that one of these two states is the ancestral state, that the other is a derived state, and that the mutation responsible is reversible.</p><p id="p-36">To characterize the outcome of a history of selection, we calculate the distribution of + and - alleles by drawing from the equilibrium probabilities of state according to a Markov model with transition probabilities characterized by the solution of selection-diffusion equations. Starting with an initial population fixed for the - allelic state and assuming that the diffusion equation results from <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Kimura (1962)</a> hold, we find that,<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-12.gif"/></noscript></span><span class="disp-formula-label">(A1)</span></span>and<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-13.gif"/></noscript></span><span class="disp-formula-label">(A2)</span></span></p><p id="p-37">To solve for the equilibrium probability of the presence of expansive alleles, one must calculate the rates of transitions between the allelic states.</p><p id="p-38">Let <em>μ</em> be the rate at which mutations that affect the trait arise. Then, the rate of beneficial transition from the - allele to the + allele is <em>Nμ</em> times the fixation probability of the + allele (<a id="xref-disp-formula-8-1" class="xref-disp-formula" href="#disp-formula-8">Equation A1</a>). Similarly, the rate of deleterious transition from the + allelic state to the - allelic state is <em>Nμ</em> times the fixation probability of the – allele (<a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">Equation A2</a>). Solution of the equilibrium state probabilities from these rates of transition yields that the<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-14.gif"/></noscript></span><span class="disp-formula-label">(A3)</span></span></p><p id="p-39">Further algebra simplifies <a id="xref-disp-formula-10-1" class="xref-disp-formula" href="#disp-formula-10">Equations A3</a> to<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8/905/embed/mml-math-15.gif"/></noscript></span><span class="disp-formula-label">(A4)</span></span></p><p id="p-40">Substituting <a id="xref-disp-formula-11-1" class="xref-disp-formula" href="#disp-formula-11">Equation A4</a> for <em>π</em>(<em>s</em>) in <a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">Equation 5</a> yields Equation 6 in the main text. Note that these equations are independent of the rate of mutation of the trait.</p><p id="p-41">Kimura's equations (<a id="xref-disp-formula-8-2" class="xref-disp-formula" href="#disp-formula-8">Equations A1</a> and <a id="xref-disp-formula-9-2" class="xref-disp-formula" href="#disp-formula-9">A2</a>) do not apply exactly in the case of multiple competing alleles. Nevertheless, a consistent relation between selection coefficient and probability of fixation does, and the result relevant to our analysis, the ratio of the fixation rates of the expansive and diminutive alleles, should not be adversely affected.</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-42"><em>Communicating editor: J. C. Fay</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-03-02"><span class="received-label">Received </span>March 2, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-06-11"><span class="accepted-label">Accepted </span>June 11, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Rice, Townsend</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.1" data-doi="10.1093/jhered/esg071"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albertson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Streelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kocher</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genetic basis of adaptive shape differences in the cichlid head</span>. <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DAlbertson%26rft.auinit1%253DR.%2BC.%26rft.volume%253D94%26rft.issue%253D4%26rft.spage%253D291%26rft.epage%253D301%26rft.atitle%253DGenetic%2BBasis%2Bof%2BAdaptive%2BShape%2BDifferences%2Bin%2Bthe%2BCichlid%2BHead%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesg071%26rft_id%253Dinfo%253Apmid%252F12920100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6Ijk0LzQvMjkxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84LzkwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Orr's quantitative trait loci sign test under conditions of trait ascertainment</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">445</span>–<span class="cit-lpage">446</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D165%26rft.spage%253D445%26rft_id%253Dinfo%253Apmid%252F14575044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=14575044&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F905.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185618200037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Vladar</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Statistical mechanics and the evolution of polygenic quantitative traits</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">997</span>–<span class="cit-lpage">1011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D108%26rft.spage%253D997%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.4" data-doi="10.1073/pnas.0812009106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bedford</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Optimization of gene expression by natural selection</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">1133</span>–<span class="cit-lpage">1138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0812009106%26rft_id%253Dinfo%253Apmid%252F19139403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA2LzQvMTEzMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOC85MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.5" data-doi="10.1086/342902"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boake</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breden</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meffert</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ritchie</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genetic tools for studying adaptation and the evolution of behavior</span>. <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">S143</span>–<span class="cit-lpage">S159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D160%26rft.spage%253DS143%26rft_id%253Dinfo%253Adoi%252F10.1086%252F342902%26rft_id%253Dinfo%253Apmid%252F18707473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/342902&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18707473&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F905.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179869000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.6" data-doi="10.1534/genetics.106.067314"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noor</span>  <span class="cit-name-given-names">M. A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The genetics of hybrid male sterility between the allopatric species pair Drosophila persimilis and D-pseudoobscura bogotana: dominant sterility alleles in collinear autosomal regions</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.067314%26rft_id%253Dinfo%253Apmid%252F17277364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc2LzEvMzQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84LzkwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Some quantitative methods for studying evolutionary patters in single characters</span>. <abbr class="cit-jnl-abbrev">Paleobiology</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">308</span>–<span class="cit-lpage">318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPaleobiology%26rft.stitle%253DPaleobiology%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253D308%26rft.epage%253D318%26rft.atitle%253DSome%2Bquantitative%2Bmethods%2Bfor%2Bstudying%2Bevolutionary%2Bpatterns%2Bin%2Bsingle%2Bcharacters%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6ImdzcGFsZW9iaW8iO3M6NToicmVzaWQiO3M6ODoiMTAvMy8zMDgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzgvOTA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.8" data-doi="10.1126/science.1070315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Unpredictable evolution in a 30-year study of Darwin's finches</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">296</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">711</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGrant%26rft.auinit1%253DP.%2BR.%26rft.volume%253D296%26rft.issue%253D5568%26rft.spage%253D707%26rft.epage%253D711%26rft.atitle%253DUnpredictable%2BEvolution%2Bin%2Ba%2B30-Year%2BStudy%2Bof%2BDarwin%2527s%2BFinches%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1070315%26rft_id%253Dinfo%253Apmid%252F11976447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTYvNTU2OC83MDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzgvOTA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1962</span> <span class="cit-article-title">On probability of fixation of mutant genes in a population</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D47%26rft.spage%253D713%26rft_id%253Dinfo%253Apmid%252F14456043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=14456043&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F905.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19621898B00007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.10" data-doi="10.1038/nmeth1084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leips</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wollenberg</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Speed-mapping quantitative trait loci using microarrays</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">839</span>–<span class="cit-lpage">841</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLai%26rft.auinit1%253DC.%2BQ.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253D839%26rft.epage%253D841%26rft.atitle%253DSpeed-mapping%2Bquantitative%2Btrait%2Bloci%2Busing%2Bmicroarrays.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth1084%26rft_id%253Dinfo%253Apmid%252F17873888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth1084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17873888&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F905.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249778200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.11" data-doi="10.2307/2407703"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lande</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Natural-selection and random genetic drift in phenotypic evolution</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">314</span>–<span class="cit-lpage">334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D30%26rft.spage%253D314%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2407703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2407703&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1976BZ57800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.12" data-doi="10.2307/2407764"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lande</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Statistical tests for natural-selection on quantitative characters</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">442</span>–<span class="cit-lpage">444</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D31%26rft.spage%253D442%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2407764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2407764&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.13" data-doi="10.2307/2408842"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lande</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">The measurment of selection on correlated characters</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1210</span>–<span class="cit-lpage">1226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D37%26rft.spage%253D1210%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408842&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1983RX46700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.14" data-doi="10.1534/genetics.104.031195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lexer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenthal</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raymond</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donovan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieseberg</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genetics of species differences in the wild annual sunflowers, Helianthus annuus and H-petiolaris</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">2225</span>–<span class="cit-lpage">2239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D169%26rft.spage%253D2225%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.031195%26rft_id%253Dinfo%253Apmid%252F15545657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.104.031195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15545657&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F905.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229263700037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.15" data-doi="10.1086/285128"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">The rate of morphological evolution in mammals from the standpopint of the neutral expectation</span>. <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">727</span>–<span class="cit-lpage">741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D136%26rft.spage%253D727%26rft_id%253Dinfo%253Adoi%252F10.1086%252F285128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/285128&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1990EX56600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.16" data-doi="10.1007/s00285-005-0368-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pugh</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">M. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A finite locus effect diffusion model for the evolution of a quantitative trait</span>. <abbr class="cit-jnl-abbrev">J. Math. Biol.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">761</span>–<span class="cit-lpage">787</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmathematical%2Bbiology%26rft.stitle%253DJ%2BMath%2BBiol%26rft.aulast%253DMiller%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D52%26rft.issue%253D6%26rft.spage%253D761%26rft.epage%253D787%26rft.atitle%253DA%2Bfinite%2Blocus%2Beffect%2Bdiffusion%2Bmodel%2Bfor%2Bthe%2Bevolution%2Bof%2Ba%2Bquantitative%2Btrait.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00285-005-0368-9%26rft_id%253Dinfo%253Apmid%252F16463184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00285-005-0368-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16463184&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F905.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orr</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Testing natural selection <em>vs.</em> genetic drift in phenotypic evolution using quantitative trait locus data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">2099</span>–<span class="cit-lpage">2104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D149%26rft.spage%253D2099%26rft_id%253Dinfo%253Apmid%252F9691061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9691061&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F905.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075232100038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.18" data-doi="10.1017/S0016672399004164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orr</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The evolutionary genetics of adaptation: a simulation study</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DOrr%26rft.auinit1%253DH.%2BA.%26rft.volume%253D74%26rft.issue%253D3%26rft.spage%253D207%26rft.epage%253D214%26rft.atitle%253DThe%2Bevolutionary%2Bgenetics%2Bof%2Badaptation%253A%2Ba%2Bsimulation%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672399004164%26rft_id%253Dinfo%253Apmid%252F10689798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672399004164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10689798&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F905.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085417000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orr</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The distribution of fitness effects among beneficial mutations</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">1519</span>–<span class="cit-lpage">1526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D163%26rft.spage%253D1519%26rft_id%253Dinfo%253Apmid%252F12702694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12702694&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F905.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182517900025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.20" data-doi="10.1534/genetics.111.137075"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Townsend</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A test for selection employing quantitative trait locus and mutation accumulation data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1533</span>–<span class="cit-lpage">1545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.137075%26rft_id%253Dinfo%253Apmid%252F22298701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE1MzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzgvOTA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.21" data-doi="10.1073/pnas.192360899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rieseberg</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Widmer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arntz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Directional selection is the primary cause of phenotypic diversification</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">12242</span>–<span class="cit-lpage">12245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.192360899%26rft_id%253Dinfo%253Apmid%252F12221290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTkvMTkvMTIyNDIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzgvOTA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.22" data-doi="10.2307/2411157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">True</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laurie</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Quantitative genetic analysis of divergence in male secondary sexual traits between Drosophila simulans and Drosophila mauritiana</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">816</span>–<span class="cit-lpage">832</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D51%26rft.spage%253D816%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2411157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2411157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.905.23" data-doi="10.2307/2408923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turelli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gillespie</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lande</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Rate tests for selection on quantitative characters during macroevolution and microevolution</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">1085</span>–<span class="cit-lpage">1089</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D42%26rft.spage%253D1085%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/8/905.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/8/891" title="Incidence of Genome Structure, DNA Asymmetry, and Cell Physiology on T-DNA Integration in Chromosomes of the Phytopathogenic Fungus Leptosphaeria maculans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/8/913" title="Genetic Interactions Between Arabidopsis DET1 and UVH6 During Development and Abiotic Stress Response" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/8"><span>Volume 2 Issue 8, August 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/8.cover-source.jpg?itok=etPFDPKt" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/8/905" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jG9vFHvh0wNuIOKhJNJVRbgxfLDHq6ZfzT1_d_XIvtI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/8/905" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/8/905" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/8/905" class="active" data-icon-position="" data-hide-link-title="0">Resampling QTL Effects in the QTL Sign Test Leads to Incongruous Sensitivity to Variance in Effect Size</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459309" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ZdNSfv6-xtmzqFz8wF00CptkZT9-Hb0eAta1Th7YoA8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/8/905.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/8/905" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-D5rpTlj_sa5UWf9RNCOQrptn4FMu3btOaczriqwK7yA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/8/905" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/8/905.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459309" id="node459309--2" data-pisa="ggg;2/8/905" data-pisa-master="ggg;2/8/905" data-apath="/ggg/2/8/905.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Resampling QTL Effects in the QTL Sign Test Leads to Incongruous Sensitivity to Variance in Effect Size</h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel P. Rice</span> and <span class="highwire-citation-author" data-delta="1">Jeffrey P. Townsend</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">905-911;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003228 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459309--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel P. Rice</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Ecology and Evolutionary Biology, Yale University, New Haven, CT 06520</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BP.%2BRice" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rice%20DP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BP.%2BRice" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jeffrey P. Townsend</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Ecology and Evolutionary Biology, Yale University, New Haven, CT 06520</span><span class="nlm-aff">Program in Computational Biology and Bioinformatics, Yale University, New Haven, CT 06520</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BP.%2BTownsend" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Townsend%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BP.%2BTownsend" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Jeffrey.Townsend@Yale.edu" class="" data-icon-position="" data-hide-link-title="0">Jeffrey.Townsend@Yale.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/8/905" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/8/905&amp;title=Resampling%20QTL%20Effects%20in%20the%20QTL%20Sign%20Test%20Leads%20to%20Incongruous%20Sensitivity%20to%20Variance%20in%20Effect%20Size" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/8/905&amp;title=Resampling%20QTL%20Effects%20in%20the%20QTL%20Sign%20Test%20Leads%20to%20Incongruous%20Sensitivity%20to%20Variance%20in%20Effect%20Size" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/8/905&amp;title=Resampling%20QTL%20Effects%20in%20the%20QTL%20Sign%20Test%20Leads%20to%20Incongruous%20Sensitivity%20to%20Variance%20in%20Effect%20Size" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/8/905&amp;text=Resampling%20QTL%20Effects%20in%20the%20QTL%20Sign%20Test%20Leads%20to%20Incongruous%20Sensitivity%20to%20Variance%20in%20Effect%20Size" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/8/905&amp;title=Resampling%20QTL%20Effects%20in%20the%20QTL%20Sign%20Test%20Leads%20to%20Incongruous%20Sensitivity%20to%20Variance%20in%20Effect%20Size" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/8/905&amp;t=Resampling%20QTL%20Effects%20in%20the%20QTL%20Sign%20Test%20Leads%20to%20Incongruous%20Sensitivity%20to%20Variance%20in%20Effect%20Size" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/8/905&amp;title=Resampling%20QTL%20Effects%20in%20the%20QTL%20Sign%20Test%20Leads%20to%20Incongruous%20Sensitivity%20to%20Variance%20in%20Effect%20Size" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/8/905&amp;title=Resampling%20QTL%20Effects%20in%20the%20QTL%20Sign%20Test%20Leads%20to%20Incongruous%20Sensitivity%20to%20Variance%20in%20Effect%20Size" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459309/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459309" id="node459309--2" data-pisa="ggg;2/8/905" data-pisa-master="ggg;2/8/905" data-apath="/ggg/2/8/905.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Resampling QTL Effects in the QTL Sign Test Leads to Incongruous Sensitivity to Variance in Effect Size</h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel P. Rice</span> and <span class="highwire-citation-author" data-delta="1">Jeffrey P. Townsend</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">905-911;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003228 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459309--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel P. Rice</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Ecology and Evolutionary Biology, Yale University, New Haven, CT 06520</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BP.%2BRice" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rice%20DP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BP.%2BRice" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jeffrey P. Townsend</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Ecology and Evolutionary Biology, Yale University, New Haven, CT 06520</span><span class="nlm-aff">Program in Computational Biology and Bioinformatics, Yale University, New Haven, CT 06520</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BP.%2BTownsend" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Townsend%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BP.%2BTownsend" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Jeffrey.Townsend@Yale.edu" class="" data-icon-position="" data-hide-link-title="0">Jeffrey.Townsend@Yale.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=905&amp;rft.epage=911&amp;rft.atitle=Resampling%20QTL%20Effects%20in%20the%20QTL%20Sign%20Test%20Leads%20to%20Incongruous%20Sensitivity%20to%20Variance%20in%20Effect%20Size&amp;rft.volume=2&amp;rft.issue=8&amp;rft.date=2012-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Rice%2C+Daniel+P.&amp;rft.au=Townsend%2C+Jeffrey+P."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459309/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459309/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459309/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459309/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459309/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459309/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459309/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459309/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459309/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459309/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459309/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459309/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494625" id="node494625" data-pisa="ggg;9/11/3891" data-pisa-master="ggg;g3.119.400723" data-apath="/ggg/9/11/3891.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3891" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Mouse Mutation That Dysregulates Neighboring <em>Galnt17</em> and <em>Auts2</em> Genes Is Associated with Phenotypes Related to the Human AUTS2 Syndrome</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494624" id="node494624" data-pisa="ggg;9/11/3781" data-pisa-master="ggg;g3.119.400647" data-apath="/ggg/9/11/3781.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3781" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Transgenic Analyses in Drosophila Reveal That mCORL1 Is Functionally Distinct from mCORL2 and dCORL</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494622" id="node494622" data-pisa="ggg;9/11/3843" data-pisa-master="ggg;g3.119.400363" data-apath="/ggg/9/11/3843.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3843" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Analysis of Cell Cycle-Regulating Genes in the Symbiotic Dinoflagellate <em>Breviolum minutum</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/8/905" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Analysis</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Simulations</a></li><li class="even"><a href="#sec-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="odd"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/8/905.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/8/905.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/2/8/905.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2019 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
  </body>
</html>
