<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/2/249" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/2/249.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/2/249.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/2/249.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2.cover.gif" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/2/2/249/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/2/249.atom" />
<meta name="HW.pisa" content="ggg;2/2/249" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Whole-Chromosome Analysis of Meiotic Recombination in Drosophila melanogaster" />
<meta name="DC.Identifier" content="10.1534/g3.111.001396" />
<meta name="DC.Date" content="2012-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Miller et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Although traditional genetic assays have characterized the pattern of crossing over across the genome in Drosophila melanogaster , these assays could not precisely define the location of crossovers. Even less is known about the frequency and distribution of noncrossover gene conversion events. To assess the specific number and positions of both meiotic gene conversion and crossover events, we sequenced the genomes of male progeny from females heterozygous for 93,538 X chromosomal single-nucleotide and InDel polymorphisms. From the analysis of the 30 F1 hemizygous X chromosomes, we detected 15 crossover and 5 noncrossover gene conversion events. Taking into account the nonuniform distribution of polymorphism along the chromosome arm, we estimate that most oocytes experience 1 crossover event and 1.6 gene conversion events per X chromosome pair per meiosis. An extrapolation to the entire genome would predict approximately 5 crossover events and 8.6 conversion events per meiosis. Mean gene conversion tract lengths were estimated to be 476 base pairs, yielding a per nucleotide conversion rate of 0.86 × 10−5 per meiosis. Both of these values are consistent with estimates of conversion frequency and tract length obtained from studies of [rosy][1] , the only gene for which gene conversion has been studied extensively in Drosophila . Motif-enrichment analysis revealed a GTGGAAA motif that was enriched near crossovers but not near gene conversions. The low-complexity and frequent occurrence of this motif may in part explain why, in contrast to mammalian systems, no meiotic crossover hotspots have been found in Drosophila .

 [1]: http://flybase.org/reports/FBgn0003308.html" />
<meta name="DC.Contributor" content="Danny E. Miller" />
<meta name="DC.Contributor" content="Satomi Takeo" />
<meta name="DC.Contributor" content="Kavyasree Nandanan" />
<meta name="DC.Contributor" content="Ariel Paulson" />
<meta name="DC.Contributor" content="Madelaine M. Gogol" />
<meta name="DC.Contributor" content="Aaron C. Noll" />
<meta name="DC.Contributor" content="Anoja G.  Perera" />
<meta name="DC.Contributor" content="Kendra N. Walton" />
<meta name="DC.Contributor" content="William D. Gilliland" />
<meta name="DC.Contributor" content="Hua Li" />
<meta name="DC.Contributor" content="Karen K.  Staehling" />
<meta name="DC.Contributor" content="Justin P. Blumenstiel" />
<meta name="DC.Contributor" content="R. Scott Hawley" />
<meta name="article:published_time" content="2012-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Whole-Chromosome Analysis of Meiotic Recombination in Drosophila melanogaster" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Although traditional genetic assays have characterized the pattern of crossing over across the genome in &lt;i&gt;Drosophila melanogaster&lt;/i&gt;, these assays could not precisely define the location of crossovers. Even less is known about the frequency and distribution of noncrossover gene conversion events. To assess the specific number and positions of both meiotic gene conversion and crossover events, we sequenced the genomes of male progeny from females heterozygous for 93,538 &lt;i&gt;X&lt;/i&gt; chromosomal single-nucleotide and InDel polymorphisms. From the analysis of the 30 F1 hemizygous X chromosomes, we detected 15 crossover and 5 noncrossover gene conversion events. Taking into account the nonuniform distribution of polymorphism along the chromosome arm, we estimate that most oocytes experience 1 crossover event and 1.6 gene conversion events per &lt;i&gt;X&lt;/i&gt; chromosome pair per meiosis. An extrapolation to the entire genome would predict approximately 5 crossover events and 8.6 conversion events per meiosis. Mean gene conversion tract lengths were estimated to be 476 base pairs, yielding a per nucleotide conversion rate of 0.86 × 10&lt;sup&gt;−5&lt;/sup&gt; per meiosis. Both of these values are consistent with estimates of conversion frequency and tract length obtained from studies of &lt;i&gt;rosy&lt;/i&gt;, the only gene for which gene conversion has been studied extensively in &lt;i&gt;Drosophila&lt;/i&gt;. Motif-enrichment analysis revealed a GTGGAAA motif that was enriched near crossovers but not near gene conversions. The low-complexity and frequent occurrence of this motif may in part explain why, in contrast to mammalian systems, no meiotic crossover hotspots have been found in &lt;i&gt;Drosophila&lt;/i&gt;.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/02/01" />
<meta name="citation_mjid" content="ggg;2/2/249" />
<meta name="citation_id" content="2/2/249" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/2/249" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/2/249.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/2/249.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/2/249.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.001396" />
<meta name="citation_pmid" content="22384403" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="2" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="249" />
<meta name="citation_lastpage" content="260" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Danny E. Miller" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, Kansas City, MO 64110" />
<meta name="citation_author" content="Satomi Takeo" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, Kansas City, MO 64110" />
<meta name="citation_author" content="Kavyasree Nandanan" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, Kansas City, MO 64110" />
<meta name="citation_author" content="Ariel Paulson" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, Kansas City, MO 64110" />
<meta name="citation_author" content="Madelaine M. Gogol" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, Kansas City, MO 64110" />
<meta name="citation_author" content="Aaron C. Noll" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, Kansas City, MO 64110" />
<meta name="citation_author" content="Anoja G.  Perera" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, Kansas City, MO 64110" />
<meta name="citation_author" content="Kendra N. Walton" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, Kansas City, MO 64110" />
<meta name="citation_author" content="William D. Gilliland" />
<meta name="citation_author_institution" content="Department of Biology, Depaul University, Chicago, IL 60604" />
<meta name="citation_author" content="Hua Li" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, Kansas City, MO 64110" />
<meta name="citation_author" content="Karen K.  Staehling" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, Kansas City, MO 64110" />
<meta name="citation_author" content="Justin P. Blumenstiel" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, KS 66045, and" />
<meta name="citation_author_email" content="jblumens@ku.edu" />
<meta name="citation_author" content="R. Scott Hawley" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, Kansas City, MO 64110" />
<meta name="citation_author_institution" content="Department of Physiology, Kansas University Medical Center, Kansas City, KS 66160" />
<meta name="citation_reference" content="citation_journal_title=Proceedings / . International Conference on Intelligent Systems for Molecular Biology ; ISMB. International Conference on Intelligent Systems for Molecular Biology;citation_journal_abbrev=Proc Int Conf Intell Syst Mol Biol;citation_author=T. L. Bailey;citation_author=C. Elkan;citation_title=Fitting a mixture model by expectation maximization to discover motifs in biopolymers.;citation_pages=28-36;citation_volume=2;citation_year=1994;citation_pmid=7584402" />
<meta name="citation_reference" content="citation_journal_title=Proceedings / . International Conference on Intelligent Systems for Molecular Biology ; ISMB. International Conference on Intelligent Systems for Molecular Biology;citation_journal_abbrev=Proc Int Conf Intell Syst Mol Biol;citation_author=T. L. Bailey;citation_author=C. Elkan;citation_title=The value of prior knowledge in discovering motifs with MEME.;citation_pages=21-29;citation_volume=3;citation_year=1995;citation_pmid=7584439" />
<meta name="citation_reference" content="citation_journal_title=Machine Learning Journal;citation_author=T. L. Bailey;citation_author=C. Elkan;citation_title=Unsupervised learning of multiple motifs in biopolymers using expectation maximization;citation_pages=51-83;citation_volume=21;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. L. Bailey;citation_author=N. Williams;citation_author=C. Misleh;citation_author=W. W. Li;citation_title=MEME: DISCOVERING AND ANALYZING DNA AND PROTEIN SEQUENCE MOTIFS;citation_pages=W369 W373;citation_volume=34;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_abbrev=Curr Genomics;citation_author=L. E. Berchowitz;citation_author=G. P. Copenhaver;citation_title=Genetic interference: don't stand so close to me.;citation_pages=91-102;citation_volume=11;citation_year=2010;citation_issue=2;citation_pmid=20885817;citation_doi=10.2174/138920210790886835" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. K. Bishop;citation_author=D. Zickler;citation_title=Early decision; meiotic crossover interference prior to stable strand exchange and synapsis.;citation_pages=9-15;citation_volume=117;citation_year=2004;citation_issue=1;citation_pmid=15066278;citation_doi=10.1016/S0092-8674(04)00297-1" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=H. L. Blanton;citation_author=S. J. Radford;citation_author=S. McMahan;citation_author=H. M. Kearney;citation_author=J. G. Ibrahim;citation_title=REC, Drosophila MCM8, drives formation of meiotic crossovers;citation_pages=343-354;citation_volume=1;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. P. Blumenstiel;citation_author=A. C. Noll;citation_author=J. A. Griffiths;citation_author=A. G. Perera;citation_author=K. N. Walton;citation_title=Identification of EMS-induced mutations in Drosophila melanogaster by whole-genome sequencing;citation_pages=25-32;citation_volume=182;citation_year=2009;citation_pmid=19307605;citation_doi=10.1534/genetics.109.101998" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. T. C. Carpenter;citation_title=Electron microscopy of meiosis in Drosophila melanogaster females: II: The recombination nodule--a recombination-associated structure at pachytene?;citation_pages=3186-3189;citation_volume=72;citation_year=1975;citation_issue=8;citation_pmid=810799;citation_doi=10.1073/pnas.72.8.3186" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. T. C. Carpenter;citation_title=Mismatch repair, gene conversion, and crossing-over in two recombination-defective mutants of Drosophila melanogaster.;citation_pages=5961-5965;citation_volume=79;citation_year=1982;citation_issue=19;citation_pmid=6821126;citation_doi=10.1073/pnas.79.19.5961" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. B. Carvalho;citation_author=A. G. Clark;citation_title=Intron size and natural selection.;citation_pages=344-344;citation_volume=401;citation_year=1999;citation_issue=6751;citation_pmid=10517631;citation_doi=10.1038/43827" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Chovnick;citation_author=G. H. Ballantyne;citation_author=D. G. Holm;citation_title=Studies on gene conversion and its relationship to linked exchange in Drosophila melanogaster;citation_pages=179-209;citation_volume=69;citation_year=1971;citation_pmid=5002749" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. M. Comeron;citation_author=M. Kreitman;citation_title=The correlation between intron length and recombination in drosophila. Dynamic equilibrium between mutational and selective forces;citation_pages=1175-1190;citation_volume=156;citation_year=2000;citation_pmid=11063693" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Curtis;citation_author=W. Bender;citation_title=GENETICS;citation_pages=739-746;citation_volume=127;citation_year=1991;citation_pmid=2029970" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Curtis;citation_author=S. H. Clark;citation_author=A. Chovnick;citation_author=W. Bender;citation_title=Molecular analysis of recombination events in Drosophila;citation_pages=653-661;citation_volume=122;citation_year=1989;citation_pmid=2503419" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. F. Dernburg;citation_author=K. McDonald;citation_author=G. Moulder;citation_author=R. Barstead;citation_author=M. Dresser;citation_title=Meiotic recombination in C. elegans initiates by a conserved mechanism and is dispensable for homologous chromosome synapsis.;citation_pages=387-398;citation_volume=94;citation_year=1998;citation_issue=3;citation_pmid=9708740;citation_doi=10.1016/S0092-8674(00)81481-6" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=P. Dhami;citation_author=P. Saffrey;citation_author=A. W. Bruce;citation_author=S. C. Dillon;citation_author=K. Chiang;citation_title=Complex exon-intron marking by histone modifications is not determined solely by nucleosome distribution.;citation_pages=e12339-e12339;citation_volume=5;citation_year=2010;citation_issue=8;citation_pmid=20808788;citation_doi=10.1371/journal.pone.0012339" />
<meta name="citation_reference" content="citation_title=The Genetics and Biology of Drosophila;citation_title=Gene conversion in Drosophila;citation_author=V. G. Finnerty;citation_pages=331-351;citation_year=1976" />
<meta name="citation_reference" content="citation_journal_abbrev=Gene;citation_author=A. S. Fiston-Lavier;citation_author=N. D. Singh;citation_author=M. Lipatov;citation_author=D. A. Petrov;citation_title=Drosophila melanogaster recombination rate calculator.;citation_pages=18-20;citation_volume=463;citation_year=2010;citation_issue=1-2;citation_pmid=20452408;citation_doi=10.1016/j.gene.2010.04.015" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=T. Goldfarb;citation_author=M. Lichten;citation_title=Frequent and efficient use of the sister chromatid for DNA double-strand break repair during budding yeast meiosis.;citation_pages=e1000520-e1000520;citation_volume=8;citation_year=2010;citation_issue=10;citation_pmid=20976044;citation_doi=10.1371/journal.pbio.1000520" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. E. Grant;citation_author=T. L. Bailey;citation_author=W. S. Noble;citation_title=FIMO: scanning for occurrences of a given motif;citation_volume=27;citation_year=2011;citation_pmid=21330290;citation_doi=10.1093/bioinformatics/btr064" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. M. Green;citation_author=K. C. Green;citation_title=Crossing-Over Between Alleles at the Lozenge Locus in Drosophila Melanogaster;citation_pages=586-591;citation_volume=35;citation_year=1949;citation_issue=10;citation_pmid=15391347;citation_doi=10.1073/pnas.35.10.586" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=M. Grelon;citation_author=D. Vezon;citation_author=G. Gendrot;citation_author=G. Pelletier;citation_title=AtSPO11-1 is necessary for efficient meiotic recombination in plants.;citation_pages=589-600;citation_volume=20;citation_year=2001;citation_issue=3;citation_pmid=11157765;citation_doi=10.1093/emboj/20.3.589" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Gupta;citation_author=J. A. Stamatoyannopoulos;citation_author=T. L. Bailey;citation_author=W. S. Noble;citation_title=Quantifying similarity between motifs.;citation_pages=R24-R24;citation_volume=8;citation_year=2007;citation_issue=2;citation_pmid=17324271;citation_doi=10.1186/gb-2007-8-2-r24" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Hey;citation_author=R. M. Kliman;citation_title=Interactions between natural selection, recombination and gene density in the genes of Drosophila;citation_pages=595-608;citation_volume=160;citation_year=2002;citation_pmid=11861564" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. J. Hilliker;citation_author=G. Harauz;citation_author=A. G. Reaume;citation_author=M. Gray;citation_author=S. H. Clark;citation_title=Meiotic gene conversion tract length distribution within the rosy locus of Drosophila melanogaster;citation_pages=1019-1024;citation_volume=137;citation_year=1994;citation_pmid=7982556" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. G. Hinch;citation_author=A. Tandon;citation_author=N. Patterson;citation_author=Y. Song;citation_author=N. Rohland;citation_title=The landscape of recombination in African Americans.;citation_pages=170-175;citation_volume=476;citation_year=2011;citation_issue=7359;citation_pmid=21775986;citation_doi=10.1038/nature10336" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Sci.;citation_author=J. K. Jang;citation_author=D. E. Sherizen;citation_author=R. Bhagat;citation_author=E. A. Manheim;citation_author=K. S. McKim;citation_title=Relationship of DNA double-strand breaks to synapsis in Drosophila;citation_volume=116;citation_year=2003;citation_pmid=12799415;citation_doi=10.1242/jcs.00614" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. Klovstad;citation_author=U. Abdu;citation_author=T. Schupbach;citation_title=Drosophila brca2 is required for mitotic and meiotic DNA repair and efficient activation of the meiotic recombination checkpoint.;citation_pages=e31-e31;citation_volume=4;citation_year=2008;citation_issue=2;citation_pmid=18266476;citation_doi=10.1371/journal.pgen.0040031" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=P. Kolasinska-Zwierz;citation_author=T. Down;citation_author=I. Latorre;citation_author=T. Liu;citation_author=X. S. Liu;citation_title=Differential chromatin marking of introns and expressed exons by H3K36me3.;citation_pages=376-381;citation_volume=41;citation_year=2009;citation_issue=3;citation_pmid=19182803;citation_doi=10.1038/ng.322" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. Kong;citation_author=D. F. Gudbjartsson;citation_author=J. Sainz;citation_author=G. M. Jonsdottir;citation_author=S. A. Gudjonsson;citation_title=A high-resolution recombination map of the human genome.;citation_pages=241-247;citation_volume=31;citation_year=2002;citation_issue=3;citation_pmid=12053178;citation_doi=10.1038/ng0302-241" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Kong;citation_author=G. Thorleifsson;citation_author=D. F. Gudbjartsson;citation_author=G. Masson;citation_author=A. Sigurdsson;citation_title=Fine-scale recombination rate differences between sexes, populations and individuals.;citation_pages=1099-1103;citation_volume=467;citation_year=2010;citation_issue=7319;citation_pmid=20981099;citation_doi=10.1038/nature09525" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. U S A;citation_author=R. J. Kulathinal;citation_author=S. M. Bennettt;citation_author=C. L. Fitzpatrick;citation_author=M. A. F. Noor;citation_title=Fine-scale mapping of recombination rate in Drosophila refines its correlation to diversity and divergence;citation_volume=105;citation_year=2008;citation_pmid=18621713;citation_doi=10.1073/pnas.0801848105" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. H. Langley;citation_author=M. Crepeau;citation_author=C. Cardeno;citation_author=R. Corbett-Detig;citation_author=K. Stevens;citation_title=Circumventing heterozygosity: sequencing the amplified genome of a single haploid drosophila melanogster embryo;citation_volume=188;citation_year=2011;citation_pmid=21441209;citation_doi=10.1534/genetics.111.127530" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=H. Li;citation_author=J. Ruan;citation_author=R. Durbin;citation_title=Mapping short DNA sequencing reads and calling variants using mapping quality scores;citation_volume=18;citation_year=2008;citation_pmid=18714091;citation_doi=10.1101/gr.078212.108" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=D. L. Lindsley;citation_author=L. Sandler;citation_title=The Genetic Analysis of Meiosis in Female Drosophila melanogaster [and Discussion];citation_pages=295-312;citation_volume=277;citation_year=1977;citation_issue=955;citation_pmid=16292;citation_doi=10.1098/rstb.1977.0019" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. Mancera;citation_author=R. Bourgon;citation_author=A. Brozzi;citation_author=W. Huber;citation_author=L. M. Steinmetz;citation_title=High-resolution mapping of meiotic crossovers and non-crossovers in yeast.;citation_pages=479-485;citation_volume=454;citation_year=2008;citation_issue=7203;citation_pmid=18615017;citation_doi=10.1038/nature07135" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=V. Matys;citation_author=O. V. Kel-Margoulis;citation_author=E. Fricke;citation_author=I. Liebich;citation_author=S. Land;citation_title=TRANSFAC(R) and its module TRANSCompel(R): transcriptional gene regulation in eukaryotes;citation_pages=D108-D110;citation_volume=34;citation_year=2006;citation_issue=suppl_1;citation_pmid=16381825;citation_doi=10.1093/nar/gkj143" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=K. S. McKim;citation_author=A. Hayashi-Hagihara;citation_title=mei-W68 in Drosophila melanogaster encodes a Spo11 homolog: evidence that the mechanism for initiating meiotic recombination is conserved;citation_pages=2932-2942;citation_volume=12;citation_year=1998;citation_issue=18;citation_pmid=9744869;citation_doi=10.1101/gad.12.18.2932" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. A. McVean;citation_author=S. R. Myers;citation_author=S. Hunt;citation_author=P. Deloukas;citation_author=D. R. Bentley;citation_title=The Fine-Scale Structure of Recombination Rate Variation in the Human Genome;citation_pages=581-584;citation_volume=304;citation_year=2004;citation_issue=5670;citation_pmid=15105499;citation_doi=10.1126/science.1092500" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=S. Mehrotra;citation_author=K. S. McKim;citation_title=Temporal analysis of meiotic DNA double-strand break formation and repair in Drosophila females;citation_pages=1883-1897;citation_volume=2;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. L. Metzker;citation_title=Sequencing technologies - the next generation.;citation_pages=31-46;citation_volume=11;citation_year=2010;citation_issue=1;citation_pmid=19997069;citation_doi=10.1038/nrg2626" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Myers;citation_author=L. Bottolo;citation_author=C. Freeman;citation_author=G. McVean;citation_author=P. Donnelly;citation_title=A Fine-Scale Map of Recombination Rates and Hotspots Across the Human Genome;citation_pages=321-324;citation_volume=310;citation_year=2005;citation_issue=5746;citation_pmid=16224025;citation_doi=10.1126/science.1117196" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. Myers;citation_author=C. Freeman;citation_author=A. Auton;citation_author=P. Donnelly;citation_author=G. McVean;citation_title=A common sequence motif associated with recombination hot spots and genome instability in humans.;citation_pages=1124-1129;citation_volume=40;citation_year=2008;citation_issue=9;citation_pmid=19165926;citation_doi=10.1038/ng.213" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=S. Myers;citation_author=R. Bowden;citation_author=A. Tumian;citation_author=R. E. Bontrop;citation_author=C. Freeman;citation_title=Drive against hotspot motifs in primates implicates the PRDM9 gene in meiotic recombination;citation_volume=327;citation_year=2010;citation_pmid=20044541;citation_doi=10.1126/science.1182363" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. Nielsen;citation_author=J. S. Paul;citation_author=A. Albrechtsen;citation_author=Y. S. Song;citation_title=Genotype and SNP calling from next-generation sequencing data.;citation_pages=443-451;citation_volume=12;citation_year=2011;citation_issue=6;citation_pmid=21587300;citation_doi=10.1038/nrg2986" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=P. F. O’Reilly;citation_author=E. Birney;citation_author=D. J. Balding;citation_title=Confounding between recombination and selection, and the Ped/Pop method for detecting selection;citation_volume=18;citation_year=2008;citation_pmid=18617692;citation_doi=10.1101/gr.067181.107" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=E. Portales-Casamar;citation_author=S. Thongjuea;citation_author=A. T. Kwon;citation_author=D. Arenillas;citation_author=X. Zhao;citation_title=JASPAR 2010: the greatly expanded open-access database of transcription factor binding profiles;citation_volume=38;citation_year=2010;citation_pmid=19906716;citation_doi=10.1093/nar/gkp950" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. J. Pugh;citation_author=A. D. Delaney;citation_author=N. Farnoud;citation_author=S. Flibotte;citation_author=M. Griffith;citation_title=Impact of whole genome amplification on analysis of copy number variants;citation_volume=36;citation_year=2008;citation_pmid=18559357;citation_doi=10.1093/nar/gkn378" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. Qi;citation_author=A. J. Wijeratne;citation_author=L. P. Tomsho;citation_author=Y. Hu;citation_author=S. C. Schuster;citation_title=Characterization of meiotic crossovers and gene conversion by whole-genome sequencing in Saccharomyces cerevisiae.;citation_pages=475-475;citation_volume=10;citation_year=2009;citation_pmid=19832984;citation_doi=10.1186/1471-2164-10-475" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. J. Radford;citation_author=S. McMahan;citation_author=H. L. Blanton;citation_author=J. Sekelsky;citation_title=Heteroduplex DNA in meiotic recombination in Drosophila mei-9 mutants;citation_volume=176;citation_year=2007;citation_pmid=17339219;citation_doi=10.1534/genetics.107.070557" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. J. Radford;citation_author=M. M. Sabourin;citation_author=S. McMahan;citation_author=J. Sekelsky;citation_title=Meiotic recombination in Drosophila Msh6 mutants yields discontinuous gene conversion tracts;citation_pages=53-62;citation_volume=176;citation_year=2007;citation_pmid=17339220;citation_doi=10.1534/genetics.107.070367" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. D. Schweitzer;citation_title=AN ANALYTICAL STUDY OF CROSSING OVER IN DROSOPHILA MELANOGASTER;citation_pages=497-527;citation_volume=20;citation_year=1935;citation_issue=5;citation_pmid=17246774" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=N. D. Singh;citation_author=C. F. Aquadro;citation_author=A. G. Clark;citation_title=Estimation of fine-scale recombination intensity variation in the white-echinus interval of D. melanogaster.;citation_pages=42-53;citation_volume=69;citation_year=2009;citation_issue=1;citation_pmid=19504037;citation_doi=10.1007/s00239-009-9250-5" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. D. Smith;citation_author=V. G. Finnerty;citation_author=A. Chovnick;citation_title=Gene conversion in Drosophila: non-reciprocal events at the maroon-like cistron.;citation_pages=442-444;citation_volume=228;citation_year=1970;citation_issue=5270;citation_pmid=5482495;citation_doi=10.1038/228442a0" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=L. S. Stevison;citation_author=M. A. F. Noor;citation_title=Genetic and evolutionary correlates of fine-scale recombination rate variation in Drosophila persimilis.;citation_pages=332-345;citation_volume=71;citation_year=2010;citation_issue=5-6;citation_pmid=20890595;citation_doi=10.1007/s00239-010-9388-1" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. Sun;citation_author=D. Treco;citation_author=N. P. Schultes;citation_author=J. W. Szostak;citation_title=Double-strand breaks at an initiation site for meiotic gene conversion.;citation_pages=87-90;citation_volume=338;citation_year=1989;citation_issue=6210;citation_pmid=2645528;citation_doi=10.1038/338087a0" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. W. Szostak;citation_author=T. L. Orr-Weaver;citation_author=R. J. Rothstein;citation_author=F. W. Stahl;citation_title=The double-strand-break repair model for recombination.;citation_pages=25-35;citation_volume=33;citation_year=1983;citation_issue=1;citation_pmid=6380756;citation_doi=10.1016/0092-8674(83)90331-8" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Tweedie;citation_author=M. Ashburner;citation_author=K. Falls;citation_author=P. Leyland;citation_author=P. McQuilton;citation_title=FlyBase: enhancing Drosophila Gene Ontology annotations;citation_volume=37;citation_year=2009;citation_pmid=18948289;citation_doi=10.1093/nar/gkn788" />
<meta name="citation_reference" content="citation_journal_title=Journal of Cell Science;citation_journal_abbrev=J. Cell Sci.;citation_author=J. L. Youds;citation_author=S. J. Boulton;citation_title=The choice in meiosis - defining the factors that influence crossover or non-crossover formation;citation_pages=501-513;citation_volume=124;citation_year=2011;citation_issue=4;citation_pmid=21282472;citation_doi=10.1242/jcs.074427" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Whole-Chromosome Analysis of Meiotic Recombination in Drosophila melanogaster" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2.cover.gif" />
<meta name="twitter:description" content="Although traditional genetic assays have characterized the pattern of crossing over across the genome in Drosophila melanogaster , these assays could not precisely define the location of crossovers. Even less is known about the frequency and distribution of noncrossover gene conversion events. To assess the specific number and positions of both meiotic gene conversion and crossover events, we sequenced the genomes of male progeny from females heterozygous for 93,538 X chromosomal single-nucleotide and InDel polymorphisms. From the analysis of the 30 F1 hemizygous X chromosomes, we detected 15 crossover and 5 noncrossover gene conversion events. Taking into account the nonuniform distribution of polymorphism along the chromosome arm, we estimate that most oocytes experience 1 crossover event and 1.6 gene conversion events per X chromosome pair per meiosis. An extrapolation to the entire genome would predict approximately 5 crossover events and 8.6 conversion events per meiosis. Mean gene conversion tract lengths were estimated to be 476 base pairs, yielding a per nucleotide conversion rate of 0.86 × 10−5 per meiosis. Both of these values are consistent with estimates of conversion frequency and tract length obtained from studies of [rosy][1] , the only gene for which gene conversion has been studied extensively in Drosophila . Motif-enrichment analysis revealed a GTGGAAA motif that was enriched near crossovers but not near gene conversions. The low-complexity and frequent occurrence of this motif may in part explain why, in contrast to mammalian systems, no meiotic crossover hotspots have been found in Drosophila .

 [1]: http://flybase.org/reports/FBgn0003308.html" />
<meta name="og-title" property="og:title" content="A Whole-Chromosome Analysis of Meiotic Recombination in Drosophila melanogaster" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/2/249" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Although traditional genetic assays have characterized the pattern of crossing over across the genome in Drosophila melanogaster , these assays could not precisely define the location of crossovers. Even less is known about the frequency and distribution of noncrossover gene conversion events. To assess the specific number and positions of both meiotic gene conversion and crossover events, we sequenced the genomes of male progeny from females heterozygous for 93,538 X chromosomal single-nucleotide and InDel polymorphisms. From the analysis of the 30 F1 hemizygous X chromosomes, we detected 15 crossover and 5 noncrossover gene conversion events. Taking into account the nonuniform distribution of polymorphism along the chromosome arm, we estimate that most oocytes experience 1 crossover event and 1.6 gene conversion events per X chromosome pair per meiosis. An extrapolation to the entire genome would predict approximately 5 crossover events and 8.6 conversion events per meiosis. Mean gene conversion tract lengths were estimated to be 476 base pairs, yielding a per nucleotide conversion rate of 0.86 × 10−5 per meiosis. Both of these values are consistent with estimates of conversion frequency and tract length obtained from studies of [rosy][1] , the only gene for which gene conversion has been studied extensively in Drosophila . Motif-enrichment analysis revealed a GTGGAAA motif that was enriched near crossovers but not near gene conversions. The low-complexity and frequent occurrence of this motif may in part explain why, in contrast to mammalian systems, no meiotic crossover hotspots have been found in Drosophila .

 [1]: http://flybase.org/reports/FBgn0003308.html" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2.cover.gif" />
    <title>A Whole-Chromosome Analysis of Meiotic Recombination in Drosophila melanogaster | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q0wst1" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__KPgiwEdjo5O9HdRryHqQs1TpsV-PeBb9B7sGBvwM9FQ__opzZWgPNgQzoEgfATRS5qCKwBw_6XcNjDHoDVOzQv8M__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__YUqwBlv6bNoCToiBWgAUSPtppdS8sbGfzmAH3qEDcI0__Mvk93h3ScFH5Xi9WhibVKlf368fHw-V7HWqWX2QMvtw__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XNWYaQubFGWYWZNytUHkWs-b34LV21DsB4s_6gchI4k__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__7cpBA0GVdrMqkZGGZEW-_SPF2HrbTCmVg9-lrP5l_AQ__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__TV2D4R18tPiSIOmeQ3vyMqHObN_FDmQ6EEIrfE5sAoA__c9BBZFy1BrgGgF_Auu9Fmysb3Z7J56eJGYvMcLQrLks__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q0wst1"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__OaHnYyci68QTwctdRYR30kEfu0eYWBrLyMqy8THv5Ik__IyfJQYM1q1QEPwYsK_Eslh2RQkxpy6yo-nR0p4DWPQQ__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__fhq_kmKS3zjIxlsAa3GE1ubLajyXUgXXTXULzAtiqfg__Lm1_Ls-bdpbWRyl0q4o0mmHyxB-139AJ68dE4zOskLQ__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"7osGanZ3KHG1ahMUnio3AIhaUIJTBOjrhhdQY4rV0LM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459241","apath":"\/ggg\/2\/2\/249.atom","pisa":"ggg;2\/2\/249","ac":{"\/ggg\/2\/2\/249.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/2\/249.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/2\/249"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/2\/249"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/2\/249.supplemental":true,"\/content\/2\/2\/249":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jpZjuTZJzllzZ+iris111CoIhbzPvBZtaI\\\\\\\/3fYgKCGPLHlYKiFZW3y7RY2Z9QjRP0K9VVJk2uyt+60Ee2vS9cIiTxpgm30XLQOmC\\\\\\\/NaRwzZNoBpuJFd7DQEwDfn5OFKACZbPY564RMZBq7687XYgjF8\\\\\\\/X2djyH1fdSFKyMTTFabZeHg1kbhkaVQkzKFj\\\\\\\/I9mugb5VkQMjURk6+WaU6T+jUhRBXFVZbYJKqbM2dmcLVeeqMxhbHarDqgQ1VRpUv6D5\\\\\\\/KxxA\\\\\\\/uWqszF45tXdIWSSfRf4pj2bV5KCtA7G34qzxAtwhue4w1ezpRMdya9eKHB12UffZoGy\\\\\\\/WKBk1hTq394tWSjHPzA3bjG28vWZVxZvjYC1irfPEoeGZYx\\\\\\\/YijrKKsWjcHd7XWJGWlBImie99P0dfHuWdVMbAGuPCIYbveEf7IzDZQeXua1LThBDybZ9AsxeU4Rj00KOXGU\\\\\\\/EkcVoGigKBABK3DvpSA5zkqfy0sOmscRpXgo8eaPI\\\\\\\/1JOsq2Cb64VxkaLhmjrLc8iubu2I3P4fJlox2ef52f7Up\\\\\\\/cw2nKmapJN+KBkFZMzFd7O\\\\\\\/6dbSxDo0Ke4ttbRvmTKqpMsqWt3VwBllYACbz+nB3Gw0hVGRrBb0xSjP2Lz8Ve\\\\\\\/waty4QBIeMi2V5YWs81V4et2VgGJ4l7mvV\\\\\\\/TOtgJf+zYp8Se0buaLrsNbYxhkA1Y4\\\\\\\/iJHmgFBujFlwG1t2L4nKVEOVJWAx2HRcGZYLxnHdxQiDM25nQvxh+0qshEYzIb9nSwJWzihJlgX322PaePw5qHDS1xf3Sx8jFDrPNCJJ0WXlz+aefNZ3vhnqcyYQFnRqf0yTGw7ILqRGw\\\\\\\/P8YllmFvPUySF+YYygwZuaa8L44rDjnGW4OYzt6MAzNGmYZVCg6F135DcD5LO0N9i0OlB5LIpjcR9izTbC\\\\\\\/56W683kBt1ErCHnDkbnVGdU\\\\\\\/hj1DpUm6L0fRjLgSpRcnvxcqMosia6W2J2qiOX42gskcyaX7DeCD8wGqjs561OZ9aR+2nyeF93AlA0toMBR8Pzh6eicr2oDjlGqJFfQ+OWVsX2NusXCX6dcP5UDS+Hq0aq4qbkZtJv6TS+lK9+QG50hZUFValMY+GHpWMUuBHL8W9P0voeTZpXEDTfnGRsBsBofFZlo4TNqAnlmdA0L9KTEQHOkIpXQKfB7sQYdcaglz0dCjuGWH5p5ha30dVxKOXPtgl\\\\\\\/U51Db98RkqkeJv4u4+BMo0kcXmGn5RkPCxo7DbwKsApcnUKgeEf0iJMt3gnX+qJ9l\\\u0022,\\\u0022iv\\\u0022:\\\u0022Q0OONMmg5GjnOA\\\\\\\/Bk9uIdw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223442f5955a7baf245ed5765a3a38ec334c4ada27aba33c9e34f81d06e10850d8\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XOEppX9rBE2IXaG6fcxb4u246IkxTh03W8eZY94VqWQd+Q4V7yFQp8marTCrOM7rPKaF+TOcY9TMlgpWVamBdQlDS0l2L\\\\\\\/KxEQh7TPX\\\\\\\/nJaJl8sa896yL\\\\\\\/yg5lfheypmKFR94rbg2ODzHwZKC9VrlhJ5VRXUyAW2qiSbJX1zbZPkCWNkQzIxpQ96KT9MpUWw\\\\\\\/i6oxjW9n5hrimoXOoSt4wyazwAkfWgiKfL+dTzBVpuhS\\\\\\\/ax+i5Jc0hzTZsW1myblSzI2m9uEdJYiS9Isq5OphkckY9jvo6EIWC+e5W7\\\\\\\/YtNo14L+jkatDgNpxfHlMi6C\\\\\\\/BO2S7E2IKvg6oS5O9ihoxoqZ0QMbOx3B23mtoYCzSOngEpvwd71J1Syhnc1LeM7GGtVbXHvrT574TY5+vlhJ3FqUc957yy\\\\\\\/gN4z4tGXDft4Luan+Ybh2P5mNJW2fjvC8kkiE4jeAnZUjWdTDUsNzqNygJ4nxB5EbA0K28\\\\\\\/v06aXzWsCtDW1EOL7CQ5i4V++tcZZ0EfJRxoViH7cWKg8t5pbfUQ2rTJrpWA+kHdD0DZdAliNEoPRf5IUhxP85wDDG2ti0hOCZLCUHGH9prMx4XDVPYEwEbGalwRzepRtrSiBdPd0Rr4PZwV19h8FALfZ5YuECj4pjgJhV22lk5+Iez0oQ4nH6Tl3EhPdQ8B7zY3PWc6mUuW0XI2qdxyBlR+C8hDRlDvil1NpCXJpjZeIV+1fOSIxHdQSk30pkW2Yk1DA14oimfs0bebF3n6nFtbaUNOnkW2D3tzhotauQjf\\\\\\\/7ecnX8Fi+v3uQ1CDRR3Fa0OapbJ+QhQNMcdZ7VRIP+CDKQpCoHNzZdFduDuBdnJlRpihOhMearSP0TBlY0E5lVWYq155m4xpkUycfG+V7RDu3Se7toOdB0va6ebsAbG3R+TKPWpswjMjCSwKcuS2mHZyUbBKMgA1oRAXJ\\\\\\\/aOmh2uElnBX2AzgUnbjR7wSd6YeadCPFvyisfOlk9BdoeV0pTP8hnNWQ8rdAtG37YWKVeZi9Z9a9o\\\\\\\/Oj403+DmF9OIJqBFu4vu2lN6FzKzr9waf9ZJa\\\\\\\/YX0VJ84NKklI3fA4iWtBL5WI+7Oo3Gr6qFazIqd3rKc\\\\\\\/MnGJwT\\\\\\\/bQHxss7Do5ohrFico7+Sjp0WsYj1ggF3g8vDbtGkifr3VGYHAAS5RIeCD9dkWG1cMMQ4rurFgpKE3i9gsrqMBrdITjiRJzTwAdIFYAsFV7wyYG1gA6Lh54J+BeBhdBYKWqk2NR8v4XzmwCFgKU1OjfCywE01\\\\\\\/WPQmzscTJqSXtBJPvkd1fHALRFUT4FKA+PGgAu10\\\\\\\/pbIR0SxQXEy77Z0kwSE+VJqjiSVbo7OxzImfNvWENxNZCIIeBRLEVAdOVkA2pJEY4CdmYUeAvwWf2HD5g7I2gpUc43R4D+PlJYgWcVKSOMzFylkrqDLtakdRNeaYQPSBl9Z\\\\\\\/leevjwYnykSdNtPUhsKfHGcivMg3mS3F5QFw5oJ\\\\\\\/rE+i5CFGRTH8QHULStleuxxxVm6fjNZWuC1MfDdm3k8X2hsUYVYC\\\\\\\/wjUt+lHmQZpe2H47Zteqql+hu3BsR2MO4Srrz5xIIdNMEUIAZb9AAWp9boLXSrCvIuHJZ4rdH5CvSPV60B5IjaGyn9qBelOnaeEnl8SboUqVzPYSynhQA0LMoLxrPd+LG3KdCFiMOPWM9V\\\\\\\/+EzG\\\\\\\/wtYtgOer6Lig0mksxMGH0xFEkRzKM13jrbKxmG+gAkZNP74z2Uneror9NdMJpWMeqDwBCrlQJDXPDkXwA\\\\\\\/QPCOQIsBEgXk1D63XQkeSgvf7+VvMbrnDBQhN\\\\\\\/59Hcx3EkFtqi4bQZR+fUlhbBXdYA6OwV+eT4QBLvLfO0BSBZ0caRribHpgSJSunUpx0mkiJpdSzTRVyF4nPFFGGpamfnCDGlY5fkAWtaQEdL\\\\\\\/Fi7I8e7i5rTmOrPm\\\\\\\/0Z8zOQgxPUEAsoh7Gq8oh34FoE5b3RtD4V0ijeQiUDDscqnRvopgoxiUKPO1spj\\\\\\\/PxQLyRXUnIdi5uX\\\\\\\/U0cCtkeNX1pQDPdmdHJVJJ1ScOKNMOHvnh1a4ljpj5YmR1rMOEa\\\\\\\/c1O2VgtpgdTqyMzypzirA3cBfMLBwZzzTs4BgHye3WDTSRDiJ6fU1V964R8YrbTdsoIOvt\\\\\\\/\\\\\\\/GTm9F365zJrvbiL33ekLznszvqk9prCb+7WX+v4WUcHmblzhICxI1fTBxiqYMo5cfUKvY3mkYj+VrN9gERgibIVUSlqCD0A6yp3UUXkmTJ17Mjh+aWycQKiupU4WhnS1fN6\\\\\\\/FQKMe0C0H1FHRwwxgddG19at3fnqvFZRBRFu23zBi2WmYwOBXedXTks+ZjcWajx+JgfwZdmHMSL7tDZpdRveRdvez2nDUMnsp8+lpPyVjklBY6wPdx9Nyxyz3k1wOGksOIEhc2sOeMR1n6t3miHVms1flcazu+2F2YMBfdgTGeA74KiweZjn3IYN7InWjchogDRFkjN2APgnyyMtoOAtqdEetfU6rMdykfxmlA08\\\\\\\/hasbqukPmErzeeRPCYOOG4\\\\\\\/NVcRy4Pdj0F6T5N0cw5I24ivC+iWClGbUNOc9cCg6Mrz+vHjrrfuNw1VeyJRXQg9HS6wxj5QMJeB1D2R5kzPCcF97cLNNgwVvwCfbIXHogRU7X9B9m4SygW2jEChwnojeyOvkythUeFyYhpaManMwoBPhL0ZrFI4z1vIQE3K8AISodzhxz93ep+wLgds1\\\\\\\/rwa6x9b\\\\\\\/\\\\\\\/c\\\\\\\/GCTaye6Y6BWKOB0PybKtcXfMb322aSoTn861\\\\\\\/j\\\\\\\/woStXDGGxwpsPkV008lRlPvXGwmTsRrxdpdJFcB8tQI4FTNAD\\\\\\\/cMe+ZPklbsRMu4yiX+tLYSl7NyD1VuQM9qFNIYN3SnHcIDsU1ya5S1imCPbAkvTizISwm4SNOo6mymTscomNsUBXgNnTQ0OJcv5wPvDUQQUQ1wEXCJtlzOGLoVMaKe981+OXYcgs379tZnE7WNU0h9Ym5q2SDr3Zqbl38ILDaa+EfYxfaiG7z8bt7oxz5ERtG6x58ViPA\\\\\\\/G7JD+jJ7heijevu2R15aqTt38RbEryGspajUKRbBbV7B6+6LQwi2xHdyEpNNpcFFlskMVDrJpWvrOSVFA5NtBo+XEsa5Nae+cswFGndwRDzk7ki7DTgN\\\\\\\/xvA8PIyQrYWjx9WCkH74KHeEY5LuA732\\\\\\\/aX2bhR17NAJcs5ubzqFrn1Q7NTUmZcnz6IajPWVAYbS38YAlTE\\\\\\\/v+i3APMUzpE9e840gNV6v\\\\\\\/7pjoAptXbaSQn0htrk36GBXcdLSjVMq7Flp37MW4GCvQBrtIbqhp7hxIrMlvj2wvAlQGyUaylcMpBSxzvUegK2lANKFd9Cb7NFHuClHEKTDty+mRUAdJfCLuMaWFSA78JoSi808YfyD6flCU7Ax5\\\\\\\/wpYrWq+YbVm6MO1ssHUfaXEcjYlhlNECVRCBTEvFSJjbk2aSQm1U0wA2wxaxPl+LXESpz18jD5yCGRzYte72DUJX8laIEu0CbGMXzCrHlwyKGHEgGXw6gVXkWe4HPL\\\\\\\/8xC\\\\\\\/vfrE++vdLkAxMsCOHrDfNam17PxgJuMi7M8rIxH7ddK4yvL28yZkYXeTG9CdWKttw1W8iCbSFrULzJrm3BlyPCmCgr+Slpsa2rGj+4lex3GHk3knVwEluZyuDZeqEJ7XLMOWxKnoVwx\\\\\\\/R8u0CfKSnLMgUV5i8j1Zdc+Y2ez8EZjJ5TvEMfBl1XTCvJg5GI4JtvTRZL5WfbsO\\\\\\\/o1AhTKmRQFzYJUW4jGBiujGZyna\\\\\\\/lsCzfytfd0V4GRY1hsdyJzDkUA7FRz5cmxGlM\\\\\\\/ukW6nhCFVEiINecRPCi4X0MCHZhct+oBVjJWcUhbU4JnmIV\\\\\\\/YD8OP03RCIHiPbh1jyHtTqIFWyGxm9AG3KVQYW4R8MfQrlR1O+NV6T35AQgZupx9p+VX1VNReDdp\\\\\\\/\\\\\\\/wQIssVbpJi+GMdMgCmLefE7BFyse5Z5ces49CO0RweYRp9+y3KLoD+IDI2j\\\\\\\/9iI3KMZMD\\\\\\\/Hzv\\\\\\\/Btl+wiZ4aWJqvJe6x\\\\\\\/lJKLKyHvq0hOoWMv2XeJsOsb8NpsC8y\\\\\\\/M0os9Lq9r8GBclewn\\\\\\\/kqEA5lhYLrI99xDgTSVsa3SE12seo\\\\\\\/DXlRfoV+90mlBiAswE1Ip+nUD51wJJ4TUdXGA3FpKYpJzt\\\\\\\/h6VOEJwxfisIXvBIabwjh2+tKHC1v4VBQvT1CiDerfZjZVsqQMuc1TvIE1j8ony3W+\\\\\\\/wsdCdT1zLSvhgbXgDONQkiNBOGwIErPlcyXBpRfDC5x5T4MFJzG6cbxYfFAnyWGYoviYTGZ33N4D+RCYGIk\\\\\\\/HOF+vPoZUFQmGuQV4R1EfcM9eXWysInvIw3fJjGdDIOQNd4oTRAHhH+QrG5uPnAqvW0RnmR59tgy397xnusz\\\\\\\/w214YDtUgaZZWShZRX9IZFQHbF9jJ0S5sMfwGjTQrFEuoKSJ2F1zFrLDejqz5q8F66cv1fZqt+02WHpgVtVRSGZuOa1dS5FCD39OmWAwLqaJbjDFbZOln\\\\\\\/PdWQra3m1BuDx58k7XRaCDQRLk1oHjsQjxR4aP8zK3SrPo5F43I9lUEVnHtROk7oRVNwlC\\\\\\\/gWE6h0y7YBaZjh\\\\\\\/ZlTrCVW5SZcr9guuJPezugrvyyL8yrMr617z8mRM3rH6IaOe\\\\\\\/J0SWQFX5TnglCuAI6vK4QF\\\\\\\/KAtgDSTyTepz8R42MSN+jMcc33lID2jG4jTUvt54Hzk\\\\\\\/5pGiligcViGuhUoWKxVWb\\\\\\\/TYxGS703rBw4KSN9BMEoHeePonjophiBDk67BUVk3\\\\\\\/J6qjPhCGNrpIPxrg0wSSczRfMdKYMfQj3FBZH40cKXy\\\\\\\/ru3KlaLK8Jy\\\\\\\/kpiZ0sKYgfzKK0t60pZMJc2z4FC2lOYCC8Po0VaJByVMbltNnEY\\\\\\\/k6\\\\\\\/1dl9mJoeXAmNGmSJVqvF5\\\\\\\/H3Xp7Xt8W6nTYNZ\\\\\\\/LF2p3cvCyBtYfD\\\\\\\/b+i40sPyqoa21DdK7orIk557wcgS0WjMiBSsQ8YI0yh\\\\\\\/DN0n+0weB3Sz4PQrGx4L9KcE0GFH0lLxMhYdYeNZVZpuxAw3E\\\\\\\/TmYvClZchT9ZK\\\\\\\/KqIQv2Xt6J5jo2rInvmwTPofbVx+0k5cgo2In3GO2RqdnFbL+P9snqlsPwmmznKpiPNGy3SNZ+Q8EctVKK7DgcUW6oRCryWrjkWXGkbkjMtAD3\\\\\\\/nATLmfRurABt9UUY9h3CHZzmCE8blwvH5n2Pf767GXEBNJoKgf1mg6LMy8NsraMkG6pSL7DFaYRReQfbuLMtIZXm2ZEor13E+sjRv7xMrRTEmWm5uvd33lOKSqOXrAPlRbt30XTVd2qWyDmEVds4czOtXG\\\\\\\/A5EIA769HyncW\\\\\\\/dKGP66EAhlim8qKIzqHN7mJBMeg9CMDCeKICdc4l33a1h\\\\\\\/m\\\\\\\/MEYy+iJyq3MXnDFA2FOUbaLzbeP5ZU6W+0B6byBPJjPOE\\\\\\\/1VH7++86dtmVLrsFC\\\\\\\/oEPlZNEEjQmGAoJ5u0O9NZah71\\\\\\\/nu\\\\\\\/xNgez0BV\\\\\\\/hqLhxr2SY348GlmqztMVuqFwogw11gcsEgUU1Y5CDCtEw8XX2xiCjZPxM72CJ9GwC3+ukWSPPHj6dK8NBL3hrhOIiH5YkRdICPxGC8Fh91xrBP1YOiHnLHeVvRfOS+TI0fwUNTczpa\\\\\\\/u46ADC7LBfPysdtXAjGp7kkBSVl7KU5P335AlAISUDKnuHGZerajS\\\\\\\/PlTBcwJwTIoyhgkaXwhMnb+xa8JJOoUE82Xy\\\\\\\/jJo2fuZZFdvGiSGMSYlWZMGeYsW88VFuE8yRmwGxvdSj7RX4Nn37q9uxpqSfmaenCzAAOga94Ec4xtV3AVR8sDCOJDpAHO2gj+4ErKU9ydaNff9BOb5cuR\\\\\\\/ZP8Q+wcrufl9q0WmW5EVKPZkrA0Cy9dHh4xNGd6ZF0sKJQab7BdlADvDEF8Gbb\\\\\\\/qZBoYD6ZcNVE0XF3rMGI1CzrE4OmKq6PKDP8Lfz4xpK3QJPl749Zl881m0wJM2\\\\\\\/+FFb+RiyHNjzLJhCMGJDqKfegQe3\\\\\\\/wJ7ULkCLi5v9UYSFJYLJQcf\\\\\\\/HKYRwaULDQ6xBwZCSLpHPMiNKTGuNmgAk+geWL5TjlqZdAk1i+FcFGKebsaOXgfGM8r3Z8aX7qVW\\\\\\\/WqbAyuUaDlx5KV2vyFGG5jHdDpsIk5Pnn\\\\\\\/gZ9Oapu0NQRkHerYTCAA+IF8qrDLxiUn1mRpopaO6xXd\\\\\\\/re2mKPz4jOmVvoKBJ7T+JKsUbJwoyIR8bo38ADLOv5sx\\\\\\\/DuC3n3lAO+8wDvm7IH1h2R8zLakVZr7Zsb09mmivjKWVRz57Iw9OOMx9vlg+GMg\\\\\\\/rdbaD+oRN5HL+BQgHrqOqr3Ztep9vDxMp+Frcp\\\\\\\/NXyhjvgCz8Pb10bJOA5owANCQhbz8m7sAcNe5S1WjNJRV9o4Wj7kCwVCzr7B966QHkhZeHkbl+PQmdMAfBlKL7l1LgfFmoV2ruldh0p7kfPKKk44Ddqg6ouDeNlJL3XepUGH4zguAyJZVzYH5xPaUb67Rx19Vwfmpz8fRnsAlMv+GClIb\\\\\\\/i\\\\\\\/0ugINwXRahai8vawO5h7DeibH\\\\\\\/+smDtVq6VGsSE5imqA0cKMJYRtjkPdtpyNlRZdXt3UPllKohroUwmLnLYKH\\\\\\\/Qhm\\\\\\\/upEWGfNKEiTckEvLBoi2CTvvHLViusUErswPhnKByAPwItLVNLgU6wpp2\\\\\\\/hNiekmbJYaHqzH4ktAN2bTASEIiCVOgAevtnoXlUo+sS9sJZDGzLWGG0CM9yT2mwQl03t8IuoaEYQaMjIy0IM4v4769WB33zXq5d+9ei9Yn7c44cnOEYR72KeYUuyTyAABp+ovfWqwf2nTIOPDXZ++Ft0UxZAuZnVTn8ndgfWENBrE0q3woe4J+YU75rByiqi5tAX\\\\\\\/RGPYqZNPmqENxYQc7D6iyc+MKvfRj0aUmbn9y6NLS8\\\\\\\/6CWCojBGxiN+3PTqMx\\\\\\\/U\\\\\\\/P2YfRwaKnlE851vOTKvUceIZqYWqYhEWwYTToL3Nb2OssHaclSrx1uqXB9jzIgifWGp7GA6qqbNIGwcTz9XGrhK0EBe265Th72CeORHKQsOYuGk+pXKPtgRvmvQLFLCHFfWJEWstW7U2Eo6K98mUw2rg8M2K4KTAkicS\\\\\\\/r9J+nR3\\\\\\\/N6QfnicHcFTLBT5XQT8AGRj5zPrfrvhbvVyLtGpnDuxGdEEKfITJlfS9v8StanKP3Z3OdOVR8RRKA3aVjOzSjGSEEtjQQYOdMF3pFq1F87V3DsGiCY21kqW5fcYopVEZq2S4JY71C9oEITKCGoHuXfWTPMyupTTKBKPECkX023Q28pi1R2zKvy3zJ5N7l0mEgpSENlm+C9ODydq17i6nsVTdL7CdXtVpnrZYA2fAEr6ZM+FeF8OFvEpKyRmRdXM9eZw3xz0vpB7CvJeCFeCfdCVKCD1nWuA+9ZOR\\\\\\\/OUGrF8D8ikW9\\\\\\\/7HD\\\\\\\/yeMajC9iveIDtdtQ7AJoWyVP0oCjus6mv3b7lKf9eC8rskZvCAlNh\\\\\\\/77arZneTvFXYqc24Rx+kLVzIsGM6byuBD4H+DRzy0I5Pk4mQ4e8UXGMtGt3jK7xO4aYl+IGfr7Nhx56VF1HupHMwNYdCvFTB6ebQ\\\\\\\/vSJw9rtDFOFevGjhx1bcQKlxTFxTja7+4Ek6RSnk9QMbWXfz08kDntx4xyFIH9MDu733yZiyvZAGHMJWQxnHzqfffXlj+em9lo0bZGbt\\\\\\\/CuFRCnHYjzu\\\\\\\/yYFmSp8FSmzBpeiAWjXoZUKFoaGsISieGBw541Tjw5xCRJiL4v2nqsLOhzzYwi\\\\\\\/6TASQm2OWYiFB9jLX\\\\\\\/JZarFM1tNo6kmI26rz7eArrzWDoHDeuN030zCEgxhMFZIGGQBA8J6dBckeuveQvdSeJuF9fQ2VyQfIt0VqABZukFtKqp5CpxZBZFS82zzek7r+jW\\\\\\\/X5G\\\\\\\/eGC0NjhmT3K8z3J6cVuyTQEsui1xk0ILbO6b2GoGuWBRxJ2Zu3KPFS5+FApDeauLFjM4WhybGvjEcmOCZK1VYenipMljGDSsxYkd+zvMwhkdWlAvJVOiXVa\\\\\\\/yb74xMBwIo1ur8kALRVkM9DW\\\\\\\/UuWtcNCokWKexx6+mMsqZICSRa7XX0Y\\\\\\\/mwurPbCWikN8qF29cXy8aA7WZVaUQ9iF15SrPvLvGxXaHiHGEkYRu04qv4TIrKrIIOC76J+gbhdT717QxqKB93P\\\\\\\/itzuScrTkCUED1FGc6DflGcTTme+WkBJZN0iYAONyiPwd4bv8FCEyPgTRMtBsBo6BK\\\\\\\/grrzJVtRxyFWi7u6lBJZ6weFPIajrUTlKcymDsQwH9hgcOtHphwmDigOJwb8ESZiV2k7ZMBnpecbINgMZRYnjj3tWpxTv\\\\\\\/uKDyickjLbZz3\\\\\\\/OJjM7grhZuzjGBxdKFSV23kHBdVXKTFDjSe9PvZoWa+sQPqscIGXEXVEaudmtZW8TiGnSdg2NPbSjAhQyiyMKppflUi4wqgvKkGmDWLrmZwpuhFMbTDqpTWsPNIaOiOLlAd+hkTXx9x+h\\\\\\\/TocwNaK5G6I\\\\\\\/y7DvVW14qIFoVTD22PgWuSYfo2CPHcOh4YddK6JNGu6WxLgD7Q+KSP9C5IHUqJIAR5FYdbzp\\\\\\\/cJLDcufBajcf8Mc9IqlkeSIUUQri2zXMKI+V6zDZRRh4LsvgvAOIgeSq5PSS3pgEaa8cjQRG1OnduRA39n4hhsx\\\\\\\/w3rluRr\\\\\\\/OduuCF1J+9a6aOayEQzYqqiENtp9NW6gIMB9+ca1IvK5cv9zZwBIcSVwRvPcbFE2FClRnq\\\\\\\/Qp3F+ra6zItT1d4yvQ3XUdgqUVwOPx+bEDE5G78CujMudRiuFy31KcLNhxZMJwG9cOKH\\\\\\\/NrKcLg5tqGeT+MqoqyOb\\\\\\\/m1HzP4ONlbO9UJHz2z0+WXrQzeM+rElOCa2PruG5ZeptYFqRhq9Ybb5WWYLtTpZ7FRvqtLNCq8DcyRb+YalZSkzzsbHcxKtl3gYMEiaNvpvejgHK77IcMTeHoRJJf8kEv8jPE\\\\\\\/LxBFNTzM\\\\\\\/VW4HDMAYbNmK3jxB7xKTpyZNun0HciZXbtFSN0OU\\\\\\\/bxJX4O3h+uPE9P9HRKON0wmmr6b0E2YDmH2TIr5kNEvr\\\\\\\/aGmLreMS8UVnNsEPFpdAgn\\\\\\\/ndzMZZK9K5vzWx6b3W\\\\\\\/quFS5VKLHadmY19C\\\\\\\/Fs5KkJX+8DMp2PT6x1XXsO6h8V+xiFG9FzOfHapn37hophbAjt+rhYK5G6poB02Ou\\\\\\\/S85BnO5hbAT8W+9fTRhpZjkiNrXxnpXIAJ6KyEeYM8u2WqaHvVVXjwTS4GYm9un3lLNF0TVr8Fd47a4Z\\\\\\\/lEtF1QgSuriB91Lzu2r6QR96IBV7sQ0VU9tx37TqLElEDnHPdBDwcaPngSymOZ1qdhf8E13I25cHM9HhGNV56xqP3yzBWvP9MMTTNablNCXr2i3D0PUxnuwMUT2iScekpxhXbJbuiTXTU6FI69axJEMuFSVONQVvy3tjs+upJUKZjxic9smq\\\\\\\/v1I+EwXGqFu3BK\\\\\\\/L4spZa6IvlsZWEXg2sM6W9VPscFVIFid+vn0QUVNrRAmjXJREfvcPHSiJINCQ1\\\\\\\/VNiSonY6vw+foe44PG2Ia9fyIxk5DkbAd9GozUx4U4ghc3d8x6j67EZrNmOERdTrDOOG3bEqGcfIc7+kVApdbYwFqVyxlOwmZpwgXLbp\\\\\\\/ENyOVsFB+7xMRNet2qEUA2ih2CtXrlpSr4J16pxMGmsZPzpzx+kidbTLnHlKh3PhMLvg\\\\\\\/AfxiCO8rqra5p0FwEkGMPXZt4+s45EzP+MsI\\\\\\\/nWdME2IJPvjAlsFLohDfkLXzc\\\\\\\/+JbwE+Qaq+ogwlBZjCqxuGFUe39qxgMY\\\\\\\/RixBHYI8\\\\\\\/A\\\\\\\/6sYhzALl+DkQLD1d5ZJu4NMxXUQGGRRh4Q08ReCq2eqvE+dP+Rtud58u8MeHWGNABMuZmgMo4B44jeBsEwJv1jrI8s86cqdH3wZqSSiRZwTvbiIUKltDbRU4pPLElMwg1MTygSi0J1MdakYCWvbbgfrWhXn66i71wwoHfFVZZjVKLU7z1EfM3wweayBqkIzB4j2ahzy0yglgVSCoNsIh\\\\\\\/FdtsSlffiga7LkHwacYYu8HxGVbA+WUJvJZP9P8PgbKt9nqllt0jjkXqBf8TZrnUlHSNTTbMnZqSOYESVfxPbZJuT4l0r3ARyBhtoYbdKALX03vntlnkRh4ZWe6EbpUxwXk0WyMFj9t3QbzMWQ+fsGrbxgNnny0PnRRx0NlacCz9\\\\\\\/NZsDxnpuVutZvJ4Y1bJzyqUTDh+KWw7z8Gd3HTPA+pmMFh9Ng5mKnqXVcQWaUHsPUj129jCO\\\\\\\/sYYvPvtqGGxlcFCt7usGo0cAciG30uQyRSLx8YpiPkBAd1Jft8HomLrMRTYB5PqG170nhGB5fJj1UXCmk6V4VsvC\\\\\\\/jrGdlElMBa+RfGzxY5rpXW9jmQ9A2Jd0q1kVRHLYKf1eoLPnkKzJnRifJNcD6um2joGmfXhv2JXtPTKde\\\\\\\/fL6mPW\\\\\\\/tn+g40JPdq7EYIJ+QbuJ\\\\\\\/phGwnxIbjuX\\\\\\\/eVNL+w5mZTTLU8IO8Ck9bJ0njK84BD2yTPkttK+nMox\\\\\\\/sdjhlSorsCF8Lt5y\\\\\\\/8MrOAq7EOsZqrmbDpdEk+\\\\\\\/FsW+GVMXOSqzeEZ1enekyBnaynrdJuoORZlcUkaFf+Ad6Vr53ZHLvmp14R5ZmavzmMx+npERb8NRnBu5lds2KiOqtUd5cD2zqk6LMjZ79719F55tXmkXyAzXaG3P8xpI+ikm2BHH8ImJB97UYNfC8TRFRzssTn0fffPdl\\\\\\\/NO8cT\\\\\\\/dwXpzvGWQHAXP8D96PORXW02ZLCtc3WE6F2rTDgxxhFcJjQeCXT+vhx\\\\\\\/0GoU1CsvKTN+b3tBrooBq4lqQCTsb99uTqvCpyDeW+iAP1inwJAxp32V6uEmewOMZ1xkXk23UDfpGFcs8kL0QnOlkvUq8HnZj4LxFvnlzuUgSaDN3qUxVIGtxQzi88M8UE\\\\\\\/BNCdpN5j55kXfVMI2xTVFMNDG7vwPAoAX4NFuEW+RzGBN5zPK+Feeb30tqCcBdYLL2wk8Vnx192NkMmVgBOneFN8Q+kKE9kwFFhpVBsl8lKLG07dZLvuoiKXU7zUrJywlg9kIaXpDdsD7\\\\\\\/5DNun8DcinTVz\\\\\\\/qBkeGbuTUTgGRmzqxwfA0PMXCKTN4Cv5vmGydKfG5AjQPIVuKm8YZnz3r4DNMVLHQkpyj2\\\\\\\/X0k0Ts4FJ+TcSNfi3aj3BDHKCTvhHc+e1r\\\\\\\/lNxEsSuI2c8GevY3ZhbnaKw7fk1RMGZ\\\\\\\/ozyLwjD0IdWZ2+BEr1Uqa2FmblVYQScxP8WMEym10PUViDKyToNFvpdQAvBuNVD\\\\\\\/CKoxww7BwCZhobQZw8gyBAWFBEMeeq7apRco26jhgJqOZ86nZoxZ5Q5\\\\\\\/xxBDcv8S0P99ofuOW8mdOKFOxcRcpisa7gWBqf64UY1nVNjSe8fzXhjU7rvjNSfuMEl6lD7MRxGjrNSrFZgGbaaB5JhY9IFwm\\\\\\\/iRCp\\\\\\\/YWfsa6Em\\\\\\\/TBs16R0Zj4xHROtSc8BCF40g4bAURjXLzeBVEegK1i6ewn7gpbgjAxYy0lt6imJ2PT0O5o\\\\\\\/Okx0\\\\\\\/MScRHC5aMiHMI3dpxaUSCtIAiQd6obO40EAFcyBeALw5An28Aw84ICThACVBmGUoHT2oIBPXIRKqWpEg6MVX2+ozc\\\\\\\/gqmtOVZm3DZ3KrntM0MansNordPgRSYjOLjc\\\\\\\/fFl8TLex91P2sIk2GgQJKb6Oh063B36KcAD9AfE4ayi0NmQTLm+MXfhudqLDDvd6G66EcsWuCoHWFXadhBXcMU3PyEOwdFgeP4LEtb8hoAInGNBQw4MmZg3vJxaCssk4unwnsqIdNCY4nohLevucxTWvT\\\\\\\/8ApfxUdGYES+Ave6ryaRzPn44hzILkpDO9FJzCMOlqOcqFQnObJOXp27XqEV9fhXacEcRCpfGIlD0KbVQetJAPNTX+bQhRY96TrncvtW3CIXq4PfGWBYPhr7Nsx6l1id\\\\\\\/epVZgoBa6WS5clb3x3GDueW15uSdWJIpQFqp5VVSX15ZB4cOnqWUUZuaP2uc16AWIpE5AmL9lDQBxynvB5ybL83FUHDzHWulPXAkcO2jExlfjCNmxuKbiFqciSCiSHtXpTaVKvAndtL2r6wFKM2w2vX2DaNMuMEuqWU+qBHtewmiUEB99G6RzSz\\\\\\\/\\\\\\\/QjtAMC7dk65\\\\\\\/y2ewqN8squJ0CAyRlWu7s6jqxoQoxYHesrDUnSB6pL85jTitpTOH0ZEJIiaRethU836QHNmV5zRfqMozBVlA4OTsf3Qb\\\\\\\/\\\\\\\/z17qORk39AQ0kPxQx+skGwEKDMZFzTSxcFO8Whrg1Ukzou3ATEUVi+\\\\\\\/AeTzVvouttpIaCRKY\\\\\\\/HGjUeW+XHDK0P6O7z9H0PSH2WsX4iPuBs5ooq\\\\\\\/T1Ov602fRL+S48w9ZrsdBGC0DN0xLVxf1puL3cKSjJ3i5w217AAzrCnlO9s7Rv2cFzbamSCWiTGSpU3u+ANtKm2t8VKXtdYAGikwvefBk5h\\\\\\\/k4QS9AQB+efUm5k5YZ9tkE\\\\\\\/d2XNPlL8+WhuaHYaHBjXswZTlsoueIQqoetTK8c6J2EwuSCdLVQOwMwgBeyetrS619OyZg8kC91si5u94OjpaXRq7WzQ3UuR7sdp1TfybnHULpiGB\\\\\\\/DvEvSmaxLC1GRtjBj4LBl92jP9XcUiwMmNtf7NLtQ61xNhCSlziPJJu+XEatjqvJDPibzJ6WrLTjNGcDMLMQJdS8fn045bcevH64LxJLaHCtDEE\\\\\\\/4sR20J0m2x5sULuF\\\\\\\/K1rT2Nn4Qox9Eu+zap5pKPGNMDXOkBUqLtEVt9bvfAXR9nCffVN131azaWC3b6KQFZbvtdQsOCxyHS2uAvDCQmNfNNPJJXfc+VWkd3CCfT4y5LcOX7g7oPmE\\\\\\\/T9peXiMz5AFN\\\\\\\/PAM6I5I+Kappi4qqA4mmllWL6x2EXfLQlow\\\\\\\/ZSDSlbIyO+nAo6USVg2tw+Z\\\\\\\/AJNtDb6Qh\\\\\\\/hiDxb5CdWPi1HeTkylmELriD4wkO0U7K0DX3Jl9q9463Xs7nsxbYAMO5vAN4+us74MIxf6zZeNBK\\\\\\\/l2U\\\\\\\/ltrtFyRTOWY2kPfJ21cYwNbQ3cJHTFbHPszElYBuDaY9bL7W9nnnIFTPFELsjXrCwx4iXmvBiKwU\\\\\\\/+lbMzqqloEtGHauabfLjb3bw2+HLFdhCQlwsNuAUFGMpxIhA3mYmWFHS3s\\\\\\\/5j1niXtD\\\\\\\/+EjRHjLeMk++IzYKgrepDozyfXBt8G5AIXlEnDMAsDRDJp4rtkJ1wN\\\\\\\/TpMQUMpDn0A3GNv8up\\\\\\\/uXAgi\\\\\\\/pu6t1lc1Rh2ENBIM+mNbBMo6alMom8BZ3qVtroxjaYCRm6v4XsMZRVGZV\\\\\\\/zkQjfLjas0ksaSTXvO2rCJdohM693itKRg\\\\\\\/HMB6Qit2GsXi0WS9l5sEJUiNQYH7TFb0DA3yWpTNBPrThM4GGg1DugBe0lIZ+21YXvzrHDreun1t8EpOz1S5Wn11lwGkrgm8D83awRuBIGKEfAm33Hot9C4IQXIRQ0JfhNxERiDL30Z8Hp0reXHwxpsH8a8Vi+ioPJz95cb7EBSX\\\\\\\/8T4ATyvbI+j3rr9y8lFhoAAGQtQ\\\\\\\/LHT7lkIFRCHjNJ\\\\\\\/IljHRJu9W1aG441K0uwfl1fhYa1L33UFDppBIdulZbcM0QmdU+KgHPybMHDNY2JfglSznBob90+ECmRcoGETKuk7dpwocqAyImO4ev5KPFEGE3g0VSacwOL3+5qaOwHoPuypB\\\\\\\/QQTXdVogFClIgckrrEckMNIks9+s960jQxqDcMxJEvQZ8wiMOVb+YAwb9tRYbmL\\\\\\\/CeBHFvAcEE0cELKZ5NhWY+7OQgKGzxmxPgy+rsMnT1xqctGeKFwc3KKjWuLGbcEUaCxcJr8lS8ra65DjSgJRYwsqLnHLIsJ5vgOoWnuf\\\\\\\/asm30CI64desW86GtEUFsDEQ67QYA7+9zHsVZe6UPF7TRj+Vf5VAoCpELEdkayB8JcO5JVNuven7pyCOGeE+JMlnXIusA0qeqiJscmUsBzsBeaUHjN5pZF6RysGf6\\\\\\\/563OXLE9BuIY3hCy\\\\\\\/c6yCkJD\\\\\\\/jtNL0ug+U4KC4aabVKu30FtBQVa94R4o5TBOvFaogdXNDOnmPrfhV\\\\\\\/XeaiMaSe1ACMJyBJyLsh2Oi9k4hQSoSOT\\\\\\\/HzucEbM4YHpKx7Wb6JW9187v1oe5JMR+7BD\\\\\\\/AW4dlPX3nn++q8DlYOQPP+PYb1KW7E15aRI\\\\\\\/nc3YRZYTt45Rzs4sPcr7e6lymGnW852fTGsi9KpUISqezwW2sMVF0nS6JHlYGFHK5f0Xq0UCQGgwfVh2haK8lM40drY0QGHWGmXyb1W2R7UnuLoXRqX2gLGjoDRqpTxqmfc+bznGKXPr6B6h\\\\\\\/mBBRW4zfmdCriBmfgepWuePm5BOB26kjXyYaebKItIl+iuR4cl\\\\\\\/N9vITD9F6Jmbw2E3zLrxVSrmIR25E3foovHtPesOe+\\\\\\\/o9wDZXUS3dbPDz7At9xZIr1BXs2vUu5eCU8fh\\\\\\\/cjf0Fv+36ucDKXVzky6Uyx8RD8\\\\\\\/WBGJT7DEi9HDtYJaH0z3wNEbhvAwzlSXiXx6liiZRfMT3IFRVKuc3kSZHrkL9PS5UEKpaP67y1p+5HuSvvWSySvxj5FFSZIsVvpv78k9Foul\\\\\\\/KBZL3VGe2Si9rQguirpAgT8KB8EmJjxHuRP1GmamLEo+W7OpevnaMcHyuP05avMS5QET6R0Cw1le15GxGyOhaeRe8OwzE+4ZYfBmU6XciQmcM5KCrKFudQYiFo\\\\\\\/berujFvHC3V5HqZ4knAbHz0ikgo5aHhtfbTkRycwt3P1sf5VAgDT9v5V5Jw3AfghkPvAMsMHJVFQafutLgRi9tz4GNs\\\\\\\/TkeayHyywgePWCRZEJyq\\\\\\\/lmueUPsCyxv28wbb3s6UJZyBZb7LHAF7t4iyeIpvVVCXpqmNVVZmn0b6yOdbmkJpmCSo8dZ8EPPQbIzqIA+GFmX3oQzHizCgx2rJ8egGv8miPN5M\\\\\\\/fUPFwJf4A8KAqE\\\\\\\/9SfS8CzX6+IlZDjiM5kxWe1Q3rTH\\\\\\\/Ab\\\\\\\/Lfw0i4fzeG\\\\\\\/RkCCkaVE8IULVhMKQIUiSTeBojrl2nmeF1ccCYqAGLammPdVmFDc6WQ7FcJiUcKHCu\\\\\\\/LlRdD4q3R8aW7OZrCCHbP\\\\\\\/CHOd8rPsy+\\\\\\\/c1galI3pw2VqI74gGlrEPc1kuO16JUEtXOP7ZdwZJ3nz7fMh\\\\\\\/dkGRxJ49wApddlQrhg9bWj8hYSd8CWf3edYeKDdI9HospDySvE0kNHrc2dAcfYiKAl\\\\\\\/VyAasx+KwwyYCGx+RQvryN7On63H9W+3ZN560OvjKll44viM3n88BUMWEqxrO0RJw+1a7ssPz504rYRSdAFZOHW8fX\\\\\\\/5+s9d3PLKJzeqJmvs5hhgpmQzPPmvJYyBy9iZQ1xrJoPR9BhPGdhUW70kU2krMimhXOGm9mlbw4qMlvGZ08GfM6ETEIebyycQsm5wJVMyZyy4e0f0DpNvO7KrOjdKeeMk5z3KYtyPjSXRrBsvKi3k1LQFNvDnceO9x4Rsm+y7kGL5gK9\\\\\\\/ntBbfRrlNz+xKDWF4QHsqFYeIzg26YEiOdb07Gcz\\\\\\\/POCg5WHmPLsBRgyDbWYRTyz0bf+O22b39TIOAlnjKTo6UypgUVrRdIDr\\\\\\\/3eO4VwPTvEtl\\\\\\\/UbFkyvmfa4tsF7poNUwFsytpGiqbOMFucXgNGvDWYH0a7IyG4ri0arLzrqjW+csZB9TRALJqeYiHt82K2EgsV9SokupPfT\\\\\\\/0nRW\\\\\\\/TlvXL8+EEuaJNi5drA2\\\\\\\/27JsMG4TAgX1o0\\\\\\\/GdZuDkBpbkZuF9+LtKeaEyCiiHu1vWmhJHNJC9phymmF8cKHDdtkSfPDlcZr0CG+d7D87kzDG1tCTXhIB1Ptj9vSmw3qNfUniCN3pdaC1TyszBCTpM8ZSjC+S1EC4roR4sZ+3Md6RVpXTmSqPS4xy40yD\\\\\\\/Uec806ha93\\\\\\\/jPwW\\\\\\\/RhNiVwAX4cvyRHgz+3TebZ9uk14pca3UCKeBOnQLw+PlOudNc2ckN5wvZNsJpjcrsIPAHz+DdO46FSax9uXVsAOdL4kd6aSeT8tm8sMcSqbiWrpmcWlTZ\\\\\\\/FpZr4WRAsPbflMd9DsjmNn7QsEx8zGE2hKhdYzaBNc6fJ9YTbjkwhDMauoPbezliSOjfGtrAL+apT+dkCIqkf0PTKCuTgtLv5VoCoa2cOib2CALppnbP7m2L7+h6FPkW03E309b8L2FRkU3f7gTO2EHgZ9H+HLGp45QqCn\\\\\\\/pE9RImIUMPURz7zDOmgsrp\\\\\\\/hQKB1+5zv4gUWBkW4RPd4purSR1tJVjOK\\\\\\\/BZOOUY0j+f9TDODBVf6brMvnyDKE0NtN+ePobzAyaEYeI+vdMebTeqJ20CZzouAfBktnUXdTYTCjrLZw8GOzNPtA8YSU5B7K31ikOX9frcZ7PROZqs7puTcc8okxm8CSCez6zHXyLtk+KBipK0r8T0xxWZh+kZ7MGMjGDMTnASFxa90dbwLh9jtq1aDI0DFpp6P61Nz1LWJjnIkDb7zfkPMokg4lpU41gqeddGR7jZaXTPv4wp2MK9gYd7wfIhdGYe6ZFOVnnhEDtpt3JNXRer6jl7wuDbIbaI18u4SCGA+Ejaj\\\\\\\/UC1PxhiZFP9ReN7rb5dKcFdwuNenYTblQG9FhA4EGPum6hOAndiVSEjJHv+jfr+KdLJxB\\\\\\\/YxF1wV7TUh2NYCMm8b5RLNUSkz7g\\\\\\\/yLrIFRnCXcGBWnavim1dC96a2f0xr9IziFNvID82ArkHQOZo+NtfLO0PA1CiJ+6eEF34FksyvRNSYZzTuaKkKjtrQdqd1LpJRCmBu4YOZy2qCPVDWz+AzYSOzKIcJmWrHTyXT09tcz920Elc4gwSV07e1WnHd\\\\\\\/KOA6nIM2t76TowGGa1anaWNvskHrFUPQg\\\\\\\/Dbv7sbeye9L8NiZ37sIcPU3BAtUB5f\\\\\\\/2c2tSF8lxQsSxF3hvea7xnqcEF6onemkbLPWFuX5HSiMwwoxG3bxc3Ar43Y1CGvADUCl5kypPbiCMVBcP1fqGo3hDFBImiSI+4Qgmrfuzlqm\\\\\\\/AL1Nh1kiW\\\\\\\/q269KyQBeg\\\\\\\/58kG6HwHwWhO1\\\\\\\/KL4fJwlYed94HVPc1BXnrQZfjaSTMFZ25++Mkoutg+kNoVWpgCFubb+\\\\\\\/imvFoFkDqumMemxzkmqhD+Q5OjD8EZ1j\\\\\\\/oMdhbdhYsg7BOhvQyxX67BhwoBcJsl\\\\\\\/PFrwL\\\\\\\/+j\\\\\\\/lBqF4OqHdyh+gxLpxKw5AQPSllqt8fPd9WiI24wKoNAAoePriujrbTKAaxLr9FcjbGjAHPWpKmxuCJrDot6rwtYcobnlUakdkoIszEKXrn2G3Fw1VpysXE7VUAaccej2pkUXgY6vvXISqdc\\\\\\\/JXGReCUNZl03QGrEaI0u+UOwJiUsfFxe7BkPDEynuDlQqlXJAGAenj7fcZdOr8b\\\\\\\/npMFhefjm8\\\\\\\/ytHQ\\\\\\\/Etamo+\\\\\\\/W7kZDKFisQJ0yauaScgsuO\\\\\\\/lZEulD0msgAssk1bM8ZraZBjeUPopgTF3rN0pzi5i6l87MmVMmRQeEZbkIQmbYGD7YbdbnCH4kdDT6w0x1kT2WPtW0oMjljza6IczyatKtDSvHrrKLm31rIgkIvPx8YoUKTLRH4NSRWJ1qCjVcm4eVe1XqSgHkTerchw+TkR\\\\\\\/A+4zvszzrWHqIgtETkEl5AA0MynLuRQdQGxgHO9t2XWWxHiJpn3p6mrxIUft6bF\\\\\\\/xgy7AMd\\\\\\\/OMDyP1oCsw+ieEVofJcMV\\\\\\\/QENh0d7icRrmTo4yBoHLYDvB+oTBgAGq77f6H83s7DDBsx1kuyA3xm9aXwhoxae3+tYxDV9yYE4OjHrjHf1wGA4yVe+GLMApgpJnSG6uyp4EOuzmrMAE\\\\\\\/Pq539Tt+yaLvfIu5TltLG4V+ytDRQ6Acv9y9WA4nq5KaNJu3avll77rks8BULUEd5xYsBj6fh7a63BI6CGrn0a9BW3y5+JQXGigNgBwqBr8T5bPv689+LWYXTCrbPejoXES2Ga2oLojOG93I4II4nzhd4sfVWx8nQHAeTbUpjFECAlHHLDLNnPpgGnf1vbB0HNavv80KXGJ4nLZ+9zuNozRqmdZO+tMndpgc9Y\\\\\\\/XNhLYbUn1QXKoBDLnnrTbqv6URuiOPpEGOzsCa3z4L+qXzNUQjwdNYZbYK8Zltfq4MitJ79KC\\\\\\\/GJAxr1zrqbj3UlhA1A3u5PiQSjOhuBO6voxhwnjtMMHg+da3Fm4PlSKi5PoaS5+V6OMp\\\\\\\/xpZPEZvtBZAbWHcspD71YIVSxHIBdJXzwst\\\\\\\/zZECk051lWZXRryYQsEIZYhwadnro6WRChZJ6r\\\\\\\/slXcL6jrfqPHT\\\\\\\/+NRkJAZrd1DxnZRcMqjT3UXNRSpOzgSeew9nIyNtQhYC5rgFkSt0NbQTbZBepRy+bAjmWsA0gl8bTS8cRfPaIgzvYSuhLHCcsbRZWtPl6TdmrUVx78nZtNHJ6eSmI2AftLtGAotarcitVFhu2MsBB0jEy5x+Yzc3hzaVNaa8pXgebUli1qn3UiQUB6abiqMnO7NBYeAksKx0lM0WO2iQNQifRXmGqHT1Ho1lJiEaiUnliLr4gIBjHsUob2ckDNOdqeMaF9cK6+NmtHNdx0k3JHTqMnEx73d34+RoFKPLsEr3YCj0SIXHA2WnWS1irdRWgq5d0\\\\\\\/DuyuIndwmV+v6NY\\\\\\\/GOYccBJ5YfVIGCREfWhXlMSyZGsKOXzx8I\\\\\\\/i4IVZ6\\\\\\\/4OcsAwZM6zVtV\\\\\\\/lpPROr0RfUtGmr+G3xTnQXm8+vG\\\\\\\/+Y1NfKTJH+U\\\\\\\/+RMOwT15tX1IgSpHP1HTPyCGV00dcRMFy5tnCrWD3pzUmoDVyNBW7DsNbwaqhfH4nTDaWaaKJ85at+FmZgZ\\\\\\\/ONUFMW+G4Ug6JgO+0ZbXFzASLdxY8bMyJWP7zDCVMZw0PI5QeXRP8cAI11e2ycb\\\\\\\/HxpP+ojI150yTSKzVWFSjUbLAhLibZfpQKq4\\\\\\\/BTUps3vmUiID5+9Pk09KrGu4CEx+7qdqvNEzcCQLlPAAEJzhHXOAe4PEgAlLkAiVGXTwu7ADN2t\\\\\\\/23dFiRAQ3lrJ7YQq8TXCxKJjuwgOseQpxyx\\\\\\\/jxJXQYRbb2fFRRZv7+lHEmY6DlCwfz920RYPuSrO7tVYrJH6EEkB67YqBWeVNtWiX2wsMPa7oiksQbH+wOjBO6Tqd03cgJGrjJefNXIe2a3LxLSLRRIDiw7lTaN2TTz1krJu2jWkl8qd+bBJfoAeaGX3VMcaoEDWs5frwz2U6F2yQmh7IPMzut8r\\\\\\\/PJE7xyV8x5RFYXjxdPwLOaZwC115\\\\\\\/t89j1GoyYwFxbNjD73hKb50TUNuvBMBVhZwQZaqJCT1p5G4nPcTEf\\\\\\\/2SqFW6not3mwy7PjFLoZz4YxZnEXnoP74w7el0mbxgsndoztsXVYbrNySBBfq3qsoLXVX1K3lZu+BiGzR5QYYE7lMGrc5esj0F5iHswghOr+KV+LbCeGxYHkOcY1pQhNjPhFxsVQlrZOGPjVmyCz10KEPA6iAHHf+Z18ijvBoftj6EY5Fz\\\\\\\/9co\\\\\\\/IK7rL0ORCFskDw1dZKAVQSkF7KtQ+v6x4xif7aju2\\\\\\\/\\\\\\\/YCwAyC40IBYMM3b\\\\\\\/kRaM5FLIP33DGuSiI2u14xv86mjl1ccmNDwDZwfP51SBeccl0VWLIdjG+abEOWi67g7hLGL2ouhHrFTwdADO+26jCyKxfWm5ig1KxmsEvC8fw4TN\\\\\\\/k44niZJsjy4AuOkGe6bgnEBLqOmIQxQHJzshfvgTnsWeN3rUHOT7bOPswnhdeHMoyPl8q7L7WywYFV6Mivpqn59E2gD9WDfDhz1ccTD\\\\\\\/MYih7AD03vAR+YS2DBPeQTbD4oczervcRKjNEOwExsfGvZcsARsXUcjucPlu49LQ852utHH+xzHeTqWA6M97rSt\\\\\\\/UZsJUZPEAKBhaNf\\\\\\\/qQ0DWNC\\\\\\\/kuzYkQqjdRyxtk9ewlRpR+xMwSBUNludis1HseA+JyVIollFNBpKCV1cpDST7RFxIwxrYHc07vPBEJ+dJHsuaR9bK+gA09D5I6MJ3hjcdqnfwnpdtR9ZUyrjonSLHHy0r\\\\\\\/JP2On5hqLlZp7To7wx+neTMc7dzZQfOtrqQSFolfyWOX6peDx28qzdBq\\\\\\\/nbzHJYEt1jQV87kUD6oI6AeSy9QgYH6s5QtKMtee3ngbi4tr7pCwrUu7ewJ50v85zHSMFeXMGE\\\\\\\/yRoKTa\\\\\\\/IRPPZsb21XJkfZuT7SWXv35V4x5zLKXsOBoDMxOV\\\\\\\/CUqk5RGJ9h79Gwe93GluG\\\\\\\/XYf6xKZTF9i6b3BjO1LRDTUhFNKDMHpq4TLvo6xpaDVih7+lWReGPfhxgpRdg0wW9ISPBPvKiHgNE5Ep52YMu\\\\\\\/RJi1YlQNiFceku+ngjrofAtbhmMRJsxuLFe1M6aKUVVnHqgP6mam05KkwTMIfa9IA9xWa4d35HQ5zb8xXQsBYSOxEgKRnCM4x232qQsjYC8wDp0AAnBXUShTaf32O1EBPoiqmi7HD6zRoe8Xy0iyeoaVPzQb2xOjX9py3NMy5+0PUqKPKpZbyIqF\\\\\\\/YO7Q4GLsfJoZasgNRxkxWFHU8fc1JhQBFqg2rGynXkOcmej7YxOuQ9k1k44iKfUUsRChO7VIGdF7tDwIFkVyrPk3HE4a7XfRPrMvpQjiFzN\\\\\\\/RwlNVgg89IixJkXQzungASydFC6lKMcZ6rfQglTq4vXtyvvHdJEDO7CSXcJowyg9UTZiTIg6AvEILsXgJH3LRJ4czRwgYeoFFR3is8j+kr1OQoxb13pMhfATvCY+QnMlNtfnplujhiJuJCzx2qO4RF70H3vCFhueqgjv0bBgSU7lxfedQSFKWFwvz8bjKF18BX5gRgCgm3T8cMn7CrjEdho1pM8G9hAWBRWeWUvSnV2Te5Ux+FGyPAAQHRcilcFDZ\\\\\\\/O1IFQMIqEFOhu0WkxitdnXJJpsbAnso+bU24Kgyl0pXV\\\\\\\/JvwyiD6gaS9RdAsW6Dw0VTutuRXnXo8e+D+mMcMsfUmFESxHFcK6Nwt43TJvOsmiw7y5o7tjTu+ydBcdIU5Lo0wVDDoaBfWjS7l40To1yuI29KipSUwqQIPfKUeMaPv2cSp5uE85X309Yfji01nWwgfGyf9x4PvtPBOKNXu+TVDyTgOXMdloueIzQNSqH9vqN6qHv\\\\\\\/l9sFBhKvgxkqoSlqdGegTk+n5GEHo7839VPplfXJKbWkbrUAer9CnjVyy+PtGqbjgwC2oiXkkLIr1L\\\\\\\/yQcNXzA7GVltQ3v0hUTgI94JF7Q9K3EqE8SZFup\\\\\\\/V3sZ0SE8cEyHCsj\\\\\\\/tUT2Z3gqoZYaMNPZvBcwxGDnLT0kJlyRNWBOvbazFTP+YBZ6RngthlblKTxJqHU\\\\\\\/KmNr7Z0fJ+GQ8BRR5FvePCTW1uD\\\\\\\/9i3deJ+Bl+wCRRRDvpKkJY6D0CHLSgk45ED3sv\\\\\\\/UOFbCRDSXTN2IMhsAak9VgqCjt6K5zbhl4ZjOuCFgkcxzCKsKnT3qJxtUTs9bfAYu+LKyBdvPGrjQAzGsWHxpveDGrRNQX0DkLRXdqVwuhxrq9UNdnvn3Xzqdn4pRm4Rg8nVUf+7EbkzVa7X9VjTe+Z3poqWFPH27mMuUC\\\\\\\/Pip9\\\\\\\/z2hV3Cwv06Ws9jwWfEnX3hImXFyphtmv\\\\\\\/e1E+Mf3eO5Y0aO350BoI5oGZVUzrbWHEbbe3\\\\\\\/q83f0QZ4rnSaCvHKLR7Lwgd215XgI8L6SMHPUNBp+2S0rlS37NwIuzjZBh\\\\\\\/yy\\\\\\\/SkHWqJOkpcqFb7q+8F+N+LJmFTiqFYy2Tsa4e4os0e6vpcUwQNOZWkPRyZPFXvjCNMdV8KiK6QkStDF2r4w1zUYxg+W4EhVkDw86dNrjFs5rERMfxSOyvQ253qrXlze4ocoZc2rBHKkcbENSzYTORYDZHr\\\\\\\/cqTqFVsi6cnETuf3GgB7h3TJHqnJ5g5DDfA4WHCNyEkg5LDxEbHY1sMnGUzz71RxkT9rZgptnRC8mpnZwsoL8dT5f03gmbJFs4q\\\\\\\/B6RSg2QGh5ytdLrnU\\\\\\\/4X1nuNmRZs2n5QEsAc+BPkGkgPiWQ6C8Zz0K+qdqME2VLV19m\\\\\\\/RV\\\\\\\/c8asUgjB1NDdhRNtrxiF77Nnl1th4PYW3JN6qAktrQ5iDJnyDr7EB6GY3gtflYbVnkhAGWuPT3z7qF8tv3MVtako1tvFGX1yO71p9Jv\\\\\\\/IPZVyM+R510BwGB4KbZmrUp9EgiNMxsvo3IeZj7On+Pz750ArTHzOQ8i+uWZ3Ed+LI32z0fGSiD50ZOFIIjc6Km+Swto2iPglxYYyo\\\\\\\/vVnKqr1aYC1CH98eUpD1PXO+C9AzDF1jzD8W03dOe4Tg+IC6ZFyGNgqg15MsoD\\\\\\\/wZeiLjxW4zy7W3e6go0V9qtmtxfjGTWswwpcpjA1RotgmGxdafYyNaHTl+wR0uzOEqJDIwkf6\\\\\\\/tmq8qFFRrZl0ZGMDWakac3SZq6d768+th5T1lh6VcOQVanuxaGad9GMYf0Y5T\\\\\\\/QGUR9nQOAPT4lThhaf74wom7H9ILe\\\\\\\/H1YoNdwiXeXKqyXvvqIL0TQ00ynIk0ROnZbWF\\\\\\\/TkeZ9559ktZpY9cd6nIc02u3QYXQ7L8yIFzs5Egs\\\\\\\/9SFQTtSo7GMhcvYJu2egqOz9iK5ZKPVI1CiT1PT\\\\\\\/TOU1UWTAmsEnV83C80Wzzw4Z9hWrFySRwvHu8eYvKBP0p8ENNYDLcQVpOgbOZCr1B4pxTy1as6T8CpTQINvEueuyhV4pYkkWEOPEyyxqTjNocUWIEYdiQfC22XAAaq9hlrV53Cx\\\\\\\/6CDJWUqzDnIa0TGGwHRC5zGltaHcwbz9y3Tw8AzSkHQ8PcPHu0c5vrlZcbZF11DxZ\\\\\\\/w7omPBrCuqX2FYX26dH\\\\\\\/pjT1d7Mr9Soej2DMB1rrBEuh8CYT8EGFyPTo1GcILKy\\\\\\\/sj40oyfEyHn8Gtc8MzpwvIllsPcoboynQVjP\\\\\\\/anoqOAAtrA9cBI8IQUSl6fJIjHuPEJPl4RGwkVqW6dXJSY1cOrTXjDMA\\\\\\\/S8UfJ6KJ7ZQANik3wtyv4V30RB4W7ehQnhB7H3orGMkY+Y961P3WXyW40+MC12NHQwsxhtt9TaPipW05fauAqPIDLylOctVj\\\\\\\/bhiWzZC95aasfeHWXYuB5kSVhymdVCfTI\\\\\\\/ORZV++lwNdEr7NGWxoy7lYClAAcJrJLmiXX4C5hD086CsOawtHQVpbSmtS1qUluhZs8GmJ+HLSXV45IzDaE8FokNSAtnodV5hDgZIM\\\\\\\/OXfGhMhjiVj11FZRGvSQSikaaAjDLWz1te1yPKjQepZv0tZTgZ3l+6oj0wYbOkGk\\\\\\\/lW0QfTIcG9JmoeWPQRC9OfMWPf9nXnpOVPJVfeJrqF3WYps9XjcKRASkGmt7PfKq64OaUbWUPoC5Tyy8lWxAUGQ+zT2CsZN0VaFxGjHpUkb290xuwr7+ZroTdIjCJa6VHVAcKFCHhxtTGt1QTd6OX+95GKpaZz6xVG6rXZwwe4TxWW\\\\\\\/gYaegRAZYtSoa2e2igakbz6QvaMwxUM6klzfuNY2abIrPlT93okXYP2IJdHfQSOepZQLuW2M+YB4uThpJIX5obShdwiu0jEdSkabyRNKs0EYRrJkLDlk3C3wHYFw0Z80GoxdvU8weE7Eov8xYJqu+LQ30LQfAd7HWWLhHLevkSf6K6oO0z8KJonwzb7fwJfQIc84KZovuKCMHYmop1w6vV+G\\\\\\\/Xa3RZe\\\\\\\/kwXmyLZNGDjw9I6IApsicxRCjOz2oXcTuq\\\\\\\/4VP3INoAE90m6BqMgKeYKdmYJtPXKRLCjegG25baAupwce2QZ2TfQ3wObeAQIVXC1GYLCA1p7\\\\\\\/34xP6h\\\\\\\/3MpMV7wViJqY36f3M1H5eKldHeDyrx1SrsDXlKYOMb+3zIrgpAqtBUs9ISXGb5evPyTXbDQ9TnzZwhb0voqnlEu09gERaDvipYiPHzjOmwf7\\\\\\\/9dF9Fm3r+pQ+pITKMJQUbPBTN8umXDFWVPUcNrC+bP\\\\\\\/3jjfRvXuS4U8a6v3FxJpZOr5HBFzwCdT3ySCtVR0jXeJL2xUlhbhMt\\\\\\\/yL5IVJgop0JPhxqZzP4\\\\\\\/7Hi9nWXlPb3lU25VBYbuBnIKydPHtMC3Bm5hbjIlEEu3Czgm+C9EMsXVdVTQ3oHadxoeJJEVxmFbx+Z41PsASGdCWU27kBAT++k9myrFeerKjDjQOYPh7Y4PZzbTz+OSfk11p0ICWH9Rfd9AxD2Wxa394XSNFHEgW\\\\\\\/GrvKxDKdUmZBDVf6pYhDYMU5R5mZ5eH4KcybQjGtoI5kOHABxv8x\\\\\\\/U1VEC0QtwxtqSEe97CZqWY\\\\\\\/KWmpUwUEYM96q4oUFrvqOqnDQX2xgVn9D\\\\\\\/9iQXT7qNi\\\\\\\/U8qZOAy7QBcf9elwP\\\\\\\/hUNrJu8OTPR9Q9fvr5w6lW8Xg8VzGiXA++\\\\\\\/N8k9EgZoKVV9haCt7sVW+vZOBsmqiS7jcqYKVtSw7M4h6Q5wciloFJe5Avj+RDIlc1Fczk80s+VZhLLBFZMvLsfnfEy55ka4NBfnGrJgKUXSzNMVUO38E5nY7SC3eWv2qframdtWyzeCShIgJQUuLFYNPg3A+OYcIyP774y6P71JAvJfWCSzz8r285lekx89mZm8iBEjuXGR6QUW5IpkTaf7RWdSbIuZv0x\\\\\\\/EabIcdIIKxnWJnRNRpBn8PqjmPSbWTU4Z5PDRMN5hx+Fp4DIJIMe+3m1JkIHolchzajwYTOkkADHh6RvNl\\\\\\\/hnt4aOVBEACpOkza9BMwlOgGHVBySLQcKkDH7eVzO7UYsvwkbdOgzUQou3lToqpNwl8fNl7Xrfdti5dci+YLGKfBSy7\\\\\\\/JTQ3NQTtpvjMbAsl1k+qsFD2iZ3edKhjR7YahJgmLni6vmE6porNxF17jP8esjrB6xcHQ64wXojdq9JtVap027SnlSqn8kXXLwkdxZZ\\\\\\\/60lIJ5oGyOCfxAEgfojWaxQkcnVtg3GQkR1E5Mlu+nlyuzr1wAZCMLB3OIQnRV\\\\\\\/4Lu1tH8LdOvzMtTLJjOFG37lk\\\\\\\/hqlhLNMq9lbP9Y69H4WzVO59YhGU8JkoQIS6kRz9dgm5Sv2mXtnQYobf28eJrbnkcygfByMrX6b6t171YnizLJYRPOPHQ8jSQlbSLA+d9xEjnE\\\\\\\/FpcaKOnem1shPH3VoLun6V9AiFLlr5EFW1I\\\\\\\/dEXlX+bpPfX8zkhmuB2wUUjhqo9VrVzos1WmrO6az\\\\\\\/On8sKEDUzLxxOF8m9Qo6i7ndJ0yR3u6yLp2sR1EiRRN0KMR8mCtkQmF+KHzltmwmO7XOB3MCQKug28vQUb7sxtPq73vh0ZUfVT0QhNuqHYP92gByNVSB1ykSLKBY74Ldzp2WM4EH4\\\\\\\/aO7mP9flx\\\\\\\/1uoZAZ7yJwhT+UQ4gWyZrgoNM5zgGcCh\\\\\\\/48e4PZWQ41EPKHHxmDnpbvk0J4IhoHQ0qlBijaYrNJX5m6GlvkEpk5PCq5zXof9YfE528QcSWoVgZFjPeVFhV4jr+IdyQugPJhT6NIsR31xy0JcABouWeSPJpdcsm4Vj4CvfARwzxu4RhbLsfnXiBkoaAqEcZgeGbMlLDc88uNVdsFYl\\\\\\\/VbB7QSZo1gJZSdFnrC5LDll2C55EW\\\\\\\/fYvyvt3x18F9s4AeOUybXlqUD5h576vGSWkiBKgT463Gjw1zTBEBm6itMF3Yb0w8m069LL93J3DK1UXMhOXL36l6ISHIgwEtZyV0c+5cN6NV5zfPu+HE9mJlXlC\\\\\\\/S8l1R4R59FIAw0TUOSFYjLaejGh6yF00OP36hDTpBhjlqBoZrukdQ+NK+wuK8As4hKlTyiUaNmS5ebjGTTK1TOBVLrWcq7vpFx4N+JBb+Jq\\\\\\\/isontESAdrKa9iXmk9L9X6SePRgnWh4ShzzQ411KBOM4CjbxsoHd\\\\\\\/Q\\\\\\\/VlIHLIrHrjfOYK26L2ESEMvDdhv7TnF6XmsjeEaAPz3JPcZggrFg00HiE0YMU145vKVbym937XPhvcfwaGxpyHcvDmcQEUfk9JLzVDLqyYOUB85+GLeh4WdLfeGT6K5lDaxb8UlmMe2gkVzKZ7ss+0sOXWl+c6KL4e+Ax7CZRDbD6gaCEUGwTH\\\\\\\/t9tXz3bsk\\\\\\\/38pU3HkZbsEkyZ5c8gWANIIjNnakAnq\\\\\\\/PyoTl8wjrKg1izMnqPO2IA92z6CfBSPGdq5iQaXrBAHeabCcmEy8G+IpRladM5Omwx7+RBuoK\\\\\\\/5+LGy5MzYXWcvjiHaqMxWfQkZDhevB9Muu0EBd2jUcYTgCNjCW\\\\\\\/qiCiVg\\\\\\\/R73D51n9gnxjPjE6HuI3MdwfGd5L34ETXqQXSMcljmk+2\\\\\\\/SrywcIkDZLy1QvznqvLkI7GHIQuaxKIcR+CBtyxVayyGppeRRtvaJY21MA6IkLi+0FKaxm6gDfw5NmW+mxueLqbCAnzyBwA6UW79PQvn09tuHLgBvXXOeElakhq5ijv2rRlgANQNfyi\\\\\\\/37VDm7xl9Bse5jXLi8io4XmJZ+qU32uKe7r\\\\\\\/j9VnhLD647dRi8Mk3dArA36kpDvyj2RRz8EX0HaTdFV0JQdDshGB1q1yGTpy\\\\\\\/atfMR7OvvLqjmhJ8k1k\\\\\\\/4uvq43Vf0xW1LysBl+6BV6oNlEWQVHb5bbok5C3+lLUhn3rZs97F0swxJtyyZQlaugk3J1N\\\\\\\/J3jFwLIc\\\\\\\/pAc9jR44RYRg87BvvuTtZpfXwkQR32HaXkWaaXRrmw91N4Cp7j3cdqvm93Qi0c71OQilhV\\\\\\\/naPV\\\\\\\/HYbV+fbzfzS9ouDp7kKiodzTSfKEssaxLAngU70GEl1AIWgKTV0Cdo\\\\\\\/GNlJJqF+Sp9zaosH4FyO6ssZGNRBf3OXVB6unTk5\\\\\\\/5iO++nlEjnGbFcf3FiWsKmoM2BXKjledLkkv1scKtfvLMAmJDqlkc6ly0okhLWj\\\\\\\/WKZmfX9rUyFBKvggn1KUIuCB0cyzQkBvrHIrQldK+grpGXFJDMQIj2q3zqX7AHzz1HpbIPD7ykEENSZ+nrm8JBjO5iZ6pjUoC2wOK7ISMlpyDONbaYSyhrcY2S53Y+PGrLH55FmEbdeTO1zTRKaBGRKEZ66gjC+xuMByYfiPOXsRWCs5RHJJepzP8nXFiMrVP0dGBXFjlyvtMjcLiQIERoNfAqTn5C0T6hnenCeCQDj25J0URJUwZY0uitkIZZ1hzXqUfBM0R+QLR8\\\\\\\/T5CDtey37vSE9FfZauetnFOI38l\\\\\\\/G76zc+yIizqhFiSOEBtRVEqYkv9GdFzXk34u6C00d4RL+WxkZhRmg2w+5f86Q50YdX1y0et0zmMoU2Yb2Ggr87C5wimlEiTGDaMPntfuGybhw9xMXTWTSKPVZFCpvx\\\\\\\/0eBpcYPP8SZM9EJjZvkna6Jnjqmf4q7QNkRICYiZE2GINUMjjPO9Chc787IPkPF8VGXjGs8LbmmEDAc60stJOw6gV2GYUzUgFM8zWwJCvO51n9rm886rBBxObLkAdSK0TnBOyR\\\\\\\/M\\\\\\\/jy2z8Tog4HoWe5dttB1xSyQuEuB2cWMROL7p8qwU3h6k5XpzpnTyeHB3OYMXFmFxnAM\\\\\\\/UaT8gzfrW05rOMe56mmF63ZXG9WYIpCQWByT5BUZQxsz1jP+BSULJhODEEveoixGIuU3NiR3uUdAICRm+SXqeLOxnRbWXeF5OB+h4Gya4PvN0V4ieaMnCTaXBfXiN3HavK8k6gh204GFrhq6sjU0a\\\\\\\/PJ\\\\\\\/fy38FJHNLR8jR\\\\\\\/vgCrGlme1dGmIM1aUV5Q3QLktMWLR6yoW1Ye5ov\\\\\\\/YalNyPNeBJ08fjLwCwAD\\\\\\\/Kpipe0jSrU\\\\\\\/EGDUVi+Oe1dnSo8Ja5NrPx34QIzvFN5PBTsgE8nRALk4unRkTgn0Aa7MHQmi1YEO6nCYKiwfzPim1zoyLRbaoL\\\\\\\/THI4DKInyvUwcju1Jbkje+Px2CZpIlEcGdXmHSIEnMVllxUYFRDlRVpMppXPDQRVom\\\\\\\/\\\\\\\/lUiSiiVIskbvDnBvjhidXk6m5m1F9JiTFhUwvoR\\\\\\\/vQTuTFAxmUroE0gzmlQzoMNEFREzCYd39lc4Qu\\\\\\\/JH5LDt7EINtNR6J5RLEKDXRMpCZ5IRcgh+yuXNBq\\\\\\\/RNPPlarELbF7n2j9el7ayRKCfqo6N1sJijaEGPFrJUxX2eeATJzkSfMUWtlY83minDoXsXu8OKUKghBkiMPa4QhTN447Mi58+JaNerQVg5fhDtIP5iySudeCNm8ii34f4VxYfUgVWgZ2ZBhy0CSmRNYKkLPEMG1TUQX4tsNOAY2OGthWYJVMx6Osl+4bpvfENenvbGKYWbaxtaxuw\\\\\\\/jUuHH6R8P13Dfjhq3TtuMni0TXVJfEECFy9nyjJ4OWBOQkwE78g\\\\\\\/HMqj+yAcQhozTzcBftbBMBTMwPXgB9hVmjd4rydi8WRcJbk4E+TdklSEEfbkmhps=\\\u0022,\\\u0022iv\\\u0022:\\\u0022dmjvDgZYcaaNhP9S\\\\\\\/HOOhw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a71cddbdeb81080385fc3573f54c671708ddb62f5814663cedb711d749c7e1b4\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CqmalsNpBX1U7jwjfK19YO21Jx\\\\\\\/ZTpLcl8Ocoa8aaPBHCSrjX7J1CwET6Oh5bigeME2iXjLg6B\\\\\\\/Q+AkH9VVTIF9weRa804PgGPO6s5s\\\\\\\/DdCtmsvj8Yq448tdeqACQGFosL9Fy2CxnNGOes30M6D2Ml3CMtvKkj74WWZEOU+NkUMdcRZ0uBebW6iHeT4HhzAwa7w0deb4RLqaJiKBMuR\\\\\\\/lzZ29IqXPJfpnr19gtBC\\\\\\\/fqykyYrfz\\\\\\\/oNNJlH6iKvgpd17m+U3eC5fhv0L33+tDnWTB+l1QvcPDwB9\\\\\\\/e1fFPlJXFU1we5IZISjqfoVOratXbIZRP2TEAWkb+dLeyCQamiIqe6KjR6aN5mYxAFCUC5rIBZhvMW4juo9KGIAmxPQjDUslNBTMenbznM7JoLDDWyaZZ9FhLZj+Mw7IkkPoXPOlmt3Uf3BqPcYe79XrAsBYvkXR3FAWslujcMpr3u+gy+uq2+OVkZefEl6LRx5EgZ+Wydy+v8PQSVK\\\\\\\/HtAfZRBJOP4UFYr1f6JNOiYw7R9fl1m9F4XSqW1OPdfX9JfXgIgqtqb\\\\\\\/pauWsIsj3iXqvWopPKysG8nXwnZrWqHSVvdeVxTa+uiOzpNCbPQW5vWqdJJH65CpB4U+62jCjXLeEnwafTRNVVTVEhN2ocvqtYTicJlSM9n8dF6sZIHNzi6exnEXrFh5BL6LkZc\\\\\\\/B1wug0viOrJRZlxjffPFHwsTHcJ+q\\\\\\\/o+VmJxmX8+46+AUulBQyDMJXeJevS\\\\\\\/N8RPAMZ1R0fKq3jsqfWEruNuedaTzwgWF3Tc3Pq1cAvn45AiJPWqEBqombEfQu5V+gRfxSZdqf1QdYnYtyF91El2fsag8dSnbFQmX\\\\\\\/OyJBy4AuGRLtiN6BpVKE7AWMwMlZ4SCRHQd8Gwfqg1lvHsaUOVJ6WU\\\\\\\/3J3eJxMEeiiBDyUQkbFwj3awHtHhcR+yefu7esTqiKLDVVMqda3z+eF\\\\\\\/OfEbcY3sTvHIpedxflMyseEM1vVJQ2Gs+tU\\\\\\\/U4oHzdgTw5sYtNYlofWK1SbXYSnMSYpugs0vKNStRr4ICuAk990PQLrQ+Yp5hMNMf\\\\\\\/lytbuIM0pPukJ20NVzFQyBK1k3r2t60h8nrt8sqfa7U\\\\\\\/HX1u43PZL16el8OjCD7QUUwbMUKryAiJQMMzHvqIkb\\\\\\\/N\\\\\\\/LbrRj8eiItHNK8KgN8Nb1iswC8h2HfgoYGS3rjH1Wxnhit6KFoi0pGEZD5nGAh8eSNtymfU5ASZBF2i\\\\\\\/Je5+2GDC12Nr5j0cbF6aOgkNKDDXppfwFUZhWuQTXSg3TW+SMCGD5cUZf5hfd+cexxQO009nVTjAmvkmRS2uCLysYhOWkA\\\\\\\/x2j3vI1Ggg74KBZMlplEbZ5b3ytz0yuGAbUgewAGCcrkmyJ1ghr0KkpfUG\\\\\\\/N45e6cFG60h9JAzNILohQyODRJ\\\\\\\/RgoqPkAgPnlb1TODOYQxcuznl0UE\\\\\\\/Gn28p+FDvt1pr2NewMSixbRYVgPnNU6+g7c91+zXO+S6WuNH0hAtis1fuHSgwtKKUl3GvApivR+BTmCLyaTboL8BkYu0b3a42v4UyFvNsmRplcxRld93a2c1s61XstNo1Ix\\\\\\\/TFpg4XqQEaTjFX+ZJMtwreAQZle+VGNNQjrvBHwIY0xIEgiWWyCN4rgqouC1y55aIsm92TnqoRhvGOONCvGSsgEilWBNFRhaOHjOSz3nHW780YZbuw0s8buHbw17dvn+jwIFnGSfLuzsmaklyjvLRcCmE7zgDM54hMeIRG6Y10VPF3rUHZ7ckTY29Bwcj1QOdD8OIN6hOakzFmNBOmCD+hNu8dp0+ylI4fpPRxSPGTe4i5Wcp1g+WOc5mO8PYND\\\\\\\/\\\\\\\/wGnoCemC\\\\\\\/kwqwChTtyaToLZDOkZ75cnCWMnlAVMG4twY\\\\\\\/YuXOKpjapO0TxOzYgs4UtUVwpBRnWi1Gm1eULElNVQriMYPx3UdBvWaOw15heCjWMBL8x7uqOo2\\\\\\\/rpeKr36J2UoeNmo\\\\\\\/fKPdTV1P14wtV+zFxrFXFcayVtQ5B23OgBzs8PSSvWyvJHK+mhaBsRDAjAScpewucb8vgLiTRwk6gbi95D4xir\\\\\\\/CSxEJzz4y0KeOrzDu3hxf9Y3G3aAtS5fb+OLbXLocTpTlMmono6mqDDf14NgS\\\\\\\/ZJgSxwQ3qA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022QkkgGsYMY39VxpJXy7m3+Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c71a2ae8097a0a79ae2bca8783a950fb35caf34d1488b5fb1c632f51c900d918\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459241 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/2/249" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rAssYxrJVBl0apeBuG1asay1Gk7V46KTg4A4z4V_EUo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/2/249" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VugP0SwjhPqPKbFSE5Eks_BZ4wpFw2Xp-10QuUhVYm8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/2/235" title="Extent With Modification: Leg Patterning in the Beetle Tribolium castaneum and the Evolution of Serial Homologs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/2/261" title="Whole-Genome Sequencing of Sordaria macrospora Mutants Identifies Developmental Genes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459241" id="node459241" data-pisa="ggg;2/2/249" data-pisa-master="ggg;2/2/249" data-apath="/ggg/2/2/249.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Whole-Chromosome Analysis of Meiotic Recombination in <em>Drosophila melanogaster</em></h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Danny E. Miller</span>, <span class="highwire-citation-author" data-delta="1">Satomi Takeo</span>, <span class="highwire-citation-author" data-delta="2">Kavyasree Nandanan</span>, <span class="highwire-citation-author" data-delta="3">Ariel Paulson</span>, <span class="highwire-citation-author" data-delta="4">Madelaine M. Gogol</span>, <span class="highwire-citation-author" data-delta="5">Aaron C. Noll</span>, <span class="highwire-citation-author" data-delta="6">Anoja G.  Perera</span>, <span class="highwire-citation-author" data-delta="7">Kendra N. Walton</span>, <span class="highwire-citation-author" data-delta="8">William D. Gilliland</span>, <span class="highwire-citation-author" data-delta="9">Hua Li</span>, <span class="highwire-citation-author" data-delta="10">Karen K.  Staehling</span>, <span class="highwire-citation-author" data-delta="11">Justin P. Blumenstiel</span> and <span class="highwire-citation-author" data-delta="12">R. Scott Hawley</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">249-260;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001396 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459241" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Danny E. Miller</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danny%2BE.%2BMiller" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADanny%2BE.%2BMiller" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Satomi Takeo</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Satomi%2BTakeo" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takeo%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASatomi%2BTakeo" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kavyasree Nandanan</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kavyasree%2BNandanan" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nandanan%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKavyasree%2BNandanan" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ariel Paulson</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ariel%2BPaulson" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paulson%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAriel%2BPaulson" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Madelaine M. Gogol</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Madelaine%2BM.%2BGogol" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gogol%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMadelaine%2BM.%2BGogol" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Aaron C. Noll</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BC.%2BNoll" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Noll%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAaron%2BC.%2BNoll" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Anoja G.  Perera</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anoja%2BG.%2B%2BPerera" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Perera%20AG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnoja%2BG.%2B%2BPerera" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kendra N. Walton</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kendra%2BN.%2BWalton" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Walton%20KN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKendra%2BN.%2BWalton" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">William D. Gilliland</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Biology, Depaul University, Chicago, IL 60604</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BD.%2BGilliland" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gilliland%20WD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BD.%2BGilliland" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Hua Li</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hua%2BLi" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHua%2BLi" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Karen K.  Staehling</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karen%2BK.%2B%2BStaehling" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Staehling%20KK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaren%2BK.%2B%2BStaehling" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Justin P. Blumenstiel</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, KS 66045, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BP.%2BBlumenstiel" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blumenstiel%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJustin%2BP.%2BBlumenstiel" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jblumens@ku.edu" class="" data-icon-position="" data-hide-link-title="0">jblumens@ku.edu</a></span></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">R. Scott Hawley</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span><span class="nlm-aff">Department of Physiology, Kansas University Medical Center, Kansas City, KS 66160</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BScott%2BHawley" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hawley%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR.%2BScott%2BHawley" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/2/249" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459241" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459241/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/2/249.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459241" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459241/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/2/249.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459241" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459241/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/2/249.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459241" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459241/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/2/249.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459241" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459241/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">Although traditional genetic assays have characterized the pattern of crossing over across the genome in <em>Drosophila melanogaster</em>, these assays could not precisely define the location of crossovers. Even less is known about the frequency and distribution of noncrossover gene conversion events. To assess the specific number and positions of both meiotic gene conversion and crossover events, we sequenced the genomes of male progeny from females heterozygous for 93,538 <em>X</em> chromosomal single-nucleotide and InDel polymorphisms. From the analysis of the 30 F1 hemizygous X chromosomes, we detected 15 crossover and 5 noncrossover gene conversion events. Taking into account the nonuniform distribution of polymorphism along the chromosome arm, we estimate that most oocytes experience 1 crossover event and 1.6 gene conversion events per <em>X</em> chromosome pair per meiosis. An extrapolation to the entire genome would predict approximately 5 crossover events and 8.6 conversion events per meiosis. Mean gene conversion tract lengths were estimated to be 476 base pairs, yielding a per nucleotide conversion rate of 0.86 × 10<sup>−5</sup> per meiosis. Both of these values are consistent with estimates of conversion frequency and tract length obtained from studies of <em><a href="http://flybase.org/reports/FBgn0003308.html">rosy</a></em>, the only gene for which gene conversion has been studied extensively in <em>Drosophila</em>. Motif-enrichment analysis revealed a GTGGAAA motif that was enriched near crossovers but not near gene conversions. The low-complexity and frequent occurrence of this motif may in part explain why, in contrast to mammalian systems, no meiotic crossover hotspots have been found in <em>Drosophila</em>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/crossing-over" class="hw-term hw-article-keyword hw-article-keyword-crossing-over" rel="nofollow">crossing over</a></li><li class="kwd"><a href="/keyword/gene-conversion" class="hw-term hw-article-keyword hw-article-keyword-gene-conversion" rel="nofollow">gene conversion</a></li><li class="kwd"><a href="/keyword/double-strand-break" class="hw-term hw-article-keyword hw-article-keyword-double-strand-break" rel="nofollow">double-strand break</a></li><li class="kwd"><a href="/keyword/genome-sequencing" class="hw-term hw-article-keyword hw-article-keyword-genome-sequencing" rel="nofollow">genome sequencing</a></li><li class="kwd"><a href="/keyword/meiosis" class="hw-term hw-article-keyword hw-article-keyword-meiosis" rel="nofollow">meiosis</a></li></ul><blockquote id="disp-quote-1" class="disp-quote"><p id="p-6"><em>This article is dedicated to the memory of Professor Arthur Chovnick, whose studies of the rosy locus form the foundation of our understanding of gene conversion in Drosophila</em>.</p></blockquote><p id="p-7">A hallmark of most meiotic processes is the induction of a substantial number of programmed double-strand breaks (DSBs) that result in meiotic recombination between homologs. A portion of these DSBs are repaired as crossover (CO) events that result in both gene conversion (GC) at the site of the DSB and reciprocal exchange between flanking markers. Alternatively, DSBs may also repair without crossing over, in which case GC may still occur, but the exchange of flanking markers does not (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Berchowitz and Copenhaver 2010</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bishop and Zickler 2004</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Youds and Boulton 2011</a>). Both forms of recombination are key determinants of diverse biological phenomena that include the dynamics of chromosome segregation, patterns of linkage disequilibrium (LD), and the evolutionary fate of beneficial alleles. However, very little is known about the molecular mechanisms that determine the landscape of meiotic recombination across species. In fact, very little is known about how meiotic recombination is distributed across the genome at a fine scale. In this study, we seek to provide insight into these questions from a <em>Drosophila</em> perspective by using the tools of whole-genome sequencing (WGS).</p><p id="p-8">In <em>Drosophila</em> females both the number of DSBs induced per meiosis (~21) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jang <em>et al.</em> 2003</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Klovstad <em>et al.</em> 2008</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Mehrotra and McKim 2006</a>) and the number of CO events per genome (~5−6) (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Carpenter 1975</a>) are well established. Unfortunately, although GC has been thoroughly studied at the <em><a href="http://flybase.org/reports/FBgn0003308.html">rosy</a></em> locus (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Blanton <em>et al.</em> 2005</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chovnick <em>et al.</em> 1971</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Curtis and Bender 1991</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Curtis <em>et al.</em> 1989</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hilliker <em>et al.</em> 1994</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Radford <em>et al.</em> 2007b</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Schweitzer 1935</a>), only limited data are available for either conversion or intragenic recombination at other loci (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Finnerty 1976</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Green and Green 1949</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Smith <em>et al.</em> 1970</a>), raising the possibility that genome-wide estimates of conversion rates may be inaccurate if the frequency of conversion at <em><a href="http://flybase.org/reports/FBgn0003308.html">rosy</a></em> is substantially different from the true genome average. Of equal concern is the paucity of data regarding the precise localization of CO events within the genome. Finally, although the classical analysis of crossing over using multiply marked chromosomes does not indicate the presence of significant hotspots for CO events (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hey and Kliman 2002</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lindsley and Sandler 1977</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Schweitzer 1935</a>), the resolution of such studies may be insufficient to identify such sites.</p><p id="p-9">Beyond the classic approach of using visible markers, two basic approaches can be taken to define the landscape of meiotic recombination with high resolution. The first approach leverages population genetic data and patterns of LD to infer ancestral recombination events. This approach has proven extremely powerful, especially in humans (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hinch <em>et al.</em> 2011</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">McVean <em>et al.</em> 2004</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Myers <em>et al.</em> 2005</a>, <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">2008</a>). One limitation of this method, however, is that it might be confounded by natural selection because selective sweeps can remove the ancestral signature of recombination faster than drift (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">O’Reilly <em>et al.</em> 2008</a>). In addition, such methods are unable to easily distinguish recombination arising from single COs (SCOs), double COs (DCOs), or non-CO GC events (hereon designated GC events). The ability to make this distinction is essential for understanding the mechanisms that determine the landscape of recombination.</p><p id="p-10">In contrast to the population genetic approach, a second approach to analyzing the landscape of recombination at high-resolution has been a pedigree-based approach. In the most basic form, this approach depends on genotyping parents and offspring, thus defining the landscape of recombination in a single meiosis. This approach has also proven very powerful in humans (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kong <em>et al.</em> 2002</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">2010</a>) and in yeast (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Mancera <em>et al.</em> 2008</a>). Similar methods have been used in <em>Drosophila</em> via two distinct approaches. First, Noor and colleagues, have used large numbers of single-nucleotide polymorphism (SNP) markers to detect recombination events and thus construct fine-scale recombination maps in <em>Drosophila pseudoobscura</em> and <em>Drosophila persimilis</em> (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kulathinal <em>et al.</em> 2008</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Stevison and Noor 2010</a>). Second, <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Singh <em>et al.</em> 2009</a> have used an approach that identified recombination events in <em>Drosophila melanogaster</em> within the <em>white-echinus</em> interval and then determined their precise location within this interval. However, in the first case, resolution was not sufficient to address important questions such as whether CO or GC events occur more frequently than expected in intergenic regions, or in regions marked by specific sequence elements or motifs. Similarly in the <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Singh <em>et al.</em> (2009)</a> study, it was not clear how the identified fine-scale heterogeneity observed in the interval studied would be distributed over the entire genome.</p><p id="p-11">Addressing such questions requires the ability to detect all CO and GC events along a large fraction of the genome, preferably over at least one whole chromosome arm, in the product of a single meiosis. Such an experiment is also best performed using sets of parents, all of which are heterozygous at the same sites, which is to say sibling females created by the mating of recently isogenized stocks. Provided that the two parental stocks differ by a sufficient number of DNA polymorphisms, sequencing of individual progeny should be able to precisely identify CO and GC events by WGS, with a resolution limited only by the number and distribution of those polymorphisms (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Qi <em>et al.</em> 2009</a>). Theoretically, more precise recombination estimates may be obtained from an analysis of progeny that are descendants of several generations because this allows more recombination events to be detected. However, in our case, single meiotic events were preferred because of our desire of being able to distinguish SCO, DCO, and GC events. If COs have a tendency to cluster as the result of recombination rate heterogeneity, it is extremely difficult to distinguish among these three forms of recombination after multiple rounds of meiosis.</p><p id="p-12">Despite the great progress in genome sequencing (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Metzker 2010</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Nielsen <em>et al.</em> 2011</a>), several challenges remain in taking a whole-genome approach to analyzing the distribution of recombination events from a single meiosis. First, assembling and scoring heterozygous reads is difficult and requires much deeper sequencing than required if simply scoring homozygous SNPs (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Langley <em>et al.</em> 2011</a>). Second, a sufficient amount of DNA may not be available from single F1 progeny to avoid whole-genome amplification (WGA), which may be susceptible to artifacts (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Pugh <em>et al.</em> 2008</a>). If follow-up verification of recombination events is desired, it may be preferable to use a sample of DNA that has not been subjected to WGA.</p><p id="p-13">We have circumvented these problems by focusing our attention on <em>X</em> chromosomal recombination. In <em>Drosophila</em>, the <em>X</em> chromosome carries nearly 20% of the euchromatic genome sequence and is hemizygous in males. Thus, by sequencing male F1 progeny, the difficulty of dealing with heterozygous SNPs can be avoided by focusing strictly on recombination on the <em>X</em>. Restricting the analysis to the <em>X</em> chromosome also allows putative sequence changes to be confirmed without WGA because the <em>X</em> chromosome can be preserved and “cloned” by crossing the single male progeny to attached-<em>X C(1)DX/Y</em> females, hereon designated <em><a href="http://flybase.org/reports/FBab0000080.html">C(1)DX</a></em>, and creating stocks (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001396/-/DC1/001396SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001396/-/DC1/FigureS1.pdf">Figure S1</a>). An alternative approach would be to use balancer chromosomes and an appropriate crossing scheme that would allow one to feasibly purify recombinant autosomes in an isogenized state. However, because we have previously shown that there is a detectable amount of GC between balancer and normal sequence chromosomes (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Blumenstiel <em>et al.</em> 2009</a>), we elected to avoid this approach.</p><p id="p-14">The two isogenic stocks we used as parental lines, Canton-S and <em><a href="http://flybase.org/reports/FBal0018186.html">w<sup>1118</sup></a></em>, differ by 93,538 SNPs and InDel polymorphisms on the 22.4-Mb <em>X</em> chromosome, allowing us a to detect virtually all CO events and, on the basis of statistical predictions of the SNP distribution, approximately 40% of all GC events assuming an average conversion tract length of 476 bp. Putative CO or GC events identified by WGS can be confirmed by using standard methods to resequence the relevant intervals from the preserved <em>X</em> chromosome in each of the stocks.</p><p id="p-15">Using this approach, we precisely localized 15 CO events and 5 GC events on the <em>X</em> chromosome arising from a total of 30 meiotic events. This allowed us to position CO events within relatively small regions, on average, 923 bp. We were also able to determine the maximum and minimum tract lengths of the five GC events, with the largest possible conversion tract comprising 3436 bp and the narrowest possible tract comprising 87 bp. Our data suggest that the previous genetic studies of GC at the <em><a href="http://flybase.org/reports/FBgn0003308.html">rosy</a></em> locus provide an excellent estimate of the incidence of GC throughout the genome, as well as the average tract lengths of those events (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Blanton <em>et al.</em> 2005</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Curtis and Bender 1991</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Curtis <em>et al.</em> 1989</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Hilliker <em>et al.</em> 1994</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Radford <em>et al.</em> 2007a</a>, <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">2007b</a>). Interestingly, we have found that CO spans are enriched for a simple GTGGAAA motif. The low complexity of this motif suggests a possible reason for why no strong recombination hotspots have been found in <em>Drosophila</em> since this motif is broadly distributed along the <em>X</em>.</p><p id="p-16">As read lengths increase and the cost of WGS decreases, we expect that this approach for the analysis of recombination and conversion will become a standard method for the analysis of recombination-defective mutants in <em>Drosophila</em>.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Fly genetics</h3><p id="p-17"><em><a href="http://flybase.org/reports/FBal0018186.html">w<sup>1118</sup></a></em> and Canton-S parental lines were isogenized for the <em>X</em>, second, and third chromosome using balancer stocks. <em>FM7</em> was used to balance the <em>X</em> chromosome and a <em>w; Sp/CyO; Pr Dr/TM3</em> stock was used for crosses in which it was necessary to balance the second and third chromosomes. <em>FM7</em> is a balancer chromosome that carries the markers <em><a href="http://flybase.org/reports/FBgn0004034.html">y</a></em>, <em><a href="http://flybase.org/reports/FBal0018074.html">w<sup>1</sup></a></em>, and <em><a href="http://flybase.org/reports/FBgn0000154.html">B</a></em> and completely suppresses crossing over with a normal <em>X</em> chromosome. <em>w; Sp/CyO; Pr Dr/TM3</em> is a double balancer stock where the <em>CyO</em> chromosome carriers the marker <em><a href="http://flybase.org/reports/FBgn0000403.html">Cy</a></em> and the <em>TM3</em> chromosome carries the marker Sb. The genotype of the fourth chromosome was not considered. <em><a href="http://flybase.org/reports/FBal0018186.html">w<sup>1118</sup></a></em> and Canton-S stocks isogenized for the <em>X</em>, second, and third chromosomes were then sequenced, and male <em><a href="http://flybase.org/reports/FBal0018186.html">w<sup>1118</sup></a></em> were crossed to Canton-S females to generate females for analysis of female recombination by sequencing their male progeny. The full cross is shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001396/-/DC1/FigureS1.pdf">Figure S1</a>. Before sequencing was initiated, recombinant <em>X</em> stocks were established by crossing single male progeny to four <em><a href="http://flybase.org/reports/FBab0000080.html">C(1)DX</a></em> females for 5 days before the male was removed and stored at −80°. The females were removed and discarded on the eighth day. Male progeny were collected from each stock as needed for sequencing the recombinant <em>X</em> chromosome within the stock.</p></div><div id="sec-3" class="subsection"><h3>Genome sequencing, SNP genotyping, and mapping recombination events</h3><p id="p-18">DNA was prepared from 10 adult males using standard procedures (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Blumenstiel <em>et al.</em> 2009</a>). A size of 300 bp of genomic DNA was selected after shearing to 200- to 800-bp fragments using sonication and DNA fragment libraries were made using the Paired End DNA Sample Prep Kit (cat. no. PE-102-1001) from Illumina following the manufacturer’s directions. The libraries were then sequenced using 40-bp paired-end reads on an Illumina Genome Analyzer IIx. Two lanes were run for each of the parental lines, and one lane was run for each of the 30 progeny lines. One lane of each flowcell was used for a Phi-X control. Images from the Illumina Genome Analyzer IIx were processed using the Illumina Analysis Pipeline version 1.6.0 (Casava) to generate FASTQ sequence files. Reads that passed through the Gerald chastity filter were aligned to the <em>D. melanogaster</em> genome (Release 5.22) using MAQ 0.7.1 (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Li <em>et al.</em> 2008</a>), allowing two mismatches in the first 24 bps and a maximum mismatch quality sum of 60. A consensus was then generated using Samtools pileup (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2009</a>) for each strain. Consensus sequences for the Canton-S and <em><a href="http://flybase.org/reports/FBal0018186.html">w<sup>1118</sup></a></em> isogenized strains were compared to identify differences. Local Smith-Waterman alignments were performed using GATK (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Depristo <em>et al.</em> 2011</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">McKenna <em>et al.</em> 2010</a>) in regions flanking the initially identified SNP to improve the alignment quality. To call a difference, a minimum MAQ consensus quality score of 30 and read depth of 4 were required at each chromosomal position for both strains. Recombination events were identified using custom PERL or R scripts that locate sites of putative CO over or GC. Most COs and all GCs were verified using Sanger sequencing on PCR products derived from individual males from both parental lines (Canton-S and <em><a href="http://flybase.org/reports/FBal0018186.html">w<sup>1118</sup></a></em>) and the recombinant <em>X</em> stocks.</p></div><div id="sec-4" class="subsection"><h3>Analysis of clustered recombination events</h3><p id="p-19">The likelihood of seeing two CO events only 25 kb apart was estimated by generating 10,000 sets of 15 random coordinates between 1 and 1.4 × 10<sup>7</sup> (14 Mb was used instead of 22 Mb to account for CO events occurring primarily in the distal two-thirds of the <em>X</em> chromosome), and the smallest distance between adjacent coordinates in each set was then calculated. This showed that the average smallest distance between any pair of 15 coordinates was approximately 65 kb and that the smallest distance was ≤25kb 30.9% of the time. Nonetheless, the possibility of increased recombination rate in several positions was further tested using polymerase chain reaction (PCR)-based restriction enzyme analysis. A total of 96 male progeny were created by crossing isogenized male <em><a href="http://flybase.org/reports/FBal0018186.html">w<sup>1118</sup></a></em> and female Canton-S parents as described, crossed to <em><a href="http://flybase.org/reports/FBab0000080.html">C(1)DX</a></em> females, and homogenate for PCR was prepared from the resulting male progeny as described. Three pairs of SNPs were identified that flanked the three intervals that showed more than one CO (8C1: 8,762,204–8,915,350. 11A3: 11,776,391–12,129,120. 18C3-18D1: 19,238,119–19,498,335: Release 5.22; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Tweedie <em>et al.</em> 2009</a>) and resulted in restriction sites in only one of the two parental lines. Six pairs of PCR primers were designed to create approximately 1-kb products centered on each SNP. Each of the 96 males was then scored for the presence of a recombination event in the relevant interval.</p></div><div id="sec-5" class="subsection"><h3>Statistical analysis of recombination</h3><p id="p-20">CO and non-CO GC (designated as GC) frequencies were analyzed separately for several reasons. First, they occur by mutually exclusive pathways (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Berchowitz and Copenhaver 2010</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bishop and Zickler 2004</a>; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Youds and Boulton 2011</a>). Second, in the case of CO, all recombination events are visible because of the exchange of flanking markers. In contrast, not all GCs are visible because they can occur entirely between two flanking SNP markers. Thus, a means of correcting the estimate applies solely to GCs. Finally, no CO in this experiment was associated with a discontinuous pattern of exchange, indicating that GC tracts to the left and right of a DSB are most often contiguous. Without sequences from the products of reciprocal exchange, there is no power to estimate GC tract lengths that arise during crossing over. Thus, tract length estimates were obtained using GCs only.</p><p id="p-21">For GCs, we used a maximum likelihood approach to jointly estimate frequency of GC DSBs per nucleotide per meiosis and the length of the GC tracts. In the case of GC DSBs, we modeled their position to be Poisson distributed between nucleotides with parameter λ. We modeled GC tracts based on a geometric process with parameter <em>p</em> (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Blanton <em>et al.</em> 2005</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Hilliker <em>et al.</em> 1994</a>) that occurs independently to both the left and right of the DSB. In this case, <em>p</em> represents the probability that a GC tract ceases to extend after <em>L</em> nucleotide extensions, each with probability (1 <em>− p</em>). Thus, the probability of either a leftward or rightward GC tract being exactly length <em>L</em> is given by 1)</p><span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/249/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/249/embed/mml-math-1.gif"/></noscript></span></span><p id="p-22">And the probability of a GC being in a specified range is the difference between the cumulative density function for the maximum value of this range subtracted by the cumulative density function for the minimum value of this range. This models the process of 5′ resection that occurs at the sites of DSBs and gives a mean leftward or rightward GC tract as (1 − <em>p</em>)<em>/p</em>. In the case of neighboring SNPs that were not converted, we modeled the likelihood as equal to the probability of no GC occurring in the span between the unconverted SNPs plus the probability of exactly one GC within the span, but with GC tract lengths short enough so as to not have converted the flanking SNPs. Thus, the probability of a given span with no converted flanking SNPs is 2)<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/249/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/249/embed/mml-math-2.gif"/></noscript></span></span>where <em>i</em> is the distance between flanking SNPs (<em>i.e.</em> the number of possible DSB locations) for the single DSB and <em>k</em> indicates the ordinates for positions that a DSB may occur within the span.</p><p id="p-23">In the case of SNPs that experience GC, the likelihood was modeled such that a single DSB could have occurred anywhere between the two outermost unconverted SNPs. Within this span, there may be either one or multiple converted SNPs. Depending on the possible location of the DSB, to the left and right the probability of observing a particular GC conformation was determined based on the maximum possible GC (bounded by unconverted SNPs) and the minimum possible GC (bounded by converted SNPs). Assuming the probability of two DSBs within the span is negligible, these probabilities were summed over all positions within the span 3)</p><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/249/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/249/embed/mml-math-3.gif"/></noscript></span></span><p id="p-24">where:</p><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/249/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/249/embed/mml-math-4.gif"/></noscript></span></span><p id="p-25">Given this formulation, there is a unique probability for each span between two unconverted SNPs of distance <em>i</em> and a unique probability for each observed GC. The full likelihood of the data are this given by 4)</p><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/249/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/249/embed/mml-math-5.gif"/></noscript></span></span><p id="p-26">Where <em>N<sub>a</sub></em> is the cumulative number of spans of length <em>a</em> that were witness to no GCs across all 30 meioses and <em>b</em> is the upper limit for span lengths used in the analysis. In this case, we set <em>b</em> to 10,000, meaning we did not include spans between SNPs greater than 10,000. Such spans yield very little information on the GC process and were thus excluded. In the second part of the likelihood function, <em>c</em> is the index for the GC tracts and in this case <em>d</em> is five. Taking the natural log of this gives 5)</p><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/249/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/2/249/embed/mml-math-6.gif"/></noscript></span></span><p id="p-27">Using this function, we jointly determined the values of <em>p</em> and λ that maximized the likelihood of the data. All analysis was performed in <em>Mathematica</em> (Wolfram Research). Then, 95% confidence intervals (95% CIs) for estimates were determined on the basis of determining parameter values that gave likelihood scores ± two ln-likelihood units while keeping the second parameter fixed for the maximum likelihood estimate. Tests of significance against previous estimates were performed using the likelihood ratio test and the χ<sup>2</sup> distribution with degrees of freedom determined based on the number of additional parameters allowed to freely vary.</p></div><div id="sec-6" class="subsection"><h3>CO motifs</h3><p id="p-28">We used MEME (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bailey and Elkan 1994</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">1995a</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bailey <em>et al.</em> 2006</a>) to identify candidate motifs in the 14 CO spans (defined as the region between flanking SNPs) excluding the CO region from 2e, whose location could not be precisely determined. MEME was run with the following considerations: first, the spans vary widely in length, and we wanted to give more weight to motifs discovered in small spans than longer ones because the signal-to-noise ratio should be greater; second, we did not want to require MEME to find each motif in all sequences; third, because of the small number of sequences and MEME’s sensitivity to initial conditions, we wanted to ensure any candidate signal was found robustly.</p><p id="p-29">To address these considerations we ran MEME with the “zoops” model (allowing zero-or-one motif instances per sequence) on 13 combinations of spans, starting with only the smallest 2 spans, then the smallest 3, working up to all 14 spans, and then looked for motifs found consistently in all runs. Motif width was capped at 20 bp and selected the top twenty motifs from each MEME run.</p><p id="p-30">After motif discovery, we used FIMO (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Grant <em>et al.</em> 2011</a>) to search for motif instances in the input sequences. FIMO matches are biased toward longer motifs, which tend to have lower <em>P</em>-values. To partially address this we required that passing matches have (<em>P</em>-value × match length) ≥ 0.001.</p><p id="p-31">To make an objective decision as to which motifs in which runs were equivalent, we used Tomtom (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gupta <em>et al.</em> 2007</a>), which makes pairwise ungapped alignments of motifs and returns a <em>P</em>-value. With 13 runs at 20 motifs each, we aligned all 260 motifs to each other and thresholded the resulting connectivity matrix with the FIMO criterion of (<em>P</em>-value × alignment length) ≥ 0.001. After thresholding, all subgraphs were extracted from the connectivity matrix, and these became “metamotifs.” Each metamotif represents a pool of motifs that can be considered equivalent at the given level of stringency. As a test of significance for metamotifs, the same process was repeated on 100 sets of 14 randomly selected regions of chromosome X with the same size distribution as the CO spans. Relative enrichment for core motifs that contributed to a metamotif was performed by examining the distribution of core motifs within the 14 CO spans relative to the 100 random sets of 14 random sequence sets by comparing Observed/Expected ratios estimated based on nucleotide composition. Finally, local enrichment was examined because a motif that facilitates the formation of CO events might be significantly enriched in CO spans relative to the immediate flanking region and show a decrease in excess enrichment further from the CO site. We tested all 260 motifs for such enrichment using the Fisher’s exact test for CO spans <em>vs.</em> three background sets: 1 kb, 5 kb, and 10 kb on either side of the span.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><p id="p-32">Our experiment set out to precisely identify and quantify locations of CO and GC events in a single <em>Drosophila melanogaster</em> female meiosis. We approached this by first isogenizing two common lab strains, Canton-S and <em><a href="http://flybase.org/reports/FBal0018186.html">w<sup>1118</sup></a></em>, and sequencing them to determine the number of either SNPs or InDels that exist between the two lines. We then crossed the isogenized stocks to one another, allowed the females to undergo meiosis, and crossed resultant males to <em><a href="http://flybase.org/reports/FBab0000080.html">C(1)DX</a></em> females. This preserved a nonrecombining clone of the original <em>X</em> in stock (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001396/-/DC1/FigureS1.pdf">Figure S1</a> for cross scheme). This allowed us to maintain the isolated <em>X</em> chromosomes without recombination.</p><p id="p-33">Illumina libraries were made from genomic DNA isolated from 10 males from either isogenized Canton-S and <em><a href="http://flybase.org/reports/FBal0018186.html">w<sup>1118</sup></a></em> stocks (parental lines) or from the resultant progeny of crosses to <em><a href="http://flybase.org/reports/FBab0000080.html">C(1)DX</a></em> females (progeny lines). The parental libraries were run on two lanes, and the progeny libraries were run on a single lane of a flow cell on an Illumina Genome Analyzer IIx using 40 bp paired-end sequencing. Using this method, we obtained <em>X</em> chromosome coverage of 99% and an average read depth of 18.1x for the parental lines and an average coverage of 93% and an average read depth of 5.6x for the progeny lines (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001396/-/DC1/TableS1.pdf">Table S1</a>).</p><p id="p-34">To identify SNPs and InDels, we independently compared each parental sequence to the reference <em>Drosophila</em> sequence (version 5.22). This generated a list of polymorphisms for each line that had a parental coverage of at least 4x and a consensus quality score of at least 30. We then compared the Canton-S and <em><a href="http://flybase.org/reports/FBal0018186.html">w<sup>1118</sup></a></em> sequences to each other, identifying polymorphisms that distinguished the two lines. This gave us 79,045 SNPs and 14,483 InDels for the two parental lines. Five more InDel polymorphisms were identified during Sanger verification of the intervals in which CO events occurred, resulting in a total of 93,528 total polymorphisms.</p><p id="p-35">Using tools developed in-house, we performed SNP calling to identify haplotype identity along the hemizygous <em>X</em> chromosome. As with the parental lines, genotypes were called only at nucleotide positions with a depth of at least 4x and a MAQ consensus quality score of 30. Since male progeny were sequenced with less depth, approximately 70% of parental variants were called with this threshold in each of the progeny lines (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001396/-/DC1/TableS1.pdf">Table S1</a>). The approximately 30% remaining were not called with this stringency and excluded from analysis.</p><p id="p-36">Initial analysis of 30 male progeny from four females identified 15 CO events and 28 putative GC events. All 15 CO events were verified with Sanger sequencing. Of 28 candidate GC events, verification using Sanger sequencing confirmed only five events, with 23 being false-positive results (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001396/-/DC1/TableS3.pdf">Table S3</a> and<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001396/-/DC1/TableS4.pdf"> Table S4</a>). Closer analysis revealed that the 82% false-positive rate for GC events was largely the result of poor sequence alignment or InDels that were not identified during the variant calling in the parental lines.</p><div id="sec-8" class="subsection"><h3>Location and positioning of COs</h3><p id="p-37">Fifteen COs were detected in 12 of the 30 progeny analyzed (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Nine chromosomes carried a single CO event, and three carried a DCO. None of the CO events displayed discontinuous tracts of GC. The presence of a CO event had no strong influence on the likelihood that a GC would be observed elsewhere on the chromosome arm (Fisher’s exact test, <em>P</em> = 0.63). To precisely define CO positions, primers were designed flanking each CO interval including the next neighboring polymorphism not defined by the CO event (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001396/-/DC1/TableS2.pdf">Table S2</a>). Verification of each CO event allowed us to narrow 14 of the events down to spans with an average length of 923 bp. The CO event from stock 2e at 9D3 was not precisely localized because the flanking SNPs identified in the parental lines were 18 kb apart. Verification of COs did not result in the discovery of new SNPs but did identify five new InDels not identified from the parental analysis. Many times these were poly-A regions or repeat-rich regions in which it is difficult to identify InDels based on the original WGS data. The five InDels found by Sanger sequencing that narrowed the boundary for a CO event were added to the total number of InDels found by whole genome sequencing and were included in the statistical analysis.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461291/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461291/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">CO span detail</span><div class="sb-div caption-clear"></div></div></div><p id="p-40">All 9 SCO chromatids carried CO events that were distributed over the distal two-thirds of the <em>X</em> chromosome (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Indeed, seven SCOs were located in the distal-most half of the <em>X</em> chromosome. The average pairwise distance between these single COs was 4.8 Mb. The three observed DCO chromatids each carry two single CO events separated by an average distance of 8.6 Mb. The greater distance between CO events on the DCO chromatids indicates the presence of CO interference. Overall, the distribution of COs along the arm of the <em>X</em> was as expected by previous genetic studies (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Lindsley and Sandler 1977</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/249/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Parental haplotype structure and the distribution of COs. Of 30 chromosomes recovered, nine experienced one CO and three experience two COs. Pairs of colored stars indicate DCO positions. For reference, the per Mb recombination rate is provided (Fiston-Lavier et al. 2010)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1485134940" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Parental haplotype structure and the distribution of COs. Of 30 chromosomes recovered, nine experienced one CO and three experience two COs. Pairs of colored stars indicate DCO positions. For reference, the per Mb recombination rate is provided (Fiston-Lavier &lt;em&gt;et al.&lt;/em&gt; 2010).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/249/F1.medium.gif" width="440" height="299"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/2/2/249/F1.medium.gif" width="440" height="299"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/249/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/249/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461297" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-41" class="first-child">Parental haplotype structure and the distribution of COs. Of 30 chromosomes recovered, nine experienced one CO and three experience two COs. Pairs of colored stars indicate DCO positions. For reference, the per Mb recombination rate is provided (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Fiston-Lavier <em>et al.</em> 2010</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">Of the 14 CO regions placed within a span of flanking SNPs, eight of these spanned entirely intronic sequence, five entirely spanned intergenic sequence, and one spanned a mixture of exonic, intronic, and intergenic sequence (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The total sum of these CO spans comprised 12,925 bp, of which 426 bp was exonic sequence, yielding the proportion of CO spans comprised of exonic sequence equal to 3.3%. This result is in contrast to the entire <em>X</em> chromosome, which comprises 23.7% exonic sequence. To test for significance, we generated 1000 random datasets with an equal size distribution of spans and estimated the proportion of exonic sequence within each of the random sets. The average percent of exonic sequence in these trials was 23.4%, and only 15 trials produced a percent exonic sequence less than or equal to 3.3%, yielding an empirical <em>P</em>-value of 0.015. These results may indicate that CO events in <em>Drosophila melanogaster</em> have a tendency to avoid exonic regions. This is a conservative conclusion, in light of the fact that intron length is lower in regions with a greater rate of crossing over (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Carvalho and Clark 1999</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Comeron and Kreitman 2000</a>) and one might therefore expect COs to be more likely to contain exonic sequence than at random.</p><p id="p-43">Interestingly, two independent CO events from different mothers occurred within the same intron of the same gene, <em><a href="http://flybase.org/reports/FBgn0261549.html">rdgA</a></em> (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). The COs were minimally 28,411 kb and maximally 28,964 kb apart and occurred in the third intron of the most common transcript of the <em><a href="http://flybase.org/reports/FBgn0261549.html">rdgA</a></em> gene, which is approximately 94 kb long.</p></div><div id="sec-9" class="subsection"><h3>CO events are not significantly clustered in 8C1, 11A3, and 18C/D</h3><p id="p-44">Although our sample of 15 COs was small, we nonetheless found three regions (8C1, 11A3, and 18C3-18D1) in which two independent CO events were detected within the same small cytological interval. In two of the three locations, the COs were in the same cytological band (8C1 and 11A3) and were approximately 28.5 kb apart, and in the third case (18C3/18D1), the COs were approximately 158 kb apart (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Although it may seem unusual to observe events only 28.5 kb apart on a 22.4-Mb chromosome, a numerical simulation (see <em>Materials and Methods</em>) showed that given a set of 15 events, observing the closest pair of two CO events occurring within a distance of 25 kb was to be expected more than 30% of the time.</p><p id="p-45">To further investigate the possibility that the frequency of meiotic recombination might be elevated in regions 8C1, 11A3, and 18C3-18D1, we generated an additional 96 recombinant chromosomes by repeating the cross described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001396/-/DC1/FigureS1.pdf">Figure S1</a>. These 96 chromosomes were then assayed for recombination within the three intervals by restriction digest genotyping of SNPs flanking each interval. Only one recombination event, between 10F3 and 11A7, was observed within the 257 tested intervals, a result that is inconsistent with substantially elevated recombination rates in these regions. We conclude that the observation of three relatively close pairs of events in our initial sample of 15 was most likely the result of chance, but note that this experiment had low power to detect recombination rate heterogeneity.</p></div><div id="sec-10" class="subsection"><h3>CO motif analysis</h3><p id="p-46">To identify potential DNA sequence motifs associated with sites of crossing over, we used the MEME software package (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bailey and Elkan 1995a</a>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">1995b</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bailey <em>et al.</em> 2006</a>). We restricted our analysis to CO spans because potential mechanistic differences between CO and non-CO events could potentially add noise to an aggregate discovery approach. Our analysis also excluded the one 18-kb minimal CO span from male stock 2e, as the signal-to-noise ratio would be substantially increased by its inclusion. This left a sample of 14 CO spans to search for motifs. Because MEME results can be somewhat sensitive to modest variation in nucleotide composition among different domains, we used an iterative procedure that varied the number of regions analyzed per run. Specifically, we performed 13 different MEME runs, the first of which contained the two smallest (and potentially most informative) CO spans, and then added the next largest CO span to each subsequent run.</p><p id="p-47">Each of these 13 MEME runs identified the top 20 motifs that were more common than expected on the basis of nucleotide composition. To identify shared features across motifs, a metamotif analysis was performed by aligning the best 20 motifs discovered from each MEME run. Three metamotifs with specific features were identified. The metamotif with the largest number of member motifs identified (MM1, not shown), comprised several poly-A stretches and was found in eight of 13 MEME runs. Such low complexity homo-polymers are statistically unexpected on the basis of single-base nucleotide composition but are obviously quite common in the genome.</p><p id="p-48">The second largest metamotif (MM2) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2a</a>) contained several variants on a core GTGGAAA sequence and was identified in 12 of 13 MEME runs (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2b</a>). Finding a sequence of this length is unexpected and the fact that 12 runs identified this common core is consistent with this motif being biologically relevant. The third largest metamotif (MM3, not shown) was only recovered from seven of 13 MEME runs; thus, it did not represent a sufficiently strong signal; smaller metamotifs were even less well represented. Therefore, we concluded that MM2 appeared to be the most promising candidate for a CO signal. No similarity to known transcription factor binding sites was identified for this motif in either the TRANSFAC (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Matys <em>et al.</em> 2006</a>) or JASPAR (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Portales-Casamar <em>et al.</em> 2010</a>) databases.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/249/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="CO motif analysis. (A) Metamotif2 (MM2) summary logo. Multiple alignment of the MM2 member motifs, one each from 12/13 MEME runs, derived from pairwise alignments by Tomtom. (B) Similar member motifs contributing to MM2, identified from 12/13 different MEME runs, using iterative addition of CO regions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1485134940" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;CO motif analysis. (A) Metamotif2 (MM2) summary logo. Multiple alignment of the MM2 member motifs, one each from 12/13 MEME runs, derived from pairwise alignments by Tomtom. (B) Similar member motifs contributing to MM2, identified from 12/13 different MEME runs, using iterative addition of CO regions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/249/F2.medium.gif" width="121" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/2/2/249/F2.medium.gif" width="121" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/249/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/249/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461206" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-49" class="first-child">CO motif analysis. (A) Metamotif2 (MM2) summary logo. Multiple alignment of the MM2 member motifs, one each from 12/13 MEME runs, derived from pairwise alignments by Tomtom. (B) Similar member motifs contributing to MM2, identified from 12/13 different MEME runs, using iterative addition of CO regions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">To further assess the significance of MM2, several statistical analyses were performed. First we determined whether the core sequences that comprise the motif were enriched in CO spans relative to the entire chromosome. Using just the core GTGGAAA sequence, we found that this sequence is enriched in CO regions by a factor of 5.9 relative to the rest of chromosome <em>X</em> (P-value <em>vs.</em> 100 random sets: 5.23E-03). If the CO signal is genuine, then, we might also assume that the core GTGGAAA sequence is significantly enriched in the CO span <em>vs.</em> the immediate flanking region and that the enrichment will decrease as the flanking regions widen, as the motif abundance begins to equalize. We tested motif enrichment in CO spans <em>vs.</em> 1 kb, 5 kb, and 10 kb left and right flanking sequences using Fisher’s exact test. As not to be biased toward members of MM2, we tested all 260 CO motifs in this manner.</p><p id="p-51">For enrichments relative to 1-kb flanks, only 11 of 260 motifs were significant after Benjamini-Hochberg correction (adjusted <em>P</em> ≤ 0.05): all were overenriched, and none was significant at 5 kb or 10 kb (although one had adjusted <em>P</em> = 0.057 at 5 kb). Three of these motifs were from MM2, the largest representation for any metamotif; no other significant metamotifs had enriched members. At adjusted <em>P</em> &lt; = 0.01, one-half of the significant motifs were contributed by these three motifs that contributed to MM2. We thus conclude that this sequence is enriched in CO regions relative to 1-kb flanking regions, suggesting that local domains of enrichment can facilitate local positioning of CO events.</p><p id="p-52">Second, to estimate the likelihood of finding a motif of equal complexity to MM2 by chance, we generated 100 sets of 14 randomly selected <em>X</em> chromosome sequences as long as the original 14 CO regions and repeated our iterative motif search analysis. If MM2 is a false-positive result, then repeating our MEME analysis on random sequences would be expected to identify similarly complex motifs quite often. Of these 100 simulated sets of 14 regions, only two sets yielded a metamotif that showed up in at least 12 of 13 MEME runs and also had as strong an e-value and information score. This yields an empirically derived <em>P</em>-value of 0.02 for MM2. Finally, given the previous observation that COs were enriched in intronic and intergenic regions, we examined the distribution of the three core components of MM2 (GTGGAAA, GTGCAAA, and ATGGAAA) in these regions. Intronic and intergenic regions comprise 76% of the X chromosome and contain 86% of these motifs located on the X. Thus, MM2 is enriched in intronic and intergenic regions on the X.</p></div><div id="sec-11" class="subsection"><h3>Analysis of GC</h3><p id="p-53">We observed five GC events (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). GC events appear to be more evenly distributed along the entire euchromatic arm of the <em>X</em> chromosome than are CO events, although a lack of statistical power arising from only recovering five GC events precludes a rigorous test of this hypothesis. That said, the distribution of GC events does appear to be different from the distribution of CO events, all of which occurred over the distal two-thirds of the <em>X</em> chromosome.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/249/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Parental haplotype structure and the distribution of non-CO GC events. Five events were identified in four progeny." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1485134940" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Parental haplotype structure and the distribution of non-CO GC events. Five events were identified in four progeny.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/249/F3.medium.gif" width="440" height="166"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/2/2/249/F3.medium.gif" width="440" height="166"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/249/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/249/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461275" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-54" class="first-child">Parental haplotype structure and the distribution of non-CO GC events. Five events were identified in four progeny.</p><div class="sb-div caption-clear"></div></div></div><p id="p-55">We defined GC tracts on the basis of their maximum possible size—the entire span between nonconverted SNPs. Of the five GC tracts placed within a span of flanking SNPs, one was entirely intronic, two entirely intergenic, and two contained an intron and exon. In aggregate, these tracts comprised 8151 bp of sequence, of which 1845 bp were exonic. This yields a total proportion of exonic sequence in GC regions being equal to 22.6% (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) . This finding is similar to the proportion of the <em>X</em> chromosome comprised of exonic sequence (23.7%) and stands in contrast to CO spans, for which only one of 14 regions contained an exon, comprising 3.3% of total sequence. To test the significance, we generated 1000 random data sets equivalent to the distribution of localized GC tracts and determined the frequency for which the proportion of exonic sequence was equal to or less than 3.3%. This experiment yielded a <em>P</em>-value of 0.096, significant only at the 0.1 level. Thus, it appears that GCs are uniformly distributed across exons, but with a sample size of five GC events, we lack power in our test for a difference in the distribution of GC and CO events.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461236/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461236/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">GC tract detail</span><div class="sb-div caption-clear"></div></div></div><p id="p-58">Although all COs are visible by this sequencing method because of the easy detection of exchange of flanking markers, GCs are invisible if they occur entirely between SNP markers. Therefore, we developed a maximum likelihood method to jointly estimate the GC frequency and tract length (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The basic principle of this method is to estimate these two parameters considering the five visible GC events, as well as the distribution of distances between SNPs for which no GC event was observed. From this analysis, the joint ML estimate for the rate of GC event occurrence was 1.8 × 10<sup>−8</sup> per bp (95% CI 0.6 × 10<sup>−8</sup> to 3.9 × 10<sup>−8</sup>) and average one-sided tract length was 238 bp (95% CI 117−543) This yields a total mean GC tract length of 476 bp, which is similar to other estimates of mean tract length estimated from the <em><a href="http://flybase.org/reports/FBgn0003308.html">rosy</a></em> locus, ranging from 352 bp (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Hilliker <em>et al.</em> 1994</a>) to 441 bp (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Blanton <em>et al.</em> 2005</a>). Considering the rate of GC event occurrence and mean tract length, our data yield a per bp GC rate of 0.86 × 10<sup>−5</sup>, which is similar to previous estimates of 1.3 × 10<sup>−5</sup> based on genetic analysis of the <em><a href="http://flybase.org/reports/FBgn0003308.html">rosy</a></em> locus (<a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Blanton <em>et al.</em> 2005</a>). As the assembled <em>X</em> chromosome scaffold is 22.4 × 10<sup>6</sup> bp, the per-chromosome arm rate of GC event occurrence is 0.40. This rate would predict 12 expected GC events across 30 <em>X</em> chromosomes. Because only five GC events were directly observed among thirty F1 progeny (16%), this finding indicates that more than one-half of the GC events were likely missed because of the fact that the conversion tract did not include a genotyped SNP marker.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/2/249/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ln-likelihood surface for joint estimation of the rate of DSB formation for non-CO GC events and the GC tract length. Non-CO GC events are the events that arise from DSBs designated as non-CO GC events. The rate of these events is given per bp per meiosis. This differs from the GC rate, which is the rate at which nucleotides become converted. The GC rate is the multiple of the non-CO GC DSB rate and the GC tract length." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1485134940" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Ln-likelihood surface for joint estimation of the rate of DSB formation for non-CO GC events and the GC tract length. Non-CO GC events are the events that arise from DSBs designated as non-CO GC events. The rate of these events is given per bp per meiosis. This differs from the GC rate, which is the rate at which nucleotides become converted. The GC rate is the multiple of the non-CO GC DSB rate and the GC tract length.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/2/249/F4.medium.gif" width="440" height="414"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/2/2/249/F4.medium.gif" width="440" height="414"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/2/249/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/2/249/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461279" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-59" class="first-child">Ln-likelihood surface for joint estimation of the rate of DSB formation for non-CO GC events and the GC tract length. Non-CO GC events are the events that arise from DSBs designated as non-CO GC events. The rate of these events is given per bp per meiosis. This differs from the GC rate, which is the rate at which nucleotides become converted. The GC rate is the multiple of the non-CO GC DSB rate and the GC tract length.</p><div class="sb-div caption-clear"></div></div></div><p id="p-60">By applying the 1.8 × 10<sup>−8</sup> GC events per bp rate to the entire genome (119 Mb, excluding the fourth chromosome and heterochromatin), the genome wide rate of euchromatic GC is estimated to be 2.14 GC events per haploid meiotic product. Because meiosis results in four haploid products, this implies 8.6 GC events per meiosis. Considering that 5 COs are estimated to occur per meiosis (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Carpenter 1975</a>), this yields a GC:CO ratio of 1.7:1. This assumes GC is an entirely asymmetrical process in which only the chromatid that has experienced the DSB receives the converted genetic information.</p><p id="p-61">Overall, we conclude there are approximately 13.6 euchromatic recombination events per meiosis. Previous studies have found there are approximately 21 DSBs per meiosis in <em>D. melanogaster</em>, and these appear to be outside of the heterochromatin (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Jang <em>et al.</em> 2003</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Klovstad <em>et al.</em> 2008</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Mehrotra and McKim 2006</a>). If heteroduplex repair always results in GC, our estimate of euchromatic DSBs is therefore 13.6. To test whether this is significantly different than the number of DSBs measured previously by cytology, we performed a likelihood-ratio test to conservatively determine whether our estimate would be inconsistent with a model of 20 DSBs which are repaired as either CO or GC events. We are unable to reject 20 recombination events (<em>P</em> = 0.15). Moreover, if one-half of the formed heteroduplex structures are repaired without GC, our estimate for non-CO DSBs would be twice our 8.6 estimate. In total, this would yield 22 DSBs, quite similar to previous estimates. Considering a low estimate of 13.5 recombination events per meiosis, these results are consistent with a modest number of meiotic DSBs in <em>D. melanogaster</em> being repaired by other mechanisms, such as sister chromatid events (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Goldfarb and Lichten 2010</a>), but it is not conclusive.</p></div><div id="sec-12" class="subsection"><h3>GC motif analysis</h3><p id="p-62">With only five GC events, we had little power to search specifically for motifs enriched in GC tracts. Instead, we examined the distribution of previously described CO motifs in the GC tracts. Focusing on the 260 CO motifs identified from the 13 MEME runs, we tested for enrichment in the GC tracts. Relative to 1-kb flanks, only three were significant, with <em>P</em>-values uncorrected for multiple tests and none after correction. Many were significant <em>vs.</em> 10-kb flanks and without exception these were all underenriched, indicating the uniqueness of the CO <em>vs.</em> GC regions.</p><p id="p-63">In addition, we sought to compare the representation of the three strongest contributors to MM2, GTGGAAA, GTGCAAA, and ATGGAAA, between CO and GC regions. In CO spans, summing to a total of 12,925 bp, these occurred 10, eight, and seven times respectively. In GC tracts, summing to a total of 8,151 bp, these occurred two, two, and three times respectively. By a Fisher’s exact test considering all possible nonoverlapping, seven bp positions, <em>P</em> = 0.038. This result indicates that the core motifs that contribute to MM2 are significantly more enriched in CO spans relative to GC tracts.</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-64">Meiotic recombination plays a key role in two fundamental aspects of biology—maintaining proper chromosome segregation in the production of gametes and allowing the evolutionary fate of alleles to become unhindered by linkage. Although many advances have been made in dissecting the machinery of meiotic recombination, very little is known about how the landscape of meiotic recombination is determined across the genome. Across all eukaryotes, the formation of DSBs is a key initiating factor of recombination (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Dernburg <em>et al.</em> 1998</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Grelon <em>et al.</em> 2001</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">McKim and Hayashi-Hagihara 1998</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Sun <em>et al.</em> 1989</a>). Thus, the rate and landscape of meiotic recombination fundamentally depends on DSB formation, but how DSBs are established is poorly understood. How the fate of these DSBs is determined—either through crossing over or through other forms of repair—is also poorly understood. This “fate decision” is crucial in gamete formation because crossing over plays an important role in proper chromosome segregation. From an evolutionary perspective, this fate also plays a key role in how patterns of LD are determined across the genome since GC breaks up LD only across shorter distances. A key to understanding the factors that determine the landscape of recombination is a determination of the chromosome-wide distribution of recombination events at the greatest resolution possible. New sequencing technologies now make this possible.</p><p id="p-65">Using a WGS approach, we have obtained the first high-resolution view of the recombination landscape across a chromosome in <em>Drosophila</em>. This approach allows a level of analysis of recombination that has not been previously available. In particular, it allows one to jointly estimate the overall rate of CO and GC as well as determine the precise location and form of recombination events without restricting analysis to a single locus. Comfortingly, this study confirms nearly a century’s worth of <em>Drosophila</em> genetics using an entirely different approach. In particular, our estimates for rate and distribution of crossing over are very similar to those using standard approaches. Likewise, using a statistical approach that jointly estimates GC frequency and GC tract length allows us to reconcile cytological studies of DSB formation with genetic studies of GC at the <em><a href="http://flybase.org/reports/FBgn0003308.html">rosy</a></em> locus. In particular, our chromosome-wide estimate of GC rate is close to that estimated from the <em><a href="http://flybase.org/reports/FBgn0003308.html">rosy</a></em> locus. Considering both COs and GCs, our lower estimate of DSBs that become repaired through recombination (13.6) is similar to, but less than, previous estimates of total meiotic DSBs (~21). This finding supports the observation that other mechanisms, such as sister chromatid repair as recently seen in yeast (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Goldfarb and Lichten 2010</a>), may also contribute to meiotic DSB repair. Further studies in a genetic background with more precisely known meiotic DSB numbers will be necessary to formally test this hypothesis.</p><p id="p-66">Aside from rate estimates for CO and GC that are consistent with many previous studies, we also find that the structure of CO and GC events is similar to that found in previous studies. In particular, we find no evidence for discontinuous tracts of GC for either CO or NCO GC. This is finding consistent with studies of the <em><a href="http://flybase.org/reports/FBgn0003308.html">rosy</a></em> locus in wild-type flies that show the great majority of GC tracts are continuous (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Carpenter 1982</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Curtis and Bender 1991</a>; <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Curtis <em>et al.</em> 1989</a>; <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Radford <em>et al.</em> 2007a</a>, <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">2007b</a>). Moreover, our estimate for GC tract length—a total mean GC tract length of 476 bp—is similar to other estimates of mean tract length estimated from the <em><a href="http://flybase.org/reports/FBgn0003308.html">rosy</a></em> locus, ranging from 352 bp (<a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Hilliker <em>et al.</em> 1994</a>) to 441 bp (<a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Blanton <em>et al.</em> 2005</a>). Overall, this analysis suggests that the <em><a href="http://flybase.org/reports/FBgn0003308.html">rosy</a></em> locus serves as an excellent model for studying the mechanisms of recombination.</p><p id="p-67">Although these results have largely confirmed previous studies, our ability to precisely define the location of recombination events across the <em>X</em> has provided novel insights. This has particular significance in explaining the mechanisms that determine fine-scale patterns of heterogeneity in the recombination rate. One striking finding is that domains of crossing over tend to avoid exonic regions. This may be mediated by chromatin marks that are enriched on exons in a manner similar to that observed in other species (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Dhami <em>et al.</em> 2010</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kolasinska-Zwierz <em>et al.</em> 2009</a>). It is also reminiscent of the observation in humans that recombination preferentially occurs outside of genes and exons (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Kong <em>et al.</em> 2010</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">McVean <em>et al.</em> 2004</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Myers <em>et al.</em> 2005</a>). In addition, we have identified a short sequence motif (GTGGAAA) that is enriched in CO spans. This differs from previously identified motifs that correlate with overall recombination rate in <em>D. pseudoobscura</em> (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Kulathinal <em>et al.</em> 2008</a>) or <em>D. persimilis</em> (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Stevison and Noor 2010</a>) <em>i.e.</em> CCCCACCCC, CCTCCT, CACAC, ATAAA, and AATAA. It also differs from the greater complexity motifs identified in mammalian systems but not found in our analysis, such as CCNCCNTNNCCNC in humans, that are associated with CO hotspots (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Myers <em>et al.</em> 2008</a>) and are also a predictor of recombination rate in <em>D. persimilis</em> (<a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Stevison and Noor 2010</a>)</p><p id="p-68">At first glance, it might not be clear why our motif results would appear different from those found in <em>D. pseudoobscura</em> and <em>D. persimilis</em>. However, the experiments performed in <em>D. pseudoobscura</em> and <em>D. persimilis</em> were different; substantially more meiotic events were analyzed (&gt;1000) but with fewer markers distributed along the chromosome arm. Thus, those studies identified a substantially greater number of recombination events at lower resolution, whereas this study examined substantially fewer CO sites at greater precision. By investigating a large number of meiotic events, <a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">Stevison and Noor (2010)</a> were able to examine broad correlations between overall recombination rate and overall genome content whereas we were unable to do so. Instead, we examined the distribution of sequences within CO and GC regions that were precisely localized. In addition, Stevison and Noor examined <em>Drosophila</em> species that have diverged substantially from <em>D. melanogaster</em>. Given the differences in the sequence composition of recombinational hotspots already present among humans and chimpanzees (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Hinch <em>et al.</em> 2011</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Myers <em>et al.</em> 2010</a>), different recombination motifs may have evolved in these two species compared with <em>D. melanogaster</em>.</p><p id="p-69">Despite finding only five GC events, their precise localization may provide at least some insight into the mechanisms of how DSBs are destined to be repaired as either COs or NCOs. In particular, the two factors that we found significant determinants of the precise localization of CO events—the avoidance of exons and the enrichment of MM2—do not appear to apply to GC events (with the caveat that there is not a substantial amount of statistical power in our sample). The proportion of GC tracts composed of exonic sequence is nearly the same as the background exonic composition of the <em>X</em>, albeit their enrichment is significantly different from CO regions only at the 0.1 level. We also find none of the motifs that are enriched in CO spans are enriched in GC tracts relative to flanking regions and there is greater enrichment of the three core motifs that comprise MM2 in CO spans relative to GC tracts.</p><p id="p-70">There are currently two models for how the CO/NCO choice is made among DSBs. In one model, after the occurrence of a DSB, a double Holliday Junction (dHJ) forms, and CO/NCO choice is determined on the basis of the resolution of the dHJ (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Szostak <em>et al.</em> 1983</a>). A more recent and well-supported model, however, suggests the CO/NCO designation is achieved before the formation of dHJ, and the NCOs occur through DSB repair without dHJ formation (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Berchowitz and Copenhaver 2010</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Bishop and Zickler 2004</a>; <a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Youds and Boulton 2011</a>). According to each of these models, CO/NCO choice occurs downstream of DSB formation. In light of these two models, these results suggest that these properties of the DNA at the site of the DSB influence CO/NCO designation, rather than the formation of a DSB <em>per se</em>. If our identified sequence features solely mediated DSB formation, we would expect to see them in CO and GC regions alike. Thus, subsequent to DSB formation, the presence of MM2 and the absence of exonic sequence may encourage CO formation. An alternative model, in which CO/NCO designation is made during or before the formation of DSB, is also possible. In this case, the presence of the MM2 motif and the absence of exonic sequence would facilitate the formation of DSBs that are destined for CO and DSBs that occur independent of these sequence characteristics would be more likely to result in NCOs. In this case, the absence of <em>Drosophila</em> CO hotpots might be explained by the fact that both nonexonic sequence and the simple MM2 motif are both broadly distributed across the <em>X</em> chromosome and thus not sufficiently localized to drive DSB formation in a manner that would have by now been detected. Further high-resolution studies will be required to test these models and also include the influence of interference.</p><p id="p-71">Overall, we have demonstrated that sequencing progeny from a single round of meiosis by WGS will prove to be a powerful method in dissecting the mechanisms of meiotic recombination. In our study, we focused only on the <em>X</em> chromosome for several reasons. By using a <em><a href="http://flybase.org/reports/FBab0000080.html">C(1)DX</a></em> stock, we were able to “clone” single recombinant <em>X</em> chromosomes without being confounded by GC events that may subsequently occur over balancer chromosomes. A second significant reason for focusing on the <em>X</em> arose from the costs related to obtaining very high levels of sequence coverage necessary for the study of recombination on autosomes. Identifying recombination events on the <em>X</em> in F1 males requires simply identifying the haplotype structure along a single hemizygous chromosome. In contrast, the scoring of recombination events on the autosomes of F1 progeny requires being able to distinguish heterozygosity from homozygosity. In the case of GC, one would in fact be required to distinguish heterozygosity and homozygosity for every nucleotide polymorphic between the parents. This becomes very challenging without great sequencing depth and would require extensive downstream verification. We also randomly selected male progeny to create <em><a href="http://flybase.org/reports/FBab0000080.html">C(1)DX</a></em>-bearing stocks for sequencing rather than selecting males know to carry recombinant <em>X</em> chromosomes. We did this so as to not bias our estimates of the rate of GC if the rate of COs or GCs were dependent. Our results suggest that the likelihood of a GC event on a chromosome was not influenced by a CO event, though we lacked power for this test.</p><p id="p-72">Of additional significance, we see this technique as the way meiotic mutants will be analyzed in the future. The rapidly decreasing cost of sequencing and the increase in data provided with each experiment is approaching a point where it is cheaper to analyze individual genomes than it is to do traditional crosses when analyzing meiotic mutants. Most importantly, very few meiotic mutants have been assessed for their effects on GC (<a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">Blanton <em>et al.</em> 2005</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Carpenter 1982</a>; <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Curtis and Bender 1991</a>; <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Radford <em>et al.</em> 2007a</a>). This missing knowledge is especially problematic for the analysis of those mutants that affect pairing and/or synapsis as <em><a href="http://flybase.org/reports/FBgn0000246.html">c(3)G</a></em>, <em><a href="http://flybase.org/reports/FBgn0038612.html">cona</a></em>, <em><a href="http://flybase.org/reports/FBgn0028525.html">c(2)M</a></em>, and <em><a href="http://flybase.org/reports/FBgn0003009.html">ord</a></em> as well as mutants likely to come out of ongoing screens.</p><p id="p-73">In summary, we provide the first step toward a WGS-based approach to the study of meiotic recombination in <em>D. melanogaster</em>. Although the step was admittedly a small one, even these limited data point to curious differences in distribution of CO and GC events, a bias against COs occurring in exonic regions, and a motif enriched in the vicinity of CO events. As technologies improve and costs continue to decrease, we expect that these inferences will be rigorously tested and the analysis extended to meiotic mutants. Indeed, we look forward to a day in the not-so-distant future when characterizing the recombination landscape with visible markers becomes a practice primarily discussed in undergraduate lecture courses.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-74">We are grateful to members of the Hawley lab and the Molecular Biology Core Facility at the Stowers Institute for helpful discussions and critical reading of the manuscript. We would like to especially thank Cathy Lake for her assistance in all phases of this effort. S.T.A. was a Japan Society for the Promotion of Science (JSPS) Postdoctoral Fellow for Research Abroad. W.D.G. was supported by URC and FSRG grants from the DePaul University College of Liberal Arts and Sciences. J.P.B. was supported by the University of Kansas and NSF Award MCB-1022165. R.S.H. is an American Cancer Society Research Professor.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001396/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001396/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">Sequence data are available at the SRA database at NCBI as Study SRP009125. <a href="http://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?study=SRP009125">http://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?study=SRP009125</a>.</p></li><li class="fn" id="fn-8"><p id="p-75"><em>Communicating editor: R. Kulathinal</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-10-12"><span class="received-label">Received </span>October 12, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-12-01"><span class="accepted-label">Accepted </span>December 1, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Miller <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elkan</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Fitting a mixture model by expectation maximization to discover motifs in biopolymers</span>. <abbr class="cit-jnl-abbrev">Proc. Int. Conf. Intell. Syst. Mol. Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">28</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2B%252F%2B...%2BInternational%2BConference%2Bon%2BIntelligent%2BSystems%2Bfor%2BMolecular%2BBiology%2B%253B%2BISMB.%2BInternational%2BConference%2Bon%2BIntelligent%2BSystems%2Bfor%2BMolecular%2BBiology%26rft.stitle%253DProc%2BInt%2BConf%2BIntell%2BSyst%2BMol%2BBiol%26rft.aulast%253DBailey%26rft.auinit1%253DT.%2BL.%26rft.volume%253D2%26rft.spage%253D28%26rft.epage%253D36%26rft.atitle%253DFitting%2Ba%2Bmixture%2Bmodel%2Bby%2Bexpectation%2Bmaximization%2Bto%2Bdiscover%2Bmotifs%2Bin%2Bbiopolymers.%26rft_id%253Dinfo%253Apmid%252F7584402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7584402&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elkan</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span><span class="cit-comment">a</span> <span class="cit-article-title">The value of prior knowledge in discovering motifs with MEME</span>. <abbr class="cit-jnl-abbrev">Proc. Int. Conf. Intell. Syst. Mol. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2B%252F%2B...%2BInternational%2BConference%2Bon%2BIntelligent%2BSystems%2Bfor%2BMolecular%2BBiology%2B%253B%2BISMB.%2BInternational%2BConference%2Bon%2BIntelligent%2BSystems%2Bfor%2BMolecular%2BBiology%26rft.stitle%253DProc%2BInt%2BConf%2BIntell%2BSyst%2BMol%2BBiol%26rft.aulast%253DBailey%26rft.auinit1%253DT.%2BL.%26rft.volume%253D3%26rft.spage%253D21%26rft.epage%253D29%26rft.atitle%253DThe%2Bvalue%2Bof%2Bprior%2Bknowledge%2Bin%2Bdiscovering%2Bmotifs%2Bwith%2BMEME.%26rft_id%253Dinfo%253Apmid%252F7584439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7584439&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elkan</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span><span class="cit-comment">b</span> <span class="cit-article-title">Unsupervised learning of multiple motifs in biopolymers using expectation maximization</span>. <abbr class="cit-jnl-abbrev">Machine Learning Journal</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMachine%2BLearning%2BJournal%26rft.volume%253D21%26rft.spage%253D51%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misleh</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">MEME: discovering and analyzing DNA and protein sequence motifs</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">W369–W373</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D34%26rft.spage%253DW369%2BW373%26rft.atitle%253DMEME%253A%2BDISCOVERING%2BAND%2BANALYZING%2BDNA%2BAND%2BPROTEIN%2BSEQUENCE%2BMOTIFS%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.5" data-doi="10.2174/138920210790886835"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berchowitz</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Copenhaver</span>  <span class="cit-name-given-names">G. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic interference: don’t stand so close to me</span>. <abbr class="cit-jnl-abbrev">Curr. Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCurr%2BGenomics%26rft.aulast%253DBerchowitz%26rft.auinit1%253DL.%2BE.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D91%26rft.epage%253D102%26rft.atitle%253DGenetic%2Binterference%253A%2Bdon%2527t%2Bstand%2Bso%2Bclose%2Bto%2Bme.%26rft_id%253Dinfo%253Adoi%252F10.2174%252F138920210790886835%26rft_id%253Dinfo%253Apmid%252F20885817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2174/138920210790886835&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20885817&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275150100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.6" data-doi="10.1016/S0092-8674(04)00297-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zickler</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Early decision: meiotic crossover interference prior to stable strand exchange and synapsis</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBishop%26rft.auinit1%253DD.%2BK.%26rft.volume%253D117%26rft.issue%253D1%26rft.spage%253D9%26rft.epage%253D15%26rft.atitle%253DEarly%2Bdecision%253B%2Bmeiotic%2Bcrossover%2Binterference%2Bprior%2Bto%2Bstable%2Bstrand%2Bexchange%2Band%2Bsynapsis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252804%252900297-1%26rft_id%253Dinfo%253Apmid%252F15066278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(04)00297-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15066278&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221500300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blanton</span>  <span class="cit-name-given-names">H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Radford</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMahan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kearney</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ibrahim</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">REC, Drosophila MCM8, drives formation of meiotic crossovers</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D1%26rft.spage%253D343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.8" data-doi="10.1534/genetics.109.101998"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blumenstiel</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noll</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perera</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walton</span>  <span class="cit-name-given-names">K. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Identification of EMS-induced mutations in <em>Drosophila melanogaster</em> by whole-genome sequencing</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D182%26rft.spage%253D25%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101998%26rft_id%253Dinfo%253Apmid%252F19307605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.109.101998&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19307605&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270213800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.9" data-doi="10.1073/pnas.72.8.3186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">A. T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Electron microscopy of meiosis in <em>Drosophila melanogaster</em> females. 2. Recombination nodule - recombination-associated structure at pachytene</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">3186</span>–<span class="cit-lpage">3189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DCarpenter%26rft.auinit1%253DA.%2BT.%2BC.%26rft.volume%253D72%26rft.issue%253D8%26rft.spage%253D3186%26rft.epage%253D3189%26rft.atitle%253DElectron%2Bmicroscopy%2Bof%2Bmeiosis%2Bin%2BDrosophila%2Bmelanogaster%2Bfemales%253A%2BII%253A%2BThe%2Brecombination%2Bnodule--a%2Brecombination-associated%2Bstructure%2Bat%2Bpachytene%253F%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.72.8.3186%26rft_id%253Dinfo%253Apmid%252F810799%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI3Mi84LzMxODYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.10" data-doi="10.1073/pnas.79.19.5961"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">A. T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Mismatch repair, gene conversion, and crossing-over in two recombination-defective mutants of <em>Drosophila melanogaster</em></span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U S A</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">5961</span>–<span class="cit-lpage">5965</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DCarpenter%26rft.auinit1%253DA.%2BT.%26rft.volume%253D79%26rft.issue%253D19%26rft.spage%253D5961%26rft.epage%253D5965%26rft.atitle%253DMismatch%2Brepair%252C%2Bgene%2Bconversion%252C%2Band%2Bcrossing-over%2Bin%2Btwo%2Brecombination-defective%2Bmutants%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.79.19.5961%26rft_id%253Dinfo%253Apmid%252F6821126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiNzkvMTkvNTk2MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8yNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.11" data-doi="10.1038/43827"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carvalho</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Intron size and natural selection</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">401</span>: <span class="cit-fpage">344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCarvalho%26rft.auinit1%253DA.%2BB.%26rft.volume%253D401%26rft.issue%253D6751%26rft.spage%253D344%26rft.epage%253D344%26rft.atitle%253DIntron%2Bsize%2Band%2Bnatural%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F43827%26rft_id%253Dinfo%253Apmid%252F10517631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/43827&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10517631&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000082822600041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chovnick</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ballantyne</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holm</span>  <span class="cit-name-given-names">D. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Studies on gene conversion and its relationship to linked exchange in Drosophila melanogaster</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D69%26rft.spage%253D179%26rft_id%253Dinfo%253Apmid%252F5002749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=5002749&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comeron</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The correlation between intron length and recombination in drosophila. Dynamic equilibrium between mutational and selective forces</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">1175</span>–<span class="cit-lpage">1190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D156%26rft.spage%253D1175%26rft_id%253Dinfo%253Apmid%252F11063693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11063693&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165143200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Curtis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bender</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Gene conversion in Drosophila and the effects of the meiotic mutants mei-9 and mei-218</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">739</span>–<span class="cit-lpage">746</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D127%26rft.spage%253D739%26rft.atitle%253DGENETICS%26rft_id%253Dinfo%253Apmid%252F2029970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2029970&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FD75300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Curtis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chovnick</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bender</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Molecular analysis of recombination events in Drosophila</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">653</span>–<span class="cit-lpage">661</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D122%26rft.spage%253D653%26rft_id%253Dinfo%253Apmid%252F2503419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2503419&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.16" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.17" data-doi="10.1016/S0092-8674(00)81481-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dernburg</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moulder</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barstead</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dresser</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Meiotic recombination in <em>C. elegans</em> initiates by a conserved mechanism and is dispensable for homologous chromosome synapsis</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DDernburg%26rft.auinit1%253DA.%2BF.%26rft.volume%253D94%26rft.issue%253D3%26rft.spage%253D387%26rft.epage%253D398%26rft.atitle%253DMeiotic%2Brecombination%2Bin%2BC.%2Belegans%2Binitiates%2Bby%2Ba%2Bconserved%2Bmechanism%2Band%2Bis%2Bdispensable%2Bfor%2Bhomologous%2Bchromosome%2Bsynapsis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981481-6%26rft_id%253Dinfo%253Apmid%252F9708740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81481-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9708740&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075308400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.18" data-doi="10.1371/journal.pone.0012339"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dhami</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saffrey</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruce</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dillon</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Complex exon-intron marking by histone modifications is not determined solely by nucleosome distribution</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e12339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDhami%26rft.auinit1%253DP.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253De12339%26rft.epage%253De12339%26rft.atitle%253DComplex%2Bexon-intron%2Bmarking%2Bby%2Bhistone%2Bmodifications%2Bis%2Bnot%2Bdetermined%2Bsolely%2Bby%2Bnucleosome%2Bdistribution.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0012339%26rft_id%253Dinfo%253Apmid%252F20808788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0012339&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20808788&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-2.2.249.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finnerty</span>  <span class="cit-name-given-names">V. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Gene conversion in <em>Drosophila</em></span>, pp. <span class="cit-fpage">331</span>–<span class="cit-lpage">351</span> in <span class="cit-source">The Genetics and Biology of Drosophila</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Novitski</span>  <span class="cit-name-given-names">E.</span></span>. <span class="cit-publ-loc">Academic Press</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.20" data-doi="10.1016/j.gene.2010.04.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fiston-Lavier</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipatov</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title"><em>Drosophila melanogaster</em> recombination rate calculator</span>. <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">463</span>: <span class="cit-fpage">18</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DGene%26rft.aulast%253DFiston-Lavier%26rft.auinit1%253DA.%2BS.%26rft.volume%253D463%26rft.issue%253D1-2%26rft.spage%253D18%26rft.epage%253D20%26rft.atitle%253DDrosophila%2Bmelanogaster%2Brecombination%2Brate%2Bcalculator.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2010.04.015%26rft_id%253Dinfo%253Apmid%252F20452408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2010.04.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20452408&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280751600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.21" data-doi="10.1371/journal.pbio.1000520"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldfarb</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Frequent and efficient use of the sister chromatid for DNA double-strand break repair during budding yeast meiosis</span>. <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1000520</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DGoldfarb%26rft.auinit1%253DT.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253De1000520%26rft.epage%253De1000520%26rft.atitle%253DFrequent%2Band%2Befficient%2Buse%2Bof%2Bthe%2Bsister%2Bchromatid%2Bfor%2BDNA%2Bdouble-strand%2Bbreak%2Brepair%2Bduring%2Bbudding%2Byeast%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000520%26rft_id%253Dinfo%253Apmid%252F20976044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000520&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20976044&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.22" data-doi="10.1093/bioinformatics/btr064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">W. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">FIMO: scanning for occurrences of a given motif</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1017</span>–<span class="cit-lpage">1018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr064%26rft_id%253Dinfo%253Apmid%252F21330290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyNy83LzEwMTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.23" data-doi="10.1073/pnas.35.10.586"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">K. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1949</span> <span class="cit-article-title">Crossing-over between alleles at the lozenge locus in <em>Drosophila melanogaster</em></span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U S A</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">586</span>–<span class="cit-lpage">591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DGreen%26rft.auinit1%253DM.%2BM.%26rft.volume%253D35%26rft.issue%253D10%26rft.spage%253D586%26rft.epage%253D591%26rft.atitle%253DCrossing-Over%2BBetween%2BAlleles%2Bat%2Bthe%2BLozenge%2BLocus%2Bin%2BDrosophila%2BMelanogaster%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.35.10.586%26rft_id%253Dinfo%253Apmid%252F15391347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIzNS8xMC81ODYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.24" data-doi="10.1093/emboj/20.3.589"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grelon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vezon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gendrot</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pelletier</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">AtSPO11-1 is necessary for efficient meiotic recombination in plants</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">600</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DGrelon%26rft.auinit1%253DM.%26rft.volume%253D20%26rft.issue%253D3%26rft.spage%253D589%26rft.epage%253D600%26rft.atitle%253DAtSPO11-1%2Bis%2Bnecessary%2Bfor%2Befficient%2Bmeiotic%2Brecombination%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F20.3.589%26rft_id%253Dinfo%253Apmid%252F11157765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo4OiIyMC8zLzU4OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8yNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.25" data-doi="10.1186/gb-2007-8-2-r24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stamatoyannopoulos</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">W. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Quantifying similarity between motifs</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">R24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DGupta%26rft.auinit1%253DS.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253DR24%26rft.epage%253DR24%26rft.atitle%253DQuantifying%2Bsimilarity%2Bbetween%2Bmotifs.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2007-8-2-r24%26rft_id%253Dinfo%253Apmid%252F17324271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2007-8-2-r24&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17324271&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kliman</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Interactions between natural selection, recombination and gene density in the genes of Drosophila</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D160%26rft.spage%253D595%26rft_id%253Dinfo%253Apmid%252F11861564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11861564&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174097600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hilliker</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harauz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reaume</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Meiotic gene conversion tract length distribution within the <em>rosy</em> locus of <em>Drosophila melanogaster</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">1019</span>–<span class="cit-lpage">1024</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D137%26rft.spage%253D1019%26rft_id%253Dinfo%253Apmid%252F7982556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7982556&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NX91200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.28" data-doi="10.1038/nature10336"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hinch</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tandon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rohland</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The landscape of recombination in African Americans</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">476</span>: <span class="cit-fpage">170</span>–<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHinch%26rft.auinit1%253DA.%2BG.%26rft.volume%253D476%26rft.issue%253D7359%26rft.spage%253D170%26rft.epage%253D175%26rft.atitle%253DThe%2Blandscape%2Bof%2Brecombination%2Bin%2BAfrican%2BAmericans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10336%26rft_id%253Dinfo%253Apmid%252F21775986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10336&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21775986&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293731900028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.29" data-doi="10.1242/jcs.00614"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jang</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherizen</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhagat</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manheim</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKim</span>  <span class="cit-name-given-names">K. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Relationship of DNA double-strand breaks to synapsis in Drosophila</span>. <abbr class="cit-jnl-abbrev">J. Cell Sci.</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">3069</span>–<span class="cit-lpage">3077</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjcs.00614%26rft_id%253Dinfo%253Apmid%252F12799415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiam9jZXMiO3M6NToicmVzaWQiO3M6MTE6IjExNi8xNS8zMDY5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8yLzI0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.30" data-doi="10.1371/journal.pgen.0040031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klovstad</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abdu</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schupbach</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Drosophila brca2 is required for mitotic and meiotic DNA repair and efficient activation of the meiotic recombination checkpoint</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKlovstad%26rft.auinit1%253DM.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De31%26rft.epage%253De31%26rft.atitle%253DDrosophila%2Bbrca2%2Bis%2Brequired%2Bfor%2Bmitotic%2Band%2Bmeiotic%2BDNA%2Brepair%2Band%2Befficient%2Bactivation%2Bof%2Bthe%2Bmeiotic%2Brecombination%2Bcheckpoint.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0040031%26rft_id%253Dinfo%253Apmid%252F18266476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0040031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18266476&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.31" data-doi="10.1038/ng.322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kolasinska-Zwierz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Down</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Latorre</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Differential chromatin marking of introns and expressed exons by H3K36me3</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">376</span>–<span class="cit-lpage">381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKolasinska-Zwierz%26rft.auinit1%253DP.%26rft.volume%253D41%26rft.issue%253D3%26rft.spage%253D376%26rft.epage%253D381%26rft.atitle%253DDifferential%2Bchromatin%2Bmarking%2Bof%2Bintrons%2Band%2Bexpressed%2Bexons%2Bby%2BH3K36me3.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.322%26rft_id%253Dinfo%253Apmid%252F19182803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19182803&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263640200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.32" data-doi="10.1038/ng0302-241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gudbjartsson</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sainz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jonsdottir</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gudjonsson</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A high-resolution recombination map of the human genome</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKong%26rft.auinit1%253DA.%26rft.volume%253D31%26rft.issue%253D3%26rft.spage%253D241%26rft.epage%253D247%26rft.atitle%253DA%2Bhigh-resolution%2Brecombination%2Bmap%2Bof%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng0302-241%26rft_id%253Dinfo%253Apmid%252F12053178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng0302-241&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12053178&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176494100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.33" data-doi="10.1038/nature09525"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorleifsson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gudbjartsson</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sigurdsson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fine-scale recombination rate differences between sexes, populations and individuals</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">1099</span>–<span class="cit-lpage">1103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKong%26rft.auinit1%253DA.%26rft.volume%253D467%26rft.issue%253D7319%26rft.spage%253D1099%26rft.epage%253D1103%26rft.atitle%253DFine-scale%2Brecombination%2Brate%2Bdifferences%2Bbetween%2Bsexes%252C%2Bpopulations%2Band%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09525%26rft_id%253Dinfo%253Apmid%252F20981099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09525&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20981099&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283548600046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.34" data-doi="10.1073/pnas.0801848105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kulathinal</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennettt</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noor</span>  <span class="cit-name-given-names">M. A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Fine-scale mapping of recombination rate in Drosophila refines its correlation to diversity and divergence</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U S A</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">10051</span>–<span class="cit-lpage">10056</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0801848105%26rft_id%253Dinfo%253Apmid%252F18621713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzI5LzEwMDUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8yLzI0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.35" data-doi="10.1534/genetics.111.127530"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crepeau</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardeno</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Circumventing heterozygosity: sequencing the amplified genome of a single haploid drosophila melanogster embryo</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.127530%26rft_id%253Dinfo%253Apmid%252F21441209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzIvMjM5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8yLzI0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.36" data-doi="10.1101/gr.078212.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mapping short DNA sequencing reads and calling variants using mapping quality scores</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1851</span>–<span class="cit-lpage">1858</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.078212.108%26rft_id%253Dinfo%253Apmid%252F18714091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMS8xODUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8yLzI0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.37" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8yNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.38" data-doi="10.1098/rstb.1977.0019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindsley</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandler</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">The genetic analysis of meiosis in female <em>Drosophila melanogaster</em></span>. <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">277</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DLindsley%26rft.auinit1%253DD.%2BL.%26rft.volume%253D277%26rft.issue%253D955%26rft.spage%253D295%26rft.epage%253D312%26rft.atitle%253DThe%2BGenetic%2BAnalysis%2Bof%2BMeiosis%2Bin%2BFemale%2BDrosophila%2Bmelanogaster%2B%255Band%2BDiscussion%255D%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.1977.0019%26rft_id%253Dinfo%253Apmid%252F16292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjExOiIyNzcvOTU1LzI5NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8yNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.39" data-doi="10.1038/nature07135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mancera</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bourgon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brozzi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">High-resolution mapping of meiotic crossovers and non-crossovers in yeast</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">454</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMancera%26rft.auinit1%253DE.%26rft.volume%253D454%26rft.issue%253D7203%26rft.spage%253D479%26rft.epage%253D485%26rft.atitle%253DHigh-resolution%2Bmapping%2Bof%2Bmeiotic%2Bcrossovers%2Band%2Bnon-crossovers%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07135%26rft_id%253Dinfo%253Apmid%252F18615017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18615017&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257860300043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.40" data-doi="10.1093/nar/gkj143"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matys</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kel-Margoulis</span>  <span class="cit-name-given-names">O. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fricke</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liebich</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Land</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">TRANSFAC and its module TRANSCompel: transcriptional gene regulation in eukaryotes</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">D108</span>–<span class="cit-lpage">D110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMatys%26rft.auinit1%253DV.%26rft.volume%253D34%26rft.issue%253Dsuppl_1%26rft.spage%253DD108%26rft.epage%253DD110%26rft.atitle%253DTRANSFAC%2528R%2529%2Band%2Bits%2Bmodule%2BTRANSCompel%2528R%2529%253A%2Btranscriptional%2Bgene%2Bregulation%2Bin%2Beukaryotes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkj143%26rft_id%253Dinfo%253Apmid%252F16381825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNC9zdXBwbF8xL0QxMDgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.41" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8yNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.42" data-doi="10.1101/gad.12.18.2932"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKim</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashi-Hagihara</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">mei-W68 in <em>Drosophila melanogaster</em> encodes a Spo11 homolog: evidence that the mechanism for initiating meiotic recombination is conserved</span>. <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">2932</span>–<span class="cit-lpage">2942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DMcKim%26rft.auinit1%253DK.%2BS.%26rft.volume%253D12%26rft.issue%253D18%26rft.spage%253D2932%26rft.epage%253D2942%26rft.atitle%253Dmei-W68%2Bin%2BDrosophila%2Bmelanogaster%2Bencodes%2Ba%2BSpo11%2Bhomolog%253A%2Bevidence%2Bthat%2Bthe%2Bmechanism%2Bfor%2Binitiating%2Bmeiotic%2Brecombination%2Bis%2Bconserved%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.12.18.2932%26rft_id%253Dinfo%253Apmid%252F9744869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjEyLzE4LzI5MzIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.43" data-doi="10.1126/science.1092500"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deloukas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The fine-scale structure of recombination rate variation in the human genome</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">304</span>: <span class="cit-fpage">581</span>–<span class="cit-lpage">584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMcVean%26rft.auinit1%253DG.%2BA.%2BT.%26rft.volume%253D304%26rft.issue%253D5670%26rft.spage%253D581%26rft.epage%253D584%26rft.atitle%253DThe%2BFine-Scale%2BStructure%2Bof%2BRecombination%2BRate%2BVariation%2Bin%2Bthe%2BHuman%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1092500%26rft_id%253Dinfo%253Apmid%252F15105499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDQvNTY3MC81ODEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mehrotra</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKim</span>  <span class="cit-name-given-names">K. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Temporal analysis of meiotic DNA double-strand break formation and repair in Drosophila females</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1883</span>–<span class="cit-lpage">1897</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D2%26rft.spage%253D1883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.45" data-doi="10.1038/nrg2626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Metzker</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Sequencing technologies—the next generation</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMetzker%26rft.auinit1%253DM.%2BL.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D46%26rft.atitle%253DSequencing%2Btechnologies%2B-%2Bthe%2Bnext%2Bgeneration.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2626%26rft_id%253Dinfo%253Apmid%252F19997069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2626&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19997069&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272867700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.46" data-doi="10.1126/science.1117196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bottolo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeman</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A fine-scale map of recombination rates and hotspots across the human genome</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">310</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMyers%26rft.auinit1%253DS.%26rft.volume%253D310%26rft.issue%253D5746%26rft.spage%253D321%26rft.epage%253D324%26rft.atitle%253DA%2BFine-Scale%2BMap%2Bof%2BRecombination%2BRates%2Band%2BHotspots%2BAcross%2Bthe%2BHuman%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1117196%26rft_id%253Dinfo%253Apmid%252F16224025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTAvNTc0Ni8zMjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.47" data-doi="10.1038/ng.213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeman</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A common sequence motif associated with recombination hot spots and genome instability in humans</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1124</span>–<span class="cit-lpage">1129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMyers%26rft.auinit1%253DS.%26rft.volume%253D40%26rft.issue%253D9%26rft.spage%253D1124%26rft.epage%253D1129%26rft.atitle%253DA%2Bcommon%2Bsequence%2Bmotif%2Bassociated%2Bwith%2Brecombination%2Bhot%2Bspots%2Band%2Bgenome%2Binstability%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.213%26rft_id%253Dinfo%253Apmid%252F19165926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.213&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19165926&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258761200028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.48" data-doi="10.1126/science.1182363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowden</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tumian</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bontrop</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeman</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Drive against hotspot motifs in primates implicates the PRDM9 gene in meiotic recombination</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">876</span>–<span class="cit-lpage">879</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1182363%26rft_id%253Dinfo%253Apmid%252F20044541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2Ny84NzYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.49" data-doi="10.1038/nrg2986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genotype and SNP calling from next-generation sequencing data</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">443</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D443%26rft.epage%253D451%26rft.atitle%253DGenotype%2Band%2BSNP%2Bcalling%2Bfrom%2Bnext-generation%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2986%26rft_id%253Dinfo%253Apmid%252F21587300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2986&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21587300&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.50" data-doi="10.1101/gr.067181.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Reilly</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balding</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Confounding between recombination and selection, and the Ped/Pop method for detecting selection</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1304</span>–<span class="cit-lpage">1313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.067181.107%26rft_id%253Dinfo%253Apmid%252F18617692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE4LzgvMTMwNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8yNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.51" data-doi="10.1093/nar/gkp950"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Portales-Casamar</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thongjuea</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwon</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arenillas</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">JASPAR 2010: the greatly expanded open-access database of transcription factor binding profiles</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">D105</span>–<span class="cit-lpage">D110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp950%26rft_id%253Dinfo%253Apmid%252F19906716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzOC9zdXBwbF8xL0QxMDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.52" data-doi="10.1093/nar/gkn378"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pugh</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delaney</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farnoud</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flibotte</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffith</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Impact of whole genome amplification on analysis of copy number variants</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">e80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn378%26rft_id%253Dinfo%253Apmid%252F18559357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjM2LzEzL2U4MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8yNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.53" data-doi="10.1186/1471-2164-10-475"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wijeratne</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomsho</span>  <span class="cit-name-given-names">L. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuster</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Characterization of meiotic crossovers and gene conversion by whole-genome sequencing in <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">475</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DQi%26rft.auinit1%253DJ.%26rft.volume%253D10%26rft.spage%253D475%26rft.epage%253D475%26rft.atitle%253DCharacterization%2Bof%2Bmeiotic%2Bcrossovers%2Band%2Bgene%2Bconversion%2Bby%2Bwhole-genome%2Bsequencing%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-475%26rft_id%253Dinfo%253Apmid%252F19832984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-475&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19832984&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.54" data-doi="10.1534/genetics.107.070557"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Radford</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMahan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanton</span>  <span class="cit-name-given-names">H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekelsky</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">a</span> <span class="cit-article-title">Heteroduplex DNA in meiotic recombination in Drosophila mei-9 mutants</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.070557%26rft_id%253Dinfo%253Apmid%252F17339219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTc2LzEvNjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.55" data-doi="10.1534/genetics.107.070367"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Radford</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabourin</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMahan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekelsky</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">b</span> <span class="cit-article-title">Meiotic recombination in Drosophila Msh6 mutants yields discontinuous gene conversion tracts</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D176%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.070367%26rft_id%253Dinfo%253Apmid%252F17339220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.107.070367&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17339220&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246960900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schweitzer</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1935</span> <span class="cit-article-title">An analytical study of crossing over in <em>Drosophila melanogaster</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">497</span>–<span class="cit-lpage">527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSchweitzer%26rft.auinit1%253DM.%2BD.%26rft.volume%253D20%26rft.issue%253D5%26rft.spage%253D497%26rft.epage%253D527%26rft.atitle%253DAN%2BANALYTICAL%2BSTUDY%2BOF%2BCROSSING%2BOVER%2BIN%2BDROSOPHILA%2BMELANOGASTER%26rft_id%253Dinfo%253Apmid%252F17246774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIyMC81LzQ5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMi8yNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.57" data-doi="10.1007/s00239-009-9250-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Estimation of fine-scale recombination intensity variation in the white-echinus interval of <em>D-melanogaster</em></span>. <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">42</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DSingh%26rft.auinit1%253DN.%2BD.%26rft.volume%253D69%26rft.issue%253D1%26rft.spage%253D42%26rft.epage%253D53%26rft.atitle%253DEstimation%2Bof%2Bfine-scale%2Brecombination%2Bintensity%2Bvariation%2Bin%2Bthe%2Bwhite-echinus%2Binterval%2Bof%2BD.%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-009-9250-5%26rft_id%253Dinfo%253Apmid%252F19504037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-009-9250-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19504037&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268577800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.58" data-doi="10.1038/228442a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finnerty</span>  <span class="cit-name-given-names">V. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chovnick</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Gene conversion in <em>Drosophila</em>: non-reciprocal events at the marron-like cistron</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">228</span>: <span class="cit-fpage">442</span>–<span class="cit-lpage">444</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSmith%26rft.auinit1%253DP.%2BD.%26rft.volume%253D228%26rft.issue%253D5270%26rft.spage%253D442%26rft.epage%253D444%26rft.atitle%253DGene%2Bconversion%2Bin%2BDrosophila%253A%2Bnon-reciprocal%2Bevents%2Bat%2Bthe%2Bmaroon-like%2Bcistron.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F228442a0%26rft_id%253Dinfo%253Apmid%252F5482495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/228442a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5482495&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1970H626700038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.59" data-doi="10.1007/s00239-010-9388-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stevison</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noor</span>  <span class="cit-name-given-names">M. A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic and evolutionary correlates of fine-scale recombination rate variation in <em>Drosophila persimilis</em></span>. <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">332</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DStevison%26rft.auinit1%253DL.%2BS.%26rft.volume%253D71%26rft.issue%253D5-6%26rft.spage%253D332%26rft.epage%253D345%26rft.atitle%253DGenetic%2Band%2Bevolutionary%2Bcorrelates%2Bof%2Bfine-scale%2Brecombination%2Brate%2Bvariation%2Bin%2BDrosophila%2Bpersimilis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-010-9388-1%26rft_id%253Dinfo%253Apmid%252F20890595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-010-9388-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20890595&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284549800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.60" data-doi="10.1038/338087a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Treco</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schultes</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szostak</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Double-strand breaks at an initiation site for meiotic gene conversion</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">338</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSun%26rft.auinit1%253DH.%26rft.volume%253D338%26rft.issue%253D6210%26rft.spage%253D87%26rft.epage%253D90%26rft.atitle%253DDouble-strand%2Bbreaks%2Bat%2Ban%2Binitiation%2Bsite%2Bfor%2Bmeiotic%2Bgene%2Bconversion.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F338087a0%26rft_id%253Dinfo%253Apmid%252F2645528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/338087a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2645528&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989T488200065&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.61" data-doi="10.1016/0092-8674(83)90331-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szostak</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orr-Weaver</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothstein</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stahl</span>  <span class="cit-name-given-names">F. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">The double-strand-break repair model for recombination</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSzostak%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D35%26rft.atitle%253DThe%2Bdouble-strand-break%2Brepair%2Bmodel%2Bfor%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252883%252990331-8%26rft_id%253Dinfo%253Apmid%252F6380756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(83)90331-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6380756&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F2%2F249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983QS06800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.62" data-doi="10.1093/nar/gkn788"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tweedie</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falls</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leyland</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McQuilton</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">FlyBase: enhancing Drosophila Gene Ontology annotations</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">D555</span>–<span class="cit-lpage">D559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn788%26rft_id%253Dinfo%253Apmid%252F18948289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNy9zdXBwbF8xL0Q1NTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzIvMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2.2.249.63" data-doi="10.1242/jcs.074427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Youds</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boulton</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The choice in meiosis—defining the factors that influence crossover or non-crossover formation</span>. <abbr class="cit-jnl-abbrev">J. Cell Sci.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">501</span>–<span class="cit-lpage">513</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BCell%2BScience%26rft.stitle%253DJ.%2BCell%2BSci.%26rft.aulast%253DYouds%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D124%26rft.issue%253D4%26rft.spage%253D501%26rft.epage%253D513%26rft.atitle%253DThe%2Bchoice%2Bin%2Bmeiosis%2B-%2Bdefining%2Bthe%2Bfactors%2Bthat%2Binfluence%2Bcrossover%2Bor%2Bnon-crossover%2Bformation%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjcs.074427%26rft_id%253Dinfo%253Apmid%252F21282472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiam9jZXMiO3M6NToicmVzaWQiO3M6OToiMTI0LzQvNTAxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8yLzI0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/2/249.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/2/235" title="Extent With Modification: Leg Patterning in the Beetle Tribolium castaneum and the Evolution of Serial Homologs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/2/261" title="Whole-Genome Sequencing of Sordaria macrospora Mutants Identifies Developmental Genes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/2"><span>Volume 2 Issue 2, February 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/2.cover.gif?itok=i7gjzosG" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/2/249.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AR4zNDk1fzTaf2YEaTkYVjNx993AUXhKiG7jCjIf5t4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/2/249" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/2/249" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/2/249" class="active" data-icon-position="" data-hide-link-title="0">A Whole-Chromosome Analysis of Meiotic Recombination in Drosophila melanogaster</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459241" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-TeAIxARXIFAxH5zJ0HudA0tj8OaJ6co24v0PfD_26Yc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/2/249.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/2/249" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-KERkntZImIdPZu_0otcn1UTGqtGkEn7tMnfrq4zAmB8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/2/249" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/2/249.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459241" id="node459241--2" data-pisa="ggg;2/2/249" data-pisa-master="ggg;2/2/249" data-apath="/ggg/2/2/249.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Whole-Chromosome Analysis of Meiotic Recombination in <em>Drosophila melanogaster</em></h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Danny E. Miller</span>, <span class="highwire-citation-author" data-delta="1">Satomi Takeo</span>, <span class="highwire-citation-author" data-delta="2">Kavyasree Nandanan</span>, <span class="highwire-citation-author" data-delta="3">Ariel Paulson</span>, <span class="highwire-citation-author" data-delta="4">Madelaine M. Gogol</span>, <span class="highwire-citation-author" data-delta="5">Aaron C. Noll</span>, <span class="highwire-citation-author" data-delta="6">Anoja G.  Perera</span>, <span class="highwire-citation-author" data-delta="7">Kendra N. Walton</span>, <span class="highwire-citation-author" data-delta="8">William D. Gilliland</span>, <span class="highwire-citation-author" data-delta="9">Hua Li</span>, <span class="highwire-citation-author" data-delta="10">Karen K.  Staehling</span>, <span class="highwire-citation-author" data-delta="11">Justin P. Blumenstiel</span> and <span class="highwire-citation-author" data-delta="12">R. Scott Hawley</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">249-260;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001396 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459241--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Danny E. Miller</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danny%2BE.%2BMiller" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADanny%2BE.%2BMiller" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Satomi Takeo</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Satomi%2BTakeo" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takeo%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASatomi%2BTakeo" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kavyasree Nandanan</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kavyasree%2BNandanan" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nandanan%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKavyasree%2BNandanan" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ariel Paulson</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ariel%2BPaulson" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paulson%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAriel%2BPaulson" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Madelaine M. Gogol</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Madelaine%2BM.%2BGogol" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gogol%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMadelaine%2BM.%2BGogol" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Aaron C. Noll</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BC.%2BNoll" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Noll%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAaron%2BC.%2BNoll" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Anoja G.  Perera</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anoja%2BG.%2B%2BPerera" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Perera%20AG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnoja%2BG.%2B%2BPerera" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kendra N. Walton</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kendra%2BN.%2BWalton" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Walton%20KN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKendra%2BN.%2BWalton" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">William D. Gilliland</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Biology, Depaul University, Chicago, IL 60604</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BD.%2BGilliland" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gilliland%20WD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BD.%2BGilliland" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Hua Li</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hua%2BLi" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHua%2BLi" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Karen K.  Staehling</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karen%2BK.%2B%2BStaehling" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Staehling%20KK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaren%2BK.%2B%2BStaehling" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Justin P. Blumenstiel</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, KS 66045, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BP.%2BBlumenstiel" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blumenstiel%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJustin%2BP.%2BBlumenstiel" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jblumens@ku.edu" class="" data-icon-position="" data-hide-link-title="0">jblumens@ku.edu</a></span></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">R. Scott Hawley</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span><span class="nlm-aff">Department of Physiology, Kansas University Medical Center, Kansas City, KS 66160</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BScott%2BHawley" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hawley%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR.%2BScott%2BHawley" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/2/249" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/2/249&amp;title=A%20Whole-Chromosome%20Analysis%20of%20Meiotic%20Recombination%20in%20Drosophila%20melanogaster" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/2/249&amp;title=A%20Whole-Chromosome%20Analysis%20of%20Meiotic%20Recombination%20in%20Drosophila%20melanogaster" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/2/249&amp;title=A%20Whole-Chromosome%20Analysis%20of%20Meiotic%20Recombination%20in%20Drosophila%20melanogaster" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/2/249&amp;text=A%20Whole-Chromosome%20Analysis%20of%20Meiotic%20Recombination%20in%20Drosophila%20melanogaster" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/2/249&amp;title=A%20Whole-Chromosome%20Analysis%20of%20Meiotic%20Recombination%20in%20Drosophila%20melanogaster" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/2/249&amp;t=A%20Whole-Chromosome%20Analysis%20of%20Meiotic%20Recombination%20in%20Drosophila%20melanogaster" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/2/249&amp;title=A%20Whole-Chromosome%20Analysis%20of%20Meiotic%20Recombination%20in%20Drosophila%20melanogaster" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/2/249&amp;title=A%20Whole-Chromosome%20Analysis%20of%20Meiotic%20Recombination%20in%20Drosophila%20melanogaster" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459241/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459241" id="node459241--2" data-pisa="ggg;2/2/249" data-pisa-master="ggg;2/2/249" data-apath="/ggg/2/2/249.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Whole-Chromosome Analysis of Meiotic Recombination in <em>Drosophila melanogaster</em></h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Danny E. Miller</span>, <span class="highwire-citation-author" data-delta="1">Satomi Takeo</span>, <span class="highwire-citation-author" data-delta="2">Kavyasree Nandanan</span>, <span class="highwire-citation-author" data-delta="3">Ariel Paulson</span>, <span class="highwire-citation-author" data-delta="4">Madelaine M. Gogol</span>, <span class="highwire-citation-author" data-delta="5">Aaron C. Noll</span>, <span class="highwire-citation-author" data-delta="6">Anoja G.  Perera</span>, <span class="highwire-citation-author" data-delta="7">Kendra N. Walton</span>, <span class="highwire-citation-author" data-delta="8">William D. Gilliland</span>, <span class="highwire-citation-author" data-delta="9">Hua Li</span>, <span class="highwire-citation-author" data-delta="10">Karen K.  Staehling</span>, <span class="highwire-citation-author" data-delta="11">Justin P. Blumenstiel</span> and <span class="highwire-citation-author" data-delta="12">R. Scott Hawley</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">249-260;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001396 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459241--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Danny E. Miller</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danny%2BE.%2BMiller" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADanny%2BE.%2BMiller" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Satomi Takeo</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Satomi%2BTakeo" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takeo%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASatomi%2BTakeo" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kavyasree Nandanan</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kavyasree%2BNandanan" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nandanan%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKavyasree%2BNandanan" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ariel Paulson</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ariel%2BPaulson" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paulson%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAriel%2BPaulson" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Madelaine M. Gogol</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Madelaine%2BM.%2BGogol" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gogol%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMadelaine%2BM.%2BGogol" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Aaron C. Noll</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BC.%2BNoll" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Noll%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAaron%2BC.%2BNoll" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Anoja G.  Perera</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anoja%2BG.%2B%2BPerera" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Perera%20AG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnoja%2BG.%2B%2BPerera" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kendra N. Walton</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kendra%2BN.%2BWalton" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Walton%20KN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKendra%2BN.%2BWalton" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">William D. Gilliland</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Biology, Depaul University, Chicago, IL 60604</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BD.%2BGilliland" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gilliland%20WD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BD.%2BGilliland" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Hua Li</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hua%2BLi" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHua%2BLi" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Karen K.  Staehling</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karen%2BK.%2B%2BStaehling" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Staehling%20KK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaren%2BK.%2B%2BStaehling" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Justin P. Blumenstiel</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, KS 66045, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BP.%2BBlumenstiel" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blumenstiel%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJustin%2BP.%2BBlumenstiel" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jblumens@ku.edu" class="" data-icon-position="" data-hide-link-title="0">jblumens@ku.edu</a></span></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">R. Scott Hawley</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Stowers Institute for Medical Research, Kansas City, MO 64110</span><span class="nlm-aff">Department of Physiology, Kansas University Medical Center, Kansas City, KS 66160</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BScott%2BHawley" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hawley%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR.%2BScott%2BHawley" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=249&amp;rft.epage=260&amp;rft.atitle=A%20Whole-Chromosome%20Analysis%20of%20Meiotic%20Recombination%20in%20Drosophila%20melanogaster&amp;rft.volume=2&amp;rft.issue=2&amp;rft.date=2012-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Miller%2C+Danny+E.&amp;rft.au=Takeo%2C+Satomi&amp;rft.au=Nandanan%2C+Kavyasree&amp;rft.au=Paulson%2C+Ariel&amp;rft.au=Gogol%2C+Madelaine+M.&amp;rft.au=Noll%2C+Aaron+C.&amp;rft.au=+Perera%2C+Anoja+G.&amp;rft.au=Walton%2C+Kendra+N.&amp;rft.au=Gilliland%2C+William+D.&amp;rft.au=Li%2C+Hua&amp;rft.au=+Staehling%2C+Karen+K.&amp;rft.au=Blumenstiel%2C+Justin+P.&amp;rft.au=Hawley%2C+R.+Scott"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459241/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459241/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459241/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459241/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459241/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459241/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459241/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459241/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459241/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459241/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459241/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459241/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494625" id="node494625" data-pisa="ggg;9/11/3891" data-pisa-master="ggg;g3.119.400723" data-apath="/ggg/9/11/3891.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3891" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Mouse Mutation That Dysregulates Neighboring <em>Galnt17</em> and <em>Auts2</em> Genes Is Associated with Phenotypes Related to the Human AUTS2 Syndrome</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494624" id="node494624" data-pisa="ggg;9/11/3781" data-pisa-master="ggg;g3.119.400647" data-apath="/ggg/9/11/3781.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3781" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Transgenic Analyses in Drosophila Reveal That mCORL1 Is Functionally Distinct from mCORL2 and dCORL</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494622" id="node494622" data-pisa="ggg;9/11/3843" data-pisa-master="ggg;g3.119.400363" data-apath="/ggg/9/11/3843.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3843" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Analysis of Cell Cycle-Regulating Genes in the Symbiotic Dinoflagellate <em>Breviolum minutum</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/2/249" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/2/249.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/2/249.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/2/249.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/2/249.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2019 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
  </body>
</html>
