<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/9/987" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/9/987.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/9/987.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/9/987.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/2/9/987/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/9/987.atom" />
<meta name="HW.pisa" content="ggg;2/9/987" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="oPOSSUM-3: Advanced Analysis of Regulatory Motif Over-Representation Across Genes or ChIP-Seq Datasets" />
<meta name="DC.Identifier" content="10.1534/g3.112.003202" />
<meta name="DC.Date" content="2012-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Kwon et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="oPOSSUM-3 is a web-accessible software system for identification of over-represented transcription factor binding sites (TFBS) and TFBS families in either DNA sequences of co-expressed genes or sequences generated from high-throughput methods, such as ChIP-Seq. Validation of the system with known sets of co-regulated genes and published ChIP-Seq data demonstrates the capacity for oPOSSUM-3 to identify mediating transcription factors (TF) for co-regulated genes or co-recovered sequences. oPOSSUM-3 is available at &lt;http://opossum.cisreg.ca&gt;." />
<meta name="DC.Contributor" content="Andrew T. Kwon" />
<meta name="DC.Contributor" content="David J. Arenillas" />
<meta name="DC.Contributor" content="Rebecca Worsley Hunt" />
<meta name="DC.Contributor" content="Wyeth W. Wasserman" />
<meta name="article:published_time" content="2012-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="oPOSSUM-3: Advanced Analysis of Regulatory Motif Over-Representation Across Genes or ChIP-Seq Datasets" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;oPOSSUM-3 is a web-accessible software system for identification of over-represented transcription factor binding sites (TFBS) and TFBS families in either DNA sequences of co-expressed genes or sequences generated from high-throughput methods, such as ChIP-Seq. Validation of the system with known sets of co-regulated genes and published ChIP-Seq data demonstrates the capacity for oPOSSUM-3 to identify mediating transcription factors (TF) for co-regulated genes or co-recovered sequences. oPOSSUM-3 is available at http://opossum.cisreg.ca.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/09/01" />
<meta name="citation_mjid" content="ggg;2/9/987" />
<meta name="citation_id" content="2/9/987" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/9/987" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/9/987.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/9/987.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/9/987.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.003202" />
<meta name="citation_pmid" content="22973536" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="987" />
<meta name="citation_lastpage" content="1002" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Andrew T. Kwon" />
<meta name="citation_author_institution" content="Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, University of British Columbia, Vancouver, British Columbia, V5Z 4H4, Canada" />
<meta name="citation_author_institution" content="Department of Medical Genetics, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada" />
<meta name="citation_author_institution" content="Genetics Graduate Program, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada" />
<meta name="citation_author" content="David J. Arenillas" />
<meta name="citation_author_institution" content="Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, University of British Columbia, Vancouver, British Columbia, V5Z 4H4, Canada" />
<meta name="citation_author_institution" content="Department of Medical Genetics, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada" />
<meta name="citation_author" content="Rebecca Worsley Hunt" />
<meta name="citation_author_institution" content="Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, University of British Columbia, Vancouver, British Columbia, V5Z 4H4, Canada" />
<meta name="citation_author_institution" content="Department of Medical Genetics, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada" />
<meta name="citation_author_institution" content="Bioinformatics Graduate Program, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada" />
<meta name="citation_author" content="Wyeth W. Wasserman" />
<meta name="citation_author_institution" content="Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, University of British Columbia, Vancouver, British Columbia, V5Z 4H4, Canada" />
<meta name="citation_author_institution" content="Department of Medical Genetics, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada" />
<meta name="citation_author_email" content="wyeth@cmmt.ubc.ca" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=S. Aerts;citation_author=P. Van Loo;citation_author=G. Thijs;citation_author=H. Mayer;citation_author=R. De Martin;citation_title=TOUCAN 2: the all-inclusive open source workbench for regulatory sequence analysis;citation_pages=W393-W396;citation_volume=33;citation_year=2005;citation_issue=suppl_2;citation_pmid=15980497;citation_doi=10.1093/nar/gki354" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Arnold;citation_author=I. Erb;citation_author=M. Pachkov;citation_author=N. Molina;citation_author=E. Van Nimwegen;citation_title=MotEvo: integrated Bayesian probabilistic methods for inferring regulatory sites and motifs on multiple alignments of DNA sequences;citation_volume=28;citation_year=2012;citation_pmid=22334039;citation_doi=10.1093/bioinformatics/btr695" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=M. I. Arnone;citation_author=E. H. Davidson;citation_title=The hardwiring of development: organization and function of genomic regulatory systems;citation_pages=1851-1864;citation_volume=124;citation_year=1997;citation_issue=10;citation_pmid=9169833" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_title=Inferring direct DNA binding from ChIP-seq;citation_author=T. L. Bailey;citation_author=P. Machanick;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=T. Blumenthal;citation_author=K. S. Gleason;citation_title=Caenorhabditis elegans operons: form and function.;citation_pages=112-120;citation_volume=4;citation_year=2003;citation_issue=2;citation_pmid=12560808" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=T. Braun;citation_author=E. Bober;citation_author=M. A. Rudnicki;citation_author=R. Jaenisch;citation_author=H. H. Arnold;citation_title=MyoD expression marks the onset of skeletal myogenesis in Myf-5 mutant mice;citation_pages=3083-3092;citation_volume=120;citation_year=1994;citation_issue=11;citation_pmid=7720554" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=L.-W. Chang;citation_author=B. R. Fontaine;citation_author=G. D. Stormo;citation_author=R. Nagarajan;citation_title=PAP: a comprehensive workbench for mammalian transcriptional regulatory sequence analysis;citation_volume=35;citation_year=2007;citation_pmid=17517777;citation_doi=10.1093/nar/gkm308" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=X. Chen;citation_author=H. Xu;citation_author=P. Yuan;citation_author=F. Fang;citation_author=M. Huss;citation_title=Integration of external signaling pathways with the core transcriptional network in embryonic stem cells.;citation_pages=1106-1117;citation_volume=133;citation_year=2008;citation_issue=6;citation_pmid=18555785;citation_doi=10.1016/j.cell.2008.04.043" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=M. Defrance;citation_author=H. Touzet;citation_title=Predicting transcription factor binding sites using local over-representation and comparative genomics.;citation_pages=396-396;citation_volume=7;citation_year=2006;citation_pmid=16945132;citation_doi=10.1186/1471-2105-7-396" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=D. Durocher;citation_author=F. Charron;citation_author=R. Warren;citation_author=R. J. Schwartz;citation_author=M. Nemer;citation_title=The cardiac transcription factors Nkx2-5 and GATA-4 are mutual cofactors;citation_pages=5687-5696;citation_volume=16;citation_year=1997;citation_issue=18;citation_pmid=9312027;citation_doi=10.1093/emboj/16.18.5687" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. Edgar;citation_author=M. Domrachev;citation_author=A. E. Lash;citation_title=Gene Expression Omnibus: NCBI gene expression and hybridization array data repository;citation_pages=207-210;citation_volume=30;citation_year=2002;citation_issue=1;citation_pmid=11752295;citation_doi=10.1093/nar/30.1.207" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=E. Efimenko;citation_author=K. Bubb;citation_author=H. Y. Mak;citation_author=T. Holzman;citation_author=M. R. Leroux;citation_title=Analysis of xbx genes in C. elegans;citation_pages=1923-1934;citation_volume=132;citation_year=2005;citation_issue=8;citation_pmid=15790967;citation_doi=10.1242/dev.01775" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=P. Emery;citation_author=M. Strubin;citation_author=K. Hofmann;citation_author=P. Bucher;citation_author=B. Mach;citation_title=A consensus motif in the RFX DNA binding domain and binding domain mutants with altered specificity;citation_pages=4486-4494;citation_volume=16;citation_year=1996;citation_issue=8;citation_pmid=8754849;citation_doi=10.1128/MCB.16.8.4486" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. P. Fejes;citation_author=G. Robertson;citation_author=M. Bilenky;citation_author=R. Varhol;citation_author=M. Bainbridge;citation_title=FindPeaks 3.1: a tool for identifying areas of enrichment from massively parallel short-read sequencing technology;citation_volume=24;citation_year=2008;citation_pmid=18599518;citation_doi=10.1093/bioinformatics/btn305" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=P. Flicek;citation_author=M. R. Amode;citation_author=D. Barrell;citation_author=K. Beal;citation_author=S. Brent;citation_title=Ensembl 2011;citation_pages=D800-D806;citation_volume=39;citation_year=2010;citation_pmid=21045057" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. S. Friling;citation_author=S. Bergelson;citation_author=V. Daniel;citation_title=Two adjacent AP-1-like binding sites form the electrophile-responsive element of the murine glutathione S-transferase Ya subunit gene.;citation_pages=668-672;citation_volume=89;citation_year=1992;citation_issue=2;citation_pmid=1731339;citation_doi=10.1073/pnas.89.2.668" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=D. L. Fulton;citation_author=S. Sundararajan;citation_author=G. Badis;citation_author=T. R. Hughes;citation_author=W. W. Wasserman;citation_title=TFCat: the curated catalog of mouse and human transcription factors.;citation_pages=R29-R29;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19284633;citation_doi=10.1186/gb-2009-10-3-r29" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. Goecks;citation_author=A. Nekrutenko;citation_author=J. Taylor;citation_title=Galaxy: a comprehensive approach for supporting accessible, reproducible, and transparent computational research in the life sciences.;citation_pages=R86-R86;citation_volume=11;citation_year=2010;citation_issue=8;citation_pmid=20738864;citation_doi=10.1186/gb-2010-11-8-r86" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. W. Harris;citation_author=I. Antoshechkin;citation_author=T. Bieri;citation_author=D. Blasiar;citation_author=J. Chan;citation_title=WormBase: a comprehensive resource for nematode research;citation_volume=38;citation_year=2010;citation_pmid=19910365;citation_doi=10.1093/nar/gkp952" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=S. J. Ho Sui;citation_author=J. R. Mortimer;citation_author=D. J. Arenillas;citation_author=J. Brumm;citation_author=C. J. Walsh;citation_title=oPOSSUM: identification of over-represented transcription factor binding sites in co-expressed genes;citation_pages=3154-3164;citation_volume=33;citation_year=2005;citation_issue=10;citation_pmid=15933209;citation_doi=10.1093/nar/gki624" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. J. Ho Sui;citation_author=D. L. Fulton;citation_author=D. J. Arenillas;citation_author=A. T. Kwon;citation_author=W. W. Wasserman;citation_title=oPOSSUM: integrated tools for analysis of regulatory motif over-representation;citation_volume=35;citation_year=2007;citation_pmid=17576675;citation_doi=10.1093/nar/gkm427" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=M. J. Hubisz;citation_author=K. S. Pollard;citation_author=A. Siepel;citation_title=PHAST and RPHAST: phylogenetic analysis with space/time models;citation_volume=12;citation_year=2011;citation_pmid=21278375;citation_doi=10.1093/bib/bbq072" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=P. N. Inglis;citation_author=K. A. Boroevich;citation_author=M. R. Leroux;citation_title=Piecing together a ciliome.;citation_pages=491-500;citation_volume=22;citation_year=2006;citation_issue=9;citation_pmid=16860433;citation_doi=10.1016/j.tig.2006.07.006" />
<meta name="citation_reference" content="citation_journal_title=Science (New York, N.Y.);citation_author=D. S. Johnson;citation_author=A. Mortazavi;citation_author=R. M. Myers;citation_author=B. Wold;citation_title=Genome-wide mapping of in vivo protein-DNA interactions;citation_pages=1497-1502;citation_volume=316;citation_year=2007;citation_doi=10.1126/science.1141319" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=S. Kumar;citation_author=A. Filipski;citation_title=Multiple sequence alignment: In pursuit of homologous DNA positions;citation_pages=127-135;citation_volume=17;citation_year=2007;citation_issue=2;citation_pmid=17272647;citation_doi=10.1101/gr.5232407" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. Malhotra;citation_author=E. Portales-Casamar;citation_author=A. Singh;citation_author=S. Srivastava;citation_author=D. Arenillas;citation_title=Global mapping of binding sites for Nrf2 identifies novel targets in cell survival response through ChIP-Seq profiling and network analysis;citation_volume=38;citation_year=2010;citation_pmid=20460467;citation_doi=10.1093/nar/gkq212" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=G. Meng;citation_author=A. Mosig;citation_author=M. Vingron;citation_title=A computational evaluation of over-representation of regulatory motifs in the promoter regions of differentially expressed genes.;citation_pages=267-267;citation_volume=11;citation_year=2010;citation_pmid=20487530;citation_doi=10.1186/1471-2105-11-267" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in cell biology;citation_journal_abbrev=Curr Opin Cell Biol;citation_author=F. J. Naya;citation_author=E. Olson;citation_title=MEF2: a transcriptional target for signaling pathways controlling skeletal muscle growth and differentiation.;citation_pages=683-688;citation_volume=11;citation_year=1999;citation_issue=6;citation_pmid=10600704;citation_doi=10.1016/S0955-0674(99)00036-8" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. Piechota;citation_author=M. Korostynski;citation_author=R. Przewlocki;citation_title=Identification of cis-regulatory elements in the mammalian genome: the cREMaG database.;citation_pages=e12465-e12465;citation_volume=5;citation_year=2010;citation_issue=8;citation_pmid=20824209;citation_doi=10.1371/journal.pone.0012465" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=E. Portales-Casamar;citation_author=S. Thongjuea;citation_author=A. T. Kwon;citation_author=D. Arenillas;citation_author=X. Zhao;citation_title=JASPAR 2010: the greatly expanded open-access database of transcription factor binding profiles;citation_volume=38;citation_year=2010;citation_pmid=19906716;citation_doi=10.1093/nar/gkp950" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. G. Robertson;citation_author=M. Bilenky;citation_author=A. Tam;citation_author=Y. Zhao;citation_author=T. Zeng;citation_title=Genome-wide relationship between histone H3 lysine 4 mono- and tri-methylation and transcription factor binding;citation_volume=18;citation_year=2008;citation_pmid=18787082;citation_doi=10.1101/gr.078519.108" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=H. G. Roider;citation_author=B. Lenhard;citation_author=A. Kanhere;citation_author=S. A. Haas;citation_author=M. Vingron;citation_title=CpG-depleted promoters harbor tissue-specific transcription factor binding signals–implications for motif overrepresentation analyses;citation_volume=37;citation_year=2009;citation_pmid=19736212;citation_doi=10.1093/nar/gkp682" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=M. Rudnicki;citation_author=R. Jaenisch;citation_title=The MyoD family of transcription factors and skeletal myogenesis.;citation_pages=203-209;citation_volume=17;citation_year=1995;citation_issue=3;citation_pmid=7748174;citation_doi=10.1002/bies.950170306" />
<meta name="citation_reference" content="citation_journal_title=Functional &amp; integrative genomics;citation_journal_abbrev=Funct Integr Genomics;citation_author=A. Sandelin;citation_author=A. Höglund;citation_author=B. Lenhard;citation_author=W. W. Wasserman;citation_title=Integrated analysis of yeast regulatory sequences for biologically linked clusters of genes.;citation_pages=125-134;citation_volume=3;citation_year=2003;citation_issue=3;citation_pmid=12827523;citation_doi=10.1007/s10142-003-0086-6" />
<meta name="citation_reference" content="citation_title=Five-Vertebrate ChIP-seq Reveals the Evolutionary Dynamics of Transcription Factor Binding.;citation_author=D. Schmidt;citation_author=M. D. Wilson;citation_author=B. Ballester;citation_author=P. C. Schwalie;citation_author=G. D. Brown;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=J. Van Helden;citation_author=M. Del Olmo;citation_author=J. E. Pérez-Ortín;citation_title=Statistical analysis of yeast genomic downstream sequences reveals putative polyadenylation signals;citation_pages=1000-1010;citation_volume=28;citation_year=2000;citation_issue=4;citation_pmid=10648794;citation_doi=10.1093/nar/28.4.1000" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=W. W. Wasserman;citation_author=J. W. Fickett;citation_title=Identification of regulatory regions which confer muscle-specific gene expression.;citation_pages=167-181;citation_volume=278;citation_year=1998;citation_issue=1;citation_pmid=9571041;citation_doi=10.1006/jmbi.1998.1700" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. Wilson;citation_author=V. Charoensawan;citation_author=S. K. Kummerfeld;citation_author=S. A. Teichmann;citation_title=DBD–taxonomically broad transcription factor predictions: new content and functionality;citation_volume=36;citation_year=2008;citation_pmid=18073188;citation_doi=10.1093/nar/gkm964" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=R. Yamashita;citation_author=Y. Suzuki;citation_author=S. Sugano;citation_author=K. Nakai;citation_title=Genome-wide analysis reveals strong correlation between CpG islands with nearby transcription start sites of genes and their tissue specificity;citation_pages=129-136;citation_volume=350;citation_year=2005;citation_pmid=15784181;citation_doi=10.1016/j.gene.2005.01.012" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=T. Yoshida;citation_author=Q. Gan;citation_author=A. S. Franke;citation_author=R. Ho;citation_author=J. Zhang;citation_title=Smooth and cardiac muscle-selective knock-out of Kruppel-like factor 4 causes postnatal death and growth retardation;citation_volume=285;citation_year=2010;citation_pmid=20439457;citation_doi=10.1074/jbc.M110.112482" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="oPOSSUM-3: Advanced Analysis of Regulatory Motif Over-Representation Across Genes or ChIP-Seq Datasets" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9.cover-source.jpg" />
<meta name="twitter:description" content="oPOSSUM-3 is a web-accessible software system for identification of over-represented transcription factor binding sites (TFBS) and TFBS families in either DNA sequences of co-expressed genes or sequences generated from high-throughput methods, such as ChIP-Seq. Validation of the system with known sets of co-regulated genes and published ChIP-Seq data demonstrates the capacity for oPOSSUM-3 to identify mediating transcription factors (TF) for co-regulated genes or co-recovered sequences. oPOSSUM-3 is available at &lt;http://opossum.cisreg.ca&gt;." />
<meta name="og-title" property="og:title" content="oPOSSUM-3: Advanced Analysis of Regulatory Motif Over-Representation Across Genes or ChIP-Seq Datasets" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/9/987" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="oPOSSUM-3 is a web-accessible software system for identification of over-represented transcription factor binding sites (TFBS) and TFBS families in either DNA sequences of co-expressed genes or sequences generated from high-throughput methods, such as ChIP-Seq. Validation of the system with known sets of co-regulated genes and published ChIP-Seq data demonstrates the capacity for oPOSSUM-3 to identify mediating transcription factors (TF) for co-regulated genes or co-recovered sequences. oPOSSUM-3 is available at &lt;http://opossum.cisreg.ca&gt;." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9.cover-source.jpg" />
    <title>oPOSSUM-3: Advanced Analysis of Regulatory Motif Over-Representation Across Genes or ChIP-Seq Datasets | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q0wst1" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__KPgiwEdjo5O9HdRryHqQs1TpsV-PeBb9B7sGBvwM9FQ__opzZWgPNgQzoEgfATRS5qCKwBw_6XcNjDHoDVOzQv8M__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__YUqwBlv6bNoCToiBWgAUSPtppdS8sbGfzmAH3qEDcI0__Mvk93h3ScFH5Xi9WhibVKlf368fHw-V7HWqWX2QMvtw__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XNWYaQubFGWYWZNytUHkWs-b34LV21DsB4s_6gchI4k__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__7cpBA0GVdrMqkZGGZEW-_SPF2HrbTCmVg9-lrP5l_AQ__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__TV2D4R18tPiSIOmeQ3vyMqHObN_FDmQ6EEIrfE5sAoA__c9BBZFy1BrgGgF_Auu9Fmysb3Z7J56eJGYvMcLQrLks__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q0wst1"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__OaHnYyci68QTwctdRYR30kEfu0eYWBrLyMqy8THv5Ik__IyfJQYM1q1QEPwYsK_Eslh2RQkxpy6yo-nR0p4DWPQQ__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__fhq_kmKS3zjIxlsAa3GE1ubLajyXUgXXTXULzAtiqfg__Lm1_Ls-bdpbWRyl0q4o0mmHyxB-139AJ68dE4zOskLQ__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"woRnGB24jr17_gLMoA0YRyvPm-Z1EuFPPBMfG9LnAsw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459330","apath":"\/ggg\/2\/9\/987.atom","pisa":"ggg;2\/9\/987","ac":{"\/ggg\/2\/9\/987.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/9\/987.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/9\/987"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/9\/987"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/9\/987":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qzOACHfS36rkpiiI7btE\\\\\\\/4GvqalILTO+VZKKevIg0q97+fqyjTi92LP6PW9WmS4GKPxtu2ubRBWS4UXZhw1ZqzrPsSfCVO9yOwNOlIf1NZfy9\\\\\\\/RH3prrNkCHvDh2Q\\\\\\\/CQG0b1mZw\\\\\\\/yzx2zZnp7oaZO86Lg88iTWYQv83XfFuwoVGcBiu8mYXm9UOSe\\\\\\\/5d1khb5SZLxm4ZsQzCQWZOCxwhFbjt0aUwCsQdYFFJ5wqyL8PtvbaCWvKtDTqFWwL9FbdEomU8EVZ26jYNVG+xwru8R0DLUeHqQRZKMvB3IOb8eAg05liOrWt9fIwqKFAIrGWdkypRAxQX\\\\\\\/tQ7baeZg1HfXUTWDRgR7EL+55CLtn0zVXQ1p1wuu8dl+qnr\\\\\\\/CKqoP6RoGTmeaRAAsCe4xDZsyWoljvPqXEztGhEk0skvJSgfO4tpt0MfU5Wi+XTzvefCVA+3FRVCrM59xp3Jc\\\\\\\/2+s8kbvW4BQ7jMGWJmbjomBdFLsRdaXdl2R8F1WVIOhkCuWC776SE1h3V7fPZyHPCvJBkbYUPpq6Bj4Bzxjk5TDx40NE3CmTkY4OXsVVNTsTFC3F7KbCHMD\\\\\\\/4cphNsmj+fkT1cWUirZFM9+GLsCjoC\\\\\\\/Kz3t3HvV3kL6mANu6fjbVybpQgeAAQ0RUNNXTX0+HhZpEPQkDYStlmrnwAH5zBUZSfdYM+GZLQixNZYlHrjge3QPI8hylL+FsCX8XYc5A\\\\\\\/qp2s8Lwk4D1\\\\\\\/a0B8Or1CIN\\\\\\\/8Y3vuI9Koh6UuMEtdprOC8az3mGQcha5o0DYBy9bzw10+we8LM0ZNTFOaO6kPeygdO+K8\\\\\\\/cn6OpcRH1uKeH9bIsT2fYnHwavGnI68vtRc9ljm4znCK\\\\\\\/APJqD9Wl7ZkpO\\\\\\\/dblrQBV7j9Xq2NmBt8uZecPhOKgtiiqfjy8Iu0n\\\\\\\/Hb3zPPnl+rq3+M95dVYFi4PY9z8eH8hJPxlELRe+Pooe7wCA0yDW\\\\\\\/tojIwDk2s9Q5HrZjC7fyT4u840kopHLp8owJOE13azlQW4LnKLRbVOdn2bIMKfYC\\\\\\\/kJH0zQ+S\\\\\\\/GExUluu0rcKPBfmhClPguz8+lRnWM6DpNoSVa7Ezg77Du7eaQfSb\\\\\\\/47ZFjPLXS8yYKKNiM+d6hu63SgsnS2KeFIoXdrd7gOMpkalWiQs6XaVMnz4IeGoGw7uBNpWopFDnBi8ucxq57ihdwSQsR8RGPob\\\\\\\/dfL9M937eIUM1Ey7x6yMnbHzoT4AX4Zgo6LM9PgeumonGrJftvTykuPeZer4iieFyI04dtxXNDZZB+CrIRb03Ioh\\\u0022,\\\u0022iv\\\u0022:\\\u0022hXovMwzBMtuEoh3O4fcj3g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220df8940422ccfcdc3d6e7f02d6682d10f1c90b3b0f383bab5c5ec661f57e97c3\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VbTqMYNKTk32wuwEV9myVN2VifcxuDv1MoHEZfPOd0CiyvGcdslTBbzZ4poF4A7P\\\\\\\/seby\\\\\\\/7SYPp0aC2KfBCaGhKRg87OUBS0FTXNqAiVjwazJkWVNQntaOLnx6MrVeF7GjjoZNBxGWhYPfQZbQ5mBDH5WqFNMUJ8Guo1YEU\\\\\\\/ia47G2wcODibyNvA9WO93GrNaNmzJP8R9md55PisfkofMiDYisWGR201Y2ba\\\\\\\/NtVGlSbA1NGaklYW\\\\\\\/Vb7eMagRxJQNF9N+7OZH4eiGs3MB5pOCrOk6d+N7JZ8MC0jCnvgTv17XCMyTNzqc2H4\\\\\\\/GE5Y3p3T8yEiivdbKmebJtIDNiohUmTeZkS+alH7mc5wfnFPRD+7vDzhjFlzTOOA6FZZX4XBCGL+OOFSy1RKBinmMctIlooOY24CPz7gHD3tAoyJDLylxcn250nt74yCtONrzxb21cgv02PQSwo4QD\\\\\\\/yBEvE\\\\\\\/hxvyE2lpOnwkWgSoCdvf\\\\\\\/cDOa9LCdbXb7Zlq7kBG1U7LQK193qOmr2C8W0edyn8tkEmrvOdLkt3tDROApJPdI2VUtIHoOH1bNhm7PtlP\\\\\\\/j9ojDAf5WG+7LwHmnrGuKm5AOLr09wE\\\\\\\/l8o7Xu+QA7\\\\\\\/ZBR4WmMefFw3QGBqEA9zCQL4+fjRab5M0TS5zUt1daWUuZ2awLhPqiidxEf6+3AugPDP5M5WWVseXbVVMnGJ3qQ7KuQwibFBsgPtmP2Ea6rMnV8g8Y8lLY0XPtPlYMKrsTBAMBcz2ThLlHLAm4cud5c4DEgofMigbccC91TSdi+vPadiU94kLAzRgxD+FNzJ+9OBtFp+j17StYMcxp\\\\\\\/mCc0lLB15\\\\\\\/PJB\\\\\\\/9LiGa+eMnzGvfgieoQC4rAzyYprZPJ+26EkDC1SQFm2vqmZ1RVm3YwGwtT\\\\\\\/Jy0kPNVOx2OlXcYIeZLIP\\\\\\\/1a8gphqofx0uiHd5mj74hluAWzvOyr8trdEP2IX8O12W2iKraPe5B6Ie6pSjv1Pt99hvXCGt1gwjgdu2XxMhphub58pxJhvH9d6sBoqeE1fC1pvJ2ZogXhbwqgD9v5wF38TuF47dA\\\\\\\/VhY0EAL9Y6s18zejukcEbzbBbHcSebp0dOZbwQlW1BBoU6KxY3DwcSQP11AzY91+xz39Ikaly1ssZg7xrtCz8fPhb7IRcJQhtErQJs0LP\\\\\\\/2LKizwumWeOKcQOOpghw1f7yTnLtRIQ9+DRDc0lizSyK4WNIjh1+jKIMX7MZETEnN0r5ia8b42IimeduoWAlv0RIatya\\\\\\\/tOvTXaHRKziY9bW9\\\\\\\/mxewsNYGa8tiQfBKvvc8PX+zkmo9+Cu6NsL6NIqVYYl5MtqEMA5PPeyaqCCVOwyMEfcPBQKSJWWQ+l\\\\\\\/rcfQ7cZiAcWyh2fAWbveUpzXTOXgoVlpnFtRsa6MqyNRsBQBFXF6c3NKY5mgLtTsm2N6HMDywIA9KiX\\\\\\\/\\\\\\\/OI+qRev\\\\\\\/apAaAYSYc7JGBRe\\\\\\\/QLNjDebvcwTSkZhqR9HQd+K7rkSCWYCzMJtDBGVV2DfFpL2yZ1yUDHKZZ9gPmMQWNBBY5NuG8A6gIImDyQRbDEIBBJddFOJrFUNESrAbGtXf15tU+NN70uu3PfIDs\\\\\\\/MSg6O38YzLxyOk2CBfy8qTc0oYZxEQOLVVZr6HKVPPSBwKWhG5FQ0s71xHFgDsocr024xb3O+9uO+9q00zT6IWT\\\\\\\/6TLKmKsWGgMeGnu6FC6uOISwptLmzDoY7DitFAzS3dDAoAY\\\\\\\/3aqRJj8QbV5jTim8qvO0VUVVTglCuJfGkD1kJxnwPdyPIOGiQfeZcX3lCL4R2q3blprQ7fhGi\\\\\\\/dbm0CMG2aj+aNNivMEVYg1HYl3pRFu4sblzzZHJOYNMVbSoP3dDYcxrOIZ8phDHlOIOTiFVuKGi1ReKl2VtsGqHWpL9Rms2bcbZXWXeQdAZUbFcGkV5Ad8sLsFQG15Bs5PXwXYyk\\\\\\\/eKaExC6N7W8173fcwb4e01\\\\\\\/uTFAsdESOTIS4\\\\\\\/Utw4CBesfkEH5DFiKQLxwX3mgqOB3Fg9XM8lMn6WFZ4xxgwBKnJFo3mW4ZlSkH\\\\\\\/Ca+oZRrH+dlBU6fN2upLx19LJuw8ZNRHBfUUDYmtZapnmAvzGbRmA8XgrEol+Xex4Ty047o1eHhURuyx0yujLHHCD6AWiwgYvh0mDB+fJNWEBa2cOP0vbn\\\\\\\/CjDH0NWJeldsZo098iwh0Q13kLdnbUYAoEil7PN7Ohr1aIC3+FlV3ISdj0X6l038VmQ475z1cbl7E2sYFgo7igRVVZ03MgLjYSzNOMMXzrCHw1CP2QvnwWY8uGdQagLAslvBlJyOlHhDBvVnfIkYqeSJMr+x8K7iImNTv07UM7HMimIiLYlH3vgezrxaDAyIqwJqATWyiea0a7SwWQVW6qGvopEzzjY71wIEgFhwA9aqoQO10olTVmfAjP\\\\\\\/qcagzh\\\\\\\/RG9LrxZcd6U38hvSys\\\\\\\/9LK11\\\\\\\/6nZTxkoW9moUfx0AGAhvKqSamd+eNTOG\\\\\\\/xIWI9bn7o3YXTo7lOI41GMw3iqDOjISQX8saub0OX6s4gNlCRkThpmUo\\\\\\\/kbgHscKajCQvIci+BQHWCZuu2BcXHRfzgIAFslkmsrVw\\\\\\\/eDqPdWH8hIr85y6p52CmK2wUVbsM1TXiOaAfMzE0xoXFXXXeW+CCA9L\\\\\\\/mPksKq4RxaZ8wrNE3cMVGL1X4g\\\\\\\/6NkL0OToPxRhqCYskCRkqICLP0KBvPVjiSpYSRZwxHuRApCEU+1R3umJrjhVCxiVembgdjWOQu5Vd2i+J74Nj6LT1BW0XdjzmBHDY1JDoIjFhK\\\\\\\/QuGJRKyJV8tYBY+vIGqCoe2tUhfu0USqm6mfI01KH\\\\\\\/YJhVRTGHW3lcEFRuGh2vpaJc16hQpcAbdAw\\\\\\\/w4JPqQ9yBrzkWTABeFn+k3mllUqWxElWiU1ZIq8Cqc9IzElyGZyOrg8t16y4hNUljo41b4an82868tdPo+7josBF6fvIrXkqGhpGmzv5IHPSP7QmUwfuh8Pi8F0Zgxn1t6YeDgfrlfZqAS8Tw4Smwa+G1XfO7IsWFc4Wph86KabAk\\\\\\\/ECrjNGooA6MsF1XKtS2wRkCb3bq65XR\\\\\\\/mCqfDzDA\\\\\\\/3OoUu1KJ4rTwvgx8Z98Qqr\\\\\\\/7ZqNC0JoIeWeaQBEHagDZOMVUAJANsYJpVItRCfW35aRv8XJVzaF9aW9etJTRLQ9EFfcMgjfIrFyCkPVjzE8yQd\\\\\\\/28ojnxk8B8QT1bS\\\\\\\/RJjVT8h46bSrKIhFQKweKKrf5MrU38D+IxksRBnWzbBvHH4\\\\\\\/RHfhWp0BuZdiSOA24ZngFWYS1Y0ExqC8yGWh8rJHYwpf\\\\\\\/KCxLXxUx7swz+dv2hS7xpCCo5g5GRlOq+cMGd\\\\\\\/qq5UNnZjt5Xo5zbTnKl0LwXlD8wYzBbgVAGqsMg8rH0CeLfIHLu+0PuQPzEPIU4JT20Bu99QiSaoih07In40atsYwwkjptmJP6GLl+dwRBXcufAEyv2Ph+0AvWrt64oFxi9wM6YFumdcneoBFo8jBW1n1vEjug2a6u98lW2H+2QnzQk8RrFey+CN6V+SqqJGzvREXyIa+EIF2TrZSd89GN6b0pyuXkL7\\\\\\\/rPi5zPsutNRRUeRP6h1+NDHbXS4Y8z9r1c+h8blia7qtGtVdTn1bUjm5tq41pQXJd60YppWiMJkVjmqgIewN0fLjWbeu0Po4o07wkofmlRz5nfhBbSGRR+qi10DmWlvm8MXtopLKPLgCyfKu3kTvbvACEzTj3BgjPEXiLs09KMPDUun9gJU0Bl5FFiUmNAMEwvLfm99v4DuRxGboMgZQx5+chnngAtYw342PaocZeCRxLwkx786\\\\\\\/eORIj0Ccj0jx9a214fGsHuHpColyJVEDeJMTBQQmidzZ+wO+a\\\\\\\/5PvLHAhXV6ADrZm+MuHn0QnRarh\\\\\\\/SB9VIIT7s87+fHNy0ketT5h6XIAPrNLMQ\\\\\\\/qVB\\\\\\\/Rqi1fphGOcNcRtkxLI46F59u6v4pU\\\\\\\/JkbRmMNOdfzU2KvtGh3HG75puQDeqFYIt1\\\\\\\/gAhKoBcxjvdfQQghhPtS\\\\\\\/tW5KO9dbdYYIxMaWOVAD50lrFJT6S6+vLMADnyMLQP3PN0Ty2Zu+25zWSx8D1Kb6vYM4LoiNpeRv0tH\\\\\\\/ZYG1ACX\\\\\\\/0UqeJAvi6n24g8ecjd0z5odC5Y15P\\\\\\\/RDcVRVEIBg6QtKIoi2LZxuEIoXPyxmBZD1PSBWcCYNmNCtZAW2WF32y1lLjSksGzfDKKGRfGsJtNfptN1fGKDUxK\\\\\\\/ZvEqh4SX9MuW2ZfDMngGHX0J9Dy3khFDCY55fJIKx5lWRGydGi5AikETJkXtC5SvMA2JdBzqXfPiXR+2fX\\\\\\\/kHfHXsO4DrJKXTNUWIGrn1S3Ww19uQbnJNZS1IU69Dcio+S0gzcT9fgVrko8evKoI89VR1kH1AIW70qQZBbeGAszpokJqAjomiKeu24y64vPXRwn1XS4NJiGJ9A94JGetJcXy3L7pLxIiXPZOTCtxYB53ItGWzcxR4CVktjoO\\\\\\\/1Fu197J\\\\\\\/bvnLb2uUj1N1\\\\\\\/8wX0IeU0bK3Plp7301aaVcj1qG8B+sFwwNsHcBCUugokAEg2pAQ7bXj9SA4aKKTVRwubWSwF9ObVpMp4OiZdgytHBn+Eq8\\\\\\\/R6Ma8gQuVRleEf0Pjvk8vq4iUB\\\\\\\/drucEFeuwLR045Kdq4upgClQk1gVgx0A5sDby0bR5077fhyQzj1pnasfOn2dvHBIC8bTjG4OePBcI\\\\\\\/tgSIo17qzprGP2nw0q3wUB9\\\\\\\/wV4ry9Ajei1B78BDLQK1gtGg5o\\\\\\\/5D4d6x8tXg98qw2gmOWk1OvR3IS4Pf4ogXFtCiD4BES81PtVC2emhLUM1MZZ7HKEHyUeO5E\\\\\\\/5z3dGivHTDceQ2ZhynnKJ47Wn1WxUlf9\\\\\\\/Uy7SIUfL+2hjoPqET1LNThswXKmHvWpHQukWdWqdHRHDCuDKNLJjE2W7RnUCGR3\\\\\\\/AMWIyAoDfNp8+wXg1F4WiGEYZ8zXJX84AK+LWZQiPBkhKCpfEq\\\\\\\/WpbJqQZE06zYkQe+y+MTISrFCOIfShBDG7VDqLtUZ5vUIBtwRyZrpHCcunFNvhdkCOocqykscdN1ZCT4ONikGWCvVbZjDszdlLbP7whqS3lbeB8urPJ2bnAz3+iBkSHwocLWsBqoXRFD3w0Y4DA6lRG9zXEr\\\\\\\/SGZdMzINQlAZZuN8XheYa7HTMz2uNJtl4o\\\\\\\/j1C+a8RbgnKLKEGGCC+h2+8rFOscF1b6\\\\\\\/rgOEfKcmzYpFTxJPCI0NuK8KJAU+ZCSV2bUGwGKXBNyvzAZ6ZoNVl+hOYNCJ6I9LJjN+U0lzaaySzPpxjZ3luZnq4JDj6ppiVXEkdh59Rrj3wE8e1Tut9XuOZs\\\\\\\/+\\\\\\\/mA8jj5jrsQ2YTr8+IhF6mSY329C652TitS1hLG7Gwwc9u+siezbIaOTmNPL0HyvH4wqq\\\\\\\/dmNWvheUDW4rLrS+rlW4ZAdnV1PyBAIa9SLFtNktuacB8rjeffPFrqfgXzC4DYC+OXtA9LrHBehcKe6PPIpo+mXt\\\\\\\/tRQsvp9rfJpF4JSsJPJ38pDj3Smepa91x34CjIKyqNpMkdf8wrglmoxg3MAuEINg\\\\\\\/4Jlnb19+jZh+1ypDot5myj5AOr9o1n1GOkwb91czVi8gLbZ4hotjpU+E8pHMBqowDLHvG23NTqDlDwe502O01ebiyZwDe4qDaYv8CYQsmRSakolIZlqANso4zOG9K3kYkRd1Z3O0rC2Xldl843LFnv1rW61\\\\\\\/IHW6+bM\\\\\\\/6XoIGANSrGF25XMGs+raodJgGQixGoYghFiCnvCh4kKEsqi9+nrqZbATMparMmnTRHeJPVNdEemyGVb9pcS9n1w5yVGFlr2Th25n0PeitpMswt5mGyPaZnCONnzfDMtNQtVuFwN+lFGPaEoBA9J+kQmA2WnRWo\\\\\\\/eYR6EIY4BRcrAPZHnQSfJx2zTFTqyf2eZp9rxGeQJZDqgadyJXZq6gvY4T9kWKwlOmNl9dBSBNKfSnNMjls1mssw1a4sPDzBiGM2z7sJZUYzm2K5CeW2n1Zy+\\\\\\\/esbuz1TSFOsuVQfCuUhEu992Y3SoptQGh0bVDp+498VAea2HF5xJYfkcr+szvcnZNPI5esKOfEDR8VqHNTYHRHhO8+2qfPQUztxQtZ1ccAhLB0+xE+LKAYHxzaU+q5\\\\\\\/oeX\\\\\\\/lwSOQDr1CquMHQnkvA3iL5c4I\\\\\\\/pFilNmC6z3rgSfK4vJFaN+e9aIzmWsQvGoewd8ePbJMlTyPx\\\\\\\/qxQdaXp6jFOv1XGBy\\\\\\\/oYBS7a+T0HCbxjLeKC\\\\\\\/p+Zav+dZXk7hg9dtHWvtHWhQFnKy5rBUBQUlZJhUknaOuB8lVoPrkNW3OYDmws1KBj5wltA5hMKIXf\\\\\\\/MMj0ML4ZnGCsCfx0vb\\\\\\\/VdNnVvmRekEavmrGBm896QrPL7o5LjTAAnyHxR1wUcHwJ8BvTcPH2rZ\\\\\\\/qjaIJtiUQ8jT7Zt0+hyPvP5cRizUIo9J\\\\\\\/trkCv2k5ql7Z7Uom3iUtjzUyPX2jT2QkVl0Xe5sX9FnVCPpAknaV9cKysquGQuvS5DSl7XQI77dp5hvb58plKrMwVtXYB0LgdMD1yyqcpqT9l3bU8DNnuNM1np6LTSQrvLs6y6oGnjy+SOEMJWII5B3MkDsYkgb2qk9In5NPaEwHLnLLf\\\\\\\/6kYX1PVHDS\\\\\\\/YHC5J\\\\\\\/WT5zhj57gi5lOHZju8jxU2+Yx5QX9NbPYf1HnbfWt+T1t9iMIDWJbb0vrwY7p232pRvaPOXlI6cH2uLhmSKYntYMWq4KT3ptd4aaBHn1d6XeqaWUGYit+KYbkCPX8ZOviaa4rqrakxtfyf2CQJjZ9ybJzTCv7lMDSSbCcp8Fkk0ECc1aRwskYnO8WzkXbww2IxXNXvFqx1hwTmF8q53KBQaashGIAynDUiaZ7sdDedZk+Tx3No40+T9HQ6O5Ghp5057IUovYchGov5joz\\\\\\\/WJ15rqZXi3ka0uGdmKj2drGqsCi1mVEOu+8sdd4VQ8Ed9145xGeckK9ZSITeEulIVUI2vzVJTN6SEKv+SrJAB8jmxpBeY3nBIhn\\\\\\\/iS2ajfbIxEljxqzWctPvXZty3XBlhl0pC7amQsZDrfxLZI3VTkNBAZQNJubPgH7rYkkclZV71YKc0u5Th1Egup5AzFMBwFWhUQgwyQ5sfr8Zo2oh6VUcqbcvZPukcMhFeQIPexlRpLaxWfydDVsPQrNapzamKms3hNhF3oWuv82McD8lQQmI6Zg5qIX8uDF+eytb2RQ5OKiBsG+BB\\\\\\\/qs1VJt9gKQdefSCOQglBBLNJoyp+3rSBi0Xnv44HTkFXd1otK1leGvPXCA\\\\\\\/uNfcNZYBjkb+BM8AOltTlIs5lGgx49t2bLnOHXsh0wDgpapaG6l1CV7\\\\\\\/ba44l9ke\\\\\\\/Cc\\\\\\\/7osDFrFy90szDCeD\\\\\\\/G2u6LvAvbmgZpBfQ0sCjCz4Uk5qHKlZTmpiJAZwwX8YxrR\\\\\\\/E5x7kqQ\\\\\\\/lSjw4yN4Xw967QOBOOaJHgnLs4QrVbMWIDY7AJG5vgsqliUWIkdNT4e8BuUdWhikyS1dXwWwdy+fsfNX1lZEyhtL\\\\\\\/LFEO+1jofCBxkUprcchr2w9DX48HqU3Aj0ggq3UcZNCtIUIv48hMrH5jUKRDNxap1VsL9CDeIuhxt\\\\\\\/VrcC2p0j7er4+7F33\\\\\\\/ynDf9ORFTGnpfAseRS6LZKptV8USjOB3G0uS+KupBDEkBmeqr+QveGxtqUnjEOV2un8BFnOwlUn+mpE6SQVsYLFw6BB3CKW81iwA6T8F5dxbsJgWoLfzyIwZnVUn\\\\\\\/0iQbVRdvAUrnzY7IqgRZGE7TaBLBbis1\\\\\\\/wUNEIZXo0e7KSKqVXCdzeQieX6DLibsJghOBRd76ZHHv8CkgwuThM0Nkoym5c0M7dHzLvTcI9sEt8UWRsfUeYHG4aZ4tXqH2Yk8Z4eNh\\\\\\\/YlrFYURuEEbsdP2kBcQ+r2YoplxjEtvTOiF9oJkJQMgKZYwrI96s+s7LdfUZwKstTpB9dm6RzVHuptnwh1MAD5AOdB0+je\\\\\\\/xiV\\\\\\\/Pg+\\\\\\\/BFllN58eqQN9ZgtrkFsiwGs3Qc0Rfq88wTWZL5A7bBCK2RTFon6pQuqjZxfFnKglN0HkQ\\\\\\\/HNd8Uf6jj2nAxpxGyJm6QWyjPL07sARnE+ECyqTsIt\\\\\\\/tCN\\\\\\\/+LUy7JzO0TEXjsKEzHJav80doq6oV11wCwAoAvEzSeylZ7jXXOvalzzZUgEaevUwb612aJivGdsi2j7pUhD+HMtTfTtoh9PK+DcGKB0iRtoLriezSg7TBPHOMKAYRX5R7kttiE6BlxiLh8iHVf+rOJMaDs4DMErP0GFYPZvzMFz\\\\\\\/3LpzBvfJin6X\\\\\\\/68uPNsJ37x3Al8x79igsapHVY3n2p66yfoMXmN8QN\\\\\\\/\\\\\\\/JiQyLtEtXM1kFGVy9MgGUcBxR\\\\\\\/Ja7iPSILHwxqEKprmIWLwMN9BOHp+GUGSBfNDoBBzfW90j96EAsSoFUzn3fC6xZwCTTdrXnYtP5J3gnClLE+h43fk02d+0HCp13lYdOTHKPkWyS4D13nIUYxCtItpITGTZbQMwoK34Ii75nhg9eaBxkB7IzOH0vQOZZjlfqrzz0lI\\\\\\\/zTrAyUMVJdvy31V\\\\\\\/2FGLODMEl\\\\\\\/wTbecry5WrF4QlkbpYygkTGi7oAZWqsRIVJ4HzaUfm+bFdQ1rW7xedSSAQJ+gia+4D79XnI0BMylazcLxhZwFER0dffmwWeQDel5KRR4tgPPqme3jP+5YsdguOaIuRFZ9Fo1\\\\\\\/cmFteNspJ3QdW95K4JYpkYmBopjBIXoXeLyqsQmmhks9SNQKkQ7+vhfiRJuEvOeudNfDUma0BoC+8z1hPVH4Q\\\\\\\/2d6kye9bJknZ\\\\\\\/6z3GNYM8Ivn3F5iC+vmm2E0ZyE3jLXfRlirtWSI9FnmXfoGXM8B0M9NCdX2ER79epsKUCzSKMpNo3MLkFyjKSNOIdHx3tJJLhg4Esre40ldsKdO\\\\\\\/OW0PqQ8TlaJ3jfAjHOHl+iRNTq24w9+WhXeAMqzzOcCnmQHKGrWUpiXs813PvzK2bUbcNfExA4mTsPRrSiX8UKQDHObF8NilGwhhs+Ab975tQxAk7xI7SBJF3ZtVPNdIHsByfSDl65cWLWxyMeW5T0C5pK+T67DOeWV3twPLFCzCLDic44w90NVQDcUoifcVBKAVgOQ\\\\\\\/0oH99+7si7KdcIq7ByPmbH5Q0RiZMP3zmgcjisKSrOf6FCN2HBsGWAubJMrb2VPQrT5J8K268qi2nBxwErb1NYHI60uBqghuCmQRcSooz91xYD1\\\\\\\/QOIz+Ww7k5mIOkd9euS0Hd4kunARx43MQ3x4aZ4aLOvxAnnjyOTkwDmUM\\\\\\\/\\\\\\\/r1vNEeyGLN5D3KWO\\\\\\\/danpbhHcEP0xGVUfSvma\\\\\\\/\\\\\\\/UMeI2CY2QBfHNPd\\\\\\\/gcPCJ\\\\\\\/8HmKecL2fPbOqnipo5GMgJ+3afZQfwGr79m1pJDGuPSnli7fNxFw8n7f\\\\\\\/AV+bO\\\\\\\/yJ23oo25Uyx\\\\\\\/bE81RAXik5xJST5MAKQfo+PEjV2M4f7NQJpYUsR8xVR530dFKK4w5TzPR3gmjuuBlbnkfRlf4g7xg3XW4oVGrjYEv\\\\\\\/bnLcHLHJ1RaVdX1IFXnldwJWCiSgXsNNG1t84HCq\\\\\\\/IA5QJNm9VgN9OC7ZjtnRjDBhzRrrTtpWK+Q65q4oEzxqYWv9sbkKSnlFrYGM1ORc\\\\\\\/hLR5C\\\\\\\/5V5lguOTMDDLXzLfuf7V9EcGUv0RsQB\\\\\\\/wIbmHRF1YLR7oJu8T\\\\\\\/nu3H7E6NjiEKxGIXcvNM+j5OCYC9M03GNjV2HYcADGAPGKKRmPyBOAgC4KPdhi2uMMBZHEKHuUIQ+43uLx7IRZgcSUA7Oy8KDZYBRCavQMAGe0oFbrCAJ+3hklwOHXjt4ZGgeM4TcEuMazIH6\\\\\\\/mXpwFW+DPM9PM7QEnvh7Me46knWoWm0C0g6zhqAo1z+ducgEqLs2eWrlKVHlOdT1B4Snb8MwL5AKzCXwon8UjBum7miDRXm02+0lTMtSqgBHwsCI4C71aEV8W+1njRwHbxIa\\\\\\\/9LLrK9elq0GW2VTBbjTJZH2CYguOrSDp+QUqiRITb1mO0eqRoaXdKGQiyvwBvUs2zRxHRyiUmLTszop018uAqJ1vZEZZ\\\\\\\/cuoGaURLLAy8GheOiAbgcauMPLjmyerfvOciCQZcUmrJLYTGESSiI7BBmrIP5vzGWzY+NV\\\\\\\/T7uur9tmrBvo3ykVgnVgbVeED\\\\\\\/7aoXbaV6DqJsn7eUc8daz5OUVV5syozcp4hFy7V\\\\\\\/H4CjZxrLtRIOMGcX5snS0yhScmPfGwUEr6hDp9TUS8o5vjIxLigOM\\\\\\\/D3c+ZXy4HLjfq77epW\\\\\\\/h0+PofNukboWKq+b322qNpWltreeKTDO8p7PX\\\\\\\/nVfOBEM4DscDXKCNqdpSLd9gN0\\\\\\\/TMJRSFD2M6VkqypD2b5IUu5BzQJnXyC8QdfXZdOkay+x9ONX1b8kwQ+ky1kpj01dp9SgC2L9hA3nTXiOmdy06dWrHf9ryCYD\\\\\\\/0dS3sqtjSKKZE6Wx6F+kGjsNRJBq6+sELqg31o3U5H0S6gd7RRjb50OXriWojCTA7zhkCSrdMvEFITAyK5nPqgsaJp5AErc3yYTZ4cbL11B1nn\\\\\\\/eiuSHQq42D0cb6JjgHx0gTK3qxxDA7+uicIZlYZXHfclwW3v7PdXvLWxAB6vIZcbOG12CVyr\\\\\\\/U3e91B48Z6m8RVY2T\\\\\\\/dYh\\\\\\\/64vOqMdTR\\\\\\\/sLHwMlt8tUgQexyg+6Qtcz29CUo1qCTf2ULO1HgLgxLUF2Hb8p\\\\\\\/w9m2ahXTxTZ5gM2ZT6D1JB4YTnP81cxrGRtIDwpNiM4qlkJM5PZ25nCYECnrFSxYgN6RQu3weu9jTNqbgd1Nw76viItK9vpSM8G1ceO7DcPAjNlg+Dqvd7QIyvNQt6uRyj6SmxEAuuYVgzzifM0v+RFmMYyf20KjMTHfM3n9ByoMtjPCfLr7AQ8hROa7+2n0im6n2l\\\\\\\/op2jOp6dfVigMm0fT8pBfKO\\\\\\\/f\\\\\\\/ZKNtY\\\\\\\/tkNoALlCeGDXseWVpnSTF03BXjujJ0M2YVpJMOfLBYObx+wGAcKf2wGcgPFRjRw\\\\\\\/FLK1TLZV8VYsafNXWlLw65WIvlcJuehc535gaYYcXpASg8IPT3trZftRDH8bFc2RgcFPpuT0D6B0wCPljDDorOLBobD3OavdlNmjaoDGZu6o0lzzC3UNEdGPUmxyu0TBzEuQBU\\\\\\\/+ree0CwpNi9vBeaBrLSpRSKRgU5CBatXrz5SloGBY0+Ds8sKVESQcgbi\\\\\\\/WpOnggzhKzmVAHkDfKIEZsVXdLZfA5ryJ7\\\\\\\/Tz4KniE3yo1tK+6LQm4UDGxeoXkz4Nqsz1QsEJD2fcDW9SCGFCyEGftMzyfSWlNuKystRAR5z8We+z24RGwenklfULddb4pu7r\\\\\\\/2UTbW4pstfZzPSOEiahvnLvCFa6S0kClVtuMDIDmoQXcT6PkTkxi6Rknxf7nhBU39bVJ218UkB7k3sX5wfM6CeMNuLo8FE3LaWWLqFt\\\\\\\/2XfJzxindxUuFR8xiUVph9QHXtUXZteLnAk6X0IWLZGeGVpRGmDPD8el4wj9j91uaF1mUZAjSK\\\\\\\/8mZTKExRIC\\\\\\\/iMEq2uQI4OkPLbRz2sxDz4JXRqyu\\\\\\\/9ylD27kjWYT6yUJY2fpn5HHWZNxwF3OKIkfLsKdGL+s8bHZvYKt2WhmeD8252yzt0wg0ky11dMftfG2ol872PXnyZDrzMHwamJ3YGlEQDO0oWRwZ9wAuZia9ykDGodFJJyUYQhO\\\\\\\/FGHw2mRxqET9z+PsCBujS0LhxIO540NrQAPTC6CFvm9wW0W1e7NeMqsqNAgmmE3RuJuACFtGpbufZrdRMWjqt5mK6Y5lAj6x40HLKaj3p8HZpxwT\\\\\\\/wr5OMaXWYlpR7PiOI6YxsNMD7JR9wLAxTm1IWm8W9l7tgGnDu52l5sjky6ZZ6cHwh07imC6WMVQBJxqsyhKjzd2jVpvUipm8JGBpXlR5VglnUydpAAWp0sJbdS3IfzAseniFwuAOj1QAQL6DGm2q4VyRotKHf4CHkMHb1iwK0yc4ejofYIuqbCxBNDKzdKVLFjKjhi3FA4yS0yA\\\\\\\/0YvwDHWm3PPPX96wzfKJ5\\\\\\\/u4pRUbIRRkS02j1j9iFCLCzfUzgmaffRmLlffLq6hiwOnbQoo6wbISwi1qKl5m67fni9jSPGPc\\\\\\\/tkdnOmZyg7MasXkf6m3iedRGM16B8FF6XWvV+Flp6dl9VBtqwmQqhZBdTb4UrWYIuhtVMO6\\\\\\\/owgUe0KeYjFTnsXF3Tx\\\\\\\/0rdo1ncD8FaJnM7U0CS6Wq82a6+AYBhrODeYIKt4rtCwBvGEjO\\\\\\\/NtUNm1uC1f6udoirZoHK+aQC\\\\\\\/NqpKcasA8\\\\\\\/DZGdnLu5OtLOc2qsC3ayZZ2bwgE70LPo9vF5ClAjA4x1gRz1DYhIWWU9faikyvkAjfCwuyjPEIgr6pXsL6fNzvcib3yP7cAZOTzqJMvw4qoIMqxDLnmCvEnN9BAPYBwrCKQQG\\\\\\\/HfwpehHj0UcWvAw08whbb\\\\\\\/i6DR+b1m5472RgWRRafs5tyHKF6oXxHb0H4bqCfXzFyC+5JH7Dxf1FjoWoCogHbK5NQrxonhM1Wj6RatUUn2SIqabXLMiSLU58tGjtqSGufupLRsd0LdWla34HLK6nBfEExleWHKi3vPdfR6CaXAozLQvCwy+1F7obaiWARuwIkJtHurdYY+EhJs0WMSltuymEfhuYIVr9EXm0sKYRD2ph6fCANcNVdI9VsCX00pQvDxjhZaujRiijbotPumZWQELl77MQV4N24hiqvqQNrElPpUNcLhqkux5DHYU7ln4WqqfpoZBCQxQen6yt1235tMbQebK1bYX19wBmOw2uiB1czF4QBVW67SNJHhD09FEzyr2IIrZ1aZ4GzbKUj+B3xD4tJU8sb7ATtOs8BaeLkMJCmg0q2Iva9Rs+3pTgVHNL89+w\\\\\\\/7S9soeq2QjhG501xXcSJdbU5Ea+wAtGj\\\\\\\/BtArd7ds5rrFsWLFpZ5NVQCEc\\\\\\\/qstdZ01+sjNTuaG+\\\\\\\/PNz7ajhPxO89QxAGob85LFaZ4GMq0rz2GjP8VP9egM1zMFxxu29gfbATHN7PzlYp0xKGmv8hCZce98Iv1KMsFCq4fORLB4TrWvY9MPyUEivQsZzDX09o9hBt36B8\\\\\\\/qCWRW5uNgLdsS2hghatapYM9FnBL+nRYKRId7qPRM7yd4CR8BVnFmd83NVLcXULm0btx7UFibTetwSJwH1MVU74no1J25KKvWx+M1zwIBoKWJqxx7rpJt8zoAqAN+NVF1+M3VZkizysaeocpDT738V+hC9gtqVh+fDnlR3CX3zrB5ow5+YcbdGxR3ODvRgElF7wjAOqTr0CdAZmvs3opRjWf730KJFaKNvElmc4cvz3SinzOVLRPG2cv2nmfdJUyF2hK0WWU3GYzW4UpUKT7kLhEOwGgqO5lsVzUA3k0KZnfCMOmYwqEABE+xwgVwYDRWzMpod24qArYkGZaaSpkgxI8yBmt8HfIZFC8jOIALTV6y6TUYC0fNhIjfwzZGiDxKXJ\\\\\\\/bSqmb3csQkPH6r3HgKQyPZBDXn25FZOnaHCrHSDixvERc0u2iNvNdCnFWR\\\\\\\/bL\\\\\\\/N9wV8rYnCWR68Ds7zD7+2JIcTF2Bvnb5xCOKNAa1tj0NZNI+z+N226K1O4KgdXGB5bD8FKEMPOkhL0FI59Wl3u\\\\\\\/jfR60h8oNF7wOg0VjUyfZqPcOt5dalkweXo4aqvewOj3Bey0I+EUC0R1Oq\\\\\\\/NE42+ZMjbIJiJiFTCQ9HwdgOwxhbtABilIlpudJ1iF+lhgOBQuoAwJU54T1figbT0\\\\\\\/tlmpis1UDDHEXDkBGO9\\\\\\\/G1fBSVcyFxXg8hheJO4Hs\\\\\\\/9wuBaXx\\\\\\\/5wzgd2ghnyCnk8PY4BWMe0\\\\\\\/0M1VmI9\\\\\\\/2PASEKCpb2RJao8T9brJb0f4rR8Z46fiTyNfGet9KMMRxXCU2kNDkBgZZGJQSUbSxUH1mf7Y5UjNc\\\\\\\/QIaEgb83ZMVWqpttAKpLU7jRTnvTdqAwxzdHdWmdBFUdKWj\\\\\\\/CZvrPMs1MbOAXXTGwE6wOC3pOQnNsL1abk5BVHdvKjj8Oip9pCjcKiLZugR\\\\\\\/5RXOGcJrgy2\\\\\\\/3Gv6uGzvir3Mygaw8PMl+NYCUiBJB4pb+nAaqN7bgGLIrqW2WMcQpgCsqXDotPa7mjsXK7x+A\\\\\\\/UvHrxNcSxHk1mtTbMBqjGKlsG4wVyS\\\\\\\/nVe5t\\\\\\\/ATrUA1o2u2M2etZDSBdpLjj0wugGhkOFsi0IzLUmpoOU8TvaQqLnlXK3mSkNA6Wv03BChsGXywGhPVM0fDpOLn+0ShTVp7FluQphd\\\\\\\/111JnmHSyYlXGUHUl8ws\\\\\\\/ADIE5yjzrMM5XOHyMW+7T0xxOUCVrtmXN\\\\\\\/afbdHx5MyhknAuYKzQT+hmoiaMNGkZfABz4xmq+wcd7sJ4JIPxw\\\\\\\/XOE9U5XuiwWmGcuF2T3+7c5ML+L4tBFS4uYyTYusPfi8+nEahgeFNmAyn8BU4+3MkpJ0fN+yLdvhQZHQULwSBu3U8KWUlctHQmZR0TOvQYEYrino6GexcUaXKeu1nC\\\\\\\/AlC3zS24VmNxR0bfhNCwfFu+TSRzgP7d5nHojptUjjGrdKdmd5Eleq4w6MBTRLTJtvVhShtEGTSykyDA8++wQt88+68f3SOsuv7x03UCkD15cV\\\\\\\/ZJ\\\\\\\/bprhiwTVesyQVApukq\\\\\\\/kyu5Vpzls1LihPm5n+7E4sBahKIwPnvi+X+yFhw3grp+CWoX\\\\\\\/PKmUgK5LLay0NENSf4YYiP\\\\\\\/0kVDJoYUYiyXrvqYVkejO783X4HE1E5xHeunQAEwTaJKa+AT3lyWzYp+q2OINSoJaXf4FQl0QCAYyHJxaXXdtQ1czTdIo6VUS7+sIIzGenkPtFdedoSz4wrudU4q2FsVbwMfDruj\\\\\\\/fiV+mjqXUTSNSa16XRaXCCpXLLWXFFfFTxNDz3Sw4pH7Tuerqtm1t1XjbIeeFnePiPCiWOU15djzjqVsLX\\\\\\\/3RYtSEZWdZotPLI1XyXcSv0hJEq9IXOKKFHHYSdYGlEnfFrRUSlasIPUXKCc8Y+JmPPje+o3LcCzvBqh\\\\\\\/ibBoWZ66qG\\\\\\\/JezaDhSpizEtb5VJ1HKX+J1l6m+8IiJVzhVv6OTp39ZgZtP8z8RN5k6fDWYnXb\\\\\\\/e2QPjPaZdBXAnpuF1fduoUw0L3viJJTCEO5zbeGnKogfh1s1o+X6Q2qcDDKhlcXWvfrmNbXYq+kh\\\\\\\/7Y7GBRzFtO52TNx+h8hzROubg4uV3\\\\\\\/b8lRZ4zwVgFH61RiFnFFnwZv+oSiMpfCqn8GsyWiYUVOfwybbtWWMsdT2mbHmbKdg47naoeGz6bvCn9QJOqwvai52sPwCMz2i2nE5utbZW\\\\\\\/i89lRXRsLNQ9pRe1Ts+OK8qZxnpQtXtuKHlkcGtBArbKPc0mkhwPaIRxTK7qzebHb3HNhH55iAOBt3LLsN4GND2r49sViATmKpMwXsC+paninkEsO+pk2n0\\\\\\\/l4M2aNZQ7zoMgwmTPr7iwSwaosf1IU9Dkg2v2kp3vFDtzmJdqW+lIfYPBWckuj1DOWBJysZ2vCIgqBpwH4Wm7iYexBkFLhqZp8V5ZT58gUm5yfhuFLeDe9YypaHZhGpENVm5Yt0\\\\\\\/urFFiDGgXYm1HowRqfCZO1JgyzlbafpdS+hsewO2\\\\\\\/pjyC0ALstu1Ce8Zvp4tITd29HUZWS6EA9sRb1oI2a\\\\\\\/AZzzEPxJFRPbkXXQpiTYmds86uj7\\\\\\\/C9M+GHDNbHZie8ENJWy27bk53v4K1F4uOQimm8bOsSKx8GQHSEGoBHTgGzc+BRiqeeEGNxbJ+sNA6ZrjXdSoSX\\\\\\\/zYKHKfjA9fl9US1cOVxp8JXLc2IuCoivFevFgxBVE+0WNTNdTm\\\\\\\/vAiKoIw1f2ZKQIuuOxJhLaOi0R1X7hxvtLQb2OHIopkagCc8LdIonpPGL0UEWzjZsis0igv\\\\\\\/4PLwYC05z2clusT8kHozyhzItxPAyyNdPpRxR5ZL72TsXP88EP\\\\\\\/QcmivlLmcEdJv3B0Nlllb4t+oy80iGF+eerhkfPgOcQdBxPZo1OJA6Q3VkLrjYqPdwBKQm+1LnL4rJsAd1mitisGhrojZVUbfaaDrFl3PKNgU0zhcgri9sZ1pDhHOGv1Mv5PForGEbjlQD3RVmUX5LYGjvwM+KHiJlw4Y0NTLZx4F0Brbtq8dzOLlZKDW+uzpeg+BhU7QI3sfkv8fRRFCr16ZCE5pXIXgGF1Iw0Vi4khPLdtC\\\\\\\/qy8zrkveWBVA6R1WTTQu1gCwy8qsHdBJTPeLJXK9mTF475Jz4TOB6J0td8+fErlmY7nRLlkmkeiwF9FstkUPJJBgxLnpRfaVQjtd3KTlUJi78oY9XOR1yJgSVjyEURhOG3EDtRBE\\\\\\\/3MN4gVR1vI228QVEY7sWTSJRHvdf1ejFUU3lKNlOWrkIwrzm8vhj4ajealaR+xZnSEJq8LWaKoTUtGYL5QZ0QJGUwIGxbekYylXseOEegHAf8D4dgsNHU4wplvCjK70Zh\\\\\\\/HbLJnQTOesHmvRNVOvxjiTZYUE4jiyePX090iyAQZRvnBwppfLWriTckYBgdfo1rW+EdRYmxtmoKTWPb9+h4PZ40xJtYPisA2ZLcGZhNf9foaEDWh7eejnWkGf+962nTxTIrA3XTsrZXcVxGeZ0Ch53UzdZwq2PHfRktl2LYt3U2SLHMxtG8pO4aF4OlCx2ZCeWTu6iSAlecN4IuHM7+VYE\\\\\\\/xHx9ou1sor5tUR8WB1PNk14p+yfy5+EjVjeXkI\\\\\\\/nTdPPc5TA\\\\\\\/f8J1RT4LXwQB0IRT8zf5AbfAdP3saI89q7ltrALGbqL5SdQXYbmxjxQhpLZh05FCAz9tgxt2wqj0NOX+7wRKLJ5TBRcQW0V49IYYRyWfE\\\\\\\/tYcBOs88IXCtvcq1KoB+NiEzQ1zGLZ3yXLBi2ba7Q20CzB1xb5YD92RLI9uhplK1DgtgCagLgrwXKXfUQY5XJgdFUkN9E5Mh+HNppxkhquqCVdaJaxVDbHDNzhuJp86w5qJVeWgpF2mcPuabQVd5nU8P\\\\\\\/hOkhUsMaDFPdHw0D5lF4FKyzoWZPjE30qxoOlrnraL8ClmfrPc8gl\\\\\\\/DXMO3m2Jp+3SNnjYFe4HZY025AoCo27iCfg4AI1VVaL3IDUmwWHgjfZ8SkH0nr6AIY9us7hysZ\\\\\\\/VkFTHzECSKbeV+pEkhm5NQB+I2fFy2XdSFS\\\\\\\/BeBLiNI3J38VzpRIWb\\\\\\\/pEImH1VxSMK91LqQW76Xmww0xHR8\\\\\\\/Syuu36FRzZFuTGE4+N2Hc8f024jw\\\\\\\/f3oqqkobEVf51QmB52LANAlRUrCzPrA7j2i61R50DFn0PzfEnB+bOy\\\\\\\/av2iXUYwXi+lO5eSf+Rmi2TQAnlgZa2Z4OpO\\\\\\\/MqrPHKHwKN6aMKS29vksYINojb7L4mr9vz5BI1SzbUAHZtLcnH6NndKfomr\\\\\\\/CV3A\\\\\\\/9nuLJZg1oQxBSRqb3u7PkY2wPPdrRq3wbZmLbQhcFpMRXs0CQoNVXwyXmpwrKeB+Zzd8oyodnGXr7fnZV\\\\\\\/KLJazDL3IQPBFgGBycXOsMo85WbrwPIqXJw6qdPRsZrf3nJ1JRQWsyNBhUgGon5+pPBStagqCALrnal2wVikIvk3QBiXI7LwxdCPYzBvlu00MmJhSl2V2nYwiyym56ELEl65nKFQNeLevE9Gu2i5c0N2T0TyaMiS+VphhQzhi+KI67FMdXxL7lSXGHvLragYoj9ngMdRdgIKpU+\\\\\\\/pJk79AZoVIVmps0tuqgOsFFofjWkLYzgi4oqGvPVo1yXwo6m7KdC60fA9iJrv7wiaE9JARHdgFABvOpuE4WDizwBdY6P8rqe2BddSJ\\\\\\\/e\\\\\\\/5QZBcYBNdGOLx5MOPaYqFG\\\\\\\/BuD2hMQ8odrzim72F5xOOw4TX96\\\\\\\/hbUocZKQg5fF4B+pmpUoHpAyZSdUU0ZYsajBMX5M6PnvCNVkQDE0uQCJtQ97r+QSfltNMYAG43oJMvXwOXA3WEriG1xGo8f904ng9fxjl\\\\\\\/hTOBro76TKY8ainuVrteoXQF2tME7nJiiEN9nlpXCb59Wwms4wExq52Wm+Dyn64lnADqa32dJi4hQFGnclYgPQyliwVLWLF1dejyopZ8wNwXEI4V46e5XIZ61JrQAY5i6AyQW4HOAKwasMD6yiN+lhhrhPa7O00U01aX7GCtFrhF9ovx++mjnHLHkby+ERHAdx0jcMqPT5vWDbAj5wqDHktFlwtDW77c7UQPecufBdEHcg1B39E52u6SBV3gAAchqEAQxORRx+SD1QLtbZ\\\\\\\/376JHMbePWBVs6w7cI+6ZbzSnW04yBs+5+qo0z2rOWdYSuOdK+zqY70l1EUVaJS\\\\\\\/38pH\\\\\\\/LEpaffTWbgsmH6VONTnnJ7xoBladd29zDT3eDksX\\\\\\\/P0vHgvAx3ai2ZfyyWfqMuKdcZleKn04NqBLtM0qurtei2U6\\\\\\\/4zMLiVMXK0kG+1Je8+MuDgE53q9henHxFYbmN\\\\\\\/P8yIAUqHCob8i63kXZ8x\\\\\\\/Jd7wB+XZnm4xLJNt3e0hWsc1YZcMC6DYXLLqPa50s040kFT8kTbkcrVQS1BPLlFb2IF\\\\\\\/jworusdDNfGyQxm\\\\\\\/svqLZQKr4SUiT3fZYWjXq9qYpifau2m1aTXK+IjJlnpn3U4+Li9oKQP\\\\\\\/08qgMN\\\\\\\/J\\\\\\\/0T9gMk4PvCwAECHcZZ\\\\\\\/SUXQnp5Oq37xCdsIChPQKlOa953LNyDWqQfr8E4+X9Cs09LMQqx0OayRBwBSYasKSeEulX0ygboJUxMIIP6EZnrwPwgDZt31DPt+5on\\\\\\\/LnhzYr3nHyIlzXLVCdFN9F8bqN\\\\\\\/ehzA3jOGkojdsuf3hV\\\\\\\/ZLx7bzz4XQpJggZXM9S2YHr+8vhqyzCap8inrCxfDM6e6Qp8DTykYXvIx6\\\\\\\/mMf6ov9zHD8ljwqfoGfyAm2DPawflAwGL1KfznXQ7Hrji3by11v2YRrmU\\\\\\\/vQ48S4hvxjo1aYyRg\\\\\\\/hwbfdEZQSa5wBa9nqupgV2xLXYXKCDD8hY38tg3\\\\\\\/b1R6PKmvabgtC9aoOCLaKtOubrNRf2exQpcFIHU5aJNNQgp1qrOtbuF8BuCU3SDGEpa2Muk2sV3IBsZfcA9SIhY8tY3RJUveAp0mV8YF0sacs295uxXP0QNx79gFvR14fRbLp350TXIogdJjAc4N9wwYesK+6q+SaCq4p6BGGI1yIBRIdH5DJd5+F7HbK62YzeZi6nMPQbF5K3rtuofSvPFL6z8ro4vvPZy7gOyqDe44BlibXVxPYyVFoQYeo7mY8vfiTNAnnCNzJazMLcpbvMk5d5uLdWcyvT+wOnYUoSHFFTxgykcD4aWKtC+67y2P\\\\\\\/2ilhe1ZLb+QFGgAGkPqmkPpcZIDhwB75W6WQ4mIsCIp4GxfaTuohfegJwk3hiAh4SzWpvKr\\\\\\\/aT556ueQS1G\\\\\\\/wQCD9UlI0kyZODx9zdSXKi1dIZ2n3wdAlrugkjdv9yLMi1jGnQZcmeNmOfSqLP0vR785tlKnVLCPdslSn67VDet76VLRlfqOr\\\\\\\/Hsn9J2SeDnuHzcbeioCHyyajBdz2f4c9aqa72j4Nc3HG5ptD2HRnxy07iJMyH9WUiMZk79l5lBnjrdzUGe3L6wily53Y27fT73IgAqDbh1o0kLSUqQs4I600oY4ZdNLJW9OrueAqvOBSMqZExYm\\\\\\\/4Pt0A3G3NkqX\\\\\\\/\\\\\\\/l1gFSDtisRgkecowL5iDsZK3UN8nzws7t4N4t7PyYtyGPFNH8dF6MO3vtQQBBy4Rua+XMxRVhdHudmrXVYk5CCQRQqU2M3nGJFD\\\\\\\/fySRXkFJd29zDQZU1tUw\\\\\\\/oEwEmhi0uu7qYmWvFhy4vx+X5CGuLpyMCMQLSKUwt3MivLSOXiG6NrT+VFGbH47JJ6vfvnQZ8hCbS5JugnkKq\\\\\\\/uqQ3AOvuyQV718VKamV26HiOIwZ9V2vbSyRUeu42FxWIjJ1qhpwCzuFPk+B6yH5IqhdWIsbq1sa2D82FUy89esKsbeHNRhS7Crnbk+rWFZd3fbloVLN8tghqmHJTcFqyuSP9NIKUB17T\\\\\\\/poOpXKbAm+aOjU0Ct4Jy975pv2nua8onR+6sgq49YS79wSxcAjW8t3udWV1hXhwrrBoMzfPxU8ltBxaU9pYsIreco0YoxleBm1RKyeeAOJ2VKrwe7BOnb9J81r9j6MHCWdesjspZ6W\\\\\\\/KGxTLCxcmgSvOGge+wZZy\\\\\\\/9rSmS2eUkFU15Y9JWlVc+o8aqHhP29SWxVhiVSlSy\\\\\\\/EL+InpuNgVNesHQXzsOZoDantyR5+inp588V2Ijm3JZ4lJdtIbawuBHIb7CRcJ0NIK0miU\\\\\\\/E6HX8imxrnLzudVKrNrdeTieJDg5eTeX8wS7Im7yjXfvZ0NMR+uL0gNAWS4M0y9rqQUevrxwBBV8clXJRgYYbTK80cs85S9uUQKUx7fDoYyN2mC1CiZ8tgAHvXkLVlThkqQIp\\\\\\\/OvDTpkrcGGxnywfKkvrGVa85z493M\\\\\\\/2LwsXUZxSX2g3C76bUmJZbpZjgCHJ2FvxqN+VIEZJsnuGOrXmVCzZM86cUlhNYQDAywBu9PDSYMF2MqnCX3GOgs1DXr\\\\\\\/weSqwtmLXkPmU0uv1aeaORdGpqNp9QOM6Ygce1qab6zRYqcSUVVS52ftghTKWAZQ7vn4tPIlguogC6PBlDL9U82VphfqJVjl7g\\\\\\\/RsuA1H3a8dsaJM35QR9LQ\\\\\\\/L7yLLNUoAVSOGAW7dl6hjxOVc2KsSXHlj2in9v9RxoJY5w6+0HVl9K3cOPNzUfrCUioPvWuPsd1LKsHvvk5CC1CwsiAxpU4nPfb6MUJglzosDNb606YEF98L6Em5fvbAmX\\\\\\\/EWxSWP\\\\\\\/fq6aT0Wm1ugCIy3PZ1revtbKLG6ednEPnmAhCI6mC4jgpJRKUK3F8HuzVjSGg3V0tf3CDszur7PqZzvtxbjVHf1bZYXu24TnsfGJDh6GsA\\\\\\\/MYwjIP31+qPOAR9liJzXhS9CKuFBLK91irHdviu1vXUsl3kiVgsSs\\\\\\\/ZNvjFnaXzcl6ti10JARWO5Uq2Yu4IuWJypsLKAA7tn8oWtIRZeHfjWb1ZJwPxe5wQ2aUPpY7\\\\\\\/zI56pb+BA+ynSY8Lu6CGnrKeZXDmGMHEj\\\\\\\/u3EXr+2WyJF9fm4ikcK87pfiH9tFpG7RXCTjOz1DeAIpBt2loIRCuxZO0QbRv8ytSTZib3eMv876nG4BC4DWA0z8ez3XGBtyoO9PvrilqTlbCoUnM5tBp8vA2TWm3R+EaB9xzgilU7RQ5ftrg7k0ZVCrVgcdxEyyo+fDmWY\\\\\\\/ugqH7QdGlhgPhjS\\\\\\\/kTvQgR8HYF2oF4A4Kcx+Bh4mMLdojMTRT4fx13KuWv\\\\\\\/PZvvdnnnyyBx2FsG1yYTC7gtpHEZHmEokWeeCHlgStNXhohY4mm3kTiJ5gsHqv8g\\\\\\\/uXinsX+Tc3J9EBg8zPCOYK1j+nCAVfCmGsZX6zK3vwHJs4FEPoqhEfNa2TGBYAECUBXl3xc5\\\\\\\/bbpoxR6LPpNbLIfWcRtbRk5lO0WAue9wM+lx3Je+AHHKu7IYFGSneZQWocZGQzVmp4dXzuDsmaMVblKGmBlwD+DCtaXlLn2H+RaIv8teBxi9em4UCpMbMr1olp0ZC9nXR+KNGdfQvYE9dq3tTZ1ntaY\\\\\\\/1Dexx0PDfeEb2ZrWlxAcZnWwsSYjmEiqFvbNp5kf9K+C8mKhX6UBv4N7iUHi9pgXbKr98DnLP7Z+ixBAQnGXWnT14CD4L8PNxdelvO\\\\\\\/3vNCLoL+XkaaocFfxsRNehTEU87J02OKqfO2cQJV9hGJYkRT5ell+ekFOQZgeDxwjcdxZE1FKoaxwh+V7Th1fQDTmRsyosRc8w2HTgPrdgQUd9Jr\\\\\\\/Qn7BzMJbeY3MnBPOEbOuuU76JSaDoqEiYe1mVBJ6yYWGJx41pirV6aFdy9rBNvV2bsOseCrrHYOU5gSsDZ84CONI3Z0x1PItBOUy\\\\\\\/gitSvB52x+MlweUw5PtBh2fHVBudbjH+BIh76fkiLJDshOqnZVQ5L\\\\\\\/Z02Nd+g2WK8bgxGOCUAZ2zTUiELsEd4B2v9469qHpqziN38qCXkaF4DCUxi8aQ+qm\\\\\\\/XjtBtRW14UvHHvJyCXVplAiVyZSvRslIF6a3tkazfF\\\\\\\/Uyb0JFy1PXNHSQNdslPTG19gAgb731ADZNURggvQkpZPtqq6SCQgF85FJSPJPchUDDrBYRdc6W2vjt84HS63MiXzgnovd+k+LMllPnp4BIdoswFLK7Yn\\\\\\\/Mwhe60p9x4bJBJW3LyRlt\\\\\\\/7sncTs5j5Vx3aCng9dmwnI+fM3dFL9WDGJMLKu1tc+0QWDg92GbvMivpsoc40M2KClAIfGQGX2Q62NOn9n0JPRI+rvnopqkbH7e+DMsm191nXDnYbwHbBi4lv6vXo5IzGM\\\\\\\/lOhhb00fK\\\\\\\/pRw58k8hvMvjueZRMXd+6iXZ1eXKLfy3V+cmNzO8SFuo09XS47MJi+5A2lol3Sptl8lbUmiE\\\\\\\/gu7pbQA76ngUALpOhejJOGfKyhHHw40xFWo0Fwv6x19UEk9flaal2cZugzD0+ekJw71zNPQGONlU7\\\\\\\/LzqLsit1wCwOhyThsy6iYNP2K91sy8j9otWznMb1NeFqUVWCj7a2BnDNFqXn7MRjgGuVXH7CNNozrsJkIk\\\\\\\/34iCnY9eej11OgGzlF7Ibc3UnqvSMQLg3Z6r7mlRYqkclKHa8RXJglz6HFyl+ViEKJAXfQoOxfH+elKhqdDZy3rkchq\\\\\\\/eBP9iVmcHXkiOx+TMjRUzGaw4DbIpCyUhSTjOn8UfNwOHOadZC7psLn1liCDJ2KAxt8KiBjt9cqwIxdAA3btjtHn3kjXYjo8sy8bVXJB2BMmO+sN5ui3Vk0FNHkJY6NKbhpKiiJMw5ZRwSyWVQ\\\\\\\/wosutlQsMVHueFXdZvsVMZqhMPPUqTGc5Zd\\\\\\\/D+Lq2tiw2By\\\\\\\/fe9zQrTYngQRXAwE1F3nu2V\\\\\\\/RSrkzgD3BYuZeTMzZu0dtkuo+Ui05OnVkoUzCwVwy\\\\\\\/Sqfab3yhPoV2I57usa21S0c8Dq0lvHIfl4f+Ek4Z30pJSjbtu3QxlZL1JRNTjjeMnLi\\\\\\\/JgF6wabVdfvrmcFDGUPZjkELTC7twBhuCJK4ZNSFyfXa6yy\\\\\\\/TrjyTVsfWbjatDOpycSfso6A+VO7TcKD6d2iBcClAPOjcYMlT\\\\\\\/SfZmTWLzeeo\\\\\\\/Y8ZuvbTt8qp7x\\\\\\\/vtKD02Rga6hEDT881tHEGURIjH\\\\\\\/lq5IC8qjNroUpY5eaHHXJxDi4Gwh0JNmHopILBcKSODXCkvRjWEcm+pecur6ZCAY2ydtKRERYaK3nq6a4hRDzMAGTqLY2wXvzL8xs4perh2LDXCzybVPuwIeghlbHu2A\\\\\\\/dFAD+xxdtXg4H5PdAuUDl2R9t0NlMVgwiJiKZ2vCXmJruEy5a1MrZ+Q3aAfFnQ35+4Ipg5D3DNM7KGMgASx3eGG7iD6KE42RqsdZlfg\\\\\\\/\\\\\\\/TmqxvObNphNoY5OuG9lu5VNwXOJxu+F+ecW2XtGnAROHblz3w7GQ52V\\\\\\\/ulzfS2vemA9XhY3MFFwGAzL\\\\\\\/EweRDjIDeEUunVu2kpIOsCVCgwl3\\\\\\\/sN356EEbJvvJ\\\\\\\/EIVZXINkuNlNSMpRP628na3X2nNqOTANIzAAi2n8srtm73soHldrtccNwCc4BOrv\\\\\\\/eFBnNznQpuTDxuDPNleFJWaP53SbzLddjH\\\\\\\/t0YI8WiargWxOtD1TD8bUedVqhUlMitB04KSlLB9MB6RnaUMC3OU7u9qFLTH9pyoxxd7NuSU4HyadD7mYuV4CRkgf\\\\\\\/J\\\\\\\/W3nKeQUU\\\\\\\/mVQpwBYcR0MuVO3I4cTNdt3l3aEyDAc3FqmELnKasF5W+wVDGg7Qk7aZElgu2cNqOgcEO25+bS+aoeieMqbljmts1ykloGlrZbCDloybMkW+eVv2RkzVlWWK7EUMf0UvkePc0FfqX\\\\\\\/cNhljhk5WoJWXMmiNFZiTzX3DNsiDLnnDXphBkKz3kMRK4EYI+erjMEVZm093kCjv2Uugkc+kGRxI858WTZvTtBKs5n0PZvH6Gqq\\\\\\\/6rtkvi+rY2DfY\\\\\\\/o75dkCH\\\\\\\/w8IGVw7TybGAXyhV5QMPZ\\\\\\\/AsscEtIzhOV71Z2dQl6DoJlHe6vFBorORLXVV0eexQ7huOgqPaiZmBE3+C9wqNS1A7wfWRFIE\\\\\\\/tX4xMBRF6NxdrO3qsxU5tdMKliZI6Y6zm7oIvN6ZXvE6Uz4ypP0ipZMfwkAE22ogJlzMpPuO8VP3\\\\\\\/UoBLEzlz9LpwpgavToSoIgI44vkTQxfGwrPyut0LQYZzL\\\\\\\/p4GCfp0evr3PDwOzJi0cOGKQxkv1gRSEAqGCHo5zBq5yEpcVQIEKr76UMmUFcwhoUpjZZDu3K12D8xIoH\\\\\\\/+b8vi1XhwuPs0OQs14ufhsMNjkiNnSdxR2Url4GbV0RPpASm13pbb3ybApDRUvitJOKBOUPVy430cFIwWn+x97sh+uZJqxCIqK1idYxVIORtY7J622WaXfoU5x+CzPX0KDu5MFIcPtCs+rdhwvJBwNiEuFHCy8nGrb75oLzMC5zByV1zn6YYu4W4oAAN57+9jAK8hXvRVitskbHMvmOhlaww350NT9ZIxBHUUVWauD97+navK57Ik2dgNwkvIaGIpBJ0VGKBIUI9VZML26ZFvJmpfuWopGuzxF2JOiSz36K0EMsURL8GnywRyfOlYFU9kH6mgoIMasqbOHeFyrpQU9dFx7j7EdVx\\\\\\\/ZfXZTbCnBos1Y3SfuzK8NlbruFTg4Xom3YqBNPl8APWBI1RiNrMyQyXt27WGNhNNYWr8Jiy1Ip+x\\\\\\\/4ABwBbELUGk9b2PmHdCJ5IlDMHnoqVVUo05eyjaxYBdaRrmQujKeR3Kd+UPUxQ5NSNNkxAP9ToiOyzc\\\\\\\/4CEUHUpQ3orDE8eJYxH57cTxXQc+Hn0eZCpBJaILPLdYnKY\\\\\\\/dqBlGbAtgid\\\\\\\/kXD9pp1jbRRG9oUYfM07u8fVhEyENcZnhEgkwu1BfdgltECD+enboJJ1utWUF\\\\\\\/z7\\\\\\\/GBPoC8N25kfw2hcVaOB+6tYm0cK3GVEXKvH7IrFt4W6HFU4XWEfYHtYHkTLGIKI87henUxRkr6p8TpEYZnvmpmRylydBDNqeq9ZQXqj9I+O\\\\\\\/RXNe3UO9WBMXrx0pItU5zMEvVlOKLw+wgJ8jryYSZffKPC+LxFBXapxy9B8RtEjkl08w8s\\\\\\\/msJzsNafWZ56WfgfomfQweNhwXj+7zrrJNCYDbTOIvmRN+haWbdnFgd7HnL6hGRG7ejI9C+GrgIvSCyG3q2K1Qhi77TVldj3LkaAUHeGgdxgzVKe+3cggdWoz87uS5kDwo\\\\\\\/cYukf8K3geS9qXymGP0ITw9\\\\\\\/AgZ\\\\\\\/QHRjRQlUMRA0Kc6v\\\\\\\/RToxUb++6\\\\\\\/9jSyfHjrkV+Xe+JjiZNdAzaBIfrDMtPK9ZAuMo0sHH0C8qCcmA8ZQeUqdREAR\\\\\\\/BtQ\\\\\\\/T+4Et7GNrTEwOsxKXTl\\\\\\\/iZaXQWMkGJGIGTTzliPZOnvse95ZUHlOhuxxgoAqVYxGfCQ1Ha+u3Hk8Y\\\\\\\/fbBQrKca9XA5fL7V8n57dN3aHr4mVsz88eeRjO+xibJy13\\\\\\\/+SnBndcS98SLH1YrpA6gUi+2f63ITxQd4D6gqzwgXZz8GNtaVTbDWrcmH9d\\\\\\\/AB48SIKiobH3sih2Lo8ZE6amKlJnVCCEjf7n9\\\\\\\/0TX89xHnmHnRuOcr2H2rHwJCCIqYa9THEG0GShkunkXNqlq481ejzMs3Zfer8GwqJqynUAqzxCpqSVHvAOwiKvS3pU+3nGgM1AyEcyyeA1RNN2LrCdm\\\\\\\/TCH9P5cAG4lKu5SqUzpzBQJcMNVNSyVSfoq0lWdJEJl020aTTfn7vVyRjA4+tykRubylyPdoYb3Kt9iiFj3NKM4Nh6zGq7F1oNzmje2MUXRR3tNbRbBi1W7CjvnQYbH0omnhyLwejhxDVtcZPGgQ0zQr4LOMVYBMC8qk29wTyvqmku7x0fJC+nONJwK8rFZ8peRl3iyST66ysXjAaCsuPo\\\\\\\/DbaTJsuY\\\\\\\/a5qO344wooTvAAvqAV\\\\\\\/iJyOu22nxm1+PQS5MXukTJM5Se2vhuXyT3Ts7RNci9yGaJdMQaZvXknSn2n0+T2+5fs5QajipZXGUFi+vjl221rLMHWWQoZ+TMb5E9wKVa5UhCBW13Wghy4SJyCGaHoF06TNMc3rRzX9ZrtKZyUudmaoQbEWfNRbLUQ3e3V0EaS5AEmKTEFMYAwp2CDVFL4JptKZCR1cO+Fb2010Cgi6+oLz4o7eH63l89LB3N\\\\\\\/E+\\\\\\\/hGDirenCb1s7WmAelQGgyNGLtGGajYGRTXcp\\\\\\\/QiQmstYDi2rjafeH+yOt2DCZCG64zD9r9vOtLDme8bm\\\\\\\/b\\\\\\\/L6zNrG+NCulSJKKkYgfwHrZPO40WJ\\\\\\\/ZBqy5OQ9TjV7U13TVJ+P8cek3I3SwFY4uch1A0vKerkByT7e\\\\\\\/Fgi0MBnLuESxQz+QcJpqHRk8x5ZPV7zJFv4RogQ2zfkmatDsIxTQ54he8Qtg80\\\\\\\/lVNbxG+a8Fz+Ph6mPoleYvooJQebSgmXqfe1uLEc3tuQnvuteOfKhpb6IEzDDQzcHJmSR3HImB1o0LH9WC8FDiFW42b3FTdNMCX8HEuijMVoKUKVIO9iQQ+spr6o0AW+lX\\\\\\\/YkCkQAm2GOukLWLhYkeT\\\\\\\/YCw14p8e7GnQf47dTzicTCUq+mTfLZMsVWpDph81AQT\\\\\\\/4h+OYzuUEdXlBIzKOz4XdSYl5MeFaECl9T0WcacxyE\\\\\\\/3JJ\\\\\\\/yqWyRB63eP5+CAwcazsqEqsp5GEFOahnuVt\\\\\\\/qbE+nWeBewGpt49F1GNX7GIpHZDjFBE2+4A\\\\\\\/J8YLnRWT3MB6AXw4DtsIC3UAFWLtN\\\\\\\/\\\\\\\/TJBBOiLjpEKbg2iYy0DNdGQh5S8ZTqWlKlDyokoRl93hqNnvL4HRJG8JV9xypk42drzbGM0MKACWUG2BpKk4T0rj+C0aY7iXhGQljQSYEXMyp9iBoT0gueWdwQMA3K35TpSqJ6dOcWRYS+65WzgTdWfb4P11QvwrDQMrIo9K6DAY1lzUvSR7VU3fa25rFjtaF0R993Zvp+6ckOrDYN5S4SRsMPSMvjLJVZReGa1wXM9WpLccQYHGUEOJ+J8tB+PAv9hfEHREqHdqY2O9saU8KbktWmmcbznW\\\\\\\/+CFbTTken9WOVXE0OxJwTndwAT0LLuZ3GN0FEJMfE980lp634jmC97LX7Iym0u3uoLNCm5fiCDVXfDYDwUwXF+I1yF8ZgSMquUMWGkDkiZeT+rs\\\\\\\/qZPg4WyndZSHD0lU\\\\\\\/g\\\\\\\/hQvmB3DkLPXJSu0o5hupc5irMyepZkTSbdyzOkaQdEDXhgMFZ704SJrYbxcRweIp3m5GvpuuGAkkntA8YPJwgZV1C3kNtF7b4EFGEwHVzY82C75uTSBJEWpgxT+OafBOdGl4Cz6RhiyDivCE\\\\\\\/RlCkm68r+z8+89mr260R8G\\\\\\\/wzhft9F\\\\\\\/5jZeQou6QnYNYX9jpKlekqVKGfzmhnx6wm\\\\\\\/nhyuNEXV2WLspma1EUJZQ+gjXLqO\\\\\\\/yyd6tg8rfIWq38w5\\\\\\\/IBM7ri3hGQFDi8iCywlliAFEI6HDh4cGzLWZwtVh80fIGeKxLniuh5AYnqred1KqUNdkfiEuKighj9rKZVy5+rYWjlelDRcnUoR9uVlPkDCrY7nKXGmGkssnvcPq+G6kqrM+yuv72vyG8KxgGkpLqDMemoyUjrjzLyQuvKCQqJdjs89jb32gCAlDesgUKrlE4dRUNr9dGVkDSkDj3RUPFpRgW1g+Y+9aSCyF327QRqNM7Jzmd6zLR+iDx6P5bMxd\\\\\\\/6KX1qOnsDbDIr+WdjY01yqH+\\\\\\\/x4AFq1tUsubn2h6ox\\\\\\\/jKrFNT2i\\\\\\\/NHlcn7OUlMt0Y3I1LORyu4i4AWgTeh65F0lgN2njapZXO2p3DBif7jCbNdCtq8XP6JJVO5o9r2TdCWs9E6dzYX0uW\\\\\\\/mih1LxeHB5yIvHWQn7u990JuHQbfLD8FTWbo9YAxMYwa3K9ADE0KZa8yq9uXtNSjKQa84wM26BYXnjZL63e5UxdZ1\\\\\\\/o4wmDywT7GcsMx1NoGAC1nSQL\\\\\\\/VqSg204\\\\\\\/y4rp169Fn9f6ieVxE0zIYmJF52JUyjwZDCfGb1ADm4f\\\\\\\/Pegh9ur+UMrGX8enX27oRFqb2oO72iFnpjuU+RsoXsigCPb37GAYG4+sCW8Xw3tjqasTiZM4yp7CPN7j4sI5k\\\\\\\/Bcm8vUGmG\\\\\\\/3mSjEUYSvT6bYm2I+AwpRvU2Ty2WwhURZ5h7X4mMgzHen3FimktWzA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022lNwrarQdOWklYKCEFHHFgA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228a4bb119043cd8feb7a1aec5c2ec7c5787c56cf8d57d0112daba93c675ce9a82\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222I34rJLMJhl6GRaj7ypDlHe\\\\\\\/Mtmz+WWNlFS8ZjjSw3YwZ+3E+2SFWVSGGWvrmWYbxX2mXEq1eWDVOZXCKPaFriW\\\\\\\/kMkAQZdv+MN+YQ9B46tzBMsmbB+Apc0PlYWBmBSqMDrLTqlCbp3snUkxxJWU6W0GoxqC5rLqHBvzed80iU60vWVxJZfptO1R8smoQcmba7VsheeDKN6CUe8T54xxtqdbcXKGca8TLdshmDkChZTOVt8lMStDKDw9uPh4ti6ZxA7wmVFqtPTHT+DXtSfz+Fm1L+aTzPMcwkXx\\\\\\\/qFAYoVlqRWpeZQQ4haqh1WRmEFVV7wf4EbbKb1FSsUS3Rwoi4hu\\\\\\\/ni4nydeuC\\\\\\\/jRO3ovwvAvtIPz050XjnfCVr7NLBKUMcV9EIn+fiyyBN6XuVsOIf\\\\\\\/5YEKDCGuGAls1nAQQ9Qiu+8DiB14m70L\\\\\\\/rrZ0CN+g6LPwyWka5Cnnol9QR1iwC6JeQgDLY25dVAfaWw+nzT7j7s8N1D9lkLk9aWBKbTaMQrDfOoy+3Wke\\\\\\\/pBtWb\\\\\\\/AVbnZ1K\\\\\\\/tE2kLflXgDfwfAoMQjjUhqO37f31fzMr8M6S\\\\\\\/caFMQJkslFIM+ZEl0j9vtZwSA\\\\\\\/6wxxXUYc1KCNBdiqykgdYRo3Gp6eDpPeJO8Qb08jgmX0l0BeBWT7nvjlODsLXoGg\\\\\\\/l1MdlHBFS+ZRJGu+jV5Zxus9gWQ6FsqH7I6kYm7TlErfOnV3xiDz2N\\\\\\\/m4a6U7IVEkBByzFdwDPhskYahEjJnN+VP21izNg9jjZ8XiWKEeD01lAp11LYqPZXSnhWCATxrCzVoUY7hHc8e9AMPNHNChE3AozAlMCLA0GSPNlIxW9oxOETdCAYb304YqYTSGsR9YHg8rS7z04MY3UIEuLfz6WHl96AGwRysyglIz0D6F1usmQJGcesNaRy7BSQwoY8JDnnJuo6HvWwD5SpIunUTu3h4vdSsJejtVa\\\\\\\/nA1Kr5xnftIu5YtbhPR7BE2TR\\\\\\\/ZZ\\\\\\\/tMTyacGGelnYT0APoO0BoNH1mpCop8vRxjTAAZWZNC82EluEy8bCgVVMVzWXW+orcAnoSxtr5ibeWtOfcvVTCPhHI7KWZIuLkqaiCyFFTLpVu35JqLR35otVQwxoKd4BexpPFPSETkkMh282CgygwUni3ed3\\\\\\\/CaeUS4XNjnply5APBrbLD+8FxCbA11tzSko83SqZO+ubVF07WgHfqIbQSELM97NwVwiwgRouufEI7tm4qJ7mjQ+ZlrTOmZkCGTPDx7KVwXhESaidcqpVWDoy7mX4fnoxquZ2ULGFglC7\\\\\\\/wQDy4hovoVKwWnvoYQaDVVt5bKzC7tTKHTIWwhrfIZJYx9EYK69wNRfRYm42ruZi+LaKQDFAYAFn+tu4n6JCXcG6bNTfKrv+++9GhUwqZwP6hrX3hGO5BFquDcLOGb5j5IX7MGic89DJ1sODROpBGIv\\\\\\\/DJUKfg+b8EVMrPnOrMJxmXeJuESSDwuKuPPW0nP8NgQy2WkVrnF1m6qzJehJxKkuPwNYnWjF5kIlEWtnxdNZldWNDvdrXTXD0A+pSXgM2WAa7kVLNqZzn7Bqbo\\\\\\\/e\\\\\\\/Pwy8dBoIWbBkp0RYa\\\\\\\/h0yZIRaKcuv9FYs6GEGySVUW1kSq9mPFxHu20PkD3G48XA1j4tqeRnP11UKTINaWvl2qdZ8Jz65kx09Yr6+lExIcG0RBu64g64zsWKAOOgLMOK9gfaal50Ye+tIYve5f7JgfheIF11uDpGjlrk6dTd3wy+ligl3o5iF\\\\\\\/jBJ8ZHodpXd0SpF1M\\\\\\\/W\\\\\\\/myk79PvZg3ImIgrJZXerZVA3Be7CGC9kGyKKXHt8qQ39PR\\\\\\\/tlIebn4O9BBlPGmsuRZipVMNRQqpWls5zd8YmCoNB6Sfs9euN\\\\\\\/\\\\\\\/JOVb0oU7wpz19UmHMPnlpcObiWlpCbiJS4Lxt5LNKrnV+5hFvPCRLNKbKCynbssAu4DPrmgThYydjmy9fz0v67GSkRGbm08ycHuKFdtSfhl+EQJxxYJvIMNk1svcD0hhlAe1wF0fvC7goaQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022J25MY2OtlLUa\\\\\\\/bHXNCXqsg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002284bed8e85b19e646a6f971a84b3364f1f17ecd9b8f4e046ad97b9f850e04d1de\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459330 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/9/987" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XpvbQoKQGqt5Z9ceLF4lyLkbLgNKw2OyuyZYbC-vr6E" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/9/987" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-S7_uXXtj7K_3k0DBkj5FtmsveYTyb_ODHxOy-yoJK5Q" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/9/977" title="Evolutionary History and Genome Organization of DUF1220 Protein Domains" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/9/1003" title="Glucose, Nitrogen, and Phosphate Repletion in Saccharomyces cerevisiae: Common Transcriptional Responses to Different Nutrient Signals" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459330" id="node459330" data-pisa="ggg;2/9/987" data-pisa-master="ggg;2/9/987" data-apath="/ggg/2/9/987.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">oPOSSUM-3: Advanced Analysis of Regulatory Motif Over-Representation Across Genes or ChIP-Seq Datasets</h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Andrew T. Kwon</span>, <span class="highwire-citation-author" data-delta="1">David J. Arenillas</span>, <span class="highwire-citation-author" data-delta="2">Rebecca Worsley Hunt</span> and <span class="highwire-citation-author" data-delta="3">Wyeth W. Wasserman</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">987-1002;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003202 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459330" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrew T. Kwon</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, University of British Columbia, Vancouver, British Columbia, V5Z 4H4, Canada</span><span class="nlm-aff">Department of Medical Genetics, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</span><span class="nlm-aff">Genetics Graduate Program, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BT.%2BKwon" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kwon%20AT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BT.%2BKwon" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">David J. Arenillas</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, University of British Columbia, Vancouver, British Columbia, V5Z 4H4, Canada</span><span class="nlm-aff">Department of Medical Genetics, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BJ.%2BArenillas" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arenillas%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BJ.%2BArenillas" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rebecca Worsley Hunt</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, University of British Columbia, Vancouver, British Columbia, V5Z 4H4, Canada</span><span class="nlm-aff">Department of Medical Genetics, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</span><span class="nlm-aff">Bioinformatics Graduate Program, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rebecca%2BWorsley%2BHunt" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hunt%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARebecca%2BWorsley%2BHunt" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Wyeth W. Wasserman</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, University of British Columbia, Vancouver, British Columbia, V5Z 4H4, Canada</span><span class="nlm-aff">Department of Medical Genetics, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wyeth%2BW.%2BWasserman" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wasserman%20WW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWyeth%2BW.%2BWasserman" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wyeth@cmmt.ubc.ca" class="" data-icon-position="" data-hide-link-title="0">wyeth@cmmt.ubc.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/9/987" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459330" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459330/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/9/987.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459330" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459330/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/9/987.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459330" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459330/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/9/987.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459330" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459330/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/9/987.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459330" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459330/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">oPOSSUM-3 is a web-accessible software system for identification of over-represented transcription factor binding sites (TFBS) and TFBS families in either DNA sequences of co-expressed genes or sequences generated from high-throughput methods, such as ChIP-Seq. Validation of the system with known sets of co-regulated genes and published ChIP-Seq data demonstrates the capacity for oPOSSUM-3 to identify mediating transcription factors (TF) for co-regulated genes or co-recovered sequences. oPOSSUM-3 is available at <a href="http://opossum.cisreg.ca">http://opossum.cisreg.ca</a>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gene-regulation" class="hw-term hw-article-keyword hw-article-keyword-gene-regulation" rel="nofollow">gene regulation</a></li><li class="kwd"><a href="/keyword/chip-seq" class="hw-term hw-article-keyword hw-article-keyword-chip-seq" rel="nofollow">ChIP-Seq</a></li><li class="kwd"><a href="/keyword/transcription-factor-binding-sites-tfbs" class="hw-term hw-article-keyword hw-article-keyword-transcription-factor-binding-sites-tfbs" rel="nofollow">transcription factor binding sites (TFBS)</a></li><li class="kwd"><a href="/keyword/transcription" class="hw-term hw-article-keyword hw-article-keyword-transcription" rel="nofollow">transcription</a></li></ul><p id="p-6">The properties of cells within an organism are defined by a complex interplay between proteins, RNA, and the genome, which can be conceptualized as the gene regulatory network. Two important components of the gene regulatory network are the DNA-binding <em>trans</em>-acting transcription factors (TF) and their corresponding transcription factor binding sites (TFBS) in the DNA. Sets of proximal TFBSs that are sufficient to cooperatively mediate TF-regulated patterns of expression constitute <em>cis</em>-regulatory modules (CRM). CRMs are the scaffold for combinatorial TF interactions, enabling a limited number of sequence-specific DNA binding TFs to participate in an exponential number of combinations, each potentially conferring specific patterns of gene activity (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Arnone and Davidson 1997</a>).</p><p id="p-7">In studying gene regulation within a cell or tissue, researchers are commonly confronted with the need to analyze sets of genes sharing a characteristic, such as co-expression, as they seek to infer properties of the gene regulatory network. A significant insight into the regulatory network structure is obtained when the mediating TFs for the observed expression patterns are identified. A key strategy in genome biology for determining such TFs is to determine the sequence motifs that are over-represented in the <em>cis</em>-regulatory regions relative to some control. The successful predecessors to oPOSSUM-3, oPOSSUM (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Ho Sui <em>et al.</em> 2005</a>) and oPOSSUM-2 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ho Sui <em>et al.</em> 2007</a>), were developed to identify statistically over-represented, predicted TFBS in co-regulated gene sets. Two complementary scoring methods measured the over-representation: (1) Z-scores based on normal approximation to the binomial distribution that measures the change in the relative number of TFBS motifs in the foreground gene set compared with the background set, and (2) Fisher scores based on a one-tailed Fisher exact probability assessing the number of genes with the TFBS motifs in the foreground set <em>vs.</em> the background set. Using the JASPAR database as the source of DNA binding profiles (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Portales-Casamar <em>et al.</em> 2010</a>), the original oPOSSUM was designed to identify over-represented TFBSs, later referred to as Single Site Analysis (SSA). The original system also incorporated a conservation filter using phylogenetic footprinting based on pairwise alignments of orthologous sequences from human and mouse. In oPOSSUM-2, an additional analysis method called Combination Site Analysis (CSA) was introduced to identify over-represented proximal pairs of TFBSs. Separate oPOSSUM-2 implementations were released for two additional model organisms (<em>C. elegans</em> and <em>S. cerevisiae</em>). The nematode oPOSSUM-2 database was based on alignments between <em>C. elegans</em> and <em>C. briggsae</em>. The oPOSSUM-2 yeast system did not incorporate conservation filters, as the compact nature of the yeast genome results in dramatically reduced search space and noise compared with larger genomes. The oPOSSUM software is a highly cited tool for TFBS motif over-representation analysis (as assessed by Google Scholar citation counts), perhaps due to the ease of use and power of the approach. On average, excluding automated internet search software, 340 unique users work with oPOSSUM-2 each month.</p><p id="p-8">Since the release of the original oPOSSUM system, a plethora of TFBS over-representation analysis tools have been introduced. TOUCAN2, a workbench system for regulatory sequence analysis implemented by <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aerts <em>et al.</em> (2005)</a>, contains features for identifying over-represented TFBS in proximal promoters of co-regulated genes. <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Defrance and Touzet (2006)</a> developed the TFM-Explorer, which assesses conservation of spatial arrangements of regulatory elements. Promoter Analysis Pipeline by <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chang <em>et al.</em> (2007)</a> includes TFBS identification in gene sets as a component of the workbench, using non-redundant profiles from public databases. <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Piechota <em>et al.</em> (2010)</a> developed the cREMaG database, which attempts to correct for the confounding influence of variable information content of TFBS profiles, distinguishes between constitutive and inducible transcriptional forms of genes and reports the presence of CpG islands. Many of the methods provide web-based user interfaces, some of which are maintained. oPOSSUM-2 was found to perform well in an independent assessment of motif over-representation analysis tools (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Meng <em>et al.</em> 2010</a>).</p><p id="p-9">Since the implementation of these approaches, technology changes have greatly affected regulatory sequence studies. First, comprehensive multi-species sequence comparison measures are conveniently available in the form of phastCons and phyloP scores from the UCSC genome databases (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hubisz <em>et al.</em> 2011</a>). Phylogenetic footprinting, used by many TFBS enrichment programs, when performed with pairwise sequence alignments places emphasis on the quality of the choice of organism with which to compare and can sharply limit the number of genes that can be analyzed. Compared with pairwise alignments, multi-species analysis improves the quality of sequence alignments (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kumar and Filipski 2007</a>) and greatly increases the number of genes available for analysis. The proliferation of large-scale regulatory sequence profiling methods such as ChIP-Seq has demonstrated that TF-DNA interactions frequently occur outside of conserved regions (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Schmidt <em>et al.</em> 2010</a>). Genomic regions bound by TFs in ChIP experiments are a snapshot of a single cell-type and set of conditions, and not all regions are necessarily functional <em>cis</em>-regulatory sequences. In the absence of or in complement with experimental data, conservation is a useful filter for enabling computational motif enrichment analysis. Second, there has been a major update to the JASPAR database, an open-source, non-redundant, curated repository of TFBS profiles (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Portales-Casamar <em>et al.</em> 2010</a>). The update provides a significant increase in non-vertebrate profiles, permitting the extension of regulatory analysis software to many non-vertebrate species, such as insects. Third, widespread application of ChIP-Seq profiling has resulted in an explosion of the number of potential regulatory sequences to be analyzed (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Johnson <em>et al.</em> 2007</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Malhotra <em>et al.</em> 2010</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Schmidt <em>et al.</em> 2010</a>). These experiments produce sets of TF bound and control sequences, in which the foreground target (TF-bound) sets purportedly contain regulatory signatures of interest, whereas the background (control) sets lack those features. Such data are optimal for TFBS over-representation analysis, creating strong demand for a new generation of software that allows analysis from both a sequence-based perspective and a gene-based perspective.</p><p id="p-10">Here we describe oPOSSUM-3, a system that capitalizes upon the aforementioned research developments. The new system features a panel of upgraded and novel approaches to regulatory sequence analysis, including Single-Site Analysis (SSA) and anchored Combination-Site Analysis (aCSA) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). A novel extension of the system addresses the challenge imposed by homologous TFs with highly similar (or identical) binding specificity. Such profile similarity will be of increasing impact on motif enrichment analysis as the number of TF profiles depicting similar binding grows. The TFBS Cluster Analysis (TCA) and anchored Combination TFBS Cluster Analysis (aCTCA) present results focused on TFBS sequence patterns rather than individual profile names. This new approach to regulatory sequence motif over-representation analysis has been assessed against reference sets of co-regulated genes and large-scale ChIP-Seq sequence collections. Assessments against reference cases exemplify the utility of oPOSSUM-3 for the identification of mediating TFs. The new system should maintain the oPOSSUM service as a popular resource for motif over-representation analysis.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/987/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of the main analysis types available in oPOSSUM-3. The input for oPOSSUM can be either gene-based, which makes use of pre-computed results based on annotated genomic information, or sequenced-based, in which the user supplies the input sequences (e.g. ChIP-Seq results) for analysis. There are four methods available: (1) Single Site Analysis (SSA), (2) TFBS Cluster Analysis (TCA), (3) anchored Combination Site Analysis (aCSA), and (4) anchored TFBS Cluster Analysis (aCTCA). The first two methods apply enrichment analyses to individual TFs or TFBS clusters, whereas the latter two methods apply enrichment analyses to pairs of individual TFs or pairs of TFBS clusters. SSA and aCSA are depicted in this figure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1712042112" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Overview of the main analysis types available in oPOSSUM-3. The input for oPOSSUM can be either gene-based, which makes use of pre-computed results based on annotated genomic information, or sequenced-based, in which the user supplies the input sequences (&lt;em&gt;e.g.&lt;/em&gt; ChIP-Seq results) for analysis. There are four methods available: (1) Single Site Analysis (SSA), (2) TFBS Cluster Analysis (TCA), (3) anchored Combination Site Analysis (aCSA), and (4) anchored TFBS Cluster Analysis (aCTCA). The first two methods apply enrichment analyses to individual TFs or TFBS clusters, whereas the latter two methods apply enrichment analyses to pairs of individual TFs or pairs of TFBS clusters. SSA and aCSA are depicted in this figure.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/987/F1.medium.gif" width="440" height="411"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/987/F1.medium.gif" width="440" height="411"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/987/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/987/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461679" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-11" class="first-child">Overview of the main analysis types available in oPOSSUM-3. The input for oPOSSUM can be either gene-based, which makes use of pre-computed results based on annotated genomic information, or sequenced-based, in which the user supplies the input sequences (<em>e.g.</em> ChIP-Seq results) for analysis. There are four methods available: (1) Single Site Analysis (SSA), (2) TFBS Cluster Analysis (TCA), (3) anchored Combination Site Analysis (aCSA), and (4) anchored TFBS Cluster Analysis (aCTCA). The first two methods apply enrichment analyses to individual TFs or TFBS clusters, whereas the latter two methods apply enrichment analyses to pairs of individual TFs or pairs of TFBS clusters. SSA and aCSA are depicted in this figure.</p><div class="sb-div caption-clear"></div></div></div><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Nomenclature</h3><p id="p-12">Throughout this article, we refer to TFs and genes using a capitalized first letter followed by lowercase letters; all suffix characters are capitalized (<em>e.g.</em>, Mef2A).</p></div><div id="sec-3" class="subsection"><h3>Data sources</h3><p id="p-13">Gene and transcript annotation and genomic sequences were retrieved from Ensembl v64 except for <em>C. elegans</em>, which uses v54 for compatibility with conservation scores provided by the UCSC Genome Browser (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Flicek <em>et al.</em> 2010</a>). For <em>C. elegans</em>, operon annotations were retrieved from Wormbase WS200 (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Harris <em>et al.</em> 2010</a>). The phastCons scores were retrieved from the UCSC Genome Browser, based on the following score sets: (1) for human (hg19), phastCons46wayPlacental; (2) for mouse (mm9), phastCons30wayPlacental; (3) for fruit fly (dm3), phastCons15way; and (4) for nematode (ce6), phastCons6way. As UCSC ce6 database for <em>C. elegans</em> is based on WS190, genomic regions that had insertions or deletions between WS190 and WS200 were excluded. All “known” genes in the Ensembl databases were included in the database. For TFBS profiles, the 2010 release of the JASPAR database was used; all profiles from the CORE and PBM collections were included. A custom profile collection (referred to as PENDING), which is not included in JASPAR, was implemented to include profiles of interest for our analysis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003202/-/DC1/FigureS1.pdf">Figure S1</a>).</p></div><div id="sec-4" class="subsection"><h3>Calculation of TFBS motif over-representation</h3><p id="p-14">Two statistical measures are used to determine the TFBS motifs that are over-represented in the foreground set <em>vs.</em> the background set, representing two different models for the occurrences of TFBSs. For sequence-based analysis, a third statistical measure of motif centrality is calculated.</p><div id="sec-5" class="subsection"><h4>Z-scores:</h4><p id="p-15">The Z-score is used to assess whether the rate of occurrence of a given TFBS in the foreground sequence set differs significantly from the expected rate calculated from the background set based on a simple binomial distribution model. For a given TFBS, let <em>X</em> denote the number of predicted binding site nucleotides in the foreground sequence set, and <em>B</em> the number of predicted binding site nucleotides in the background set. If <em>n</em> is the total number of nucleotides in the foreground set, and <em>N</em> the total number for the background set, the expected value of <em>X</em> is given by <em>µ = BC</em>, where <em>C = n/N</em>. The probability of success is given by <em>P = B/N</em>, and the standard deviation is <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/987/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/987/embed/mml-math-1.gif"/></noscript></span></span>. Then, if <em>x</em> is the observed number of binding site nucleotides in the foreground set, using the normal approximation to the binomial distribution with a continuity correction of 0.5, the Z-score can be calculated as <em>Z = (x−µ−0.5)/σ</em>. The continuity correction is applied to correct for the normal distribution being continuous while the binomial distribution is discrete. The continuity correction (0.5) is subtracted from <em>x</em> because we are interested in the probability that <em>x</em> is strictly less than a given value. The calculation was implemented in the R statistics package, as were the subsequent enrichment scores (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">R Development Core Team 2008</a>).</p></div><div id="sec-6" class="subsection"><h4>Fisher scores:</h4><p id="p-16">The Fisher probability test is used to determine the probability of a non-random association between the foreground sequence set and a given TFBS by comparing the proportion of foreground sequences containing a given TFBS with the proportion of background sequences with that site. The Fisher probability is calculated using a hypergeometric probability distribution, which describes sampling without replacement from a finite population consisting of two types of elements. The Fisher scores are obtained by taking the negative logarithm of the probabilities (natural logarithm is used). In contrast to the Z-score, only the presence or absence of a TFBS in a given sequence is considered; the number of occurrences of a TFBS is not included in the probability calculation.</p></div><div id="sec-7" class="subsection"><h4>Centrality KS scores:</h4><p id="p-17">For sequence-based analysis methods, the Kolmogorov-Smirnoff (KS) test is used to compare the empirical distributions of the TFBS locations between target and background sets. In ChIP-Seq experiments, the highest number of sequence tags under a peak region is expected to occur where there is specific binding by the target TF to the DNA, such that functional TFBSs are located at or in close proximity to this maximum confidence position (MCP). Thus, in target sequences, the distances of the target TFBSs to the MCP (which we refer to as the DistMCP) are expected to be clustered around zero, whereas in background sequences, the binding sites would be distributed randomly. An example is given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003202/-/DC1/FigureS2.pdf">Figure S2</a>, which shows the binding site distributions in the Nfe2L2 ChIP-Seq dataset by <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Malhotra <em>et al.</em> (2010)</a>. <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Van Helden <em>et al.</em> (2000)</a> employed such position analysis for detecting 3′ signals in yeast genes within the RSAT package. Building on this concept, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bailey and Machanick (2012)</a> implemented the CentriMo tool as part of the MEME suite. By comparing the distributions of DistMCPs for a given TF in the target and background sequences, we can identify those TFBSs that are positionally enriched relative to the MCP in the target set. As for Fisher analysis, KS scores are obtained by taking negative logarithms of the p-values obtained from KS tests.</p></div><div id="sec-8" class="subsection"><h4>False Discovery Rate calculation:</h4><p id="p-18">The False Discovery Rate (FDR) procedure was used to calculate adjusted p-values for both Z-scores and Fisher scores in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Tables 2</a>–<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">5</a> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Benjamini and Hochberg 1995</a>). Z-scores were first converted to p-values using the standard normal table, whereas for Fisher scores, the p-values obtained during Fisher exact probability calculation were used. All calculations were performed in the R statistics package (p.adjust for FDR and pnorm for the standard normal table). As mentioned in the original oPOSSUM article (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Ho Sui <em>et al.</em> 2005</a>), we do not perceive that TFBS over-representation measures are appropriate significance tests, as the procedures do not fully account for the non-random properties of genome sequences. The FDR values in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Tables 2</a>–<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">5</a> were provided for completeness, but they do not affect the relative rankings.</p></div></div><div id="sec-9" class="subsection"><h3>Single Site Analysis</h3><p id="p-19">Single Site Analysis (SSA) is the original over-representation analysis procedure developed for oPOSSUM. Individual TFBS hits within the user-specified search regions are counted for both the foreground gene/sequence set and the background set. For gene-based analysis, the search regions are determined by selecting the conserved regions located within a given distance from the gene transcription start sites. Both the Z-scores and Fisher scores are calculated for each TFBS based on the counts, which are then ranked accordingly. For a detailed explanation of the oPOSSUM SSA, please refer to <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Ho Sui <em>et al.</em> (2005)</a>. The major difference in the SSA implementation between the previous versions of oPOSSUM and version 3 is the replacement of the pairwise sequence alignment-based phylogenetic footprinting procedure with phastCons scores. Conserved regions are defined as those genomic regions of at least 20 nucleotides in length with average phastCons scores over a given threshold (two sets of conserved regions are determined using thresholds of 0.4 and 0.6), and TFBS searches are restricted to motifs within or overlapping (by at least 1 bp) these regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003202/-/DC1/FigureS3.pdf">Figure S3</a>).</p></div><div id="sec-10" class="subsection"><h3>Anchored Combination Site Analysis (aCSA)</h3><p id="p-20">In many cases, given a gene set of interest, the user has beforehand knowledge of a TF that is central to the observed co-regulation. To focus the intensive computational analysis of motif combinations, the aCSA method restricts the over-representation assessment to sequence regions proximal to predicted TFBS for a user-specified TF. Given an anchoring TF and an inter-binding site distance parameter value, pairings of the anchoring TFBS and all available secondary TFBSs within the inter-binding site distance are counted and over-representation scores are calculated. It is possible to observe self-interactions, when TFBSs for the anchor TF frequently occur in clusters.</p></div><div id="sec-11" class="subsection"><h3>Gene-based <em>vs.</em> sequence-based</h3><p id="p-21">Increasingly researchers focus on sets of sequences likely to be bound by a TF, rather than a list of genes. To facilitate the analysis of sequence-based data, such as those from ChIP-Seq studies, new functions within oPOSSUM-3 were implemented. The sequence-based oPOSSUM systems take as input a foreground sequence set and a background sequence set. The supplied sequences are searched for TFBS motifs, and over-representation scores are calculated. There is no conservation filtering applied to the input sequences; the entire set of submitted sequences is screened for TFBS motifs.</p></div><div id="sec-12" class="subsection"><h3>TFBS clustering</h3><p id="p-22">Transcription factors can be classified according to structural characteristics of DNA binding domains (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). It is often the case that TFs belonging to the same structural classes share similar DNA binding profiles, such that distinct TFs of the same class may bind to similar sequence patterns. If a given class contains numerous TFs with almost identical consensus sequences, oPOSSUM-3 analysis results can be dominated by subsets of profiles that are nearly identical. In such cases, it is useful to condense the redundant results to allow the user to identify independent enriched profiles. It is not suitable to combine the results simply based on the TF class, as the extent of the binding sequence similarity is variable among the different classes. Although some classes are defined by a characteristic consensus sequence, other classes, such as zinc fingers, have low profile similarities among the member TFs. Thus, it is necessary to divide each structural class into clusters based on profile similarity. <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> illustrates the idea behind TFBS clustering and its application to oPOSSUM-3.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/987/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="TF structural families and TFBS clusters. As many TFs cannot be distinguished in their binding specificity, clustering TF binding profiles promotes user consideration of all members of a functionally equivalent group. As TFs within a group bind to essentially identical sequences, users should focus on those TFs within a group likely to be active in a cell or condition relevant to their research. In the figure, the outer shapes and color represent the structural class, and the inner shapes symbolize the binding specificity of a cluster. The different shading within the shapes denotes individual TFs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1712042112" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;TF structural families and TFBS clusters. As many TFs cannot be distinguished in their binding specificity, clustering TF binding profiles promotes user consideration of all members of a functionally equivalent group. As TFs within a group bind to essentially identical sequences, users should focus on those TFs within a group likely to be active in a cell or condition relevant to their research. In the figure, the outer shapes and color represent the structural class, and the inner shapes symbolize the binding specificity of a cluster. The different shading within the shapes denotes individual TFs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/987/F2.medium.gif" width="440" height="238"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/987/F2.medium.gif" width="440" height="238"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/987/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/987/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461826" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-23" class="first-child">TF structural families and TFBS clusters. As many TFs cannot be distinguished in their binding specificity, clustering TF binding profiles promotes user consideration of all members of a functionally equivalent group. As TFs within a group bind to essentially identical sequences, users should focus on those TFs within a group likely to be active in a cell or condition relevant to their research. In the figure, the outer shapes and color represent the structural class, and the inner shapes symbolize the binding specificity of a cluster. The different shading within the shapes denotes individual TFs.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/987/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="TFBS cluster analysis. (A) TFBS clustering process. Binding site profiles (represented by solid blue circles) are compared, and those exhibiting similarity (all pairwise similarity scores within a threshold T) are classified as a group (placed within the blue ring). In a second step, a consensus of the initial set is generated, and all additional profiles within the radius margin threshold (represented by the green ring) are added to the group. (B) Individual TFBS hits within a DNA sequence are grouped together according to the clustered groups, and over-representation analysis is performed from the group perspective. Each labeled red rectangle (e.g., CR1) represents a cluster of TFBSs (individually displayed as small rectangles below)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1712042112" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;TFBS cluster analysis. (A) TFBS clustering process. Binding site profiles (represented by solid blue circles) are compared, and those exhibiting similarity (all pairwise similarity scores within a threshold T) are classified as a group (placed within the blue ring). In a second step, a consensus of the initial set is generated, and all additional profiles within the radius margin threshold (represented by the green ring) are added to the group. (B) Individual TFBS hits within a DNA sequence are grouped together according to the clustered groups, and over-representation analysis is performed from the group perspective. Each labeled red rectangle (&lt;em&gt;e.g.&lt;/em&gt;, CR1) represents a cluster of TFBSs (individually displayed as small rectangles below).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/987/F3.medium.gif" width="307" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/987/F3.medium.gif" width="307" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/987/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/987/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461771" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-24" class="first-child">TFBS cluster analysis. (A) TFBS clustering process. Binding site profiles (represented by solid blue circles) are compared, and those exhibiting similarity (all pairwise similarity scores within a threshold T) are classified as a group (placed within the blue ring). In a second step, a consensus of the initial set is generated, and all additional profiles within the radius margin threshold (represented by the green ring) are added to the group. (B) Individual TFBS hits within a DNA sequence are grouped together according to the clustered groups, and over-representation analysis is performed from the group perspective. Each labeled red rectangle (<em>e.g.</em>, CR1) represents a cluster of TFBSs (individually displayed as small rectangles below).</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Profiles in JASPAR 2010 have been annotated for TF structural class and family. Based on these classifications, the profiles in the given family are subject to a refined clustering process in oPOSSUM-3. First, using the MatrixAligner similarity scoring program (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sandelin <em>et al.</em> 2003</a>), a pairwise similarity score table is calculated for the entire set of profiles in JASPAR. Two thresholds are set: (1) cluster score threshold T, which is the MatrixAligner score above which the two matrices being compared are deemed to be similar, and (2) radius margin R, which is a secondary score threshold used to determine whether those TFs at the boundary of the cluster join the cluster (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The process is based on tree traversal, with nodes being the profiles and the edges being the similarity scores. One profile within a given family is randomly chosen to act as the seed node for the cluster, and a tree is constructed between this seed profile and all other profiles in the family. The nodes are traversed in sequence, and the traversed nodes are added to the cluster if (1) the similarity scores between the parent node and the child node are lower than the cluster score threshold T, and (2) the average score S between the cluster member nodes (the parent nodes that have already been included in the cluster) and the child node is below T. If the child node in question qualifies for condition 1 but not condition 2, it can still be included in the cluster if S exceeds T by less than R. A pseudocode of the clustering process is given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003202/-/DC1/FigureS4.pdf">Figure S4</a>. From the JASPAR 2010 database, 250 profiles from the CORE collection and 184 from the PBM collection were analyzed, along with the 4 profiles from the custom PENDING collection. A cluster score threshold of 1.8 and a radius margin of 0.1 were used. These values were selected empirically, based on the distribution of pairwise similarity scores among all available JASPAR profiles.</p><p id="p-26">When a TFBS cluster-based analysis is performed, any overlapping TFBS hits that belong to the same TFBS cluster are merged to form a single cluster hit. Only the merged cluster hits are counted for TFBS over-representation calculations (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>).</p></div><div id="sec-13" class="subsection"><h3>Species-specific implementation details</h3><p id="p-27">The amount of sequence analyzed for the pre-computation of putative TFBS locations for the gene-based over-representation variants was adjusted to reflect the intergenic distances (unpublished observations) of the target organism. For human and mouse, 10,000 bp upstream and 10,000 bp downstream from the Ensembl-annotated TSS were searched for TFBS hits. For fruit fly, 3000 bp upstream and 3000 bp downstream were searched, and for nematode, 1500 bp in each direction. For yeast, 1000 bp upstream of TSSs and downstream to the 3′ end of each gene were searched. In gene-based pre-calculations for invertebrates, exons were excluded, as in general they are highly conserved and thus the conservation filter employed to reduce false positive TFBSs is rendered ineffective. As opposed to vertebrates and insects, nematodes exhibit some operon structures of gene organization (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Blumenthal and Gleason 2003</a>). An operon consists of multiple adjacent genes that are transcribed as a single unit, which is then spliced into separate mRNAs for translation. When analyzing genes that are part of an annotated operon, oPOSSUM-3 accounts for the operon structures by restricting the search space to the regions flanking the annotated start position of the first gene in the operon. TFBS predictions in the search region of the first gene are deemed to apply to all other genes in the operon (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003202/-/DC1/FigureS5.pdf">Figure S5</a>).</p><p id="p-28">Although JASPAR now includes distinct divisions for nematode and insect TFBS profiles, coverage is not optimal. For insects, the homeodomain family heavily dominates the set of profiles (90 of 125). For worms, there are currently only 4 profiles in the database. Thus, whereas human and mouse oPOSSUM-3 databases are built with vertebrate profiles only, fruit fly and nematode versions are built with all available metazoan profiles in JASPAR 2010.</p><p id="p-29">The yeast gene-based oPOSSUM-3 database differs from the metazoan versions of oPOSSUM-3 in that it does not use conservation filtering, reflecting the compact promoter regions analyzed.</p></div><div id="sec-14" class="subsection"><h3>Build process for oPOSSUM-3</h3><p id="p-30"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> outlines the build process for gene-based versions of oPOSSUM-3. The system imports the gene and transcript annotation data from the Ensembl database and the phastCons conservation scores from the UCSC database. These data are combined to define the conserved segments in each gene. TFBS profiles from the JASPAR database are used to search for putative TFBSs within or overlapping the conserved regions. These TFBSs are stored in the oPOSSUM-3 database as well as the resulting counts (both single site and cluster sites) for pre-determined conservation thresholds, search region lengths, and profile matrix scoring thresholds. This pre-computation facilitates faster analysis for the user if these pre-determined search regions and thresholds are used for analysis. If the user chooses custom values of these search region lengths and thresholds, TFBS counts are computed at the time of the analysis. The sequence-based oPOSSUM-3 foregoes this pre-computation pipeline, and all calculations are done at the time of analysis.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/987/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The build process for oPOSSUM 3 gene-based analysis. The system incorporates data from the gene annotation data from Ensembl, TFBS profiles from JASPAR, and multi-species conservation information based on phastCons scores from UCSC Genome Browser. The oPOSSUM database incorporates the data from these sources to pre-compute TFBS profile hits for gene-based analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1712042112" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The build process for oPOSSUM 3 gene-based analysis. The system incorporates data from the gene annotation data from Ensembl, TFBS profiles from JASPAR, and multi-species conservation information based on phastCons scores from UCSC Genome Browser. The oPOSSUM database incorporates the data from these sources to pre-compute TFBS profile hits for gene-based analysis.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/987/F4.medium.gif" width="440" height="246"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/987/F4.medium.gif" width="440" height="246"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/987/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/987/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461825" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-31" class="first-child">The build process for oPOSSUM 3 gene-based analysis. The system incorporates data from the gene annotation data from Ensembl, TFBS profiles from JASPAR, and multi-species conservation information based on phastCons scores from UCSC Genome Browser. The oPOSSUM database incorporates the data from these sources to pre-compute TFBS profile hits for gene-based analysis.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Data sources for ChIP-Seq–based analyses</h3><p id="p-32">ChIP-Seq datasets for Nfe2L2 (mouse embryonic fibroblast) and FoxA2 (mouse liver) were obtained in their final processed form from their respective authors (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Malhotra <em>et al.</em> 2010</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Robertson <em>et al.</em> 2008</a>). The cMyc and Sox2 ChIP-Seq datasets [mouse embryonic stem cells (mES)] (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chen <em>et al.</em> 2008</a>) were obtained as BED files from Gene Expression Omnibus (GEO accession GSE11431) (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Edgar <em>et al.</em> 2002</a>) and processed into peaks (putative TF bound regions) using the FindPeaks algorithm (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Fejes <em>et al.</em> 2008</a>) with the following settings: -control -dist_type 1 200 -subpeaks 0.6 -trim 0.2 -duplicatefilter. All NCBI36/mm8 datasets were converted to build NCBI37/mm9 using the UCSC Utility: Batch Coordinate Conversion (liftOver).</p></div><div id="sec-16" class="subsection"><h3>ChIP-Seq–related analysis</h3><p id="p-33">Data analyses and plots were derived using the R statistical package (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">R Development Core Team 2008</a>).</p><div id="sec-17" class="subsection"><h4>Selection of foreground datasets:</h4><p id="p-34">The FoxA2-, cMyc-, and Sox2-bound regions for our analyses were identified using a ratio of peak height to width, and the top 1200 ranked regions were selected. Nfe2L2 had undergone stringent filtering by the authors and had a strong signal in the data; thus the full dataset of 1256 sequences was used. For consistency, we chose similarly sized datasets for our analyses because the over-representation scores are dependent upon the data size (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003202/-/DC1/FigureS6.pdf">Figure S6</a>).</p></div><div id="sec-18" class="subsection"><h4>Selection of background datasets:</h4><p id="p-35">Background datasets were extracted from the control data of the related ChIP-Seq experiments. All backgrounds were selected to match the mononucleotide GC composition of the foreground, except where we tested the impact of using different background GC compositions. The default background size for our analyses was 2-fold greater than the matching foreground, except for Nfe2L2 where the fibroblast background was limited to the same size as the foreground to obtain sequences with GC composition distribution similar to the foreground. Data analyses and plots were derived using the R statistical package (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">R Development Core Team 2008</a>).</p></div><div id="sec-19" class="subsection"><h4>Thresholding by score:</h4><p id="p-36">The ranges of the Z-score and Fisher score enrichment values are dependent on foreground dataset size (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003202/-/DC1/FigureS6.pdf">Figure S6</a>). A threshold was derived using the mean plus N times the standard deviation of the score of interest (where N = 2 for Z-score thresholds, and N = 1 for Fisher score thresholds) (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003202/-/DC1/FigureS7.pdf">Figure S7</a>). KS scores can be undefined (infinite) for TFs of interest; for depiction, a score of 100 is assigned. Calculation of candidate thresholds excludes these values (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003202/-/DC1/FigureS8.pdf">Figure S8</a>). Hereafter, the threshold is referred to as the applied threshold. In cases where we assess the correlation between reported results, we used lists of length 5 through M, where M was calculated to be four times the number of TFs with scores greater than the applied threshold. We chose to make M lenient in terms of the number of TFs that are enriched to be thorough.</p></div><div id="sec-20" class="subsection"><h4>GC composition calculation:</h4><p id="p-37">Sequence composition of ChIP-Seq sequences was determined at the mononucleotide level, using counts of A, G, C, and T. TF profile composition was determined similarly by counting the frequency of the nucleotides in all the sequences contributing to the profile. All backgrounds used in testing were selected to reflect the sequence GC composition distribution of the foreground dataset.</p></div></div><div id="sec-21" class="subsection"><h3>Analysis parameters</h3><p id="p-38">All analyses were performed with the default parameters, unless otherwise stated. Default settings restrict oPOSSUM-3 results to those profiles with information content of at least 8 bits, and to putative TFBSs whose score is at least 85% of the optimal score. For gene-based analyses, species-specific default search region distances were used (corresponding to Level 3 in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461803/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461803/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Search region level distances for human/mouse, fly and nematode in oPOSSUM-3. oPOSSUM-3 searches for the presence of motifs in non-coding regions meeting a specified conservation threshold and situated within a selected search region.</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section other1" id="sec-22"><h2 class="">System Walk-Through</h2><p id="p-39">oPOSSUM-3 is available as a web service at <a href="http://opossum.cisreg.ca">http://opossum.cisreg.ca</a>. The main page of oPOSSUM-3 lists each of the analysis methods offered as well as the species supported, allowing the user to select the appropriate combination. <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> depicts the oPOSSUM-3 analysis pipeline for both the gene-based and sequence-based variants, and <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> depicts the user interface.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/987/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="oPOSSUM analysis pipeline. Dashed lines indicate optional stages." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1712042112" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;oPOSSUM analysis pipeline. Dashed lines indicate optional stages.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/987/F5.medium.gif" width="440" height="375"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/987/F5.medium.gif" width="440" height="375"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/987/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/987/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461693" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-40" class="first-child">oPOSSUM analysis pipeline. Dashed lines indicate optional stages.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/987/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Web interface. (A) oPOSSUM home page listing the various analyses and organisms available. (B) Four steps for initiating a Single Site Analysis (SSA) with user-chosen parameters." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1712042112" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Web interface. (A) oPOSSUM home page listing the various analyses and organisms available. (B) Four steps for initiating a Single Site Analysis (SSA) with user-chosen parameters.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/987/F6.medium.gif" width="229" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/987/F6.medium.gif" width="229" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/987/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/987/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461795" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 </span> <p id="p-41" class="first-child">Web interface. (A) oPOSSUM home page listing the various analyses and organisms available. (B) Four steps for initiating a Single Site Analysis (SSA) with user-chosen parameters.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-23" class="subsection"><h3>Input for gene-based analysis</h3><p id="p-42">For a gene-based analysis, the system takes as input the list of identifiers (ID) of the genes to be analyzed. Although the default input ID type is an Ensembl gene ID, the system accepts nine alternatives, including official gene symbols and Uniprot IDs. For the background gene set, the user can (1) choose the entire gene set from the oPOSSUM-3 database, (2) specify a number of genes to be randomly chosen, or (3) supply a list of gene IDs.</p><p id="p-43">The gene-based analysis can be performed in either default or custom modes. In default mode, the user is restricted to pre-defined search regions, sequence conservation levels, and TFBS score thresholds. While in custom mode, the user can specify the parameter values within ranges allowed by the system. The default mode is faster as it makes use of pre-computed TFBS counts from the database, whereas the custom mode requires the system to compile binding site frequencies. For SSA, the user may select profiles from species-dependent subsets of JASPAR CORE, JASPAR PBM, or oPOSSUM-3-specific PENDING collections. The PENDING collection contains profiles that are not available from the 2010 release of JASPAR. The user specifies whether to use the entire set or a selected subset of the profiles. For aCSA, the user must select an anchor profile. For the TFBS cluster-based versions (both TCA and aCTCA), instead of selecting individual TFBS profiles, the user selects the TF of interest. The system will then include all TFBS clusters that belong to the selected TF families.</p></div><div id="sec-24" class="subsection"><h3>Input for sequence-based analysis</h3><p id="p-44">The input parameter selection for the sequence-based methods is simpler than it is for the gene-based methods, as there are no search region or conservation level settings to specify. The user must supply fasta formatted sequences for both foreground and background datasets to be scanned for TFBS hits. oPOSSUM-3 provides a link to the Galaxy service for users needing to generate fasta files from sequence coordinates (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Goecks <em>et al.</em> 2010</a>). oPOSSUM-3 provides background sets for users lacking a matched background for their data. Lastly, the user must select TFBS profiles to be included in the analysis or provide a set of custom profiles.</p></div><div id="sec-25" class="subsection"><h3>Parameter considerations</h3><p id="p-45">For all analyses, we suggest using default parameters, as we have done in this article, to get an analysis started when a user is uncertain of what to expect from their data. After the default analysis, users may elect to alter parameters for further investigation. It can be informative to compare how results change as parameters are altered. For instance, a score of 85% is the default threshold for reporting TFBSs; we have found this setting to work well with our datasets. Users may start with this threshold and decide that the results returned are missing a known TFBS, and so on the next analyses, they might reduce the TFBS stringency from 85 to 80%. Another example would be the region to analyze for gene-based analyses. The default distances are 5000 bp both up- and downstream; however, users may be interested to determine whether there are TFBSs enriched predominantly in upstream regions that differ from TFBSs enriched in downstream regions. In this case, two separate analyses would be run with different restrictions on the search regions.</p></div><div id="sec-26" class="subsection"><h3>Results output</h3><p id="p-46">The results are generated and returned in a table format, along with a summary of the user-specified parameters and the nucleotide compositions of the foreground and background sequences used in the analysis. In gene-based SSA, the results table is generated and returned to the user immediately, whereas for gene-based aCSA, TCA, and aCTCA, as well as for all sequence-based methods, a link to the results is emailed to the user after computations are complete. If needed, the user can download the results as a tab-delimited text file for further analysis or record keeping. By default, the results in the table are ranked by descending Z-score, and the user can specify the number of motifs to report based on one score (default is all results). The table can be re-sorted by any column by clicking on the column header. The TF names are linked to the corresponding entries from the JASPAR web site. As a profile’s information content (IC) can affect its rate of occurrence, those profiles with IC lower than 9 or greater than 19 are highlighted to inform the user of extreme cases. The TFBS counts are linked to a separate TFBS details page that specifies the sequences and locations. For sequence-based analysis, the report specifies the overall GC mononucleotide content for both the background and foreground sequence sets. For both gene-based and sequence-based SSA and aCSA, the results table reports the GC content of each TF binding profile and highlights the extreme values (<em>i.e.</em>, those that are below 0.33 or above 0.66). For TFBS cluster-based versions, the results table entries are based on TFBS cluster names instead of individual profiles. The TFBS cluster hit coordinates and the corresponding sequences represent the merged TFBS hits. Each TFBS cluster name is linked to a separate page with the summary information on the cluster, such as TF class, family, and the member profiles that constitute the cluster.</p><p id="p-47">Regarding interpretation of the results, we strongly recommend that users consider results plotted as we have done in this article [<em>i.e.</em>, in <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003202/-/DC1/FigureS8.pdf">Figure S8</a>, the score of interest (Z-score, Fisher score, or KS score) <em>vs.</em> the GC composition of the TF profile (only for SSA and aCSA), and in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003202/-/DC1/FigureS7.pdf">Figure S7</a>, the Fisher score <em>vs.</em> Z-score]. This visualization of the data will alert the user if a skew in TF ranking has occurred due to imbalanced composition of the background relative to the foreground. If the user wants an alternative computable metric, we recommend as a first pass setting a score threshold using the median plus N times the standard deviation (see <em>Materials and Methods</em>). For SSA and TCA, start with N = 2 for Z-score, and N = 1 for Fisher score; for aCSA or aTCA, start with N = 2 or 3 for either score. It is useful to consider these thresholds in conjunction with the aforementioned plots as a means for deciding whether the threshold is too stringent for the user’s purposes.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/987/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between TF profile GC content and enrichment statistics. The percentage of G and C nucleotides in the TF profile models are plotted against the motif enrichment Z-scores. The three panels represent analysis results for the same 1256 Nfe2L2 ChIP-Seq regions (GC composition avg. 43%) compared with three background sets of different GC composition: (A) elevated background GC (avg. 51% GC); (B) low background GC (avg. 37% GC); and (C) background with GC composition matched to the distribution of the ChIP-Seq regions (avg. 43% GC). The GC composition of the background used in B is that of the control associated with the Nfe2L2 ChIP-Seq data. The plotted Z-scores represent the enrichment of TFs in 1256 ChIP-Seq Nfe2L2-bound regions. The Nfe2L2 profile can be distinguished in all three cases (A, B, and C). However, detection of the Ap1 profile, representing a TF with a known Nfe2L2-related biological function, is sensitive to background selection. Most TFs that would be ranked highly by Z-score when the background is not corrected for GC composition (seen in A and B) are not ranked highly when the background is matched to the foreground distribution of GC composition (C)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1712042112" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relationship between TF profile GC content and enrichment statistics. The percentage of G and C nucleotides in the TF profile models are plotted against the motif enrichment Z-scores. The three panels represent analysis results for the same 1256 Nfe2L2 ChIP-Seq regions (GC composition avg. 43%) compared with three background sets of different GC composition: (A) elevated background GC (avg. 51% GC); (B) low background GC (avg. 37% GC); and (C) background with GC composition matched to the distribution of the ChIP-Seq regions (avg. 43% GC). The GC composition of the background used in B is that of the control associated with the Nfe2L2 ChIP-Seq data. The plotted Z-scores represent the enrichment of TFs in 1256 ChIP-Seq Nfe2L2-bound regions. The Nfe2L2 profile can be distinguished in all three cases (A, B, and C). However, detection of the Ap1 profile, representing a TF with a known Nfe2L2-related biological function, is sensitive to background selection. Most TFs that would be ranked highly by Z-score when the background is not corrected for GC composition (seen in A and B) are not ranked highly when the background is matched to the foreground distribution of GC composition (C).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/987/F7.medium.gif" width="440" height="141"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/987/F7.medium.gif" width="440" height="141"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/987/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/987/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461747" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7 </span> <p id="p-48" class="first-child">Relationship between TF profile GC content and enrichment statistics. The percentage of G and C nucleotides in the TF profile models are plotted against the motif enrichment Z-scores. The three panels represent analysis results for the same 1256 Nfe2L2 ChIP-Seq regions (GC composition avg. 43%) compared with three background sets of different GC composition: (A) elevated background GC (avg. 51% GC); (B) low background GC (avg. 37% GC); and (C) background with GC composition matched to the distribution of the ChIP-Seq regions (avg. 43% GC). The GC composition of the background used in B is that of the control associated with the Nfe2L2 ChIP-Seq data. The plotted Z-scores represent the enrichment of TFs in 1256 ChIP-Seq Nfe2L2-bound regions. The Nfe2L2 profile can be distinguished in all three cases (A, B, and C). However, detection of the Ap1 profile, representing a TF with a known Nfe2L2-related biological function, is sensitive to background selection. Most TFs that would be ranked highly by Z-score when the background is not corrected for GC composition (seen in A and B) are not ranked highly when the background is matched to the foreground distribution of GC composition (C).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section results" id="sec-27"><h2 class="">Results</h2><div id="sec-28" class="subsection"><h3>Application to reference collections</h3><p id="p-49">To validate the performance of the oPOSSUM-3 system, each of the analysis methods was tested using either sets of co-expressed genes or sequences identified in ChIP-Seq studies. The gene sets were restricted to cases in which there was prior knowledge of TF(s) responsible for co-expression. The ChIP-Seq sequences, by the nature of the ChIP method, are already self-restricted to a primary TF (the target of the antibody used in the experiments).</p></div><div id="sec-29" class="subsection"><h3>Skeletal muscle reference gene set</h3><p id="p-50">Skeletal muscle-specific genes are known to be regulated by a core set of TFs including (but not limited to) Mef2A, Myf, Srf, Tead1, and Sp1, with the first three performing prominent roles (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Braun <em>et al.</em> 1994</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Naya and Olson 1999</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Rudnicki and Jaenisch 1995</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Wasserman and Fickett 1998</a>). From the literature, a collection of 25 human genes regulated by muscle-specific enhancers was prepared (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003202/-/DC1/TableS1.pdf">Table S1</a>). This set was analyzed with the full panel of oPOSSUM-3 gene-based methods for human, and the results are shown in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>. Although rankings differ depending on the type of analysis performed and the scoring method used, one or more of the core muscle-specific TFs of Mef2A, Myf, and Srf are included in the top five enriched TF profiles by either the Z- or Fisher scores. SSA (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2A</a>) is able to identify the muscle factors Mef2A, Myf, Srf, and Sp1 as candidate TFs for regulating the muscle gene set, along with Nkx2-5, which is also a known muscle-specific TF (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Durocher <em>et al.</em> 1997</a>). Likewise, TCA (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2B</a>) yields the clusters C113 (Nkx2-5), C130 (Mef2A), C143 (Myf), and C129 (Srf). When run with Mef2A as the anchoring TF, aCSA (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2C</a>) reports known muscle regulatory TF candidate pairings with Mef2A, including Sp1, Nkx2-5, and Klf4 (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Yoshida <em>et al.</em> 2010</a>). Similarly for aCTCA (<a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2D</a>), Mef2A pairs with ETS domain TFs (a known muscle regulatory TF family) in addition to clusters containing Sp1 and Klf4.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461810/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461810/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">oPOSSUM-3 results for the muscle reference gene set (human).</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-30" class="subsection"><h3>Cilia gene set for nematodes</h3><p id="p-52"><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Inglis <em>et al.</em> (2006)</a> curated a collection of genes known to be involved in cilia function and structure, making the set available within Ciliome DB. In vertebrates, RFX TFs are the key regulators of cilia gene expression. The RFX TFs bind to a regulatory motif termed the X-box. The regulation is conserved between mammals and nematodes; X-box sequences are associated with orthologous cilia genes (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Efimenko <em>et al.</em> 2005</a>). Although a nematode-specific RFX motif has been computed from these sequences, these sequences were not experimentally validated in a rigorous manner. There are two RFX profiles in the research literature, differing in the distance between two half sites (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Emery <em>et al.</em> 1996</a>). These RFX profiles were added to the oPOSSUM-3 custom collection of TFBS profiles and were used to analyze 531 nematode cilia genes. The gene set was obtained from Ciliome DB by combining the nematode genes from Avidor-Reiss, Blacque SAGE, and Efimenko datasets and selecting for those genes that were included in at least two of the studies. The results are listed in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. RFX profiles received the highest scores in both SSA and TCA. The high scores obtained using the vertebrate RFX profiles in nematode genomic sequences support the conservation of the TFBS sequences across large evolutionary distances.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461845/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461845/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">oPOSSUM-3 results for the cilia gene set in nematodes.</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-31" class="subsection"><h3>Results for ChIP-Seq reference datasets</h3><p id="p-54">We used oPOSSUM-3 to analyze ChIP-Seq datasets, of which we describe two results here. The Nfe2L2 data from <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Malhotra <em>et al.</em> (2010)</a> contains relatively few ChIP-Seq regions (1256 sequences) derived from studies of mouse embryonic fibroblasts. Nfe2L2, previously known as Nrf2, is a stress-activated TF linked to the regulation of detoxification enzymes. We analyzed the Nfe2L2 data with oPOSSUM-3 (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). The Nfe2L2 profile is consistently found to be a top-scoring motif in all analyses. The Ap1 motif, known to overlap and coordinate with the Nfe2L2 binding sequence (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Friling <em>et al.</em> 1992</a>), was also enriched in the SSA results, and it was clustered with the Nfe2L2 profile (cluster <sup>11</sup>C) in the TCA and aCTCA analyses.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461727/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461727/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">oPOSSUM-3 results for Nfe2L2 ChIP-Seq dataset, using JASPAR CORE vertebrate profiles.</span><div class="sb-div caption-clear"></div></div></div><p id="p-56">For contrast, the FoxA2 data from <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Robertson <em>et al.</em> (2008)</a> shows a large number of TF bound regions (&gt;10,000 sequences) using mouse liver as the source. We took a subset of 1200 sequences for analysis (see <em>Materials and Methods</em>). FoxA2 (also known as Hnf3α), a TF linked to differentiation, is a member of the forkhead-box family of TF proteins for which the JASPAR 2010 database has eight profiles. Using the sequence-based SSA, we recovered FoxA2 as the top ranking TF, while additional forkhead-box family profiles were ranked in the top five enriched TFBSs (<a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a>). The other prominently enriched profile represents the Hnf4 hepatocyte nuclear factor, which is predominantly expressed in the liver. The TCA analysis complements the SSA results, with the clusters C57 representing forkhead-box binding profiles ranking first in all three scores. C19 (Hnf4A) ranks second by Z and Fisher scores, whereas C41 (Sox2, SRY) ranks second by KS score. aCSA and aCTCA analyses also support the importance of forkhead-box binding and Hnf4 profiles.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461749/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461749/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">oPOSSUM-3 results for FoxA2 ChIP-Seq dataset, using JASPAR CORE vertebrate profiles.</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-32" class="subsection"><h3>Accounting for GC composition in over-representation analyses</h3><p id="p-58">The genomes are composed of regions that are distinct by the nature of their nucleotide composition. Promoters for housekeeping genes in particular are often found to be rich in G and C nucleotides or in CpG islands (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Yamashita <em>et al.</em> 2005</a>), whereas tissue-specific promoters are often found to be relatively more AT-rich (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Roider <em>et al.</em> 2009</a>). These broad composition differences are reflected among the sequences derived from ChIP experiments, such as ChIP-Seq. The %GC content of the published ChIP-Seq experiments we examined ranged from ∼30–57% GC (unpublished observations). The mean and variance of the %GC composition distribution differed between experiments for different TFs. The binding profiles of the TFs themselves also varied in GC content. Given a set of GC-rich sequences to analyze, TFs with GC-rich profiles will have a greater probability of having a motif present than AT-rich profiles. As over-representation analysis is designed to report TFBS predictions that exist to a greater extent in a set of chosen sequences than in a background set of sequences, we examined the effect of GC composition on motif enrichment scoring using backgrounds with varying GC content (<em>i.e.</em>, high, low, and neutral GC composition) against several ChIP-Seq datasets.</p><p id="p-59">We used mouse-derived ChIP-Seq data for several TFs with varying GC composition in their profiles: Sox2 (ES cells), 36% GC; Nfe2L2 (fibroblast), 43% GC; FoxA2 (liver), 46% GC; and cMyc (ES cells), 61% GC. For each ChIP-Seq dataset, we analyzed a subset of the TF bound regions (foreground data) against background sequence sets with a variety of %GC distributions. The over-representation Z-score for each TF was then plotted against the %GC composition of the TF’s profile. As illustrated with Nfe2L2 in <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>, composition differences between the foreground and background sequences will affect the over-representation scores of TFs. When the background %GC composition is lower than that of the foreground sequences, the TFs with low GC profiles are reported as under-represented (low Z-score) in the foreground sequences relative to the background sequences, while the TFs with high GC profiles are over-represented (high Z-score) in the foreground sequences <em>vs.</em> the background sequences. The opposite trend occurs when the background %GC composition is higher than the foreground sequences; in this case the TFs with low GC profiles are reported as over-represented. A background with the same GC composition distribution (mean and variance) as the foreground dataset removes bias arising from GC composition differences and controls against TFBSs being unduly assigned high or low over-representation scores.</p></div><div id="sec-33" class="subsection"><h3>Scoring measures for ChIP-Seq–based analyses</h3><p id="p-60">In combination, the Z-score and Fisher score inform the user whether enrichment is (1) due to a high frequency of TFBSs in some of the sequences and absence in others (Z-score high rank, Fisher score low rank; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003202/-/DC1/FigureS7.pdf">Figure S7</a>A, TF: Ap1); (2) due to a majority of regions carrying a TFBS but at low frequency per sequence (Fisher score high rank, Z-score low rank; or (3) due to both types of representation being elevated (Z-score and Fisher score both rank high). For sequence-based analyses on ChIP-Seq data, we find that ordering by either score presents similar ranks. To measure the agreement between the Z- and Fisher scores, we took the top 10 motifs ranked by each of the two types of scores and counted the fraction of TF motifs that were common to both lists. We based our choice of 10 top motifs on our lenient applied threshold (see <em>Materials and Methods</em>). Using the results from SSA, there was 100% agreement within the top 10 motifs for cMyc, and 82% agreement within both Sox2 and Foxa2. The Chip-Seq target TF ranked in the top two enriched motifs for both measures. For the most stringent results, users may use both Z- and Fisher over-representation scores in combination when selecting motifs for further analysis (<em>i.e.</em>, motifs scoring above both scores’ applied thresholds, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003202/-/DC1/FigureS7.pdf">Figure S7</a>).</p><p id="p-61">Unique to sequence-based SSA and TCA, KS scores are available, complementing Z- and Fisher scores. For the Nfe2L2 and FoxA2 ChIP-Seq datasets analyzed in this study, the target TF binding profiles receive high KS scores, as well as high Z- and Fisher scores (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Tables 4</a> and <a id="xref-table-wrap-5-4" class="xref-table" href="#T5">5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003202/-/DC1/FigureS8.pdf">Figure S8</a>). The KS scores obtained for these datasets were extreme; “infinity” values were returned for the target profiles. The accompanying false discovery rates support the utility of these scores.</p></div><div id="sec-34" class="subsection"><h3>Assessing impact of background dataset size on ChIP-Seq–based analyses</h3><p id="p-62">For some foreground sets, extreme composition properties can make it difficult to obtain composition-matched background sets of comparable size from the associated control dataset, let alone a background with 2-fold or more sequences. To determine the impact of background set size on results, we assessed the impact of background-to-foreground ratios on results. Datasets from ChIP-Seq analysis of the Sox2 and FoxA2 TFs were used for the assessment. We first assessed the variability between two different background sets of the same size, selected from the pool of control sequences to match the composition of the same associated foreground dataset. SSA analysis was performed for each. The Spearman correlation between the top TFs, for lists of length 5 through M (where M is derived from the lenient threshold; see <em>Materials and Methods</em>), was ≥0.97 for both Sox2 and FoxA2 Z-scores, ≥0.93 for FoxA2 Fisher scores, and ≥0.86 for Sox2 Fisher scores. Thus the selection of background sequences, which are matched to the foreground composition, has a subtle influence on the rankings of enriched TFs. We then ran SSA analyses using 0.5-, 1-, 2-, 3-, and 4-fold background-to-foreground ratios. For this comparison of different sized backgrounds, we again used lists of length 5 through M. The Spearman correlation between any two backgrounds, regardless of size, was ≥0.89 for both the Z- and Fisher scores of FoxA2, whereas the correlation value for Sox2 was ≥0.90 for either score. Thus we conclude that using background datasets larger than the foreground is not critical as it does not strongly influence the rankings of enriched motifs.</p></div></div><div class="section discussion" id="sec-35"><h2 class="">Discussion</h2><p id="p-63">The oPOSSUM-3 system enables researchers interested in the study of gene regulatory networks to identify TFs that may be acting in a biological context. Several key advances in the implementation of oPOSSUM-3 make the system suitable for analysis of emerging high-throughput data. Using phastCons multi-species conservation scores obviates the restrictive pairwise sequence alignment phylogenetic footprinting procedures of past releases, which greatly increases the number of genes represented within the system. This conversion to phastCons-based phylogenetic footprinting enables the construction of oPOSSUM services for any species represented in the Ensembl genome annotation database and the UCSC Genome Browser. The new anchored approach to combination site analysis (aCSA) allows for computationally tractable identification of interacting TFs. Motivated by the highly similar profiles for structurally related TFs emerging from large-scale studies, the clustering of highly similar binding site profiles allows a more focused report to users. Most importantly, the new sequence-based methods enable the oPOSSUM system to function as an analysis tool for large-scale ChIP-Seq sequence sets. Taken as a whole, oPOSSUM-3 is a powerful tool for biologists seeking insight into gene regulatory networks.</p><p id="p-64">One of the key considerations in motif enrichment analysis of mammalian regulatory sequences is how to filter the sequences to enrich for regulatory regions. The presented methods have two approaches to this problem. First, it is now widely recognized that using experimental data arising from chromatin immunoprecipitation (ChIP) studies can enrich for regulatory regions. The new oPOSSUM system features the capacity to process user-supplied DNA sequences. Although the cases presented in this report focus on TF-specific ChIP-Seq data, users may analyze any set of sequences they generate. Research that combines subsets of gene expression, epigenetic marks, and TF binding data could yield interesting sequence sets for analysis with oPOSSUM-3. For gene-based analysis, principally used by researchers working with gene expression data, the use of phylogenetic footprinting to restrict analysis remains the most common approach. The transition from pairwise alignment analysis to multiple sequence alignment analysis presents key benefits and raises challenges. Multiple sequence alignment approaches to phylogenetic footprinting can be performed for a dramatically higher portion of genes (a 66% increase observed in this study) and lead to improved alignment quality (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Kumar and Filipski 2007</a>). There remains a need to refine the multiple species footprinting approach using more advanced methods that incorporate motif conservation as well as sequence conservation. The recently published MotEvo method may address this challenge (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arnold <em>et al.</em> 2012</a>). In the near future, a broad and systematic benchmarking study of multiple species phylogenetic footprinting procedures for motif enrichment and <em>de novo</em> motif discovery would advance the field.</p><p id="p-65">Although running analyses on oPOSSUM-3 is relatively straightforward, interpretation of results requires some consideration, as expected for any over-representation analysis. First, as shown in the analysis of the effects of background GC-content, one should be aware of nucleotide composition differences between the foreground target set and the background set, as such differences can bias the TFBS profile enrichment scores. oPOSSUM-3 reports the GC-content of the foreground and background sets used in each analysis to empower users to make such assessments. A second consideration is the selection of the enrichment scoring method. While in some cases the profile of a mediating TF scores high on both Z-score and Fisher score rankings, empirical observations indicate that the two metrics can differ substantially. The differences reflect properties of the frequency of predicted TFBS. Fisher scores indicate the number of sequences or genes containing a predicted TFBS, whereas Z-scores reflect the frequency of the predicted TFBSs. The KS scores, which are new to oPOSSUM, measure the tendency for the predicted TF binding sites to cluster near a position of maximum confidence in the peak, or near the peak center if the user does not provide a position of confidence.</p><p id="p-66">There are opportunities to enhance oPOSSUM-3 in the future. The oPOSSUM system should be linked to a curated database of TFs, such as DBD or TFCat (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Fulton <em>et al.</em> 2009</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wilson <em>et al.</em> 2008</a>). By linking the systems, we can indicate which genes in the foreground target set are encoding TFs and therefore which genes may be candidates for participation in regulatory feedback loops. Based on similar considerations, the regulatory ncRNA genes should be reported. Finally, the system can be modified to incorporate automated methods to correct for biases in the gene or sequence foreground sets that are not represented in the default background set.</p><p id="p-67">Ultimately, we expect the oPOSSUM-3 system to be a convenient and useful tool for transcriptional regulation analysis, providing researchers with insight into the transcription factors acting on their sets of genes or sequences. The user-friendly interface provides researchers with access to a powerful bioinformatics tool.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-68">We gratefully recognize key contributors to past versions of oPOSSUM, as the discussions of the earlier implementations informed this work. In particular, we thank Shannan Ho Sui and Debra Fulton. We thank Miroslav Hatas for information technology support and Dora Pak for management support. The work was supported by funding from the National Institutes of Health (USA) grant 1R01GM084875, the Canadian Institutes for Health Research, the National Science and Engineering Research Council (NSERC), GenomeBC (CanEuCre Project), and Genome Canada (Pleiades Promoter Project). A.T.K. was supported by a NSERC fellowship, and R.W.H. was supported by fellowships from the Canadian Institutes of Health Research and the Michael Smith Foundation for Health Research.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003202/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003202/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">The software code for the oPOSSUM-3 system may be obtained directly from the oPOSSUM download page (<a href="http://opossum.cisreg.ca/oPOSSUM3/download.html">http://opossum.cisreg.ca/oPOSSUM3/download.html</a>) or can be downloaded from the GitHub repository (<a href="https://github.com/wassermanlab/oPOSSUM3">https://github.com/wassermanlab/oPOSSUM3</a>).</p></li><li class="fn" id="fn-8"><p id="p-69"><em>Communicating editor: T. R. Hughes</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-05-27"><span class="received-label">Received </span>May 27, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-06-11"><span class="accepted-label">Accepted </span>June 11, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Kwon <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.1" data-doi="10.1093/nar/gki354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aerts</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Loo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thijs</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayer</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Martin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">TOUCAN 2: the all-inclusive open source workbench for regulatory sequence analysis</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">W393</span>–<span class="cit-lpage">W396</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DAerts%26rft.auinit1%253DS.%26rft.volume%253D33%26rft.issue%253Dsuppl_2%26rft.spage%253DW393%26rft.epage%253DW396%26rft.atitle%253DTOUCAN%2B2%253A%2Bthe%2Ball-inclusive%2Bopen%2Bsource%2Bworkbench%2Bfor%2Bregulatory%2Bsequence%2Banalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki354%26rft_id%253Dinfo%253Apmid%252F15980497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzMy9zdXBwbF8yL1czOTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzkvOTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.2" data-doi="10.1093/bioinformatics/btr695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erb</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pachkov</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molina</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Nimwegen</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">MotEvo: integrated Bayesian probabilistic methods for inferring regulatory sites and motifs on multiple alignments of DNA sequences</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">487</span>–<span class="cit-lpage">494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr695%26rft_id%253Dinfo%253Apmid%252F22334039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyOC80LzQ4NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOS85ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arnone</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">E. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The hardwiring of development: organization and function of genomic regulatory systems</span>. <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">1851</span>–<span class="cit-lpage">1864</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DArnone%26rft.auinit1%253DM.%26rft.volume%253D124%26rft.issue%253D10%26rft.spage%253D1851%26rft.epage%253D1864%26rft.atitle%253DThe%2Bhardwiring%2Bof%2Bdevelopment%253A%2Borganization%2Band%2Bfunction%2Bof%2Bgenomic%2Bregulatory%2Bsystems%26rft_id%253Dinfo%253Apmid%252F9169833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTI0LzEwLzE4NTEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzkvOTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Machanick</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Inferring direct DNA binding from ChIP-seq</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> DOI: 10.1093/nar/gks433.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-other" id="cit-2.9.987.5"><div class="cit-metadata unstructured">Benjamini, Y., and Y. Hochberg, 1995 Controlling the false discovery rate: a practical and powerful approach to multiple testing. J. R. Stat. Soc. B 57: 289–300.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blumenthal</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gleason</span>  <span class="cit-name-given-names">K. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Caenorhabditis elegans operons: form and function</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">112</span>–<span class="cit-lpage">120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBlumenthal%26rft.auinit1%253DT.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253D112%26rft.epage%253D120%26rft.atitle%253DCaenorhabditis%2Belegans%2Boperons%253A%2Bform%2Band%2Bfunction.%26rft_id%253Dinfo%253Apmid%252F12560808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12560808&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F987.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Braun</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bober</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rudnicki</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaenisch</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">H. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">MyoD expression marks the onset of skeletal myogenesis in Myf-5 mutant mice</span>. <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">3083</span>–<span class="cit-lpage">3092</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DBraun%26rft.auinit1%253DT%26rft.volume%253D120%26rft.issue%253D11%26rft.spage%253D3083%26rft.epage%253D3092%26rft.atitle%253DMyoD%2Bexpression%2Bmarks%2Bthe%2Bonset%2Bof%2Bskeletal%2Bmyogenesis%2Bin%2BMyf-5%2Bmutant%2Bmice%26rft_id%253Dinfo%253Apmid%252F7720554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTIwLzExLzMwODMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzkvOTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.8" data-doi="10.1093/nar/gkm308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">L.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fontaine</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stormo</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagarajan</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PAP: a comprehensive workbench for mammalian transcriptional regulatory sequence analysis</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">W238</span>–<span class="cit-lpage">W244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm308%26rft_id%253Dinfo%253Apmid%252F17517777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNS9zdXBwbF8yL1cyMzgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzkvOTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.9" data-doi="10.1016/j.cell.2008.04.043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huss</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Integration of external signaling pathways with the core transcriptional network in embryonic stem cells</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">1106</span>–<span class="cit-lpage">1117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DChen%26rft.auinit1%253DX.%26rft.volume%253D133%26rft.issue%253D6%26rft.spage%253D1106%26rft.epage%253D1117%26rft.atitle%253DIntegration%2Bof%2Bexternal%2Bsignaling%2Bpathways%2Bwith%2Bthe%2Bcore%2Btranscriptional%2Bnetwork%2Bin%2Bembryonic%2Bstem%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.04.043%26rft_id%253Dinfo%253Apmid%252F18555785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.04.043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18555785&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F987.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256693400024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.10" data-doi="10.1186/1471-2105-7-396"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Defrance</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Touzet</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Predicting transcription factor binding sites using local over-representation and comparative genomics</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">396</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DDefrance%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.spage%253D396%26rft.epage%253D396%26rft.atitle%253DPredicting%2Btranscription%2Bfactor%2Bbinding%2Bsites%2Busing%2Blocal%2Bover-representation%2Band%2Bcomparative%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-7-396%26rft_id%253Dinfo%253Apmid%252F16945132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-7-396&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16945132&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F987.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.11" data-doi="10.1093/emboj/16.18.5687"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durocher</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charron</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warren</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nemer</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The cardiac transcription factors Nkx2–5 and GATA-4 are mutual cofactors</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">5687</span>–<span class="cit-lpage">5696</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DDurocher%26rft.auinit1%253DD.%26rft.volume%253D16%26rft.issue%253D18%26rft.spage%253D5687%26rft.epage%253D5696%26rft.atitle%253DThe%2Bcardiac%2Btranscription%2Bfactors%2BNkx2-5%2Band%2BGATA-4%2Bare%2Bmutual%2Bcofactors%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F16.18.5687%26rft_id%253Dinfo%253Apmid%252F9312027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMTYvMTgvNTY4NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOS85ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.12" data-doi="10.1093/nar/30.1.207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Domrachev</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lash</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Gene Expression Omnibus: NCBI gene expression and hybridization array data repository</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%26rft.volume%253D30%26rft.issue%253D1%26rft.spage%253D207%26rft.epage%253D210%26rft.atitle%253DGene%2BExpression%2BOmnibus%253A%2BNCBI%2Bgene%2Bexpression%2Band%2Bhybridization%2Barray%2Bdata%2Brepository%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F30.1.207%26rft_id%253Dinfo%253Apmid%252F11752295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjMwLzEvMjA3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi85Lzk4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.13" data-doi="10.1242/dev.01775"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Efimenko</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bubb</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mak</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holzman</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leroux</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Analysis of xbx genes in C. elegans</span>. <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">1923</span>–<span class="cit-lpage">1934</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DEfimenko%26rft.auinit1%253DE.%26rft.volume%253D132%26rft.issue%253D8%26rft.spage%253D1923%26rft.epage%253D1934%26rft.atitle%253DAnalysis%2Bof%2Bxbx%2Bgenes%2Bin%2BC.%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.01775%26rft_id%253Dinfo%253Apmid%252F15790967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMDoiMTMyLzgvMTkyMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOS85ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.14" data-doi="10.1128/MCB.16.8.4486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emery</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strubin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hofmann</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bucher</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mach</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">A consensus motif in the RFX DNA binding domain and binding domain mutants with altered specificity</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">4486</span>–<span class="cit-lpage">4494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DEmery%26rft.auinit1%253DP.%26rft.volume%253D16%26rft.issue%253D8%26rft.spage%253D4486%26rft.epage%253D4494%26rft.atitle%253DA%2Bconsensus%2Bmotif%2Bin%2Bthe%2BRFX%2BDNA%2Bbinding%2Bdomain%2Band%2Bbinding%2Bdomain%2Bmutants%2Bwith%2Baltered%2Bspecificity%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.16.8.4486%26rft_id%253Dinfo%253Apmid%252F8754849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE2LzgvNDQ4NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOS85ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.15" data-doi="10.1093/bioinformatics/btn305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fejes</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bilenky</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varhol</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bainbridge</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">FindPeaks 3.1: a tool for identifying areas of enrichment from massively parallel short-read sequencing technology</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1729</span>–<span class="cit-lpage">1730</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn305%26rft_id%253Dinfo%253Apmid%252F18599518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjQvMTUvMTcyOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOS85ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flicek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amode</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrell</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beal</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brent</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Ensembl 2011</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">D800</span>–<span class="cit-lpage">D806</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D39%26rft.spage%253DD800%26rft_id%253Dinfo%253Apmid%252F21045057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21045057&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F987.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285831700126&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.17" data-doi="10.1073/pnas.89.2.668"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friling</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergelson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daniel</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Two adjacent AP-1-like binding sites form the electrophile-responsive element of the murine glutathione S-transferase Ya subunit gene</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">668</span>–<span class="cit-lpage">672</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DFriling%26rft.auinit1%253DR.%2BS.%26rft.volume%253D89%26rft.issue%253D2%26rft.spage%253D668%26rft.epage%253D672%26rft.atitle%253DTwo%2Badjacent%2BAP-1-like%2Bbinding%2Bsites%2Bform%2Bthe%2Belectrophile-responsive%2Belement%2Bof%2Bthe%2Bmurine%2Bglutathione%2BS-transferase%2BYa%2Bsubunit%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.89.2.668%26rft_id%253Dinfo%253Apmid%252F1731339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI4OS8yLzY2OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOS85ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.18" data-doi="10.1186/gb-2009-10-3-r29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sundararajan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Badis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wasserman</span>  <span class="cit-name-given-names">W. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">TFCat: the curated catalog of mouse and human transcription factors</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DFulton%26rft.auinit1%253DD.%2BL.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR29%26rft.epage%253DR29%26rft.atitle%253DTFCat%253A%2Bthe%2Bcurated%2Bcatalog%2Bof%2Bmouse%2Band%2Bhuman%2Btranscription%2Bfactors.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r29%26rft_id%253Dinfo%253Apmid%252F19284633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r29&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19284633&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F987.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.19" data-doi="10.1186/gb-2010-11-8-r86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goecks</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nekrutenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Galaxy: a comprehensive approach for supporting accessible, reproducible, and transparent computational research in the life sciences</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DGoecks%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D8%26rft.spage%253DR86%26rft.epage%253DR86%26rft.atitle%253DGalaxy%253A%2Ba%2Bcomprehensive%2Bapproach%2Bfor%2Bsupporting%2Baccessible%252C%2Breproducible%252C%2Band%2Btransparent%2Bcomputational%2Bresearch%2Bin%2Bthe%2Blife%2Bsciences.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-8-r86%26rft_id%253Dinfo%253Apmid%252F20738864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-8-r86&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20738864&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F987.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.20" data-doi="10.1093/nar/gkp952"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antoshechkin</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bieri</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blasiar</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">WormBase: a comprehensive resource for nematode research</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">D463</span>–<span class="cit-lpage">D467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp952%26rft_id%253Dinfo%253Apmid%252F19910365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzOC9zdXBwbF8xL0Q0NjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzkvOTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.21" data-doi="10.1093/nar/gki624"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ho Sui</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mortimer</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arenillas</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brumm</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">oPOSSUM: identification of over-represented transcription factor binding sites in co-expressed genes</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">3154</span>–<span class="cit-lpage">3164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DHo%2BSui%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D33%26rft.issue%253D10%26rft.spage%253D3154%26rft.epage%253D3164%26rft.atitle%253DoPOSSUM%253A%2Bidentification%2Bof%2Bover-represented%2Btranscription%2Bfactor%2Bbinding%2Bsites%2Bin%2Bco-expressed%2Bgenes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki624%26rft_id%253Dinfo%253Apmid%252F15933209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMy8xMC8zMTU0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi85Lzk4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.22" data-doi="10.1093/nar/gkm427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ho Sui</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arenillas</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwon</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wasserman</span>  <span class="cit-name-given-names">W. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">oPOSSUM: integrated tools for analysis of regulatory motif over-representation</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">W245</span>–<span class="cit-lpage">W252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm427%26rft_id%253Dinfo%253Apmid%252F17576675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNS9zdXBwbF8yL1cyNDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzkvOTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.23" data-doi="10.1093/bib/bbq072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pollard</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siepel</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">PHAST and RPHAST: phylogenetic analysis with space/time models</span>. <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbq072%26rft_id%253Dinfo%253Apmid%252F21278375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYmliIjtzOjU6InJlc2lkIjtzOjc6IjEyLzEvNDEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzkvOTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.24" data-doi="10.1016/j.tig.2006.07.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Inglis</span>  <span class="cit-name-given-names">P. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boroevich</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leroux</span>  <span class="cit-name-given-names">M. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Piecing together a ciliome</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DInglis%26rft.auinit1%253DP.%2BN.%26rft.volume%253D22%26rft.issue%253D9%26rft.spage%253D491%26rft.epage%253D500%26rft.atitle%253DPiecing%2Btogether%2Ba%2Bciliome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2006.07.006%26rft_id%253Dinfo%253Apmid%252F16860433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2006.07.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16860433&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F987.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240722800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.25" data-doi="10.1126/science.1141319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mortazavi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wold</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide mapping of in vivo protein-DNA interactions</span>. <abbr class="cit-jnl-abbrev">Science (New York, N.Y.)</abbr> <span class="cit-vol">316</span>: <span class="cit-fpage">1497</span>–<span class="cit-lpage">1502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2B%2528New%2BYork%252C%2BN.Y.%2529%26rft.volume%253D316%26rft.spage%253D1497%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1141319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1126/science.1141319&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.26" data-doi="10.1101/gr.5232407"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filipski</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Multiple sequence alignment: in pursuit of homologous DNA positions</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DKumar%26rft.auinit1%253DS.%26rft.volume%253D17%26rft.issue%253D2%26rft.spage%253D127%26rft.epage%253D135%26rft.atitle%253DMultiple%2Bsequence%2Balignment%253A%2BIn%2Bpursuit%2Bof%2Bhomologous%2BDNA%2Bpositions%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5232407%26rft_id%253Dinfo%253Apmid%252F17272647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzIvMTI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi85Lzk4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.27" data-doi="10.1093/nar/gkq212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malhotra</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Portales-Casamar</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arenillas</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Global mapping of binding sites for Nrf2 identifies novel targets in cell survival response through ChIP-Seq profiling and network analysis</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">5718</span>–<span class="cit-lpage">5734</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq212%26rft_id%253Dinfo%253Apmid%252F20460467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzOC8xNy81NzE4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi85Lzk4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.28" data-doi="10.1186/1471-2105-11-267"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mosig</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vingron</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A computational evaluation of over-representation of regulatory motifs in the promoter regions of differentially expressed genes</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DMeng%26rft.auinit1%253DG.%26rft.volume%253D11%26rft.spage%253D267%26rft.epage%253D267%26rft.atitle%253DA%2Bcomputational%2Bevaluation%2Bof%2Bover-representation%2Bof%2Bregulatory%2Bmotifs%2Bin%2Bthe%2Bpromoter%2Bregions%2Bof%2Bdifferentially%2Bexpressed%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-267%26rft_id%253Dinfo%253Apmid%252F20487530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-267&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20487530&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F987.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.29" data-doi="10.1016/S0955-0674(99)00036-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">MEF2: a transcriptional target for signaling pathways controlling skeletal muscle growth and differentiation</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Cell Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">683</span>–<span class="cit-lpage">688</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bcell%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BCell%2BBiol%26rft.aulast%253DNaya%26rft.auinit1%253DF.%2BJ.%26rft.volume%253D11%26rft.issue%253D6%26rft.spage%253D683%26rft.epage%253D688%26rft.atitle%253DMEF2%253A%2Ba%2Btranscriptional%2Btarget%2Bfor%2Bsignaling%2Bpathways%2Bcontrolling%2Bskeletal%2Bmuscle%2Bgrowth%2Band%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0955-0674%252899%252900036-8%26rft_id%253Dinfo%253Apmid%252F10600704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0955-0674(99)00036-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10600704&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F987.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084010000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.30" data-doi="10.1371/journal.pone.0012465"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piechota</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korostynski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przewlocki</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Identification of cis-regulatory elements in the mammalian genome: the cREMaG database</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e12465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPiechota%26rft.auinit1%253DM.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253De12465%26rft.epage%253De12465%26rft.atitle%253DIdentification%2Bof%2Bcis-regulatory%2Belements%2Bin%2Bthe%2Bmammalian%2Bgenome%253A%2Bthe%2BcREMaG%2Bdatabase.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0012465%26rft_id%253Dinfo%253Apmid%252F20824209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0012465&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20824209&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F987.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.31" data-doi="10.1093/nar/gkp950"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Portales-Casamar</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thongjuea</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwon</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arenillas</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">JASPAR 2010: the greatly expanded open-access database of transcription factor binding profiles</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">D105</span>–<span class="cit-lpage">D110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp950%26rft_id%253Dinfo%253Apmid%252F19906716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzOC9zdXBwbF8xL0QxMDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzkvOTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-other" id="cit-2.9.987.32"><div class="cit-metadata unstructured">R Development Core Team, 2008 <em>R: A Language and Environment for Statistical Computing</em>. R Foundation for Statistical Computing, Vienna, Austria.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.33" data-doi="10.1101/gr.078519.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bilenky</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tam</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome-wide relationship between histone H3 lysine 4 mono- and tri-methylation and transcription factor binding</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1906</span>–<span class="cit-lpage">1917</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.078519.108%26rft_id%253Dinfo%253Apmid%252F18787082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMi8xOTA2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi85Lzk4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.34" data-doi="10.1093/nar/gkp682"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roider</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenhard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanhere</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vingron</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">CpG-depleted promoters harbor tissue-specific transcription factor binding signals–implications for motif overrepresentation analyses</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">6305</span>–<span class="cit-lpage">6315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp682%26rft_id%253Dinfo%253Apmid%252F19736212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzNy8xOS82MzA1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi85Lzk4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.35" data-doi="10.1002/bies.950170306"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rudnicki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaenisch</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">The MyoD family of transcription factors and skeletal myogenesis</span>. <abbr class="cit-jnl-abbrev">Bioessays</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DRudnicki%26rft.auinit1%253DM.%2BA.%26rft.volume%253D17%26rft.issue%253D3%26rft.spage%253D203%26rft.epage%253D209%26rft.atitle%253DThe%2BMyoD%2Bfamily%2Bof%2Btranscription%2Bfactors%2Band%2Bskeletal%2Bmyogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.950170306%26rft_id%253Dinfo%253Apmid%252F7748174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.950170306&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7748174&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F987.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QQ31100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.36" data-doi="10.1007/s10142-003-0086-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sandelin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Höglund</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenhard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wasserman</span>  <span class="cit-name-given-names">W. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Integrated analysis of yeast regulatory sequences for biologically linked clusters of genes</span>. <abbr class="cit-jnl-abbrev">Funct. Integr. Genomics</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunctional%2B%2526%2Bintegrative%2Bgenomics%26rft.stitle%253DFunct%2BIntegr%2BGenomics%26rft.aulast%253DSandelin%26rft.auinit1%253DA.%26rft.volume%253D3%26rft.issue%253D3%26rft.spage%253D125%26rft.epage%253D134%26rft.atitle%253DIntegrated%2Banalysis%2Bof%2Byeast%2Bregulatory%2Bsequences%2Bfor%2Bbiologically%2Blinked%2Bclusters%2Bof%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10142-003-0086-6%26rft_id%253Dinfo%253Apmid%252F12827523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10142-003-0086-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12827523&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F987.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-2.9.987.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ballester</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwalie</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source">Five-Vertebrate ChIP-seq Reveals the Evolutionary Dynamics of Transcription Factor Binding.</span> <span class="cit-publ-name">Science</span>, <span class="cit-publ-loc">New York, N.Y</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.38" data-doi="10.1093/nar/28.4.1000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Helden</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Del Olmo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Ortín</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Statistical analysis of yeast genomic downstream sequences reveals putative polyadenylation signals</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1000</span>–<span class="cit-lpage">1010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DHelden%26rft.auinit1%253DJ.%2Bv.%26rft.volume%253D28%26rft.issue%253D4%26rft.spage%253D1000%26rft.epage%253D1010%26rft.atitle%253DStatistical%2Banalysis%2Bof%2Byeast%2Bgenomic%2Bdownstream%2Bsequences%2Breveals%2Bputative%2Bpolyadenylation%2Bsignals%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.4.1000%26rft_id%253Dinfo%253Apmid%252F10648794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjI4LzQvMTAwMCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOS85ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.39" data-doi="10.1006/jmbi.1998.1700"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wasserman</span>  <span class="cit-name-given-names">W. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fickett</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Identification of regulatory regions which confer muscle-specific gene expression</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">278</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DWasserman%26rft.auinit1%253DW.%2BW.%26rft.volume%253D278%26rft.issue%253D1%26rft.spage%253D167%26rft.epage%253D181%26rft.atitle%253DIdentification%2Bof%2Bregulatory%2Bregions%2Bwhich%2Bconfer%2Bmuscle-specific%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1998.1700%26rft_id%253Dinfo%253Apmid%252F9571041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1998.1700&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9571041&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F987.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073500600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.40" data-doi="10.1093/nar/gkm964"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charoensawan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kummerfeld</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teichmann</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">DBD–taxonomically broad transcription factor predictions: new content and functionality</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">D88</span>–<span class="cit-lpage">D92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm964%26rft_id%253Dinfo%253Apmid%252F18073188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE0OiIzNi9zdXBwbF8xL0Q4OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOS85ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.41" data-doi="10.1016/j.gene.2005.01.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamashita</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugano</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakai</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genome-wide analysis reveals strong correlation between CpG islands with nearby transcription start sites of genes and their tissue specificity</span>. <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">350</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D350%26rft.spage%253D129%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2005.01.012%26rft_id%253Dinfo%253Apmid%252F15784181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2005.01.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15784181&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F987.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229559800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.987.42" data-doi="10.1074/jbc.M110.112482"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoshida</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gan</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franke</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Smooth and cardiac muscle-selective knock-out of Kruppel-like factor 4 causes postnatal death and growth retardation</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">285</span>: <span class="cit-fpage">21175</span>–<span class="cit-lpage">21184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M110.112482%26rft_id%253Dinfo%253Apmid%252F20439457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvMjcvMjExNzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzkvOTg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/9/987.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/9/977" title="Evolutionary History and Genome Organization of DUF1220 Protein Domains" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/9/1003" title="Glucose, Nitrogen, and Phosphate Repletion in Saccharomyces cerevisiae: Common Transcriptional Responses to Different Nutrient Signals" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/9"><span>Volume 2 Issue 9, September 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/9.cover-source.jpg?itok=JMb6jwWa" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/9/987" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MHbbcsZZjAx2dr0BFM-eWtIqaDtxhCT3BCF3dyzNe0s" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/9/987" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/9/987" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/9/987" class="active" data-icon-position="" data-hide-link-title="0">oPOSSUM-3: Advanced Analysis of Regulatory Motif Over-Representation Across Genes or ChIP-Seq Datasets</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459330" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-s04KoofjQkgJLzZPb1y8zlQvKNFQIqaVnIa_MBv2JH8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/9/987.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/9/987" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-xMEH4pr3Oc7IOJinld-YwO47MeOzwK-9raokcmUp7NQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/9/987" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/9/987.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459330" id="node459330--2" data-pisa="ggg;2/9/987" data-pisa-master="ggg;2/9/987" data-apath="/ggg/2/9/987.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">oPOSSUM-3: Advanced Analysis of Regulatory Motif Over-Representation Across Genes or ChIP-Seq Datasets</h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Andrew T. Kwon</span>, <span class="highwire-citation-author" data-delta="1">David J. Arenillas</span>, <span class="highwire-citation-author" data-delta="2">Rebecca Worsley Hunt</span> and <span class="highwire-citation-author" data-delta="3">Wyeth W. Wasserman</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">987-1002;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003202 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459330--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrew T. Kwon</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, University of British Columbia, Vancouver, British Columbia, V5Z 4H4, Canada</span><span class="nlm-aff">Department of Medical Genetics, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</span><span class="nlm-aff">Genetics Graduate Program, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BT.%2BKwon" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kwon%20AT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BT.%2BKwon" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">David J. Arenillas</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, University of British Columbia, Vancouver, British Columbia, V5Z 4H4, Canada</span><span class="nlm-aff">Department of Medical Genetics, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BJ.%2BArenillas" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arenillas%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BJ.%2BArenillas" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rebecca Worsley Hunt</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, University of British Columbia, Vancouver, British Columbia, V5Z 4H4, Canada</span><span class="nlm-aff">Department of Medical Genetics, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</span><span class="nlm-aff">Bioinformatics Graduate Program, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rebecca%2BWorsley%2BHunt" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hunt%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARebecca%2BWorsley%2BHunt" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Wyeth W. Wasserman</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, University of British Columbia, Vancouver, British Columbia, V5Z 4H4, Canada</span><span class="nlm-aff">Department of Medical Genetics, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wyeth%2BW.%2BWasserman" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wasserman%20WW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWyeth%2BW.%2BWasserman" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wyeth@cmmt.ubc.ca" class="" data-icon-position="" data-hide-link-title="0">wyeth@cmmt.ubc.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/9/987" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/9/987&amp;title=oPOSSUM-3%3A%20Advanced%20Analysis%20of%20Regulatory%20Motif%20Over-Representation%20Across%20Genes%20or%20ChIP-Seq%20Datasets" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/9/987&amp;title=oPOSSUM-3%3A%20Advanced%20Analysis%20of%20Regulatory%20Motif%20Over-Representation%20Across%20Genes%20or%20ChIP-Seq%20Datasets" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/9/987&amp;title=oPOSSUM-3%3A%20Advanced%20Analysis%20of%20Regulatory%20Motif%20Over-Representation%20Across%20Genes%20or%20ChIP-Seq%20Datasets" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/9/987&amp;text=oPOSSUM-3%3A%20Advanced%20Analysis%20of%20Regulatory%20Motif%20Over-Representation%20Across%20Genes%20or%20ChIP-Seq%20Datasets" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/9/987&amp;title=oPOSSUM-3%3A%20Advanced%20Analysis%20of%20Regulatory%20Motif%20Over-Representation%20Across%20Genes%20or%20ChIP-Seq%20Datasets" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/9/987&amp;t=oPOSSUM-3%3A%20Advanced%20Analysis%20of%20Regulatory%20Motif%20Over-Representation%20Across%20Genes%20or%20ChIP-Seq%20Datasets" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/9/987&amp;title=oPOSSUM-3%3A%20Advanced%20Analysis%20of%20Regulatory%20Motif%20Over-Representation%20Across%20Genes%20or%20ChIP-Seq%20Datasets" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/9/987&amp;title=oPOSSUM-3%3A%20Advanced%20Analysis%20of%20Regulatory%20Motif%20Over-Representation%20Across%20Genes%20or%20ChIP-Seq%20Datasets" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459330/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459330" id="node459330--2" data-pisa="ggg;2/9/987" data-pisa-master="ggg;2/9/987" data-apath="/ggg/2/9/987.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">oPOSSUM-3: Advanced Analysis of Regulatory Motif Over-Representation Across Genes or ChIP-Seq Datasets</h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Andrew T. Kwon</span>, <span class="highwire-citation-author" data-delta="1">David J. Arenillas</span>, <span class="highwire-citation-author" data-delta="2">Rebecca Worsley Hunt</span> and <span class="highwire-citation-author" data-delta="3">Wyeth W. Wasserman</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">987-1002;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003202 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459330--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrew T. Kwon</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, University of British Columbia, Vancouver, British Columbia, V5Z 4H4, Canada</span><span class="nlm-aff">Department of Medical Genetics, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</span><span class="nlm-aff">Genetics Graduate Program, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BT.%2BKwon" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kwon%20AT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BT.%2BKwon" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">David J. Arenillas</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, University of British Columbia, Vancouver, British Columbia, V5Z 4H4, Canada</span><span class="nlm-aff">Department of Medical Genetics, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BJ.%2BArenillas" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arenillas%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BJ.%2BArenillas" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rebecca Worsley Hunt</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, University of British Columbia, Vancouver, British Columbia, V5Z 4H4, Canada</span><span class="nlm-aff">Department of Medical Genetics, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</span><span class="nlm-aff">Bioinformatics Graduate Program, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rebecca%2BWorsley%2BHunt" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hunt%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARebecca%2BWorsley%2BHunt" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Wyeth W. Wasserman</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, University of British Columbia, Vancouver, British Columbia, V5Z 4H4, Canada</span><span class="nlm-aff">Department of Medical Genetics, University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wyeth%2BW.%2BWasserman" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wasserman%20WW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWyeth%2BW.%2BWasserman" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wyeth@cmmt.ubc.ca" class="" data-icon-position="" data-hide-link-title="0">wyeth@cmmt.ubc.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=987&amp;rft.epage=1002&amp;rft.atitle=oPOSSUM-3%3A%20Advanced%20Analysis%20of%20Regulatory%20Motif%20Over-Representation%20Across%20Genes%20or%20ChIP-Seq%20Datasets&amp;rft.volume=2&amp;rft.issue=9&amp;rft.date=2012-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kwon%2C+Andrew+T.&amp;rft.au=Arenillas%2C+David+J.&amp;rft.au=Hunt%2C+Rebecca+Worsley&amp;rft.au=Wasserman%2C+Wyeth+W."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459330/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459330/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459330/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459330/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459330/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459330/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459330/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459330/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459330/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459330/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459330/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459330/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494625" id="node494625" data-pisa="ggg;9/11/3891" data-pisa-master="ggg;g3.119.400723" data-apath="/ggg/9/11/3891.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3891" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Mouse Mutation That Dysregulates Neighboring <em>Galnt17</em> and <em>Auts2</em> Genes Is Associated with Phenotypes Related to the Human AUTS2 Syndrome</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494624" id="node494624" data-pisa="ggg;9/11/3781" data-pisa-master="ggg;g3.119.400647" data-apath="/ggg/9/11/3781.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3781" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Transgenic Analyses in Drosophila Reveal That mCORL1 Is Functionally Distinct from mCORL2 and dCORL</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494622" id="node494622" data-pisa="ggg;9/11/3843" data-pisa-master="ggg;g3.119.400363" data-apath="/ggg/9/11/3843.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3843" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Analysis of Cell Cycle-Regulating Genes in the Symbiotic Dinoflagellate <em>Breviolum minutum</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/9/987" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">System Walk-Through</a></li><li class="even"><a href="#sec-27" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="odd"><a href="#sec-35" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/9/987.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/9/987.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/9/987.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/9/987.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2019 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
  </body>
</html>
