<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/8/891" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/8/891.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/8/891.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/8/891.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/8/891.atom" />
<meta name="HW.pisa" content="ggg;2/8/891" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Incidence of Genome Structure, DNA Asymmetry, and Cell Physiology on T-DNA Integration in Chromosomes of the Phytopathogenic Fungus Leptosphaeria maculans" />
<meta name="DC.Identifier" content="10.1534/g3.112.002048" />
<meta name="DC.Date" content="2012-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Bourras et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The ever-increasing generation of sequence data is accompanied by unsatisfactory functional annotation, and complex genomes, such as those of plants and filamentous fungi, show a large number of genes with no predicted or known function. For functional annotation of unknown or hypothetical genes, the production of collections of mutants using Agrobacterium tumefaciens –mediated transformation (ATMT) associated with genotyping and phenotyping has gained wide acceptance. ATMT is also widely used to identify pathogenicity determinants in pathogenic fungi. A systematic analysis of T-DNA borders was performed in an ATMT-mutagenized collection of the phytopathogenic fungus Leptosphaeria maculans to evaluate the features of T-DNA integration in its particular transposable element-rich compartmentalized genome. A total of 318 T-DNA tags were recovered and analyzed for biases in chromosome and genic compartments, existence of CG/AT skews at the insertion site, and occurrence of microhomologies between the T-DNA left border (LB) and the target sequence. Functional annotation of targeted genes was done using the Gene Ontology annotation. The T-DNA integration mainly targeted gene-rich, transcriptionally active regions, and it favored biological processes consistent with the physiological status of a germinating spore. T-DNA integration was strongly biased toward regulatory regions, and mainly promoters. Consistent with the T-DNA intranuclear-targeting model, the density of T-DNA insertion correlated with CG skew near the transcription initiation site. The existence of microhomologies between promoter sequences and the T-DNA LB flanking sequence was also consistent with T-DNA integration to host DNA mediated by homologous recombination based on the microhomology-mediated end-joining pathway." />
<meta name="DC.Contributor" content="Salim Bourras" />
<meta name="DC.Contributor" content="Michel Meyer" />
<meta name="DC.Contributor" content="Jonathan Grandaubert" />
<meta name="DC.Contributor" content="Nicolas Lapalu" />
<meta name="DC.Contributor" content="Isabelle Fudal" />
<meta name="DC.Contributor" content="Juliette Linglin" />
<meta name="DC.Contributor" content="Benedicte Ollivier" />
<meta name="DC.Contributor" content="Françoise Blaise" />
<meta name="DC.Contributor" content="Marie-Hélène Balesdent" />
<meta name="DC.Contributor" content="Thierry Rouxel" />
<meta name="article:published_time" content="2012-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Incidence of Genome Structure, DNA Asymmetry, and Cell Physiology on T-DNA Integration in Chromosomes of the Phytopathogenic Fungus Leptosphaeria maculans" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The ever-increasing generation of sequence data is accompanied by unsatisfactory functional annotation, and complex genomes, such as those of plants and filamentous fungi, show a large number of genes with no predicted or known function. For functional annotation of unknown or hypothetical genes, the production of collections of mutants using &lt;i&gt;Agrobacterium tumefaciens&lt;/i&gt;–mediated transformation (ATMT) associated with genotyping and phenotyping has gained wide acceptance. ATMT is also widely used to identify pathogenicity determinants in pathogenic fungi. A systematic analysis of T-DNA borders was performed in an ATMT-mutagenized collection of the phytopathogenic fungus &lt;i&gt;Leptosphaeria maculans&lt;/i&gt; to evaluate the features of T-DNA integration in its particular transposable element-rich compartmentalized genome. A total of 318 T-DNA tags were recovered and analyzed for biases in chromosome and genic compartments, existence of CG/AT skews at the insertion site, and occurrence of microhomologies between the T-DNA left border (LB) and the target sequence. Functional annotation of targeted genes was done using the Gene Ontology annotation. The T-DNA integration mainly targeted gene-rich, transcriptionally active regions, and it favored biological processes consistent with the physiological status of a germinating spore. T-DNA integration was strongly biased toward regulatory regions, and mainly promoters. Consistent with the T-DNA intranuclear-targeting model, the density of T-DNA insertion correlated with CG skew near the transcription initiation site. The existence of microhomologies between promoter sequences and the T-DNA LB flanking sequence was also consistent with T-DNA integration to host DNA mediated by homologous recombination based on the microhomology-mediated end-joining pathway.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/08/01" />
<meta name="citation_mjid" content="ggg;2/8/891" />
<meta name="citation_id" content="2/8/891" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/8/891" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/8/891.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/8/891.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/8/891.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.002048" />
<meta name="citation_pmid" content="22908038" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="891" />
<meta name="citation_lastpage" content="904" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Salim Bourras" />
<meta name="citation_author_institution" content="Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and" />
<meta name="citation_author" content="Michel Meyer" />
<meta name="citation_author_institution" content="Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and" />
<meta name="citation_author" content="Jonathan Grandaubert" />
<meta name="citation_author_institution" content="Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and" />
<meta name="citation_author" content="Nicolas Lapalu" />
<meta name="citation_author_institution" content="INRA, Research Unit 1290 BIOGER, F-78026 Versailles Cedex, France" />
<meta name="citation_author" content="Isabelle Fudal" />
<meta name="citation_author_institution" content="Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and" />
<meta name="citation_author" content="Juliette Linglin" />
<meta name="citation_author_institution" content="Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and" />
<meta name="citation_author" content="Benedicte Ollivier" />
<meta name="citation_author_institution" content="Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and" />
<meta name="citation_author" content="Françoise Blaise" />
<meta name="citation_author_institution" content="Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and" />
<meta name="citation_author" content="Marie-Hélène Balesdent" />
<meta name="citation_author_institution" content="Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and" />
<meta name="citation_author" content="Thierry Rouxel" />
<meta name="citation_author_institution" content="Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and" />
<meta name="citation_author_email" content="rouxel@versailles.inra.fr" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=N. N. Alexandrov;citation_author=M. E. Troukhan;citation_author=V. V. Brover;citation_author=T. Tatarinova;citation_author=R. B. Flavell;citation_title=Features of Arabidopsis genes and genome discovered using full-length cDNAs.;citation_pages=69-85;citation_volume=60;citation_year=2006;citation_issue=1;citation_pmid=16463100;citation_doi=10.1007/s11103-005-2564-9" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. M. Alonso;citation_author=A. N. Stepanova;citation_author=T. J. Leisse;citation_author=C. J. Kim;citation_author=H. Chen;citation_title=Genome-Wide Insertional Mutagenesis of Arabidopsis thaliana;citation_pages=653-657;citation_volume=301;citation_year=2003;citation_issue=5633;citation_pmid=12893945;citation_doi=10.1126/science.1086391" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Arabidopsis Genome Initiative;citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana.;citation_pages=796-815;citation_volume=408;citation_year=2000;citation_issue=6814;citation_pmid=11130711;citation_doi=10.1038/35048692" />
<meta name="citation_reference" content="citation_journal_title=Molecular plant-microbe interactions :  MPMI;citation_journal_abbrev=Mol Plant Microbe Interact;citation_author=A. Attard;citation_author=L. Gout;citation_author=M. Gourgues;citation_author=M. L. Kuhn;citation_author=J. Schmit;citation_title=Analysis of molecular markers genetically linked to the Leptosphaeria maculans avirulence gene AvrLm1 in field populations indicates a highly conserved event leading to virulence on Rlm1 genotypes.;citation_pages=672-682;citation_volume=15;citation_year=2002;citation_issue=7;citation_pmid=12118883" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. Bakó;citation_author=M. Umeda;citation_author=A. F. Tiburcio;citation_author=J. Schell;citation_author=C. Koncz;citation_title=The VirD2 pilot protein of Agrobacterium-transferred DNA interacts with the TATA box-binding protein and a nuclear protein kinase in plants;citation_volume=100;citation_year=2003;citation_pmid=12900506;citation_doi=10.1073/pnas.1733208100" />
<meta name="citation_reference" content="citation_journal_title=Biotechniques;citation_author=S. Balzergue;citation_author=B. Dubreucq;citation_author=S. Chauvin;citation_author=I. Le-Clainche;citation_author=F. Le Boulaire;citation_title=Improved PCR-walking for large-scale isolation of plant T-DNA borders;citation_pages=496-504;citation_volume=30;citation_year=2001;citation_pmid=11252785" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=F. Baudat;citation_author=A. Nicolas;citation_title=Clustering of meiotic double-strand breaks on yeast chromosome III;citation_pages=5213-5218;citation_volume=94;citation_year=1997;citation_issue=10;citation_pmid=9144217;citation_doi=10.1073/pnas.94.10.5213" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=L. E. Berchowitz;citation_author=S. E. Hanlon;citation_author=J. D. Lieb;citation_author=G. P. Copenhaver;citation_title=A positive but complex association between meiotic double-strand break hotspots and open chromatin in Saccharomyces cerevisiae;citation_volume=19;citation_year=2009;citation_pmid=19801530;citation_doi=10.1101/gr.096297.109" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=F. Blaise;citation_author=E. Remy;citation_author=M. Meyer;citation_author=L. Zhou;citation_author=J. P. Narcy;citation_title=A critical assessment of Agrobacterium tumefaciens-mediated transformation as a tool for pathogenicity gene discovery in the phytopathogenic fungus Leptosphaeria maculans.;citation_pages=123-138;citation_volume=44;citation_year=2007;citation_issue=2;citation_pmid=16979359;citation_doi=10.1016/j.fgb.2006.07.006" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=F. R. Blattner;citation_author=G. Plunkett;citation_author=C. A. Bloch;citation_author=N. T. Perna;citation_author=V. Burland;citation_title=The Complete Genome Sequence of Escherichia coli K-12;citation_pages=1453-1462;citation_volume=277;citation_year=1997;citation_issue=5331;citation_pmid=9278503;citation_doi=10.1126/science.277.5331.1453" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=H. G. Blitzblau;citation_author=G. W. Bell;citation_author=J. Rodriguez;citation_author=S. P. Bell;citation_author=A. Hochwagen;citation_title=Mapping of meiotic single-stranded DNA reveals double-stranded-break hotspots near centromeres and telomeres.;citation_pages=2003-2012;citation_volume=17;citation_year=2007;citation_issue=23;citation_pmid=18060788;citation_doi=10.1016/j.cub.2007.10.066" />
<meta name="citation_reference" content="citation_journal_title=Photochemistry and Photobiology;citation_journal_abbrev=Photochemistry and Photobiology;citation_author=G. U. L. Brags;citation_author=D. E. Rangel;citation_author=S. D. Flint;citation_author=A. J. Anderson;citation_author=D. W. Roberts;citation_title=Conidial pigmentation is important to tolerance against solar-simulated radiation in the entomopathogenic fungus Metarhizium anisopliae.;citation_pages=418-422;citation_volume=82;citation_year=2006;citation_issue=2;citation_pmid=16613494;citation_doi=10.1562/2005-05-08-RA-52" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=R. Breathnach;citation_author=P. Chambon;citation_title=Organization and expression of eucaryotic split genes coding for proteins.;citation_pages=349-383;citation_volume=50;citation_year=1981;citation_pmid=6791577;citation_doi=10.1146/annurev.bi.50.070181.002025" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=P. Bucher;citation_title=Weight matrix descriptions of four eukaryotic RNA polymerase II promoter elements derived from 502 unrelated promoter sequences.;citation_pages=563-578;citation_volume=212;citation_year=1990;citation_issue=4;citation_pmid=2329577;citation_doi=10.1016/0022-2836(90)90223-9" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=C. Buhler;citation_author=V. Borde;citation_author=M. Lichten;citation_title=Mapping meiotic single-strand DNA reveals a new landscape of DNA double-strand breaks in Saccharomyces cerevisiae.;citation_pages=e324-e324;citation_volume=5;citation_year=2007;citation_issue=12;citation_pmid=18076285;citation_doi=10.1371/journal.pbio.0050324" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in structural biology;citation_journal_abbrev=Curr Opin Struct Biol;citation_author=S. K. Burley;citation_title=The TATA box binding protein.;citation_pages=69-75;citation_volume=6;citation_year=1996;citation_issue=1;citation_pmid=8696975;citation_doi=10.1016/S0959-440X(96)80097-2" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=J. Choi;citation_author=J. Park;citation_author=J. Jeon;citation_author=M. H. Chi;citation_author=J. Goh;citation_title=Genome-wide analysis of T-DNA integration into the chromosomes of Magnaporthe oryzae.;citation_pages=371-382;citation_volume=66;citation_year=2007;citation_issue=2;citation_pmid=17850257;citation_doi=10.1111/j.1365-2958.2007.05918.x" />
<meta name="citation_reference" content="citation_journal_title=Trends in microbiology;citation_journal_abbrev=Trends Microbiol;citation_author=K. R. Christie;citation_author=E. L. Hong;citation_author=J. M. Cherry;citation_title=Functional annotations for the Saccharomyces cerevisiae genome: the knowns and the known unknowns.;citation_pages=286-294;citation_volume=17;citation_year=2009;citation_issue=7;citation_pmid=19577472;citation_doi=10.1016/j.tim.2009.04.005" />
<meta name="citation_reference" content="citation_journal_title=Cellular microbiology;citation_journal_abbrev=Cell Microbiol;citation_author=V. Citovsky;citation_author=S. V. Kozlovsky;citation_author=B. Lacroix;citation_author=A. Zaltsman;citation_author=M. Dafny-Yelin;citation_title=Biological systems of the host cell involved in Agrobacterium infection.;citation_pages=9-20;citation_volume=9;citation_year=2007;citation_issue=1;citation_pmid=17222189;citation_doi=10.1111/j.1462-5822.2006.00830.x" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Biol.;citation_author=C. D'Enfert;citation_title=Fungal spore germination: insights from the molecular genetics of Aspergillus nidulans and Neurospora crassa;citation_pages=163-172;citation_volume=21;citation_year=1997;citation_doi=10.1006/fgbi.1997.0975" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Djamei;citation_author=A. Pitzschke;citation_author=H. Nakagami;citation_author=I. Rajh;citation_author=H. Hirt;citation_title=Trojan Horse Strategy in Agrobacterium Transformation: Abusing MAPK Defense Signaling;citation_pages=453-456;citation_volume=318;citation_year=2007;citation_issue=5849;citation_pmid=17947581;citation_doi=10.1126/science.1148110" />
<meta name="citation_reference" content="citation_journal_title=Molecular plant-microbe interactions :  MPMI;citation_journal_abbrev=Mol Plant Microbe Interact;citation_author=C. E. Elliott;citation_author=B. J. Howlett;citation_title=Overexpression of a 3-ketoacyl-CoA thiolase in Leptosphaeria maculans causes reduced pathogenicity on Brassica napus.;citation_pages=588-596;citation_volume=19;citation_year=2006;citation_issue=6;citation_pmid=16776292;citation_doi=10.1094/MPMI-19-0588" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. P. Francino;citation_author=H. Ochman;citation_title=Strand asymmetries in DNA evolution.;citation_pages=240-245;citation_volume=13;citation_year=1997;citation_issue=6;citation_pmid=9196330;citation_doi=10.1016/S0168-9525(97)01118-9" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=A. C. Frank;citation_author=J. R. Lobry;citation_title=Asymmetric substitution patterns: a review of possible underlying mutational or selective mechanisms;citation_pages=65-77;citation_volume=238;citation_year=1999;citation_pmid=10570985;citation_doi=10.1016/S0378-1119(99)00297-8" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. Fujimori;citation_author=T. Washio;citation_author=M. Tomita;citation_title=GC-compositional strand bias around transcription start sites in plants and fungi.;citation_pages=26-26;citation_volume=6;citation_year=2005;citation_pmid=15733327;citation_doi=10.1186/1471-2164-6-26" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. L. Gasior;citation_author=T. P. Wakeman;citation_author=B. Xu;citation_author=P. L. Deininger;citation_title=The human LINE-1 retrotransposon creates DNA double-strand breaks.;citation_pages=1383-1393;citation_volume=357;citation_year=2006;citation_issue=5;citation_pmid=16490214;citation_doi=10.1016/j.jmb.2006.01.089" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Goffeau;citation_author=B. G. Barrell;citation_author=H. Bussey;citation_author=R. W. Davis;citation_author=B. Dujon;citation_title=Life with 6000 Genes;citation_pages=546-567;citation_volume=274;citation_year=1996;citation_issue=5287;citation_pmid=8849441;citation_doi=10.1126/science.274.5287.546" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Götz;citation_author=J. M. García-Gómez;citation_author=J. Terol;citation_author=T. D. Williams;citation_author=S. H. Nagaraj;citation_title=High-throughput functional annotation and data mining with the blast2GO suite;citation_volume=36;citation_year=2008;citation_pmid=18445632;citation_doi=10.1093/nar/gkn176" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Grigoriev;citation_title=Analyzing genomes with cumulative skew diagrams;citation_volume=26;citation_year=1998;citation_pmid=9580676;citation_doi=10.1093/nar/26.10.2286" />
<meta name="citation_reference" content="citation_title=Mycology;citation_author=I. V. Grigoriev;citation_author=D. Cullen;citation_author=S. B. Goodwin;citation_author=D. Hibbett;citation_author=T. W. Jeffries;citation_title=Fueling the future with fungal genomics;citation_pages=192-209;citation_volume=2;citation_year=2011;citation_doi=10.1080/21501203.2011.584577" />
<meta name="citation_reference" content="citation_journal_title=Technometrics;citation_author=F. E. Grubbs;citation_title=Procedures for detecting outlying observations in samples;citation_pages=1-21;citation_volume=11;citation_year=1969;citation_doi=10.2307/1266761" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=W. K. Huh;citation_author=J. V. Falvo;citation_author=L. C. Gerke;citation_author=A. S. Carroll;citation_author=R. W. Howson;citation_title=Global analysis of protein localization in budding yeast.;citation_pages=686-691;citation_volume=425;citation_year=2003;citation_issue=6959;citation_pmid=14562095;citation_doi=10.1038/nature02026" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=R. Javahery;citation_author=A. Khachi;citation_author=K. Lo;citation_author=B. Zenzie-Gregory;citation_author=S. T. Smale;citation_title=DNA sequence requirements for transcriptional initiator activity in mammalian cells.;citation_pages=116-127;citation_volume=14;citation_year=1994;citation_issue=1;citation_pmid=8264580;citation_doi=10.1128/MCB.14.1.116" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=G. M. Jones;citation_author=J. Stalker;citation_author=S. Humphray;citation_author=A. West;citation_author=T. Cox;citation_title=A systematic library for comprehensive overexpression screens in Saccharomyces cerevisiae.;citation_pages=239-241;citation_volume=5;citation_year=2008;citation_issue=3;citation_pmid=18246075;citation_doi=10.1038/nmeth.1181" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=Y. A. Kourmpetis;citation_author=A. D. van Dijk;citation_author=R. C. van Ham;citation_author=C. J. ter Braak;citation_title=Genome-wide computational function prediction of Arabidopsis proteins by integration of multiple data sources;citation_volume=155;citation_year=2011;citation_pmid=21098674;citation_doi=10.1104/pp.110.162164" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=A. Krishnan;citation_author=E. Guiderdoni;citation_author=G. An;citation_author=Y. I. Hsing;citation_author=C. D. Han;citation_title=Mutant Resources in Rice for Functional Genomics of the Grasses;citation_pages=165-170;citation_volume=149;citation_year=2009;citation_issue=1;citation_pmid=19126710;citation_doi=10.1104/pp.108.128918" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=F. Kunst;citation_author=N. Ogasawara;citation_author=I. Moszer;citation_author=A. M. Albertini;citation_author=G. Alloni;citation_title=The complete genome sequence of the gram-positive bacterium Bacillus subtilis.;citation_pages=249-256;citation_volume=390;citation_year=1997;citation_issue=6657;citation_pmid=9384377;citation_doi=10.1038/36786" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Li;citation_author=A. Krichevsky;citation_author=M. Vaidya;citation_author=T. Tzfira;citation_author=V. Citovsky;citation_title=Uncoupling of the functions of the Arabidopsis VIP1 protein in transient and stable plant genetic transformation by Agrobacterium;citation_volume=102;citation_year=2005;citation_pmid=15824315;citation_doi=10.1073/pnas.0404118102" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=Y. G. Liu;citation_author=N. Mitsukawa;citation_author=T. Oosumi;citation_author=R. F. Whittier;citation_title=Efficient isolation and mapping of Arabidopsis thaliana T-DNA insert junctions by thermal asymmetric interlaced PCR.;citation_pages=457-463;citation_volume=8;citation_year=1995;citation_issue=3;citation_pmid=7550382;citation_doi=10.1046/j.1365-313X.1995.08030457.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. R. Lobry;citation_title=Origin of Replication of Mycoplasma genitalium;citation_pages=745-746;citation_volume=272;citation_year=1996;citation_issue=5262;citation_pmid=8614839;citation_doi=10.1126/science.272.5262.745" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=J. R. Lobry;citation_title=Asymmetric substitution patterns in the two DNA strands of bacteria;citation_pages=660-665;citation_volume=13;citation_year=1996;citation_issue=5;citation_pmid=8676740;citation_doi=10.1093/oxfordjournals.molbev.a025626" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=A. Loyter;citation_author=J. Rosenbluh;citation_author=N. Zakai;citation_author=J. Li;citation_author=S. V. Kozlovsky;citation_title=The Plant VirE2 Interacting Protein 1. A Molecular Link between the Agrobacterium T-Complex and the Host Cell Chromatin?;citation_pages=1318-1321;citation_volume=138;citation_year=2005;citation_issue=3;citation_pmid=16010006;citation_doi=10.1104/pp.105.062547" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=L. J. Ma;citation_author=H. C. van der Does;citation_author=K. A. Borkovich;citation_author=J. J. Coleman;citation_author=M. J. Daboussi;citation_title=Comparative genomics reveals mobile pathogenicity chromosomes in Fusarium.;citation_pages=367-373;citation_volume=464;citation_year=2010;citation_issue=7287;citation_pmid=20237561;citation_doi=10.1038/nature08850" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. McVey;citation_author=S. E. Lee;citation_title=MMEJ repair of double-strand breaks (director's cut): deleted sequences and alternative endings.;citation_pages=529-538;citation_volume=24;citation_year=2008;citation_issue=11;citation_pmid=18809224;citation_doi=10.1016/j.tig.2008.08.007" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=Y. Meng;citation_author=G. Patel;citation_author=M. Heist;citation_author=M. F. Betts;citation_author=S. L. Tucker;citation_title=A systematic analysis of T-DNA insertion events in Magnaporthe oryzae.;citation_pages=1050-1064;citation_volume=44;citation_year=2007;citation_issue=10;citation_pmid=17544743;citation_doi=10.1016/j.fgb.2007.04.002" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=C. B. Michielse;citation_author=P. J. Hooykaas;citation_author=C. van den Hondel;citation_author=A. F. Ram;citation_title=Agrobacterium-mediated transformation as a tool for functional genomics in fungi.;citation_pages=1-17;citation_volume=48;citation_year=2005;citation_issue=1;citation_pmid=15889258;citation_doi=10.1007/s00294-005-0578-0" />
<meta name="citation_reference" content="citation_journal_title=Environ. Health Perspect.;citation_author=A. T. Natarajan;citation_title=Mechanisms for induction of mutations and chromosome alterations;citation_pages=225-229;citation_volume=101;citation_year=1993" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiology Letters;citation_journal_abbrev=FEMS Microbiol Lett;citation_author=N. Osherov;citation_author=G. S. May;citation_title=The molecular mechanisms of conidial germination.;citation_pages=153-160;citation_volume=199;citation_year=2001;citation_issue=2;citation_pmid=11377860;citation_doi=10.1111/j.1574-6968.2001.tb10667.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Pena-Castillo;citation_author=T. R. Hughes;citation_title=Why are there still over 1000 uncharacterized yeast genes?;citation_pages=7-14;citation_volume=176;citation_year=2007;citation_pmid=17435240;citation_doi=10.1534/genetics.107.074468" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Pitzschke;citation_author=A. Djamei;citation_author=M. Teige;citation_author=H. Hirt;citation_title=VIP1 response elements mediate mitogen-activated protein kinase 3-induced stress gene expression;citation_volume=106;citation_year=2009;citation_pmid=19820165;citation_doi=10.1073/pnas.0905599106" />
<meta name="citation_reference" content="citation_journal_title=Journal of invertebrate pathology;citation_journal_abbrev=J Invertebr Pathol;citation_author=D. E. Rangel;citation_author=M. J. Butler;citation_author=J. Torabinejad;citation_author=A. J. Anderson;citation_author=G. U. Braga;citation_title=Mutants and isolates of Metarhizium anisopliae are diverse in their relationships between conidial pigmentation and stress tolerance.;citation_pages=170-182;citation_volume=93;citation_year=2006;citation_issue=3;citation_pmid=16934287;citation_doi=10.1016/j.jip.2006.06.008" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=E. Remy;citation_author=M. Meyer;citation_author=F. Blaise;citation_author=M. Chabirand;citation_author=N. Wolff;citation_title=The Lmpma1 gene of Leptosphaeria maculans encodes a plasma membrane H+-ATPase isoform essential for pathogenicity towards oilseed rape.;citation_pages=1122-1134;citation_volume=45;citation_year=2008;citation_issue=7;citation_pmid=18538267;citation_doi=10.1016/j.fgb.2008.04.008" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=E. Remy;citation_author=M. Meyer;citation_author=F. Blaise;citation_author=U. K. Simon;citation_author=D. Kuhn;citation_title=The Lmgpi15 gene, encoding a component of the glycosylphosphatidylinositol anchor biosynthesis pathway, is required for morphogenesis and pathogenicity in Leptosphaeria maculans.;citation_pages=1105-1120;citation_volume=179;citation_year=2008;citation_issue=4;citation_pmid=18557818;citation_doi=10.1111/j.1469-8137.2008.02522.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular plant-microbe interactions :  MPMI;citation_journal_abbrev=Mol Plant Microbe Interact;citation_author=E. Remy;citation_author=M. Meyer;citation_author=F. Blaise;citation_author=U. K. Simon;citation_author=D. Kuhn;citation_title=A key enzyme of the Leloir pathway is involved in pathogenicity of Leptosphaeria maculans toward oilseed rape.;citation_pages=725-736;citation_volume=22;citation_year=2009;citation_issue=6;citation_pmid=19445597;citation_doi=10.1094/MPMI-22-6-0725" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=M. Ricchetti;citation_author=B. Dujon;citation_author=C. Fairhead;citation_title=Distance from the chromosome end determines the efficiency of double strand break repair in subtelomeres of haploid yeast.;citation_pages=847-862;citation_volume=328;citation_year=2003;citation_issue=4;citation_pmid=12729759;citation_doi=10.1016/S0022-2836(03)00315-2" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Commun;citation_author=T. Rouxel;citation_author=J. Grandaubert;citation_author=J. K. Hane;citation_author=C. Hoede;citation_author=A. P. van de Wouw;citation_title=Effector diversification within compartments of the Leptosphaeria maculans genome affected by Repeat-Induced Point mutations.;citation_pages=202-202;citation_volume=2;citation_year=2011;citation_pmid=21326234;citation_doi=10.1038/ncomms1189" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=S. T. Smale;citation_author=J. T. Kadonaga;citation_title=The RNA polymerase II core promoter;citation_pages=449-479;citation_volume=72;citation_year=2003;citation_pmid=12651739;citation_doi=10.1146/annurev.biochem.72.121801.161520" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. D. Spanu;citation_author=J. C. Abbott;citation_author=J. Amselem;citation_author=T. A. Burgis;citation_author=D. M. Soanes;citation_title=Genome Expansion and Gene Loss in Powdery Mildew Fungi Reveal Tradeoffs in Extreme Parasitism;citation_pages=1543-1546;citation_volume=330;citation_year=2010;citation_issue=6010;citation_pmid=21148392;citation_doi=10.1126/science.1194573" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=T. Tatarinova;citation_author=V. Brover;citation_author=M. Troukhan;citation_author=N. Alexandrov;citation_title=Skew in CG content near the transcription start site in Arabidopsis thaliana;citation_pages=i313-314;citation_volume=19;citation_year=2003;citation_issue=90001;citation_pmid=12855475;citation_doi=10.1093/bioinformatics/btg1043" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=V. Thole;citation_author=B. Worland;citation_author=J. Wright;citation_author=M. W. Bevan;citation_author=P. Vain;citation_title=Distribution and characterization of more than 1000 T-DNA tags in the genome of Brachypodium distachyon community standard line Bd21.;citation_pages=734-747;citation_volume=8;citation_year=2010;citation_issue=6;citation_pmid=20374523;citation_doi=10.1111/j.1467-7652.2010.00518.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=E. R. Tillier;citation_author=R. A. Collins;citation_title=The contributions of replication orientation, gene direction, and signal sequences to base-composition asymmetries in bacterial genomes.;citation_pages=249-257;citation_volume=50;citation_year=2000;citation_issue=3;citation_pmid=10754068" />
<meta name="citation_reference" content="citation_journal_title= Biochimie;citation_author=M. Touchon;citation_author=E. P. C. Rocha;citation_title=A gentle guide to the analysis of strand asymmetry in genome sequences.;citation_pages=648-659;citation_volume=90;citation_year=2008;citation_pmid=17988781;citation_doi=10.1016/j.biochi.2007.09.015" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Touchon;citation_author=A. Arneodo;citation_author=Y. d'Aubenton-Carafa;citation_author=C. Thermes;citation_title=Transcription-coupled and splicing-coupled strand asymmetries in eukaryotic genomes;citation_pages=4969-4978;citation_volume=32;citation_year=2004;citation_issue=17;citation_pmid=15388799;citation_doi=10.1093/nar/gkh823" />
<meta name="citation_reference" content="citation_journal_title=Annual review of microbiology;citation_journal_abbrev=Annu Rev Microbiol;citation_author=T. Tzfira;citation_author=J. Li;citation_author=B. Lacroix;citation_author=V. Citovsky;citation_title=NUCLEIC ACID TRANSPORT IN PLANT-MICROBE INTERACTIONS: The Molecules That Walk Through the Walls;citation_pages=187-219;citation_volume=54;citation_year=2000;citation_issue=1;citation_pmid=11018128;citation_doi=10.1146/annurev.micro.54.1.187" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=T. Tzfira;citation_author=Y. Rhee;citation_author=M.-H. Chen;citation_author=V. Citovsky;citation_title=Agrobacterium T-DNA integration: molecules and models.;citation_pages=375-383;citation_volume=20;citation_year=2004;citation_issue=8;citation_pmid=15262410;citation_doi=10.1016/j.tig.2004.06.004" />
<meta name="citation_reference" content="citation_journal_title=Can. J. Plant Pathol.;citation_author=O. H. Will;citation_author=D. Dixon;citation_author=A. Birney;citation_author=P. L. Thomas;citation_title=Effects of far UV and visible light on germination of wild type and albino teliospores of Ustilago nuda;citation_pages=225-229;citation_volume=9;citation_year=1987;citation_doi=10.1080/07060668709501878" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. A. Winzeler;citation_author=D. D. Shoemaker;citation_author=A. Astromoff;citation_author=H. Liang;citation_author=K. Anderson;citation_title=Functional Characterization of the S. cerevisiae Genome by Gene Deletion and Parallel Analysis;citation_pages=901-906;citation_volume=285;citation_year=1999;citation_issue=5429;citation_pmid=10436161;citation_doi=10.1126/science.285.5429.901" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=T. C. Wu;citation_author=M. Lichten;citation_title=Meiosis-induced double-strand break sites determined by yeast chromatin structure;citation_pages=515-518;citation_volume=263;citation_year=1994;citation_issue=5146;citation_pmid=8290959;citation_doi=10.1126/science.8290959" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. Yuryev;citation_author=M. Patturajan;citation_author=Y. Litingtung;citation_author=R. V. Joshi;citation_author=C. Gentile;citation_title=The C-terminal domain of the largest subunit of RNA polymerase II interacts with a novel set of serine/arginine-rich proteins.;citation_pages=6975-6980;citation_volume=93;citation_year=1996;citation_issue=14;citation_pmid=8692929;citation_doi=10.1073/pnas.93.14.6975" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=J. Zhang;citation_author=D. Gu;citation_author=Y. X. Chang;citation_author=C. J. You;citation_author=X. W. Li;citation_title=Non-random distribution of T-DNA insertions at various levels of the genome hierarchy as revealed by analyzing 13 804 T-DNA flanking sequences from an enhancer-trap mutant library.;citation_pages=947-959;citation_volume=49;citation_year=2007;citation_issue=5;citation_pmid=17253985;citation_doi=10.1111/j.1365-313X.2006.03001.x" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=J. Zupan;citation_author=T. R. Muth;citation_author=O. Draper;citation_author=P. C. Zambryski;citation_title=The transfer of DNA from agrobacterium tumefaciens into plants: a feast of fundamental insights.;citation_pages=11-28;citation_volume=23;citation_year=2000;citation_issue=1;citation_pmid=10929098;citation_doi=10.1046/j.1365-313x.2000.00808.x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Incidence of Genome Structure, DNA Asymmetry, and Cell Physiology on T-DNA Integration in Chromosomes of the Phytopathogenic Fungus Leptosphaeria maculans" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8.cover-source.jpg" />
<meta name="twitter:description" content="The ever-increasing generation of sequence data is accompanied by unsatisfactory functional annotation, and complex genomes, such as those of plants and filamentous fungi, show a large number of genes with no predicted or known function. For functional annotation of unknown or hypothetical genes, the production of collections of mutants using Agrobacterium tumefaciens –mediated transformation (ATMT) associated with genotyping and phenotyping has gained wide acceptance. ATMT is also widely used to identify pathogenicity determinants in pathogenic fungi. A systematic analysis of T-DNA borders was performed in an ATMT-mutagenized collection of the phytopathogenic fungus Leptosphaeria maculans to evaluate the features of T-DNA integration in its particular transposable element-rich compartmentalized genome. A total of 318 T-DNA tags were recovered and analyzed for biases in chromosome and genic compartments, existence of CG/AT skews at the insertion site, and occurrence of microhomologies between the T-DNA left border (LB) and the target sequence. Functional annotation of targeted genes was done using the Gene Ontology annotation. The T-DNA integration mainly targeted gene-rich, transcriptionally active regions, and it favored biological processes consistent with the physiological status of a germinating spore. T-DNA integration was strongly biased toward regulatory regions, and mainly promoters. Consistent with the T-DNA intranuclear-targeting model, the density of T-DNA insertion correlated with CG skew near the transcription initiation site. The existence of microhomologies between promoter sequences and the T-DNA LB flanking sequence was also consistent with T-DNA integration to host DNA mediated by homologous recombination based on the microhomology-mediated end-joining pathway." />
<meta name="og-title" property="og:title" content="Incidence of Genome Structure, DNA Asymmetry, and Cell Physiology on T-DNA Integration in Chromosomes of the Phytopathogenic Fungus Leptosphaeria maculans" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/8/891" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The ever-increasing generation of sequence data is accompanied by unsatisfactory functional annotation, and complex genomes, such as those of plants and filamentous fungi, show a large number of genes with no predicted or known function. For functional annotation of unknown or hypothetical genes, the production of collections of mutants using Agrobacterium tumefaciens –mediated transformation (ATMT) associated with genotyping and phenotyping has gained wide acceptance. ATMT is also widely used to identify pathogenicity determinants in pathogenic fungi. A systematic analysis of T-DNA borders was performed in an ATMT-mutagenized collection of the phytopathogenic fungus Leptosphaeria maculans to evaluate the features of T-DNA integration in its particular transposable element-rich compartmentalized genome. A total of 318 T-DNA tags were recovered and analyzed for biases in chromosome and genic compartments, existence of CG/AT skews at the insertion site, and occurrence of microhomologies between the T-DNA left border (LB) and the target sequence. Functional annotation of targeted genes was done using the Gene Ontology annotation. The T-DNA integration mainly targeted gene-rich, transcriptionally active regions, and it favored biological processes consistent with the physiological status of a germinating spore. T-DNA integration was strongly biased toward regulatory regions, and mainly promoters. Consistent with the T-DNA intranuclear-targeting model, the density of T-DNA insertion correlated with CG skew near the transcription initiation site. The existence of microhomologies between promoter sequences and the T-DNA LB flanking sequence was also consistent with T-DNA integration to host DNA mediated by homologous recombination based on the microhomology-mediated end-joining pathway." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/8.cover-source.jpg" />
    <title>Incidence of Genome Structure, DNA Asymmetry, and Cell Physiology on T-DNA Integration in Chromosomes of the Phytopathogenic Fungus Leptosphaeria maculans | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q0wst1" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__KPgiwEdjo5O9HdRryHqQs1TpsV-PeBb9B7sGBvwM9FQ__opzZWgPNgQzoEgfATRS5qCKwBw_6XcNjDHoDVOzQv8M__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__YUqwBlv6bNoCToiBWgAUSPtppdS8sbGfzmAH3qEDcI0__Mvk93h3ScFH5Xi9WhibVKlf368fHw-V7HWqWX2QMvtw__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XNWYaQubFGWYWZNytUHkWs-b34LV21DsB4s_6gchI4k__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__7cpBA0GVdrMqkZGGZEW-_SPF2HrbTCmVg9-lrP5l_AQ__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__TV2D4R18tPiSIOmeQ3vyMqHObN_FDmQ6EEIrfE5sAoA__c9BBZFy1BrgGgF_Auu9Fmysb3Z7J56eJGYvMcLQrLks__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q0wst1"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__OaHnYyci68QTwctdRYR30kEfu0eYWBrLyMqy8THv5Ik__IyfJQYM1q1QEPwYsK_Eslh2RQkxpy6yo-nR0p4DWPQQ__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__fhq_kmKS3zjIxlsAa3GE1ubLajyXUgXXTXULzAtiqfg__Lm1_Ls-bdpbWRyl0q4o0mmHyxB-139AJ68dE4zOskLQ__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"IYI3WNXDYSDA0VuDg0SG94Fg-BL4ywi9nJzqY5xObIU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459307","apath":"\/ggg\/2\/8\/891.atom","pisa":"ggg;2\/8\/891","ac":{"\/ggg\/2\/8\/891.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/8\/891.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/8\/891"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/8\/891"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/8\/891":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022aWtI+WjaI43go4Cd+by1lbR+tlWrohfcnrsBo0RKKlG8Dvn0DrBsxnH4Uh1btizcaBsFtq85wHPUjSuJG\\\\\\\/bctN7pc8tyiS9+ToF3qhNSz9GTDkzXDONBfmbiTMBw6cHIqhPKjJqfFjL11\\\\\\\/1mv1iLZ21\\\\\\\/2\\\\\\\/SGzRtCePgW0uB14yBUEUVlQWTJlsG6krGbl56MYUAjatKPlfFqIBfqESL8f\\\\\\\/U4cLXfEMbcay++NJx97NvNHfCr\\\\\\\/DJwzb386nBQRJ0gmz+LChaQ36tc9Bt\\\\\\\/znFrxC28AWFacvZ3K6b6Xj0GYcHpdhxs\\\\\\\/KyuHo\\\\\\\/48NrXXk\\\\\\\/zlR+MUJOtBIGsgeI7VrX07fi8UuKaF8Ou12RT4A37PPMOyYWB8RkMx07sv5VvZAJgKn6UoVS+BobHUfX6cr48Bb+Mbnyk3gHTOzGZsES\\\\\\\/kxmXIU30onz57fQ6Wl0q9cMwNv8JGLUVZEJigOW0\\\\\\\/TE\\\\\\\/k+3EabPFVO2p\\\\\\\/ODIGg0drknbR11EeSiU+6qs23aQZ70uUrGz93E1Ih40UVO3e7Nq\\\\\\\/7L2Jnw2Q8pZ9mjc8Ui\\\\\\\/2nXe26Z1pZHayFbsDYXAwjn0df2i4iWJ0gqLYoTkXzd5hdx0c7smKy8nghCxn+6\\\\\\\/iRyLurUa3UKr3OCAX4Va\\\\\\\/w0jfapQ3If5nRldoEHzto5YpPjaX19rOhf4XNqrSaCvvCBLJwWAvCMQV1S2zVBpEkjS6uS6Bqz3T3uW6\\\\\\\/J3pQqb1WJ85wnJGdbt1\\\\\\\/XMVczt15e\\\\\\\/lJmpSBofZBSES4iumaAY2ZTheomnLxx90OLpVMLf6I1ZeK0+fVNKlOdosYOXoh9duhakew9AbYm9Csli1w\\\\\\\/lGXHo4Pe9Plqc2TpEG6AAu+xQWdkULpGCfDR0mkNKURpDqFmKhPCu32DBndWVyUPeCUZh9R2A+\\\\\\\/9SIdIAyFmsnuEZp55Z\\\\\\\/ob++iR8zroctt9D1XT6gvYhopHMmIb1quqrh5N9I7LzPGxncb\\\\\\\/KbJK+DslNly5VkrlFMi4fhirQfKiYne6BzB+4cJrHrmTDlltw+wv132lBjgDx+iJmvSqgtFsiKSi7xNP0q9MZjEVEdexNnVNmeTSzlwZLF+w8DhxUdfWtUMUVB6xTzFuKCItwHo3Nn8btbMPAqWLzHCzYix4TNVylm6PtgEgpkV6FAhRNTxRL4C68v3Jy2Czk4o6HujhHUv5F8kbmI\\\\\\\/stFk11LHOQr3cOsbf7QerDMYIRs2IkJpRN\\\\\\\/PpUM3uARik46\\\\\\\/ADdRFuN5nU1cpkSnMuSp7E7\\\\\\\/Wk5opK92JQHFUIbgBjdLt8\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/pfZQ6bBQc8dLfV7mLxjHA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222a5f1fbb0bcddb6697192976e31c44e0862c194239cde1bd879414fe3532e52f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022SNOeGnUmhUS9nhMgbPhwqvQCfeFDBHBOue6sn179XFwjbLlJ1P\\\\\\\/kK\\\\\\\/IzWSZRl9jg\\\\\\\/JG+VqFhECrDwm5F29oAxGZOXbb7VoyacVc5WX4e1AelyKkItxvFe5R+\\\\\\\/LjpTFt65iESJcRHSKNwyNhljZqqP5acf\\\\\\\/w4oPaFv9TBfB6zYBExu0WItWGfQPgbASFBDQnbRCMzSKfq7G4CpUBLdEpyNELA2MxKOFOHHkgohc6AUwtnhVB+uQ3nwTJrkMUyjZBYu5zARoC4hRqMYz+gdy0\\\\\\\/0fCH71HquXd2A13M5Hij07Be75BE4KNubDBlOxvjwZcHBBbRGAkb+sAAJNmZ6EhFiyhDCgygf8kWi+bbBJqCXlcvG+9tZfoF2O+jMIaA42hcs+T6sA\\\\\\\/SZRQtyiBqOQnQDuG+tw4\\\\\\\/lcfErX2yuzORBxkq9zje+R51wwAl0gJYt43bgRcs0\\\\\\\/Xk1vZhAXx+5RJEMt+gzygDZ0qYybbTkxAU7waA5fWSee1LwU1KWHudznZiuSNiulpEJIqDLqqAvhIQT4saplm83e3DTQG+EILOxAbUuo1iLFJJOzliCEtPR91tEiJX4L0kDstMOhcy3HWPCf+Cc0p6+hsNcQylZU+NRlfi2K7FKO5tlBY2GYtqaeUbdUjPr3AeKxxxHYaUYpiZzjpufc\\\\\\\/BcsoCZf1aVbVj8TLwYpcbqS\\\\\\\/3YYzwEwBDPqduqW0A+QmMrllqygwIi4sEGQ\\\\\\\/XOy5WcvaXBNQPPE43eIFfI1UkxA41m5EeJnqGT18QLtxDUQFSy+lm+CM2Z\\\\\\\/TJQ17yS3JTvrdYNfEJRQu71M4fb58n01mlTtQQeMzrnaSDw014XW5gu3l\\\\\\\/tRfqcN+UAasJ38snA9wwzqP+QAsJz8Aiz7MrykEG4JNyFpxvqCZJZ2M2djgaQqMy+sQ60ByZ7apdNPFik05ZIJPMi0evtcbTLRHRCvKovYlv16J6B7223nXj80CXVqktGNqOFoPTQjrJtOdhMX9lAwk3G+PX3GzMJXe058eajtKwlS0wQrwvf\\\\\\\/6L6YEhh0sTbJSRhMItXgbXwhRP68\\\\\\\/Ek8bVuYuN3qYNs8X0D+JDMrB0iZUGTpj6ivXrPvaOD2XSKJU0QqrN0WbVcq2keWoHesCFcorBqUHv3dkOHPC4ju2QOLgrU+xVRfGtpSPlhrHi51a+wFULNFYimfUDTajP8ns83qwDmqlnoANLC2NnQlSI8xBSrWGl8bHcWSk3ImMrwbWRuWexuDP09Yo8g9cZXi3irLWN4HmSmR+Ax8kJqJ1YLj4gUpgNcfxpMlFUNWboU1ZPnBoft6HZ9Eo\\\\\\\/WbLYpnGdOi9ag+D8oj4cyg9Q\\\\\\\/L0+cmPktoNrjlgzOksYkoVORkrUdoAOnfsFVzZUqwm8233unLtg65GLYk92hCIlOAuIGWEuB2k9wmPgIPtv65B\\\\\\\/PgK++ytlbAwJ95nOfKSCdarK389gY18AdANNEUCRXrQmRL21eiQwVlI6W07+Efa+BqCl0LYvoz5+Qa84paV8UIzr9GisY2+tyV2g7hEOGSNkF3s29lQ+JpNUkR1whZRo7DVpSOXWtrmcVAYIYUhczV8+WZ9kXJmAxrWpnGwu7iO3SIDD3\\\\\\\/W2FZt2Sl+BPGrwvjGrgNZzEXAQuw5RzA8\\\\\\\/OeheC5H7KMXuCo7UzLjuiTg6zkQkiffoQDlXUO1cnamCuCnNEZXkmyIbueWMXFI4aKNMaW6dum64QGecyreaUgo+EFBokPPGxqc9cEsNU0nRyBUkAannm2Bd2tC+9AchrvzIqUXSdo07rWBZykYi050LmKL+7R6zmZC44SMCfiHcT\\\\\\\/9gio7KWMBVn9H4ODngCWy+TCJ6T86EsNJMynb84p2iCSbnV4yZVEoj8+ShGuIZMR7pvJA2+bxhmLwZSFg4XPHZdQiCxnYmOCwvAKHiOzzLDTG7NQ+ycU5FyMZs9\\\\\\\/\\\\\\\/grmoTz9PLQM2E+dyjh8Nh\\\\\\\/O+VaaH\\\\\\\/VJom+XAOdEM04wNi0nIY4eHWHxcygwTpK\\\\\\\/+umKWjsaCOXmtEIHmLT9+eTSx1IIYGock3U+k3gjwBa8fHrf\\\\\\\/lYr2EMcJZh3jwo1FH\\\\\\\/jdXZnNSlw9w6DrE9hiexX2Y6J19Jizro8Cgyxfh7X6U97b3AfAh0l+iU16dXDdOx\\\\\\\/K5t394nveYLuleS2\\\\\\\/Y0wfgEPnx\\\\\\\/I3zTMuPL47QLOvqgoGMj1yHDki3\\\\\\\/bnlalPPy\\\\\\\/uIhUJL41mvsu+b6roNeu90Di09c6nqs0rakaFiUcDXWlCt0Rs9GJz1eBffXAZoGnD7YvCpWf9kjFDECrye0N6M14pRtDLjF95qrG6hDqp5QT5GHZAERNDJcy3m3EMDxb8XixqA6dSKsFUxcCZow8g7OlLM19N807x7t9pL\\\\\\\/f2tAtwOZFMbHUUbr0QU7T5fs0xF\\\\\\\/2AJOQK6D+Cd7eCSwW8l4tNEHCV+KOZ9BjiBCDMkzvvwgZGz8sS9uizP0ZzsqVub4MWRqFGvlt8v5C\\\\\\\/wenoUgWpSDq0+G+eRhq3P6s57b2p736+O8fWSsDXnfI93yf2ahQYkO\\\\\\\/036opw3PHzJw9kZWwccb8Rpk1GGqu3iho\\\\\\\/JbvreJP1VqVSL5RuH1vuZ\\\\\\\/z\\\\\\\/eYzECpTCTZcYRA7pho9RS4KJLwb\\\\\\\/TDtZPtD9fOKIaNJq8G2pub5BhSOooqYp1hRFrxUtInI0+KmkbKM+tBHqAIdljcl82ye2bHwPtttkhTC0i3upiwXpeOEjqX\\\\\\\/C7LomnphIAEkjBJjuWI\\\\\\\/ElOrBRZeVqa1jyf8mSK4FKVKyWTnTH2xV\\\\\\\/5093OQR7DAalcSDtdJ7EWm03CvOVUuqEh3BLLKCu3LDz8MOKyZsdtNytK2stB6KJSbidsDrn2miAMCDYIbQIyyyhmPhB6WHytbL0frl6CPkEs3SeyvPYhQCUUznxtqPqTwKJXz9LONLxgvazBA0AUiCkuUUFVQ+\\\\\\\/qjZWDNmtZ8iylbv\\\\\\\/2n5ec\\\\\\\/Cf4j8H+CAjQPnryXwX5\\\\\\\/ihMTEt58awpFEM9pPp73mScBVVXz0GfcXpqgzEOUUkz6KNEe6Y8XLopMOcQVOWdthO3\\\\\\\/KMywrp7x5Hq4RzKehugnk5i4joR9+0Or013m6gdxw\\\\\\\/\\\\\\\/h7eJBUU5ySogyGgjZaky5GAwkrCr1+aOtdoAs0F1jN2PzqvRlDUG9xyzJyBpR9EuBbdjCrbYQOi8+FaTOOVcKn2LHh2LXwKQMvMTJ2jlJ93JJen6EmFnX35HCqPYQ\\\\\\\/FzH7xOSWTSPXpzsLIj08VU2WzBnA8GgrG3mijGMcd4qS0Gdej\\\\\\\/GSpxb4mLbSREKdQF0UnZU7v2WeVABrQS39dmwB7HbzQAWwjU9YhlKc0kelGJlBJE9kJMJQs8zaiTUSmsKKMxo1aWyKKh8l0S+j+lTdYWCNREsOWUddG\\\\\\\/v7rSEZQ2afvVkWDtuzOu2HMk1lh4Q6LgfCjM\\\\\\\/NMAmHa2vWIohTnNzKQBx85PsTZHmq7fRPIR5JI\\\\\\\/nMfM+eZYUFZZIU\\\\\\\/QFm4tj+hGOLvVK0MIWsdvhIcdQ20xhZZ77seaMD9Qbb\\\\\\\/ihQdVDXb3p7kPM671ZBuGJM2\\\\\\\/6ZNqEnP26nNNPtfqRcyYqQgC\\\\\\\/LuHNwOCiTL+WyhuDSAGnXmdrXpaNKKZraUoMfKnwk0KSUCf8uS\\\\\\\/jSCn+Pf3iMuuwvoRoqXiidLSV9aIbG7YVZUVn2hlLP72tQCIIgoSTw0Aw2ci7iid8AfT3iUipGegR\\\\\\\/0VxfjOVEFJ5TfYqFaQz7YKpYC4EzTsWPIYzD3J9dmTUcpmnfrYAoZ4eeDu39yYUie3H3PZTw+s9UBpShXHdc2xzfS8BYYS5L+9TXD9RoId4AAJ8aSJADynB\\\\\\\/jtEynX3icyDqBqkwuCp69EjOJv1orQl04Wddq1wtsMyE0M7BlJ3vulutLFBYeQNNccGqRd0QWipo8vdJ+O1s\\\\\\\/O5dVxF3W9pe+GjN59zzv8bkjevq0iHuh2mqZxYUGZqXE4wG2KhjsgRq\\\\\\\/hCRVsBCnLjrengIBQ\\\\\\\/PW\\\\\\\/jw9rc5RhZluLZDx4S570btI1aw9JBAkrAoVSZUEJrtiDKdSg3R+5BXTvRiNrVlbSvSukOyUZd7JskzLlZN+Vs3fHmODr5yRIlOK8BIdmIVKOgmwKnIMdZQ7UXmtAx7gWE2hS5EuuXlw89G5Ys9EiHk5mrgk1UUC4pJMEHcekDLVGLXKQqhU197wCOkKjgmgJgqMB2JY\\\\\\\/5+c9R7PrS0sqMN4HKM\\\\\\\/5rMW5wmPtCaIrkV6ahMLCVVSa9M8ssPU+qc3Vv0Pt5Yw4z4jV0VXmCdVBQhoDmMfOD8+uFJRuoXZmZwlcRRsexuEzOSbpztu9ZHn1IMLyDc7mB2n3fhICQtD7kBYJ+jtcpEF57GJyL2E6LsPgJE6Cl86mTcOrXPQICBmbxR6z70Qe2kyRUq3AitNd7Vovw\\\\\\\/37H9IxbuiPHIvjw\\\\\\\/eCYu5++kjciUkhtdj8+\\\\\\\/Q5ZxBH0U4qvqyWA6PL+p1gUVn8NXSkZofPVb1CsLlJHvYIzZdTiy56KDNuiXgj2bNfjZwzwVY4Xci1NXxj8q67kbflvE6OmqL42dYGBWAapyWsjEi4bySH4A+OH7Eed\\\\\\\/GYiooy2GRdq7TKb1SWYUrhCgDPm27HLnjwuB\\\\\\\/QycsTCzQkGct4kzN5JDokTYnZJpCTmTGOs2xR11UsOqBdIzkRW\\\\\\\/k4M73hJRLkv6gftxoO+XTblsBYM80XpI8rjI+wduxr2eZuBvTyE\\\\\\\/+i8Ecm7WBnxXKJ0cQVlDeyad0\\\\\\\/QwpVKPx2IWe78\\\\\\\/VYaGrFZQRwGjot79cqKGLJMzlg6lP8c7BPWQYeDXB5OpcaHKNmBTARBcyHdLFg\\\\\\\/NA5qj9DI1X1TGZREH6Zq+MozOl7czGk8ZnmE9KDSx0Js82gLUYxFIHwFDm8P\\\\\\\/oWqEptCsnZUChWhpZV6BeHu1rvg3wfGPy3mZTpZeUYTOAodDhSM0ZUEKGFAhG84D4EevwYhpUiKc8Fo5jOjTLdBxIeboqM4ljUhyOC+XDwRZ9SU06J81Mr6WhHvIfKwRWeYoBfDX4G2pBKbNyadzeEO558kzfBvrJ+03d82RvpqEt5cutsHkprY8fhqtPuMJ7o4MvYmTLlo1GplqyCX7zUmvd++R6+N0jP\\\\\\\/4hwk5AtngmNfQvgIilGpoVfntmWmox\\\\\\\/FIs04A8uuDDlWt+v\\\\\\\/fp6MSMXb2S3gSQrZqeVSoFsGAVi2kDZlyoFs4KQnTy5no84p7K+zgV4pWlee\\\\\\\/LMWlSRqY7RvCTTE+ZJb+6AnYCGD8lac5oLutogNXuJGV1mAffPdzAVfP67xboqOEcHn1q4Kzt5MqSqMtuuapxJOKWyzioakfLrrZz98SCFnNh4OVAaNLP6i+WrYJX9IY6eI8RWH95bsILTMW0zfbS7UO5JlXvI\\\\\\\/Sle7VUjp+Di0m1dnV765Nid\\\\\\\/701ngzUaboQCoIYHGRW9A7WeEaq4UmIQtRghte2dQEtagKlp5bg483id+YrymKVIsdxhBwM8u1bAKqadU7XmuU0rpSXB+6HMLZe4lDQcsM5y9n8Hozrm9HJ4TwTe5203oFZN16DfH6W+wfVG4mPYNfIzJqovLbHdF\\\\\\\/ylYctV9NC5H+QrFk\\\\\\\/RQCki6yLQtz\\\\\\\/KbwfNsQ+siSvPSu0ldZKM7D0RwLIRDfUsyQuuQIMmfddabkfVd4HaA3kkRhnmWAiQ5FtKgWKTMAUR3XLY5Gb8H3RWG4mBGsickJVvvZA145MY4fMi9WJMudg7asVnR4bcfbQhbQYMRwI01pxCvFeZHd9fDXswGrgjIxsy589BzNxwiXimfx3Y+qVUkyrwipCdVTSUHmXhcYbDeOa7\\\\\\\/XjdJh\\\\\\\/oiaB7rIe6eqpyvP\\\\\\\/0qqD7PlB5FfIcLjWixmaf7BiMztqnKjUv7do20A6rOgOh6qi5bKXNkOd5ELrHUKcWprnIPbo\\\\\\\/ImworS7V0SBwEcxRCzKuz4NiZxjmj\\\\\\\/joGoZKg+CR3r5Tc3S2rfrHJYKy31lt+rxwhnI9Q18wtHpaumXokVvtJQKz2t0tid4SrJpDq3DnM+\\\\\\\/A8kFowQw9vUqukXAoHR7ww9NDMOk8YupHVwS7+EY+hPHCe1xp+b+Xx2iBGHORIGjS42TPxvGB9NxlO8cbG9Fvy8MkhxbKn2SK+nIN6VXlaQqxJAaCx0RvzAXW6hui+UwzdHttnXjzTZ\\\\\\\/vfzdaAVfA\\\\\\\/g0+j+p9u6TBqC2j1+uAUvDitnOpLlRfzFCPx44waKo76HWxp0wwFk9mXv9GkDtPc\\\\\\\/jlUEkWTkjToBTz0a4AVSdSEOiMJWT2EuGyAOnQSOX4lXoGWsevwki+J\\\\\\\/vjvKt8GBCDmRa56GzrhtUk2nfxj6\\\\\\\/nEgC5RHksKg7cptaqlf9F6N+vZWh2L1+7aiZnj2Dr151rEwfyqp9E61Y3VeQHNUJS3hj88eYDiawoo5E1lK51enbrMkyIaM+V6nC\\\\\\\/mwgyo4+ajr3TUxjfJjdb4UPo62W2IYwBQbe7wLnzSHbaojKY7fVf\\\\\\\/a6Tm6Zf8FHOUBy5V9iArWYR\\\\\\\/OKh61B776DuZOvtrlp3WwgdOafUoShOkq8nZd172Z4UIWDEnLSj3GMJycYMVqLjxgBcziti2G0tMgENfocqdMDzTEQ7t3K4qRMbx7Fv6HB9GKMhJHs+CXKgzD3QeXIDOe3Ekd4ECQ+LNjRQCW42zwgLFTEUplSCnSmhLzeJv2fzvrhF2nQJ3a43ZxbiU+K4z5YH7w6X\\\\\\\/jlIa\\\\\\\/6ayhOdA1yLXI40VUE94G1syBOsUFvnUGErqYF\\\\\\\/\\\\\\\/YPZ2FNimkvWHTuMoTI0Lc19ldzPqggTcDwFYeG+hxQRe9tmLh2xKV\\\\\\\/KkcYGtr8gd23bhjnWOgCTXZGLbg0ybnzJkq6ToSqOd29WO8bCCA9y7E3aL280hcXB6K1rAi8TNEQxSUS3XEg5MZbM5IU7U9TqzZ2ysXH7juSjxg7Ii3odJp+5b\\\\\\\/ULdXtl\\\\\\\/CHqa+VUok0pT4CT6\\\\\\\/coexOVxFEEQtLVj81rH4Fx+LNrkqFXUfqE7a5Lpzd8ac6ifx+pCRp\\\\\\\/Ocg\\\\\\\/LTwrTMEUOUkgSCMQVeH\\\\\\\/SuyscdK+dSLPRiedqHeS1n\\\\\\\/nLBwycBnd92Rz7hzLfNsog3zxLyRQNFDAiUrOWZSZ2dNRs97BnWRiwaUYurU+vBrT3BEHP7Tas7IKczViUkKniueDGcXIMGZG2vhf66tJfJumQYNHNcIOP1xTTl7EvWdSMQrmTCcO0a47CcJxbIrf0FaF4yrQw9kps+IcXL\\\\\\\/pplB8VPkiREQ82XyAx+kondSQd9tpmygbpcz4t4eROHWcn4Uq3q5QfsJ4xrJBfBLE9NKiZ8py2gqKAkWcnK+byQu6By7TktXtUGOdF6bie\\\\\\\/3aE6Oku2NUhIxGxj3Ahhr+qZmRmQs9D92QhqTg7SmqOKgBEJXHTuxJ4FE9AWtNNBPdRz+boaZ6L3jW3Ss1+vPKgM5dqJZerozRv5oSvCrCzkuPt+WnGMe+JcjQqsO39jH5tpjSSN8o8+m4u7+tOo5oS9K+JmXx+BV9b42MXzOt9rq6u19vZf0Bl+3q8zkjM3BOWD7NfdHRe0SWM9L645HJJYSf9+rHYYBgSjr1xixYIaB6ACmpZXZCMvokAaW03fS+qdxmhgo6Vj+1xq2YX7S7ISAmcWygkizVTMQlii1HunLttFa4tB9aFh2oBtDS7pf8OezOX+hg3htIR7PktAwWDyc9byPuTanHA09yGnE5JSTyHKMzCZyrBUmbQlBfKyWcjDWRgwEKUfk8OFh00e1YWWd36EhtJQpYHNuOloulkoB1ROPw6\\\\\\\/PXBRRlsRzx2g6ZNP9ZSI6ReB6X1kaR8q5OmnM8hoiOWEuwU32jR2Ty0PsSby3kcnWevNQTm1NmwWvbJXoK4sIqwb4VhdNbrA2zdkT0ap4AxTs6J713B664uh+E1Pq0HwPfX9wgxZmz0Qu0axbpXc0SD\\\\\\\/KK2k6XwTFYLkTGDbFEl4WyUKtP9FSfbxNZPtWeAQg3THcFzANqt0JOdbZff5UIXFwq1JDuX+pAcL5kU57kcJzhVwl\\\\\\\/ZnAjX4rEiKdByj3LJY4lIZSkmRAsLl2WxdX0tJw01jPbWZT3ZmnIpPwpKt3jMdqbW\\\\\\\/2lU1OdgdBUxpXuTfdakKbErl5EnGfu9zNyAmBNDl9RNi7Q3gAXaw+kmIYNiKVat\\\\\\\/3NhjrNkau1P4UkIquXwCVMjFnKL+sf0\\\\\\\/amCCVyfC2KXiy6rngTOVeiyrhQPFyWAkF1hg5rRbcl9HmoKDFYq2sDfu38iGuHAMpgB\\\\\\\/nPiqCtnqwtQHjb+ypd7Q\\\\\\\/aM75dMKq5CqGUatXENpimveAgLdFpJz3bRyvDueH5KN7GtPPnNCwJQyar\\\\\\\/sExbOERjMUTRZlvvhEmSLlA9kPmFZIcnE1iHCNRu0MDNzxNaMPV0f4uio\\\\\\\/na8jxSg\\\\\\\/ZOz3NvXjHJoG\\\\\\\/alTxh7vbx+TUF74f1Ss6l16TBNpKsZ2OHyAQaiZ5frAlOHM0driUpXUQPoX7LUpwvjuod2o\\\\\\\/ouzqfIFhUhsKbOnWr4isBqb6yhNlQKx4eCZrw8Unq8aJUsxJQp0L1vPa67jTT1RHnccnZ\\\\\\\/+navkJhp1CzyDYHrHRrBDFVuXRL8K735GKg4g2LzsijR\\\\\\\/q+e\\\\\\\/MVBmRwVAVdg+nLvQA0VKbySfUO7gPRGjEy4Fz\\\\\\\/j7vPOtG8TortwVoCvs35gJ4h60rlcCYhYsIhTVPEhUBMUYnuSuaZ+mzm1egnHhlwjFYIIOHfvqgPQ\\\\\\\/j4UHl4Y2gwndNqtTij4J8z8BptF1bdnzPuclcR92Gvkxqo0yXItbLFBtrh11+A63dKwiFAM59XIPzs5CsN0SviMYYiXqwvXwSZ8+BptcwDQp58ZM7XJWXDSK19ZY\\\\\\\/VeknCvzkxVCMRCmOCRKB5so4cklDSH7k7lujuck+lMqu4afK8bbKqVXq1H+NaN5KprnhyGYA4uNbwJhOiL9MZksR2vX2rN2fDh\\\\\\\/MtxtdTSC73IpG7V2PPEVARhnkaUoJ5KBicvHPJs3wr25c7hfP4g+QGW6Fjwa7EneOmOKbImAp2MMrf+lJfYkmhfuvNHLE2orUsZnnkRKGozXE1oRB52yhb3Eu6M6iBcYw3alLqJN+o4mx43Yq6NdxQaQo7vMfOdErKtXxzt+UinjQ0XLR3Kav6eLtHU5GZpp\\\\\\\/quI07LXKNh8j+JdWRkSee6S\\\\\\\/rTAHtN58OtV2NzyrVnJFnzssu36O\\\\\\\/iuhsFGDKge4pOYhSIUEJ9QhQGFqfHlKbDk049t2ZAHvAIoBEtnRctOTDId23kL4WYeIQX0Zt6UGdQ\\\\\\\/+LRxttpmk1UyqFvEmlNzTERr3GnUtpMNY9\\\\\\\/pvWv2DHR10QkcQoqBuvVhMlo6BawFGmI55eZuSaXGLYsgVnDduIHcE+PP3u7IMz0ShMXwU3Di8TV2tzfSPAsCBbRfqlGXaNCJ\\\\\\\/oKRb4BHA5lU1s2Bq+0fIv98cF9FTSss2U+Qo2K1QbTOPCoc5pTzu+a7nD2pXFVoGDC7ikU+a7LLTSJ5LPFwxTfqc2Txt+p6iJB7io9p+86vdsHiwbYrmCZkjc3KriW3Q1IaiHZXeTjUfJIdqQbcR7\\\\\\\/7EcqxKM5\\\\\\\/gypJgH6BKDy+l8NOlEE\\\\\\\/OU0AW9sY6Y94tl59xBREWl+HFcQspc7rq5eN\\\\\\\/iKwygx4TGarcjQjzHdAep9gYYkZQs8evhogPqyJc8OpRbGj\\\\\\\/ksNja\\\\\\\/nJyjBHe2PBMzpdxofXsxG4LMrxsbXMZjAuNSwM85UzKgKCxtR7xstij88eF9STBWUT8X+sAlkmkUsrPRwxBncfX8DAwALNRZDRRB1sCcACbepxlJG\\\\\\\/iJP9+UsO0BqMXjoncMW9cJGiGlCHk3fPevfSoB26T45+Ilrgw9dudpV2ciBM9drdzElLv8xRJ\\\\\\\/NrlF1IS1P1NbFO7+w672SzzykxLSBAV1UFr8ID9a397ohKY0UkAQrnmBciSVj7Wd9fOtdN5Gckhr5fJg9xZ7RMo1FMtU0C0bF\\\\\\\/WS5DLhvDrKb8r+yWSs6V9Y1uhFaBIlhjueofj3NZe+5CLaCRarNO6x75E2mDyG7KJvEefE44Nd5yOBngxn2sfJr9P7QmLolOORoHwbgWjyPTjTCmBz0ovFFHiwMWjYqOIUBit0gUeCdtWhzajPMMVC9u0bakdueko\\\\\\\/BeUjGvlmjYl+U6DsXoZsnVzEsapFRAb4XjMvvJsEWx34TG38IGvUXYr6M0CeZVV0eSDCyCEes1dXXhsfloItRfTeG0eBktzN6ZvPqIir+Rc\\\\\\\/9S\\\\\\\/+eP5T6drKmvrYcT69CwDIou3YPJPmpN9TveoQLd+lHboEyO003uK4LBhF5s08hygWjKclAjB364\\\\\\\/q9XpS6aWO9ZdRrg7g3kesINHaItwRbGlh8\\\\\\\/HTJPxi7rM4\\\\\\\/ckyiRcGfKNsxVt3LqJKUUBp9J4hv596J8RB0I6ojTkAACGjvnNrYkVH22b8rbPQZBRmYZ6NQeeKOZAKuRiUnZRh3ZgwxZs+lk1PMuC6z9yhaq6NYsUBUAOYZ6I6GDujr\\\\\\\/0ZhxQuOmNFn9NpweNdVv2IAUQx5S6apuihgX2Wra8tOuDuI+2ARwqwpsmvYBA4RICSUHw2T4of3wuEMoo3mSWUmi4Vu9sPqCquhts3PgUBhLdTb1\\\\\\\/hCLfhfkdWu8KZmcuhCWEDxovPbZKF873izs36sec+d1JnA7CbppG1CWJXnW6XqMQXO0onB26m9fkEGI7ietZnL0\\\\\\\/IPjXJTZ6oOvAN5jeqsgzNY1J4ocsP2wi9tGFN84VkRCWiIjbWPYT7qsR\\\\\\\/8ecuMu1k5XYk66gRdp3sUfc6lyO0HpSWiSEl5yyv7kZYz64MDweP7QfnMF2xnR4NRwvBLnUN8z8a6WVAmknbEATWhrj9HMZ+BlN2aR\\\\\\\/w\\\\\\\/zZoqUhhlg\\\\\\\/MjnKQyemr36EAuD\\\\\\\/zBM6EAWnNC1t\\\\\\\/4dItqEfxWe5C+6TRQjiUMKHrPo3Yi5BOOMzGRFb2jfadjd9BJr6RqaC\\\\\\\/BjcaoJgdaWX5zm7cC8oefUvRao24msE95Ky+UQXV+PxCCMQwDHlXMtm8qV2rrb68FXvr0mTPyrTg6RgYHr2ZKuV9X4icgHr0OeFk3e0cvQxLy4Q8eBO1x1OtRkSZ5WrCye3nPz59oa2ldw0lsDZZG2NiGqrCAdzArW01sTv130XwImVCvBaV8XHXuwreeI+sqrfq50XqGCSCSj2n2Z8mJfyWfveHz0zxwdzjBGUyyR7U22BpdZ7cCoNCBlsvW\\\\\\\/oFo3hEX+sW20H3BJPHzsd6DqgXj6PQ\\\\\\\/otRx\\\\\\\/l8OrUiRJ+m\\\\\\\/frT00f+XOcZsUN9n6+6E16RwN8JVTAGBWVySVGk0FZKrcRDEHwaopMEhbwUBfaI4+ls4riYoqVltER3fgttdp7j8zE3JXpNsZ1xFKyBRKLIZWWUe\\\\\\\/HSR5tuGTxSQ4BVkXuV8AY5TYFQcLV1iXuX9AVr\\\\\\\/21Lg93B4u2cmz79GPi5+sKEy82rzenn3kUpZG0BxUtmyJILJV2wk+\\\\\\\/ZFe5L8L8mMd+D+wKzK\\\\\\\/Vj3YT4A\\\\\\\/AZBU\\\\\\\/dhGfp9gx\\\\\\\/Ptt6XVxHYRkw8y8KRH1jDhlK1CfXfDUCdQbHMIljoHtCZaQGM4+1qFuWlJ1+qkBUPPczHbrxxfHi5LsQenWZvoa\\\\\\\/fQjmkYmH6QLy04UDRdWUBw\\\\\\\/+7DZmMF78yIR0fOsuk2Acfhbnu469fAhj0RKJUlp2hI87zHsWf+yc1S7ydeMsfopTC9b1Bvm4QcDf38566xqcVeg7xVyVnHD58eXcUoriAN7roUrLvJBAkVHSTPPFOUDcHR9m+FDVl8\\\\\\\/+Nd8Q0A9bghvA9jnEagtpVQieDMA2ibDovJC62vk4lfNZRNEfmBLwv4G5reYy+7OaU6j9O0kODPdkTaahXImDcqKsD\\\\\\\/UfH7RAqceKaOn16fckriuE2VyAtHYAM2wZ+HZdlwNZC7vDFK6XF\\\\\\\/9OOra+BHpRca8gbXF6XRZ8c5SVqN3lQFVCabJZUDERrM6YCR90Dsz5MyLwCOIaYiM2AsHkTT+e3wjkEiZuv5N45U6qgp75N+4n+dMJ\\\\\\\/SnBTAJX4ciHGlMDDXz6aUgTy7VFD7qeKBe31fnvi1kdRRbm+uYjK55wdfUFKcdrx20gKJbQqGzAmvHP1L\\\\\\\/u\\\\\\\/Y8lB8udjWYK9+Ph4UBDikdI33m8zFVHmIX1wo3GHE4smOiELeKTonc8XjlvWuiNs8A9okmk+gR3Y1vXKfeYKFvEsQfqbhQvO+d5SaGGmSSIoIeJzouu1cljIKpNTTi962scGsM4PfvuvwVIcHumq76torLdXP65MLPPnv0QzrxVWianQVRvrBevj+nIwYTUooTCubbPnGQiO5tyvJ68M5Wm++r4wntPHjRlpbulllYrjRbATy\\\\\\\/sUPVIHAz\\\\\\\/pGdDxI7iJzBnGav74PNccV+nwS9fmf\\\\\\\/nvVj72lJ2GHwSqfov2SsVbHnEI3h6tntXJFjxbb64EUuBvD79nJxxoYIaFqRiZ20gFtLcq7XBjcz1s9kFoqs3fhHDl7ALoZ8gk4SWZAcSHzpRz4riJw1HOxwxv8WqLHbWPx\\\\\\\/qIWhx9vhmqIHDPCNOZJSkRX3YxqBqgGYJ0iG6HOqPWEiCjTZKxIWYk8Ho7cFG6L0WvLgIqRuBUga2efLHa8qfK9HfpOnwcOpr3dp6NKyBhEOY1Kp5dvSbBS02KKKdNYdF4dfKP3m7Zd77aWMRgcDQhn9KvZnmqrhqsWyRDoxeV\\\\\\\/3AEk4NLQ7HBzmA1J1STw8GDrNUextuFIfEZnTs1v58GB7TunyOixHYF4m9zGgLp+raN7CIISIQhaQRx44CPOe3n4AxEDKvCAh0M0sERBFJvONVA2oOASuUheFw1QXkS3EFEguR0vuCjWKYTcCj5vY2yiPeXkEIyfx61DNKYg\\\\\\\/qluh+AipJVbQfl9gVv2ZY6YYRYz2+go9nB+R5c7mUbW2Dh1oMsYdJV793dD3Duvv1ZpUBqkP3HtWi7k7nXNHKe+gEEpQfkpG3O4ABS97PwJr2cF\\\\\\\/6+fX+67k49eCP7OSgHwQOs5JGSk\\\\\\\/XF+F5zz5yFbzbK86\\\\\\\/3eVf+wfJN4tf5XJOikRIi0KNDAEQ6QqIZrhDJiByzYXqBj2Bzn15tH0Hpy6Q49y\\\\\\\/pBaiSfM6fafJxWexuU4cyJQpprZf3N2y1qPgzzYjILGas4a++MAxrT1TvpWp2MQfurGVVNyihxYUXKO7RQ0JZt9ZAysfsDCGJoskDfut7PeHFNsfBwQB0USgYDq213Z3gCh2wcxlKZzDUXwxKu16Rxijaq1cVuWjkBf\\\\\\\/iLAiiLjZbQPgG101HlewPDOzVwYA5J3k9OOdMXHGNXrt07Y1bi0qSou815sFUQPDLC5BJXFwFUvzukKjtRnoOaU7rJQ1F0Pleatqc7GjonTj71mq9hdFwwahGSSX2SctJ6C9fjQezsScunoS80HL+dA0I+jbri62gWFWbyABjowT3yU6sICZXBgQfi4IXuMNLFJNocTRrH5e5Qdz9sfFP1rAr6CDyHIwjpn9xLv0P6Tn2sgjHH9kES4M4fpwv8AqwGCOM4Mu4ZaP1BqOzBJqZGOIPHNAhQdo3Y6Wg2iRDuJI4\\\\\\\/RziFZpDM76orDXz1OEU6ezh2AUSIyt17ujc43WsRNjuWtbis8gvLbhxOectVLU+qoVnAGSrlBVZPd\\\\\\\/PtNMkaJOaFQBE5KPVvNKwGL58FQf7jVusKyzj4SmBwTB4aULX3V28OpJh2VEUSMW3LZqi88N1KrOIoMtQa6u6x9q1VjhO7jihBpjg6\\\\\\\/zONjGMItNujPM8ugOVHVEJBkNztkiFPwrOBx9qk2h3pJSfHRwYNkoU0uDNovXy9w7iB4X1cnUcrdaZueIKpe70cxeg8T\\\\\\\/ivhifCzIytcY7QzoIvKJ+KvGQ98WWtke7AI5Qwr82j6KKnNtpq3qK1zgIk2U3r4gOdSqRHh6TL651\\\\\\\/VYGDannzF+P9NJJov7o+pVS0MOMQaZ2eFhZ5nh5dwD8vyNRqG1PRMFySxS838l1mrmFh01M7nPZ+uJXw92s2zLVzUQbR\\\\\\\/IT4d6F\\\\\\\/CHxyOrxnJBVgpU8d06F6rkJKj49BNVfxXTQKHkod6bIGvn4PvjOk2YpceNSFYnynKD5cbTiCyJnMf5H5XC\\\\\\\/IBL0U0dqOz1sK5+Eu1axn7Ysozsy+lYyoXBIY9nQLKRQk4I56jeE0k59RC9\\\\\\\/P3kWWMWs3Ht4xEHtCoVVUDaOr8Es4fd6EbLgcTBysIQObyNzUzTWzs9iZGaj\\\\\\\/jbOe31WuzGGgR\\\\\\\/N0v8eJQ\\\\\\\/ZSlXx1Ie7JpKCLUv\\\\\\\/ZZvAQuxj\\\\\\\/IImWJSJuRpmywH3DRM47BaYYMPPjzNmuQPCgQXGcsBjIqzE4ew2uJ\\\\\\\/ppJpg8GSCKDS0TwUecBtMSzyNVo1xCiYgD6N2B+lUoGNkRV6FnuLDkjRVOEbAW+9X20mnNrSlpGvAXk9N5Rah++xGgy3cWXboKnHK00a2fOg5gcdBm509Fw3bvgqlV\\\\\\\/axshgEi54M3dTWBk\\\\\\\/It+0Y\\\\\\\/hhGSa\\\\\\\/Mh9mFEIKKa124Hrepa30kaXdOCZfJH4BRYyPe2d\\\\\\\/kNKKYSPU7mu8p4xY39zcAx1kXzD0PCwwSEIgNhxMOdCwHJsaEFsd3+mQjUJTMkFmaDnrdeuamWxa8S0MZbAx2yBbdCx1Zpi0aCwuo2AkxZxlfylUk1fG1YoMtqPRs3vpNqHMuLQSL1t6LHagFMTniC2Dsa5w28tYEpw9oMdcCA37M6SMsNfpx1CCNmAn1CBsRW\\\\\\\/w0ZQdzT\\\\\\\/asFFkrF+YujDxXGuAZWLor5MsyVsEc53Z9dYgIVugTM8l0F\\\\\\\/8XN8DrYoGQNHSjTKk9mLwfApLBPxk5yRMHAXn7lL4hNzjZWRlZt0YNwsXbOGAp87KntqrcTnVGhZ64iQms14j5kccnXMZPL0PnxSjcJEI9DpDgmn9MVhFkaxAmQ+\\\\\\\/vLgp93ucQK+z8WP+MdOMXI3C3klBixPitbLn\\\\\\\/VXJoYJ\\\\\\\/lmGD9Hkiyxwlee7SFTL+M8\\\\\\\/BrRJ7+3LQi45yWJTTJQ3e6KlbuPGmo4Tv9PKOG3Zcx6Sz0PPiN07szN6h2YlGccmDe+LW\\\\\\\/NIXV9ZHrHs8AsZJlHVzU+dEoSEPSSkcg2OwT8FXpr5ntMmyOWvZg1ktbsShkGEy5CyYlnbJxLY1\\\\\\\/nv0vpjLI716dEiKpttN7Kg4nWL618PVBjGhXftwNkFhfV1SW8\\\\\\\/wWbAgpv4g6ivplt3SIq9EVgI3PQGLEObNYfI4bo1amUKWp9Z0h4g6rzwwi5IKqiH2SVzqAw11AGV9gV9LvoG0EGPVEn\\\\\\\/vJOXJJndrd72M\\\\\\\/rMo8UWGBxqym0Tdov\\\\\\\/qvDedUSYM4ch6Q65EKsyLpdHLmbHRk4PPFZAk0lbv6nPVf\\\\\\\/+\\\\\\\/4e+dw0lzo+iG19TNi2g0WUgF0ko0YMJg95Q55IW2JLjBimha0WNvePGrICAH1uBs7zUBSoOYESfc8jkJjzLQZMqHkuBR0\\\\\\\/\\\\\\\/a2f7lp9RmVYL1ohPhNjmP96WWesyKykVLx+mhfAsjetqvASp7bm7xquN76zB+2dyKFfhkrl0naMxmrr3zgCsyiTqXslqo+T+Ouvk6Y40wTGWfrq62GsofgLYaJOfQmer0A1l31ztOrTznz1ZKuO2SriaeVfTakypk4zN4k0jeDizoeiKJH3e8CMQ9mURQEqQxq74p1JJhE8KIa0zz\\\\\\\/8V\\\\\\\/UnxbvvqPrIoolVzbKoSmhVzadt7TgTE672PuombL8ZJSSRc3bBX3biVWdUWm1+sWio8Bz\\\\\\\/vYDgLqPFyI\\\\\\\/57N5DwqZaK\\\\\\\/v+t\\\\\\\/nU5Y5thynr+kf+k4LgeNGmh\\\\\\\/oaaeMUSspe\\\\\\\/1bnjLkJ\\\\\\\/rYhdbc3tw3nZFEYE2lI4qCyb6uMn1RjjwJMULpZrsEXeCIRLSCHU+CvzfKVLLyf4IANOhD2bOrtY1lY5kft1nQpukfd1eN4KKsvBrOvQz5Avde2ANgTuW9AOb6nBfIpiutV\\\\\\\/Pc9gnQcWAcfm3b\\\\\\\/fICc0074mH14pt80\\\\\\\/1ttRpffOI0e7gJueVMZkCI1+73VZBM+pgsqlRtAnt524gIC9SdYdB3DQMOXRGEUaOoO8NrtwT4+DKAIYGepmfYe0rxTJitpoi+f+QnNdoMu9OrOVgQQW8gRG5qSGu+HPlYavJg4f9Er\\\\\\\/1s7w6xzo5NBGdlT9TBoW04Lp1SqUYGQxtpzAEBVt9aGGjZY66eC23AK8U0WVMMDwWSr2V5SxU5bymI11b\\\\\\\/wVxfgFKIyvAEwjPk01C4jHb5qHlOWYg\\\\\\\/YDagkYQIMbnOYMUARS\\\\\\\/ccvHuaKDEZWGJrQIyWw8RPoQ95iUKdfdCw71BpwKaDr\\\\\\\/R+eVr9TISjUGafHfXxUeH8TSCrF8gO23qDf+mCcHczmQ9UHLFKP2p7AfEyArBdEPe8OKao60p5DVqtLAVX8g0vNgJPCdRrnxdI3p7W1Qg10SU4vIxAJRRsdpZ9+KCNysBx+5Ut6NGUJ2Y5d7wyepbqXxEC8czy56F+2FnxmZ3UWMNcD1vsvf7JdTWsiRgQHMcNOotA6+BPL6zz849Zpupx6V2pc0QQpIcAj+LJVogFQqdkvq49jE8v6TexLxai5YY8HWhT\\\\\\\/21OFNXBA\\\\\\\/I1sECqjbScy6JKQAlcux992ckHyyK0v2+RHm14QDJB4ub5KlzHnY5BfTEyvWXKn7iltW+4Un8Lv0QYzJfAjW\\\\\\\/XvvB6ER\\\\\\\/oMOEXJaUz2VkyOLxE7FRhi+4KL7uA4c48NH1BE0HzgLnET9AeEPoopdAdfx3uEs9EIk4Kx+fvxlUGQJPuGjefQD+lttyN3yby+i5ltRQGLfr7PVcr0uRj0qi1zMbUhCSVkOJFrWi78iwvU8v1gqr7NzUriq8ELcXIAhxcwhUJakj2xmKwjn3qDtZQA6mNAYTH4UriboOsqH8alhMA42S5rzWXMhlQzh8OLl4zkEXJ72qknGoJ\\\\\\\/Ws\\\\\\\/6xLYGmNaPnUDyU3jL4MgjT8d9iG2oo7\\\\\\\/3DZW+raBV\\\\\\\/fpUfyWk5w3klnyGP69tNHUC85s9paYjcVq3ShRSseDBYf+M5GLDM0\\\\\\\/zh4vpDgY5E5kDlPsMxOX2yWiprigWo6dDziMDNsNu19TpqbK9F72jQuG9ILGcspiuHiij+DKzleQ5VAWtxU7jHMoNMVA4QJ8NxUgPGUbhjYIXrOu3CRNbM5qqdoBMxw4Pv7FqjXnMyetz+Z9jIyk91nfijfMHfIq18dJNN7QzolH9rfyh+TL2hhwVXnR03X3Z5ebfn\\\\\\\/ZAXOkqNKuRZISb7X6k8ZFjJWgSH0frHWimluHfjt0igLOy3sNagYm\\\\\\\/M1AfSF8n8BsEnqSMUhfyuiO9mIEElw6sqDMlNr57vYdcUEBM1Mcbo+82LMN7t3XQNk68EGY7M2BZW7BIzqYlJc3RKgFgh4O\\\\\\\/BJu19GKHXOFo9xhL1mJSRK37LCfZyIXcq\\\\\\\/VPbx4bQ\\\\\\\/tX+68KgklOo4bUAC3NKjHLpUkIywAMzzUt3D2XQFD6cy87Wg9ICKBjysXozXXJdPbcIG7+1vLHCBDkUK40xhFZpFgi2MrrMf7Q5cxdrp9CntUPj83g8202i6Bd5pnfAudxSKqUy+urNgK0H7bUvEWzVK6hOkTbDHiFCan5ASY3mHu4T2lDWA7NZaAC1njhE7fa4zhr5ZpMmsbNqUciiwrBIgaKwbxouNGbF2ALw93hb\\\\\\\/ErGoGYODFd0ZScnRSphmPkCtEJ7TKUwwuczYTdFf4JvcSX16FebYN187qQxKANiRA3IXclnytO6nhKtspqzTTq\\\\\\\/gNr8KMVKUBmX5MIqZDXKER02fLA4PsPx98DNVRtKfMO0+T\\\\\\\/XzdJ+c5Wl1Vj574eqzAj8Ztk7hYIxU5kQyu0ybLDLI7EOnFTER1ty5+R2XhS9Kxg9H999lUgIWdP5RCOcs\\\\\\\/GeUdwE+glltC7iWabL1i5U+QIYZVqWDVWqp7p1ZxmQoytszJEDcx1LQmPPN+JvwfFclgeeVIf05nw8sVKaZIXEreL49U9LB75KmYvLlDuEhls1giQSq7wgTEPn7UNMt6eak95gp39ufPPqo1VKFBkKpY89a2jT92DRSOlS9mK3Zq6IgRLpZIv16+S83hRyTe0O4\\\\\\\/fzY37+y3QK32FpDKyhuPD9crefEgI2nt7EbGdRaZpwzqBMXcL1livQto2bTPdwmjrUeY9ekzKjzb490a6zhN+Tvkl1gyZTrljjRffXh0V7AuPUl\\\\\\\/t0jOkt\\\\\\\/VeEDpqlhyqsyOR6hq04BN03\\\\\\\/6wX+RmnpuX5OYiOPhQ2yQr0T3+HN7J8bCp96ck2Sg6yzzRy1plkdHRMLEYsSlogw7e6EtPo6u5ZHBG\\\\\\\/+DPXPdoZZ2JzGVRk\\\\\\\/r2cNde0EucqPjOmIx2fu3ge15S9DivefC7fT7yNmZGHRw7lSoFWguQPlI+fUSZiBZUh+tVU+ZCi8ylxwlERniQdvIlxzOVamrSLgtl42nPrOsCLpgxFcLP7tQ7JaGwtfqUNI9PwZ5TIl\\\\\\\/7Ws1Ggs6WVIQ+vk8HYV04\\\\\\\/KGucdjTJpBd8LsfSlN\\\\\\\/wePfhz8UcP507BdUjyabVHhWp6g2J7DwBRRoG\\\\\\\/Gfj5tYvEPk3CypGsR\\\\\\\/T824En\\\\\\\/WxYLpNVDVzxT1vpkQRth6V\\\\\\\/1j6wR8vBYScuJhzxtDzQSPhlxkjBz02Gt2NOe0u\\\\\\\/iSJtWV0Mb1tczn+mfNV64wrH7mb731D5tAcZoURZZ0KDmrhX6jX0igfMsjcjJcyxvAF4LOtJXiVH2GdOOvOEM+3f9+A0Jj+IO9iVwodIgjM0O0r4IoAu6qyMShsWWrJgW0mV\\\\\\\/r6e7tmWTp991sR3KMl8L4+ebvfrY6TDEyuU1BxYDZ+dEbN42zJjylYMwCTWaCDQCCFMCFnd3KCEKxxXug18vCbnjd+Voqt6xmeOPW\\\\\\\/uHhhSRSkn0mCxvOACFYoSvo2x5wqVODKmqDg+efMe0lTcrdERCY\\\\\\\/gyWdwvEcTwoP28FLDByPmmdOEJ6coL7MMxG5ZbHumyeXp00ZYvWfAwzzhQQaD1PM2lARFBWT0lRLxBqlek2tY+7uhVhxdfOQw\\\\\\\/hVH9X9e7lGbNiAFcbDT74ExZl+yoFj1iuFO8ur5SY0T8mytYEnUktWeh3VP1MboiIsaxw1npChkq9Ftv1a5esOyOaCch\\\\\\\/xUsSaJ1nbzqJSALl8Vh9kU2yvITP8DGQAYY9u39Hi+mskDP42wfoMbu1YoaOAqs65bbTeveFJWRROBkNlTxOh7jJZRaLqMLLtmmU8Y76M7YJ1btrRqpWNRX4SEPQcHXE78nhnhSOA8ojY1k42WtOvXHPkPkERd+WEQm8c9PUeLKCMubB6UAaIlfVxENp3AL8l6ySrgJqxzDT\\\\\\\/D9cDME6sWsEbvJecYgUQEsUBVgrbj6bjnP4kNBgESiY7IDY7dcB3qmY9NmUz6aJGq88BoOam\\\\\\\/ipz0MzB5By+SdC0FZOQYZABR3xMYZmUwtaBKJhIQenORoaPxZMzfwRX8XgBcmXE5vn7t208QAHOLOYNKggIwaQzMo\\\\\\\/aIOQWc9dyq0bRG6c\\\\\\\/Z4Um3OvAA6VyjpCLps3l1Mye8leYLnpfLpNcJejTDe3ou6h\\\\\\\/MuknD+L\\\\\\\/Hlu1Ap8CTWcLVWtAfJACCyldOxRIZfvLXtf9ZjTVbZ+p1PHClcIkbHNuCWPYn2w+meVz+eM3FC4LTYxFkKvaaWPUYf6zBybb8A6lUFLTVEKcENUzrVVTYFdPy4b9DD1Wk6EWoczSjjoCyoD1wyZRBXc9S+5TSiNZlAfe46a3iTWowrKgr7eWwlBtBB0dyoY\\\\\\\/ADlYDfKPHbGk1gLtB9tbThQfvaEA+FGNAmau+HABTeSldqiBPfuhhEm4KjY\\\\\\\/kKAn5\\\\\\\/AwqqrMvOjV+Z6L3t3SvzkBDZblFfMiHjbvx0vcnv1M7cz1XWFd62jPSWlRs4xvJ+iA59X+dstvdeP\\\\\\\/pRnoGz3igfZ2GfF5cfx9KCTIv2HHF4kdQadOAnZMuHLOkc9vn5Zk+D\\\\\\\/lSSx5fjjFM+kRd6ebx6TM26GmkoMtByrOK7jsSzUIHJLMyp9Srg4xX88X2idJrHrlmIqgoLampZSXthyW6uR6EEScoJkjsguuX6OkEzmxK+Eb0pO3qnZ75gB4P0uAR\\\\\\\/uo2nfpwWc1aGjMyHmNmvb7R7luIwWsh3DnsmWUrYSVzIfCxkPHRtEdr1\\\\\\\/cUWMHOzwXvo1HE9uzNryqn\\\\\\\/anrK80TpM90GTQPNs1ntpN8LUC13eNYpQO3sTOItviFKlP3Yo55fsVDvnmIk0VJFkJkFSNkOfGwewCdTwI4EbObLNL5bTQjDxPP+F8zZe0UC7Gu7xxFUmnF1OJMisEK8gxqXG0fLaoPOBUnNr8XZRuKCdiC6lVQV1hGjVlBSdKNdHy+mh8CxP25XRccrDpS6Vz2JE6UdR0UAqFXt4Xu\\\\\\\/ePzdQ0O087kqj7I7RhfcQcwq7PbBQ1ZD3IF0VMqGceoVkWw4apvxj6TcooSFGbrLdm2k6SWCSCy5LxDfRRnJ6ynium94btr+uF+Jrb4iomvIDhZuSEX07FAD3n1GwotUe0K60UQf+Yez+5QSPhkGbDu7hVj0G26enGO1nq2T63WvbENvpakWXqmOs4Dx\\\\\\\/yZY5Qac7TG+gi24s+1\\\\\\\/zGqzX6UgxkABoli3ApQ73snZ\\\\\\\/Gtki3ukHnCOYT1N3GSENRoMPY0Z9FrHfBdL2s1fxA25I4fFLvWlTtT4bNHwWP3BoxoWI7Z3smDB1TfMvOb0EY8UKyXyl6uD3Q8LDJeLtIUKS+C2thA9TDXqSARPLltw2m9yobJWj2XAkdldR\\\\\\\/xaONx29TSwVs5JFZK\\\\\\\/CCWzStz27BufsFNH61CpEnc9hQvNXM0P\\\\\\\/6PGfQmaKx5Nf\\\\\\\/EX9CRPOUQmth732cjWAxpkno9EV6D5PJHTm9vQIWqo8\\\\\\\/zcXQJupC1hpyjvC85ttniTdbx\\\\\\\/UMjiE9VfRLGWH8389dbJVoD\\\\\\\/y6tiuHCeWhS0df3PMXQHF7FlLR3rD5a6GeXxmmT9Q80qyGcbFeP6Myyd1XdaGkb1fFRQOrDm7FeKm6cpF0H\\\\\\\/JzJ+QC9qCIqy6x9TZCkOdUaEMVHhf5bk0kLfAPJ\\\\\\\/Susk5o6SG73v0UtA8RvKlqIQvG\\\\\\\/5ma4iSCdtJ4Y1AFxyEekK8ZB4SQ6N1GeoszfcWReZVDGValCRg330yCGYEhqZxK3Ppp+TXEM8f880bQnbaZDk3xfN7HhDNdK1zOII4VzcY2pplx0zKRiLBNTxhEWw\\\\\\\/7Tv+qPrBLdTVWGZYaSTIIXkPXCG4ePKvoOwgFQlFOrUpKM0gfSc4nhHyVkKBnjlRWQbVP+OaIO\\\\\\\/T\\\\\\\/R4R0E2K9QiHAslYLqicMN+xffOBMEKn5dkY6TTjyXXSYo\\\\\\\/9uisMjL2jxQIAPET1d1AjkvWFb2QB210\\\\\\\/eMoroSA99jGK8Yi4Kz9+H9zEnW5owZaJEtlO0goyWZ9T15HRtH+qpmaETRX1CwFo6U7DXL5S85g8DbAWsOTYNQVSOpdyh2WRkXk2CUXwJ5Tj5VIThAzJbGzOeYgDnZmYT906UHPO6Z1OpLp+YSOY3Xd\\\\\\\/vx5VSYFTAHsWQinCftT0anaxAyoC8YZm\\\\\\\/qeCKdWF+hLEPsAOGEg\\\\\\\/QNoL8RSjGKNkHio+qmRsBJvUeFOD8pTYtu75WmuOxCsCnEYUe3YzpGclPH3ieL2FupxKSXcKyq6b62R+jEuCxIJbFn4joYpqep3zziumPssWWVkP7ut+Ut+\\\\\\\/ncFdqcNXUE5CRITAeemcaFq3Bin7nT6bvIymNwVpBXWgf4nCpEKP96ib7kssO1EcU6hRTSgq3R3nCtF9oqlXWnM7vYbGfv999iyfhf7TeYIaYRffdwvE+tKKaDhmQY8d5E4306M\\\\\\\/vYxcYPTMSUmYbsJI\\\\\\\/RBquHh0Qr9VwtJM5wlRA2MrIXlK0ZD3ZtvIeERHCADIQIDYucbY36522vuSym0\\\\\\\/o\\\\\\\/wvFAV46AkfvghpPqXFCLZckNwn\\\\\\\/Imjae8jN2gqjZtnNzWJKWxo2NHsaAG2Dn0gxT8PMTntgVgeCpnrp0gAFL\\\\\\\/Ubf8dM5IWoBXOdIN7v032kRAD5yoOtXZ0VS14OXEDP1Lie58zE6o\\\\\\\/ZEvm2xLAdrpPRU3YPEmHIAd4HUlDzqXkxutdvlKMphkAyLpSP0d57pmwbcPY+ZONjeWIMv4gFXLWus2gRX2CikdvBjYw78ntG4bZ\\\\\\\/UHv2ztmwCt54bnoaJTk6ed1p3bBiPKumYrd\\\\\\\/YrBmIUFxf6hSpy1uqKxyCn1NZ7uH0p5SGM2yCkOBxORnUWWCmnRHJVkdg1vC0xTMyXNcLj2YWQsugVQ\\\\\\\/ezKAzeWZAVi924rSoZeepxVl1Gu8cAJYs6bjASo9XVgmeZ8192hpZK5o8NLnDo\\\\\\\/KZQgLTZmul4wykq7N0fon9AvV7wFMb+a7Z\\\\\\\/mLDxIx4\\\\\\\/cNFkCeHeyMpEvUSMaogHWQ2eRCQ00KFWAYVrjyHlz3GJ3alGeo+oxdDpNqQkwKJBH7fjlTE92IE3g3o48h79JGXTmun+Z21tF86YSPaWLhCvGJmGX3CzyE+n292aOzhmrgWpN5cci1fTawjWh65+1nxCxN0pzniM5C9PewPwWzwXcrgBUIdH1YwKTQP9r6Jx41agbSnlcIx\\\\\\\/xTP2d3uOto8OHAdsuuxFNilRJaScK7yYIUlzqlGUFroONrm0OVft1tYm5KjC8eIUg17jdaSi2VWRYgN2tEEQQ6w\\\\\\\/0tBfhcz1aHuKjUhV8Yi3asZT4gPteU8\\\\\\\/vCetFRq6Zhe\\\\\\\/BYp8Lyrs7D0G3AzsnaWs0p7CHx74Q54D2oMUpw2Q5t2BVFI4BqyLIM7575t6BxfaQ4JY6F\\\\\\\/JZbdovO3Fshf\\\\\\\/djq2q+7DvSrO8zGJLQEEFd024PNuc4BLGJwvkEhA\\\\\\\/HZBXiNHz58ZYVRmaSMKYuej0g7kyte6j68DR3N8dZljfWg+gz4ScIJssnRPnSaIOmqapxpR9OC8AvZzp6+f1\\\\\\\/luelKXKt4oJ7f6Z2cZlxsfl2PaRMoqIfbX9sHkqizRETfSu0\\\\\\\/uwM60ukdI4j8JOBsou0G7663G8qGS\\\\\\\/5bHX40KvxwP+8Qq5WJteetpe3fu5MdLOjjrOPnGyAEkLUNqQdDo2UJThytKZzATW7IN5INA+oMei5RZDW13P+nzsi2PEJdVpkIDmch2oTwqrOY\\\\\\\/ZUw1T0biJaVQ5i0dBPc3XATcZ6jEzGdMiOQtTC3ODCdeNiGckG3PFyzdNW8b2aZIrlgVEyjj+HyX8Ysn6h+689903zyCzeXvjH0gDf3W4\\\\\\\/reTsxYadwcJO1OUz5fQvCYtmXTpMBMWPfslt8r18uYjm09E\\\\\\\/oyWE0yPFqfW5QcCpXciU0ySV3wRLCrScwfoOgB2Cat0XhdzSFMnA2M\\\\\\\/LLWcok+hQiU9j+7Y07opSGOfoaPpLVSWAdQSNvGc4tCYaiXTHNTtaQXepGfugbCrYGCZRAxR47zig4FJbK9wPI2Z49SC73kPcyxSGwX6mixThFCwqFWXp9XtIwDlrwp1U7lI3LkH+hC9\\\\\\\/06FxMReR8IewkxpGQA8EdcT4d4mcjKK3r\\\\\\\/L3+rENNkMhE70NVyJNKRU0Z5XKafBTqO3OPevKGCJthhy5ABWrcA2MXUZhIsUuxaKhVRaFw+QOyihKGWz4a+IGiFxq\\\\\\\/pUUr8sn0R8bzePyW8QOZtNuT50swdJaxkUZx4ouQVYlSY5uAJV83LaSHwMDYeZuF9EF9s6ZtaNB3wrfkzNJf2Qzrw0CYX1QSNdlGpfs1x6RmmXoKB\\\\\\\/BRbpx0iw4craNMIgjby6d\\\\\\\/+LhuEKaSfkYKaKWMPguJY6ikZ+e8xWZiXEHPc8Qt2aUJ8rUa7EfFcBjYeG7r4wgYPOOzZrTA\\\\\\\/dGH5rhOcClseyVQ4dn8zK\\\\\\\/eKSSk59LwxWon8WSc7XmzbF5OmWIzacHgPrlmuhVHjEQUfoiOYrJPULQm2nLEzKLcgn5djCO9tVJHuvUNE6rqsJVicaHObp6LnFqTY\\\\\\\/wRqmBVVHsUHmfidv37fnPKGgBz1WuWeADvoObYpuQWr1HifqyNVfNFKZaAe7PFQB5MxecsmXhbLf7gOaoswZId+nKFiaDx7Se7uE0vcN\\\\\\\/5y\\\\\\\/eoZTntW\\\\\\\/JKuo0xc6xwXKFlLmC+WhhBeA6Qri4\\\\\\\/ogwDD\\\\\\\/bs58LySmpfsHVuKhG3SONqlJh2kjei50Lkv7cn7jNrZIfWnIHxp9GoqeV4NM0WVDdLrlj80l3LwNj554J6Vf\\\\\\\/\\\\\\\/Dk7bSquElr\\\\\\\/+x8VHKwvAZDs1bAH2w7JNeGFMlmbCobd8BMRtBrgjEXTDhQYCcXkb\\\\\\\/f9tiWkc07CAwCk7Tw9tSf7HsIWN2+Lc9oobSOnqKlBFpdTnLK5RwH2TDZv8Ft4BKVM\\\\\\\/Vnk+o+1sR\\\\\\\/E2aN9ZmuU2QKYfmnjNN0I6E0f3+Taj9LPP8ciPSrKcaGGYK8OJ4yaZ9jC9vU7MDNYL4Aoyt\\\\\\\/itpTlc+k+Gm0JpAA+HGDuR7ias5BOjsblRx7QweAVeZ31\\\\\\\/ypOyLNdlF4b7xWHCkGjf\\\\\\\/OohNG62ch04EcvoMMneuCXoJESCxtMOYVF5wNmGHlxEOVM6h\\\\\\\/D0ZJLcG0kTAbNHQSMfDmgYCHAP4uU8OZdq1jxllRfONu83Y89b4UE\\\\\\\/iZnU5WrLdGaKijMQsWyaFjnVGJogsLTEMJNfP5\\\\\\\/s5zlkVl8U+JiadWD5i5cUkbZqFEmhD3DFmXB\\\\\\\/DyArakvC55qDxT+dr2ifU9vIeGLEq\\\\\\\/B7Eo26RSAKF+kz8tM\\\\\\\/Y6fw46XPiyyz90E1b5g7t+JT7iXTTlexIP69PAlv1uR\\\\\\\/HLsyCYsSaa5kkUyCBISsALU5XKjCljaxtzWp3fLp0JyPRAb6oI\\\\\\\/Fpp83xbdreaC2E0QHai06CyjKzdS+AjPaRrOuVfpyzDncXKM4fZ09XojKPZOYmztFKas7Mm5tTVg6Ku9pTiMaNx3DZB9h2ocCVbxzXwThr8k9uu013\\\\\\\/qoACBw1wWrBXniXaraVRm\\\\\\\/5LSCpH6nXpy47ixD305k6LULmTU4HyClJUSE1Q27t9c8WGhNJUsYGFD8K4VZG3jconmCEUnM5xVTdjvOJTDkrURXEfyVUU0SnkPRGNZ03UPAMvBLnOuH7RTXjoIViLIeUcGaiujx\\\\\\\/BIfItyvqQ7A42IypGTuQWyatwV4exXCUf38+4wsoFB1Hv5cej4+A02ZpPGja7arEoCD+C7vKMNJA0aHWLtvyK0NsG7WgQ6IQGaZmpJGYQjtp9BNhEbDsO0b2c8yUzGHvUwbmuXHVl8Tumbys9mtowI2GPREkWF23WNaplnFZ2wVkxWpiNuQgNGftGXwyOhcAQtfDzKYqsKTM\\\\\\\/QpqJ\\\\\\\/aLqYXZ0KJnL0YjzgclcR8\\\\\\\/ViTpsPLz4C5YZztOWybN\\\\\\\/F0kPVMfNQfqjnDWFMq4TV+tG4u+pKYJH2TXGttRh7SPRiBX\\\\\\\/NwgBnDYLFmhZoaiB1UJe1p3c8IDPOW+oU+fX1WV9cDRuK9F95An4qLapE8XEaA+Jy1UxFs7dOJ\\\\\\\/cHCtyqG5egT7lqTyQCmLxoYSWWIufc3lSz6rUq+qI43a02WpGadq9mdu+gXSWmmCTRf0Iul3L2p\\\\\\\/K2ECCW2PPZzH2iIcCEj4\\\\\\\/fRJA7td5vp9R1sveyQosmDRA0h6FimXa2xiWeeCDSqLStWaw4yWcD2qrNSNl2NTZTV39zaP85WGbO6OCRgwuVBUHQCelL6AA5JIFyzh1qO1yFTyrhfs6jM6PtFKMBJwe9uzqp+RnuRKMPviq0Yx+XqU5cm5ZpuecIg5Cv5MsdV1nvTrPwS1e5nCRG1B9GL8ECEoSQi0iISwtpSJJ4BgUzefiLad1mXVQ7BGJYkBE4wUMGHBFxcZNiBmSqogc\\\\\\\/cCyIPzOGGjV8l1ttZpCqBC8ZP00s6F7CvQ3R1bEYBNnchGTevMts8gnNwtMBtRYf52JqO5yCAthQbPIZlx0PaxBDeSZHPtcup5grNHwBLywGJ2napTiZRAw3v9LNGLXdg5RrrOzciBiJgqdC2YVL1DSPp8\\\\\\\/dyrFBexiSxhU1KAGIIrRM9CW74qRrRkHk3+oE6KvkKW68D0Jf\\\\\\\/TDD299hnlI4qcrYgeL9GYyfwyQwdBoODNiRqOManwFrX2d+Z2wmvTFxT\\\\\\\/o5x8pH6kiy2wT1CosRhf7qHSxO6ZuWGmUJvUT6wLel67J3ccruGV3szqGxypj6\\\\\\\/faNFHT+gP8o3oiVOaN0\\\\\\\/Xy\\\\\\\/R0UeytT1oJqArqSQIXzuL7g04WFIjekUy29VhHMaIWYkIgGG\\\\\\\/2e18StRLss+e1b24O3Itqdr61YkCBv9HrumvYam\\\\\\\/EgcJErmJZk8bsImkmDAi9Ux9cuXZgpHK\\\\\\\/e7Z3+YzmSMefKDtu8MLGr7R7SA\\\\\\\/VgKfKuq9t11jCZLn75v5GcfSa4GlX2t1sz9k6xSddm2NNWy9uqXT6TAllyAHkdjoCOCl0DTaUp\\\\\\\/okuOOy9stTAJ1Su7Y967w3lTmWa3656pExIO45GiGzqQRyFIs\\\\\\\/YQz3adHL0nxPTsAmDd+cONhi1cFpnX9rnsBDGrZsjzsh+pFh9fAB5wqfeWtKuO9b7sIHPKoT0hF0GN3E28eiQkFfe98WHBK\\\\\\\/fcF\\\\\\\/BSnmkqSh+4GdWRh0steJCeAWg0EvDtF4zo5Ch6CtHUrVCk\\\\\\\/LMYSW05xRhwhnbAoRsZkbcpXsArscD\\\\\\\/2Bc5us06LiDqnJpeZpGhizqtiDogi+RzBuoB+YSyhItNp\\\\\\\/miSz1tgJerl943S+hDz1HsBCNqvdhoc2GNAduJrZT6b4th\\\\\\\/1\\\\\\\/InwuEokaZ47lLd3ah0aRBpFdShFhzoXkB0d+MYHx\\\\\\\/RkjKEH0oCbubWY0luU6eYUhPDu8I4l+nkHS7LGedZa5KYpGdSnh+83x030hB+lAx0sQxKs4ZbI0GLAB04kVMeY\\\\\\\/eskZiyY8sy6Gb8wQLFg\\\\\\\/EUldTHkoxOrXxUtgbrkR6Yub7Fv8uf0xpc19\\\\\\\/wYihuXvTi8\\\\\\\/PO5VSj9cXcZrYvK0ya1IOyqP67poVAD4pLz\\\\\\\/BlgDdmHEQy4chaKnO795tfCjVjmd+zAlNid154pnmI6b3ikXvp9wTJsU1ciSkfN4xl5qEb3GZll7tTHCj0JMODr7XyIoQzQ5MyKYf+bQMlfNv3XKD8U4ayrTlSETtn7XYU+VYC+i0tLKYl6FaNzp0xR8Rc02hz8btaDRAAR3x9l9NYh2AMSeoRo3EHnrXHJAik3CZSQalXfCUs7aDOkEGYMXwSaSXEsnMsax7Nzl53oD8BM93512av3nv5VeyOf+leRGQ1BzOZIfbAwGGghDrExMlzQnn+Ad26zXOZ7smOdM7dS16JnKwY\\\\\\\/hoRVogKRKSoDhRXCiyNPRgigebcHVZnd+XaFNyAw3pW45SHMkCEDY1L0gLt6VkmGT2eZVdOmQJPWBEKikZMzNUb7H19ztPq+S94fkLQKc8iwHF2Duw34IUC3TLSxmJclufDAqNUw9cZVvY43nT9XysOvi\\\\\\\/CgOTCOtBp3oYnvxnqUhWU4X9Cg2jUm8agrg3mBtIPcqE5AwVwz63s7ZoOraZmsF1Rp1ju610XLiRlUUpUmfSuZgZQm5E1woXBvGoxO88J7Ep0mlNW9GoodaLnYYmqDgIQQgEtWf\\\\\\\/5dU5iwBKTm\\\\\\\/IKXxo\\\\\\\/h5iGHuZ6rh2Jlib5aqGa0D1sGdRqf496nr9s6OhAo5CNm0KPROi1qmARmWLPfkzpntGUhVDH\\\\\\\/NIVl9g4bPRwsB3i3u6mYg8ZqxJAcdn4bhBHabmNK1goI7iLAOOy42d8GJBuG0m90bkhlbREWQat4\\\\\\\/PNIyFaDp0fIa2gDwl2yx1dxyoeFaErzB+L2quaNQER6gsotmTpj12cD\\\\\\\/SEiIE87XFsPGITFDOpJ4CyAmNuhr8GWsF3Ykl3FqKfX9vCKeulD8t2J2D4\\\\\\\/iUNlubF5Y3yshxsU6X6g1n4r5Ry\\\\\\\/uexnKYCdQUfZTR1ltcIGJBMV622VLhm7nqmrHs4GCrZD5RypVVWd1Gjm5UzfUrXuF4uMLZB4qLBmAPR1eOE5WDLsyPyAP2Hdqjo5nR3gqRVrf+NfJ7sXtvWkO+TG\\\\\\\/HWR0+xjA7bC+wBeZ\\\\\\\/o7Q7KDLRziYqV\\\\\\\/LjvpeTMbXPMu\\\\\\\/siI+Izfi16s52UXR+1fiEnFyIvN00Msi7mvJokSensB6w4wIW7WI+GYWp0z9+7eIk1ER7jUgu4Vi3icG8QLMx2WaPYkdbp8UTlVNIZ2a5waqCSBTeTpTp2\\\\\\\/KzXyB9qlvbRLCwuW2tkRTr4UY\\\\\\\/2WFpzS2h08PP7isr\\\\\\\/0UjQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00222zIC37wtkeVkKx0LTBHJeg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b5b652a8245708684a6781a08546bf29c9d0e1a210f1e1a18aa961c52833741f\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022m2XbY58xnwUGUxUcAAbqb1bleVgBuNZO2kKppuqAWRhxSt4h\\\\\\\/CBgK+wAitZ3saAsUm5qiVTxQSnBBakexiEmT6yaFue6td\\\\\\\/RF0sEbvOhHr+Rn1Qsxige1lfZLTZeC4tKkn2C4gasljxMzPIDdcoropn2Pm14r1AIfudeFq8sC\\\\\\\/Px0fYGSLNp5+ZBeLD72yF2UpEALcZsao4EC2UF3tB\\\\\\\/h7Fol16\\\\\\\/HFPr3QXEJ7sHure4zeYZLewStWAyIhvqNmEwJDODtmRKicl1MbydpUfXzej\\\\\\\/onohrg9\\\\\\\/msbCLnCuuY2\\\\\\\/zPfYiBoYTYBMbGtEQYbjrmn8ZqR2I6PRsV50rAGzYLY0t4Gym79agAu555tjElj7sv\\\\\\\/X0UbsDVPMP2enWZ8Ua2YxA0n3OWRNk2N6\\\\\\\/dD7Klpf\\\\\\\/9NakzyM+W91\\\\\\\/MBQYyT1IUF0aanKSKqhQd3QtUf4mxtcrsqYg28lu5ieUMgiR5T+PR5qqHmiUuNQt3wK0pKsbMyW6E4YyqqpKdKnlLjCGjqdWBNTw737B8B2zDENfZlxZGjIv+DCg\\\\\\\/VyhquGwyZ4Yup2IxXSxddCF5YxsxWeRGh8uyLiVMbXpv6ol\\\\\\\/MpApW2Pr5mbh4fDxeHzAfqYGtgtLpvFX+Ro+8eK8OJtPPwCcW3m5mypVGt7u3W263UmRVh7lDH3amzhf0WAnb23vEYlv7RdhHjkXeuY07p5KEKzLrIkoTO7SRHStMdXBnPDxIFNCppHvetg5J6LAnhVzUdwoxSHFeaqoTNNO5Aw\\\\\\\/K5dJoQ5SZfX04Nm3slcC2L+I7DFuVvfd9GohX5BrXMI9nLAX7UDbY\\\\\\\/LyEx1VgK1xztKCPlc5FnyyKUAudIjNlJqyctsjTKMC4fglIR\\\\\\\/Ys9gJg\\\\\\\/4lcOK52eiTNcvPwXfUWAd2fWnRzao8R4Ukq4SNNulyxGO+eG2pokB00H6Bu4oa0qmsjAtTGxvq6FsulFaVXe9yvfQPcjlX\\\\\\\/XpFk5o6nAGRL2P9QsYz6ZIpAAlLUxplzqtLl3RCQcDsqx87E5jTjOwk6F6wrp3TfUV0iTBlvn\\\\\\\/ZIC5vCEneKuXaNmmXBVW659HOLJwrmIyP\\\\\\\/4MzY7qkOczReAtlV7ic7a8A7LdjNQuqsQsmTZyRsEy1+39wkGOnnL3hMInEXV4AwdIxUosWKnccsyOKI5f\\\\\\\/6PmtDsDSQRIEPmPUReq25u+7aXF7htQK+UJljjPGSSa+SJrkjXdqyHxBW9Q6Juj+wFrF2mCPNTgqulkEo5nDutRK9NR7bnHpM4GvFWYKR\\\\\\\/GLOVmtL5iGMx3g4Zl4Znd6hTHe0KqOkKMTmhe\\\\\\\/LZIZvcy19dpERh\\\\\\\/gaTKn14MqzVW3P+Vq+XgTdAbmk2EAKRPqHuu\\\\\\\/1aRTaddjDpRLFp26\\\\\\\/6KzRgVIqT1znL104Jvsf3nO4Z7SqRJON9mTNhCpbYuj0eGQR7zhciNUQP6sNrEtpGcS83\\\\\\\/050xnUCfICKTd4YeN6E0MAaWVfWcaxpLTnHuiH+Tl4Xj\\\\\\\/bmtQVEwdCN9x4xrh3N9QqZzAAdioFjaj7o7ZZXMYgFpHo4HI3FR+yBENHGOByoG27WEFpEn9Ua1KjD2F9zX9tr8MaMSTsJ1nAR1pO0oL7qLOtHXnemnseBY+RW\\\\\\\/y1VSHbqhNjBLZYMFLjHiKw3tDWfwOl3acvm3e3ZIDfeDgXYGkQqVbuYJUujumlAxbm\\\\\\\/UuLyq9c8eSNHkCLkp8cu8v\\\\\\\/\\\\\\\/REddHPwH2nMGiYR2ToEmfw+ANVoi6VySSyx5YHoqcnF5kh5Dbv7fFRoM9gH32S71tOw1ubosAGJ2tSo2UcR70H2ffAWR2RqrjTXscK4kfw2C1cDP9nOOWHw9zSl1b49FaF\\\\\\\/Z+T0Mphwv24Y\\\\\\\/y+B3poqlADbaJ9hhOycQr\\\\\\\/EOzR6CDdnQKcWtULvBCVkZEO5jYTr7zwbOi3JZii6aXglZceIgGmnqAdO9lwQEm0\\\\\\\/miPGuStidMOU+5vdsnsnYxUNprecWztKCBpdlOdmCVx\\\\\\\/PyvxK0BDN3FEH667msFDFwOAHPg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022rLQ7uhwHxDBSGqrq9Akzng==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002298457bb15af338408d55581790443ce5f14571a3e0461a15b9802ce4cc027f62\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459307 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/8/891" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2zjrLFNkJAGXTnVvKpkd6PGN1KKg2X-91UfKJjH2s6E" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/8/891" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XPAkIqJQIhpTLO835G12IdhOFImg33XkaqF-u4ZBvNg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/8/883" title="Alignment-Free Population Genomics: An Efficient Estimator of Sequence Diversity" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/8/905" title="Resampling QTL Effects in the QTL Sign Test Leads to Incongruous Sensitivity to Variance in Effect Size" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459307" id="node459307" data-pisa="ggg;2/8/891" data-pisa-master="ggg;2/8/891" data-apath="/ggg/2/8/891.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Incidence of Genome Structure, DNA Asymmetry, and Cell Physiology on T-DNA Integration in Chromosomes of the Phytopathogenic Fungus <em>Leptosphaeria maculans</em></h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Salim Bourras</span>, <span class="highwire-citation-author" data-delta="1">Michel Meyer</span>, <span class="highwire-citation-author" data-delta="2">Jonathan Grandaubert</span>, <span class="highwire-citation-author" data-delta="3">Nicolas Lapalu</span>, <span class="highwire-citation-author" data-delta="4">Isabelle Fudal</span>, <span class="highwire-citation-author" data-delta="5">Juliette Linglin</span>, <span class="highwire-citation-author" data-delta="6">Benedicte Ollivier</span>, <span class="highwire-citation-author" data-delta="7">Françoise Blaise</span>, <span class="highwire-citation-author" data-delta="8">Marie-Hélène Balesdent</span> and <span class="highwire-citation-author" data-delta="9">Thierry Rouxel</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">891-904;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002048 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459307" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Salim Bourras</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Salim%2BBourras" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bourras%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASalim%2BBourras" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michel Meyer</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michel%2BMeyer" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meyer%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichel%2BMeyer" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jonathan Grandaubert</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BGrandaubert" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grandaubert%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BGrandaubert" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nicolas Lapalu</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, Research Unit 1290 BIOGER, F-78026 Versailles Cedex, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BLapalu" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lapalu%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicolas%2BLapalu" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Isabelle Fudal</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Isabelle%2BFudal" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fudal%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIsabelle%2BFudal" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Juliette Linglin</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juliette%2BLinglin" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Linglin%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuliette%2BLinglin" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Benedicte Ollivier</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benedicte%2BOllivier" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ollivier%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABenedicte%2BOllivier" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Françoise Blaise</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fran%C3%A7oise%2BBlaise" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blaise%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFran%25C3%25A7oise%2BBlaise" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Marie-Hélène Balesdent</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-H%C3%A9l%C3%A8ne%2BBalesdent" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Balesdent%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie-H%25C3%25A9l%25C3%25A8ne%2BBalesdent" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Thierry Rouxel</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thierry%2BRouxel" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rouxel%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThierry%2BRouxel" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rouxel@versailles.inra.fr" class="" data-icon-position="" data-hide-link-title="0">rouxel@versailles.inra.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/8/891" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459307" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459307/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/8/891.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459307" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459307/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/8/891.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459307" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459307/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/8/891.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459307" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459307/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The ever-increasing generation of sequence data is accompanied by unsatisfactory functional annotation, and complex genomes, such as those of plants and filamentous fungi, show a large number of genes with no predicted or known function. For functional annotation of unknown or hypothetical genes, the production of collections of mutants using <em>Agrobacterium tumefaciens</em>–mediated transformation (ATMT) associated with genotyping and phenotyping has gained wide acceptance. ATMT is also widely used to identify pathogenicity determinants in pathogenic fungi. A systematic analysis of T-DNA borders was performed in an ATMT-mutagenized collection of the phytopathogenic fungus <em>Leptosphaeria maculans</em> to evaluate the features of T-DNA integration in its particular transposable element-rich compartmentalized genome. A total of 318 T-DNA tags were recovered and analyzed for biases in chromosome and genic compartments, existence of CG/AT skews at the insertion site, and occurrence of microhomologies between the T-DNA left border (LB) and the target sequence. Functional annotation of targeted genes was done using the Gene Ontology annotation. The T-DNA integration mainly targeted gene-rich, transcriptionally active regions, and it favored biological processes consistent with the physiological status of a germinating spore. T-DNA integration was strongly biased toward regulatory regions, and mainly promoters. Consistent with the T-DNA intranuclear-targeting model, the density of T-DNA insertion correlated with CG skew near the transcription initiation site. The existence of microhomologies between promoter sequences and the T-DNA LB flanking sequence was also consistent with T-DNA integration to host DNA mediated by homologous recombination based on the microhomology-mediated end-joining pathway.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/t-dna" class="hw-term hw-article-keyword hw-article-keyword-t-dna" rel="nofollow">T-DNA</a></li><li class="kwd"><a href="/keyword/mutagenesis" class="hw-term hw-article-keyword hw-article-keyword-mutagenesis" rel="nofollow">mutagenesis</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/leptosphaeria-maculans" class="hw-term hw-article-keyword hw-article-keyword-leptosphaeria-maculans" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Leptosphaeria maculans</em></a></li><li class="kwd"><a href="/keyword/genome-structure" class="hw-term hw-article-keyword hw-article-keyword-genome-structure" rel="nofollow">genome structure</a></li><li class="kwd"><a href="/keyword/gene-ontology" class="hw-term hw-article-keyword hw-article-keyword-gene-ontology" rel="nofollow">Gene Ontology</a></li></ul><p id="p-3">The first eukaryotic (and fungal genome) to be sequenced was that of the budding yeast <em>Saccharomyces cerevisiae</em> (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Goffeau <em>et al.</em> 1996</a>). Since then, an ever-expanding number of fungal genomes has been made available, and the genome sequence of more than 300 isolates from more than 150 fungal species is currently available or in progress (<a href="http://cfgp.riceblast.snu.ac.kr/main.php">http://cfgp.riceblast.snu.ac.kr/main.php</a>; <a href="http://fungalgenomes.org/wiki/Fungal_Genome_Links">http://fungalgenomes.org/wiki/Fungal_Genome_Links</a>), with prospects for more fungal genome sequencing, such as the 1000 fungal genome initiative (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Grigoriev <em>et al.</em> 2011</a>). Whereas high-throughput approaches, such as transcriptomics, proteomics, and comparative genomics between related species, have proved useful in eukaryotic genome annotation to predict the correct gene structure, functional annotation lags behind, and complex genomes, such as those of plants and filamentous fungi, show a large number of genes with no predicted or known function [<em>e.g.</em>, Arabidopsis Genome Initiative (2000)]. The dramatic increase in whole-genome sequencing is thus accompanied by a dramatic difficulty to reach the full biological value of the sequenced genomes with accurate identification of the protein-coding genes, as well as the nature of the functional protein products. In yeast and in some model plants, such as <em>Arabidopsis thaliana</em>, this was partly achieved with the involvement of a wide community, which promoted the development of strain/line collections in which virtually every protein-coding gene in the genome was modified, for example, by deleting, tagging with green fluorescent protein (GFP), or engineering for overexpression (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Jones <em>et al.</em> 2008</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Huh <em>et al.</em> 2003</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Winzeler <em>et al.</em> 1999</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alonso <em>et al.</em> 2003</a>). Even with this wide involvement of the research community, <em>ca</em>. 1000 of 5796 (17%) of protein-encoding genes in yeast and <em>ca</em>. one third of <em>A. thaliana</em> proteins still lack a functional annotation to date (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Pena-Castillo and Hughes 2007</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kourmpetis <em>et al.</em> 2011</a>).</p><p id="p-4">Although the production of large collections of mutants with disrupted or inactivated genes associated with genotyping and phenotyping has gained wide acceptance for functional annotation of unknown or hypothetical genes, this has only been developed in a few tractable model plant or fungal species, mainly using <em>Agrobacterium tumefaciens</em>–mediated transformation (ATMT) (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Alonso <em>et al.</em> 2003</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Michielse <em>et al.</em> 2005</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Krishnan <em>et al.</em> 2009</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Thole <em>et al.</em> 2010</a>). Furthermore, the whole-genome investigation for T-DNA tag distribution and the biases linked with integration conditioning the possibility to reach saturation mutagenesis has only been investigated for very few model plant species [<em>i.e.</em>, <em>A. thaliana</em>, rice, and <em>Brachypodium distachyon</em> (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Alonso <em>et al.</em> 2003</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Krishnan <em>et al.</em> 2009</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Thole <em>et al.</em> 2010</a>)], and only one phytopathogenic filamentous fungus, <em>Magnaporthe oryzae</em> (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Choi <em>et al.</em> 2007</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Meng <em>et al.</em> 2007</a>).</p><p id="p-5">Filamentous fungi, including <em>M. oryzae</em>, were first believed to have compact genomes with very few repeated elements and repeat-rich genomic regions. Filamentous fungi are amenable to ATMT (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Michielse <em>et al.</em> 2005</a>). Furthermore, biases linked with T-DNA integration in the genome of <em>M. oryzae</em> were indicated to be lower than in plants and the T-DNA integration was suggested to be “more canonical” than in plants (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Choi <em>et al.</em> 2007</a>). However, the sequencing of numerous fungal species indicates an extreme diversity of genomic complexity, genome size, and genomic landscapes, ranging from those fungi with compact genomes to fungi where massive transposable element (TE) invasion took place, eventually resulting in genome sizes larger than that of <em>A. thaliana</em> (<em>e.g.</em>, <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Spanu <em>et al.</em> 2010</a>). Filamentous fungi with complex genomes also are characterized by compartmentalized, “two-speed” genomes in which specific compartments of the genome, usually TE-rich, are also enriched in genes involved in niche adaptation, such as pathogenicity effectors in phytopathogenic fungi. Examples of this encompass dispensable ‘B’ chromosomes of <em>Fusarium</em> (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Ma <em>et al.</em> 2010</a>), TE-rich regions of the powdery mildew fungi (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Spanu <em>et al.</em> 2010</a>), and AT-rich isochores comprising one third of the genome of <em>Leptosphaeria maculans</em> (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Rouxel <em>et al.</em> 2011</a>). By comparison with <em>M. oryzae</em>, there is only preliminary information on how T-DNA integration will happen in such genomes and how it will be useable for saturation mutagenesis of genes involved in niche adaptation. For example, in the case of the fungal pathogen of oilseed rape, <em>L. maculans</em>, analysis of 135 T-DNA integration events at a time when the genome sequence was not available indicated T-DNA preferentially integrated as a single copy in gene-rich regions of the fungal genome, but not in AT isochores (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Blaise <em>et al.</em> 2007</a>). The low frequency of T-DNA tags corresponding to known or putative protein-coding genes (19.3%) also suggested a probable bias toward intergenic and/or regulatory regions (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Blaise <em>et al.</em> 2007</a>). However, as underlined by some authors (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Meng <em>et al.</em> 2007</a>), the absence of genome sequence for <em>L. maculans</em> limited the analyses that could be performed (<em>i.e.</em>, on favored targets for T-DNA integration), and conclusions about the possible bias toward promoter regions drawn by <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Blaise <em>et al.</em> (2007)</a> could not be substantiated.</p><p id="p-6">The objective of this article was to further evaluate the suitability of ATMT for random saturation mutagenesis in the compartmentalized fungal genome of <em>L. maculans</em> and to further establish the mechanism of T-DNA integration in fungal genomes, taking advantage of an increased collection of T-DNA–mutagenized isolates and availability of the <em>L. maculans</em> genome sequence. One of the main questions to be addressed regarded the accessibility of AT isochores of the genome and thus of genes involved in pathogenicity by the T-DNA. A total of 400 T-DNA tags were generated, and their pattern of integration in the genome was investigated in terms of chromosomal biases, distribution within chromosomes, distribution within protein-coding regions, and targeted motifs. In addition, a Gene Ontology (GO) annotation was done and compared with that of the whole genome to identify possible insertion biases due to the physiological stage of the germinating conidia at the time of ATMT process.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Transformation of <em>L. maculans</em> germinating conidia</h3><p id="p-7">All <em>L. maculans</em> transformants were issued from the <em>A. tumefaciens</em>–mediated transformation (ATMT) of the reference isolate v23.1.3 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Attard <em>et al.</em> 2002</a>) sequenced by Genoscope (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Rouxel <em>et al.</em> 2011</a>). Generation of the collection was described by <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Blaise <em>et al.</em> (2007)</a> and increased for this study to <em>ca</em>. 5000 T-DNA–tagged lines. Briefly, ATMT was performed on germinating conidia using the <em>A. tumefaciens</em> strain C58pGV2260 harboring the binary vector pBBH. The vector contains a hygromycin B resistance gene (<em>hph</em>) under the control of the <em>Aspergillus nidulans</em> gpdA promoter (<a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Blaise <em>et al.</em> 2007</a>).</p></div><div id="sec-3" class="subsection"><h3>Definition of genome compartment for insertion of T-DNA tags</h3><p id="p-8"><em>L. maculans</em> chromosomes were first compartmentalized based on nucleic acids composition in AT-rich and GC-equilibrated isochores as described in <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Rouxel <em>et al.</em> (2011)</a>.</p><p id="p-9">Following the automated annotation of the genome (<a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Rouxel <em>et al.</em> 2011</a>) and regardless of the isochore structure of the genome, we generated four gene-centered collections of sequences: (i) coding regions, defined as sequences from start to stop codons, and further subdivided to introns and exons; (ii) gene-promoter regions; (iii) terminator regions; and (iv) intergenic regions, defined as the remaining genomic sequences. Regulatory regions, and mainly promoters, are often ill-defined in fungi. For this reason, when analyzing T-DNA tag position relative to genes in the <em>M. oryzae</em> genome, <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Meng <em>et al.</em> (2007)</a> defined three sets of 500 bp, 750 bp, and 1000 bp for 5′ promoter regions and downstream regulatory regions. However, the increase in size of regulatory regions did not drastically change the features associated with the T-DNA–favored target (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Meng <em>et al.</em> 2007</a>). In addition, the genome of <em>L. maculans</em> is more compact than that of <em>M. oryzae</em> in GC isochores [<em>e.g.</em>, median size of intergenic regions in the case of head-to-tail ORFs is 670 bp (<a id="xref-ref-56-5" class="xref-bibr" href="#ref-56">Rouxel <em>et al.</em> 2011</a>)]. We thus only investigated here one range of size for promoters and terminators: 500 bp upstream of gene start codons or downstream of gene stop codons, respectively. Collections of gene-promoter, terminator, and intergenic regions were extracted using a Python script, departing from gene coordinates.</p></div><div id="sec-4" class="subsection"><h3>Recovery of T-DNA–flanking sequences and analysis of T-DNA–targeted genes</h3><p id="p-10">T-DNA–flanking sequences were recovered from genomic DNA by thermal asymmetric interlaced-PCR (tail-PCR) and PCR-walking techniques as described in <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Liu <em>et al.</em> (1995)</a> and <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Balzergue <em>et al.</em> (2001)</a>, respectively. Sequencing was performed on PCR products using a Beckman Coulter CEQ 8000 automated sequencer (Beckman Coulter, Fullerton, CA, USA) according to the manufacturer's instructions. All sequences were cured manually and aligned to <em>L. maculans</em> genome sequence using BLASTn with a cutoff e-value of 1e−10. The latter step was automated using a homemade script in Python. The position of an insertion site was defined as the position of the first aligned nucleotide to a flanking sequence. All extracted positions were mapped and plotted on the <em>L. maculans</em>–assembled genome using homemade scripts in Python and R. Based on mapping of T-DNA insertion sites, genes with a T-DNA tag in their promoter, terminator, or coding region were extracted, mapped, and analyzed for size, compositional, and structural features. The latter step was automated using homemade scripts in Python.</p></div><div id="sec-5" class="subsection"><h3>Functional annotation using GO</h3><p id="p-11">GO annotations of <em>L. maculans</em>–predicted genes were done with Blast2GO (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Götz <em>et al.</em> 2008</a>) as described in <a id="xref-ref-56-6" class="xref-bibr" href="#ref-56">Rouxel <em>et al.</em> (2011)</a>. The NCBI “NR” database (October 16, 2009, release version) was queried with all predicted genes using BLAST algorithm version 2.2.21 on the URGI high-throughput computing cluster (128 Intel Xeon E5450). All genes were mapped according to GO, GeneInfo, Gene2accession, and PIR data, and then analyzed with Blast2GO, which applies GO annotations from BLAST search results. This process takes into account sequence similarity and the evidence code (EC) associated with GO annotations. Finally, GO annotations were enriched using Annex and Interproscan data. In this work, we chose to use the “biological process” vocabulary for functional annotation and comparison between T-DNA–tagged genes and all genes of the genome, because this GO vocabulary was found to better fit fungal behavior when described from a physiological or phenotypical point of view. In addition, it is the vocabulary for which the highest annotation number was obtained in yeast (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Christie <em>et al.</em> 2009</a>).</p></div><div id="sec-6" class="subsection"><h3>Statistical analyses</h3><p id="p-12">Biases were assessed by calculating the standardized residuals between observed and expected values as follows: SR = (Observed – Expected) / √Expected. SR calculation allows the detection of outlying observations [<em>i.e.</em>, those that appear to deviate from other members of the sample in which they occur (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Grubbs 1969</a>)]. In general, SR &gt; 0 means the observed value is greater than expected, and by contrast, SR &lt; 0 means the observed value is smaller than expected. To test whether the outlying observations deviate significantly from what is expected, the SR distribution following a normal distribution was estimated using the Kolmogorov-Smirnov test embedded in XLSTAT statistical analysis software version 2009.6.02 (with default parameters). Therefore, when the hypothesis of normal distribution was not rejected, SR exceeding the absolute value of 1.96 was considered a bias (<em>i.e.</em>, significantly deviant from the rest of the data).</p><p id="p-13">The Monte Carlo test on contingency tables was used as an alternative to assess biases of T-DNA tags mapping. This nonparametric test based on simulations assesses the independence between rows and columns. Then, when coupled with Fisher's exact test, it determines whether the difference between the observed and the theoretical values is significant. All calculations were performed using the appropriate XLSTAT function with default parameters.</p><p id="p-14">The linear regression option of XLSTAT was used to model the relationships between data sets. A graphical output comprising the regression line and the 95% confidence intervals area was generated using the embedded function of the software.</p></div></div><div class="section results" id="sec-7"><h2 class="">RESULTS</h2><div id="sec-8" class="subsection"><h3>Generation of the repertoire of T-DNA–flanking sequences</h3><p id="p-15">A subset of the collection of 5000 transformants of <em>L. maculans</em> obtained by ATMT was selected for sequencing the T-DNA insertion borders. Four-hundred sequences were obtained. Of these, 40 T-DNA–flanking sequences were generated by PCR walking (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Balzergue <em>et al.</em> 2001</a>) and 360 by tail-PCR (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Liu <em>et al.</em> 1995</a>). BLASTn searches against the <em>L. maculans</em> genome indicated 33 sequences (8.25%) had no BLAST hit (sequences too short for the BLASTn algorithm and sequences corresponding to the bacterial vector). The remaining 367 sequences were filtered for ambiguous BLAST hits (poor homology below the cutoff e-value of 1e−10), resulting in the final repertoire of 318 flanking sequences corresponding to single-locus T-DNA integration events in unique transformants. Of these, 217 sequences were obtained by sequencing the left border (LB) of the T-DNA insertion, and 101 by sequencing its right border (RB).</p></div><div id="sec-9" class="subsection"><h3>Compartmentalization of the genome and T-DNA integrations</h3><p id="p-16">The <em>L. maculans</em> genome is compartmentalized into two distinct genomic landscapes: GC isochores (summing up 64% of the genome and containing 95% of the genes) and AT isochores (summing up 36% of the genome and 5% of the genes, but mainly consisting of mosaics of inactivated and truncated TEs) (<a id="xref-ref-56-7" class="xref-bibr" href="#ref-56">Rouxel <em>et al.</em> 2011</a>). The T-DNA insertions were graphically coincident to GC isochores in an almost systematic fashion (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), and 96.9% of T-DNA tags were mapped to GC isochores <em>vs.</em> only 3.1% that were mapped to AT isochores. AT isochores are further subdivided into three categories: telomeres (representing <em>ca</em>. 2% of the genome assembly); large-sized isochores (13–325 kb, representing <em>ca</em>. 31% of the genome assembly) corresponding to complex mosaics of TEs; and mid-sized isochores (1–13 kb, summing up <em>ca</em>. 3% of the genome assembly) generally corresponding to the insertion of a single TE within a GC isochore (<a id="xref-ref-56-8" class="xref-bibr" href="#ref-56">Rouxel <em>et al.</em> 2011</a>). The above-mentioned depletion of T-DNA integration in AT isochores was mainly due to a very low amount of integrations in large AT isochores with only two cases observed (0.6%), whereas 1.6% of the tags were found in telomeres that may contain active genes, including numerous copies of a RecQ telomere-linked helicase (<a id="xref-ref-56-9" class="xref-bibr" href="#ref-56">Rouxel <em>et al.</em> 2011</a>). No T-DNA tag was coincident with multiple loci in the genome, even in AT-rich compartments and telomeres, due to sequence divergence generated by RIP acting on repeated copies of TEs. Actually, even when mapping to a given TE family, the tag sequence, when unambiguous, was always sufficient to derive a single locus due to these sequence polymorphisms.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/891/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A schematic representation of occurrence of T-DNA insertion events along four L. maculans chromosomes. For each chromosome, the upper plot shows the location of the T-DNA integration events, and the lower plot schematizes variations in GC content along the chromosome, defining AT-rich and GC-equilibrated isochores. The average GC percentage of the chromosome is indicated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-502616278" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A schematic representation of occurrence of T-DNA insertion events along four &lt;em&gt;L. maculans&lt;/em&gt; chromosomes. For each chromosome, the upper plot shows the location of the T-DNA integration events, and the lower plot schematizes variations in GC content along the chromosome, defining AT-rich and GC-equilibrated isochores. The average GC percentage of the chromosome is indicated.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/891/F1.medium.gif" width="440" height="178"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/891/F1.medium.gif" width="440" height="178"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/891/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/891/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461706" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-17" class="first-child">A schematic representation of occurrence of T-DNA insertion events along four <em>L. maculans</em> chromosomes. For each chromosome, the upper plot shows the location of the T-DNA integration events, and the lower plot schematizes variations in GC content along the chromosome, defining AT-rich and GC-equilibrated isochores. The average GC percentage of the chromosome is indicated.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">In addition to these two distinct compartments, the rDNA array summing up 1.7% of the genome assembly (<a id="xref-ref-56-10" class="xref-bibr" href="#ref-56">Rouxel <em>et al.</em> 2011</a>) was also underrepresented with no T-DNA tags targeting it.</p></div><div id="sec-10" class="subsection"><h3>Chromosomal features and T-DNA integrations</h3><p id="p-19">The number of T-DNA insertions per chromosome was then compared with seven chromosomal features (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>), and the distribution of T-DNA insertions was plotted against each of these features. Globally, the number of T-DNA integrations was linearly correlated with all investigated features, but it better correlated with size of the GC isochores within chromosomes (<em>R<sup>2</sup></em> = 0.703) than with total size of the chromosome (<em>R<sup>2</sup></em> = 0.573). The favored insertion sites were transcriptional regions (<em>R<sup>2</sup></em> = 0.720), mainly regulatory regions (<em>R<sup>2</sup></em> = 0.736) and introns (<em>R<sup>2</sup></em> = 0.705) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Most chromosomes, except chromosomes 5, 10, and to a lesser extent, 1, showed such a linear correlation between the number of T-DNA integrations and chromosomal features (see below).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/891/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlation between the number of T-DNA integrations and chromosomal features. The features investigated for each chromosome were (A) chromosome size; (B) total size of the GC isochores; (C) total size of the transcriptional regions [defined as the sum of regulatory sequences (promoter + terminator) and gene-coding sequences (exons + introns)]; (D) total size of the regulatory regions (defined as the sum of promoter and terminator sequences); (E) total size of gene-coding regions (defined as the sum of exonic and intronic sequences); (F) total size of the exonic sequences; and (G) total size of the intronic sequences. Regression curves and the 95% confidence intervals are plotted in continuous and discontinuous lines, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-502616278" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Correlation between the number of T-DNA integrations and chromosomal features. The features investigated for each chromosome were (A) chromosome size; (B) total size of the GC isochores; (C) total size of the transcriptional regions [defined as the sum of regulatory sequences (promoter + terminator) and gene-coding sequences (exons + introns)]; (D) total size of the regulatory regions (defined as the sum of promoter and terminator sequences); (E) total size of gene-coding regions (defined as the sum of exonic and intronic sequences); (F) total size of the exonic sequences; and (G) total size of the intronic sequences. Regression curves and the 95% confidence intervals are plotted in continuous and discontinuous lines, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/891/F2.medium.gif" width="340" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/891/F2.medium.gif" width="340" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/891/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/891/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461802" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-20" class="first-child">Correlation between the number of T-DNA integrations and chromosomal features. The features investigated for each chromosome were (A) chromosome size; (B) total size of the GC isochores; (C) total size of the transcriptional regions [defined as the sum of regulatory sequences (promoter + terminator) and gene-coding sequences (exons + introns)]; (D) total size of the regulatory regions (defined as the sum of promoter and terminator sequences); (E) total size of gene-coding regions (defined as the sum of exonic and intronic sequences); (F) total size of the exonic sequences; and (G) total size of the intronic sequences. Regression curves and the 95% confidence intervals are plotted in continuous and discontinuous lines, respectively.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Favored T-DNA insertion events in genic regions</h3><p id="p-21">Noticing that the chromosomal distribution of T-DNA tags was correlated to the size of gene-regulatory and gene-coding regions within chromosomes, we studied to what extent compartmentalization features are involved in profiling whole-genome T-DNA insertion occurrence. In contrast to what is described in <em>M. oryzae</em> (<a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">Meng <em>et al.</em> 2007</a>; <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Choi <em>et al.</em> 2007</a>), targeting of ORF was not significantly different in <em>L. maculans</em> to what would be expected under the hypothesis of random integration in the genome (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Biases assessment using the SR method showed that T-DNA insertions were less common than expected in intergenic regions (SR = −6.67) and exons (SR = −2.04) and more common in gene regulatory regions (SR = 12.92) and gene introns (SR = 4.24) (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Biases in favor of regulatory regions were corroborated by the Monte-Carlo test. However, no significant bias was observed in intergenic regions and introns according to this analysis (data not shown).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461754/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461754/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Distribution of T-DNA insertion events within <em>L. maculans</em> genomic regions</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Promoter features favoring T-DNA targeting</h3><p id="p-25">Because promoters are the main genomic regions in which T-DNA integration occurred, we analyzed further promoter regions to investigate the involvement of host-DNA asymmetry and T-DNA–host-DNA shared microhomologies to favor the T-DNA targeting. Previous studies noticed an increased CG skew around transcription start site in <em>A. thaliana</em> and other eukaryotes (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Tatarinova <em>et al.</em> 2003</a>). The targeted promoter sequences were thus analyzed for DNA asymmetry by calculating CG and AT skews. Positive CG and AT skew values indicate an overabundance of C and A residues, respectively, whereas negative CG and AT skew values indicate an overabundance of G and T residues, respectively. Sequences 500 bp upstream the transcription initiation start codon of 122 T-DNA–targeted genes harboring a T-DNA tag in their promoter region were first extracted, and then CG and AT skews were calculated, plotted, and compared with the density of T-DNA integration events in the same intervals (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The T-DNA tag density and CG skew increased gradually when getting closer from the start codon, to reach their maximum value at positions −113 and −50 respectively, and then decreased dramatically (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>), whereas CG and AT skews plotted differently but shared the same peak position at −50. To corroborate the functional meaning of CG skew peak in promoters targeted by T-DNA, we compared it with the CG skew profile of whole <em>L. maculans</em> promoters following the extraction of sequences 500 bp upstream of the transcription initiation start codon for all 12,469 <em>L. maculans</em>–predicted genes. Comparison of AT/CG skews between promoters of both collections showed that, in both cases, CG skews reached their peak value at position −50. By contrast, AT skew peak in whole-genome promoters profile plotted at position −25, closer to gene start (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/891/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The link between CG skew and AT skew in gene promoter regions and favored T-DNA integration events. A. Density of T-DNA insertions in promoter regions (green curve), CG skew (red curve) and AT skew (blue curve) variations along T-DNA-targeted gene promoter regions, as a function of location from the ATG. B. Comparison of CG skew (red curve) and AT skew (blue curve) variations between promoter regions of T-DNA&#x2013;targeted genes (plain lines) and promoters of all L. maculans predicted genes (dotted lines)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-502616278" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The link between CG skew and AT skew in gene promoter regions and favored T-DNA integration events. A. Density of T-DNA insertions in promoter regions (green curve), CG skew (red curve) and AT skew (blue curve) variations along T-DNA-targeted gene promoter regions, as a function of location from the ATG. B. Comparison of CG skew (red curve) and AT skew (blue curve) variations between promoter regions of T-DNA&#x2013;targeted genes (plain lines) and promoters of all &lt;em&gt;L. maculans&lt;/em&gt; predicted genes (dotted lines).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/891/F3.medium.gif" width="341" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/891/F3.medium.gif" width="341" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/891/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/891/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461674" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-26" class="first-child">The link between CG skew and AT skew in gene promoter regions and favored T-DNA integration events. A. Density of T-DNA insertions in promoter regions (green curve), CG skew (red curve) and AT skew (blue curve) variations along T-DNA-targeted gene promoter regions, as a function of location from the ATG. B. Comparison of CG skew (red curve) and AT skew (blue curve) variations between promoter regions of T-DNA–targeted genes (plain lines) and promoters of all <em>L. maculans</em> predicted genes (dotted lines).</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">To assess to what extent DNA asymmetry impacted T-DNA integration, the occurrence of CG and AT skews contexts at T-DNA insertion sites were also analyzed in gene terminator, gene coding and intergenic regions. Sequences 200 bp upstream and downstream the 318 insertion sites were extracted and split out into three groups: (i) gene terminator regions (78 sequences), (ii) gene coding regions (119 sequences) and (iii) intergenic regions (40 sequences). For comparison purposes, the 122 sequences corresponding to gene promoter regions were added.</p><p id="p-28">Skew graphics showed that T-DNA insertions occurred preferentially in increased CG skew context, in all genomic compartments (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>) and also in a weak AT skewed context for promoter, terminator and gene coding regions. In contrast, AT skew was increased at the insertion sites in intergenic regions (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/891/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of CG (A) and AT skews (B) at T-DNA insertion sites in four targeted compartments of the genome. Sequences 200 bp upstream and downstream of the integration sites were extracted and CG/AT skews were calculated. The sequences were then grouped according to four compartments of the genome: promoter (red curves), terminator (green curves), intergenic (blue curves), and protein coding (black curves) regions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-502616278" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Analysis of CG (A) and AT skews (B) at T-DNA insertion sites in four targeted compartments of the genome. Sequences 200 bp upstream and downstream of the integration sites were extracted and CG/AT skews were calculated. The sequences were then grouped according to four compartments of the genome: promoter (red curves), terminator (green curves), intergenic (blue curves), and protein coding (black curves) regions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/891/F4.medium.gif" width="307" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/891/F4.medium.gif" width="307" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/891/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/891/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461700" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-29" class="first-child">Analysis of CG (A) and AT skews (B) at T-DNA insertion sites in four targeted compartments of the genome. Sequences 200 bp upstream and downstream of the integration sites were extracted and CG/AT skews were calculated. The sequences were then grouped according to four compartments of the genome: promoter (red curves), terminator (green curves), intergenic (blue curves), and protein coding (black curves) regions.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Microhomologies between the T-DNA left border and T-DNA preinsertion sites</h3><p id="p-30">T-DNA integration to host DNA is mediated by two major mechanisms: nonhomologous recombination (NHR) and homologous recombination (HR) via the T-DNA LB [for review, see <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Tzfira <em>et al.</em> (2004)</a> and <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Citovsky <em>et al.</em> (2007)</a>]. The former is in fact a HR-like mechanism relying on the microhomology-mediated end-joining (MMEJ) pathway and its property to use 5–25 bp microhomologous sequences to anneal and join free single-stranded DNA ends [for review, see <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">McVey and Lee (2008)</a>]. We thus investigated whether microhomologies between the 25-bp T-DNA LB and host DNA could be found at the insertion site.</p><p id="p-31">Sequences 25-bp backward of the insertion sites were extracted and chosen so that (i) they correspond to sequences upstream of a junction between T-DNA LB and host DNA; (ii) they were exempt from potential filler DNA at T-DNA–host-DNA junction; and (iii) they were exempt from gaps and undetermined nucleotides (N) in the current version of genome assembly (<a id="xref-ref-56-11" class="xref-bibr" href="#ref-56">Rouxel <em>et al.</em> 2011</a>). A total of 160 25-bp sequences was thus obtained. We also divided the T-DNA LB into 5-bp-long successive sequences that we named “microhomology motifs” and aligned the 21 resulting motifs to the 160 selected preinsertion sites (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Nineteen putative microhomology motifs were found in 69 locations, distributed unequally among 41 preinsertion sites (25.6%) (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). No single motif was common to all sequences, but 3 microhomology motifs (14.2%) TTGGC (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>, alignments 22–27), ATATA (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>, alignments 45–53), and TATAT (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>, alignments 54–62), were found in 32.4% of the locations, suggesting the presence of homology islands. In addition, TATA-containing motifs were the most represented (32.4%) (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>, alignments 42–65).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/891/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The search for microhomology between the host-DNA and T-DNA left border. One hundred and sixty 25-bp preinsertion sites were investigated for occurrence of 5-bp-long consecutive motifs corresponding to identical motifs in the T-DNA left border. The 41 sequences of preinsertion sites that show identity with consecutive, 1-bp sliding window, and 5-bp-long motifs are displayed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-502616278" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The search for microhomology between the host-DNA and T-DNA left border. One hundred and sixty 25-bp preinsertion sites were investigated for occurrence of 5-bp-long consecutive motifs corresponding to identical motifs in the T-DNA left border. The 41 sequences of preinsertion sites that show identity with consecutive, 1-bp sliding window, and 5-bp-long motifs are displayed.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/891/F5.medium.gif" width="440" height="322"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/891/F5.medium.gif" width="440" height="322"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/891/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/891/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461732" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-32" class="first-child">The search for microhomology between the host-DNA and T-DNA left border. One hundred and sixty 25-bp preinsertion sites were investigated for occurrence of 5-bp-long consecutive motifs corresponding to identical motifs in the T-DNA left border. The 41 sequences of preinsertion sites that show identity with consecutive, 1-bp sliding window, and 5-bp-long motifs are displayed.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Assessment of T-DNA LB sequence affinity with common core promoter elements of eukaryotic genes (TATA box with consensus TATA(T/A)A(A/T) (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Breathnach and Chambon 1981</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Burley 1996</a>), CAT box with core consensus CCAAT (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Bucher 1990</a>), and initiator (Inr) with consensus PPAN(T/A)PP (P, pyrimidine; N, any nucleotide) (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Javahery <em>et al.</em> 1994</a>) showed that the 25-bp-long T-DNA LB harbored islands of homologies with both TATA box (positions −5 to −10) and Inr (positions −20 to −25) elements. In addition, when we extended the alignment to 15 upstream supplementary bases, an additional homology with CAT box was found (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). We lastly calculated the frequency of occurrence of each base of the T-DNA LB sequence in microhomology motifs and plotted the values along the T-DNA LB sequence. As shown in <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>, T-DNA–host-DNA base identity increased approaching the LB free end. Corroborating the previous observation, the TATA island, but not Inr island, can be postulated to be frequently represented in T-DNA–host-DNA shared microhomologies due to the high ratio of identical bases at this location (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/891/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Occurrence of sequence microhomologies to eukaryotic core promoter elements (TATA box, CAT box, and Initiator) in the T-DNA LB and 15 upstream supplementary bases." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-502616278" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Occurrence of sequence microhomologies to eukaryotic core promoter elements (TATA box, CAT box, and Initiator) in the T-DNA LB and 15 upstream supplementary bases.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/891/F6.medium.gif" width="440" height="172"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/891/F6.medium.gif" width="440" height="172"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/891/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/891/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461692" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 </span> <p id="p-34" class="first-child">Occurrence of sequence microhomologies to eukaryotic core promoter elements (TATA box, CAT box, and Initiator) in the T-DNA LB and 15 upstream supplementary bases.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/8/891/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of microhomology at T-DNA preinsertion sites. Frequency of occurrence of single bases identical to those of the 25-bp T-DNA left border in the genome preinsertion sites were analyzed. The T-DNA LB sequence is illustrated, and homologs of the TATA box and Inr in the LB sequence are boxed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-502616278" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Analysis of microhomology at T-DNA preinsertion sites. Frequency of occurrence of single bases identical to those of the 25-bp T-DNA left border in the genome preinsertion sites were analyzed. The T-DNA LB sequence is illustrated, and homologs of the TATA box and Inr in the LB sequence are boxed.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/8/891/F7.medium.gif" width="440" height="300"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7&#x2002;" src="https://www.g3journal.org/content/ggg/2/8/891/F7.medium.gif" width="440" height="300"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/8/891/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/8/891/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461686" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7 </span> <p id="p-35" class="first-child">Analysis of microhomology at T-DNA preinsertion sites. Frequency of occurrence of single bases identical to those of the 25-bp T-DNA left border in the genome preinsertion sites were analyzed. The T-DNA LB sequence is illustrated, and homologs of the TATA box and Inr in the LB sequence are boxed.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>GO annotation of T-DNA–targeted genes</h3><p id="p-36">The whole-genome mapping of the 318 T-DNA insertions showed that 279 of these were in gene-coding or regulatory regions, whereas the other T-DNA tags were located in intergenic regions, including AT-rich, gene-poor isochores. A functional profile of the collection of T-DNA–targeted genes was performed by coupling the GO annotation of the “biological process” vocabulary with an assessment of representation bias by calculating the SRs between observed and expected number of annotations per functional category. The proportion of genes coding for hypothetical or predicted proteins of unknown function in the T-DNA–targeted gene collection (73.1%) was comparable to that observed in the whole genome (71.8%). Most of the functional categories represented in the “biological process” vocabulary (15 of 22) were represented in genes tagged by T-DNA (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The SR values for “pigmentation” (SR = 5.94), “growth” (SR = 2.09), and “cell wall organization or biogenesis” (SR = 3.21) revealed an overrepresentation in the T-DNA–targeted gene collection compared with all predicted genes of the genome, whereas the “signaling” functional category was underrepresented (SR = −2.03) (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Similar biases were identified using the Monte Carlo method (data not shown).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461730/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461730/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Gene Ontology annotation of T-DNA–targeted genes using the “biological process” vocabulary</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Functional significance of chromosome bias in T-DNA insertions?</h3><p id="p-41">T-DNA insertion events were mapped onto the <em>L. maculans</em> genome and plotted along its 18 chromosomes to investigate distribution biases. The T-DNA insertion density varied from 0 insertion event/Mb (chromosome 18) to 14.7 insertion events/Mb (chromosome 11) (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). In most of the cases, the number of tags per chromosome was compliant with a random integration of the T-DNA. However, T-DNA insertion events were found to be statistically more common than expected into chromosomes 5 (SR = 4.08) and 10 (SR = 4.16) (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> and <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>), whereas they were less common than expected into chromosome 18 (SR = −2.45) (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). Chromosome 1, whose number of T-DNA integrations was consistent with chromosome size (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>), showed a number of tags markedly lower than the mean confidence intervals for other criteria, such as size of coding or regulatory regions (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461781/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461781/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Distribution of T-DNA insertion events along the <em>L. maculans</em> chromosomes</span><div class="sb-div caption-clear"></div></div></div><p id="p-49">Chromosome 18 is very rich in TE and poor in genes (<a id="xref-ref-56-12" class="xref-bibr" href="#ref-56">Rouxel <em>et al.</em> 2011</a>), and the lack of T-DNA integration in this chromosome is consistent with the above-mentioned preferred integration in gene-rich genomics regions. GO annotation indicated that chromosome 5 was significantly enriched in the “pigmentation” (SR = 2.93) and “carbon utilization” (SR = 2.14) functional categories, whereas chromosome 10 showed no significant enrichment in any functional categories (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Of these, only the “pigmentation” functional category was found to be overrepresented in the collection of T-DNA–targeted genes (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). These data suggest that functional bias is unrelated to genome distribution of T-DNA insertions. Biases assessment using the Monte Carlo method led to the same results (data not shown).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461766/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461766/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Gene Ontology annotation of “biological process” for chromosomes 5 and 10</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-16"><h2 class="">DISCUSSION</h2><p id="p-54">Although many fungal species are amenable to ATMT, the mechanisms of T-DNA integration in the fungal genomes are largely unknown compared with what is known in plants, and it is still a matter of debate to know whether T-DNA integration will be random enough to allow a systematic targeting of all genes in the genome for functional identification. In phytopathogenic filamentous fungi, numerous pathogenicity mutants were generated by ATMT, but a systematic analysis of T-DNA integration in the genomes has only been performed in <em>M. oryzae</em> (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Choi <em>et al.</em> 2007</a>; <a id="xref-ref-45-6" class="xref-bibr" href="#ref-45">Meng <em>et al.</em> 2007</a>) and in <em>L. maculans</em> prior to obtainment of the whole-genome sequence (<a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">Blaise <em>et al.</em> 2007</a>). Here we exploited the <em>L. maculans</em> genome sequence to investigate how “canonical” T-DNA integration patterns are in a fungal genome with such contrasted genomic landscapes compared with what is known in <em>M. oryzae</em>. This comparison, however, has to be taken with care, as the two articles on <em>M. oryzae</em> show some experimental differences with ours. In <a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Choi <em>et al.</em> (2007)</a>, a very large number of 1246 transformants were investigated, but more than 1100 were chosen so that they harbor phenotypic defects, thus suggesting a bias toward T-DNA integration within coding sequences, in detriment to lines in which noncoding regions were targeted. In this sense, <a id="xref-ref-45-7" class="xref-bibr" href="#ref-45">Meng <em>et al.</em> (2007)</a>, who reported on characterization of a much lower number of 175 T-DNA integrations into random T-DNA tagged lines, was less biased for a systematic analysis of T-DNA patterns in filamentous fungi. In addition, the GO annotation of targeted proteins was used to have a better insight into T-DNA integration mechanisms in <em>L. maculans</em>, which has not been done in fungi, except budding yeast, to date (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Christie <em>et al.</em> 2009</a>).</p><p id="p-55">The ultimate goal of ATMT mutagenesis in plants or fungi is to reach saturation mutagenesis in order to eventually reach a functional annotation of the numerous unknown or hypothetical genes in the genomes. For example, in <em>L. maculans</em>, only 43% of the predicted proteins in the genome have strongly supported functional annotation, 45% are similar to hypothetical proteins for which no functional annotation is available, and 12% are predicted proteins with no annotation whatsoever (<a id="xref-ref-56-13" class="xref-bibr" href="#ref-56">Rouxel <em>et al.</em> 2011</a>; J. Grandaubert, unpublished data). In addition, in the case of phytopathogens, the initial objective of the T-DNA insertional mutagenesis strategy is the generation of mutants showing pathogenicity defects, as well as the discovery of novel genes and novel functions involved in pathogenesis. For these objectives, ATMT has to target mostly genic compartments of the genomes and show limited biases in targeted genes or genomic regions. The first advantage of ATMT for this objective is the common single-copy integration of the T-DNA in genomes, and mainly in fungal genomes, including that of <em>L. maculans</em> (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Michielse <em>et al.</em> 2005</a>; <a id="xref-ref-9-7" class="xref-bibr" href="#ref-9">Blaise <em>et al.</em> 2007</a>). The second point to be stressed in <em>L. maculans</em> is the high percentage of recovery of flanking sequences with matches in the fungal genome (around 80%) as was also observed for <em>M. oryzae</em> (<a id="xref-ref-45-8" class="xref-bibr" href="#ref-45">Meng <em>et al.</em> 2007</a>), whereas in plants, the frequencies usually amount to 60–65% [<em>e.g.</em> <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Thole <em>et al.</em> (2010)</a>]. As shown for <em>M. oryzae</em>, or <em>Arabidopsis</em> and other model plant species, the T-DNA integration is shown here to be nonrandom. First T-DNA integration was much rarer than expected in TEs, as is generally the case in plants (<a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Thole <em>et al.</em> 2010</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Zhang <em>et al.</em> 2007</a>), although this bias was not found when analyzing random transformants of <em>M. oryzae</em> (<a id="xref-ref-45-9" class="xref-bibr" href="#ref-45">Meng <em>et al.</em> 2007</a>). One possible explanation for this discrepancy would lie in the fact that all TEs in the genome of <em>L. maculans</em> are strongly degenerated and inactivated (<a id="xref-ref-56-14" class="xref-bibr" href="#ref-56">Rouxel <em>et al.</em> 2011</a>) and that, as discussed below, T-DNA integration favors transcriptionally active regions of the genome. Also, as is the case for <em>M. oryzae</em>, T-DNA tags were not recovered from other large arrays of repeats, such as the rDNA array, or from the mitochondrial genome (<a id="xref-ref-45-10" class="xref-bibr" href="#ref-45">Meng <em>et al.</em> 2007</a>), whereas the tagging of the rDNA array by T-DNA is overrepresented in some plant species, such as <em>B. distachyon</em> (<a id="xref-ref-60-5" class="xref-bibr" href="#ref-60">Thole <em>et al.</em> 2010</a>). Similarly to what was observed in <em>M. oryzae</em>, a marginal chromosomal bias showed some favored or disfavored chromosomes for T-DNA integration in <em>L. maculans</em>. As was noticed for <em>M. oryzae</em>, the biological significance of this fact remains obscure because no functional specificity was associated with these chromosomes. This bias may only be due to the limited number of tags analyzed in randomly tagged fungal isolates as it does not seem to occur in plants where the nonrandom integrations are observed within a chromosome rather than between chromosomes (<a id="xref-ref-60-6" class="xref-bibr" href="#ref-60">Thole <em>et al.</em> 2010</a>). More importantly, one main feature of T-DNA integration in the genome [<em>i.e.</em>, the favored targeting of 5′ 500-bp regions of genes assumed to be promoters] is a widely shared trait for plants and fungi (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Alonso <em>et al.</em> 2003</a>; <a id="xref-ref-45-11" class="xref-bibr" href="#ref-45">Meng <em>et al.</em> 2007</a>; <a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">Choi <em>et al.</em> 2007</a>; <a id="xref-ref-60-7" class="xref-bibr" href="#ref-60">Thole <em>et al.</em> 2010</a>). By comparison with <em>M. oryzae</em>, the bias toward promoter regions was even more marked in the genome of <em>L. maculans</em>, consistent with the common recovery of pathogenic mutants for which the altered pathogenicity was due to T-DNA integration in promoters of genes (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Elliott and Howlett 2006</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Remy <em>et al.</em> 2008a</a>,<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">b</a>, <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">2009</a>). Lastly, the favored targeting of promoters is consistent with the presence of microhomology motifs (see below) involved in the homologous recombination with the T-DNA border.</p><p id="p-56">When compiled, the observed T-DNA integration biases seem to share at least one common denominator: T-DNA integration takes place in transcriptionally active regions. In a cell, transcriptional activity should be considered the first step in the translation of genomic information into physiological state. Hence, starting from this postulate, it is logical to suppose that cellular activity affects T-DNA–favored insertion sites and that targeted genes should reflect, to a certain extent, the physiological state of the transformed cell. In this study, T-DNA insertion loci were recovered from <em>L. maculans</em> transformants obtained by ATMT for which germinating conidia (incubated for 48 hr) were used (<a id="xref-ref-9-8" class="xref-bibr" href="#ref-9">Blaise <em>et al.</em> 2007</a>). Conidial germination is commonly described as a three-step mechanism (D'enfert 1997; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Osherov and May 2001</a>): (i) activation, during which appropriate amounts of water and low-molecular-weight nutrients trigger conidial cell activation for germination; (ii) isotropic growth, during which the conidial cell undergoes morphological changes, uptakes water, and increases its physiological activity, which leads to an increase in size and mass; and (iii) polarized growth, during which a germ tube emerges from the conidial cell and develops, which requires <em>de novo</em> synthesis of wall materials. Germination is an asynchronic phenomenon that may differ from one conidia to another. ATMT is thus performed on conidial populations at four physiological stages: (i) ungerminated conidia, (ii) conidia at germination activation, (iii) conidia at isotropic growth, and (iv) conidia at polarized growth. The overrepresentation of “pigmentation,” “growth,” and “cell wall organization or biogenesis” functional categories in genes targeted by the T-DNA would be consistent with the hypothesis that targeted genes reflect the physiological state of the germinating conidia. First, even though the conidia of <em>L. maculans</em> are hyaline under the microscope, deposition of melanin and other pigments is generally associated with spore production in fungi in which they seem to function in the protection of microbes against environmental stress such as UV light and heat (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Will <em>et al.</em> 1987</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Brags <em>et al.</em> 2006</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Rangel <em>et al.</em> 2006</a>), consistent with the overrepresentation of the “pigmentation” category. Second, “growth” fits the isotropic growth step of conidial germination. Third, “cell wall organization or biogenesis” fits the polarized growth step of conidial germination.</p><p id="p-57">T-DNA intranuclear targeting is assumed to result from a long evolution of <em>Agrobacterium</em> species' transfection mechanisms to fit host cellular machinery. Starting from this postulate, we analyzed this phenomenon mainly from the host point of view. Consequently, we considered T-DNA insertion biases not only as resulting from T-DNA characteristics but also largely depending on the following: (i) host genome characteristics (in <em>L. maculans</em> GC isochores/gene-rich <em>vs.</em> AT isochores/gene-poor compartments); (ii) gene expression at both transcriptional (machinery) and functional (cell physiological state) levels; and (iii) DNA features characterized by heterogeneity, unequal sensitivity to DNA damages, and organization in a gene-dependent fashion.</p><p id="p-58">Locus biases, which are due to a chromatin-targeting process that guides T-DNA from its entry into the nucleus to its anchorage to host chromatin, were investigated. Previous studies have shown that T-DNA nuclear import is mediated by two bacterial virulence (Vir) proteins, VirD2 and VirE2, which directly associates with T-DNA to form the transport (T) complex [for review, see <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Tzfira <em>et al.</em> (2000)</a> and <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Zupan <em>et al.</em> (2000)</a>]. In addition to T-DNA encapsidation, Vir proteins act as interfaces with host machinery: VirD2 is phosphorylated <em>in vivo</em> by CAK2M, a cyclin-dependent kinase-activating kinase, and is tightly associated with TATA box-binding protein (TBP) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bakó <em>et al.</em> 2003</a>), and VirE2 binds VIP1 (VirE2-interacting protein 1), a bZIP transcription factor capable of binding core histones (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Li <em>et al.</em> 2005</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Loyter <em>et al.</em> 2005</a>). VIP1, CAK2M, and TBP profile T-DNA insertion loci distribution within the host genome, and their proper functions and properties generate locus biases: CAK2M phosphorylates the C-terminal domain of the RNA polymerase II (RNA Pol II) largest subunit (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bakó <em>et al.</em> 2003</a>), which serves as a TBP-binding platform (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Yuryev <em>et al.</em> 1996</a>); TBP binds the TATA box core promoter element, whose recognition nucleates the assembly of transcription preinitiation complex [for review, see <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Smale and Kadonaga (2003)</a>]; and VIP1 precisely binds a DNA hexamer motif found in the promoters of various stress-responsive genes and plays a role in immunity signaling by stimulating stress-dependent gene expression, at least in plants (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Djamei <em>et al.</em> 2007</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Pitzschke <em>et al.</em> 2009</a>). However, because VIP1 shows no significant homology to known animal or fungal proteins, it could be plant specific. Nevertheless, because T-complex anchorage to host chromatin seems to be a key step for further T-DNA integration, it is consistent to consider the existence of animal and fungal VIP1-like proteins interacting with both T-complex and host chromatin. The T-DNA insertion pattern in <em>L. maculans</em> corroborates this tight relation between T-DNA and gene transcription machinery, because T-DNA insertions were predominant in the following: (i) GC isochores, which are gene-rich islands frequently targeted by gene transcription machinery and therefore more likely to be in a relaxed, opened state, rather than AT isochores, which are TE-rich, gene-poor regions assimilated to heterochromatin, therefore condensed and closed (<a id="xref-ref-56-15" class="xref-bibr" href="#ref-56">Rouxel <em>et al.</em> 2011</a>); (ii) gene-regulatory regions, which border zones between a histone-containing region capable of anchoring the T-complex and a histone-less region that is the gene-expressing DNA; and (iii) promoter region, in which additional binding opportunities (CAK2M, TBP) increase the probability of T-complex anchorage and strengthen it.</p><p id="p-59">T-complex anchorage to host chromatin is not synonymous with T-DNA insertion. For the latter to occur, DNA damage is mandatory, because the T-DNA integration process abuses HR and NHR pathways, two host-DNA double-strand break (DSB) repair machineries [for review, see <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Tzfira <em>et al.</em> (2004)</a> and <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Citovsky <em>et al.</em> (2007)</a>]. Consequently, as additional factors that affect occurrence of T-DNA integration events, we must consider both DSBs hotspots and DSB repair efficiency. In eukaryotic cells, DSBs are common events resulting from both environmental and endogenous factors. DSBs are also created by converting single-strand lesions (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Natarajan 1993</a>) and retrotransposon activity (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gasior <em>et al.</em> 2006</a>), and they occur preferentially in opened chromatin (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Berchowitz <em>et al.</em> 2009</a>) and transcriptionally active promoters, telomeres, and centromeres (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Wu and Lichten 1994</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Baudat and Nicolas 1997</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Blitzblau <em>et al.</em> 2007</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Buhler <em>et al.</em> 2007</a>). However, not all occurring DSBs are repaired with the same efficiency. In fact, telomeric regions and packed heterochromatin are deficient in repair of DSBs (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Ricchetti <em>et al.</em> 2003</a>). Together, these studies highlight biases of DSB occurrence and repair that correlate with T-DNA mapping biases in <em>L. maculans</em>. Actually, T-DNA integration favored gene-rich GC isochores and not AT isochores that exhibit heterochromatin characteristics in which chromatin is packed and DSB repair is likely to be deficient, and T-DNA integration events overmapped to gene promoter regions where DSBs are assumed to occur frequently.</p><p id="p-60">To corroborate mapping biases in T-DNA integration events, we analyzed T-DNA LB, T-DNA preinsertion and insertion site sequences, and T-DNA–targeted genes for particular compositional, structural, and functional signatures, and we showed that T-DNA LB shares microhomologies with preinsertion sites, suggesting that T-DNA integration may occur at least by HR in <em>L. maculans</em>. The same was observed by <a id="xref-ref-45-12" class="xref-bibr" href="#ref-45">Meng <em>et al.</em> (2007)</a> and by <a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">Choi <em>et al.</em> (2007)</a> in the <em>M. oryzae</em> genome, but the authors did not reach conclusive evidence regarding the targeted motifs. Hence, T-DNA LB sequence affinity with host DNA may affect T-DNA integration event distribution. Our results highlighted that T-DNA LB harbored microhomologies with CAAT box, Inr, and TATA box of eukaryotic promoters. Also, TATA-containing micromology motifs were frequently shared between T-DNA LB and its target sequence. These observations correlate with frequent mapping of T-DNA insertions to gene promoter regions, in the sense that sequence affinity is mandatory for DNA end joining by MMEJ [for review, see McVey and Lee (2008)].</p><p id="p-61">DNA asymmetry was observed in both prokaryotic and eukaryotic genomes. It is a consequence of many mechanisms, among which gene expression is one of the better studied, and in which DNA asymmetry is seen as signatures indicating functional signals and DNA modifications (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Touchon and Rocha 2008</a>). DNA asymmetry is revealed by the CG skew and AT skew. Previous studies noticed that the CG skew is stronger than the AT skew, at least in eubacteria (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Francino and Ochman 1997</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Frank and Lobry 1999</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Tillier and Collins 2000</a>) and that skew curves are associated with replication origin (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lobry, 1996a</a>,<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">b</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Blattner <em>et al.</em> 1997</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kunst <em>et al.</em> 1997</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Grigoriev 1998</a>) and transcription-coupled and splicing-coupled mutations (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Touchon <em>et al.</em> 2004</a>). In particular, CG skew peak is associated with gene expression level, at least in plants (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alexandrov <em>et al.</em> 2006</a>), and transcription initiation starts in eukaryotes, including fungi (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Tatarinova <em>et al.</em> 2003</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Fujimori <em>et al.</em> 2005</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Alexandrov <em>et al.</em> 2006</a>). Altogether, these studies highlight a correlation between DNA asymmetry and cellular activity–driven DNA manipulations and modifications in general (replication, gene expression, mutations) and a tight association between an increased CG skew and gene transcription in particular. T-DNA targeted to CG asymmetric DNA is thus consistent with frequent insertions in transcriptionally active regions and gene promoter sequences.</p></div><div class="section conclusions" id="sec-17"><h2 class="">CONCLUSIONS</h2><p id="p-62">Using a fungal genome showing contrasted genomic landscapes, our data substantiate the advantages of ATMT to reach a functional annotation of genes, but they cast doubts on whether this strategy will be able to target species-specific genes involved in pathogenicity that reside in specific AT-rich compartments of the fungal genome. The main particularities substantiating these points are (i) the common single-copy integration of the T-DNA; (ii) the high frequency of integration within protein-coding genes (even if the introns are favored targets), amounting to one third of the integration events in <em>L. maculans</em>; and (iii) the common occurrence in promoters favoring the access to genes whose complete inactivation would be too detrimental for the fungus. In contrast, under the hypothesis that genes specifically involved in plant pathogenicity are hosted in specific compartments of the genome, we notice that T-DNA targeting to AT isochores is very low compared with the percentage of these landscapes in the genome, but it is nevertheless consistent with the amount of genes hosted in AT isochores. More importantly, our work suggests the importance of the physiology of the fungus at the time of ATMT and the favored targeting of transcriptionally active regions of the genome. With most of the genes involved in pathogenicity, such as those encoding effector proteins, repressed during the vegetative growth of the fungus and overexpressed at the onset of plant infection (<a id="xref-ref-56-16" class="xref-bibr" href="#ref-56">Rouxel <em>et al.</em> 2011</a>), these are unlikely to be targeted by the T-DNA.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgment</h2><p id="p-63">S.B. was funded by a grant from the French Ministry of Research.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-19"><p id="p-64"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-01-21"><span class="received-label">Received </span>January 21, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-06-07"><span class="accepted-label">Accepted </span>June 7, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Bourras <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.1" data-doi="10.1007/s11103-005-2564-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexandrov</span>  <span class="cit-name-given-names">N. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troukhan</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brover</span>  <span class="cit-name-given-names">V. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatarinova</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flavell</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Features of <em>Arabidopsis</em> genes and genome discovered using full-length cDNAs</span>. <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DAlexandrov%26rft.auinit1%253DN.%2BN.%26rft.volume%253D60%26rft.issue%253D1%26rft.spage%253D69%26rft.epage%253D85%26rft.atitle%253DFeatures%2Bof%2BArabidopsis%2Bgenes%2Band%2Bgenome%2Bdiscovered%2Busing%2Bfull-length%2BcDNAs.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11103-005-2564-9%26rft_id%253Dinfo%253Apmid%252F16463100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11103-005-2564-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16463100&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235202000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.2" data-doi="10.1126/science.1086391"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alonso</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stepanova</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leisse</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genome-wide insertional mutagenesis of <em>Arabidopsis thaliana</em></span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">301</span>: <span class="cit-fpage">653</span>–<span class="cit-lpage">657</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAlonso%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D301%26rft.issue%253D5633%26rft.spage%253D653%26rft.epage%253D657%26rft.atitle%253DGenome-Wide%2BInsertional%2BMutagenesis%2Bof%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1086391%26rft_id%253Dinfo%253Apmid%252F12893945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDEvNTYzMy82NTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzgvODkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.8.891.3" data-doi="10.1038/35048692"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Arabidopsis Genome Initiative</span>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Analysis of the genome sequence of the flowering plant <em>Arabidopsis thaliana</em></span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">408</span>: <span class="cit-fpage">796</span>–<span class="cit-lpage">815</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D408%26rft.issue%253D6814%26rft.spage%253D796%26rft.epage%253D815%26rft.atitle%253DAnalysis%2Bof%2Bthe%2Bgenome%2Bsequence%2Bof%2Bthe%2Bflowering%2Bplant%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35048692%26rft_id%253Dinfo%253Apmid%252F11130711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35048692&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11130711&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165831300037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Attard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gout</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gourgues</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmit</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Analysis of molecular markers genetically linked to the <em>Leptosphaeria maculans</em> avirulence gene <em>AvrLm1</em> in field populations indicates a highly conserved event leading to virulence on <em>Rlm1</em> genotypes</span>. <abbr class="cit-jnl-abbrev">Mol. Plant-Microbe Interact.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">672</span>–<span class="cit-lpage">682</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bplant-microbe%2Binteractions%2B%253A%2B%2BMPMI%26rft.stitle%253DMol%2BPlant%2BMicrobe%2BInteract%26rft.aulast%253DAttard%26rft.auinit1%253DA.%26rft.volume%253D15%26rft.issue%253D7%26rft.spage%253D672%26rft.epage%253D682%26rft.atitle%253DAnalysis%2Bof%2Bmolecular%2Bmarkers%2Bgenetically%2Blinked%2Bto%2Bthe%2BLeptosphaeria%2Bmaculans%2Bavirulence%2Bgene%2BAvrLm1%2Bin%2Bfield%2Bpopulations%2Bindicates%2Ba%2Bhighly%2Bconserved%2Bevent%2Bleading%2Bto%2Bvirulence%2Bon%2BRlm1%2Bgenotypes.%26rft_id%253Dinfo%253Apmid%252F12118883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12118883&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.5" data-doi="10.1073/pnas.1733208100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bakó</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Umeda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiburcio</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koncz</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The VirD2 pilot protein of <em>Agrobacterium</em>-transferred DNA interacts with the TATA box-binding protein and a nuclear protein kinase in plants</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">10108</span>–<span class="cit-lpage">10113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1733208100%26rft_id%253Dinfo%253Apmid%252F12900506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAwLzE3LzEwMTA4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84Lzg5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balzergue</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubreucq</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chauvin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le-Clainche</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Boulaire</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Improved PCR-walking for large-scale isolation of plant T-DNA borders</span>. <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">496</span>–<span class="cit-lpage">504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechniques%26rft.volume%253D30%26rft.spage%253D496%26rft_id%253Dinfo%253Apmid%252F11252785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11252785&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167339800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.7" data-doi="10.1073/pnas.94.10.5213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baudat</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Clustering of meiotic double-strand breaks on yeast chromosome III</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">5213</span>–<span class="cit-lpage">5218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBaudat%26rft.auinit1%253DF.%26rft.volume%253D94%26rft.issue%253D10%26rft.spage%253D5213%26rft.epage%253D5218%26rft.atitle%253DClustering%2Bof%2Bmeiotic%2Bdouble-strand%2Bbreaks%2Bon%2Byeast%2Bchromosome%2BIII%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.10.5213%26rft_id%253Dinfo%253Apmid%252F9144217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTQvMTAvNTIxMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOC84OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.8" data-doi="10.1101/gr.096297.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berchowitz</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanlon</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lieb</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Copenhaver</span>  <span class="cit-name-given-names">G. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A positive but complex association between meiotic double-strand break hotspots and open chromatin in <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2245</span>–<span class="cit-lpage">2257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.096297.109%26rft_id%253Dinfo%253Apmid%252F19801530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMi8yMjQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84Lzg5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.9" data-doi="10.1016/j.fgb.2006.07.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blaise</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Remy</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Narcy</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A critical assessment of <em>Agrobacterium tumefaciens</em>-mediated transformation as a tool for pathogenicity gene discovery in the phytopathogenic fungus <em>Leptosphaeria maculans</em></span>. <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DBlaise%26rft.auinit1%253DF.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D123%26rft.epage%253D138%26rft.atitle%253DA%2Bcritical%2Bassessment%2Bof%2BAgrobacterium%2Btumefaciens-mediated%2Btransformation%2Bas%2Ba%2Btool%2Bfor%2Bpathogenicity%2Bgene%2Bdiscovery%2Bin%2Bthe%2Bphytopathogenic%2Bfungus%2BLeptosphaeria%2Bmaculans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2006.07.006%26rft_id%253Dinfo%253Apmid%252F16979359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2006.07.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16979359&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.10" data-doi="10.1126/science.277.5331.1453"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blattner</span>  <span class="cit-name-given-names">F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plunkett</span>  <span class="cit-name-given-names">G.</span>  <span class="cit-name-suffix">III</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloch</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perna</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burland</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The complete genome sequence of <em>Escherichia coli</em> K-12</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">277</span>: <span class="cit-fpage">1453</span>–<span class="cit-lpage">1462</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBlattner%26rft.auinit1%253DF.%2BR.%26rft.volume%253D277%26rft.issue%253D5331%26rft.spage%253D1453%26rft.epage%253D1462%26rft.atitle%253DThe%2BComplete%2BGenome%2BSequence%2Bof%2BEscherichia%2Bcoli%2BK-12%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.277.5331.1453%26rft_id%253Dinfo%253Apmid%252F9278503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNzcvNTMzMS8xNDUzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84Lzg5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.11" data-doi="10.1016/j.cub.2007.10.066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blitzblau</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochwagen</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mapping of meiotic single-stranded DNA reveals double-stranded-break hotspots near centromeres and telomeres</span>. <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">2003</span>–<span class="cit-lpage">2012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DBlitzblau%26rft.auinit1%253DH.%2BG.%26rft.volume%253D17%26rft.issue%253D23%26rft.spage%253D2003%26rft.epage%253D2012%26rft.atitle%253DMapping%2Bof%2Bmeiotic%2Bsingle-stranded%2BDNA%2Breveals%2Bdouble-stranded-break%2Bhotspots%2Bnear%2Bcentromeres%2Band%2Btelomeres.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2007.10.066%26rft_id%253Dinfo%253Apmid%252F18060788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2007.10.066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18060788&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251482100021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.12" data-doi="10.1562/2005-05-08-RA-52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brags</span>  <span class="cit-name-given-names">G. U. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rangel</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Conidial pigmentation is important to tolerance against solar-simulated radiation in the entomopathogenic fungus <em>Metarhizium anisopliae</em></span>. <abbr class="cit-jnl-abbrev">Photochem. Photobiol.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">418</span>–<span class="cit-lpage">422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhotochemistry%2Band%2BPhotobiology%26rft.stitle%253DPhotochemistry%2Band%2BPhotobiology%26rft.aulast%253DBraga%26rft.auinit1%253DG.%2BU.%26rft.volume%253D82%26rft.issue%253D2%26rft.spage%253D418%26rft.epage%253D422%26rft.atitle%253DConidial%2Bpigmentation%2Bis%2Bimportant%2Bto%2Btolerance%2Bagainst%2Bsolar-simulated%2Bradiation%2Bin%2Bthe%2Bentomopathogenic%2Bfungus%2BMetarhizium%2Banisopliae.%26rft_id%253Dinfo%253Adoi%252F10.1562%252F2005-05-08-RA-52%26rft_id%253Dinfo%253Apmid%252F16613494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1562/2005-05-08-RA-52&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16613494&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.13" data-doi="10.1146/annurev.bi.50.070181.002025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breathnach</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chambon</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Organization and expression of eucaryotic split genes coding for proteins</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DBreathnach%26rft.auinit1%253DR.%26rft.volume%253D50%26rft.spage%253D349%26rft.epage%253D383%26rft.atitle%253DOrganization%2Band%2Bexpression%2Bof%2Beucaryotic%2Bsplit%2Bgenes%2Bcoding%2Bfor%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.bi.50.070181.002025%26rft_id%253Dinfo%253Apmid%252F6791577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.bi.50.070181.002025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6791577&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981LW78300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.14" data-doi="10.1016/0022-2836(90)90223-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bucher</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Weight matrix descriptions of four eukaryotic RNA polymerase II promoter elements derived from 502 unrelated promoter sequences</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">212</span>: <span class="cit-fpage">563</span>–<span class="cit-lpage">578</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DBucher%26rft.auinit1%253DP.%26rft.volume%253D212%26rft.issue%253D4%26rft.spage%253D563%26rft.epage%253D578%26rft.atitle%253DWeight%2Bmatrix%2Bdescriptions%2Bof%2Bfour%2Beukaryotic%2BRNA%2Bpolymerase%2BII%2Bpromoter%2Belements%2Bderived%2Bfrom%2B502%2Bunrelated%2Bpromoter%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252890%252990223-9%26rft_id%253Dinfo%253Apmid%252F2329577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(90)90223-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2329577&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DB07400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.15" data-doi="10.1371/journal.pbio.0050324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buhler</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borde</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mapping meiotic single-strand DNA reveals a new landscape of DNA double-strand breaks in <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DBuhler%26rft.auinit1%253DC.%26rft.volume%253D5%26rft.issue%253D12%26rft.spage%253De324%26rft.epage%253De324%26rft.atitle%253DMapping%2Bmeiotic%2Bsingle-strand%2BDNA%2Breveals%2Ba%2Bnew%2Blandscape%2Bof%2BDNA%2Bdouble-strand%2Bbreaks%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0050324%26rft_id%253Dinfo%253Apmid%252F18076285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0050324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076285&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.16" data-doi="10.1016/S0959-440X(96)80097-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burley</span>  <span class="cit-name-given-names">S. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The TATA box binding protein</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Struct. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bstructural%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BStruct%2BBiol%26rft.aulast%253DBurley%26rft.auinit1%253DS.%2BK.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D69%26rft.epage%253D75%26rft.atitle%253DThe%2BTATA%2Bbox%2Bbinding%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-440X%252896%252980097-2%26rft_id%253Dinfo%253Apmid%252F8696975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-440X(96)80097-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8696975&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996TW57300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.17" data-doi="10.1111/j.1365-2958.2007.05918.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chi</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide analysis of T-DNA integration into the chromosomes of <em>Magnaporthe oryzae</em></span>. <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">382</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DChoi%26rft.auinit1%253DJ.%26rft.volume%253D66%26rft.issue%253D2%26rft.spage%253D371%26rft.epage%253D382%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2BT-DNA%2Bintegration%2Binto%2Bthe%2Bchromosomes%2Bof%2BMagnaporthe%2Boryzae.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2007.05918.x%26rft_id%253Dinfo%253Apmid%252F17850257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2007.05918.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17850257&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249834700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.18" data-doi="10.1016/j.tim.2009.04.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christie</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherry</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Functional annotations for the <em>Saccharomyces cerevisiae</em> genome: the knowns and the known unknowns</span>. <abbr class="cit-jnl-abbrev">Trends Microbiol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">286</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.stitle%253DTrends%2BMicrobiol%26rft.aulast%253DChristie%26rft.auinit1%253DK.%2BR.%26rft.volume%253D17%26rft.issue%253D7%26rft.spage%253D286%26rft.epage%253D294%26rft.atitle%253DFunctional%2Bannotations%2Bfor%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome%253A%2Bthe%2Bknowns%2Band%2Bthe%2Bknown%2Bunknowns.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2009.04.005%26rft_id%253Dinfo%253Apmid%252F19577472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2009.04.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19577472&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268616600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.19" data-doi="10.1111/j.1462-5822.2006.00830.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Citovsky</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozlovsky</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacroix</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaltsman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dafny-Yelin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Biological systems of the host cell involved in <em>Agrobacterium</em> infection</span>. <abbr class="cit-jnl-abbrev">Cell. Microbiol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCellular%2Bmicrobiology%26rft.stitle%253DCell%2BMicrobiol%26rft.aulast%253DCitovsky%26rft.auinit1%253DV.%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253D9%26rft.epage%253D20%26rft.atitle%253DBiological%2Bsystems%2Bof%2Bthe%2Bhost%2Bcell%2Binvolved%2Bin%2BAgrobacterium%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1462-5822.2006.00830.x%26rft_id%253Dinfo%253Apmid%252F17222189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1462-5822.2006.00830.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17222189&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242780900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.8.891.20" data-doi="10.1006/fgbi.1997.0975"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">D'Enfert</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Fungal spore germination: insights from the molecular genetics of <em>Aspergillus nidulans</em> and <em>Neurospora crassa</em></span>. <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BBiol.%26rft.volume%253D21%26rft.spage%253D163%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ffgbi.1997.0975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/fgbi.1997.0975&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XH94900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.21" data-doi="10.1126/science.1148110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Djamei</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pitzschke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakagami</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajh</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirt</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Trojan horse strategy in <em>Agrobacterium</em> transformation: abusing MAPK defense signaling</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">318</span>: <span class="cit-fpage">453</span>–<span class="cit-lpage">456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDjamei%26rft.auinit1%253DA.%26rft.volume%253D318%26rft.issue%253D5849%26rft.spage%253D453%26rft.epage%253D456%26rft.atitle%253DTrojan%2BHorse%2BStrategy%2Bin%2BAgrobacterium%2BTransformation%253A%2BAbusing%2BMAPK%2BDefense%2BSignaling%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1148110%26rft_id%253Dinfo%253Apmid%252F17947581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTgvNTg0OS80NTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzgvODkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.22" data-doi="10.1094/MPMI-19-0588"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elliott</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howlett</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Overexpression of a 3-ketoacyl-CoA thiolase in <em>Leptosphaeria maculans</em> causes reduced pathogenicity on <em>Brassica napus</em></span>. <abbr class="cit-jnl-abbrev">Mol. Plant-Microbe Interact.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">588</span>–<span class="cit-lpage">596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bplant-microbe%2Binteractions%2B%253A%2B%2BMPMI%26rft.stitle%253DMol%2BPlant%2BMicrobe%2BInteract%26rft.aulast%253DElliott%26rft.auinit1%253DC.%2BE.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D588%26rft.epage%253D596%26rft.atitle%253DOverexpression%2Bof%2Ba%2B3-ketoacyl-CoA%2Bthiolase%2Bin%2BLeptosphaeria%2Bmaculans%2Bcauses%2Breduced%2Bpathogenicity%2Bon%2BBrassica%2Bnapus.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FMPMI-19-0588%26rft_id%253Dinfo%253Apmid%252F16776292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/MPMI-19-0588&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16776292&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237757200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.23" data-doi="10.1016/S0168-9525(97)01118-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Francino</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochman</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Strand asymmetries in DNA evolution</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">240</span>–<span class="cit-lpage">245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DFrancino%26rft.auinit1%253DM.%2BP.%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253D240%26rft.epage%253D245%26rft.atitle%253DStrand%2Basymmetries%2Bin%2BDNA%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252897%252901118-9%26rft_id%253Dinfo%253Apmid%252F9196330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(97)01118-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9196330&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XD50800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.24" data-doi="10.1016/S0378-1119(99)00297-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frank</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lobry</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Asymmetric substitution patterns: a review of possible underlying mutational or selective mechanisms</span>. <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">238</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">77</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D238%26rft.spage%253D65%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252899%252900297-8%26rft_id%253Dinfo%253Apmid%252F10570985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(99)00297-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10570985&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083005400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.25" data-doi="10.1186/1471-2164-6-26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fujimori</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Washio</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomita</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">GC-compositional strand bias around transcription start sites in plants and fungi</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DFujimori%26rft.auinit1%253DS.%26rft.volume%253D6%26rft.spage%253D26%26rft.epage%253D26%26rft.atitle%253DGC-compositional%2Bstrand%2Bbias%2Baround%2Btranscription%2Bstart%2Bsites%2Bin%2Bplants%2Band%2Bfungi.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-6-26%26rft_id%253Dinfo%253Apmid%252F15733327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-6-26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15733327&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.26" data-doi="10.1016/j.jmb.2006.01.089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gasior</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wakeman</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deininger</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The human LINE-1 retrotransposon creates DNA double-strand breaks</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">357</span>: <span class="cit-fpage">1383</span>–<span class="cit-lpage">1393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DGasior%26rft.auinit1%253DS.%2BL.%26rft.volume%253D357%26rft.issue%253D5%26rft.spage%253D1383%26rft.epage%253D1393%26rft.atitle%253DThe%2Bhuman%2BLINE-1%2Bretrotransposon%2Bcreates%2BDNA%2Bdouble-strand%2Bbreaks.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2006.01.089%26rft_id%253Dinfo%253Apmid%252F16490214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2006.01.089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16490214&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236629400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.27" data-doi="10.1126/science.274.5287.546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goffeau</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrell</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bussey</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Life with 6000 genes</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">546</span>–<span class="cit-lpage">567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGoffeau%26rft.auinit1%253DA.%26rft.volume%253D274%26rft.issue%253D5287%26rft.spage%253D546%26rft.epage%253D567%26rft.atitle%253DLife%2Bwith%2B6000%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.274.5287.546%26rft_id%253Dinfo%253Apmid%252F8849441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNzQvNTI4Ny81NDYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzgvODkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.28" data-doi="10.1093/nar/gkn176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Götz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">García-Gómez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terol</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagaraj</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">High-throughput functional annotation and data mining with the blast2GO suite</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">3420</span>–<span class="cit-lpage">3435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn176%26rft_id%253Dinfo%253Apmid%252F18445632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzNi8xMC8zNDIwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84Lzg5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.29" data-doi="10.1093/nar/26.10.2286"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grigoriev</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Analyzing genomes with cumulative skew diagrams</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2286</span>–<span class="cit-lpage">2290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F26.10.2286%26rft_id%253Dinfo%253Apmid%252F9580676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMC8yMjg2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84Lzg5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-other" id="cit-2.8.891.30" data-doi="10.1080/21501203.2011.584577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grigoriev</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodwin</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hibbett</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeffries</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Fueling the future with fungal genomics</span>. <span class="cit-source">Mycology</span> <span class="cit-vol">2</span>: <span class="cit-fpage">192</span>–<span class="cit-lpage">209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycology%26rft.volume%253D2%26rft.spage%253D192%26rft_id%253Dinfo%253Adoi%252F10.1080%252F21501203.2011.584577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/21501203.2011.584577&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.31" data-doi="10.2307/1266761"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grubbs</span>  <span class="cit-name-given-names">F. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-article-title">Procedures for detecting outlying observations in samples</span>. <abbr class="cit-jnl-abbrev">Technometrics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTechnometrics%26rft.volume%253D11%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1266761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1266761&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1969C675800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.32" data-doi="10.1038/nature02026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huh</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falvo</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howson</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Global analysis of protein localization in budding yeast</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">425</span>: <span class="cit-fpage">686</span>–<span class="cit-lpage">691</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHuh%26rft.auinit1%253DW.%2BK.%26rft.volume%253D425%26rft.issue%253D6959%26rft.spage%253D686%26rft.epage%253D691%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bprotein%2Blocalization%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02026%26rft_id%253Dinfo%253Apmid%252F14562095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14562095&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185924500033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.33" data-doi="10.1128/MCB.14.1.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Javahery</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khachi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zenzie-Gregory</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smale</span>  <span class="cit-name-given-names">S. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">DNA sequence requirements for transcriptional initiator activity in mammalian cells</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">116</span>–<span class="cit-lpage">127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DJavahery%26rft.auinit1%253DR%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253D116%26rft.epage%253D127%26rft.atitle%253DDNA%2Bsequence%2Brequirements%2Bfor%2Btranscriptional%2Binitiator%2Bactivity%2Bin%2Bmammalian%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.14.1.116%26rft_id%253Dinfo%253Apmid%252F8264580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjE0LzEvMTE2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84Lzg5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.34" data-doi="10.1038/nmeth.1181"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stalker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Humphray</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A systematic library for comprehensive overexpression screens in <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DJones%26rft.auinit1%253DG.%2BM.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253D239%26rft.epage%253D241%26rft.atitle%253DA%2Bsystematic%2Blibrary%2Bfor%2Bcomprehensive%2Boverexpression%2Bscreens%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1181%26rft_id%253Dinfo%253Apmid%252F18246075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1181&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18246075&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253777900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.35" data-doi="10.1104/pp.110.162164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kourmpetis</span>  <span class="cit-name-given-names">Y. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Dijk</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Ham</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">ter Braak</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide computational function prediction of <em>Arabidopsis</em> proteins by integration of multiple data sources</span>. <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">271</span>–<span class="cit-lpage">281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.110.162164%26rft_id%253Dinfo%253Apmid%252F21098674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTUvMS8yNzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzgvODkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.36" data-doi="10.1104/pp.108.128918"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krishnan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guiderdoni</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsing</span>  <span class="cit-name-given-names">Y. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mutant resources in rice for functional genomics of the grasses</span>. <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DKrishnan%26rft.auinit1%253DA.%26rft.volume%253D149%26rft.issue%253D1%26rft.spage%253D165%26rft.epage%253D170%26rft.atitle%253DMutant%2BResources%2Bin%2BRice%2Bfor%2BFunctional%2BGenomics%2Bof%2Bthe%2BGrasses%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.108.128918%26rft_id%253Dinfo%253Apmid%252F19126710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNDkvMS8xNjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzgvODkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.37" data-doi="10.1038/36786"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kunst</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogasawara</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moszer</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albertini</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alloni</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The complete genome sequence of the Gram-positive bacterium <em>Bacillus subtilis</em></span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">390</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">256</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKunst%26rft.auinit1%253DF.%26rft.volume%253D390%26rft.issue%253D6657%26rft.spage%253D249%26rft.epage%253D256%26rft.atitle%253DThe%2Bcomplete%2Bgenome%2Bsequence%2Bof%2Bthe%2Bgram-positive%2Bbacterium%2BBacillus%2Bsubtilis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F36786%26rft_id%253Dinfo%253Apmid%252F9384377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/36786&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9384377&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997YG66700052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.38" data-doi="10.1073/pnas.0404118102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krichevsky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaidya</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tzfira</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Citovsky</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Uncoupling of the functions of the <em>Arabidopsis</em> VIP1 protein in transient and stable plant genetic transformation by <em>Agrobacterium</em></span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">5733</span>–<span class="cit-lpage">5738</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0404118102%26rft_id%253Dinfo%253Apmid%252F15824315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAyLzE2LzU3MzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzgvODkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.39" data-doi="10.1046/j.1365-313X.1995.08030457.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitsukawa</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oosumi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whittier</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Efficient isolation and mapping of <em>Arabidopsis thaliana</em> T-DNA insert junctions by thermal asymmetric interlaced PCR</span>. <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">463</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DLiu%26rft.auinit1%253DY.%2BG.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D457%26rft.epage%253D463%26rft.atitle%253DEfficient%2Bisolation%2Band%2Bmapping%2Bof%2BArabidopsis%2Bthaliana%2BT-DNA%2Binsert%2Bjunctions%2Bby%2Bthermal%2Basymmetric%2Binterlaced%2BPCR.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-313X.1995.08030457.x%26rft_id%253Dinfo%253Apmid%252F7550382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-313X.1995.08030457.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7550382&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RV49700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.40" data-doi="10.1126/science.272.5262.745"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lobry</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span><span class="cit-comment">a</span> <span class="cit-article-title">Origin of replication of <em>Mycoplasma genitalium</em></span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">272</span>: <span class="cit-fpage">745</span>–<span class="cit-lpage">746</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLobry%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D272%26rft.issue%253D5262%26rft.spage%253D745%26rft.epage%253D746%26rft.atitle%253DOrigin%2Bof%2BReplication%2Bof%2BMycoplasma%2Bgenitalium%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.272.5262.745%26rft_id%253Dinfo%253Apmid%252F8614839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNzIvNTI2Mi83NDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzgvODkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.41" data-doi="10.1093/oxfordjournals.molbev.a025626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lobry</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span><span class="cit-comment">b</span> <span class="cit-article-title">Asymmetric substitution patterns in the two DNA strands of bacteria</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">660</span>–<span class="cit-lpage">665</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DLobry%26rft.auinit1%253DJ.%26rft.volume%253D13%26rft.issue%253D5%26rft.spage%253D660%26rft.epage%253D665%26rft.atitle%253DAsymmetric%2Bsubstitution%2Bpatterns%2Bin%2Bthe%2Btwo%2BDNA%2Bstrands%2Bof%2Bbacteria%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a025626%26rft_id%253Dinfo%253Apmid%252F8676740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjEzLzUvNjYwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84Lzg5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.42" data-doi="10.1104/pp.105.062547"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loyter</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenbluh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zakai</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozlovsky</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The plant VirE2 interacting protein 1. A molecular link between the <em>Agrobacterium</em> T-complex and the host cell chromatin?</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">1318</span>–<span class="cit-lpage">1321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DLoyter%26rft.auinit1%253DA.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D1318%26rft.epage%253D1321%26rft.atitle%253DThe%2BPlant%2BVirE2%2BInteracting%2BProtein%2B1.%2BA%2BMolecular%2BLink%2Bbetween%2Bthe%2BAgrobacterium%2BT-Complex%2Band%2Bthe%2BHost%2BCell%2BChromatin%253F%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.105.062547%26rft_id%253Dinfo%253Apmid%252F16010006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTM4LzMvMTMxOCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOC84OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.43" data-doi="10.1038/nature08850"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Does</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borkovich</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coleman</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daboussi</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Comparative genomics reveals mobile pathogenicity chromosomes in <em>Fusarium</em></span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">373</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMa%26rft.auinit1%253DL.%2BJ.%26rft.volume%253D464%26rft.issue%253D7287%26rft.spage%253D367%26rft.epage%253D373%26rft.atitle%253DComparative%2Bgenomics%2Breveals%2Bmobile%2Bpathogenicity%2Bchromosomes%2Bin%2BFusarium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08850%26rft_id%253Dinfo%253Apmid%252F20237561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08850&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20237561&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275657100037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.44" data-doi="10.1016/j.tig.2008.08.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">MMEJ repair of double-strand breaks director's cut: deleted sequences and alternative endings</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">529</span>–<span class="cit-lpage">538</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DMcVey%26rft.auinit1%253DM.%26rft.volume%253D24%26rft.issue%253D11%26rft.spage%253D529%26rft.epage%253D538%26rft.atitle%253DMMEJ%2Brepair%2Bof%2Bdouble-strand%2Bbreaks%2B%2528director%2527s%2Bcut%2529%253A%2Bdeleted%2Bsequences%2Band%2Balternative%2Bendings.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2008.08.007%26rft_id%253Dinfo%253Apmid%252F18809224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2008.08.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18809224&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261533700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.45" data-doi="10.1016/j.fgb.2007.04.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heist</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Betts</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A systematic analysis of T-DNA insertion events in <em>Magnaporthe oryzae</em></span>. <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1050</span>–<span class="cit-lpage">1064</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DMeng%26rft.auinit1%253DY.%26rft.volume%253D44%26rft.issue%253D10%26rft.spage%253D1050%26rft.epage%253D1064%26rft.atitle%253DA%2Bsystematic%2Banalysis%2Bof%2BT-DNA%2Binsertion%2Bevents%2Bin%2BMagnaporthe%2Boryzae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2007.04.002%26rft_id%253Dinfo%253Apmid%252F17544743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2007.04.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17544743&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249637000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.46" data-doi="10.1007/s00294-005-0578-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michielse</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hooykaas</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van den Hondel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ram</span>  <span class="cit-name-given-names">A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title"><em>Agrobacterium</em>-mediated transformation as a tool for functional genomics in fungi</span>. <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DMichielse%26rft.auinit1%253DC.%2BB.%26rft.volume%253D48%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D17%26rft.atitle%253DAgrobacterium-mediated%2Btransformation%2Bas%2Ba%2Btool%2Bfor%2Bfunctional%2Bgenomics%2Bin%2Bfungi.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00294-005-0578-0%26rft_id%253Dinfo%253Apmid%252F15889258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00294-005-0578-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15889258&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230624600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Natarajan</span>  <span class="cit-name-given-names">A. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Mechanisms for induction of mutations and chromosome alterations</span>. <abbr class="cit-jnl-abbrev">Environ. Health Perspect.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnviron.%2BHealth%2BPerspect.%26rft.volume%253D101%26rft.spage%253D225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.48" data-doi="10.1111/j.1574-6968.2001.tb10667.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Osherov</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The molecular mechanisms of conidial germination</span>. <abbr class="cit-jnl-abbrev">FEMS Microbiol. Lett.</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">153</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiology%2BLetters%26rft.stitle%253DFEMS%2BMicrobiol%2BLett%26rft.aulast%253DOsherov%26rft.auinit1%253DN.%26rft.volume%253D199%26rft.issue%253D2%26rft.spage%253D153%26rft.epage%253D160%26rft.atitle%253DThe%2Bmolecular%2Bmechanisms%2Bof%2Bconidial%2Bgermination.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6968.2001.tb10667.x%26rft_id%253Dinfo%253Apmid%252F11377860%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1574-6968.2001.tb10667.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11377860&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169275800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.49" data-doi="10.1534/genetics.107.074468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pena-Castillo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Why are there still over 1000 uncharacterized yeast genes?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D176%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.074468%26rft_id%253Dinfo%253Apmid%252F17435240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.107.074468&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17435240&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246960900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.50" data-doi="10.1073/pnas.0905599106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pitzschke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Djamei</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teige</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirt</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">VIP1 response elements mediate mitogen-activated protein kinase 3-induced stress gene expression</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">18414</span>–<span class="cit-lpage">18419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0905599106%26rft_id%253Dinfo%253Apmid%252F19820165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzQzLzE4NDE0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84Lzg5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.51" data-doi="10.1016/j.jip.2006.06.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rangel</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torabinejad</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braga</span>  <span class="cit-name-given-names">G. U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mutants and isolates of <em>Metarhizium anisopliae</em> are diverse in their relationships between conidial pigmentation and stress tolerance</span>. <abbr class="cit-jnl-abbrev">J. Invertebr. Pathol.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">170</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Binvertebrate%2Bpathology%26rft.stitle%253DJ%2BInvertebr%2BPathol%26rft.aulast%253DRangel%26rft.auinit1%253DD.%2BE.%26rft.volume%253D93%26rft.issue%253D3%26rft.spage%253D170%26rft.epage%253D182%26rft.atitle%253DMutants%2Band%2Bisolates%2Bof%2BMetarhizium%2Banisopliae%2Bare%2Bdiverse%2Bin%2Btheir%2Brelationships%2Bbetween%2Bconidial%2Bpigmentation%2Band%2Bstress%2Btolerance.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jip.2006.06.008%26rft_id%253Dinfo%253Apmid%252F16934287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jip.2006.06.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16934287&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.52" data-doi="10.1016/j.fgb.2008.04.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Remy</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blaise</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chabirand</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolff</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span><span class="cit-comment">a</span> <span class="cit-article-title">The <em>Lmpma1</em> gene of <em>Leposphaeria maculans</em> encodes a plasma membrane H<sup>+</sup>-ATPase isoform essential for pathogenicity towards oilseed rape</span>. <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1122</span>–<span class="cit-lpage">1134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DRemy%26rft.auinit1%253DE.%26rft.volume%253D45%26rft.issue%253D7%26rft.spage%253D1122%26rft.epage%253D1134%26rft.atitle%253DThe%2BLmpma1%2Bgene%2Bof%2BLeptosphaeria%2Bmaculans%2Bencodes%2Ba%2Bplasma%2Bmembrane%2BH%252B-ATPase%2Bisoform%2Bessential%2Bfor%2Bpathogenicity%2Btowards%2Boilseed%2Brape.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2008.04.008%26rft_id%253Dinfo%253Apmid%252F18538267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2008.04.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18538267&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.53" data-doi="10.1111/j.1469-8137.2008.02522.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Remy</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blaise</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">U. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span><span class="cit-comment">b</span> <span class="cit-article-title">The <em>Lmgpi15</em> gene, encoding a component of the glycosylphosphatidylinositol anchor biosynthesis pathway, is required for morphogenesis and pathogenicity in <em>Leptosphaeria maculans</em></span>. <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DRemy%26rft.auinit1%253DE.%26rft.volume%253D179%26rft.issue%253D4%26rft.spage%253D1105%26rft.epage%253D1120%26rft.atitle%253DThe%2BLmgpi15%2Bgene%252C%2Bencoding%2Ba%2Bcomponent%2Bof%2Bthe%2Bglycosylphosphatidylinositol%2Banchor%2Bbiosynthesis%2Bpathway%252C%2Bis%2Brequired%2Bfor%2Bmorphogenesis%2Band%2Bpathogenicity%2Bin%2BLeptosphaeria%2Bmaculans.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2008.02522.x%26rft_id%253Dinfo%253Apmid%252F18557818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2008.02522.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18557818&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.54" data-doi="10.1094/MPMI-22-6-0725"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Remy</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blaise</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">U. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A key enzyme of the Leloir pathway is involved in pathogenicity of <em>Leptosphaeria maculans</em> towards oilseed rape</span>. <abbr class="cit-jnl-abbrev">Mol. Plant-Microbe Interact.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">725</span>–<span class="cit-lpage">736</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bplant-microbe%2Binteractions%2B%253A%2B%2BMPMI%26rft.stitle%253DMol%2BPlant%2BMicrobe%2BInteract%26rft.aulast%253DRemy%26rft.auinit1%253DE.%26rft.volume%253D22%26rft.issue%253D6%26rft.spage%253D725%26rft.epage%253D736%26rft.atitle%253DA%2Bkey%2Benzyme%2Bof%2Bthe%2BLeloir%2Bpathway%2Bis%2Binvolved%2Bin%2Bpathogenicity%2Bof%2BLeptosphaeria%2Bmaculans%2Btoward%2Boilseed%2Brape.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FMPMI-22-6-0725%26rft_id%253Dinfo%253Apmid%252F19445597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/MPMI-22-6-0725&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19445597&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.55" data-doi="10.1016/S0022-2836(03)00315-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ricchetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fairhead</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Distance from the chromosome end determines the efficiency of double strand break repair in subtelomeres of haploid yeast</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">328</span>: <span class="cit-fpage">847</span>–<span class="cit-lpage">862</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DRicchetti%26rft.auinit1%253DM.%26rft.volume%253D328%26rft.issue%253D4%26rft.spage%253D847%26rft.epage%253D862%26rft.atitle%253DDistance%2Bfrom%2Bthe%2Bchromosome%2Bend%2Bdetermines%2Bthe%2Befficiency%2Bof%2Bdouble%2Bstrand%2Bbreak%2Brepair%2Bin%2Bsubtelomeres%2Bof%2Bhaploid%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-2836%252803%252900315-2%26rft_id%253Dinfo%253Apmid%252F12729759%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-2836(03)00315-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12729759&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182616000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.56" data-doi="10.1038/ncomms1189"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rouxel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grandaubert</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hane</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoede</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van de Wouw</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Diversification of effectors within compartments of the <em>Leptosphaeria maculans</em> genome affected by RIP mutations</span>. <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DRouxel%26rft.auinit1%253DT.%26rft.volume%253D2%26rft.spage%253D202%26rft.epage%253D202%26rft.atitle%253DEffector%2Bdiversification%2Bwithin%2Bcompartments%2Bof%2Bthe%2BLeptosphaeria%2Bmaculans%2Bgenome%2Baffected%2Bby%2BRepeat-Induced%2BPoint%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms1189%26rft_id%253Dinfo%253Apmid%252F21326234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms1189&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21326234&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.57" data-doi="10.1146/annurev.biochem.72.121801.161520"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smale</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadonaga</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The RNA polymerase II core promoter</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">479</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.volume%253D72%26rft.spage%253D449%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biochem.72.121801.161520%26rft_id%253Dinfo%253Apmid%252F12651739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biochem.72.121801.161520&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12651739&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185092500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.58" data-doi="10.1126/science.1194573"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spanu</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abbott</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amselem</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgis</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soanes</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome expansion and gene loss in powdery mildew fungi reveal tradeoffs in extreme parasitism</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>: <span class="cit-fpage">1543</span>–<span class="cit-lpage">1546</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSpanu%26rft.auinit1%253DP.%2BD.%26rft.volume%253D330%26rft.issue%253D6010%26rft.spage%253D1543%26rft.epage%253D1546%26rft.atitle%253DGenome%2BExpansion%2Band%2BGene%2BLoss%2Bin%2BPowdery%2BMildew%2BFungi%2BReveal%2BTradeoffs%2Bin%2BExtreme%2BParasitism%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1194573%26rft_id%253Dinfo%253Apmid%252F21148392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzAvNjAxMC8xNTQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84Lzg5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.59" data-doi="10.1093/bioinformatics/btg1043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tatarinova</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brover</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troukhan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexandrov</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Skew in CG content near the transcription start site in <em>Arabidopsis thaliana</em></span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">i313</span>–<span class="cit-lpage">i314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DTatarinova%26rft.auinit1%253DT.%26rft.volume%253D19%26rft.issue%253D90001%26rft.spage%253Di313%26rft.epage%253D314%26rft.atitle%253DSkew%2Bin%2BCG%2Bcontent%2Bnear%2Bthe%2Btranscription%2Bstart%2Bsite%2Bin%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg1043%26rft_id%253Dinfo%253Apmid%252F12855475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxNToiMTkvc3VwcGxfMS9pMzEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84Lzg5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.60" data-doi="10.1111/j.1467-7652.2010.00518.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thole</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Worland</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bevan</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vain</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Distribution and characterization of more than 1000 T-DNA tags in the genome of <em>Brachypodium distachyon</em> community standard line Bd21</span>. <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">734</span>–<span class="cit-lpage">747</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DThole%26rft.auinit1%253DV.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253D734%26rft.epage%253D747%26rft.atitle%253DDistribution%2Band%2Bcharacterization%2Bof%2Bmore%2Bthan%2B1000%2BT-DNA%2Btags%2Bin%2Bthe%2Bgenome%2Bof%2BBrachypodium%2Bdistachyon%2Bcommunity%2Bstandard%2Bline%2BBd21.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2010.00518.x%26rft_id%253Dinfo%253Apmid%252F20374523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2010.00518.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20374523&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279616100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tillier</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The contributions of replication orientation, gene direction, and signal sequences to base-composition asymmetries in bacterial genomes</span>. <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DTillier%26rft.auinit1%253DE.%2BR.%26rft.volume%253D50%26rft.issue%253D3%26rft.spage%253D249%26rft.epage%253D257%26rft.atitle%253DThe%2Bcontributions%2Bof%2Breplication%2Borientation%252C%2Bgene%2Bdirection%252C%2Band%2Bsignal%2Bsequences%2Bto%2Bbase-composition%2Basymmetries%2Bin%2Bbacterial%2Bgenomes.%26rft_id%253Dinfo%253Apmid%252F10754068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10754068&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086183400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.62" data-doi="10.1016/j.biochi.2007.09.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Touchon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rocha</span>  <span class="cit-name-given-names">E. P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A gentle guide to the analysis of strand asymmetry in genome sequences.</span><abbr class="cit-jnl-abbrev"> Biochimie</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">648</span>–<span class="cit-lpage">659</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253D%2BBiochimie%26rft.volume%253D90%26rft.spage%253D648%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biochi.2007.09.015%26rft_id%253Dinfo%253Apmid%252F17988781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biochi.2007.09.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17988781&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255701600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.63" data-doi="10.1093/nar/gkh823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Touchon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arneodo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">d'Aubenton-Carafa</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thermes</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Transcription-coupled and splicing-coupled strand asymmetries in eukaryotic genomes</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">4969</span>–<span class="cit-lpage">4978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DTouchon%26rft.auinit1%253DM.%26rft.volume%253D32%26rft.issue%253D17%26rft.spage%253D4969%26rft.epage%253D4978%26rft.atitle%253DTranscription-coupled%2Band%2Bsplicing-coupled%2Bstrand%2Basymmetries%2Bin%2Beukaryotic%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh823%26rft_id%253Dinfo%253Apmid%252F15388799%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMi8xNy80OTY5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi84Lzg5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.64" data-doi="10.1146/annurev.micro.54.1.187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tzfira</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacroix</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Citovsky</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Nucleic acid transport in plant-microbe interactions: the molecules that walk through the walls</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Microbiol.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bmicrobiology%26rft.stitle%253DAnnu%2BRev%2BMicrobiol%26rft.aulast%253DTzfira%26rft.auinit1%253DT.%26rft.volume%253D54%26rft.issue%253D1%26rft.spage%253D187%26rft.epage%253D219%26rft.atitle%253DNUCLEIC%2BACID%2BTRANSPORT%2BIN%2BPLANT-MICROBE%2BINTERACTIONS%253A%2BThe%2BMolecules%2BThat%2BWalk%2BThrough%2Bthe%2BWalls%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.micro.54.1.187%26rft_id%253Dinfo%253Apmid%252F11018128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.micro.54.1.187&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11018128&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165272300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.65" data-doi="10.1016/j.tig.2004.06.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tzfira</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rhee</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">M.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Citovsky</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title"><em>Agrobacterium</em> T-DNA integration: molecules and models</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DTzfira%26rft.auinit1%253DT.%26rft.volume%253D20%26rft.issue%253D8%26rft.spage%253D375%26rft.epage%253D383%26rft.atitle%253DAgrobacterium%2BT-DNA%2Bintegration%253A%2Bmolecules%2Band%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2004.06.004%26rft_id%253Dinfo%253Apmid%252F15262410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2004.06.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15262410&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223238900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.66" data-doi="10.1080/07060668709501878"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Will</span>  <span class="cit-name-given-names">O. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Effects of far UV and visible light on germination of wild type and albino teliospores of <em>Ustilago nuda</em></span>. <abbr class="cit-jnl-abbrev">Can. J. Plant Pathol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BPlant%2BPathol.%26rft.volume%253D9%26rft.spage%253D225%26rft_id%253Dinfo%253Adoi%252F10.1080%252F07060668709501878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/07060668709501878&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.67" data-doi="10.1126/science.285.5429.901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winzeler</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Astromoff</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Functional characterization of the <em>S. cerevisiae</em> genome by gene deletion and parallel analysis</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">285</span>: <span class="cit-fpage">901</span>–<span class="cit-lpage">906</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWinzeler%26rft.auinit1%253DE.%2BA.%26rft.volume%253D285%26rft.issue%253D5429%26rft.spage%253D901%26rft.epage%253D906%26rft.atitle%253DFunctional%2BCharacterization%2Bof%2Bthe%2BS.%2Bcerevisiae%2BGenome%2Bby%2BGene%2BDeletion%2Band%2BParallel%2BAnalysis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.285.5429.901%26rft_id%253Dinfo%253Apmid%252F10436161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvNTQyOS85MDEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzgvODkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.68" data-doi="10.1126/science.8290959"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Meiosis-induced double-strand break sites determined by yeast chromatin structure</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">263</span>: <span class="cit-fpage">515</span>–<span class="cit-lpage">518</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWu%26rft.auinit1%253DT.%26rft.volume%253D263%26rft.issue%253D5146%26rft.spage%253D515%26rft.epage%253D518%26rft.atitle%253DMeiosis-induced%2Bdouble-strand%2Bbreak%2Bsites%2Bdetermined%2Bby%2Byeast%2Bchromatin%2Bstructure%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.8290959%26rft_id%253Dinfo%253Apmid%252F8290959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNjMvNTE0Ni81MTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzgvODkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.69" data-doi="10.1073/pnas.93.14.6975"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yuryev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patturajan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Litingtung</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joshi</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gentile</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The C-terminal domain of the largest subunit of RNA polymerase II interacts with a novel set of serine/arginine-rich proteins</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">6975</span>–<span class="cit-lpage">6980</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DYuryev%26rft.auinit1%253DA.%26rft.volume%253D93%26rft.issue%253D14%26rft.spage%253D6975%26rft.epage%253D6980%26rft.atitle%253DThe%2BC-terminal%2Bdomain%2Bof%2Bthe%2Blargest%2Bsubunit%2Bof%2BRNA%2Bpolymerase%2BII%2Binteracts%2Bwith%2Ba%2Bnovel%2Bset%2Bof%2Bserine%252Farginine-rich%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.93.14.6975%26rft_id%253Dinfo%253Apmid%252F8692929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTMvMTQvNjk3NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvOC84OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.70" data-doi="10.1111/j.1365-313X.2006.03001.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">Y. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">You</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Non-random distribution of T-DNA insertions at various levels of the genome hierarchy as revealed by analyzing 13 804 T-DNA flanking sequences from an enhancer-trap mutant library</span>. <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">947</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DZhang%26rft.auinit1%253DJ.%26rft.volume%253D49%26rft.issue%253D5%26rft.spage%253D947%26rft.epage%253D959%26rft.atitle%253DNon-random%2Bdistribution%2Bof%2BT-DNA%2Binsertions%2Bat%2Bvarious%2Blevels%2Bof%2Bthe%2Bgenome%2Bhierarchy%2Bas%2Brevealed%2Bby%2Banalyzing%2B13%2B804%2BT-DNA%2Bflanking%2Bsequences%2Bfrom%2Ban%2Benhancer-trap%2Bmutant%2Blibrary.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2006.03001.x%26rft_id%253Dinfo%253Apmid%252F17253985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2006.03001.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17253985&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244280300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-2.8.891.71" data-doi="10.1046/j.1365-313x.2000.00808.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zupan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muth</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Draper</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zambryski</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The transfer of DNA from <em>Agrobacterium tumefaciens</em> into plants: a feast of fundamental insights</span>. <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DZupan%26rft.auinit1%253DJ.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D28%26rft.atitle%253DThe%2Btransfer%2Bof%2BDNA%2Bfrom%2Bagrobacterium%2Btumefaciens%2Binto%2Bplants%253A%2Ba%2Bfeast%2Bof%2Bfundamental%2Binsights.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-313x.2000.00808.x%26rft_id%253Dinfo%253Apmid%252F10929098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-313x.2000.00808.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10929098&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F8%2F891.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088453900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/8/891.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/8/883" title="Alignment-Free Population Genomics: An Efficient Estimator of Sequence Diversity" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/8/905" title="Resampling QTL Effects in the QTL Sign Test Leads to Incongruous Sensitivity to Variance in Effect Size" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/8"><span>Volume 2 Issue 8, August 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/8.cover-source.jpg?itok=etPFDPKt" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/8/891" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bdgqBDBt2No2rcZrblUhvg7NWuuKW5l3YwyVELD49yk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/8/891" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/8/891" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/8/891" class="active" data-icon-position="" data-hide-link-title="0">Incidence of Genome Structure, DNA Asymmetry, and Cell Physiology on T-DNA Integration in Chromosomes of the Phytopathogenic Fungus Leptosphaeria maculans</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459307" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-SJgNxxuXLzba93H45QRm0eXoJGyYTlC2ZlBBlKipOYI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/8/891.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/8/891" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form--pMiE424Cc9Dg-1eqlHsiEpbeGZBUUpr0BikQ-gtznI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/8/891" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/8/891.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459307" id="node459307--2" data-pisa="ggg;2/8/891" data-pisa-master="ggg;2/8/891" data-apath="/ggg/2/8/891.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Incidence of Genome Structure, DNA Asymmetry, and Cell Physiology on T-DNA Integration in Chromosomes of the Phytopathogenic Fungus <em>Leptosphaeria maculans</em></h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Salim Bourras</span>, <span class="highwire-citation-author" data-delta="1">Michel Meyer</span>, <span class="highwire-citation-author" data-delta="2">Jonathan Grandaubert</span>, <span class="highwire-citation-author" data-delta="3">Nicolas Lapalu</span>, <span class="highwire-citation-author" data-delta="4">Isabelle Fudal</span>, <span class="highwire-citation-author" data-delta="5">Juliette Linglin</span>, <span class="highwire-citation-author" data-delta="6">Benedicte Ollivier</span>, <span class="highwire-citation-author" data-delta="7">Françoise Blaise</span>, <span class="highwire-citation-author" data-delta="8">Marie-Hélène Balesdent</span> and <span class="highwire-citation-author" data-delta="9">Thierry Rouxel</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">891-904;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002048 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459307--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Salim Bourras</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Salim%2BBourras" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bourras%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASalim%2BBourras" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michel Meyer</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michel%2BMeyer" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meyer%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichel%2BMeyer" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jonathan Grandaubert</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BGrandaubert" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grandaubert%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BGrandaubert" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nicolas Lapalu</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, Research Unit 1290 BIOGER, F-78026 Versailles Cedex, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BLapalu" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lapalu%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicolas%2BLapalu" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Isabelle Fudal</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Isabelle%2BFudal" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fudal%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIsabelle%2BFudal" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Juliette Linglin</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juliette%2BLinglin" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Linglin%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuliette%2BLinglin" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Benedicte Ollivier</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benedicte%2BOllivier" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ollivier%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABenedicte%2BOllivier" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Françoise Blaise</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fran%C3%A7oise%2BBlaise" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blaise%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFran%25C3%25A7oise%2BBlaise" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Marie-Hélène Balesdent</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-H%C3%A9l%C3%A8ne%2BBalesdent" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Balesdent%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie-H%25C3%25A9l%25C3%25A8ne%2BBalesdent" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Thierry Rouxel</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thierry%2BRouxel" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rouxel%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThierry%2BRouxel" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rouxel@versailles.inra.fr" class="" data-icon-position="" data-hide-link-title="0">rouxel@versailles.inra.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/8/891" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/8/891&amp;title=Incidence%20of%20Genome%20Structure%2C%20DNA%20Asymmetry%2C%20and%20Cell%20Physiology%20on%20T-DNA%20Integration%20in%20Chromosomes%20of%20the%20Phytopathogenic%20Fungus%20Leptosphaeria%20maculans" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/8/891&amp;title=Incidence%20of%20Genome%20Structure%2C%20DNA%20Asymmetry%2C%20and%20Cell%20Physiology%20on%20T-DNA%20Integration%20in%20Chromosomes%20of%20the%20Phytopathogenic%20Fungus%20Leptosphaeria%20maculans" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/8/891&amp;title=Incidence%20of%20Genome%20Structure%2C%20DNA%20Asymmetry%2C%20and%20Cell%20Physiology%20on%20T-DNA%20Integration%20in%20Chromosomes%20of%20the%20Phytopathogenic%20Fungus%20Leptosphaeria%20maculans" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/8/891&amp;text=Incidence%20of%20Genome%20Structure%2C%20DNA%20Asymmetry%2C%20and%20Cell%20Physiology%20on%20T-DNA%20Integration%20in%20Chromosomes%20of%20the%20Phytopathogenic%20Fungus%20Leptosphaeria%20maculans" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/8/891&amp;title=Incidence%20of%20Genome%20Structure%2C%20DNA%20Asymmetry%2C%20and%20Cell%20Physiology%20on%20T-DNA%20Integration%20in%20Chromosomes%20of%20the%20Phytopathogenic%20Fungus%20Leptosphaeria%20maculans" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/8/891&amp;t=Incidence%20of%20Genome%20Structure%2C%20DNA%20Asymmetry%2C%20and%20Cell%20Physiology%20on%20T-DNA%20Integration%20in%20Chromosomes%20of%20the%20Phytopathogenic%20Fungus%20Leptosphaeria%20maculans" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/8/891&amp;title=Incidence%20of%20Genome%20Structure%2C%20DNA%20Asymmetry%2C%20and%20Cell%20Physiology%20on%20T-DNA%20Integration%20in%20Chromosomes%20of%20the%20Phytopathogenic%20Fungus%20Leptosphaeria%20maculans" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/8/891&amp;title=Incidence%20of%20Genome%20Structure%2C%20DNA%20Asymmetry%2C%20and%20Cell%20Physiology%20on%20T-DNA%20Integration%20in%20Chromosomes%20of%20the%20Phytopathogenic%20Fungus%20Leptosphaeria%20maculans" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459307/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459307" id="node459307--2" data-pisa="ggg;2/8/891" data-pisa-master="ggg;2/8/891" data-apath="/ggg/2/8/891.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Incidence of Genome Structure, DNA Asymmetry, and Cell Physiology on T-DNA Integration in Chromosomes of the Phytopathogenic Fungus <em>Leptosphaeria maculans</em></h1>  
    	<div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Salim Bourras</span>, <span class="highwire-citation-author" data-delta="1">Michel Meyer</span>, <span class="highwire-citation-author" data-delta="2">Jonathan Grandaubert</span>, <span class="highwire-citation-author" data-delta="3">Nicolas Lapalu</span>, <span class="highwire-citation-author" data-delta="4">Isabelle Fudal</span>, <span class="highwire-citation-author" data-delta="5">Juliette Linglin</span>, <span class="highwire-citation-author" data-delta="6">Benedicte Ollivier</span>, <span class="highwire-citation-author" data-delta="7">Françoise Blaise</span>, <span class="highwire-citation-author" data-delta="8">Marie-Hélène Balesdent</span> and <span class="highwire-citation-author" data-delta="9">Thierry Rouxel</span></span></div>
  
    	<div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">891-904;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002048 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459307--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Salim Bourras</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Salim%2BBourras" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bourras%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASalim%2BBourras" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michel Meyer</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michel%2BMeyer" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meyer%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichel%2BMeyer" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jonathan Grandaubert</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BGrandaubert" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grandaubert%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BGrandaubert" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nicolas Lapalu</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">INRA, Research Unit 1290 BIOGER, F-78026 Versailles Cedex, France</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BLapalu" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lapalu%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicolas%2BLapalu" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Isabelle Fudal</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Isabelle%2BFudal" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fudal%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIsabelle%2BFudal" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Juliette Linglin</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juliette%2BLinglin" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Linglin%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuliette%2BLinglin" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Benedicte Ollivier</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benedicte%2BOllivier" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ollivier%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABenedicte%2BOllivier" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Françoise Blaise</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fran%C3%A7oise%2BBlaise" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blaise%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFran%25C3%25A7oise%2BBlaise" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Marie-Hélène Balesdent</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-H%C3%A9l%C3%A8ne%2BBalesdent" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Balesdent%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie-H%25C3%25A9l%25C3%25A8ne%2BBalesdent" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Thierry Rouxel</div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff">Institut National de la Recherche Agronomique (INRA), Research Unit 1290 BIOGER, F-78850 Thiverval-Grignon, France, and</span></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thierry%2BRouxel" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rouxel%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThierry%2BRouxel" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rouxel@versailles.inra.fr" class="" data-icon-position="" data-hide-link-title="0">rouxel@versailles.inra.fr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=891&amp;rft.epage=904&amp;rft.atitle=Incidence%20of%20Genome%20Structure%2C%20DNA%20Asymmetry%2C%20and%20Cell%20Physiology%20on%20T-DNA%20Integration%20in%20Chromosomes%20of%20the%20Phytopathogenic%20Fungus%20Leptosphaeria%20maculans&amp;rft.volume=2&amp;rft.issue=8&amp;rft.date=2012-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Bourras%2C+Salim&amp;rft.au=Meyer%2C+Michel&amp;rft.au=Grandaubert%2C+Jonathan&amp;rft.au=Lapalu%2C+Nicolas&amp;rft.au=Fudal%2C+Isabelle&amp;rft.au=Linglin%2C+Juliette&amp;rft.au=Ollivier%2C+Benedicte&amp;rft.au=Blaise%2C+Fran%C3%A7oise&amp;rft.au=Balesdent%2C+Marie-H%C3%A9l%C3%A8ne&amp;rft.au=Rouxel%2C+Thierry"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459307/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459307/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459307/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459307/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459307/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459307/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459307/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459307/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459307/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459307/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459307/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459307/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494625" id="node494625" data-pisa="ggg;9/11/3891" data-pisa-master="ggg;g3.119.400723" data-apath="/ggg/9/11/3891.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3891" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Mouse Mutation That Dysregulates Neighboring <em>Galnt17</em> and <em>Auts2</em> Genes Is Associated with Phenotypes Related to the Human AUTS2 Syndrome</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494624" id="node494624" data-pisa="ggg;9/11/3781" data-pisa-master="ggg;g3.119.400647" data-apath="/ggg/9/11/3781.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3781" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Transgenic Analyses in Drosophila Reveal That mCORL1 Is Functionally Distinct from mCORL2 and dCORL</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494622" id="node494622" data-pisa="ggg;9/11/3843" data-pisa-master="ggg;g3.119.400363" data-apath="/ggg/9/11/3843.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3843" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Analysis of Cell Cycle-Regulating Genes in the Symbiotic Dinoflagellate <em>Breviolum minutum</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/8/891" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">RESULTS</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">DISCUSSION</a></li><li class="odd"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">CONCLUSIONS</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgment</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/8/891.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/8/891.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/2/8/891.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2019 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__Gk8ter50DWCZUhSfS3IqlmiQ6T-aNb48R5smU7zB-i8.js"></script>
  </body>
</html>
